--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = off;
SET check_function_bodies = false;
SET client_min_messages = warning;
SET escape_string_warning = off;

--
-- Name: gmodnatdiv_development; Type: DATABASE; Schema: -; Owner: gmod
--

CREATE DATABASE gmodnatdiv_development WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'C' LC_CTYPE = 'C';


ALTER DATABASE gmodnatdiv_development OWNER TO gmod;

\connect gmodnatdiv_development

SET statement_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = off;
SET check_function_bodies = false;
SET client_min_messages = warning;
SET escape_string_warning = off;

--
-- Name: plpgsql; Type: PROCEDURAL LANGUAGE; Schema: -; Owner: postgres
--

CREATE PROCEDURAL LANGUAGE plpgsql;


ALTER PROCEDURAL LANGUAGE plpgsql OWNER TO postgres;

SET search_path = public, pg_catalog;

--
-- Name: tablefunc_crosstab_2; Type: TYPE; Schema: public; Owner: postgres
--

CREATE TYPE tablefunc_crosstab_2 AS (
	row_name text,
	category_1 text,
	category_2 text
);


ALTER TYPE public.tablefunc_crosstab_2 OWNER TO postgres;

--
-- Name: tablefunc_crosstab_3; Type: TYPE; Schema: public; Owner: postgres
--

CREATE TYPE tablefunc_crosstab_3 AS (
	row_name text,
	category_1 text,
	category_2 text,
	category_3 text
);


ALTER TYPE public.tablefunc_crosstab_3 OWNER TO postgres;

--
-- Name: tablefunc_crosstab_4; Type: TYPE; Schema: public; Owner: postgres
--

CREATE TYPE tablefunc_crosstab_4 AS (
	row_name text,
	category_1 text,
	category_2 text,
	category_3 text,
	category_4 text
);


ALTER TYPE public.tablefunc_crosstab_4 OWNER TO postgres;

--
-- Name: connectby(text, text, text, text, integer, text); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION connectby(text, text, text, text, integer, text) RETURNS SETOF record
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'connectby_text';


ALTER FUNCTION public.connectby(text, text, text, text, integer, text) OWNER TO postgres;

--
-- Name: connectby(text, text, text, text, integer); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION connectby(text, text, text, text, integer) RETURNS SETOF record
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'connectby_text';


ALTER FUNCTION public.connectby(text, text, text, text, integer) OWNER TO postgres;

--
-- Name: connectby(text, text, text, text, text, integer, text); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION connectby(text, text, text, text, text, integer, text) RETURNS SETOF record
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'connectby_text_serial';


ALTER FUNCTION public.connectby(text, text, text, text, text, integer, text) OWNER TO postgres;

--
-- Name: connectby(text, text, text, text, text, integer); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION connectby(text, text, text, text, text, integer) RETURNS SETOF record
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'connectby_text_serial';


ALTER FUNCTION public.connectby(text, text, text, text, text, integer) OWNER TO postgres;

--
-- Name: crosstab(text); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION crosstab(text) RETURNS SETOF record
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'crosstab';


ALTER FUNCTION public.crosstab(text) OWNER TO postgres;

--
-- Name: crosstab(text, integer); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION crosstab(text, integer) RETURNS SETOF record
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'crosstab';


ALTER FUNCTION public.crosstab(text, integer) OWNER TO postgres;

--
-- Name: crosstab(text, text); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION crosstab(text, text) RETURNS SETOF record
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'crosstab_hash';


ALTER FUNCTION public.crosstab(text, text) OWNER TO postgres;

--
-- Name: crosstab2(text); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION crosstab2(text) RETURNS SETOF tablefunc_crosstab_2
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'crosstab';


ALTER FUNCTION public.crosstab2(text) OWNER TO postgres;

--
-- Name: crosstab3(text); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION crosstab3(text) RETURNS SETOF tablefunc_crosstab_3
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'crosstab';


ALTER FUNCTION public.crosstab3(text) OWNER TO postgres;

--
-- Name: crosstab4(text); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION crosstab4(text) RETURNS SETOF tablefunc_crosstab_4
    LANGUAGE c STABLE STRICT
    AS '$libdir/tablefunc', 'crosstab';


ALTER FUNCTION public.crosstab4(text) OWNER TO postgres;

--
-- Name: get_cvterm_id(character varying, character varying); Type: FUNCTION; Schema: public; Owner: gmod
--

CREATE FUNCTION get_cvterm_id(p_cv_name character varying, p_cvterm_name character varying) RETURNS integer
    LANGUAGE plpgsql
    AS $$
   DECLARE
     ret_val INTEGER;
   BEGIN
     SELECT cvterm_id INTO ret_val FROM cvterm 
       INNER JOIN cv ON cvterm.cv_id = cv.cv_id 
       WHERE cv.name = p_cv_name AND cvterm.name = p_cvterm_name; 
--     IF NOT FOUND
--       RAISE EXCEPTION 'cv % and cvterm % not found', p_cv_name, p_cvterm_name;
--     END IF
     RETURN ret_val;
   END;
   $$;


ALTER FUNCTION public.get_cvterm_id(p_cv_name character varying, p_cvterm_name character varying) OWNER TO gmod;

--
-- Name: gwas_query(refcursor, integer, boolean); Type: FUNCTION; Schema: public; Owner: gmod
--

CREATE FUNCTION gwas_query(p_rc refcursor, p_proj_id integer, p_get_header boolean) RETURNS refcursor
    LANGUAGE plpgsql
    AS $$
  DECLARE
    v_plant_id INTEGER;
  BEGIN
    v_plant_id = get_cvterm_id('diversity_experiment', 'plant id');
    IF p_get_header THEN
      OPEN p_rc FOR
        SELECT 'experiment_id', 'genotype', 'plant_id', treatment_str,
	  phenotype_str
	  FROM gwas
          WHERE project_id = p_proj_id AND sp_type_id = v_plant_id
          LIMIT 1;
    ELSE
      OPEN p_rc FOR 
        SELECT nd_experiment_id, genotype, sp_value_id AS plant_id,
	  treatment_val, phenotype_val
          FROM gwas
          WHERE project_id = p_proj_id AND sp_type_id = v_plant_id;
    END IF;
  RETURN p_rc;
  END;
$$;


ALTER FUNCTION public.gwas_query(p_rc refcursor, p_proj_id integer, p_get_header boolean) OWNER TO gmod;

--
-- Name: normal_rand(integer, double precision, double precision); Type: FUNCTION; Schema: public; Owner: postgres
--

CREATE FUNCTION normal_rand(integer, double precision, double precision) RETURNS SETOF double precision
    LANGUAGE c STRICT
    AS '$libdir/tablefunc', 'normal_rand';


ALTER FUNCTION public.normal_rand(integer, double precision, double precision) OWNER TO postgres;

SET default_tablespace = '';

SET default_with_oids = false;

--
-- Name: acquisition; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE acquisition (
    acquisition_id integer NOT NULL,
    assay_id integer NOT NULL,
    protocol_id integer,
    channel_id integer,
    acquisitiondate timestamp without time zone DEFAULT now(),
    name text,
    uri text
);


ALTER TABLE public.acquisition OWNER TO gmod;

--
-- Name: TABLE acquisition; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE acquisition IS 'This represents the scanning of hybridized material. The output of this process is typically a digital image of an array.';


--
-- Name: acquisition_acquisition_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE acquisition_acquisition_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.acquisition_acquisition_id_seq OWNER TO gmod;

--
-- Name: acquisition_acquisition_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE acquisition_acquisition_id_seq OWNED BY acquisition.acquisition_id;


--
-- Name: acquisition_acquisition_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('acquisition_acquisition_id_seq', 1, false);


--
-- Name: acquisition_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE acquisition_relationship (
    acquisition_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    type_id integer NOT NULL,
    object_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.acquisition_relationship OWNER TO gmod;

--
-- Name: TABLE acquisition_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE acquisition_relationship IS 'Multiple monochrome images may be merged to form a multi-color image. Red-green images of 2-channel hybridizations are an example of this.';


--
-- Name: acquisition_relationship_acquisition_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE acquisition_relationship_acquisition_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.acquisition_relationship_acquisition_relationship_id_seq OWNER TO gmod;

--
-- Name: acquisition_relationship_acquisition_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE acquisition_relationship_acquisition_relationship_id_seq OWNED BY acquisition_relationship.acquisition_relationship_id;


--
-- Name: acquisition_relationship_acquisition_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('acquisition_relationship_acquisition_relationship_id_seq', 1, false);


--
-- Name: acquisitionprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE acquisitionprop (
    acquisitionprop_id integer NOT NULL,
    acquisition_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.acquisitionprop OWNER TO gmod;

--
-- Name: TABLE acquisitionprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE acquisitionprop IS 'Parameters associated with image acquisition.';


--
-- Name: acquisitionprop_acquisitionprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE acquisitionprop_acquisitionprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.acquisitionprop_acquisitionprop_id_seq OWNER TO gmod;

--
-- Name: acquisitionprop_acquisitionprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE acquisitionprop_acquisitionprop_id_seq OWNED BY acquisitionprop.acquisitionprop_id;


--
-- Name: acquisitionprop_acquisitionprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('acquisitionprop_acquisitionprop_id_seq', 1, false);


--
-- Name: analysis; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE analysis (
    analysis_id integer NOT NULL,
    name character varying(255),
    description text,
    program character varying(255) NOT NULL,
    programversion character varying(255) NOT NULL,
    algorithm character varying(255),
    sourcename character varying(255),
    sourceversion character varying(255),
    sourceuri text,
    timeexecuted timestamp without time zone DEFAULT now() NOT NULL
);


ALTER TABLE public.analysis OWNER TO gmod;

--
-- Name: TABLE analysis; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE analysis IS 'An analysis is a particular type of a computational analysis; it may be a blast of one sequence against another, or an all by all blast, or a different kind of analysis altogether. It is a single unit of computation.';


--
-- Name: COLUMN analysis.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysis.name IS 'A way of grouping analyses. This should be a handy short identifier that can help people find an analysis they want. For instance "tRNAscan", "cDNA", "FlyPep", "SwissProt", and it should not be assumed to be unique. For instance, there may be lots of separate analyses done against a cDNA database.';


--
-- Name: COLUMN analysis.program; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysis.program IS 'Program name, e.g. blastx, blastp, sim4, genscan.';


--
-- Name: COLUMN analysis.programversion; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysis.programversion IS 'Version description, e.g. TBLASTX 2.0MP-WashU [09-Nov-2000].';


--
-- Name: COLUMN analysis.algorithm; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysis.algorithm IS 'Algorithm name, e.g. blast.';


--
-- Name: COLUMN analysis.sourcename; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysis.sourcename IS 'Source name, e.g. cDNA, SwissProt.';


--
-- Name: COLUMN analysis.sourceuri; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysis.sourceuri IS 'This is an optional, permanent URL or URI for the source of the analysis. The idea is that someone could recreate the analysis directly by going to this URI and fetching the source data (e.g. the blast database, or the training model).';


--
-- Name: analysis_analysis_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE analysis_analysis_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.analysis_analysis_id_seq OWNER TO gmod;

--
-- Name: analysis_analysis_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE analysis_analysis_id_seq OWNED BY analysis.analysis_id;


--
-- Name: analysis_analysis_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('analysis_analysis_id_seq', 9, true);


--
-- Name: analysisfeature; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE analysisfeature (
    analysisfeature_id integer NOT NULL,
    feature_id integer NOT NULL,
    analysis_id integer NOT NULL,
    rawscore double precision,
    normscore double precision,
    significance double precision,
    identity double precision
);


ALTER TABLE public.analysisfeature OWNER TO gmod;

--
-- Name: TABLE analysisfeature; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE analysisfeature IS 'Computational analyses generate features (e.g. Genscan generates transcripts and exons; sim4 alignments generate similarity/match features). analysisfeatures are stored using the feature table from the sequence module. The analysisfeature table is used to decorate these features, with analysis specific attributes. A feature is an analysisfeature if and only if there is a corresponding entry in the analysisfeature table. analysisfeatures will have two or more featureloc entries, with rank indicating query/subject';


--
-- Name: COLUMN analysisfeature.rawscore; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysisfeature.rawscore IS 'This is the native score generated by the program; for example, the bitscore generated by blast, sim4 or genscan scores. One should not assume that high is necessarily better than low.';


--
-- Name: COLUMN analysisfeature.normscore; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysisfeature.normscore IS 'This is the rawscore but semi-normalized. Complete normalization to allow comparison of features generated by different programs would be nice but too difficult. Instead the normalization should strive to enforce the following semantics: * normscores are floating point numbers >= 0, * high normscores are better than low one. For most programs, it would be sufficient to make the normscore the same as this rawscore, providing these semantics are satisfied.';


--
-- Name: COLUMN analysisfeature.significance; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysisfeature.significance IS 'This is some kind of expectation or probability metric, representing the probability that the analysis would appear randomly given the model. As such, any program or person querying this table can assume the following semantics: * 0 <= significance <= n, where n is a positive number, theoretically unbounded but unlikely to be more than 10 * low numbers are better than high numbers.';


--
-- Name: COLUMN analysisfeature.identity; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN analysisfeature.identity IS 'Percent identity between the locations compared. Note that these 4 metrics do not cover the full range of scores possible; it would be undesirable to list every score possible, as this should be kept extensible. instead, for non-standard scores, use the analysisprop table.';


--
-- Name: analysisfeature_analysisfeature_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE analysisfeature_analysisfeature_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.analysisfeature_analysisfeature_id_seq OWNER TO gmod;

--
-- Name: analysisfeature_analysisfeature_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE analysisfeature_analysisfeature_id_seq OWNED BY analysisfeature.analysisfeature_id;


--
-- Name: analysisfeature_analysisfeature_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('analysisfeature_analysisfeature_id_seq', 1, false);


--
-- Name: analysisfeatureprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE analysisfeatureprop (
    analysisfeatureprop_id integer NOT NULL,
    analysisfeature_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer NOT NULL
);


ALTER TABLE public.analysisfeatureprop OWNER TO gmod;

--
-- Name: analysisfeatureprop_analysisfeatureprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE analysisfeatureprop_analysisfeatureprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.analysisfeatureprop_analysisfeatureprop_id_seq OWNER TO gmod;

--
-- Name: analysisfeatureprop_analysisfeatureprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE analysisfeatureprop_analysisfeatureprop_id_seq OWNED BY analysisfeatureprop.analysisfeatureprop_id;


--
-- Name: analysisfeatureprop_analysisfeatureprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('analysisfeatureprop_analysisfeatureprop_id_seq', 1, false);


--
-- Name: analysisprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE analysisprop (
    analysisprop_id integer NOT NULL,
    analysis_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.analysisprop OWNER TO gmod;

--
-- Name: analysisprop_analysisprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE analysisprop_analysisprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.analysisprop_analysisprop_id_seq OWNER TO gmod;

--
-- Name: analysisprop_analysisprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE analysisprop_analysisprop_id_seq OWNED BY analysisprop.analysisprop_id;


--
-- Name: analysisprop_analysisprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('analysisprop_analysisprop_id_seq', 1, false);


--
-- Name: arraydesign; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE arraydesign (
    arraydesign_id integer NOT NULL,
    manufacturer_id integer NOT NULL,
    platformtype_id integer NOT NULL,
    substratetype_id integer,
    protocol_id integer,
    dbxref_id integer,
    name text NOT NULL,
    version text,
    description text,
    array_dimensions text,
    element_dimensions text,
    num_of_elements integer,
    num_array_columns integer,
    num_array_rows integer,
    num_grid_columns integer,
    num_grid_rows integer,
    num_sub_columns integer,
    num_sub_rows integer
);


ALTER TABLE public.arraydesign OWNER TO gmod;

--
-- Name: TABLE arraydesign; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE arraydesign IS 'General properties about an array. An array is a template used to generate physical slides, etc. It contains layout information, as well as global array properties, such as material (glass, nylon) and spot dimensions (in rows/columns).';


--
-- Name: arraydesign_arraydesign_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE arraydesign_arraydesign_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.arraydesign_arraydesign_id_seq OWNER TO gmod;

--
-- Name: arraydesign_arraydesign_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE arraydesign_arraydesign_id_seq OWNED BY arraydesign.arraydesign_id;


--
-- Name: arraydesign_arraydesign_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('arraydesign_arraydesign_id_seq', 38, true);


--
-- Name: arraydesignprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE arraydesignprop (
    arraydesignprop_id integer NOT NULL,
    arraydesign_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.arraydesignprop OWNER TO gmod;

--
-- Name: TABLE arraydesignprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE arraydesignprop IS 'Extra array design properties that are not accounted for in arraydesign.';


--
-- Name: arraydesignprop_arraydesignprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE arraydesignprop_arraydesignprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.arraydesignprop_arraydesignprop_id_seq OWNER TO gmod;

--
-- Name: arraydesignprop_arraydesignprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE arraydesignprop_arraydesignprop_id_seq OWNED BY arraydesignprop.arraydesignprop_id;


--
-- Name: arraydesignprop_arraydesignprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('arraydesignprop_arraydesignprop_id_seq', 1, false);


--
-- Name: assay; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE assay (
    assay_id integer NOT NULL,
    arraydesign_id integer NOT NULL,
    protocol_id integer,
    assaydate timestamp without time zone DEFAULT now(),
    arrayidentifier text,
    arraybatchidentifier text,
    operator_id integer NOT NULL,
    dbxref_id integer,
    name text,
    description text
);


ALTER TABLE public.assay OWNER TO gmod;

--
-- Name: TABLE assay; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE assay IS 'An assay consists of a physical instance of an array, combined with the conditions used to create the array (protocols, technician information). The assay can be thought of as a hybridization.';


--
-- Name: assay_assay_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE assay_assay_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.assay_assay_id_seq OWNER TO gmod;

--
-- Name: assay_assay_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE assay_assay_id_seq OWNED BY assay.assay_id;


--
-- Name: assay_assay_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('assay_assay_id_seq', 1, false);


--
-- Name: assay_biomaterial; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE assay_biomaterial (
    assay_biomaterial_id integer NOT NULL,
    assay_id integer NOT NULL,
    biomaterial_id integer NOT NULL,
    channel_id integer,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.assay_biomaterial OWNER TO gmod;

--
-- Name: TABLE assay_biomaterial; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE assay_biomaterial IS 'A biomaterial can be hybridized many times (technical replicates), or combined with other biomaterials in a single hybridization (for two-channel arrays).';


--
-- Name: assay_biomaterial_assay_biomaterial_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE assay_biomaterial_assay_biomaterial_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.assay_biomaterial_assay_biomaterial_id_seq OWNER TO gmod;

--
-- Name: assay_biomaterial_assay_biomaterial_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE assay_biomaterial_assay_biomaterial_id_seq OWNED BY assay_biomaterial.assay_biomaterial_id;


--
-- Name: assay_biomaterial_assay_biomaterial_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('assay_biomaterial_assay_biomaterial_id_seq', 1, false);


--
-- Name: assay_project; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE assay_project (
    assay_project_id integer NOT NULL,
    assay_id integer NOT NULL,
    project_id integer NOT NULL
);


ALTER TABLE public.assay_project OWNER TO gmod;

--
-- Name: TABLE assay_project; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE assay_project IS 'Link assays to projects.';


--
-- Name: assay_project_assay_project_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE assay_project_assay_project_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.assay_project_assay_project_id_seq OWNER TO gmod;

--
-- Name: assay_project_assay_project_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE assay_project_assay_project_id_seq OWNED BY assay_project.assay_project_id;


--
-- Name: assay_project_assay_project_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('assay_project_assay_project_id_seq', 1, false);


--
-- Name: assayprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE assayprop (
    assayprop_id integer NOT NULL,
    assay_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.assayprop OWNER TO gmod;

--
-- Name: TABLE assayprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE assayprop IS 'Extra assay properties that are not accounted for in assay.';


--
-- Name: assayprop_assayprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE assayprop_assayprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.assayprop_assayprop_id_seq OWNER TO gmod;

--
-- Name: assayprop_assayprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE assayprop_assayprop_id_seq OWNED BY assayprop.assayprop_id;


--
-- Name: assayprop_assayprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('assayprop_assayprop_id_seq', 1, false);


--
-- Name: biomaterial; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE biomaterial (
    biomaterial_id integer NOT NULL,
    taxon_id integer,
    biosourceprovider_id integer,
    dbxref_id integer,
    name text,
    description text
);


ALTER TABLE public.biomaterial OWNER TO gmod;

--
-- Name: TABLE biomaterial; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE biomaterial IS 'A biomaterial represents the MAGE concept of BioSource, BioSample, and LabeledExtract. It is essentially some biological material (tissue, cells, serum) that may have been processed. Processed biomaterials should be traceable back to raw biomaterials via the biomaterialrelationship table.';


--
-- Name: biomaterial_biomaterial_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE biomaterial_biomaterial_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.biomaterial_biomaterial_id_seq OWNER TO gmod;

--
-- Name: biomaterial_biomaterial_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE biomaterial_biomaterial_id_seq OWNED BY biomaterial.biomaterial_id;


--
-- Name: biomaterial_biomaterial_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('biomaterial_biomaterial_id_seq', 1, false);


--
-- Name: biomaterial_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE biomaterial_dbxref (
    biomaterial_dbxref_id integer NOT NULL,
    biomaterial_id integer NOT NULL,
    dbxref_id integer NOT NULL
);


ALTER TABLE public.biomaterial_dbxref OWNER TO gmod;

--
-- Name: biomaterial_dbxref_biomaterial_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE biomaterial_dbxref_biomaterial_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.biomaterial_dbxref_biomaterial_dbxref_id_seq OWNER TO gmod;

--
-- Name: biomaterial_dbxref_biomaterial_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE biomaterial_dbxref_biomaterial_dbxref_id_seq OWNED BY biomaterial_dbxref.biomaterial_dbxref_id;


--
-- Name: biomaterial_dbxref_biomaterial_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('biomaterial_dbxref_biomaterial_dbxref_id_seq', 1, false);


--
-- Name: biomaterial_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE biomaterial_relationship (
    biomaterial_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    type_id integer NOT NULL,
    object_id integer NOT NULL
);


ALTER TABLE public.biomaterial_relationship OWNER TO gmod;

--
-- Name: TABLE biomaterial_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE biomaterial_relationship IS 'Relate biomaterials to one another. This is a way to track a series of treatments or material splits/merges, for instance.';


--
-- Name: biomaterial_relationship_biomaterial_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE biomaterial_relationship_biomaterial_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.biomaterial_relationship_biomaterial_relationship_id_seq OWNER TO gmod;

--
-- Name: biomaterial_relationship_biomaterial_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE biomaterial_relationship_biomaterial_relationship_id_seq OWNED BY biomaterial_relationship.biomaterial_relationship_id;


--
-- Name: biomaterial_relationship_biomaterial_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('biomaterial_relationship_biomaterial_relationship_id_seq', 1, false);


--
-- Name: biomaterial_treatment; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE biomaterial_treatment (
    biomaterial_treatment_id integer NOT NULL,
    biomaterial_id integer NOT NULL,
    treatment_id integer NOT NULL,
    unittype_id integer,
    value real,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.biomaterial_treatment OWNER TO gmod;

--
-- Name: TABLE biomaterial_treatment; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE biomaterial_treatment IS 'Link biomaterials to treatments. Treatments have an order of operations (rank), and associated measurements (unittype_id, value).';


--
-- Name: biomaterial_treatment_biomaterial_treatment_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE biomaterial_treatment_biomaterial_treatment_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.biomaterial_treatment_biomaterial_treatment_id_seq OWNER TO gmod;

--
-- Name: biomaterial_treatment_biomaterial_treatment_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE biomaterial_treatment_biomaterial_treatment_id_seq OWNED BY biomaterial_treatment.biomaterial_treatment_id;


--
-- Name: biomaterial_treatment_biomaterial_treatment_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('biomaterial_treatment_biomaterial_treatment_id_seq', 1, false);


--
-- Name: biomaterialprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE biomaterialprop (
    biomaterialprop_id integer NOT NULL,
    biomaterial_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.biomaterialprop OWNER TO gmod;

--
-- Name: TABLE biomaterialprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE biomaterialprop IS 'Extra biomaterial properties that are not accounted for in biomaterial.';


--
-- Name: biomaterialprop_biomaterialprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE biomaterialprop_biomaterialprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.biomaterialprop_biomaterialprop_id_seq OWNER TO gmod;

--
-- Name: biomaterialprop_biomaterialprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE biomaterialprop_biomaterialprop_id_seq OWNED BY biomaterialprop.biomaterialprop_id;


--
-- Name: biomaterialprop_biomaterialprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('biomaterialprop_biomaterialprop_id_seq', 1, false);


--
-- Name: cell_line; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line (
    cell_line_id integer NOT NULL,
    name character varying(255),
    uniquename character varying(255) NOT NULL,
    organism_id integer NOT NULL,
    timeaccessioned timestamp without time zone DEFAULT now() NOT NULL,
    timelastmodified timestamp without time zone DEFAULT now() NOT NULL
);


ALTER TABLE public.cell_line OWNER TO gmod;

--
-- Name: cell_line_cell_line_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_cell_line_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_cell_line_id_seq OWNER TO gmod;

--
-- Name: cell_line_cell_line_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_cell_line_id_seq OWNED BY cell_line.cell_line_id;


--
-- Name: cell_line_cell_line_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_cell_line_id_seq', 1, false);


--
-- Name: cell_line_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_cvterm (
    cell_line_cvterm_id integer NOT NULL,
    cell_line_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    pub_id integer NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.cell_line_cvterm OWNER TO gmod;

--
-- Name: cell_line_cvterm_cell_line_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_cvterm_cell_line_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_cvterm_cell_line_cvterm_id_seq OWNER TO gmod;

--
-- Name: cell_line_cvterm_cell_line_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_cvterm_cell_line_cvterm_id_seq OWNED BY cell_line_cvterm.cell_line_cvterm_id;


--
-- Name: cell_line_cvterm_cell_line_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_cvterm_cell_line_cvterm_id_seq', 1, false);


--
-- Name: cell_line_cvtermprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_cvtermprop (
    cell_line_cvtermprop_id integer NOT NULL,
    cell_line_cvterm_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.cell_line_cvtermprop OWNER TO gmod;

--
-- Name: cell_line_cvtermprop_cell_line_cvtermprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_cvtermprop_cell_line_cvtermprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_cvtermprop_cell_line_cvtermprop_id_seq OWNER TO gmod;

--
-- Name: cell_line_cvtermprop_cell_line_cvtermprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_cvtermprop_cell_line_cvtermprop_id_seq OWNED BY cell_line_cvtermprop.cell_line_cvtermprop_id;


--
-- Name: cell_line_cvtermprop_cell_line_cvtermprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_cvtermprop_cell_line_cvtermprop_id_seq', 1, false);


--
-- Name: cell_line_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_dbxref (
    cell_line_dbxref_id integer NOT NULL,
    cell_line_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    is_current boolean DEFAULT true NOT NULL
);


ALTER TABLE public.cell_line_dbxref OWNER TO gmod;

--
-- Name: cell_line_dbxref_cell_line_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_dbxref_cell_line_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_dbxref_cell_line_dbxref_id_seq OWNER TO gmod;

--
-- Name: cell_line_dbxref_cell_line_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_dbxref_cell_line_dbxref_id_seq OWNED BY cell_line_dbxref.cell_line_dbxref_id;


--
-- Name: cell_line_dbxref_cell_line_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_dbxref_cell_line_dbxref_id_seq', 1, false);


--
-- Name: cell_line_feature; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_feature (
    cell_line_feature_id integer NOT NULL,
    cell_line_id integer NOT NULL,
    feature_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.cell_line_feature OWNER TO gmod;

--
-- Name: cell_line_feature_cell_line_feature_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_feature_cell_line_feature_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_feature_cell_line_feature_id_seq OWNER TO gmod;

--
-- Name: cell_line_feature_cell_line_feature_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_feature_cell_line_feature_id_seq OWNED BY cell_line_feature.cell_line_feature_id;


--
-- Name: cell_line_feature_cell_line_feature_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_feature_cell_line_feature_id_seq', 1, false);


--
-- Name: cell_line_library; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_library (
    cell_line_library_id integer NOT NULL,
    cell_line_id integer NOT NULL,
    library_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.cell_line_library OWNER TO gmod;

--
-- Name: cell_line_library_cell_line_library_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_library_cell_line_library_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_library_cell_line_library_id_seq OWNER TO gmod;

--
-- Name: cell_line_library_cell_line_library_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_library_cell_line_library_id_seq OWNED BY cell_line_library.cell_line_library_id;


--
-- Name: cell_line_library_cell_line_library_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_library_cell_line_library_id_seq', 1, false);


--
-- Name: cell_line_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_pub (
    cell_line_pub_id integer NOT NULL,
    cell_line_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.cell_line_pub OWNER TO gmod;

--
-- Name: cell_line_pub_cell_line_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_pub_cell_line_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_pub_cell_line_pub_id_seq OWNER TO gmod;

--
-- Name: cell_line_pub_cell_line_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_pub_cell_line_pub_id_seq OWNED BY cell_line_pub.cell_line_pub_id;


--
-- Name: cell_line_pub_cell_line_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_pub_cell_line_pub_id_seq', 1, false);


--
-- Name: cell_line_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_relationship (
    cell_line_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    object_id integer NOT NULL,
    type_id integer NOT NULL
);


ALTER TABLE public.cell_line_relationship OWNER TO gmod;

--
-- Name: cell_line_relationship_cell_line_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_relationship_cell_line_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_relationship_cell_line_relationship_id_seq OWNER TO gmod;

--
-- Name: cell_line_relationship_cell_line_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_relationship_cell_line_relationship_id_seq OWNED BY cell_line_relationship.cell_line_relationship_id;


--
-- Name: cell_line_relationship_cell_line_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_relationship_cell_line_relationship_id_seq', 1, false);


--
-- Name: cell_line_synonym; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_line_synonym (
    cell_line_synonym_id integer NOT NULL,
    cell_line_id integer NOT NULL,
    synonym_id integer NOT NULL,
    pub_id integer NOT NULL,
    is_current boolean DEFAULT false NOT NULL,
    is_internal boolean DEFAULT false NOT NULL
);


ALTER TABLE public.cell_line_synonym OWNER TO gmod;

--
-- Name: cell_line_synonym_cell_line_synonym_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_line_synonym_cell_line_synonym_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_line_synonym_cell_line_synonym_id_seq OWNER TO gmod;

--
-- Name: cell_line_synonym_cell_line_synonym_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_line_synonym_cell_line_synonym_id_seq OWNED BY cell_line_synonym.cell_line_synonym_id;


--
-- Name: cell_line_synonym_cell_line_synonym_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_line_synonym_cell_line_synonym_id_seq', 1, false);


--
-- Name: cell_lineprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_lineprop (
    cell_lineprop_id integer NOT NULL,
    cell_line_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.cell_lineprop OWNER TO gmod;

--
-- Name: cell_lineprop_cell_lineprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_lineprop_cell_lineprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_lineprop_cell_lineprop_id_seq OWNER TO gmod;

--
-- Name: cell_lineprop_cell_lineprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_lineprop_cell_lineprop_id_seq OWNED BY cell_lineprop.cell_lineprop_id;


--
-- Name: cell_lineprop_cell_lineprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_lineprop_cell_lineprop_id_seq', 1, false);


--
-- Name: cell_lineprop_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cell_lineprop_pub (
    cell_lineprop_pub_id integer NOT NULL,
    cell_lineprop_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.cell_lineprop_pub OWNER TO gmod;

--
-- Name: cell_lineprop_pub_cell_lineprop_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cell_lineprop_pub_cell_lineprop_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cell_lineprop_pub_cell_lineprop_pub_id_seq OWNER TO gmod;

--
-- Name: cell_lineprop_pub_cell_lineprop_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cell_lineprop_pub_cell_lineprop_pub_id_seq OWNED BY cell_lineprop_pub.cell_lineprop_pub_id;


--
-- Name: cell_lineprop_pub_cell_lineprop_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cell_lineprop_pub_cell_lineprop_pub_id_seq', 1, false);


--
-- Name: channel; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE channel (
    channel_id integer NOT NULL,
    name text NOT NULL,
    definition text NOT NULL
);


ALTER TABLE public.channel OWNER TO gmod;

--
-- Name: TABLE channel; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE channel IS 'Different array platforms can record signals from one or more channels (cDNA arrays typically use two CCD, but Affymetrix uses only one).';


--
-- Name: channel_channel_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE channel_channel_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.channel_channel_id_seq OWNER TO gmod;

--
-- Name: channel_channel_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE channel_channel_id_seq OWNED BY channel.channel_id;


--
-- Name: channel_channel_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('channel_channel_id_seq', 1, false);


--
-- Name: contact; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE contact (
    contact_id integer NOT NULL,
    type_id integer,
    name character varying(255) NOT NULL,
    description character varying(255)
);


ALTER TABLE public.contact OWNER TO gmod;

--
-- Name: TABLE contact; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE contact IS 'Model persons, institutes, groups, organizations, etc.';


--
-- Name: COLUMN contact.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN contact.type_id IS 'What type of contact is this? E.g. "person", "lab".';


--
-- Name: contact_contact_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE contact_contact_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.contact_contact_id_seq OWNER TO gmod;

--
-- Name: contact_contact_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE contact_contact_id_seq OWNED BY contact.contact_id;


--
-- Name: contact_contact_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('contact_contact_id_seq', 11, true);


--
-- Name: contact_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE contact_relationship (
    contact_relationship_id integer NOT NULL,
    type_id integer NOT NULL,
    subject_id integer NOT NULL,
    object_id integer NOT NULL
);


ALTER TABLE public.contact_relationship OWNER TO gmod;

--
-- Name: TABLE contact_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE contact_relationship IS 'Model relationships between contacts';


--
-- Name: COLUMN contact_relationship.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN contact_relationship.type_id IS 'Relationship type between subject and object. This is a cvterm, typically from the OBO relationship ontology, although other relationship types are allowed.';


--
-- Name: COLUMN contact_relationship.subject_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN contact_relationship.subject_id IS 'The subject of the subj-predicate-obj sentence. In a DAG, this corresponds to the child node.';


--
-- Name: COLUMN contact_relationship.object_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN contact_relationship.object_id IS 'The object of the subj-predicate-obj sentence. In a DAG, this corresponds to the parent node.';


--
-- Name: contact_relationship_contact_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE contact_relationship_contact_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.contact_relationship_contact_relationship_id_seq OWNER TO gmod;

--
-- Name: contact_relationship_contact_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE contact_relationship_contact_relationship_id_seq OWNED BY contact_relationship.contact_relationship_id;


--
-- Name: contact_relationship_contact_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('contact_relationship_contact_relationship_id_seq', 1, false);


--
-- Name: contactprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE contactprop (
    contactprop_id integer NOT NULL,
    contact_id integer NOT NULL,
    type_id integer NOT NULL,
    value text
);


ALTER TABLE public.contactprop OWNER TO gmod;

--
-- Name: contactprop_contactprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE contactprop_contactprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.contactprop_contactprop_id_seq OWNER TO gmod;

--
-- Name: contactprop_contactprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE contactprop_contactprop_id_seq OWNED BY contactprop.contactprop_id;


--
-- Name: contactprop_contactprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('contactprop_contactprop_id_seq', 1, false);


--
-- Name: control; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE control (
    control_id integer NOT NULL,
    type_id integer NOT NULL,
    assay_id integer NOT NULL,
    tableinfo_id integer NOT NULL,
    row_id integer NOT NULL,
    name text,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.control OWNER TO gmod;

--
-- Name: control_control_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE control_control_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.control_control_id_seq OWNER TO gmod;

--
-- Name: control_control_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE control_control_id_seq OWNED BY control.control_id;


--
-- Name: control_control_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('control_control_id_seq', 1, false);


--
-- Name: cv; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cv (
    cv_id integer NOT NULL,
    name character varying(255) NOT NULL,
    definition text
);


ALTER TABLE public.cv OWNER TO gmod;

--
-- Name: TABLE cv; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE cv IS 'A controlled vocabulary or ontology. A cv is composed of cvterms (AKA terms, classes, types, universals - relations and properties are also stored in cvterm) and the relationships between them.';


--
-- Name: COLUMN cv.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cv.name IS 'The name of the ontology. This corresponds to the obo-format -namespace-. cv names uniquely identify the cv. In OBO file format, the cv.name is known as the namespace.';


--
-- Name: COLUMN cv.definition; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cv.definition IS 'A text description of the criteria for membership of this ontology.';


--
-- Name: cv_cv_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cv_cv_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cv_cv_id_seq OWNER TO gmod;

--
-- Name: cv_cv_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cv_cv_id_seq OWNED BY cv.cv_id;


--
-- Name: cv_cv_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cv_cv_id_seq', 14, true);


--
-- Name: cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cvterm (
    cvterm_id integer NOT NULL,
    cv_id integer NOT NULL,
    name character varying(1024) NOT NULL,
    definition text,
    dbxref_id integer NOT NULL,
    is_obsolete integer DEFAULT 0 NOT NULL,
    is_relationshiptype integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.cvterm OWNER TO gmod;

--
-- Name: TABLE cvterm; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE cvterm IS 'A term, class, universal or type within an ontology or controlled vocabulary. This table is also used for relations and properties. cvterms constitute nodes in the graph defined by the collection of cvterms and cvterm_relationships.';


--
-- Name: COLUMN cvterm.cv_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm.cv_id IS 'The cv or ontology or namespace to which this cvterm belongs.';


--
-- Name: COLUMN cvterm.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm.name IS 'A concise human-readable name or label for the cvterm. Uniquely identifies a cvterm within a cv.';


--
-- Name: COLUMN cvterm.definition; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm.definition IS 'A human-readable text definition.';


--
-- Name: COLUMN cvterm.dbxref_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm.dbxref_id IS 'Primary identifier dbxref - The unique global OBO identifier for this cvterm. Note that a cvterm may have multiple secondary dbxrefs - see also table: cvterm_dbxref.';


--
-- Name: COLUMN cvterm.is_obsolete; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm.is_obsolete IS 'Boolean 0=false,1=true; see GO documentation for details of obsoletion. Note that two terms with different primary dbxrefs may exist if one is obsolete.';


--
-- Name: COLUMN cvterm.is_relationshiptype; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm.is_relationshiptype IS 'Boolean 0=false,1=true relations or relationship types (also known as Typedefs in OBO format, or as properties or slots) form a cv/ontology in themselves. We use this flag to indicate whether this cvterm is an actual term/class/universal or a relation. Relations may be drawn from the OBO Relations ontology, but are not exclusively drawn from there.';


--
-- Name: cvterm_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cvterm_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cvterm_cvterm_id_seq OWNER TO gmod;

--
-- Name: cvterm_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cvterm_cvterm_id_seq OWNED BY cvterm.cvterm_id;


--
-- Name: cvterm_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cvterm_cvterm_id_seq', 2316, true);


--
-- Name: cvterm_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cvterm_dbxref (
    cvterm_dbxref_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    is_for_definition integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.cvterm_dbxref OWNER TO gmod;

--
-- Name: TABLE cvterm_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE cvterm_dbxref IS 'In addition to the primary identifier (cvterm.dbxref_id) a cvterm can have zero or more secondary identifiers/dbxrefs, which may refer to records in external databases. The exact semantics of cvterm_dbxref are not fixed. For example: the dbxref could be a pubmed ID that is pertinent to the cvterm, or it could be an equivalent or similar term in another ontology. For example, GO cvterms are typically linked to InterPro IDs, even though the nature of the relationship between them is largely one of statistical association. The dbxref may be have data records attached in the same database instance, or it could be a "hanging" dbxref pointing to some external database. NOTE: If the desired objective is to link two cvterms together, and the nature of the relation is known and holds for all instances of the subject cvterm then consider instead using cvterm_relationship together with a well-defined relation.';


--
-- Name: COLUMN cvterm_dbxref.is_for_definition; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm_dbxref.is_for_definition IS 'A cvterm.definition should be supported by one or more references. If this column is true, the dbxref is not for a term in an external database - it is a dbxref for provenance information for the definition.';


--
-- Name: cvterm_dbxref_cvterm_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cvterm_dbxref_cvterm_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cvterm_dbxref_cvterm_dbxref_id_seq OWNER TO gmod;

--
-- Name: cvterm_dbxref_cvterm_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cvterm_dbxref_cvterm_dbxref_id_seq OWNED BY cvterm_dbxref.cvterm_dbxref_id;


--
-- Name: cvterm_dbxref_cvterm_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cvterm_dbxref_cvterm_dbxref_id_seq', 2339, true);


--
-- Name: cvterm_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cvterm_relationship (
    cvterm_relationship_id integer NOT NULL,
    type_id integer NOT NULL,
    subject_id integer NOT NULL,
    object_id integer NOT NULL
);


ALTER TABLE public.cvterm_relationship OWNER TO gmod;

--
-- Name: TABLE cvterm_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE cvterm_relationship IS 'A relationship linking two cvterms. Each cvterm_relationship constitutes an edge in the graph defined by the collection of cvterms and cvterm_relationships. The meaning of the cvterm_relationship depends on the definition of the cvterm R refered to by type_id. However, in general the definitions are such that the statement "all SUBJs REL some OBJ" is true. The cvterm_relationship statement is about the subject, not the object. For example "insect wing part_of thorax".';


--
-- Name: COLUMN cvterm_relationship.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm_relationship.type_id IS 'The nature of the relationship between subject and object. Note that relations are also housed in the cvterm table, typically from the OBO relationship ontology, although other relationship types are allowed.';


--
-- Name: COLUMN cvterm_relationship.subject_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm_relationship.subject_id IS 'The subject of the subj-predicate-obj sentence. The cvterm_relationship is about the subject. In a graph, this typically corresponds to the child node.';


--
-- Name: COLUMN cvterm_relationship.object_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvterm_relationship.object_id IS 'The object of the subj-predicate-obj sentence. The cvterm_relationship refers to the object. In a graph, this typically corresponds to the parent node.';


--
-- Name: cvterm_relationship_cvterm_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cvterm_relationship_cvterm_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cvterm_relationship_cvterm_relationship_id_seq OWNER TO gmod;

--
-- Name: cvterm_relationship_cvterm_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cvterm_relationship_cvterm_relationship_id_seq OWNED BY cvterm_relationship.cvterm_relationship_id;


--
-- Name: cvterm_relationship_cvterm_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cvterm_relationship_cvterm_relationship_id_seq', 2459, true);


--
-- Name: cvtermpath; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cvtermpath (
    cvtermpath_id integer NOT NULL,
    type_id integer,
    subject_id integer NOT NULL,
    object_id integer NOT NULL,
    cv_id integer NOT NULL,
    pathdistance integer
);


ALTER TABLE public.cvtermpath OWNER TO gmod;

--
-- Name: TABLE cvtermpath; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE cvtermpath IS 'The reflexive transitive closure of the cvterm_relationship relation.';


--
-- Name: COLUMN cvtermpath.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvtermpath.type_id IS 'The relationship type that this is a closure over. If null, then this is a closure over ALL relationship types. If non-null, then this references a relationship cvterm - note that the closure will apply to both this relationship AND the OBO_REL:is_a (subclass) relationship.';


--
-- Name: COLUMN cvtermpath.cv_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvtermpath.cv_id IS 'Closures will mostly be within one cv. If the closure of a relationship traverses a cv, then this refers to the cv of the object_id cvterm.';


--
-- Name: COLUMN cvtermpath.pathdistance; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvtermpath.pathdistance IS 'The number of steps required to get from the subject cvterm to the object cvterm, counting from zero (reflexive relationship).';


--
-- Name: cvtermpath_cvtermpath_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cvtermpath_cvtermpath_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cvtermpath_cvtermpath_id_seq OWNER TO gmod;

--
-- Name: cvtermpath_cvtermpath_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cvtermpath_cvtermpath_id_seq OWNED BY cvtermpath.cvtermpath_id;


--
-- Name: cvtermpath_cvtermpath_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cvtermpath_cvtermpath_id_seq', 6231, true);


--
-- Name: cvtermprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cvtermprop (
    cvtermprop_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    type_id integer NOT NULL,
    value text DEFAULT ''::text NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.cvtermprop OWNER TO gmod;

--
-- Name: TABLE cvtermprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE cvtermprop IS 'Additional extensible properties can be attached to a cvterm using this table. Corresponds to -AnnotationProperty- in W3C OWL format.';


--
-- Name: COLUMN cvtermprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvtermprop.type_id IS 'The name of the property or slot is a cvterm. The meaning of the property is defined in that cvterm.';


--
-- Name: COLUMN cvtermprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvtermprop.value IS 'The value of the property, represented as text. Numeric values are converted to their text representation.';


--
-- Name: COLUMN cvtermprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvtermprop.rank IS 'Property-Value ordering. Any cvterm can have multiple values for any particular property type - these are ordered in a list using rank, counting from zero. For properties that are single-valued rather than multi-valued, the default 0 value should be used.';


--
-- Name: cvtermprop_cvtermprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cvtermprop_cvtermprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cvtermprop_cvtermprop_id_seq OWNER TO gmod;

--
-- Name: cvtermprop_cvtermprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cvtermprop_cvtermprop_id_seq OWNED BY cvtermprop.cvtermprop_id;


--
-- Name: cvtermprop_cvtermprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cvtermprop_cvtermprop_id_seq', 320, true);


--
-- Name: cvtermsynonym; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE cvtermsynonym (
    cvtermsynonym_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    synonym character varying(1024) NOT NULL,
    type_id integer
);


ALTER TABLE public.cvtermsynonym OWNER TO gmod;

--
-- Name: TABLE cvtermsynonym; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE cvtermsynonym IS 'A cvterm actually represents a distinct class or concept. A concept can be refered to by different phrases or names. In addition to the primary name (cvterm.name) there can be a number of alternative aliases or synonyms. For example, "T cell" as a synonym for "T lymphocyte".';


--
-- Name: COLUMN cvtermsynonym.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN cvtermsynonym.type_id IS 'A synonym can be exact, narrower, or broader than.';


--
-- Name: cvtermsynonym_cvtermsynonym_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE cvtermsynonym_cvtermsynonym_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.cvtermsynonym_cvtermsynonym_id_seq OWNER TO gmod;

--
-- Name: cvtermsynonym_cvtermsynonym_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE cvtermsynonym_cvtermsynonym_id_seq OWNED BY cvtermsynonym.cvtermsynonym_id;


--
-- Name: cvtermsynonym_cvtermsynonym_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('cvtermsynonym_cvtermsynonym_id_seq', 2908, true);


--
-- Name: db; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE db (
    db_id integer NOT NULL,
    name character varying(255) NOT NULL,
    description character varying(255),
    urlprefix character varying(255),
    url character varying(255)
);


ALTER TABLE public.db OWNER TO gmod;

--
-- Name: TABLE db; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE db IS 'A database authority. Typical databases in bioinformatics are FlyBase, GO, UniProt, NCBI, MGI, etc. The authority is generally known by this shortened form, which is unique within the bioinformatics and biomedical realm. To Do - add support for URIs, URNs (e.g. LSIDs). We can do this by treating the URL as a URI - however, some applications may expect this to be resolvable - to be decided.';


--
-- Name: db_db_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE db_db_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.db_db_id_seq OWNER TO gmod;

--
-- Name: db_db_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE db_db_id_seq OWNED BY db.db_id;


--
-- Name: db_db_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('db_db_id_seq', 92, true);


--
-- Name: dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE dbxref (
    dbxref_id integer NOT NULL,
    db_id integer NOT NULL,
    accession character varying(255) NOT NULL,
    version character varying(255) DEFAULT ''::character varying NOT NULL,
    description text
);


ALTER TABLE public.dbxref OWNER TO gmod;

--
-- Name: TABLE dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE dbxref IS 'A unique, global, public, stable identifier. Not necessarily an external reference - can reference data items inside the particular chado instance being used. Typically a row in a table can be uniquely identified with a primary identifier (called dbxref_id); a table may also have secondary identifiers (in a linking table T_dbxref). A dbxref is generally written as DB:ACCESSION or as DB:ACCESSION:VERSION.';


--
-- Name: COLUMN dbxref.accession; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN dbxref.accession IS 'The local part of the identifier. Guaranteed by the db authority to be unique for that db.';


--
-- Name: dbxref_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE dbxref_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.dbxref_dbxref_id_seq OWNER TO gmod;

--
-- Name: dbxref_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE dbxref_dbxref_id_seq OWNED BY dbxref.dbxref_id;


--
-- Name: dbxref_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('dbxref_dbxref_id_seq', 3023, true);


--
-- Name: dbxrefprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE dbxrefprop (
    dbxrefprop_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    type_id integer NOT NULL,
    value text DEFAULT ''::text NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.dbxrefprop OWNER TO gmod;

--
-- Name: TABLE dbxrefprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE dbxrefprop IS 'Metadata about a dbxref. Note that this is not defined in the dbxref module, as it depends on the cvterm table. This table has a structure analagous to cvtermprop.';


--
-- Name: dbxrefprop_dbxrefprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE dbxrefprop_dbxrefprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.dbxrefprop_dbxrefprop_id_seq OWNER TO gmod;

--
-- Name: dbxrefprop_dbxrefprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE dbxrefprop_dbxrefprop_id_seq OWNED BY dbxrefprop.dbxrefprop_id;


--
-- Name: dbxrefprop_dbxrefprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('dbxrefprop_dbxrefprop_id_seq', 1, false);


--
-- Name: eimage; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE eimage (
    eimage_id integer NOT NULL,
    eimage_data text,
    eimage_type character varying(255) NOT NULL,
    image_uri character varying(255)
);


ALTER TABLE public.eimage OWNER TO gmod;

--
-- Name: COLUMN eimage.eimage_data; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN eimage.eimage_data IS 'We expect images in eimage_data (e.g. JPEGs) to be uuencoded.';


--
-- Name: COLUMN eimage.eimage_type; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN eimage.eimage_type IS 'Describes the type of data in eimage_data.';


--
-- Name: eimage_eimage_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE eimage_eimage_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.eimage_eimage_id_seq OWNER TO gmod;

--
-- Name: eimage_eimage_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE eimage_eimage_id_seq OWNED BY eimage.eimage_id;


--
-- Name: eimage_eimage_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('eimage_eimage_id_seq', 1, false);


--
-- Name: element; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE element (
    element_id integer NOT NULL,
    feature_id integer,
    arraydesign_id integer NOT NULL,
    type_id integer,
    dbxref_id integer
);


ALTER TABLE public.element OWNER TO gmod;

--
-- Name: TABLE element; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE element IS 'Represents a feature of the array. This is typically a region of the array coated or bound to DNA.';


--
-- Name: element_element_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE element_element_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.element_element_id_seq OWNER TO gmod;

--
-- Name: element_element_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE element_element_id_seq OWNED BY element.element_id;


--
-- Name: element_element_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('element_element_id_seq', 1, false);


--
-- Name: element_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE element_relationship (
    element_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    type_id integer NOT NULL,
    object_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.element_relationship OWNER TO gmod;

--
-- Name: TABLE element_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE element_relationship IS 'Sometimes we want to combine measurements from multiple elements to get a composite value. Affymetrix combines many probes to form a probeset measurement, for instance.';


--
-- Name: element_relationship_element_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE element_relationship_element_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.element_relationship_element_relationship_id_seq OWNER TO gmod;

--
-- Name: element_relationship_element_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE element_relationship_element_relationship_id_seq OWNED BY element_relationship.element_relationship_id;


--
-- Name: element_relationship_element_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('element_relationship_element_relationship_id_seq', 1, false);


--
-- Name: elementresult; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE elementresult (
    elementresult_id integer NOT NULL,
    element_id integer NOT NULL,
    quantification_id integer NOT NULL,
    signal double precision NOT NULL
);


ALTER TABLE public.elementresult OWNER TO gmod;

--
-- Name: TABLE elementresult; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE elementresult IS 'An element on an array produces a measurement when hybridized to a biomaterial (traceable through quantification_id). This is the base data from which tables that actually contain data inherit.';


--
-- Name: elementresult_elementresult_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE elementresult_elementresult_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.elementresult_elementresult_id_seq OWNER TO gmod;

--
-- Name: elementresult_elementresult_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE elementresult_elementresult_id_seq OWNED BY elementresult.elementresult_id;


--
-- Name: elementresult_elementresult_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('elementresult_elementresult_id_seq', 1, false);


--
-- Name: elementresult_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE elementresult_relationship (
    elementresult_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    type_id integer NOT NULL,
    object_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.elementresult_relationship OWNER TO gmod;

--
-- Name: TABLE elementresult_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE elementresult_relationship IS 'Sometimes we want to combine measurements from multiple elements to get a composite value. Affymetrix combines many probes to form a probeset measurement, for instance.';


--
-- Name: elementresult_relationship_elementresult_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE elementresult_relationship_elementresult_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.elementresult_relationship_elementresult_relationship_id_seq OWNER TO gmod;

--
-- Name: elementresult_relationship_elementresult_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE elementresult_relationship_elementresult_relationship_id_seq OWNED BY elementresult_relationship.elementresult_relationship_id;


--
-- Name: elementresult_relationship_elementresult_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('elementresult_relationship_elementresult_relationship_id_seq', 1, false);


--
-- Name: environment; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE environment (
    environment_id integer NOT NULL,
    uniquename text NOT NULL,
    description text
);


ALTER TABLE public.environment OWNER TO gmod;

--
-- Name: TABLE environment; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE environment IS 'The environmental component of a phenotype description.';


--
-- Name: environment_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE environment_cvterm (
    environment_cvterm_id integer NOT NULL,
    environment_id integer NOT NULL,
    cvterm_id integer NOT NULL
);


ALTER TABLE public.environment_cvterm OWNER TO gmod;

--
-- Name: environment_cvterm_environment_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE environment_cvterm_environment_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.environment_cvterm_environment_cvterm_id_seq OWNER TO gmod;

--
-- Name: environment_cvterm_environment_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE environment_cvterm_environment_cvterm_id_seq OWNED BY environment_cvterm.environment_cvterm_id;


--
-- Name: environment_cvterm_environment_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('environment_cvterm_environment_cvterm_id_seq', 1, false);


--
-- Name: environment_environment_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE environment_environment_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.environment_environment_id_seq OWNER TO gmod;

--
-- Name: environment_environment_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE environment_environment_id_seq OWNED BY environment.environment_id;


--
-- Name: environment_environment_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('environment_environment_id_seq', 1, false);


--
-- Name: expression; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE expression (
    expression_id integer NOT NULL,
    uniquename text NOT NULL,
    md5checksum character(32),
    description text
);


ALTER TABLE public.expression OWNER TO gmod;

--
-- Name: TABLE expression; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE expression IS 'The expression table is essentially a bridge table.';


--
-- Name: expression_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE expression_cvterm (
    expression_cvterm_id integer NOT NULL,
    expression_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    rank integer DEFAULT 0 NOT NULL,
    cvterm_type_id integer NOT NULL
);


ALTER TABLE public.expression_cvterm OWNER TO gmod;

--
-- Name: expression_cvterm_expression_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE expression_cvterm_expression_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.expression_cvterm_expression_cvterm_id_seq OWNER TO gmod;

--
-- Name: expression_cvterm_expression_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE expression_cvterm_expression_cvterm_id_seq OWNED BY expression_cvterm.expression_cvterm_id;


--
-- Name: expression_cvterm_expression_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('expression_cvterm_expression_cvterm_id_seq', 1, false);


--
-- Name: expression_cvtermprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE expression_cvtermprop (
    expression_cvtermprop_id integer NOT NULL,
    expression_cvterm_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.expression_cvtermprop OWNER TO gmod;

--
-- Name: TABLE expression_cvtermprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE expression_cvtermprop IS 'Extensible properties for expression to cvterm associations. Examples: qualifiers.';


--
-- Name: COLUMN expression_cvtermprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN expression_cvtermprop.type_id IS 'The name of the property/slot is a cvterm. The meaning of the property is defined in that cvterm. For example, cvterms may come from the FlyBase miscellaneous cv.';


--
-- Name: COLUMN expression_cvtermprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN expression_cvtermprop.value IS 'The value of the property, represented as text. Numeric values are converted to their text representation. This is less efficient than using native database types, but is easier to query.';


--
-- Name: COLUMN expression_cvtermprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN expression_cvtermprop.rank IS 'Property-Value ordering. Any expression_cvterm can have multiple values for any particular property type - these are ordered in a list using rank, counting from zero. For properties that are single-valued rather than multi-valued, the default 0 value should be used.';


--
-- Name: expression_cvtermprop_expression_cvtermprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE expression_cvtermprop_expression_cvtermprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.expression_cvtermprop_expression_cvtermprop_id_seq OWNER TO gmod;

--
-- Name: expression_cvtermprop_expression_cvtermprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE expression_cvtermprop_expression_cvtermprop_id_seq OWNED BY expression_cvtermprop.expression_cvtermprop_id;


--
-- Name: expression_cvtermprop_expression_cvtermprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('expression_cvtermprop_expression_cvtermprop_id_seq', 1, false);


--
-- Name: expression_expression_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE expression_expression_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.expression_expression_id_seq OWNER TO gmod;

--
-- Name: expression_expression_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE expression_expression_id_seq OWNED BY expression.expression_id;


--
-- Name: expression_expression_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('expression_expression_id_seq', 1, false);


--
-- Name: expression_image; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE expression_image (
    expression_image_id integer NOT NULL,
    expression_id integer NOT NULL,
    eimage_id integer NOT NULL
);


ALTER TABLE public.expression_image OWNER TO gmod;

--
-- Name: expression_image_expression_image_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE expression_image_expression_image_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.expression_image_expression_image_id_seq OWNER TO gmod;

--
-- Name: expression_image_expression_image_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE expression_image_expression_image_id_seq OWNED BY expression_image.expression_image_id;


--
-- Name: expression_image_expression_image_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('expression_image_expression_image_id_seq', 1, false);


--
-- Name: expression_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE expression_pub (
    expression_pub_id integer NOT NULL,
    expression_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.expression_pub OWNER TO gmod;

--
-- Name: expression_pub_expression_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE expression_pub_expression_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.expression_pub_expression_pub_id_seq OWNER TO gmod;

--
-- Name: expression_pub_expression_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE expression_pub_expression_pub_id_seq OWNED BY expression_pub.expression_pub_id;


--
-- Name: expression_pub_expression_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('expression_pub_expression_pub_id_seq', 1, false);


--
-- Name: expressionprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE expressionprop (
    expressionprop_id integer NOT NULL,
    expression_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.expressionprop OWNER TO gmod;

--
-- Name: expressionprop_expressionprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE expressionprop_expressionprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.expressionprop_expressionprop_id_seq OWNER TO gmod;

--
-- Name: expressionprop_expressionprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE expressionprop_expressionprop_id_seq OWNED BY expressionprop.expressionprop_id;


--
-- Name: expressionprop_expressionprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('expressionprop_expressionprop_id_seq', 1, false);


--
-- Name: feature; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature (
    feature_id integer NOT NULL,
    dbxref_id integer,
    organism_id integer NOT NULL,
    name character varying(255),
    uniquename text NOT NULL,
    residues text,
    seqlen integer,
    md5checksum character(32),
    type_id integer NOT NULL,
    is_analysis boolean DEFAULT false NOT NULL,
    is_obsolete boolean DEFAULT false NOT NULL,
    timeaccessioned timestamp without time zone DEFAULT now() NOT NULL,
    timelastmodified timestamp without time zone DEFAULT now() NOT NULL
);
ALTER TABLE ONLY feature ALTER COLUMN residues SET STORAGE EXTERNAL;


ALTER TABLE public.feature OWNER TO gmod;

--
-- Name: TABLE feature; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature IS 'A feature is a biological sequence or a section of a biological sequence, or a collection of such sections. Examples include genes, exons, transcripts, regulatory regions, polypeptides, protein domains, chromosome sequences, sequence variations, cross-genome match regions such as hits and HSPs and so on; see the Sequence Ontology for more. The combination of organism_id, uniquename and type_id should be unique.';


--
-- Name: COLUMN feature.dbxref_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.dbxref_id IS 'An optional primary public stable identifier for this feature. Secondary identifiers and external dbxrefs go in the table feature_dbxref.';


--
-- Name: COLUMN feature.organism_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.organism_id IS 'The organism to which this feature belongs. This column is mandatory.';


--
-- Name: COLUMN feature.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.name IS 'The optional human-readable common name for a feature, for display purposes.';


--
-- Name: COLUMN feature.uniquename; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.uniquename IS 'The unique name for a feature; may not be necessarily be particularly human-readable, although this is preferred. This name must be unique for this type of feature within this organism.';


--
-- Name: COLUMN feature.residues; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.residues IS 'A sequence of alphabetic characters representing biological residues (nucleic acids, amino acids). This column does not need to be manifested for all features; it is optional for features such as exons where the residues can be derived from the featureloc. It is recommended that the value for this column be manifested for features which may may non-contiguous sublocations (e.g. transcripts), since derivation at query time is non-trivial. For expressed sequence, the DNA sequence should be used rather than the RNA sequence. The default storage method for the residues column is EXTERNAL, which will store it uncompressed to make substring operations faster.';


--
-- Name: COLUMN feature.seqlen; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.seqlen IS 'The length of the residue feature. See column:residues. This column is partially redundant with the residues column, and also with featureloc. This column is required because the location may be unknown and the residue sequence may not be manifested, yet it may be desirable to store and query the length of the feature. The seqlen should always be manifested where the length of the sequence is known.';


--
-- Name: COLUMN feature.md5checksum; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.md5checksum IS 'The 32-character checksum of the sequence, calculated using the MD5 algorithm. This is practically guaranteed to be unique for any feature. This column thus acts as a unique identifier on the mathematical sequence.';


--
-- Name: COLUMN feature.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.type_id IS 'A required reference to a table:cvterm giving the feature type. This will typically be a Sequence Ontology identifier. This column is thus used to subclass the feature table.';


--
-- Name: COLUMN feature.is_analysis; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.is_analysis IS 'Boolean indicating whether this feature is annotated or the result of an automated analysis. Analysis results also use the companalysis module. Note that the dividing line between analysis and annotation may be fuzzy, this should be determined on a per-project basis in a consistent manner. One requirement is that there should only be one non-analysis version of each wild-type gene feature in a genome, whereas the same gene feature can be predicted multiple times in different analyses.';


--
-- Name: COLUMN feature.is_obsolete; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.is_obsolete IS 'Boolean indicating whether this feature has been obsoleted. Some chado instances may choose to simply remove the feature altogether, others may choose to keep an obsolete row in the table.';


--
-- Name: COLUMN feature.timeaccessioned; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.timeaccessioned IS 'For handling object accession or modification timestamps (as opposed to database auditing data, handled elsewhere). The expectation is that these fields would be available to software interacting with chado.';


--
-- Name: COLUMN feature.timelastmodified; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature.timelastmodified IS 'For handling object accession or modification timestamps (as opposed to database auditing data, handled elsewhere). The expectation is that these fields would be available to software interacting with chado.';


--
-- Name: feature_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_cvterm (
    feature_cvterm_id integer NOT NULL,
    feature_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    pub_id integer NOT NULL,
    is_not boolean DEFAULT false NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.feature_cvterm OWNER TO gmod;

--
-- Name: TABLE feature_cvterm; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_cvterm IS 'Associate a term from a cv with a feature, for example, GO annotation.';


--
-- Name: COLUMN feature_cvterm.pub_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_cvterm.pub_id IS 'Provenance for the annotation. Each annotation should have a single primary publication (which may be of the appropriate type for computational analyses) where more details can be found. Additional provenance dbxrefs can be attached using feature_cvterm_dbxref.';


--
-- Name: COLUMN feature_cvterm.is_not; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_cvterm.is_not IS 'If this is set to true, then this annotation is interpreted as a NEGATIVE annotation - i.e. the feature does NOT have the specified function, process, component, part, etc. See GO docs for more details.';


--
-- Name: feature_cvterm_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_cvterm_dbxref (
    feature_cvterm_dbxref_id integer NOT NULL,
    feature_cvterm_id integer NOT NULL,
    dbxref_id integer NOT NULL
);


ALTER TABLE public.feature_cvterm_dbxref OWNER TO gmod;

--
-- Name: TABLE feature_cvterm_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_cvterm_dbxref IS 'Additional dbxrefs for an association. Rows in the feature_cvterm table may be backed up by dbxrefs. For example, a feature_cvterm association that was inferred via a protein-protein interaction may be backed by by refering to the dbxref for the alternate protein. Corresponds to the WITH column in a GO gene association file (but can also be used for other analagous associations). See http://www.geneontology.org/doc/GO.annotation.shtml#file for more details.';


--
-- Name: feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq OWNER TO gmod;

--
-- Name: feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq OWNED BY feature_cvterm_dbxref.feature_cvterm_dbxref_id;


--
-- Name: feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq', 1, false);


--
-- Name: feature_cvterm_feature_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_cvterm_feature_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_cvterm_feature_cvterm_id_seq OWNER TO gmod;

--
-- Name: feature_cvterm_feature_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_cvterm_feature_cvterm_id_seq OWNED BY feature_cvterm.feature_cvterm_id;


--
-- Name: feature_cvterm_feature_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_cvterm_feature_cvterm_id_seq', 1, false);


--
-- Name: feature_cvterm_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_cvterm_pub (
    feature_cvterm_pub_id integer NOT NULL,
    feature_cvterm_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.feature_cvterm_pub OWNER TO gmod;

--
-- Name: TABLE feature_cvterm_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_cvterm_pub IS 'Secondary pubs for an association. Each feature_cvterm association is supported by a single primary publication. Additional secondary pubs can be added using this linking table (in a GO gene association file, these corresponding to any IDs after the pipe symbol in the publications column.';


--
-- Name: feature_cvterm_pub_feature_cvterm_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_cvterm_pub_feature_cvterm_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_cvterm_pub_feature_cvterm_pub_id_seq OWNER TO gmod;

--
-- Name: feature_cvterm_pub_feature_cvterm_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_cvterm_pub_feature_cvterm_pub_id_seq OWNED BY feature_cvterm_pub.feature_cvterm_pub_id;


--
-- Name: feature_cvterm_pub_feature_cvterm_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_cvterm_pub_feature_cvterm_pub_id_seq', 1, false);


--
-- Name: feature_cvtermprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_cvtermprop (
    feature_cvtermprop_id integer NOT NULL,
    feature_cvterm_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.feature_cvtermprop OWNER TO gmod;

--
-- Name: TABLE feature_cvtermprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_cvtermprop IS 'Extensible properties for feature to cvterm associations. Examples: GO evidence codes; qualifiers; metadata such as the date on which the entry was curated and the source of the association. See the featureprop table for meanings of type_id, value and rank.';


--
-- Name: COLUMN feature_cvtermprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_cvtermprop.type_id IS 'The name of the property/slot is a cvterm. The meaning of the property is defined in that cvterm. cvterms may come from the OBO evidence code cv.';


--
-- Name: COLUMN feature_cvtermprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_cvtermprop.value IS 'The value of the property, represented as text. Numeric values are converted to their text representation. This is less efficient than using native database types, but is easier to query.';


--
-- Name: COLUMN feature_cvtermprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_cvtermprop.rank IS 'Property-Value ordering. Any feature_cvterm can have multiple values for any particular property type - these are ordered in a list using rank, counting from zero. For properties that are single-valued rather than multi-valued, the default 0 value should be used.';


--
-- Name: feature_cvtermprop_feature_cvtermprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_cvtermprop_feature_cvtermprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_cvtermprop_feature_cvtermprop_id_seq OWNER TO gmod;

--
-- Name: feature_cvtermprop_feature_cvtermprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_cvtermprop_feature_cvtermprop_id_seq OWNED BY feature_cvtermprop.feature_cvtermprop_id;


--
-- Name: feature_cvtermprop_feature_cvtermprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_cvtermprop_feature_cvtermprop_id_seq', 1, false);


--
-- Name: feature_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_dbxref (
    feature_dbxref_id integer NOT NULL,
    feature_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    is_current boolean DEFAULT true NOT NULL
);


ALTER TABLE public.feature_dbxref OWNER TO gmod;

--
-- Name: TABLE feature_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_dbxref IS 'Links a feature to dbxrefs. This is for secondary identifiers; primary identifiers should use feature.dbxref_id.';


--
-- Name: COLUMN feature_dbxref.is_current; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_dbxref.is_current IS 'True if this secondary dbxref is the most up to date accession in the corresponding db. Retired accessions should set this field to false';


--
-- Name: feature_dbxref_feature_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_dbxref_feature_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_dbxref_feature_dbxref_id_seq OWNER TO gmod;

--
-- Name: feature_dbxref_feature_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_dbxref_feature_dbxref_id_seq OWNED BY feature_dbxref.feature_dbxref_id;


--
-- Name: feature_dbxref_feature_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_dbxref_feature_dbxref_id_seq', 1, false);


--
-- Name: feature_expression; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_expression (
    feature_expression_id integer NOT NULL,
    expression_id integer NOT NULL,
    feature_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.feature_expression OWNER TO gmod;

--
-- Name: feature_expression_feature_expression_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_expression_feature_expression_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_expression_feature_expression_id_seq OWNER TO gmod;

--
-- Name: feature_expression_feature_expression_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_expression_feature_expression_id_seq OWNED BY feature_expression.feature_expression_id;


--
-- Name: feature_expression_feature_expression_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_expression_feature_expression_id_seq', 1, false);


--
-- Name: feature_expressionprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_expressionprop (
    feature_expressionprop_id integer NOT NULL,
    feature_expression_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.feature_expressionprop OWNER TO gmod;

--
-- Name: TABLE feature_expressionprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_expressionprop IS 'Extensible properties for feature_expression (comments, for example). Modeled on feature_cvtermprop.';


--
-- Name: feature_expressionprop_feature_expressionprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_expressionprop_feature_expressionprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_expressionprop_feature_expressionprop_id_seq OWNER TO gmod;

--
-- Name: feature_expressionprop_feature_expressionprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_expressionprop_feature_expressionprop_id_seq OWNED BY feature_expressionprop.feature_expressionprop_id;


--
-- Name: feature_expressionprop_feature_expressionprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_expressionprop_feature_expressionprop_id_seq', 1, false);


--
-- Name: feature_feature_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_feature_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_feature_id_seq OWNER TO gmod;

--
-- Name: feature_feature_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_feature_id_seq OWNED BY feature.feature_id;


--
-- Name: feature_feature_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_feature_id_seq', 1, false);


--
-- Name: feature_genotype; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_genotype (
    feature_genotype_id integer NOT NULL,
    feature_id integer NOT NULL,
    genotype_id integer NOT NULL,
    chromosome_id integer,
    rank integer NOT NULL,
    cgroup integer NOT NULL,
    cvterm_id integer NOT NULL
);


ALTER TABLE public.feature_genotype OWNER TO gmod;

--
-- Name: COLUMN feature_genotype.chromosome_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_genotype.chromosome_id IS 'A feature of SO type "chromosome".';


--
-- Name: COLUMN feature_genotype.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_genotype.rank IS 'rank can be used for n-ploid organisms or to preserve order.';


--
-- Name: COLUMN feature_genotype.cgroup; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_genotype.cgroup IS 'Spatially distinguishable group. group can be used for distinguishing the chromosomal groups, for example (RNAi products and so on can be treated as different groups, as they do not fall on a particular chromosome).';


--
-- Name: feature_genotype_feature_genotype_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_genotype_feature_genotype_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_genotype_feature_genotype_id_seq OWNER TO gmod;

--
-- Name: feature_genotype_feature_genotype_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_genotype_feature_genotype_id_seq OWNED BY feature_genotype.feature_genotype_id;


--
-- Name: feature_genotype_feature_genotype_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_genotype_feature_genotype_id_seq', 1, false);


--
-- Name: feature_phenotype; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_phenotype (
    feature_phenotype_id integer NOT NULL,
    feature_id integer NOT NULL,
    phenotype_id integer NOT NULL
);


ALTER TABLE public.feature_phenotype OWNER TO gmod;

--
-- Name: feature_phenotype_feature_phenotype_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_phenotype_feature_phenotype_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_phenotype_feature_phenotype_id_seq OWNER TO gmod;

--
-- Name: feature_phenotype_feature_phenotype_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_phenotype_feature_phenotype_id_seq OWNED BY feature_phenotype.feature_phenotype_id;


--
-- Name: feature_phenotype_feature_phenotype_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_phenotype_feature_phenotype_id_seq', 1, false);


--
-- Name: feature_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_pub (
    feature_pub_id integer NOT NULL,
    feature_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.feature_pub OWNER TO gmod;

--
-- Name: TABLE feature_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_pub IS 'Provenance. Linking table between features and publications that mention them.';


--
-- Name: feature_pub_feature_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_pub_feature_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_pub_feature_pub_id_seq OWNER TO gmod;

--
-- Name: feature_pub_feature_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_pub_feature_pub_id_seq OWNED BY feature_pub.feature_pub_id;


--
-- Name: feature_pub_feature_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_pub_feature_pub_id_seq', 1, false);


--
-- Name: feature_pubprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_pubprop (
    feature_pubprop_id integer NOT NULL,
    feature_pub_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.feature_pubprop OWNER TO gmod;

--
-- Name: TABLE feature_pubprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_pubprop IS 'Property or attribute of a feature_pub link.';


--
-- Name: feature_pubprop_feature_pubprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_pubprop_feature_pubprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_pubprop_feature_pubprop_id_seq OWNER TO gmod;

--
-- Name: feature_pubprop_feature_pubprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_pubprop_feature_pubprop_id_seq OWNED BY feature_pubprop.feature_pubprop_id;


--
-- Name: feature_pubprop_feature_pubprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_pubprop_feature_pubprop_id_seq', 1, false);


--
-- Name: feature_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_relationship (
    feature_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    object_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.feature_relationship OWNER TO gmod;

--
-- Name: TABLE feature_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_relationship IS 'Features can be arranged in graphs, e.g. "exon part_of transcript part_of gene"; If type is thought of as a verb, the each arc or edge makes a statement [Subject Verb Object]. The object can also be thought of as parent (containing feature), and subject as child (contained feature or subfeature). We include the relationship rank/order, because even though most of the time we can order things implicitly by sequence coordinates, we can not always do this - e.g. transpliced genes. It is also useful for quickly getting implicit introns.';


--
-- Name: COLUMN feature_relationship.subject_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationship.subject_id IS 'The subject of the subj-predicate-obj sentence. This is typically the subfeature.';


--
-- Name: COLUMN feature_relationship.object_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationship.object_id IS 'The object of the subj-predicate-obj sentence. This is typically the container feature.';


--
-- Name: COLUMN feature_relationship.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationship.type_id IS 'Relationship type between subject and object. This is a cvterm, typically from the OBO relationship ontology, although other relationship types are allowed. The most common relationship type is OBO_REL:part_of. Valid relationship types are constrained by the Sequence Ontology.';


--
-- Name: COLUMN feature_relationship.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationship.value IS 'Additional notes or comments.';


--
-- Name: COLUMN feature_relationship.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationship.rank IS 'The ordering of subject features with respect to the object feature may be important (for example, exon ordering on a transcript - not always derivable if you take trans spliced genes into consideration). Rank is used to order these; starts from zero.';


--
-- Name: feature_relationship_feature_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_relationship_feature_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_relationship_feature_relationship_id_seq OWNER TO gmod;

--
-- Name: feature_relationship_feature_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_relationship_feature_relationship_id_seq OWNED BY feature_relationship.feature_relationship_id;


--
-- Name: feature_relationship_feature_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_relationship_feature_relationship_id_seq', 1, false);


--
-- Name: feature_relationship_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_relationship_pub (
    feature_relationship_pub_id integer NOT NULL,
    feature_relationship_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.feature_relationship_pub OWNER TO gmod;

--
-- Name: TABLE feature_relationship_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_relationship_pub IS 'Provenance. Attach optional evidence to a feature_relationship in the form of a publication.';


--
-- Name: feature_relationship_pub_feature_relationship_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_relationship_pub_feature_relationship_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_relationship_pub_feature_relationship_pub_id_seq OWNER TO gmod;

--
-- Name: feature_relationship_pub_feature_relationship_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_relationship_pub_feature_relationship_pub_id_seq OWNED BY feature_relationship_pub.feature_relationship_pub_id;


--
-- Name: feature_relationship_pub_feature_relationship_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_relationship_pub_feature_relationship_pub_id_seq', 1, false);


--
-- Name: feature_relationshipprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_relationshipprop (
    feature_relationshipprop_id integer NOT NULL,
    feature_relationship_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.feature_relationshipprop OWNER TO gmod;

--
-- Name: TABLE feature_relationshipprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_relationshipprop IS 'Extensible properties for feature_relationships. Analagous structure to featureprop. This table is largely optional and not used with a high frequency. Typical scenarios may be if one wishes to attach additional data to a feature_relationship - for example to say that the feature_relationship is only true in certain contexts.';


--
-- Name: COLUMN feature_relationshipprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationshipprop.type_id IS 'The name of the property/slot is a cvterm. The meaning of the property is defined in that cvterm. Currently there is no standard ontology for feature_relationship property types.';


--
-- Name: COLUMN feature_relationshipprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationshipprop.value IS 'The value of the property, represented as text. Numeric values are converted to their text representation. This is less efficient than using native database types, but is easier to query.';


--
-- Name: COLUMN feature_relationshipprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_relationshipprop.rank IS 'Property-Value ordering. Any feature_relationship can have multiple values for any particular property type - these are ordered in a list using rank, counting from zero. For properties that are single-valued rather than multi-valued, the default 0 value should be used.';


--
-- Name: feature_relationshipprop_feature_relationshipprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_relationshipprop_feature_relationshipprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_relationshipprop_feature_relationshipprop_id_seq OWNER TO gmod;

--
-- Name: feature_relationshipprop_feature_relationshipprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_relationshipprop_feature_relationshipprop_id_seq OWNED BY feature_relationshipprop.feature_relationshipprop_id;


--
-- Name: feature_relationshipprop_feature_relationshipprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_relationshipprop_feature_relationshipprop_id_seq', 1, false);


--
-- Name: feature_relationshipprop_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_relationshipprop_pub (
    feature_relationshipprop_pub_id integer NOT NULL,
    feature_relationshipprop_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.feature_relationshipprop_pub OWNER TO gmod;

--
-- Name: TABLE feature_relationshipprop_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_relationshipprop_pub IS 'Provenance for feature_relationshipprop.';


--
-- Name: feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq OWNER TO gmod;

--
-- Name: feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq OWNED BY feature_relationshipprop_pub.feature_relationshipprop_pub_id;


--
-- Name: feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq', 1, false);


--
-- Name: feature_synonym; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE feature_synonym (
    feature_synonym_id integer NOT NULL,
    synonym_id integer NOT NULL,
    feature_id integer NOT NULL,
    pub_id integer NOT NULL,
    is_current boolean DEFAULT false NOT NULL,
    is_internal boolean DEFAULT false NOT NULL
);


ALTER TABLE public.feature_synonym OWNER TO gmod;

--
-- Name: TABLE feature_synonym; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE feature_synonym IS 'Linking table between feature and synonym.';


--
-- Name: COLUMN feature_synonym.pub_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_synonym.pub_id IS 'The pub_id link is for relating the usage of a given synonym to the publication in which it was used.';


--
-- Name: COLUMN feature_synonym.is_current; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_synonym.is_current IS 'The is_current boolean indicates whether the linked synonym is the current -official- symbol for the linked feature.';


--
-- Name: COLUMN feature_synonym.is_internal; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN feature_synonym.is_internal IS 'Typically a synonym exists so that somebody querying the db with an obsolete name can find the object theyre looking for (under its current name. If the synonym has been used publicly and deliberately (e.g. in a paper), it may also be listed in reports as a synonym. If the synonym was not used deliberately (e.g. there was a typo which went public), then the is_internal boolean may be set to -true- so that it is known that the synonym is -internal- and should be queryable but should not be listed in reports as a valid synonym.';


--
-- Name: feature_synonym_feature_synonym_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_synonym_feature_synonym_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_synonym_feature_synonym_id_seq OWNER TO gmod;

--
-- Name: feature_synonym_feature_synonym_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE feature_synonym_feature_synonym_id_seq OWNED BY feature_synonym.feature_synonym_id;


--
-- Name: feature_synonym_feature_synonym_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_synonym_feature_synonym_id_seq', 1, false);


--
-- Name: feature_uniquename_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE feature_uniquename_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.feature_uniquename_seq OWNER TO gmod;

--
-- Name: feature_uniquename_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('feature_uniquename_seq', 1, false);


--
-- Name: featureloc; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featureloc (
    featureloc_id integer NOT NULL,
    feature_id integer NOT NULL,
    srcfeature_id integer,
    fmin integer,
    is_fmin_partial boolean DEFAULT false NOT NULL,
    fmax integer,
    is_fmax_partial boolean DEFAULT false NOT NULL,
    strand smallint,
    phase integer,
    residue_info text,
    locgroup integer DEFAULT 0 NOT NULL,
    rank integer DEFAULT 0 NOT NULL,
    CONSTRAINT featureloc_c2 CHECK ((fmin <= fmax))
);


ALTER TABLE public.featureloc OWNER TO gmod;

--
-- Name: TABLE featureloc; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE featureloc IS 'The location of a feature relative to another feature. Important: interbase coordinates are used. This is vital as it allows us to represent zero-length features e.g. splice sites, insertion points without an awkward fuzzy system. Features typically have exactly ONE location, but this need not be the case. Some features may not be localized (e.g. a gene that has been characterized genetically but no sequence or molecular information is available). Note on multiple locations: Each feature can have 0 or more locations. Multiple locations do NOT indicate non-contiguous locations (if a feature such as a transcript has a non-contiguous location, then the subfeatures such as exons should always be manifested). Instead, multiple featurelocs for a feature designate alternate locations or grouped locations; for instance, a feature designating a blast hit or hsp will have two locations, one on the query feature, one on the subject feature. Features representing sequence variation could have alternate locations instantiated on a feature on the mutant strain. The column:rank is used to differentiate these different locations. Reflexive locations should never be stored - this is for -proper- (i.e. non-self) locations only; nothing should be located relative to itself.';


--
-- Name: COLUMN featureloc.feature_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.feature_id IS 'The feature that is being located. Any feature can have zero or more featurelocs.';


--
-- Name: COLUMN featureloc.srcfeature_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.srcfeature_id IS 'The source feature which this location is relative to. Every location is relative to another feature (however, this column is nullable, because the srcfeature may not be known). All locations are -proper- that is, nothing should be located relative to itself. No cycles are allowed in the featureloc graph.';


--
-- Name: COLUMN featureloc.fmin; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.fmin IS 'The leftmost/minimal boundary in the linear range represented by the featureloc. Sometimes (e.g. in Bioperl) this is called -start- although this is confusing because it does not necessarily represent the 5-prime coordinate. Important: This is space-based (interbase) coordinates, counting from zero. To convert this to the leftmost position in a base-oriented system (eg GFF, Bioperl), add 1 to fmin.';


--
-- Name: COLUMN featureloc.is_fmin_partial; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.is_fmin_partial IS 'This is typically false, but may be true if the value for column:fmin is inaccurate or the leftmost part of the range is unknown/unbounded.';


--
-- Name: COLUMN featureloc.fmax; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.fmax IS 'The rightmost/maximal boundary in the linear range represented by the featureloc. Sometimes (e.g. in bioperl) this is called -end- although this is confusing because it does not necessarily represent the 3-prime coordinate. Important: This is space-based (interbase) coordinates, counting from zero. No conversion is required to go from fmax to the rightmost coordinate in a base-oriented system that counts from 1 (e.g. GFF, Bioperl).';


--
-- Name: COLUMN featureloc.is_fmax_partial; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.is_fmax_partial IS 'This is typically false, but may be true if the value for column:fmax is inaccurate or the rightmost part of the range is unknown/unbounded.';


--
-- Name: COLUMN featureloc.strand; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.strand IS 'The orientation/directionality of the location. Should be 0, -1 or +1.';


--
-- Name: COLUMN featureloc.phase; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.phase IS 'Phase of translation with respect to srcfeature_id. Values are 0, 1, 2. It may not be possible to manifest this column for some features such as exons, because the phase is dependant on the spliceform (the same exon can appear in multiple spliceforms). This column is mostly useful for predicted exons and CDSs.';


--
-- Name: COLUMN featureloc.residue_info; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.residue_info IS 'Alternative residues, when these differ from feature.residues. For instance, a SNP feature located on a wild and mutant protein would have different alternative residues. for alignment/similarity features, the alternative residues is used to represent the alignment string (CIGAR format). Note on variation features; even if we do not want to instantiate a mutant chromosome/contig feature, we can still represent a SNP etc with 2 locations, one (rank 0) on the genome, the other (rank 1) would have most fields null, except for alternative residues.';


--
-- Name: COLUMN featureloc.locgroup; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.locgroup IS 'This is used to manifest redundant, derivable extra locations for a feature. The default locgroup=0 is used for the DIRECT location of a feature. Important: most Chado users may never use featurelocs WITH logroup > 0. Transitively derived locations are indicated with locgroup > 0. For example, the position of an exon on a BAC and in global chromosome coordinates. This column is used to differentiate these groupings of locations. The default locgroup 0 is used for the main or primary location, from which the others can be derived via coordinate transformations. Another example of redundant locations is storing ORF coordinates relative to both transcript and genome. Redundant locations open the possibility of the database getting into inconsistent states; this schema gives us the flexibility of both warehouse instantiations with redundant locations (easier for querying) and management instantiations with no redundant locations. An example of using both locgroup and rank: imagine a feature indicating a conserved region between the chromosomes of two different species. We may want to keep redundant locations on both contigs and chromosomes. We would thus have 4 locations for the single conserved region feature - two distinct locgroups (contig level and chromosome level) and two distinct ranks (for the two species).';


--
-- Name: COLUMN featureloc.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureloc.rank IS 'Used when a feature has >1 location, otherwise the default rank 0 is used. Some features (e.g. blast hits and HSPs) have two locations - one on the query and one on the subject. Rank is used to differentiate these. Rank=0 is always used for the query, Rank=1 for the subject. For multiple alignments, assignment of rank is arbitrary. Rank is also used for sequence_variant features, such as SNPs. Rank=0 indicates the wildtype (or baseline) feature, Rank=1 indicates the mutant (or compared) feature.';


--
-- Name: featureloc_featureloc_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featureloc_featureloc_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featureloc_featureloc_id_seq OWNER TO gmod;

--
-- Name: featureloc_featureloc_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featureloc_featureloc_id_seq OWNED BY featureloc.featureloc_id;


--
-- Name: featureloc_featureloc_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featureloc_featureloc_id_seq', 1, false);


--
-- Name: featureloc_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featureloc_pub (
    featureloc_pub_id integer NOT NULL,
    featureloc_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.featureloc_pub OWNER TO gmod;

--
-- Name: TABLE featureloc_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE featureloc_pub IS 'Provenance of featureloc. Linking table between featurelocs and publications that mention them.';


--
-- Name: featureloc_pub_featureloc_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featureloc_pub_featureloc_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featureloc_pub_featureloc_pub_id_seq OWNER TO gmod;

--
-- Name: featureloc_pub_featureloc_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featureloc_pub_featureloc_pub_id_seq OWNED BY featureloc_pub.featureloc_pub_id;


--
-- Name: featureloc_pub_featureloc_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featureloc_pub_featureloc_pub_id_seq', 1, false);


--
-- Name: featuremap; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featuremap (
    featuremap_id integer NOT NULL,
    name character varying(255),
    description text,
    unittype_id integer
);


ALTER TABLE public.featuremap OWNER TO gmod;

--
-- Name: featuremap_featuremap_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featuremap_featuremap_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featuremap_featuremap_id_seq OWNER TO gmod;

--
-- Name: featuremap_featuremap_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featuremap_featuremap_id_seq OWNED BY featuremap.featuremap_id;


--
-- Name: featuremap_featuremap_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featuremap_featuremap_id_seq', 1, false);


--
-- Name: featuremap_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featuremap_pub (
    featuremap_pub_id integer NOT NULL,
    featuremap_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.featuremap_pub OWNER TO gmod;

--
-- Name: featuremap_pub_featuremap_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featuremap_pub_featuremap_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featuremap_pub_featuremap_pub_id_seq OWNER TO gmod;

--
-- Name: featuremap_pub_featuremap_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featuremap_pub_featuremap_pub_id_seq OWNED BY featuremap_pub.featuremap_pub_id;


--
-- Name: featuremap_pub_featuremap_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featuremap_pub_featuremap_pub_id_seq', 1, false);


--
-- Name: featurepos; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featurepos (
    featurepos_id integer NOT NULL,
    featuremap_id integer NOT NULL,
    feature_id integer NOT NULL,
    map_feature_id integer NOT NULL,
    mappos double precision NOT NULL
);


ALTER TABLE public.featurepos OWNER TO gmod;

--
-- Name: COLUMN featurepos.map_feature_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featurepos.map_feature_id IS 'map_feature_id links to the feature (map) upon which the feature is being localized.';


--
-- Name: featurepos_featuremap_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featurepos_featuremap_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featurepos_featuremap_id_seq OWNER TO gmod;

--
-- Name: featurepos_featuremap_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featurepos_featuremap_id_seq OWNED BY featurepos.featuremap_id;


--
-- Name: featurepos_featuremap_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featurepos_featuremap_id_seq', 1, false);


--
-- Name: featurepos_featurepos_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featurepos_featurepos_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featurepos_featurepos_id_seq OWNER TO gmod;

--
-- Name: featurepos_featurepos_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featurepos_featurepos_id_seq OWNED BY featurepos.featurepos_id;


--
-- Name: featurepos_featurepos_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featurepos_featurepos_id_seq', 1, false);


--
-- Name: featureprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featureprop (
    featureprop_id integer NOT NULL,
    feature_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.featureprop OWNER TO gmod;

--
-- Name: TABLE featureprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE featureprop IS 'A feature can have any number of slot-value property tags attached to it. This is an alternative to hardcoding a list of columns in the relational schema, and is completely extensible.';


--
-- Name: COLUMN featureprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureprop.type_id IS 'The name of the property/slot is a cvterm. The meaning of the property is defined in that cvterm. Certain property types will only apply to certain feature types (e.g. the anticodon property will only apply to tRNA features) ; the types here come from the sequence feature property ontology.';


--
-- Name: COLUMN featureprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureprop.value IS 'The value of the property, represented as text. Numeric values are converted to their text representation. This is less efficient than using native database types, but is easier to query.';


--
-- Name: COLUMN featureprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featureprop.rank IS 'Property-Value ordering. Any feature can have multiple values for any particular property type - these are ordered in a list using rank, counting from zero. For properties that are single-valued rather than multi-valued, the default 0 value should be used';


--
-- Name: featureprop_featureprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featureprop_featureprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featureprop_featureprop_id_seq OWNER TO gmod;

--
-- Name: featureprop_featureprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featureprop_featureprop_id_seq OWNED BY featureprop.featureprop_id;


--
-- Name: featureprop_featureprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featureprop_featureprop_id_seq', 1, false);


--
-- Name: featureprop_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featureprop_pub (
    featureprop_pub_id integer NOT NULL,
    featureprop_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.featureprop_pub OWNER TO gmod;

--
-- Name: TABLE featureprop_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE featureprop_pub IS 'Provenance. Any featureprop assignment can optionally be supported by a publication.';


--
-- Name: featureprop_pub_featureprop_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featureprop_pub_featureprop_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featureprop_pub_featureprop_pub_id_seq OWNER TO gmod;

--
-- Name: featureprop_pub_featureprop_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featureprop_pub_featureprop_pub_id_seq OWNED BY featureprop_pub.featureprop_pub_id;


--
-- Name: featureprop_pub_featureprop_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featureprop_pub_featureprop_pub_id_seq', 1, false);


--
-- Name: featurerange; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE featurerange (
    featurerange_id integer NOT NULL,
    featuremap_id integer NOT NULL,
    feature_id integer NOT NULL,
    leftstartf_id integer NOT NULL,
    leftendf_id integer,
    rightstartf_id integer,
    rightendf_id integer NOT NULL,
    rangestr character varying(255)
);


ALTER TABLE public.featurerange OWNER TO gmod;

--
-- Name: TABLE featurerange; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE featurerange IS 'In cases where the start and end of a mapped feature is a range, leftendf and rightstartf are populated. leftstartf_id, leftendf_id, rightstartf_id, rightendf_id are the ids of features with respect to which the feature is being mapped. These may be cytological bands.';


--
-- Name: COLUMN featurerange.featuremap_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN featurerange.featuremap_id IS 'featuremap_id is the id of the feature being mapped.';


--
-- Name: featurerange_featurerange_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE featurerange_featurerange_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.featurerange_featurerange_id_seq OWNER TO gmod;

--
-- Name: featurerange_featurerange_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE featurerange_featurerange_id_seq OWNED BY featurerange.featurerange_id;


--
-- Name: featurerange_featurerange_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('featurerange_featurerange_id_seq', 1, false);


--
-- Name: genotype; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE genotype (
    genotype_id integer NOT NULL,
    name text,
    uniquename text NOT NULL,
    description character varying(255)
);


ALTER TABLE public.genotype OWNER TO gmod;

--
-- Name: TABLE genotype; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE genotype IS 'Genetic context. A genotype is defined by a collection of features, mutations, balancers, deficiencies, haplotype blocks, or engineered constructs.';


--
-- Name: COLUMN genotype.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN genotype.name IS 'Optional alternative name for a genotype, for display purposes.';


--
-- Name: COLUMN genotype.uniquename; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN genotype.uniquename IS 'The unique name for a genotype; typically derived from the features making up the genotype.';


--
-- Name: genotype_genotype_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE genotype_genotype_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.genotype_genotype_id_seq OWNER TO gmod;

--
-- Name: genotype_genotype_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE genotype_genotype_id_seq OWNED BY genotype.genotype_id;


--
-- Name: genotype_genotype_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('genotype_genotype_id_seq', 40, true);


--
-- Name: nd_experiment_phenotype; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_phenotype (
    nd_experiment_phenotype_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    phenotype_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_phenotype OWNER TO gmod;

--
-- Name: TABLE nd_experiment_phenotype; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_phenotype IS 'Linking table: experiments to the phenotypes they produce. There is a one-to-one relationship between an experiment and a phenotype since each phenotype record should point to one experiment. Add a new experiment_id for each phenotype record.';


--
-- Name: nd_experiment_phenotypeprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_phenotypeprop (
    nd_experiment_phenotypeprop_id integer NOT NULL,
    nd_experiment_phenotype_id integer NOT NULL,
    type_id integer NOT NULL,
    value character varying(255),
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.nd_experiment_phenotypeprop OWNER TO gmod;

--
-- Name: TABLE nd_experiment_phenotypeprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_phenotypeprop IS 'Property/value associations for experiment_phenotypes. This table can store the properties such as observation values.';


--
-- Name: COLUMN nd_experiment_phenotypeprop.nd_experiment_phenotype_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_phenotypeprop.nd_experiment_phenotype_id IS 'The experiment_phenotype to which the property applies.';


--
-- Name: COLUMN nd_experiment_phenotypeprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_phenotypeprop.type_id IS 'The name of the property as a reference to a controlled vocabulary term.';


--
-- Name: COLUMN nd_experiment_phenotypeprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_phenotypeprop.value IS 'The value of the property.';


--
-- Name: COLUMN nd_experiment_phenotypeprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_phenotypeprop.rank IS 'The rank of the property value, if the property has an array of values.';


--
-- Name: nd_experiment_project; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_project (
    nd_experiment_project_id integer NOT NULL,
    project_id integer NOT NULL,
    nd_experiment_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_project OWNER TO gmod;

--
-- Name: nd_experiment_stock; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_stock (
    nd_experiment_stock_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    stock_id integer NOT NULL,
    type_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_stock OWNER TO gmod;

--
-- Name: TABLE nd_experiment_stock; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_stock IS 'Part of a stock or a clone of a stock that is used in an experiment';


--
-- Name: COLUMN nd_experiment_stock.stock_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_stock.stock_id IS 'stock used in the extraction or the corresponding stock for the clone';


--
-- Name: nd_experiment_stockprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_stockprop (
    nd_experiment_stockprop_id integer NOT NULL,
    nd_experiment_stock_id integer NOT NULL,
    type_id integer NOT NULL,
    value character varying(255),
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.nd_experiment_stockprop OWNER TO gmod;

--
-- Name: TABLE nd_experiment_stockprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_stockprop IS 'Property/value associations for experiment_stocks. This table can store the properties such as treatment';


--
-- Name: COLUMN nd_experiment_stockprop.nd_experiment_stock_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_stockprop.nd_experiment_stock_id IS 'The experiment_stock to which the property applies.';


--
-- Name: COLUMN nd_experiment_stockprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_stockprop.type_id IS 'The name of the property as a reference to a controlled vocabulary term.';


--
-- Name: COLUMN nd_experiment_stockprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_stockprop.value IS 'The value of the property.';


--
-- Name: COLUMN nd_experiment_stockprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_experiment_stockprop.rank IS 'The rank of the property value, if the property has an array of values.';


--
-- Name: phenotype; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phenotype (
    phenotype_id integer NOT NULL,
    uniquename text NOT NULL,
    observable_id integer,
    attr_id integer,
    value text,
    cvalue_id integer,
    assay_id integer
);


ALTER TABLE public.phenotype OWNER TO gmod;

--
-- Name: TABLE phenotype; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phenotype IS 'A phenotypic statement, or a single atomic phenotypic observation, is a controlled sentence describing observable effects of non-wild type function. E.g. Obs=eye, attribute=color, cvalue=red.';


--
-- Name: COLUMN phenotype.observable_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phenotype.observable_id IS 'The entity: e.g. anatomy_part, biological_process.';


--
-- Name: COLUMN phenotype.attr_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phenotype.attr_id IS 'Phenotypic attribute (quality, property, attribute, character) - drawn from PATO.';


--
-- Name: COLUMN phenotype.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phenotype.value IS 'Value of attribute - unconstrained free text. Used only if cvalue_id is not appropriate.';


--
-- Name: COLUMN phenotype.cvalue_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phenotype.cvalue_id IS 'Phenotype attribute value (state).';


--
-- Name: COLUMN phenotype.assay_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phenotype.assay_id IS 'Evidence type.';


--
-- Name: project; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE project (
    project_id integer NOT NULL,
    name character varying(255) NOT NULL,
    description character varying(255) NOT NULL
);


ALTER TABLE public.project OWNER TO gmod;

--
-- Name: stock; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock (
    stock_id integer NOT NULL,
    dbxref_id integer,
    organism_id integer,
    name character varying(255),
    uniquename text NOT NULL,
    description text,
    type_id integer NOT NULL,
    is_obsolete boolean DEFAULT false NOT NULL
);


ALTER TABLE public.stock OWNER TO gmod;

--
-- Name: TABLE stock; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stock IS 'Any stock can be globally identified by the combination of organism, uniquename and stock type. A stock is the physical entities, either living or preserved, held by collections. Stocks belong to a collection; they have IDs, type, organism, description and may have a genotype.';


--
-- Name: COLUMN stock.dbxref_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock.dbxref_id IS 'The dbxref_id is an optional primary stable identifier for this stock. Secondary indentifiers and external dbxrefs go in table: stock_dbxref.';


--
-- Name: COLUMN stock.organism_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock.organism_id IS 'The organism_id is the organism to which the stock belongs. This column should only be left blank if the organism cannot be determined.';


--
-- Name: COLUMN stock.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock.name IS 'The name is a human-readable local name for a stock.';


--
-- Name: COLUMN stock.description; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock.description IS 'The description is the genetic description provided in the stock list.';


--
-- Name: COLUMN stock.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock.type_id IS 'The type_id foreign key links to a controlled vocabulary of stock types. The would include living stock, genomic DNA, preserved specimen. Secondary cvterms for stocks would go in stock_cvterm.';


--
-- Name: stock_genotype; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock_genotype (
    stock_genotype_id integer NOT NULL,
    stock_id integer NOT NULL,
    genotype_id integer NOT NULL
);


ALTER TABLE public.stock_genotype OWNER TO gmod;

--
-- Name: TABLE stock_genotype; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stock_genotype IS 'Simple table linking a stock to a genotype. Features with genotypes can be linked to stocks thru feature_genotype -> genotype -> stock_genotype -> stock.';


--
-- Name: stockprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stockprop (
    stockprop_id integer NOT NULL,
    stock_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.stockprop OWNER TO gmod;

--
-- Name: TABLE stockprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stockprop IS 'A stock can have any number of slot-value property tags attached to it. This is an alternative to hardcoding a list of columns in the relational schema, and is completely extensible. There is a unique constraint, stockprop_c1, for the combination of stock_id, rank, and type_id. Multivalued property-value pairs must be differentiated by rank.';


--
-- Name: gwas; Type: VIEW; Schema: public; Owner: gmod
--

CREATE VIEW gwas AS
    SELECT ndep.nd_experiment_id, p.project_id, g.uniquename AS genotype, sp.type_id AS sp_type_id, sp.value AS sp_value_id, treatment.str AS treatment_str, treatment.val AS treatment_val, pheno.str AS phenotype_str, pheno.val AS phenotype_val FROM ((((((((project p JOIN nd_experiment_project ndep USING (project_id)) JOIN nd_experiment_stock ndes USING (nd_experiment_id)) JOIN stock s USING (stock_id)) JOIN stockprop sp USING (stock_id)) JOIN stock_genotype sg USING (stock_id)) JOIN genotype g USING (genotype_id)) JOIN (SELECT ndes.nd_experiment_stock_id, array_agg((SELECT cvterm.name FROM cvterm WHERE (cvterm.cvterm_id = ndes.type_id))) AS str, array_agg(ndes.value) AS val FROM nd_experiment_stockprop ndes WHERE (EXISTS (SELECT 1 FROM cvtermpath WHERE ((((cvtermpath.type_id = get_cvterm_id('relationship'::character varying, 'is_a'::character varying)) AND (cvtermpath.subject_id = ndes.type_id)) AND (cvtermpath.object_id = get_cvterm_id('diversity_experiment'::character varying, 'treatment'::character varying))) AND (cvtermpath.cv_id = (SELECT cv.cv_id FROM cv WHERE ((cv.name)::text = 'diversity_experiment'::text)))))) GROUP BY ndes.nd_experiment_stock_id) treatment USING (nd_experiment_stock_id)) JOIN (SELECT ndept.nd_experiment_id, array_agg((SELECT phenotype.uniquename FROM phenotype WHERE (phenotype.phenotype_id = ndept.phenotype_id))) AS str, array_agg(ndeptp.value) AS val FROM (nd_experiment_phenotype ndept JOIN nd_experiment_phenotypeprop ndeptp USING (nd_experiment_phenotype_id)) GROUP BY ndept.nd_experiment_id) pheno USING (nd_experiment_id)) ORDER BY ndep.nd_experiment_id;


ALTER TABLE public.gwas OWNER TO gmod;

--
-- Name: library; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE library (
    library_id integer NOT NULL,
    organism_id integer NOT NULL,
    name character varying(255),
    uniquename text NOT NULL,
    type_id integer NOT NULL,
    is_obsolete integer DEFAULT 0 NOT NULL,
    timeaccessioned timestamp without time zone DEFAULT now() NOT NULL,
    timelastmodified timestamp without time zone DEFAULT now() NOT NULL
);


ALTER TABLE public.library OWNER TO gmod;

--
-- Name: COLUMN library.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN library.type_id IS 'The type_id foreign key links to a controlled vocabulary of library types. Examples of this would be: "cDNA_library" or "genomic_library"';


--
-- Name: library_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE library_cvterm (
    library_cvterm_id integer NOT NULL,
    library_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.library_cvterm OWNER TO gmod;

--
-- Name: TABLE library_cvterm; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE library_cvterm IS 'The table library_cvterm links a library to controlled vocabularies which describe the library. For instance, there might be a link to the anatomy cv for "head" or "testes" for a head or testes library.';


--
-- Name: library_cvterm_library_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE library_cvterm_library_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.library_cvterm_library_cvterm_id_seq OWNER TO gmod;

--
-- Name: library_cvterm_library_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE library_cvterm_library_cvterm_id_seq OWNED BY library_cvterm.library_cvterm_id;


--
-- Name: library_cvterm_library_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('library_cvterm_library_cvterm_id_seq', 1, false);


--
-- Name: library_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE library_dbxref (
    library_dbxref_id integer NOT NULL,
    library_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    is_current boolean DEFAULT true NOT NULL
);


ALTER TABLE public.library_dbxref OWNER TO gmod;

--
-- Name: library_dbxref_library_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE library_dbxref_library_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.library_dbxref_library_dbxref_id_seq OWNER TO gmod;

--
-- Name: library_dbxref_library_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE library_dbxref_library_dbxref_id_seq OWNED BY library_dbxref.library_dbxref_id;


--
-- Name: library_dbxref_library_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('library_dbxref_library_dbxref_id_seq', 1, false);


--
-- Name: library_feature; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE library_feature (
    library_feature_id integer NOT NULL,
    library_id integer NOT NULL,
    feature_id integer NOT NULL
);


ALTER TABLE public.library_feature OWNER TO gmod;

--
-- Name: TABLE library_feature; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE library_feature IS 'library_feature links a library to the clones which are contained in the library. Examples of such linked features might be "cDNA_clone" or "genomic_clone".';


--
-- Name: library_feature_library_feature_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE library_feature_library_feature_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.library_feature_library_feature_id_seq OWNER TO gmod;

--
-- Name: library_feature_library_feature_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE library_feature_library_feature_id_seq OWNED BY library_feature.library_feature_id;


--
-- Name: library_feature_library_feature_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('library_feature_library_feature_id_seq', 1, false);


--
-- Name: library_library_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE library_library_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.library_library_id_seq OWNER TO gmod;

--
-- Name: library_library_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE library_library_id_seq OWNED BY library.library_id;


--
-- Name: library_library_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('library_library_id_seq', 1, false);


--
-- Name: library_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE library_pub (
    library_pub_id integer NOT NULL,
    library_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.library_pub OWNER TO gmod;

--
-- Name: library_pub_library_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE library_pub_library_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.library_pub_library_pub_id_seq OWNER TO gmod;

--
-- Name: library_pub_library_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE library_pub_library_pub_id_seq OWNED BY library_pub.library_pub_id;


--
-- Name: library_pub_library_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('library_pub_library_pub_id_seq', 1, false);


--
-- Name: library_synonym; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE library_synonym (
    library_synonym_id integer NOT NULL,
    synonym_id integer NOT NULL,
    library_id integer NOT NULL,
    pub_id integer NOT NULL,
    is_current boolean DEFAULT true NOT NULL,
    is_internal boolean DEFAULT false NOT NULL
);


ALTER TABLE public.library_synonym OWNER TO gmod;

--
-- Name: COLUMN library_synonym.pub_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN library_synonym.pub_id IS 'The pub_id link is for relating the usage of a given synonym to the publication in which it was used.';


--
-- Name: COLUMN library_synonym.is_current; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN library_synonym.is_current IS 'The is_current bit indicates whether the linked synonym is the current -official- symbol for the linked library.';


--
-- Name: COLUMN library_synonym.is_internal; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN library_synonym.is_internal IS 'Typically a synonym exists so that somebody querying the database with an obsolete name can find the object they are looking for under its current name. If the synonym has been used publicly and deliberately (e.g. in a paper), it my also be listed in reports as a synonym. If the synonym was not used deliberately (e.g., there was a typo which went public), then the is_internal bit may be set to "true" so that it is known that the synonym is "internal" and should be queryable but should not be listed in reports as a valid synonym.';


--
-- Name: library_synonym_library_synonym_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE library_synonym_library_synonym_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.library_synonym_library_synonym_id_seq OWNER TO gmod;

--
-- Name: library_synonym_library_synonym_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE library_synonym_library_synonym_id_seq OWNED BY library_synonym.library_synonym_id;


--
-- Name: library_synonym_library_synonym_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('library_synonym_library_synonym_id_seq', 1, false);


--
-- Name: libraryprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE libraryprop (
    libraryprop_id integer NOT NULL,
    library_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.libraryprop OWNER TO gmod;

--
-- Name: libraryprop_libraryprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE libraryprop_libraryprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.libraryprop_libraryprop_id_seq OWNER TO gmod;

--
-- Name: libraryprop_libraryprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE libraryprop_libraryprop_id_seq OWNED BY libraryprop.libraryprop_id;


--
-- Name: libraryprop_libraryprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('libraryprop_libraryprop_id_seq', 1, false);


--
-- Name: libraryprop_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE libraryprop_pub (
    libraryprop_pub_id integer NOT NULL,
    libraryprop_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.libraryprop_pub OWNER TO gmod;

--
-- Name: libraryprop_pub_libraryprop_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE libraryprop_pub_libraryprop_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.libraryprop_pub_libraryprop_pub_id_seq OWNER TO gmod;

--
-- Name: libraryprop_pub_libraryprop_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE libraryprop_pub_libraryprop_pub_id_seq OWNED BY libraryprop_pub.libraryprop_pub_id;


--
-- Name: libraryprop_pub_libraryprop_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('libraryprop_pub_libraryprop_pub_id_seq', 1, false);


--
-- Name: magedocumentation; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE magedocumentation (
    magedocumentation_id integer NOT NULL,
    mageml_id integer NOT NULL,
    tableinfo_id integer NOT NULL,
    row_id integer NOT NULL,
    mageidentifier text NOT NULL
);


ALTER TABLE public.magedocumentation OWNER TO gmod;

--
-- Name: magedocumentation_magedocumentation_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE magedocumentation_magedocumentation_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.magedocumentation_magedocumentation_id_seq OWNER TO gmod;

--
-- Name: magedocumentation_magedocumentation_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE magedocumentation_magedocumentation_id_seq OWNED BY magedocumentation.magedocumentation_id;


--
-- Name: magedocumentation_magedocumentation_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('magedocumentation_magedocumentation_id_seq', 1, false);


--
-- Name: mageml; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE mageml (
    mageml_id integer NOT NULL,
    mage_package text NOT NULL,
    mage_ml text NOT NULL
);


ALTER TABLE public.mageml OWNER TO gmod;

--
-- Name: TABLE mageml; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE mageml IS 'This table is for storing extra bits of MAGEml in a denormalized form. More normalization would require many more tables.';


--
-- Name: mageml_mageml_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE mageml_mageml_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.mageml_mageml_id_seq OWNER TO gmod;

--
-- Name: mageml_mageml_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE mageml_mageml_id_seq OWNED BY mageml.mageml_id;


--
-- Name: mageml_mageml_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('mageml_mageml_id_seq', 1, false);


--
-- Name: materialized_view; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE materialized_view (
    materialized_view_id integer NOT NULL,
    last_update timestamp without time zone,
    refresh_time integer,
    name character varying(64),
    mv_schema character varying(64),
    mv_table character varying(128),
    mv_specs text,
    indexed text,
    query text,
    special_index text
);


ALTER TABLE public.materialized_view OWNER TO gmod;

--
-- Name: materialized_view_materialized_view_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE materialized_view_materialized_view_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.materialized_view_materialized_view_id_seq OWNER TO gmod;

--
-- Name: materialized_view_materialized_view_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE materialized_view_materialized_view_id_seq OWNED BY materialized_view.materialized_view_id;


--
-- Name: materialized_view_materialized_view_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('materialized_view_materialized_view_id_seq', 1, false);


--
-- Name: nd_experiment; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment (
    nd_experiment_id integer NOT NULL,
    nd_geolocation_id integer NOT NULL,
    type_id integer NOT NULL
);


ALTER TABLE public.nd_experiment OWNER TO gmod;

--
-- Name: nd_experiment_contact; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_contact (
    nd_experiment_contact_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    contact_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_contact OWNER TO gmod;

--
-- Name: nd_experiment_contact_nd_experiment_contact_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_contact_nd_experiment_contact_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_contact_nd_experiment_contact_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_contact_nd_experiment_contact_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_contact_nd_experiment_contact_id_seq OWNED BY nd_experiment_contact.nd_experiment_contact_id;


--
-- Name: nd_experiment_contact_nd_experiment_contact_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_contact_nd_experiment_contact_id_seq', 1, false);


--
-- Name: nd_experiment_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_dbxref (
    nd_experiment_dbxref_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    dbxref_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_dbxref OWNER TO gmod;

--
-- Name: TABLE nd_experiment_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_dbxref IS 'Cross-reference experiment to accessions, images, etc';


--
-- Name: nd_experiment_dbxref_nd_experiment_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_dbxref_nd_experiment_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_dbxref_nd_experiment_dbxref_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_dbxref_nd_experiment_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_dbxref_nd_experiment_dbxref_id_seq OWNED BY nd_experiment_dbxref.nd_experiment_dbxref_id;


--
-- Name: nd_experiment_dbxref_nd_experiment_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_dbxref_nd_experiment_dbxref_id_seq', 1, false);


--
-- Name: nd_experiment_genotype; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_genotype (
    nd_experiment_genotype_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    genotype_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_genotype OWNER TO gmod;

--
-- Name: TABLE nd_experiment_genotype; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_genotype IS 'Linking table: experiments to the genotypes they produce. There is a one-to-one relationship between an experiment and a genotype since each genotype record should point to one experiment. Add a new experiment_id for each genotype record.';


--
-- Name: nd_experiment_genotype_nd_experiment_genotype_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_genotype_nd_experiment_genotype_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_genotype_nd_experiment_genotype_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_genotype_nd_experiment_genotype_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_genotype_nd_experiment_genotype_id_seq OWNED BY nd_experiment_genotype.nd_experiment_genotype_id;


--
-- Name: nd_experiment_genotype_nd_experiment_genotype_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_genotype_nd_experiment_genotype_id_seq', 1, false);


--
-- Name: nd_experiment_nd_experiment_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_nd_experiment_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_nd_experiment_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_nd_experiment_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_nd_experiment_id_seq OWNED BY nd_experiment.nd_experiment_id;


--
-- Name: nd_experiment_nd_experiment_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_nd_experiment_id_seq', 240, true);


--
-- Name: nd_experiment_phenotype_nd_experiment_phenotype_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_phenotype_nd_experiment_phenotype_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_phenotype_nd_experiment_phenotype_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_phenotype_nd_experiment_phenotype_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_phenotype_nd_experiment_phenotype_id_seq OWNED BY nd_experiment_phenotype.nd_experiment_phenotype_id;


--
-- Name: nd_experiment_phenotype_nd_experiment_phenotype_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_phenotype_nd_experiment_phenotype_id_seq', 22320, true);


--
-- Name: nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq OWNED BY nd_experiment_phenotypeprop.nd_experiment_phenotypeprop_id;


--
-- Name: nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq', 22320, true);


--
-- Name: nd_experiment_project_nd_experiment_project_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_project_nd_experiment_project_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_project_nd_experiment_project_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_project_nd_experiment_project_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_project_nd_experiment_project_id_seq OWNED BY nd_experiment_project.nd_experiment_project_id;


--
-- Name: nd_experiment_project_nd_experiment_project_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_project_nd_experiment_project_id_seq', 240, true);


--
-- Name: nd_experiment_protocol; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_protocol (
    nd_experiment_protocol_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    nd_protocol_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_protocol OWNER TO gmod;

--
-- Name: TABLE nd_experiment_protocol; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_protocol IS 'Linking table: experiments to the protocols they involve.';


--
-- Name: nd_experiment_protocol_nd_experiment_protocol_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_protocol_nd_experiment_protocol_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_protocol_nd_experiment_protocol_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_protocol_nd_experiment_protocol_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_protocol_nd_experiment_protocol_id_seq OWNED BY nd_experiment_protocol.nd_experiment_protocol_id;


--
-- Name: nd_experiment_protocol_nd_experiment_protocol_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_protocol_nd_experiment_protocol_id_seq', 1, false);


--
-- Name: nd_experiment_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_pub (
    nd_experiment_pub_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_pub OWNER TO gmod;

--
-- Name: TABLE nd_experiment_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_pub IS 'Linking nd_experiment(s) to publication(s)';


--
-- Name: nd_experiment_pub_nd_experiment_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_pub_nd_experiment_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_pub_nd_experiment_pub_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_pub_nd_experiment_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_pub_nd_experiment_pub_id_seq OWNED BY nd_experiment_pub.nd_experiment_pub_id;


--
-- Name: nd_experiment_pub_nd_experiment_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_pub_nd_experiment_pub_id_seq', 1, false);


--
-- Name: nd_experiment_stock_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experiment_stock_dbxref (
    nd_experiment_stock_dbxref_id integer NOT NULL,
    nd_experiment_stock_id integer NOT NULL,
    dbxref_id integer NOT NULL
);


ALTER TABLE public.nd_experiment_stock_dbxref OWNER TO gmod;

--
-- Name: TABLE nd_experiment_stock_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_experiment_stock_dbxref IS 'Cross-reference experiment_stock to accessions, images, etc';


--
-- Name: nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq OWNED BY nd_experiment_stock_dbxref.nd_experiment_stock_dbxref_id;


--
-- Name: nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq', 1, false);


--
-- Name: nd_experiment_stock_nd_experiment_stock_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_stock_nd_experiment_stock_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_stock_nd_experiment_stock_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_stock_nd_experiment_stock_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_stock_nd_experiment_stock_id_seq OWNED BY nd_experiment_stock.nd_experiment_stock_id;


--
-- Name: nd_experiment_stock_nd_experiment_stock_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_stock_nd_experiment_stock_id_seq', 240, true);


--
-- Name: nd_experiment_stockprop_nd_experiment_stockprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experiment_stockprop_nd_experiment_stockprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experiment_stockprop_nd_experiment_stockprop_id_seq OWNER TO gmod;

--
-- Name: nd_experiment_stockprop_nd_experiment_stockprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experiment_stockprop_nd_experiment_stockprop_id_seq OWNED BY nd_experiment_stockprop.nd_experiment_stockprop_id;


--
-- Name: nd_experiment_stockprop_nd_experiment_stockprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experiment_stockprop_nd_experiment_stockprop_id_seq', 240, true);


--
-- Name: nd_experimentprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_experimentprop (
    nd_experimentprop_id integer NOT NULL,
    nd_experiment_id integer NOT NULL,
    type_id integer NOT NULL,
    value character varying(255) NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.nd_experimentprop OWNER TO gmod;

--
-- Name: nd_experimentprop_nd_experimentprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_experimentprop_nd_experimentprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_experimentprop_nd_experimentprop_id_seq OWNER TO gmod;

--
-- Name: nd_experimentprop_nd_experimentprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_experimentprop_nd_experimentprop_id_seq OWNED BY nd_experimentprop.nd_experimentprop_id;


--
-- Name: nd_experimentprop_nd_experimentprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_experimentprop_nd_experimentprop_id_seq', 3840, true);


--
-- Name: nd_geolocation; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_geolocation (
    nd_geolocation_id integer NOT NULL,
    description character varying(255),
    latitude real,
    longitude real,
    geodetic_datum character varying(32),
    altitude real
);


ALTER TABLE public.nd_geolocation OWNER TO gmod;

--
-- Name: TABLE nd_geolocation; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_geolocation IS 'The geo-referencable location of the stock. NOTE: This entity is subject to change as a more general and possibly more OpenGIS-compliant geolocation module may be introduced into Chado.';


--
-- Name: COLUMN nd_geolocation.description; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocation.description IS 'A textual representation of the location, if this is the original georeference. Optional if the original georeference is available in lat/long coordinates.';


--
-- Name: COLUMN nd_geolocation.latitude; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocation.latitude IS 'The decimal latitude coordinate of the georeference, using positive and negative sign to indicate N and S, respectively.';


--
-- Name: COLUMN nd_geolocation.longitude; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocation.longitude IS 'The decimal longitude coordinate of the georeference, using positive and negative sign to indicate E and W, respectively.';


--
-- Name: COLUMN nd_geolocation.geodetic_datum; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocation.geodetic_datum IS 'The geodetic system on which the geo-reference coordinates are based. For geo-references measured between 1984 and 2010, this will typically be WGS84.';


--
-- Name: COLUMN nd_geolocation.altitude; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocation.altitude IS 'The altitude (elevation) of the location in meters. If the altitude is only known as a range, this is the average, and altitude_dev will hold half of the width of the range.';


--
-- Name: nd_geolocation_nd_geolocation_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_geolocation_nd_geolocation_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_geolocation_nd_geolocation_id_seq OWNER TO gmod;

--
-- Name: nd_geolocation_nd_geolocation_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_geolocation_nd_geolocation_id_seq OWNED BY nd_geolocation.nd_geolocation_id;


--
-- Name: nd_geolocation_nd_geolocation_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_geolocation_nd_geolocation_id_seq', 3, true);


--
-- Name: nd_geolocationprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_geolocationprop (
    nd_geolocationprop_id integer NOT NULL,
    nd_geolocation_id integer NOT NULL,
    type_id integer NOT NULL,
    value character varying(250),
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.nd_geolocationprop OWNER TO gmod;

--
-- Name: TABLE nd_geolocationprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_geolocationprop IS 'Property/value associations for geolocations. This table can store the properties such as location and environment';


--
-- Name: COLUMN nd_geolocationprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocationprop.type_id IS 'The name of the property as a reference to a controlled vocabulary term.';


--
-- Name: COLUMN nd_geolocationprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocationprop.value IS 'The value of the property.';


--
-- Name: COLUMN nd_geolocationprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_geolocationprop.rank IS 'The rank of the property value, if the property has an array of values.';


--
-- Name: nd_geolocationprop_nd_geolocationprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_geolocationprop_nd_geolocationprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_geolocationprop_nd_geolocationprop_id_seq OWNER TO gmod;

--
-- Name: nd_geolocationprop_nd_geolocationprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_geolocationprop_nd_geolocationprop_id_seq OWNED BY nd_geolocationprop.nd_geolocationprop_id;


--
-- Name: nd_geolocationprop_nd_geolocationprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_geolocationprop_nd_geolocationprop_id_seq', 9, true);


--
-- Name: nd_protocol; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_protocol (
    nd_protocol_id integer NOT NULL,
    name character varying(255) NOT NULL
);


ALTER TABLE public.nd_protocol OWNER TO gmod;

--
-- Name: TABLE nd_protocol; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_protocol IS 'A protocol can be anything that is done as part of the experiment.';


--
-- Name: COLUMN nd_protocol.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_protocol.name IS 'The protocol name.';


--
-- Name: nd_protocol_nd_protocol_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_protocol_nd_protocol_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_protocol_nd_protocol_id_seq OWNER TO gmod;

--
-- Name: nd_protocol_nd_protocol_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_protocol_nd_protocol_id_seq OWNED BY nd_protocol.nd_protocol_id;


--
-- Name: nd_protocol_nd_protocol_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_protocol_nd_protocol_id_seq', 1, false);


--
-- Name: nd_protocol_reagent; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_protocol_reagent (
    nd_protocol_reagent_id integer NOT NULL,
    nd_protocol_id integer NOT NULL,
    reagent_id integer NOT NULL,
    type_id integer NOT NULL
);


ALTER TABLE public.nd_protocol_reagent OWNER TO gmod;

--
-- Name: nd_protocol_reagent_nd_protocol_reagent_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_protocol_reagent_nd_protocol_reagent_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_protocol_reagent_nd_protocol_reagent_id_seq OWNER TO gmod;

--
-- Name: nd_protocol_reagent_nd_protocol_reagent_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_protocol_reagent_nd_protocol_reagent_id_seq OWNED BY nd_protocol_reagent.nd_protocol_reagent_id;


--
-- Name: nd_protocol_reagent_nd_protocol_reagent_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_protocol_reagent_nd_protocol_reagent_id_seq', 1, false);


--
-- Name: nd_protocolprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_protocolprop (
    nd_protocolprop_id integer NOT NULL,
    nd_protocol_id integer NOT NULL,
    type_id integer NOT NULL,
    value character varying(255),
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.nd_protocolprop OWNER TO gmod;

--
-- Name: TABLE nd_protocolprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_protocolprop IS 'Property/value associations for protocol.';


--
-- Name: COLUMN nd_protocolprop.nd_protocol_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_protocolprop.nd_protocol_id IS 'The protocol to which the property applies.';


--
-- Name: COLUMN nd_protocolprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_protocolprop.type_id IS 'The name of the property as a reference to a controlled vocabulary term.';


--
-- Name: COLUMN nd_protocolprop.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_protocolprop.value IS 'The value of the property.';


--
-- Name: COLUMN nd_protocolprop.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_protocolprop.rank IS 'The rank of the property value, if the property has an array of values.';


--
-- Name: nd_protocolprop_nd_protocolprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_protocolprop_nd_protocolprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_protocolprop_nd_protocolprop_id_seq OWNER TO gmod;

--
-- Name: nd_protocolprop_nd_protocolprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_protocolprop_nd_protocolprop_id_seq OWNED BY nd_protocolprop.nd_protocolprop_id;


--
-- Name: nd_protocolprop_nd_protocolprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_protocolprop_nd_protocolprop_id_seq', 1, false);


--
-- Name: nd_reagent; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_reagent (
    nd_reagent_id integer NOT NULL,
    name character varying(80) NOT NULL,
    type_id integer NOT NULL,
    feature_id integer
);


ALTER TABLE public.nd_reagent OWNER TO gmod;

--
-- Name: TABLE nd_reagent; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_reagent IS 'A reagent such as a primer, an enzyme, an adapter oligo, a linker oligo. Reagents are used in genotyping experiments, or in any other kind of experiment.';


--
-- Name: COLUMN nd_reagent.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_reagent.name IS 'The name of the reagent. The name should be unique for a given type.';


--
-- Name: COLUMN nd_reagent.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_reagent.type_id IS 'The type of the reagent, for example linker oligomer, or forward primer.';


--
-- Name: COLUMN nd_reagent.feature_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_reagent.feature_id IS 'If the reagent is a primer, the feature that it corresponds to. More generally, the corresponding feature for any reagent that has a sequence that maps to another sequence.';


--
-- Name: nd_reagent_nd_reagent_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_reagent_nd_reagent_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_reagent_nd_reagent_id_seq OWNER TO gmod;

--
-- Name: nd_reagent_nd_reagent_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_reagent_nd_reagent_id_seq OWNED BY nd_reagent.nd_reagent_id;


--
-- Name: nd_reagent_nd_reagent_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_reagent_nd_reagent_id_seq', 1, false);


--
-- Name: nd_reagent_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_reagent_relationship (
    nd_reagent_relationship_id integer NOT NULL,
    subject_reagent_id integer NOT NULL,
    object_reagent_id integer NOT NULL,
    type_id integer NOT NULL
);


ALTER TABLE public.nd_reagent_relationship OWNER TO gmod;

--
-- Name: TABLE nd_reagent_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE nd_reagent_relationship IS 'Relationships between reagents. Some reagents form a group. i.e., they are used all together or not at all. Examples are adapter/linker/enzyme experiment reagents.';


--
-- Name: COLUMN nd_reagent_relationship.subject_reagent_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_reagent_relationship.subject_reagent_id IS 'The subject reagent in the relationship. In parent/child terminology, the subject is the child. For example, in "linkerA 3prime-overhang-linker enzymeA" linkerA is the subject, 3prime-overhand-linker is the type, and enzymeA is the object.';


--
-- Name: COLUMN nd_reagent_relationship.object_reagent_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_reagent_relationship.object_reagent_id IS 'The object reagent in the relationship. In parent/child terminology, the object is the parent. For example, in "linkerA 3prime-overhang-linker enzymeA" linkerA is the subject, 3prime-overhand-linker is the type, and enzymeA is the object.';


--
-- Name: COLUMN nd_reagent_relationship.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN nd_reagent_relationship.type_id IS 'The type (or predicate) of the relationship. For example, in "linkerA 3prime-overhang-linker enzymeA" linkerA is the subject, 3prime-overhand-linker is the type, and enzymeA is the object.';


--
-- Name: nd_reagent_relationship_nd_reagent_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_reagent_relationship_nd_reagent_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_reagent_relationship_nd_reagent_relationship_id_seq OWNER TO gmod;

--
-- Name: nd_reagent_relationship_nd_reagent_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_reagent_relationship_nd_reagent_relationship_id_seq OWNED BY nd_reagent_relationship.nd_reagent_relationship_id;


--
-- Name: nd_reagent_relationship_nd_reagent_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_reagent_relationship_nd_reagent_relationship_id_seq', 1, false);


--
-- Name: nd_reagentprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE nd_reagentprop (
    nd_reagentprop_id integer NOT NULL,
    nd_reagent_id integer NOT NULL,
    type_id integer NOT NULL,
    value character varying(255),
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.nd_reagentprop OWNER TO gmod;

--
-- Name: nd_reagentprop_nd_reagentprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE nd_reagentprop_nd_reagentprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.nd_reagentprop_nd_reagentprop_id_seq OWNER TO gmod;

--
-- Name: nd_reagentprop_nd_reagentprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE nd_reagentprop_nd_reagentprop_id_seq OWNED BY nd_reagentprop.nd_reagentprop_id;


--
-- Name: nd_reagentprop_nd_reagentprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('nd_reagentprop_nd_reagentprop_id_seq', 1, false);


--
-- Name: organism; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE organism (
    organism_id integer NOT NULL,
    abbreviation character varying(255),
    genus character varying(255) NOT NULL,
    species character varying(255) NOT NULL,
    common_name character varying(255),
    comment text
);


ALTER TABLE public.organism OWNER TO gmod;

--
-- Name: TABLE organism; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE organism IS 'The organismal taxonomic classification. Note that phylogenies are represented using the phylogeny module, and taxonomies can be represented using the cvterm module or the phylogeny module.';


--
-- Name: COLUMN organism.species; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN organism.species IS 'A type of organism is always uniquely identified by genus and species. When mapping from the NCBI taxonomy names.dmp file, this column must be used where it is present, as the common_name column is not always unique (e.g. environmental samples). If a particular strain or subspecies is to be represented, this is appended onto the species name. Follows standard NCBI taxonomy pattern.';


--
-- Name: organism_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE organism_dbxref (
    organism_dbxref_id integer NOT NULL,
    organism_id integer NOT NULL,
    dbxref_id integer NOT NULL
);


ALTER TABLE public.organism_dbxref OWNER TO gmod;

--
-- Name: organism_dbxref_organism_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE organism_dbxref_organism_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.organism_dbxref_organism_dbxref_id_seq OWNER TO gmod;

--
-- Name: organism_dbxref_organism_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE organism_dbxref_organism_dbxref_id_seq OWNED BY organism_dbxref.organism_dbxref_id;


--
-- Name: organism_dbxref_organism_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('organism_dbxref_organism_dbxref_id_seq', 1, false);


--
-- Name: organism_organism_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE organism_organism_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.organism_organism_id_seq OWNER TO gmod;

--
-- Name: organism_organism_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE organism_organism_id_seq OWNED BY organism.organism_id;


--
-- Name: organism_organism_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('organism_organism_id_seq', 15, true);


--
-- Name: organismprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE organismprop (
    organismprop_id integer NOT NULL,
    organism_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.organismprop OWNER TO gmod;

--
-- Name: TABLE organismprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE organismprop IS 'Tag-value properties - follows standard chado model.';


--
-- Name: organismprop_organismprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE organismprop_organismprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.organismprop_organismprop_id_seq OWNER TO gmod;

--
-- Name: organismprop_organismprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE organismprop_organismprop_id_seq OWNED BY organismprop.organismprop_id;


--
-- Name: organismprop_organismprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('organismprop_organismprop_id_seq', 1, false);


--
-- Name: phendesc; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phendesc (
    phendesc_id integer NOT NULL,
    genotype_id integer NOT NULL,
    environment_id integer NOT NULL,
    description text NOT NULL,
    type_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.phendesc OWNER TO gmod;

--
-- Name: TABLE phendesc; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phendesc IS 'A summary of a _set_ of phenotypic statements for any one gcontext made in any one publication.';


--
-- Name: phendesc_phendesc_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phendesc_phendesc_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phendesc_phendesc_id_seq OWNER TO gmod;

--
-- Name: phendesc_phendesc_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phendesc_phendesc_id_seq OWNED BY phendesc.phendesc_id;


--
-- Name: phendesc_phendesc_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phendesc_phendesc_id_seq', 1, false);


--
-- Name: phenotype_comparison; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phenotype_comparison (
    phenotype_comparison_id integer NOT NULL,
    genotype1_id integer NOT NULL,
    environment1_id integer NOT NULL,
    genotype2_id integer NOT NULL,
    environment2_id integer NOT NULL,
    phenotype1_id integer NOT NULL,
    phenotype2_id integer,
    pub_id integer NOT NULL,
    organism_id integer NOT NULL
);


ALTER TABLE public.phenotype_comparison OWNER TO gmod;

--
-- Name: TABLE phenotype_comparison; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phenotype_comparison IS 'Comparison of phenotypes e.g., genotype1/environment1/phenotype1 "non-suppressible" with respect to genotype2/environment2/phenotype2.';


--
-- Name: phenotype_comparison_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phenotype_comparison_cvterm (
    phenotype_comparison_cvterm_id integer NOT NULL,
    phenotype_comparison_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    pub_id integer NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.phenotype_comparison_cvterm OWNER TO gmod;

--
-- Name: phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq OWNER TO gmod;

--
-- Name: phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq OWNED BY phenotype_comparison_cvterm.phenotype_comparison_cvterm_id;


--
-- Name: phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq', 1, false);


--
-- Name: phenotype_comparison_phenotype_comparison_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phenotype_comparison_phenotype_comparison_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phenotype_comparison_phenotype_comparison_id_seq OWNER TO gmod;

--
-- Name: phenotype_comparison_phenotype_comparison_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phenotype_comparison_phenotype_comparison_id_seq OWNED BY phenotype_comparison.phenotype_comparison_id;


--
-- Name: phenotype_comparison_phenotype_comparison_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phenotype_comparison_phenotype_comparison_id_seq', 1, false);


--
-- Name: phenotype_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phenotype_cvterm (
    phenotype_cvterm_id integer NOT NULL,
    phenotype_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.phenotype_cvterm OWNER TO gmod;

--
-- Name: phenotype_cvterm_phenotype_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phenotype_cvterm_phenotype_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phenotype_cvterm_phenotype_cvterm_id_seq OWNER TO gmod;

--
-- Name: phenotype_cvterm_phenotype_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phenotype_cvterm_phenotype_cvterm_id_seq OWNED BY phenotype_cvterm.phenotype_cvterm_id;


--
-- Name: phenotype_cvterm_phenotype_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phenotype_cvterm_phenotype_cvterm_id_seq', 1, false);


--
-- Name: phenotype_phenotype_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phenotype_phenotype_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phenotype_phenotype_id_seq OWNER TO gmod;

--
-- Name: phenotype_phenotype_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phenotype_phenotype_id_seq OWNED BY phenotype.phenotype_id;


--
-- Name: phenotype_phenotype_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phenotype_phenotype_id_seq', 130, true);


--
-- Name: phenotypeprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phenotypeprop (
    phenotypeprop_id integer NOT NULL,
    phenotype_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.phenotypeprop OWNER TO gmod;

--
-- Name: phenotypeprop_phenotypeprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phenotypeprop_phenotypeprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phenotypeprop_phenotypeprop_id_seq OWNER TO gmod;

--
-- Name: phenotypeprop_phenotypeprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phenotypeprop_phenotypeprop_id_seq OWNED BY phenotypeprop.phenotypeprop_id;


--
-- Name: phenotypeprop_phenotypeprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phenotypeprop_phenotypeprop_id_seq', 255, true);


--
-- Name: phenstatement; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phenstatement (
    phenstatement_id integer NOT NULL,
    genotype_id integer NOT NULL,
    environment_id integer NOT NULL,
    phenotype_id integer NOT NULL,
    type_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.phenstatement OWNER TO gmod;

--
-- Name: TABLE phenstatement; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phenstatement IS 'Phenotypes are things like "larval lethal". Phenstatements are things like "dpp-1 is recessive larval lethal". So essentially phenstatement is a linking table expressing the relationship between genotype, environment, and phenotype.';


--
-- Name: phenstatement_phenstatement_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phenstatement_phenstatement_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phenstatement_phenstatement_id_seq OWNER TO gmod;

--
-- Name: phenstatement_phenstatement_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phenstatement_phenstatement_id_seq OWNED BY phenstatement.phenstatement_id;


--
-- Name: phenstatement_phenstatement_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phenstatement_phenstatement_id_seq', 1, false);


--
-- Name: phylonode; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylonode (
    phylonode_id integer NOT NULL,
    phylotree_id integer NOT NULL,
    parent_phylonode_id integer,
    left_idx integer NOT NULL,
    right_idx integer NOT NULL,
    type_id integer,
    feature_id integer,
    label character varying(255),
    distance double precision
);


ALTER TABLE public.phylonode OWNER TO gmod;

--
-- Name: TABLE phylonode; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phylonode IS 'This is the most pervasive element in the phylogeny module, cataloging the "phylonodes" of tree graphs. Edges are implied by the parent_phylonode_id reflexive closure. For all nodes in a nested set implementation the left and right index will be *between* the parents left and right indexes.';


--
-- Name: COLUMN phylonode.parent_phylonode_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phylonode.parent_phylonode_id IS 'Root phylonode can have null parent_phylonode_id value.';


--
-- Name: COLUMN phylonode.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phylonode.type_id IS 'Type: e.g. root, interior, leaf.';


--
-- Name: COLUMN phylonode.feature_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phylonode.feature_id IS 'Phylonodes can have optional features attached to them e.g. a protein or nucleotide sequence usually attached to a leaf of the phylotree for non-leaf nodes, the feature may be a feature that is an instance of SO:match; this feature is the alignment of all leaf features beneath it.';


--
-- Name: phylonode_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylonode_dbxref (
    phylonode_dbxref_id integer NOT NULL,
    phylonode_id integer NOT NULL,
    dbxref_id integer NOT NULL
);


ALTER TABLE public.phylonode_dbxref OWNER TO gmod;

--
-- Name: TABLE phylonode_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phylonode_dbxref IS 'For example, for orthology, paralogy group identifiers; could also be used for NCBI taxonomy; for sequences, refer to phylonode_feature, feature associated dbxrefs.';


--
-- Name: phylonode_dbxref_phylonode_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylonode_dbxref_phylonode_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylonode_dbxref_phylonode_dbxref_id_seq OWNER TO gmod;

--
-- Name: phylonode_dbxref_phylonode_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylonode_dbxref_phylonode_dbxref_id_seq OWNED BY phylonode_dbxref.phylonode_dbxref_id;


--
-- Name: phylonode_dbxref_phylonode_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylonode_dbxref_phylonode_dbxref_id_seq', 1, false);


--
-- Name: phylonode_organism; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylonode_organism (
    phylonode_organism_id integer NOT NULL,
    phylonode_id integer NOT NULL,
    organism_id integer NOT NULL
);


ALTER TABLE public.phylonode_organism OWNER TO gmod;

--
-- Name: TABLE phylonode_organism; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phylonode_organism IS 'This linking table should only be used for nodes in taxonomy trees; it provides a mapping between the node and an organism. One node can have zero or one organisms, one organism can have zero or more nodes (although typically it should only have one in the standard NCBI taxonomy tree).';


--
-- Name: COLUMN phylonode_organism.phylonode_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phylonode_organism.phylonode_id IS 'One phylonode cannot refer to >1 organism.';


--
-- Name: phylonode_organism_phylonode_organism_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylonode_organism_phylonode_organism_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylonode_organism_phylonode_organism_id_seq OWNER TO gmod;

--
-- Name: phylonode_organism_phylonode_organism_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylonode_organism_phylonode_organism_id_seq OWNED BY phylonode_organism.phylonode_organism_id;


--
-- Name: phylonode_organism_phylonode_organism_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylonode_organism_phylonode_organism_id_seq', 1, false);


--
-- Name: phylonode_phylonode_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylonode_phylonode_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylonode_phylonode_id_seq OWNER TO gmod;

--
-- Name: phylonode_phylonode_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylonode_phylonode_id_seq OWNED BY phylonode.phylonode_id;


--
-- Name: phylonode_phylonode_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylonode_phylonode_id_seq', 1, false);


--
-- Name: phylonode_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylonode_pub (
    phylonode_pub_id integer NOT NULL,
    phylonode_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.phylonode_pub OWNER TO gmod;

--
-- Name: phylonode_pub_phylonode_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylonode_pub_phylonode_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylonode_pub_phylonode_pub_id_seq OWNER TO gmod;

--
-- Name: phylonode_pub_phylonode_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylonode_pub_phylonode_pub_id_seq OWNED BY phylonode_pub.phylonode_pub_id;


--
-- Name: phylonode_pub_phylonode_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylonode_pub_phylonode_pub_id_seq', 1, false);


--
-- Name: phylonode_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylonode_relationship (
    phylonode_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    object_id integer NOT NULL,
    type_id integer NOT NULL,
    rank integer,
    phylotree_id integer NOT NULL
);


ALTER TABLE public.phylonode_relationship OWNER TO gmod;

--
-- Name: TABLE phylonode_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phylonode_relationship IS 'This is for relationships that are not strictly hierarchical; for example, horizontal gene transfer. Most phylogenetic trees are strictly hierarchical, nevertheless it is here for completeness.';


--
-- Name: phylonode_relationship_phylonode_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylonode_relationship_phylonode_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylonode_relationship_phylonode_relationship_id_seq OWNER TO gmod;

--
-- Name: phylonode_relationship_phylonode_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylonode_relationship_phylonode_relationship_id_seq OWNED BY phylonode_relationship.phylonode_relationship_id;


--
-- Name: phylonode_relationship_phylonode_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylonode_relationship_phylonode_relationship_id_seq', 1, false);


--
-- Name: phylonodeprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylonodeprop (
    phylonodeprop_id integer NOT NULL,
    phylonode_id integer NOT NULL,
    type_id integer NOT NULL,
    value text DEFAULT ''::text NOT NULL,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.phylonodeprop OWNER TO gmod;

--
-- Name: COLUMN phylonodeprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phylonodeprop.type_id IS 'type_id could designate phylonode hierarchy relationships, for example: species taxonomy (kingdom, order, family, genus, species), "ortholog/paralog", "fold/superfold", etc.';


--
-- Name: phylonodeprop_phylonodeprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylonodeprop_phylonodeprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylonodeprop_phylonodeprop_id_seq OWNER TO gmod;

--
-- Name: phylonodeprop_phylonodeprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylonodeprop_phylonodeprop_id_seq OWNED BY phylonodeprop.phylonodeprop_id;


--
-- Name: phylonodeprop_phylonodeprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylonodeprop_phylonodeprop_id_seq', 1, false);


--
-- Name: phylotree; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylotree (
    phylotree_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    name character varying(255),
    type_id integer,
    analysis_id integer,
    comment text
);


ALTER TABLE public.phylotree OWNER TO gmod;

--
-- Name: TABLE phylotree; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phylotree IS 'Global anchor for phylogenetic tree.';


--
-- Name: COLUMN phylotree.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN phylotree.type_id IS 'Type: protein, nucleotide, taxonomy, for example. The type should be any SO type, or "taxonomy".';


--
-- Name: phylotree_phylotree_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylotree_phylotree_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylotree_phylotree_id_seq OWNER TO gmod;

--
-- Name: phylotree_phylotree_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylotree_phylotree_id_seq OWNED BY phylotree.phylotree_id;


--
-- Name: phylotree_phylotree_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylotree_phylotree_id_seq', 1, false);


--
-- Name: phylotree_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE phylotree_pub (
    phylotree_pub_id integer NOT NULL,
    phylotree_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.phylotree_pub OWNER TO gmod;

--
-- Name: TABLE phylotree_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE phylotree_pub IS 'Tracks citations global to the tree e.g. multiple sequence alignment supporting tree construction.';


--
-- Name: phylotree_pub_phylotree_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE phylotree_pub_phylotree_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.phylotree_pub_phylotree_pub_id_seq OWNER TO gmod;

--
-- Name: phylotree_pub_phylotree_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE phylotree_pub_phylotree_pub_id_seq OWNED BY phylotree_pub.phylotree_pub_id;


--
-- Name: phylotree_pub_phylotree_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('phylotree_pub_phylotree_pub_id_seq', 1, false);


--
-- Name: project_contact; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE project_contact (
    project_contact_id integer NOT NULL,
    project_id integer NOT NULL,
    contact_id integer NOT NULL
);


ALTER TABLE public.project_contact OWNER TO gmod;

--
-- Name: TABLE project_contact; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE project_contact IS 'Linking project(s) to contact(s)';


--
-- Name: project_contact_project_contact_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE project_contact_project_contact_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.project_contact_project_contact_id_seq OWNER TO gmod;

--
-- Name: project_contact_project_contact_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE project_contact_project_contact_id_seq OWNED BY project_contact.project_contact_id;


--
-- Name: project_contact_project_contact_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('project_contact_project_contact_id_seq', 1, false);


--
-- Name: project_project_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE project_project_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.project_project_id_seq OWNER TO gmod;

--
-- Name: project_project_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE project_project_id_seq OWNED BY project.project_id;


--
-- Name: project_project_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('project_project_id_seq', 3, true);


--
-- Name: project_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE project_pub (
    project_pub_id integer NOT NULL,
    project_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.project_pub OWNER TO gmod;

--
-- Name: TABLE project_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE project_pub IS 'Linking project(s) to publication(s)';


--
-- Name: project_pub_project_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE project_pub_project_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.project_pub_project_pub_id_seq OWNER TO gmod;

--
-- Name: project_pub_project_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE project_pub_project_pub_id_seq OWNED BY project_pub.project_pub_id;


--
-- Name: project_pub_project_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('project_pub_project_pub_id_seq', 1, false);


--
-- Name: project_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE project_relationship (
    project_relationship_id integer NOT NULL,
    subject_project_id integer NOT NULL,
    object_project_id integer NOT NULL,
    type_id integer NOT NULL
);


ALTER TABLE public.project_relationship OWNER TO gmod;

--
-- Name: TABLE project_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE project_relationship IS 'A project can be composed of several smaller scale projects';


--
-- Name: COLUMN project_relationship.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN project_relationship.type_id IS 'The type of relationship being stated, such as "is part of".';


--
-- Name: project_relationship_project_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE project_relationship_project_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.project_relationship_project_relationship_id_seq OWNER TO gmod;

--
-- Name: project_relationship_project_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE project_relationship_project_relationship_id_seq OWNED BY project_relationship.project_relationship_id;


--
-- Name: project_relationship_project_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('project_relationship_project_relationship_id_seq', 1, false);


--
-- Name: projectprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE projectprop (
    projectprop_id integer NOT NULL,
    project_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.projectprop OWNER TO gmod;

--
-- Name: projectprop_projectprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE projectprop_projectprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.projectprop_projectprop_id_seq OWNER TO gmod;

--
-- Name: projectprop_projectprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE projectprop_projectprop_id_seq OWNED BY projectprop.projectprop_id;


--
-- Name: projectprop_projectprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('projectprop_projectprop_id_seq', 83, true);


--
-- Name: protocol; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE protocol (
    protocol_id integer NOT NULL,
    type_id integer NOT NULL,
    pub_id integer,
    dbxref_id integer,
    name text NOT NULL,
    uri text,
    protocoldescription text,
    hardwaredescription text,
    softwaredescription text
);


ALTER TABLE public.protocol OWNER TO gmod;

--
-- Name: TABLE protocol; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE protocol IS 'Procedural notes on how data was prepared and processed.';


--
-- Name: protocol_protocol_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE protocol_protocol_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.protocol_protocol_id_seq OWNER TO gmod;

--
-- Name: protocol_protocol_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE protocol_protocol_id_seq OWNED BY protocol.protocol_id;


--
-- Name: protocol_protocol_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('protocol_protocol_id_seq', 1, false);


--
-- Name: protocolparam; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE protocolparam (
    protocolparam_id integer NOT NULL,
    protocol_id integer NOT NULL,
    name text NOT NULL,
    datatype_id integer,
    unittype_id integer,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.protocolparam OWNER TO gmod;

--
-- Name: TABLE protocolparam; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE protocolparam IS 'Parameters related to a protocol. For example, if the protocol is a soak, this might include attributes of bath temperature and duration.';


--
-- Name: protocolparam_protocolparam_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE protocolparam_protocolparam_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.protocolparam_protocolparam_id_seq OWNER TO gmod;

--
-- Name: protocolparam_protocolparam_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE protocolparam_protocolparam_id_seq OWNED BY protocolparam.protocolparam_id;


--
-- Name: protocolparam_protocolparam_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('protocolparam_protocolparam_id_seq', 1, false);


--
-- Name: pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE pub (
    pub_id integer NOT NULL,
    title text,
    volumetitle text,
    volume character varying(255),
    series_name character varying(255),
    issue character varying(255),
    pyear character varying(255),
    pages character varying(255),
    miniref character varying(255),
    uniquename text NOT NULL,
    type_id integer NOT NULL,
    is_obsolete boolean DEFAULT false,
    publisher character varying(255),
    pubplace character varying(255)
);


ALTER TABLE public.pub OWNER TO gmod;

--
-- Name: TABLE pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE pub IS 'A documented provenance artefact - publications, documents, personal communication.';


--
-- Name: COLUMN pub.title; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pub.title IS 'Descriptive general heading.';


--
-- Name: COLUMN pub.volumetitle; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pub.volumetitle IS 'Title of part if one of a series.';


--
-- Name: COLUMN pub.series_name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pub.series_name IS 'Full name of (journal) series.';


--
-- Name: COLUMN pub.pages; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pub.pages IS 'Page number range[s], e.g. 457--459, viii + 664pp, lv--lvii.';


--
-- Name: COLUMN pub.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pub.type_id IS 'The type of the publication (book, journal, poem, graffiti, etc). Uses pub cv.';


--
-- Name: pub_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE pub_dbxref (
    pub_dbxref_id integer NOT NULL,
    pub_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    is_current boolean DEFAULT true NOT NULL
);


ALTER TABLE public.pub_dbxref OWNER TO gmod;

--
-- Name: TABLE pub_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE pub_dbxref IS 'Handle links to repositories, e.g. Pubmed, Biosis, zoorec, OCLC, Medline, ISSN, coden...';


--
-- Name: pub_dbxref_pub_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE pub_dbxref_pub_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.pub_dbxref_pub_dbxref_id_seq OWNER TO gmod;

--
-- Name: pub_dbxref_pub_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE pub_dbxref_pub_dbxref_id_seq OWNED BY pub_dbxref.pub_dbxref_id;


--
-- Name: pub_dbxref_pub_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('pub_dbxref_pub_dbxref_id_seq', 1, false);


--
-- Name: pub_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE pub_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.pub_pub_id_seq OWNER TO gmod;

--
-- Name: pub_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE pub_pub_id_seq OWNED BY pub.pub_id;


--
-- Name: pub_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('pub_pub_id_seq', 1, true);


--
-- Name: pub_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE pub_relationship (
    pub_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    object_id integer NOT NULL,
    type_id integer NOT NULL
);


ALTER TABLE public.pub_relationship OWNER TO gmod;

--
-- Name: TABLE pub_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE pub_relationship IS 'Handle relationships between publications, e.g. when one publication makes others obsolete, when one publication contains errata with respect to other publication(s), or when one publication also appears in another pub.';


--
-- Name: pub_relationship_pub_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE pub_relationship_pub_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.pub_relationship_pub_relationship_id_seq OWNER TO gmod;

--
-- Name: pub_relationship_pub_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE pub_relationship_pub_relationship_id_seq OWNED BY pub_relationship.pub_relationship_id;


--
-- Name: pub_relationship_pub_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('pub_relationship_pub_relationship_id_seq', 1, false);


--
-- Name: pubauthor; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE pubauthor (
    pubauthor_id integer NOT NULL,
    pub_id integer NOT NULL,
    rank integer NOT NULL,
    editor boolean DEFAULT false,
    surname character varying(100) NOT NULL,
    givennames character varying(100),
    suffix character varying(100)
);


ALTER TABLE public.pubauthor OWNER TO gmod;

--
-- Name: TABLE pubauthor; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE pubauthor IS 'An author for a publication. Note the denormalisation (hence lack of _ in table name) - this is deliberate as it is in general too hard to assign IDs to authors.';


--
-- Name: COLUMN pubauthor.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pubauthor.rank IS 'Order of author in author list for this pub - order is important.';


--
-- Name: COLUMN pubauthor.editor; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pubauthor.editor IS 'Indicates whether the author is an editor for linked publication. Note: this is a boolean field but does not follow the normal chado convention for naming booleans.';


--
-- Name: COLUMN pubauthor.givennames; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pubauthor.givennames IS 'First name, initials';


--
-- Name: COLUMN pubauthor.suffix; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN pubauthor.suffix IS 'Jr., Sr., etc';


--
-- Name: pubauthor_pubauthor_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE pubauthor_pubauthor_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.pubauthor_pubauthor_id_seq OWNER TO gmod;

--
-- Name: pubauthor_pubauthor_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE pubauthor_pubauthor_id_seq OWNED BY pubauthor.pubauthor_id;


--
-- Name: pubauthor_pubauthor_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('pubauthor_pubauthor_id_seq', 1, false);


--
-- Name: pubprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE pubprop (
    pubprop_id integer NOT NULL,
    pub_id integer NOT NULL,
    type_id integer NOT NULL,
    value text NOT NULL,
    rank integer
);


ALTER TABLE public.pubprop OWNER TO gmod;

--
-- Name: TABLE pubprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE pubprop IS 'Property-value pairs for a pub. Follows standard chado pattern.';


--
-- Name: pubprop_pubprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE pubprop_pubprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.pubprop_pubprop_id_seq OWNER TO gmod;

--
-- Name: pubprop_pubprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE pubprop_pubprop_id_seq OWNED BY pubprop.pubprop_id;


--
-- Name: pubprop_pubprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('pubprop_pubprop_id_seq', 1, false);


--
-- Name: quantification; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE quantification (
    quantification_id integer NOT NULL,
    acquisition_id integer NOT NULL,
    operator_id integer,
    protocol_id integer,
    analysis_id integer NOT NULL,
    quantificationdate timestamp without time zone DEFAULT now(),
    name text,
    uri text
);


ALTER TABLE public.quantification OWNER TO gmod;

--
-- Name: TABLE quantification; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE quantification IS 'Quantification is the transformation of an image acquisition to numeric data. This typically involves statistical procedures.';


--
-- Name: quantification_quantification_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE quantification_quantification_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.quantification_quantification_id_seq OWNER TO gmod;

--
-- Name: quantification_quantification_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE quantification_quantification_id_seq OWNED BY quantification.quantification_id;


--
-- Name: quantification_quantification_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('quantification_quantification_id_seq', 1, false);


--
-- Name: quantification_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE quantification_relationship (
    quantification_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    type_id integer NOT NULL,
    object_id integer NOT NULL
);


ALTER TABLE public.quantification_relationship OWNER TO gmod;

--
-- Name: TABLE quantification_relationship; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE quantification_relationship IS 'There may be multiple rounds of quantification, this allows us to keep an audit trail of what values went where.';


--
-- Name: quantification_relationship_quantification_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE quantification_relationship_quantification_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.quantification_relationship_quantification_relationship_id_seq OWNER TO gmod;

--
-- Name: quantification_relationship_quantification_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE quantification_relationship_quantification_relationship_id_seq OWNED BY quantification_relationship.quantification_relationship_id;


--
-- Name: quantification_relationship_quantification_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('quantification_relationship_quantification_relationship_id_seq', 1, false);


--
-- Name: quantificationprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE quantificationprop (
    quantificationprop_id integer NOT NULL,
    quantification_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.quantificationprop OWNER TO gmod;

--
-- Name: TABLE quantificationprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE quantificationprop IS 'Extra quantification properties that are not accounted for in quantification.';


--
-- Name: quantificationprop_quantificationprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE quantificationprop_quantificationprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.quantificationprop_quantificationprop_id_seq OWNER TO gmod;

--
-- Name: quantificationprop_quantificationprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE quantificationprop_quantificationprop_id_seq OWNED BY quantificationprop.quantificationprop_id;


--
-- Name: quantificationprop_quantificationprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('quantificationprop_quantificationprop_id_seq', 1, false);


--
-- Name: stock_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock_cvterm (
    stock_cvterm_id integer NOT NULL,
    stock_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.stock_cvterm OWNER TO gmod;

--
-- Name: TABLE stock_cvterm; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stock_cvterm IS 'stock_cvterm links a stock to cvterms. This is for secondary cvterms; primary cvterms should use stock.type_id.';


--
-- Name: stock_cvterm_stock_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_cvterm_stock_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_cvterm_stock_cvterm_id_seq OWNER TO gmod;

--
-- Name: stock_cvterm_stock_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_cvterm_stock_cvterm_id_seq OWNED BY stock_cvterm.stock_cvterm_id;


--
-- Name: stock_cvterm_stock_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_cvterm_stock_cvterm_id_seq', 1, false);


--
-- Name: stock_dbxref; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock_dbxref (
    stock_dbxref_id integer NOT NULL,
    stock_id integer NOT NULL,
    dbxref_id integer NOT NULL,
    is_current boolean DEFAULT true NOT NULL
);


ALTER TABLE public.stock_dbxref OWNER TO gmod;

--
-- Name: TABLE stock_dbxref; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stock_dbxref IS 'stock_dbxref links a stock to dbxrefs. This is for secondary identifiers; primary identifiers should use stock.dbxref_id.';


--
-- Name: COLUMN stock_dbxref.is_current; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock_dbxref.is_current IS 'The is_current boolean indicates whether the linked dbxref is the current -official- dbxref for the linked stock.';


--
-- Name: stock_dbxref_stock_dbxref_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_dbxref_stock_dbxref_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_dbxref_stock_dbxref_id_seq OWNER TO gmod;

--
-- Name: stock_dbxref_stock_dbxref_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_dbxref_stock_dbxref_id_seq OWNED BY stock_dbxref.stock_dbxref_id;


--
-- Name: stock_dbxref_stock_dbxref_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_dbxref_stock_dbxref_id_seq', 1, false);


--
-- Name: stock_genotype_stock_genotype_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_genotype_stock_genotype_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_genotype_stock_genotype_id_seq OWNER TO gmod;

--
-- Name: stock_genotype_stock_genotype_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_genotype_stock_genotype_id_seq OWNED BY stock_genotype.stock_genotype_id;


--
-- Name: stock_genotype_stock_genotype_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_genotype_stock_genotype_id_seq', 240, true);


--
-- Name: stock_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock_pub (
    stock_pub_id integer NOT NULL,
    stock_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.stock_pub OWNER TO gmod;

--
-- Name: TABLE stock_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stock_pub IS 'Provenance. Linking table between stocks and, for example, a stocklist computer file.';


--
-- Name: stock_pub_stock_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_pub_stock_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_pub_stock_pub_id_seq OWNER TO gmod;

--
-- Name: stock_pub_stock_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_pub_stock_pub_id_seq OWNED BY stock_pub.stock_pub_id;


--
-- Name: stock_pub_stock_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_pub_stock_pub_id_seq', 1, false);


--
-- Name: stock_relationship; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock_relationship (
    stock_relationship_id integer NOT NULL,
    subject_id integer NOT NULL,
    object_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.stock_relationship OWNER TO gmod;

--
-- Name: COLUMN stock_relationship.subject_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock_relationship.subject_id IS 'stock_relationship.subject_id is the subject of the subj-predicate-obj sentence. This is typically the substock.';


--
-- Name: COLUMN stock_relationship.object_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock_relationship.object_id IS 'stock_relationship.object_id is the object of the subj-predicate-obj sentence. This is typically the container stock.';


--
-- Name: COLUMN stock_relationship.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock_relationship.type_id IS 'stock_relationship.type_id is relationship type between subject and object. This is a cvterm, typically from the OBO relationship ontology, although other relationship types are allowed.';


--
-- Name: COLUMN stock_relationship.value; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock_relationship.value IS 'stock_relationship.value is for additional notes or comments.';


--
-- Name: COLUMN stock_relationship.rank; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stock_relationship.rank IS 'stock_relationship.rank is the ordering of subject stocks with respect to the object stock may be important where rank is used to order these; starts from zero.';


--
-- Name: stock_relationship_cvterm; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock_relationship_cvterm (
    stock_relationship_cvterm_id integer NOT NULL,
    stock_relatiohship_id integer NOT NULL,
    cvterm_id integer NOT NULL,
    pub_id integer
);


ALTER TABLE public.stock_relationship_cvterm OWNER TO gmod;

--
-- Name: TABLE stock_relationship_cvterm; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stock_relationship_cvterm IS 'For germplasm maintenance and pedigree data, stock_relationship. type_id will record cvterms such as "is a female parent of", "a parent for mutation", "is a group_id of", "is a source_id of", etc The cvterms for higher categories such as "generative", "derivative" or "maintenance" can be stored in table stock_relationship_cvterm';


--
-- Name: stock_relationship_cvterm_stock_relationship_cvterm_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_relationship_cvterm_stock_relationship_cvterm_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_relationship_cvterm_stock_relationship_cvterm_id_seq OWNER TO gmod;

--
-- Name: stock_relationship_cvterm_stock_relationship_cvterm_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_relationship_cvterm_stock_relationship_cvterm_id_seq OWNED BY stock_relationship_cvterm.stock_relationship_cvterm_id;


--
-- Name: stock_relationship_cvterm_stock_relationship_cvterm_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_relationship_cvterm_stock_relationship_cvterm_id_seq', 1, false);


--
-- Name: stock_relationship_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stock_relationship_pub (
    stock_relationship_pub_id integer NOT NULL,
    stock_relationship_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.stock_relationship_pub OWNER TO gmod;

--
-- Name: TABLE stock_relationship_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stock_relationship_pub IS 'Provenance. Attach optional evidence to a stock_relationship in the form of a publication.';


--
-- Name: stock_relationship_pub_stock_relationship_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_relationship_pub_stock_relationship_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_relationship_pub_stock_relationship_pub_id_seq OWNER TO gmod;

--
-- Name: stock_relationship_pub_stock_relationship_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_relationship_pub_stock_relationship_pub_id_seq OWNED BY stock_relationship_pub.stock_relationship_pub_id;


--
-- Name: stock_relationship_pub_stock_relationship_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_relationship_pub_stock_relationship_pub_id_seq', 1, false);


--
-- Name: stock_relationship_stock_relationship_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_relationship_stock_relationship_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_relationship_stock_relationship_id_seq OWNER TO gmod;

--
-- Name: stock_relationship_stock_relationship_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_relationship_stock_relationship_id_seq OWNED BY stock_relationship.stock_relationship_id;


--
-- Name: stock_relationship_stock_relationship_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_relationship_stock_relationship_id_seq', 1, false);


--
-- Name: stock_stock_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stock_stock_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stock_stock_id_seq OWNER TO gmod;

--
-- Name: stock_stock_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stock_stock_id_seq OWNED BY stock.stock_id;


--
-- Name: stock_stock_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stock_stock_id_seq', 240, true);


--
-- Name: stockcollection; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stockcollection (
    stockcollection_id integer NOT NULL,
    type_id integer NOT NULL,
    contact_id integer,
    name character varying(255),
    uniquename text NOT NULL
);


ALTER TABLE public.stockcollection OWNER TO gmod;

--
-- Name: TABLE stockcollection; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stockcollection IS 'The lab or stock center distributing the stocks in their collection.';


--
-- Name: COLUMN stockcollection.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stockcollection.type_id IS 'type_id is the collection type cv.';


--
-- Name: COLUMN stockcollection.contact_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stockcollection.contact_id IS 'contact_id links to the contact information for the collection.';


--
-- Name: COLUMN stockcollection.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stockcollection.name IS 'name is the collection.';


--
-- Name: COLUMN stockcollection.uniquename; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stockcollection.uniquename IS 'uniqename is the value of the collection cv.';


--
-- Name: stockcollection_stock; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stockcollection_stock (
    stockcollection_stock_id integer NOT NULL,
    stockcollection_id integer NOT NULL,
    stock_id integer NOT NULL
);


ALTER TABLE public.stockcollection_stock OWNER TO gmod;

--
-- Name: TABLE stockcollection_stock; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stockcollection_stock IS 'stockcollection_stock links a stock collection to the stocks which are contained in the collection.';


--
-- Name: stockcollection_stock_stockcollection_stock_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stockcollection_stock_stockcollection_stock_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stockcollection_stock_stockcollection_stock_id_seq OWNER TO gmod;

--
-- Name: stockcollection_stock_stockcollection_stock_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stockcollection_stock_stockcollection_stock_id_seq OWNED BY stockcollection_stock.stockcollection_stock_id;


--
-- Name: stockcollection_stock_stockcollection_stock_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stockcollection_stock_stockcollection_stock_id_seq', 1, false);


--
-- Name: stockcollection_stockcollection_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stockcollection_stockcollection_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stockcollection_stockcollection_id_seq OWNER TO gmod;

--
-- Name: stockcollection_stockcollection_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stockcollection_stockcollection_id_seq OWNED BY stockcollection.stockcollection_id;


--
-- Name: stockcollection_stockcollection_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stockcollection_stockcollection_id_seq', 1, false);


--
-- Name: stockcollectionprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stockcollectionprop (
    stockcollectionprop_id integer NOT NULL,
    stockcollection_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.stockcollectionprop OWNER TO gmod;

--
-- Name: TABLE stockcollectionprop; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stockcollectionprop IS 'The table stockcollectionprop contains the value of the stock collection such as website/email URLs; the value of the stock collection order URLs.';


--
-- Name: COLUMN stockcollectionprop.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN stockcollectionprop.type_id IS 'The cv for the type_id is "stockcollection property type".';


--
-- Name: stockcollectionprop_stockcollectionprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stockcollectionprop_stockcollectionprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stockcollectionprop_stockcollectionprop_id_seq OWNER TO gmod;

--
-- Name: stockcollectionprop_stockcollectionprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stockcollectionprop_stockcollectionprop_id_seq OWNED BY stockcollectionprop.stockcollectionprop_id;


--
-- Name: stockcollectionprop_stockcollectionprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stockcollectionprop_stockcollectionprop_id_seq', 1, false);


--
-- Name: stockprop_pub; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE stockprop_pub (
    stockprop_pub_id integer NOT NULL,
    stockprop_id integer NOT NULL,
    pub_id integer NOT NULL
);


ALTER TABLE public.stockprop_pub OWNER TO gmod;

--
-- Name: TABLE stockprop_pub; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE stockprop_pub IS 'Provenance. Any stockprop assignment can optionally be supported by a publication.';


--
-- Name: stockprop_pub_stockprop_pub_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stockprop_pub_stockprop_pub_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stockprop_pub_stockprop_pub_id_seq OWNER TO gmod;

--
-- Name: stockprop_pub_stockprop_pub_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stockprop_pub_stockprop_pub_id_seq OWNED BY stockprop_pub.stockprop_pub_id;


--
-- Name: stockprop_pub_stockprop_pub_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stockprop_pub_stockprop_pub_id_seq', 1, false);


--
-- Name: stockprop_stockprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE stockprop_stockprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.stockprop_stockprop_id_seq OWNER TO gmod;

--
-- Name: stockprop_stockprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE stockprop_stockprop_id_seq OWNED BY stockprop.stockprop_id;


--
-- Name: stockprop_stockprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('stockprop_stockprop_id_seq', 1200, true);


--
-- Name: study; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE study (
    study_id integer NOT NULL,
    contact_id integer NOT NULL,
    pub_id integer,
    dbxref_id integer,
    name text NOT NULL,
    description text
);


ALTER TABLE public.study OWNER TO gmod;

--
-- Name: study_assay; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE study_assay (
    study_assay_id integer NOT NULL,
    study_id integer NOT NULL,
    assay_id integer NOT NULL
);


ALTER TABLE public.study_assay OWNER TO gmod;

--
-- Name: study_assay_study_assay_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE study_assay_study_assay_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.study_assay_study_assay_id_seq OWNER TO gmod;

--
-- Name: study_assay_study_assay_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE study_assay_study_assay_id_seq OWNED BY study_assay.study_assay_id;


--
-- Name: study_assay_study_assay_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('study_assay_study_assay_id_seq', 1, false);


--
-- Name: study_study_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE study_study_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.study_study_id_seq OWNER TO gmod;

--
-- Name: study_study_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE study_study_id_seq OWNED BY study.study_id;


--
-- Name: study_study_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('study_study_id_seq', 1, false);


--
-- Name: studydesign; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE studydesign (
    studydesign_id integer NOT NULL,
    study_id integer NOT NULL,
    description text
);


ALTER TABLE public.studydesign OWNER TO gmod;

--
-- Name: studydesign_studydesign_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE studydesign_studydesign_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.studydesign_studydesign_id_seq OWNER TO gmod;

--
-- Name: studydesign_studydesign_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE studydesign_studydesign_id_seq OWNED BY studydesign.studydesign_id;


--
-- Name: studydesign_studydesign_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('studydesign_studydesign_id_seq', 1, false);


--
-- Name: studydesignprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE studydesignprop (
    studydesignprop_id integer NOT NULL,
    studydesign_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.studydesignprop OWNER TO gmod;

--
-- Name: studydesignprop_studydesignprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE studydesignprop_studydesignprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.studydesignprop_studydesignprop_id_seq OWNER TO gmod;

--
-- Name: studydesignprop_studydesignprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE studydesignprop_studydesignprop_id_seq OWNED BY studydesignprop.studydesignprop_id;


--
-- Name: studydesignprop_studydesignprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('studydesignprop_studydesignprop_id_seq', 1, false);


--
-- Name: studyfactor; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE studyfactor (
    studyfactor_id integer NOT NULL,
    studydesign_id integer NOT NULL,
    type_id integer,
    name text NOT NULL,
    description text
);


ALTER TABLE public.studyfactor OWNER TO gmod;

--
-- Name: studyfactor_studyfactor_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE studyfactor_studyfactor_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.studyfactor_studyfactor_id_seq OWNER TO gmod;

--
-- Name: studyfactor_studyfactor_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE studyfactor_studyfactor_id_seq OWNED BY studyfactor.studyfactor_id;


--
-- Name: studyfactor_studyfactor_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('studyfactor_studyfactor_id_seq', 1, false);


--
-- Name: studyfactorvalue; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE studyfactorvalue (
    studyfactorvalue_id integer NOT NULL,
    studyfactor_id integer NOT NULL,
    assay_id integer NOT NULL,
    factorvalue text,
    name text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.studyfactorvalue OWNER TO gmod;

--
-- Name: studyfactorvalue_studyfactorvalue_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE studyfactorvalue_studyfactorvalue_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.studyfactorvalue_studyfactorvalue_id_seq OWNER TO gmod;

--
-- Name: studyfactorvalue_studyfactorvalue_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE studyfactorvalue_studyfactorvalue_id_seq OWNED BY studyfactorvalue.studyfactorvalue_id;


--
-- Name: studyfactorvalue_studyfactorvalue_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('studyfactorvalue_studyfactorvalue_id_seq', 1, false);


--
-- Name: studyprop; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE studyprop (
    studyprop_id integer NOT NULL,
    study_id integer NOT NULL,
    type_id integer NOT NULL,
    value text,
    rank integer DEFAULT 0 NOT NULL
);


ALTER TABLE public.studyprop OWNER TO gmod;

--
-- Name: studyprop_feature; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE studyprop_feature (
    studyprop_feature_id integer NOT NULL,
    studyprop_id integer NOT NULL,
    feature_id integer NOT NULL,
    type_id integer
);


ALTER TABLE public.studyprop_feature OWNER TO gmod;

--
-- Name: studyprop_feature_studyprop_feature_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE studyprop_feature_studyprop_feature_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.studyprop_feature_studyprop_feature_id_seq OWNER TO gmod;

--
-- Name: studyprop_feature_studyprop_feature_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE studyprop_feature_studyprop_feature_id_seq OWNED BY studyprop_feature.studyprop_feature_id;


--
-- Name: studyprop_feature_studyprop_feature_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('studyprop_feature_studyprop_feature_id_seq', 1, false);


--
-- Name: studyprop_studyprop_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE studyprop_studyprop_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.studyprop_studyprop_id_seq OWNER TO gmod;

--
-- Name: studyprop_studyprop_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE studyprop_studyprop_id_seq OWNED BY studyprop.studyprop_id;


--
-- Name: studyprop_studyprop_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('studyprop_studyprop_id_seq', 1, false);


--
-- Name: synonym; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE synonym (
    synonym_id integer NOT NULL,
    name character varying(255) NOT NULL,
    type_id integer NOT NULL,
    synonym_sgml character varying(255) NOT NULL
);


ALTER TABLE public.synonym OWNER TO gmod;

--
-- Name: TABLE synonym; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE synonym IS 'A synonym for a feature. One feature can have multiple synonyms, and the same synonym can apply to multiple features.';


--
-- Name: COLUMN synonym.name; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN synonym.name IS 'The synonym itself. Should be human-readable machine-searchable ascii text.';


--
-- Name: COLUMN synonym.type_id; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN synonym.type_id IS 'Types would be symbol and fullname for now.';


--
-- Name: COLUMN synonym.synonym_sgml; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON COLUMN synonym.synonym_sgml IS 'The fully specified synonym, with any non-ascii characters encoded in SGML.';


--
-- Name: synonym_synonym_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE synonym_synonym_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.synonym_synonym_id_seq OWNER TO gmod;

--
-- Name: synonym_synonym_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE synonym_synonym_id_seq OWNED BY synonym.synonym_id;


--
-- Name: synonym_synonym_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('synonym_synonym_id_seq', 1, false);


--
-- Name: tableinfo; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE tableinfo (
    tableinfo_id integer NOT NULL,
    name character varying(30) NOT NULL,
    primary_key_column character varying(30),
    is_view integer DEFAULT 0 NOT NULL,
    view_on_table_id integer,
    superclass_table_id integer,
    is_updateable integer DEFAULT 1 NOT NULL,
    modification_date date DEFAULT now() NOT NULL
);


ALTER TABLE public.tableinfo OWNER TO gmod;

--
-- Name: tableinfo_tableinfo_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE tableinfo_tableinfo_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.tableinfo_tableinfo_id_seq OWNER TO gmod;

--
-- Name: tableinfo_tableinfo_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE tableinfo_tableinfo_id_seq OWNED BY tableinfo.tableinfo_id;


--
-- Name: tableinfo_tableinfo_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('tableinfo_tableinfo_id_seq', 1, false);


--
-- Name: treatment; Type: TABLE; Schema: public; Owner: gmod; Tablespace: 
--

CREATE TABLE treatment (
    treatment_id integer NOT NULL,
    rank integer DEFAULT 0 NOT NULL,
    biomaterial_id integer NOT NULL,
    type_id integer NOT NULL,
    protocol_id integer,
    name text
);


ALTER TABLE public.treatment OWNER TO gmod;

--
-- Name: TABLE treatment; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON TABLE treatment IS 'A biomaterial may undergo multiple treatments. Examples of treatments: apoxia, fluorophore and biotin labeling.';


--
-- Name: treatment_treatment_id_seq; Type: SEQUENCE; Schema: public; Owner: gmod
--

CREATE SEQUENCE treatment_treatment_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MAXVALUE
    NO MINVALUE
    CACHE 1;


ALTER TABLE public.treatment_treatment_id_seq OWNER TO gmod;

--
-- Name: treatment_treatment_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gmod
--

ALTER SEQUENCE treatment_treatment_id_seq OWNED BY treatment.treatment_id;


--
-- Name: treatment_treatment_id_seq; Type: SEQUENCE SET; Schema: public; Owner: gmod
--

SELECT pg_catalog.setval('treatment_treatment_id_seq', 1, false);


--
-- Name: acquisition_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE acquisition ALTER COLUMN acquisition_id SET DEFAULT nextval('acquisition_acquisition_id_seq'::regclass);


--
-- Name: acquisition_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE acquisition_relationship ALTER COLUMN acquisition_relationship_id SET DEFAULT nextval('acquisition_relationship_acquisition_relationship_id_seq'::regclass);


--
-- Name: acquisitionprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE acquisitionprop ALTER COLUMN acquisitionprop_id SET DEFAULT nextval('acquisitionprop_acquisitionprop_id_seq'::regclass);


--
-- Name: analysis_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE analysis ALTER COLUMN analysis_id SET DEFAULT nextval('analysis_analysis_id_seq'::regclass);


--
-- Name: analysisfeature_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE analysisfeature ALTER COLUMN analysisfeature_id SET DEFAULT nextval('analysisfeature_analysisfeature_id_seq'::regclass);


--
-- Name: analysisfeatureprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE analysisfeatureprop ALTER COLUMN analysisfeatureprop_id SET DEFAULT nextval('analysisfeatureprop_analysisfeatureprop_id_seq'::regclass);


--
-- Name: analysisprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE analysisprop ALTER COLUMN analysisprop_id SET DEFAULT nextval('analysisprop_analysisprop_id_seq'::regclass);


--
-- Name: arraydesign_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE arraydesign ALTER COLUMN arraydesign_id SET DEFAULT nextval('arraydesign_arraydesign_id_seq'::regclass);


--
-- Name: arraydesignprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE arraydesignprop ALTER COLUMN arraydesignprop_id SET DEFAULT nextval('arraydesignprop_arraydesignprop_id_seq'::regclass);


--
-- Name: assay_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE assay ALTER COLUMN assay_id SET DEFAULT nextval('assay_assay_id_seq'::regclass);


--
-- Name: assay_biomaterial_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE assay_biomaterial ALTER COLUMN assay_biomaterial_id SET DEFAULT nextval('assay_biomaterial_assay_biomaterial_id_seq'::regclass);


--
-- Name: assay_project_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE assay_project ALTER COLUMN assay_project_id SET DEFAULT nextval('assay_project_assay_project_id_seq'::regclass);


--
-- Name: assayprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE assayprop ALTER COLUMN assayprop_id SET DEFAULT nextval('assayprop_assayprop_id_seq'::regclass);


--
-- Name: biomaterial_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE biomaterial ALTER COLUMN biomaterial_id SET DEFAULT nextval('biomaterial_biomaterial_id_seq'::regclass);


--
-- Name: biomaterial_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE biomaterial_dbxref ALTER COLUMN biomaterial_dbxref_id SET DEFAULT nextval('biomaterial_dbxref_biomaterial_dbxref_id_seq'::regclass);


--
-- Name: biomaterial_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE biomaterial_relationship ALTER COLUMN biomaterial_relationship_id SET DEFAULT nextval('biomaterial_relationship_biomaterial_relationship_id_seq'::regclass);


--
-- Name: biomaterial_treatment_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE biomaterial_treatment ALTER COLUMN biomaterial_treatment_id SET DEFAULT nextval('biomaterial_treatment_biomaterial_treatment_id_seq'::regclass);


--
-- Name: biomaterialprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE biomaterialprop ALTER COLUMN biomaterialprop_id SET DEFAULT nextval('biomaterialprop_biomaterialprop_id_seq'::regclass);


--
-- Name: cell_line_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line ALTER COLUMN cell_line_id SET DEFAULT nextval('cell_line_cell_line_id_seq'::regclass);


--
-- Name: cell_line_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_cvterm ALTER COLUMN cell_line_cvterm_id SET DEFAULT nextval('cell_line_cvterm_cell_line_cvterm_id_seq'::regclass);


--
-- Name: cell_line_cvtermprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_cvtermprop ALTER COLUMN cell_line_cvtermprop_id SET DEFAULT nextval('cell_line_cvtermprop_cell_line_cvtermprop_id_seq'::regclass);


--
-- Name: cell_line_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_dbxref ALTER COLUMN cell_line_dbxref_id SET DEFAULT nextval('cell_line_dbxref_cell_line_dbxref_id_seq'::regclass);


--
-- Name: cell_line_feature_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_feature ALTER COLUMN cell_line_feature_id SET DEFAULT nextval('cell_line_feature_cell_line_feature_id_seq'::regclass);


--
-- Name: cell_line_library_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_library ALTER COLUMN cell_line_library_id SET DEFAULT nextval('cell_line_library_cell_line_library_id_seq'::regclass);


--
-- Name: cell_line_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_pub ALTER COLUMN cell_line_pub_id SET DEFAULT nextval('cell_line_pub_cell_line_pub_id_seq'::regclass);


--
-- Name: cell_line_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_relationship ALTER COLUMN cell_line_relationship_id SET DEFAULT nextval('cell_line_relationship_cell_line_relationship_id_seq'::regclass);


--
-- Name: cell_line_synonym_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_line_synonym ALTER COLUMN cell_line_synonym_id SET DEFAULT nextval('cell_line_synonym_cell_line_synonym_id_seq'::regclass);


--
-- Name: cell_lineprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_lineprop ALTER COLUMN cell_lineprop_id SET DEFAULT nextval('cell_lineprop_cell_lineprop_id_seq'::regclass);


--
-- Name: cell_lineprop_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cell_lineprop_pub ALTER COLUMN cell_lineprop_pub_id SET DEFAULT nextval('cell_lineprop_pub_cell_lineprop_pub_id_seq'::regclass);


--
-- Name: channel_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE channel ALTER COLUMN channel_id SET DEFAULT nextval('channel_channel_id_seq'::regclass);


--
-- Name: contact_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE contact ALTER COLUMN contact_id SET DEFAULT nextval('contact_contact_id_seq'::regclass);


--
-- Name: contact_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE contact_relationship ALTER COLUMN contact_relationship_id SET DEFAULT nextval('contact_relationship_contact_relationship_id_seq'::regclass);


--
-- Name: contactprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE contactprop ALTER COLUMN contactprop_id SET DEFAULT nextval('contactprop_contactprop_id_seq'::regclass);


--
-- Name: control_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE control ALTER COLUMN control_id SET DEFAULT nextval('control_control_id_seq'::regclass);


--
-- Name: cv_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cv ALTER COLUMN cv_id SET DEFAULT nextval('cv_cv_id_seq'::regclass);


--
-- Name: cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cvterm ALTER COLUMN cvterm_id SET DEFAULT nextval('cvterm_cvterm_id_seq'::regclass);


--
-- Name: cvterm_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cvterm_dbxref ALTER COLUMN cvterm_dbxref_id SET DEFAULT nextval('cvterm_dbxref_cvterm_dbxref_id_seq'::regclass);


--
-- Name: cvterm_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cvterm_relationship ALTER COLUMN cvterm_relationship_id SET DEFAULT nextval('cvterm_relationship_cvterm_relationship_id_seq'::regclass);


--
-- Name: cvtermpath_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cvtermpath ALTER COLUMN cvtermpath_id SET DEFAULT nextval('cvtermpath_cvtermpath_id_seq'::regclass);


--
-- Name: cvtermprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cvtermprop ALTER COLUMN cvtermprop_id SET DEFAULT nextval('cvtermprop_cvtermprop_id_seq'::regclass);


--
-- Name: cvtermsynonym_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE cvtermsynonym ALTER COLUMN cvtermsynonym_id SET DEFAULT nextval('cvtermsynonym_cvtermsynonym_id_seq'::regclass);


--
-- Name: db_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE db ALTER COLUMN db_id SET DEFAULT nextval('db_db_id_seq'::regclass);


--
-- Name: dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE dbxref ALTER COLUMN dbxref_id SET DEFAULT nextval('dbxref_dbxref_id_seq'::regclass);


--
-- Name: dbxrefprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE dbxrefprop ALTER COLUMN dbxrefprop_id SET DEFAULT nextval('dbxrefprop_dbxrefprop_id_seq'::regclass);


--
-- Name: eimage_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE eimage ALTER COLUMN eimage_id SET DEFAULT nextval('eimage_eimage_id_seq'::regclass);


--
-- Name: element_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE element ALTER COLUMN element_id SET DEFAULT nextval('element_element_id_seq'::regclass);


--
-- Name: element_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE element_relationship ALTER COLUMN element_relationship_id SET DEFAULT nextval('element_relationship_element_relationship_id_seq'::regclass);


--
-- Name: elementresult_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE elementresult ALTER COLUMN elementresult_id SET DEFAULT nextval('elementresult_elementresult_id_seq'::regclass);


--
-- Name: elementresult_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE elementresult_relationship ALTER COLUMN elementresult_relationship_id SET DEFAULT nextval('elementresult_relationship_elementresult_relationship_id_seq'::regclass);


--
-- Name: environment_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE environment ALTER COLUMN environment_id SET DEFAULT nextval('environment_environment_id_seq'::regclass);


--
-- Name: environment_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE environment_cvterm ALTER COLUMN environment_cvterm_id SET DEFAULT nextval('environment_cvterm_environment_cvterm_id_seq'::regclass);


--
-- Name: expression_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE expression ALTER COLUMN expression_id SET DEFAULT nextval('expression_expression_id_seq'::regclass);


--
-- Name: expression_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE expression_cvterm ALTER COLUMN expression_cvterm_id SET DEFAULT nextval('expression_cvterm_expression_cvterm_id_seq'::regclass);


--
-- Name: expression_cvtermprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE expression_cvtermprop ALTER COLUMN expression_cvtermprop_id SET DEFAULT nextval('expression_cvtermprop_expression_cvtermprop_id_seq'::regclass);


--
-- Name: expression_image_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE expression_image ALTER COLUMN expression_image_id SET DEFAULT nextval('expression_image_expression_image_id_seq'::regclass);


--
-- Name: expression_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE expression_pub ALTER COLUMN expression_pub_id SET DEFAULT nextval('expression_pub_expression_pub_id_seq'::regclass);


--
-- Name: expressionprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE expressionprop ALTER COLUMN expressionprop_id SET DEFAULT nextval('expressionprop_expressionprop_id_seq'::regclass);


--
-- Name: feature_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature ALTER COLUMN feature_id SET DEFAULT nextval('feature_feature_id_seq'::regclass);


--
-- Name: feature_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_cvterm ALTER COLUMN feature_cvterm_id SET DEFAULT nextval('feature_cvterm_feature_cvterm_id_seq'::regclass);


--
-- Name: feature_cvterm_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_cvterm_dbxref ALTER COLUMN feature_cvterm_dbxref_id SET DEFAULT nextval('feature_cvterm_dbxref_feature_cvterm_dbxref_id_seq'::regclass);


--
-- Name: feature_cvterm_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_cvterm_pub ALTER COLUMN feature_cvterm_pub_id SET DEFAULT nextval('feature_cvterm_pub_feature_cvterm_pub_id_seq'::regclass);


--
-- Name: feature_cvtermprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_cvtermprop ALTER COLUMN feature_cvtermprop_id SET DEFAULT nextval('feature_cvtermprop_feature_cvtermprop_id_seq'::regclass);


--
-- Name: feature_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_dbxref ALTER COLUMN feature_dbxref_id SET DEFAULT nextval('feature_dbxref_feature_dbxref_id_seq'::regclass);


--
-- Name: feature_expression_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_expression ALTER COLUMN feature_expression_id SET DEFAULT nextval('feature_expression_feature_expression_id_seq'::regclass);


--
-- Name: feature_expressionprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_expressionprop ALTER COLUMN feature_expressionprop_id SET DEFAULT nextval('feature_expressionprop_feature_expressionprop_id_seq'::regclass);


--
-- Name: feature_genotype_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_genotype ALTER COLUMN feature_genotype_id SET DEFAULT nextval('feature_genotype_feature_genotype_id_seq'::regclass);


--
-- Name: feature_phenotype_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_phenotype ALTER COLUMN feature_phenotype_id SET DEFAULT nextval('feature_phenotype_feature_phenotype_id_seq'::regclass);


--
-- Name: feature_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_pub ALTER COLUMN feature_pub_id SET DEFAULT nextval('feature_pub_feature_pub_id_seq'::regclass);


--
-- Name: feature_pubprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_pubprop ALTER COLUMN feature_pubprop_id SET DEFAULT nextval('feature_pubprop_feature_pubprop_id_seq'::regclass);


--
-- Name: feature_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_relationship ALTER COLUMN feature_relationship_id SET DEFAULT nextval('feature_relationship_feature_relationship_id_seq'::regclass);


--
-- Name: feature_relationship_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_relationship_pub ALTER COLUMN feature_relationship_pub_id SET DEFAULT nextval('feature_relationship_pub_feature_relationship_pub_id_seq'::regclass);


--
-- Name: feature_relationshipprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_relationshipprop ALTER COLUMN feature_relationshipprop_id SET DEFAULT nextval('feature_relationshipprop_feature_relationshipprop_id_seq'::regclass);


--
-- Name: feature_relationshipprop_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_relationshipprop_pub ALTER COLUMN feature_relationshipprop_pub_id SET DEFAULT nextval('feature_relationshipprop_pub_feature_relationshipprop_pub_i_seq'::regclass);


--
-- Name: feature_synonym_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE feature_synonym ALTER COLUMN feature_synonym_id SET DEFAULT nextval('feature_synonym_feature_synonym_id_seq'::regclass);


--
-- Name: featureloc_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featureloc ALTER COLUMN featureloc_id SET DEFAULT nextval('featureloc_featureloc_id_seq'::regclass);


--
-- Name: featureloc_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featureloc_pub ALTER COLUMN featureloc_pub_id SET DEFAULT nextval('featureloc_pub_featureloc_pub_id_seq'::regclass);


--
-- Name: featuremap_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featuremap ALTER COLUMN featuremap_id SET DEFAULT nextval('featuremap_featuremap_id_seq'::regclass);


--
-- Name: featuremap_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featuremap_pub ALTER COLUMN featuremap_pub_id SET DEFAULT nextval('featuremap_pub_featuremap_pub_id_seq'::regclass);


--
-- Name: featurepos_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featurepos ALTER COLUMN featurepos_id SET DEFAULT nextval('featurepos_featurepos_id_seq'::regclass);


--
-- Name: featuremap_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featurepos ALTER COLUMN featuremap_id SET DEFAULT nextval('featurepos_featuremap_id_seq'::regclass);


--
-- Name: featureprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featureprop ALTER COLUMN featureprop_id SET DEFAULT nextval('featureprop_featureprop_id_seq'::regclass);


--
-- Name: featureprop_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featureprop_pub ALTER COLUMN featureprop_pub_id SET DEFAULT nextval('featureprop_pub_featureprop_pub_id_seq'::regclass);


--
-- Name: featurerange_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE featurerange ALTER COLUMN featurerange_id SET DEFAULT nextval('featurerange_featurerange_id_seq'::regclass);


--
-- Name: genotype_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE genotype ALTER COLUMN genotype_id SET DEFAULT nextval('genotype_genotype_id_seq'::regclass);


--
-- Name: library_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE library ALTER COLUMN library_id SET DEFAULT nextval('library_library_id_seq'::regclass);


--
-- Name: library_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE library_cvterm ALTER COLUMN library_cvterm_id SET DEFAULT nextval('library_cvterm_library_cvterm_id_seq'::regclass);


--
-- Name: library_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE library_dbxref ALTER COLUMN library_dbxref_id SET DEFAULT nextval('library_dbxref_library_dbxref_id_seq'::regclass);


--
-- Name: library_feature_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE library_feature ALTER COLUMN library_feature_id SET DEFAULT nextval('library_feature_library_feature_id_seq'::regclass);


--
-- Name: library_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE library_pub ALTER COLUMN library_pub_id SET DEFAULT nextval('library_pub_library_pub_id_seq'::regclass);


--
-- Name: library_synonym_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE library_synonym ALTER COLUMN library_synonym_id SET DEFAULT nextval('library_synonym_library_synonym_id_seq'::regclass);


--
-- Name: libraryprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE libraryprop ALTER COLUMN libraryprop_id SET DEFAULT nextval('libraryprop_libraryprop_id_seq'::regclass);


--
-- Name: libraryprop_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE libraryprop_pub ALTER COLUMN libraryprop_pub_id SET DEFAULT nextval('libraryprop_pub_libraryprop_pub_id_seq'::regclass);


--
-- Name: magedocumentation_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE magedocumentation ALTER COLUMN magedocumentation_id SET DEFAULT nextval('magedocumentation_magedocumentation_id_seq'::regclass);


--
-- Name: mageml_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE mageml ALTER COLUMN mageml_id SET DEFAULT nextval('mageml_mageml_id_seq'::regclass);


--
-- Name: materialized_view_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE materialized_view ALTER COLUMN materialized_view_id SET DEFAULT nextval('materialized_view_materialized_view_id_seq'::regclass);


--
-- Name: nd_experiment_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment ALTER COLUMN nd_experiment_id SET DEFAULT nextval('nd_experiment_nd_experiment_id_seq'::regclass);


--
-- Name: nd_experiment_contact_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_contact ALTER COLUMN nd_experiment_contact_id SET DEFAULT nextval('nd_experiment_contact_nd_experiment_contact_id_seq'::regclass);


--
-- Name: nd_experiment_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_dbxref ALTER COLUMN nd_experiment_dbxref_id SET DEFAULT nextval('nd_experiment_dbxref_nd_experiment_dbxref_id_seq'::regclass);


--
-- Name: nd_experiment_genotype_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_genotype ALTER COLUMN nd_experiment_genotype_id SET DEFAULT nextval('nd_experiment_genotype_nd_experiment_genotype_id_seq'::regclass);


--
-- Name: nd_experiment_phenotype_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_phenotype ALTER COLUMN nd_experiment_phenotype_id SET DEFAULT nextval('nd_experiment_phenotype_nd_experiment_phenotype_id_seq'::regclass);


--
-- Name: nd_experiment_phenotypeprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_phenotypeprop ALTER COLUMN nd_experiment_phenotypeprop_id SET DEFAULT nextval('nd_experiment_phenotypeprop_nd_experiment_phenotypeprop_id_seq'::regclass);


--
-- Name: nd_experiment_project_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_project ALTER COLUMN nd_experiment_project_id SET DEFAULT nextval('nd_experiment_project_nd_experiment_project_id_seq'::regclass);


--
-- Name: nd_experiment_protocol_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_protocol ALTER COLUMN nd_experiment_protocol_id SET DEFAULT nextval('nd_experiment_protocol_nd_experiment_protocol_id_seq'::regclass);


--
-- Name: nd_experiment_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_pub ALTER COLUMN nd_experiment_pub_id SET DEFAULT nextval('nd_experiment_pub_nd_experiment_pub_id_seq'::regclass);


--
-- Name: nd_experiment_stock_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_stock ALTER COLUMN nd_experiment_stock_id SET DEFAULT nextval('nd_experiment_stock_nd_experiment_stock_id_seq'::regclass);


--
-- Name: nd_experiment_stock_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_stock_dbxref ALTER COLUMN nd_experiment_stock_dbxref_id SET DEFAULT nextval('nd_experiment_stock_dbxref_nd_experiment_stock_dbxref_id_seq'::regclass);


--
-- Name: nd_experiment_stockprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experiment_stockprop ALTER COLUMN nd_experiment_stockprop_id SET DEFAULT nextval('nd_experiment_stockprop_nd_experiment_stockprop_id_seq'::regclass);


--
-- Name: nd_experimentprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_experimentprop ALTER COLUMN nd_experimentprop_id SET DEFAULT nextval('nd_experimentprop_nd_experimentprop_id_seq'::regclass);


--
-- Name: nd_geolocation_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_geolocation ALTER COLUMN nd_geolocation_id SET DEFAULT nextval('nd_geolocation_nd_geolocation_id_seq'::regclass);


--
-- Name: nd_geolocationprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_geolocationprop ALTER COLUMN nd_geolocationprop_id SET DEFAULT nextval('nd_geolocationprop_nd_geolocationprop_id_seq'::regclass);


--
-- Name: nd_protocol_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_protocol ALTER COLUMN nd_protocol_id SET DEFAULT nextval('nd_protocol_nd_protocol_id_seq'::regclass);


--
-- Name: nd_protocol_reagent_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_protocol_reagent ALTER COLUMN nd_protocol_reagent_id SET DEFAULT nextval('nd_protocol_reagent_nd_protocol_reagent_id_seq'::regclass);


--
-- Name: nd_protocolprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_protocolprop ALTER COLUMN nd_protocolprop_id SET DEFAULT nextval('nd_protocolprop_nd_protocolprop_id_seq'::regclass);


--
-- Name: nd_reagent_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_reagent ALTER COLUMN nd_reagent_id SET DEFAULT nextval('nd_reagent_nd_reagent_id_seq'::regclass);


--
-- Name: nd_reagent_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_reagent_relationship ALTER COLUMN nd_reagent_relationship_id SET DEFAULT nextval('nd_reagent_relationship_nd_reagent_relationship_id_seq'::regclass);


--
-- Name: nd_reagentprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE nd_reagentprop ALTER COLUMN nd_reagentprop_id SET DEFAULT nextval('nd_reagentprop_nd_reagentprop_id_seq'::regclass);


--
-- Name: organism_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE organism ALTER COLUMN organism_id SET DEFAULT nextval('organism_organism_id_seq'::regclass);


--
-- Name: organism_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE organism_dbxref ALTER COLUMN organism_dbxref_id SET DEFAULT nextval('organism_dbxref_organism_dbxref_id_seq'::regclass);


--
-- Name: organismprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE organismprop ALTER COLUMN organismprop_id SET DEFAULT nextval('organismprop_organismprop_id_seq'::regclass);


--
-- Name: phendesc_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phendesc ALTER COLUMN phendesc_id SET DEFAULT nextval('phendesc_phendesc_id_seq'::regclass);


--
-- Name: phenotype_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phenotype ALTER COLUMN phenotype_id SET DEFAULT nextval('phenotype_phenotype_id_seq'::regclass);


--
-- Name: phenotype_comparison_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phenotype_comparison ALTER COLUMN phenotype_comparison_id SET DEFAULT nextval('phenotype_comparison_phenotype_comparison_id_seq'::regclass);


--
-- Name: phenotype_comparison_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phenotype_comparison_cvterm ALTER COLUMN phenotype_comparison_cvterm_id SET DEFAULT nextval('phenotype_comparison_cvterm_phenotype_comparison_cvterm_id_seq'::regclass);


--
-- Name: phenotype_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phenotype_cvterm ALTER COLUMN phenotype_cvterm_id SET DEFAULT nextval('phenotype_cvterm_phenotype_cvterm_id_seq'::regclass);


--
-- Name: phenotypeprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phenotypeprop ALTER COLUMN phenotypeprop_id SET DEFAULT nextval('phenotypeprop_phenotypeprop_id_seq'::regclass);


--
-- Name: phenstatement_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phenstatement ALTER COLUMN phenstatement_id SET DEFAULT nextval('phenstatement_phenstatement_id_seq'::regclass);


--
-- Name: phylonode_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylonode ALTER COLUMN phylonode_id SET DEFAULT nextval('phylonode_phylonode_id_seq'::regclass);


--
-- Name: phylonode_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylonode_dbxref ALTER COLUMN phylonode_dbxref_id SET DEFAULT nextval('phylonode_dbxref_phylonode_dbxref_id_seq'::regclass);


--
-- Name: phylonode_organism_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylonode_organism ALTER COLUMN phylonode_organism_id SET DEFAULT nextval('phylonode_organism_phylonode_organism_id_seq'::regclass);


--
-- Name: phylonode_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylonode_pub ALTER COLUMN phylonode_pub_id SET DEFAULT nextval('phylonode_pub_phylonode_pub_id_seq'::regclass);


--
-- Name: phylonode_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylonode_relationship ALTER COLUMN phylonode_relationship_id SET DEFAULT nextval('phylonode_relationship_phylonode_relationship_id_seq'::regclass);


--
-- Name: phylonodeprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylonodeprop ALTER COLUMN phylonodeprop_id SET DEFAULT nextval('phylonodeprop_phylonodeprop_id_seq'::regclass);


--
-- Name: phylotree_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylotree ALTER COLUMN phylotree_id SET DEFAULT nextval('phylotree_phylotree_id_seq'::regclass);


--
-- Name: phylotree_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE phylotree_pub ALTER COLUMN phylotree_pub_id SET DEFAULT nextval('phylotree_pub_phylotree_pub_id_seq'::regclass);


--
-- Name: project_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE project ALTER COLUMN project_id SET DEFAULT nextval('project_project_id_seq'::regclass);


--
-- Name: project_contact_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE project_contact ALTER COLUMN project_contact_id SET DEFAULT nextval('project_contact_project_contact_id_seq'::regclass);


--
-- Name: project_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE project_pub ALTER COLUMN project_pub_id SET DEFAULT nextval('project_pub_project_pub_id_seq'::regclass);


--
-- Name: project_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE project_relationship ALTER COLUMN project_relationship_id SET DEFAULT nextval('project_relationship_project_relationship_id_seq'::regclass);


--
-- Name: projectprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE projectprop ALTER COLUMN projectprop_id SET DEFAULT nextval('projectprop_projectprop_id_seq'::regclass);


--
-- Name: protocol_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE protocol ALTER COLUMN protocol_id SET DEFAULT nextval('protocol_protocol_id_seq'::regclass);


--
-- Name: protocolparam_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE protocolparam ALTER COLUMN protocolparam_id SET DEFAULT nextval('protocolparam_protocolparam_id_seq'::regclass);


--
-- Name: pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE pub ALTER COLUMN pub_id SET DEFAULT nextval('pub_pub_id_seq'::regclass);


--
-- Name: pub_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE pub_dbxref ALTER COLUMN pub_dbxref_id SET DEFAULT nextval('pub_dbxref_pub_dbxref_id_seq'::regclass);


--
-- Name: pub_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE pub_relationship ALTER COLUMN pub_relationship_id SET DEFAULT nextval('pub_relationship_pub_relationship_id_seq'::regclass);


--
-- Name: pubauthor_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE pubauthor ALTER COLUMN pubauthor_id SET DEFAULT nextval('pubauthor_pubauthor_id_seq'::regclass);


--
-- Name: pubprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE pubprop ALTER COLUMN pubprop_id SET DEFAULT nextval('pubprop_pubprop_id_seq'::regclass);


--
-- Name: quantification_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE quantification ALTER COLUMN quantification_id SET DEFAULT nextval('quantification_quantification_id_seq'::regclass);


--
-- Name: quantification_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE quantification_relationship ALTER COLUMN quantification_relationship_id SET DEFAULT nextval('quantification_relationship_quantification_relationship_id_seq'::regclass);


--
-- Name: quantificationprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE quantificationprop ALTER COLUMN quantificationprop_id SET DEFAULT nextval('quantificationprop_quantificationprop_id_seq'::regclass);


--
-- Name: stock_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock ALTER COLUMN stock_id SET DEFAULT nextval('stock_stock_id_seq'::regclass);


--
-- Name: stock_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock_cvterm ALTER COLUMN stock_cvterm_id SET DEFAULT nextval('stock_cvterm_stock_cvterm_id_seq'::regclass);


--
-- Name: stock_dbxref_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock_dbxref ALTER COLUMN stock_dbxref_id SET DEFAULT nextval('stock_dbxref_stock_dbxref_id_seq'::regclass);


--
-- Name: stock_genotype_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock_genotype ALTER COLUMN stock_genotype_id SET DEFAULT nextval('stock_genotype_stock_genotype_id_seq'::regclass);


--
-- Name: stock_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock_pub ALTER COLUMN stock_pub_id SET DEFAULT nextval('stock_pub_stock_pub_id_seq'::regclass);


--
-- Name: stock_relationship_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock_relationship ALTER COLUMN stock_relationship_id SET DEFAULT nextval('stock_relationship_stock_relationship_id_seq'::regclass);


--
-- Name: stock_relationship_cvterm_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock_relationship_cvterm ALTER COLUMN stock_relationship_cvterm_id SET DEFAULT nextval('stock_relationship_cvterm_stock_relationship_cvterm_id_seq'::regclass);


--
-- Name: stock_relationship_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stock_relationship_pub ALTER COLUMN stock_relationship_pub_id SET DEFAULT nextval('stock_relationship_pub_stock_relationship_pub_id_seq'::regclass);


--
-- Name: stockcollection_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stockcollection ALTER COLUMN stockcollection_id SET DEFAULT nextval('stockcollection_stockcollection_id_seq'::regclass);


--
-- Name: stockcollection_stock_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stockcollection_stock ALTER COLUMN stockcollection_stock_id SET DEFAULT nextval('stockcollection_stock_stockcollection_stock_id_seq'::regclass);


--
-- Name: stockcollectionprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stockcollectionprop ALTER COLUMN stockcollectionprop_id SET DEFAULT nextval('stockcollectionprop_stockcollectionprop_id_seq'::regclass);


--
-- Name: stockprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stockprop ALTER COLUMN stockprop_id SET DEFAULT nextval('stockprop_stockprop_id_seq'::regclass);


--
-- Name: stockprop_pub_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE stockprop_pub ALTER COLUMN stockprop_pub_id SET DEFAULT nextval('stockprop_pub_stockprop_pub_id_seq'::regclass);


--
-- Name: study_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE study ALTER COLUMN study_id SET DEFAULT nextval('study_study_id_seq'::regclass);


--
-- Name: study_assay_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE study_assay ALTER COLUMN study_assay_id SET DEFAULT nextval('study_assay_study_assay_id_seq'::regclass);


--
-- Name: studydesign_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE studydesign ALTER COLUMN studydesign_id SET DEFAULT nextval('studydesign_studydesign_id_seq'::regclass);


--
-- Name: studydesignprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE studydesignprop ALTER COLUMN studydesignprop_id SET DEFAULT nextval('studydesignprop_studydesignprop_id_seq'::regclass);


--
-- Name: studyfactor_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE studyfactor ALTER COLUMN studyfactor_id SET DEFAULT nextval('studyfactor_studyfactor_id_seq'::regclass);


--
-- Name: studyfactorvalue_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE studyfactorvalue ALTER COLUMN studyfactorvalue_id SET DEFAULT nextval('studyfactorvalue_studyfactorvalue_id_seq'::regclass);


--
-- Name: studyprop_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE studyprop ALTER COLUMN studyprop_id SET DEFAULT nextval('studyprop_studyprop_id_seq'::regclass);


--
-- Name: studyprop_feature_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE studyprop_feature ALTER COLUMN studyprop_feature_id SET DEFAULT nextval('studyprop_feature_studyprop_feature_id_seq'::regclass);


--
-- Name: synonym_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE synonym ALTER COLUMN synonym_id SET DEFAULT nextval('synonym_synonym_id_seq'::regclass);


--
-- Name: tableinfo_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE tableinfo ALTER COLUMN tableinfo_id SET DEFAULT nextval('tableinfo_tableinfo_id_seq'::regclass);


--
-- Name: treatment_id; Type: DEFAULT; Schema: public; Owner: gmod
--

ALTER TABLE treatment ALTER COLUMN treatment_id SET DEFAULT nextval('treatment_treatment_id_seq'::regclass);


--
-- Data for Name: acquisition; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY acquisition (acquisition_id, assay_id, protocol_id, channel_id, acquisitiondate, name, uri) FROM stdin;
\.


--
-- Data for Name: acquisition_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY acquisition_relationship (acquisition_relationship_id, subject_id, type_id, object_id, value, rank) FROM stdin;
\.


--
-- Data for Name: acquisitionprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY acquisitionprop (acquisitionprop_id, acquisition_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: analysis; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY analysis (analysis_id, name, description, program, programversion, algorithm, sourcename, sourceversion, sourceuri, timeexecuted) FROM stdin;
1	dabg	\N	dabg	dabg	\N	\N	\N	\N	2010-09-22 16:05:26.540862
2	dchip	\N	dchip	dchip	\N	\N	\N	\N	2010-09-22 16:05:26.542202
3	gcrma	\N	gcrma	gcrma	\N	\N	\N	\N	2010-09-22 16:05:26.543046
4	mas5	\N	mas5	mas5	\N	\N	\N	\N	2010-09-22 16:05:26.543547
5	mpam	\N	mpam	mpam	\N	\N	\N	\N	2010-09-22 16:05:26.544044
6	plier	\N	plier	plier	\N	\N	\N	\N	2010-09-22 16:05:26.544528
7	rma	\N	rma	rma	\N	\N	\N	\N	2010-09-22 16:05:26.545005
8	sea	\N	sea	sea	\N	\N	\N	\N	2010-09-22 16:05:26.546367
9	vsn	\N	vsn	vsn	\N	\N	\N	\N	2010-09-22 16:05:26.546844
\.


--
-- Data for Name: analysisfeature; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY analysisfeature (analysisfeature_id, feature_id, analysis_id, rawscore, normscore, significance, identity) FROM stdin;
\.


--
-- Data for Name: analysisfeatureprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY analysisfeatureprop (analysisfeatureprop_id, analysisfeature_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: analysisprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY analysisprop (analysisprop_id, analysis_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: arraydesign; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY arraydesign (arraydesign_id, manufacturer_id, platformtype_id, substratetype_id, protocol_id, dbxref_id, name, version, description, array_dimensions, element_dimensions, num_of_elements, num_array_columns, num_array_rows, num_grid_columns, num_grid_rows, num_sub_columns, num_sub_rows) FROM stdin;
\.


--
-- Data for Name: arraydesignprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY arraydesignprop (arraydesignprop_id, arraydesign_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: assay; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY assay (assay_id, arraydesign_id, protocol_id, assaydate, arrayidentifier, arraybatchidentifier, operator_id, dbxref_id, name, description) FROM stdin;
\.


--
-- Data for Name: assay_biomaterial; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY assay_biomaterial (assay_biomaterial_id, assay_id, biomaterial_id, channel_id, rank) FROM stdin;
\.


--
-- Data for Name: assay_project; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY assay_project (assay_project_id, assay_id, project_id) FROM stdin;
\.


--
-- Data for Name: assayprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY assayprop (assayprop_id, assay_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: biomaterial; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY biomaterial (biomaterial_id, taxon_id, biosourceprovider_id, dbxref_id, name, description) FROM stdin;
\.


--
-- Data for Name: biomaterial_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY biomaterial_dbxref (biomaterial_dbxref_id, biomaterial_id, dbxref_id) FROM stdin;
\.


--
-- Data for Name: biomaterial_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY biomaterial_relationship (biomaterial_relationship_id, subject_id, type_id, object_id) FROM stdin;
\.


--
-- Data for Name: biomaterial_treatment; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY biomaterial_treatment (biomaterial_treatment_id, biomaterial_id, treatment_id, unittype_id, value, rank) FROM stdin;
\.


--
-- Data for Name: biomaterialprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY biomaterialprop (biomaterialprop_id, biomaterial_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: cell_line; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line (cell_line_id, name, uniquename, organism_id, timeaccessioned, timelastmodified) FROM stdin;
\.


--
-- Data for Name: cell_line_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_cvterm (cell_line_cvterm_id, cell_line_id, cvterm_id, pub_id, rank) FROM stdin;
\.


--
-- Data for Name: cell_line_cvtermprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_cvtermprop (cell_line_cvtermprop_id, cell_line_cvterm_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: cell_line_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_dbxref (cell_line_dbxref_id, cell_line_id, dbxref_id, is_current) FROM stdin;
\.


--
-- Data for Name: cell_line_feature; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_feature (cell_line_feature_id, cell_line_id, feature_id, pub_id) FROM stdin;
\.


--
-- Data for Name: cell_line_library; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_library (cell_line_library_id, cell_line_id, library_id, pub_id) FROM stdin;
\.


--
-- Data for Name: cell_line_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_pub (cell_line_pub_id, cell_line_id, pub_id) FROM stdin;
\.


--
-- Data for Name: cell_line_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_relationship (cell_line_relationship_id, subject_id, object_id, type_id) FROM stdin;
\.


--
-- Data for Name: cell_line_synonym; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_line_synonym (cell_line_synonym_id, cell_line_id, synonym_id, pub_id, is_current, is_internal) FROM stdin;
\.


--
-- Data for Name: cell_lineprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_lineprop (cell_lineprop_id, cell_line_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: cell_lineprop_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cell_lineprop_pub (cell_lineprop_pub_id, cell_lineprop_id, pub_id) FROM stdin;
\.


--
-- Data for Name: channel; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY channel (channel_id, name, definition) FROM stdin;
\.


--
-- Data for Name: contact; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY contact (contact_id, type_id, name, description) FROM stdin;
9	\N	Matilde Cordeiro	\N
10	\N	Ken Moriuchi	\N
11	\N	Eric von Wettberg	\N
\.


--
-- Data for Name: contact_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY contact_relationship (contact_relationship_id, type_id, subject_id, object_id) FROM stdin;
\.


--
-- Data for Name: contactprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY contactprop (contactprop_id, contact_id, type_id, value) FROM stdin;
\.


--
-- Data for Name: control; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY control (control_id, type_id, assay_id, tableinfo_id, row_id, name, value, rank) FROM stdin;
\.


--
-- Data for Name: cv; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cv (cv_id, name, definition) FROM stdin;
1	null	\N
2	local	Locally created terms
3	Statistical Terms	Locally created terms for statistics
4	developmental stages	\N
5	autocreated	Terms that are automatically inserted by loading software
6	relationship	\N
7	synonym_type	\N
8	cvterm_property_type	\N
9	anonymous	\N
10	sequence	\N
11	unit.ontology	\N
12	quality	\N
13	pato.ontology	\N
14	diversity_experiment	\N
\.


--
-- Data for Name: cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cvterm (cvterm_id, cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype) FROM stdin;
1	1	null	\N	1	0	0
2	1	computer file	\N	2	0	0
3	2	glass	glass array	3	0	0
4	2	photochemical_oligo	in-situ photochemically synthesized oligoes	4	0	0
5	2	fetus	\N	5	0	0
6	4	neonate	\N	6	0	0
7	4	child	\N	7	0	0
8	4	adult_young	\N	8	0	0
9	4	adult	\N	9	0	0
10	4	adult_old	\N	10	0	0
11	2	survival_time	\N	11	0	0
12	3	n	sensu statistica	12	0	0
13	3	minimum	sensu statistica	13	0	0
14	3	maximum	sensu statistica	14	0	0
15	3	modality	sensu statistica	15	0	0
16	3	modality p	sensu statistica	16	0	0
17	3	mean	sensu statistica	17	0	0
18	3	median	sensu statistica	18	0	0
19	3	mode	sensu statistica	19	0	0
20	3	quartile 1	sensu statistica	20	0	0
21	3	quartile 3	sensu statistica	21	0	0
22	3	skewness	sensu statistica	22	0	0
23	3	kurtosis	sensu statistica	23	0	0
24	3	chi square p	sensu statistica	24	0	0
25	3	standard deviation	sensu statistica	25	0	0
26	3	expectation maximization gaussian mean	sensu statistica	26	0	0
27	3	expectation maximization p	sensu statistica	27	0	0
28	3	histogram	sensu statistica	28	0	0
79	10	integral_part_of		105	0	1
80	10	member_of	\N	106	0	1
81	10	non_functional_homolog_of		107	0	1
37	6	has_part	\N	40	0	1
82	10	orthologous_to	\N	108	0	1
104	10	G_quartet	G-quartets are unusual nucleic acid structures consisting of a planar arrangement where each guanine is hydrogen bonded by hoogsteen pairing to another guanine in the quartet.	131	0	0
83	10	paralogous_to	\N	109	0	1
38	6	integral_part_of		42	0	1
106	10	satellite_DNA	The many tandem repeats (identical or related) of a short basic repeating unit; many have a base composition or other property different from the genome average that allows them to be separated from the bulk (main band) genomic DNA.	135	0	0
84	10	part_of		110	0	1
29	8	comment	\N	29	0	0
39	6	has_integral_part	\N	44	0	1
85	10	partial_evidence_for_feature		111	0	1
33	8	is_anti_symmetric	\N	33	0	0
86	10	position_of	\N	112	0	1
40	6	proper_part_of		46	0	1
87	10	processed_from		113	0	1
41	6	has_proper_part	\N	48	0	1
88	10	processed_into		114	0	1
42	6	located_in		50	0	1
89	10	recombined_from	\N	115	0	1
90	10	recombined_to	\N	116	0	1
43	6	location_of	\N	52	0	1
91	10	regulated_by	\N	117	1	1
92	10	sequence_of	\N	118	0	1
44	6	contained_in		54	0	1
45	6	contains	\N	56	0	1
46	6	adjacent_to		58	0	1
47	6	transformation_of		60	0	1
93	10	similar_to	\N	119	0	1
48	6	transformed_into	\N	62	1	1
49	6	derives_from		64	0	1
50	6	derived_into	\N	66	0	1
94	10	trans_spliced_from	\N	120	0	1
51	6	preceded_by		68	0	1
95	10	trans_spliced_to	\N	121	0	1
52	6	precedes	\N	70	0	1
96	10	transcribed_from		122	0	1
53	6	has_participant		72	0	1
54	6	participates_in	\N	74	0	1
55	6	has_agent		76	0	1
56	6	agent_in	\N	78	0	1
57	6	instance_of		80	0	1
58	6	has_improper_part	\N	82	1	1
97	10	transcribed_to		123	0	1
98	10	translates_to		124	0	1
59	6	improper_part_of		84	1	1
32	8	is_reflexive	\N	32	0	0
99	10	translation_of		125	0	1
60	6	relationship		86	1	1
31	6	is_a		31	0	1
78	8	is_symmetric	\N	104	0	0
75	12	has_part		101	0	1
61	10	adjacent_to		87	0	1
62	10	associated_with	\N	88	0	1
63	10	complete_evidence_for_feature		89	0	1
100	10	variant_of		126	0	1
64	10	derives_from	\N	90	0	1
101	10	Sequence_Ontology (obsolete SO:0000000)	\N	127	1	0
65	10	edited_from	\N	91	0	1
66	10	edited_to	\N	92	0	1
67	10	evidence_for_feature		93	0	1
107	10	PCR_product	A region amplified by a PCR reaction.	138	0	0
68	10	exemplar_of		94	0	1
69	10	genome_of	\N	95	0	1
70	10	guided_by	\N	96	0	1
71	10	guides	\N	97	0	1
72	10	has_genome_location	\N	98	1	1
73	10	has_intergral_part		99	0	1
74	10	has_origin	\N	100	0	1
76	10	has_quality	\N	102	0	1
77	10	homologous_to	\N	103	0	1
102	10	region	A sequence_feature with an extent greater than zero. A nucleotide region is composed of bases and a polypeptide region is composed of amino acids.	128	0	0
155	10	sequence_variant_affecting_regulatory_region	A sequence_variant_effect which changes the regulatory region of a gene.	208	0	0
103	10	sequence_secondary_structure	A folded sequence.	130	0	0
109	10	read_pair	A pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert.	141	0	0
30	8	is_anonymous	\N	30	0	0
36	12	part_of		38	0	1
105	10	interior_coding_exon	\N	134	0	0
110	10	gene_sensu_your_favorite_organism (obsolete SO:0000008)	\N	143	1	0
111	10	gene_class (obsolete SO:0000009)	\N	144	1	0
112	10	protein_coding	\N	145	0	0
113	10	non_protein_coding	\N	146	0	0
114	10	scRNA_primary_transcript	The primary transcript of any one of several small cytoplasmic RNA molecules present in the cytoplasm and sometimes nucleus of a eukaryote.	147	0	0
147	10	delete (obsolete SO:0000045)	To remove a subsection of sequence.	199	1	0
115	10	scRNA	Any one of several small cytoplasmic RNA molecules present in the cytoplasm and sometimes nucleus of a eukaryote.	149	0	0
137	10	riboswitch	A riboswitch is a part of an mRNA that can act as a direct sensor of small molecules to control their own expression. A riboswitch is a cis element in the 5' end of an mRNA, that acts as a direct sensor of metabolites.	183	0	0
116	10	INR_motif	A sequence element characteristic of some RNA polymerase II promoters required for the correct positioning of the polymerase for the start of transcription. Overlaps the TSS. The mammalian consensus sequence is YYAN(T|A)YY; the Drosophila consensus sequence is TCA(G|T)t(T|C). In each the A is at position +1 with respect to the TSS. Functionally similar to the TATA box element.	150	0	0
144	10	pseudogene_attribute (obsolete SO:0000042)	An attribute of a pseudogene (SO:0000336).	195	1	0
138	10	matrix_attachment_site	A DNA region that is required for the binding of chromatin to the nuclear matrix.	186	0	0
148	10	insert (obsolete SO:0000046)	To insert a subsection of sequence.	200	1	0
145	10	processed_pseudogene	A pseudogene where by an mRNA was retrotransposed. The mRNA sequence is transcribed back into the genome, lacking introns and promotors, but often including a polyA tail.	196	0	0
117	10	DPE_motif	A sequence element characteristic of some RNA polymerase II promoters; Positioned from +28 to +32 with respect to the TSS (+1). Experimental results suggest that the DPE acts in conjunction with the INR_motif to provide a binding site for TFIID in the absence of a TATA box to mediate transcription of TATA-less promoters. Consensus sequence (A|G)G(A|T)(C|T)(G|A|C).	152	0	0
149	10	invert (obsolete SO:0000047)	To invert a subsection of sequence.	201	1	0
150	10	substitute (obsolete SO:0000048)	To substitute a subsection of sequence for another.	202	1	0
119	10	BRE_motif	A sequence element characteristic of some RNA polymerase II promoters, located immediately upstream of some TATA box elements at -37 to -32 with respect to the TSS (+1). Consensus sequence is (G|C)(G|C)(G|A)CGCC. Binds TFIIB.	155	0	0
151	10	translocate (obsolete SO:0000049)	To translocate a subsection of sequence.	203	1	0
146	10	pseudogene_by_unequal_crossing_over	A pseudogene caused by unequal crossing over at recombination.	198	0	0
120	10	PSE_motif	A sequence element characteristic of the promoters of snRNA genes transcribed by RNA polymerase II or by RNA polymerase III. Located between -45 and -60 relative to the TSS. The human PSE_motif consensus sequence is TCACCNTNA(C|G)TNAAAAG(T|G).	156	0	0
152	10	gene_part (obsolete SO:0000050)	A part of a gene, that has no other route in the ontology back to region. This concept is necessary for logical inference as these parts must have the properties of region. It also allows us to associate all the parts of genes with a gene.	204	1	0
153	10	probe	A DNA sequence used experimentally to detect the presence or absence of a complementary nucleic acid.	205	0	0
121	10	linkage_group	A group of loci that can be grouped in a linear order representing the different degrees of linkage among the genes concerned.	157	0	0
154	10	assortment_derived_deficiency (obsolete SO:0000052)	\N	207	1	0
122	10	RNA_internal_loop	A region of double stranded RNA where the bases do not conform to WC base pairing. The loop is closed on both sides by canonical base pairing. If the interruption to base pairing occurs on one strand only, it is known as a bulge.	160	0	0
139	10	locus_control_region	A DNA region that includes DNAse hypersensitive sites located 5' to a gene that confers the high-level, position-independent, and copy number-dependent expression to that gene.	189	0	0
123	10	asymmetric_RNA_internal_loop	An internal RNA loop where one of the strands includes more bases than the corresponding region on the other strand.	161	0	0
124	10	A_minor_RNA_motif	A region forming a motif, composed of adenines, where the minor groove edges are inserted into the minor groove of another helix.	162	0	0
125	10	K_turn_RNA_motif	The kink turn (K-turn) is an RNA structural motif that creates a sharp (~120 degree) bend between two continuous helices.	163	0	0
140	10	match_set (obsolete SO:0000038)	A collection of match parts.	191	1	0
141	10	match_part	A part of a match, for example an hsp from blast is a match_part.	192	0	0
126	10	sarcin_like_RNA_motif	A loop in ribosomal RNA containing the sites of attack for ricin and sarcin.	165	0	0
142	10	genomic_clone	A clone of a DNA region of a genome.	193	0	0
143	10	sequence_operation (obsolete SO:0000041)	An operation that can be applied to a sequence, that results in a change.	194	1	0
127	10	symmetric_RNA_internal_loop	An internal RNA loop where the extent of the loop on both stands is the same size.	167	0	0
128	10	RNA_junction_loop	\N	168	0	0
129	10	RNA_hook_turn	\N	169	0	0
130	10	base_pair	\N	170	0	0
131	10	WC_base_pair	The canonical base pair, where two bases interact via WC edges, with glycosidic bonds oriented cis relative to the axis of orientation.	172	0	0
132	10	sugar_edge_base_pair	A type of non-canonical base-pairing.	174	0	0
133	10	aptamer	DNA or RNA molecules that have been selected from random pools based on their ability to bind other molecules.	175	0	0
134	10	DNA_aptamer	DNA molecules that have been selected from random pools based on their ability to bind other molecules.	178	0	0
135	10	RNA_aptamer	RNA molecules that have been selected from random pools based on their ability to bind other molecules.	180	0	0
136	10	morpholino_oligo	Morpholino oligos are synthesized from four different Morpholino subunits, each of which contains one of the four genetic bases (A, C, G, T) linked to a 6-membered morpholine ring. Eighteen to 25 subunits of these four subunit types are joined in a specific order by non-ionic phosphorodiamidate intersubunit linkages to give a Morpholino.	181	0	0
156	10	aneuploid	A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number.	209	0	0
157	10	hyperploid	A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number as extra chromosomes are present.	211	0	0
158	10	hypoploid	A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number as some chromosomes are missing.	213	0	0
159	10	operator	A regulatory element of an operon to which activators or repressors bind thereby effecting translation of genes in that operon.	215	0	0
160	10	assortment_derived_aneuploid (obsolete SO:0000058)	\N	217	1	0
161	10	nuclease_binding_site	A region of a molecule that binds to a nuclease.	218	0	0
162	10	compound_chromosome_arm	\N	220	0	0
163	10	restriction_enzyme_binding_site	A region of a molecule that binds to a restriction enzyme.	221	0	0
199	10	nucleomorph_gene	A gene from nucleomorph sequence.	261	0	0
164	10	deficient_intrachromosomal_transposition	An intrachromosomal transposition whereby a translocation in which one of the four broken ends loses a segment before re-joining.	222	0	0
208	10	non_capped_primary_transcript (obsolete SO:0000106)	\N	275	1	0
165	10	deficient_interchromosomal_transposition	An interchromosomal transposition whereby a translocation in which one of the four broken ends loses a segment before re-joining.	224	0	0
200	10	plasmid_gene	A gene from plasmid sequence.	262	0	0
166	10	gene_by_transcript_attribute (obsolete SO:0000064)	\N	225	1	0
167	10	free_chromosome_arm	A chromosome structure variation whereby an arm exists as an individual chromosome element.	226	0	0
209	10	sequencing_primer	\N	276	0	0
168	10	gene_by_polyadenylation_attribute (obsolete SO:0000066)	\N	227	1	0
169	10	gene_to_gene_feature	\N	228	0	0
201	10	proviral_gene	A gene from proviral sequence.	263	0	0
170	10	overlapping	An attribute describing a gene that has a sequence that overlaps the sequence of another gene.	229	0	0
171	10	inside_intron	An attribute to describe a gene when it is located within the intron of another gene.	230	0	0
214	10	primer	A short preexisting polynucleotide chain to which new deoxyribonucleotides can be added by DNA polymerase.	281	0	0
172	10	inside_intron_antiparallel	An attribute to describe a gene when it is located within the intron of another gene and on the opposite strand.	231	0	0
202	10	endogenous_retroviral_gene	A proviral gene with origin endogenous retrovirus.	264	0	0
173	10	inside_intron_parallel	An attribute to describe a gene when it is located within the intron of another gene and on the same strand.	232	0	0
210	10	mRNA_with_frameshift	An mRNA with a frameshift.	277	0	0
174	10	end_overlapping_gene (obsolete SO:0000072)	\N	233	1	0
175	10	five_prime_three_prime_overlap	An attribute to describe a gene when the five prime region overlaps with another gene's 3' region.	234	0	0
203	10	transposable_element	A transposon or insertion sequence. An element that can insert in a variety of DNA sequences.	265	0	0
176	10	five_prime_five_prime_overlap	An attribute to describe a gene when the five prime region overlaps with another gene's five prime region.	235	0	0
212	10	sequence_feature	An extent of biological sequence.	279	0	0
177	10	three_prime_three_prime_overlap	An attribute to describe a gene when the 3' region overlaps with another gene's 3' region.	236	0	0
211	10	sequence_variant_obs (obsolete SO:0000109)	A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration.	278	1	0
178	10	three_prime_five_prime_overlap	An attribute to describe a gene when the 3' region overlaps with another gene's 5' region.	237	0	0
204	10	expressed_sequence_match	A match to an EST or cDNA sequence.	268	0	0
108	7	related	\N	139	0	0
179	10	antisense	A region sequence that is complementary to a sequence of messenger RNA.	238	0	0
180	10	polycistronic_transcript	A transcript that is polycistronic.	240	0	0
205	10	clone_insert_end	The end of the clone insert.	269	0	0
181	10	dicistronic_transcript	A transcript that is dicistronic.	241	0	0
182	10	operon_member	\N	242	0	0
206	10	polypeptide	A sequence of amino acids linked by peptide bonds which may lack appreciable tertiary structure and may not be liable to irreversible denaturation.	270	0	0
183	10	gene_array_member	\N	243	0	0
213	10	transposable_element_gene	A gene encoded within a transposable element. For example gag, int, env and pol are the transposable element genes of the TY element in yeast.	280	0	0
184	10	processed_transcript_attribute (obsolete SO:0000082)	\N	244	1	0
185	10	macronuclear_sequence	\N	245	0	0
207	10	chromosome_arm	A region of the chromosome between the centromere and the telomere. Human chromosomes have two arms, the p arm (short) and the q arm (long) which are separated from each other by the centromere.	273	0	0
186	10	micronuclear_sequence	\N	246	0	0
187	10	gene_by_genome_location (obsolete SO:0000085)	\N	247	1	0
188	10	gene_by_organelle_of_genome (obsolete SO:0000086)	\N	248	1	0
189	10	nuclear_gene	A gene from nuclear sequence.	249	0	0
190	10	mt_gene	A gene located in mitochondrial sequence.	251	0	0
191	10	kinetoplast_gene	A gene located in kinetoplast sequence.	253	0	0
192	10	plastid_gene	A gene from plastid sequence.	254	0	0
193	10	apicoplast_gene	A gene from apicoplast sequence.	255	0	0
194	10	ct_gene	A gene from chloroplast sequence.	256	0	0
195	10	chromoplast_gene	A gene from chromoplast_sequence.	257	0	0
196	10	cyanelle_gene	A gene from cyanelle sequence.	258	0	0
197	10	leucoplast_gene	A plastid gene from leucoplast sequence.	259	0	0
198	10	proplastid_gene	A gene from proplastid sequence.	260	0	0
215	10	proviral_region	A viral sequence which has integrated into a host genome.	284	0	0
216	10	methylated_C	A methylated deoxy-cytosine.	285	0	0
217	10	transcript_feature (obsolete SO:0000115)	\N	286	1	0
218	10	edited	An attribute describing a sequence that is modified by editing.	287	0	0
219	10	transcript_with_readthrough_stop_codon (obsolete SO:0000117)	\N	288	1	0
220	10	transcript_with_translational_frameshift	A transcript with a translational frameshift.	289	0	0
221	10	regulated	An attribute to describe a sequence that is regulated.	290	0	0
222	10	protein_coding_primary_transcript	A primary transcript that, at least in part, encodes one or more proteins.	291	0	0
223	10	forward_primer	A single stranded oligo used for polymerase chain reaction.	292	0	0
253	10	clone	A piece of DNA that has been inserted in a vector so that it can be propagated in a host bacterium or some other organism.	330	0	0
224	10	RNA_sequence_secondary_structure	A folded RNA sequence.	294	0	0
254	10	YAC	Yeast Artificial Chromosome, a vector constructed from the telomeric, centromeric, and replication origin sequences needed for replication in yeast cells.	332	0	0
225	10	transcriptionally_regulated	An attribute describing a gene that is regulated at transcription.	295	0	0
258	10	cosmid	A cloning vector that is a hybrid of lambda phages and a plasmid that can be propagated as a plasmid or packaged as a phage,since they retain the lambda cos sites.	337	0	0
226	10	transcriptionally_constitutive	Expressed in relatively constant amounts without regard to cellular environmental conditions such as the concentration of a particular substrate.	296	0	0
255	10	BAC	Bacterial Artificial Chromosome, a cloning vector that can be propagated as mini-chromosomes in a bacterial host.	333	0	0
227	10	transcriptionally_induced	An inducer molecule is required for transcription to occur.	297	0	0
228	10	transcriptionally_repressed	A repressor molecule is required for transcription to stop.	298	0	0
256	10	PAC	The P1-derived artificial chromosome are DNA constructs that are derived from the DNA of P1 bacteriophage. They can carry large amounts (about 100-300 kilobases) of other sequences for a variety of bioengineering purposes. It is one type of vector used to clone DNA fragments (100- to 300-kb insert size; average, 150 kb) in Escherichia coli cells.	334	0	0
229	10	silenced_gene	A gene that is silenced.	299	0	0
259	10	phagemid	A plasmid which carries within its sequence a bacteriophage replication origin. When the host bacterium is infected with "helper" phage, a phagemid is replicated along with the phage DNA and packaged into phage capsids.	339	0	0
230	10	gene_silenced_by_DNA_modification	A gene that is silenced by DNA modification.	300	0	0
260	10	fosmid	A cloning vector that utilises the E. coli F factor.	341	0	0
231	10	gene_silenced_by_DNA_methylation	A gene that is silenced by DNA methylation.	301	0	0
257	10	plasmid	A self replicating, using the hosts cellular machinery, often circular nucleic acid molecule that is distinct from a chromosome in the organism.	336	0	0
261	10	deletion	The point at which one or more contiguous nucleotides were excised.	343	0	0
232	10	post_translationally_regulated	An attribute describing a gene that is regulated after it has been translated.	302	0	0
233	10	translationally_regulated	An attribute describing a gene that is regulated as it is translated.	303	0	0
234	10	reverse_primer	A single stranded oligo used for polymerase chain reaction.	304	0	0
361	10	glutaminyl_tRNA	A tRNA sequence that has a glutamine anticodon, and a 3' glutamine binding region.	476	0	0
235	10	epigenetically_modified	This attribute describes a gene where heritable changes other than those in the DNA sequence occur. These changes include: modification to the DNA (such as DNA methylation, the covalent modification of cytosine), and post-translational modification of histones.	305	0	0
236	10	imprinted	Imprinted genes are epigenetically modified genes that are expressed monoallelically according to their parent of origin.	306	0	0
237	10	maternally_imprinted	The maternal copy of the gene is modified, rendering it transcriptionally silent.	308	0	0
238	10	paternally_imprinted	The paternal copy of the gene is modified, rendering it transcriptionally silent.	309	0	0
239	10	allelically_excluded	Allelic exclusion is a process occuring in diploid organisms, where a gene is inactivated and not expressed in that cell.	310	0	0
240	10	gene_rearranged_at_DNA_level	An epigenetically modified gene, rearranged at the DNA level.	311	0	0
241	10	ribosome_entry_site	Region in mRNA where ribosome assembles.	312	0	0
242	10	attenuator	A sequence segment located within the five prime end of an mRNA that causes premature termination of translation.	313	0	0
243	10	terminator	The sequence of DNA located either at the end of the transcript that causes RNA polymerase to terminate transcription.	316	0	0
244	10	DNA_sequence_secondary_structure	A folded DNA sequence.	318	0	0
245	10	assembly_component	A region of known length which may be used to manufacture a longer region.	319	0	0
246	10	primary_transcript_attribute (obsolete SO:0000144)	\N	320	1	0
247	10	recoded_codon	A codon that has been redefined at translation. The redefinition may be as a result of translational bypass, translational frameshifting or stop codon readthrough.	321	0	0
248	10	capped	An attribute describing when a sequence, usually an mRNA is capped by the addition of a modified guanine nucleotide at the 5' end.	322	0	0
249	10	exon	A region of the transcript sequence within a gene which is not removed from the primary RNA transcript by RNA splicing.	323	0	0
250	10	supercontig	One or more contigs that have been ordered and oriented using end-read information. Contains gaps that are filled with N's.	325	0	0
251	10	contig	A contiguous sequence derived from sequence assembly. Has no gaps, but may contain N's from unvailable bases.	326	0	0
252	10	read	A sequence obtained from a single sequencing experiment. Typically a read is produced when a base calling program interprets information from a chromatogram trace file produced from a sequencing machine.	328	0	0
262	10	lambda_clone (obsolete SO:0000160)	A linear clone derived from lambda bacteriophage. The genes involved in the lysogenic pathway are removed from the from the viral DNA. Up to 25 kb of foreign DNA can then be inserted into the lambda genome.	346	1	0
263	10	methylated_A	A modified RNA base in which adenine has been methylated.	348	0	0
264	10	splice_site	Consensus region of primary transcript bordering junction of splicing. A region that overlaps exactly 2 base and adjacent_to splice_junction.	349	0	0
265	10	five_prime_cis_splice_site	Intronic 2 bp region bordering the exon, at the 5' edge of the intron. A splice_site that is downstream_adjacent_to exon and starts intron.	352	0	0
266	10	three_prime_cis_splice_site	Intronic 2 bp region bordering the exon, at the 3' edge of the intron. A splice_site that is upstream_adjacent_to exon and finishes intron.	354	0	0
267	10	enhancer	A cis-acting sequence that increases the utilization of (some) eukaryotic promoters, and can function in either orientation and in any location (upstream or downstream) relative to the promoter.	355	0	0
268	10	enhancer_bound_by_factor	An enhancer bound by a factor.	357	0	0
269	10	promoter	A regulatory_region composed of the TSS(s) and binding sites for TF_complexes of the basal transcription machinery.	358	0	0
270	10	restriction_enzyme_cut_site (obsolete SO:0000168)	A specific nucleotide sequence of DNA at or near which a particular restriction enzyme cuts the DNA.	361	1	0
271	10	RNApol_I_promoter	A DNA sequence in eukaryotic DNA to which RNA polymerase I binds, to begin transcription.	362	0	0
272	10	RNApol_II_promoter	A DNA sequence in eukaryotic DNA to which RNA polymerase II binds, to begin transcription.	363	0	0
273	10	RNApol_III_promoter	A DNA sequence in eukaryotic DNA to which RNA polymerase III binds, to begin transcription.	364	0	0
274	10	CAAT_signal	Part of a conserved sequence located about 75-bp upstream of the start point of eukaryotic transcription units which may be involved in RNA polymerase binding; consensus=GG(C|T)CAATCT.	365	0	0
275	10	GC_rich_promoter_region	A conserved GC-rich region located upstream of the start point of eukaryotic transcription units which may occur in multiple copies or in either orientation; consensus=GGGCGG.	367	0	0
276	10	TATA_box	A conserved AT-rich septamer found about 25-bp before the start point of many eukaryotic RNA polymerase II transcript units; may be involved in positioning the enzyme for correct initiation; consensus=TATA(A|T)A(A|T).	368	0	0
277	10	minus_10_signal	A conserved region about 10-bp upstream of the start point of bacterial transcription units which may be involved in binding RNA polymerase; consensus=TAtAaT.	370	0	0
278	10	minus_35_signal	A conserved hexamer about 35-bp upstream of the start point of bacterial transcription units; consensus=TTGACa or TGTTGACA.	372	0	0
279	10	cross_genome_match	A nucleotide match against a sequence from another organism.	373	0	0
280	10	operon	A group of contiguous genes transcribed as a single (polycistronic) mRNA from a single regulatory region.	374	0	0
281	10	clone_insert_start	The start of the clone insert.	376	0	0
282	10	retrotransposon	A transposable element that is incorporated into a chromosome by a mechanism that requires reverse transcriptase.	377	0	0
283	10	translated_nucleotide_match	A match against a translated sequence.	380	0	0
284	10	DNA_transposon	A transposon where the mechanism of transposition is via a DNA intermediate.	381	0	0
285	10	non_transcribed_region	A region of the gene which is not transcribed.	382	0	0
286	10	U2_intron	A major type of spliceosomal intron spliced by the U2 spliceosome, that includes U1, U2, U4/U6 and U5 snRNAs.	383	0	0
287	10	primary_transcript	A transcript that in its initial state requires modification to be functional.	385	0	0
288	10	LTR_retrotransposon	A retrotransposon flanked by long terminal repeat sequences.	387	0	0
289	10	repeat_family (obsolete SO:0000187)	A group of characterized repeat sequences.	388	1	0
290	10	intron	A region of a primary transcript that is transcribed, but removed from within the transcript by splicing together the sequences (exons) on either side of it.	389	0	0
291	10	non_LTR_retrotransposon	A retrotransposon without long terminal repeat sequences.	391	0	0
292	10	five_prime_intron	\N	392	0	0
293	10	interior_intron	\N	393	0	0
294	10	three_prime_intron	\N	394	0	0
295	10	RFLP_fragment	A DNA fragment used as a reagent to detect the polymorphic genomic loci by hybridizing against the genomic DNA digested with a given restriction enzyme.	395	0	0
296	10	LINE_element	A dispersed repeat family with many copies, each from 1 to 6 kb long. New elements are generated by retroposition of a transcribed copy. Typically the LINE contains 2 ORF's one of which is reverse transcriptase, and 3'and 5' direct repeats.	398	0	0
297	10	coding_exon	An exon whereby at least one base is part of a codon (here, 'codon'is inclusive of the stop_codon).	399	0	0
298	10	five_prime_coding_exon_coding_region	The sequence of the five_prime_coding_exon that codes for protein.	400	0	0
299	10	three_prime_coding_exon_coding_region	The sequence of the three_prime_coding_exon that codes for protein.	401	0	0
300	10	noncoding_exon	An exon that does not contain any codons.	402	0	0
301	10	translocation	A region of nucleotide sequence that has translocated to a new position.	403	0	0
302	10	five_prime_coding_exon	The 5' most coding exon.	404	0	0
303	10	interior_exon	An exon that is bounded by 5' and 3' splice sites.	405	0	0
304	10	three_prime_coding_exon	The coding exon that is most 3-prime on a given transcript.	407	0	0
305	10	UTR	Messenger RNA sequences that are untranslated and lie five prime or three prime to sequences which are translated.	408	0	0
306	10	five_prime_UTR	A region at the 5' end of a mature transcript (preceding the initiation codon) that is not translated into a protein.	409	0	0
307	10	three_prime_UTR	A region at the 3' end of a mature transcript (following the stop codon) that is not translated into a protein.	411	0	0
308	10	SINE_element	A repetitive element, a few hundred base pairs long, that is dispersed throughout the genome. A common human SINE is the Alu element.	413	0	0
309	10	simple_sequence_length_variation	\N	415	0	0
362	10	glutamyl_tRNA	A tRNA sequence that has a glutamic acid anticodon, and a 3' glutamic acid binding region.	477	0	0
310	10	terminal_inverted_repeat_element	A DNA transposable element defined as having termini with perfect, or nearly perfect short inverted repeats, generally 10 - 40 nucleotides long.	416	0	0
311	10	rRNA_primary_transcript	A primary transcript encoding a ribosomal RNA.	418	0	0
312	10	tRNA_primary_transcript	A primary transcript encoding a transfer RNA (SO:0000253).	419	0	0
313	10	alanine_tRNA_primary_transcript	A primary transcript encoding alanyl tRNA.	420	0	0
314	10	arginine_tRNA_primary_transcript	A primary transcript encoding arginyl tRNA (SO:0000255).	421	0	0
315	10	asparagine_tRNA_primary_transcript	A primary transcript encoding asparaginyl tRNA (SO:0000256).	422	0	0
316	10	aspartic_acid_tRNA_primary_transcript	A primary transcript encoding aspartyl tRNA (SO:0000257).	423	0	0
317	10	cysteine_tRNA_primary_transcript	A primary transcript encoding cysteinyl tRNA (SO:0000258).	424	0	0
318	10	glutamic_acid_tRNA_primary_transcript	A primary transcript encoding glutaminyl tRNA (SO:0000260).	425	0	0
319	10	glutamine_tRNA_primary_transcript	A primary transcript encoding glutamyl tRNA (SO:0000260).	426	0	0
320	10	glycine_tRNA_primary_transcript	A primary transcript encoding glycyl tRNA (SO:0000263).	427	0	0
321	10	histidine_tRNA_primary_transcript	A primary transcript encoding histidyl tRNA (SO:0000262).	428	0	0
322	10	isoleucine_tRNA_primary_transcript	A primary transcript encoding isoleucyl tRNA (SO:0000263).	429	0	0
323	10	leucine_tRNA_primary_transcript	A primary transcript encoding leucyl tRNA (SO:0000264).	430	0	0
324	10	lysine_tRNA_primary_transcript	A primary transcript encoding lysyl tRNA (SO:0000265).	431	0	0
325	10	methionine_tRNA_primary_transcript	A primary transcript encoding methionyl tRNA (SO:0000266).	432	0	0
326	10	phenylalanine_tRNA_primary_transcript	A primary transcript encoding phenylalanyl tRNA (SO:0000267).	433	0	0
327	10	proline_tRNA_primary_transcript	A primary transcript encoding prolyl tRNA (SO:0000268).	434	0	0
328	10	serine_tRNA_primary_transcript	A primary transcript encoding seryl tRNA (SO:000269).	435	0	0
329	10	threonine_tRNA_primary_transcript	A primary transcript encoding threonyl tRNA (SO:000270).	436	0	0
330	10	tryptophan_tRNA_primary_transcript	A primary transcript encoding tryptophanyl tRNA (SO:000271).	437	0	0
331	10	tyrosine_tRNA_primary_transcript	A primary transcript encoding tyrosyl tRNA (SO:000272).	438	0	0
332	10	valine_tRNA_primary_transcript	A primary transcript encoding valyl tRNA (SO:000273).	439	0	0
333	10	snRNA_primary_transcript	A primary transcript encoding a small nuclear RNA (SO:0000274).	440	0	0
334	10	snoRNA_primary_transcript	A primary transcript encoding a small nucleolar mRNA (SO:0000275).	441	0	0
335	10	mature_transcript	A transcript which has undergone the necessary modifications, if any, for its function. In eukaryotes this includes, for example, processing of introns, cleavage, base modification, and modifications to the 5' and/or the 3' ends, other than addition of bases. In bacteria functional mRNAs are usually not modified.	442	0	0
336	10	mRNA	Messenger RNA is the intermediate molecule between DNA and protein. It includes UTR and coding sequences. It does not contain introns.	444	0	0
337	10	TF_binding_site	A region of a molecule that binds a TF complex [GO:0005667].	446	0	0
338	10	ORF	The inframe interval between the stop codons of a reading frame which when read as sequential triplets, has the potential of encoding a sequential string of amino acids. TER(NNN)nTER.	447	0	0
339	10	transcript_attribute	\N	449	0	0
340	10	foldback_element	A transposable element with extensive secondary structure, characterised by large modular imperfect long inverted repeats.	450	0	0
341	10	flanking_region	The sequences extending on either side of a specific region.	451	0	0
342	10	chromosome_variation	\N	452	0	0
343	10	internal_UTR	A UTR bordered by the terminal and initial codons of two CDSs in a polycistronic transcript. Every UTR is either 5', 3' or internal.	453	0	0
344	10	untranslated_region_polycistronic_mRNA	The untranslated sequence separating the 'cistrons' of multicistronic mRNA.	454	0	0
345	10	internal_ribosome_entry_site	Sequence element that recruits a ribosomal subunit to internal mRNA for translation initiation.	455	0	0
346	10	four_cutter_restriction_site (obsolete SO:0000244)	\N	457	1	0
347	10	mRNA_by_polyadenylation_status (obsolete SO:0000245)	\N	458	1	0
348	10	polyadenylated	A attribute describing the addition of a poly A tail to the 3' end of a mRNA molecule.	459	0	0
349	10	mRNA_not_polyadenylated (obsolete SO:0000247)	\N	460	1	0
350	10	sequence_length_variation	\N	461	0	0
351	10	six_cutter_restriction_site (obsolete SO:0000249)	\N	462	1	0
352	10	modified_RNA_base_feature	A post_transcriptionally modified base.	463	0	0
353	10	eight_cutter_restriction_site (obsolete SO:0000251)	\N	464	1	0
354	10	rRNA	RNA that comprises part of a ribosome, and that can provide both structural scaffolding and catalytic activity.	465	0	0
355	10	tRNA	Transfer RNA (tRNA) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-codon, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. Transfer RNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3' end, to which the tRNA's corresponding amino acid is attached (by aminoacyl-tRNA synthetases). Transfer RNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and 'wobble' base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position.	468	0	0
356	10	alanyl_tRNA	A tRNA sequence that has an alanine anticodon, and a 3' alanine binding region.	471	0	0
357	10	rRNA_small_subunit_primary_transcript	A primary transcript encoding a small ribosomal subunit RNA.	472	0	0
358	10	asparaginyl_tRNA	A tRNA sequence that has an asparagine anticodon, and a 3' asparagine binding region.	473	0	0
359	10	aspartyl_tRNA	A tRNA sequence that has an aspartic acid anticodon, and a 3' aspartic acid binding region.	474	0	0
360	10	cysteinyl_tRNA	A tRNA sequence that has a cysteine anticodon, and a 3' cysteine binding region.	475	0	0
363	10	glycyl_tRNA	A tRNA sequence that has a glycine anticodon, and a 3' glycine binding region.	478	0	0
364	10	histidyl_tRNA	A tRNA sequence that has a histidine anticodon, and a 3' histidine binding region.	479	0	0
365	10	isoleucyl_tRNA	A tRNA sequence that has an isoleucine anticodon, and a 3' isoleucine binding region.	480	0	0
366	10	leucyl_tRNA	A tRNA sequence that has a leucine anticodon, and a 3' leucine binding region.	481	0	0
367	10	lysyl_tRNA	A tRNA sequence that has a lysine anticodon, and a 3' lysine binding region.	482	0	0
368	10	methionyl_tRNA	A tRNA sequence that has a methionine anticodon, and a 3' methionine binding region.	483	0	0
369	10	phenylalanyl_tRNA	A tRNA sequence that has a phenylalanine anticodon, and a 3' phenylalanine binding region.	484	0	0
370	10	prolyl_tRNA	A tRNA sequence that has a proline anticodon, and a 3' proline binding region.	485	0	0
371	10	seryl_tRNA	A tRNA sequence that has a serine anticodon, and a 3' serine binding region.	486	0	0
372	10	threonyl_tRNA	A tRNA sequence that has a threonine anticodon, and a 3' threonine binding region.	487	0	0
373	10	tryptophanyl_tRNA	A tRNA sequence that has a tryptophan anticodon, and a 3' tryptophan binding region.	488	0	0
374	10	tyrosyl_tRNA	A tRNA sequence that has a tyrosine anticodon, and a 3' tyrosine binding region.	489	0	0
375	10	valyl_tRNA	A tRNA sequence that has a valine anticodon, and a 3' valine binding region.	490	0	0
376	10	snRNA	A small nuclear RNA molecule involved in pre-mRNA splicing and processing.	491	0	0
377	10	snoRNA	A snoRNA (small nucleolar RNA) is any one of a class of small RNAs that are associated with the eukaryotic nucleus as components of small nucleolar ribonucleoproteins. They participate in the processing or modifications of many RNAs, mostly ribosomal RNAs (rRNAs) though snoRNAs are also known to target other classes of RNA, including spliceosomal RNAs, tRNAs, and mRNAs via a stretch of sequence that is complementary to a sequence in the targeted RNA.	495	0	0
378	10	miRNA	Small, ~22-nt, RNA molecule that is the endogenous transcript of a miRNA gene. Micro RNAs are produced from precursor molecules (SO:0000647) that can form local hairpin structures, which ordinarily are processed (via the Dicer pathway) such that a single miRNA molecule accumulates from one arm of a hairpin precursor molecule. Micro RNAs may trigger the cleavage of their target molecules or act as translational repressors.	498	0	0
379	10	bound_by_factor	An attribute describing a sequence that is bound by another molecule.	501	0	0
380	10	transcript_bound_by_nucleic_acid	A transcript that is bound by a nucleic acid.	502	0	0
381	10	transcript_bound_by_protein	A transcript that is bound by a protein.	503	0	0
382	10	engineered_gene	A gene that is engineered.	504	0	0
383	10	engineered_foreign_gene	A gene that is engineered and foreign.	505	0	0
384	10	mRNA_with_minus_1_frameshift	An mRNA with a minus 1 frameshift.	506	0	0
385	10	engineered_foreign_transposable_element_gene	A transposible_element that is engineered and foreign.	507	0	0
386	10	type_I_enzyme_restriction_site (obsolete SO:0000284)	The recognition site is bipartite and interrupted.	508	1	0
387	10	foreign_gene	A gene that is foreign.	510	0	0
388	10	long_terminal_repeat	A sequence directly repeated at both ends of a defined sequence, of the sort typically found in retroviruses.	511	0	0
389	10	fusion_gene	A gene that is a fusion.	513	0	0
390	10	engineered_fusion_gene	A fusion gene that is engineered.	515	0	0
391	10	microsatellite	A repeat_region containing repeat_units (2 to 4 bp) that is repeated multiple times in tandem.	516	0	0
392	10	dinucleotide_repeat_microsatellite_feature	\N	519	0	0
393	10	trinucleotide_repeat_microsatellite_feature	\N	520	0	0
414	10	experimentally_determined	Attribute to describe a feature that has been experiemntally verified.	546	0	0
409	10	CpG_island	Regions of a few hundred to a few thousand bases in vertebrate genomes that are relatively GC and CpG rich; they are typically unmethylated and often found near the 5' ends of genes.	540	0	0
405	10	clip	Part of the primary transcript that is clipped off during processing.	536	0	0
394	10	repetitive_element (obsolete SO:0000292)	\N	521	1	0
395	10	engineered_foreign_repetitive_element	A repetitive element that is engineered and foreign.	522	0	0
406	10	type_II_enzyme_restriction_site (obsolete SO:0000304)	The recognition site is either palindromic, partially palindromic or an interrupted palindrome. Cleavage occurs within the recognition site.	537	1	0
396	10	inverted_repeat	The sequence is complementarily repeated on the opposite strand. It is a palindrome, and it may, or may not be hyphenated. Examples: GCTGATCAGC, or GCTGA-----TCAGC.	523	0	0
407	10	modified_base_site	A modified nucleotide, i.e. a nucleotide other than A, T, C. G or (in RNA) U.	538	0	0
397	10	U12_intron	A type of spliceosomal intron spliced by the U12 spliceosome, that includes U11, U12, U4atac/U6atac and U5 snRNAs.	525	0	0
408	10	methylated_base_feature	A nucleotide modified by methylation.	539	0	0
398	10	origin_of_replication	The origin of replication; starting site for duplication of a nucleic acid molecule to give two identical copies.	526	0	0
410	10	sequence_feature_locating_method (obsolete SO:0000308)	\N	542	1	0
411	10	computed_feature (obsolete SO:0000309)	\N	543	1	0
399	10	D_loop	Displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region; also used to describe the displacement of a region of one strand of duplex DNA by a single stranded invader in the reaction catalyzed by RecA protein.	528	0	0
412	10	predicted_ab_initio_computation (obsolete SO:0000310)	\N	544	1	0
400	10	recombination_feature	\N	530	0	0
413	10	computed_feature_by_similarity (obsolete SO:0000311)	.	545	1	0
401	10	specific_recombination_site	\N	531	0	0
402	10	recombination_feature_of_rearranged_gene	\N	532	0	0
403	10	vertebrate_immune_system_gene_recombination_feature	\N	533	0	0
404	10	J_gene_recombination_feature	Recombination signal including J-heptamer, J-spacer and J-nonamer in 5' of J-region of a J-gene or J-sequence.	534	0	0
415	10	stem_loop	A double-helical region of nucleic acid formed by base-pairing between adjacent (inverted) complementary sequences.	547	0	0
416	10	direct_repeat	A repeat where the same sequence is repeated in the same direction. Example: GCTGA-----GCTGA.	550	0	0
448	10	loxP_site	\N	594	0	0
417	10	TSS	The first base where RNA polymerase begins to synthesize the RNA transcript.	552	0	0
456	10	group_1_intron_homing_endonuclease_target_region	\N	606	0	0
418	10	CDS	A contiguous sequence which begins with, and includes, a start codon and ends with, and includes, a stop codon.	553	0	0
449	10	nucleotide_match	A match against a nucleotide sequence.	595	0	0
419	10	cDNA_clone	Complementary DNA; A piece of DNA copied from an mRNA and spliced into a vector for propagation in a suitable host.	554	0	0
459	10	flanked	An attribute describing a region that is bounded either side by a paricular kind of region.	609	0	0
420	10	start_codon	First codon to be translated by a ribosome.	556	0	0
450	10	nucleic_acid	An attribute describing a sequence consisting of nucleobases bound to repeating units. The forms found in nature are deoxyribonucleic acid (DNA), where the repeating units are 2-deoxy-D-ribose rings connected to a phosphate backbone, and ribonucleic acid (RNA), where the repeating units are D-ribose rings connected to a phosphate backbone.	596	0	0
457	10	haplotype_block	A region of the genome which is co-inherited as the result of the lack of historic recombination within it.	607	0	0
421	10	stop_codon	In mRNA, a set of three nucleotides that indicates the end of information for protein synthesis.	558	0	0
451	10	protein_match	A match against a protein sequence.	600	0	0
422	10	intronic_splice_enhancer	Sequences within the intron that modulate splice site selection for some introns.	560	0	0
458	10	RNA	An attribute describing a sequence consisting of nucleobases bound to a repeating unit made of a D-ribose ring connected to a phosphate backbone.	608	0	0
423	10	mRNA_with_plus_1_frameshift	An mRNA with a plus 1 frameshift.	561	0	0
452	10	FRT_site	An inversion site found on the Saccharomyces cerevisiae 2 micron plasmid.	601	0	0
424	10	nuclease_hypersensitive_site	\N	562	0	0
460	10	floxed	An attribute describing sequence that is flanked by Lox-P sites.	610	0	0
425	10	coding_start	The first base to be translated into protein.	563	0	0
461	10	codon	A set of (usually) three nucleotide bases in a DNA or RNA sequence, which together code for a unique amino acid or the termination of translation and are contained within the CDS.	612	0	0
453	10	synthetic_sequence	An attribute to decide a sequence of nucleotides, nucleotide analogs, or amino acids that has been designed by an experimenter and which may, or may not, correspond with any natural sequence.	602	0	0
426	10	tag	A nucleotide sequence that may be used to identify a larger sequence.	564	0	0
427	10	rRNA_large_subunit_primary_transcript	A primary transcript encoding a large ribosomal subunit RNA.	565	0	0
462	10	FRT_flanked	An attribute to describe sequence that is flanked by the FLP recombinase recognition site, FRT.	615	0	0
428	10	SAGE_tag	A short diagnostic sequence tag, serial analysis of gene expression (SAGE), that allows the quantitative and simultaneous analysis of a large number of transcripts.	566	0	0
454	10	DNA	An attribute describing a sequence consisting of nucleobases bound to a repeating unit made of a 2-deoxy-D-ribose ring connected to a phosphate backbone.	603	0	0
429	10	coding_end	The last base to be translated into protein. It does not include the stop codon.	568	0	0
455	10	sequence_assembly	A sequence of nucleotides that has been algorithmically derived from an alignment of two or more different sequences.	604	0	0
430	10	microarray_oligo	\N	569	0	0
431	10	mRNA_with_plus_2_frameshift	An mRNA with a plus 2 frameshift.	570	0	0
432	10	conserved_region	Region of sequence similarity by descent from a common ancestor.	571	0	0
433	10	STS	Short (typically a few hundred base pairs) DNA sequence that has a single occurrence in a genome and whose location and base sequence are known.	573	0	0
434	10	coding_conserved_region	Coding region of sequence similarity by descent from a common ancestor.	575	0	0
435	10	exon_junction	The boundary between two exons in a processed transcript.	576	0	0
436	10	nc_conserved_region	Non-coding region of sequence similarity by descent from a common ancestor.	577	0	0
437	10	mRNA_with_minus_2_frameshift	A mRNA with a minus 2 frameshift.	578	0	0
438	10	pseudogene	A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their "normal" paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its "normal" paralog).	579	0	0
439	10	RNAi_reagent	A double stranded RNA duplex, at least 20bp long, used experimentally to inhibit gene function by RNA interference.	581	0	0
440	10	MITE	A highly repetitive and short (100-500 base pair) transposable element with terminal inverted repeats (TIR) and target site duplication (TSD). MITEs do not encode proteins.	582	0	0
441	10	recombination_hotspot	A region in a genome which promotes recombination.	584	0	0
442	10	chromosome	Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication.	586	0	0
443	10	chromosome_band	A cytologically distinguishable feature of a chromosome, often made visible by staining, and usually alternating light and dark.	588	0	0
444	10	site_specific_recombination_target_region	\N	590	0	0
445	10	match	A region of sequence, aligned to another sequence with some statistical significance, using an algorithm such as BLAST or SIM4.	591	0	0
446	10	splice_enhancer	Region of a transcript that regulates splicing.	592	0	0
447	10	EST	A tag produced from a single sequencing read from a cDNA clone or PCR product; typically a few hundred base pairs long.	593	0	0
463	10	invalidated_by_chimeric_cDNA	A cDNA clone constructed from more than one mRNA. Usually an experimental artifact.	616	0	0
464	10	floxed_gene	A transgene that is floxed.	617	0	0
465	10	transposable_element_flanking_region	The region of sequence surrounding a transposible element.	618	0	0
466	10	integron	A region encoding an integrase which acts at a site adjacent to it (attI_site) to insert DNA which must include but is not limited to an attC_site.	619	0	0
467	10	insertion_site	The junction where an insertion occurred.	621	0	0
468	10	attI_site	A region within an integron, adjacent to an integrase, at which site specific recombination involving an attC_site takes place.	622	0	0
469	10	transposable_element_insertion_site	The junction in a genome where a transposable_element has inserted.	623	0	0
470	10	integrase_coding_region (obsolete SO:0000369)	\N	624	1	0
471	10	small_regulatory_ncRNA	A non-coding RNA, usually with a specific secondary structure, that acts to regulate gene expression.	625	0	0
472	10	conjugative_transposon	A transposon that encodes function required for conjugation.	626	0	0
473	10	enzymatic_RNA	An RNA sequence that has catalytic activity with or without an associated ribonucleoprotein.	628	0	0
474	10	recombinationally_inverted_gene	A recombinationally rearranged gene by inversion.	629	0	0
475	10	ribozyme	An RNA with catalytic activity.	630	0	0
476	10	rRNA_5_8S	5_8S ribosomal RNA (5. 8S rRNA) is a component of the large subunit of the eukaryotic ribosome. It is transcribed by RNA polymerase I as part of the 45S precursor that also contains 18S and 28S rRNA. Functionally, it is thought that 5.8S rRNA may be involved in ribosome translocation. It is also known to form covalent linkage to the p53 tumour suppressor protein. 5_8S rRNA is also found in archaea.	632	0	0
488	10	RprA_RNA	Translational regulation of the stationary phase sigma factor RpoS is mediated by the formation of a double-stranded RNA stem-loop structure in the upstream region of the rpoS messenger RNA, occluding the translation initiation site. Clones carrying rprA (RpoS regulator RNA) increased the translation of RpoS. The rprA gene encodes a 106 nucleotide regulatory RNA. As with DsrA Rfam:RF00014, RprA is predicted to form three stem-loops. Thus, at least two small RNAs, DsrA and RprA, participate in the positive regulation of RpoS translation. Unlike DsrA, RprA does not have an extensive region of complementarity to the RpoS leader, leaving its mechanism of action unclear. RprA is non-essential.	661	0	0
477	10	RNA_6S	A small (184-nt in E. coli) RNA that forms a hairpin type structure. 6S RNA associates with RNA polymerase in a highly specific manner. 6S RNA represses expression from a sigma70-dependent promoter during stationary phase.	635	0	0
478	10	CsrB_RsmB_RNA	An enterobacterial RNA that binds the CsrA protein. The CsrB RNAs contain a conserved motif CAGGXXG that is found in up to 18 copies and has been suggested to bind CsrA. The Csr regulatory system has a strong negative regulatory effect on glycogen biosynthesis, glyconeogenesis and glycogen catabolism and a positive regulatory effect on glycolysis. In other bacteria such as Erwinia caratovara the RsmA protein has been shown to regulate the production of virulence determinants, such extracellular enzymes. RsmA binds to RsmB regulatory RNA which is also a member of this family.	638	0	0
521	10	mature_protein_region	The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide.	736	0	0
479	10	DsrA_RNA	DsrA RNA regulates both transcription, by overcoming transcriptional silencing by the nucleoid-associated H-NS protein, and translation, by promoting efficient translation of the stress sigma factor, RpoS. These two activities of DsrA can be separated by mutation: the first of three stem-loops of the 85 nucleotide RNA is necessary for RpoS translation but not for anti-H-NS action, while the second stem-loop is essential for antisilencing and less critical for RpoS translation. The third stem-loop, which behaves as a transcription terminator, can be substituted by the trp transcription terminator without loss of either DsrA function. The sequence of the first stem-loop of DsrA is complementary with the upstream leader portion of RpoS messenger RNA, suggesting that pairing of DsrA with the RpoS message might be important for translational regulation.	640	0	0
480	10	GcvB_RNA	A small untranslated RNA involved in expression of the dipeptide and oligopeptide transport systems in Escherichia coli.	643	0	0
481	10	hammerhead_ribozyme	A small catalytic RNA motif that catalyzes self-cleavage reaction. Its name comes from its secondary structure which resembles a carpenter's hammer. The hammerhead ribozyme is involved in the replication of some viroid and some satellite RNAs.	646	0	0
482	10	group_IIA_intron	\N	649	0	0
483	10	group_IIB_intron	\N	650	0	0
484	10	MicF_RNA	A non-translated 93 nt antisense RNA that binds its target ompF mRNA and regulates ompF expression by inhibiting translation and inducing degradation of the message.	651	0	0
485	10	OxyS_RNA	A small untranslated RNA which is induced in response to oxidative stress in Escherichia coli. Acts as a global regulator to activate or repress the expression of as many as 40 genes, including the fhlA-encoded transcriptional activator and the rpoS-encoded sigma(s) subunit of RNA polymerase. OxyS is bound by the Hfq protein, that increases the OxyS RNA interaction with its target messages.	654	0	0
486	10	RNase_MRP_RNA	The RNA molecule essential for the catalytic activity of RNase MRP, an enzymatically active ribonucleoprotein with two distinct roles in eukaryotes. In mitochondria it plays a direct role in the initiation of mitochondrial DNA replication. In the nucleus it is involved in precursor rRNA processing, where it cleaves the internal transcribed spacer 1 between 18S and 5.8S rRNAs.	657	0	0
487	10	RNase_P_RNA	The RNA component of Ribonuclease P (RNase P), a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterised activity is the generation of mature 5 prime ends of tRNAs by cleaving the 5 prime leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzymes have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous. Helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs.	659	0	0
489	10	RRE_RNA	The Rev response element (RRE) is encoded within the HIV-env gene. Rev is an essential regulatory protein of HIV that binds an internal loop of the RRE leading, encouraging further Rev-RRE binding. This RNP complex is critical for mRNA export and hence for expression of the HIV structural proteins.	664	0	0
490	10	spot_42_RNA	A 109-nucleotide RNA of E. coli that seems to have a regulatory role on the galactose operon. Changes in Spot 42 levels are implicated in affecting DNA polymerase I levels.	666	0	0
491	10	telomerase_RNA	The RNA component of telomerase, a reverse transcriptase that synthesises telomeric DNA.	669	0	0
492	10	U1_snRNA	U1 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Its 5' end forms complementary base pairs with the 5' splice junction, thus defining the 5' donor site of an intron. There are significant differences in sequence and secondary structure between metazoan and yeast U1 snRNAs, the latter being much longer (568 nucleotides as compared to 164 nucleotides in human). Nevertheless, secondary structure predictions suggest that all U1 snRNAs share a 'common core' consisting of helices I, II, the proximal region of III, and IV.	672	0	0
493	10	U2_snRNA	U2 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Complementary binding between U2 snRNA (in an area lying towards the 5' end but 3' to hairpin I) and the branchpoint sequence (BPS) of the intron results in the bulging out of an unpaired adenine, on the BPS, which initiates a nucleophilic attack at the intronic 5' splice site, thus starting the first of two transesterification reactions that mediate splicing.	675	0	0
494	10	U4_snRNA	U4 small nuclear RNA (U4 snRNA) is a component of the major U2-dependent spliceosome. It forms a duplex with U6, and with each splicing round, it is displaced from U6 (and the spliceosome) in an ATP-dependent manner, allowing U6 to refold and create the active site for splicing catalysis. A recycling process involving protein Prp24 re-anneals U4 and U6.	678	0	0
495	10	U4atac_snRNA	An snRNA required for the splicing of the minor U12-dependent class of eukaryotic nuclear introns. It forms a base paired complex with U6atac_snRNA (SO:0000397).	681	0	0
496	10	U5_snRNA	U5 RNA is a component of both types of known spliceosome. The precise function of this molecule is unknown, though it is known that the 5' loop is required for splice site selection and p220 binding, and that both the 3' stem-loop and the Sm site are important for Sm protein binding and cap methylation.	683	0	0
497	10	U6_snRNA	U6 snRNA is a component of the spliceosome which is involved in splicing pre-mRNA. The putative secondary structure consensus base pairing is confined to a short 5' stem loop, but U6 snRNA is thought to form extensive base-pair interactions with U4 snRNA.	686	0	0
498	10	U6atac_snRNA	U6atac_snRNA is an snRNA required for the splicing of the minor U12-dependent class of eukaryotic nuclear introns. It forms a base paired complex with U4atac_snRNA (SO:0000394).	688	0	0
499	10	U11_snRNA	U11 snRNA plays a role in splicing of the minor U12-dependent class of eukaryotic nuclear introns, similar to U1 snRNA in the major class spliceosome it base pairs to the conserved 5' splice site sequence.	690	0	0
500	10	U12_snRNA	The U12 small nuclear (snRNA), together with U4atac/U6atac, U5, and U11 snRNAs and associated proteins, forms a spliceosome that cleaves a divergent class of low-abundance pre-mRNA introns.	693	0	0
501	10	sequence_attribute	An attribute describes a quality of sequence.	696	0	0
502	10	gene_attribute	\N	697	0	0
503	10	enhancer_attribute (obsolete SO:0000402)	\N	698	1	0
504	10	U14_snoRNA	U14 small nucleolar RNA (U14 snoRNA) is required for early cleavages of eukaryotic precursor rRNAs. In yeasts, this molecule possess a stem-loop region (known as the Y-domain) which is essential for function. A similar structure, but with a different consensus sequence, is found in plants, but is absent in vertebrates.	699	0	0
505	10	vault_RNA	A family of RNAs are found as part of the enigmatic vault ribonucleoprotein complex. The complex consists of a major vault protein (MVP), two minor vault proteins (VPARP and TEP1), and several small untranslated RNA molecules. It has been suggested that the vault complex is involved in drug resistance.	703	0	0
506	10	Y_RNA	Y RNAs are components of the Ro ribonucleoprotein particle (Ro RNP), in association with Ro60 and La proteins. The Y RNAs and Ro60 and La proteins are well conserved, but the function of the Ro RNP is not known. In humans the RNA component can be one of four small RNAs: hY1, hY3, hY4 and hY5. These small RNAs are predicted to fold into a conserved secondary structure containing three stem structures. The largest of the four, hY1, contains an additional hairpin.	706	0	0
507	10	twintron	An intron within an intron. Twintrons are group II or III introns, into which another group II or III intron has been transposed.	709	0	0
508	10	rRNA_18S	A large polynucleotide in eukaryotes, which functions as the small subunit of the ribosome.	713	0	0
509	10	site (obsolete SO:0000408)	The interbase position where something (eg an aberration) occurred.	715	1	0
510	10	binding_site	A region on the surface of a molecule that may interact with another molecule. When applied to polypeptides: Amino acids involved in binding or interactions. It can also apply to an amino acid bond which is represented by the positions of the two flanking amino acids.	716	0	0
511	10	protein_binding_site	A region of a molecule that binds to a protein.	720	0	0
512	10	rescue_region	A region that rescues.	721	0	0
513	10	restriction_fragment	A region of polynucleotide sequence produced by digestion with a restriction endonuclease.	722	0	0
514	10	sequence_difference	A region where the sequence differs from that of a specified sequence.	724	0	0
515	10	invalidated_by_genomic_contamination	An attribute to describe a feature that is invalidated due to genomic contamination.	725	0	0
516	10	invalidated_by_genomic_polyA_primed_cDNA	An attribute to describe a feature that is invalidated due to polyA priming.	726	0	0
517	10	invalidated_by_partial_processing	An attribute to describe a feature that is invalidated due to partial processing.	727	0	0
518	10	polypeptide_domain	A structurally or functionally defined protein region. In proteins with multiple domains, the combination of the domains determines the function of the protein. A region which has been shown to recur throughout evolution.	728	0	0
520	10	signal_peptide	The signal_peptide is a short region of the peptide located at the N-terminus that directs the protein to be secreted or part of membrane components.	733	0	0
522	10	five_prime_terminal_inverted_repeat	\N	738	0	0
523	10	three_prime_terminal_inverted_repeat	\N	739	0	0
524	10	U5_LTR_region	\N	740	0	0
525	10	R_LTR_region	\N	741	0	0
526	10	U3_LTR_region	\N	742	0	0
527	10	five_prime_LTR	\N	743	0	0
528	10	three_prime_LTR	\N	744	0	0
867	10	cosmid_clone (obsolete SO:0000765)	\N	1155	1	0
563	10	inversion_derived_bipartite_deficiency	A chromosomal deletion whereby a chromosome generated by recombination between two inversions; has a deficiency at each end of the inversion.	783	0	0
577	10	positively_autoregulated	The gene product is involved in its own transcriptional regulation, where it increases transcription.	798	0	0
529	10	R_five_prime_LTR_region	\N	745	0	0
564	10	pseudogenic_region	A non-functional descendent of a functional entity.	785	0	0
573	10	autoregulated	The gene product is involved in its own transcriptional regulation.	794	0	0
530	10	U5_five_prime_LTR_region	\N	746	0	0
565	10	encodes_alternately_spliced_transcripts	A gene that encodes more than one transcript.	786	0	0
574	10	tiling_path	A set of regions which overlap with minimal polymorphism to form a linear sequence.	795	0	0
531	10	U3_five_prime_LTR_region	\N	747	0	0
566	10	decayed_exon	A non-functional descendant of an exon.	787	0	0
580	10	C_gene	Genomic DNA of immunoglobulin/T-cell receptor gene including C-region (and introns if present) with 5' UTR (SO:0000204) and 3' UTR (SO:0000205).	801	0	0
532	10	R_three_prime_LTR_region	\N	748	0	0
567	10	inversion_derived_deficiency_plus_duplication	A chromosome deletion whereby a chromosome is generated by recombination between two inversions; there is a deficiency at one end of the inversion and a duplication at the other end of the inversion.	788	0	0
575	10	negatively_autoregulated	The gene product is involved in its own transcriptional regulation where it decreases transcription.	796	0	0
533	10	U3_three_prime_LTR_region	\N	749	0	0
568	10	V_gene	Germline genomic DNA including L-part1, V-intron and V-exon, with the 5' UTR and 3' UTR.	789	0	0
578	10	contig_read	A DNA sequencer read which is part of a contig.	799	0	0
534	10	U5_three_prime_LTR_region	\N	750	0	0
576	10	tiling_path_fragment	A piece of sequence that makes up a tiling_path (SO:0000472).	797	0	0
579	10	polycistronic_gene (obsolete SO:0000477)	A gene that is polycistronic.	800	1	0
535	10	non_LTR_retrotransposon_polymeric_tract	A polymeric tract, such as poly(dA), within a non_LTR_retrotransposon.	751	0	0
569	10	post_translationally_regulated_by_protein_stability	An attribute describing a gene sequence where the resulting protein is regulated by the stability of the resulting protein.	790	0	0
536	10	target_site_duplication	A sequence of the target DNA that is duplicated when a transposable element or phage inserts; usually found at each end the insertion.	752	0	0
868	10	pyrrolysyl_tRNA	A tRNA sequence that has a pyrrolysine anticodon, and a 3' pyrrolysine binding region.	1156	0	0
537	10	RR_tract	A polypurine tract within an LTR_retrotransposon.	754	0	0
665	10	vertebrate_immune_system_gene_recombination_spacer	\N	892	0	0
538	10	ARS	A sequence that can autonomously replicate, as a plasmid, when transformed into a bacterial host.	755	0	0
570	10	golden_path_fragment	One of the pieces of sequence that make up a golden path.	791	0	0
539	10	assortment_derived_duplication (obsolete SO:0000437)	\N	756	1	0
540	10	gene_not_polyadenylated (obsolete SO:0000438)	\N	757	1	0
541	10	inverted_ring_chromosome	\N	758	0	0
542	10	vector_replicon	A replicon that has been modified to act as a vector for foreign sequence.	759	0	0
571	10	post_translationally_regulated_by_protein_modification	An attribute describing a gene sequence where the resulting protein is modified to regulate it.	792	0	0
581	10	trans_spliced_transcript	A transcript that is trans-spliced.	802	0	0
543	10	ss_oligo	A single stranded oligonucleotide.	761	0	0
572	10	J_gene	Germline genomic DNA of an immunoglobulin/T-cell receptor gene including J-region with 5' UTR (SO:0000204) and 3' UTR (SO:0000205), also designated as J-segment.	793	0	0
544	10	ds_oligo	A double stranded oligonucleotide.	762	0	0
545	10	polymer_attribute	An attribute to describe the kind of biological sequence.	763	0	0
546	10	three_prime_noncoding_exon	Non-coding exon in the 3' UTR.	764	0	0
547	10	five_prime_noncoding_exon	Non-coding exon in the 5' UTR.	765	0	0
548	10	UTR_intron	Intron located in the untranslated region.	766	0	0
549	10	five_prime_UTR_intron	An intron located in the 5' UTR.	767	0	0
550	10	three_prime_UTR_intron	An intron located in the 3' UTR.	768	0	0
551	10	random_sequence	A sequence of nucleotides or amino acids which, by design, has a "random" order of components, given a predetermined input frequency of these components.	769	0	0
552	10	interband	A light region between two darkly staining bands in a polytene chromosome.	770	0	0
553	10	gene_with_polyadenylated_mRNA	A gene that encodes a polyadenylated mRNA.	771	0	0
554	10	transgene_attribute (obsolete SO:0000452)	\N	772	1	0
555	10	chromosomal_transposition	A chromosome structure variant whereby a region of a chromosome has been transferred to another position. Among interchromosomal rearrangements, the term transposition is reserved for that class in which the telomeres of the chromosomes involved are coupled (that is to say, form the two ends of a single DNA molecule) as in wild-type.	773	0	0
556	10	rasiRNA	A 17-28-nt, small interfering RNA derived from transcripts of repetitive elements.	774	0	0
557	10	gene_with_mRNA_with_frameshift	A gene that encodes an mRNA with a frameshift.	777	0	0
558	10	recombinationally_rearranged_gene	A gene that is recombinationally rearranged.	778	0	0
559	10	interchromosomal_duplication	A chromosome duplication involving an insertion from another chromosome.	779	0	0
560	10	D_gene	Germline genomic DNA including D-region with 5' UTR and 3' UTR, also designated as D-segment.	780	0	0
561	10	gene_with_trans_spliced_transcript	A gene with a transcript that is trans-spliced.	781	0	0
562	10	vertebrate_immunoglobulin_T_cell_receptor_segment	\N	782	0	0
582	10	tiling_path_clone	A clone which is part of a tiling path. A tiling path is a set of sequencing substrates, typically clones, which have been selected in order to efficiently cover a region of the genome in preparation for sequencing and assembly.	803	0	0
583	10	terminal_inverted_repeat	An inverted repeat (SO:0000294) occuring at the termini of a DNA transposon.	804	0	0
584	10	vertebrate_immunoglobulin_T_cell_receptor_gene_cluster	\N	805	0	0
585	10	nc_primary_transcript	A primary transcript that is never translated into a protein.	806	0	0
586	10	three_prime_coding_exon_noncoding_region	The sequence of the 3' exon that is not coding.	807	0	0
587	10	DJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one DJ-gene, and one J-gene.	808	0	0
588	10	five_prime_coding_exon_noncoding_region	The sequence of the 5' exon preceding the start codon.	809	0	0
589	10	VDJ_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene, one J-gene and one C-gene.	810	0	0
590	10	VDJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene and one J-gene.	811	0	0
591	10	VJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene and one C-gene.	812	0	0
592	10	VJ_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene, one J-gene and one C-gene.	813	0	0
593	10	VJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene and one J-gene.	814	0	0
594	10	D_gene_recombination_feature	\N	815	0	0
595	10	three_prime_D_heptamer	7 nucleotide recombination site like CACAGTG, part of a 3' D-recombination signal sequence of an immunoglobulin/T-cell receptor gene.	816	0	0
596	10	three_prime_D_nonamer	A 9 nucleotide recombination site (e.g. ACAAAAACC), part of a 3' D-recombination signal sequence of an immunoglobulin/T-cell receptor gene.	817	0	0
597	10	three_prime_D_spacer	A 12 or 23 nucleotide spacer between the 3'D-HEPTAMER and 3'D-NONAMER of a 3'D-RS.	818	0	0
598	10	five_prime_D_heptamer	7 nucleotide recombination site (e.g. CACTGTG), part of a 5' D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene.	819	0	0
599	10	five_prime_D_nonamer	9 nucleotide recombination site (e.g. GGTTTTTGT), part of a five_prime_D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene.	820	0	0
600	10	five_prime_D_spacer	12 or 23 nucleotide spacer between the 5' D-heptamer (SO:0000496) and 5' D-nonamer (SO:0000497) of a 5' D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene.	821	0	0
601	10	virtual_sequence	A continuous piece of sequence similar to the 'virtual contig' concept of the Ensembl database.	822	0	0
602	10	Hoogsteen_base_pair	A type of non-canonical base-pairing. This is less energetically favourable than watson crick base pairing. Hoogsteen GC base pairs only have two hydrogen bonds.	823	0	0
603	10	reverse_Hoogsteen_base_pair	A type of non-canonical base-pairing.	825	0	0
604	10	transcribed_region (obsolete SO:0000502)	A region of sequence that is transcribed. This region may cover the transcript of a gene, it may emcompas the sequence covered by all of the transcripts of a alternately spliced gene, or it may cover the region transcribed by a polycistronic transcript. A gene may have 1 or more transcribed regions and a transcribed_region may belong to one or more genes.	826	1	0
605	10	alternately_spliced_gene_encodeing_one_transcript (obsolete SO:0000503)	\N	827	1	0
606	10	D_DJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene and one C-gene.	828	0	0
607	10	D_DJ_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene and one DJ-gene.	829	0	0
608	10	D_DJ_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene, one J-gene and one C-gene.	830	0	0
609	10	pseudogenic_exon	A non functional descendant of an exon, part of a pseudogene.	831	0	0
610	10	D_DJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene, and one J-gene.	832	0	0
611	10	D_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one D-gene, one J-gene and one C-gene.	833	0	0
612	10	VD_gene	Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA including L-part1, V-intron and V-D-exon, with the 5' UTR (SO:0000204) and 3' UTR (SO:0000205).	834	0	0
613	10	J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one J-gene and one C-gene.	835	0	0
614	10	inversion_derived_deficiency_plus_aneuploid	A chromosomal deletion whereby a chromosome generated by recombination between two inversions; has a deficiency at one end and presumed to have a deficiency or duplication at the other end of the inversion.	836	0	0
615	10	J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one J-gene.	837	0	0
616	10	J_nonamer	9 nucleotide recombination site (e.g. GGTTTTTGT), part of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene.	838	0	0
617	10	J_heptamer	7 nucleotide recombination site (e.g. CACAGTG), part of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene.	839	0	0
618	10	pseudogenic_transcript	A non functional descendant of a transcript, part of a pseudogene.	840	0	0
619	10	J_spacer	12 or 23 nucleotide spacer between the J-nonamer and the J-heptamer of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene.	841	0	0
620	10	V_DJ_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one DJ-gene.	842	0	0
621	10	V_DJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene and one J-gene.	843	0	0
622	10	V_VDJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene and one C-gene.	844	0	0
801	10	junction	A sequence_feature with an extent of zero.	1071	0	0
623	10	V_VDJ_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one VDJ-gene.	845	0	0
624	10	V_VDJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene and one J-gene.	846	0	0
625	10	V_VJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene and one C-gene.	847	0	0
626	10	V_VJ_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one VJ-gene.	848	0	0
627	10	V_VJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene and one J-gene.	849	0	0
628	10	V_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one V-gene.	850	0	0
629	10	V_D_DJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene and one C-gene.	851	0	0
630	10	V_D_DJ_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene.	852	0	0
631	10	V_D_DJ_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene, one J-gene and one C-gene.	853	0	0
632	10	V_D_DJ_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene and one J-gene.	854	0	0
633	10	V_D_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one D-gene and one J-gene and one C-gene.	855	0	0
634	10	V_D_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one D-gene and one J-gene.	856	0	0
635	10	V_heptamer	7 nucleotide recombination site (e.g. CACAGTG), part of V-gene recombination feature of an immunoglobulin/T-cell receptor gene.	857	0	0
636	10	V_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene and one J-gene.	858	0	0
637	10	V_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one J-gene and one C-gene.	859	0	0
638	10	V_nonamer	9 nucleotide recombination site (e.g. ACAAAAACC), part of V-gene recombination feature of an immunoglobulin/T-cell receptor gene.	860	0	0
639	10	V_spacer	12 or 23 nucleotide spacer between the V-heptamer and the V-nonamer of a V-gene recombination feature of an immunoglobulin/T-cell receptor gene.	861	0	0
640	10	V_gene_recombination_feature	Recombination signal including V-heptamer, V-spacer and V-nonamer in 3' of V-region of a V-gene or V-sequence of an immunoglobulin/T-cell receptor gene.	862	0	0
641	10	DJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one DJ-gene and one C-gene.	863	0	0
642	10	DJ_J_C_cluster	Genomic DNA in rearranged configuration including at least one D-J-GENE, one J-GENE and one C-GENE.	864	0	0
643	10	VDJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene and one C-gene.	865	0	0
644	10	V_DJ_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene and one C-gene.	866	0	0
645	10	alternately_spliced_gene_encoding_greater_than_one_transcript (obsolete SO:0000543)	\N	867	1	0
646	10	helitron	A rolling circle transposon. Autonomous helitrons encode a 5'-to-3' DNA helicase and nuclease/ligase similar to those encoded by known rolling-circle replicons.	868	0	0
647	10	recoding_pseudoknot	The pseudoknots involved in recoding are unique in that, as they play their role as a structure, they are immediately unfolded and their now linear sequence serves as a template for decoding.	871	0	0
648	10	designed_sequence	\N	873	0	0
649	10	inversion_derived_bipartite_duplication	A chromosome generated by recombination between two inversions; there is a duplication at each end of the inversion.	874	0	0
650	10	gene_with_edited_transcript	A gene that encodes a transcript that is edited.	875	0	0
651	10	inversion_derived_duplication_plus_aneuploid	A chromosome generated by recombination between two inversions; has a duplication at one end and presumed to have a deficiency or duplication at the other end of the inversion.	876	0	0
652	10	aneuploid_chromosome	A chromosome structural variation whereby either a chromosome exists in addition to the normal chromosome complement or is lacking.	877	0	0
653	10	polyA_signal_sequence	The recognition sequence necessary for endonuclease cleavage of an RNA transcript that is followed by polyadenylation; consensus=AATAAA.	878	0	0
654	10	Shine_Dalgarno_sequence	Region in 5' UTR where ribosome assembles on mRNA.	879	0	0
655	10	polyA_site	The site on an RNA transcript to which will be added adenine residues by post-transcriptional polyadenylation.	881	0	0
656	10	assortment_derived_deficiency_plus_duplication (obsolete SO:0000554)	\N	882	1	0
657	10	five_prime_clip	5' most region of a precursor transcript that is clipped off during processing.	883	0	0
658	10	five_prime_D_recombination_signal_sequence	Recombination signal of an immunoglobulin/T-cell receptor gene, including the 5' D-nonamer (SO:0000497), 5' D-spacer (SO:0000498), and 5' D-heptamer (SO:0000396) in 5' of the D-region of a D-gene, or in 5' of the D-region of DJ-gene.	884	0	0
659	10	three_prime_clip	3'-most region of a precursor transcript that is clipped off during processing.	885	0	0
660	10	C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene including more than one C-gene.	886	0	0
661	10	D_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one D-gene.	888	0	0
662	10	D_J_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one D-gene and one J-gene.	889	0	0
663	10	heptamer_of_recombination_feature_of_vertebrate_immune_system_gene	Seven nucleotide recombination site (e.g. CACAGTG), part of V-gene, D-gene or J-gene recombination feature of an immunoglobulin or T-cell receptor gene.	890	0	0
664	10	nonamer_of_recombination_feature_of_vertebrate_immune_system_gene	\N	891	0	0
666	10	V_DJ_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene, one J-gene and one C-gene.	893	0	0
667	10	V_VDJ_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene, one J-gene and one C-gene.	894	0	0
668	10	V_VJ_J_C_cluster	Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene, one J-gene and one C-gene.	895	0	0
669	10	inversion_derived_aneuploid_chromosome	A chromosome may be generated by recombination between two inversions; presumed to have a deficiency or duplication at each end of the inversion.	896	0	0
670	10	bidirectional_promoter	\N	897	0	0
671	10	retrotransposed	An attribute of a feature that occured as the product of a reverse transcriptase mediated event.	898	0	0
672	10	three_prime_D_recombination_signal_sequence	Recombination signal of an immunoglobulin/T-cell receptor gene, including the 3' D-heptamer (SO:0000493), 3' D-spacer, and 3' D-nonamer (SO:0000494) in 3' of the D-region of a D-gene.	901	0	0
673	10	miRNA_encoding	\N	902	0	0
674	10	DJ_gene	Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA including D-J-region with 5' UTR and 3' UTR, also designated as D-J-segment.	903	0	0
675	10	rRNA_encoding	\N	904	0	0
676	10	VDJ_gene	Rearranged genomic DNA of immunoglobulin/T-cell receptor gene including L-part1, V-intron and V-D-J-exon, with the 5'UTR (SO:0000204) and 3'UTR (SO:0000205).	905	0	0
677	10	scRNA_encoding	\N	906	0	0
678	10	VJ_gene	Rearranged genomic DNA of immunoglobulin/T-cell receptor gene including L-part1, V-intron and V-J-exon, with the 5'UTR (SO:0000204) and 3'UTR (SO:0000205).	907	0	0
679	10	centromere	A region of chromosome where the spindle fibers attach during mitosis and meiosis.	908	0	0
680	10	snoRNA_encoding	\N	910	0	0
681	10	edited_transcript_feature	A locatable feature on a transcript that is edited.	911	0	0
682	10	methylation_guide_snoRNA_primary_transcript	A primary transcript encoding a methylation guide small nucleolar RNA.	912	0	0
683	10	cap	A structure consisting of a 7-methylguanosine in 5'-5' triphosphate linkage with the first nucleotide of an mRNA. It is added post-transcriptionally, and is not encoded in the DNA.	913	0	0
684	10	rRNA_cleavage_snoRNA_primary_transcript	A primary transcript encoding an rRNA cleavage snoRNA.	915	0	0
685	10	pre_edited_region	The region of a transcript that will be edited.	916	0	0
686	10	tmRNA	A tmRNA liberates a mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNAs genes have been found encoded in two parts. TmRNAs have been identified in eubacteria and some chloroplasts but are absent from archeal and eukaryote nuclear genomes.	918	0	0
687	10	C_D_box_snoRNA_encoding	\N	921	0	0
688	10	tmRNA_primary_transcript	A primary transcript encoding a tmRNA (SO:0000584).	922	0	0
689	10	group_I_intron	Group I catalytic introns are large self-splicing ribozymes. They catalyse their own excision from mRNA, tRNA and rRNA precursors in a wide range of organisms. The core secondary structure consists of 9 paired regions (P1-P9). These fold to essentially two domains, the P4-P6 domain (formed from the stacking of P5, P4, P6 and P6a helices) and the P3-P9 domain (formed from the P8, P3, P7 and P9 helices). Group I catalytic introns often have long ORFs inserted in loop regions.	923	0	0
690	10	autocatalytically_spliced_intron	A self spliced intron.	926	0	0
691	10	SRP_RNA_primary_transcript	A primary transcript encoding a signal recognition particle RNA.	927	0	0
692	10	SRP_RNA	The signal recognition particle (SRP) is a universally conserved ribonucleoprotein. It is involved in the co-translational targeting of proteins to membranes. The eukaryotic SRP consists of a 300-nucleotide 7S RNA and six proteins: SRPs 72, 68, 54, 19, 14, and 9. Archaeal SRP consists of a 7S RNA and homologues of the eukaryotic SRP19 and SRP54 proteins. In most eubacteria, the SRP consists of a 4.5S RNA and the Ffh protein (a homologue of the eukaryotic SRP54 protein). Eukaryotic and archaeal 7S RNAs have very similar secondary structures, with eight helical elements. These fold into the Alu and S domains, separated by a long linker region. Eubacterial SRP is generally a simpler structure, with the M domain of Ffh bound to a region of the 4.5S RNA that corresponds to helix 8 of the eukaryotic and archaeal SRP S domain. Some Gram-positive bacteria (e.g. Bacillus subtilis), however, have a larger SRP RNA that also has an Alu domain. The Alu domain is thought to mediate the peptide chain elongation retardation function of the SRP. The universally conserved helix which interacts with the SRP54/Ffh M domain mediates signal sequence recognition. In eukaryotes and archaea, the SRP19-helix 6 complex is thought to be involved in SRP assembly and stabilizes helix 8 for SRP54 binding.	928	0	0
693	10	pseudoknot	A tertiary structure in RNA where nucleotides in a loop form base pairs with a region of RNA downstream of the loop.	930	0	0
694	10	H_pseudoknot	A pseudoknot which contains two stems and at least two loops.	932	0	0
2250	14	root length	\N	2957	0	0
695	10	C_D_box_snoRNA	Most box C/D snoRNAs also contain long (>10 nt) sequences complementary to rRNA. Boxes C and D, as well as boxes C' and D', are usually located in close proximity, and form a structure known as the box C/D motif. This motif is important for snoRNA stability, processing, nucleolar targeting and function. A small number of box C/D snoRNAs are involved in rRNA processing; most, however, are known or predicted to serve as guide RNAs in ribose methylation of rRNA. Targeting involves direct base pairing of the snoRNA at the rRNA site to be modified and selection of a rRNA nucleotide a fixed distance from box D or D'.	934	0	0
2251	14	root diameter	\N	2958	0	0
2252	14	nodule number inside pot	\N	2959	0	0
2253	14	nodule number outside pot	\N	2960	0	0
696	10	H_ACA_box_snoRNA	Members of the box H/ACA family contain an ACA triplet, exactly 3 nt upstream from the 3' end and an H-box in a hinge region that links two structurally similar functional domains of the molecule. Both boxes are important for snoRNA biosynthesis and function. A few box H/ACA snoRNAs are involved in rRNA processing; most others are known or predicted to participate in selection of uridine nucleosides in rRNA to be converted to pseudouridines. Site selection is mediated by direct base pairing of the snoRNA with rRNA through one or both targeting domains.	936	0	0
697	10	C_D_box_snoRNA_primary_transcript	A primary transcript encoding a small nucleolar RNA of the box C/D family.	937	0	0
698	10	H_ACA_box_snoRNA_primary_transcript	A primary transcript encoding a small nucleolar RNA of the box H/ACA family.	938	0	0
699	10	transcript_edited_by_U_insertion/deletion (obsolete SO:0000597)	The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa.	939	1	0
700	10	edited_by_C_insertion_and_dinucleotide_insertion (obsolete SO:0000598)	\N	941	1	0
701	10	edited_by_C_to_U_substitution (obsolete SO:0000599)	\N	942	1	0
702	10	edited_by_A_to_I_substitution (obsolete SO:0000600)	\N	943	1	0
703	10	edited_by_G_addition (obsolete SO:0000601)	\N	944	1	0
704	10	guide_RNA	A short 3'-uridylated RNA that can form a duplex (except for its post-transcriptionally added oligo_U tail (SO:0000609)) with a stretch of mature edited mRNA.	945	0	0
705	10	group_II_intron	Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI). A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny.	947	0	0
706	10	editing_block	Edited mRNA sequence mediated by a single guide RNA (SO:0000602).	950	0	0
707	10	intergenic_region	A region containing or overlapping no genes that is bounded on either side by a gene, or bounded by a gene and the end of the chromosome.	951	0	0
708	10	editing_domain	Edited mRNA sequence mediated by two or more overlapping guide RNAs (SO:0000602).	953	0	0
709	10	unedited_region	The region of an edited transcript that will not be edited.	954	0	0
710	10	H_ACA_box_snoRNA_encoding	\N	955	0	0
711	10	oligo_U_tail	The string of non-encoded U's at the 3' end of a guide RNA (SO:0000602).	956	0	0
712	10	polyA_sequence	Sequence of about 100 nucleotides of A added to the 3' end of most eukaryotic mRNAs.	958	0	0
713	10	branch_site	A pyrimidine rich sequence near the 3' end of an intron to which the 5'end becomes covalently bound during nuclear splicing. The resulting structure resembles a lariat.	959	0	0
714	10	polypyrimidine_tract	The polypyrimidine tract is one of the cis-acting sequence elements directing intron removal in pre-mRNA splicing.	960	0	0
715	10	bacterial_RNApol_promoter	A DNA sequence to which bacterial RNA polymerase binds, to begin transcription.	963	0	0
716	10	bacterial_terminator	A terminator signal for bacterial transcription.	964	0	0
717	10	terminator_of_type_2_RNApol_III_promoter	A terminator signal for RNA polymerase III transcription.	965	0	0
718	10	transcription_end_site	The base where transcription ends.	966	0	0
719	10	RNApol_III_promoter_type_1	\N	967	0	0
720	10	RNApol_III_promoter_type_2	\N	968	0	0
721	10	A_box	A variably distant linear promoter region recognised by TFIIIC, with consensus sequence TGGCnnAGTGG.	969	0	0
722	10	B_box	A variably distant linear promoter region recognised by TFIIIC, with consensus sequence AGGTTCCAnnCC.	971	0	0
723	10	RNApol_III_promoter_type_3	\N	972	0	0
724	10	C_box	An RNA polymerase III type 1 promoter with consensus sequence CAnnCCn.	973	0	0
725	10	snRNA_encoding	\N	974	0	0
726	10	telomere	A specific structure at the end of a linear chromosome, required for the integrity and maintenance of the end.	975	0	0
727	10	silencer	A regulatory region which upon binding of transcription factors, suppress the transcription of the gene or genes they control.	977	0	0
728	10	chromosomal_regulatory_element	\N	979	0	0
729	10	insulator	A trancriptional cis regulatory region that when located between a CM and a gene's promoter prevents the CRM from modulating that genes expression.	980	0	0
730	10	chromosomal_structural_element	\N	982	0	0
731	10	five_prime_open_reading_frame	\N	983	0	0
732	10	upstream_AUG_codon	A start codon upstream of the ORF.	984	0	0
733	10	polycistronic_primary_transcript	A primary transcript encoding for more than one gene product.	985	0	0
734	10	monocistronic_primary_transcript	A primary transcript encoding for one gene product.	986	0	0
735	10	monocistronic_mRNA	An mRNA with either a single protein product, or for which the regions encoding all its protein products overlap.	987	0	0
736	10	polycistronic_mRNA	An mRNA that encodes multiple proteins from at least two non-overlapping regions.	989	0	0
737	10	mini_exon_donor_RNA	A primary transcript that donates the spliced leader to other mRNA.	991	0	0
738	10	spliced_leader_RNA	\N	992	0	0
739	10	engineered_plasmid	A plasmid that is engineered.	993	0	0
740	10	transcribed_spacer_region	Part of an rRNA transcription unit that is transcribed but discarded during maturation, not giving rise to any part of rRNA.	994	0	0
741	10	internal_transcribed_spacer_region	Non-coding regions of DNA sequence that separate genes coding for the 28S, 5.8S, and 18S ribosomal RNAs.	996	0	0
742	10	external_transcribed_spacer_region	Non-coding regions of DNA that precede the sequence that codes for the ribosomal RNA.	997	0	0
743	10	tetranucleotide_repeat_microsatellite_feature	\N	998	0	0
744	10	SRP_RNA_encoding	\N	999	0	0
745	10	minisatellite	A repeat region containing tandemly repeated sequences having a unit length of 10 to 40 bp.	1000	0	0
746	10	antisense_RNA	Antisense RNA is RNA that is transcribed from the coding, rather than the template, strand of DNA. It is therefore complementary to mRNA.	1002	0	0
747	10	antisense_primary_transcript	The reverse complement of the primary transcript.	1004	0	0
802	10	remark	A comment about the sequence.	1072	0	0
748	10	siRNA	A small RNA molecule that is the product of a longer exogenous or endogenous dsRNA, which is either a bimolecular duplex or very long hairpin, processed (via the Dicer pathway) such that numerous siRNAs accumulate from both strands of the dsRNA. SRNAs trigger the cleavage of their target molecules.	1005	0	0
749	10	miRNA_primary_transcript	A primary transcript encoding a micro RNA.	1007	0	0
750	10	stRNA_primary_transcript	A primary transcript encoding a small temporal mRNA (SO:0000649).	1008	0	0
751	10	stRNA	Non-coding RNAs of about 21 nucleotides in length that regulate temporal development; first discovered in C. elegans.	1009	0	0
752	10	small_subunit_rRNA	Ribosomal RNA transcript that structures the small subunit of the ribosome.	1012	0	0
753	10	large_subunit_rRNA	Ribosomal RNA transcript that structures the large subunit of the ribosome.	1013	0	0
754	10	rRNA_5S	5S ribosomal RNA (5S rRNA) is a component of the large ribosomal subunit in both prokaryotes and eukaryotes. In eukaryotes, it is synthesised by RNA polymerase III (the other eukaryotic rRNAs are cleaved from a 45S precursor synthesised by RNA polymerase I). In Xenopus oocytes, it has been shown that fingers 4-7 of the nine-zinc finger transcription factor TFIIIA can bind to the central region of 5S RNA. Thus, in addition to positively regulating 5S rRNA transcription, TFIIIA also stabilises 5S rRNA until it is required for transcription.	1014	0	0
755	10	rRNA_28S	A component of the large ribosomal subunit.	1017	0	0
756	10	maxicircle_gene	A mitochondrial gene located in a maxicircle.	1019	0	0
757	10	ncRNA	An RNA transcript that does not encode for a protein rather the RNA molecule is the gene product.	1020	0	0
758	10	stRNA_encoding	\N	1022	0	0
759	10	repeat_region	A region of sequence containing one or more repeat units.	1023	0	0
760	10	dispersed_repeat	A repeat that is located at dispersed sites in the genome.	1024	0	0
761	10	tmRNA_encoding	\N	1026	0	0
762	10	DNA_invertase_target_sequence (obsolete SO:0000660)	\N	1027	1	0
763	10	intron_attribute (obsolete SO:0000661)	\N	1028	1	0
764	10	spliceosomal_intron	An intron which is spliced by the spliceosome.	1029	0	0
765	10	tRNA_encoding	\N	1030	0	0
766	10	introgressed_chromosome_region	\N	1031	0	0
767	10	monocistronic_transcript	A transcript that is monocistronic.	1032	0	0
768	10	mobile_intron	An intron (mitochondrial, chloroplast, nuclear or prokaryotic) that encodes a double strand sequence specific endonuclease allowing for mobility.	1033	0	0
769	10	insertion	The sequence of one or more nucleotides added between two adjacent nucleotides in the sequence.	1034	0	0
770	10	EST_match	A match against an EST sequence.	1036	0	0
771	10	sequence_rearrangement_feature	\N	1037	0	0
772	10	chromosome_breakage_sequence	A sequence within the micronuclear DNA of ciliates at which chromosome breakage and telomere addition occurs during nuclear differentiation.	1038	0	0
773	10	internal_eliminated_sequence	A sequence eliminated from the genome of ciliates during nuclear differentiation.	1039	0	0
774	10	macronucleus_destined_segment	A sequence that is conserved, although rearranged relative to the micronucleus, in the macronucleus of a ciliate genome.	1040	0	0
775	10	transcript	An RNA synthesized on a DNA or RNA template by an RNA polymerase.	1041	0	0
776	10	non_canonical_splice_site (obsolete SO:0000674)	A splice site where the donor and acceptor sites differ from the canonical form.	1043	1	0
777	10	canonical_splice_site (obsolete SO:0000675)	The major class of splice site with dinucleotides GT and AG for donor and acceptor sites, respectively.	1044	1	0
778	10	canonical_three_prime_splice_site	The canonical 3' splice site has the sequence "AG".	1045	0	0
779	10	canonical_five_prime_splice_site	The canonical 5' splice site has the sequence "GT".	1046	0	0
780	10	non_canonical_three_prime_splice_site	A 3' splice site that does not have the sequence "AG".	1047	0	0
781	10	non_canonical_five_prime_splice_site	A 5' splice site which does not have the sequence "GT".	1048	0	0
782	10	non_canonical_start_codon	A start codon that is not the usual AUG sequence.	1049	0	0
783	10	aberrant_processed_transcript	A transcript that has been processed "incorrectly", for example by the failure of splicing of one or more exons.	1050	0	0
784	10	splicing_feature (obsolete SO:0000682)	\N	1051	1	0
785	10	exonic_splice_enhancer	Exonic splicing enhancers (ESEs) facilitate exon definition by assisting in the recruitment of splicing factors to the adjacent intron.	1052	0	0
786	10	nuclease_sensitive_site	A region of nucleotide sequence targeted by a nuclease enzyme.	1054	0	0
787	10	DNAseI_hypersensitive_site	\N	1055	0	0
788	10	translocation_element	A chromosomal translocation whereby  the chromosomes carrying non-homologous centromeres may be recovered independently. These chromosomes are described as translocation elements. This occurs for some translocations, particularly but not exclusively, reciprocal translocations,	1056	0	0
789	10	deletion_junction	The space between two bases in a sequence which marks the position where a deletion has occurred.	1057	0	0
790	10	golden_path	A set of subregions selected from sequence contigs which when concatenated form a nonredundant linear sequence.	1058	0	0
791	10	cDNA_match	A match against cDNA sequence.	1059	0	0
792	10	gene_with_polycistronic_transcript	A gene that encodes a polycistronic transcript.	1060	0	0
793	10	cleaved_initiator_methionine	The initiator methionine that has been cleaved from a mature polypeptide sequence.	1061	0	0
794	10	gene_with_dicistronic_transcript	A gene that encodes a dicistronic transcript.	1063	0	0
795	10	gene_with_recoded_mRNA	A gene that encodes an mRNA that is recoded.	1064	0	0
796	10	SNP	SNPs are single base pair positions in genomic DNA at which different sequence alternatives exist in normal individuals in some population(s), wherein the least frequent variant has an abundance of 1% or greater.	1065	0	0
797	10	reagent	A sequence used in experiment.	1066	0	0
798	10	oligo	A short oligonucleotide sequence, of length on the order of 10's of bases; either single or double stranded.	1067	0	0
799	10	gene_with_stop_codon_read_through	A gene that encodes a transcript with stop codon readthrough.	1069	0	0
800	10	gene_with_stop_codon_redefined_as_pyrrolysine	A gene encoding an mRNA that has the stop codon redefined as pyrrolysine.	1070	0	0
803	10	possible_base_call_error	A region of sequence where the validity of the base calling is questionable.	1073	0	0
804	10	possible_assembly_error	A region of sequence where there may have been an error in the assembly.	1074	0	0
805	10	experimental_result_region	A region of sequence implicated in an experimental result.	1075	0	0
806	10	gene	A region (or regions) that includes all of the sequence elements necessary to encode a functional transcript. A gene may include regulatory regions, transcribed regions and/or other functional sequence regions.	1076	0	0
807	10	tandem_repeat	Two or more adjcent copies of a region (of length greater than 1).	1079	0	0
808	10	trans_splice_acceptor_site	The 3' splice site of the acceptor primary transcript.	1081	0	0
809	10	trans_splice_donor_site	The 5' five prime splice site region of the donor RNA.	1082	0	0
810	10	SL1_acceptor_site	\N	1083	0	0
811	10	SL2_acceptor_site	\N	1084	0	0
812	10	gene_with_stop_codon_redefined_as_selenocysteine	A gene encoding an mRNA that has the stop codon redefined as selenocysteine.	1085	0	0
813	10	gene_with_mRNA_recoded_by_translational_bypass	A gene with mRNA recoded by translational bypass.	1086	0	0
814	10	gene_with_transcript_with_translational_frameshift	A gene encoding a transcript that has a translational frameshift.	1087	0	0
815	10	DNA_motif	A motif that is active in the DNA form of the sequence.	1088	0	0
816	10	nucleotide_motif	A region of nucleotide sequence corresponding to a known motif.	1090	0	0
817	10	RNA_motif	A motif that is active in RNA sequence.	1091	0	0
818	10	dicistronic_mRNA	An mRNA that has the quality dicistronic.	1092	0	0
819	10	reading_frame	A nucleic acid sequence that when read as sequential triplets, has the potential of encoding a sequential string of amino acids. It need not contain the start or stop codon.	1093	0	0
820	10	blocked_reading_frame	A reading_frame that is interrupted by one or more stop codons; usually identified through intergenomic sequence comparisons.	1095	0	0
821	10	ultracontig	An ordered and oriented set of scaffolds based on somewhat weaker sets of inferential evidence such as one set of mate pair reads together with supporting evidence from ESTs or location of markers from SNP or microsatellite maps, or cytogenetic localization of contained markers.	1096	0	0
822	10	foreign_transposable_element	A transposable element that is foreign.	1098	0	0
823	10	gene_with_dicistronic_primary_transcript	A gene that encodes a dicistronic primary transcript.	1099	0	0
824	10	gene_with_dicistronic_mRNA	A gene that encodes a polycistronic mRNA.	1100	0	0
825	10	iDNA	Genomic sequence removed from the genome, as a normal event, by a process of recombination.	1101	0	0
826	10	oriT	A region of a DNA molecule where transfer is initiated during the process of conjugation or mobilization.	1103	0	0
827	10	transit_peptide	The transit_peptide is a short region at the N-terminus of the peptide that directs the protein to an organelle (chloroplast, mitochondrion, microbody or cyanelle).	1105	0	0
828	10	repeat_unit	The simplest repeated component of a repeat region. A single repeat.	1107	0	0
829	10	CRM	A regulatory_region where more than 1 TF_binding_site together are regulatorily active.	1108	0	0
830	10	intein	A region of a peptide that is able to excise itself and rejoin the remaining portions with a peptide bond.	1110	0	0
831	10	intein_containing	An attribute of protein-coding genes where the initial protein product contains an intein.	1112	0	0
832	10	gap	A gap in the sequence of known length. The unknown bases are filled in with N's.	1113	0	0
833	10	fragmentary	An attribute to describe a feature that is incomplete.	1114	0	0
834	10	predicted	An attribute describing an unverified region.	1115	0	0
835	10	feature_attribute	An attribute describing a located_sequence_feature.	1117	0	0
836	10	exemplar_mRNA	An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group.	1118	0	0
837	10	sequence_location	\N	1119	0	0
838	10	organelle_sequence	\N	1120	0	0
839	10	mitochondrial_sequence	\N	1121	0	0
840	10	nuclear_sequence	\N	1122	0	0
841	10	nucleomorphic_sequence	\N	1123	0	0
842	10	plastid_sequence	\N	1124	0	0
843	10	kinetoplast	A kinetoplast is an interlocked network of thousands of minicircles and tens of maxi circles, located near the base of the flagellum of some protozoan species.	1125	0	0
844	10	maxicircle	A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method.	1129	0	0
845	10	apicoplast_sequence	\N	1131	0	0
846	10	chromoplast_sequence	\N	1132	0	0
847	10	chloroplast_sequence	\N	1133	0	0
848	10	cyanelle_sequence	\N	1134	0	0
849	10	leucoplast_sequence	\N	1135	0	0
850	10	proplastid_sequence	\N	1136	0	0
851	10	plasmid_location	\N	1137	0	0
852	10	amplification_origin	An origin_of_replication that is used for the amplification of a chromosomal nucleic acid sequence.	1138	0	0
853	10	proviral_location	\N	1139	0	0
854	10	gene_group_regulatory_region	\N	1140	0	0
855	10	clone_insert	The region of sequence that has been inserted and is being propogated by the clone.	1141	0	0
856	10	lambda_vector	The lambda bacteriophage is the vector for the linear lambda clone. The genes involved in the lysogenic pathway are removed from the from the viral DNA. Up to 25 kb of foreign DNA can then be inserted into the lambda genome.	1142	0	0
857	10	plasmid_vector	\N	1143	0	0
858	10	cDNA	DNA synthesized by reverse transcriptase using RNA as a template.	1145	0	0
859	10	single_stranded_cDNA	\N	1147	0	0
860	10	double_stranded_cDNA	\N	1148	0	0
861	10	plasmid_clone (obsolete SO:0000759)	\N	1149	1	0
862	10	YAC_clone (obsolete SO:0000760)	\N	1150	1	0
863	10	phagemid_clone (obsolete SO:0000761)	\N	1151	1	0
864	10	PAC_clone (obsolete SO:0000762)	\N	1152	1	0
865	10	fosmid_clone (obsolete SO:0000763)	\N	1153	1	0
866	10	BAC_clone (obsolete SO:0000764)	\N	1154	1	0
869	10	clone_insert_start (obsolete SO:0000767)	\N	1157	1	0
870	10	episome	A plasmid that may integrate with a chromosome.	1158	0	0
871	10	tmRNA_coding_piece	The region of a two-piece tmRNA that bears the reading frame encoding the proteolysis tag. The tmRNA gene undergoes circular permutation in some groups of bacteria. Processing of the transcripts from such a gene leaves the mature tmRNA in two pieces, base-paired together.	1159	0	0
872	10	tmRNA_acceptor_piece	The acceptor region of a two-piece tmRNA that when mature is charged at its 3' end with alanine. The tmRNA gene undergoes circular permutation in some groups of bacteria; processing of the transcripts from such a gene leaves the mature tmRNA in two pieces, base-paired together.	1163	0	0
873	10	QTL	A quantitative trait locus (QTL) is a polymorphic locus which contains alleles that differentially affect the expression of a continuously distributed phenotypic trait. Usually it is a marker described by statistical association to quantitative variation in the particular phenotypic trait that is thought to be controlled by the cumulative action of alleles at multiple loci.	1164	0	0
874	10	genomic_island	A genomic island is an integrated mobile genetic element, characterized by size (over 10 Kb). It that has features that suggest a foreign origin. These can include nucleotide distribution (oligonucleotides signature, CG content etc.) that differs from the bulk of the chromosome and/or genes suggesting DNA mobility.	1166	0	0
875	10	pathogenic_island	Mobile genetic elements that contribute to rapid changes in virulence potential. They are present on the genomes of pathogenic strains but absent from the genomes of non pathogenic members of the same or related species.	1169	0	0
876	10	metabolic_island	A transmissible element containing genes involved in metabolism, analogous to the pathogenicity islands of gram negative bacteria.	1170	0	0
877	10	adaptive_island	An adaptive island is a genomic island that provides an adaptive advantage to the host.	1171	0	0
878	10	symbiosis_island	A transmissible element containing genes involved in symbiosis, analogous to the pathogenicity islands of gram negative bacteria.	1172	0	0
879	10	pseudogenic_rRNA	A non functional descendent of an rRNA.	1173	0	0
880	10	pseudogenic_tRNA	A non functional descendent of a tRNA.	1174	0	0
881	10	engineered_episome	An episome that is engineered.	1175	0	0
882	10	transposable_element_attribute (obsolete SO:0000780)	\N	1176	1	0
883	10	transgenic	Attribute describing sequence that has been integrated with foreign sequence.	1177	0	0
884	10	natural	An attribute describing a feature that occurs in nature.	1178	0	0
885	10	engineered	An attribute to describe a region that was modified in vitro.	1179	0	0
886	10	foreign	An attribute to describe a region from another species.	1180	0	0
887	10	cloned_region	\N	1181	0	0
888	10	reagent_attribute (obsolete SO:0000786)	\N	1182	1	0
889	10	clone_attribute (obsolete SO:0000787)	\N	1183	1	0
890	10	cloned (obsolete SO:0000788)	\N	1184	1	0
891	10	validated	An attribute to describe a feature that has been proven.	1185	0	0
892	10	invalidated	An attribute describing a feature that is invalidated.	1186	0	0
893	10	cloned_genomic (obsolete SO:0000791)	\N	1187	1	0
894	10	cloned_cDNA (obsolete SO:0000792)	\N	1188	1	0
895	10	engineered_DNA (obsolete SO:0000793)	\N	1189	1	0
896	10	engineered_rescue_region	A rescue region that is engineered.	1190	0	0
897	10	rescue_mini_gene	A mini_gene that rescues.	1191	0	0
898	10	transgenic_transposable_element	TE that has been modified in vitro, including insertion of DNA derived from a source other than the originating TE.	1192	0	0
899	10	natural_transposable_element	TE that exists (or existed) in nature.	1194	0	0
900	10	engineered_transposable_element	TE that has been modified by manipulations in vitro.	1195	0	0
901	10	engineered_foreign_transposable_element	A transposable_element that is engineered and foreign.	1196	0	0
902	10	assortment_derived_duplication	A multi-chromosome duplication aberration generated by reassortment of other aberration components.	1197	0	0
903	10	assortment_derived_deficiency_plus_duplication	A multi-chromosome aberration generated by reassortment of other aberration components; presumed to have a deficiency and a duplication.	1199	0	0
904	10	assortment_derived_deficiency	A multi-chromosome deficiency aberration generated by reassortment of other aberration components.	1200	0	0
905	10	assortment_derived_aneuploid	A multi-chromosome aberration generated by reassortment of other aberration components; presumed to have a deficiency or a duplication.	1201	0	0
906	10	engineered_region	A region that is engineered.	1202	0	0
907	10	engineered_foreign_region	A region that is engineered and foreign.	1203	0	0
908	10	fusion	\N	1204	0	0
909	10	engineered_tag	A tag that is engineered.	1205	0	0
910	10	validated_cDNA_clone	A cDNA clone that has been validated.	1206	0	0
911	10	invalidated_cDNA_clone	A cDNA clone that is invalid.	1207	0	0
912	10	chimeric_cDNA_clone	A cDNA clone invalidated because it is chimeric.	1208	0	0
913	10	genomically_contaminated_cDNA_clone	A cDNA clone invalidated by genomic contamination.	1209	0	0
914	10	polyA_primed_cDNA_clone	A cDNA clone invalidated by polyA priming.	1210	0	0
915	10	partially_processed_cDNA_clone	A cDNA invalidated clone by partial processing.	1211	0	0
916	10	rescue	An attribute describing a region's ability, when introduced to a mutant organism, to re-establish (rescue) a phenotype.	1212	0	0
917	10	mini_gene	By definition, minigenes are short open-reading frames (ORF), usually encoding approximately 9 to 20 amino acids, which are expressed in vivo (as distinct from being synthesized as peptide or protein ex vivo and subsequently injected). The in vivo synthesis confers a distinct advantage: the expressed sequences can enter both antigen presentation pathways, MHC I (inducing CD8+ T- cells, which are usually cytotoxic T-lymphocytes (CTL)) and MHC II (inducing CD4+ T-cells, usually 'T-helpers' (Th)); and can encounter B-cells, inducing antibody responses. Three main vector approaches have been used to deliver minigenes: viral vectors, bacterial vectors and plasmid DNA.	1213	0	0
918	10	rescue_gene	A gene that rescues.	1215	0	0
919	10	wild_type	An attribute describing sequence with the genotype found in nature and/or standard laboratory stock.	1216	0	0
920	10	wild_type_rescue_gene	A gene that rescues.	1218	0	0
921	10	mitochondrial_chromosome	A chromosome originating in a mitochondria.	1219	0	0
922	10	chloroplast_chromosome	A chromosome originating in a chloroplast.	1220	0	0
923	10	chromoplast_chromosome	A chromosome originating in a chromoplast.	1221	0	0
924	10	cyanelle_chromosome	A chromosome originating in a cyanelle.	1222	0	0
925	10	leucoplast_chromosome	A chromosome with origin in a leucoplast.	1223	0	0
926	10	macronuclear_chromosome	A chromosome originating in a macronucleus.	1224	0	0
927	10	micronuclear_chromosome	A chromosome originating in a micronucleus.	1225	0	0
928	10	nuclear_chromosome	A chromosome originating in a nucleus.	1226	0	0
929	10	nucleomorphic_chromosome	A chromosome originating in a nucleomorph.	1227	0	0
930	10	chromosome_part	A region of a chromosome.	1228	0	0
931	10	gene_member_region	A region of a gene.	1229	0	0
932	10	promoter_region (obsolete SO:0000832)	A region of sequence which is part of a promoter.	1230	1	0
933	10	transcript_region	A region of a transcript.	1231	0	0
934	10	mature_transcript_region	A region of a mature transcript.	1232	0	0
935	10	primary_transcript_region	A part of a primary transcript.	1233	0	0
936	10	mRNA_region	A region of an mRNA.	1234	0	0
937	10	UTR_region	A region of UTR.	1235	0	0
938	10	rRNA_primary_transcript_region	A region of an rRNA primary transcript.	1236	0	0
939	10	polypeptide_region	Biological sequence region that can be assigned to a specific subsequence of a polypeptide.	1237	0	0
940	10	repeat_component	A region of a repeated sequence.	1241	0	0
941	10	spliceosomal_intron_region	A region within an intron.	1242	0	0
942	10	gene_component_region	\N	1243	0	0
943	10	bacterial_RNApol_promoter_region (obsolete SO:0000843)	A region which is part of a bacterial RNA polymerase promoter.	1244	1	0
944	10	RNApol_II_promoter_region (obsolete SO:0000844)	A region of sequence which is a promoter for RNA polymerase II.	1245	1	0
945	10	RNApol_III_promoter_type_1_region (obsolete SO:0000845)	A region of sequence which is a promoter for RNA polymerase III type 1.	1246	1	0
946	10	RNApol_III_promoter_type_2_region (obsolete SO:0000846)	A region of sequence which is a promoter for RNA polymerase III type 2.	1247	1	0
947	10	tmRNA_region	A region of a tmRNA.	1248	0	0
948	10	LTR_component	\N	1249	0	0
949	10	three_prime_LTR_component	\N	1250	0	0
950	10	five_prime_LTR_component	\N	1251	0	0
951	10	CDS_region	A region of a CDS.	1252	0	0
952	10	exon_region	A region of an exon.	1253	0	0
953	10	homologous_region	A region that is homologous to another region.	1254	0	0
954	10	paralogous_region	A homologous_region that is paralogous to another region.	1256	0	0
955	10	orthologous_region	A homologous_region that is orthologous to another region.	1258	0	0
956	10	conserved	\N	1260	0	0
957	10	homologous	Similarity due to common ancestry.	1261	0	0
958	10	orthologous	An attribute describing a kind of homology where divergence occured after a speciation event.	1262	0	0
959	10	paralogous	An attribute describing a kind of homology where divergence occurred after a duplication event.	1263	0	0
960	10	syntenic	Attribute describing sequence regions occurring in same order on chromosome of different species.	1264	0	0
961	10	capped_primary_transcript	A primary transcript that is capped.	1266	0	0
962	10	capped_mRNA	An mRNA that is capped.	1267	0	0
963	10	mRNA_attribute	An attribute describing an mRNA feature.	1268	0	0
964	10	exemplar	An attribute describing a sequence is representative of a class of similar sequences.	1269	0	0
965	10	frameshift	An attribute describing a sequence that contains a mutation involving the deletion or insertion of one or more bases, where this number is not divisible by 3.	1270	0	0
966	10	minus_1_frameshift	A frameshift caused by deleting one base.	1272	0	0
967	10	minus_2_frameshift	A frameshift caused by deleting two bases.	1273	0	0
968	10	plus_1_frameshift	A frameshift caused by inserting one base.	1274	0	0
969	10	plus_2_framshift	A frameshift caused by inserting two bases.	1275	0	0
970	10	trans_spliced	An attribute describing transcript sequence that is created by splicing exons from diferent genes.	1276	0	0
971	10	polyadenylated_mRNA	An mRNA that is polyadenylated.	1277	0	0
972	10	trans_spliced_mRNA	An mRNA that is trans-spliced.	1278	0	0
973	10	edited_transcript	A transcript that is edited.	1279	0	0
974	10	edited_transcript_by_A_to_I_substitution	A transcript that has been edited by A to I substitution.	1280	0	0
975	10	bound_by_protein	An attribute describing a sequence that is bound by a protein.	1281	0	0
976	10	bound_by_nucleic_acid	An attribute describing a sequence that is bound by a nucleic acid.	1282	0	0
977	10	alternatively_spliced	An attribute describing a situation where a gene may encode for more than 1 transcript.	1283	0	0
978	10	monocistronic	An attribute describing a sequence that contains the code for one gene product.	1284	0	0
979	10	dicistronic	An attribute describing a sequence that contains the code for two gene products.	1285	0	0
980	10	polycistronic	An attribute describing a sequence that contains the code for more than one gene product.	1286	0	0
981	10	recoded	An attribute describing an mRNA sequence that has been reprogrammed at translation, causing localized alterations.	1287	0	0
982	10	codon_redefined	An attribute describing the alteration of codon meaning.	1288	0	0
983	10	stop_codon_read_through	A stop codon redefined to be a new amino acid.	1289	0	0
984	10	stop_codon_redefined_as_pyrrolysine	A stop codon redefined to be the new amino acid, pyrrolysine.	1290	0	0
985	10	stop_codon_redefined_as_selenocysteine	A stop codon redefined to be the new amino acid, selenocysteine.	1291	0	0
986	10	recoded_by_translational_bypass	Recoded mRNA where a block of nucleotides is not translated.	1292	0	0
1638	10	gene_cassette_member	\N	2293	0	0
987	10	translationally_frameshifted	Recoding by frameshifting a particular site.	1293	0	0
988	10	maternally_imprinted_gene	A gene that is maternally_imprinted.	1294	0	0
989	10	paternally_imprinted_gene	A gene that is paternally imprinted.	1295	0	0
990	10	post_translationally_regulated_gene	A gene that is post translationally regulated.	1296	0	0
991	10	negatively_autoregulated_gene	A gene that is negatively autoreguated.	1297	0	0
992	10	positively_autoregulated_gene	A gene that is positively autoregulated.	1298	0	0
993	10	silenced	An attribute describing an epigenetic process where a gene is inactivated at transcriptional or translational level.	1299	0	0
994	10	silenced_by_DNA_modification	An attribute describing an epigenetic process where a gene is inactivated by DNA modifications, resulting in repression of transcription.	1301	0	0
995	10	silenced_by_DNA_methylation	An attribute describing an epigenetic process where a gene is inactivated by DNA methylation, resulting in repression of transcription.	1302	0	0
996	10	translationally_regulated_gene	A gene that is translationally regulated.	1303	0	0
997	10	allelically_excluded_gene	A gene that is allelically_excluded.	1304	0	0
998	10	epigenetically_modified_gene	A gene that is epigenetically modified.	1305	0	0
999	10	nuclear_mitochondrial (obsolete SO:0000899)	An attribute describing a nuclear pseudogene of a mitochndrial gene.	1306	1	0
1000	10	processed (obsolete SO:0000900)	An attribute describing a pseudogene where by an mRNA was retrotransposed. The mRNA sequence is transcribed back into the genome, lacking introns and promotors, but often including a polyA tail.	1307	1	0
1001	10	unequally_crossed_over (obsolete SO:0000901)	An attribute describing a pseudogene that was created by tandem duplication and unequal crossing over during recombination.	1308	1	0
1002	10	transgene	A transgene is a gene that has been transferred naturally or by any of a number of genetic engineering techniques from one organism to another.	1309	0	0
1003	10	endogenous_retroviral_sequence	\N	1311	0	0
1004	10	rearranged_at_DNA_level	An attribute to describe the sequence of a feature, where the DNA is rearranged.	1312	0	0
1005	10	status	An attribute describing the status of a feature, based on the available evidence.	1313	0	0
1006	10	independently_known	Attribute to describe a feature that is independently known - not predicted.	1314	0	0
1007	10	supported_by_sequence_similarity	An attribute to describe a feature that has been predicted using sequence similarity techniques.	1315	0	0
1008	10	supported_by_domain_match	An attribute to describe a feature that has been predicted using sequence similarity of a known domain.	1316	0	0
1009	10	supported_by_EST_or_cDNA	An attribute to describe a feature that has been predicted using sequence similarity to EST or cDNA data.	1317	0	0
1010	10	orphan	\N	1318	0	0
1011	10	predicted_by_ab_initio_computation	An attribute describing a feature that is predicted by a computer program that did not rely on sequence similarity.	1319	0	0
1012	10	asx_turn	A motif of three consecutive residues and one H-bond in which: residue(i) is Aspartate or Asparagine (Asx), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2).	1320	0	0
1013	10	cloned_cDNA_insert	A clone insert made from cDNA.	1323	0	0
1014	10	cloned_genomic_insert	A clone insert made from genomic DNA.	1324	0	0
1015	10	engineered_insert	A clone insert that is engineered.	1325	0	0
1016	10	edit_operation (obsolete SO:0000916)	\N	1326	1	0
1017	10	insert_U (obsolete SO:0000917)	An edit to insert a U.	1327	1	0
1018	10	delete_U (obsolete SO:0000918)	An edit to delete a uridine.	1328	1	0
1019	10	substitute_A_to_I (obsolete SO:0000919)	An edit to substitute an I for an A.	1329	1	0
1020	10	insert_C (obsolete SO:0000920)	An edit to insert a cytidine.	1330	1	0
1021	10	insert_dinucleotide (obsolete SO:0000921)	An edit to insert a dinucleotide.	1331	1	0
1022	10	substitute_C_to_U (obsolete SO:0000922)	An edit to substitute an U for a C.	1332	1	0
1023	10	insert_G (obsolete SO:0000923)	An edit to insert a G.	1333	1	0
1024	10	insert_GC (obsolete SO:0000924)	An edit to insert a GC dinucleotide.	1334	1	0
1025	10	insert_GU (obsolete SO:0000925)	An edit to insert a GU dinucleotide.	1335	1	0
1026	10	insert_CU (obsolete SO:0000926)	An edit to insert a CU dinucleotide.	1336	1	0
1027	10	insert_AU (obsolete SO:0000927)	An edit to insert a AU dinucleotide.	1337	1	0
1028	10	insert_AA (obsolete SO:0000928)	An edit to insert a AA dinucleotide.	1338	1	0
1029	10	edited_mRNA	An mRNA that is edited.	1339	0	0
1030	10	guide_RNA_region	A region of guide RNA.	1340	0	0
1031	10	anchor_region	A region of a guide_RNA that base-pairs to a target mRNA.	1341	0	0
1032	10	pre_edited_mRNA	\N	1343	0	0
1033	10	intermediate	An attribute to describe a feature between stages of processing.	1344	0	0
1034	10	miRNA_target_site	A miRNA target site is a binding site where the molecule is a micro RNA.	1345	0	0
1035	10	edited_CDS	A CDS that is edited.	1347	0	0
1036	10	vertebrate_immunoglobulin_T_cell_receptor_rearranged_segment	\N	1348	0	0
1037	10	vertebrate_immune_system_feature (obsolete SO:0000937)	\N	1349	1	0
1038	10	vertebrate_immunoglobulin_T_cell_receptor_rearranged_gene_cluster	\N	1350	0	0
1039	10	vertebrate_immune_system_gene_recombination_signal_feature	\N	1351	0	0
1040	10	recombinationally_rearranged	\N	1352	0	0
1041	10	recombinationally_rearranged_vertebrate_immune_system_gene	A recombinationally rearranged gene of the vertebrate immune system.	1353	0	0
1042	10	attP_site	An integration/excision site of a phage chromosome at which a recombinase acts to insert the phage DNA at a cognate integration/excision site on a bacterial chromosome.	1354	0	0
1043	10	attB_site	An integration/excision site of a bacterial chromosome at which a recombinase acts to insert foreign DNA containing a cognate integration/excision site.	1355	0	0
1044	10	attL_site	A region that results from recombination between attP_site and attB_site, composed of the 5' portion of attB_site and the 3' portion of attP_site.	1356	0	0
1334	10	amino_acid	A sequence feature that corresponds to a single amino acid residue in a polypeptide.	1829	0	0
1045	10	attR_site	A region that results from recombination between attP_site and attB_site, composed of the 5' portion of attP_site and the 3' portion of attB_site.	1357	0	0
1046	10	integration_excision_site	A region specifically recognised by a recombinase, which inserts or removes another region marked by a distinct cognate integration/excision site.	1358	0	0
1047	10	resolution_site	A region specifically recognised by a recombinase, which separates a physically contiguous circle of DNA into two physically separate circles.	1359	0	0
1048	10	inversion_site	A region specifically recognised by a recombinase, which inverts the region flanked by a pair of sites.	1360	0	0
1049	10	dif_site	A site at which replicated bacterial circular chromosomes are decatenated by site specific resolvase.	1361	0	0
1050	10	attC_site	An attC site is a sequence required for the integration of a DNA of an integron.	1362	0	0
1051	10	eukaryotic_terminator	\N	1363	0	0
1052	10	oriV	An origin of vegetative replication in plasmids and phages.	1364	0	0
1053	10	oriC	An origin of bacterial chromosome replication.	1365	0	0
1054	10	DNA_chromosome	Structural unit composed of a self-replicating, DNA molecule.	1366	0	0
1055	10	double_stranded_DNA_chromosome	Structural unit composed of a self-replicating, double-stranded DNA molecule.	1367	0	0
1056	10	single_stranded_DNA_chromosome	Structural unit composed of a self-replicating, single-stranded DNA molecule.	1368	0	0
1057	10	linear_double_stranded_DNA_chromosome	Structural unit composed of a self-replicating, double-stranded, linear DNA molecule.	1369	0	0
1058	10	circular_double_stranded_DNA_chromosome	Structural unit composed of a self-replicating, double-stranded, circular DNA molecule.	1370	0	0
1059	10	linear_single_stranded_DNA_chromosome	Structural unit composed of a self-replicating, single-stranded, linear DNA molecule.	1371	0	0
1060	10	circular_single_stranded_DNA_chromosome	Structural unit composed of a self-replicating, single-stranded, circular DNA molecule.	1372	0	0
1061	10	RNA_chromosome	Structural unit composed of a self-replicating, RNA molecule.	1373	0	0
1062	10	single_stranded_RNA_chromosome	Structural unit composed of a self-replicating, single-stranded RNA molecule.	1374	0	0
1063	10	linear_single_stranded_RNA_chromosome	Structural unit composed of a self-replicating, single-stranded, linear RNA molecule.	1375	0	0
1064	10	linear_double_stranded_RNA_chromosome	Structural unit composed of a self-replicating, double-stranded, linear RNA molecule.	1376	0	0
1065	10	double_stranded_RNA_chromosome	Structural unit composed of a self-replicating, double-stranded RNA molecule.	1377	0	0
1066	10	circular_single_stranded_RNA_chromosome	Structural unit composed of a self-replicating, single-stranded, circular DNA molecule.	1378	0	0
1067	10	circular_double_stranded_RNA_chromosome	Structural unit composed of a self-replicating, double-stranded, circular RNA molecule.	1379	0	0
1068	10	sequence_replication_mode (obsolete SO:0000968)	\N	1380	1	0
1069	10	rolling_circle (obsolete SO:0000969)	\N	1381	1	0
1070	10	theta_replication (obsolete SO:0000970)	\N	1383	1	0
1071	10	DNA_replication_mode (obsolete SO:0000971)	\N	1384	1	0
1072	10	RNA_replication_mode (obsolete SO:0000972)	\N	1385	1	0
1073	10	insertion_sequence	A terminal_inverted_repeat_element that is bacterial and only encodes the functions required for its transposition between these inverted repeats.	1386	0	0
1074	10	minicircle_gene	\N	1388	0	0
1075	10	cryptic	A feature_attribute describing a feature that is not manifest under normal conditions.	1389	0	0
1076	10	anchor_binding_site	\N	1390	0	0
1077	10	template_region	A region of a guide_RNA that specifies the insertions and deletions of bases in the editing of a target mRNA.	1391	0	0
1078	10	gRNA_encoding	A non-protein_coding gene that encodes a guide_RNA.	1392	0	0
1079	10	minicircle	A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs.	1393	0	0
1080	10	rho_dependent_bacterial_terminator	\N	1396	0	0
1081	10	rho_independent_bacterial_terminator	\N	1397	0	0
1082	10	strand_attribute	\N	1398	0	0
1083	10	single	\N	1399	0	0
1084	10	double	\N	1400	0	0
1085	10	topology_attribute	\N	1401	0	0
1086	10	linear	A quality of a nucleotide polymer that has a 3'-terminal residue and a 5'-terminal residue.	1402	0	0
1087	10	circular	A quality of a nucleotide polymer that has no terminal nucleotide residues.	1403	0	0
1088	10	class_II_RNA	Small non-coding RNA (59-60 nt long) containing 5' and 3' ends that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm.	1404	0	0
1089	10	class_I_RNA	Small non-coding RNA (55-65 nt long) containing highly conserved 5' and 3' ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm.	1406	0	0
1090	10	genomic_DNA	\N	1407	0	0
1091	10	BAC_cloned_genomic_insert	\N	1408	0	0
1092	10	consensus	\N	1409	0	0
1093	10	consensus_region	\N	1410	0	0
1094	10	consensus_mRNA	\N	1411	0	0
1095	10	predicted_gene	\N	1412	0	0
1096	10	gene_fragment	\N	1413	0	0
1097	10	recursive_splice_site	A recursive splice site is a splice site which subdivides a large intron. Recursive splicing is a mechanism that splices large introns by sub dividing the intron at non exonic elements and alternate exons.	1414	0	0
1098	10	BAC_end	A region of sequence from the end of a BAC clone that may provide a highly specific marker.	1416	0	0
1099	10	rRNA_16S	A large polynucleotide in Bacteria and Archaea, which functions as the small subunit of the ribosome.	1417	0	0
1100	10	rRNA_23S	A large polynucleotide in Bacteria and Archaea, which functions as the large subunit of the ribosome.	1419	0	0
1101	10	rRNA_25S	A large polynucleotide which functions as part of the large subunit of the ribosome in some eukaryotes.	1420	0	0
1102	10	solo_LTR	A recombination product between the 2 LTR of the same element.	1421	0	0
1103	10	low_complexity	\N	1422	0	0
1104	10	low_complexity_region	\N	1423	0	0
1335	10	major_TSS	\N	1831	0	0
1336	10	minor_TSS	\N	1832	0	0
1105	10	prophage	A phage genome after it has established in the host genome in a latent/immune state either as a plasmid or as an integrated "island".	1424	0	0
1106	10	cryptic_prophage	A remnant of an integrated prophage in the host genome or an "island" in the host genome that includes phage like-genes.	1427	0	0
1107	10	tetraloop	A base-paired stem with loop of 4 non-hydrogen bonded nucleotides.	1429	0	0
1108	10	DNA_constraint_sequence	A double-stranded DNA used to control macromolecular structure and function.	1431	0	0
1109	10	i_motif	A cytosine rich domain whereby strands associate both inter- and intramolecularly at moderately acidic pH.	1432	0	0
1110	10	PNA_oligo	Peptide nucleic acid, is a chemical not known to occur naturally but is artificially synthesized and used in some biological research and medical treatments. The PNA backbone is composed of repeating N-(2-aminoethyl)-glycine units linked by peptide bonds. The purine and pyrimidine bases are linked to the backbone by methylene carbonyl bonds.	1434	0	0
1111	10	DNAzyme	A DNA sequence with catalytic activity.	1436	0	0
1112	10	MNP	A multiple nucleotide polymorphism with alleles of common length > 1, for example AAA/TTT.	1437	0	0
1113	10	intron_domain	\N	1439	0	0
1114	10	wobble_base_pair	A type of non-canonical base pairing, most commonly between G and U, which is important for the secondary structure of RNAs. It has similar thermodynamic stability to the Watson-Crick pairing. Wobble base pairs only have two hydrogen bonds. Other wobble base pair possibilities are I-A, I-U and I-C.	1440	0	0
1115	10	internal_guide_sequence	A purine-rich sequence in the group I introns which determines the locations of the splice sites in group I intron splicing and has catalytic activity.	1443	0	0
1116	10	silent_mutation	\N	1444	0	0
1117	10	epitope	A region of a macromolecule that is recognized by the immune system.	1446	0	0
1118	10	copy_number_variation	A variation that increases or decreases the copy number of a given region.	1448	0	0
1119	10	sequence_variant_affecting_copy_number	\N	1450	0	0
1120	10	chromosome_breakpoint	\N	1451	0	0
1121	10	inversion_breakpoint	The point within a chromosome where an inversion begins or ends.	1453	0	0
1122	10	allele	An allele is one of a set of coexisting sequence variants of a gene.	1454	0	0
1123	10	haplotype	A haplotype is one of a set of coexisting sequence variants of a haplotype block.	1456	0	0
1124	10	polymorphic_sequence_variant	A sequence variant that is segregating in one or more natural populations of a species.	1458	0	0
1125	10	genome	A genome is the sum of genetic material within a cell or virion.	1459	0	0
1126	10	genotype	A genotype is a variant genome, complete or incomplete.	1461	0	0
1127	10	diplotype	A diplotype is a pair of haplotypes from a given individual. It is a genotype where the phase is known.	1463	0	0
1128	10	direction_attribute	\N	1464	0	0
1129	10	forward	Forward is an attribute of the feature, where the feature is in the 5' to 3' direction.	1465	0	0
1130	10	reverse	Reverse is an attribute of the feature, where the feature is in the 3' to 5' direction. Again could be applied to primer.	1466	0	0
1131	10	mitochondrial_DNA	\N	1467	0	0
1132	10	chloroplast_DNA	\N	1469	0	0
1133	10	mirtron	A debranched intron which mimics the structure of pre-miRNA and enters the miRNA processing pathway without Drosha mediated cleavage.	1470	0	0
1134	10	piRNA	A small non coding RNA, part of a silencing system that prevents the spreading of selfish genetic elements.	1472	0	0
1135	10	arginyl_tRNA	A tRNA sequence that has an arginine anticodon, and a 3' arginine binding region.	1474	0	0
1136	10	mobile_genetic_element	A nucleotide region with either intra-genome or intracellular moblity, of varying length, which often carry the information necessary for transfer and recombination with the host genome.	1475	0	0
1137	10	extrachromosomal_mobile_genetic_element	An MGE that is not integrated into the host chromosome.	1478	0	0
1138	10	integrated_mobile_genetic_element	An MGE that is integrated into the host chromosome.	1479	0	0
1139	10	integrated_plasmid	A plasmid sequence that is integrated within the host chromosome.	1480	0	0
1140	10	viral_sequence	The region of nucleotide sequence of a virus, a submicroscopic particle that replicates by infecting a host cell.	1481	0	0
1141	10	phage_sequence	The nucleotide sequence of a virus that infects bacteria.	1482	0	0
1142	10	attCtn_site	An attachment site located on a conjugative transposon and used for site-specific integration of a conjugative transposon.	1484	0	0
1143	10	nuclear_mt_pseudogene	A nuclear pseudogene of a mitochndrial gene.	1485	0	0
1144	10	cointegrated_plasmid	A MGE region consisting of two fused plasmids resulting from a replicative transposition event.	1486	0	0
1145	10	IRLinv_site	Component of the inversion site located at the left of a region susceptible to site-specific inversion.	1488	0	0
1146	10	IRRinv_site	Component of the inversion site located at the right of a region susceptible to site-specific inversion.	1489	0	0
1147	10	inversion_site_part	A region located within an inversion site.	1490	0	0
1148	10	defective_conjugative_transposon	An island that contains genes for integration/excision and the gene and site for the initiation of intercellular transfer by conjugation. It can be complemented for transfer by a conjugative transposon.	1491	0	0
1149	10	repeat_fragment	A portion of a repeat, interrupted by the insertion of another element.	1493	0	0
1150	10	nested_region (obsolete SO:0001051)	\N	1494	1	0
1151	10	nested_repeat (obsolete SO:0001052)	\N	1495	1	0
1152	10	nested_transposon (obsolete SO:0001053)	\N	1496	1	0
1153	10	transposon_fragment	\N	1497	0	0
1154	10	transcriptional_cis_regulatory_region	A regulatory_region that modulates the transcription of a gene or genes.	1498	0	0
1155	10	splicing_regulatory_region	A regulatory_region that modulates splicing.	1499	0	0
1156	10	enhanceosome (obsolete SO:0001057)	\N	1500	1	0
1157	10	promoter_targeting_sequence	A transcriptional_cis_regulatory_region that restricts the activity of a CRM to a single promoter and which functions only when both itself and an insulator are located between the CRM and the promoter.	1501	0	0
1158	10	sequence_alteration	A sequence_alteration is a sequence_feature whose extent is the deviation from another sequence.	1502	0	0
1159	10	sequence_variant	A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration.	1505	0	0
1160	10	propeptide_cleavage_site	The propeptide_cleavage_site is the arginine/lysine boundary on a propeptide where cleavage occurs.	1506	0	0
1161	10	propeptide	Part of a peptide chain which is cleaved off during the formation of the mature protein.	1508	0	0
1162	10	immature_peptide_region	An immature_peptide_region is the extent of the peptide after it has been translated and before any processing occurs.	1511	0	0
1163	10	active_peptide	Active peptides are proteins which are biologically active, released from a precursor molecule.	1513	0	0
1164	10	compositionally_biased_region_of_peptide	Polypeptide region that is rich in a particular amino acid or homopolymeric and greater than three residues in length.	1517	0	0
1165	10	polypeptide_motif	A sequence motif is a short (up to 20 amino acids) region of biological interest. Such motifs, although they are too short to constitute functional domains, share sequence similarities and are conserved in different proteins. They display a common function (protein-binding, subcellular location etc.).	1519	0	0
1166	10	polypeptide_repeat	A polypeptide_repeat is a single copy of an internal sequence repetition.	1521	0	0
1167	10	polypeptide_structural_region	Region of polypeptide with a given structural property.	1523	0	0
1168	10	membrane_structure	Arrangement of the polypeptide with respect to the lipid bilayer.	1525	0	0
1169	10	extramembrane_polypeptide_region	Polypeptide region that is localized outside of a lipid bilayer.	1527	0	0
1170	10	cytoplasmic_polypeptide_region	Polypeptide region that is localized inside the cytoplasm.	1529	0	0
1171	10	non_cytoplasmic_polypeptide_region	Polypeptide region that is localized outside of a lipid bilayer and outside of the cytoplasm.	1531	0	0
1172	10	intramembrane_polypeptide_region	Polypeptide region present in the lipid bilayer.	1533	0	0
1173	10	membrane_peptide_loop	Polypeptide region localized within the lipid bilayer where both ends traverse the same membrane.	1535	0	0
1174	10	transmembrane_polypeptide_region	Polypeptide region traversing the lipid bilayer.	1537	0	0
1175	10	polypeptide_secondary_structure	A region of peptide with secondary structure has hydrogen bonding along the peptide chain that causes a defined conformation of the chain.	1540	0	0
1176	10	polypeptide_structural_motif	Motif is a three-dimensional structural element within the chain, which appears also in a variety of other molecules. Unlike a domain, a motif does not need to form a stable globular unit.	1543	0	0
1177	10	coiled_coil	A coiled coil is a structural motif in proteins, in which alpha-helices are coiled together like the strands of a rope.	1546	0	0
1178	10	helix_turn_helix	A motif comprising two helices separated by a turn.	1549	0	0
1179	10	polypeptide_sequencing_information	Incompatibility in the sequence due to some experimental problem.	1551	0	0
1180	10	non_adjacent_residues	Indicates that two consecutive residues in a fragment sequence are not consecutive in the full-length protein and that there are a number of unsequenced residues between them.	1553	0	0
1181	10	non_terminal_residue	The residue at an extremity of the sequence is not the terminal residue.	1555	0	0
1182	10	sequence_conflict	Different sources report differing sequences.	1557	0	0
1183	10	sequence_uncertainty	Describes the positions in a sequence where the authors are unsure about the sequence assignment.	1559	0	0
1184	10	cross_link (obsolete SO:0001087)	Posttranslationally formed amino acid bonds.	1561	1	0
1185	10	disulfide_bond (obsolete SO:0001088)	The covalent bond between sulfur atoms that binds two peptide chains or different parts of one peptide chain and is a structural determinant in many protein molecules.	1563	1	0
1186	10	post_translationally_modified_region	A region where a transformation occurs in a protein after it has been synthesized. This which may regulate, stabilize, crosslink or introduce new chemical functionalities in the protein.	1565	0	0
1187	10	covalent_binding_site (obsolete SO:0001090)	Binding involving a covalent bond.	1568	1	0
1188	10	non_covalent_binding_site (obsolete SO:0001091)	Binding site for any chemical group (co-enzyme, prosthetic group, etc.).	1570	1	0
1189	10	polypeptide_metal_contact	Residue is part of a binding site for a metal ion.	1572	0	0
1190	10	protein_protein_contact	Residues involved in protein-protein interactions.	1574	0	0
1191	10	polypeptide_calcium_ion_contact_site	Residue involved in contact with calcium.	1578	0	0
1192	10	polypeptide_cobalt_ion_contact_site	Residue involved in contact with cobalt.	1580	0	0
1193	10	polypeptide_copper_ion_contact_site	Residue involved in contact with copper.	1582	0	0
1194	10	polypeptide_iron_ion_contact_site	Residue involved in contact with iron.	1584	0	0
1195	10	polypeptide_magnesium_ion_contact_site	Residue involved in contact with magnesium.	1586	0	0
1196	10	polypeptide_manganese_ion_contact_site	Residue involved in contact with manganese.	1588	0	0
1197	10	polypeptide_molybdenum_ion_contact_site	Residue involved in contact with molybdenum.	1590	0	0
1198	10	polypeptide_nickel_ion_contact_site	Residue involved in contact with nickel.	1592	0	0
1199	10	polypeptide_tungsten_ion_contact_site	Residue involved in contact with tungsten.	1594	0	0
1200	10	polypeptide_zinc_ion_contact_site	Residue involved in contact with zinc.	1596	0	0
1201	10	catalytic_residue	Amino acid involved in the activity of an enzyme.	1598	0	0
1202	10	polypeptide_ligand_contact	Residues which interact with a ligand.	1600	0	0
1203	10	asx_motif	A motif of five consecutive residues and two H-bonds in which: Residue(i) is Aspartate or Asparagine (Asx), side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2) or (i+3), main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+3) or (i+4).	1602	0	0
1204	10	beta_bulge	A motif of three residues within a beta-sheet in which the main chains of two consecutive residues are H-bonded to that of the third, and in which the dihedral angles are as follows: Residue(i): -140 degrees < phi(l) -20 degrees , -90 degrees < psi(l) < 40 degrees. Residue (i+1): -180 degrees < phi < -25 degrees or +120 degrees < phi < +180 degrees, +40 degrees < psi < +180 degrees or -180 degrees < psi < -120 degrees.	1604	0	0
1205	10	beta_bulge_loop	A motif of three residues within a beta-sheet consisting of two H-bonds. Beta bulge loops often occur at the loop ends of beta-hairpins.	1607	0	0
1206	10	beta_bulge_loop_five	A motif of three residues within a beta-sheet consisting of two H-bonds in which: the main-chain NH of residue(i) is H-bonded to the main-chain CO of residue(i+4), the main-chain CO of residue i is H-bonded to the main-chain NH of residue(i+3), these loops have an RL nest at residues i+2 and i+3.	1610	0	0
1207	10	beta_bulge_loop_six	A motif of three residues within a beta-sheet consisting of two H-bonds in which: the main-chain NH of residue(i) is H-bonded to the main-chain CO of residue(i+5), the main-chain CO of residue i is H-bonded to the main-chain NH of residue(i+4), these loops have an RL nest at residues i+3 and i+4.	1612	0	0
1208	10	beta_strand	A beta strand describes a single length of polypeptide chain that forms part of a beta sheet. A single continuous stretch of amino acids adopting an extended conformation of hydrogen bonds between the N-O and the C=O of another part of the peptide. This forms a secondary protein structure in which two or more extended polypeptide regions are hydrogen-bonded to one another in a planar array.	1614	0	0
1209	10	antiparallel_beta_strand	A peptide region which hydrogen bonded to another region of peptide running in the oposite direction (one running N-terminal to C-terminal and one running C-terminal to N-terminal). Hydrogen bonding occurs between every other C=O from one strand to every other N-H on the adjacent strand. In this case, if two atoms C-alpha (i) and C-alpha (j) are adjacent in two hydrogen-bonded beta strands, then they form two mutual backbone hydrogen bonds to each other's flanking peptide groups; this is known as a close pair of hydrogen bonds. The peptide backbone dihedral angles (phi, psi) are about (-140 degrees, 135 degrees) in antiparallel sheets.	1617	0	0
1210	10	parallel_beta_strand	A peptide region which hydrogen bonded to another region of peptide running in the oposite direction (both running N-terminal to C-terminal). This orientation is slightly less stable because it introduces nonplanarity in the inter-strand hydrogen bonding pattern. Hydrogen bonding occurs between every other C=O from one strand to every other N-H on the adjacent strand. In this case, if two atoms C-alpha (i)and C-alpha (j) are adjacent in two hydrogen-bonded beta strands, then they do not hydrogen bond to each other; rather, one residue forms hydrogen bonds to the residues that flank the other (but not vice versa). For example, residue i may form hydrogen bonds to residues j - 1 and j + 1; this is known as a wide pair of hydrogen bonds. By contrast, residue j may hydrogen-bond to different residues altogether, or to none at all. The dihedral angles (phi, psi) are about (-120 degrees, 115 degrees) in parallel sheets.	1619	0	0
1211	10	peptide_helix	A helix is a secondary_structure conformation where the peptide backbone forms a coil.	1621	0	0
1212	10	left_handed_peptide_helix	A left handed helix is a region of peptide where the coiled conformation turns in an anticlockwise, left handed screw.	1623	0	0
1213	10	right_handed_peptide_helix	A right handed helix is a region of peptide where the coiled conformation turns in a clockwise, right handed screw.	1625	0	0
1214	10	alpha_helix	The helix has 3.6 residues per turn which corersponds to a translation of 1.5 angstroms (= 0.15 nm) along the helical axis. Every backbone N-H group donates a hydrogen bond to the backbone C=O group of the amino acid four residues earlier.	1627	0	0
1215	10	pi_helix	The pi helix has 4.1 residues per turn and a translation of 1.15  (=0.115 nm) along the helical axis. The N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid five residues earlier.	1630	0	0
1216	10	three_ten_helix	The 3-10 helix has 3 residues per turn with a translation of 2.0 angstroms (=0.2 nm) along the helical axis. The N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid three residues earlier.	1633	0	0
1217	10	polypeptide_nest_motif	A motif of two consecutive residues with dihedral angles. Nest should not have Proline as any residue. Nests frequently occur as parts of other motifs such as Schellman loops.	1636	0	0
1218	10	polypeptide_nest_left_right_motif	A motif of two consecutive residues with dihedral angles: Residue(i): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees.	1638	0	0
1219	10	polypeptide_nest_right_left_motif	A motif of two consecutive residues with dihedral angles: Residue(i): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees.	1640	0	0
1220	10	schellmann_loop	A motif of six or seven consecutive residues that contains two H-bonds.	1642	0	0
1221	10	schellmann_loop_seven	Wild type: A motif of seven consecutive residues that contains two H-bonds in which: the main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+6), the main-chain CO of residue(i+1) is H-bonded to the main-chain NH of residue(i+5).	1644	0	0
1222	10	schellmann_loop_six	Common Type: A motif of six consecutive residues that contains two H-bonds in which: the main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+5) the main-chain CO of residue(i+1) is H-bonded to the main-chain NH of residue(i+4).	1646	0	0
1223	10	serine_threonine_motif	A motif of five consecutive residues and two hydrogen bonds in which: residue(i) is Serine (S) or Threonine (T), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2) or (i+3) , the main-chain CO group of residue(i) is H-bonded to the main-chain NH of residue(i+3) or (i+4).	1648	0	0
1224	10	serine_threonine_staple_motif	A motif of four or five consecutive residues and one H-bond in which: residue(i) is Serine (S) or Threonine (T), the side-chain OH of residue(i) is H-bonded to the main-chain CO of residue(i3) or (i4), Phi angles of residues(i1), (i2) and (i3) are negative.	1650	0	0
1225	10	polypeptide_turn_motif	A reversal in the direction of the backbone of a protein that is stabilized by hydrogen bond between backbone NH and CO groups, involving no more than 4 amino acid residues.	1652	0	0
1226	10	asx_turn_left_handed_type_one	Left handed type I (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, -90 degrees < psi +120 degrees < +40 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees.	1655	0	0
1227	10	asx_turn_left_handed_type_two	Left handed type II (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, +80 degrees < psi +120 degrees < +180 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees.	1657	0	0
1228	10	asx_turn_right_handed_type_two	Right handed type II (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, +80 degrees < psi +120 degrees < +180 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees.	1659	0	0
1229	10	asx_turn_right_handed_type_one	Right handed type I (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, -90 degrees < psi +120 degrees < +40 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees.	1661	0	0
1230	10	beta_turn	A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles of the second and third residues, which are the basis for sub-categorization.	1663	0	0
1231	10	beta_turn_left_handed_type_one	Left handed type I:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles:- Residue(i+1): -140 degrees > phi > -20 degrees, -90 degrees > psi > +40 degrees. Residue(i+2): -140 degrees > phi > -20 degrees, -90 degrees > psi > +40 degrees.	1665	0	0
1232	10	beta_turn_left_handed_type_two	Left handed type II: A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees > phi > -20 degrees, +80 degrees > psi > +180 degrees. Residue(i+2): +20 degrees > phi > +140 degrees, -40 degrees > psi > +90 degrees.	1667	0	0
1233	10	beta_turn_right_handed_type_one	Right handed type I:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. Residue(i+2): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees.	1669	0	0
1234	10	beta_turn_right_handed_type_two	Right handed type II:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees < phi < -20 degrees, +80 degrees < psi < +180 degrees. Residue(i+2): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees.	1671	0	0
1235	10	gamma_turn	Gamma turns, defined for 3 residues i, i+1, i+2 if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees.	1673	0	0
1236	10	gamma_turn_classic	Gamma turns, defined for 3 residues i, i+1, i+2 if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees: phi(i+1)=75.0 - psi(i+1)=-64.0.	1675	0	0
1237	10	gamma_turn_inverse	Gamma turns, defined for 3 residues i, i+1, i+2 if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees: phi(i+1)=-79.0 - psi(i+1)=69.0.	1677	0	0
1238	10	serine_threonine_turn	A motif of three consecutive residues and one H-bond in which: residue(i) is Serine (S) or Threonine (T), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2).	1679	0	0
1239	10	st_turn_left_handed_type_one	The peptide twists in an anticlockwise, left handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, -90 degrees psi +120 degrees < +40 degrees, residue(i+1): -140 degrees < phi < -20 degrees, -90 < psi < +40 degrees.	1681	0	0
1240	10	st_turn_left_handed_type_two	The peptide twists in an anticlockwise, left handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, +80 degrees psi +120 degrees < +180 degrees, residue(i+1): +20 degrees < phi < +140 degrees, -40 < psi < +90 degrees.	1683	0	0
1241	10	st_turn_right_handed_type_one	The peptide twists in an clockwise, right handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, -90 degrees psi +120 degrees < +40 degrees, residue(i+1): -140 degrees < phi < -20 degrees, -90 < psi < +40 degrees.	1685	0	0
1242	10	st_turn_right_handed_type_two	The peptide twists in an clockwise, right handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, +80 degrees psi +120 degrees < +180 degrees, residue(i+1): +20 degrees < phi < +140 degrees, -40 < psi < +90 degrees.	1687	0	0
1243	10	polypeptide_variation_site	A site of sequence variation (alteration). Alternative sequence due to naturally occuring events such as polymorphisms and altermatve splicing or experimental methods such as site directed mutagenesis.	1689	0	0
1244	10	natural_variant_site	Describes the natural sequence variants due to polymorphisms, disease-associated mutations, RNA editing and variations between strains, isolates or cultivars.	1691	0	0
519	7	broad	\N	729	0	0
1245	10	mutated_variant_site	Site which has been experimentally altered.	1693	0	0
1246	10	alternate_sequence_site	Description of sequence variants produced by alternative splicing, alternative promoter usage, alternative initiation and ribosomal frameshifting.	1695	0	0
1247	10	beta_turn_type_six	A motif of four consecutive peptide resides of type VIa or type VIb and where the i+2 residue is cis-proline.	1698	0	0
1248	10	beta_turn_type_six_a	A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -90 degrees, psi ~ 0 degrees.	1699	0	0
1249	10	beta_turn_type_six_a_one	\N	1701	0	0
1250	10	beta_turn_type_six_a_two	\N	1702	0	0
1251	10	beta_turn_type_six_b	A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -120 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -60 degrees, psi ~ 0 degrees.	1703	0	0
1252	10	beta_turn_type_eight	A motif of four consecutive peptide residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ -30 degrees. Residue(i+2): phi ~ -120 degrees, psi ~ 120 degrees.	1704	0	0
1253	10	DRE_motif	A sequence element characteristic of some RNA polymerase II promoters, usually located between -10 and -60 relative to the TSS. Consensus sequence is WATCGATW.	1705	0	0
1338	10	encodes_alternate_transcription_start_sites	\N	1835	0	0
118	7	narrow	\N	154	0	0
1337	10	TSS_region	The region of a gene from the 5' most TSS to the 3' TSS.	1833	0	0
1254	10	DMv4_motif	A sequence element characteristic of some RNA polymerase II promoters, located immediately upstream of some TATA box elements with respect to the TSS (+1). Consensus sequence is YGGTCACACTR. Marked spatial preference within core promoter; tend to occur near the TSS, although not as tightly as INR (SO:0000014).	1707	0	0
1255	10	E_box_motif	A sequence element characteristic of some RNA polymerase II promoters, usually located between -60 and +1 relative to the TSS. Consensus sequence is AWCAGCTGWT. Tends to co-occur with DMv2 (SO:0001161). Tends to not occur with DPE motif (SO:0000015).	1709	0	0
1281	10	PNA	An attribute describing a sequence composed of peptide nucleic acid (CHEBI:48021), a chemical consisting of nucleobases bound to a backbone composed of repeating N-(2-aminoethyl)-glycine units linked by peptide bonds. The purine and pyrimidine bases are linked to the backbone by methylene carbonyl bonds.	1751	0	0
1276	10	U3_snoRNA	U3 snoRNA is a member of the box C/D class of small nucleolar RNAs. The U3 snoRNA secondary structure is characterised by a small 5' domain (with boxes A and A'), and a larger 3' domain (with boxes B, C, C', and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C' and D form the C'/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5' domain and the hinge region act as a pre-rRNA-binding domain. The 3' domain has conserved protein-binding sites. Both the box B/C and box C'/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C'/D motif is also necessary for nucleolar localization, stability and hypermethylation of U3 snoRNA. Both box B/C and C'/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA.	1737	0	0
1256	10	DMv5_motif	A sequence element characteristic of some RNA polymerase II promoters, usually located between -50 and -10 relative to the TSS. Consensus sequence is KTYRGTATWTTT. Tends to co-occur with DMv4 (SO:0001157) . Tends to not occur with DPE motif (SO:0000015) or MTE (SO:0001162).	1711	0	0
1282	10	enzymatic	An attribute describing the sequence of a transcript that has catalytic activity with or without an associated ribonucleoprotein.	1752	0	0
1283	10	ribozymic	An attribute describing the sequence of a transcript that has catalytic activity even without an associated ribonucleoprotein.	1753	0	0
1284	10	pseudouridylation_guide_snoRNA	A snoRNA that specifies the site of pseudouridylation in an RNA molecule by base pairing with a short sequence around the target residue.	1754	0	0
1257	10	DMv3_motif	A sequence element characteristic of some RNA polymerase II promoters, usually located between -30 and +15 relative to the TSS. Consensus sequence is KNNCAKCNCTRNY. Tends to co-occur with DMv2 (SO:0001161). Tends to not occur with DPE motif (SO:0000015) or MTE (0001162).	1712	0	0
1403	10	N6_methyl_N6_threonylcarbamoyladenosine	N6_methyl_N6_threonylcarbamoyladenosine is a modified adenosine.	1936	0	0
1277	10	AU_rich_element	A cis-acting element found in the 3' UTR of some mRNA which is rich in AUUUA pentamers. Messenger RNAs bearing multiple AU-rich elements are often unstable.	1740	0	0
1520	10	pyrosequenced_read	A read produced by pyrosequencing technology.	2152	0	0
1339	10	miRNA_primary_transcript_region	A part of an miRNA primary_transcript.	1836	0	0
1258	10	DMv2_motif	A sequence element characteristic of some RNA polymerase II promoters, usually located between -60 and -45 relative to the TSS. Consensus sequence is MKSYGGCARCGSYSS. Tends to co-occur with DMv3 (SO:0001160). Tends to not occur with DPE motif (SO:0000015) or MTE (SO:0001162).	1713	0	0
1278	10	Bruno_response_element	A cis-acting element found in the 3' UTR of some mRNA which is bound by the Drosophila Bruno protein and its homologs.	1743	0	0
1279	10	iron_responsive_element	A regulatory sequence found in the 5' and 3' UTRs of many mRNAs which encode iron-binding proteins. It has a hairpin structure and is recognized by trans-acting proteins known as iron-regulatory proteins.	1745	0	0
1259	10	MTE	A sequence element characteristic of some RNA polymerase II promoters, usually located between +20 and +30 relative to the TSS. Consensus sequence is CSARCSSAACGS. Tends to co-occur with INR motif (SO:0000014). Tends to not occur with DPE motif (SO:0000015) or DMv5 (SO:0001159).	1714	0	0
1280	10	morpholino	An attribute describing a sequence composed of nucleobases bound to a morpholino backbone. A morpholino backbone consists of morpholine (CHEBI:34856) rings connected by phosphorodiamidate linkages.	1749	0	0
1260	10	INR1_motif	A promoter motif with consensus sequence TCATTCG.	1716	0	0
1261	10	DPE1_motif	A promoter motif with consensus sequence CGGACGT.	1718	0	0
1262	10	DMv1_motif	A promoter motif with consensus sequence CARCCCT.	1719	0	0
1263	10	GAGA_motif	A non directional promoter motif with consensus sequence GAGAGCG.	1720	0	0
1264	10	NDM2_motif	A non directional promoter motif with consensus CGMYGYCR.	1721	0	0
1265	10	NDM3_motif	A non directional promoter motif with consensus sequence GAAAGCT.	1722	0	0
1266	10	ds_RNA_viral_sequence	A ds_RNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as double stranded RNA.	1723	0	0
1267	10	polinton	A kind of DNA transposon that populates the genomes of protists, fungi, and animals, characterized by a unique set of proteins necessary for their transposition, including a protein-primed DNA polymerase B, retroviral integrase, cysteine protease, and ATPase. Polintons are characterized by 6-bp target site duplications, terminal-inverted repeats that are several hundred nucleotides long, and 5'-AG and TC-3' termini. Polintons exist as autonomous and nonautonomous elements.	1724	0	0
1268	10	rRNA_21S	A component of the large ribosomal subunit in mitochondrial rRNA.	1726	0	0
1269	10	tRNA_region	A region of a tRNA.	1727	0	0
1270	10	anticodon_loop	A sequence of seven nucleotide bases in tRNA which contains the anticodon. It has the sequence 5'-pyrimidine-purine-anticodon-modified purine-any base-3.	1728	0	0
1271	10	anticodon	A sequence of three nucleotide bases in tRNA which recognizes a codon in mRNA.	1730	0	0
1272	10	CCA_tail	Base sequence at the 3' end of a tRNA. The 3'-hydroxyl group on the terminal adenosine is the attachment point for the amino acid.	1732	0	0
1273	10	DHU_loop	Non-base-paired sequence of nucleotide bases in tRNA. It contains several dihydrouracil residues.	1733	0	0
1274	10	T_loop	Non-base-paired sequence of three nucleotide bases in tRNA. It has sequence T-Psi-C.	1735	0	0
1275	10	pyrrolysine_tRNA_primary_transcript	A primary transcript encoding pyrrolysyl tRNA (SO:0000766).	1736	0	0
1285	10	LNA	An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of 'locked' deoxyribose rings connected to a phosphate backbone. The deoxyribose unit's conformation is 'locked' by a 2'-C,4'-C-oxymethylene link.	1757	0	0
1286	10	LNA_oligo	An oligo composed of LNA residues.	1759	0	0
1287	10	TNA	An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of threose rings connected to a phosphate backbone.	1761	0	0
1288	10	TNA_oligo	An oligo composed of TNA residues.	1763	0	0
1289	10	GNA	An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of an acyclic three-carbon propylene glycol connected to a phosphate backbone. It has two enantiomeric forms, (R)-GNA and (S)-GNA.	1765	0	0
1290	10	GNA_oligo	An oligo composed of GNA residues.	1767	0	0
1291	10	R_GNA	An attribute describing a GNA sequence in the (R)-GNA enantiomer.	1769	0	0
1292	10	R_GNA_oligo	An oligo composed of (R)-GNA residues.	1771	0	0
1293	10	S_GNA	An attribute describing a GNA sequence in the (S)-GNA enantiomer.	1772	0	0
1294	10	S_GNA_oligo	An oligo composed of (S)-GNA residues.	1774	0	0
1295	10	ds_DNA_viral_sequence	A ds_DNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as double stranded DNA.	1775	0	0
1296	10	ss_RNA_viral_sequence	A ss_RNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as single stranded RNA.	1776	0	0
1297	10	negative_sense_ssRNA_viral_sequence	A negative_sense_RNA_viral_sequence is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus that is complementary to mRNA and must be converted to positive sense RNA by RNA polymerase before translation.	1777	0	0
1298	10	positive_sense_ssRNA_viral_sequence	A positive_sense_RNA_viral_sequence is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus that can be immediately translated by the host.	1778	0	0
1299	10	ambisense_ssRNA_viral_sequence	A ambisense_RNA_virus is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus with both messenger and anti messenger polarity.	1779	0	0
1300	10	RNA_polymerase_promoter	A region (DNA) to which RNA polymerase binds, to begin transcription.	1780	0	0
1301	10	Phage_RNA_Polymerase_Promoter	A region (DNA) to which Bacteriophage RNA polymerase binds, to begin transcription.	1782	0	0
1302	10	SP6_RNA_Polymerase_Promoter	A region (DNA) to which the SP6 RNA polymerase binds, to begin transcription.	1783	0	0
1303	10	T3_RNA_Polymerase_Promoter	A DNA sequence to which the T3 RNA polymerase binds, to begin transcription.	1784	0	0
1304	10	T7_RNA_Polymerase_Promoter	A region (DNA) to which the T7 RNA polymerase binds, to begin transcription.	1785	0	0
1305	10	five_prime_EST	An EST read from the 5' end of a transcript that usually codes for a protein. These regions tend to be conserved across species and do not change much within a gene family.	1786	0	0
1306	10	three_prime_EST	An EST read from the 3' end of a transcript. They are more likely to fall within non-coding, or untranslated regions(UTRs).	1788	0	0
1307	10	translational_frameshift	The region of mRNA (not divisible by 3 bases) that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different.	1789	0	0
1308	10	plus_1_translational_frameshift	The region of mRNA 1 base long that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different.	1791	0	0
1309	10	plus_2_translational_frameshift	The region of mRNA 2 bases long that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different.	1792	0	0
1310	10	group_III_intron	Group III introns are introns found in the mRNA of the plastids of euglenoid protists. They are spliced by a two step transesterification with bulged adenosine as initiating nucleophile.	1793	0	0
1311	10	noncoding_region_of_exon	The maximal intersection of exon and UTR.	1796	0	0
1312	10	coding_region_of_exon	The region of an exon that encodes for protein sequence.	1797	0	0
1313	10	endonuclease_spliced_intron	An intron that spliced via endonucleolytic cleavage and ligation rather than transesterification.	1798	0	0
1314	10	protein_coding_gene	\N	1799	0	0
1315	10	transgenic_insertion	An insertion that derives from another organism, via the use of recombinant DNA technology.	1800	0	0
1316	10	retrogene	\N	1802	0	0
1317	10	silenced_by_RNA_interference	An attribute describing an epigenetic process where a gene is inactivated by RNA interference.	1803	0	0
1318	10	silenced_by_histone_modification	An attribute describing an epigenetic process where a gene is inactivated by histone modification.	1804	0	0
1319	10	silenced_by_histone_methylation	An attribute describing an epigenetic process where a gene is inactivated by histone methylation.	1805	0	0
1320	10	silenced_by_histone_deacetylation	An attribute describing an epigenetic process where a gene is inactivated by histone deacetylation.	1806	0	0
1321	10	gene_silenced_by_RNA_interference	A gene that is silenced by RNA interference.	1807	0	0
1322	10	gene_silenced_by_histone_modification	A gene that is silenced by histone modification.	1808	0	0
1323	10	gene_silenced_by_histone_methylation	A gene that is silenced by histone methylation.	1809	0	0
1324	10	gene_silenced_by_histone_deacetylation	A gene that is silenced by histone deacetylation.	1810	0	0
1325	10	dihydrouridine	A modified RNA base in which the 5,6-dihydrouracil is bound to the ribose ring.	1811	0	0
1326	10	pseudouridine	A modified RNA base in which the 5- position of the uracil is bound to the ribose ring instead of the 4- position.	1814	0	0
1327	10	inosine	A modified RNA base in which hypoxanthine is bound to the ribose ring.	1817	0	0
1328	10	seven_methylguanine	A modified RNA base in which guanine is methylated at the 7- position.	1820	0	0
1329	10	ribothymidine	A modified RNA base in which thymine is bound to the ribose ring.	1821	0	0
1330	10	methylinosine	A modified RNA base in which methylhypoxanthine is bound to the ribose ring.	1822	0	0
1331	10	mobile	An attribute describing a feature that has either intra-genome or intracellular mobility.	1823	0	0
1332	10	replicon	A region containing at least one unique origin of replication and a unique termination site.	1825	0	0
1333	10	base	A base is a sequence feature that corresponds to a single unit of a nucleotide polymer.	1827	0	0
1340	10	pre_miRNA	The 60-70 nucleotide region remain after Drosha processing of the primary transcript, that folds back upon itself to form a hairpin sructure.	1837	0	0
1341	10	miRNA_stem	The stem of the hairpin loop formed by folding of the pre-miRNA.	1838	0	0
1342	10	miRNA_loop	The loop of the hairpin loop formed by folding of the pre-miRNA.	1839	0	0
1343	10	synthetic_oligo	An oligo composed of synthetic nucleotides.	1840	0	0
1344	10	assembly	A region of the genome of known length that is composed by ordering and aligning two or more different regions.	1841	0	0
1345	10	fragment_assembly	A fragment assembly is a genome assembly that orders overlapping fragments of the genome based on landmark sequences. The base pair distance between the landmarks is known allowing additivity of lengths.	1843	0	0
1346	10	fingerprint_map	A fingerprint_map is a physical map composed of restriction fragments.	1844	0	0
1347	10	STS_map	An STS map is a physical map organized by the unique STS landmarks.	1845	0	0
1393	10	N6_methyladenosine	N6_methyladenosine is a modified adenosine.	1916	0	0
1348	10	RH_map	A radiation hybrid map is a physical map.	1846	0	0
1380	10	two_thiocytidine	2-thiocytidine is a modified cytidine.	1889	0	0
1349	10	sonicate_fragment	A DNA fragment generated by sonication. Sonication is a technique used to sheer DNA into smaller fragments.	1847	0	0
1397	10	two_methylthio_N6_isopentenyladenosine	2_methylthio_N6_isopentenyladenosine is a modified adenosine.	1924	0	0
1350	10	polyploid	A kind of chromosome variation where the chromosome complement is an exact multiple of the haploid number and is greater than the diploid number.	1848	0	0
1351	10	autopolyploid	A polyploid where the multiple chromosome set was derived from the same organism.	1850	0	0
1352	10	allopolyploid	A polyploid where the multiple chromosome set was derived from a different organism.	1852	0	0
1353	10	homing_endonuclease_binding_site	The binding site (recognition site) of a homing endonuclease. The binding site is typically large.	1854	0	0
1389	10	five_formyl_two_prime_O_methylcytidine	5-formyl-2'-O-methylcytidine is a modified cytidine.	1908	0	0
1354	10	octamer_motif	A sequence element characteristic of some RNA polymerase II promoters with sequence ATTGCAT that binds Pou-domain transcription factors.	1855	0	0
1381	10	N4_acetylcytidine	N4-acetylcytidine is a modified cytidine.	1891	0	0
1355	10	apicoplast_chromosome	A chromosome originating in an apicoplast.	1858	0	0
1356	10	sequence_collection	A collection of discontinuous sequences.	1859	0	0
1394	10	two_prime_O_methyladenosine	2prime_O_methyladenosine is a modified adenosine.	1918	0	0
1357	10	overlapping_feature_set	A continuous region of sequence composed of the overlapping of multiple sequence_features, which ultimately provides evidence for another sequence_feature.	1860	0	0
1396	10	N6_isopentenyladenosine	N6_isopentenyladenosine is a modified adenosine.	1922	0	0
1358	10	overlapping_EST_set	A continous experimental result region extending the length of multiple overlapping EST's.	1861	0	0
1382	10	five_formylcytidine	5-formylcytidine is a modified cytidine.	1893	0	0
1359	10	ncRNA_gene	\N	1862	0	0
1390	10	N4_N4_2_prime_O_trimethylcytidine	N4_N4_2_prime_O_trimethylcytidine is a modified cytidine.	1910	0	0
1360	10	gRNA_gene	\N	1863	0	0
1361	10	miRNA_gene	\N	1864	0	0
1383	10	five_two_prime_O_dimethylcytidine	5,2'-O-dimethylcytidine is a modified cytidine.	1895	0	0
1362	10	scRNA_gene	\N	1865	0	0
1391	10	one_methyladenosine	1_methyladenosine is a modified adenosine.	1912	0	0
1363	10	snoRNA_gene	\N	1866	0	0
1395	10	two_methylthio_N6_methyladenosine	2_methylthio_N6_methyladenosine is a modified adenosine.	1920	0	0
1364	10	snRNA_gene	\N	1867	0	0
1398	10	N6_cis_hydroxyisopentenyl_adenosine	N6_cis_hydroxyisopentenyl_adenosine is a modified adenosine.	1926	0	0
1365	10	SRP_RNA_gene	\N	1868	0	0
1384	10	N4_acetyl_2_prime_O_methylcytidine	N4-acetyl-2'-O-methylcytidine is a modified cytidine.	1897	0	0
1366	10	stRNA_gene	\N	1869	0	0
1399	10	two_methylthio_N6_cis_hydroxyisopentenyl_adenosine	2_methylthio_N6_cis_hydroxyisopentenyl_adenosine is a modified adenosine.	1928	0	0
1367	10	tmRNA_gene	\N	1870	0	0
1392	10	two_methyladenosine	2_methyladenosine is a modified adenosine.	1914	0	0
1368	10	tRNA_gene	\N	1871	0	0
1400	10	N6_glycinylcarbamoyladenosine	N6_glycinylcarbamoyladenosine is a modified adenosine.	1930	0	0
1369	10	modified_adenosine	A modified adenine is an adenine base feature that has been altered.	1872	0	0
1385	10	lysidine	Lysidine is a modified cytidine.	1899	0	0
1370	10	modified_inosine	A modified inosine is an inosine base feature that has been altered.	1873	0	0
1401	10	N6_threonylcarbamoyladenosine	N6_threonylcarbamoyladenosine is a modified adenosine.	1932	0	0
1371	10	modified_cytidine	A modified cytidine is a cytidine base feature which has been altered.	1874	0	0
1386	10	N4_methylcytidine	N4-methylcytidine is a modified cytidine.	1902	0	0
1372	10	modified_guanosine	\N	1875	0	0
1402	10	two_methylthio_N6_threonyl_carbamoyladenosine	2_methylthio_N6_threonyl_carbamoyladenosine is a modified adenosine.	1934	0	0
1373	10	modified_uridine	\N	1876	0	0
1374	10	one_methylinosine	1-methylinosine is a modified insosine.	1877	0	0
1387	10	N4_2_prime_O_dimethylcytidine	N4,2'-O-dimethylcytidine is a modified cytidine.	1904	0	0
1375	10	one_two_prime_O_dimethylinosine	1,2'-O-dimethylinosine is a modified inosine.	1879	0	0
1388	10	five_hydroxymethylcytidine	5-hydroxymethylcytidine is a modified cytidine.	1906	0	0
1376	10	two_prime_O_methylinosine	2'-O-methylinosine is a modified inosine.	1881	0	0
1377	10	three_methylcytidine	3-methylcytidine is a modified cytidine.	1883	0	0
1378	10	five_methylcytidine	5-methylcytidine is a modified cytidine.	1885	0	0
1379	10	two_prime_O_methylcytidine	2'-O-methylcytidine is a modified cytidine.	1887	0	0
1404	10	N6_hydroxynorvalylcarbamoyladenosine	N6_hydroxynorvalylcarbamoyladenosine is a modified adenosine.	1938	0	0
1405	10	two_methylthio_N6_hydroxynorvalyl_carbamoyladenosine	2_methylthio_N6_hydroxynorvalyl_carbamoyladenosine is a modified adenosine.	1940	0	0
1406	10	two_prime_O_ribosyladenosine_phosphate	2prime_O_ribosyladenosine_phosphate is a modified adenosine.	1942	0	0
1407	10	N6_N6_dimethyladenosine	N6_N6_dimethyladenosine is a modified adenosine.	1944	0	0
1408	10	N6_2_prime_O_dimethyladenosine	N6_2prime_O_dimethyladenosine is a modified adenosine.	1946	0	0
1409	10	N6_N6_2_prime_O_trimethyladenosine	N6_N6_2prime_O_trimethyladenosine is a modified adenosine.	1948	0	0
1410	10	one_two_prime_O_dimethyladenosine	1,2'-O-dimethyladenosine is a modified adenosine.	1950	0	0
1411	10	N6_acetyladenosine	N6_acetyladenosine is a modified adenosine.	1952	0	0
1412	10	seven_deazaguanosine	7-deazaguanosine is a moddified guanosine.	1954	0	0
1413	10	queuosine	Queuosine is a modified 7-deazoguanosine.	1955	0	0
1414	10	epoxyqueuosine	Epoxyqueuosine is a modified 7-deazoguanosine.	1958	0	0
1415	10	galactosyl_queuosine	Galactosyl_queuosine is a modified 7-deazoguanosine.	1960	0	0
1416	10	mannosyl_queuosine	Mannosyl_queuosine is a modified 7-deazoguanosine.	1962	0	0
1417	10	seven_cyano_seven_deazaguanosine	7_cyano_7_deazaguanosine is a modified 7-deazoguanosine.	1964	0	0
1418	10	seven_aminomethyl_seven_deazaguanosine	7_aminomethyl_7_deazaguanosine is a modified 7-deazoguanosine.	1966	0	0
1419	10	archaeosine	Archaeosine is a modified 7-deazoguanosine.	1968	0	0
1420	10	one_methylguanosine	1_methylguanosine is a modified guanosine base feature.	1970	0	0
1421	10	N2_methylguanosine	N2_methylguanosine is a modified guanosine base feature.	1972	0	0
1422	10	seven_methylguanosine	7_methylguanosine is a modified guanosine base feature.	1974	0	0
1423	10	two_prime_O_methylguanosine	2prime_O_methylguanosine is a modified guanosine base feature.	1976	0	0
1424	10	N2_N2_dimethylguanosine	N2_N2_dimethylguanosine is a modified guanosine base feature.	1978	0	0
1425	10	N2_2_prime_O_dimethylguanosine	N2_2prime_O_dimethylguanosine is a modified guanosine base feature.	1980	0	0
1426	10	N2_N2_2_prime_O_trimethylguanosine	N2_N2_2prime_O_trimethylguanosine is a modified guanosine base feature.	1982	0	0
1427	10	two_prime_O_ribosylguanosine_phosphate	2prime_O_ribosylguanosine_phosphate is a modified guanosine base feature.	1984	0	0
1428	10	wybutosine	Wybutosine is a modified guanosine base feature.	1986	0	0
1429	10	peroxywybutosine	Peroxywybutosine is a modified guanosine base feature.	1988	0	0
1430	10	hydroxywybutosine	Hydroxywybutosine is a modified guanosine base feature.	1990	0	0
1431	10	undermodified_hydroxywybutosine	Undermodified_hydroxywybutosine is a modified guanosine base feature.	1992	0	0
1432	10	wyosine	Wyosine is a modified guanosine base feature.	1994	0	0
1433	10	methylwyosine	Methylwyosine is a modified guanosine base feature.	1996	0	0
1434	10	N2_7_dimethylguanosine	N2_7_dimethylguanosine is a modified guanosine base feature.	1998	0	0
1435	10	N2_N2_7_trimethylguanosine	N2_N2_7_trimethylguanosine is a modified guanosine base feature.	2000	0	0
1436	10	one_two_prime_O_dimethylguanosine	1_2prime_O_dimethylguanosine is a modified guanosine base feature.	2002	0	0
1437	10	four_demethylwyosine	4_demethylwyosine is a modified guanosine base feature.	2004	0	0
1438	10	isowyosine	Isowyosine is a modified guanosine base feature.	2006	0	0
1439	10	N2_7_2prirme_O_trimethylguanosine	N2_7_2prirme_O_trimethylguanosine is a modified guanosine base feature.	2008	0	0
1440	10	five_methyluridine	5_methyluridine is a modified uridine base feature.	2010	0	0
1441	10	two_prime_O_methyluridine	2prime_O_methyluridine is a modified uridine base feature.	2013	0	0
1442	10	five_two_prime_O_dimethyluridine	5_2_prime_O_dimethyluridine is a modified uridine base feature.	2015	0	0
1443	10	one_methylpseudouridine	1_methylpseudouridine is a modified uridine base feature.	2017	0	0
1444	10	two_prime_O_methylpseudouridine	2prime_O_methylpseudouridine is a modified uridine base feature.	2019	0	0
1445	10	two_thiouridine	2_thiouridine is a modified uridine base feature.	2021	0	0
1446	10	four_thiouridine	4_thiouridine is a modified uridine base feature.	2023	0	0
1447	10	five_methyl_2_thiouridine	5_methyl_2_thiouridine is a modified uridine base feature.	2025	0	0
1448	10	two_thio_two_prime_O_methyluridine	2_thio_2prime_O_methyluridine is a modified uridine base feature.	2027	0	0
1449	10	three_three_amino_three_carboxypropyl_uridine	3_3_amino_3_carboxypropyl_uridine is a modified uridine base feature.	2029	0	0
1450	10	five_hydroxyuridine	5_hydroxyuridine is a modified uridine base feature.	2031	0	0
1451	10	five_methoxyuridine	5_methoxyuridine is a modified uridine base feature.	2032	0	0
1452	10	uridine_five_oxyacetic_acid	Uridine_5_oxyacetic_acid is a modified uridine base feature.	2034	0	0
1453	10	uridine_five_oxyacetic_acid_methyl_ester	Uridine_5_oxyacetic_acid_methyl_ester is a modified uridine base feature.	2036	0	0
1454	10	five_carboxyhydroxymethyl_uridine	5_carboxyhydroxymethyl_uridine is a modified uridine base feature.	2038	0	0
1455	10	five_carboxyhydroxymethyl_uridine_methyl_ester	5_carboxyhydroxymethyl_uridine_methyl_ester is a modified uridine base feature.	2040	0	0
1456	10	five_methoxycarbonylmethyluridine	Five_methoxycarbonylmethyluridine is a modified uridine base feature.	2042	0	0
1457	10	five_methoxycarbonylmethyl_two_prime_O_methyluridine	Five_methoxycarbonylmethyl_2_prime_O_methyluridine is a modified uridine base feature.	2044	0	0
1458	10	five_methoxycarbonylmethyl_two_thiouridine	5_methoxycarbonylmethyl_2_thiouridine is a modified uridine base feature.	2046	0	0
1459	10	five_aminomethyl_two_thiouridine	5_aminomethyl_2_thiouridine is a modified uridine base feature.	2048	0	0
1460	10	five_methylaminomethyluridine	5_methylaminomethyluridine is a modified uridine base feature.	2050	0	0
1461	10	five_methylaminomethyl_two_thiouridine	5_methylaminomethyl_2_thiouridine is a modified uridine base feature.	2052	0	0
1462	10	five_methylaminomethyl_two_selenouridine	5_methylaminomethyl_2_selenouridine is a modified uridine base feature.	2054	0	0
1463	10	five_carbamoylmethyluridine	5_carbamoylmethyluridine is a modified uridine base feature.	2056	0	0
1464	10	five_carbamoylmethyl_two_prime_O_methyluridine	5_carbamoylmethyl_2_prime_O_methyluridine is a modified uridine base feature.	2058	0	0
1465	10	five_carboxymethylaminomethyluridine	5_carboxymethylaminomethyluridine is a modified uridine base feature.	2060	0	0
1466	10	five_carboxymethylaminomethyl_two_prime_O_methyluridine	5_carboxymethylaminomethyl_2_prime_O_methyluridine is a modified uridine base feature.	2062	0	0
1467	10	five_carboxymethylaminomethyl_two_thiouridine	5_carboxymethylaminomethyl_2_thiouridine is a modified uridine base feature.	2064	0	0
1468	10	three_methyluridine	3_methyluridine is a modified uridine base feature.	2066	0	0
1469	10	one_methyl_three_three_amino_three_carboxypropyl_pseudouridine	1_methyl_3_3_amino_3_carboxypropyl_pseudouridine is a modified uridine base feature.	2068	0	0
1470	10	five_carboxymethyluridine	5_carboxymethyluridine is a modified uridine base feature.	2070	0	0
1471	10	three_two_prime_O_dimethyluridine	3_2prime_O_dimethyluridine is a modified uridine base feature.	2072	0	0
1472	10	five_methyldihydrouridine	5_methyldihydrouridine is a modified uridine base feature.	2074	0	0
1473	10	three_methylpseudouridine	3_methylpseudouridine is a modified uridine base feature.	2076	0	0
1474	10	five_taurinomethyluridine	5_taurinomethyluridine is a modified uridine base feature.	2078	0	0
1475	10	five_taurinomethyl_two_thiouridine	5_taurinomethyl_2_thiouridineis a modified uridine base feature.	2080	0	0
1476	10	five_isopentenylaminomethyl_uridine	5_isopentenylaminomethyl_uridine is a modified uridine base feature.	2082	0	0
1477	10	five_isopentenylaminomethyl_two_thiouridine	5_isopentenylaminomethyl_2_thiouridine is a modified uridine base feature.	2084	0	0
1478	10	five_isopentenylaminomethyl_two_prime_O_methyluridine	5_isopentenylaminomethyl_2prime_O_methyluridine is a modified uridine base feature.	2086	0	0
1479	10	histone_binding_site	A region of a DNA molecule that is bound by a histone.	2088	0	0
1480	10	CDS_fragment	\N	2089	0	0
1481	10	modified_amino_acid_feature	A post translationally modified amino acid feature.	2090	0	0
1482	10	modified_glycine	A post translationally modified glycine amino acid feature.	2091	0	0
1483	10	modified_L_alanine	A post translationally modified alanine amino acid feature.	2093	0	0
1484	10	modified_L_asparagine	A post translationally modified asparagine amino acid feature.	2095	0	0
1485	10	modified_L_aspartic_acid	A post translationally modified aspartic acid amino acid feature.	2097	0	0
1486	10	modified_L_cysteine	A post translationally modified cysteine amino acid feature.	2099	0	0
1487	10	modified_L_glutamic_acid	\N	2101	0	0
1488	10	modified_L_threonine	A post translationally modified threonine amino acid feature.	2103	0	0
1489	10	modified_L_tryptophan	A post translationally modified tryptophan amino acid feature.	2105	0	0
1490	10	modified_L_glutamine	A post translationally modified glutamine amino acid feature.	2107	0	0
1491	10	modified_L_methionine	A post translationally modified methionine amino acid feature.	2109	0	0
1492	10	modified_L_isoleucine	A post translationally modified isoleucine amino acid feature.	2111	0	0
1493	10	modified_L_phenylalanine	A post translationally modified phenylalanine amino acid feature.	2113	0	0
1494	10	modified_L_histidine	A post translationally modified histidie amino acid feature.	2115	0	0
1495	10	modified_L_serine	A post translationally modified serine amino acid feature.	2117	0	0
1496	10	modified_L_lysine	A post translationally modified lysine amino acid feature.	2119	0	0
1497	10	modified_L_leucine	A post translationally modified leucine amino acid feature.	2121	0	0
1498	10	modified_L_selenocysteine	A post translationally modified selenocysteine amino acid feature.	2123	0	0
1499	10	modified_L_valine	A post translationally modified valine amino acid feature.	2125	0	0
1500	10	modified_L_proline	A post translationally modified proline amino acid feature.	2127	0	0
1501	10	modified_L_tyrosine	A post translationally modified tyrosine amino acid feature.	2129	0	0
1502	10	modified_L_arginine	A post translationally modified arginine amino acid feature.	2131	0	0
1503	10	peptidyl	An attribute describing the nature of a proteinaceous polymer, where by the amino acid units are joined by peptide bonds.	2133	0	0
1504	10	cleaved_for_gpi_anchor_region	The C-terminal residues of a polypeptide which are exchanged for a GPI-anchor.	2134	0	0
1505	10	biomaterial_region	A region which is intended for use in an experiment.	2136	0	0
1506	10	experimental_feature	A region which is the result of some arbitrary experimental procedure. The procedure may be carried out with biological material or inside a computer.	2137	0	0
1507	10	biological_region	A region defined by its disposition to be involved in a biological process.	2138	0	0
1508	10	topologically_defined_region	A region that is defined according to its relations with other regions within the same sequence.	2139	0	0
1509	10	translocation_breakpoint	The point within a chromosome where a translocation begins or ends.	2140	0	0
1510	10	insertion_breakpoint	The point within a chromosome where a insertion begins or ends.	2141	0	0
1511	10	deletion_breakpoint	The point within a chromosome where a deletion begins or ends.	2142	0	0
1512	10	five_prime_flanking_region	A flanking region located five prime of a specific region.	2143	0	0
1513	10	three_prime_flanking_region	A flanking region located three prime of a specific region.	2145	0	0
1514	10	transcribed_fragment	An experimental region, defined by a tiling array experiment to be transcribed at some level.	2146	0	0
1515	10	cis_splice_site	Intronic 2 bp region bordering exon. A splice_site that adjacent_to exon and overlaps intron.	2147	0	0
1516	10	trans_splice_site	Primary transcript region bordering trans-splice junction.	2148	0	0
1517	10	splice_junction	The boundary between an intron and an exon.	2149	0	0
1518	10	conformational_switch	A region of a polypeptide, involved in the transition from one conformational state to another.	2150	0	0
1519	10	dye_terminator_read	A read produced by the dye terminator method of sequencing.	2151	0	0
1521	10	ligation_based_read	A read produced by ligation based sequencing technologies.	2153	0	0
1522	10	polymerase_synthesis_read	A read produced by the polymerase based sequence by synthesis method.	2154	0	0
1523	10	cis_regulatory_frameshift_element	A structural region in an RNA molecule which promotes ribosomal frameshifting of cis coding sequence.	2155	0	0
1524	10	expressed_sequence_assembly	A sequence assembly derived from expressed sequences.	2157	0	0
1525	10	DNA_binding_site	A region of a molecule that binds to DNA.	2158	0	0
1526	10	polyA_junction	The boundary between the UTR and the polyA sequence.	2159	0	0
1527	10	cryptic_gene	A gene that is not transcribed under normal conditions and is not critical to normal cellular functioning.	2160	0	0
1528	10	sequence_variant_affecting_polyadenylation	\N	2161	0	0
1529	10	three_prime_RACE_clone	A three prime RACE (Rapid Amplification of cDNA Ends) clone is a cDNA clone copied from the 3' end of an mRNA (using a poly-dT primer to capture the polyA tail and a gene-specific or randomly primed 5' primer), and spliced into a vector for propagation in a suitable host.	2162	0	0
1530	10	cassette_pseudogene	A cassette pseudogene is a kind of gene in an innactive form which may recombine at a telomeric locus to form a functional copy.	2164	0	0
1531	10	alanine	\N	2165	0	0
1532	10	valine	\N	2166	0	0
1533	10	leucine	\N	2167	0	0
1534	10	isoleucine	\N	2168	0	0
1535	10	proline	\N	2169	0	0
1536	10	tryptophan	\N	2170	0	0
1537	10	phenylalanine	\N	2171	0	0
1538	10	methionine	\N	2172	0	0
1539	10	glycine	\N	2173	0	0
1540	10	serine	\N	2174	0	0
1541	10	threonine	\N	2175	0	0
1542	10	tyrosine	\N	2176	0	0
1543	10	cysteine	\N	2177	0	0
1544	10	glutamine	\N	2178	0	0
1545	10	asparagine	\N	2179	0	0
1546	10	lysine	\N	2180	0	0
1547	10	argenine	\N	2181	0	0
1548	10	histidine	\N	2182	0	0
1549	10	aspartic_acid	\N	2183	0	0
1550	10	glutamic_acid	\N	2184	0	0
1551	10	selenocysteine	\N	2185	0	0
1552	10	pyrrolysine	\N	2186	0	0
1553	10	transcribed_cluster	A region defined by a set of transcribed sequences from the same gene or expressed pseudogene.	2187	0	0
1554	10	unigene_cluster	A kind of transcribed_cluster defined by a set of transcribed sequences from the a unique gene.	2188	0	0
1555	10	CRISPR	Clustered Palindromic Repeats interspersed with bacteriophage derived spacer sequences.	2189	0	0
1556	10	insulator_binding_site	A protein_binding_site located within an insulator.	2191	0	0
1557	10	enhancer_binding_site	A protein_binding_site located within an enhancer.	2192	0	0
1558	10	contig_collection	A collection of contigs.	2193	0	0
1559	10	lincRNA	A multiexonic non-coding RNA transcribed by RNA polymerase II.	2194	0	0
1560	10	UST	An EST spanning part or all of the untranslated regions of a protein-coding transcript.	2196	0	0
1561	10	three_prime_UST	A UST located in the 3'UTR of a protein-coding transcript.	2198	0	0
1562	10	five_prime_UST	An UST located in the 5'UTR of a protein-coding transcript.	2199	0	0
1563	10	RST	A tag produced from a single sequencing read from a RACE product; typically a few hundred base pairs long.	2200	0	0
1564	10	three_prime_RST	A tag produced from a single sequencing read from a 3'-RACE product; typically a few hundred base pairs long.	2201	0	0
1565	10	five_prime_RST	A tag produced from a single sequencing read from a 5'-RACE product; typically a few hundred base pairs long.	2202	0	0
1566	10	UST_match	A match against an UST sequence.	2203	0	0
1567	10	RST_match	A match against an RST sequence.	2204	0	0
1568	10	primer_match	A nucleotide match to a primer sequence.	2205	0	0
1569	10	miRNA_antiguide	A region of the pri miRNA that basepairs with the guide to form the hairpin.	2206	0	0
1570	10	trans_splice_junction	The boundary between the spliced leader and the first exon of the mRNA.	2207	0	0
1571	10	outron	A region of a primary transcript, that is removed via trans splicing.	2208	0	0
1572	10	natural_plasmid	A plasmid that occurs naturally.	2210	0	0
1573	10	gene_trap_construct	A gene trap construct is a type of engineered plasmid which is designed to integrate into a genome and produce a fusion transcript between exons of the gene into which it inserts and a reporter element in the construct. Gene traps contain a splice acceptor, do not contain promoter elements for the reporter, and are mutagenic. Gene traps may be bicistronic with the second cassette containing a promoter driving an a selectable marker.	2211	0	0
1574	10	promoter_trap_construct	A promoter trap construct is a type of engineered plasmid which is designed to integrate into a genome and express a reporter when inserted in close proximity to a promoter element. Promoter traps typically do not contain promoter elements and are mutagenic.	2213	0	0
1575	10	enhancer_trap_construct	An enhancer trap construct is a type of engineered plasmid which is designed to integrate into a genome and express a reporter when the expression from a basic minimal promoter is enhanced by genomic enhancer elements. Enhancer traps contain promoter elements and are not usually mutagenic.	2214	0	0
1576	10	PAC_end	A region of sequence from the end of a PAC clone that may provide a highly specific marker.	2215	0	0
1577	10	RAPD	RAPD is a 'PCR product' where a sequence variant is identified through the use of PCR with random primers.	2217	0	0
1578	10	shadow_enhancer	\N	2218	0	0
1579	10	SNV	SNVs are single base pair positions in genomic DNA at which different sequence alternatives exist.	2219	0	0
1580	10	X_element_combinatorial_repeat	An X element combinatorial repeat is a repeat region located between the X element and the telomere or adjacent Y' element.	2220	0	0
1581	10	Y_prime_ element	A Y' element is a repeat region (SO:0000657) located adjacent to telomeric repeats or X element combinatorial repeats, either as a single copy or tandem repeat of two to four copies.	2222	0	0
1582	10	standard_draft	The status of a whole genome sequence, where the data is minimally filtered or unfiltered, from any number of sequencing platforms,  and is assembled into contigs. Genome sequence of this quality may harbor regions of poor quality and can be relatively incomplete.	2224	0	0
2254	14	leaf area	\N	2961	0	0
1583	10	high_quality_draft	The status of a whole genome sequence, where overall coverage represents at least 90 percent of the genome.	2226	0	0
1584	10	improved_high_quality_draft	The status of a whole genome sequence, where additional work has been performed, using either manual or automated methods, such as gap resolution.	2227	0	0
1585	10	annotation_directed_improved_draft	The status of a whole genome sequence,where annotation, and verification of coding regions has occurred.	2228	0	0
1586	10	noncontiguous_finished	The status of a whole genome sequence, where the assembly is high quality, closure approaches have been successful for most gaps, misassemblies and low quality regions.	2229	0	0
1587	10	finished_genome	The status of a whole genome sequence, with less than 1 error per 100,000 base pairs.	2230	0	0
1588	10	intronic_regulatory_region	A regulatory region that is part of an intron.	2231	0	0
1589	10	centromere_DNA_Element_I	A centromere DNA Element I (CDEI) is a conserved region,  part of the centromere, consisting of a consensus regionncomposed of 8-11bp which enables binding by the centromere binding factor 1(Cbf1p).	2232	0	0
1590	10	centromere_DNA_Element_II	A centromere DNA Element II (CDEII) is part a conserved region of the centromere, consisting of a consensus region that is AT-rich and ~ 75-100 bp in length.	2234	0	0
1591	10	centromere DNA Element III	A centromere DNA Element I (CDEI) is a conserved region,  part of the centromere, consisting of a consensus region that consists of a 25-bp which enables binding by the centromere DNA binding factor 3 (CBF3) complex.	2235	0	0
1592	10	telomeric_repeat	The telomeric repeat is a repeat region, part of the chromosome, which in yeast, is a G-rich terminal sequence of the form (TG(1-3))n or more precisely ((TG)(1-6)TG(2-3))n.	2236	0	0
1593	10	X_element	The X element is a conserved region, of the telomere, of ~475 bp that contains an ARS sequence and in most cases an Abf1p binding site.	2238	0	0
1594	10	YAC_end	A region of sequence from the end of a YAC clone that may provide a highly specific marker.	2240	0	0
1595	10	whole_genome_sequence_status	The status of whole genome sequence.	2241	0	0
1596	10	heritable_phenotypic_marker	A biological_region characterized as a single heritable trait in a phenotype screen. The heritable phenotype may be mapped to a chromosome but generally has not been characterized to a specific gene locus.	2242	0	0
1597	10	peptide_collection	A collection of peptide sequences.	2244	0	0
1598	10	high_identity_region	An experimental feature with high sequence identity to another sequence.	2246	0	0
1599	10	processed_transcript	A transcript for which no open reading frame has been identified and for which no other function has been determined.	2247	0	0
1600	10	assortment_derived_variation	A chromosome variation derived from an event during meiosis.	2249	0	0
1601	10	reference_genome	A collection of sequences (often chromosomes) taken as the standard for a given organism and genome assembly.	2250	0	0
1602	10	variant_genome	A collection of sequences (often chromosomes) of  an individual.	2251	0	0
1603	10	variant_collection	A collection of one or more sequences of an individual.	2252	0	0
1604	10	alteration_attribute	\N	2253	0	0
1605	10	chromosomal_variation_attribute	\N	2254	0	0
1606	10	intrachromosomal	\N	2255	0	0
1607	10	interchromosomal	\N	2256	0	0
1608	10	insertion_attribute	A quality of a chromosomal insertion,.	2257	0	0
1609	10	tandem	\N	2258	0	0
1610	10	direct	A quality of an insertion where the insert is not in a cytologically inverted orientation.	2259	0	0
1611	10	inverted	A quality of an insertion where the insert is in a cytologically inverted orientation.	2260	0	0
1612	10	free	The quality of a duplication where the new region exists independently of the original.	2261	0	0
1613	10	inversion_attribute	\N	2262	0	0
1614	10	pericentric	\N	2263	0	0
1615	10	paracentric	\N	2264	0	0
1616	10	translocaton_attribute	\N	2265	0	0
1617	10	reciprocal	\N	2266	0	0
1618	10	insertional	\N	2267	0	0
1619	10	duplication_attribute	\N	2268	0	0
1620	10	chromosomally_aberrant_genome	\N	2269	0	0
1621	10	assembly_error_correction	A region of sequence where the final nucleotide assignment differs from the original assembly due to an improvement that replaces a mistake.	2270	0	0
1622	10	base_call_error_correction	A region of sequence where the final nucleotide assignment is different from that given by the base caller due to an improvement that replaces a mistake.	2271	0	0
1623	10	peptide_localization_signal	A region of peptide sequence used to target the polypeptide molecule to a specific organelle.	2272	0	0
1624	10	nuclear_localization_signal	A polypeptide region that targets a polypeptide to the nucleus.	2273	0	0
1625	10	endosomal_localization_signal	A polypeptide region that targets a polypeptide to the endosome.	2275	0	0
1626	10	lysosomal_localization_signal	A polypeptide region that targets a polypeptide to the lysosome.	2276	0	0
1627	10	nuclear_export_signal	A polypeptide region that targets a polypeptide to he cytoplasm.	2277	0	0
1628	10	recombination_signal_sequence	A region recognized by a recombinase.	2279	0	0
1629	10	cryptic_splice_site	A splice site that is in part of the transcript not normally spliced. They occur via mutation or transcriptional error.	2281	0	0
1630	10	nuclear_rim_localization_signal	A polypeptide region that targets a polypeptide to the nuclear rim.	2282	0	0
1631	10	p_element	A P_element is a DNA transposon responsible for hybrid dysgenesis.	2284	0	0
1632	10	regulatory_region	A DNA sequence that controls the expression of a gene.	2285	0	0
1633	10	U14_snoRNA_primary_transcript	The primary transcript of an evolutionarily conserved eukaryotic low molecular weight RNA capable of intermolecular hybridization with both homologous and heterologous 18S rRNA.	2287	0	0
1634	10	methylation_guide_snoRNA	A snoRNA that specifies the site of 2'-O-ribose methylation in an RNA molecule by base pairing with a short sequence around the target residue.	2289	0	0
1635	10	rRNA_cleavage_RNA	An ncRNA that is part of a ribonucleoprotein that cleaves the primary pre-rRNA transcript in the process of producing mature rRNA molecules.	2290	0	0
1636	10	exon_of_single_exon_gene	An exon that is the only exon in a gene.	2291	0	0
1637	10	cassette_array_member	\N	2292	0	0
1639	10	gene_subarray_member	\N	2294	0	0
1640	10	primer_binding_site	Non-covalent primer binding site for initiation of replication, transcription, or reverse transcription.	2295	0	0
1641	10	gene_array	An array includes two or more genes, or two or more gene subarrays, contiguously arranged where the individual genes, or subarrays, are either identical in sequence, or essentially so.	2297	0	0
1642	10	gene_subarray	A subarray is, by defintition, a member of a gene array (SO:0005851); the members of a subarray may differ substantially in sequence, but are closely related in function.	2298	0	0
1643	10	gene_cassette	A gene that can be substituted for a related gene at a different site in the genome.	2299	0	0
1644	10	gene_cassette_array	An array of non-functional genes whose members, when captured by recombination form functional genes.	2302	0	0
1645	10	gene_group	A collection of related genes.	2303	0	0
1646	10	selenocysteine_tRNA_primary_transcript	A primary transcript encoding seryl tRNA (SO:000269).	2304	0	0
1647	10	selenocysteinyl_tRNA	A tRNA sequence that has a selenocysteine anticodon, and a 3' selenocysteine binding region.	2305	0	0
1648	10	syntenic_region	A region in which two or more pairs of homologous markers occur on the same chromosome in two or more species.	2306	0	0
1649	10	biochemical_region_of_peptide	A region of a peptide that is involved in a biochemical function.	2308	0	0
1650	10	molecular_contact_region	A region that is involved a contact with another molecule.	2309	0	0
1651	10	intrinsically_unstructured_polypeptide_region	A region of polypeptide chain with high conformational flexibility.	2310	0	0
1652	10	catmat_left_handed_three	A motif of 3 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -75 bounds -100 to -50, res i+1: psi 140 bounds 110 to 170. An extra restriction of the length of the O to O distance would be useful, that it be less than 5 Angstrom. More precisely these two oxygens are the main chain carbonyl oxygen atoms of residues i-1 and i+1.	2311	0	0
1653	10	catmat_left_handed_four	A motif of 4 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i psi -10 bounds -50 to 30, res i+1: phi -90 bounds -120 to -60, res i+1: psi -10 bounds -50 to 30, res i+2: phi -75 bounds -100 to -50, res i+2: psi 140 bounds 110 to 170.  The extra restriction of the length of the O to O distance is similar, that it be less than 5 Angstrom. In this case these two Oxygen atoms are the main chain carbonyl oxygen atoms of residues i-1 and i+2.	2312	0	0
1654	10	catmat_right_handed_three	A motif of 3 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -75 bounds -100 to -50, res i+1: psi 140 bounds 110 to 170. An extra restriction of the length of the O to O distance would be useful, that it be less than 5 Angstrom. More precisely these two oxygens are the main chain carbonyl oxygen atoms of residues i-1 and i+1.	2313	0	0
1655	10	catmat_right_handed_four	A motif of 4 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -90 bounds -120 to -60, res i+1: psi -10 bounds -50 to 30, res i+2: phi -75 bounds -100 to -50, res i+2: psi 140 bounds 110 to 170. The extra restriction of the length of the O to O distance is similar, that it be less than 5 Angstrom. In this case these two Oxygen atoms are the main chain carbonyl oxygen atoms of residues i-1 and i+2.	2314	0	0
1656	10	alpha_beta_motif	A motif of five consecutive residues and two H-bonds in which: H-bond between CO of residue(i) and NH of residue(i+4), H-bond between CO of residue(i) and NH of residue(i+3),Phi angles of residues(i+1), (i+2) and (i+3) are negative.	2315	0	0
1657	10	lipoprotein_signal_peptide	A peptide that acts as a signal for both membrane translocation and lipid attachment in prokaryotes.	2316	0	0
1658	10	no_output	An experimental region wherean analysis has been run and not produced any annotation.	2317	0	0
1659	10	cleaved_peptide_region	The cleaved_peptide_regon is the a region of peptide sequence that is cleaved during maturation.	2318	0	0
1660	10	peptide_coil	Irregular, unstructured regions of a protein's backbone, as distinct from the regular region (namely alpha helix and beta strand - characterised by specific patterns of main-chain hydrogen bonds).	2319	0	0
1661	10	hydrophobic_region_of_peptide	Hydrophobic regions are regions with a low affinity for water.	2320	0	0
1662	10	n_terminal_region	The amino-terminal positively-charged region of a signal peptide (approx 1-5 aa).	2321	0	0
1663	10	c_terminal_region	The more polar, carboxy-terminal region of the signal peptide (approx 3-7 aa).	2322	0	0
1664	10	central_hydrophobic_region_of_signal_peptide	The central, hydrophobic region of the signal peptide (approx 7-15 aa).	2323	0	0
1665	10	polypeptide_conserved_motif	A conserved motif is a short (up to 20 amino acids) region of biological interest that is conserved in different proteins. They may or may not have functional or structural significance within the proteins in which they are found.	2324	0	0
1666	10	polypeptide_binding_motif	A polypeptide binding motif is a short (up to 20 amino acids) polypeptide region of biological interest that contains one or more amino acids experimentally shown to bind to a ligand.	2325	0	0
1667	10	polypeptide_catalytic_motif	A polypeptide catalytic motif is a short (up to 20 amino acids) polypeptide region that contains one or more active site residues.	2326	0	0
1668	10	polypeptide_DNA_contact	Residues involved in interactions with DNA.	2327	0	0
1669	10	polypeptide_conserved_region	A subsection of sequence with biological interest that is conserved in different proteins. They may or may not have functional or structural significance within the proteins in which they are found.	2328	0	0
1670	10	substitution	Any change in genomic DNA caused by a single event.	2329	0	0
1671	10	complex_substitution	When no simple or well defined DNA mutation event describes the observed DNA change, the keyword "complex" should be used. Usually there are multiple equally plausible explanations for the change.	2331	0	0
1672	10	point_mutation	A single nucleotide change which has occurred at the same position of a corresponding nucleotide in a reference sequence.	2332	0	0
1673	10	transition	Change of a pyrimidine nucleotide, C or T, into an other pyrimidine nucleotide, or change of a purine nucleotide, A or G, into an other purine nucleotide.	2334	0	0
1674	10	pyrimidine_transition	A substitution of a pyrimidine, C or T, for another pyrimidine.	2335	0	0
1675	10	C_to_T_transition	A transition of a cytidine to a thymine.	2336	0	0
2255	14	leaf fresh weight	\N	2962	0	0
1676	10	C_to_T_transition_at_pCpG_site	The transition of cytidine to thymine occurring at a pCpG site as a consequence of the spontaneous deamination of 5'-methylcytidine.	2337	0	0
1677	10	T_to_C_transition	\N	2338	0	0
1678	10	purine_transition	A substitution of a purine, A or G, for another purine.	2339	0	0
1679	10	A_to_G_transition	A transition of an adenine to a guanine.	2340	0	0
1680	10	G_to_A_transition	A transition of a guanine to an adenine.	2341	0	0
1681	10	transversion	Change of a pyrimidine nucleotide, C or T, into a purine nucleotide, A or G, or vice versa.	2342	0	0
1682	10	pyrimidine_to_purine_transversion	Change of a pyrimidine nucleotide, C or T, into a purine nucleotide, A or G.	2344	0	0
1683	10	C_to_A_transversion	A transversion from cytidine to adenine.	2345	0	0
1684	10	C_to_G_transversion	\N	2346	0	0
1685	10	T_to_A_transversion	A transversion from T to A.	2347	0	0
1686	10	T_to_G_transversion	A transversion from T to G.	2348	0	0
1687	10	purine_to_pyrimidine_transversion	Change of a purine nucleotide, A or G , into a pyrimidine nucleotide C or T.	2349	0	0
1688	10	A_to_C_transversion	A transversion from adenine to cytidine.	2350	0	0
1689	10	A_to_T_transversion	A transversion from adenine to thymine.	2351	0	0
1690	10	G_to_C_transversion	A transversion from guanine to cytidine.	2352	0	0
1691	10	G_to_T_transversion	A transversion from guanine to thymine.	2353	0	0
1692	10	intrachromosomal_mutation	A chromosomal structure variation within a single chromosome.	2354	0	0
1693	10	chromosomal_deletion	An incomplete chromosome.	2355	0	0
1694	10	chromosomal_inversion	An interchromosomal mutation where a region of the chromosome is inverted with respect to wild type.	2357	0	0
1695	10	interchromosomal_mutation	A chromosomal structure variation whereby more than one chromosome is involved.	2359	0	0
1696	10	indel	A sequence alteration which included an insertion and a deletion, affecting 2 or more bases.	2360	0	0
1697	10	duplication	One or more nucleotides are added between two adjacent nucleotides in the sequence; the inserted sequence derives from, or is identical in sequence to, nucleotides adjacent to insertion point.	2363	0	0
1698	10	inversion	A continuous nucleotide sequence is inverted in the same position.	2364	0	0
1699	10	chromosomal_duplication	An extra chromosome.	2365	0	0
1700	10	intrachromosomal_duplication	A duplication that occurred within a chromosome.	2367	0	0
1701	10	direct_tandem_duplication	A tandem duplication where the individual regions are in the same orientation.	2368	0	0
1702	10	inverted_tandem_duplication	A tandem duplication where the individual regions are not in the same orientation.	2369	0	0
1703	10	intrachromosomal_transposition	A chromosome structure variation whereby a transposition occurred within a chromosome.	2370	0	0
1704	10	compound_chromosome	A chromosome structure variant where a monocentric element is caused by the fusion of two chromosome arms.	2371	0	0
1705	10	Robertsonian_fusion	A non reciprocal translocation whereby the participating chromosomes break at their centromeres and the long arms fuse to form a single chromosome with a single centromere.	2372	0	0
1706	10	chromosomal_translocation	An interchromosomal mutation. Rearrangements  that alter the pairing of telomeres are classified as translocations.	2375	0	0
1707	10	ring_chromosome	A ring chromosome is a chromosome whose arms have fused together to form a ring, often with the loss of the ends of the chromosome.	2377	0	0
1708	10	pericentric_inversion	A chromosomal inversion that includes the centromere.	2379	0	0
1709	10	paracentric_inversion	A chromosomal inversion that does not include the centromere.	2380	0	0
1710	10	reciprocal_chromosomal_translocation	A chromosomal translocation with two breaks; two chromosome segments have simply been exchanged.	2381	0	0
1711	10	sequence_variation_affecting_transcript	Any change in mature, spliced and processed, RNA that results from a change in the corresponding DNA sequence.	2382	0	0
1769	10	sequence_variant_causing_no_3D_structural_change	\N	2447	0	0
1720	10	sequence_variant_causing_conservative_missense_codon_change_in_transcript	The amino acid change following from the codon change does not change the gross properties (size, charge, hydrophobicity) of the amino acid at that position.	2393	0	0
1712	10	sequence_variant_causing_no_change_in_transcript	No effect on the state of the RNA.	2385	0	0
1713	10	sequence_variation_affecting_complex_change_in_transcript	\N	2386	0	0
1721	10	sequence_variant_causing_nonconservative_missense_codon_change_in_transcript	The amino acid change following from the codon change changes the gross properties (size, charge, hydrophobicity) of the amino acid in that position.	2394	0	0
1722	10	sequence_variant_causing_nonsense_codon_change_in_transcript	The nucleotide change in the codon triplet creates a terminator codon.	2395	0	0
1714	10	sequence_variation_affecting_coding_sequence	Any of the amino acid coding triplets of a gene are affected by the DNA mutation.	2387	0	0
1723	10	sequence_variant_causing_terminator_codon_change_in_transcript	The nucleotide change in the codon triplet changes the stop codon, causing an elongated transcript sequence.	2396	0	0
1715	10	sequence_variant_causing_initiator_codon_change_in_transcript	The DNA mutation changes, usually destroys, the first coding triplet of a gene. Usually prevents translation although another initiator codon may be used.	2388	0	0
1724	10	sequence_variation_affecting_reading_frame	An umbrella term for terms describing an effect of a sequence variation on the frame of translation.	2397	0	0
1716	10	sequence_variant_causing_amino_acid_coding_codon_change_in_transcript	The DNA mutation affects the amino acid coding sequence of a gene; this region includes both the initiator and terminator codons.	2389	0	0
1770	10	sequence_variant_causing_complex_3D_structural_change	\N	2448	0	0
1717	10	sequence_variant_causing_synonymous_codon_change_in_transcript	The changed codon has the same translation product as the original codon.	2390	0	0
1718	10	sequence_variant_causing_non_synonymous_codon_change_in_transcript	A DNA point mutation that causes a substitution of an amino acid by an other.	2391	0	0
1719	10	sequence_variant_causing_missense_codon_change_in_transcript	The nucleotide change in the codon leads to a new codon coding for a new amino acid.	2392	0	0
1725	10	frameshift_sequence_variation	A mutation causing a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three.	2398	0	0
1726	10	sequence_variant_causing_plus_1_frameshift_mutation	A mutation causing a disruption of the translational reading frame, due to the insertion of a nucleotide.	2400	0	0
1727	10	sequence_variant_causing_minus_1_frameshift	A mutation causing a disruption of the translational reading frame, due to the deletion of a nucleotide.	2401	0	0
1728	10	sequence_variant_causing_plus_2_frameshift	A mutation causing a disruption of the translational reading frame, due to the insertion of two nucleotides.	2402	0	0
1729	10	sequence_variant_causing_minus_2_frameshift	A mutation causing a disruption of the translational reading frame, due to the deletion of two nucleotides.	2403	0	0
1730	10	sequence_variant_affecting_transcript_processing	Sequence variant affects the way in which the primary transcriptional product is processed to form the mature transcript.	2404	0	0
1731	10	sequence_variant_affecting_splicing	A sequence_variant_effect where the way in which the primary transcriptional product is processed to form the mature transcript, specifically by the removal (splicing) of intron sequences is changed.	2405	0	0
1732	10	sequence_variant_affecting_splice_donor	A sequence_variant_effect that changes the splice donor sequence.	2406	0	0
1733	10	sequence_variant_affecting_splice_acceptor	A sequence_variant_effect that changes the splice acceptor sequence.	2407	0	0
1734	10	sequence_variant_causing_cryptic_splice_activation	A sequence variant causing a new (functional) splice site.	2408	0	0
1735	10	sequence_variant_affecting_editing	Sequence variant affects the editing of the transcript.	2409	0	0
1736	10	sequence_variant_affecting_transcription	Mutation affects the process of transcription, its initiation, progression or termination.	2410	0	0
1737	10	sequence_variant_decreasing_rate_of_transcription	A sequence variation that decreases the rate a which transcription of the sequence occurs.	2411	0	0
1738	10	sequence_variation_affecting_transcript_sequence	\N	2412	0	0
1739	10	sequence_variant_increasing_rate_of_transcription	\N	2413	0	0
1740	10	sequence_variant_affecting_rate_of_transcription	A mutation that alters the rate a which transcription of the sequence occurs.	2414	0	0
1741	10	sequence variant_affecting_transcript_stability	Sequence variant affects the stability of the transcript.	2415	0	0
1742	10	sequence_variant_increasing_transcript_stability	Sequence variant increases the stability (half-life) of the transcript.	2416	0	0
1743	10	sequence_variant_decreasing_transcript_stability	Sequence variant decreases the stability (half-life) of the transcript.	2417	0	0
1744	10	sequence_variation_affecting_level_of_transcript	A sequence variation that causes a change in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence.	2418	0	0
1745	10	sequence_variation_decreasing_level_of_transcript	A sequence variation that causes a decrease in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence.	2419	0	0
1746	10	sequence_variation_increasing_level_of_transcript	A sequence_variation that causes an increase in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence.	2420	0	0
1747	10	sequence_variant_affecting_translational_product	MERGED DEFINITION:nTARGET DEFINITION: MERGED DEFINITION:nTARGET DEFINITION: Mutation causes a change in primary translation product of a transcript.n--------------------nSOURCE DEFINITION: A sequence variant causing a partially uncharacterised change in translational product.n--------------------nSOURCE DEFINITION: A sequence variant causing an uncharacterized change of translational product.	2421	0	0
1748	10	sequence_variant_causing_no_change_of_translational_product	The sequence variant at RNA level does not lead to any change in polypeptide.	2424	0	0
1749	10	sequence_variant_causing_complex_change_of_translational_product	Any sequence variant effect that is known at nucleotide level but cannot be explained by using other key terms.	2425	0	0
1760	10	sequence_variant_decreasing_level_of_translation_product	\N	2436	0	0
1750	10	sequence_variant_causing_amino_acid_substitution	The replacement of a single amino acid by another.	2426	0	0
1751	10	sequence_variant_causing_conservative_amino_acid_substitution	\N	2427	0	0
1761	10	sequence_variant_increasing_level_of_translation_product	\N	2437	0	0
1752	10	sequence_variant_causing_nonconservative_amino_acid_substitution	\N	2428	0	0
1764	10	mutation_causing_out_of_frame_polypeptide_N_terminal_elongation	\N	2440	0	0
1753	10	sequence_variant_causing_amino_acid_insertion	The insertion of one or more amino acids from the polypeptide, without affecting the surrounding sequence.	2429	0	0
1762	10	sequence_variant_affecting_polypeptide_amino_acid_sequence	\N	2438	0	0
1754	10	sequence_variant_causing_amino_acid_deletion	The deletion of one or more amino acids from the polypeptide, without affecting the surrounding sequence.	2430	0	0
1755	10	sequence_variant_causing_polypeptide_truncation	The translational product is truncated at its C-terminus, usually a result of a nonsense codon change in transcript (SO:1000062).	2431	0	0
1763	10	mutation_causing_inframe_polypeptide_N_terminal_elongation	\N	2439	0	0
1756	10	sequence_variant_causing_polypeptide_elongation	The extension of the translational product at either (or both) the N-terminus and/or the C-terminus.	2432	0	0
1765	10	mutaton_causing_inframe_polypeptide_C_terminal_elongation	\N	2441	0	0
1757	10	mutation_causing_polypeptide_N_terminal_elongation	.	2433	0	0
1766	10	mutation_causing_out_of_frame_polypeptide_C_terminal_elongation	\N	2442	0	0
1767	10	frame_restoring_sequence_variant	A mutation that reverts the sequence of a previous frameshift mutation back to the initial frame.	2443	0	0
1758	10	mutation_causing_polypeptide_C_terminal_elongation	.	2434	0	0
1768	10	sequence_variant_affecting_3D_structure_of_polypeptide	A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule.	2444	0	0
1759	10	sequence_variant_affecting_level_of_translational_product	\N	2435	0	0
1771	10	sequence_variant_causing_conformational_change	\N	2449	0	0
1772	10	sequence_variant_affecting_polypeptide_function	\N	2450	0	0
2256	14	leaf dry weight	\N	2963	0	0
1773	10	sequence_variant_causing_loss_of_function_of_polypeptide	\N	2451	0	0
1803	10	inverted_interchromosomal_transposition	An interchromosomal transposition whereby a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segment.	2483	0	0
1774	10	sequence_variant_causing_inactive_ligand_binding_site	\N	2452	0	0
1808	10	unoriented_interchromosomal_transposition	An interchromosomal transposition whereby a copy of the segment between the first two breaks listed is inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded.	2488	0	0
1775	10	sequence_variant_causing_inactive_catalytic_site	\N	2453	0	0
1804	10	uninverted_interchromosomal_transposition	An interchromosomal transition where the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments.	2484	0	0
1776	10	sequence_variant_causing_polypeptide_localization_change	\N	2454	0	0
2257	14	plant condition	\N	2964	0	0
1777	10	sequence_variant_causing_polypeptide_post_translational_processing_change	\N	2455	0	0
1805	10	inverted_intrachromosomal_transposition	An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segments.	2485	0	0
1809	10	unoriented_intrachromosomal_transposition	An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded.	2489	0	0
1778	10	polypeptide_post_translational_processing_affected (obsolete SO:1000123)	\N	2456	1	0
1779	10	sequence_variant_causing_partial_loss_of_function_of_polypeptide	\N	2457	0	0
1806	10	uninverted_intrachromosomal_transposition	An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments.	2486	0	0
1810	10	uncharacterised_chromosomal_mutation	\N	2490	0	0
1780	10	sequence_variant_causing_gain_of_function_of_polypeptide	\N	2458	0	0
1807	10	unoriented_insertional_duplication	An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded.	2487	0	0
2258	14	thrip	\N	2965	0	0
1781	10	sequence_variant_affecting_transcript_secondary_structure	A sequence variant that affects the secondary structure (folding) of the RNA transcript molecule.	2459	0	0
1782	10	sequence_variant_causing_compensatory_transcript_secondary_structure_mutation	\N	2460	0	0
1783	10	sequence_variant_effect	The effect of a change in nucleotide sequence.	2461	0	0
1784	10	sequence_variant_causing_polypeptide_fusion	\N	2462	0	0
1785	10	autosynaptic_chromosome	An autosynaptic chromosome is the aneuploid product of recombination between a pericentric inversion and a cytologically wild-type chromosome.	2463	0	0
1786	10	homo_compound_chromosome	A compound chromosome whereby two copies of the same chromosomal arm attached to a common centromere. The chromosome is diploid for the arm involved.	2465	0	0
1787	10	hetero_compound_chromosome	A compound chromosome whereby two arms from different chromosomes are connected through the centromere of one of them.	2466	0	0
1788	10	chromosome_fission	A chromosome that occurred by the division of a larger chromosome.	2467	0	0
1789	10	dexstrosynaptic_chromosome	An autosynaptic chromosome carrying the two right (D = dextro) telomeres.	2468	0	0
1790	10	laevosynaptic_chromosome	LS is an autosynaptic chromosome carrying the two left (L = levo) telomeres.	2470	0	0
1791	10	free_duplication	A chromosome structure variation whereby the duplicated sequences are carried as a free centric element.	2471	0	0
1792	10	free_ring_duplication	A ring chromosome which is a copy of another chromosome.	2472	0	0
1793	10	complex_chromosomal_mutation	A chromosome structure variant with 4 or more breakpoints.	2473	0	0
1794	10	deficient_translocation	A chromosomal deletion whereby a translocation occurs in which one of the four broken ends loses a segment before re-joining.	2474	0	0
1795	10	inversion_cum_translocation	A chromosomal translocation whereby the first two breaks are in the same chromosome, and the region between them is rejoined in inverted order to the other side of the first break, such that both sides of break one are present on the same chromosome. The remaining free ends are joined as a translocation with those resulting from the third break.	2475	0	0
1796	10	bipartite_duplication	An interchromosomal mutation whereby the (large) region between the first two breaks listed is lost, and the two flanking segments (one of them centric) are joined as a translocation to the free ends resulting from the third break.	2476	0	0
1797	10	cyclic_translocation	A chromosomal translocation whereby three breaks occurred in three different chromosomes. The centric segment resulting from the first break listed is joined to the acentric segment resulting from the second, rather than the third.	2477	0	0
1798	10	bipartite_inversion	A chromosomal inversion caused by three breaks in the same chromosome; both central segments are inverted in place (i.e., they are not transposed).	2478	0	0
1799	10	uninverted_insertional_duplication	An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments.	2479	0	0
1800	10	inverted_insertional_duplication	An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segments.	2480	0	0
1801	10	insertional_duplication	A chromosome duplication involving the insertion of a duplicated region (as opposed to a free duplication).	2481	0	0
1802	10	interchromosomal_transposition	A chromosome structure variation whereby a transposition occurred between chromosomes.	2482	0	0
1811	10	deficient_inversion	A chromosomal deletion whereby three breaks occur in the same chromosome; one central region is lost, and the other is inverted.	2491	0	0
1812	10	tandem_duplication	A duplication consisting of 2 identical regions, which are adjacent.	2492	0	0
1813	10	partially_characterised_chromosomal_mutation	\N	2493	0	0
1814	10	sequence_variant_affecting_gene_structure	A sequence_variant_effect that changes the gene structure.	2494	0	0
1815	10	sequence_variant_causing_gene_fusion	A sequence_variant_effect that changes the gene structure by causing a fusion to another gene.	2495	0	0
1816	10	chromosome_number_variation	A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number.	2496	0	0
1817	10	chromosome_structure_variation	\N	2497	0	0
1818	10	sequence_variant_causes_exon_loss	A sequence variant affecting splicing and causes an exon loss.	2498	0	0
1819	10	sequence_variant_causes_intron_gain	A sequence variant effect, causing an intron to be gained by the processed transcript; usually a result of a donor acceptor mutation (SO:1000072).	2499	0	0
1820	10	sequence_variant_causing_cryptic_splice_donor_activation	\N	2500	0	0
1821	10	sequence_variant_causing_cryptic_splice_acceptor_activation	\N	2501	0	0
1822	10	alternatively_spliced_transcript	A transcript that is alternatively spliced.	2502	0	0
1823	10	encodes_1_polypeptide	A gene that is alternately spliced, but encodes only one polypeptide.	2503	0	0
1824	10	encodes_greater_than_1_polypeptide	A gene that is alternately spliced, and encodes more than one polypeptide.	2504	0	0
1825	10	encodes_different_polypeptides_different_stop	A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different stop codons.	2505	0	0
1826	10	encodes_overlapping_peptides_different_start	A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different start codons.	2506	0	0
1827	10	encodes_disjoint_polypeptides	A gene that is alternately spliced, and encodes more than one polypeptide, that do not have overlapping peptide sequences.	2507	0	0
1828	10	encodes_overlapping_polypeptides_different_start_and_stop	A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different start and stop codons.	2508	0	0
1829	10	alternatively_spliced_gene_encoding_greater_than_1_polypeptide_coding_regions_overlapping (obsolete SO:1001194)	\N	2509	1	0
1830	10	encodes_overlapping_peptides	A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences.	2510	0	0
1831	10	cryptogene	A maxicircle gene so extensively edited that it cannot be matched to its edited mRNA sequence.	2511	0	0
1832	10	dicistronic_primary_transcript	A primary transcript that has the quality dicistronic.	2512	0	0
1833	10	member_of_regulon	\N	2513	0	0
1834	10	alternatively_spliced_transcript_encoding_greater_than_1_polypeptide_different_start_codon_different_stop_codon_coding_regions_non_overlapping (obsolete SO:1001244)	\N	2514	1	0
1835	10	CDS_independently_known	A CDS with the evidence status of being independently known.	2515	0	0
1836	10	orphan_CDS	A CDS whose predicted amino acid sequence is unsupported by any experimental evidence or by any match with any other known sequence.	2516	0	0
1837	10	CDS_supported_by_domain_match_data	A CDS that is supported by domain similarity.	2517	0	0
1838	10	CDS_supported_by_sequence_similarity_data	A CDS that is supported by sequence similarity data.	2518	0	0
1839	10	CDS_predicted	A CDS that is predicted.	2519	0	0
1840	10	status_of_coding_sequence (obsolete SO:1001255)	\N	2520	1	0
1841	10	CDS_supported_by_EST_or_cDNA_data	A CDS that is supported by similarity to EST or cDNA data.	2521	0	0
1842	10	internal_Shine_Dalgarno_sequence	A Shine-Dalgarno sequence that stimulates recoding through interactions with the anti-Shine-Dalgarno in the RNA of small ribosomal subunits of translating ribosomes. The signal is only operative in Bacteria.	2522	0	0
1843	10	recoded_mRNA	The sequence of a mature mRNA transcript, modified before translation or during translation, usually by special cis-acting signals.	2524	0	0
1844	10	minus_1_translationally_frameshifted	An attribute describing a translational frameshift of -1.	2526	0	0
1845	10	plus_1_translationally_frameshifted	An attribute describing a translational frameshift of +1.	2527	0	0
1846	10	mRNA_recoded_by_translational_bypass	A recoded_mRNA where translation was suspended at a particular codon and resumed at a particular non-overlapping downstream codon.	2528	0	0
1847	10	mRNA_recoded_by_codon_redefinition	A recoded_mRNA that was modified by an alteration of codon meaning.	2529	0	0
1848	10	stop_codon_redefinition_as_selenocysteine (obsolete SO:1001266)	\N	2530	1	0
1849	10	stop_codon_readthrough (obsolete SO:1001267)	\N	2531	1	0
1850	10	recoding_stimulatory_region	A site in an mRNA sequence that stimulates the recoding of a region in the same mRNA.	2532	0	0
1851	10	four_bp_start_codon	A non-canonical start codon with 4 base pairs.	2534	0	0
1852	10	stop_codon_redefinition_as_pyrrolysine (obsolete SO:1001270)	\N	2535	1	0
1853	10	archaeal_intron	An intron characteristic of Archaeal tRNA and rRNA genes, where intron transcript generates a bulge-helix-bulge motif that is recognised by a splicing endoribonuclease.	2536	0	0
1854	10	tRNA_intron	An intron found in tRNA that is spliced via endonucleolytic cleavage and ligation rather than transesterification.	2538	0	0
1855	10	CTG_start_codon	A non-canonical start codon of sequence CTG.	2539	0	0
1856	10	SECIS_element	The incorporation of selenocysteine into a protein sequence is directed by an in-frame UGA codon (usually a stop codon) within the coding region of the mRNA. Selenoprotein mRNAs contain a conserved secondary structure in the 3' UTR that is required for the distinction of UGA stop from UGA selenocysteine. The selenocysteine insertion sequence (SECIS) is around 60 nt in length and adopts a hairpin structure which is sufficiently well-defined and conserved to act as a computational screen for selenoprotein genes.	2540	0	0
1857	10	retron	Sequence coding for a short, single-stranded, DNA sequence via a retrotransposed RNA intermediate; characteristic of some microbial genomes.	2543	0	0
1858	10	three_prime_recoding_site	The recoding stimulatory signal located downstream of the recoding site.	2544	0	0
1859	10	three_prime_stem_loop_structure	A recoding stimulatory region, the stem-loop secondary structural element is downstream of the redefined region.	2545	0	0
1860	10	five_prime_recoding_site	The recoding stimulatory signal located upstream of the recoding site.	2546	0	0
1861	10	flanking_three_prime_quadruplet_recoding_signal	Four base pair sequence immediately downstream of the redefined region. The redefined region is a frameshift site. The quadruplet is 2 overlapping codons.	2547	0	0
1862	10	UAG_stop_codon_signal	A stop codon signal for a UAG stop codon redefinition.	2548	0	0
1863	10	UAA_stop_codon_signal	A stop codon signal for a UAA stop codon redefinition.	2549	0	0
1864	10	regulon	A group of genes, whether linked as a cluster or not, that respond to a common regulatory signal.	2550	0	0
1865	10	UGA_stop_codon_signal	A stop codon signal for a UGA stop codon redefinition.	2552	0	0
1866	10	three_prime_repeat_recoding_signal	A recoding stimulatory signal, downstream sequence important for recoding that contains repetitive elements.	2553	0	0
1867	10	distant_three_prime_recoding_signal	A recoding signal that is found many hundreds of nucleotides 3' of a redefined stop codon.	2554	0	0
1868	10	stop_codon_signal	A recoding stimulatory signal that is a stop codon and has effect on efficiency of recoding.	2556	0	0
1869	10	databank_entry	The sequence referred to by an entry in a databank such as Genbank or SwissProt.	2557	0	0
1870	10	gene_segment	A gene component region which acts as a recombinational unit of a gene whose functional form is generated through somatic recombination.	2558	0	0
1871	12	decreased_in_magnitude_relative_to		2560	0	1
1872	12	has_cross_section		2561	0	1
1873	12	has_dividend_entity	\N	2562	0	1
1874	12	has_dividend_quality	\N	2563	0	1
1875	12	has_divisor_entity	\N	2564	0	1
1876	12	has_divisor_quality	\N	2565	0	1
1877	12	has_ratio_quality	\N	2566	0	1
1878	12	has_relative_magnitude	\N	2567	0	1
1879	12	increased_in_magnitude_relative_to		2568	0	1
1880	13	is_magnitude_of	\N	2569	0	1
1881	13	is_measurement_of	\N	2570	0	1
1882	13	is_unit_of	\N	2571	0	1
1883	12	reciprocal_of		2572	0	1
1884	12	singly_occurring_form_of	\N	2573	0	1
1885	12	towards	\N	2574	0	1
1886	13	unit_of	\N	2575	0	1
1887	11	unit	A unit of measurement is a standardized quantity of a physical quality.	2576	0	0
1888	11	length unit	A base unit which is a standard measure of the distance between two points.	2578	0	0
1889	11	mass unit	A base unit which is a standard measure of the amount of matter/energy of a physical object.	2579	0	0
1890	11	time unit	A base unit which is a standard measure of the dimension in which events occur in sequence.	2580	0	0
1891	11	electric current unit	A base unit which is a standard measure of the flow of electric charge.	2581	0	0
1892	11	temperature unit	A base unit which is a standard measure of the average kinetic energy of the particles in a sample of matter.	2582	0	0
1893	11	substance unit	A base unit which is a standardized quantity of an element or compound with uniform composition.	2583	0	0
1894	11	luminous intensity unit	A base unit which is a standard measure of the wavelength-weighted power emitted by a light source in a particular direction.	2584	0	0
1895	11	meter	A length unit which is equal to the length of the path traveled by light in vacuum during a time interval of 1/299 792 458 of a second.	2585	0	0
1896	11	kilogram	A mass unit which is equal to the mass of the International Prototype Kilogram kept by the BIPM at Svres, France.	2587	0	0
1897	11	second	A time unit which is equal to the duration of 9 192 631 770 periods of the radiation corresponding to the transition between the two hyperfine levels of the ground state of the caesium 133 atom.	2588	0	0
1898	11	ampere	An electric current unit which is equal to the constant current which, if maintained in two straight parallel conductors of infinite length, of negligible circular cross-section, and placed 1 m apart in vacuum, would produce between these conductors a force equal to 2 x 10^[-7] newton per meter of length.	2589	0	0
1899	11	kelvin	A thermodynamic temperature unit which is equal to the fraction 1/273.16 of the thermodynamic temperature of the triple point of water.	2590	0	0
1900	11	mole	A substance unit which is equal to the amount of substance of a molecular system which contains as many elementary entities as there are atoms in 0.012 kilogram of carbon 12.	2591	0	0
1901	11	candela	A luminous intensity unit which equal to the luminous intensity, in a given direction, of a source that emits monochromatic radiation of frequency 540 x 1012 hertz and that has a radiant intensity in that direction of 1/683 watt per steradian.	2592	0	0
1902	11	centimeter	A length unit which is equal to one hundredth of a meter or 10^[-2] m.	2593	0	0
1903	11	millimeter	A length unit which is equal to one thousandth of a meter or 10^[-3] m.	2594	0	0
1904	11	micrometer	A length unit which is equal to one millionth of a meter or 10^[-6] m.	2595	0	0
1905	11	nanometer	A length unit which is equal to one thousandth of one millionth of a meter or 10^[-9] m.	2596	0	0
1906	11	angstrom	A length unit which is equal to 10 [-10] m.	2597	0	0
1907	11	picometer	A length unit which is equal to 10^[-12] m.	2598	0	0
1908	11	gram	A mass unit which is equal to one thousandth of a kilogram or 10^[-3] kg.	2599	0	0
1909	11	milligram	A mass unit which is equal to one thousandth of a gram or 10^[-3] g.	2600	0	0
1910	11	microgram	A mass unit which is equal to one millionth of a gram or 10^[-6] g.	2601	0	0
1911	11	nanogram	A mass unit which is equal to one thousandth of one millionth of a gram or 10^[-9] g.	2602	0	0
1912	11	picogram	A mass unit which is equal to 10^[-12] g.	2603	0	0
1913	11	femtogram	A mass unit which is equal to 10^[-15] g.	2604	0	0
1914	11	degree celsius	A temperature derived unit which is equal to one Kelvin degree. However, they have their zeros at different points. The Centigrade scale has its zero at 273.15 K.	2605	0	0
1915	11	millisecond	A time unit which is equal to one thousandth of a second or 10^[-3] s.	2606	0	0
1916	11	microsecond	A time unit which is equal to one millionth of a second or 10^[-6] s.	2607	0	0
1917	11	picosecond	A time unit which is equal to 10^[-12] s.	2608	0	0
1918	11	minute	A time unit which is equal to 60 seconds.	2609	0	0
1919	11	hour	A time unit which is equal to 3600 seconds or 60 minutes.	2610	0	0
1920	11	day	A time unit which is equal to 24 hours.	2611	0	0
1921	11	week	A time unit which is equal to 7 days.	2612	0	0
1922	11	month	A time unit which is approximately equal to the length of time of one of cycle of the moon's phases which is science is taken to be equal to 30 days.	2613	0	0
1923	11	year	A time unit which is equal to 12 months which is science is taken to be equal to 365.25 days.	2614	0	0
1924	11	milliampere	An electric current unit current which is equal to one thousandth of an ampere or 10^[-3] A.	2615	0	0
1925	11	microampere	An electric current unit current which is equal to one millionth of an ampere or 10^[-6] A.	2616	0	0
1926	11	micromole	A substance unit equal to a millionth of a mol or 10^[-6] mol.	2617	0	0
1927	11	millimole	A substance unit equal to a thousandth of a mol or 10^[-3] mol.	2618	0	0
1928	11	nanomole	A substance unit equal to one thousandth of one millionth of a mole or 10^[-9] mol.	2619	0	0
1929	11	picomole	A substance unit equal to 10^[-12] mol.	2620	0	0
1930	11	femtomole	A substance unit equal to 10^[-15] mol.	2621	0	0
1931	11	attomole	A substance unit equal to 10^[-18] mol.	2622	0	0
1932	11	base unit	A unit which is one of a particular measure to which all measures of that type can be related.	2623	0	0
1933	11	derived unit	A unit which is derived from base units.	2624	0	0
1934	11	area unit	A derived unit which is a standard measure of the amount of a 2-dimensional flat surface.	2625	0	0
1935	11	acceleration unit	A derived unit which is a standard measure of the rate of change of velocity in either speed or direction.	2626	0	0
1936	11	angular velocity unit	A derived unit which is a standard measure of the rate of angular movement about an axis; the angle rotated in a given time.	2627	0	0
1937	11	angular acceleration unit	A derived unit which is a standard measure of the rate of change of angular velocity.	2628	0	0
1938	11	concentration unit	A derived unit which represents a standard measurement of how much of a given substance there is mixed with another substance.	2629	0	0
1939	11	mass density unit	A density unit which is a standard measure of the mass of a substance in a given volume.	2630	0	0
1940	11	luminance unit	A derived unit which is a standard measure of the luminous intensity impinging on a given area.	2631	0	0
1941	11	area density unit	A density unit which is a standard measure of the mass exerting an influence on a given area.	2632	0	0
1942	11	molar mass unit	A derived unit which is a standard measure of the mass of a homogeneous substance containing 6.02 x 1023 atoms or molecules.	2633	0	0
1943	11	molar volume unit	A derived unit which is a standard measure of the volume of a homogeneous substance containing 6.02 x 1023 atoms or molecules.	2634	0	0
1944	11	momentum unit	A derived unit which is a standard measure of the quantity of motion measured by the product of mass and velocity.	2635	0	0
1945	11	rotational frequency unit	A derived unit which is a standard measure of the number of rotations in a given time.	2636	0	0
1946	11	specific volume unit	A derived unit which is a standard measure of the volume of a given mass of substance (the reciprocal of density).	2637	0	0
1947	11	speed/velocity unit	A derived unit which is a standard measure of the rate of movement. Speed is measured in the same physical units of measurement as velocity, but does not contain the element of direction that velocity has. Speed is thus the magnitude component of velocity.	2638	0	0
1948	11	unit of molarity	A derived concentration unit which is a standard measure of the number of moles of a given substance per liter of solution.	2639	0	0
1949	11	molar	A unit of concentration which expresses a concentration of 1 mole of solute per liter of solution (mol/L).	2640	0	0
1950	11	millimolar	A unit of molarity which is equal to one thousandth of a molar or 10^[-3] M.	2641	0	0
1951	11	micromolar	A unit of molarity which is equal to one millionth of a molar or 10^[-6] M.	2642	0	0
1952	11	nanomolar	A unit of molarity which is equal to one thousandth of one millionth of a molar or 10^[-9] M.	2643	0	0
1953	11	picomolar	A unit of molarity which is equal to 10^[-12] M.	2644	0	0
1954	11	unit of molality	A derived concentration unit which is a standard measure of the number of moles of a given substance per kilogram of solvent.	2645	0	0
1955	11	molal	A unit of concentration which expresses a concentration of a solution of 1 mole per kilogram of solvent (mol/kg).	2646	0	0
1956	11	millimolal	A molality unit which is equal to one thousandth of a molal or 10^[-3] m.	2647	0	0
1957	11	micromolal	A molality unit which is equal to one millionth of a molal or 10^[-6] m.	2648	0	0
1958	11	nanomolal	A molality unit which is equal to one thousandth of one millionth of a molal or 10^[-9] m.	2649	0	0
1959	11	picomolal	A molality unit which is equal to 10^[-12] m.	2650	0	0
1960	11	femtomolar	A unit of molarity which is equal to 10^[-15] M.	2651	0	0
1961	11	unit of normality	A unit of concentration which highlights the chemical nature of salts.	2652	0	0
1962	11	normal	A unit of concentration which is one gram equivalent of a solute per liter of solution. A gram equivalent weight or equivalent is a measure of the reactive capacity of a given molecule.	2653	0	0
1963	11	mole fraction	A concentration unit which denotes the number of moles of solute as a proportion of the total number of moles in a solution.	2654	0	0
1964	11	meter per second per second	An acceleration unit which is equal to the acceleration an object changing its velocity by 1meter/s over a time period that equals one second.	2655	0	0
1965	11	radian per second per second	An angular unit acceleration which is equal to the angular acceleration of an object changing its angular velocity by 1rad/s over a time period that equals one second.	2656	0	0
1966	11	radian per second	An angular unit velocity which is equal to about 9.54930 rpm (revolutions per minute).	2657	0	0
1967	11	square meter	An area unit which is equal to an area enclosed by a square with sides each 1 meter long.	2658	0	0
1968	11	square centimeter	An area unit which is equal to one thousand of square meter or 10^[-3] m^[2].	2659	0	0
2259	14	powdery mildew	\N	2966	0	0
1969	11	square millimeter	An area unit which is equal to one millionth of a square meter or 10^[-6] m^[2].	2660	0	0
1970	11	kilogram per cubic meter	A mass unit density which is equal to mass of an object in kilograms divided by the volume in cubic meters.	2661	0	0
1971	11	gram per cubic centimeter	A mass unit density which is equal to mass of an object in grams divided by the volume in cubic centimeters.	2662	0	0
1972	11	microgram per liter	A mass unit density which is equal to mass of an object in micrograms divided by the volume in liters.	2663	0	0
1973	11	candela per square meter	A luminance unit which is equal to a luminous intensity of one candela radiating from a surface whose area is one square meter.	2665	0	0
1974	11	kilogram per square meter	An area density unit which is equal to the mass of an object in kilograms divided by the surface area in meters squared.	2666	0	0
1975	11	kilogram per mole	A molar mass unit which is equal to one kilogram of mass of one mole of chemical element or chemical compound.	2667	0	0
1976	11	gram per mole	A molar mass unit which is equal to one gram of mass of one mole of chemical element or chemical compound.	2668	0	0
1977	11	cubic meter per mole	A molar volume unit which is equal to 1 cubic meter occupied by one mole of a substance in the form of a solid, liquid, or gas.	2669	0	0
1978	11	cubic centimeter per mole	A molar volume unit which is equal to 1 cubic centimeter occupied by one mole of a substance in the form of a solid, liquid, or gas.	2670	0	0
1979	11	kilogram meter per second	A momentum unit which is equal to the momentum of a one kilogram mass object with a speed of one meter per second.	2671	0	0
1980	11	one turn per second	A rotational frequency unit which is equal to 1 complete turn in a period of time that equals to 1 second.	2672	0	0
1981	11	cubic meter per kilogram	A specific volume unit which is equal to one cubic meter volume occupied by one kilogram of a particular substance.	2673	0	0
1982	11	meter per second	A speed/velocity unit which is equal to the speed of an object traveling 1 meter distance in one second.	2674	0	0
1983	11	volume unit	A derived unit which is a standard measure of the amount of space occupied by any substance, whether solid, liquid, or gas.	2675	0	0
1984	11	cubic meter	A volume unit which is equal to the volume of a cube with edges one meter in length. One cubic meter equals to 1000 liters.	2676	0	0
1985	11	cubic centimeter	A volume unit which is equal to one millionth of a cubic meter or 10^[-9] m^[3], or to 1 ml.	2677	0	0
1986	11	milliliter	A volume unit which is equal to one thousandth of a liter or 10^[-3] L, or to 1 cubic centimeter.	2678	0	0
1987	11	liter	A volume unit which is equal to one thousandth of a cubic meter or 10^[-3] m^[3], or to 1 decimeter.	2679	0	0
1988	11	cubic decimeter	A volume unit which is equal to one thousand of a cubic meter or 10^[-3] m^[3], or to 1 L.	2680	0	0
1989	11	microliter	A volume unit which is equal to one millionth of a liter or 10^[-6] L.	2681	0	0
1990	11	nanoliter	A volume unit which is equal to one thousandth of one millionth of a liter or 10^[-9] L.	2682	0	0
1991	11	picoliter	A volume unit which is equal to 10^[-12] L.	2683	0	0
1992	11	femtoliter	A volume unit which is equal to 10^[-15] L.	2684	0	0
1993	11	frequency unit	A derived unit which is a standard measure of the number of repetitive actions in a particular time.	2685	0	0
1994	11	hertz	A frequency unit which is equal to 1 complete cycle of a recurring phenomenon in 1 second.	2686	0	0
1995	11	force unit	A derived unit which is a standard measure of the force is applied when a mass is accelerated.	2687	0	0
1996	11	newton	A force unit which is equal to the force required to cause an acceleration of 1m/s2 of a mass of 1 Kg in the direction of the force.	2688	0	0
1997	11	pressure unit	A derived unit which is a standard measure of the force applied to a given area.	2689	0	0
1998	11	pascal	A pressure unit which is equal to the pressure or stress on a surface caused by a force of 1 newton spread over a surface of 1 m^[2].	2690	0	0
1999	11	energy unit	A derived unit which is a standard measure of the work done by a certain force (gravitational, electric, magnetic, force of inertia, etc).	2691	0	0
2000	11	joule	An energy unit which is equal to the energy required when a force of 1 newton moves an object 1 meter in the direction of the force.	2692	0	0
2001	11	power unit	A derived unit which is a standard measure power or the rate of doing work.	2693	0	0
2002	11	watt	A power unit which is equal to the power used when work is done at the rate of 1 joule per second.	2694	0	0
2003	11	illuminance unit	A derived unit which is a standard measure of the luminous flux incident on a unit area.	2695	0	0
2004	11	lux	An illuminance unit which is equal to the illuminance produced by 1 lumen evenly spread over an area 1 m^[2].	2696	0	0
2005	11	luminous flux unit	A derived unit which is a standard measure of the flow of radiant energy.	2697	0	0
2006	11	lumen	A luminous flux unit which is equal to the luminous flux emitted into 1 steradian by a point source of 1 candela.	2698	0	0
2007	11	catalytic activity unit	A derived unit which is a standard measure of the amount of the action of a catalyst.	2699	0	0
2008	11	katal	A catalytic unit activity which is equal to the activity of a catalyst in moles per second, such as the amount of an enzyme needed to transform one mole of substrate per second.	2700	0	0
2009	11	angle unit	A derived unit which is a standard measure of the figure or space formed by the junction of two lines or planes.	2701	0	0
2010	11	plane angle unit	A derived unit which is a standard measure of the angle formed by two straight lines in the same plane.	2702	0	0
2011	11	radian	A plane angle unit which is equal to the angle subtended at the center of a circle by an arc equal in length to the radius of the circle, approximately 57 degrees 17 minutes and 44.6 seconds.	2703	0	0
2012	11	solid angle unit	A derived unit which is a standard measure of the angle formed by three or more planes intersecting at a common point.	2704	0	0
2013	11	steradian	A solid angle unit which is equal to the solid angle subtended at the center of a sphere by an area on the surface of the sphere that is equal to the radius squared.	2705	0	0
2014	11	temperature derived unit	A derived unit which represents a standard measurement of temperature in a particular temperature scale.	2706	0	0
2099	11	plaque forming unit	A dimensionless count unit which a measure of plague forming units in a given volume.	2797	0	0
2015	11	radiation unit	A derived unit which is a standard measure of the amount of radiation emitted by a given radiation source as well as the amount of radiation absorbed or deposited in a specific material by a radiation source.	2707	0	0
2016	11	activity (of a radionuclide) unit	A derived unit which is a standard measure of the transformation (disintegration) rate of a radioactive substance.	2709	0	0
2017	11	absorbed dose unit	A derived unit which is a standard measure of the energy imparted by ionizing radiation to unit mass of matter such as tissue.	2711	0	0
2018	11	dose equivalent unit	A derived unit which is a standard measure of the expression of dose in terms of its biological effect.	2712	0	0
2019	11	exposure unit	A derived unit which is a standard measure of the quantity that expresses the ability of radiation to ionize air and thereby create electric charges which can be collected and measured.	2714	0	0
2020	11	becquerel	An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which one nucleus decays per second or there is one atom disintegration per second (dps).	2715	0	0
2021	11	curie	An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which there are 3.7 x 10^[10] atom disintegration per second (dps).	2716	0	0
2022	11	gray	An absorbed dose unit which is equal to the absorption of one joule of radiation energy by one kilogram of matter.	2717	0	0
2023	11	rad	An absorbed dose unit which is equal to 0.01 gray (Gy).	2718	0	0
2024	11	roentgen	An exposure unit which is equal to the amount of radiation required to liberate positive and negative charges of one electrostatic unit of charge in 1 cm^[3] of air at standard temperature and pressure (STP). This corresponds to the generation of approximately 2.0810^[9] ion pairs.	2719	0	0
2025	11	sievert	A dose equivalent unit which is equal to the absorption of one joule of radiation energy by one kilogram of matter.	2720	0	0
2026	11	millisievert	A dose equivalent unit which is equal to one thousandth of a sievert or 10^[-3] Sv.	2721	0	0
2027	11	microsievert	A dose equivalent unit which is equal to one millionth of a sievert or 10^[-6] Sv.	2722	0	0
2028	11	Roentgen equivalent man	A dose equivalent unit which when multiplied by hundred is equal to one sievert or 1 Sv. 1 Sv is equal to 100 rem.	2723	0	0
2029	11	microgray	An absorbed dose unit which is equal to one millionth of a gray or 10^[-6] Gy.	2724	0	0
2030	11	milligray	An absorbed dose unit which is equal to one thousandth of a gray or 10^[-3] Gy.	2725	0	0
2031	11	nanogray	An absorbed dose unit which is equal to one thousandth of a millionth of a gray or 10^[-9] Gy.	2726	0	0
2032	11	nanosievert	A dose equivalent unit which is equal to one thousandth of a millionth of a sievert or 10^[-9] Sv.	2727	0	0
2033	11	millicurie	An activity (of a radionuclide) unit which is equal to one thousandth of a curie or 10^[-3] Ci.	2728	0	0
2034	11	microcurie	An activity (of a radionuclide) unit which is equal to one millionth of a curie or 10^[-6] Ci.	2729	0	0
2035	11	disintegrations per minute	An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which one nucleus decays per minute or there is one atom disintegration per minute.	2730	0	0
2036	11	counts per minute	An activity (of a radionuclide) unit which is equal to the number of light emissions produced by ionizing radiation in one minute.	2731	0	0
2037	11	time derived unit	A derived unit which is a standard measure of the dimension in which events occur in sequence.	2732	0	0
2038	11	nanosecond	A time unit which is equal to one thousandth of one millionth of a second or 10^[-9] s.	2733	0	0
2039	11	century	A time unit which is equal to 100 years.	2734	0	0
2040	11	half life	A time unit which represents the period over which the activity or concentration of a specified chemical or element falls to half its original activity or concentration.	2735	0	0
2041	11	foot candle	An illuminance unit which is equal to the illuminance produced by 1 lumen evenly spread over an area 1 foot^[2]. One footcandle is equal to 10.76 lux.	2737	0	0
2042	11	irradiance unit	A derived unit which is a standard measure of the power of electromagnetic radiation at a surface, per unit area.	2738	0	0
2043	11	watt per square meter	An irradiance unit which is equal to 1 watt of radiant power incident per one square meter surface area.	2739	0	0
2044	11	einstein per square meter per second	An irradiance unit which is equal to one einstein per square meter per second. One einstein is one mole of photons, regardless of their frequency. Therefore, the number of photons in an einstein is Avogadro's number.	2740	0	0
2045	11	light unit	A derived unit which is a standard measure of the intensity of light.	2741	0	0
2046	11	watt per steradian per square meter	A radiance unit which is equal to one watt of radiant power incident per steradian solid angle per one square meter projected area of the source, as viewed from the given direction.	2742	0	0
2047	11	radiant intensity unit	A derived unit which is a standard measure of the intensity of electromagnetic radiation.	2743	0	0
2048	11	microeinstein per square meter per second	An irradiance unit which is equal to one microeinstein per square meter per second or 10^[-6] microeinstein/sm^[2].	2744	0	0
2049	11	radiance unit	A derived unit which is a standard measure of the power of electromagnetic radiation through space or through a material medium in the form of electromagnetic waves.	2745	0	0
2050	11	watt per steradian	A radiant intensity unit which is equal to one kilogram meter squared per second cubed per steradian.	2746	0	0
2051	11	mass percentage	A dimensionless concentration unit which denotes the mass of a substance in a mixture as a percentage of the mass of the entire mixture.	2747	0	0
2052	11	mass volume percentage	A dimensionless concentration unit which denotes the mass of the substance in a mixture as a percentage of the volume of the entire mixture.	2748	0	0
2053	11	volume percentage	A dimensionless concentration unit which denotes the volume of the solute in mL per 100 mL of the resulting solution.	2749	0	0
2054	11	parts per notation	A dimensionless concentration notation which describes the amount of one substance in another. It is the ratio of the amount of the substance of interest to the amount of that substance plus the amount of the substance.	2750	0	0
2055	11	parts per hundred	A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 100 regardless of the units of measure as long as they are the same.	2751	0	0
2056	11	parts per thousand	A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1000 regardless of the units of measure as long as they are the same.	2752	0	0
2057	11	parts per million	A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000 regardless of the units of measure used as long as they are the same or 1 part in 10^[6].	2753	0	0
2058	11	parts per billion	A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000,000 regardless of the units of measure as long as they are the same or 1 part in 10^[9].	2754	0	0
2059	11	parts per trillion	A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000,000 regardless of the units of measure used as long as they are the same or 1 part in 10^[12].	2755	0	0
2060	11	parts per quadrillion	A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000,000,000 regardless of the units of measure used as long as they are the same or 1 part in 10^[15].	2756	0	0
2061	11	gram per milliliter	A mass unit density which is equal to mass of an object in grams divided by the volume in milliliter.	2757	0	0
2062	11	kilogram per liter	A mass unit density which is equal to mass of an object in kilograms divided by the volume in liters.	2758	0	0
2063	11	gram per liter	A mass unit density which is equal to mass of an object in grams divided by the volume in liters.	2759	0	0
2064	11	milligram per milliliter	A mass unit density which is equal to mass of an object in milligrams divided by the volume in milliliters.	2760	0	0
2065	11	unit per volume unit	A derived concentration unit which is a standard measure of the number of units, as an agreed arbitrary amount, of a given substance per a specific volume of solution.	2761	0	0
2066	11	unit per milliliter	A unit per milliliter unit which is equal to one unit of an agreed arbitrary amount per one milliliter.	2763	0	0
2067	11	unit per liter	A unit per milliliter unit which is equal to one unit of an agreed arbitrary amount per one liter.	2764	0	0
2068	11	mass per unit volume	A concentration unit which is a standard measure of the mass of a substance in a given volume (density).	2765	0	0
2069	11	enzyme unit	A catalytic unit activity which is equal to the amount of the enzyme that catalyzes the conversion of 1 micro mole of substrate per minute.	2766	0	0
2070	11	density unit	A derived unit which is a standard measure of the influence exerted by some mass.	2767	0	0
2071	11	linear density unit	A density unit which is a standard measure of the mass exerting an influence on a one-dimensional object.	2768	0	0
2072	11	kilogram per meter	An area density unit which is equal to the mass of an object in kilograms divided by one meter.	2769	0	0
2073	11	degree	A plane angle unit which is equal to 1/360 of a full rotation or 1.7453310^[-2] rad.	2770	0	0
2074	11	dimensionless unit	A derived unit which is a standard measure of physical quantity consisting of only a numerical number without any units.	2771	0	0
2075	11	percent	A dimensionless ratio unit which denotes numbers as fractions of 100.	2772	0	0
2076	11	pi	A dimensionless unit which denoted an irrational real number, approximately equal to 3.14159 which is the ratio of a circle's circumference to its diameter in Euclidean geometry.	2773	0	0
2077	11	count unit	A dimensionless unit which denotes a simple count of things.	2774	0	0
2078	11	ratio	A dimensionless unit which denotes an amount or magnitude of one quantity relative to another.	2775	0	0
2079	11	fraction	A dimensionless ratio unit which relates the part (the numerator) to the whole (the denominator).	2776	0	0
2080	11	molecule count	A dimensionless count unit which denotes the number of molecules.	2777	0	0
2081	11	purity percentage	A dimensionless percent unit which denotes the homogeneity of a biomaterial.	2778	0	0
2082	11	confluence percentage	A dimensionless percent unit which denotes the density of an attached or monolayer culture (e.g., cell culture).	2779	0	0
2083	11	degree fahrenheit	A temperature derived unit which is equal to 5/9ths of a kelvin. Negative 40 degrees Fahrenheit is equal to negative 40 degrees Celsius.	2780	0	0
2084	11	pH	A dimensionless concentration notation which denotes the acidity of a solution in terms of activity of hydrogen ions (H+).	2781	0	0
2085	11	liter per kilogram	A specific volume unit which is equal to one liter volume occupied by one kilogram of a particular substance.	2782	0	0
2086	11	milliliter per kilogram	A specific volume unit which is equal to a thousandth of a liter per kilogram or 10^[-3] l/kg.	2783	0	0
2087	11	microliter per kilogram	A specific volume unit which is equal to one millionth of a liter per kilogram or 10^[-6] l/kg.	2784	0	0
2088	11	cell concentration unit	A concentration unit which denotes the average cell number in a given volume.	2785	0	0
2089	11	cells per milliliter	A unit of cell concentration which is equal to one cell in a volume of 1 milliliter.	2787	0	0
2090	11	catalytic (activity) concentration unit	A concentration unit which is a standard measure of the amount of the action of a catalyst in a given volume.	2788	0	0
2091	11	katal per cubic meter	A catalytic (activity) concentration unit which is equal to 1 katal activity of a catalyst in a given volume of one cubic meter.	2789	0	0
2092	11	katal per liter	A catalytic (activity) concentration unit which is equal to 1 katal activity of a catalyst in a given volume of one thousandth of a cubic meter.	2790	0	0
2093	11	volume per unit volume	A dimensionless concentration unit which denotes the given volume of the solute in the total volume of the resulting solution.	2791	0	0
2094	11	milliliter per cubic meter	A volume per unit volume unit which is equal to one millionth of a liter of solute in one cubic meter of solution.	2792	0	0
2246	14	root top wet weight	\N	2953	0	0
2095	11	milliliter per liter	A volume per unit volume unit which is equal to one millionth of a liter of solute in one liter of solution.	2793	0	0
35	7	exact	\N	35	0	0
2096	11	gram per deciliter	A mass density unit which is equal to mass of an object in grams divided by the volume in deciliters.	2794	0	0
2097	11	deciliter	A volume unit which is equal to one tenth of a liter or 10^[-1] L.	2795	0	0
2098	11	colony forming unit	A dimensionless count unit which a measure of viable bacterial numbers.	2796	0	0
2247	14	root top dry weight	\N	2954	0	0
2100	11	colony forming unit per volume	A concentration unit which a measure of viable bacterial numbers in a given volume.	2798	0	0
2101	11	colony forming unit per milliliter	A colony forming unit which a measure of viable bacterial numbers in one milliliter.	2799	0	0
2102	11	plaque forming unit per volume	A concentration unit which a measure of plague forming units in a given volume.	2800	0	0
2103	11	plaque forming unit per milliliter	A concentration unit which a measure of plague forming units in one milliliter.	2801	0	0
2104	11	disintegrations per second	An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which one nucleus decays per second or there is one atom disintegration per second.	2802	0	0
2105	11	electric potential difference	A derived unit which is a standard measure of the work done per unit charge as a charge is moved between two points in an electric field.	2803	0	0
2106	11	volt	An electric potential difference unit which is equal to the work per unit charge. One volt is the potential difference required to move one coulomb of charge between two points in a circuit while using one joule of energy.	2804	0	0
2107	11	electric charge	A derived unit which is a standard measure of the quantity of unbalanced electricity in a body (either positive or negative) and construed as an excess or deficiency of electrons.	2805	0	0
2108	11	coulomb	An electrical charge unit which is equal to the amount of charge transferred by a current of 1 ampere in 1 second.	2807	0	0
2109	11	dalton	An independently to the base SI units defined mass unit which is equal to one twelfth of the mass of an unbound atom of the carbon-12 nuclide, at rest and in its ground state.	2808	0	0
2110	11	kilodalton	A mass unit which is equal to one thousand daltons.	2809	0	0
2111	11	watt-hour	An energy unit which is equal to the amount of electrical energy equivalent to a one-watt load drawing power for one hour.	2810	0	0
2112	11	kilowatt-hour	An energy unit which is equal to 1,000 watt-hours.	2811	0	0
2113	11	magnetic flux unit	A derived unit which is a standard measure of quantity of magnetism, taking account of the strength and the extent of a magnetic field.	2812	0	0
2114	11	weber	A magnetic flux unit which is equal to the amount of flux that when linked with a single turn of wire for an interval of one second will induce an electromotive force of one volt.	2813	0	0
2115	11	magnetic flux density	A derived unit which is a standard measure of the strength of a magnetic field.	2815	0	0
2116	11	tesla	A magnetic flux density unit which is equal to one weber per square meter.	2817	0	0
2117	11	volt-hour	A magnetic flux unit which is equal to 3600 Wb.	2818	0	0
2118	11	kilovolt-hour	A magnetic flux unit which is equal to one thousand volt-hours.	2820	0	0
2119	11	information unit	A unit which is a standard measure of the amount of information.	2821	0	0
2120	11	bit	An information unit which refers to a digit in the binary numeral system, which consists of base 2 digits (ie there are only 2 possible values: 0 or 1).	2822	0	0
2121	11	byte	An information unit which is equal to 8 bits.	2823	0	0
2122	11	kilobyte	An information unit which is equal to 1000 bytes.	2824	0	0
2123	11	megabyte	An information unit which is equal to 1000 kB.	2825	0	0
2124	11	image resolution unit	An information unit which is a standard measure of the detail an image holds.	2826	0	0
2125	11	chroma sampling unit	An image resolution unit which is a standard measure of the amount of spatial detail in an image.	2827	0	0
2126	11	dynamic range unit	An image resolution unit which is a standard measure of the amount of contrast available in a pixel.	2828	0	0
2127	11	spatial resolution unit	An image resolution unit which is a standard measure of the way luminance and chrominance may be sampled at different levels.	2829	0	0
2128	11	dots per inch	A spatial resolution unit which is a standard measure of the printing resolution, in particular the number of individual dots of ink a printer or toner can produce within a linear one-inch space.	2830	0	0
2129	11	micron	A spatial resolution unit which is equal to a pixel size of one micrometer.	2831	0	0
2130	11	pixels per inch	A spatial resolution unit which is a standard measure of the resolution of a computer display, related to the size of the display in inches and the total number of pixels in the horizontal and vertical directions.	2832	0	0
2131	11	pixels per millimeter	A spatial resolution unit which is a standard measure of the number of pixels in one millimeter length or width of a digital image divided by the physical length or width of a printed image.	2833	0	0
2132	11	base pair	A count unit which contains one nucleotide.	2834	0	0
2133	11	kibibyte	An information unit which is equal to 1024 B.	2836	0	0
2134	11	mebibyte	An information unit which is equal to 1024 KiB.	2837	0	0
2135	11	millivolt	An electric potential difference unit which is equal to one thousandth of a volt or 10^[-3] V.	2838	0	0
2136	11	kilovolt	An electric potential difference unit which is equal to one thousand volts or 10^[3] V.	2839	0	0
2137	11	microvolt	An electric potential difference unit which is equal to one millionth of a volt or 10^[-6] V.	2840	0	0
2138	11	nanovolt	An electric potential difference unit which is equal to one billionth of a volt or 10^[-12] V.	2841	0	0
2139	11	picovolt	An electric potential difference unit which is equal to one trillionth of a volt or 10^[-12] V.	2842	0	0
2140	11	megavolt	An electric potential difference unit which is equal to one million volts or 10^[6] V.	2843	0	0
2141	11	surface tension unit	A unit which is a standard measure of the ability of a liguid to attraction of molecules at its surface as a result of unbalanced molecular cohesive forces.	2844	0	0
2142	11	newton per meter	A surface tension unit which is equal to one newton per meter.	2845	0	0
2143	11	dyne per cm	A surface tension unit which is equal to one dyne per centimeter.	2846	0	0
2144	11	viscosity unit	A derived unit which is a standard measure of the internal resistance of fluids to flow.	2847	0	0
2145	11	pascal second	A viscosity unit which is equal to one pascale per second.	2848	0	0
2146	11	poise	A viscosity unit which is equal to one dyne second per square centimeter.	2849	0	0
2147	11	decibel	An ratio unit which is an indicator of sound power per unit area.	2850	0	0
2248	14	root bottom wet weight	\N	2955	0	0
2249	14	root bottom dry weight	\N	2956	0	0
2148	11	effective dose unit	A derived unit which is a standard measure of the estimate of the stochastic effect that a non-uniform radiation dose has on a human.	2852	0	0
2149	11	conduction unit	A derived unit which represents a standard measurement of the transmission of an entity through a medium.	2853	0	0
2150	11	electrical conduction unit	A derived unit which represents a standard measurement of the movement of electrically charged particles through a transmission medium (electrical conductor).	2855	0	0
2151	11	heat conduction unit	A derived unit which represents a standard measurement of the spontaneous transfer of thermal energy through matter, from a region of higher temperature to a region of lower temperature.	2856	0	0
2152	11	siemens	An electrical conduction unit which is equal to A/V.	2857	0	0
2153	11	watt per meter kelvin	An heat conduction unit which is equal to one watt divided by meter kelvin.	2858	0	0
2154	11	electronvolt	A non-SI unit of energy (eV) defined as the energy acquired by a single unbound electron when it passes through an electrostatic potential difference of one volt. An electronvolt is equal to 1.602 176 53(14) x 10^-19 J.	2859	0	0
2155	11	electric field strength unit	The electric field strength is a derived unit which is a measure of the potential difference between two points some distance apart.	2860	0	0
2156	11	volt per meter	The volt per meter is a unit of electric field strength equal to the a potential difference of 1 volt existing between two points that are 1 meter apart.	2862	0	0
2157	11	absorbance unit	A dimensionless logarithmic unit assigned to a measure of absorbance of light through a partially absorbing substance, defined as -log10(I/I_0) where I = transmitted light and I_0 = incident light.	2863	0	0
2158	11	volumetric flow rate unit	A derived unit which is a standard measure of the volume of fluid which passes through a given surface per unit time .	2865	0	0
2159	11	microliters per minute	A volumetric flow rate unit which is equal to one microliter volume through a given surface in one minute.	2866	0	0
2161	14	connected_to	\N	2868	0	1
34	8	is_transitive	\N	34	0	0
2162	14	trait	\N	2869	0	0
2163	14	leaf number	\N	2870	0	0
2164	14	alive leaf number	\N	2871	0	0
2165	14	branch number	\N	2872	0	0
2166	14	harvest date	\N	2873	0	0
2167	14	developmental trait	\N	2874	0	0
2168	14	morphological trait	\N	2875	0	0
2169	14	physiological trait	\N	2876	0	0
2170	14	environmental trait	\N	2877	0	0
2171	14	seedling quality	\N	2878	0	0
2172	14	pre-reproductive quality	\N	2879	0	0
2173	14	reproductive quality	\N	2880	0	0
2174	14	vegetative	\N	2881	0	0
2175	14	reproductive trait	\N	2882	0	0
2176	14	shoot trait	\N	2883	0	0
2177	14	leaf trait	\N	2884	0	0
2178	14	root trait	\N	2885	0	0
2179	14	nodule trait	\N	2886	0	0
2180	14	mycorrhizae trait	\N	2887	0	0
2181	14	flower trait	\N	2888	0	0
2182	14	fruit trait	\N	2889	0	0
2183	14	seed trait	\N	2890	0	0
2184	14	meta data term	\N	2891	0	0
2185	14	death quality	\N	2892	0	0
2186	14	senescence	\N	2893	0	0
2187	14	first leaf emergence	\N	2894	0	0
2188	14	second leaf emergence	\N	2895	0	0
2189	14	third leaf emergence	\N	2896	0	0
2190	14	cotyledon death	\N	2897	0	0
2191	14	first leaf death	\N	2898	0	0
2192	14	second leaf death	\N	2899	0	0
2193	14	third leaf death	\N	2900	0	0
2194	14	flowering	\N	2901	0	0
2195	14	fruiting	\N	2902	0	0
2196	14	cotyledon expansion	\N	2903	0	0
2197	14	reproductive unit number	\N	2904	0	0
2198	14	mature fruit number	\N	2905	0	0
2199	14	fruit number	\N	2906	0	0
2200	14	green fruit number	\N	2907	0	0
2201	14	flower number	\N	2908	0	0
2202	14	bud number	\N	2909	0	0
2203	14	NaCl concentration	\N	2910	0	0
2204	14	cotyledon width	\N	2911	0	0
2205	14	leaf length	\N	2912	0	0
2206	14	leaf width	\N	2913	0	0
2207	14	first leaf width	\N	2914	0	0
2208	14	second leaf width	\N	2915	0	0
2209	14	third leaf width	\N	2916	0	0
2210	14	first leaf length	\N	2917	0	0
2211	14	cotyledon length	\N	2918	0	0
2212	14	leaf category for length and width measurements	\N	2919	0	0
2213	14	treatment type	\N	2920	0	0
2214	14	treatment amount (obsolete DE:0000052)	\N	2921	1	0
2215	14	treatment unit	\N	2922	0	0
2216	14	deleted	\N	2923	0	0
2217	14	geolocation	\N	2924	0	0
2218	14	cotyledon number	\N	2925	0	0
2219	14	B0	\N	2926	0	0
2220	14	B01	\N	2927	0	0
2221	14	B02	\N	2928	0	0
2222	14	B03	\N	2929	0	0
2223	14	B04	\N	2930	0	0
2224	14	B05	\N	2931	0	0
2225	14	B06	\N	2932	0	0
2226	14	B011	\N	2933	0	0
2227	14	B012	\N	2934	0	0
2228	14	B021	\N	2935	0	0
2229	14	B022	\N	2936	0	0
2230	14	B031	\N	2937	0	0
2231	14	primary branch position	\N	2938	0	0
2232	14	primary branch length	\N	2939	0	0
2233	14	secondary branch number	\N	2940	0	0
2234	14	secondary branch length	\N	2941	0	0
2235	14	tertiary branch number	\N	2942	0	0
2236	14	tertiary branch length	\N	2943	0	0
2237	14	branch length max	\N	2944	0	0
2238	14	dead stem number	\N	2945	0	0
2239	14	stem diameter	\N	2946	0	0
2240	14	stem fresh weight	\N	2947	0	0
2241	14	stem dry weight	\N	2948	0	0
2242	14	partial shoot dry weight	\N	2949	0	0
2243	14	shoot dry weight	\N	2950	0	0
2244	14	root dry weight	\N	2951	0	0
2245	14	partial root dry weight	\N	2952	0	0
2260	14	experiment id	\N	2967	0	0
2261	14	title	\N	2968	0	0
2262	14	start date	\N	2969	0	0
2263	14	end date	\N	2970	0	0
2264	14	experimenter	\N	2971	0	0
2265	14	related experiment	\N	2972	0	0
2266	14	city	\N	2973	0	0
2267	14	state	\N	2974	0	0
2268	14	zipcode	\N	2975	0	0
2269	14	abstract	\N	2976	0	0
2270	14	substrate	\N	2977	0	0
2271	14	container	\N	2978	0	0
2272	14	media	\N	2979	0	0
2273	14	soil sterilization	\N	2980	0	0
2274	14	growth condition	\N	2981	0	0
2275	14	watering system	\N	2982	0	0
2276	14	watering schedule	\N	2983	0	0
2277	14	watering amount	\N	2984	0	0
2278	14	light source	\N	2985	0	0
2279	14	low brightness	\N	2986	0	0
2280	14	high brightness	\N	2987	0	0
2281	14	light photoperiod	\N	2988	0	0
2282	14	dark photoperiod	\N	2989	0	0
2283	14	day temperature	\N	2990	0	0
2284	14	night temperature	\N	2991	0	0
2285	14	scarification	\N	2992	0	0
2286	14	seed sterilization	\N	2993	0	0
2287	14	stock term	\N	2994	0	0
2288	14	plant id	\N	2995	0	0
2289	14	sequence	\N	2996	0	0
2290	14	genotype	\N	2997	0	0
2291	14	stock type	\N	2998	0	0
2292	14	tub	\N	2999	0	0
2293	14	treatment	\N	3000	0	0
2294	14	location in holder	\N	3001	0	0
2295	14	soil replaced date	\N	3002	0	0
2296	14	term	\N	3003	0	0
2297	14	species	\N	3004	0	0
2298	14	second leaf length	\N	3005	0	0
2299	14	comment	\N	3006	0	0
2300	14	comment one (obsolete DE:0000138)	\N	3007	1	0
2301	14	comment two (obsolete DE:0000139)	\N	3008	1	0
2302	14	comment three (obsolete DE:0000140)	\N	3009	1	0
2303	14	powdery mildew comment	\N	3010	0	0
2304	14	quality	\N	3011	0	0
2305	14	temporal quality	\N	3012	0	0
2306	14	measurement unit	\N	3013	0	0
2307	14	third leaf length	\N	3014	0	0
2308	14	inbred line	\N	3015	0	0
2309	14	experiment type	\N	3016	0	0
2310	14	phenotype experiment	\N	3017	0	0
2311	14	genotype experiment	\N	3018	0	0
2312	14	input stock	\N	3019	0	0
2313	14	measurement	\N	3020	0	0
2314	14	measurement value	\N	3021	0	0
2315	14	measurement state	\N	3022	0	0
2316	14	comment temporal	\N	3023	0	0
\.


--
-- Data for Name: cvterm_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cvterm_dbxref (cvterm_dbxref_id, cvterm_id, dbxref_id, is_for_definition) FROM stdin;
1	31	36	0
2	31	37	0
3	36	39	0
4	37	41	0
5	38	43	0
6	39	45	0
7	40	47	0
8	41	49	0
9	42	51	0
10	43	53	0
11	44	55	0
12	45	57	0
13	46	59	0
14	47	61	0
15	48	63	0
16	49	65	0
17	50	67	0
18	51	69	0
19	52	71	0
20	53	73	0
21	54	75	0
22	55	77	0
23	56	79	0
24	57	81	0
25	58	83	0
26	59	85	0
27	102	129	1
28	103	129	1
29	104	132	1
30	104	133	0
31	106	136	1
32	106	137	0
33	107	129	1
34	107	140	0
35	109	142	1
36	114	148	1
37	115	148	1
38	116	151	1
39	117	153	1
40	119	151	1
41	120	151	1
42	121	158	1
43	121	159	0
44	122	129	1
45	123	129	1
46	124	129	1
47	125	129	1
48	125	164	0
49	126	166	1
50	127	129	1
51	130	171	0
52	131	173	1
53	132	173	1
54	133	176	1
55	133	177	0
56	134	179	1
57	135	176	1
58	136	182	1
59	137	184	1
60	137	185	0
61	138	187	1
62	138	188	0
63	139	187	1
64	139	190	0
65	140	129	1
66	141	129	1
67	142	187	1
68	143	129	1
69	144	187	1
70	145	197	1
71	146	129	1
72	147	129	1
73	148	129	1
74	149	129	1
75	150	129	1
76	151	129	1
77	152	129	1
78	153	187	1
79	153	206	0
80	155	129	1
81	156	129	1
82	156	210	0
83	157	129	1
84	157	212	0
85	158	129	1
86	158	214	0
87	159	187	1
88	159	216	0
89	161	219	1
90	163	219	1
91	164	223	1
92	165	129	1
93	167	129	1
94	170	129	1
95	171	129	1
96	172	129	1
97	173	129	1
98	175	129	1
99	176	129	1
100	177	129	1
101	178	129	1
102	179	129	1
103	179	239	0
104	180	197	1
105	181	129	1
106	189	197	1
107	189	250	0
108	190	197	1
109	190	252	0
110	191	197	1
111	192	197	1
112	193	197	1
113	194	197	1
114	195	197	1
115	196	197	1
116	197	197	1
117	198	129	1
118	199	197	1
119	200	197	1
120	201	197	1
121	202	197	1
122	203	266	1
123	203	267	0
124	204	129	1
125	205	129	1
126	206	187	1
127	206	271	0
128	206	272	0
129	207	274	1
130	210	197	1
131	211	129	1
132	212	129	1
133	213	129	1
134	214	282	1
135	214	283	0
136	215	129	1
137	216	129	1
138	218	129	1
139	220	197	1
140	221	129	1
141	222	129	1
142	223	293	1
143	224	129	1
144	225	187	1
145	226	129	1
146	227	129	1
147	228	129	1
148	229	197	1
149	230	197	1
150	231	197	1
151	232	129	1
152	233	129	1
153	234	293	1
154	235	129	1
155	236	129	1
156	236	307	0
157	237	129	1
158	238	129	1
159	239	129	1
160	240	197	1
161	241	129	1
162	242	314	1
163	242	315	0
164	243	136	1
165	243	317	0
166	244	129	1
167	245	129	1
168	247	197	1
169	248	129	1
170	249	129	1
171	249	324	0
172	250	142	1
173	251	142	1
174	251	327	0
175	252	329	1
176	253	129	1
177	253	331	0
178	254	187	1
179	255	187	1
180	256	335	1
181	257	187	1
182	258	187	1
183	258	338	0
184	259	187	1
185	259	340	0
186	260	187	1
187	260	342	0
188	261	129	1
189	261	344	0
190	261	345	0
191	262	347	1
192	263	129	1
193	264	350	1
194	264	129	1
195	264	351	0
196	265	353	1
197	265	350	1
198	265	129	1
199	266	353	1
200	266	350	1
201	266	129	1
202	267	136	1
203	267	356	0
204	268	197	1
205	269	359	1
206	269	360	0
207	270	187	1
208	271	129	1
209	272	129	1
210	273	129	1
211	274	136	1
212	274	366	0
213	275	136	1
214	276	136	1
215	276	369	0
216	277	136	1
217	277	371	0
218	278	136	1
219	279	187	1
220	280	187	1
221	280	375	0
222	281	129	1
223	282	378	1
224	282	379	0
225	283	129	1
226	284	129	1
227	285	129	1
228	286	384	1
229	287	187	1
230	287	386	0
231	288	129	1
232	289	129	1
233	290	136	1
234	290	390	0
235	291	129	1
236	295	396	1
237	295	397	0
238	296	353	1
239	297	129	1
240	298	350	1
241	299	350	1
242	300	129	1
243	301	129	1
244	302	129	1
245	303	406	1
246	304	187	1
247	305	129	1
248	306	136	1
249	306	410	0
250	307	136	1
251	307	412	0
252	308	129	1
253	308	414	0
254	310	417	1
255	311	129	1
256	312	129	1
257	313	129	1
258	314	129	1
259	315	129	1
260	316	129	1
261	317	129	1
262	318	129	1
263	319	129	1
264	320	129	1
265	321	129	1
266	322	129	1
267	323	129	1
268	324	129	1
269	325	129	1
270	326	129	1
271	327	129	1
272	328	129	1
273	329	129	1
274	330	129	1
275	331	129	1
276	332	129	1
277	333	129	1
278	334	129	1
279	335	129	1
280	335	443	0
281	336	187	1
282	336	445	0
283	337	129	1
284	338	448	1
285	338	187	1
286	340	417	1
287	341	129	1
288	343	350	1
289	344	129	1
290	345	129	1
291	345	456	0
292	348	129	1
293	352	129	1
294	354	136	1
295	354	466	1
296	354	467	0
297	355	469	1
298	355	466	1
299	355	470	0
300	356	129	1
301	357	129	1
302	358	129	1
303	359	129	1
304	360	129	1
305	361	129	1
306	362	129	1
307	363	129	1
308	364	129	1
309	365	129	1
310	366	129	1
311	367	129	1
312	368	129	1
313	369	129	1
314	370	129	1
315	371	129	1
316	372	129	1
317	373	129	1
318	374	129	1
319	375	129	1
320	376	136	1
321	376	492	1
322	376	493	1
323	376	494	0
324	377	496	1
325	377	497	0
326	378	499	1
327	378	500	0
328	379	129	1
329	380	197	1
330	381	197	1
331	382	197	1
332	383	197	1
333	384	197	1
334	385	197	1
335	386	509	1
336	387	197	1
337	388	136	1
338	388	512	0
339	389	197	1
340	389	514	0
341	390	197	1
342	391	517	1
343	391	518	0
344	395	197	1
345	396	129	1
346	396	524	0
347	397	384	1
348	398	136	1
349	398	527	0
350	399	136	1
351	399	529	0
352	404	535	1
353	405	129	1
354	406	509	1
355	407	136	1
356	408	129	1
357	409	329	1
358	409	541	0
359	413	187	1
360	414	129	1
361	415	136	1
362	415	548	0
363	415	549	0
364	416	129	1
365	416	551	0
366	417	129	1
367	418	187	1
368	419	555	1
369	420	129	1
370	420	557	0
371	421	129	1
372	421	559	0
373	422	129	1
374	423	129	1
375	425	129	1
376	426	129	1
377	427	129	1
378	428	567	1
379	429	129	1
380	431	197	1
381	432	129	1
382	432	572	0
383	433	574	1
384	434	129	1
385	435	129	1
386	436	129	1
387	437	129	1
388	438	353	1
389	438	580	0
390	439	329	1
391	440	583	1
392	441	329	1
393	441	585	0
394	442	187	1
395	442	587	0
396	443	187	1
397	443	589	0
398	445	129	1
399	446	129	1
400	447	129	1
401	449	129	1
402	450	597	1
403	450	598	1
404	450	599	0
405	451	129	1
406	452	187	1
407	453	187	1
408	454	598	1
409	455	187	1
410	455	605	0
411	457	187	1
412	458	598	1
413	459	129	1
414	460	129	1
415	460	611	0
416	461	613	1
417	461	129	1
418	461	614	0
419	462	129	1
420	463	187	1
421	464	197	1
422	465	129	1
423	466	314	1
424	466	620	0
425	467	129	1
426	468	314	1
427	469	129	1
428	471	187	1
429	472	627	1
430	473	598	1
431	474	197	1
432	475	187	1
433	475	631	0
434	476	633	1
435	476	634	0
436	477	636	1
437	477	637	0
438	478	639	1
439	479	641	1
440	479	642	0
441	480	644	1
442	480	645	0
443	481	647	1
444	481	648	0
445	484	652	1
446	484	653	0
447	485	655	1
448	485	656	0
449	486	658	1
450	487	660	1
451	488	662	1
452	488	663	0
453	489	665	1
454	490	667	1
455	490	668	0
456	491	670	1
457	491	671	0
458	492	673	1
459	492	674	0
460	493	676	1
461	493	677	0
462	494	679	1
463	494	680	0
464	495	682	1
465	496	684	1
466	496	685	0
467	497	679	1
468	497	687	0
469	498	689	1
470	499	691	1
471	499	692	0
472	500	694	1
473	500	695	0
474	501	129	1
475	504	700	1
476	504	701	1
477	504	702	0
478	505	704	1
479	505	705	0
480	506	707	1
481	506	708	0
482	507	710	1
483	507	711	1
484	507	712	0
485	508	129	1
486	508	714	0
487	509	129	1
488	510	717	1
489	510	129	1
490	510	718	0
491	510	719	0
492	511	129	1
493	512	197	1
494	513	129	1
495	513	723	0
496	514	129	1
497	515	129	1
498	516	129	1
499	517	129	1
500	518	717	1
501	518	730	0
502	518	731	0
503	518	732	0
504	520	136	1
505	520	734	0
506	520	735	0
507	521	717	1
508	521	136	1
509	521	219	1
510	521	737	0
511	535	129	1
512	536	753	1
513	537	129	1
514	538	187	1
515	542	187	1
516	542	760	0
517	543	129	1
518	544	129	1
519	545	129	1
520	546	129	1
521	547	129	1
522	548	129	1
523	549	129	1
524	550	129	1
525	551	187	1
526	552	187	1
527	553	197	1
528	555	223	1
529	555	129	1
530	556	775	1
531	556	776	0
532	557	197	1
533	558	129	1
534	559	129	1
535	560	535	1
536	561	197	1
537	563	784	1
538	564	350	1
539	565	129	1
540	566	129	1
541	567	784	1
542	568	535	1
543	569	129	1
544	570	329	1
545	571	129	1
546	572	535	1
547	573	129	1
548	574	350	1
549	575	129	1
550	576	129	1
551	577	129	1
552	578	129	1
553	579	129	1
554	580	535	1
555	581	197	1
556	582	129	1
557	583	129	1
558	585	129	1
559	586	129	1
560	587	535	1
561	588	129	1
562	589	535	1
563	590	535	1
564	591	535	1
565	592	535	1
566	593	535	1
567	595	535	1
568	596	535	1
569	597	535	1
570	598	535	1
571	599	535	1
572	600	535	1
573	601	129	1
574	602	173	1
575	602	824	0
576	603	129	1
577	604	129	1
578	606	535	1
579	607	535	1
580	608	535	1
581	609	129	1
582	610	535	1
583	611	535	1
584	612	535	1
585	613	535	1
586	614	784	1
587	615	535	1
588	616	535	1
589	617	535	1
590	618	129	1
591	619	535	1
592	620	535	1
593	621	535	1
594	622	535	1
595	623	535	1
596	624	535	1
597	625	535	1
598	626	535	1
599	627	535	1
600	628	535	1
601	629	535	1
602	630	535	1
603	631	535	1
604	632	535	1
605	633	535	1
606	634	535	1
607	635	535	1
608	636	535	1
609	637	535	1
610	638	535	1
611	639	535	1
612	640	535	1
613	641	535	1
614	642	535	1
615	643	535	1
616	644	535	1
617	646	869	1
618	646	870	0
619	647	872	1
620	649	784	1
621	650	197	1
622	651	784	1
623	652	129	1
624	653	136	1
625	654	129	1
626	654	880	0
627	655	136	1
628	657	136	1
629	658	535	1
630	659	136	1
631	660	887	1
632	661	535	1
633	662	535	1
634	663	535	1
635	666	535	1
636	667	535	1
637	668	535	1
638	669	784	1
639	671	129	1
640	671	899	0
641	671	900	0
642	672	535	1
643	674	535	1
644	676	535	1
645	678	535	1
646	679	129	1
647	679	909	0
648	681	187	1
649	682	129	1
650	683	555	1
651	683	914	0
652	684	129	1
653	685	917	1
654	686	919	1
655	686	920	0
656	688	129	1
657	689	924	1
658	689	925	0
659	690	129	1
660	691	129	1
661	692	929	1
662	693	598	1
663	693	931	0
664	694	933	1
665	695	935	1
666	696	935	1
667	697	129	1
668	698	129	1
669	699	940	1
670	704	940	1
671	704	946	0
672	705	948	1
673	705	949	0
674	706	917	1
675	707	350	1
676	707	952	0
677	708	917	1
678	709	917	1
679	711	957	1
680	712	129	1
681	713	129	1
682	714	961	1
683	714	962	0
684	715	129	1
685	716	129	1
686	717	129	1
687	718	129	1
688	721	129	1
689	721	970	0
690	722	129	1
691	724	129	1
692	726	187	1
693	726	976	0
694	727	129	1
695	727	978	0
696	729	359	1
697	729	981	0
698	732	129	1
699	733	129	1
700	734	129	1
701	735	329	1
702	735	988	0
703	736	329	1
704	736	990	0
705	737	129	1
706	739	197	1
707	740	995	1
708	741	129	1
709	742	129	1
710	745	517	1
711	745	1001	0
712	746	129	1
713	746	1003	0
714	747	129	1
715	748	499	1
716	748	1006	0
717	749	129	1
718	750	129	1
719	751	1010	1
720	751	1011	0
721	752	129	1
722	753	129	1
723	754	1015	1
724	754	1016	0
725	755	129	1
726	755	1018	0
727	756	197	1
728	757	129	1
729	757	1021	0
730	759	129	1
731	760	129	1
732	760	1025	0
733	764	129	1
734	767	197	1
735	768	129	1
736	769	129	1
737	769	1035	0
738	770	129	1
739	772	187	1
740	773	187	1
741	774	187	1
742	775	187	1
743	775	1042	0
744	776	129	1
745	777	129	1
746	778	129	1
747	779	129	1
748	780	129	1
749	781	129	1
750	782	129	1
751	783	129	1
752	785	1053	1
753	786	187	1
754	788	187	1
755	789	129	1
756	790	142	1
757	791	129	1
758	792	197	1
759	793	717	1
760	793	1062	0
761	794	197	1
762	795	197	1
763	796	219	1
764	797	129	1
765	798	187	1
766	798	1068	0
767	799	197	1
768	800	197	1
769	801	129	1
770	802	129	1
771	803	129	1
772	804	129	1
773	805	129	1
774	806	1077	1
775	806	1078	0
776	807	129	1
777	807	1080	0
778	807	266	0
779	808	129	1
780	809	129	1
781	812	197	1
782	813	197	1
783	814	197	1
784	815	129	1
785	815	1089	0
786	816	129	1
787	817	129	1
788	818	129	1
789	819	448	1
790	819	1094	0
791	820	448	1
792	821	1097	1
793	822	129	1
794	823	197	1
795	824	197	1
796	825	187	1
797	825	1102	0
798	826	136	1
799	826	1104	0
800	827	136	1
801	827	1106	0
802	828	129	1
803	828	136	0
804	829	1109	1
805	830	129	1
806	830	1111	0
807	831	129	1
808	832	129	1
809	833	129	1
810	834	129	1
811	834	1116	0
812	835	129	1
813	836	293	1
814	843	1126	1
815	843	1127	0
816	843	1128	0
817	844	1126	1
818	844	1130	0
819	852	187	1
820	855	129	1
821	856	347	1
822	857	1144	0
823	858	187	1
824	858	1146	0
825	868	129	1
826	870	187	1
827	871	1160	1
828	871	1161	1
829	871	1162	1
830	872	1160	1
831	872	1161	1
832	873	1165	1
833	874	1167	1
834	874	129	1
835	874	1168	0
836	875	129	1
837	876	129	1
838	877	129	1
839	878	129	1
840	879	129	1
841	880	129	1
842	881	197	1
843	883	129	1
844	884	129	1
845	885	129	1
846	886	129	1
847	891	129	1
848	892	129	1
849	896	197	1
850	897	197	1
851	898	1193	1
852	899	1193	1
853	900	1193	1
854	901	1193	1
855	902	1198	1
856	903	1198	1
857	904	1198	1
858	905	1198	1
859	906	197	1
860	907	197	1
861	909	197	1
862	910	197	1
863	911	197	1
864	912	197	1
865	913	197	1
866	914	197	1
867	915	197	1
868	916	129	1
869	917	1214	1
870	918	197	1
871	919	129	1
872	919	1217	0
873	920	197	1
874	921	197	1
875	922	197	1
876	923	197	1
877	924	197	1
878	925	197	1
879	926	197	1
880	927	197	1
881	928	197	1
882	929	197	1
883	930	129	1
884	931	129	1
885	932	129	1
886	933	129	1
887	934	129	1
888	935	129	1
889	936	219	1
890	937	129	1
891	938	129	1
892	939	1238	1
893	939	129	1
894	939	1239	0
895	939	1240	0
896	940	129	1
897	941	129	1
898	943	129	1
899	944	129	1
900	945	129	1
901	946	129	1
902	947	219	1
903	951	219	1
904	952	598	1
905	953	129	1
906	953	1255	0
907	954	129	1
908	954	1257	0
909	955	129	1
910	955	1259	0
911	957	129	1
912	958	129	1
913	959	129	1
914	960	129	1
915	960	1265	0
916	961	197	1
917	962	197	1
918	963	129	1
919	964	129	1
920	965	129	1
921	965	1271	0
922	966	129	1
923	967	129	1
924	968	129	1
925	969	129	1
926	970	129	1
927	971	197	1
928	972	197	1
929	973	129	1
930	974	129	1
931	975	129	1
932	976	129	1
933	977	129	1
934	978	129	1
935	979	129	1
936	980	129	1
937	981	129	1
938	982	129	1
939	983	129	1
940	984	129	1
941	985	129	1
942	986	129	1
943	987	129	1
944	988	197	1
945	989	197	1
946	990	197	1
947	991	197	1
948	992	197	1
949	993	129	1
950	993	1300	0
951	994	129	1
952	995	129	1
953	996	197	1
954	997	197	1
955	998	129	1
956	999	129	1
957	1000	129	1
958	1001	129	1
959	1002	197	1
960	1002	1310	0
961	1004	129	1
962	1005	129	1
963	1006	129	1
964	1007	129	1
965	1008	129	1
966	1009	129	1
967	1011	129	1
968	1012	1321	1
969	1012	1322	0
970	1013	197	1
971	1014	197	1
972	1015	197	1
973	1017	129	1
974	1018	129	1
975	1019	129	1
976	1020	129	1
977	1021	129	1
978	1022	129	1
979	1023	129	1
980	1024	129	1
981	1025	129	1
982	1026	129	1
983	1027	129	1
984	1028	129	1
985	1029	197	1
986	1030	187	1
987	1031	1342	1
988	1033	129	1
989	1034	1346	1
990	1035	197	1
991	1041	197	1
992	1042	314	1
993	1043	314	1
994	1044	314	1
995	1045	314	1
996	1046	314	1
997	1047	314	1
998	1048	187	1
999	1049	314	1
1000	1050	314	1
1001	1052	314	1
1002	1053	314	1
1003	1054	187	1
1004	1055	187	1
1005	1056	187	1
1006	1057	187	1
1007	1058	187	1
1008	1059	187	1
1009	1060	187	1
1010	1061	187	1
1011	1062	187	1
1012	1063	187	1
1013	1064	187	1
1014	1065	187	1
1015	1066	187	1
1016	1067	187	1
1017	1069	1382	0
1018	1073	314	1
1019	1073	1387	0
1020	1075	129	1
1021	1077	1342	1
1022	1078	187	1
1023	1079	1126	1
1024	1079	1394	0
1025	1079	1395	0
1026	1086	219	1
1027	1087	219	1
1028	1088	1405	1
1029	1089	1405	1
1030	1097	1415	1
1031	1098	129	1
1032	1099	129	1
1033	1099	1418	0
1034	1100	129	1
1035	1101	598	1
1036	1102	129	1
1037	1105	1425	1
1038	1105	1426	0
1039	1106	1425	1
1040	1106	1428	0
1041	1107	129	1
1042	1107	1430	0
1043	1109	1433	1
1044	1110	129	1
1045	1110	1435	0
1046	1111	219	1
1047	1112	1438	1
1048	1114	1441	1
1049	1114	1442	0
1050	1115	219	1
1051	1116	1445	0
1052	1117	1447	1
1053	1118	129	1
1054	1118	1449	0
1055	1120	1452	0
1056	1121	219	1
1057	1122	1077	1
1058	1122	1455	0
1059	1123	1077	1
1060	1123	1457	0
1061	1124	1077	1
1062	1125	1077	1
1063	1125	1460	0
1064	1126	1077	1
1065	1126	1462	0
1066	1127	1077	1
1067	1129	129	1
1068	1130	129	1
1069	1131	1468	0
1070	1133	1471	1
1071	1133	187	1
1072	1134	129	1
1073	1134	1473	0
1074	1135	129	1
1075	1136	1476	1
1076	1136	1477	0
1077	1137	129	1
1078	1138	129	1
1079	1139	129	1
1080	1140	129	1
1081	1141	129	1
1082	1141	1483	0
1083	1142	1167	1
1084	1143	197	1
1085	1144	1487	1
1086	1145	1167	1
1087	1146	1167	1
1088	1147	129	1
1089	1148	1492	1
1090	1149	129	1
1091	1154	359	1
1092	1155	129	1
1093	1157	359	1
1094	1158	129	1
1095	1158	1503	0
1096	1158	1504	0
1097	1159	129	1
1098	1160	717	1
1099	1160	1507	0
1100	1161	717	1
1101	1161	1509	0
1102	1161	1510	0
1103	1162	717	1
1104	1162	1512	0
1105	1163	717	1
1106	1163	1514	1
1107	1163	1515	0
1108	1163	1516	0
1109	1164	717	1
1110	1164	1514	1
1111	1164	1518	0
1112	1165	717	1
1113	1165	1514	1
1114	1165	1520	0
1115	1166	717	1
1116	1166	1522	0
1117	1167	717	1
1118	1167	219	1
1119	1167	1524	0
1120	1168	717	1
1121	1168	1526	0
1122	1169	717	1
1123	1169	219	1
1124	1169	1528	0
1125	1170	717	1
1126	1170	219	1
1127	1170	1530	0
1128	1171	717	1
1129	1171	219	1
1130	1171	1532	0
1131	1172	717	1
1132	1172	1534	0
1133	1173	717	1
1134	1173	219	1
1135	1173	1536	0
1136	1174	717	1
1137	1174	1538	1
1138	1174	1539	0
1139	1175	717	1
1140	1175	1541	0
1141	1175	1542	0
1142	1176	717	1
1143	1176	1544	0
1144	1176	1545	0
1145	1177	717	1
1146	1177	1514	1
1147	1177	1547	0
1148	1177	1548	0
1149	1178	717	1
1150	1178	1550	0
1151	1179	717	1
1152	1179	1552	0
1153	1180	717	1
1154	1180	1514	1
1155	1180	1554	0
1156	1181	717	1
1157	1181	1514	1
1158	1181	1556	0
1159	1182	717	1
1160	1182	1514	1
1161	1182	1558	0
1162	1183	717	1
1163	1183	1514	1
1164	1183	1560	0
1165	1184	717	1
1166	1184	1514	1
1167	1184	1562	0
1168	1185	717	1
1169	1185	1514	1
1170	1185	1564	0
1171	1186	717	1
1172	1186	1514	1
1173	1186	1566	0
1174	1186	1567	0
1175	1187	717	1
1176	1187	1569	0
1177	1188	717	1
1178	1188	1571	0
1179	1189	717	1
1180	1189	1514	1
1181	1189	1573	0
1182	1190	717	1
1183	1190	1575	1
1184	1190	1576	0
1185	1190	1577	0
1186	1191	717	1
1187	1191	1579	0
1188	1192	717	1
1189	1192	1581	0
1190	1193	717	1
1191	1193	1583	0
1192	1194	717	1
1193	1194	1585	0
1194	1195	717	1
1195	1195	1587	0
1196	1196	717	1
1197	1196	1589	0
1198	1197	717	1
1199	1197	1591	0
1200	1198	717	1
1201	1198	1593	0
1202	1199	717	1
1203	1199	1595	0
1204	1200	717	1
1205	1200	1597	0
1206	1201	717	1
1207	1201	1514	1
1208	1201	1599	0
1209	1202	717	1
1210	1202	1601	0
1211	1203	717	1
1212	1203	1321	1
1213	1203	1603	0
1214	1204	717	1
1215	1204	1321	1
1216	1204	1605	0
1217	1204	1606	0
1218	1205	717	1
1219	1205	1608	1
1220	1205	1609	0
1221	1206	717	1
1222	1206	1321	1
1223	1206	1611	0
1224	1207	717	1
1225	1207	1321	1
1226	1207	1613	0
1227	1208	717	1
1228	1208	1514	1
1229	1208	1615	0
1230	1208	1616	0
1231	1209	717	1
1232	1209	1514	1
1233	1209	1618	0
1234	1210	717	1
1235	1210	1514	1
1236	1210	1620	0
1237	1211	717	1
1238	1211	1622	0
1239	1212	717	1
1240	1212	1624	0
1241	1213	717	1
1242	1213	1626	0
1243	1214	717	1
1244	1214	1628	0
1245	1214	1629	0
1246	1215	717	1
1247	1215	1631	0
1248	1215	1632	0
1249	1216	717	1
1250	1216	1634	0
1251	1216	1635	0
1252	1217	717	1
1253	1217	1321	1
1254	1217	1637	0
1255	1218	717	1
1256	1218	1321	1
1257	1218	1639	0
1258	1219	717	1
1259	1219	1321	1
1260	1219	1641	0
1261	1220	717	1
1262	1220	1321	1
1263	1220	1643	0
1264	1221	717	1
1265	1221	1321	1
1266	1221	1645	0
1267	1222	717	1
1268	1222	1321	1
1269	1222	1647	0
1270	1223	717	1
1271	1223	1321	1
1272	1223	1649	0
1273	1224	717	1
1274	1224	1321	1
1275	1224	1651	0
1276	1225	717	1
1277	1225	1653	1
1278	1225	1654	0
1279	1226	717	1
1280	1226	1321	1
1281	1226	1656	0
1282	1227	717	1
1283	1227	1321	1
1284	1227	1658	0
1285	1228	717	1
1286	1228	1321	1
1287	1228	1660	0
1288	1229	717	1
1289	1229	1321	1
1290	1229	1662	0
1291	1230	717	1
1292	1230	1321	1
1293	1230	1664	0
1294	1231	717	1
1295	1231	1321	1
1296	1231	1666	0
1297	1232	717	1
1298	1232	1321	1
1299	1232	1668	0
1300	1233	717	1
1301	1233	1321	1
1302	1233	1670	0
1303	1234	717	1
1304	1234	1321	1
1305	1234	1672	0
1306	1235	717	1
1307	1235	1321	1
1308	1235	1674	0
1309	1236	717	1
1310	1236	1321	1
1311	1236	1676	0
1312	1237	717	1
1313	1237	1321	1
1314	1237	1678	0
1315	1238	717	1
1316	1238	1321	1
1317	1238	1680	0
1318	1239	717	1
1319	1239	1321	1
1320	1239	1682	0
1321	1240	717	1
1322	1240	1321	1
1323	1240	1684	0
1324	1241	717	1
1325	1241	1321	1
1326	1241	1686	0
1327	1242	717	1
1328	1242	1321	1
1329	1242	1688	0
1330	1243	717	1
1331	1243	129	1
1332	1243	1690	0
1333	1244	717	1
1334	1244	1514	1
1335	1244	1692	0
1336	1245	717	1
1337	1245	1514	1
1338	1245	1694	0
1339	1246	717	1
1340	1246	1514	1
1341	1246	1696	0
1342	1246	1697	0
1343	1247	219	1
1344	1248	1700	1
1345	1248	219	1
1346	1251	1700	1
1347	1251	219	1
1348	1252	1700	1
1349	1252	219	1
1350	1253	1706	1
1351	1254	1708	1
1352	1255	1710	1
1353	1256	1710	1
1354	1257	1710	1
1355	1258	1710	1
1356	1259	1715	1
1357	1260	1717	1
1358	1261	1717	1
1359	1262	1717	1
1360	1263	1717	1
1361	1264	1717	1
1362	1265	1717	1
1363	1266	129	1
1364	1267	1725	1
1365	1268	598	1
1366	1269	598	1
1367	1270	1729	1
1368	1271	598	1
1369	1271	1731	0
1370	1272	1729	1
1371	1273	1734	1
1372	1274	1729	1
1373	1275	598	1
1374	1276	1738	1
1375	1276	1739	0
1376	1277	1741	1
1377	1277	1742	0
1378	1278	1744	1
1379	1279	1746	1
1380	1279	1747	1
1381	1279	1748	0
1382	1280	598	1
1383	1280	1750	0
1384	1281	598	1
1385	1282	598	1
1386	1283	598	1
1387	1284	1755	1
1388	1284	1756	1
1389	1285	1758	1
1390	1286	598	1
1391	1286	1760	0
1392	1287	1762	1
1393	1288	598	1
1394	1288	1764	0
1395	1289	1766	1
1396	1290	598	1
1397	1290	1768	0
1398	1291	1770	1
1399	1292	598	1
1400	1293	1773	1
1401	1294	598	1
1402	1295	129	1
1403	1296	129	1
1404	1297	129	1
1405	1298	129	1
1406	1299	129	1
1407	1300	1781	1
1408	1301	1781	1
1409	1302	1781	1
1410	1303	1781	1
1411	1304	1781	1
1412	1305	1787	1
1413	1306	1787	1
1414	1307	129	1
1415	1307	1790	0
1416	1308	129	1
1417	1309	129	1
1418	1310	1794	1
1419	1310	1795	0
1420	1311	129	1
1421	1312	129	1
1422	1313	129	1
1423	1315	1801	1
1424	1317	598	1
1425	1318	598	1
1426	1319	598	1
1427	1320	598	1
1428	1321	197	1
1429	1322	197	1
1430	1323	197	1
1431	1324	197	1
1432	1325	598	1
1433	1325	1812	0
1434	1325	1813	0
1435	1326	598	1
1436	1326	1815	0
1437	1326	1816	0
1438	1327	1818	1
1439	1327	598	1
1440	1327	1819	0
1441	1328	598	1
1442	1329	598	1
1443	1330	598	1
1444	1331	598	1
1445	1331	1824	0
1446	1332	1729	1
1447	1332	1826	0
1448	1333	129	1
1449	1333	1828	0
1450	1334	598	1
1451	1334	1830	0
1452	1337	1834	1
1453	1339	129	1
1454	1340	129	1
1455	1341	129	1
1456	1342	129	1
1457	1343	129	1
1458	1344	129	1
1459	1344	1842	0
1460	1345	129	1
1461	1346	129	1
1462	1347	129	1
1463	1348	129	1
1464	1349	129	1
1465	1350	129	1
1466	1350	1849	0
1467	1351	129	1
1468	1351	1851	0
1469	1352	129	1
1470	1352	1853	0
1471	1353	129	1
1472	1354	1856	1
1473	1354	1857	1
1474	1355	197	1
1475	1356	129	1
1476	1357	129	1
1477	1358	129	1
1478	1369	129	1
1479	1370	129	1
1480	1371	129	1
1481	1374	1818	1
1482	1374	1878	0
1483	1375	1818	1
1484	1375	1880	0
1485	1376	1818	1
1486	1376	1882	0
1487	1377	1818	1
1488	1377	1884	0
1489	1378	1818	1
1490	1378	1886	0
1491	1379	1818	1
1492	1379	1888	0
1493	1380	1818	1
1494	1380	1890	0
1495	1381	1818	1
1496	1381	1892	0
1497	1382	1818	1
1498	1382	1894	0
1499	1383	1818	1
1500	1383	1896	0
1501	1384	1818	1
1502	1384	1898	0
1503	1385	1818	1
1504	1385	1900	0
1505	1385	1901	0
1506	1386	1818	1
1507	1386	1903	0
1508	1387	1818	1
1509	1387	1905	0
1510	1388	1818	1
1511	1388	1907	0
1512	1389	1818	1
1513	1389	1909	0
1514	1390	1818	1
1515	1390	1911	0
1516	1391	1818	1
1517	1391	1913	0
1518	1392	1818	1
1519	1392	1915	0
1520	1393	1818	1
1521	1393	1917	0
1522	1394	1818	1
1523	1394	1919	0
1524	1395	1818	1
1525	1395	1921	0
1526	1396	1818	1
1527	1396	1923	0
1528	1397	1818	1
1529	1397	1925	0
1530	1398	1818	1
1531	1398	1927	0
1532	1399	1818	1
1533	1399	1929	0
1534	1400	1818	1
1535	1400	1931	0
1536	1401	1818	1
1537	1401	1933	0
1538	1402	1818	1
1539	1402	1935	0
1540	1403	1818	1
1541	1403	1937	0
1542	1404	1818	1
1543	1404	1939	0
1544	1405	1818	1
1545	1405	1941	0
1546	1406	1818	1
1547	1406	1943	0
1548	1407	1818	1
1549	1407	1945	0
1550	1408	1818	1
1551	1408	1947	0
1552	1409	1818	1
1553	1409	1949	0
1554	1410	1818	1
1555	1410	1951	0
1556	1411	1818	1
1557	1411	1953	0
1558	1412	1818	1
1559	1413	1818	1
1560	1413	1956	0
1561	1413	1957	0
1562	1414	1818	1
1563	1414	1959	0
1564	1415	1818	1
1565	1415	1961	0
1566	1416	1818	1
1567	1416	1963	0
1568	1417	1818	1
1569	1417	1965	0
1570	1418	1818	1
1571	1418	1967	0
1572	1419	1818	1
1573	1419	1969	0
1574	1420	1818	1
1575	1420	1971	0
1576	1421	1818	1
1577	1421	1973	0
1578	1422	1818	1
1579	1422	1975	0
1580	1423	1818	1
1581	1423	1977	0
1582	1424	1818	1
1583	1424	1979	0
1584	1425	1818	1
1585	1425	1981	0
1586	1426	1818	1
1587	1426	1983	0
1588	1427	1818	1
1589	1427	1985	0
1590	1428	1818	1
1591	1428	1987	0
1592	1429	1818	1
1593	1429	1989	0
1594	1430	1818	1
1595	1430	1991	0
1596	1431	1818	1
1597	1431	1993	0
1598	1432	1818	1
1599	1432	1995	0
1600	1433	1818	1
1601	1433	1997	0
1602	1434	1818	1
1603	1434	1999	0
1604	1435	1818	1
1605	1435	2001	0
1606	1436	1818	1
1607	1436	2003	0
1608	1437	1818	1
1609	1437	2005	0
1610	1438	1818	1
1611	1438	2007	0
1612	1439	1818	1
1613	1439	2009	0
1614	1440	1818	1
1615	1440	2011	0
1616	1440	2012	0
1617	1441	1818	1
1618	1441	2014	0
1619	1442	1818	1
1620	1442	2016	0
1621	1443	1818	1
1622	1443	2018	0
1623	1444	1818	1
1624	1444	2020	0
1625	1445	1818	1
1626	1445	2022	0
1627	1446	1818	1
1628	1446	2024	0
1629	1447	1818	1
1630	1447	2026	0
1631	1448	1818	1
1632	1448	2028	0
1633	1449	1818	1
1634	1449	2030	0
1635	1450	1818	1
1636	1450	2028	0
1637	1451	1818	1
1638	1451	2033	0
1639	1452	1818	1
1640	1452	2035	0
1641	1453	1818	1
1642	1453	2037	0
1643	1454	1818	1
1644	1454	2039	0
1645	1455	1818	1
1646	1455	2041	0
1647	1456	1818	1
1648	1456	2043	0
1649	1457	1818	1
1650	1457	2045	0
1651	1458	1818	1
1652	1458	2047	0
1653	1459	1818	1
1654	1459	2049	0
1655	1460	1818	1
1656	1460	2051	0
1657	1461	1818	1
1658	1461	2053	0
1659	1462	1818	1
1660	1462	2055	0
1661	1463	1818	1
1662	1463	2057	0
1663	1464	1818	1
1664	1464	2059	0
1665	1465	1818	1
1666	1465	2061	0
1667	1466	1818	1
1668	1466	2063	0
1669	1467	1818	1
1670	1467	2065	0
1671	1468	1818	1
1672	1468	2067	0
1673	1469	1818	1
1674	1469	2069	0
1675	1470	1818	1
1676	1470	2071	0
1677	1471	1818	1
1678	1471	2073	0
1679	1472	1818	1
1680	1472	2075	0
1681	1473	1818	1
1682	1473	2077	0
1683	1474	1818	1
1684	1474	2079	0
1685	1475	1818	1
1686	1475	2081	0
1687	1476	1818	1
1688	1476	2083	0
1689	1477	1818	1
1690	1477	2085	0
1691	1478	1818	1
1692	1478	2087	0
1693	1479	129	1
1694	1481	129	1
1695	1482	129	1
1696	1482	2092	0
1697	1483	129	1
1698	1483	2094	0
1699	1484	129	1
1700	1484	2096	0
1701	1485	129	1
1702	1485	2098	0
1703	1486	129	1
1704	1486	2100	0
1705	1487	2102	0
1706	1488	129	1
1707	1488	2104	0
1708	1489	129	1
1709	1489	2106	0
1710	1490	129	1
1711	1490	2108	0
1712	1491	129	1
1713	1491	2110	0
1714	1492	129	1
1715	1492	2112	0
1716	1493	129	1
1717	1493	2114	0
1718	1494	129	1
1719	1494	2116	0
1720	1495	129	1
1721	1495	2118	0
1722	1496	129	1
1723	1496	2120	0
1724	1497	129	1
1725	1497	2122	0
1726	1498	129	1
1727	1498	2124	0
1728	1499	129	1
1729	1499	2126	0
1730	1500	129	1
1731	1500	2128	0
1732	1501	129	1
1733	1501	2130	0
1734	1502	129	1
1735	1502	2132	0
1736	1503	129	1
1737	1504	2135	1
1738	1505	219	1
1739	1506	219	1
1740	1507	219	1
1741	1508	219	1
1742	1509	219	1
1743	1510	219	1
1744	1511	219	1
1745	1512	2144	1
1746	1513	2144	1
1747	1514	129	1
1748	1515	350	1
1749	1515	129	1
1750	1516	129	1
1751	1517	129	1
1752	1518	129	1
1753	1519	129	1
1754	1520	129	1
1755	1521	129	1
1756	1522	129	1
1757	1523	2156	1
1758	1524	129	1
1759	1525	129	1
1760	1526	129	1
1761	1527	129	1
1762	1529	2163	1
1763	1530	129	1
1764	1553	129	1
1765	1554	129	1
1766	1555	2156	1
1767	1555	2190	0
1768	1556	129	1
1769	1557	129	1
1770	1558	129	1
1771	1559	2195	1
1772	1559	129	1
1773	1560	2197	1
1774	1561	2197	1
1775	1562	2197	1
1776	1563	2197	1
1777	1564	2197	1
1778	1565	2197	1
1779	1566	2197	1
1780	1567	2197	1
1781	1568	2197	1
1782	1569	129	1
1783	1570	129	1
1784	1571	2209	1
1785	1571	129	1
1786	1572	197	1
1787	1573	2212	1
1788	1574	2212	1
1789	1575	2212	1
1790	1576	2216	1
1791	1577	2216	1
1792	1579	1801	1
1793	1580	2221	1
1794	1581	2223	1
1795	1582	2225	1
1796	1583	2225	1
1797	1584	2225	1
1798	1585	2225	1
1799	1586	2225	1
1800	1587	2225	1
1801	1588	129	1
1802	1589	2233	1
1803	1590	2233	1
1804	1591	2233	1
1805	1592	2237	1
1806	1593	2239	1
1807	1594	129	1
1808	1595	2225	1
1809	1596	2243	1
1810	1597	2245	1
1811	1598	129	1
1812	1599	2248	1
1813	1600	129	1
1814	1601	129	1
1815	1602	129	1
1816	1603	129	1
1817	1608	129	1
1818	1610	129	1
1819	1611	129	1
1820	1612	129	1
1821	1621	129	1
1822	1622	129	1
1823	1623	129	1
1824	1624	129	1
1825	1624	2274	0
1826	1625	129	1
1827	1626	129	1
1828	1627	129	1
1829	1627	2278	0
1830	1628	129	1
1831	1628	2280	0
1832	1629	129	1
1833	1630	129	1
1834	1630	2283	0
1835	1631	129	1
1836	1632	129	1
1837	1632	2286	0
1838	1633	2288	1
1839	1634	1755	1
1840	1634	1756	1
1841	1635	496	1
1842	1636	598	1
1843	1640	136	1
1844	1640	2296	0
1845	1641	187	1
1846	1642	187	1
1847	1643	2300	1
1848	1643	2301	0
1849	1644	187	1
1850	1645	187	1
1851	1646	129	1
1852	1647	129	1
1853	1648	2307	1
1854	1649	717	1
1855	1650	717	1
1856	1651	717	1
1857	1652	717	1
1858	1652	1321	1
1859	1653	717	1
1860	1653	1321	1
1861	1654	717	1
1862	1654	1321	1
1863	1655	717	1
1864	1655	1321	1
1865	1656	717	1
1866	1656	1321	1
1867	1657	717	1
1868	1658	717	1
1869	1659	717	1
1870	1660	717	1
1871	1661	717	1
1872	1662	717	1
1873	1663	717	1
1874	1664	717	1
1875	1665	717	1
1876	1666	717	1
1877	1667	717	1
1878	1668	717	1
1879	1669	717	1
1880	1670	2330	1
1881	1671	2330	1
1882	1672	1077	1
1883	1672	2333	0
1884	1673	2330	1
1885	1674	129	1
1886	1675	129	1
1887	1676	2330	1
1888	1678	129	1
1889	1679	129	1
1890	1680	129	1
1891	1681	2330	1
1892	1681	2343	0
1893	1682	129	1
1894	1683	129	1
1895	1685	129	1
1896	1686	129	1
1897	1687	129	1
1898	1688	129	1
1899	1689	129	1
1900	1690	129	1
1901	1691	129	1
1902	1692	129	1
1903	1693	129	1
1904	1693	2356	0
1905	1694	129	1
1906	1694	2358	0
1907	1695	129	1
1908	1696	2330	1
1909	1696	2361	1
1910	1696	2362	0
1911	1697	2330	1
1912	1698	2330	1
1913	1699	129	1
1914	1699	2366	0
1915	1700	129	1
1916	1701	129	1
1917	1702	129	1
1918	1703	129	1
1919	1704	129	1
1920	1705	2373	1
1921	1705	2374	0
1922	1706	223	1
1923	1706	2376	0
1924	1707	2378	1
1925	1708	223	1
1926	1709	223	1
1927	1710	223	1
1928	1711	2330	1
1929	1711	2383	0
1930	1711	2384	0
1931	1712	2330	1
1932	1714	2330	1
1933	1715	2330	1
1934	1716	2330	1
1935	1717	2330	1
1936	1718	2330	1
1937	1719	2330	1
1938	1720	2330	1
1939	1721	2330	1
1940	1722	2330	1
1941	1723	129	1
1942	1724	2330	1
1943	1725	129	1
1944	1725	2399	0
1945	1726	129	1
1946	1727	129	1
1947	1728	129	1
1948	1729	129	1
1949	1730	2330	1
1950	1731	2330	1
1951	1732	129	1
1952	1733	129	1
1953	1734	2330	1
1954	1735	2330	1
1955	1736	2330	1
1956	1737	129	1
1957	1740	129	1
1958	1741	2330	1
1959	1742	2330	1
1960	1743	2330	1
1961	1744	129	1
1962	1745	129	1
1963	1746	129	1
1964	1747	2330	1
1965	1747	2422	0
1966	1747	2423	0
1967	1748	2330	1
1968	1749	2330	1
1969	1750	2330	1
1970	1753	2330	1
1971	1754	2330	1
1972	1755	2330	1
1973	1756	2330	1
1974	1757	2330	1
1975	1758	2330	1
1976	1767	129	1
1977	1768	129	1
1978	1768	2445	0
1979	1768	2446	0
1980	1781	129	1
1981	1783	129	1
1982	1785	2464	1
1983	1786	129	1
1984	1787	223	1
1985	1787	129	1
1986	1788	129	1
1987	1789	2469	1
1988	1790	2469	1
1989	1791	223	1
1990	1792	129	1
1991	1793	223	1
1992	1793	129	1
1993	1794	223	1
1994	1795	223	1
1995	1796	223	1
1996	1797	223	1
1997	1798	223	1
1998	1799	223	1
1999	1800	223	1
2000	1801	129	1
2001	1802	129	1
2002	1803	223	1
2003	1804	223	1
2004	1805	223	1
2005	1806	223	1
2006	1807	223	1
2007	1808	223	1
2008	1809	223	1
2009	1811	223	1
2010	1811	129	1
2011	1812	129	1
2012	1814	129	1
2013	1815	129	1
2014	1816	129	1
2015	1818	129	1
2016	1819	2330	1
2017	1822	197	1
2018	1823	129	1
2019	1824	129	1
2020	1825	129	1
2021	1826	129	1
2022	1827	129	1
2023	1828	129	1
2024	1830	129	1
2025	1831	187	1
2026	1832	197	1
2027	1835	197	1
2028	1836	187	1
2029	1837	197	1
2030	1838	197	1
2031	1839	129	1
2032	1841	197	1
2033	1842	2523	1
2034	1842	129	1
2035	1843	2525	1
2036	1844	129	1
2037	1845	129	1
2038	1846	2525	1
2039	1847	187	1
2040	1850	2533	1
2041	1851	129	1
2042	1853	2537	1
2043	1853	187	1
2044	1854	129	1
2045	1855	129	1
2046	1856	2541	1
2047	1856	2542	0
2048	1857	187	1
2049	1858	129	1
2050	1859	2523	1
2051	1859	129	1
2052	1860	129	1
2053	1861	2523	1
2054	1861	129	1
2055	1862	129	1
2056	1863	129	1
2057	1864	466	1
2058	1864	2551	0
2059	1865	129	1
2060	1866	2523	1
2061	1866	129	1
2062	1867	2555	1
2063	1868	2523	1
2064	1868	129	1
2065	1869	129	1
2066	1870	2559	1
2067	1887	2577	1
2068	1888	2577	1
2069	1889	2577	1
2070	1890	2577	1
2071	1891	2577	1
2072	1892	2577	1
2073	1893	2577	1
2074	1894	2577	1
2075	1895	2586	1
2076	1896	2586	1
2077	1897	2586	1
2078	1898	2586	1
2079	1899	2586	1
2080	1900	2586	1
2081	1901	2586	1
2082	1902	2586	1
2083	1903	2586	1
2084	1904	2586	1
2085	1905	2586	1
2086	1906	2586	1
2087	1907	2586	1
2088	1908	2586	1
2089	1909	2586	1
2090	1910	2586	1
2091	1911	2586	1
2092	1912	2586	1
2093	1913	2586	1
2094	1914	2586	1
2095	1915	2586	1
2096	1916	2586	1
2097	1917	2586	1
2098	1918	2577	1
2099	1919	2577	1
2100	1920	2577	1
2101	1921	2577	1
2102	1922	2577	1
2103	1923	2577	1
2104	1924	2586	1
2105	1925	2586	1
2106	1926	2586	1
2107	1927	2586	1
2108	1928	2586	1
2109	1929	2586	1
2110	1930	2586	1
2111	1931	2586	1
2112	1932	2586	1
2113	1933	2577	1
2114	1934	2577	1
2115	1935	2577	1
2116	1936	2577	1
2117	1937	2577	1
2118	1938	2577	1
2119	1939	2577	1
2120	1940	2577	1
2121	1941	2577	1
2122	1942	2577	1
2123	1943	2577	1
2124	1944	2577	1
2125	1945	2586	1
2126	1946	2577	1
2127	1947	2577	1
2128	1948	2577	1
2129	1949	2577	1
2130	1950	2577	1
2131	1951	2577	1
2132	1952	2577	1
2133	1953	2577	1
2134	1954	2577	1
2135	1955	2577	1
2136	1956	2577	1
2137	1957	2577	1
2138	1958	2577	1
2139	1959	2577	1
2140	1960	2577	1
2141	1961	2577	1
2142	1962	2577	1
2143	1963	2577	1
2144	1964	2586	1
2145	1965	2586	1
2146	1966	2586	1
2147	1967	2586	1
2148	1968	2586	1
2149	1969	2586	1
2150	1970	2586	1
2151	1971	2586	1
2152	1972	2664	1
2153	1973	2586	1
2154	1974	2586	1
2155	1975	2586	1
2156	1976	2586	1
2157	1977	2586	1
2158	1978	2586	1
2159	1979	2586	1
2160	1980	2586	1
2161	1981	2586	1
2162	1982	2586	1
2163	1983	2586	1
2164	1984	2586	1
2165	1985	2586	1
2166	1986	2586	1
2167	1987	2586	1
2168	1988	2586	1
2169	1989	2586	1
2170	1990	2586	1
2171	1991	2586	1
2172	1992	2586	1
2173	1993	2586	1
2174	1994	2586	1
2175	1995	2586	1
2176	1996	2586	1
2177	1997	2586	1
2178	1998	2586	1
2179	1999	2586	1
2180	2000	2586	1
2181	2001	2586	1
2182	2002	2586	1
2183	2003	2577	1
2184	2004	2586	1
2185	2005	2577	1
2186	2006	2586	1
2187	2007	2586	1
2188	2008	2586	1
2189	2009	2577	1
2190	2010	2577	1
2191	2011	2586	1
2192	2012	2577	1
2193	2013	2586	1
2194	2014	2577	1
2195	2015	2708	1
2196	2016	2710	1
2197	2017	2710	1
2198	2018	2713	1
2199	2019	2713	1
2200	2020	2586	1
2201	2021	2713	1
2202	2022	2586	1
2203	2023	2577	1
2204	2024	2577	1
2205	2025	2586	1
2206	2026	2586	1
2207	2027	2586	1
2208	2028	2577	1
2209	2029	2586	1
2210	2030	2586	1
2211	2031	2586	1
2212	2032	2586	1
2213	2033	2713	1
2214	2034	2713	1
2215	2035	2713	1
2216	2036	2713	1
2217	2037	2577	1
2218	2038	2586	1
2219	2039	2577	1
2220	2040	2736	1
2221	2041	2577	1
2222	2042	2577	1
2223	2043	2586	1
2224	2044	2577	1
2225	2045	2586	1
2226	2046	2586	1
2227	2047	2577	1
2228	2048	2577	1
2229	2049	2577	1
2230	2050	2586	1
2231	2051	2577	1
2232	2052	2577	1
2233	2053	2577	1
2234	2054	2577	1
2235	2055	2577	1
2236	2056	2577	1
2237	2057	2577	1
2238	2058	2577	1
2239	2059	2577	1
2240	2060	2577	1
2241	2061	2586	1
2242	2062	2586	1
2243	2063	2586	1
2244	2064	2586	1
2245	2065	2762	1
2246	2066	2762	1
2247	2067	2762	1
2248	2068	2577	1
2249	2069	2577	1
2250	2070	2577	1
2251	2071	2577	1
2252	2072	2586	1
2253	2073	2577	1
2254	2074	2577	1
2255	2075	2577	1
2256	2076	2577	1
2257	2077	2736	1
2258	2078	2577	1
2259	2079	2577	1
2260	2080	2736	1
2261	2081	2736	1
2262	2082	2736	1
2263	2083	2577	1
2264	2084	2577	1
2265	2085	2586	1
2266	2086	2586	1
2267	2087	2586	1
2268	2088	2786	1
2269	2089	2786	1
2270	2090	2664	1
2271	2091	2586	1
2272	2092	2586	1
2273	2093	2586	1
2274	2094	2586	1
2275	2095	2586	1
2276	2096	2586	1
2277	2097	2586	1
2278	2098	2577	1
2279	2099	2577	1
2280	2100	2577	1
2281	2101	2577	1
2282	2102	2664	1
2283	2103	2664	1
2284	2104	2713	1
2285	2105	2577	1
2286	2106	2577	1
2287	2107	2806	1
2288	2108	2806	1
2289	2109	2577	1
2290	2110	2577	1
2291	2111	2577	1
2292	2112	2577	1
2293	2113	2577	1
2294	2114	2814	1
2295	2115	2816	1
2296	2116	2806	1
2297	2117	2819	1
2298	2118	2819	1
2299	2119	2577	1
2300	2120	2577	1
2301	2121	2577	1
2302	2122	2577	1
2303	2123	2577	1
2304	2124	2577	1
2305	2125	2577	1
2306	2126	2577	1
2307	2127	2577	1
2308	2128	2577	1
2309	2129	2577	1
2310	2130	2577	1
2311	2131	2577	1
2312	2132	2835	1
2313	2133	2586	1
2314	2134	2586	1
2315	2135	2586	1
2316	2136	2586	1
2317	2137	2586	1
2318	2138	2586	1
2319	2139	2586	1
2320	2140	2586	1
2321	2141	2586	1
2322	2142	2586	1
2323	2143	2577	1
2324	2144	2664	1
2325	2145	2586	1
2326	2146	2577	1
2327	2147	2851	1
2328	2148	2577	1
2329	2149	2854	1
2330	2150	2577	1
2331	2151	2577	1
2332	2152	2586	1
2333	2153	2586	1
2334	2154	2577	1
2335	2155	2861	1
2336	2156	2861	1
2337	2157	2864	1
2338	2158	2577	1
2339	2159	2854	1
\.


--
-- Data for Name: cvterm_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cvterm_relationship (cvterm_relationship_id, type_id, subject_id, object_id) FROM stdin;
1	31	38	36
2	31	39	37
3	31	40	36
4	31	41	37
5	31	55	53
6	31	56	54
7	31	58	37
8	31	59	36
9	31	102	212
10	31	103	1507
11	31	104	103
12	31	105	297
13	31	106	807
14	31	107	797
15	31	109	245
16	31	112	502
17	31	113	502
18	31	114	585
19	31	115	757
20	31	116	337
21	31	117	337
22	31	119	337
23	31	120	815
24	31	121	1507
25	31	122	817
26	31	123	122
27	31	124	817
28	31	125	123
29	31	126	123
30	31	127	122
31	31	128	817
32	31	129	128
33	31	130	103
34	31	131	130
35	31	132	130
36	31	133	798
37	31	134	133
38	31	135	133
39	31	136	1343
40	31	137	936
41	31	138	728
42	31	139	829
43	31	141	1506
44	31	142	253
45	31	145	438
46	31	146	438
47	31	153	798
48	31	155	1783
49	31	156	1816
50	31	157	156
51	31	158	156
52	31	159	854
53	31	161	511
54	31	162	1704
55	31	163	161
56	31	164	1693
57	31	164	1703
58	31	165	1802
59	31	167	1817
60	31	169	502
61	31	170	169
62	31	171	170
63	31	172	171
64	31	173	171
65	31	175	170
66	31	176	170
67	31	177	170
68	31	178	170
69	31	179	170
70	31	180	775
71	31	181	180
72	31	182	183
73	31	183	502
74	31	185	838
75	31	186	838
76	31	189	806
77	31	190	806
78	31	191	190
79	31	192	806
80	31	193	192
81	31	194	192
82	31	195	192
83	31	196	192
84	31	197	192
85	31	198	192
86	31	199	806
87	31	200	806
88	31	201	806
89	31	202	201
90	31	203	1138
91	31	204	449
92	31	205	801
93	31	206	1507
94	31	207	930
95	31	209	214
96	31	210	336
97	31	213	806
98	31	214	543
99	31	215	1138
100	31	216	408
101	31	218	339
102	31	220	775
103	31	221	502
104	31	222	287
105	31	223	214
106	31	224	103
107	31	225	221
108	31	226	225
109	31	227	225
110	31	228	225
111	31	229	806
112	31	230	229
113	31	231	230
114	31	232	221
115	31	233	221
116	31	234	214
117	31	235	502
118	31	236	221
119	31	236	235
120	31	237	236
121	31	238	236
122	31	239	235
123	31	240	998
124	31	241	937
125	31	242	1632
126	31	243	1632
127	31	244	103
128	31	245	1506
129	31	247	461
130	31	248	339
131	31	249	933
132	31	250	455
133	31	251	245
134	31	251	455
135	31	252	245
136	31	253	797
137	31	254	542
138	31	255	542
139	31	256	542
140	31	257	1332
141	31	258	542
142	31	259	542
143	31	260	542
144	31	261	1158
145	31	261	1507
146	31	263	408
147	31	264	935
148	31	265	1515
149	31	266	1515
150	31	267	829
151	31	268	267
152	31	269	1154
153	31	271	1300
154	31	272	829
155	31	272	1300
156	31	273	829
157	31	273	1300
158	31	274	337
159	31	275	815
160	31	276	337
161	31	277	815
162	31	278	815
163	31	279	449
164	31	280	1645
165	31	281	801
166	31	282	203
167	31	283	449
168	31	284	203
169	31	285	942
170	31	286	764
171	31	287	775
172	31	288	282
173	31	290	935
174	31	291	282
175	31	292	290
176	31	293	290
177	31	294	290
178	31	295	513
179	31	296	291
180	31	297	249
181	31	298	1312
182	31	299	1312
183	31	300	249
184	31	301	1158
185	31	302	297
186	31	303	249
187	31	304	297
188	31	305	936
189	31	306	305
190	31	307	305
191	31	308	291
192	31	309	350
193	31	310	284
194	31	311	585
195	31	312	585
196	31	313	312
197	31	314	312
198	31	315	312
199	31	316	312
200	31	317	312
201	31	318	312
202	31	319	312
203	31	320	312
204	31	321	312
205	31	322	312
206	31	323	312
207	31	324	312
208	31	325	312
209	31	326	312
210	31	327	312
211	31	328	312
212	31	329	312
213	31	330	312
214	31	331	312
215	31	332	312
216	31	333	585
217	31	334	585
218	31	335	775
219	31	336	335
220	31	337	511
221	31	337	1632
222	31	338	819
223	31	339	835
224	31	340	284
225	31	341	1508
226	31	342	1603
227	31	343	305
228	31	344	305
229	31	345	241
230	31	348	963
231	31	350	1670
232	31	352	1333
233	31	354	757
234	31	355	757
235	31	356	355
236	31	357	311
237	31	358	355
238	31	359	355
239	31	360	355
240	31	361	355
241	31	362	355
242	31	363	355
243	31	364	355
244	31	365	355
245	31	366	355
246	31	367	355
247	31	368	355
248	31	369	355
249	31	370	355
250	31	371	355
251	31	372	355
252	31	373	355
253	31	374	355
254	31	375	355
255	31	376	757
256	31	377	757
257	31	378	471
258	31	379	835
259	31	380	775
260	31	381	775
261	31	382	806
262	31	382	906
263	31	383	382
264	31	383	387
265	31	383	907
266	31	384	210
267	31	385	213
268	31	385	383
269	31	387	806
270	31	388	759
271	31	389	806
272	31	390	382
273	31	390	389
274	31	391	106
275	31	392	391
276	31	393	391
277	31	395	759
278	31	395	907
279	31	396	759
280	31	397	764
281	31	398	1507
282	31	399	398
283	31	400	1507
284	31	401	771
285	31	402	401
286	31	403	402
287	31	404	1039
288	31	405	935
289	31	407	1333
290	31	408	407
291	31	409	1507
292	31	414	891
293	31	415	224
294	31	416	759
295	31	417	935
296	31	418	936
297	31	419	253
298	31	420	461
299	31	421	461
300	31	422	446
301	31	422	941
302	31	423	210
303	31	424	786
304	31	425	951
305	31	426	798
306	31	427	311
307	31	428	426
308	31	429	951
309	31	430	153
310	31	431	210
311	31	432	1506
312	31	433	426
313	31	434	432
314	31	435	801
315	31	436	432
316	31	437	210
317	31	438	564
318	31	439	544
319	31	440	310
320	31	441	400
321	31	442	1332
322	31	443	930
323	31	444	401
324	31	445	1506
325	31	446	1155
326	31	447	426
327	31	448	1047
328	31	449	445
329	31	450	545
330	31	451	445
331	31	452	1048
332	31	453	545
333	31	454	450
334	31	455	1344
335	31	456	786
336	31	457	400
337	31	458	450
338	31	459	835
339	31	460	459
340	31	461	951
341	31	462	459
342	31	463	892
343	31	464	1002
344	31	465	341
345	31	466	1138
346	31	467	801
347	31	468	1046
348	31	469	467
349	31	471	757
350	31	472	284
351	31	473	775
352	31	474	558
353	31	475	473
354	31	476	753
355	31	477	471
356	31	478	471
357	31	479	471
358	31	480	479
359	31	481	817
360	31	482	705
361	31	483	705
362	31	484	746
363	31	485	471
364	31	486	757
365	31	487	757
366	31	488	471
367	31	489	471
368	31	490	471
369	31	491	757
370	31	492	376
371	31	493	376
372	31	494	376
373	31	495	376
374	31	496	376
375	31	497	376
376	31	498	376
377	31	499	376
378	31	500	376
379	31	502	835
380	31	504	695
381	31	505	757
382	31	506	757
383	31	507	290
384	31	508	752
385	31	510	1507
386	31	511	510
387	31	512	797
388	31	513	245
389	31	514	802
390	31	515	892
391	31	516	892
392	31	517	892
393	31	518	1167
394	31	518	1669
395	31	520	1659
396	31	521	939
397	31	522	583
398	31	523	583
399	31	524	948
400	31	525	948
401	31	526	948
402	31	527	388
403	31	528	388
404	31	529	525
405	31	529	950
406	31	530	524
407	31	530	950
408	31	531	526
409	31	531	950
410	31	532	949
411	31	533	949
412	31	534	949
413	31	535	759
414	31	535	940
415	31	536	416
416	31	537	432
417	31	538	398
418	31	541	1694
419	31	541	1707
420	31	542	1332
421	31	543	798
422	31	544	798
423	31	545	501
424	31	546	300
425	31	547	300
426	31	548	290
427	31	549	548
428	31	550	548
429	31	551	453
430	31	552	930
431	31	553	1314
432	31	555	1817
433	31	556	757
434	31	557	1314
435	31	558	806
436	31	559	1699
437	31	560	562
438	31	561	806
439	31	562	403
440	31	563	1693
441	31	564	1507
442	31	565	502
443	31	566	564
444	31	567	1693
445	31	567	1700
446	31	568	562
447	31	569	232
448	31	570	245
449	31	571	232
450	31	572	562
451	31	573	225
452	31	574	455
453	31	575	228
454	31	575	573
455	31	576	245
456	31	577	227
457	31	577	573
458	31	578	252
459	31	580	562
460	31	581	775
461	31	582	253
462	31	582	576
463	31	583	396
464	31	584	403
465	31	585	287
466	31	586	1311
467	31	587	1038
468	31	588	1311
469	31	589	1038
470	31	590	1038
471	31	591	1038
472	31	592	1038
473	31	593	1038
474	31	594	1039
475	31	595	663
476	31	596	664
477	31	597	665
478	31	598	663
479	31	599	664
480	31	600	665
481	31	601	455
482	31	602	130
483	31	603	130
484	31	606	1038
485	31	607	1038
486	31	608	1038
487	31	609	564
488	31	610	1038
489	31	611	584
490	31	612	1036
491	31	613	584
492	31	614	1693
493	31	615	584
494	31	616	664
495	31	617	663
496	31	618	564
497	31	619	665
498	31	620	1038
499	31	621	1038
500	31	622	1038
501	31	623	1038
502	31	624	1038
503	31	625	1038
504	31	626	1038
505	31	627	1038
506	31	628	584
507	31	629	1038
508	31	630	1038
509	31	631	1038
510	31	632	1038
511	31	633	1038
512	31	634	1038
513	31	635	663
514	31	636	584
515	31	637	584
516	31	638	664
517	31	639	665
518	31	640	1039
519	31	641	1038
520	31	642	1038
521	31	643	1038
522	31	644	1038
523	31	646	284
524	31	647	693
525	31	648	453
526	31	649	1700
527	31	650	1314
528	31	651	1700
529	31	652	1817
530	31	653	1632
531	31	654	241
532	31	655	937
533	31	657	405
534	31	658	594
535	31	659	405
536	31	660	584
537	31	661	584
538	31	662	584
539	31	663	1039
540	31	664	1039
541	31	665	403
542	31	666	1038
543	31	667	1038
544	31	668	1038
545	31	669	652
546	31	670	269
547	31	671	835
548	31	672	594
549	31	673	113
550	31	674	1036
551	31	675	113
552	31	676	1036
553	31	677	113
554	31	678	1036
555	31	679	730
556	31	680	113
557	31	681	933
558	31	682	334
559	31	683	1507
560	31	684	334
561	31	685	681
562	31	686	471
563	31	687	680
564	31	688	585
565	31	689	690
566	31	690	290
567	31	691	585
568	31	692	757
569	31	693	103
570	31	694	693
571	31	695	377
572	31	696	377
573	31	697	334
574	31	698	334
575	31	704	757
576	31	705	690
577	31	706	681
578	31	707	1507
579	31	708	681
580	31	709	681
581	31	710	680
582	31	711	1507
583	31	712	1507
584	31	713	941
585	31	714	941
586	31	715	854
587	31	715	1300
588	31	716	243
589	31	716	854
590	31	717	1051
591	31	718	935
592	31	719	273
593	31	720	273
594	31	721	337
595	31	722	337
596	31	723	273
597	31	724	337
598	31	725	113
599	31	726	730
600	31	727	829
601	31	728	930
602	31	729	1154
603	31	730	930
604	31	731	936
605	31	732	937
606	31	733	180
607	31	733	287
608	31	734	287
609	31	734	767
610	31	735	336
611	31	735	767
612	31	736	180
613	31	736	336
614	31	737	287
615	31	738	935
616	31	739	257
617	31	739	906
618	31	740	938
619	31	741	740
620	31	742	740
621	31	743	391
622	31	744	113
623	31	745	106
624	31	746	757
625	31	747	287
626	31	748	757
627	31	749	585
628	31	750	749
629	31	751	757
630	31	752	354
631	31	753	354
632	31	754	753
633	31	755	753
634	31	756	191
635	31	757	335
636	31	758	113
637	31	759	1508
638	31	760	759
639	31	761	113
640	31	764	290
641	31	765	113
642	31	766	930
643	31	767	775
644	31	768	290
645	31	768	1136
646	31	769	1158
647	31	769	1507
648	31	770	204
649	31	771	400
650	31	772	771
651	31	773	771
652	31	774	771
653	31	775	931
654	31	778	266
655	31	779	265
656	31	780	266
657	31	781	265
658	31	782	420
659	31	783	775
660	31	785	446
661	31	786	161
662	31	787	424
663	31	788	1706
664	31	789	801
665	31	790	455
666	31	791	204
667	31	792	806
668	31	793	1659
669	31	794	792
670	31	795	1314
671	31	796	1579
672	31	797	1505
673	31	798	797
674	31	799	795
675	31	800	799
676	31	801	212
677	31	802	1506
678	31	803	514
679	31	804	514
680	31	805	802
681	31	806	1507
682	31	807	759
683	31	808	1516
684	31	809	1516
685	31	810	808
686	31	811	808
687	31	812	799
688	31	813	795
689	31	814	795
690	31	815	816
691	31	816	1507
692	31	817	816
693	31	818	181
694	31	818	736
695	31	819	1506
696	31	820	819
697	31	821	455
698	31	822	203
699	31	823	794
700	31	824	794
701	31	825	400
702	31	826	398
703	31	827	1659
704	31	828	1508
705	31	829	1154
706	31	830	1659
707	31	831	112
708	31	832	245
709	31	833	1005
710	31	834	1005
711	31	835	501
712	31	836	336
713	31	837	501
714	31	838	837
715	31	839	838
716	31	840	838
717	31	841	838
718	31	842	838
719	31	843	1125
720	31	843	1356
721	31	844	1332
722	31	845	842
723	31	846	842
724	31	847	842
725	31	848	842
726	31	849	842
727	31	850	842
728	31	851	837
729	31	852	398
730	31	853	837
731	31	854	1632
732	31	855	797
733	31	856	542
734	31	857	542
735	31	858	454
736	31	859	858
737	31	860	858
738	31	868	355
739	31	870	257
740	31	871	947
741	31	872	947
742	31	873	1507
743	31	874	1138
744	31	875	874
745	31	876	874
746	31	877	874
747	31	878	874
748	31	879	564
749	31	880	564
750	31	881	739
751	31	881	870
752	31	883	835
753	31	884	835
754	31	885	835
755	31	886	835
756	31	887	797
757	31	891	1005
758	31	892	1005
759	31	896	512
760	31	896	906
761	31	897	917
762	31	898	203
763	31	899	203
764	31	899	1572
765	31	900	203
766	31	900	906
767	31	901	822
768	31	901	900
769	31	901	907
770	31	902	1600
771	31	903	1600
772	31	904	1600
773	31	905	1600
774	31	906	1505
775	31	907	906
776	31	908	835
777	31	909	426
778	31	909	906
779	31	910	419
780	31	911	419
781	31	912	911
782	31	913	911
783	31	914	911
784	31	915	911
785	31	916	835
786	31	917	338
787	31	918	806
788	31	919	835
789	31	920	918
790	31	921	442
791	31	922	442
792	31	923	442
793	31	924	442
794	31	925	442
795	31	926	442
796	31	927	442
797	31	928	442
798	31	929	442
799	31	930	1507
800	31	931	1507
801	31	933	1507
802	31	934	933
803	31	935	933
804	31	936	934
805	31	937	936
806	31	938	935
807	31	939	1507
808	31	940	1508
809	31	941	935
810	31	942	1507
811	31	947	934
812	31	948	940
813	31	949	948
814	31	950	948
815	31	951	936
816	31	952	933
817	31	953	432
818	31	954	953
819	31	955	953
820	31	956	835
821	31	957	956
822	31	958	957
823	31	959	957
824	31	960	956
825	31	961	287
826	31	962	336
827	31	963	339
828	31	964	963
829	31	965	963
830	31	966	965
831	31	967	965
832	31	968	965
833	31	969	965
834	31	970	339
835	31	971	336
836	31	972	336
837	31	972	581
838	31	973	775
839	31	974	973
840	31	974	1029
841	31	975	379
842	31	976	379
843	31	977	339
844	31	978	339
845	31	979	980
846	31	980	339
847	31	981	963
848	31	982	981
849	31	983	247
850	31	984	983
851	31	985	983
852	31	986	981
853	31	987	981
854	31	988	998
855	31	989	998
856	31	990	806
857	31	991	806
858	31	992	806
859	31	993	228
860	31	994	993
861	31	995	994
862	31	996	806
863	31	997	998
864	31	998	806
865	31	1002	806
866	31	1003	853
867	31	1004	235
868	31	1005	835
869	31	1006	1005
870	31	1007	834
871	31	1008	1007
872	31	1009	1007
873	31	1010	834
874	31	1011	834
875	31	1012	1225
876	31	1013	855
877	31	1014	855
878	31	1015	855
879	31	1015	906
880	31	1029	973
881	31	1030	934
882	31	1031	1030
883	31	1032	222
884	31	1033	835
885	31	1034	510
886	31	1035	418
887	31	1036	403
888	31	1038	403
889	31	1039	403
890	31	1040	835
891	31	1041	558
892	31	1042	1046
893	31	1043	1046
894	31	1044	1046
895	31	1045	1046
896	31	1046	444
897	31	1047	444
898	31	1048	444
899	31	1049	1047
900	31	1050	1046
901	31	1051	243
902	31	1052	398
903	31	1053	398
904	31	1054	442
905	31	1055	1054
906	31	1056	1054
907	31	1057	1055
908	31	1058	1055
909	31	1059	1056
910	31	1060	1056
911	31	1061	442
912	31	1062	1061
913	31	1063	1062
914	31	1064	1065
915	31	1065	1061
916	31	1066	1062
917	31	1067	1065
918	31	1073	310
919	31	1074	191
920	31	1075	835
921	31	1076	933
922	31	1077	1030
923	31	1078	113
924	31	1079	1332
925	31	1080	716
926	31	1081	716
927	31	1082	835
928	31	1083	1082
929	31	1084	1082
930	31	1085	545
931	31	1086	1085
932	31	1087	1085
933	31	1088	757
934	31	1089	757
935	31	1090	454
936	31	1091	1014
937	31	1092	1005
938	31	1093	1506
939	31	1094	336
940	31	1094	1093
941	31	1095	806
942	31	1096	942
943	31	1097	1515
944	31	1098	252
945	31	1099	752
946	31	1100	753
947	31	1101	753
948	31	1102	388
949	31	1103	1005
950	31	1104	1506
951	31	1105	215
952	31	1106	874
953	31	1107	415
954	31	1108	544
955	31	1109	244
956	31	1110	1343
957	31	1111	798
958	31	1112	1670
959	31	1113	935
960	31	1114	130
961	31	1115	1113
962	31	1116	1783
963	31	1117	510
964	31	1118	1158
965	31	1119	1783
966	31	1120	801
967	31	1121	1120
968	31	1122	1603
969	31	1123	1603
970	31	1124	1122
971	31	1125	1356
972	31	1126	1603
973	31	1127	1603
974	31	1128	835
975	31	1129	1128
976	31	1130	1128
977	31	1131	839
978	31	1132	847
979	31	1133	1113
980	31	1134	757
981	31	1135	355
982	31	1136	1507
983	31	1137	1136
984	31	1138	1136
985	31	1139	1138
986	31	1140	1137
987	31	1140	1332
988	31	1141	1140
989	31	1142	1046
990	31	1143	438
991	31	1144	1138
992	31	1145	1147
993	31	1146	1147
994	31	1147	444
995	31	1148	874
996	31	1149	759
997	31	1149	940
998	31	1153	203
999	31	1154	1632
1000	31	1155	1632
1001	31	1157	1154
1002	31	1158	212
1003	31	1160	1659
1004	31	1161	1659
1005	31	1162	939
1006	31	1163	521
1007	31	1164	939
1008	31	1165	1669
1009	31	1166	1669
1010	31	1167	939
1011	31	1168	1167
1012	31	1169	1167
1013	31	1170	1169
1014	31	1171	1169
1015	31	1172	1167
1016	31	1173	1172
1017	31	1174	1172
1018	31	1175	1167
1019	31	1176	1167
1020	31	1177	1176
1021	31	1178	1176
1022	31	1179	802
1023	31	1180	1179
1024	31	1181	1179
1025	31	1182	1179
1026	31	1183	1179
1027	31	1186	1649
1028	31	1189	510
1029	31	1189	1650
1030	31	1190	510
1031	31	1190	1650
1032	31	1191	1189
1033	31	1192	1189
1034	31	1193	1189
1035	31	1194	1189
1036	31	1195	1189
1037	31	1196	1189
1038	31	1197	1189
1039	31	1198	1189
1040	31	1199	1189
1041	31	1200	1189
1042	31	1201	1334
1043	31	1202	510
1044	31	1202	1650
1045	31	1203	1175
1046	31	1204	1175
1047	31	1205	1175
1048	31	1206	1205
1049	31	1207	1205
1050	31	1208	1175
1051	31	1209	1208
1052	31	1210	1208
1053	31	1211	1175
1054	31	1212	1211
1055	31	1213	1211
1056	31	1214	1213
1057	31	1215	1213
1058	31	1216	1213
1059	31	1217	1175
1060	31	1218	1217
1061	31	1219	1217
1062	31	1220	1175
1063	31	1221	1220
1064	31	1222	1220
1065	31	1223	1175
1066	31	1224	1175
1067	31	1225	1175
1068	31	1226	1012
1069	31	1227	1012
1070	31	1228	1012
1071	31	1229	1012
1072	31	1230	1225
1073	31	1231	1230
1074	31	1232	1230
1075	31	1233	1230
1076	31	1234	1230
1077	31	1235	1225
1078	31	1236	1235
1079	31	1237	1235
1080	31	1238	1225
1081	31	1239	1238
1082	31	1240	1238
1083	31	1241	1238
1084	31	1242	1238
1085	31	1243	939
1086	31	1244	1243
1087	31	1245	1243
1088	31	1246	1243
1089	31	1247	1230
1090	31	1248	1247
1091	31	1249	1248
1092	31	1250	1248
1093	31	1251	1247
1094	31	1252	1230
1095	31	1253	337
1096	31	1254	815
1097	31	1255	337
1098	31	1256	815
1099	31	1257	815
1100	31	1258	815
1101	31	1259	337
1102	31	1260	337
1103	31	1261	815
1104	31	1262	815
1105	31	1263	337
1106	31	1264	815
1107	31	1265	815
1108	31	1266	1140
1109	31	1267	310
1110	31	1268	753
1111	31	1269	934
1112	31	1270	1269
1113	31	1271	1269
1114	31	1272	1269
1115	31	1273	1269
1116	31	1274	1269
1117	31	1275	312
1118	31	1276	695
1119	31	1277	937
1120	31	1278	937
1121	31	1279	937
1122	31	1280	450
1123	31	1281	450
1124	31	1282	835
1125	31	1283	1282
1126	31	1284	696
1127	31	1285	450
1128	31	1286	1343
1129	31	1287	450
1130	31	1288	1343
1131	31	1289	450
1132	31	1290	1343
1133	31	1291	1289
1134	31	1292	1290
1135	31	1293	1289
1136	31	1294	1290
1137	31	1295	1140
1138	31	1296	1140
1139	31	1297	1296
1140	31	1298	1296
1141	31	1299	1296
1142	31	1300	269
1143	31	1301	1300
1144	31	1302	1301
1145	31	1303	1301
1146	31	1304	1301
1147	31	1305	447
1148	31	1306	447
1149	31	1307	936
1150	31	1308	1307
1151	31	1309	1307
1152	31	1310	690
1153	31	1311	952
1154	31	1312	952
1155	31	1313	290
1156	31	1314	806
1157	31	1315	769
1158	31	1316	806
1159	31	1317	993
1160	31	1318	993
1161	31	1319	1318
1162	31	1320	1318
1163	31	1321	229
1164	31	1322	229
1165	31	1323	1322
1166	31	1324	1322
1167	31	1325	1373
1168	31	1326	1373
1169	31	1327	352
1170	31	1328	352
1171	31	1329	352
1172	31	1330	1370
1173	31	1331	835
1174	31	1332	1507
1175	31	1333	1507
1176	31	1334	1507
1177	31	1335	417
1178	31	1336	417
1179	31	1337	942
1180	31	1338	502
1181	31	1339	935
1182	31	1340	1339
1183	31	1341	1339
1184	31	1342	1339
1185	31	1343	798
1186	31	1344	1506
1187	31	1345	1344
1188	31	1346	1345
1189	31	1347	1345
1190	31	1348	1345
1191	31	1349	245
1192	31	1350	1816
1193	31	1351	1350
1194	31	1352	1350
1195	31	1353	161
1196	31	1354	337
1197	31	1355	442
1198	31	1357	805
1199	31	1358	1357
1200	31	1359	806
1201	31	1360	1359
1202	31	1361	1359
1203	31	1362	1359
1204	31	1363	1359
1205	31	1364	1359
1206	31	1365	1359
1207	31	1366	1359
1208	31	1367	1359
1209	31	1368	1359
1210	31	1369	352
1211	31	1370	1327
1212	31	1371	352
1213	31	1372	352
1214	31	1373	352
1215	31	1374	1370
1216	31	1375	1370
1217	31	1376	1370
1218	31	1377	1371
1219	31	1378	1371
1220	31	1379	1371
1221	31	1380	1371
1222	31	1381	1371
1223	31	1382	1371
1224	31	1383	1371
1225	31	1384	1371
1226	31	1385	1371
1227	31	1386	1371
1228	31	1387	1371
1229	31	1388	1371
1230	31	1389	1371
1231	31	1390	1371
1232	31	1391	1369
1233	31	1392	1369
1234	31	1393	1369
1235	31	1394	1369
1236	31	1395	1369
1237	31	1396	1369
1238	31	1397	1369
1239	31	1398	1369
1240	31	1399	1369
1241	31	1400	1369
1242	31	1401	1369
1243	31	1402	1369
1244	31	1403	1369
1245	31	1404	1369
1246	31	1405	1369
1247	31	1406	1369
1248	31	1407	1369
1249	31	1408	1369
1250	31	1409	1369
1251	31	1410	1369
1252	31	1411	1369
1253	31	1412	1372
1254	31	1413	1412
1255	31	1414	1412
1256	31	1415	1412
1257	31	1416	1412
1258	31	1417	1412
1259	31	1418	1412
1260	31	1419	1412
1261	31	1420	1372
1262	31	1421	1372
1263	31	1422	1372
1264	31	1423	1372
1265	31	1424	1372
1266	31	1425	1372
1267	31	1426	1372
1268	31	1427	1372
1269	31	1428	1372
1270	31	1429	1372
1271	31	1430	1372
1272	31	1431	1372
1273	31	1432	1372
1274	31	1433	1372
1275	31	1434	1372
1276	31	1435	1372
1277	31	1436	1372
1278	31	1437	1372
1279	31	1438	1372
1280	31	1439	1372
1281	31	1440	1373
1282	31	1441	1373
1283	31	1442	1373
1284	31	1443	1373
1285	31	1444	1373
1286	31	1445	1373
1287	31	1446	1373
1288	31	1447	1373
1289	31	1448	1373
1290	31	1449	1373
1291	31	1450	1373
1292	31	1451	1373
1293	31	1452	1373
1294	31	1453	1373
1295	31	1454	1373
1296	31	1455	1373
1297	31	1456	1373
1298	31	1457	1373
1299	31	1458	1373
1300	31	1459	1373
1301	31	1460	1373
1302	31	1461	1373
1303	31	1462	1373
1304	31	1463	1373
1305	31	1464	1373
1306	31	1465	1373
1307	31	1466	1373
1308	31	1467	1373
1309	31	1468	1373
1310	31	1469	1373
1311	31	1470	1373
1312	31	1471	1373
1313	31	1472	1373
1314	31	1473	1373
1315	31	1474	1373
1316	31	1475	1373
1317	31	1476	1373
1318	31	1477	1373
1319	31	1478	1373
1320	31	1479	511
1321	31	1480	418
1322	31	1481	1334
1323	31	1482	1481
1324	31	1483	1481
1325	31	1484	1481
1326	31	1485	1481
1327	31	1486	1481
1328	31	1487	1481
1329	31	1488	1481
1330	31	1489	1481
1331	31	1490	1481
1332	31	1491	1481
1333	31	1492	1481
1334	31	1493	1481
1335	31	1494	1481
1336	31	1495	1481
1337	31	1496	1481
1338	31	1497	1481
1339	31	1498	1481
1340	31	1499	1481
1341	31	1500	1481
1342	31	1501	1481
1343	31	1502	1481
1344	31	1503	545
1345	31	1504	1659
1346	31	1505	102
1347	31	1506	102
1348	31	1507	102
1349	31	1508	102
1350	31	1509	1120
1351	31	1510	1120
1352	31	1511	1120
1353	31	1512	341
1354	31	1513	341
1355	31	1514	1506
1356	31	1515	264
1357	31	1516	264
1358	31	1517	801
1359	31	1518	1649
1360	31	1519	252
1361	31	1520	252
1362	31	1521	252
1363	31	1522	252
1364	31	1523	1632
1365	31	1524	455
1366	31	1525	510
1367	31	1526	801
1368	31	1527	806
1369	31	1528	1730
1370	31	1529	419
1371	31	1530	438
1372	31	1531	1334
1373	31	1532	1334
1374	31	1533	1334
1375	31	1534	1334
1376	31	1535	1334
1377	31	1536	1334
1378	31	1537	1334
1379	31	1538	1334
1380	31	1539	1334
1381	31	1540	1334
1382	31	1541	1334
1383	31	1542	1334
1384	31	1543	1334
1385	31	1544	1334
1386	31	1545	1334
1387	31	1546	1334
1388	31	1547	1334
1389	31	1548	1334
1390	31	1549	1334
1391	31	1550	1334
1392	31	1551	1334
1393	31	1552	1334
1394	31	1553	1506
1395	31	1554	1553
1396	31	1555	416
1397	31	1556	511
1398	31	1557	511
1399	31	1558	1182
1400	31	1558	1356
1401	31	1559	757
1402	31	1560	447
1403	31	1561	1560
1404	31	1562	1560
1405	31	1563	447
1406	31	1564	1563
1407	31	1565	1563
1408	31	1566	204
1409	31	1567	204
1410	31	1568	449
1411	31	1569	1339
1412	31	1570	801
1413	31	1571	935
1414	31	1572	257
1415	31	1572	1137
1416	31	1573	739
1417	31	1574	739
1418	31	1575	739
1419	31	1576	252
1420	31	1577	107
1421	31	1578	267
1422	31	1579	1670
1423	31	1580	759
1424	31	1581	759
1425	31	1582	1595
1426	31	1583	1595
1427	31	1584	1595
1428	31	1585	1595
1429	31	1586	1595
1430	31	1587	1595
1431	31	1588	1632
1432	31	1589	432
1433	31	1590	432
1434	31	1591	432
1435	31	1592	759
1436	31	1593	432
1437	31	1594	252
1438	31	1595	1005
1439	31	1596	1507
1440	31	1597	1356
1441	31	1598	1506
1442	31	1599	775
1443	31	1600	342
1444	31	1601	1125
1445	31	1602	1125
1446	31	1603	1356
1447	31	1604	835
1448	31	1605	1604
1449	31	1606	1605
1450	31	1607	1605
1451	31	1608	1604
1452	31	1609	1608
1453	31	1610	1608
1454	31	1611	1608
1455	31	1612	1619
1456	31	1613	1604
1457	31	1614	1613
1458	31	1615	1613
1459	31	1616	1604
1460	31	1617	1616
1461	31	1618	1616
1462	31	1619	1604
1463	31	1620	1602
1464	31	1621	514
1465	31	1622	514
1466	31	1623	939
1467	31	1624	1623
1468	31	1625	1623
1469	31	1626	1623
1470	31	1627	1623
1471	31	1628	401
1472	31	1629	264
1473	31	1630	1623
1474	31	1631	284
1475	31	1632	931
1476	31	1633	334
1477	31	1634	695
1478	31	1635	757
1479	31	1636	249
1480	31	1637	1638
1481	31	1638	183
1482	31	1639	183
1483	31	1640	510
1484	31	1641	1645
1485	31	1642	1645
1486	31	1643	806
1487	31	1644	1645
1488	31	1645	1507
1489	31	1646	312
1490	31	1647	355
1491	31	1648	432
1492	31	1649	1165
1493	31	1650	1649
1494	31	1651	1167
1495	31	1652	1175
1496	31	1653	1175
1497	31	1654	1175
1498	31	1655	1175
1499	31	1656	1175
1500	31	1657	1659
1501	31	1658	805
1502	31	1659	939
1503	31	1660	1175
1504	31	1661	939
1505	31	1662	1659
1506	31	1663	1659
1507	31	1664	1659
1508	31	1665	1165
1509	31	1666	1649
1510	31	1667	1649
1511	31	1668	510
1512	31	1668	1650
1513	31	1669	939
1514	31	1670	1158
1515	31	1670	1507
1516	31	1671	1670
1517	31	1672	1579
1518	31	1673	1579
1519	31	1674	1673
1520	31	1675	1674
1521	31	1676	1675
1522	31	1677	1674
1523	31	1678	1673
1524	31	1679	1678
1525	31	1680	1678
1526	31	1681	1579
1527	31	1682	1681
1528	31	1683	1682
1529	31	1684	1682
1530	31	1685	1682
1531	31	1686	1682
1532	31	1687	1681
1533	31	1688	1687
1534	31	1689	1687
1535	31	1690	1687
1536	31	1691	1687
1537	31	1692	1817
1538	31	1693	1692
1539	31	1694	1692
1540	31	1695	1817
1541	31	1696	1158
1542	31	1697	769
1543	31	1698	1158
1544	31	1698	1507
1545	31	1699	1817
1546	31	1700	1692
1547	31	1700	1699
1548	31	1701	1812
1549	31	1702	1812
1550	31	1703	555
1551	31	1703	1700
1552	31	1704	1817
1553	31	1705	1706
1554	31	1706	1695
1555	31	1707	1692
1556	31	1708	1694
1557	31	1709	1694
1558	31	1710	1706
1559	31	1711	1783
1560	31	1712	1711
1561	31	1713	1711
1562	31	1714	1738
1563	31	1715	1716
1564	31	1716	1714
1565	31	1717	1716
1566	31	1718	1716
1567	31	1719	1718
1568	31	1720	1719
1569	31	1721	1719
1570	31	1722	1716
1571	31	1723	1714
1572	31	1724	1714
1573	31	1725	1724
1574	31	1726	1725
1575	31	1727	1725
1576	31	1728	1725
1577	31	1729	1725
1578	31	1730	1738
1579	31	1731	1783
1580	31	1732	1731
1581	31	1733	1731
1582	31	1734	1731
1583	31	1735	1730
1584	31	1736	1711
1585	31	1737	1740
1586	31	1738	1711
1587	31	1739	1740
1588	31	1740	1736
1589	31	1741	1738
1590	31	1742	1741
1591	31	1743	1741
1592	31	1744	1711
1593	31	1745	1744
1594	31	1746	1744
1595	31	1747	1783
1596	31	1748	1747
1597	31	1749	1747
1598	31	1750	1762
1599	31	1751	1750
1600	31	1752	1750
1601	31	1753	1762
1602	31	1754	1762
1603	31	1755	1762
1604	31	1756	1762
1605	31	1757	1756
1606	31	1758	1756
1607	31	1759	1747
1608	31	1760	1759
1609	31	1761	1759
1610	31	1762	1747
1611	31	1763	1757
1612	31	1764	1757
1613	31	1765	1758
1614	31	1766	1758
1615	31	1767	1725
1616	31	1768	1747
1617	31	1769	1768
1618	31	1770	1768
1619	31	1771	1768
1620	31	1772	1747
1621	31	1773	1772
1622	31	1774	1773
1623	31	1775	1774
1624	31	1776	1772
1625	31	1777	1772
1626	31	1777	1773
1627	31	1779	1773
1628	31	1780	1772
1629	31	1781	1738
1630	31	1782	1781
1631	31	1784	1762
1632	31	1785	1817
1633	31	1786	1704
1634	31	1787	1704
1635	31	1788	1692
1636	31	1789	1785
1637	31	1790	1785
1638	31	1791	1699
1639	31	1792	1707
1640	31	1792	1791
1641	31	1793	1817
1642	31	1794	1693
1643	31	1794	1706
1644	31	1795	1694
1645	31	1795	1706
1646	31	1796	1695
1647	31	1796	1700
1648	31	1797	1706
1649	31	1798	1694
1650	31	1799	1801
1651	31	1800	1801
1652	31	1801	1699
1653	31	1802	555
1654	31	1802	1695
1655	31	1803	1802
1656	31	1804	1802
1657	31	1805	1694
1658	31	1805	1703
1659	31	1805	1795
1660	31	1806	1703
1661	31	1807	1801
1662	31	1808	1802
1663	31	1809	1703
1664	31	1810	1817
1665	31	1811	1693
1666	31	1811	1694
1667	31	1812	1697
1668	31	1813	1810
1669	31	1814	1783
1670	31	1815	1814
1671	31	1816	342
1672	31	1817	342
1673	31	1818	1731
1674	31	1819	1731
1675	31	1820	1734
1676	31	1821	1734
1677	31	1822	775
1678	31	1823	565
1679	31	1824	565
1680	31	1825	1830
1681	31	1826	1830
1682	31	1827	1824
1683	31	1828	1830
1684	31	1830	1824
1685	31	1831	756
1686	31	1831	1527
1687	31	1832	181
1688	31	1832	733
1689	31	1833	183
1690	31	1835	418
1691	31	1836	1839
1692	31	1837	1838
1693	31	1838	1839
1694	31	1839	418
1695	31	1841	1838
1696	31	1842	345
1697	31	1842	1850
1698	31	1843	336
1699	31	1844	987
1700	31	1845	987
1701	31	1846	1843
1702	31	1847	1843
1703	31	1850	936
1704	31	1851	782
1705	31	1853	1313
1706	31	1854	1313
1707	31	1855	782
1708	31	1856	1850
1709	31	1857	1507
1710	31	1858	1850
1711	31	1859	1858
1712	31	1860	1850
1713	31	1861	1858
1714	31	1862	1868
1715	31	1863	1868
1716	31	1864	1645
1717	31	1865	1868
1718	31	1866	1858
1719	31	1867	1858
1720	31	1868	1850
1721	31	1869	797
1722	31	1870	942
1723	84	109	251
1724	64	115	114
1725	84	116	272
1726	84	117	272
1727	84	119	272
1728	84	120	269
1729	84	137	336
1730	84	141	445
1731	84	205	855
1732	64	206	418
1733	84	213	203
1734	84	241	305
1735	84	242	336
1736	84	243	775
1737	84	250	821
1738	84	251	250
1739	84	252	251
1740	75	272	276
1741	75	273	276
1742	84	274	272
1743	84	275	272
1744	84	277	715
1745	84	278	715
1746	84	281	855
1747	84	298	302
1748	84	299	304
1749	64	335	287
1750	84	342	1620
1751	64	354	311
1752	64	355	312
1753	64	356	313
1754	64	358	315
1755	64	359	316
1756	64	360	317
1757	64	361	318
1758	64	362	319
1759	64	363	320
1760	64	364	321
1761	64	365	322
1762	64	366	323
1763	64	367	324
1764	64	368	325
1765	64	369	326
1766	64	370	327
1767	64	371	328
1768	64	372	329
1769	64	373	330
1770	64	374	331
1771	64	375	332
1772	64	376	333
1773	64	377	334
1774	64	378	749
1775	84	388	288
1776	84	398	1332
1777	84	435	335
1778	81	438	806
1779	64	447	336
1780	84	468	466
1781	64	504	1633
1782	84	521	1162
1783	84	535	291
1784	64	536	203
1785	84	537	288
1786	84	542	253
1787	81	566	249
1788	84	570	790
1789	84	576	574
1790	84	583	310
1791	84	586	304
1792	75	587	572
1793	75	587	674
1794	84	588	302
1795	75	589	572
1796	75	589	580
1797	75	589	676
1798	75	590	572
1799	75	590	676
1800	75	591	580
1801	75	591	678
1802	75	592	572
1803	75	592	580
1804	75	592	678
1805	75	593	572
1806	75	593	678
1807	84	595	672
1808	84	596	672
1809	84	597	672
1810	84	598	658
1811	84	599	658
1812	84	600	658
1813	75	606	560
1814	75	606	580
1815	75	606	674
1816	75	607	560
1817	75	607	674
1818	75	608	560
1819	75	608	572
1820	75	608	580
1821	75	608	674
1822	81	609	249
1823	84	609	618
1824	75	610	560
1825	75	610	572
1826	75	610	674
1827	75	611	560
1828	75	611	572
1829	75	611	580
1830	75	613	572
1831	75	613	580
1832	75	615	572
1833	84	616	404
1834	84	617	404
1835	81	618	775
1836	84	618	438
1837	84	619	404
1838	75	620	568
1839	75	620	674
1840	75	621	568
1841	75	621	572
1842	75	621	674
1843	75	622	568
1844	75	622	580
1845	75	622	676
1846	75	623	568
1847	75	623	676
1848	75	624	568
1849	75	624	572
1850	75	624	676
1851	75	625	568
1852	75	625	580
1853	75	625	678
1854	75	626	568
1855	75	626	678
1856	75	627	568
1857	75	627	572
1858	75	627	678
1859	75	628	568
1860	75	629	560
1861	75	629	568
1862	75	629	580
1863	75	629	674
1864	75	630	560
1865	75	630	568
1866	75	630	674
1867	75	631	560
1868	75	631	568
1869	75	631	572
1870	75	631	580
1871	75	631	674
1872	75	632	560
1873	75	632	568
1874	75	632	572
1875	75	632	674
1876	75	633	560
1877	75	633	568
1878	75	633	572
1879	75	633	580
1880	75	634	560
1881	75	634	568
1882	75	634	572
1883	84	635	640
1884	75	636	568
1885	75	636	572
1886	75	637	568
1887	75	637	572
1888	75	637	580
1889	84	638	640
1890	84	639	640
1891	75	641	580
1892	75	641	674
1893	75	642	572
1894	75	642	580
1895	75	642	674
1896	75	643	580
1897	75	643	676
1898	75	644	568
1899	75	644	580
1900	75	644	674
1901	84	647	1850
1902	84	655	307
1903	84	655	335
1904	75	660	580
1905	75	661	560
1906	75	662	560
1907	75	662	572
1908	75	666	568
1909	75	666	572
1910	75	666	580
1911	75	666	674
1912	75	667	568
1913	75	667	572
1914	75	667	580
1915	75	667	676
1916	75	668	568
1917	75	668	572
1918	75	668	580
1919	75	668	678
1920	64	692	691
1921	64	695	697
1922	64	696	698
1923	61	711	704
1924	61	712	336
1925	84	721	720
1926	84	722	720
1927	84	724	719
1928	84	731	306
1929	84	732	305
1930	84	738	737
1931	64	746	747
1932	64	752	357
1933	64	753	427
1934	75	759	828
1935	80	806	1645
1936	84	832	455
1937	84	844	843
1938	80	854	1645
1939	84	855	253
1940	64	868	1275
1941	84	887	855
1942	84	930	442
1943	80	931	806
1944	84	933	775
1945	84	935	287
1946	84	936	336
1947	84	938	311
1948	84	939	206
1949	84	941	764
1950	84	942	806
1951	84	947	686
1952	84	948	388
1953	84	949	528
1954	84	950	527
1955	84	951	418
1956	84	952	249
1957	84	1030	704
1958	84	1042	1141
1959	84	1050	466
1960	84	1079	843
1961	84	1098	255
1962	84	1113	290
1963	84	1115	689
1964	84	1120	442
1965	100	1122	806
1966	100	1123	457
1967	100	1126	1125
1968	64	1135	314
1969	84	1142	472
1970	84	1145	1048
1971	84	1146	1048
1972	84	1160	1161
1973	84	1169	1168
1974	84	1172	1168
1975	75	1178	1211
1976	75	1178	1225
1977	84	1201	1667
1978	84	1253	272
1979	84	1255	272
1980	84	1256	272
1981	84	1257	272
1982	84	1258	272
1983	84	1259	272
1984	84	1260	272
1985	84	1261	272
1986	84	1262	272
1987	84	1263	272
1988	84	1264	272
1989	84	1265	272
1990	84	1269	355
1991	84	1271	1270
1992	84	1277	307
1993	84	1278	307
1994	84	1279	305
1995	84	1333	102
1996	84	1334	206
1997	75	1337	417
1998	84	1340	749
1999	84	1341	1340
2000	84	1342	1340
2001	75	1346	513
2002	75	1347	433
2003	75	1348	433
2004	84	1354	272
2005	75	1358	447
2006	75	1553	797
2007	84	1556	729
2008	84	1557	267
2009	84	1576	256
2010	84	1580	726
2011	84	1581	726
2012	84	1588	290
2013	84	1589	679
2014	84	1590	679
2015	84	1591	679
2016	84	1592	726
2017	84	1593	726
2018	84	1594	254
2019	64	1634	682
2020	64	1635	684
2021	84	1640	288
2022	75	1644	1643
2023	64	1647	1646
2024	84	1659	1162
2025	84	1662	520
2026	84	1663	520
2027	84	1664	520
2028	31	63	67
2029	31	77	93
2030	31	81	77
2031	31	82	77
2032	31	83	77
2033	31	85	67
2034	31	1888	1932
2035	31	1889	1932
2036	31	1890	1932
2037	31	1891	1932
2038	31	1892	1932
2039	31	1893	1932
2040	31	1894	1932
2041	31	1895	1888
2042	31	1896	1889
2043	31	1897	1890
2044	31	1898	1891
2045	31	1899	1892
2046	31	1900	1893
2047	31	1901	1894
2048	31	1902	1888
2049	31	1903	1888
2050	31	1904	1888
2051	31	1905	1888
2052	31	1906	1888
2053	31	1907	1888
2054	31	1908	1889
2055	31	1909	1889
2056	31	1910	1889
2057	31	1911	1889
2058	31	1912	1889
2059	31	1913	1889
2060	31	1914	2014
2061	31	1915	1890
2062	31	1916	1890
2063	31	1917	1890
2064	31	1918	2037
2065	31	1919	2037
2066	31	1920	2037
2067	31	1921	2037
2068	31	1922	2037
2069	31	1923	2037
2070	31	1924	1891
2071	31	1925	1891
2072	31	1926	1893
2073	31	1927	1893
2074	31	1928	1893
2075	31	1929	1893
2076	31	1930	1893
2077	31	1931	1893
2078	31	1932	1887
2079	31	1933	1887
2080	31	1934	1933
2081	31	1935	1933
2082	31	1936	1933
2083	31	1937	1933
2084	31	1938	1933
2085	31	1939	2070
2086	31	1940	2045
2087	31	1941	2070
2088	31	1942	1933
2089	31	1943	1933
2090	31	1944	1933
2091	31	1945	1933
2092	31	1946	1933
2093	31	1947	1933
2094	31	1948	1938
2095	31	1949	1948
2096	31	1950	1948
2097	31	1951	1948
2098	31	1952	1948
2099	31	1953	1948
2100	31	1954	1938
2101	31	1955	1954
2102	31	1956	1954
2103	31	1957	1954
2104	31	1958	1954
2105	31	1959	1954
2106	31	1960	1948
2107	31	1961	1938
2108	31	1962	1961
2109	31	1963	1938
2110	31	1963	2079
2111	31	1964	1935
2112	31	1965	1937
2113	31	1966	1936
2114	31	1967	1934
2115	31	1968	1934
2116	31	1969	1934
2117	31	1970	1939
2118	31	1971	1939
2119	31	1972	1939
2120	31	1973	1940
2121	31	1974	1941
2122	31	1975	1942
2123	31	1976	1942
2124	31	1977	1943
2125	31	1978	1943
2126	31	1979	1944
2127	31	1980	1945
2128	31	1981	1946
2129	31	1982	1947
2130	31	1983	1933
2131	31	1984	1983
2132	31	1985	1983
2133	31	1986	1983
2134	31	1987	1983
2135	31	1988	1983
2136	31	1989	1983
2137	31	1990	1983
2138	31	1991	1983
2139	31	1992	1983
2140	31	1993	1933
2141	31	1994	1993
2142	31	1995	1933
2143	31	1996	1995
2144	31	1997	1933
2145	31	1998	1997
2146	31	1999	1933
2147	31	2000	1999
2148	31	2001	1933
2149	31	2002	2001
2150	31	2003	2045
2151	31	2004	2003
2152	31	2005	2045
2153	31	2006	2005
2154	31	2007	1933
2155	31	2008	2007
2156	31	2009	1933
2157	31	2010	2009
2158	31	2011	2010
2159	31	2012	2009
2160	31	2013	2012
2161	31	2014	1933
2162	31	2015	1933
2163	31	2016	2015
2164	31	2017	2015
2165	31	2018	2015
2166	31	2019	2015
2167	31	2020	2016
2168	31	2021	2016
2169	31	2022	2017
2170	31	2023	2017
2171	31	2024	2019
2172	31	2025	2018
2173	31	2026	2018
2174	31	2027	2018
2175	31	2028	2018
2176	31	2029	2017
2177	31	2030	2017
2178	31	2031	2017
2179	31	2032	2018
2180	31	2033	2016
2181	31	2034	2016
2182	31	2035	2016
2183	31	2036	2016
2184	31	2037	1933
2185	31	2038	1890
2186	31	2039	2037
2187	31	2040	2037
2188	31	2041	2003
2189	31	2042	2045
2190	31	2043	2042
2191	31	2044	2042
2192	31	2045	1933
2193	31	2046	2049
2194	31	2047	2045
2195	31	2048	2042
2196	31	2049	2045
2197	31	2050	2047
2198	31	2051	1938
2199	31	2051	2075
2200	31	2052	1938
2201	31	2052	2075
2202	31	2053	2075
2203	31	2053	2093
2204	31	2054	1938
2205	31	2054	2074
2206	31	2055	2054
2207	31	2056	2054
2208	31	2057	2054
2209	31	2058	2054
2210	31	2059	2054
2211	31	2060	2054
2212	31	2061	1939
2213	31	2062	1939
2214	31	2063	1939
2215	31	2064	1939
2216	31	2065	1938
2217	31	2066	2065
2218	31	2067	2065
2219	31	2068	1938
2220	31	2069	2007
2221	31	2070	1933
2222	31	2071	2070
2223	31	2072	2071
2224	31	2073	2010
2225	31	2074	1933
2226	31	2075	2078
2227	31	2076	2074
2228	31	2077	2074
2229	31	2078	2074
2230	31	2079	2078
2231	31	2080	2077
2232	31	2081	2075
2233	31	2082	2075
2234	31	2083	2014
2235	31	2084	1938
2236	31	2084	2074
2237	31	2085	1946
2238	31	2086	1946
2239	31	2087	1946
2240	31	2088	1938
2241	31	2089	2088
2242	31	2090	1938
2243	31	2091	2090
2244	31	2092	2090
2245	31	2093	1938
2246	31	2094	2093
2247	31	2095	2093
2248	31	2096	1939
2249	31	2097	1983
2250	31	2098	2077
2251	31	2099	2077
2252	31	2100	1938
2253	31	2101	2100
2254	31	2102	1938
2255	31	2103	2102
2256	31	2104	2016
2257	31	2105	1933
2258	31	2106	2105
2259	31	2107	1933
2260	31	2108	2107
2261	31	2109	1889
2262	31	2110	1889
2263	31	2111	1999
2264	31	2112	1999
2265	31	2113	1933
2266	31	2114	2113
2267	31	2115	1933
2268	31	2116	2115
2269	31	2117	2113
2270	31	2118	2113
2271	31	2119	1933
2272	31	2120	2119
2273	31	2121	2119
2274	31	2122	2119
2275	31	2123	2119
2276	31	2124	2119
2277	31	2125	2124
2278	31	2126	2124
2279	31	2127	2124
2280	31	2128	2127
2281	31	2129	2127
2282	31	2130	2127
2283	31	2131	2127
2284	31	2132	2077
2285	31	2133	2119
2286	31	2134	2119
2287	31	2135	2105
2288	31	2136	2105
2289	31	2137	2105
2290	31	2138	2105
2291	31	2139	2105
2292	31	2140	2105
2293	31	2141	1933
2294	31	2142	2141
2295	31	2143	2141
2296	31	2144	1933
2297	31	2145	2144
2298	31	2146	2144
2299	31	2147	2078
2300	31	2148	2015
2301	31	2149	1933
2302	31	2150	2149
2303	31	2151	2149
2304	31	2152	2150
2305	31	2153	2151
2306	31	2154	1999
2307	31	2155	1933
2308	31	2156	2155
2309	31	2157	2074
2310	31	2158	1933
2311	31	2159	2158
2312	31	2163	2177
2313	31	2164	2177
2314	31	2165	2176
2315	31	2166	2185
2316	31	2167	2162
2317	31	2168	2162
2318	31	2169	2162
2319	31	2170	2162
2320	31	2171	2305
2321	31	2172	2305
2322	31	2173	2305
2323	31	2174	2168
2324	31	2175	2168
2325	31	2176	2174
2326	31	2177	2174
2327	31	2178	2174
2328	31	2179	2174
2329	31	2180	2174
2330	31	2181	2175
2331	31	2182	2175
2332	31	2183	2175
2333	31	2184	2296
2334	31	2185	2305
2335	31	2186	2185
2336	31	2187	2172
2337	31	2188	2172
2338	31	2189	2172
2339	31	2190	2172
2340	31	2191	2172
2341	31	2192	2172
2342	31	2193	2172
2343	31	2194	2173
2344	31	2195	2173
2345	31	2196	2171
2346	31	2197	2175
2347	31	2198	2182
2348	31	2199	2182
2349	31	2200	2182
2350	31	2201	2181
2351	31	2202	2175
2352	31	2203	2213
2353	31	2204	2177
2354	31	2205	2177
2355	31	2206	2177
2356	31	2207	2206
2357	31	2208	2206
2358	31	2209	2206
2359	31	2210	2205
2360	31	2211	2177
2361	31	2212	2177
2362	31	2213	2293
2363	31	2215	2293
2364	31	2216	2184
2365	31	2217	2184
2366	31	2218	2177
2367	31	2219	2212
2368	31	2220	2212
2369	31	2221	2212
2370	31	2222	2212
2371	31	2223	2212
2372	31	2224	2212
2373	31	2225	2212
2374	31	2226	2212
2375	31	2227	2212
2376	31	2228	2212
2377	31	2229	2212
2378	31	2230	2212
2379	31	2231	2176
2380	31	2232	2176
2381	31	2233	2176
2382	31	2234	2176
2383	31	2235	2176
2384	31	2236	2176
2385	31	2237	2176
2386	31	2238	2176
2387	31	2239	2176
2388	31	2240	2176
2389	31	2241	2176
2390	31	2242	2176
2391	31	2243	2176
2392	31	2244	2178
2393	31	2245	2178
2394	31	2246	2178
2395	31	2247	2178
2396	31	2248	2178
2397	31	2249	2178
2398	31	2250	2178
2399	31	2251	2178
2400	31	2252	2179
2401	31	2253	2179
2402	31	2254	2177
2403	31	2255	2177
2404	31	2256	2177
2405	31	2257	2176
2406	31	2258	2170
2407	31	2259	2170
2408	31	2260	2184
2409	31	2261	2184
2410	31	2262	2184
2411	31	2263	2184
2412	31	2264	2184
2413	31	2265	2184
2414	31	2266	2184
2415	31	2267	2184
2416	31	2268	2184
2417	31	2269	2184
2418	31	2270	2184
2419	31	2271	2184
2420	31	2272	2184
2421	31	2273	2184
2422	31	2274	2184
2423	31	2275	2184
2424	31	2276	2184
2425	31	2277	2184
2426	31	2278	2184
2427	31	2279	2184
2428	31	2280	2184
2429	31	2281	2184
2430	31	2282	2184
2431	31	2283	2184
2432	31	2284	2184
2433	31	2285	2184
2434	31	2286	2184
2435	31	2287	2296
2436	31	2288	2287
2437	31	2289	2287
2438	31	2290	2287
2439	31	2291	2287
2440	31	2292	2287
2441	31	2293	2287
2442	31	2294	2287
2443	31	2295	2287
2444	31	2297	2287
2445	31	2298	2205
2446	31	2299	2296
2447	31	2303	2299
2448	31	2305	2304
2449	31	2306	2313
2450	31	2307	2205
2451	31	2308	2291
2452	31	2309	2184
2453	31	2310	2309
2454	31	2311	2309
2455	31	2312	2287
2456	31	2313	2304
2457	31	2314	2313
2458	31	2315	2313
2459	31	2316	2299
\.


--
-- Data for Name: cvtermpath; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cvtermpath (cvtermpath_id, type_id, subject_id, object_id, cv_id, pathdistance) FROM stdin;
5581	31	2226	2212	14	1
5582	31	2212	2226	14	-1
5583	31	2226	2177	14	2
5584	31	2177	2212	14	-2
5585	31	2212	2177	14	1
5586	31	2177	2212	14	-1
5587	31	2226	2174	14	3
5588	31	2174	2177	14	-3
5589	31	2212	2174	14	2
5590	31	2174	2177	14	-2
5591	31	2177	2174	14	1
5592	31	2174	2177	14	-1
5593	31	2226	2168	14	4
5594	31	2168	2174	14	-4
5595	31	2212	2168	14	3
5596	31	2168	2174	14	-3
5597	31	2177	2168	14	2
5598	31	2168	2174	14	-2
5599	31	2174	2168	14	1
5600	31	2168	2174	14	-1
5601	31	2226	2162	14	5
5602	31	2162	2168	14	-5
5603	31	2212	2162	14	4
5604	31	2162	2168	14	-4
5605	31	2177	2162	14	3
5606	31	2162	2168	14	-3
5607	31	2174	2162	14	2
5608	31	2162	2168	14	-2
5609	31	2168	2162	14	1
5610	31	2162	2168	14	-1
5611	31	2231	2176	14	1
5612	31	2176	2231	14	-1
5613	31	2231	2174	14	2
5614	31	2174	2176	14	-2
5615	31	2176	2174	14	1
5616	31	2174	2176	14	-1
5617	31	2231	2168	14	3
5618	31	2176	2168	14	2
5619	31	2231	2162	14	4
5620	31	2176	2162	14	3
5621	31	2168	2162	14	2
5622	31	2265	2184	14	1
5623	31	2184	2265	14	-1
5624	31	2265	2296	14	2
5625	31	2296	2184	14	-2
5626	31	2184	2296	14	1
5627	31	2296	2184	14	-1
5628	31	2180	2174	14	1
5629	31	2174	2180	14	-1
5630	31	2180	2168	14	2
5631	31	2180	2162	14	3
5632	31	2199	2182	14	1
5633	31	2182	2199	14	-1
5634	31	2199	2175	14	2
5635	31	2175	2182	14	-2
5636	31	2182	2175	14	1
5637	31	2175	2182	14	-1
5638	31	2199	2168	14	3
5639	31	2168	2175	14	-3
5640	31	2182	2168	14	2
5641	31	2168	2175	14	-2
5642	31	2175	2168	14	1
5643	31	2168	2175	14	-1
5644	31	2199	2162	14	4
5645	31	2182	2162	14	3
5646	31	2175	2162	14	2
5647	31	2207	2206	14	1
5648	31	2206	2207	14	-1
5649	31	2207	2177	14	2
5650	31	2177	2206	14	-2
5651	31	2206	2177	14	1
5652	31	2177	2206	14	-1
5653	31	2207	2174	14	3
5654	31	2206	2174	14	2
5655	31	2207	2168	14	4
5656	31	2206	2168	14	3
5657	31	2174	2168	14	2
5658	31	2207	2162	14	5
5659	31	2206	2162	14	4
5660	31	2174	2162	14	3
5661	31	2258	2170	14	1
5662	31	2170	2258	14	-1
5663	31	2258	2162	14	2
5664	31	2162	2170	14	-2
5665	31	2170	2162	14	1
5666	31	2162	2170	14	-1
5667	31	2186	2185	14	1
5668	31	2185	2186	14	-1
5669	31	2186	2305	14	2
5670	31	2305	2185	14	-2
5671	31	2185	2305	14	1
5672	31	2305	2185	14	-1
5673	31	2186	2304	14	3
5674	31	2304	2305	14	-3
5675	31	2185	2304	14	2
5676	31	2304	2305	14	-2
5677	31	2305	2304	14	1
5678	31	2304	2305	14	-1
5679	31	2310	2309	14	1
5680	31	2309	2310	14	-1
5681	31	2310	2184	14	2
5682	31	2184	2309	14	-2
5683	31	2309	2184	14	1
5684	31	2184	2309	14	-1
5685	31	2310	2296	14	3
5686	31	2296	2184	14	-3
5687	31	2309	2296	14	2
5688	31	2234	2176	14	1
5689	31	2176	2234	14	-1
5690	31	2234	2174	14	2
5691	31	2234	2168	14	3
5692	31	2234	2162	14	4
5693	31	2168	2162	14	3
5694	31	2266	2184	14	1
5695	31	2184	2266	14	-1
5696	31	2266	2296	14	2
5697	31	2250	2178	14	1
5698	31	2178	2250	14	-1
5699	31	2250	2174	14	2
5700	31	2174	2178	14	-2
5701	31	2178	2174	14	1
5702	31	2174	2178	14	-1
5703	31	2250	2168	14	3
5704	31	2178	2168	14	2
5705	31	2250	2162	14	4
5706	31	2178	2162	14	3
5707	31	2286	2184	14	1
5708	31	2184	2286	14	-1
5709	31	2286	2296	14	2
5710	31	2228	2212	14	1
5711	31	2212	2228	14	-1
5712	31	2228	2177	14	2
5713	31	2228	2174	14	3
5714	31	2177	2174	14	2
5715	31	2228	2168	14	4
5716	31	2177	2168	14	3
5717	31	2228	2162	14	5
5718	31	2177	2162	14	4
5719	31	2230	2212	14	1
5720	31	2212	2230	14	-1
5721	31	2230	2177	14	2
5722	31	2230	2174	14	3
5723	31	2230	2168	14	4
5724	31	2174	2168	14	3
5725	31	2230	2162	14	5
5726	31	2174	2162	14	4
5727	31	2164	2177	14	1
5728	31	2177	2164	14	-1
5729	31	2164	2174	14	2
5730	31	2164	2168	14	3
5731	31	2164	2162	14	4
5732	31	2279	2184	14	1
5733	31	2184	2279	14	-1
5734	31	2279	2296	14	2
5735	31	2193	2172	14	1
5736	31	2172	2193	14	-1
5737	31	2193	2305	14	2
5738	31	2305	2172	14	-2
5739	31	2172	2305	14	1
5740	31	2305	2172	14	-1
5741	31	2193	2304	14	3
5742	31	2172	2304	14	2
5743	31	2311	2309	14	1
5744	31	2309	2311	14	-1
5745	31	2311	2184	14	2
5746	31	2311	2296	14	3
5747	31	2184	2296	14	2
5748	31	2292	2287	14	1
5749	31	2287	2292	14	-1
5750	31	2292	2296	14	2
5751	31	2296	2287	14	-2
5752	31	2287	2296	14	1
5753	31	2296	2287	14	-1
5754	31	2227	2212	14	1
5755	31	2212	2227	14	-1
5756	31	2227	2177	14	2
5757	31	2227	2174	14	3
5758	31	2227	2168	14	4
5759	31	2227	2162	14	5
5760	31	2168	2162	14	4
5761	31	2209	2206	14	1
5762	31	2206	2209	14	-1
5763	31	2209	2177	14	2
5764	31	2209	2174	14	3
5765	31	2209	2168	14	4
5766	31	2209	2162	14	5
5767	31	2198	2182	14	1
5768	31	2182	2198	14	-1
5769	31	2198	2175	14	2
5770	31	2198	2168	14	3
5771	31	2175	2168	14	2
5772	31	2198	2162	14	4
5773	31	2175	2162	14	3
5774	31	2169	2162	14	1
5775	31	2162	2169	14	-1
5776	31	2272	2184	14	1
5777	31	2184	2272	14	-1
5778	31	2272	2296	14	2
5779	31	2240	2176	14	1
5780	31	2176	2240	14	-1
5781	31	2240	2174	14	2
5782	31	2240	2168	14	3
5783	31	2240	2162	14	4
5784	31	2211	2177	14	1
5785	31	2177	2211	14	-1
5786	31	2211	2174	14	2
5787	31	2211	2168	14	3
5788	31	2211	2162	14	4
5789	31	2288	2287	14	1
5790	31	2287	2288	14	-1
5791	31	2288	2296	14	2
5792	31	2253	2179	14	1
5793	31	2179	2253	14	-1
5794	31	2253	2174	14	2
5795	31	2174	2179	14	-2
5796	31	2179	2174	14	1
5797	31	2174	2179	14	-1
5798	31	2253	2168	14	3
5799	31	2179	2168	14	2
5800	31	2253	2162	14	4
5801	31	2179	2162	14	3
5802	31	2282	2184	14	1
5803	31	2184	2282	14	-1
5804	31	2282	2296	14	2
5805	31	2237	2176	14	1
5806	31	2176	2237	14	-1
5807	31	2237	2174	14	2
5808	31	2237	2168	14	3
5809	31	2237	2162	14	4
5810	31	2191	2172	14	1
5811	31	2172	2191	14	-1
5812	31	2191	2305	14	2
5813	31	2191	2304	14	3
5814	31	2305	2304	14	2
5815	31	2241	2176	14	1
5816	31	2176	2241	14	-1
5817	31	2241	2174	14	2
5818	31	2241	2168	14	3
5819	31	2241	2162	14	4
5820	31	2295	2287	14	1
5821	31	2287	2295	14	-1
5822	31	2295	2296	14	2
5823	31	2315	2313	14	1
5824	31	2313	2315	14	-1
5825	31	2315	2304	14	2
5826	31	2304	2313	14	-2
5827	31	2313	2304	14	1
5828	31	2304	2313	14	-1
5829	31	2249	2178	14	1
5830	31	2178	2249	14	-1
5831	31	2249	2174	14	2
5832	31	2249	2168	14	3
5833	31	2249	2162	14	4
5834	31	2307	2205	14	1
5835	31	2205	2307	14	-1
5836	31	2307	2177	14	2
5837	31	2177	2205	14	-2
5838	31	2205	2177	14	1
5839	31	2177	2205	14	-1
5840	31	2307	2174	14	3
5841	31	2205	2174	14	2
5842	31	2307	2168	14	4
5843	31	2205	2168	14	3
5844	31	2307	2162	14	5
5845	31	2205	2162	14	4
5846	31	2251	2178	14	1
5847	31	2178	2251	14	-1
5848	31	2251	2174	14	2
5849	31	2251	2168	14	3
5850	31	2251	2162	14	4
5851	31	2271	2184	14	1
5852	31	2184	2271	14	-1
5853	31	2271	2296	14	2
5854	31	2257	2176	14	1
5855	31	2176	2257	14	-1
5856	31	2257	2174	14	2
5857	31	2257	2168	14	3
5858	31	2257	2162	14	4
5859	31	2316	2299	14	1
5860	31	2299	2316	14	-1
5861	31	2316	2296	14	2
5862	31	2296	2299	14	-2
5863	31	2299	2296	14	1
5864	31	2296	2299	14	-1
5865	31	2218	2177	14	1
5866	31	2177	2218	14	-1
5867	31	2218	2174	14	2
5868	31	2218	2168	14	3
5869	31	2218	2162	14	4
5870	31	2189	2172	14	1
5871	31	2172	2189	14	-1
5872	31	2189	2305	14	2
5873	31	2189	2304	14	3
5874	31	2215	2293	14	1
5875	31	2293	2215	14	-1
5876	31	2215	2287	14	2
5877	31	2287	2293	14	-2
5878	31	2293	2287	14	1
5879	31	2287	2293	14	-1
5880	31	2215	2296	14	3
5881	31	2296	2287	14	-3
5882	31	2293	2296	14	2
5883	31	2260	2184	14	1
5884	31	2184	2260	14	-1
5885	31	2260	2296	14	2
5886	31	2246	2178	14	1
5887	31	2178	2246	14	-1
5888	31	2246	2174	14	2
5889	31	2246	2168	14	3
5890	31	2246	2162	14	4
5891	31	2208	2206	14	1
5892	31	2206	2208	14	-1
5893	31	2208	2177	14	2
5894	31	2208	2174	14	3
5895	31	2208	2168	14	4
5896	31	2208	2162	14	5
5897	31	2284	2184	14	1
5898	31	2184	2284	14	-1
5899	31	2284	2296	14	2
5900	31	2203	2213	14	1
5901	31	2213	2203	14	-1
5902	31	2203	2293	14	2
5903	31	2293	2213	14	-2
5904	31	2213	2293	14	1
5905	31	2293	2213	14	-1
5906	31	2203	2287	14	3
5907	31	2287	2293	14	-3
5908	31	2213	2287	14	2
5909	31	2203	2296	14	4
5910	31	2296	2287	14	-4
5911	31	2213	2296	14	3
5912	31	2287	2296	14	2
5913	31	2252	2179	14	1
5914	31	2179	2252	14	-1
5915	31	2252	2174	14	2
5916	31	2252	2168	14	3
5917	31	2252	2162	14	4
5918	31	2219	2212	14	1
5919	31	2212	2219	14	-1
5920	31	2219	2177	14	2
5921	31	2219	2174	14	3
5922	31	2219	2168	14	4
5923	31	2219	2162	14	5
5924	31	2289	2287	14	1
5925	31	2287	2289	14	-1
5926	31	2289	2296	14	2
5927	31	2294	2287	14	1
5928	31	2287	2294	14	-1
5929	31	2294	2296	14	2
5930	31	2197	2175	14	1
5931	31	2175	2197	14	-1
5932	31	2197	2168	14	2
5933	31	2197	2162	14	3
5934	31	2276	2184	14	1
5935	31	2184	2276	14	-1
5936	31	2276	2296	14	2
5937	31	2312	2287	14	1
5938	31	2287	2312	14	-1
5939	31	2312	2296	14	2
5940	31	2236	2176	14	1
5941	31	2176	2236	14	-1
5942	31	2236	2174	14	2
5943	31	2236	2168	14	3
5944	31	2236	2162	14	4
5945	31	2167	2162	14	1
5946	31	2162	2167	14	-1
5947	31	2245	2178	14	1
5948	31	2178	2245	14	-1
5949	31	2245	2174	14	2
5950	31	2245	2168	14	3
5951	31	2245	2162	14	4
5952	31	2275	2184	14	1
5953	31	2184	2275	14	-1
5954	31	2275	2296	14	2
5955	31	2223	2212	14	1
5956	31	2212	2223	14	-1
5957	31	2223	2177	14	2
5958	31	2223	2174	14	3
5959	31	2223	2168	14	4
5960	31	2223	2162	14	5
5961	31	2255	2177	14	1
5962	31	2177	2255	14	-1
5963	31	2255	2174	14	2
5964	31	2255	2168	14	3
5965	31	2255	2162	14	4
5966	31	2308	2291	14	1
5967	31	2291	2308	14	-1
5968	31	2308	2287	14	2
5969	31	2287	2291	14	-2
5970	31	2291	2287	14	1
5971	31	2287	2291	14	-1
5972	31	2308	2296	14	3
5973	31	2291	2296	14	2
5974	31	2225	2212	14	1
5975	31	2212	2225	14	-1
5976	31	2225	2177	14	2
5977	31	2225	2174	14	3
5978	31	2225	2168	14	4
5979	31	2225	2162	14	5
5980	31	2290	2287	14	1
5981	31	2287	2290	14	-1
5982	31	2290	2296	14	2
5983	31	2192	2172	14	1
5984	31	2172	2192	14	-1
5985	31	2192	2305	14	2
5986	31	2192	2304	14	3
5987	31	2204	2177	14	1
5988	31	2177	2204	14	-1
5989	31	2204	2174	14	2
5990	31	2204	2168	14	3
5991	31	2204	2162	14	4
5992	31	2202	2175	14	1
5993	31	2175	2202	14	-1
5994	31	2202	2168	14	2
5995	31	2202	2162	14	3
5996	31	2280	2184	14	1
5997	31	2184	2280	14	-1
5998	31	2280	2296	14	2
5999	31	2261	2184	14	1
6000	31	2184	2261	14	-1
6001	31	2261	2296	14	2
6002	31	2270	2184	14	1
6003	31	2184	2270	14	-1
6004	31	2270	2296	14	2
6005	31	2314	2313	14	1
6006	31	2313	2314	14	-1
6007	31	2314	2304	14	2
6008	31	2216	2184	14	1
6009	31	2184	2216	14	-1
6010	31	2216	2296	14	2
6011	31	2238	2176	14	1
6012	31	2176	2238	14	-1
6013	31	2238	2174	14	2
6014	31	2238	2168	14	3
6015	31	2238	2162	14	4
6016	31	2233	2176	14	1
6017	31	2176	2233	14	-1
6018	31	2233	2174	14	2
6019	31	2233	2168	14	3
6020	31	2233	2162	14	4
6021	31	2232	2176	14	1
6022	31	2176	2232	14	-1
6023	31	2232	2174	14	2
6024	31	2232	2168	14	3
6025	31	2232	2162	14	4
6026	31	2217	2184	14	1
6027	31	2184	2217	14	-1
6028	31	2217	2296	14	2
6029	31	2210	2205	14	1
6030	31	2205	2210	14	-1
6031	31	2210	2177	14	2
6032	31	2210	2174	14	3
6033	31	2210	2168	14	4
6034	31	2210	2162	14	5
6035	31	2262	2184	14	1
6036	31	2184	2262	14	-1
6037	31	2262	2296	14	2
6038	31	2183	2175	14	1
6039	31	2175	2183	14	-1
6040	31	2183	2168	14	2
6041	31	2183	2162	14	3
6042	31	2196	2171	14	1
6043	31	2171	2196	14	-1
6044	31	2196	2305	14	2
6045	31	2305	2171	14	-2
6046	31	2171	2305	14	1
6047	31	2305	2171	14	-1
6048	31	2196	2304	14	3
6049	31	2171	2304	14	2
6050	31	2256	2177	14	1
6051	31	2177	2256	14	-1
6052	31	2256	2174	14	2
6053	31	2256	2168	14	3
6054	31	2256	2162	14	4
6055	31	2242	2176	14	1
6056	31	2176	2242	14	-1
6057	31	2242	2174	14	2
6058	31	2242	2168	14	3
6059	31	2242	2162	14	4
6060	31	2222	2212	14	1
6061	31	2212	2222	14	-1
6062	31	2222	2177	14	2
6063	31	2222	2174	14	3
6064	31	2222	2168	14	4
6065	31	2222	2162	14	5
6066	31	2298	2205	14	1
6067	31	2205	2298	14	-1
6068	31	2298	2177	14	2
6069	31	2298	2174	14	3
6070	31	2298	2168	14	4
6071	31	2298	2162	14	5
6072	31	2264	2184	14	1
6073	31	2184	2264	14	-1
6074	31	2264	2296	14	2
6075	31	2165	2176	14	1
6076	31	2176	2165	14	-1
6077	31	2165	2174	14	2
6078	31	2165	2168	14	3
6079	31	2165	2162	14	4
6080	31	2254	2177	14	1
6081	31	2177	2254	14	-1
6082	31	2254	2174	14	2
6083	31	2254	2168	14	3
6084	31	2254	2162	14	4
6085	31	2247	2178	14	1
6086	31	2178	2247	14	-1
6087	31	2247	2174	14	2
6088	31	2247	2168	14	3
6089	31	2247	2162	14	4
6090	31	2239	2176	14	1
6091	31	2176	2239	14	-1
6092	31	2239	2174	14	2
6093	31	2239	2168	14	3
6094	31	2239	2162	14	4
6095	31	2194	2173	14	1
6096	31	2173	2194	14	-1
6097	31	2194	2305	14	2
6098	31	2305	2173	14	-2
6099	31	2173	2305	14	1
6100	31	2305	2173	14	-1
6101	31	2194	2304	14	3
6102	31	2173	2304	14	2
6103	31	2274	2184	14	1
6104	31	2184	2274	14	-1
6105	31	2274	2296	14	2
6106	31	2285	2184	14	1
6107	31	2184	2285	14	-1
6108	31	2285	2296	14	2
6109	31	2190	2172	14	1
6110	31	2172	2190	14	-1
6111	31	2190	2305	14	2
6112	31	2190	2304	14	3
6113	31	2229	2212	14	1
6114	31	2212	2229	14	-1
6115	31	2229	2177	14	2
6116	31	2229	2174	14	3
6117	31	2229	2168	14	4
6118	31	2229	2162	14	5
6119	31	2277	2184	14	1
6120	31	2184	2277	14	-1
6121	31	2277	2296	14	2
6122	31	2201	2181	14	1
6123	31	2181	2201	14	-1
6124	31	2201	2175	14	2
6125	31	2175	2181	14	-2
6126	31	2181	2175	14	1
6127	31	2175	2181	14	-1
6128	31	2201	2168	14	3
6129	31	2181	2168	14	2
6130	31	2201	2162	14	4
6131	31	2181	2162	14	3
6132	31	2297	2287	14	1
6133	31	2287	2297	14	-1
6134	31	2297	2296	14	2
6135	31	2163	2177	14	1
6136	31	2177	2163	14	-1
6137	31	2163	2174	14	2
6138	31	2163	2168	14	3
6139	31	2163	2162	14	4
6140	31	2188	2172	14	1
6141	31	2172	2188	14	-1
6142	31	2188	2305	14	2
6143	31	2188	2304	14	3
6144	31	2248	2178	14	1
6145	31	2178	2248	14	-1
6146	31	2248	2174	14	2
6147	31	2248	2168	14	3
6148	31	2248	2162	14	4
6149	31	2303	2299	14	1
6150	31	2299	2303	14	-1
6151	31	2303	2296	14	2
6152	31	2306	2313	14	1
6153	31	2313	2306	14	-1
6154	31	2306	2304	14	2
6155	31	2187	2172	14	1
6156	31	2172	2187	14	-1
6157	31	2187	2305	14	2
6158	31	2187	2304	14	3
6159	31	2235	2176	14	1
6160	31	2176	2235	14	-1
6161	31	2235	2174	14	2
6162	31	2235	2168	14	3
6163	31	2235	2162	14	4
6164	31	2268	2184	14	1
6165	31	2184	2268	14	-1
6166	31	2268	2296	14	2
6167	31	2259	2170	14	1
6168	31	2170	2259	14	-1
6169	31	2259	2162	14	2
6170	31	2166	2185	14	1
6171	31	2185	2166	14	-1
6172	31	2166	2305	14	2
6173	31	2166	2304	14	3
6174	31	2200	2182	14	1
6175	31	2182	2200	14	-1
6176	31	2200	2175	14	2
6177	31	2200	2168	14	3
6178	31	2200	2162	14	4
6179	31	2263	2184	14	1
6180	31	2184	2263	14	-1
6181	31	2263	2296	14	2
6182	31	2269	2184	14	1
6183	31	2184	2269	14	-1
6184	31	2269	2296	14	2
6185	31	2195	2173	14	1
6186	31	2173	2195	14	-1
6187	31	2195	2305	14	2
6188	31	2195	2304	14	3
6189	31	2281	2184	14	1
6190	31	2184	2281	14	-1
6191	31	2281	2296	14	2
6192	31	2224	2212	14	1
6193	31	2212	2224	14	-1
6194	31	2224	2177	14	2
6195	31	2224	2174	14	3
6196	31	2224	2168	14	4
6197	31	2224	2162	14	5
6198	31	2283	2184	14	1
6199	31	2184	2283	14	-1
6200	31	2283	2296	14	2
6201	31	2244	2178	14	1
6202	31	2178	2244	14	-1
6203	31	2244	2174	14	2
6204	31	2244	2168	14	3
6205	31	2244	2162	14	4
6206	31	2221	2212	14	1
6207	31	2212	2221	14	-1
6208	31	2221	2177	14	2
6209	31	2221	2174	14	3
6210	31	2221	2168	14	4
6211	31	2221	2162	14	5
6212	31	2243	2176	14	1
6213	31	2176	2243	14	-1
6214	31	2243	2174	14	2
6215	31	2243	2168	14	3
6216	31	2243	2162	14	4
6217	31	2273	2184	14	1
6218	31	2184	2273	14	-1
6219	31	2273	2296	14	2
6220	31	2267	2184	14	1
6221	31	2184	2267	14	-1
6222	31	2267	2296	14	2
6223	31	2278	2184	14	1
6224	31	2184	2278	14	-1
6225	31	2278	2296	14	2
6226	31	2220	2212	14	1
6227	31	2212	2220	14	-1
6228	31	2220	2177	14	2
6229	31	2220	2174	14	3
6230	31	2220	2168	14	4
6231	31	2220	2162	14	5
\.


--
-- Data for Name: cvtermprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cvtermprop (cvtermprop_id, cvterm_id, type_id, value, rank) FROM stdin;
1	31	32	1	0
2	31	33	1	0
3	31	34	1	0
4	36	32	1	0
5	36	33	1	0
6	36	34	1	0
7	37	32	1	0
8	37	33	1	0
9	37	34	1	0
10	38	32	1	0
11	38	33	1	0
12	38	34	1	0
13	39	32	1	0
14	39	33	1	0
15	39	34	1	0
16	40	34	1	0
17	41	34	1	0
18	42	34	1	0
19	42	32	1	0
20	43	34	1	0
21	43	32	1	0
22	47	34	1	0
23	49	34	1	0
24	50	34	1	0
25	51	34	1	0
26	52	34	1	0
27	58	32	1	0
28	58	34	1	0
29	59	32	1	0
30	59	34	1	0
31	63	34	1	0
32	64	34	1	0
33	67	34	1	0
34	77	78	1	0
35	80	34	1	0
36	82	78	1	0
37	83	78	1	0
38	84	34	1	0
39	93	78	1	0
40	107	29	This term is mapped to MGED. This term is now located in OBI, with the following ID OBI_0000406.	0
41	145	29	Please not the synonym R psi M uses the spelled out form of the greek letter.	0
42	162	29	FLAG - this term is should probably be a part of rather than an is_a.	0
43	166	29	This classes of attributes was added by MA to allow the broad description of genes based on qualities of the transcript(s). A product of SO meeting 2004.	0
44	206	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology. The term 'protein' was merged with 'polypeptide'. Although 'protein' was a sequenece_attribute and therefore meant to describe the quality rather than an actual feature, it was being used erroneously. It is replaced by 'peptidyl' as the polymer attribute.	0
45	222	29	May contain introns.	0
46	223	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
47	225	29	By:<protein_id>.	0
48	234	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
49	239	29	Exapmles are x-innactivation and immunoglobulin formation.	0
50	249	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
51	254	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
52	255	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
53	256	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology. Drosophila melanogaster PACs carry an average insert size of 80 kb. The library represents a 6-fold coverage of the genome.	0
54	257	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
55	258	29	Paper: vans GA et al. High efficiency vectors for cosmid microcloning and genomic analysis. Gene 1989; 79(1):9-20. This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
56	260	29	Birren BW et al. A human chromosome 22 fosmid resource: mapping and analysis of 96 clones. Genomics 1996.	0
57	264	29	With spliceosomal introns, the splice sites bind the spliceosomal machinary.	0
58	267	29	An enhancer may participate in an enhanceosome GO:0034206. A protein-DNA complex formed by the association of a distinct set of general and specific transcription factors with a region of enhancer DNA. The cooperative assembly of an enhanceosome confers specificity of transcriptional regulation. This comment is a placeholder should we start to make cross products with GO.	0
59	269	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology. The region on a DNA molecule involved in RNA polymerase binding to initiate transcription.	0
60	280	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
61	286	29	May have either GT-AG or AT-AG 5' and 3' boundaries.	0
62	290	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
63	335	29	A processed transcript cannot contain introns.	0
64	336	29	An mRNA does not contain introns as it is a processed_transcript. The equivalent kind of primary_transcript is protein_coding_primary_transcript (SO:0000120) which may contain introns. This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
65	338	29	The definition was modified by Rama. ORF is defined by the sequence, whereas the CDS is defined according to whether a polypeptide is made. This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
66	355	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
67	376	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
68	379	29	Formerly called transcript_by_bound_factor.	0
69	380	29	Formerly called transcript_by_bound_nucleic_acid.	0
70	381	29	Formerly called transcript_by_bound_protein.	0
71	397	29	May have either GT-AC or AT-AC 5' and 3' boundaries.	0
72	407	29	Modified base:<modified_base>.	0
73	413	29	similar to:<sequence_id>	0
74	419	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
75	442	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
76	447	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
77	458	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
78	473	29	This was moved to be a child of transcript (SO:0000673) because some enzymatic RNA regions are part of primary transcripts and some are part of processed transcripts.	0
79	504	29	An evolutionarily conserved eukaryotic low molecular weight RNA capable of intermolecular hybridization with both homologous and heterologous 18S rRNA.	0
80	510	29	Discrete.	0
255	1521	29	An example of this kind of read is one produced by ABI SOLiD.	0
81	518	29	Range. Old definition from before biosapiens: A region of a single polypeptide chain that folds into an independent unit and exhibits biological activity. A polypeptide chain may have multiple domains.	0
82	520	29	Old def before biosapiens:The sequence for an N-terminal domain of a secreted protein; this domain is involved in attaching nascent polypeptide to the membrane leader sequence.	0
83	521	29	This term mature peptide, merged with the biosapiens term mature protein region and took that to be the new name. Old def: The coding sequence for the mature or final peptide or protein product following post-translational modification.	0
84	542	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
85	543	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
86	544	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
87	562	29	I am using the term segment instead of gene here to avoid confusion with the region 'gene'.	0
88	604	29	This concept cam about as a direct result of the SO meeting August 2004.nThe exact nature of the relationship between transcribed_region and gene is still up for discussion. We are going with 'associated_with' for the time being.	0
89	609	29	This is the analog of the exon of a functional gene. The term was requested by Rama - SGD to allow the annotation of the parts of a pseudogene. Non-functional is defined as either its transcription or translation (or both) are prevented due to one or more mutatations.	0
90	618	29	This is the analog of the transcript of a functional gene. The term was requested by Rama - SGD to allow the annotation of the parts of a pseudogene. Non-functional is defined as either its transcription or translation (or both) are prevented due to one or more mutatations.	0
91	652	29	Examples are Nullo-4, Haplo-4 and tiplo-4 in Drosophila.	0
92	671	29	GO:0003964 RNA-directed DNA polymerase activity.	0
93	689	29	GO:0000372.	0
94	705	29	GO:0000373.	0
95	707	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
96	757	29	A ncRNA is a processed_transcript, so it may not contain parts such as transcribed_spacer_regions that are removed in the act of processing. For the corresponding primary_transcripts, please see term SO:0000483 nc_primary_transcript.	0
97	764	29	GO:0000398.	0
98	797	29	Requested by Lynn Crosby, jan 2006.	0
99	801	29	A junction is a boundary between regions. A boundary has an extent of zero.	0
100	806	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology. A gene may be considered as a unit of inheritance.	0
101	808	29	This region contains a polypyridine tract and AG dinucleotide in some organisms and is UUUCAG in C. elegans.	0
102	809	29	SL RNA contains a donor site.	0
103	819	29	This term was added after a request by SGD. August 2004. Modified after SO meeting in Cambridge to not include start or stop.	0
104	820	29	Term requested by Rama from SGD.	0
105	822	29	requested by Michael on 19 Nov 2004.	0
106	823	29	Requested by Michael, 19 nov 2004.	0
107	824	29	Requested by MA nov 19 2004.	0
108	827	29	Added to bring SO inline with the embl ddbj genbank feature table. Old definition before biosapiens: The coding sequence for an N-terminal domain of a nuclear-encoded organellar protein. This domain is involved in post translational import of the protein into the organelle.	0
109	828	29	Added to comply with the feature table. A single repeat.	0
110	829	29	Requested by Stepen Grossmann Dec 2004.	0
111	830	29	Intein-mediated protein splicing occurs after mRNA has been translated into a protein.	0
112	833	29	Term added because of request by MO people.	0
113	836	29	Added for the MO people.	0
114	839	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
115	871	29	Added in response to comment from Kelly Williams from Indiana. Nov 2005.	0
116	872	29	Added in response to Kelly Williams from Indiana. Date: Nov 2005.	0
117	873	29	Added in respose to request by Simon Twigger November 14th 2005.	0
118	874	29	Genomic islands are transmissible elements characterized by large size (>10kb).	0
119	875	29	Nature Reviews Microbiology 2, 414-424 (2004); doi:10.1038 micro 884 GENOMIC ISLANDS IN PATHOGENIC AND ENVIRONMENTAL MICROORGANISMS Ulrich Dobrindt, Bianca Hochhut, Ute Hentschel & Jorg Hacker.	0
120	876	29	Genes for phenolic compound degradation in Pseudomonas putida are found on metabolic islands.	0
121	877	29	The iron-uptake ability of many pathogens are conveyed by adaptive islands. Nature Reviews Microbiology 2, 414-424 (2004); doi:10.1038 micro 884 GENOMIC ISLANDS IN PATHOGENIC AND ENVIRONMENTAL MICROORGANISMS Ulrich Dobrindt, Bianca Hochhut, Ute Hentschel & Jorg Hacker.	0
122	878	29	Nitrogen fixation in Rhizobiaceae species is encoded by symbiosis islands. Evolution of rhizobia by acquisition of a 500-kb symbiosis island that integrates into a phe-tRNA gene. John T. Sullivan and Clive W. Ronso PNAS 1998 Apr 28 95 (9) 5145-5149.	0
123	879	29	Added Jan 2006 to allow the annotation of the pseudogenic rRNA by flybase. Non-functional is defined as its transcription is prevented due to one or more mutatations.	0
124	880	29	Added Jan 2006 to allow the annotation of the pseudogenic tRNA by flybase. Non-functional is defined as its transcription is prevented due to one or more mutatations.	0
125	881	29	Requested by Lynn Crosby Jan 2006.	0
126	882	29	Added by KE Jan 2006 to capture the kinds of attributes of TEs	0
127	887	29	Added in response to Lynn Crosby. A clone insert may be composed of many cloned regions.	0
128	888	29	Added jan 2006 by KE.	0
129	898	29	Modified as requested by Lynn - FB. May 2007.	0
130	930	29	This is a manufactured term, that serves the purpose of allow the parts of a chromosome to have an is_a path to the root.	0
131	931	29	A manufactured term used to allow the parts of a gene to have an is_a path to the root.	0
132	932	29	This is a manufactured term to allow the parts of promoter to have an is_a path back to the root.	0
133	933	29	This term was added to provide a grouping term for the region parts of transcript, thus giving them an is_a path back to the root.	0
134	934	29	A manufactured term to collect together the parts of a mature transcript and give them an is_a path to the root.	0
135	935	29	This term was added to provide a grouping term for the region parts of primary_transcript, thus giving them an is_a path back to the root.	0
136	936	29	This term was added to provide a grouping term for the region parts of mRNA, thus giving them an is_a path back to the root.	0
137	937	29	A region of UTR. This term is a grouping term to allow the parts of UTR to have an is_a path to the root.	0
138	938	29	To allow transcribed_spacer_region to have a path to the root.	0
139	939	29	Added to allow the polypeptide regions to have is_a paths back to the root.	0
140	940	29	A manufactured to group the parts of repeats, to give them an is_a path back to the root.	0
141	941	29	A terms added to allow the parts of introns to have is_a paths to the root.	0
142	943	29	This is a manufactured term to allow the parts of bacterial_RNApol_promoter to have an is_a path back to the root.	0
143	944	29	This is a manufactured term to allow the parts of RNApol_II_promoter to have an is_a path back to the root.	0
144	945	29	This is a manufactured term to allow the parts of RNApol_III_promoter_type_1 to have an is_a path back to the root.	0
145	946	29	This is a manufactured term to allow the parts of RNApol_III_promoter_type_2 to have an is_a path back to the root.	0
146	947	29	This term was added to provide a grouping term for the region parts of tmRNA, thus giving them an is_a path back to the root.	0
147	954	29	A term to be used in conjunction with the paralogous_to relationship.	0
148	955	29	This term should be used in conjunction with the similarity relationships defined in SO.	0
149	1005	29	This term is the hypernym of attributes and should not be annotated to.	0
150	1017	29	The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa.	0
151	1018	29	The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa.	0
152	1024	29	The type of RNA editing found in the mitochondria of Myxomycota, characterized by the insertion of mono- and dinucleotides in RNAs relative to their mtDNA template and in addition, C to U base conversion. The most common mononucleotide insertion is cytidine, although a number of uridine mononucleotides are inserted at specific sites. Adenine and guanine have not been observed in mononucleotide insertions. Five different dinucleotide insertions have been observed, GC, GU, CU, AU and AA. Both mono- and dinucleotide insertions create open reading frames in mRNA and contribute to highly conserved structural features of rRNAs and tRNAs.	0
153	1025	29	The type of RNA editing found in the mitochondria of Myxomycota, characterized by the insertion of mono- and dinucleotides in RNAs relative to their mtDNA template and in addition, C to U base conversion. The most common mononucleotide insertion is cytidine, although a number of uridine mononucleotides are inserted at specific sites. Adenine and guanine have not been observed in mononucleotide insertions. Five different dinucleotide insertions have been observed, GC, GU, CU, AU and AA. Both mono- and dinucleotide insertions create open reading frames in mRNA and contribute to highly conserved structural features of rRNAs and tRNAs.	0
154	1026	29	The type of RNA editing found in the mitochondria of Myxomycota, characterized by the insertion of mono- and dinucleotides in RNAs relative to their mtDNA template and in addition, C to U base conversion. The most common mononucleotide insertion is cytidine, although a number of uridine mononucleotides are inserted at specific sites. Adenine and guanine have not been observed in mononucleotide insertions. Five different dinucleotide insertions have been observed, GC, GU, CU, AU and AA. Both mono- and dinucleotide insertions create open reading frames in mRNA and contribute to highly conserved structural features of rRNAs and tRNAs.	0
155	1027	29	The type of RNA editing found in the mitochondria of Myxomycota, characterized by the insertion of mono- and dinucleotides in RNAs relative to their mtDNA template and in addition, C to U base conversion. The most common mononucleotide insertion is cytidine, although a number of uridine mononucleotides are inserted at specific sites. Adenine and guanine have not been observed in mononucleotide insertions. Five different dinucleotide insertions have been observed, GC, GU, CU, AU and AA. Both mono- and dinucleotide insertions create open reading frames in mRNA and contribute to highly conserved structural features of rRNAs and tRNAs.	0
156	1028	29	The type of RNA editing found in the mitochondria of Myxomycota, characterized by the insertion of mono- and dinucleotides in RNAs relative to their mtDNA template and in addition, C to U base conversion. The most common mononucleotide insertion is cytidine, although a number of uridine mononucleotides are inserted at specific sites. Adenine and guanine have not been observed in mononucleotide insertions. Five different dinucleotide insertions have been observed, GC, GU, CU, AU and AA. Both mono- and dinucleotide insertions create open reading frames in mRNA and contribute to highly conserved structural features of rRNAs and tRNAs.	0
157	1048	29	A target region for site-specific inversion of a DNA region and which carries binding sites for a site-specific recombinase and accessory proteins as well as the site for specific cleavage by the recombinase.	0
158	1068	29	This has been obsoleted as it represents a process. replaced_by: GO:0034961.	0
159	1069	29	This has been obsoleted as it represents a process. replaced_by: GO:0070581.	0
160	1070	29	This has been obsoleted as it represents a process. replaced_by: GO:0070582	0
161	1071	29	This has been obsoleted as it represents a process. replaced_by: GO:0006260.	0
162	1072	29	This has been obsoleted as it represents a process. replaced_by: GO:0034961.	0
163	1076	29	Part of an edited transcript only.	0
164	1082	29	Attributes added to describe the different kinds of replicon. SO workshop, September 2006.	0
165	1083	29	Attributes added to describe the different kinds of replicon. SO workshop, September 2006.	0
166	1084	29	Attributes added to describe the different kinds of replicon. SO workshop, September 2006.	0
167	1085	29	Attributes added to describe the different kinds of replicon. SO workshop, September 2006.	0
168	1086	29	Attributes added to describe the different kinds of replicon. SO workshop, September 2006.	0
169	1087	29	Attributes added to describe the different kinds of replicon. SO workshop, September 2006.	0
170	1089	29	Requested by Karen Pilcher - Dictybase. song-Term Tracker-1574577.	0
171	1090	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
172	1091	29	Requested by Andy Schroder - Flybase Harvard, Nov 2006.	0
173	1092	29	Term added Dec 06 to comply with mapping to MGED terms.It should be used to generate consensus regions. The specific cross product terms they require are consensus_region and consensus_mRNA.	0
174	1093	29	DO not obsolete without considering MGED mapping.	0
175	1094	29	DO not obsolete without considering MGED mapping.	0
176	1095	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
177	1096	29	This term is mapped to MGED. Do not obsolete without consulting MGED ontology.	0
178	1098	29	Requested by Keith Boroevich December, 2006.	0
179	1102	29	Requested by Hadi Quesneville January 2007.	0
180	1106	29	This is not cryptic in the same sense as a cryptic gene or cryptic splice site.	0
181	1111	29	Added by request from Colin Batchelor.	0
182	1113	29	Requested by Colin Batchelor, Feb 2007.	0
183	1116	29	Added in March 2007 in after meeting with pharmgkb.	0
184	1117	29	Requested by Trish Whetzel.	0
185	1131	29	This terms is used by MO.	0
186	1132	29	This term is used by MO.	0
187	1133	29	Ruby et al. Nature 448:83 desribe a new class of miRNAs that are derived from debranched introns.	0
188	1140	29	The definitions of the children of this term were revised Decemeber 2007 after discussion on song-devel. The resulting definitions are slightly unweildy but hopefully more logically correct.	0
189	1147	29	A term created to allow the parts of an inversion site have an is_a path back to the root.	0
190	1149	29	Requested by Chris Smith, and others at Flybase to help annotate nested repeats.	0
191	1158	29	Merged with partially characterized change in nucleotide sequence.	0
192	1160	29	Discrete.	0
193	1161	29	Range.	0
194	1162	29	Range.	0
195	1163	29	Hormones, neuropeptides, antimicrobial peptides, are active peptides. They are typically short (<40 amino acids) in length.	0
196	1164	29	Range.	0
197	1165	29	Range.	0
198	1166	29	Range.	0
199	1167	29	Range.	0
200	1168	29	Range.	0
201	1169	29	Range.	0
202	1171	29	This could be inside an organelle within the cell.	0
203	1175	29	Biosapien term was secondary_structure.	0
204	1177	29	Range.	0
205	1179	29	Range.	0
206	1181	29	Discrete.	0
207	1182	29	Discrete.	0
208	1185	29	2 discreet & joined.	0
209	1186	29	Discrete.	0
210	1188	29	Discrete.	0
211	1189	29	Discrete.	0
212	1201	29	Discrete.	0
213	1208	29	Range.	0
214	1209	29	Range.	0
215	1210	29	Range.	0
216	1211	29	Range.	0
217	1214	29	Range.	0
218	1215	29	Range.	0
219	1216	29	Range.	0
220	1225	29	Range.	0
221	1243	29	For example, was a substitution natural or mutated as part of an experiment? This term is added to merge the biosapiens term sequence_variations.	0
222	1244	29	Discrete.	0
223	1245	29	Discrete.	0
224	1246	29	Discrete.	0
225	1253	29	This consensus sequence was identified computationally using the MEME algorithm within core promoter sequences from -60 to +40, with an E value of 1.7e-183.  Tends to co-occur with Motif 7. Tends to not occur with DPE motif (SO:0000015) or motif 10.	0
226	1276	29	The definition is most of the old definition for snoRNA (SO:0000275).	0
227	1278	29	Not to be confused with BRE_motif (SO:0000016), which binds transcription factor II B.	0
228	1280	29	Do not use this for feature annotation. Use morpholino_oligo (SO:0000034) instead.	0
229	1281	29	Do not use this term for feature annotation. Use PNA_oligo (SO:0001011) instead.	0
230	1282	29	Do not use this for feature annotation. Use enzymatic_RNA (SO:0000372) instead.	0
231	1283	29	Do not use this for feature annotation. Use ribozyme (SO:0000374) instead.	0
232	1284	29	Has RNA pseudouridylation guide activity (GO:0030558).	0
233	1285	29	Do not use this term for feature annotation. Use LNA_oligo (SO:0001189) instead.	0
234	1287	29	Do not use this term for feature annotation. Use TNA_oligo (SO:0001191) instead.	0
235	1289	29	Do not use this term for feature annotation. Use GNA_oligo (SO:0001192) instead.	0
236	1291	29	Do not use this term for feature annotation. Use R_GNA_oligo (SO:0001195) instead.	0
237	1293	29	Do not use this term for feature annotation. Use S_GNA_oligo (SO:0001197) instead.	0
238	1310	29	GO:0000374.	0
239	1311	29	An exon either containing but not starting with a start codon or containing but not ending with astop codon will be partially coding and partially non coding.	0
240	1312	29	An exon containing either a start or stop codon will be partially coding and partially non coding.	0
241	1317	29	RNA interference is GO:0016246.	0
242	1318	29	Histone modification is GO:0016570.	0
243	1319	29	Histone methylation is GO:0016571.	0
244	1320	29	Histone deacetylation is GO:0016573.	0
245	1326	29	The free molecule is CHEBI:17802.	0
246	1327	29	The free molecule is CHEBI:17596.	0
247	1328	29	The free molecule is CHEBI:2274.	0
248	1329	29	The free molecule is CHEBI:30832.	0
249	1334	29	Probably in the future this will cross reference to Chebi.	0
250	1354	29	Nature. 1986 Oct 16-22;323(6089):640-3.	0
251	1357	29	This feature was requested by nicole, tracker id 1911479. It is required to gather evidense together for annotation. An example would be overlapping ESTs that support an mRNA.	0
252	1514	29	Term requested by the MODencode group.	0
253	1518	29	MM Young, K Kirshenbaum, KA Dill & S Highsmith. Predicting conformational switches in proteins. Protein Science, 1999, 8, 1752-64. K. Kirshenbaum, M.M. Young and S. Highsmith. Predicting Allosteric Switches in Myosins. Protein Science 8(9):1806-1815. 1999.	0
254	1520	29	An example is a read produced by Roche 454 technology.	0
256	1522	29	An example is a read produced by Illumina technology.	0
257	1524	29	From tracker [ 2372385 ] expressed_sequence_assembly.	0
258	1530	29	Requested by the trypanosome community.	0
259	1531	29	A place holder for a cross product with chebi.	0
260	1532	29	A place holder for a cross product with chebi.	0
261	1533	29	A place holder for a cross product with chebi.	0
262	1534	29	A place holder for a cross product with chebi.	0
263	1535	29	A place holder for a cross product with chebi.	0
264	1536	29	A place holder for a cross product with chebi.	0
265	1537	29	A place holder for a cross product with chebi.	0
266	1538	29	A place holder for a cross product with chebi.	0
267	1539	29	A place holder for a cross product with chebi.	0
268	1540	29	A place holder for a cross product with chebi.	0
269	1541	29	A place holder for a cross product with chebi.	0
270	1542	29	A place holder for a cross product with chebi.	0
271	1543	29	A place holder for a cross product with chebi.	0
272	1544	29	A place holder for a cross product with chebi.	0
273	1545	29	A place holder for a cross product with chebi.	0
274	1546	29	A place holder for a cross product with chebi.	0
275	1547	29	A place holder for a cross product with chebi.	0
276	1548	29	A place holder for a cross product with chebi.	0
277	1549	29	A place holder for a cross product with chebi.	0
278	1550	29	A place holder for a cross product with chebi.	0
279	1551	29	A place holder for a cross product with chebi.	0
280	1552	29	A place holder for a cross product with chebi.	0
281	1553	29	This term was requested by Jeff Bowes, using the tracker, ID = 2594157.	0
282	1554	29	This term was requested by Jeff Bowes, using the tracker, ID = 2594157.	0
283	1556	29	See tracker ID 2060908.	0
284	1558	29	See tracker ID: 2138359.	0
285	1580	29	X element combinatorial repeats contain Tbf1p binding sites,nand possible functions include a role in telomerase-independent telomerenmaintenance via recombination or as a barrier against transcriptionalnsilencing. These are usually present as a combination of one or more ofnseveral types of smaller elements (designated A, B, C, or D).	0
286	1592	29	The repeats are maintained by telomerase and there is generally 300 (+/-) 75 bp of TG(1-3) at a given end. Telomeric repeats function in completing chromosome replication and protecting the ends from degradation and end-to-end fusions.	0
287	1593	29	Possible functions include roles in chromosomal segregation,nmaintenance of chromosome stability, recombinational sequestering, or as anbarrier to transcriptional silencing.	0
288	1595	29	This terms and children were added to SO in resonse to tracker request by patrick chain. The pape:r Genome Project Standards in a New Era of Sequencing. Science October 9th 2009, addresses these terms.	0
289	1597	29	Term requested via tracker  ID: 2910829.	0
290	1598	29	Requested by tracker ID: 2902685.	0
291	1599	29	Ensembl and Vega also use this term name. Requested by Howard Deen of MGI.	0
292	1634	29	Has RNA 2'-O-ribose methylation guide activity (GO:0030561).	0
293	1641	29	This would include, for example, a cluster of genes each encoding the major ribosomal RNAs and a cluster of histone gene subarrays.	0
294	1642	29	This would include, for example, a cluster of genes encoding different histones.	0
295	1643	29	This would include, for example, the mating type gene cassettes of S. cerevisiae. Gene cassettes usually exist as linear sequences as part of a larger DNA molecule, such as a chromosome or plasmid.	0
296	1644	29	This would include, for example, the arrays of non-functional VSG genes of Trypanosomes.	0
297	1649	29	Range.	0
298	1650	29	Range.	0
299	1659	29	Range.	0
300	1661	29	Range.	0
301	1696	29	Indels can have a different number of bases than the corresponding reference sequence.	0
302	1720	29	The exact rules need to be stated, a common set of rules can be derived from e.g. BLOSUM62 amino acid distance matrix.	0
303	1721	29	The exact rules need to be stated, a common set of rules can be derived from e.g. BLOSUM62 amino acid distance matrix.	0
304	1734	29	A cryptic splice site is only used when the natural splice site has been disrupted by a sequence alteration.	0
305	1747	29	The nature of the mutation event is only partially characterised.	0
306	1783	29	Updated after discussion with Peter Taschner - Feb 09.	0
307	1807	29	Flag - unknown in the definition.	0
308	1808	29	FLAG - term describes an unknown.	0
309	1809	29	FLAG - definition describes an unknown.	0
310	1853	29	Intron characteristic of tRNA genes; splices by an endonuclease-ligase mediated mechanism.	0
311	1854	29	Could be a cross product with Gene ontology, GO:0006388.	0
312	1868	29	This term does not include the stop codons that are redefined. An example would be a stop codon that partially overlapped a frame shifting site would be an example stimulatory signal.	0
313	1870	29	Requested by tracker 2021594, July 2008, by Alex.	0
314	2028	29	Rem = absorbed dose (rad) x quality factor (Q). Q is unique to the type of incident radiation.	0
315	2040	29	Typically applied to the half life of radioactive atoms but also applicable to any other situation where the population is of molecules of diminishing concentration or activity.	0
316	2068	29	For units and further information look under the mass density unit. For a sample of a specific pure elemental substance, the concentration is directly proportional to the density. However, concentration is not proportional to density in general.	0
317	2081	29	An example of a biomaterial could be an e.g. tumor biopsy.	0
318	2083	29	In this scale, the freezing point of water is 32 degrees Fahrenheit and the boiling point is 212 degrees, placing the boiling and freezing points of water 180 degrees apart. -40 degrees Fahrenheit is equal to -40 degrees Celsius.	0
319	2161	78	1	0
320	2161	34	1	0
\.


--
-- Data for Name: cvtermsynonym; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY cvtermsynonym (cvtermsynonym_id, cvterm_id, synonym, type_id) FROM stdin;
1	31	is_subtype_of	35
2	49	derived_from	35
3	102	sequence	35
4	103	sequence secondary structure	35
5	104	G quartet	35
6	104	G tetrad	35
7	104	G-quadruplex	35
8	104	G-quartet	35
9	104	G-tetrad	35
10	104	G_quadruplex	35
11	104	guanine tetrad	35
12	105	interior coding exon	35
13	106	satellite DNA	35
14	107	amplicon	108
15	107	PCR product	35
16	109	read-pair	35
17	112	protein-coding	35
18	113	non protein-coding	35
19	114	scRNA primary transcript	35
20	114	scRNA transcript	35
21	114	small cytoplasmic RNA	108
22	114	small cytoplasmic RNA transcript	35
23	114	small_cytoplasmic_RNA	108
24	115	small cytoplasmic RNA	35
25	116	DMp2	108
26	116	initiator	35
27	116	initiator motif	35
28	116	INR motif	35
29	117	CRWMGCGWKCGCTTS	118
30	117	downstream core promoter element	35
31	117	DPE motif	35
32	119	B-recognition element	35
33	119	BRE motif	35
34	119	TFIIB recognition element	108
35	119	transcription factor B-recognition element	35
36	120	proximal sequence element	35
37	120	PSE motif	35
38	121	linkage group	35
39	122	RNA internal loop	35
40	123	asymmetric RNA internal loop	35
41	124	A minor RNA motif	35
42	125	K turn RNA motif	35
43	125	K-turn	35
44	125	kink turn	35
45	125	kink-turn motif	35
46	126	sarcin like RNA motif	35
47	126	sarcin/ricin domain	35
48	126	sarcin/ricin loop	35
49	126	sarcin/ricin RNA domain	35
50	127	A-minor RNA motif	35
51	128	RNA junction loop	35
52	129	hook turn	108
53	129	hook-turn motif	35
54	129	RNA hook turn	35
55	130	base pair	35
56	131	canonical base pair	35
57	131	Watson Crick base pair	35
58	131	Watson-Crick base pair	108
59	131	Watson-Crick pair	35
60	131	WC base pair	35
61	132	sugar edge base pair	35
62	134	DNA aptamer	35
63	135	RNA aptamer	35
64	136	morpholino oligo	35
65	137	riboswitch RNA	35
66	138	MAR	35
67	138	matrix association region	35
68	138	matrix attachment region	35
69	138	matrix attachment site	35
70	138	nuclear matrix association region	35
71	138	nuclear matrix attachment site	35
72	138	S/MAR	35
73	138	S/MAR element	108
74	138	scaffold attachment site	35
75	138	scaffold matrix attachment region	35
76	138	SMAR	35
77	139	LCR	35
78	139	locus control element	108
79	139	locus control region	35
80	141	match part	35
81	142	genomic clone	35
82	143	sequence operation	35
83	144	pseudogene attribute	35
84	145	processed pseudogene	35
85	145	pseudogene by reverse transcription	108
86	145	R psi G	108
87	145	retropseudogene	35
88	146	pseudogene by unequal crossing over	35
89	154	assortment-derived_deficiency	108
90	155	mutation affecting regulatory region	108
91	155	sequence variant affecting regulatory region	35
92	159	operator segment	35
93	160	assortment-derived_aneuploid	108
94	161	nuclease binding site	35
95	162	compound chromosome arm	35
96	163	restriction endonuclease binding site	35
97	163	restriction endonuclease recognition site	108
98	163	restriction enzyme binding site	35
99	163	restriction enzyme recognition site	108
100	164	deficient intrachromosomal transposition	35
101	165	deficient interchromosomal transposition	35
102	167	free chromosome arm	35
103	169	gene to gene feature	35
104	171	inside intron	35
105	172	inside intron antiparallel	35
106	173	inside intron parallel	35
107	175	five prime-three prime overlap	35
108	176	five prime-five prime overlap	35
109	177	three prime-three prime overlap	35
110	178	5' 3' overlap	35
111	178	three prime five prime overlap	35
112	180	polycistronic transcript	35
113	181	dicistronic transcript	35
114	182	operon member	35
115	183	gene array member	35
116	185	macronuclear sequence	35
117	186	micronuclear sequence	35
118	189	nuclear gene	35
119	190	mitochondrial gene	35
120	190	mt gene	35
121	191	kinetoplast gene	35
122	192	plastid gene	35
123	193	apicoplast gene	35
124	194	chloroplast gene	35
125	194	ct gene	35
126	195	chromoplast gene	35
127	196	cyanelle gene	35
128	197	leucoplast gene	35
129	198	proplastid gene	35
130	199	nucleomorph gene	35
131	200	plasmid gene	35
132	201	proviral gene	35
133	202	endogenous retroviral gene	35
134	203	transposable element	35
135	203	transposon	35
136	204	expressed sequence match	35
137	205	clone insert end	35
138	206	protein	35
139	207	chromosome arm	35
140	209	sequencing primer	35
141	210	frameshifted mRNA	35
142	210	mRNA with frameshift	35
143	211	mutation	108
144	212	located sequence feature	108
145	212	located_sequence_feature	35
146	212	sequence feature	35
147	213	transposable element gene	35
148	214	DNA primer	35
149	214	primer oligonucleotide	35
150	214	primer polynucleotide	35
151	214	primer sequence	35
152	215	proviral region	35
153	215	proviral sequence	108
154	216	methylated C	35
155	216	methylated cytosine	35
156	216	methylated cytosine base	35
157	216	methylated cytosine residue	35
158	220	transcript with translational frameshift	35
159	222	pre mRNA	108
160	222	protein coding primary transcript	35
161	223	DNA forward primer	35
162	223	forward DNA primer	35
163	223	forward primer	35
164	223	forward primer oligo	35
165	223	forward primer oligonucleotide	35
166	223	forward primer polynucleotide	35
167	223	forward primer sequence	35
168	224	RNA sequence secondary structure	35
169	225	transcriptionally regulated	35
170	226	transcriptionally constitutive	35
171	227	transcriptionally induced	35
172	228	transcriptionally repressed	35
173	229	silenced gene	35
174	230	gene silenced by DNA modification	35
175	231	gene silenced by DNA methylation	35
176	231	methylation-silenced gene	35
177	232	post translationally regulated	35
178	232	post-translationally regulated	35
179	233	translationally regulated	35
180	234	DNA reverse primer	35
181	234	reverse DNA primer	35
182	234	reverse primer	35
183	234	reverse primer oligo	35
184	234	reverse primer oligonucleotide	35
185	234	reverse primer sequence	35
186	235	epigenetically modified	35
187	237	maternally imprinted	35
188	238	paternally imprinted	35
189	239	allelically excluded	35
190	240	gene rearranged at DNA level	35
191	241	ribosome entry site	35
192	242	attenuator sequence	35
193	243	terminator sequence	35
194	244	DNA sequence secondary structure	35
195	245	assembly component	35
196	247	recoded codon	35
197	250	scaffold	108
198	254	yeast artificial chromosome	35
199	255	bacterial artificial chromosome	35
200	256	P1	35
201	256	P1 artificial chromosome	35
202	257	plasmid sequence	35
203	258	cosmid vector	35
204	259	phagemid vector	108
205	260	fosmid vector	108
206	261	deleted_sequence	35
207	261	nucleotide deletion	35
208	261	nucleotide_deletion	35
209	263	methylated A	35
210	263	methylated adenine	35
211	263	methylated adenine base	35
212	263	methylated adenine residue	35
213	264	splice site	35
214	265	5' splice site	35
215	265	donor	108
216	265	donor splice site	35
217	265	five prime splice site	35
218	265	splice donor site	35
219	266	3' splice site	108
220	266	acceptor	108
221	266	acceptor splice site	35
222	266	splice acceptor site	35
223	266	three prime splice site	35
224	268	enhancer bound by factor	35
225	269	promoter sequence	35
226	271	pol I promoter	35
227	271	polymerase I promoter	35
228	271	RNA polymerase A promoter	35
229	271	RNApol I promoter	35
230	272	pol II promoter	108
231	272	polymerase II promoter	35
232	272	RNA polymerase B promoter	35
233	272	RNApol II promoter	35
234	273	pol III promoter	35
235	273	polymerase III promoter	35
236	273	RNA polymerase C promoter	35
237	273	RNApol III promoter	35
238	274	CAAT box	35
239	274	CAAT signal	35
240	274	CAAT-box	35
241	275	GC rich promoter region	35
242	275	GC-rich region	35
243	276	Goldstein-Hogness box	35
244	276	TATA box	35
245	277	-10 signal	35
246	277	minus 10 signal	35
247	277	Pribnow box	35
248	277	Pribnow Schaller box	35
249	277	Pribnow-Schaller box	35
250	278	-35 signal	35
251	278	minus 35 signal	35
252	279	cross genome match	35
253	281	clone insert start	35
254	282	class I	108
255	282	class I transposon	35
256	282	retrotransposon element	35
257	283	translated nucleotide match	35
258	284	class II	108
259	284	class II transposon	35
260	284	DNA transposon	35
261	285	non transcribed region	35
262	285	non-transcribed sequence	35
263	285	nontranscribed region	35
264	285	nontranscribed sequence	35
265	286	U2 intron	35
266	287	precursor RNA	35
267	287	primary transcript	35
268	288	long terminal repeat retrotransposon	35
269	288	LTR retrotransposon	35
270	291	non LTR retrotransposon	35
271	292	5' intron	35
272	292	5' intron sequence	35
273	292	five prime intron	35
274	293	interior intron	35
275	294	3' intron	35
276	294	3' intron sequence	108
277	294	three prime intron	35
278	295	restriction fragment length polymorphism	35
279	295	RFLP	35
280	295	RFLP fragment	35
281	296	LINE	35
282	296	LINE element	35
283	296	Long interspersed element	35
284	296	Long interspersed nuclear element	35
285	297	coding exon	35
286	298	five prime exon coding region	35
287	299	three prime exon coding region	35
288	300	noncoding exon	35
289	301	translocated sequence	35
290	302	5' coding exon	35
291	302	five prime coding exon	35
292	303	interior exon	35
293	304	3' coding exon	108
294	304	three prime coding exon	35
295	305	untranslated region	35
296	306	5' UTR	35
297	306	five prime UTR	35
298	306	five_prime_untranslated_region	35
299	307	three prime untranslated region	35
300	307	three prime UTR	35
301	308	Short interspersed element	35
302	308	Short interspersed nuclear element	35
303	308	SINE element	35
304	309	simple sequence length polymorphism	108
305	309	simple sequence length variation	35
306	309	SSLP	108
307	310	terminal inverted repeat element	35
308	310	TIR element	35
309	311	ribosomal RNA primary transcript	35
310	311	rRNA primary transcript	35
311	312	tRNA primary transcript	35
312	313	alanine tRNA primary transcript	35
313	314	arginine tRNA primary transcript	35
314	315	asparagine tRNA primary transcript	35
315	316	aspartic acid tRNA primary transcript	35
316	317	cysteine tRNA primary transcript	35
317	318	glutamic acid tRNA primary transcript	35
318	319	glutamine tRNA primary transcript	35
319	320	glycine tRNA primary transcript	35
320	321	histidine tRNA primary transcript	35
321	322	isoleucine tRNA primary transcript	35
322	323	leucine tRNA primary transcript	35
323	324	lysine tRNA primary transcript	35
324	325	methionine tRNA primary transcript	35
325	326	phenylalanine tRNA primary transcript	35
326	327	proline tRNA primary transcript	35
327	328	serine tRNA primary transcript	35
328	329	threonine tRNA primary transcript	35
329	330	tryptophan tRNA primary transcript	35
330	331	tyrosine tRNA primary transcript	35
331	332	valine tRNA primary transcript	35
332	333	snRNA primary transcript	35
333	334	snoRNA primary transcript	35
334	335	mature transcript	35
335	336	messenger RNA	35
336	337	TF binding site	35
337	337	transcription factor binding site	35
338	338	open reading frame	35
339	339	transcript attribute	35
340	340	foldback element	35
341	340	long inverted repeat element	108
342	340	LVR element	108
343	341	flanking region	35
344	342	chromosome variation	35
345	343	internal UTR	35
346	344	untranslated region polycistronic mRNA	35
347	345	internal ribosomal entry sequence	35
348	345	internal ribosomal entry site	35
349	345	internal ribosome entry sequence	108
350	345	internal ribosome entry site	35
351	345	IRES	35
352	346	4-cutter_restriction_site	108
353	346	four-cutter_restriction_sit	108
354	350	sequence length variation	35
355	351	6-cutter_restriction_site	108
356	351	six-cutter_restriction_site	108
357	352	modified RNA base feature	35
358	353	8-cutter_restriction_site	108
359	353	eight-cutter_restriction_site	108
360	354	ribosomal ribonucleic acid	35
361	354	ribosomal RNA	35
362	355	transfer ribonucleic acid	108
363	355	transfer RNA	108
364	356	alanyl tRNA	35
365	356	alanyl-transfer ribonucleic acid	35
366	356	alanyl-transfer RNA	35
367	357	rRNA small subunit primary transcript	35
368	358	asparaginyl tRNA	35
369	358	asparaginyl-transfer ribonucleic acid	35
370	358	asparaginyl-transfer RNA	35
371	359	aspartyl tRNA	35
372	359	aspartyl-transfer ribonucleic acid	35
373	359	aspartyl-transfer RNA	35
374	360	cysteinyl tRNA	35
375	360	cysteinyl-transfer ribonucleic acid	35
376	360	cysteinyl-transfer RNA	35
377	361	glutaminyl tRNA	35
378	361	glutaminyl-transfer ribonucleic acid	35
379	361	glutaminyl-transfer RNA	35
380	362	glutamyl tRNA	35
381	362	glutamyl-transfer ribonucleic acid	35
382	362	glutamyl-transfer RNA	108
383	363	glycyl tRNA	35
384	363	glycyl-transfer ribonucleic acid	108
385	363	glycyl-transfer RNA	108
386	364	histidyl tRNA	35
387	364	histidyl-transfer ribonucleic acid	35
388	364	histidyl-transfer RNA	35
389	365	isoleucyl tRNA	35
390	365	isoleucyl-transfer ribonucleic acid	35
391	365	isoleucyl-transfer RNA	35
392	366	leucyl tRNA	35
393	366	leucyl-transfer ribonucleic acid	35
394	366	leucyl-transfer RNA	35
395	367	lysyl tRNA	35
396	367	lysyl-transfer ribonucleic acid	35
397	367	lysyl-transfer RNA	35
398	368	methionyl tRNA	35
399	368	methionyl-transfer ribonucleic acid	35
400	368	methionyl-transfer RNA	35
401	369	phenylalanyl tRNA	35
402	369	phenylalanyl-transfer ribonucleic acid	35
403	369	phenylalanyl-transfer RNA	35
404	370	prolyl tRNA	35
405	370	prolyl-transfer ribonucleic acid	35
406	370	prolyl-transfer RNA	35
407	371	seryl tRNA	35
408	371	seryl-transfer ribonucleic acid	108
409	371	seryl-transfer RNA	35
410	372	threonyl tRNA	35
411	372	threonyl-transfer ribonucleic acid	35
412	372	threonyl-transfer RNA	108
413	373	tryptophanyl tRNA	35
414	373	tryptophanyl-transfer ribonucleic acid	35
415	373	tryptophanyl-transfer RNA	35
416	374	tyrosyl tRNA	35
417	374	tyrosyl-transfer ribonucleic acid	35
418	374	tyrosyl-transfer RNA	108
419	375	valyl tRNA	35
420	375	valyl-transfer ribonucleic acid	35
421	375	valyl-transfer RNA	108
422	376	small nuclear RNA	35
423	377	small nucleolar RNA	35
424	378	micro RNA	35
425	378	microRNA	35
426	379	bound by factor	35
427	380	transcript bound by nucleic acid	35
428	381	transcript bound by protein	35
429	382	engineered gene	35
430	383	engineered foreign gene	35
431	384	mRNA with minus 1 frameshift	35
432	385	engineered foreign transposable element gene	35
433	387	foreign gene	35
434	388	direct terminal repeat	108
435	388	long terminal repeat	35
436	388	LTR	35
437	389	fusion gene	35
438	390	engineered fusion gene	35
439	391	microsatellite locus	35
440	391	microsatellite marker	35
441	391	VNTR	35
442	392	dinucleotide repeat microsatellite	35
443	392	dinucleotide repeat microsatellite feature	35
444	392	dinucleotide repeat microsatellite locus	35
445	392	dinucleotide repeat microsatellite marker	35
446	393	dinucleotide repeat microsatellite marker	108
447	393	rinucleotide repeat microsatellite	35
448	393	trinucleotide repeat microsatellite feature	35
449	393	trinucleotide repeat microsatellite locus	35
450	395	engineered foreign repetitive element	35
451	396	inverted repeat	35
452	396	inverted repeat sequence	35
453	397	U12 intron	35
454	397	U12-dependent intron	35
455	398	ori	35
456	398	origin of replication	35
457	399	D-loop	35
458	399	displacement loop	108
459	400	recombination feature	35
460	401	specific recombination site	35
461	402	recombination feature of rearranged gene	35
462	403	vertebrate immune system gene recombination feature	35
463	404	J gene recombination feature	35
464	404	J-RS	35
465	407	modified base site	35
466	408	methylated base feature	35
467	409	CG island	35
468	409	CpG island	35
469	414	experimentally determined	35
470	415	RNA_hairpin_loop	35
471	415	stem loop	35
472	415	stem-loop	35
473	416	direct repeat	35
474	417	transcription start site	35
475	418	coding sequence	35
476	419	cDNA clone	35
477	420	initiation codon	35
478	420	start codon	35
479	421	stop codon	35
480	422	intronic splice enhancer	35
481	423	mRNA with plus 1 frameshift	35
482	424	nuclease hypersensitive site	35
483	425	coding start	35
484	425	translation initiation site	35
485	425	translation start	108
486	427	rRNA large subunit primary transcript	35
487	428	SAGE tag	35
488	429	coding end	35
489	429	translation termination site	35
490	429	translation_end	35
491	430	microarray oligo	35
492	430	microarray oligonucleotide	35
493	431	mRNA with plus 2 frameshift	35
494	432	conserved region	35
495	433	sequence tag site	35
496	434	coding conserved region	35
497	435	exon junction	35
498	436	nc conserved region	35
499	436	noncoding conserved region	35
500	437	mRNA with minus 2 frameshift	35
501	439	RNAi reagent	35
502	440	miniature inverted repeat transposable element	35
503	441	recombination hotspot	35
504	443	chromosome band	35
505	443	cytoband	35
506	443	cytological band	35
507	444	site specific recombination target region	35
508	446	splice enhancer	35
509	447	expressed sequence tag	35
510	448	Cre-recombination target region	108
511	448	loxP site	35
512	449	nucleotide match	35
513	450	nucleic acid	35
514	451	protein match	35
515	452	FLP recombination target region	35
516	452	FRT site	35
517	453	synthetic sequence	35
518	455	sequence assembly	35
519	456	group 1 intron homing endonuclease target region	35
520	457	haplotype block	35
521	462	FRT flanked	35
522	463	invalidated by chimeric cDNA	35
523	464	floxed gene	35
524	465	transposable element flanking region	35
525	467	insertion site	35
526	468	attI site	35
527	469	transposable element insertion site	35
528	471	small regulatory ncRNA	35
529	472	conjugative transposon	35
530	473	enzymatic RNA	35
531	474	recombinationally inverted gene	35
532	476	5.8S LSU rRNA	35
533	476	5.8S ribosomal RNA	35
534	476	5.8S rRNA	35
535	476	rRNA 5 8S	35
536	477	6S RNA	35
537	477	RNA 6S	35
538	478	CsrB RsmB RNA	35
539	478	CsrB-RsmB RNA	35
540	479	DsrA RNA	35
541	480	GcvB RNA	35
542	481	hammerhead ribozyme	35
543	482	group IIA intron	35
544	483	group IIB intron	35
545	484	MicF RNA	35
546	485	OxyS RNA	35
547	486	RNase MRP RNA	35
548	487	RNase P RNA	35
549	488	RprA RNA	35
550	489	RRE RNA	35
551	490	spot-42 RNA	35
552	491	telomerase RNA	35
553	492	small nuclear RNA U1	35
554	492	snRNA U1	35
555	492	U1 small nuclear RNA	35
556	492	U1 snRNA	35
557	493	small nuclear RNA U2	35
558	493	snRNA U2	35
559	493	U2 small nuclear RNA	35
560	493	U2 snRNA	35
561	494	small nuclear RNA U4	35
562	494	snRNA U4	35
563	494	U4 small nuclear RNA	35
564	494	U4 snRNA	35
565	495	small nuclear RNA U4atac	35
566	495	snRNA U4atac	35
567	495	U4atac small nuclear RNA	35
568	495	U4atac snRNA	35
569	496	small nuclear RNA U5	35
570	496	snRNA U5	35
571	496	U5 small nuclear RNA	35
572	496	U5 snRNA	35
573	497	small nuclear RNA U6	35
574	497	snRNA U6	35
575	497	U6 small nuclear RNA	35
576	497	U6 snRNA	35
577	498	snRNA U6atac	35
578	498	U6atac small nuclear RNA	35
579	498	U6atac snRNA	35
580	499	small nuclear RNA U11	35
581	499	snRNA U11	35
582	499	U11 small nuclear RNA	35
583	499	U11 snRNA	35
584	500	small nuclear RNA U12	35
585	500	snRNA U12	35
586	500	U12 small nuclear RNA	35
587	500	U12 snRNA	35
588	501	sequence attribute	35
589	502	gene attribute	35
590	504	small nucleolar RNA U14	35
591	504	snoRNA U14	35
592	504	U14 small nucleolar RNA	35
593	504	U14 snoRNA	35
594	505	vault RNA	35
595	506	Y RNA	35
596	508	18S ribosomal RNA	35
597	508	18S rRNA	35
598	508	rRNA 18S	35
599	510	binding_or_interaction_site	35
600	510	site	108
601	511	protein binding site	35
602	512	rescue fragment	35
603	512	rescue region	35
604	512	rescue segment	108
605	513	restriction fragment	35
606	514	sequence difference	35
607	515	invalidated by genomic contamination	35
608	516	invalidated by genomic polyA primed cDNA	35
609	517	invalidated by partial processing	35
610	518	ca_bind	108
611	518	DNA_bind	108
612	518	domain	519
613	518	np_bind	108
614	518	polypeptide domain	35
615	518	polypeptide_structural_domain	35
616	518	structural domain	519
617	518	zn_fing	108
618	520	signal	108
619	520	signal peptide	35
620	520	signal peptide coding sequence	35
621	521	chain	108
622	521	mature peptide	108
623	521	mature protein region	35
624	522	5' TIR	35
625	522	five prime terminal inverted repeat	35
626	523	3' TIR	35
627	523	three prime terminal inverted repeat	35
628	524	U5 long terminal repeat region	35
629	524	U5 LTR region	35
630	525	R long terminal repeat region	35
631	525	R LTR region	35
632	526	U3 long terminal repeat region	35
633	526	U3 LTR region	35
634	527	5' long terminal repeat	35
635	527	5' LTR	35
636	527	five prime LTR	35
637	528	3' long terminal repeat	35
638	528	3' LTR	35
639	528	three prime LTR	35
640	529	R 5' long term repeat region	35
641	529	R five prime LTR region	35
642	530	U5 5' long terminal repeat region	35
643	530	U5 five prime LTR region	35
644	531	U3 5' long term repeat region	35
645	531	U3 five prime LTR region	35
646	532	R 3' long terminal repeat region	35
647	532	R three prime LTR region	35
648	533	U3 3' long terminal repeat region	35
649	533	U3 three prime LTR region	35
650	534	U5 3' long terminal repeat region	35
651	534	U5 three prime LTR region	35
652	535	non LTR retrotransposon polymeric tract	35
653	536	target site duplication	35
654	537	LTR retrotransposon poly purine tract	108
655	537	RR tract	35
656	538	autonomously replicating sequence	35
657	541	inverted ring chromosome	35
658	542	vector	35
659	542	vector replicon	35
660	543	single strand oligo	35
661	543	single strand oligonucleotide	35
662	543	single stranded oligonucleotide	35
663	543	ss oligo	35
664	543	ss oligonucleotide	35
665	544	double stranded oligonucleotide	35
666	544	ds oligo	35
667	544	ds-oligonucleotide	35
668	545	polymer attribute	35
669	546	three prime noncoding exon	35
670	547	5' nc exon	35
671	547	5' non coding exon	35
672	547	five prime noncoding exon	35
673	548	UTR intron	35
674	549	five prime UTR intron	35
675	550	three prime UTR intron	35
676	551	random sequence	35
677	552	chromosome interband	108
678	553	gene with polyadenylated mRNA	35
679	555	chromosomal transposition	35
680	555	transposition	118
681	556	repeat associated small interfering RNA	35
682	557	gene with mRNA with frameshift	35
683	558	recombinationally rearranged gene	35
684	559	interchromosomal duplication	35
685	560	D gene	35
686	560	D-GENE	35
687	561	gene with trans spliced transcript	35
688	562	vertebrate immunoglobulin T cell receptor segment	35
689	562	vertebrate_immunoglobulin/T-cell receptor gene	35
690	563	inversion derived bipartite deficiency	35
691	564	pseudogenic region	35
692	565	encodes alternately spliced transcripts	35
693	566	decayed exon	35
694	567	inversion derived deficiency plus duplication	35
695	568	V gene	35
696	568	V-GENE	35
697	568	variable_gene	35
698	569	post translationally regulated by protein stability	35
699	569	post-translationally regulated by protein stability	35
700	570	golden path fragment	35
701	571	post translationally regulated by protein modification	35
702	571	post-translationally regulated by protein modification	35
703	572	J gene	35
704	572	J-GENE	35
705	574	tiling path	35
706	575	negatively autoregulated	35
707	576	tiling path fragment	35
708	577	positively autoregulated	35
709	578	contig read	35
710	580	C gene	35
711	580	C_GENE	35
712	580	constant gene	35
713	581	trans spliced transcript	35
714	581	trans-spliced transcript	35
715	582	tiling path clone	35
716	583	terminal inverted repeat	35
717	583	TIR	35
718	584	vertebrate immunoglobulin T cell receptor gene cluster	35
719	584	vertebrate_immunoglobulin/T-cell receptor gene cluster	35
720	585	nc primary transcript	35
721	585	noncoding primary transcript	35
722	586	three prime coding exon noncoding region	35
723	586	three_prime_exon_noncoding_region	35
724	587	(DJ)-J-CLUSTER	35
725	587	DJ J cluster	35
726	588	five prime coding exon noncoding region	35
727	588	five_prime_exon_noncoding_region	35
728	589	(VDJ)-J-C-CLUSTER	35
729	589	VDJ J C cluster	35
730	590	(VDJ)-J-CLUSTER	35
731	590	VDJ J cluster	35
732	591	(VJ)-C-CLUSTER	108
733	591	VJ C cluster	35
734	592	(VJ)-J-C-CLUSTER	35
735	592	VJ J C cluster	35
736	593	(VJ)-J-CLUSTER	35
737	593	VJ J cluster	35
738	594	D gene recombination feature	35
739	595	3'D-HEPTAMER	35
740	595	three prime D heptamer	35
741	596	3'D-NOMAMER	35
742	596	three prime D nonamer	35
743	597	3'D-SPACER	35
744	597	three prime D spacer	35
745	598	5'D-HEPTAMER	35
746	598	five prime D heptamer	35
747	599	5'D-NONAMER	35
748	599	five prime D nonamer	35
749	600	5'-SPACER	35
750	600	five prime D spacer	35
751	600	five prime D-spacer	35
752	601	virtual sequence	35
753	602	Hoogsteen base pair	35
754	603	reverse Hoogsteen base pair	35
755	606	D DJ C cluster	35
756	606	D-(DJ)-C-CLUSTER	35
757	607	D DJ cluster	35
758	607	D-(DJ)-CLUSTER	35
759	608	D DJ J C cluster	35
760	608	D-(DJ)-J-C-CLUSTER	35
761	609	pseudogenic exon	35
762	610	D DJ J cluster	35
763	610	D-(DJ)-J-CLUSTER	35
764	611	D J C cluster	35
765	611	D-J-C-CLUSTER	35
766	612	V_D_GENE	35
767	612	VD gene	35
768	613	J C cluster	35
769	613	J-C-CLUSTER	35
770	614	inversion derived deficiency plus aneuploid	35
771	615	J cluster	35
772	615	J-CLUSTER	35
773	616	J nonamer	35
774	616	J-NONAMER	35
775	617	J heptamer	35
776	617	J-HEPTAMER	35
777	618	pseudogenic transcript	35
778	619	J spacer	35
779	619	J-SPACER	35
780	620	V DJ cluster	35
781	620	V-(DJ)-CLUSTER	35
782	621	V DJ J cluster	35
783	621	V-(DJ)-J-CLUSTER	108
784	622	V VDJ C cluster	35
785	622	V-(VDJ)-C-CLUSTER	35
786	623	V VDJ cluster	35
787	623	V-(VDJ)-CLUSTER	35
788	624	V VDJ J cluster	35
789	624	V-(VDJ)-J-CLUSTER	108
790	625	V VJ C cluster	35
791	625	V-(VJ)-C-CLUSTER	35
792	626	V VJ cluster	35
793	626	V-(VJ)-CLUSTER	35
794	627	V VJ J cluster	35
795	627	V-(VJ)-J-CLUSTER	35
796	628	V cluster	35
797	628	V-CLUSTER	35
798	629	V D DJ C cluster	35
799	629	V-D-(DJ)-C-CLUSTER	35
800	630	V D DJ cluster	35
801	630	V-D-(DJ)-CLUSTER	35
802	631	V D DJ J C cluster	35
803	631	V-D-(DJ)-J-C-CLUSTER	35
804	632	V D DJ J cluster	35
805	632	V-D-(DJ)-J-CLUSTER	35
806	633	V D J C cluster	35
807	633	V-D-J-C-CLUSTER	35
808	634	V D J cluster	35
809	634	V-D-J-CLUSTER	35
810	635	V heptamer	35
811	635	V-HEPTAMER	35
812	636	V J cluster	35
813	636	V-J-CLUSTER	35
814	637	V J C cluster	35
815	637	V-J-C-CLUSTER	35
816	638	V nonamer	35
817	638	V-NONAMER	35
818	639	V spacer	35
819	639	V-SPACER	35
820	640	V gene recombination feature	35
821	640	V-RS	35
822	641	(DJ)-C-CLUSTER	35
823	641	DJ C cluster	35
824	642	(DJ)-J-C-CLUSTER	35
825	642	DJ J C cluster	35
826	643	(VDJ)-C-CLUSTER	35
827	643	VDJ C cluster	35
828	644	V DJ C cluster	35
829	644	V-(DJ)-C-CLUSTER	35
830	646	ISCR	108
831	647	recoding pseudoknot	35
832	648	designed sequence	35
833	649	inversion derived bipartite duplication	35
834	650	gene with edited transcript	35
835	651	inversion derived duplication plus aneuploid	35
836	652	aneuploid chromosome	35
837	653	poly(A) signal	35
838	653	polyA signal sequence	35
839	653	polyadenylation termination signal	35
840	654	five prime ribosome binding site	35
841	654	RBS	108
842	654	Shine Dalgarno sequence	35
843	654	Shine-Dalgarno sequence	35
844	655	polyA cleavage site	35
845	655	polyA site	35
846	655	polyadenylation site	108
847	657	5' clip	108
848	657	five prime clip	35
849	658	5'RS	35
850	658	five prime D recombination signal sequence	35
851	658	five prime D-recombination signal sequence	35
852	659	3'-clip	35
853	659	three prime clip	35
854	660	C cluster	35
855	660	C-CLUSTER	35
856	661	D cluster	35
857	661	D-CLUSTER	35
858	662	D J cluster	35
859	662	D-J-CLUSTER	35
860	663	HEPTAMER	108
861	663	heptamer of recombination feature of vertebrate immune system gene	35
862	664	nonamer of recombination feature of vertebrate immune system gene	35
863	665	vertebrate immune system gene recombination spacer	35
864	666	V DJ J C cluster	35
865	666	V-(DJ)-J-C-CLUSTER	35
866	667	V VDJ J C cluster	35
867	667	V-(VDJ)-J-C-CLUSTER	35
868	668	V VJ J C cluster	35
869	668	V-(VJ)-J-C-CLUSTER	35
870	669	inversion derived aneuploid chromosome	35
871	670	bidirectional promoter	35
872	672	3'D-RS	35
873	672	three prime D recombination signal sequence	35
874	672	three_prime_D-recombination_signal_sequence	35
875	673	miRNA encoding	35
876	674	D-J-GENE	35
877	674	DJ gene	35
878	675	rRNA encoding	35
879	676	V-D-J-GENE	35
880	676	VDJ gene	35
881	677	scRNA encoding	35
882	678	V-J-GENE	35
883	678	VJ gene	35
884	680	snoRNA encoding	35
885	681	edited transcript feature	35
886	682	methylation guide snoRNA primary transcript	35
887	684	rRNA cleavage snoRNA primary transcript	35
888	685	pre edited region	35
889	685	pre-edited region	35
890	686	10Sa RNA	108
891	686	ssrA	108
892	687	C/D box snoRNA encoding	35
893	688	10Sa RNA primary transcript	108
894	688	ssrA RNA primary transcript	108
895	688	tmRNA primary transcript	35
896	689	group I intron	35
897	690	autocatalytically spliced intron	35
898	691	SRP RNA primary transcript	35
899	692	7S RNA	108
900	692	signal recognition particle RNA	108
901	692	SRP RNA	35
902	694	classical pseudoknot	35
903	694	H pseudoknot	35
904	694	H-pseudoknot	35
905	694	H-type pseudoknot	35
906	694	hairpin-type pseudoknot	35
907	695	box C/D snoRNA	35
908	695	C D box snoRNA	35
909	695	C/D box snoRNA	35
910	696	box H/ACA snoRNA	35
911	696	H ACA box snoRNA	35
912	696	H/ACA box snoRNA	35
913	697	C/D box snoRNA primary transcript	35
914	698	H ACA box snoRNA primary transcript	35
915	700	transcript_edited_by_C-insertion_and_dinucleotide_insertion	108
916	704	gRNA	35
917	704	guide RNA	35
918	705	group II intron	35
919	706	editing block	35
920	707	intergenic region	35
921	708	editing domain	35
922	709	unedited region	35
923	710	H ACA box snoRNA encoding	35
924	711	oligo U tail	35
925	712	polyA sequence	35
926	713	branch point	35
927	713	branch site	35
928	713	branch_point	35
929	714	polypyrimidine tract	35
930	715	bacterial RNApol promoter	35
931	716	bacterial terminator	35
932	717	terminator of type 2 RNApol III promoter	35
933	718	transcription end site	35
934	719	RNApol III promoter type 1	35
935	720	RNApol III promoter type 2	35
936	720	tRNA promoter	108
937	721	A-box	35
938	722	B-box	35
939	723	RNApol III promoter type 3	35
940	724	C-box	35
941	725	snRNA encoding	35
942	726	telomeric DNA	35
943	726	telomeric sequence	35
944	728	chromosomal regulatory element	35
945	729	insulator element	35
946	730	chromosomal structural element	35
947	731	five prime open reading frame	35
948	732	upstream AUG codon	35
949	733	polycistronic primary transcript	35
950	734	monocistronic primary transcript	35
951	735	monocistronic mRNA	35
952	735	monocistronic processed transcript	35
953	736	polycistronic mRNA	35
954	736	polycistronic processed transcript	108
955	737	mini exon donor RNA	35
956	737	mini-exon donor RNA	35
957	738	mini-exon	108
958	738	spliced leader RNA	35
959	739	engineered plasmid	35
960	739	engineered plasmid gene	108
961	740	transcribed spacer region	35
962	741	internal transcribed spacer region	35
963	742	external transcribed spacer region	35
964	743	tetranucleotide repeat microsatellite feature	35
965	744	SRP RNA encoding	35
966	746	antisense RNA	35
967	747	antisense primary transcript	35
968	748	small interfering RNA	35
969	749	micro RNA primary transcript	35
970	749	miRNA primary transcript	35
971	750	small temporal RNA primary transcript	35
972	750	stRNA primary transcript	35
973	751	small temporal RNA	35
974	752	small subunit rRNA	35
975	752	SSU RNA	35
976	752	SSU rRNA	35
977	753	large subunit rRNA	35
978	753	LSU RNA	35
979	753	LSU rRNA	35
980	754	5S LSU rRNA	35
981	754	5S ribosomal RNA	35
982	754	5S rRNA	35
983	754	rRNA 5S	35
984	755	28S LSU rRNA	35
985	755	28S ribosomal RNA	35
986	755	28S rRNA	35
987	755	rRNA 28S	35
988	756	maxi-circle gene	35
989	756	maxicircle gene	35
990	757	noncoding RNA	35
991	758	stRNA encoding	35
992	759	repeat region	35
993	760	dispersed repeat	35
994	760	interspersed repeat	35
995	761	tmRNA encoding	35
996	764	spliceosomal intron	35
997	765	tRNA encoding	35
998	766	introgressed chromosome region	35
999	767	monocistronic transcript	35
1000	768	mobile intron	35
1001	769	nucleotide insertion	35
1002	769	nucleotide_insertion	35
1003	770	EST match	35
1004	771	sequence rearrangement feature	35
1005	772	chromosome breakage sequence	35
1006	773	internal eliminated sequence	35
1007	774	macronucleus destined segment	35
1008	776	non canonical splice site	35
1009	776	non-canonical splice site	35
1010	777	canonical splice site	35
1011	778	canonical 3' splice site	35
1012	778	canonical three prime splice site	35
1013	779	canonical 5' splice site	35
1014	779	canonical five prime splice site	35
1015	780	non canonical 3' splice site	108
1016	780	non canonical three prime splice site	35
1017	780	non-canonical three prime splice site	35
1018	781	non canonical 5' splice site	35
1019	781	non canonical five prime splice site	35
1020	781	non-canonical five prime splice site	35
1021	782	non ATG start codon	35
1022	782	non canonical start codon	35
1023	782	non-canonical start codon	35
1024	783	aberrant processed transcript	35
1025	785	exonic splice enhancer	35
1026	786	nuclease sensitive site	35
1027	787	DHS	35
1028	787	DNAseI hypersensitive site	35
1029	788	translocation element	35
1030	789	deletion junction	35
1031	790	golden path	35
1032	791	cDNA match	35
1033	792	gene with polycistronic transcript	35
1034	793	cleaved initiator methionine	35
1035	793	init_met	108
1036	793	initiator methionine	108
1037	794	gene with dicistronic transcript	35
1038	795	gene with recoded mRNA	35
1039	796	single nucleotide polymorphism	35
1040	798	oligonucleotide	35
1041	799	gene with stop codon read through	35
1042	800	gene with stop codon redefined as pyrrolysine	35
1043	801	boundary	35
1044	801	breakpoint	35
1045	803	possible base call error	35
1046	804	possible assembly error	35
1047	805	experimental result region	35
1048	807	tandem repeat	35
1049	808	3' trans splice site	108
1050	808	trans splice acceptor site	35
1051	809	5 prime trans splice site	108
1052	809	trans splice donor site	35
1053	809	trans-splice donor site	35
1054	810	SL1 acceptor site	35
1055	811	SL2 acceptor site	35
1056	812	gene with stop codon redefined as selenocysteine	35
1057	813	gene with mRNA recoded by translational bypass	35
1058	814	gene with transcript with translational frameshift	35
1059	815	DNA motif	35
1060	816	nucleotide motif	35
1061	817	RNA motif	35
1062	818	dicistronic mRNA	35
1063	818	dicistronic processed transcript	108
1064	819	reading frame	35
1065	820	blocked reading frame	35
1066	822	foreign transposable element	35
1067	823	gene with dicistronic primary transcript	35
1068	824	gene with dicistronic mRNA	35
1069	824	gene with dicistronic processed transcript	35
1070	825	intervening DNA	35
1071	826	origin of transfer	35
1072	827	signal	108
1073	827	transit	108
1074	827	transit peptide	35
1075	828	repeat unit	35
1076	829	cis regulatory module	35
1077	829	TF module	35
1078	830	protein intron	108
1079	831	intein containing	35
1080	833	fragment	35
1081	835	feature attribute	35
1082	836	exemplar mRNA	35
1083	837	sequence location	35
1084	838	organelle sequence	35
1085	839	mitochondrial sequence	35
1086	840	nuclear sequence	35
1087	841	nucleomorphic sequence	35
1088	842	plastid sequence	35
1089	843	kinetoplast_chromosome	35
1090	844	maxicircle_chromosome	35
1091	845	apicoplast sequence	35
1092	846	chromoplast sequence	35
1093	847	chloroplast sequence	35
1094	848	cyanelle sequence	35
1095	849	leucoplast sequence	35
1096	850	proplastid sequence	35
1097	851	plasmid location	35
1098	852	amplification origin	35
1099	853	proviral location	35
1100	854	gene group regulatory region	35
1101	855	clone insert	35
1102	856	lambda vector	35
1103	857	plasmid vector	35
1104	859	single strand cDNA	35
1105	859	single stranded cDNA	35
1106	859	single-strand cDNA	108
1107	860	double strand cDNA	108
1108	860	double stranded cDNA	35
1109	860	double-strand cDNA	108
1110	864	P1_clone	108
1111	868	pyrrolysyl tRNA	35
1112	868	pyrrolysyl-transfer ribonucleic acid	35
1113	868	pyrrolysyl-transfer RNA	35
1114	871	tmRNA coding piece	35
1115	872	tmRNA acceptor piece	35
1116	873	quantitative trait locus	35
1117	874	genomic island	35
1118	875	pathogenic island	35
1119	876	metabolic island	35
1120	877	adaptive island	35
1121	878	symbiosis island	35
1122	879	pseudogenic rRNA	35
1123	880	pseudogenic tRNA	35
1124	881	engineered episome	35
1125	887	cloned region	35
1126	887	cloned segment	35
1127	888	reagent attribute	35
1128	896	engineered rescue fragment	35
1129	896	engineered rescue region	35
1130	896	engineered rescue segment	35
1131	897	rescue mini gene	35
1132	897	rescue mini-gene	35
1133	898	transgenic transposable element	35
1134	899	natural transposable element	35
1135	900	engineered transposable element	35
1136	901	engineered foreign transposable element	35
1137	902	assortment derived duplication	35
1138	903	assortment derived deficiency plus duplication	35
1139	904	assortment-derived deficiency	35
1140	905	assortment derived aneuploid	35
1141	906	construct	35
1142	906	engineered region	35
1143	906	engineered sequence	35
1144	907	engineered foreign region	35
1145	909	engineered tag	35
1146	910	validated cDNA clone	35
1147	911	invalidated cDNA clone	35
1148	912	chimeric cDNA clone	35
1149	913	genomically contaminated cDNA clone	35
1150	914	polyA primed cDNA clone	35
1151	915	partially processed cDNA clone	35
1152	917	mini gene	35
1153	918	rescue gene	35
1154	919	wild type	35
1155	920	wild type rescue gene	35
1156	921	mitochondrial chromosome	35
1157	922	chloroplast chromosome	35
1158	923	chromoplast chromosome	35
1159	924	cyanelle chromosome	35
1160	925	leucoplast chromosome	35
1161	926	macronuclear chromosome	35
1162	927	micronuclear chromosome	35
1163	928	nuclear chromosome	35
1164	929	nucleomorphic chromosome	35
1165	930	chromosome part	35
1166	931	gene member region	35
1167	933	transcript region	35
1168	934	mature transcript region	35
1169	935	primary transcript region	35
1170	936	mRNA region	35
1171	937	UTR region	35
1172	938	rRNA primary transcript region	35
1173	939	positional	108
1174	939	positional polypeptide feature	108
1175	939	region	118
1176	939	region or site annotation	108
1177	939	site	118
1178	940	repeat component	35
1179	941	spliceosomal intron region	35
1180	942	gene component region	35
1181	947	tmRNA region	35
1182	948	long term repeat component	35
1183	948	LTR component	35
1184	949	3' long terminal repeat component	35
1185	949	three prime LTR component	35
1186	950	5' long term repeat component	35
1187	950	five prime LTR component	35
1188	951	CDS region	35
1189	952	exon region	35
1190	953	homolog	35
1191	953	homologous region	35
1192	953	homologue	35
1193	954	paralog	35
1194	954	paralogous region	35
1195	954	paralogue	35
1196	955	ortholog	35
1197	955	orthologous region	35
1198	955	orthologue	35
1199	961	capped primary transcript	35
1200	962	capped mRNA	35
1201	963	mRNA attribute	35
1202	966	minus 1 frameshift	35
1203	967	minus 2 frameshift	35
1204	968	plus 1 frameshift	35
1205	969	plus 2 framshift	35
1206	970	trans-spliced	35
1207	971	polyadenylated mRNA	35
1208	972	trans-spliced mRNA	35
1209	973	edited transcript	35
1210	974	edited transcript by A to I substitution	35
1211	975	bound by protein	35
1212	976	bound by nucleic acid	35
1213	977	alternatively spliced	35
1214	982	codon redefined	35
1215	983	stop codon read through	35
1216	983	stop codon readthrough	108
1217	984	stop codon redefined as pyrrolysine	35
1218	985	stop codon redefined as selenocysteine	35
1219	986	recoded by translational bypass	35
1220	987	translationally frameshifted	35
1221	988	maternally imprinted gene	35
1222	989	paternally imprinted gene	35
1223	990	post translationally regulated gene	35
1224	991	negatively autoregulated gene	35
1225	992	positively autoregulated gene	35
1226	994	silenced by DNA modification	35
1227	995	silenced by DNA methylation	35
1228	996	translationally regulated gene	35
1229	997	allelically excluded gene	35
1230	998	epigenetically modified gene	35
1231	999	nuclear mitochondrial	35
1232	1001	unequally crossed over	35
1233	1003	endogenous retroviral sequence	35
1234	1004	rearranged at DNA level	35
1235	1006	independently known	35
1236	1007	supported by sequence similarity	35
1237	1008	supported by domain match	35
1238	1009	supported by EST or cDNA	35
1239	1011	predicted by ab initio computation	35
1240	1012	asx turn	35
1241	1013	cloned cDNA insert	35
1242	1014	cloned genomic insert	35
1243	1015	engineered insert	35
1244	1016	edit operation	35
1245	1017	insert U	35
1246	1018	delete U	35
1247	1019	substitute A to I	35
1248	1020	insert C	35
1249	1021	insert dinucleotide	35
1250	1022	substitute C to U	35
1251	1023	insert G	35
1252	1024	insert GC	35
1253	1025	insert GU	35
1254	1026	insert CU	35
1255	1027	insert AU	35
1256	1028	insert AA	35
1257	1029	edited mRNA	35
1258	1030	guide RNA region	35
1259	1031	anchor region	35
1260	1032	pre-edited mRNA	35
1261	1034	miRNA target site	35
1262	1035	edited CDS	35
1263	1036	vertebrate immunoglobulin T cell receptor rearranged segment	35
1264	1038	vertebrate immunoglobulin T cell receptor rearranged gene cluster	35
1265	1039	vertebrate immune system gene recombination signal feature	35
1266	1040	recombinationally rearranged	35
1267	1041	recombinationally rearranged vertebrate immune system gene	35
1268	1042	attP site	35
1269	1043	attB site	35
1270	1044	attBP'	108
1271	1044	attL site	108
1272	1045	attPB'	108
1273	1045	attR site	35
1274	1046	attachment site	108
1275	1046	integration excision site	35
1276	1047	res site	35
1277	1047	resolution site	35
1278	1048	inversion site	35
1279	1049	dif site	35
1280	1050	attC site	35
1281	1051	eukaryotic terminator	35
1282	1052	origin of vegetative replication	35
1283	1053	origin of bacterial chromosome replication	35
1284	1054	DNA chromosome	35
1285	1055	double stranded DNA chromosome	35
1286	1056	single stranded DNA chromosome	35
1287	1057	linear double stranded DNA chromosome	35
1288	1058	circular double stranded DNA chromosome	35
1289	1059	linear single stranded DNA chromosome	35
1290	1060	circular single stranded DNA chromosome	35
1291	1061	RNA chromosome	35
1292	1062	single stranded RNA chromosome	35
1293	1063	linear single stranded RNA chromosome	35
1294	1064	linear double stranded RNA chromosome	35
1295	1065	double stranded RNA chromosome	35
1296	1066	circular single stranded RNA chromosome	35
1297	1067	circular double stranded RNA chromosome	35
1298	1068	sequence replication mode	35
1299	1069	rolling circle	35
1300	1070	theta replication	35
1301	1071	DNA replication mode	35
1302	1072	RNA replication mode	35
1303	1073	insertion sequence	35
1304	1073	IS	108
1305	1074	minicircle gene	35
1306	1076	anchor binding site	35
1307	1077	information region	35
1308	1077	template region	35
1309	1078	gRNA encoding	35
1310	1079	minicircle_chromosome	35
1311	1080	rho dependent bacterial terminator	35
1312	1081	rho independent bacterial terminator	35
1313	1082	strand attribute	35
1314	1085	topology attribute	35
1315	1086	two-ended	108
1316	1087	zero-ended	108
1317	1088	class II RNA	35
1318	1089	class I RNA	35
1319	1090	genomic DNA	35
1320	1091	BAC cloned genomic insert	35
1321	1093	consensus region	35
1322	1094	consensus mRNA	35
1323	1095	predicted gene	35
1324	1096	gene fragment	35
1325	1097	recursive splice site	35
1326	1098	BAC end	35
1327	1099	16S ribosomal RNA	35
1328	1099	16S rRNA	108
1329	1099	16S SSU RNA	35
1330	1099	rRNA 16S	35
1331	1100	23S LSU rRNA	35
1332	1100	23S ribosomal RNA	108
1333	1100	23S rRNA	35
1334	1100	rRNA 23S	35
1335	1101	25S LSU rRNA	35
1336	1101	25S ribosomal RNA	35
1337	1101	25S rRNA	35
1338	1101	rRNA 25S	35
1339	1102	solo LTR	35
1340	1103	low complexity	35
1341	1104	low complexity region	35
1342	1106	cryptic prophage	35
1343	1108	DNA constraint	35
1344	1108	DNA constraint sequence	35
1345	1109	i motif	35
1346	1109	short intercalated motif	35
1347	1110	peptide nucleic acid	35
1348	1110	PNA oligo	35
1349	1111	catalytic DNA	35
1350	1111	deoxyribozyme	108
1351	1111	DNA enzyme	35
1352	1112	multiple nucleotide polymorphism	108
1353	1113	intron domain	35
1354	1114	wobble base pair	35
1355	1114	wobble pair	35
1356	1115	IGS	35
1357	1115	internal guide sequence	35
1358	1116	silent mutation	35
1359	1118	CNP	35
1360	1118	CNV	35
1361	1118	copy number polymorphism	35
1362	1118	copy number variation	35
1363	1119	mutation affecting copy number	35
1364	1119	sequence variant affecting copy number	35
1365	1120	aberration breakpoint	35
1366	1120	aberration_junction	35
1367	1120	chromosome breakpoint	35
1368	1121	inversion breakpoint	35
1369	1122	allelomorph	35
1370	1124	polymorphic sequence variant	35
1371	1128	direction attribute	35
1372	1131	mitochondrial DNA	35
1373	1132	chloroplast DNA	35
1374	1134	piwi-associated RNA	35
1375	1135	arginyl tRNA	35
1376	1136	MGE	35
1377	1136	mobile genetic element	35
1378	1137	extrachromosomal mobile genetic element	35
1379	1138	integrated mobile genetic element	35
1380	1139	integrated plasmid	35
1381	1140	viral sequence	35
1382	1140	virus sequence	35
1383	1141	bacteriophage	35
1384	1141	phage	35
1385	1141	phage sequence	35
1386	1142	attCtn site	35
1387	1143	nuclear mitochondrial pseudogene	35
1388	1143	nuclear mt pseudogene	35
1389	1143	NUMT	35
1390	1144	cointegrated plasmid	35
1391	1144	cointegrated replicon	35
1392	1145	IRLinv site	35
1393	1146	IRRinv site	35
1394	1147	inversion site part	35
1395	1148	defective conjugative transposon	35
1396	1149	repeat fragment	35
1397	1153	transposon fragment	35
1398	1154	transcriptional cis regulatory region	35
1399	1155	splicing regulatory region	35
1400	1157	promoter targeting sequence	35
1401	1158	partially characterised change in DNA sequence	35
1402	1158	partially_characterised_change_in_DNA_sequence	35
1403	1158	sequence alteration	35
1404	1159	sequence variant	35
1405	1160	propeptide cleavage site	35
1406	1161	propep	108
1407	1162	immature_peptide_region	35
1408	1163	active peptide	35
1409	1163	peptide	519
1410	1164	compbias	108
1411	1164	compositional bias	108
1412	1164	compositionally biased	108
1413	1164	compositionally biased region of peptide	108
1414	1164	compositionally_biased_region	35
1415	1165	motif	519
1416	1165	polypeptide motif	35
1417	1166	polypeptide repeat	35
1418	1166	repeat	108
1419	1167	polypeptide structural region	35
1420	1167	structural_region	108
1421	1168	membrane_structure	35
1422	1169	extramembrane	108
1423	1169	extramembrane polypeptide region	35
1424	1169	extramembrane_region	108
1425	1169	topo_dom	108
1426	1170	cytoplasm_location	35
1427	1170	cytoplasmic polypeptide region	35
1428	1170	inside	108
1429	1171	non cytoplasmic polypeptide region	35
1430	1171	non_cytoplasm_location	35
1431	1171	outside	108
1432	1172	intramembrane	108
1433	1172	intramembrane polypeptide region	35
1434	1173	membrane peptide loop	35
1435	1173	membrane_loop	108
1436	1174	transmem	108
1437	1174	transmembrane	108
1438	1174	transmembrane polypeptide region	35
1439	1175	2nary structure	108
1440	1175	polypeptide secondary structure	35
1441	1175	secondary structure	108
1442	1175	secondary structure region	108
1443	1175	secondary_structure	108
1444	1176	polypeptide structural motif	108
1445	1176	structural_motif	108
1446	1177	coiled	108
1447	1177	coiled coil	35
1448	1178	helix turn helix	35
1449	1178	helix-turn-helix	35
1450	1178	HTH	108
1451	1179	sequencing_information	35
1452	1180	non consecutive	35
1453	1180	non_cons	35
1454	1181	non terminal	35
1455	1181	non_ter	35
1456	1182	conflict	35
1457	1183	unsure	35
1458	1184	cross_link	35
1459	1184	crosslink	108
1460	1185	disulfid	108
1461	1185	disulfide	108
1462	1185	disulfide bond	108
1463	1185	disulfide_bond	35
1464	1185	disulphide	35
1465	1185	disulphide bond	108
1466	1186	mod_res	35
1467	1186	modified residue	35
1468	1186	post_translational_modification	35
1469	1187	covalent_binding_site	35
1470	1188	binding	108
1471	1188	binding site	108
1472	1188	non_covalent_binding_site	35
1473	1189	metal_binding	108
1474	1190	protein protein contact	35
1475	1190	protein protein contact site	35
1476	1190	protein_protein_interaction	108
1477	1191	ca bind	108
1478	1191	Ca_contact_site	35
1479	1191	polypeptide calcium ion contact site	35
1480	1192	Co_contact_site	35
1481	1192	polypeptide cobalt ion contact site	35
1482	1193	Cu_contact_site	35
1483	1193	polypeptide copper ion contact site	35
1484	1194	Fe_contact_site	35
1485	1194	polypeptide iron ion contact site	35
1486	1195	Mg_contact_site	35
1487	1195	polypeptide magnesium ion contact site	35
1488	1196	Mn_contact_site	35
1489	1196	polypeptide manganese ion contact site	35
1490	1197	Mo_contact_site	35
1491	1197	polypeptide molybdenum ion contact site	35
1492	1198	Ni_contact_site	35
1493	1198	polypeptide nickel ion contact site	35
1494	1199	polypeptide tungsten ion contact site	35
1495	1199	W_contact_site	35
1496	1200	polypeptide zinc ion contact site	35
1497	1200	Zn_contact_site	35
1498	1201	act_site	108
1499	1201	active site residue	35
1500	1201	catalytic residue	35
1501	1202	polypeptide ligand contact	35
1502	1202	protein-ligand interaction	108
1503	1203	asx motif	35
1504	1204	beta bulge	35
1505	1205	beta bulge loop	35
1506	1206	beta bulge loop five	35
1507	1207	beta bulge loop six	35
1508	1208	strand	108
1509	1209	antiparallel beta strand	35
1510	1210	parallel beta strand	35
1511	1211	helix	108
1512	1212	helix-l	108
1513	1212	left handed helix	35
1514	1213	helix	108
1515	1213	right handed helix	35
1516	1214	a-helix	108
1517	1214	helix	108
1518	1215	pi helix	35
1519	1216	3(10) helix	35
1520	1216	3-10 helix	35
1521	1216	310 helix	35
1522	1216	three_ten_helix	35
1523	1217	nest	108
1524	1217	nest_motif	35
1525	1217	polypeptide nest motif	108
1526	1218	nest_left_right	35
1527	1218	nest_lr	35
1528	1218	polypeptide nest left right motif	35
1529	1219	nest_right_left	35
1530	1219	nest_rl	35
1531	1219	polypeptide nest right left motif	35
1532	1220	paperclip	108
1533	1220	paperclip loop	108
1534	1220	schellmann_loop	35
1535	1221	schellmann loop seven	35
1536	1221	seven-residue schellmann loop	35
1537	1222	schellmann loop six	35
1538	1222	six-residue schellmann loop	35
1539	1223	serine/threonine motif	35
1540	1223	st motif	35
1541	1223	st_motif	35
1542	1224	serine threonine staple motif	35
1543	1224	st_staple	35
1544	1225	turn	108
1545	1226	asx turn left handed type one	35
1546	1226	asx_turn_il	108
1547	1227	asx turn left handed type two	35
1548	1227	asx_turn_iil	35
1549	1228	asx turn right handed type two	35
1550	1228	asx_turn_iir	35
1551	1229	asx turn type right handed type one	35
1552	1229	asx_turn_ir	35
1553	1230	beta turn	35
1554	1231	beta turn left handed type one	35
1555	1231	beta_turn_il	35
1556	1231	type I' beta turn	35
1557	1231	type I' turn	35
1558	1232	beta turn left handed type two	35
1559	1232	beta_turn_iil	35
1560	1232	type II' beta turn	35
1561	1232	type II' turn	35
1562	1233	beta turn right handed type one	35
1563	1233	beta_turn_ir	35
1564	1233	type I beta turn	35
1565	1233	type I turn	35
1566	1234	beta turn right handed type two	35
1567	1234	beta_turn_iir	35
1568	1234	type II beta turn	35
1569	1234	type II turn	35
1570	1235	gamma turn	35
1571	1236	classic gamma turn	35
1572	1236	gamma turn classic	35
1573	1237	gamma turn inverse	35
1574	1238	serine/threonine turn	35
1575	1238	st_turn	35
1576	1239	st turn left handed type one	35
1577	1239	st_turn_il	35
1578	1240	st turn left handed type two	35
1579	1240	st_turn_iil	35
1580	1241	st turn right handed type one	35
1581	1241	st_turn_ir	35
1582	1242	st turn right handed type two	35
1583	1242	st_turn_iir	35
1584	1243	sequence_variations	35
1585	1244	natural_variant	519
1586	1244	sequence variation	519
1587	1244	variant	519
1588	1245	mutagen	35
1589	1245	mutagenesis	35
1590	1245	mutated_site	35
1591	1246	alternative_sequence	35
1592	1246	isoform	118
1593	1246	sequence variation	118
1594	1246	var_seq	35
1595	1246	varsplic	118
1596	1247	beta turn type six	35
1597	1247	cis-proline loop	35
1598	1247	type VI beta turn	35
1599	1247	type VI turn	35
1600	1248	beta turn type six a	35
1601	1248	type VIa beta turn	35
1602	1248	type VIa turn	35
1603	1249	beta turn type six a one	35
1604	1249	type VIa1 beta turn	35
1605	1249	type VIa1 turn	35
1606	1250	beta turn type six a two	35
1607	1250	type VIa2 beta turn	35
1608	1250	type VIa2 turn	35
1609	1251	beta turn type six b	35
1610	1251	type VIb beta turn	35
1611	1251	type VIb turn	35
1612	1252	beta turn type eight	35
1613	1252	type VIII beta turn	35
1614	1252	type VIII turn	35
1615	1253	DRE motif	35
1616	1253	NDM4	35
1617	1253	WATCGATW_motif	35
1618	1254	directional motif v4	35
1619	1254	DMv4	35
1620	1254	DMv4 motif	35
1621	1254	motif 1 element	35
1622	1254	promoter motif 1	35
1623	1254	YGGTCACATR	118
1624	1255	AWCAGCTGWT	118
1625	1255	E box motif	35
1626	1255	generic E box motif	35
1627	1255	NDM5	108
1628	1256	directional motif v5	35
1629	1256	DMv5	35
1630	1256	DMv5 motif	35
1631	1256	KTYRGTATWTTT	118
1632	1256	promoter motif 6	108
1633	1257	directional motif v3	35
1634	1257	DMv3	35
1635	1257	DMv3 motif	35
1636	1257	KNNCAKCNCTRNY	118
1637	1257	promoter motif 7	35
1638	1258	directional motif v2	35
1639	1258	DMv2	35
1640	1258	DMv2 motif	35
1641	1258	MKSYGGCARCGSYSS	118
1642	1258	promoter motif 8	35
1643	1259	CSARCSSAACGS	118
1644	1259	motif ten element	35
1645	1259	motif_ten_element	35
1646	1260	directional motif p3	35
1647	1260	directional promoter motif 3	35
1648	1260	DMp3	35
1649	1260	INR1 motif	35
1650	1261	directional motif 5	35
1651	1261	directional promoter motif 5	108
1652	1261	DMp5	35
1653	1261	DPE1 motif	35
1654	1262	directional promoter motif v1	108
1655	1262	DMv1	108
1656	1262	DMv1 motif	35
1657	1263	GAGA	35
1658	1263	GAGA motif	35
1659	1263	NDM1	35
1660	1264	NDM2	35
1661	1264	NDM2 motif	35
1662	1264	non directional promoter motif 2	35
1663	1265	NDM3	35
1664	1265	NDM3 motif	35
1665	1265	non directional motif 3	35
1666	1266	double stranded RNA virus sequence	35
1667	1266	ds RNA viral sequence	35
1668	1267	maverick element	108
1669	1268	21S LSU rRNA	35
1670	1268	21S ribosomal RNA	35
1671	1268	21S rRNA	35
1672	1268	rRNA 21S	35
1673	1269	tRNA region	35
1674	1270	anti-codon loop	35
1675	1270	anticodon loop	35
1676	1271	anti-codon	35
1677	1272	CCA sequence	35
1678	1272	CCA tail	35
1679	1273	D loop	108
1680	1273	DHU loop	35
1681	1274	T loop	35
1682	1274	TpsiC loop	35
1683	1275	pyrrolysine tRNA primary transcript	35
1684	1276	small nucleolar RNA U3	35
1685	1276	snoRNA U3	35
1686	1276	U3 small nucleolar RNA	35
1687	1276	U3 snoRNA	35
1688	1277	ARE	108
1689	1277	AU rich element	35
1690	1277	AU-rich element	35
1691	1278	BRE	108
1692	1278	Bruno response element	35
1693	1279	IRE	35
1694	1279	iron responsive element	35
1695	1281	peptide nucleic acid	108
1696	1284	pseudouridylation guide snoRNA	35
1697	1286	LNA oligo	35
1698	1286	locked nucleic acid	35
1699	1288	threose nucleic acid	35
1700	1288	TNA oligo	35
1701	1290	glycerol nucleic acid	35
1702	1290	glycol nucleic acid	35
1703	1290	GNA oligo	35
1704	1291	R GNA	35
1705	1292	(R)-glycerol nucleic acid	35
1706	1292	(R)-glycol nucleic acid	35
1707	1292	R GNA oligo	35
1708	1293	S GNA	35
1709	1294	(S)-glycerol nucleic acid	35
1710	1294	(S)-glycol nucleic acid	35
1711	1294	S GNA oligo	35
1712	1295	double stranded DNA virus	35
1713	1295	ds DNA viral sequence	35
1714	1296	single strand RNA virus	35
1715	1296	ss RNA viral sequence	35
1716	1297	negative sense single stranded RNA virus	108
1717	1297	negative sense ssRNA viral sequence	35
1718	1298	positive sense single stranded RNA virus	108
1719	1298	positive sense ssRNA viral sequence	35
1720	1299	ambisense single stranded RNA virus	35
1721	1299	ambisense ssRNA viral sequence	35
1722	1300	RNA polymerase promoter	35
1723	1301	Phage RNA Polymerase Promoter	35
1724	1302	SP6 RNA Polymerase Promoter	35
1725	1303	T3 RNA Polymerase Promoter	35
1726	1304	T7 RNA Polymerase Promoter	35
1727	1305	5' EST	35
1728	1305	five prime EST	35
1729	1306	3' EST	35
1730	1306	three prime EST	35
1731	1307	ribosomal frameshift	35
1732	1307	translational frameshift	35
1733	1308	plus 1 ribosomal frameshift	35
1734	1308	plus 1 translational frameshift	35
1735	1309	plus 2 ribosomal frameshift	35
1736	1309	plus 2 translational frameshift	35
1737	1310	group III intron	35
1738	1311	noncoding region of exon	35
1739	1312	coding region of exon	35
1740	1313	endonuclease spliced intron	35
1741	1314	protein coding gene	35
1742	1315	transgenic insertion	35
1743	1317	silenced by RNA interference	35
1744	1318	silenced by histone modification	35
1745	1319	silenced by histone methylation	35
1746	1320	silenced by histone deacetylation	35
1747	1321	gene silenced by RNA interference	35
1748	1321	RNA interference silenced gene	35
1749	1321	RNAi silenced gene	35
1750	1322	gene silenced by histone modification	35
1751	1323	gene silenced by histone methylation	35
1752	1324	gene silenced by histone deacetylation	35
1753	1325	D	35
1754	1326	Y	35
1755	1327	I	108
1756	1327	RNAMOD:017	108
1757	1328	7-methylguanine	35
1758	1328	seven methylguanine	35
1759	1334	amino acid	35
1760	1335	major transcription start site	35
1761	1335	major TSS	35
1762	1336	minor TSS	35
1763	1337	TSS region	35
1764	1338	encodes alternate transcription start sites	35
1765	1339	miRNA primary transcript region	35
1766	1340	pre-miRNA	35
1767	1341	miRNA stem	35
1768	1342	miRNA loop	35
1769	1343	synthetic oligo	35
1770	1345	fragment assembly	35
1771	1345	physical map	35
1772	1346	BACmap	35
1773	1346	fingerprint map	35
1774	1346	FPC	35
1775	1346	FPCmap	35
1776	1346	restriction map	35
1777	1347	STS map	35
1778	1348	radiation hybrid map	35
1779	1348	RH map	35
1780	1349	sonicate fragment	35
1781	1353	homing endonuclease binding site	35
1782	1354	octamer motif	35
1783	1355	apicoplast chromosome	35
1784	1356	sequence collection	35
1785	1357	overlapping feature set	35
1786	1358	overlapping EST set	35
1787	1359	ncRNA gen	35
1788	1359	ncRNA gene	35
1789	1359	non-coding RNA gene	108
1790	1360	gRNA gene	35
1791	1361	miRNA gene	35
1792	1362	scRNA gene	35
1793	1363	snoRNA gene	35
1794	1364	snRNA gene	35
1795	1365	SRP RNA gene	35
1796	1366	stRNA gene	35
1797	1367	tmRNA gene	35
1798	1368	tRNA gene	35
1799	1369	modified adenosine	35
1800	1370	modified inosine	35
1801	1371	modified cytidine	35
1802	1372	modified guanosine	35
1803	1373	modified uridine	35
1804	1374	1-methylinosine	35
1805	1374	m1I	35
1806	1374	one methylinosine	35
1807	1375	1,2'-O-dimethylinosine	35
1808	1375	m'Im	35
1809	1375	one two prime O dimethylinosine	35
1810	1376	2'-O-methylinosine	35
1811	1376	Im	35
1812	1376	two prime O methylinosine	35
1813	1377	3-methylcytidine	35
1814	1377	m3C	35
1815	1377	three methylcytidine	35
1816	1378	5-methylcytidine	35
1817	1378	five methylcytidine	35
1818	1378	m5C	35
1819	1379	2'-O-methylcytidine	35
1820	1379	Cm	35
1821	1379	two prime O methylcytidine	35
1822	1380	2-thiocytidine	35
1823	1380	s2C	35
1824	1380	two thiocytidine	35
1825	1381	ac4C	35
1826	1381	N4 acetylcytidine	35
1827	1381	N4-acetylcytidine	35
1828	1382	5-formylcytidine	35
1829	1382	f5C	35
1830	1382	five formylcytidine	35
1831	1383	5,2'-O-dimethylcytidine	35
1832	1383	five two prime O dimethylcytidine	35
1833	1383	m5Cm	35
1834	1384	ac4Cm	35
1835	1384	N4 acetyl 2 prime O methylcytidine	35
1836	1384	N4-acetyl-2'-O-methylcytidine	35
1837	1385	k2C	35
1838	1386	m4C	35
1839	1386	N4 methylcytidine	35
1840	1386	N4-methylcytidine	35
1841	1387	m4Cm	35
1842	1387	N4 2 prime O dimethylcytidine	35
1843	1387	N4,2'-O-dimethylcytidine	35
1844	1388	5-hydroxymethylcytidine	35
1845	1388	five hydroxymethylcytidine	35
1846	1388	hm5C	35
1847	1389	5-formyl-2'-O-methylcytidine	35
1848	1389	f5Cm	35
1849	1389	five formyl two prime O methylcytidine	35
1850	1390	m42Cm	35
1851	1390	N4,N4,2'-O-trimethylcytidine	35
1852	1391	1-methyladenosine	35
1853	1391	m1A	35
1854	1391	one methyladenosine	35
1855	1392	2-methyladenosine	35
1856	1392	m2A	35
1857	1392	two methyladenosine	35
1858	1393	m6A	35
1859	1393	N6 methyladenosine	35
1860	1393	N6-methyladenosine	35
1861	1394	2'-O-methyladenosine	35
1862	1394	Am	35
1863	1394	two prime O methyladenosine	35
1864	1395	2-methylthio-N6-methyladenosine	35
1865	1395	ms2m6A	35
1866	1395	two methylthio N6 methyladenosine	35
1867	1396	i6A	35
1868	1396	N6 isopentenyladenosine	35
1869	1396	N6-isopentenyladenosine	35
1870	1397	2-methylthio-N6-isopentenyladenosine	35
1871	1397	ms2i6A	35
1872	1397	two methylthio N6 isopentenyladenosine	35
1873	1398	io6A	35
1874	1398	N6 cis hydroxyisopentenyl adenosine	35
1875	1398	N6-(cis-hydroxyisopentenyl)adenosine	35
1876	1399	2-methylthio-N6-(cis-hydroxyisopentenyl) adenosine	35
1877	1399	ms2io6A	35
1878	1399	two methylthio N6 cis hydroxyisopentenyl adenosine	35
1879	1400	g6A	35
1880	1400	N6 glycinylcarbamoyladenosine	35
1881	1400	N6-glycinylcarbamoyladenosine	35
1882	1401	N6 threonylcarbamoyladenosine	35
1883	1401	N6-threonylcarbamoyladenosine	35
1884	1401	t6A	35
1885	1402	2-methylthio-N6-threonyl carbamoyladenosine	35
1886	1402	ms2t6A	35
1887	1402	two methylthio N6 threonyl carbamoyladenosine	35
1888	1403	m6t6A	35
1889	1403	N6 methyl N6 threonylcarbamoyladenosine	35
1890	1403	N6-methyl-N6-threonylcarbamoyladenosine	35
1891	1404	hn6A	35
1892	1404	N6 hydroxynorvalylcarbamoyladenosine	35
1893	1404	N6-hydroxynorvalylcarbamoyladenosine	35
1894	1405	2-methylthio-N6-hydroxynorvalyl carbamoyladenosine	35
1895	1405	ms2hn6A	35
1896	1405	two methylthio N6 hydroxynorvalyl carbamoyladenosine	35
1897	1406	2'-O-ribosyladenosine (phosphate)	35
1898	1406	Ar(p)	35
1899	1406	two prime O ribosyladenosine phosphate	35
1900	1407	m62A	35
1901	1407	N6,N6-dimethyladenosine	35
1902	1408	m6Am	35
1903	1408	N6 2 prime O dimethyladenosine	35
1904	1408	N6,2'-O-dimethyladenosine	35
1905	1409	m62Am	35
1906	1409	N6,N6,2'-O-trimethyladenosine	35
1907	1410	1,2'-O-dimethyladenosine	35
1908	1410	m1Am	35
1909	1410	one two prime O dimethyladenosine	35
1910	1411	ac6A	35
1911	1411	N6 acetyladenosine	35
1912	1411	N6-acetyladenosine	35
1913	1412	7-deazaguanosine	108
1914	1412	seven deazaguanosine	35
1915	1413	Q	35
1916	1414	eQ	35
1917	1415	galactosyl queuosine	35
1918	1415	galactosyl-queuosine	35
1919	1415	galQ	35
1920	1416	mannosyl queuosine	35
1921	1416	mannosyl-queuosine	35
1922	1416	manQ	35
1923	1417	7-cyano-7-deazaguanosine	35
1924	1417	preQ0	35
1925	1417	seven cyano seven deazaguanosine	35
1926	1418	7-aminomethyl-7-deazaguanosine	35
1927	1418	preQ1	35
1928	1418	seven aminomethyl seven deazaguanosine	35
1929	1419	G+	35
1930	1420	1-methylguanosine	35
1931	1420	m1G	35
1932	1420	one methylguanosine	35
1933	1421	m2G	35
1934	1421	N2 methylguanosine	35
1935	1421	N2-methylguanosine	35
1936	1422	7-methylguanosine	35
1937	1422	m7G	35
1938	1422	seven methylguanosine	35
1939	1423	2'-O-methylguanosine	35
1940	1423	Gm	35
1941	1423	two prime O methylguanosine	35
1942	1424	m22G	35
1943	1424	N2,N2-dimethylguanosine	35
1944	1425	m2Gm	35
1945	1425	N2 2 prime O dimethylguanosine	35
1946	1425	N2,2'-O-dimethylguanosine	35
1947	1426	m22Gmv	35
1948	1426	N2,N2,2'-O-trimethylguanosine	35
1949	1427	2'-O-ribosylguanosine (phosphate)	35
1950	1427	Gr(p)	35
1951	1427	two prime O ribosylguanosine phosphate	35
1952	1428	yW	35
1953	1429	o2yW	35
1954	1430	OHyW	35
1955	1431	OHyW*	35
1956	1431	undermodified hydroxywybutosine	35
1957	1432	IMG	35
1958	1433	mimG	35
1959	1434	m2,7G	35
1960	1434	N2 7 dimethylguanosine	35
1961	1434	N2,7-dimethylguanosine	35
1962	1435	m2,2,7G	35
1963	1435	N2,N2,7-trimethylguanosine	35
1964	1436	1,2'-O-dimethylguanosine	35
1965	1436	m1Gm	35
1966	1436	one two prime O dimethylguanosine	35
1967	1437	4-demethylwyosine	35
1968	1437	four demethylwyosine	35
1969	1437	imG-14	35
1970	1438	imG2	35
1971	1439	m2,7Gm	35
1972	1439	N2 7 2prirme O trimethylguanosine	35
1973	1439	N2,7,2'-O-trimethylguanosine	35
1974	1440	5-methyluridine	35
1975	1440	five methyluridine	35
1976	1440	m5U	35
1977	1441	2'-O-methyluridine	35
1978	1441	two prime O methyluridine	35
1979	1441	Um	35
1980	1442	5,2'-O-dimethyluridine	35
1981	1442	five two prime O dimethyluridine	35
1982	1442	m5Um	35
1983	1443	1-methylpseudouridine	35
1984	1443	m1Y	35
1985	1443	one methylpseudouridine	35
1986	1444	2'-O-methylpseudouridine	35
1987	1444	two prime O methylpseudouridine	35
1988	1444	Ym	35
1989	1445	2-thiouridine	35
1990	1445	s2U	35
1991	1445	two thiouridine	35
1992	1446	4-thiouridine	35
1993	1446	four thiouridine	35
1994	1446	s4U	35
1995	1447	5-methyl-2-thiouridine	35
1996	1447	five methyl 2 thiouridine	35
1997	1447	m5s2U	35
1998	1448	2-thio-2'-O-methyluridine	35
1999	1448	s2Um	35
2000	1448	two thio two prime O methyluridine	35
2001	1449	3-(3-amino-3-carboxypropyl)uridine	35
2002	1449	acp3U	35
2003	1450	5-hydroxyuridine	35
2004	1450	five hydroxyuridine	35
2005	1450	ho5U	35
2006	1451	5-methoxyuridine	35
2007	1451	five methoxyuridine	35
2008	1451	mo5U	35
2009	1452	cmo5U	35
2010	1452	uridine 5-oxyacetic acid	35
2011	1452	uridine five oxyacetic acid	35
2012	1453	mcmo5U	35
2013	1453	uridine 5-oxyacetic acid methyl ester	35
2014	1453	uridine five oxyacetic acid methyl ester	35
2015	1454	5-(carboxyhydroxymethyl)uridine	35
2016	1454	chm5U	35
2017	1454	five carboxyhydroxymethyl uridine	35
2018	1455	5-(carboxyhydroxymethyl)uridine methyl ester	35
2019	1455	five carboxyhydroxymethyl uridine methyl ester	35
2020	1455	mchm5U	35
2021	1456	5-methoxycarbonylmethyluridine	35
2022	1456	five methoxycarbonylmethyluridine	35
2023	1456	mcm5U	35
2024	1457	5-methoxycarbonylmethyl-2'-O-methyluridine	35
2025	1457	five methoxycarbonylmethyl two prime O methyluridine	35
2026	1457	mcm5Um	35
2027	1458	5-methoxycarbonylmethyl-2-thiouridine	35
2028	1458	five methoxycarbonylmethyl two thiouridine	35
2029	1458	mcm5s2U	35
2030	1459	5-aminomethyl-2-thiouridine	35
2031	1459	five aminomethyl two thiouridine	35
2032	1459	nm5s2U	35
2033	1460	5-methylaminomethyluridine	35
2034	1460	five methylaminomethyluridine	35
2035	1460	mnm5U	35
2036	1461	5-methylaminomethyl-2-thiouridine	35
2037	1461	five methylaminomethyl two thiouridine	35
2038	1461	mnm5s2U	35
2039	1462	5-methylaminomethyl-2-selenouridine	35
2040	1462	five methylaminomethyl two selenouridine	35
2041	1462	mnm5se2U	35
2042	1463	5-carbamoylmethyluridine	35
2043	1463	five carbamoylmethyluridine	35
2044	1463	ncm5U	35
2045	1464	5-carbamoylmethyl-2'-O-methyluridine	35
2046	1464	five carbamoylmethyl two prime O methyluridine	35
2047	1464	ncm5Um	35
2048	1465	5-carboxymethylaminomethyluridine	35
2049	1465	cmnm5U	35
2050	1465	five carboxymethylaminomethyluridine	35
2051	1466	5-carboxymethylaminomethyl- 2'-O-methyluridine	35
2052	1466	cmnm5Um	35
2053	1466	five carboxymethylaminomethyl two prime O methyluridine	35
2054	1467	5-carboxymethylaminomethyl-2-thiouridine	35
2055	1467	cmnm5s2U	35
2056	1467	five carboxymethylaminomethyl two thiouridine	35
2057	1468	3-methyluridine	35
2058	1468	m3U	35
2059	1468	three methyluridine	35
2060	1469	1-methyl-3-(3-amino-3-carboxypropyl) pseudouridine	35
2061	1469	m1acp3Y	35
2062	1470	5-carboxymethyluridine	35
2063	1470	cm5U	35
2064	1470	five carboxymethyluridine	35
2065	1471	3,2'-O-dimethyluridine	35
2066	1471	m3Um	35
2067	1471	three two prime O dimethyluridine	35
2068	1472	5-methyldihydrouridine	35
2069	1472	five methyldihydrouridine	35
2070	1472	m5D	35
2071	1473	3-methylpseudouridine	35
2072	1473	m3Y	35
2073	1473	three methylpseudouridine	35
2074	1474	5-taurinomethyluridine	35
2075	1474	five taurinomethyluridine	35
2076	1474	tm5U	35
2077	1475	5-taurinomethyl-2-thiouridine	35
2078	1475	five taurinomethyl two thiouridine	35
2079	1475	tm5s2U	35
2080	1476	5-(isopentenylaminomethyl)uridine	35
2081	1476	five isopentenylaminomethyl uridine	35
2082	1476	inm5U	35
2083	1477	5-(isopentenylaminomethyl)- 2-thiouridine	35
2084	1477	five isopentenylaminomethyl two thiouridine	35
2085	1477	inm5s2U	35
2086	1478	5-(isopentenylaminomethyl)- 2'-O-methyluridine	35
2087	1478	five isopentenylaminomethyl two prime O methyluridine	35
2088	1478	inm5Um	35
2089	1479	histone binding site	35
2090	1480	CDS fragment	35
2091	1480	incomplete CDS	35
2092	1481	modified amino acid feature	35
2093	1482	ModGly	35
2094	1482	modified glycine	35
2095	1483	ModAla	35
2096	1483	modified L alanine	35
2097	1483	modified L-alanine	35
2098	1484	ModAsn	35
2099	1484	modified L asparagine	35
2100	1484	modified L-asparagine	35
2101	1485	ModAsp	35
2102	1485	modified L aspartic acid	35
2103	1485	modified L-aspartic acid	35
2104	1486	ModCys	35
2105	1486	modified L cysteine	35
2106	1486	modified L-cysteine	35
2107	1487	ModGlu	35
2108	1487	modified L glutamic acid	35
2109	1487	modified L-glutamic acid	35
2110	1488	modified L threonine	35
2111	1488	modified L-threonine	35
2112	1488	ModThr	35
2113	1489	modified L tryptophan	35
2114	1489	modified L-tryptophan	35
2115	1489	ModTrp	35
2116	1490	ModGln	35
2117	1490	modified L glutamine	35
2118	1490	modified L-glutamine	35
2119	1491	modified L methionine	35
2120	1491	modified L-methionine	35
2121	1491	ModMet	35
2122	1492	modified L isoleucine	35
2123	1492	modified L-isoleucine	35
2124	1492	ModIle	35
2125	1493	modified L phenylalanine	35
2126	1493	modified L-phenylalanine	35
2127	1493	ModPhe	35
2128	1494	ModHis	35
2129	1494	modified L histidine	35
2130	1494	modified L-histidine	35
2131	1495	modified L serine	35
2132	1495	modified L-serine	35
2133	1495	MosSer	35
2134	1496	modified L lysine	35
2135	1496	modified L-lysine	35
2136	1496	ModLys	35
2137	1497	modified L leucine	35
2138	1497	modified L-leucine	35
2139	1497	ModLeu	35
2140	1498	modified L selenocysteine	35
2141	1498	modified L-selenocysteine	35
2142	1499	modified L valine	35
2143	1499	modified L-valine	35
2144	1499	ModVal	35
2145	1500	modified L proline	35
2146	1500	modified L-proline	35
2147	1500	ModPro	35
2148	1501	modified L tyrosine	35
2149	1501	modified L-tyrosine	35
2150	1501	ModTry	35
2151	1502	ModArg	35
2152	1502	modified L arginine	35
2153	1502	modified L-arginine	35
2154	1504	cleaved for gpi anchor region	35
2155	1505	biomaterial region	35
2156	1506	analysis feature	108
2157	1506	experimental output artefact	35
2158	1506	experimental_output_artefact	35
2159	1507	biological region	35
2160	1508	topologically defined region	35
2161	1509	translocation breakpoint	35
2162	1510	insertion breakpoint	35
2163	1511	deletion breakpoint	35
2164	1512	5' flanking region	108
2165	1512	five prime flanking region	35
2166	1513	3' flanking region	108
2167	1513	three prime flanking region	35
2168	1514	transcribed fragment	35
2169	1514	transfrag	108
2170	1515	cis splice site	35
2171	1516	trans splice site	35
2172	1517	splice boundary	35
2173	1517	splice junction	35
2174	1518	polypeptide conformational switch	35
2175	1519	dye terminator read	108
2176	1520	pyorsequenced read	108
2177	1521	ligation based read	108
2178	1522	polymerase synthesis read	108
2179	1523	cis regulatory frameshift element	35
2180	1524	expressed sequence assembly	35
2181	1525	DNA binding site	35
2182	1526	polyA junction	35
2183	1527	cryptic gene	35
2184	1528	mutation affecting polyadenylation	108
2185	1528	sequence variant affecting polyadenylation	35
2186	1529	3' RACE clone	108
2187	1530	cassette pseudogene	35
2188	1530	cassette type psedogene	108
2189	1531	A	35
2190	1531	Ala	35
2191	1532	V	35
2192	1532	Val	35
2193	1533	L	35
2194	1533	Leu	35
2195	1534	I	35
2196	1534	Ile	35
2197	1535	P	35
2198	1535	Pro	35
2199	1536	Trp	35
2200	1536	W	35
2201	1537	F	35
2202	1537	Phe	35
2203	1538	M	35
2204	1538	Met	35
2205	1539	G	35
2206	1539	Gly	35
2207	1540	S	35
2208	1540	Ser	35
2209	1541	T	35
2210	1541	Thr	35
2211	1542	Tyr	35
2212	1542	Y	35
2213	1543	C	35
2214	1543	Cys	35
2215	1544	Gln	35
2216	1544	Q	35
2217	1545	Asn	35
2218	1545	N	35
2219	1546	K	35
2220	1546	Lys	35
2221	1547	Arg	35
2222	1547	R	35
2223	1548	H	35
2224	1548	His	35
2225	1549	Asp	35
2226	1549	aspartic acid	35
2227	1549	D	35
2228	1550	E	35
2229	1550	Glu	35
2230	1550	glutamic acid	35
2231	1551	Sec	35
2232	1551	U	35
2233	1552	O	35
2234	1552	Pyl	35
2235	1553	transcribed cluster	35
2236	1553	unigene cluster	108
2237	1554	unigene cluster	108
2238	1555	Clustered_Regularly_Interspaced_Short_Palindromic_Repeat	35
2239	1555	CRISPR element	35
2240	1556	insulator binding site	108
2241	1557	enhancer binding site	108
2242	1558	contig collection	35
2243	1559	large intervening non-coding RNA	35
2244	1560	UTR sequence tag	35
2245	1561	3' UST	108
2246	1562	5' UST	108
2247	1563	RACE sequence tag	35
2248	1564	3' RST	35
2249	1565	5' RST	108
2250	1566	UST match	35
2251	1567	RST match	35
2252	1568	primer match	35
2253	1569	miRNA antiguide	35
2254	1569	miRNA passenger strand	35
2255	1569	miRNA star	35
2256	1570	trans-splice junction	35
2257	1572	natural plasmid	35
2258	1573	gene trap construct	35
2259	1574	promoter trap construct	35
2260	1575	enhancer trap construct	35
2261	1576	PAC end	35
2262	1577	Random Amplification Polymorphic DNA	35
2263	1578	shadow enhancer	35
2264	1579	single nucleotide variant	35
2265	1580	X element combinatorial repeat	35
2266	1581	Y' element	35
2267	1582	standard draft	35
2268	1583	high quality draft	35
2269	1584	improved high quality draft	35
2270	1585	annotation directed improvement	35
2271	1586	non contiguous finished	35
2272	1587	finished	35
2273	1587	finished genome	35
2274	1588	intronic regulatory region	35
2275	1589	CDEI	35
2276	1589	Centromere DNA Element I	35
2277	1590	CDEII	35
2278	1590	centromere DNA Element II	35
2279	1591	CDEIII	35
2280	1591	centromere DNA Element III	35
2281	1592	telomeric repeat	35
2282	1593	X element	108
2283	1594	YAC end	35
2284	1595	whole genome sequence status	35
2285	1596	heritable phenotypic marker	35
2286	1596	phenotypic marker	35
2287	1597	peptide collection	35
2288	1597	peptide set	35
2289	1598	high identity region	35
2290	1599	processed transcript	35
2291	1604	alteration attribute	35
2292	1605	chromosomal variation attribute	35
2293	1608	insertion attribute	35
2294	1613	inversion attribute	35
2295	1616	translocation attribute	35
2296	1623	localization signal	108
2297	1623	peptide localization signal	35
2298	1624	NLS	35
2299	1625	endosomal localization signal	35
2300	1626	lysosomal localization signal	35
2301	1627	NES	35
2302	1627	nuclear export signal	35
2303	1628	recombination signal sequence	108
2304	1629	cryptic splice signal	108
2305	1629	cryptic splice site	35
2306	1631	P element	108
2307	1632	regulatory region	35
2308	1633	4.5S snRNA primary transcript	35
2309	1633	U14 snoRNA primary transcript	35
2310	1634	methylation guide snoRNA	35
2311	1635	rRNA cleavage RNA	35
2312	1636	exon of single exon gene	35
2313	1636	single_exon	108
2314	1636	singleton exon	35
2315	1637	cassette array member	35
2316	1638	gene cassette member	35
2317	1639	gene subarray member	35
2318	1640	primer binding site	35
2319	1641	gene array	35
2320	1642	gene subarray	35
2321	1643	gene cassette	35
2322	1644	gene cassette array	35
2323	1645	gene group	35
2324	1646	selenocysteine tRNA primary transcript	35
2325	1647	selenocysteinyl tRNA	35
2326	1647	selenocysteinyl-transfer ribonucleic acid	35
2327	1647	selenocysteinyl-transfer RNA	35
2328	1648	syntenic region	35
2329	1649	biochemical motif	35
2330	1649	biochemical region of peptide	35
2331	1649	biochemical_region	108
2332	1650	molecular contact region	108
2333	1651	disordered region	108
2334	1651	intrinsically unstructured polypeptide region	35
2335	1652	catmat-3l	35
2336	1653	catmat-4l	35
2337	1654	catmat-3r	35
2338	1655	catmat-4r	35
2339	1656	alpha beta motif	35
2340	1657	lipoprotein signal peptide	35
2341	1657	prokaryotic membrane lipoprotein lipid attachment site	35
2342	1658	no output	35
2343	1659	cleaved peptide region	35
2344	1660	coil	108
2345	1660	peptide coil	35
2346	1660	random coil	108
2347	1661	hydropathic	108
2348	1661	hydrophobic region of peptide	108
2349	1661	hydrophobic_region	35
2350	1661	hydrophobicity	108
2351	1662	N-region	108
2352	1663	C-region	108
2353	1664	central hydrophobic region of signal peptide	35
2354	1664	central_hydrophobic_region	108
2355	1664	H-region	108
2356	1665	motif	108
2357	1666	binding	108
2358	1666	polypeptide binding motif	35
2359	1667	catalytic_motif	108
2360	1667	polypeptide catalytic motif	35
2361	1668	polypeptide DNA contact	35
2362	1669	polypeptide conserved region	35
2363	1671	complex substitution	35
2364	1672	point mutation	35
2365	1674	pyrimidine transition	35
2366	1675	C to T transition	35
2367	1676	C to T transition at pCpG site	35
2368	1677	T to C transition	35
2369	1678	purine transition	35
2370	1679	A to G transition	35
2371	1680	G to A transition	35
2372	1682	pyrimidine to purine transversion	35
2373	1683	C to A transversion	35
2374	1684	C to G transversion	35
2375	1685	T to A transversion	35
2376	1686	T to G transversion	35
2377	1687	purine to pyrimidine transversion	35
2378	1688	A to C transversion	35
2379	1689	A to T transversion	35
2380	1690	G to C transversion	35
2381	1691	G to T transversion	35
2382	1692	intrachromosomal mutation	35
2383	1693	(bacteria)&ampDgr;	108
2384	1693	(Drosophila)Df	108
2385	1693	(fungi)D	108
2386	1693	chromosomal deletion	35
2387	1693	deficiency	35
2388	1694	(bacteria)IN	108
2389	1694	(Drosophila)In	108
2390	1694	(fungi)In	108
2391	1694	chromosomal inversion	35
2392	1695	interchromosomal mutation	35
2393	1697	nucleotide duplication	35
2394	1697	nucleotide_duplication	108
2395	1699	(Drosophila)Dp	108
2396	1699	(fungi)Dp	108
2397	1699	chromosomal duplication	35
2398	1700	intrachromosomal duplication	35
2399	1701	direct tandem duplication	35
2400	1702	inverted tandem duplication	35
2401	1702	mirror duplication	108
2402	1703	(Drosophila)Tp	108
2403	1703	intrachromosomal transposition	35
2404	1704	compound chromosome	35
2405	1705	centric-fusion translocations	35
2406	1705	Robertsonian fusion	35
2407	1705	whole-arm translocations	35
2408	1706	(Drosophila)T	108
2409	1706	(fungi)T	108
2410	1706	chromosomal translocation	35
2411	1707	(Drosophila)R	108
2412	1707	(fungi)C	108
2413	1707	ring chromosome	35
2414	1708	pericentric inversion	35
2415	1709	paracentric inversion	35
2416	1710	reciprocal chromosomal translocation	35
2417	1711	mutation affecting transcript	35
2418	1711	mutation causing partially characterised change in transcript	108
2419	1711	mutation causing uncharacterised change in transcript	108
2420	1711	sequence variant causing partially characterised change in transcript	35
2421	1711	sequence variant causing uncharacterised change in transcript	35
2422	1711	sequence variation affecting transcript	35
2423	1711	sequence_variant_causing_partially_characterised_change_in_transcript	35
2424	1711	sequence_variant_causing_uncharacterised_change_in_transcript	35
2425	1712	mutation causing no change in transcript	108
2426	1712	sequence variant causing no change in transcript	35
2427	1713	mutation affecting complex change in transcript	35
2428	1713	sequence variation affecting complex change in transcript	35
2429	1714	mutation affecting coding sequence	35
2430	1714	sequence variation affecting coding sequence	108
2431	1715	mutation causing initiator codon change in transcript	108
2432	1715	sequence variant causing initiator codon change in transcript	35
2433	1716	mutaton causing amino acid coding codon change in transcript	108
2434	1716	sequence variant causing amino acid coding codon change in transcript	35
2435	1717	mutation causing synonymous codon change in transcript	108
2436	1717	sequence variant causing synonymous codon change in transcript	35
2437	1718	mutation causing non synonymous codon change in transcript	108
2438	1718	non-synonymous codon change in transcript	35
2439	1718	sequence variant causing non synonymous codon change in transcript	35
2440	1719	mutation causing missense codon change in transcript	108
2441	1719	sequence variant causing missense codon change in transcript	35
2442	1720	mutation causing conservative missense codon change in transcript	108
2443	1720	sequence variant causing conservative missense codon change in transcript	35
2444	1721	mutation causing nonconservative missense codon change in transcript	108
2445	1721	sequence variant causing nonconservative missense codon change in transcript	35
2446	1722	mutation causing nonsense codon change in transcript	108
2447	1722	sequence variant causing nonsense codon change in transcript	35
2448	1723	mutation causing terminator codon change in transcript	108
2449	1723	sequence variant causing terminator codon change in transcript	35
2450	1724	mutation affecting reading frame	35
2451	1724	sequence variation affecting reading frame	108
2452	1725	frameshift mutation	35
2453	1725	frameshift sequence variation	108
2454	1725	out of frame mutation	108
2455	1726	plus 1 frameshift mutation	35
2456	1726	sequence variant causing plus 1 frameshift mutation	35
2457	1727	minus 1 frameshift mutation	35
2458	1727	sequence variant causing minus 1 frameshift	35
2459	1728	plus 2 frameshift mutation	35
2460	1728	sequence variant causing plus 2 frameshift	35
2461	1729	minus 2 frameshift mutation	35
2462	1729	sequence variant causing minus 2 frameshift	35
2463	1730	mutation affecting transcript processing	108
2464	1730	sequence variant affecting transcript processing	35
2465	1731	mutation affecting splicing	108
2466	1731	sequence variant affecting splicing	35
2467	1732	mutation affecting splice donor	108
2468	1732	sequence variant affecting splice donor	108
2469	1732	splice donor mutation	35
2470	1733	mutation affecting splicing	108
2471	1733	sequence variant affecting splice acceptor	108
2472	1733	splice acceptor mutation	35
2473	1734	cryptic splice activator sequence variant	35
2474	1734	mutation causing cryptic splice activator	108
2475	1734	sequence variant causing cryptic splice activator	35
2476	1735	mutation affecting editing	108
2477	1735	sequence variant affecting editing	35
2478	1736	mutation affecting transcription	108
2479	1736	sequence variant affecting transcription	35
2480	1737	mutation decreasing rate of transcription	108
2481	1737	sequence variation decreasing rate of transcription	35
2482	1738	mutation affecting transcript sequence	35
2483	1738	sequence variation affecting transcript sequence	35
2484	1739	mutation increasing rate of transcription	108
2485	1739	sequence variation increasing rate of transcription	35
2486	1740	mutation affecting rate of transcription	108
2487	1740	sequence variant affecting rate of transcription	35
2488	1741	mutation affecting transcript stability	108
2489	1741	sequence variant affecting transcript stability	35
2490	1742	mutation increasing transcript stability	108
2491	1742	sequence variant increasing transcript stability	35
2492	1743	mutation decreasing transcript stability	108
2493	1743	sequence variant decreasing transcript stability	35
2494	1744	mutation affecting level of transcript	108
2495	1744	sequence variation affecting level of transcript	35
2496	1745	mutation decreasing level of transcript	35
2497	1745	sequence variation decreasing level of transcript	108
2498	1746	mutation increasing level of transcript	35
2499	1746	sequence variation increasing level of transcript	35
2500	1747	mutation affecting translational product	108
2501	1747	mutation causing partially characterised change of translational product	108
2502	1747	mutation causing uncharacterised change of translational product	108
2503	1747	sequence variant affecting translational product	35
2504	1747	sequence variant causing partially characterised change of translational product	35
2505	1747	sequence variant causing uncharacterised change of translational product	35
2506	1747	sequence_variant_causing_partially_characterised_change_of_translational_product	35
2507	1747	sequence_variant_causing_uncharacterised_change_of_translational_product	35
2508	1748	mutation causing no change of translational product	108
2509	1748	sequence variant causing no change of translational product	35
2510	1749	mutation causing complex change of translational product	108
2511	1749	sequence variant causing complex change of translational product	35
2512	1750	mutation causing amino acid substitution	108
2513	1750	sequence variant causing amino acid substitution	35
2514	1751	mutation causing conservative amino acid substitution	108
2515	1751	sequence variant causing conservative amino acid substitution	35
2516	1752	mutation causing nonconservative amino acid substitution	108
2517	1752	sequence variant causing nonconservative amino acid substitution	35
2518	1753	mutation causing amino acid insertion	108
2519	1753	sequence variant causing amino acid insertion	35
2520	1754	mutation causing amino acid deletion	108
2521	1754	sequence variant causing amino acid deletion	35
2522	1755	mutation causing polypeptide truncation	108
2523	1755	sequence variant causing polypeptide truncation	35
2524	1756	mutation causing polypeptide elongation	108
2525	1756	sequence variant causing polypeptide elongation	35
2526	1757	mutation causing polypeptide N terminal elongation	35
2527	1757	polypeptide N-terminal elongation	35
2528	1758	mutation causing polypeptide C terminal elongation	35
2529	1758	polypeptide C-terminal elongation	35
2530	1759	mutation affecting level of translational product	108
2531	1759	sequence variant affecting level of translational product	35
2532	1760	mutationdecreasing level of translation product	108
2533	1760	sequence variant decreasing level of translation product	35
2534	1761	mutationt increasing level of translation product	108
2535	1761	sequence variant increasing level of translation product	35
2536	1762	mutation affecting polypeptide amino acid sequence	108
2537	1762	sequence variant affecting polypeptide amino acid sequence	35
2538	1763	inframe polypeptide N-terminal elongation	35
2539	1763	mutation causing inframe polypeptide N terminal elongation	35
2540	1764	mutation causing out of frame polypeptide N terminal elongation	35
2541	1764	out of frame polypeptide N-terminal elongation	35
2542	1765	inframe_polypeptide C-terminal elongation	35
2543	1765	mutaton causing inframe polypeptide C terminal elongation	35
2544	1766	mutation causing out of frame polypeptide C terminal elongation	35
2545	1766	out of frame polypeptide C-terminal elongation	35
2546	1767	frame restoring mutation	35
2547	1767	frame restoring sequence variant	35
2548	1768	mutation affecting 3D structure of polypeptide	108
2549	1768	mutation causing partially characterised 3D structural change	108
2550	1768	mutation causing uncharacterised 3D structural change	108
2551	1768	sequence variant affecting 3D structure of polypeptide	35
2552	1768	sequence variant affecting 3D-structure of polypeptide	35
2553	1768	sequence variant causing partially characterised 3D structural change	35
2554	1768	sequence variant causing uncharacterised 3D structural change	35
2555	1768	sequence_variant_causing_partially_characterised_3D_structural_change	35
2556	1768	sequence_variant_causing_uncharacterised_3D_structural_change	35
2557	1769	mutation causing no 3D structural change	108
2558	1769	sequence variant causing no 3D structural change	35
2559	1770	mutation causing complex 3D structural change	108
2560	1770	sequence variant causing complex 3D structural change	35
2561	1771	mutation causing conformational change	108
2562	1771	sequence variant causing conformational change	35
2563	1772	mutation affecting polypeptide function	108
2564	1772	sequence variant affecting polypeptide function	35
2565	1773	loss of function of polypeptide	108
2566	1773	mutation causing loss of function of polypeptide	108
2567	1773	sequence variant causing loss of function of polypeptide	35
2568	1774	mutation causing inactive ligand binding site	108
2569	1774	sequence variant causing inactive ligand binding site	35
2570	1775	mutation causing inactive catalytic site	108
2571	1775	sequence variant causing inactive catalytic site	35
2572	1776	mutation causing polypeptide localization change	108
2573	1776	sequence variant causing polypeptide localization change	35
2574	1777	mutation causing polypeptide post translational processing change	108
2575	1777	polypeptide post-translational processing affected	35
2576	1777	sequence variant causing polypeptide post translational processing change	35
2577	1778	polypeptide_post-translational_processing_affected	108
2578	1779	mutation causing partial loss of function of polypeptide	108
2579	1779	partial loss of function of polypeptide	35
2580	1779	sequence variant causing partial loss of function of polypeptide	35
2581	1780	gain of function of polypeptide	35
2582	1780	mutation causing gain of function of polypeptide	108
2583	1780	sequence variant causing gain of function of polypeptide	35
2584	1781	mutation affecting transcript secondary structure	108
2585	1781	sequence variant affecting transcript secondary structure	35
2586	1782	mutation causing compensatory transcript secondary structure mutation	108
2587	1782	sequence variant causing compensatory transcript secondary structure mutation	35
2588	1783	sequence variant effect	108
2589	1784	mutation causing polypeptide fusion	108
2590	1784	sequence variant causing polypeptide fusion	35
2591	1785	(Drosophila)A	108
2592	1785	autosynaptic chromosome	35
2593	1786	homo compound chromosome	35
2594	1786	homo-compound chromosome	35
2595	1787	hetero compound chromosome	35
2596	1787	hetero-compound chromosome	35
2597	1788	chromosome fission	35
2598	1789	dexstrosynaptic chromosome	35
2599	1790	laevosynaptic chromosome	35
2600	1791	free duplication	35
2601	1792	(Drosophila)R	108
2602	1792	free ring duplication	35
2603	1793	complex chromosomal mutation	35
2604	1794	(Drosophila)Df	108
2605	1794	(Drosophila)DfT	108
2606	1794	deficient translocation	35
2607	1795	(Drosophila)InT	108
2608	1795	(Drosophila)T	108
2609	1795	inversion cum translocation	35
2610	1796	(Drosophila)bDp	108
2611	1796	bipartite duplication	35
2612	1797	cyclic translocation	35
2613	1798	(Drosophila)bIn	108
2614	1798	bipartite inversion	35
2615	1799	(Drosophila)eDp	108
2616	1799	uninverted insertional duplication	35
2617	1800	(Drosophila)iDp	108
2618	1800	inverted insertional duplication	35
2619	1801	(Drosophila)Dpp	108
2620	1801	insertional duplication	35
2621	1802	(Drosophila)Tp	108
2622	1802	interchromosomal transposition	35
2623	1803	(Drosophila)iTp	108
2624	1803	inverted interchromosomal transposition	35
2625	1804	(Drosophila)eTp	108
2626	1804	uninverted interchromosomal transposition	35
2627	1805	(Drosophila)iTp	108
2628	1805	inverted intrachromosomal transposition	35
2629	1806	(Drosophila)eTp	108
2630	1806	uninverted intrachromosomal transposition	35
2631	1807	(Drosophila)uDp	108
2632	1807	unoriented insertional duplication	35
2633	1808	(Drosophila)uTp	108
2634	1808	unorientated interchromosomal transposition	35
2635	1809	(Drosophila)uTp	108
2636	1809	unorientated intrachromosomal transposition	35
2637	1810	uncharacterised chromosomal mutation	35
2638	1811	(Drosophila)Df	108
2639	1811	(Drosophila)DfIn	108
2640	1811	deficient inversion	35
2641	1812	tandem duplication	35
2642	1813	partially characterised chromosomal mutation	35
2643	1814	mutation affecting gene structure	108
2644	1814	sequence variant affecting gene structure	35
2645	1815	mutation causing gene fusion	108
2646	1815	sequence variant causing gene fusion	35
2647	1816	chromosome number variation	35
2648	1817	chromosome structure variation	35
2649	1818	mutation causes exon loss	108
2650	1818	sequence variant causes exon loss	35
2651	1819	mutation causes intron gain	108
2652	1819	sequence variant causes intron gain	35
2653	1820	sequence variant causing cryptic splice donor activation	35
2654	1821	sequence variant causing cryptic splice acceptor activation	35
2655	1822	alternatively spliced transcript	35
2656	1823	encodes 1 polypeptide	35
2657	1824	encodes greater than 1 polypeptide	35
2658	1825	encodes different polypeptides different stop	35
2659	1826	encodes overlapping peptides different start	35
2660	1827	encodes disjoint polypeptides	35
2661	1828	encodes overlapping polypeptides different start and stop	35
2662	1830	encodes overlapping peptides	35
2663	1832	dicistronic primary transcript	35
2664	1833	member of regulon	35
2665	1834	alternatively_spliced_transcript_encoding_greater_than_1_polypeptide_different_start_codon_different_stop_codon_coding_regions_non-overlapping	108
2666	1835	CDS independently known	35
2667	1836	orphan CDS	35
2668	1837	CDS supported by domain match data	35
2669	1838	CDS supported by sequence similarity data	35
2670	1839	CDS predicted	35
2671	1841	CDS supported by EST or cDNA data	35
2672	1842	internal Shine Dalgarno sequence	35
2673	1842	internal Shine-Dalgarno sequence	35
2674	1843	recoded mRNA	35
2675	1844	minus 1 translationally frameshifted	35
2676	1845	plus 1 translationally frameshifted	35
2677	1846	mRNA recoded by translational bypass	35
2678	1847	mRNA recoded by codon redefinition	35
2679	1850	recoding stimulatory region	35
2680	1850	recoding stimulatory signal	35
2681	1851	4bp start codon	35
2682	1851	four bp start codon	35
2683	1853	archaeal intron	35
2684	1854	pre-tRNA intron	35
2685	1854	tRNA intron	35
2686	1855	CTG start codon	35
2687	1856	SECIS element	35
2688	1858	three prime recoding site	35
2689	1859	three prime stem loop structure	35
2690	1860	five prime recoding site	35
2691	1861	flanking three prime quadruplet recoding signal	35
2692	1862	UAG stop codon signal	35
2693	1863	UAA stop codon signal	35
2694	1865	UGA stop codon signal	35
2695	1866	three prime repeat recoding signal	35
2696	1867	distant three prime recoding signal	35
2697	1868	stop codon signal	35
2698	1869	accession	108
2699	1869	databank entry	35
2700	1870	gene segment	35
2701	1884	monadic_form_of	35
2702	1895	m	35
2703	1896	kg	35
2704	1897	s	35
2705	1898	A	35
2706	1899	K	35
2707	1900	mol	35
2708	1901	cd	35
2709	1902	cm	35
2710	1903	mm	35
2711	1904	um	35
2712	1905	nm	35
2713	1906	A	35
2714	1907	pm	35
2715	1908	g	35
2716	1909	mg	35
2717	1910	ug	35
2718	1911	ng	35
2719	1912	pg	35
2720	1913	fg	35
2721	1914	C	35
2722	1915	ms	35
2723	1916	us	35
2724	1917	ps	35
2725	1918	min	35
2726	1919	h	35
2727	1924	mA	35
2728	1925	uA	35
2729	1926	umol	35
2730	1927	mmol	35
2731	1928	nmol	35
2732	1929	pmol	35
2733	1930	fmol	35
2734	1931	amol	35
2735	1939	mass per unit volume	35
2736	1941	mass per unit area unit	35
2737	1949	M	35
2738	1950	mM	35
2739	1951	uM	35
2740	1952	nM	35
2741	1953	pM	35
2742	1955	m	35
2743	1956	mm	35
2744	1957	um	35
2745	1958	nm	35
2746	1959	pm	35
2747	1960	fM	35
2748	1962	N	35
2749	1963	(x)	35
2750	1963	chi	35
2751	1964	m/s^[2]	35
2752	1965	alpha	35
2753	1965	rad/s^[2]	35
2754	1966	rad/s	35
2755	1967	m^[2]	35
2756	1968	cm^[2]	35
2757	1969	mm^[2]	35
2758	1970	kg/m^[3]	35
2759	1971	g/cm^[3]	35
2760	1972	ug/L	35
2761	1973	cd/m^[2]	35
2762	1974	Body Mass Index (BMI)	35
2763	1974	kg/m^[2]	35
2764	1975	kg/mol	35
2765	1976	g/mol	35
2766	1977	m^[3]/mol	35
2767	1978	cm^[3]/mol	35
2768	1979	kg.m/s	35
2769	1980	1/s	35
2770	1981	m^[3]/kg	35
2771	1982	m/s	35
2772	1984	m^[3]	35
2773	1985	cc	35
2774	1985	cm^3	35
2775	1986	ml	35
2776	1987	l	35
2777	1987	L	35
2778	1988	dm^[3]	35
2779	1989	ul	35
2780	1990	nl	35
2781	1991	pl	35
2782	1992	fl	35
2783	1994	Hz	35
2784	1996	N	35
2785	1998	Pa	35
2786	2000	J	35
2787	2002	W	35
2788	2004	lx	35
2789	2006	lm	35
2790	2008	kat	35
2791	2011	rad	35
2792	2013	sr	35
2793	2020	Bq	35
2794	2021	Ci	35
2795	2022	Gy	35
2796	2023	rad	35
2797	2024	R	35
2798	2025	Sv	35
2799	2026	mSv	35
2800	2027	uSv	35
2801	2028	rem	35
2802	2029	uGy	35
2803	2030	mGy	35
2804	2031	nGy	35
2805	2032	nSv	35
2806	2033	mCi	35
2807	2034	uCi	35
2808	2035	dpm	35
2809	2036	cpm	35
2810	2038	ns	35
2811	2041	ft-c	35
2812	2043	W/m^[2]	35
2813	2044	einstein/sm^[2]	35
2814	2044	mole per second and square meter mol/sm^2	35
2815	2046	W/sr m^[2]	35
2816	2048	micromole per second and square meter mmol/sm^2	35
2817	2048	umicroeinstein/sm^[2]	35
2818	2050	W/sr	35
2819	2051	w/w	35
2820	2051	weight-weight percentage	35
2821	2052	(w/v)	35
2822	2052	weight-volume percentage	35
2823	2053	% (v/v)	35
2824	2055	10^[-2]	35
2825	2055	pph	35
2826	2056	10^[-3]	35
2827	2056	ppth	35
2828	2057	10^[-6]	35
2829	2057	ppm	35
2830	2058	10^[-9]	35
2831	2058	ppb	35
2832	2059	10^[-12]	35
2833	2059	ppt	35
2834	2060	10^[-15]	35
2835	2060	ppq	35
2836	2061	g/ml	35
2837	2062	kg/L	35
2838	2063	g/L	35
2839	2064	mg/ml	35
2840	2066	U/ml	35
2841	2067	U/l	35
2842	2069	U	35
2843	2072	kg/m	35
2844	2075	%	35
2845	2077	count	35
2846	2083	F	35
2847	2085	l/kg	35
2848	2086	ml/kg	35
2849	2087	ul/kg	35
2850	2089	cells per ml	35
2851	2091	kat/m^[3]	35
2852	2092	kat/l	35
2853	2094	ml/m^[3]	35
2854	2095	ml/l	35
2855	2096	g/dl	35
2856	2097	dl	35
2857	2098	cfu	35
2858	2099	pfu	35
2859	2101	cfu/ml	35
2860	2103	pfu/ml	35
2861	2104	dps	35
2862	2106	V	35
2863	2108	C	35
2864	2109	Da	35
2865	2109	u	35
2866	2109	unified atomic mass unit	35
2867	2110	kDa	35
2868	2111	Wh	35
2869	2114	V s	35
2870	2114	volt-second	108
2871	2114	Wb	35
2872	2115	B	35
2873	2116	T	35
2874	2116	Wb/m2	35
2875	2117	Vh	35
2876	2118	kVh	35
2877	2121	B	35
2878	2122	kB	35
2879	2123	MB	35
2880	2128	dpi	35
2881	2129	micrometer	35
2882	2130	pixel density	35
2883	2130	ppi	35
2884	2132	bp	35
2885	2133	KiB	35
2886	2134	MiB	35
2887	2135	mV	35
2888	2136	kV	35
2889	2137	uV	35
2890	2138	nV	35
2891	2139	pV	35
2892	2140	MV	35
2893	2142	N/m	35
2894	2143	dyn/cm	35
2895	2145	Pa  s	35
2896	2146	dyne s/cm^2	35
2897	2146	P	35
2898	2147	dB	35
2899	2152	A V^-1	35
2900	2152	mho	108
2901	2152	S	108
2902	2153	W/m K	35
2903	2154	electron volt	35
2904	2154	eV	35
2905	2155	E-field strength	35
2906	2156	V/m	35
2907	2157	AU	35
2908	2159	uL/min	35
\.


--
-- Data for Name: db; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY db (db_id, name, description, urlprefix, url) FROM stdin;
1	null	a fake database for local items	\N	\N
2	GFF_source	A collection of sources (ie, column 2) from GFF files	\N	\N
3	ATCC	\N	\N	\N
4	DB:refseq	\N	\N	\N
5	DB:genbank	\N	\N	\N
6	DB:EMBL	\N	\N	\N
7	DB:TIGR	\N	\N	\N
8	DB:ucsc	\N	\N	\N
9	DB:ucla	\N	\N	\N
10	DB:SGD	\N	\N	\N
11	DB:PFAM	\N	\N	\N
12	DB:SUPERFAMILY	\N	\N	\N
13	DB:PROFILE	\N	\N	\N
14	DB:PRODOM	\N	\N	\N
15	DB:PRINTS	\N	\N	\N
16	DB:SMART	\N	\N	\N
17	DB:TIGRFAMs	\N	\N	\N
18	DB:PIR	\N	\N	\N
19	DB:Affymetrix_U133	\N	\N	\N
20	DB:Affymetrix_U133PLUS	\N	\N	\N
21	DB:Affymetrix_U95	\N	\N	\N
22	DB:LocusLink	\N	\N	\N
23	DB:RefSeq_protein	\N	\N	\N
24	DB:GenBank_protein	\N	\N	\N
25	DB:OMIM	\N	\N	\N
26	DB:Swiss	\N	\N	\N
27	DB:RefSNP	\N	\N	\N
28	DB:TSC	\N	\N	\N
29	DB:GR	Gramene	\N	\N
30	DB:uniprot	UniProt/TrEMBL	http://us.expasy.org/cgi-bin/niceprot.pl?	\N
31	DB:refseq:mrna	RefSeq mRNA	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=nucleotide&dopt=GenBank&term=	\N
32	DB:refseq:protein	RefSeq Protein	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=protein&dopt=GenBank&term=	\N
33	DB:unigene	Unigene	http://www.ncbi.nih.gov/entrez/query.fcgi?db=unigene&cmd=search&term=	\N
34	DB:omim	OMIM	http://www.ncbi.nlm.nih.gov/entrez/dispomim.cgi?id=	\N
35	DB:locuslink	LocusLink	http://www.ncbi.nlm.nih.gov/LocusLink/LocRpt.cgi?l=	\N
36	DB:genbank:mrna	GenBank mRNA	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=nucleotide&dopt=GenBank&term=	\N
37	DB:genbank:protein	GenBank Protein	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=protein&dopt=GenBank&term=	\N
38	DB:swissprot:display	SwissProt	http://us.expasy.org/cgi-bin/niceprot.pl?	\N
39	DB:pfam	Pfam	http://www.sanger.ac.uk/cgi-bin/Pfam/dql.pl?query=	\N
40	OBO_REL	\N	\N	\N
41	_global	\N	\N	\N
42	internal	\N	\N	\N
43	owl	\N	\N	\N
44	SO	\N	\N	\N
45	URL	\N	\N	\N
46	PMID	\N	\N	\N
47	ISBN	\N	\N	\N
48	http	\N	\N	\N
49	FB	\N	\N	\N
50	GOC	\N	\N	\N
51	SGD	\N	\N	\N
52	WB	\N	\N	\N
53	CHEBI	\N	\N	\N
54	RSC	\N	\N	\N
55	EBIBS	\N	\N	\N
56	BS	\N	\N	\N
57	doi	\N	\N	\N
58	Indiana	\N	\N	\N
59	issn	\N	\N	\N
60	Phigo	\N	\N	\N
61	phigo	\N	\N	\N
62	UniProt	\N	\N	\N
63	Http	\N	\N	\N
64	uniprot	\N	\N	\N
65	xenbase	\N	\N	\N
66	RNAMOD	\N	\N	\N
67	BBOP	\N	\N	\N
68	MOD	\N	\N	\N
69	EBI	\N	\N	\N
70	RFAM	\N	\N	\N
71	modENCODE	\N	\N	\N
72	ZFIN	\N	\N	\N
73	DOI	\N	\N	\N
74	JAX	\N	\N	\N
75	MGI	\N	\N	\N
76	UO	\N	\N	\N
77	Wikipedia	\N	\N	\N
78	NIST	\N	\N	\N
79	UOC	\N	\N	\N
80	OCRBS	\N	\N	\N
81	DEFRA	\N	\N	\N
82	ORCBS	\N	\N	\N
83	MGED	\N	\N	\N
84	Webmd	\N	\N	\N
85	Bioedonline	\N	\N	\N
86	WordNet	\N	\N	\N
87	ScienceLobby	\N	\N	\N
88	allnet	\N	\N	\N
89	PATO	\N	\N	\N
90	techtarget	\N	\N	\N
91	PATOC	\N	\N	\N
92	DE	\N	\N	\N
\.


--
-- Data for Name: dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY dbxref (dbxref_id, db_id, accession, version, description) FROM stdin;
1	1	local:null		\N
2	1	local:computer file		\N
3	1	local:glass		\N
4	1	local:photochemical_oligo		\N
5	1	developmental stages:fetus		\N
6	1	developmental stages:neonate		\N
7	1	developmental stages:child		\N
8	1	developmental stages:adult_young		\N
9	1	developmental stages:adult		\N
10	1	developmental stages:adult_old		\N
11	1	local:survival_time		\N
12	1	Statistical Terms:n		\N
13	1	Statistical Terms:minimum		\N
14	1	Statistical Terms:maximum		\N
15	1	Statistical Terms:modality		\N
16	1	Statistical Terms:modality p		\N
17	1	Statistical Terms:mean		\N
18	1	Statistical Terms:median		\N
19	1	Statistical Terms:mode		\N
20	1	Statistical Terms:quartile 1		\N
21	1	Statistical Terms:quartile 3		\N
22	1	Statistical Terms:skewness		\N
23	1	Statistical Terms:kurtosis		\N
24	1	Statistical Terms:chi square p		\N
25	1	Statistical Terms:standard deviation		\N
26	1	Statistical Terms:expectation maximization gaussian mean		\N
27	1	Statistical Terms:expectation maximization p		\N
28	1	Statistical Terms:histogram		\N
29	42	comment		\N
30	42	is_anonymous		\N
31	40	is_a		\N
32	42	is_reflexive		\N
33	42	is_anti_symmetric		\N
34	42	is_transitive		\N
35	42	exact		\N
36	40	0000001		\N
37	43	subClassOf		\N
38	40	part_of		\N
39	40	0000002		\N
40	40	has_part		\N
41	40	0000003		\N
42	40	integral_part_of		\N
43	40	0000004		\N
44	40	has_integral_part		\N
45	40	0000005		\N
46	40	proper_part_of		\N
47	40	0000006		\N
48	40	has_proper_part		\N
49	40	0000007		\N
50	40	located_in		\N
51	40	0000008		\N
52	40	location_of		\N
53	40	0000009		\N
54	40	contained_in		\N
55	40	0000010		\N
56	40	contains		\N
57	40	0000011		\N
58	40	adjacent_to		\N
59	40	0000012		\N
60	40	transformation_of		\N
61	40	0000013		\N
62	40	transformed_into		\N
63	40	0000014		\N
64	40	derives_from		\N
65	40	0000015		\N
66	40	derived_into		\N
67	40	0000016		\N
68	40	preceded_by		\N
69	40	0000017		\N
70	40	precedes		\N
71	40	0000018		\N
72	40	has_participant		\N
73	40	0000019		\N
74	40	participates_in		\N
75	40	0000020		\N
76	40	has_agent		\N
77	40	0000021		\N
78	40	agent_in		\N
79	40	0000022		\N
80	40	instance_of		\N
81	40	0000023		\N
82	40	has_improper_part		\N
83	40	0000024		\N
84	40	improper_part_of		\N
85	40	0000025		\N
86	40	relationship		\N
87	41	adjacent_to		\N
88	41	associated_with		\N
89	41	complete_evidence_for_feature		\N
90	41	derives_from		\N
91	41	edited_from		\N
92	41	edited_to		\N
93	41	evidence_for_feature		\N
94	41	exemplar_of		\N
95	41	genome_of		\N
96	41	guided_by		\N
97	41	guides		\N
98	41	has_genome_location		\N
99	41	has_intergral_part		\N
100	41	has_origin		\N
101	41	has_part		\N
102	41	has_quality		\N
103	41	homologous_to		\N
104	42	is_symmetric		\N
105	41	integral_part_of		\N
106	41	member_of		\N
107	41	non_functional_homolog_of		\N
108	41	orthologous_to		\N
109	41	paralogous_to		\N
110	41	part_of		\N
111	41	partial_evidence_for_feature		\N
112	41	position_of		\N
113	41	processed_from		\N
114	41	processed_into		\N
115	41	recombined_from		\N
116	41	recombined_to		\N
117	41	regulated_by		\N
118	41	sequence_of		\N
119	41	similar_to		\N
120	41	trans_spliced_from		\N
121	41	trans_spliced_to		\N
122	41	transcribed_from		\N
123	41	transcribed_to		\N
124	41	translates_to		\N
125	41	translation_of		\N
126	41	variant_of		\N
127	44	0000000		\N
128	44	0000001		\N
129	44	ke		\N
130	44	0000002		\N
131	44	0000003		\N
132	45	http://www.ncbi.nlm.nih.gov/pubmed/7919797?dopt=Abstract		\N
133	45	http://en.wikipedia.org/wiki/G-quadruplex		\N
134	44	0000004		\N
135	44	0000005		\N
136	45	http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html		\N
137	45	http://en.wikipedia.org/wiki/Satellite_DNA		\N
138	44	0000006		\N
139	42	related		\N
140	45	http://en.wikipedia.org/wiki/RAPD		\N
141	44	0000007		\N
142	44	ls		\N
143	44	0000008		\N
144	44	0000009		\N
145	44	0000010		\N
146	44	0000011		\N
147	44	0000012		\N
148	45	http://www.ebi.ac.uk/embl/WebFeat/align/scRNA_s.html		\N
149	44	0000013		\N
150	44	0000014		\N
151	46	12651739		\N
152	44	0000015		\N
153	46	12651739:12537576		\N
154	42	narrow		\N
155	44	0000016		\N
156	44	0000017		\N
157	44	0000018		\N
158	47	038752046		\N
159	45	http://en.wikipedia.org/wiki/Linkage_group		\N
160	44	0000020		\N
161	44	0000021		\N
162	44	0000022		\N
163	44	0000023		\N
164	45	http://en.wikipedia.org/wiki/K-turn		\N
165	44	0000024		\N
166	45	http://proton.chem.yale.edu/pdf/7897662.pdf		\N
167	44	0000025		\N
168	44	0000026		\N
169	44	0000027		\N
170	44	0000028		\N
171	45	http://en.wikipedia.org/wiki/Base_pair		\N
172	44	0000029		\N
173	46	12177293		\N
174	44	0000030		\N
175	44	0000031		\N
176	45	http://aptamer.icmb.utexas.edu		\N
177	45	http://en.wikipedia.org/wiki/Aptamer		\N
178	44	0000032		\N
179	48	aptamer.icmb.utexas.edu		\N
180	44	0000033		\N
181	44	0000034		\N
182	45	http://www.gene-tools.com/		\N
183	44	0000035		\N
184	46	2820954		\N
185	45	http://en.wikipedia.org/wiki/Riboswitch		\N
186	44	0000036		\N
187	44	ma		\N
188	45	http://en.wikipedia.org/wiki/Matrix_attachment_site		\N
189	44	0000037		\N
190	45	http://en.wikipedia.org/wiki/Locus_control_region		\N
191	44	0000038		\N
192	44	0000039		\N
193	44	0000040		\N
194	44	0000041		\N
195	44	0000042		\N
196	44	0000043		\N
197	44	xp		\N
198	44	0000044		\N
199	44	0000045		\N
200	44	0000046		\N
201	44	0000047		\N
202	44	0000048		\N
203	44	0000049		\N
204	44	0000050		\N
205	44	0000051		\N
206	45	http://en.wikipedia.org/wiki/Hybridization_probe		\N
207	44	0000052		\N
208	44	0000053		\N
209	44	0000054		\N
210	45	http://en.wikipedia.org/wiki/Aneuploid		\N
211	44	0000055		\N
212	45	http://en.wikipedia.org/wiki/Hyperploid		\N
213	44	0000056		\N
214	45	http://en.wikipedia.org/wiki/Hypoploid		\N
215	44	0000057		\N
216	45	http://en.wikipedia.org/wiki/Operator_(biology)#Operator		\N
217	44	0000058		\N
218	44	0000059		\N
219	44	cb		\N
220	44	0000060		\N
221	44	0000061		\N
222	44	0000062		\N
223	49	reference_manual		\N
224	44	0000063		\N
225	44	0000064		\N
226	44	0000065		\N
227	44	0000066		\N
228	44	0000067		\N
229	44	0000068		\N
230	44	0000069		\N
231	44	0000070		\N
232	44	0000071		\N
233	44	0000072		\N
234	44	0000073		\N
235	44	0000074		\N
236	44	0000075		\N
237	44	0000076		\N
238	44	0000077		\N
239	45	http://en.wikipedia.org/wiki/Antisense		\N
240	44	0000078		\N
241	44	0000079		\N
242	44	0000080		\N
243	44	0000081		\N
244	44	0000082		\N
245	44	0000083		\N
246	44	0000084		\N
247	44	0000085		\N
248	44	0000086		\N
249	44	0000087		\N
250	45	http://en.wikipedia.org/wiki/Nuclear_gene		\N
251	44	0000088		\N
252	45	http://en.wikipedia.org/wiki/Mitochondrial_gene		\N
253	44	0000089		\N
254	44	0000090		\N
255	44	0000091		\N
256	44	0000092		\N
257	44	0000093		\N
258	44	0000094		\N
259	44	0000095		\N
260	44	0000096		\N
261	44	0000097		\N
262	44	0000098		\N
263	44	0000099		\N
264	44	0000100		\N
265	44	0000101		\N
266	45	http://www.sci.sdsu.edu/~smaloy/Glossary/T.html		\N
267	45	http://en.wikipedia.org/wiki/Transposable_element		\N
268	44	0000102		\N
269	44	0000103		\N
270	44	0000104		\N
271	44	0000358		\N
272	45	http://en.wikipedia.org/wiki/Polypeptide		\N
273	44	0000105		\N
274	45	http://www.medterms.com/script/main/art.asp?articlekey=5152		\N
275	44	0000106		\N
276	44	0000107		\N
277	44	0000108		\N
278	44	0000109		\N
279	44	0000110		\N
280	44	0000111		\N
281	44	0000112		\N
282	45	http://www.ornl.gov/TechResources/Human_Genome/publicat/primer2001/glossary.html		\N
283	45	http://en.wikipedia.org/wiki/Primer_(molecular_biology)		\N
284	44	0000113		\N
285	44	0000114		\N
286	44	0000115		\N
287	44	0000116		\N
288	44	0000117		\N
289	44	0000118		\N
290	44	0000119		\N
291	44	0000120		\N
292	44	0000121		\N
293	45	http://mged.sourceforge.net/ontologies/MGEDontology.php		\N
294	44	0000122		\N
295	44	0000123		\N
296	44	0000124		\N
297	44	0000125		\N
298	44	0000126		\N
299	44	0000127		\N
300	44	0000128		\N
301	44	0000129		\N
302	44	0000130		\N
303	44	0000131		\N
304	44	0000132		\N
305	44	0000133		\N
306	44	0000134		\N
307	48	http://en.wikipedia.org/wiki/Genomic_imprinting		\N
308	44	0000135		\N
309	44	0000136		\N
310	44	0000137		\N
311	44	0000138		\N
312	44	0000139		\N
313	44	0000140		\N
314	44	as		\N
315	45	http://en.wikipedia.org/wiki/Attenuator		\N
316	44	0000141		\N
317	45	http://en.wikipedia.org/wiki/Terminator_(genetics)		\N
318	44	0000142		\N
319	44	0000143		\N
320	44	0000144		\N
321	44	0000145		\N
322	44	0000146		\N
323	44	0000147		\N
324	45	http://en.wikipedia.org/wiki/Exon		\N
325	44	0000148		\N
326	44	0000149		\N
327	45	http://en.wikipedia.org/wiki/Contig		\N
328	44	0000150		\N
329	44	rd		\N
330	44	0000151		\N
331	48	http://en.wikipedia.org/wiki/Clone_(genetics)		\N
332	44	0000152		\N
333	44	0000153		\N
334	44	0000154		\N
335	45	http://en.wikipedia.org/wiki/P1-derived_artificial_chromosome		\N
336	44	0000155		\N
337	44	0000156		\N
338	45	http://en.wikipedia.org/wiki/Cosmid		\N
339	44	0000157		\N
340	45	http://en.wikipedia.org/wiki/Phagemid		\N
341	44	0000158		\N
342	45	http://en.wikipedia.org/wiki/Fosmid		\N
343	44	0000159		\N
344	44	1000033		\N
345	45	http://en.wikipedia.org/wiki/Nucleotide_deletion		\N
346	44	0000160		\N
347	47	0-1767-2380-8		\N
348	44	0000161		\N
349	44	0000162		\N
350	44	cjm		\N
351	45	http://en.wikipedia.org/wiki/Splice_site		\N
352	44	0000163		\N
353	45	http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html		\N
354	44	0000164		\N
355	44	0000165		\N
356	45	http://en.wikipedia.org/wiki/Enhancer_(genetics)		\N
357	44	0000166		\N
358	44	0000167		\N
359	44	regcreative		\N
360	45	http://en.wikipedia.org/wiki/Promoter		\N
361	44	0000168		\N
362	44	0000169		\N
363	44	0000170		\N
364	44	0000171		\N
365	44	0000172		\N
366	45	http://en.wikipedia.org/wiki/CAAT_box		\N
367	44	0000173		\N
368	44	0000174		\N
369	45	http://en.wikipedia.org/wiki/TATA_box		\N
370	44	0000175		\N
371	45	http://en.wikipedia.org/wiki/Pribnow_box		\N
372	44	0000176		\N
373	44	0000177		\N
374	44	0000178		\N
375	45	http://en.wikipedia.org/wiki/Operon		\N
376	44	0000179		\N
377	44	0000180		\N
378	45	http://www.dddmag.com/Glossary.aspx#r		\N
379	45	http://en.wikipedia.org/wiki/Retrotransposon		\N
380	44	0000181		\N
381	44	0000182		\N
382	44	0000183		\N
383	44	0000184		\N
384	46	9428511		\N
385	44	0000185		\N
386	45	http://en.wikipedia.org/wiki/Primary_transcript		\N
387	44	0000186		\N
388	44	0000187		\N
389	44	0000188		\N
390	45	http://en.wikipedia.org/wiki/Intron		\N
391	44	0000189		\N
392	44	0000190		\N
393	44	0000191		\N
394	44	0000192		\N
395	44	0000193		\N
396	50	pj		\N
397	45	http://en.wikipedia.org/wiki/Restriction_fragment_length_polymorphism		\N
398	44	0000194		\N
399	44	0000195		\N
400	44	0000196		\N
401	44	0000197		\N
402	44	0000198		\N
403	44	0000199		\N
404	44	0000200		\N
405	44	0000201		\N
406	46	10373547		\N
407	44	0000202		\N
408	44	0000203		\N
409	44	0000204		\N
410	45	http://en.wikipedia.org/wiki/5'_UTR		\N
411	44	0000205		\N
412	45	http://en.wikipedia.org/wiki/Three_prime_untranslated_region		\N
413	44	0000206		\N
414	45	http://en.wikipedia.org/wiki/Short_interspersed_nuclear_element		\N
415	44	0000207		\N
416	44	0000208		\N
417	45	http://www.genetics.org/cgi/reprint/156/4/1983.pdf		\N
418	44	0000209		\N
419	44	0000210		\N
420	44	0000211		\N
421	44	0000212		\N
422	44	0000213		\N
423	44	0000214		\N
424	44	0000215		\N
425	44	0000216		\N
426	44	0000217		\N
427	44	0000218		\N
428	44	0000219		\N
429	44	0000220		\N
430	44	0000221		\N
431	44	0000222		\N
432	44	0000223		\N
433	44	0000224		\N
434	44	0000225		\N
435	44	0000226		\N
436	44	0000227		\N
437	44	0000228		\N
438	44	0000229		\N
439	44	0000230		\N
440	44	0000231		\N
441	44	0000232		\N
442	44	0000233		\N
443	45	http://en.wikipedia.org/wiki/Mature_transcript		\N
444	44	0000234		\N
445	45	http://en.wikipedia.org/wiki/MRNA		\N
446	44	0000235		\N
447	44	0000236		\N
448	51	rb		\N
449	44	0000237		\N
450	44	0000238		\N
451	44	0000239		\N
452	44	0000240		\N
453	44	0000241		\N
454	44	0000242		\N
455	44	0000243		\N
456	45	http://en.wikipedia.org/wiki/Internal_ribosome_entry_site		\N
457	44	0000244		\N
458	44	0000245		\N
459	44	0000246		\N
460	44	0000247		\N
461	44	0000248		\N
462	44	0000249		\N
463	44	0000250		\N
464	44	0000251		\N
465	44	0000252		\N
466	47	0198506732		\N
467	45	http://en.wikipedia.org/wiki/RRNA		\N
468	44	0000253		\N
469	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00005		\N
470	45	http://en.wikipedia.org/wiki/TRNA		\N
471	44	0000254		\N
472	44	0000255		\N
473	44	0000256		\N
474	44	0000257		\N
475	44	0000258		\N
476	44	0000259		\N
477	44	0000260		\N
478	44	0000261		\N
479	44	0000262		\N
480	44	0000263		\N
481	44	0000264		\N
482	44	0000265		\N
483	44	0000266		\N
484	44	0000267		\N
485	44	0000268		\N
486	44	0000269		\N
487	44	0000270		\N
488	44	0000271		\N
489	44	0000272		\N
490	44	0000273		\N
491	44	0000274		\N
492	46	11733745		\N
493	52	ems		\N
494	45	http://en.wikipedia.org/wiki/SnRNA		\N
495	44	0000275		\N
496	50	kgc		\N
497	45	http://en.wikipedia.org/wiki/SnoRNA		\N
498	44	0000276		\N
499	46	12592000		\N
500	45	http://en.wikipedia.org/wiki/MiRNA		\N
501	44	0000277		\N
502	44	0000278		\N
503	44	0000279		\N
504	44	0000280		\N
505	44	0000281		\N
506	44	0000282		\N
507	44	0000283		\N
508	44	0000284		\N
509	45	http://www.promega.com		\N
510	44	0000285		\N
511	44	0000286		\N
512	45	http://en.wikipedia.org/wiki/Long_terminal_repeat		\N
513	44	0000287		\N
514	45	http://en.wikipedia.org/wiki/Fusion_gene		\N
515	44	0000288		\N
516	44	0000289		\N
517	45	http://www.informatics.jax.org/silver/glossary.shtml		\N
518	45	http://en.wikipedia.org/wiki/Microsatellite		\N
519	44	0000290		\N
520	44	0000291		\N
521	44	0000292		\N
522	44	0000293		\N
523	44	0000294		\N
524	45	http://en.wikipedia.org/wiki/Inverted_repeat		\N
525	44	0000295		\N
526	44	0000296		\N
527	45	http://en.wikipedia.org/wiki/Origin_of_replication		\N
528	44	0000297		\N
529	45	http://en.wikipedia.org/wiki/D_loop		\N
530	44	0000298		\N
531	44	0000299		\N
532	44	0000300		\N
533	44	0000301		\N
534	44	0000302		\N
535	45	http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#		\N
536	44	0000303		\N
537	44	0000304		\N
538	44	0000305		\N
539	44	0000306		\N
540	44	0000307		\N
541	45	http://en.wikipedia.org/wiki/CpG_island		\N
542	44	0000308		\N
543	44	0000309		\N
544	44	0000310		\N
545	44	0000311		\N
546	44	0000312		\N
547	44	0000313		\N
548	44	0000019		\N
549	45	http://en.wikipedia.org/wiki/Stem_loop		\N
550	44	0000314		\N
551	45	http://en.wikipedia.org/wiki/Direct_repeat		\N
552	44	0000315		\N
553	44	0000316		\N
554	44	0000317		\N
555	45	http://seqcore.brcf.med.umich.edu/doc/educ/dnapr/mbglossary/mbgloss.html		\N
556	44	0000318		\N
557	45	http://en.wikipedia.org/wiki/Start_codon		\N
558	44	0000319		\N
559	45	http://en.wikipedia.org/wiki/Stop_codon		\N
560	44	0000320		\N
561	44	0000321		\N
562	44	0000322		\N
563	44	0000323		\N
564	44	0000324		\N
565	44	0000325		\N
566	44	0000326		\N
567	45	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7570003&amp;dopt=Abstract		\N
568	44	0000327		\N
569	44	0000328		\N
570	44	0000329		\N
571	44	0000330		\N
572	45	http://en.wikipedia.org/wiki/Conserved_region		\N
573	44	0000331		\N
574	45	http://www.biospace.com		\N
575	44	0000332		\N
576	44	0000333		\N
577	44	0000334		\N
578	44	0000335		\N
579	44	0000336		\N
580	45	http://en.wikipedia.org/wiki/Pseudogene		\N
581	44	0000337		\N
582	44	0000338		\N
583	45	http://www.pnas.org/cgi/content/full/97/18/10083		\N
584	44	0000339		\N
585	45	http://en.wikipedia.org/wiki/Recombination_hotspot		\N
586	44	0000340		\N
587	45	http://en.wikipedia.org/wiki/Chromosome		\N
588	44	0000341		\N
589	45	http://en.wikipedia.org/wiki/Cytological_band		\N
590	44	0000342		\N
591	44	0000343		\N
592	44	0000344		\N
593	44	0000345		\N
594	44	0000346		\N
595	44	0000347		\N
596	44	0000348		\N
597	53	33696		\N
598	54	cb		\N
599	45	http://en.wikipedia.org/wiki/Nucleic_acid		\N
600	44	0000349		\N
601	44	0000350		\N
602	44	0000351		\N
603	44	0000352		\N
604	44	0000353		\N
605	45	http://en.wikipedia.org/wiki/Sequence_assembly		\N
606	44	0000354		\N
607	44	0000355		\N
608	44	0000356		\N
609	44	0000357		\N
610	44	0000359		\N
611	45	http://en.wikipedia.org/wiki/Floxed		\N
612	44	0000360		\N
613	45	http://www.everythingbio.com/glos/definition.php?word=codon		\N
614	45	http://en.wikipedia.org/wiki/Codon		\N
615	44	0000361		\N
616	44	0000362		\N
617	44	0000363		\N
618	44	0000364		\N
619	44	0000365		\N
620	45	http://en.wikipedia.org/wiki/Integron		\N
621	44	0000366		\N
622	44	0000367		\N
623	44	0000368		\N
624	44	0000369		\N
625	44	0000370		\N
626	44	0000371		\N
627	45	http://www.sci.sdsu.edu/~smaloy/Glossary/C.html		\N
628	44	0000372		\N
629	44	0000373		\N
630	44	0000374		\N
631	45	http://en.wikipedia.org/wiki/Ribozyme		\N
632	44	0000375		\N
633	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00002		\N
634	45	http://en.wikipedia.org/wiki/5.8S_ribosomal_RNA		\N
635	44	0000376		\N
636	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00013		\N
637	45	http://en.wikipedia.org/wiki/6S_RNA		\N
638	44	0000377		\N
639	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00018		\N
640	44	0000378		\N
641	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00014		\N
642	45	http://en.wikipedia.org/wiki/DsrA_RNA		\N
643	44	0000379		\N
644	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00022		\N
645	45	http://en.wikipedia.org/wiki/GcvB_RNA		\N
646	44	0000380		\N
647	45	http://rnaworld.bio.ku.edu/class/RNA/RNA00/RNA_World_3.html		\N
648	45	http://en.wikipedia.org/wiki/Hammerhead_ribozyme		\N
649	44	0000381		\N
650	44	0000382		\N
651	44	0000383		\N
652	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00033		\N
653	45	http://en.wikipedia.org/wiki/MicF_RNA		\N
654	44	0000384		\N
655	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00035		\N
656	45	http://en.wikipedia.org/wiki/OxyS_RNA		\N
657	44	0000385		\N
658	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00030		\N
659	44	0000386		\N
660	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00010		\N
661	44	0000387		\N
662	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00034		\N
663	45	http://en.wikipedia.org/wiki/RprA_RNA		\N
664	44	0000388		\N
665	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00036		\N
666	44	0000389		\N
667	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00021		\N
668	45	http://en.wikipedia.org/wiki/Spot_42_RNA		\N
669	44	0000390		\N
670	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00025		\N
671	45	http://en.wikipedia.org/wiki/Telomerase_RNA		\N
672	44	0000391		\N
673	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00003		\N
674	45	http://en.wikipedia.org/wiki/U1_snRNA		\N
675	44	0000392		\N
676	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00004		\N
677	45	http://en.wikipedia.org/wiki/U2_snRNA		\N
678	44	0000393		\N
679	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00015		\N
680	45	http://en.wikipedia.org/wiki/U4_snRNA		\N
681	44	0000394		\N
682	46	=12409455		\N
683	44	0000395		\N
684	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00020		\N
685	45	http://en.wikipedia.org/wiki/U5_snRNA		\N
686	44	0000396		\N
687	45	http://en.wikipedia.org/wiki/U6_snRNA		\N
688	44	0000397		\N
689	45	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=retrieve&amp;db=pubmed&amp;list_uids=12409455&amp;dopt=Abstract		\N
690	44	0000398		\N
691	46	9622129		\N
692	45	http://en.wikipedia.org/wiki/U11_snRNA		\N
693	44	0000399		\N
694	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00007		\N
695	45	http://en.wikipedia.org/wiki/U12_snRNA		\N
696	44	0000400		\N
697	44	0000401		\N
698	44	0000402		\N
699	44	0000403		\N
700	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00016		\N
701	46	2551119		\N
702	44	0005839		\N
703	44	0000404		\N
704	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00006		\N
705	45	http://en.wikipedia.org/wiki/Vault_RNA		\N
706	44	0000405		\N
707	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00019		\N
708	45	http://en.wikipedia.org/wiki/Y_RNA		\N
709	44	0000406		\N
710	46	1899376		\N
711	46	7823908		\N
712	45	http://en.wikipedia.org/wiki/Twintron		\N
713	44	0000407		\N
714	45	http://en.wikipedia.org/wiki/18S_ribosomal_RNA		\N
715	44	0000408		\N
716	44	0000409		\N
717	55	GAR		\N
718	56	00033		\N
719	45	http://en.wikipedia.org/wiki/Binding_site		\N
720	44	0000410		\N
721	44	0000411		\N
722	44	0000412		\N
883	44	0000555		\N
723	45	http://en.wikipedia.org/wiki/Restriction_fragment		\N
724	44	0000413		\N
725	44	0000414		\N
726	44	0000415		\N
727	44	0000416		\N
728	44	0000417		\N
729	42	broad		\N
730	56	00012		\N
731	56	00134		\N
732	44	0001069		\N
733	44	0000418		\N
734	56	00159		\N
735	45	http://en.wikipedia.org/wiki/Signal_peptide		\N
736	44	0000419		\N
737	56	00149		\N
738	44	0000420		\N
739	44	0000421		\N
740	44	0000422		\N
741	44	0000423		\N
742	44	0000424		\N
743	44	0000425		\N
744	44	0000426		\N
745	44	0000427		\N
746	44	0000428		\N
747	44	0000429		\N
748	44	0000430		\N
749	44	0000431		\N
750	44	0000432		\N
751	44	0000433		\N
752	44	0000434		\N
753	45	http://www.koko.gov.my/CocoaBioTech/Glossaryt.html		\N
754	44	0000435		\N
755	44	0000436		\N
756	44	0000437		\N
757	44	0000438		\N
758	44	0000439		\N
759	44	0000440		\N
760	45	http://en.wikipedia.org/wiki/Vector_(molecular_biology)		\N
761	44	0000441		\N
762	44	0000442		\N
763	44	0000443		\N
764	44	0000444		\N
765	44	0000445		\N
766	44	0000446		\N
767	44	0000447		\N
768	44	0000448		\N
769	44	0000449		\N
770	44	0000450		\N
771	44	0000451		\N
772	44	0000452		\N
773	44	0000453		\N
774	44	0000454		\N
775	45	http://www.developmentalcell.com/content/article/abstract?uid=PIIS1534580703002284		\N
776	45	http://en.wikipedia.org/wiki/RasiRNA		\N
777	44	0000455		\N
778	44	0000456		\N
779	44	0000457		\N
780	44	0000458		\N
781	44	0000459		\N
782	44	0000460		\N
783	44	0000461		\N
784	49	km		\N
785	44	0000462		\N
786	44	0000463		\N
787	44	0000464		\N
788	44	0000465		\N
789	44	0000466		\N
790	44	0000467		\N
791	44	0000468		\N
792	44	0000469		\N
793	44	0000470		\N
794	44	0000471		\N
795	44	0000472		\N
796	44	0000473		\N
797	44	0000474		\N
798	44	0000475		\N
799	44	0000476		\N
800	44	0000477		\N
801	44	0000478		\N
802	44	0000479		\N
803	44	0000480		\N
804	44	0000481		\N
805	44	0000482		\N
806	44	0000483		\N
807	44	0000484		\N
808	44	0000485		\N
809	44	0000486		\N
810	44	0000487		\N
811	44	0000488		\N
812	44	0000489		\N
813	44	0000490		\N
814	44	0000491		\N
815	44	0000492		\N
816	44	0000493		\N
817	44	0000494		\N
818	44	0000495		\N
819	44	0000496		\N
820	44	0000497		\N
821	44	0000498		\N
822	44	0000499		\N
823	44	0000500		\N
824	45	http://en.wikipedia.org/wiki/Hoogsteen_base_pair		\N
825	44	0000501		\N
826	44	0000502		\N
827	44	0000503		\N
828	44	0000504		\N
829	44	0000505		\N
830	44	0000506		\N
831	44	0000507		\N
832	44	0000508		\N
833	44	0000509		\N
834	44	0000510		\N
835	44	0000511		\N
836	44	0000512		\N
837	44	0000513		\N
838	44	0000514		\N
839	44	0000515		\N
840	44	0000516		\N
841	44	0000517		\N
842	44	0000518		\N
843	44	0000519		\N
844	44	0000520		\N
845	44	0000521		\N
846	44	0000522		\N
847	44	0000523		\N
848	44	0000524		\N
849	44	0000525		\N
850	44	0000526		\N
851	44	0000527		\N
852	44	0000528		\N
853	44	0000529		\N
854	44	0000530		\N
855	44	0000531		\N
856	44	0000532		\N
857	44	0000533		\N
858	44	0000534		\N
859	44	0000535		\N
860	44	0000536		\N
861	44	0000537		\N
862	44	0000538		\N
863	44	0000539		\N
864	44	0000540		\N
865	44	0000541		\N
866	44	0000542		\N
867	44	0000543		\N
868	44	0000544		\N
869	45	http://www.pnas.org/cgi/content/full/100/11/6569		\N
870	45	http://en.wikipedia.org/wiki/Helitron		\N
871	44	0000545		\N
872	45	http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=33937		\N
873	44	0000546		\N
874	44	0000547		\N
875	44	0000548		\N
876	44	0000549		\N
877	44	0000550		\N
878	44	0000551		\N
879	44	0000552		\N
880	45	http://en.wikipedia.org/wiki/Shine-Dalgarno_sequence		\N
881	44	0000553		\N
882	44	0000554		\N
884	44	0000556		\N
885	44	0000557		\N
886	44	0000558		\N
887	45	http://www.imgt.org/ligmb/LIGMlect?query=7		\N
888	44	0000559		\N
889	44	0000560		\N
890	44	0000561		\N
891	44	0000562		\N
892	44	0000563		\N
893	44	0000564		\N
894	44	0000565		\N
895	44	0000566		\N
896	44	0000567		\N
897	44	0000568		\N
898	44	0000569		\N
899	44	0100042		\N
900	45	http://en.wikipedia.org/wiki/Retrotransposed		\N
901	44	0000570		\N
902	44	0000571		\N
903	44	0000572		\N
904	44	0000573		\N
905	44	0000574		\N
906	44	0000575		\N
907	44	0000576		\N
908	44	0000577		\N
909	45	http://en.wikipedia.org/wiki/Centromere		\N
910	44	0000578		\N
911	44	0000579		\N
912	44	0000580		\N
913	44	0000581		\N
914	45	http://en.wikipedia.org/wiki/5%27_cap		\N
915	44	0000582		\N
916	44	0000583		\N
917	45	http://dna.kdna.ucla.edu/rna/index.aspx		\N
918	44	0000584		\N
919	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00023		\N
920	45	http://en.wikipedia.org/wiki/TmRNA		\N
921	44	0000585		\N
922	44	0000586		\N
923	44	0000587		\N
924	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00028		\N
925	45	http://en.wikipedia.org/wiki/Group_I_intron		\N
926	44	0000588		\N
927	44	0000589		\N
928	44	0000590		\N
929	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00017		\N
930	44	0000591		\N
931	45	http://en.wikipedia.org/wiki/Pseudoknot		\N
932	44	0000592		\N
933	45	http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10334330&amp;dopt=Abstract		\N
934	44	0000593		\N
935	45	http://www.bio.umass.edu/biochem/rna-sequence/Yeast_snoRNA_Database/snoRNA_DataBase.html		\N
936	44	0000594		\N
937	44	0000595		\N
938	44	0000596		\N
939	44	0000597		\N
940	45	http://www.rna.ucla.edu/index.html		\N
941	44	0000598		\N
942	44	0000599		\N
943	44	0000600		\N
944	44	0000601		\N
945	44	0000602		\N
946	45	http://en.wikipedia.org/wiki/Guide_RNA		\N
947	44	0000603		\N
948	45	http://www.sanger.ac.uk/Software/Rfam/browse/index.shtml		\N
949	45	http://en.wikipedia.org/wiki/Group_II_intron		\N
950	44	0000604		\N
951	44	0000605		\N
952	45	http://en.wikipedia.org/wiki/Intergenic_region		\N
953	44	0000606		\N
954	44	0000607		\N
955	44	0000608		\N
956	44	0000609		\N
957	45	http://www.rna.ucla.edu/		\N
958	44	0000610		\N
959	44	0000611		\N
960	44	0000612		\N
961	45	http://nar.oupjournals.org/cgi/content/full/25/4/888		\N
962	45	http://en.wikipedia.org/wiki/Polypyrimidine_tract		\N
963	44	0000613		\N
964	44	0000614		\N
965	44	0000615		\N
966	44	0000616		\N
967	44	0000617		\N
968	44	0000618		\N
969	44	0000619		\N
970	45	http://en.wikipedia.org/wiki/A-box		\N
971	44	0000620		\N
972	44	0000621		\N
973	44	0000622		\N
974	44	0000623		\N
975	44	0000624		\N
976	45	http://en.wikipedia.org/wiki/Telomere		\N
977	44	0000625		\N
978	45	http://en.wikipedia.org/wiki/Silencer_(DNA)		\N
979	44	0000626		\N
980	44	0000627		\N
981	45	http://en.wikipedia.org/wiki/Insulator_(genetics)		\N
982	44	0000628		\N
983	44	0000629		\N
984	44	0000630		\N
985	44	0000631		\N
986	44	0000632		\N
987	44	0000633		\N
988	45	http://en.wikipedia.org/wiki/Monocistronic_mRNA		\N
989	44	0000634		\N
990	45	http://en.wikipedia.org/wiki/Polycistronic_mRNA		\N
991	44	0000635		\N
992	44	0000636		\N
993	44	0000637		\N
994	44	0000638		\N
995	45	http://oregonstate.edu/instruction/bb492/general/glossary.html		\N
996	44	0000639		\N
997	44	0000640		\N
998	44	0000641		\N
999	44	0000642		\N
1000	44	0000643		\N
1001	45	http://en.wikipedia.org/wiki/Minisatellite		\N
1002	44	0000644		\N
1003	45	http://en.wikipedia.org/wiki/Antisense_RNA		\N
1004	44	0000645		\N
1005	44	0000646		\N
1006	45	http://en.wikipedia.org/wiki/SiRNA		\N
1007	44	0000647		\N
1008	44	0000648		\N
1009	44	0000649		\N
1010	46	11081512		\N
1011	45	http://en.wikipedia.org/wiki/StRNA		\N
1012	44	0000650		\N
1013	44	0000651		\N
1014	44	0000652		\N
1015	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00001		\N
1016	45	http://en.wikipedia.org/wiki/5S_ribosomal_RNA		\N
1017	44	0000653		\N
1018	45	http://en.wikipedia.org/wiki/28S_ribosomal_RNA		\N
1019	44	0000654		\N
1020	44	0000655		\N
1021	45	http://en.wikipedia.org/wiki/NcRNA		\N
1022	44	0000656		\N
1023	44	0000657		\N
1024	44	0000658		\N
1025	45	http://en.wikipedia.org/wiki/Interspersed_repeat		\N
1026	44	0000659		\N
1027	44	0000660		\N
1028	44	0000661		\N
1029	44	0000662		\N
1030	44	0000663		\N
1031	44	0000664		\N
1032	44	0000665		\N
1033	44	0000666		\N
1034	44	0000667		\N
1035	44	1000034		\N
1036	44	0000668		\N
1037	44	0000669		\N
1038	44	0000670		\N
1039	44	0000671		\N
1040	44	0000672		\N
1041	44	0000673		\N
1042	45	http://en.wikipedia.org/wiki/RNA		\N
1043	44	0000674		\N
1044	44	0000675		\N
1045	44	0000676		\N
1046	44	0000677		\N
1047	44	0000678		\N
1048	44	0000679		\N
1049	44	0000680		\N
1050	44	0000681		\N
1051	44	0000682		\N
1052	44	0000683		\N
1053	45	http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12403462&amp;dopt=Abstract		\N
1054	44	0000684		\N
1055	44	0000685		\N
1056	44	0000686		\N
1057	44	0000687		\N
1058	44	0000688		\N
1059	44	0000689		\N
1060	44	0000690		\N
1061	44	0000691		\N
1062	56	00067		\N
1063	44	0000692		\N
1064	44	0000693		\N
1065	44	0000694		\N
1066	44	0000695		\N
1067	44	0000696		\N
1068	45	http://en.wikipedia.org/wiki/Oligonucleotide		\N
1069	44	0000697		\N
1070	44	0000698		\N
1071	44	0000699		\N
1072	44	0000700		\N
1073	44	0000701		\N
1074	44	0000702		\N
1075	44	0000703		\N
1076	44	0000704		\N
1077	44	immuno_workshop		\N
1078	45	http://en.wikipedia.org/wiki/Gene		\N
1079	44	0000705		\N
1080	45	http://en.wikipedia.org/wiki/Tandem_repeat		\N
1081	44	0000706		\N
1082	44	0000707		\N
1083	44	0000708		\N
1084	44	0000709		\N
1085	44	0000710		\N
1086	44	0000711		\N
1087	44	0000712		\N
1088	44	0000713		\N
1089	45	http://en.wikipedia.org/wiki/DNA_motif		\N
1090	44	0000714		\N
1091	44	0000715		\N
1092	44	0000716		\N
1093	44	0000717		\N
1094	45	http://en.wikipedia.org/wiki/Reading_frame		\N
1095	44	0000718		\N
1096	44	0000719		\N
1097	49	WG		\N
1098	44	0000720		\N
1099	44	0000721		\N
1100	44	0000722		\N
1101	44	0000723		\N
1102	45	http://en.wikipedia.org/wiki/IDNA		\N
1103	44	0000724		\N
1104	45	http://en.wikipedia.org/wiki/Origin_of_transfer		\N
1105	44	0000725		\N
1106	56	00055		\N
1107	44	0000726		\N
1108	44	0000727		\N
1109	44	SG		\N
1110	44	0000728		\N
1111	45	http://en.wikipedia.org/wiki/Intein		\N
1112	44	0000729		\N
1113	44	0000730		\N
1114	44	0000731		\N
1115	44	0000732		\N
1116	45	http://en.wikipedia.org/wiki/Predicted		\N
1117	44	0000733		\N
1118	44	0000734		\N
1119	44	0000735		\N
1120	44	0000736		\N
1121	44	0000737		\N
1122	44	0000738		\N
1123	44	0000739		\N
1124	44	0000740		\N
1125	44	0000741		\N
1126	46	8395055		\N
1127	44	0000826		\N
1128	45	http://en.wikipedia.org/wiki/Kinetoplast		\N
1129	44	0000742		\N
1130	44	0000827		\N
1131	44	0000743		\N
1132	44	0000744		\N
1133	44	0000745		\N
1134	44	0000746		\N
1135	44	0000747		\N
1136	44	0000748		\N
1137	44	0000749		\N
1138	44	0000750		\N
1139	44	0000751		\N
1140	44	0000752		\N
1141	44	0000753		\N
1142	44	0000754		\N
1143	44	0000755		\N
1144	45	http://en.wikipedia.org/wiki/Plasmid_vector#Vectors		\N
1145	44	0000756		\N
1146	45	http://en.wikipedia.org/wiki/CDNA		\N
1147	44	0000757		\N
1148	44	0000758		\N
1149	44	0000759		\N
1150	44	0000760		\N
1151	44	0000761		\N
1152	44	0000762		\N
1153	44	0000763		\N
1154	44	0000764		\N
1155	44	0000765		\N
1156	44	0000766		\N
1157	44	0000767		\N
1158	44	0000768		\N
1159	44	0000769		\N
1160	57	10.1093/nar/gkh795		\N
1161	58	kw		\N
1162	59	1362-4962		\N
1163	44	0000770		\N
1164	44	0000771		\N
1165	45	http://rgd.mcw.edu/tu/qtls/		\N
1166	44	0000772		\N
1167	60	at		\N
1168	45	http://en.wikipedia.org/wiki/Genomic_island		\N
1169	44	0000773		\N
1170	44	0000774		\N
1171	44	0000775		\N
1172	44	0000776		\N
1173	44	0000777		\N
1174	44	0000778		\N
1175	44	0000779		\N
1176	44	0000780		\N
1177	44	0000781		\N
1178	44	0000782		\N
1179	44	0000783		\N
1180	44	0000784		\N
1181	44	0000785		\N
1182	44	0000786		\N
1183	44	0000787		\N
1184	44	0000788		\N
1185	44	0000789		\N
1186	44	0000790		\N
1187	44	0000791		\N
1188	44	0000792		\N
1189	44	0000793		\N
1190	44	0000794		\N
1191	44	0000795		\N
1192	44	0000796		\N
1193	49	mc		\N
1194	44	0000797		\N
1195	44	0000798		\N
1196	44	0000799		\N
1197	44	0000800		\N
1198	49	gm		\N
1199	44	0000801		\N
1200	44	0000802		\N
1201	44	0000803		\N
1202	44	0000804		\N
1203	44	0000805		\N
1204	44	0000806		\N
1205	44	0000807		\N
1206	44	0000808		\N
1207	44	0000809		\N
1208	44	0000810		\N
1209	44	0000811		\N
1210	44	0000812		\N
1211	44	0000813		\N
1212	44	0000814		\N
1213	44	0000815		\N
1214	46	15992143		\N
1215	44	0000816		\N
1216	44	0000817		\N
1217	45	http://en.wikipedia.org/wiki/Wild_type		\N
1218	44	0000818		\N
1219	44	0000819		\N
1220	44	0000820		\N
1221	44	0000821		\N
1222	44	0000822		\N
1223	44	0000823		\N
1224	44	0000824		\N
1225	44	0000825		\N
1226	44	0000828		\N
1227	44	0000829		\N
1228	44	0000830		\N
1229	44	0000831		\N
1230	44	0000832		\N
1231	44	0000833		\N
1232	44	0000834		\N
1233	44	0000835		\N
1234	44	0000836		\N
1235	44	0000837		\N
1236	44	0000838		\N
1237	44	0000839		\N
1238	44	GAR		\N
1239	56	00124		\N
1240	56	00331		\N
1241	44	0000840		\N
1242	44	0000841		\N
1243	44	0000842		\N
1244	44	0000843		\N
1245	44	0000844		\N
1246	44	0000845		\N
1247	44	0000846		\N
1248	44	0000847		\N
1249	44	0000848		\N
1250	44	0000849		\N
1251	44	0000850		\N
1252	44	0000851		\N
1253	44	0000852		\N
1254	44	0000853		\N
1255	45	http://en.wikipedia.org/wiki/Homology_(biology)		\N
1256	44	0000854		\N
1257	45	http://en.wikipedia.org/wiki/Paralog#Paralogy		\N
1258	44	0000855		\N
1259	45	http://en.wikipedia.org/wiki/Ortholog#Orthology		\N
1260	44	0000856		\N
1261	44	0000857		\N
1262	44	0000858		\N
1263	44	0000859		\N
1264	44	0000860		\N
1265	45	http://en.wikipedia.org/wiki/Syntenic		\N
1266	44	0000861		\N
1267	44	0000862		\N
1268	44	0000863		\N
1269	44	0000864		\N
1270	44	0000865		\N
1271	45	http://en.wikipedia.org/wiki/Frameshift		\N
1272	44	0000866		\N
1273	44	0000867		\N
1274	44	0000868		\N
1275	44	0000869		\N
1276	44	0000870		\N
1277	44	0000871		\N
1278	44	0000872		\N
1279	44	0000873		\N
1280	44	0000874		\N
1281	44	0000875		\N
1282	44	0000876		\N
1283	44	0000877		\N
1284	44	0000878		\N
1285	44	0000879		\N
1286	44	0000880		\N
1287	44	0000881		\N
1288	44	0000882		\N
1289	44	0000883		\N
1290	44	0000884		\N
1291	44	0000885		\N
1292	44	0000886		\N
1293	44	0000887		\N
1294	44	0000888		\N
1295	44	0000889		\N
1296	44	0000890		\N
1297	44	0000891		\N
1298	44	0000892		\N
1299	44	0000893		\N
1300	45	http://en.wikipedia.org/wiki/Silenced		\N
1301	44	0000894		\N
1302	44	0000895		\N
1303	44	0000896		\N
1304	44	0000897		\N
1305	44	0000898		\N
1306	44	0000899		\N
1307	44	0000900		\N
1308	44	0000901		\N
1309	44	0000902		\N
1310	45	http://en.wikipedia.org/wiki/Transgene		\N
1311	44	0000903		\N
1312	44	0000904		\N
1313	44	0000905		\N
1314	44	0000906		\N
1315	44	0000907		\N
1316	44	0000908		\N
1317	44	0000909		\N
1318	44	0000910		\N
1319	44	0000911		\N
1320	44	0000912		\N
1321	45	http://www.ebi.ac.uk/msd-srv/msdmotif/		\N
1322	56	00203		\N
1323	44	0000913		\N
1324	44	0000914		\N
1325	44	0000915		\N
1326	44	0000916		\N
1327	44	0000917		\N
1328	44	0000918		\N
1329	44	0000919		\N
1330	44	0000920		\N
1331	44	0000921		\N
1332	44	0000922		\N
1333	44	0000923		\N
1334	44	0000924		\N
1335	44	0000925		\N
1336	44	0000926		\N
1337	44	0000927		\N
1338	44	0000928		\N
1339	44	0000929		\N
1340	44	0000930		\N
1341	44	0000931		\N
1342	44	jk		\N
1343	44	0000932		\N
1344	44	0000933		\N
1345	44	0000934		\N
1346	49	cds		\N
1347	44	0000935		\N
1348	44	0000936		\N
1349	44	0000937		\N
1350	44	0000938		\N
1351	44	0000939		\N
1352	44	0000940		\N
1353	44	0000941		\N
1354	44	0000942		\N
1355	44	0000943		\N
1356	44	0000944		\N
1357	44	0000945		\N
1358	44	0000946		\N
1359	44	0000947		\N
1360	44	0000948		\N
1361	44	0000949		\N
1362	44	0000950		\N
1363	44	0000951		\N
1364	44	0000952		\N
1365	44	0000953		\N
1366	44	0000954		\N
1367	44	0000955		\N
1368	44	0000956		\N
1369	44	0000957		\N
1370	44	0000958		\N
1371	44	0000959		\N
1372	44	0000960		\N
1373	44	0000961		\N
1374	44	0000962		\N
1375	44	0000963		\N
1376	44	0000964		\N
1377	44	0000965		\N
1378	44	0000966		\N
1379	44	0000967		\N
1380	44	0000968		\N
1381	44	0000969		\N
1382	45	http://en.wikipedia.org/wiki/Rolling_circle		\N
1383	44	0000970		\N
1384	44	0000971		\N
1385	44	0000972		\N
1386	44	0000973		\N
1387	45	http://en.wikipedia.org/wiki/Insertion_sequence		\N
1388	44	0000975		\N
1389	44	0000976		\N
1390	44	0000977		\N
1391	44	0000978		\N
1392	44	0000979		\N
1393	44	0000980		\N
1394	44	0000974		\N
1395	45	http://en.wikipedia.org/wiki/Minicircle		\N
1396	44	0000981		\N
1397	44	0000982		\N
1398	44	0000983		\N
1399	44	0000984		\N
1400	44	0000985		\N
1401	44	0000986		\N
1402	44	0000987		\N
1403	44	0000988		\N
1404	44	0000989		\N
1405	46	15333696		\N
1406	44	0000990		\N
1407	44	0000991		\N
1408	44	0000992		\N
1409	44	0000993		\N
1410	44	0000994		\N
1411	44	0000995		\N
1412	44	0000996		\N
1413	44	0000997		\N
1414	44	0000998		\N
1415	45	http://www.genetics.org/cgi/content/full/170/2/661		\N
1416	44	0000999		\N
1417	44	0001000		\N
1418	45	http://en.wikipedia.org/wiki/16S_ribosomal_RNA		\N
1419	44	0001001		\N
1420	44	0001002		\N
1421	44	0001003		\N
1422	44	0001004		\N
1423	44	0001005		\N
1424	44	0001006		\N
1425	50	jl		\N
1426	45	http://en.wikipedia.org/wiki/Prophage		\N
1427	44	0001007		\N
1428	45	http://ecoliwiki.net/colipedia/index.php/Category:Cryptic_Prophage.w%20dbxref		\N
1429	44	0001008		\N
1430	45	http://en.wikipedia.org/wiki/Tetraloop		\N
1431	44	0001009		\N
1432	44	0001010		\N
1433	46	9753739		\N
1434	44	0001011		\N
1435	45	http://en.wikipedia.org/wiki/Peptide_nucleic_acid		\N
1436	44	0001012		\N
1437	44	0001013		\N
1438	45	http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=rs2067431		\N
1439	44	0001014		\N
1440	44	0001015		\N
1441	46	11256617		\N
1442	45	http://en.wikipedia.org/wiki/Wobble_base_pair		\N
1443	44	0001016		\N
1444	44	0001017		\N
1445	45	http://en.wikipedia.org/wiki/Silent_mutation		\N
1446	44	0001018		\N
1447	45	http://en.wikipedia.org/wiki/Epitope		\N
1448	44	0001019		\N
1449	45	http://en.wikipedia.org/wiki/Copy_number_variation		\N
1450	44	0001020		\N
1451	44	0001021		\N
1452	44	0001242		\N
1453	44	0001022		\N
1454	44	0001023		\N
1455	45	http://en.wikipedia.org/wiki/Allele		\N
1456	44	0001024		\N
1457	45	http://en.wikipedia.org/wiki/Haplotype		\N
1458	44	0001025		\N
1459	44	0001026		\N
1460	45	http://en.wikipedia.org/wiki/Genome		\N
1461	44	0001027		\N
1462	45	http://en.wikipedia.org/wiki/Genotype		\N
1463	44	0001028		\N
1464	44	0001029		\N
1465	44	0001030		\N
1466	44	0001031		\N
1467	44	0001032		\N
1468	45	http://en.wikipedia.org/wiki/Mitochondrial_DNA		\N
1469	44	0001033		\N
1470	44	0001034		\N
1471	46	17589500		\N
1472	44	0001035		\N
1473	45	http://en.wikipedia.org/wiki/PiRNA		\N
1474	44	0001036		\N
1475	44	0001037		\N
1476	46	14681355		\N
1477	45	http://en.wikipedia.org/wiki/Mobile_genetic_element		\N
1478	44	0001038		\N
1479	44	0001039		\N
1480	44	0001040		\N
1481	44	0001041		\N
1482	44	0001042		\N
1483	45	http://en.wikipedia.org/wiki/Bacteriophage		\N
1484	44	0001043		\N
1485	44	0001044		\N
1486	44	0001045		\N
1487	61	at		\N
1488	44	0001046		\N
1489	44	0001047		\N
1490	44	0001048		\N
1491	44	0001049		\N
1492	60	ariane		\N
1493	44	0001050		\N
1494	44	0001051		\N
1495	44	0001052		\N
1496	44	0001053		\N
1497	44	0001054		\N
1498	44	0001055		\N
1499	44	0001056		\N
1500	44	0001057		\N
1501	44	0001058		\N
1502	44	0001059		\N
1503	44	1000004		\N
1504	44	1000007		\N
1505	44	0001060		\N
1506	44	0001061		\N
1507	56	00063		\N
1508	44	0001062		\N
1509	56	00077		\N
1510	45	http://en.wikipedia.org/wiki/Propeptide		\N
1511	44	0001063		\N
1512	56	00129		\N
1513	44	0001064		\N
1514	62	curation_manual		\N
1515	56	00076		\N
1516	45	http://en.wikipedia.org/wiki/Peptide		\N
1517	44	0001066		\N
1518	56	00068		\N
1519	44	0001067		\N
1520	56	00032		\N
1521	44	0001068		\N
1522	56	00070		\N
1523	44	0001070		\N
1524	56	00337		\N
1525	44	0001071		\N
1526	56	00128		\N
1527	44	0001072		\N
1528	56	00154		\N
1529	44	0001073		\N
1530	56	00145		\N
1531	44	0001074		\N
1532	56	00144		\N
1533	44	0001075		\N
1534	56	00156		\N
1535	44	0001076		\N
1536	56	00155		\N
1537	44	0001077		\N
1538	62	curator_manual		\N
1539	56	00158		\N
1540	44	0001078		\N
1541	56	00003		\N
1542	45	http://en.wikipedia.org/wiki/Secondary_structure		\N
1543	44	0001079		\N
1544	56	0000338		\N
1545	45	http://en.wikipedia.org/wiki/Structural_motif		\N
1546	44	0001080		\N
1547	56	00041		\N
1548	45	http://en.wikipedia.org/wiki/Coiled_coil		\N
1549	44	0001081		\N
1550	56	00147		\N
1551	44	0001082		\N
1552	56	00125		\N
1553	44	0001083		\N
1554	56	00182		\N
1555	44	0001084		\N
1556	56	00072		\N
1557	44	0001085		\N
1558	56	00069		\N
1559	44	0001086		\N
1560	56	00181		\N
1561	44	0001087		\N
1562	56	00178		\N
1563	44	0001088		\N
1564	56	00028		\N
1565	44	0001089		\N
1566	56	00052		\N
1567	45	http://en.wikipedia.org/wiki/Post_translational_modification		\N
1568	44	0001090		\N
1569	56	00246		\N
1570	44	0001091		\N
1571	56	00029		\N
1572	44	0001092		\N
1573	56	00027		\N
1574	44	0001093		\N
1575	62	Curation_manual		\N
1576	56	00131		\N
1577	45	http://en.wikipedia.org/wiki/Protein_protein_interaction		\N
1578	44	0001094		\N
1579	56	00186		\N
1580	44	0001095		\N
1581	56	00136		\N
1582	44	0001096		\N
1583	56	00146		\N
1584	44	0001097		\N
1585	56	00137		\N
1586	44	0001098		\N
1587	56	00187		\N
1588	44	0001099		\N
1589	56	00140		\N
1590	44	0001100		\N
1591	56	00141		\N
1592	44	0001101		\N
1593	56	00142		\N
1594	44	0001102		\N
1595	56	00143		\N
1596	44	0001103		\N
1597	56	00185		\N
1598	44	0001104		\N
1599	56	00026		\N
1600	44	0001105		\N
1601	56	00157		\N
1602	44	0001106		\N
1603	56	00202		\N
1604	44	0001107		\N
1605	56	00208		\N
1606	45	http://en.wikipedia.org/wiki/Beta_bulge		\N
1607	44	0001108		\N
1608	63	//www.ebi.ac.uk/msd-srv/msdmotif/		\N
1609	56	00209		\N
1610	44	0001109		\N
1611	56	00210		\N
1612	44	0001110		\N
1613	56	00211		\N
1614	44	0001111		\N
1615	56	00042		\N
1616	45	http://en.wikipedia.org/wiki/Beta_sheet		\N
1617	44	0001112		\N
1618	56	0000341		\N
1619	44	0001113		\N
1620	56	00151		\N
1621	44	0001114		\N
1622	56	00152		\N
1623	44	0001115		\N
1624	56	00222		\N
1625	44	0001116		\N
1626	56	0000339		\N
1627	44	0001117		\N
1628	56	00040		\N
1629	45	http://en.wikipedia.org/wiki/Alpha_helix		\N
1630	44	0001118		\N
1631	56	00153		\N
1632	45	http://en.wikipedia.org/wiki/Pi_helix		\N
1633	44	0001119		\N
1634	56	0000340		\N
1635	45	http://en.wikipedia.org/wiki/310_helix		\N
1636	44	0001120		\N
1637	56	00223		\N
1638	44	0001121		\N
1639	56	00224		\N
1640	44	0001122		\N
1641	56	00225		\N
1642	44	0001123		\N
1643	56	00226		\N
1644	44	0001124		\N
1645	56	00228		\N
1646	44	0001125		\N
1647	56	00227		\N
1648	44	0001126		\N
1649	56	00229		\N
1650	44	0001127		\N
1651	56	00230		\N
1652	44	0001128		\N
1653	64	feature_type		\N
1654	56	00148		\N
1655	44	0001129		\N
1656	56	00206		\N
1657	44	0001130		\N
1658	56	00204		\N
1659	44	0001131		\N
1660	56	00205		\N
1661	44	0001132		\N
1662	56	00207		\N
1663	44	0001133		\N
1664	56	00212		\N
1665	44	0001134		\N
1666	56	00215		\N
1667	44	0001135		\N
1668	56	00213		\N
1669	44	0001136		\N
1670	56	00216		\N
1671	44	0001137		\N
1672	56	00214		\N
1673	44	0001138		\N
1674	56	00219		\N
1675	44	0001139		\N
1676	56	00220		\N
1677	44	0001140		\N
1678	56	00221		\N
1679	44	0001141		\N
1680	56	00231		\N
1681	44	0001142		\N
1682	56	00234		\N
1683	44	0001143		\N
1684	56	00232		\N
1685	44	0001144		\N
1686	56	00235		\N
1687	44	0001145		\N
1688	56	00233		\N
1689	44	0001146		\N
1690	56	00336		\N
1691	44	0001147		\N
1692	56	00071		\N
1693	44	0001148		\N
1694	56	00036		\N
1695	44	0001149		\N
1696	56	00073		\N
1697	44	0001065		\N
1698	44	0001150		\N
1699	44	0001151		\N
1700	46	2371257		\N
1701	44	0001152		\N
1702	44	0001153		\N
1703	44	0001154		\N
1704	44	0001155		\N
1705	44	0001156		\N
1706	46	12537576		\N
1707	44	0001157		\N
1708	46	16827941:12537576		\N
1709	44	0001158		\N
1710	46	12537576:16827941		\N
1711	44	0001159		\N
1712	44	0001160		\N
1713	44	0001161		\N
1714	44	0001162		\N
1715	46	12537576:15231738		\N
1716	44	0001163		\N
1717	46	16827941		\N
1718	44	0001164		\N
1719	44	0001165		\N
1720	44	0001166		\N
1721	44	0001167		\N
1722	44	0001168		\N
1723	44	0001169		\N
1724	44	0001170		\N
1725	46	16537396		\N
1726	44	0001171		\N
1727	44	0001172		\N
1728	44	0001173		\N
1729	47	0716719207		\N
1730	44	0001174		\N
1731	45	http://en.wikipedia.org/wiki/Anticodon		\N
1732	44	0001175		\N
1733	44	0001176		\N
1734	47	071671920		\N
1735	44	0001177		\N
1736	44	0001178		\N
1737	44	0001179		\N
1738	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00012		\N
1739	45	http://en.wikipedia.org/wiki/Small_nucleolar_RNA_U3		\N
1740	44	0001180		\N
1741	46	7892223		\N
1742	45	http://en.wikipedia.org/wiki/AU-rich_element		\N
1743	44	0001181		\N
1744	46	10893231		\N
1745	44	0001182		\N
1746	46	3198610		\N
1747	46	8710843		\N
1748	45	http://en.wikipedia.org/wiki/Iron_responsive_element		\N
1749	44	0001183		\N
1750	45	http://en.wikipedia.org/wiki/Morpholino		\N
1751	44	0001184		\N
1752	44	0001185		\N
1753	44	0001186		\N
1754	44	0001187		\N
1755	50	mah		\N
1756	46	12457565		\N
1757	44	0001188		\N
1758	53	48010		\N
1759	44	0001189		\N
1760	45	http://en.wikipedia.org/wiki/Locked_nucleic_acid		\N
1761	44	0001190		\N
1762	53	48019		\N
1763	44	0001191		\N
1764	45	http://en.wikipedia.org/wiki/Threose_nucleic_acid		\N
1765	44	0001192		\N
1766	53	48015		\N
1767	44	0001193		\N
1768	45	http://en.wikipedia.org/wiki/Glycerol_nucleic_acid		\N
1769	44	0001194		\N
1770	53	48016		\N
1771	44	0001195		\N
1772	44	0001196		\N
1773	53	48017		\N
1774	44	0001197		\N
1775	44	0001198		\N
1776	44	0001199		\N
1777	44	0001200		\N
1778	44	0001201		\N
1779	44	0001202		\N
1780	44	0001203		\N
1781	65	jb		\N
1782	44	0001204		\N
1783	44	0001205		\N
1784	44	0001206		\N
1785	44	0001207		\N
1786	44	0001208		\N
1787	45	http://www.ncbi.nlm.nih.gov/About/primer/est.html		\N
1788	44	0001209		\N
1789	44	0001210		\N
1790	45	http://en.wikipedia.org/wiki/Translational_frameshift		\N
1791	44	0001211		\N
1792	44	0001212		\N
1793	44	0001213		\N
1794	46	11377794		\N
1795	45	http://en.wikipedia.org/wiki/Group_III_intron		\N
1796	44	0001214		\N
1797	44	0001215		\N
1798	44	0001216		\N
1799	44	0001217		\N
1800	44	0001218		\N
1801	44	bm		\N
1802	44	0001219		\N
1803	44	0001220		\N
1804	44	0001221		\N
1805	44	0001222		\N
1806	44	0001223		\N
1807	44	0001224		\N
1808	44	0001225		\N
1809	44	0001226		\N
1810	44	0001227		\N
1811	44	0001228		\N
1812	45	http://en.wikipedia.org/wiki/Dihydrouridine		\N
1813	66	051		\N
1814	44	0001229		\N
1815	45	http://en.wikipedia.org/wiki/Pseudouridine		\N
1816	66	050		\N
1817	44	0001230		\N
1818	45	http://library.med.utah.edu/RNAmods/		\N
1819	45	http://en.wikipedia.org/wiki/Inosine		\N
1820	44	0001231		\N
1821	44	0001232		\N
1822	44	0001233		\N
1823	44	0001234		\N
1824	45	http://en.wikipedia.org/wiki/Mobile		\N
1825	44	0001235		\N
1826	45	http://en.wikipedia.org/wiki/Replicon_(genetics)		\N
1827	44	0001236		\N
1828	45	http://en.wikipedia.org/wiki/Nucleobase		\N
1829	44	0001237		\N
1830	45	http://en.wikipedia.org/wiki/Amino_acid		\N
1831	44	0001238		\N
1832	44	0001239		\N
1833	44	0001240		\N
1834	67	nw		\N
1835	44	0001241		\N
1836	44	0001243		\N
1837	44	0001244		\N
1838	44	0001245		\N
1839	44	0001246		\N
1840	44	0001247		\N
1841	44	0001248		\N
1842	45	http://en.wikipedia.org/wiki/Genome_assembly#Genome_assembly		\N
1843	44	0001249		\N
1844	44	0001250		\N
1845	44	0001251		\N
1846	44	0001252		\N
1847	44	0001253		\N
1848	44	0001254		\N
1849	45	http://en.wikipedia.org/wiki/Polyploid		\N
1850	44	0001255		\N
1851	45	http://en.wikipedia.org/wiki/Autopolyploid		\N
1852	44	0001256		\N
1853	45	http://en.wikipedia.org/wiki/Allopolyploid		\N
1854	44	0001257		\N
1855	44	0001258		\N
1856	50	dh		\N
1857	46	3095662		\N
1858	44	0001259		\N
1859	44	0001260		\N
1860	44	0001261		\N
1861	44	0001262		\N
1862	44	0001263		\N
1863	44	0001264		\N
1864	44	0001265		\N
1865	44	0001266		\N
1866	44	0001267		\N
1867	44	0001268		\N
1868	44	0001269		\N
1869	44	0001270		\N
1870	44	0001271		\N
1871	44	0001272		\N
1872	44	0001273		\N
1873	44	0001274		\N
1874	44	0001275		\N
1875	44	0001276		\N
1876	44	0001277		\N
1877	44	0001278		\N
1878	66	018		\N
1879	44	0001279		\N
1880	66	019		\N
1881	44	0001280		\N
1882	66	081		\N
1883	44	0001281		\N
1884	66	020		\N
1885	44	0001282		\N
1886	66	021		\N
1887	44	0001283		\N
1888	66	022		\N
1889	44	0001284		\N
1890	66	023		\N
1891	44	0001285		\N
1892	66	024		\N
1893	44	0001286		\N
1894	66	025		\N
1895	44	0001287		\N
1896	66	026		\N
1897	44	0001288		\N
1898	66	027		\N
1899	44	0001289		\N
1900	45	http://en.wikipedia.org/wiki/Lysidine		\N
1901	66	028		\N
1902	44	0001290		\N
1903	66	082		\N
1904	44	0001291		\N
1905	66	083		\N
1906	44	0001292		\N
1907	66	084		\N
1908	44	0001293		\N
1909	66	095		\N
1910	44	0001294		\N
1911	66	107		\N
1912	44	0001295		\N
1913	66	001		\N
1914	44	0001296		\N
1915	66	002		\N
1916	44	0001297		\N
1917	66	003		\N
1918	44	0001298		\N
1919	66	004		\N
1920	44	0001299		\N
1921	66	005		\N
1922	44	0001300		\N
1923	66	006		\N
1924	44	0001301		\N
1925	66	007		\N
1926	44	0001302		\N
1927	66	008		\N
1928	44	0001303		\N
1929	66	009		\N
1930	44	0001304		\N
1931	66	010		\N
1932	44	0001305		\N
1933	66	011		\N
1934	44	0001306		\N
1935	66	012		\N
1936	44	0001307		\N
1937	66	013		\N
1938	44	0001308		\N
1939	66	014		\N
1940	44	0001309		\N
1941	66	015		\N
1942	44	0001310		\N
1943	66	016		\N
1944	44	0001311		\N
1945	66	080		\N
1946	44	0001312		\N
1947	66	088		\N
1948	44	0001313		\N
1949	66	089		\N
1950	44	0001314		\N
1951	66	097		\N
1952	44	0001315		\N
1953	66	102		\N
1954	44	0001316		\N
1955	44	0001317		\N
1956	45	http://en.wikipedia.org/wiki/Queuosine		\N
1957	66	043		\N
1958	44	0001318		\N
1959	66	044		\N
1960	44	0001319		\N
1961	66	045		\N
1962	44	0001320		\N
1963	66	046		\N
1964	44	0001321		\N
1965	66	047		\N
1966	44	0001322		\N
1967	66	048		\N
1968	44	0001323		\N
1969	66	049		\N
1970	44	0001324		\N
1971	66	029		\N
1972	44	0001325		\N
1973	66	030		\N
1974	44	0001326		\N
1975	66	031		\N
1976	44	0001327		\N
1977	66	032		\N
1978	44	0001328		\N
1979	66	033		\N
1980	44	0001329		\N
1981	66	034		\N
1982	44	0001330		\N
1983	66	035		\N
1984	44	0001331		\N
1985	66	036		\N
1986	44	0001332		\N
1987	66	037		\N
1988	44	0001333		\N
1989	66	038		\N
1990	44	0001334		\N
1991	66	039		\N
1992	44	0001335		\N
1993	66	040		\N
1994	44	0001336		\N
1995	66	041		\N
1996	44	0001337		\N
1997	66	042		\N
1998	44	0001338		\N
1999	66	090		\N
2000	44	0001339		\N
2001	66	091		\N
2002	44	0001340		\N
2003	66	096		\N
2004	44	0001341		\N
2005	66	100		\N
2006	44	0001342		\N
2007	66	101		\N
2008	44	0001343		\N
2009	66	106		\N
2010	44	0001344		\N
2011	45	http://en.wikipedia.org/wiki/5-methyluridine		\N
2012	66	052		\N
2013	44	0001345		\N
2014	66	053		\N
2015	44	0001346		\N
2016	66	054		\N
2017	44	0001347		\N
2018	66	055		\N
2019	44	0001348		\N
2020	66	056		\N
2021	44	0001349		\N
2022	66	057		\N
2023	44	0001350		\N
2024	66	058		\N
2025	44	0001351		\N
2026	66	059		\N
2027	44	0001352		\N
2028	66	060		\N
2029	44	0001353		\N
2030	66	061		\N
2031	44	0001354		\N
2032	44	0001355		\N
2033	66	063		\N
2034	44	0001356		\N
2035	66	064		\N
2036	44	0001357		\N
2037	66	065		\N
2038	44	0001358		\N
2039	66	066		\N
2040	44	0001359		\N
2041	66	067		\N
2042	44	0001360		\N
2043	66	068		\N
2044	44	0001361		\N
2045	66	069		\N
2046	44	0001362		\N
2047	66	070		\N
2048	44	0001363		\N
2049	66	071		\N
2050	44	0001364		\N
2051	66	072		\N
2052	44	0001365		\N
2053	66	073		\N
2054	44	0001366		\N
2055	66	074		\N
2056	44	0001367		\N
2057	66	075		\N
2058	44	0001368		\N
2059	66	076		\N
2060	44	0001369		\N
2061	66	077		\N
2062	44	0001370		\N
2063	66	078		\N
2064	44	0001371		\N
2065	66	079		\N
2066	44	0001372		\N
2067	66	085		\N
2068	44	0001373		\N
2069	66	086		\N
2070	44	0001374		\N
2071	66	087		\N
2072	44	0001375		\N
2073	66	092		\N
2074	44	0001376		\N
2075	66	093		\N
2076	44	0001377		\N
2077	66	094		\N
2078	44	0001378		\N
2079	66	098		\N
2080	44	0001379		\N
2081	66	099		\N
2082	44	0001380		\N
2083	66	103		\N
2084	44	0001381		\N
2085	66	104		\N
2086	44	0001382		\N
2087	66	105		\N
2088	44	0001383		\N
2089	44	0001384		\N
2090	44	0001385		\N
2091	44	0001386		\N
2092	68	00908		\N
2093	44	0001387		\N
2094	68	00901		\N
2095	44	0001388		\N
2096	68	00903		\N
2097	44	0001389		\N
2098	68	00904		\N
2099	44	0001390		\N
2100	68	00905		\N
2101	44	0001391		\N
2102	68	00906		\N
2103	44	0001392		\N
2104	68	00917		\N
2105	44	0001393		\N
2106	68	00918		\N
2107	44	0001394		\N
2108	68	00907		\N
2109	44	0001395		\N
2110	68	00913		\N
2111	44	0001396		\N
2112	68	00910		\N
2113	44	0001397		\N
2114	68	00914		\N
2115	44	0001398		\N
2116	68	00909		\N
2117	44	0001399		\N
2118	68	00916		\N
2119	44	0001400		\N
2120	68	00912		\N
2121	44	0001401		\N
2122	68	00911		\N
2123	44	0001402		\N
2124	68	01158		\N
2125	44	0001403		\N
2126	68	00920		\N
2127	44	0001404		\N
2128	68	00915		\N
2129	44	0001405		\N
2130	68	00919		\N
2131	44	0001406		\N
2132	68	00902		\N
2133	44	0001407		\N
2134	44	0001408		\N
2135	69	rh		\N
2136	44	0001409		\N
2137	44	0001410		\N
2138	44	0001411		\N
2139	44	0001412		\N
2140	44	0001413		\N
2141	44	0001414		\N
2142	44	0001415		\N
2143	44	0001416		\N
2144	44	chado		\N
2145	44	0001417		\N
2146	44	0001418		\N
2147	44	0001419		\N
2148	44	0001420		\N
2149	44	0001421		\N
2150	44	0001422		\N
2151	44	0001423		\N
2152	44	0001424		\N
2153	44	0001425		\N
2154	44	0001426		\N
2155	44	0001427		\N
2156	70	jd		\N
2157	44	0001428		\N
2158	44	0001429		\N
2159	44	0001430		\N
2160	44	0001431		\N
2161	44	0001432		\N
2162	44	0001433		\N
2163	71	nlw		\N
2164	44	0001434		\N
2165	44	0001435		\N
2166	44	0001436		\N
2167	44	0001437		\N
2168	44	0001438		\N
2169	44	0001439		\N
2170	44	0001440		\N
2171	44	0001441		\N
2172	44	0001442		\N
2173	44	0001443		\N
2174	44	0001444		\N
2175	44	0001445		\N
2176	44	0001446		\N
2177	44	0001447		\N
2178	44	0001448		\N
2179	44	0001449		\N
2180	44	0001450		\N
2181	44	0001451		\N
2182	44	0001452		\N
2183	44	0001453		\N
2184	44	0001454		\N
2185	44	0001455		\N
2186	44	0001456		\N
2187	44	0001457		\N
2188	44	0001458		\N
2189	44	0001459		\N
2190	48	en.wikipedia.org/wiki/CRISPR		\N
2191	44	0001460		\N
2192	44	0001461		\N
2193	44	0001462		\N
2194	44	0001463		\N
2195	46	19182780		\N
2196	44	0001464		\N
2197	44	nlw		\N
2198	44	0001465		\N
2199	44	0001466		\N
2200	44	0001467		\N
2201	44	0001468		\N
2202	44	0001469		\N
2203	44	0001470		\N
2204	44	0001471		\N
2205	44	0001472		\N
2206	44	0001473		\N
2207	44	0001474		\N
2208	44	0001475		\N
2209	46	16401417		\N
2210	44	0001476		\N
2211	44	0001477		\N
2212	72	dh		\N
2213	44	0001478		\N
2214	44	0001479		\N
2215	44	0001480		\N
2216	72	mh		\N
2217	44	0001481		\N
2218	44	0001482		\N
2219	44	0001483		\N
2220	44	0001484		\N
2221	45	http://www.yeastgenome.org/help/glossary.html		\N
2222	44	0001485		\N
2223	48	http://www.yeastgenome.org/help/glossary.html		\N
2224	44	0001486		\N
2225	73	10.1126		\N
2226	44	0001487		\N
2227	44	0001488		\N
2228	44	0001489		\N
2229	44	0001490		\N
2230	44	0001491		\N
2231	44	0001492		\N
2232	44	0001493		\N
2233	46	11222754		\N
2234	44	0001494		\N
2235	44	0001495		\N
2236	44	0001496		\N
2237	46	8720065		\N
2238	44	0001497		\N
2239	45	http://www.yeastgenome.org/help/glossary.html#xelemcoresequence		\N
2240	44	0001498		\N
2241	44	0001499		\N
2242	44	0001500		\N
2243	74	hdene		\N
2244	44	0001501		\N
2245	67	nlw		\N
2246	44	0001502		\N
2247	44	0001503		\N
2248	75	hdeen		\N
2249	44	0001504		\N
2250	44	0001505		\N
2251	44	0001506		\N
2252	44	0001507		\N
2253	44	0001508		\N
2254	44	0001509		\N
2255	44	0001510		\N
2256	44	0001511		\N
2257	44	0001512		\N
2258	44	0001513		\N
2259	44	0001514		\N
2260	44	0001515		\N
2261	44	0001516		\N
2262	44	0001517		\N
2263	44	0001518		\N
2264	44	0001519		\N
2265	44	0001520		\N
2266	44	0001521		\N
2267	44	0001522		\N
2268	44	0001523		\N
2269	44	0001524		\N
2270	44	0001525		\N
2271	44	0001526		\N
2272	44	0001527		\N
2273	44	0001528		\N
2274	45	http://en.wikipedia.org/wiki/Nuclear_localization_signal		\N
2275	44	0001529		\N
2276	44	0001530		\N
2277	44	0001531		\N
2278	45	http://en.wikipedia.org/wiki/Nuclear_export_signal		\N
2279	44	0001532		\N
2280	45	http://en.wikipedia.org/wiki/Recombination_Signal_Sequences		\N
2281	44	0001533		\N
2282	44	0001534		\N
2283	46	16027110		\N
2284	44	0001535		\N
2285	44	0005836		\N
2286	45	http://en.wikipedia.org/wiki/Regulatory_region		\N
2287	44	0005837		\N
2288	46	2251119		\N
2289	44	0005841		\N
2290	44	0005843		\N
2291	44	0005845		\N
2292	44	0005847		\N
2293	44	0005848		\N
2294	44	0005849		\N
2295	44	0005850		\N
2296	45	http://en.wikipedia.org/wiki/Primer_binding_site		\N
2297	44	0005851		\N
2298	44	0005852		\N
2299	44	0005853		\N
2300	51	se		\N
2301	45	http://en.wikipedia.org/wiki/Gene_cassette		\N
2302	44	0005854		\N
2303	44	0005855		\N
2304	44	0005856		\N
2305	44	0005857		\N
2306	44	0005858		\N
2307	45	http://www.informatics.jax.org/silverbook/glossary.shtml		\N
2308	44	0100001		\N
2309	44	0100002		\N
2310	44	0100003		\N
2311	44	0100004		\N
2312	44	0100005		\N
2313	44	0100006		\N
2314	44	0100007		\N
2315	44	0100008		\N
2316	44	0100009		\N
2317	44	0100010		\N
2318	44	0100011		\N
2319	44	0100012		\N
2320	44	0100013		\N
2321	44	0100014		\N
2322	44	0100015		\N
2323	44	0100016		\N
2324	44	0100017		\N
2325	44	0100018		\N
2326	44	0100019		\N
2327	44	0100020		\N
2328	44	0100021		\N
2329	44	1000002		\N
2330	45	http://www.ebi.ac.uk/mutations/recommendations/mutevent.html		\N
2331	44	1000005		\N
2332	44	1000008		\N
2333	45	http://en.wikipedia.org/wiki/Point_mutation		\N
2334	44	1000009		\N
2335	44	1000010		\N
2336	44	1000011		\N
2337	44	1000012		\N
2338	44	1000013		\N
2339	44	1000014		\N
2340	44	1000015		\N
2341	44	1000016		\N
2342	44	1000017		\N
2343	45	http://en.wikipedia.org/wiki/Transversion		\N
2344	44	1000018		\N
2345	44	1000019		\N
2346	44	1000020		\N
2347	44	1000021		\N
2348	44	1000022		\N
2349	44	1000023		\N
2350	44	1000024		\N
2351	44	1000025		\N
2352	44	1000026		\N
2353	44	1000027		\N
2354	44	1000028		\N
2355	44	1000029		\N
2356	45	http://en.wikipedia.org/wiki/Chromosomal_deletion		\N
2357	44	1000030		\N
2358	45	http://en.wikipedia.org/wiki/Chromosomal_inversion		\N
2359	44	1000031		\N
2360	44	1000032		\N
2361	48	http://www.hgvs.org/mutnomen/recs-DNA.html#indel		\N
2362	45	http://en.wikipedia.org/wiki/Indel		\N
2363	44	1000035		\N
2364	44	1000036		\N
2365	44	1000037		\N
2366	45	http://en.wikipedia.org/wiki/Chromosomal_duplication		\N
2367	44	1000038		\N
2368	44	1000039		\N
2369	44	1000040		\N
2370	44	1000041		\N
2371	44	1000042		\N
2372	44	1000043		\N
2373	45	http://en.wikipedia.org/wiki/Robertsonian_translocation		\N
2374	45	http://en.wikipedia.org/wiki/Robertsonian_fusion		\N
2375	44	1000044		\N
2376	45	http://en.wikipedia.org/wiki/Chromosomal_translocation		\N
2377	44	1000045		\N
2378	45	http://en.wikipedia.org/wiki/Ring_chromosome		\N
2379	44	1000046		\N
2380	44	1000047		\N
2381	44	1000048		\N
2382	44	1000049		\N
2383	44	1000177		\N
2384	44	1000179		\N
2385	44	1000050		\N
2386	44	1000052		\N
2387	44	1000054		\N
2388	44	1000055		\N
2389	44	1000056		\N
2390	44	1000057		\N
2391	44	1000058		\N
2392	44	1000059		\N
2393	44	1000060		\N
2394	44	1000061		\N
2395	44	1000062		\N
2396	44	1000063		\N
2397	44	1000064		\N
2398	44	1000065		\N
2399	45	http://en.wikipedia.org/wiki/Frameshift_mutation		\N
2400	44	1000066		\N
2401	44	1000067		\N
2402	44	1000068		\N
2403	44	1000069		\N
2404	44	1000070		\N
2405	44	1000071		\N
2406	44	1000072		\N
2407	44	1000073		\N
2408	44	1000074		\N
2409	44	1000075		\N
2410	44	1000076		\N
2411	44	1000078		\N
2412	44	1000079		\N
2413	44	1000080		\N
2414	44	1000081		\N
2415	44	1000082		\N
2416	44	1000083		\N
2417	44	1000084		\N
2418	44	1000085		\N
2419	44	1000086		\N
2420	44	1000087		\N
2421	44	1000088		\N
2422	44	1000090		\N
2423	44	1000091		\N
2424	44	1000089		\N
2425	44	1000092		\N
2426	44	1000093		\N
2427	44	1000094		\N
2428	44	1000095		\N
2429	44	1000096		\N
2430	44	1000097		\N
2431	44	1000098		\N
2432	44	1000099		\N
2433	44	1000100		\N
2434	44	1000101		\N
2435	44	1000102		\N
2436	44	1000103		\N
2437	44	1000104		\N
2438	44	1000105		\N
2439	44	1000106		\N
2440	44	1000107		\N
2441	44	1000108		\N
2442	44	1000109		\N
2443	44	1000110		\N
2444	44	1000111		\N
2445	44	1000113		\N
2446	44	1000114		\N
2447	44	1000112		\N
2448	44	1000115		\N
2449	44	1000116		\N
2450	44	1000117		\N
2451	44	1000118		\N
2452	44	1000119		\N
2453	44	1000120		\N
2454	44	1000121		\N
2455	44	1000122		\N
2456	44	1000123		\N
2457	44	1000124		\N
2458	44	1000125		\N
2459	44	1000126		\N
2460	44	1000127		\N
2461	44	1000132		\N
2462	44	1000134		\N
2463	44	1000136		\N
2464	46	6804304		\N
2465	44	1000138		\N
2466	44	1000140		\N
2467	44	1000141		\N
2468	44	1000142		\N
2469	49	manual		\N
2470	44	1000143		\N
2471	44	1000144		\N
2472	44	1000145		\N
2473	44	1000146		\N
2474	44	1000147		\N
2475	44	1000148		\N
2476	44	1000149		\N
2477	44	1000150		\N
2478	44	1000151		\N
2479	44	1000152		\N
2480	44	1000153		\N
2481	44	1000154		\N
2482	44	1000155		\N
2483	44	1000156		\N
2484	44	1000157		\N
2485	44	1000158		\N
2486	44	1000159		\N
2487	44	1000160		\N
2488	44	1000161		\N
2489	44	1000162		\N
2490	44	1000170		\N
2491	44	1000171		\N
2492	44	1000173		\N
2493	44	1000175		\N
2494	44	1000180		\N
2495	44	1000181		\N
2496	44	1000182		\N
2497	44	1000183		\N
2498	44	1000184		\N
2499	44	1000185		\N
2500	44	1000186		\N
2501	44	1001186		\N
2502	44	1001187		\N
2503	44	1001188		\N
2504	44	1001189		\N
2505	44	1001190		\N
2506	44	1001191		\N
2507	44	1001192		\N
2508	44	1001193		\N
2509	44	1001194		\N
2510	44	1001195		\N
2511	44	1001196		\N
2512	44	1001197		\N
2513	44	1001217		\N
2514	44	1001244		\N
2515	44	1001246		\N
2516	44	1001247		\N
2517	44	1001249		\N
2518	44	1001251		\N
2519	44	1001254		\N
2520	44	1001255		\N
2521	44	1001259		\N
2522	44	1001260		\N
2523	46	12519954		\N
2524	44	1001261		\N
2525	45	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8811194&amp;dopt=Abstract		\N
2526	44	1001262		\N
2527	44	1001263		\N
2528	44	1001264		\N
2529	44	1001265		\N
2530	44	1001266		\N
2531	44	1001267		\N
2532	44	1001268		\N
2533	45	http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12519954&amp;dopt=Abstract		\N
2534	44	1001269		\N
2535	44	1001270		\N
2536	44	1001271		\N
2537	46	9301331		\N
2538	44	1001272		\N
2539	44	1001273		\N
2540	44	1001274		\N
2541	45	http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00031		\N
2542	45	http://en.wikipedia.org/wiki/SECIS_element		\N
2543	44	1001275		\N
2544	44	1001277		\N
2545	44	1001279		\N
2546	44	1001280		\N
2547	44	1001281		\N
2548	44	1001282		\N
2549	44	1001283		\N
2550	44	1001284		\N
2551	45	http://en.wikipedia.org/wiki/Regulon		\N
2552	44	1001285		\N
2553	44	1001286		\N
2554	44	1001287		\N
2555	45	http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8709208&amp;dopt=Abstract		\N
2556	44	1001288		\N
2557	44	2000061		\N
2558	44	3000000		\N
2559	50	add		\N
2560	41	decreased_in_magnitude_relative_to		\N
2561	41	has_cross_section		\N
2562	41	has_dividend_entity		\N
2563	41	has_dividend_quality		\N
2564	41	has_divisor_entity		\N
2565	41	has_divisor_quality		\N
2566	41	has_ratio_quality		\N
2567	41	has_relative_magnitude		\N
2568	41	increased_in_magnitude_relative_to		\N
2569	41	is_magnitude_of		\N
2570	41	is_measurement_of		\N
2571	41	is_unit_of		\N
2572	41	reciprocal_of		\N
2573	41	singly_occurring_form_of		\N
2574	41	towards		\N
2575	41	unit_of		\N
2576	76	0000000		\N
2577	77	Wikipedia		\N
2578	76	0000001		\N
2579	76	0000002		\N
2580	76	0000003		\N
2581	76	0000004		\N
2582	76	0000005		\N
2583	76	0000006		\N
2584	76	0000007		\N
2585	76	0000008		\N
2586	78	NIST		\N
2587	76	0000009		\N
2588	76	0000010		\N
2589	76	0000011		\N
2590	76	0000012		\N
2591	76	0000013		\N
2592	76	0000014		\N
2593	76	0000015		\N
2594	76	0000016		\N
2595	76	0000017		\N
2596	76	0000018		\N
2597	76	0000019		\N
2598	76	0000020		\N
2599	76	0000021		\N
2600	76	0000022		\N
2601	76	0000023		\N
2602	76	0000024		\N
2603	76	0000025		\N
2604	76	0000026		\N
2605	76	0000027		\N
2606	76	0000028		\N
2607	76	0000029		\N
2608	76	0000030		\N
2609	76	0000031		\N
2610	76	0000032		\N
2611	76	0000033		\N
2612	76	0000034		\N
2613	76	0000035		\N
2614	76	0000036		\N
2615	76	0000037		\N
2616	76	0000038		\N
2617	76	0000039		\N
2618	76	0000040		\N
2619	76	0000041		\N
2620	76	0000042		\N
2621	76	0000043		\N
2622	76	0000044		\N
2623	76	0000045		\N
2624	76	0000046		\N
2625	76	0000047		\N
2626	76	0000048		\N
2627	76	0000049		\N
2628	76	0000050		\N
2629	76	0000051		\N
2630	76	0000052		\N
2631	76	0000053		\N
2632	76	0000054		\N
2633	76	0000055		\N
2634	76	0000056		\N
2635	76	0000057		\N
2636	76	0000058		\N
2637	76	0000059		\N
2638	76	0000060		\N
2639	76	0000061		\N
2640	76	0000062		\N
2641	76	0000063		\N
2642	76	0000064		\N
2643	76	0000065		\N
2644	76	0000066		\N
2645	76	0000067		\N
2646	76	0000068		\N
2647	76	0000069		\N
2648	76	0000070		\N
2649	76	0000071		\N
2650	76	0000072		\N
2651	76	0000073		\N
2652	76	0000074		\N
2653	76	0000075		\N
2654	76	0000076		\N
2655	76	0000077		\N
2656	76	0000078		\N
2657	76	0000079		\N
2658	76	0000080		\N
2659	76	0000081		\N
2660	76	0000082		\N
2661	76	0000083		\N
2662	76	0000084		\N
2663	76	0000301		\N
2664	79	GVG		\N
2665	76	0000085		\N
2666	76	0000086		\N
2667	76	0000087		\N
2668	76	0000088		\N
2669	76	0000089		\N
2670	76	0000090		\N
2671	76	0000091		\N
2672	76	0000092		\N
2673	76	0000093		\N
2674	76	0000094		\N
2675	76	0000095		\N
2676	76	0000096		\N
2677	76	0000097		\N
2678	76	0000098		\N
2679	76	0000099		\N
2680	76	0000100		\N
2681	76	0000101		\N
2682	76	0000102		\N
2683	76	0000103		\N
2684	76	0000104		\N
2685	76	0000105		\N
2686	76	0000106		\N
2687	76	0000107		\N
2688	76	0000108		\N
2689	76	0000109		\N
2690	76	0000110		\N
2691	76	0000111		\N
2692	76	0000112		\N
2693	76	0000113		\N
2694	76	0000114		\N
2695	76	0000115		\N
2696	76	0000116		\N
2697	76	0000117		\N
2698	76	0000118		\N
2699	76	0000119		\N
2700	76	0000120		\N
2701	76	0000121		\N
2702	76	0000122		\N
2703	76	0000123		\N
2704	76	0000124		\N
2705	76	0000125		\N
2706	76	0000126		\N
2707	76	0000127		\N
2708	80	OCRBS		\N
2709	76	0000128		\N
2710	81	DEFRA		\N
2711	76	0000129		\N
2712	76	0000130		\N
2713	82	ORCBS		\N
2714	76	0000131		\N
2715	76	0000132		\N
2716	76	0000133		\N
2717	76	0000134		\N
2718	76	0000135		\N
2719	76	0000136		\N
2720	76	0000137		\N
2721	76	0000138		\N
2722	76	0000139		\N
2723	76	0000140		\N
2724	76	0000141		\N
2725	76	0000142		\N
2726	76	0000143		\N
2727	76	0000144		\N
2728	76	0000145		\N
2729	76	0000146		\N
2730	76	0000147		\N
2731	76	0000148		\N
2732	76	0000149		\N
2733	76	0000150		\N
2734	76	0000151		\N
2735	76	0000152		\N
2736	83	MGED		\N
2737	76	0000153		\N
2738	76	0000154		\N
2739	76	0000155		\N
2740	76	0000156		\N
2741	76	0000157		\N
2742	76	0000158		\N
2743	76	0000159		\N
2744	76	0000160		\N
2745	76	0000161		\N
2746	76	0000162		\N
2747	76	0000163		\N
2748	76	0000164		\N
2749	76	0000165		\N
2750	76	0000166		\N
2751	76	0000167		\N
2752	76	0000168		\N
2753	76	0000169		\N
2754	76	0000170		\N
2755	76	0000171		\N
2756	76	0000172		\N
2757	76	0000173		\N
2758	76	0000174		\N
2759	76	0000175		\N
2760	76	0000176		\N
2761	76	0000177		\N
2762	84	Webmd		\N
2763	76	0000178		\N
2764	76	0000179		\N
2765	76	0000180		\N
2766	76	0000181		\N
2767	76	0000182		\N
2768	76	0000183		\N
2769	76	0000184		\N
2770	76	0000185		\N
2771	76	0000186		\N
2772	76	0000187		\N
2773	76	0000188		\N
2774	76	0000189		\N
2775	76	0000190		\N
2776	76	0000191		\N
2777	76	0000192		\N
2778	76	0000193		\N
2779	76	0000194		\N
2780	76	0000195		\N
2781	76	0000196		\N
2782	76	0000197		\N
2783	76	0000198		\N
2784	76	0000199		\N
2785	76	0000200		\N
2786	85	Bioedonline		\N
2787	76	0000201		\N
2788	76	0000202		\N
2789	76	0000203		\N
2790	76	0000204		\N
2791	76	0000205		\N
2792	76	0000206		\N
2793	76	0000207		\N
2794	76	0000208		\N
2795	76	0000209		\N
2796	76	0000210		\N
2797	76	0000211		\N
2798	76	0000212		\N
2799	76	0000213		\N
2800	76	0000214		\N
2801	76	0000215		\N
2802	76	0000216		\N
2803	76	0000217		\N
2804	76	0000218		\N
2805	76	0000219		\N
2806	86	WordNet		\N
2807	76	0000220		\N
2808	76	0000221		\N
2809	76	0000222		\N
2810	76	0000223		\N
2811	76	0000224		\N
2812	76	0000225		\N
2813	76	0000226		\N
2814	87	ScienceLobby		\N
2815	76	0000227		\N
2816	88	allnet		\N
2817	76	0000228		\N
2818	76	0000229		\N
2819	89	GVG		\N
2820	76	0000230		\N
2821	76	0000231		\N
2822	76	0000232		\N
2823	76	0000233		\N
2824	76	0000234		\N
2825	76	0000235		\N
2826	76	0000236		\N
2827	76	0000237		\N
2828	76	0000238		\N
2829	76	0000239		\N
2830	76	0000240		\N
2831	76	0000241		\N
2832	76	0000242		\N
2833	76	0000243		\N
2834	76	0000244		\N
2835	76	GVG		\N
2836	76	0000245		\N
2837	76	0000246		\N
2838	76	0000247		\N
2839	76	0000248		\N
2840	76	0000249		\N
2841	76	0000250		\N
2842	76	0000251		\N
2843	76	0000252		\N
2844	76	0000253		\N
2845	76	0000254		\N
2846	76	0000255		\N
2847	76	0000256		\N
2848	76	0000257		\N
2849	76	0000258		\N
2850	76	0000259		\N
2851	90	techtarget		\N
2852	76	0000260		\N
2853	76	0000261		\N
2854	91	GVG		\N
2855	76	0000262		\N
2856	76	0000263		\N
2857	76	0000264		\N
2858	76	0000265		\N
2859	76	0000266		\N
2860	76	0000267		\N
2861	77	http://en.wikipedia.org/wiki/Electric_field		\N
2862	76	0000268		\N
2863	76	0000269		\N
2864	77	http://en.wikipedia.org/wiki/Absorbance_		\N
2865	76	0000270		\N
2866	76	0000271		\N
2868	92	connected_to		\N
2869	92	0000000		\N
2870	92	0000001		\N
2871	92	0000002		\N
2872	92	0000003		\N
2873	92	0000004		\N
2874	92	0000005		\N
2875	92	0000006		\N
2876	92	0000007		\N
2877	92	0000008		\N
2878	92	0000009		\N
2879	92	0000010		\N
2880	92	0000011		\N
2881	92	0000012		\N
2882	92	0000013		\N
2883	92	0000014		\N
2884	92	0000015		\N
2885	92	0000016		\N
2886	92	0000017		\N
2887	92	0000018		\N
2888	92	0000019		\N
2889	92	0000020		\N
2890	92	0000021		\N
2891	92	0000022		\N
2892	92	0000023		\N
2893	92	0000024		\N
2894	92	0000025		\N
2895	92	0000026		\N
2896	92	0000027		\N
2897	92	0000028		\N
2898	92	0000029		\N
2899	92	0000030		\N
2900	92	0000031		\N
2901	92	0000032		\N
2902	92	0000033		\N
2903	92	0000034		\N
2904	92	0000035		\N
2905	92	0000036		\N
2906	92	0000037		\N
2907	92	0000038		\N
2908	92	0000039		\N
2909	92	0000040		\N
2910	92	0000041		\N
2911	92	0000042		\N
2912	92	0000043		\N
2913	92	0000044		\N
2914	92	0000045		\N
2915	92	0000046		\N
2916	92	0000047		\N
2917	92	0000048		\N
2918	92	0000049		\N
2919	92	0000050		\N
2920	92	0000051		\N
2921	92	0000052		\N
2922	92	0000053		\N
2923	92	0000054		\N
2924	92	0000055		\N
2925	92	0000056		\N
2926	92	0000057		\N
2927	92	0000058		\N
2928	92	0000059		\N
2929	92	0000060		\N
2930	92	0000061		\N
2931	92	0000062		\N
2932	92	0000063		\N
2933	92	0000064		\N
2934	92	0000065		\N
2935	92	0000066		\N
2936	92	0000067		\N
2937	92	0000068		\N
2938	92	0000069		\N
2939	92	0000070		\N
2940	92	0000071		\N
2941	92	0000072		\N
2942	92	0000073		\N
2943	92	0000074		\N
2944	92	0000075		\N
2945	92	0000076		\N
2946	92	0000077		\N
2947	92	0000078		\N
2948	92	0000079		\N
2949	92	0000080		\N
2950	92	0000081		\N
2951	92	0000082		\N
2952	92	0000083		\N
2953	92	0000084		\N
2954	92	0000085		\N
2955	92	0000086		\N
2956	92	0000087		\N
2957	92	0000088		\N
2958	92	0000089		\N
2959	92	0000090		\N
2960	92	0000091		\N
2961	92	0000092		\N
2962	92	0000093		\N
2963	92	0000094		\N
2964	92	0000095		\N
2965	92	0000096		\N
2966	92	0000097		\N
2967	92	0000098		\N
2968	92	0000099		\N
2969	92	0000100		\N
2970	92	0000101		\N
2971	92	0000102		\N
2972	92	0000103		\N
2973	92	0000104		\N
2974	92	0000105		\N
2975	92	0000106		\N
2976	92	0000107		\N
2977	92	0000108		\N
2978	92	0000109		\N
2979	92	0000110		\N
2980	92	0000111		\N
2981	92	0000112		\N
2982	92	0000113		\N
2983	92	0000114		\N
2984	92	0000115		\N
2985	92	0000116		\N
2986	92	0000117		\N
2987	92	0000118		\N
2988	92	0000119		\N
2989	92	0000120		\N
2990	92	0000121		\N
2991	92	0000122		\N
2992	92	0000123		\N
2993	92	0000124		\N
2994	92	0000125		\N
2995	92	0000126		\N
2996	92	0000127		\N
2997	92	0000128		\N
2998	92	0000129		\N
2999	92	0000130		\N
3000	92	0000131		\N
3001	92	0000132		\N
3002	92	0000133		\N
3003	92	0000134		\N
3004	92	0000135		\N
3005	92	0000136		\N
3006	92	0000137		\N
3007	92	0000138		\N
3008	92	0000139		\N
3009	92	0000140		\N
3010	92	0000141		\N
3011	92	0000142		\N
3012	92	0000143		\N
3013	92	0000144		\N
3014	92	0000145		\N
3015	92	0000146		\N
3016	92	0000147		\N
3017	92	0000148		\N
3018	92	0000149		\N
3019	92	0000150		\N
3020	92	0000151		\N
3021	92	0000152		\N
3022	92	0000153		\N
3023	92	0000154		\N
\.


--
-- Data for Name: dbxrefprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY dbxrefprop (dbxrefprop_id, dbxref_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: eimage; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY eimage (eimage_id, eimage_data, eimage_type, image_uri) FROM stdin;
\.


--
-- Data for Name: element; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY element (element_id, feature_id, arraydesign_id, type_id, dbxref_id) FROM stdin;
\.


--
-- Data for Name: element_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY element_relationship (element_relationship_id, subject_id, type_id, object_id, value, rank) FROM stdin;
\.


--
-- Data for Name: elementresult; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY elementresult (elementresult_id, element_id, quantification_id, signal) FROM stdin;
\.


--
-- Data for Name: elementresult_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY elementresult_relationship (elementresult_relationship_id, subject_id, type_id, object_id, value, rank) FROM stdin;
\.


--
-- Data for Name: environment; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY environment (environment_id, uniquename, description) FROM stdin;
\.


--
-- Data for Name: environment_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY environment_cvterm (environment_cvterm_id, environment_id, cvterm_id) FROM stdin;
\.


--
-- Data for Name: expression; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY expression (expression_id, uniquename, md5checksum, description) FROM stdin;
\.


--
-- Data for Name: expression_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY expression_cvterm (expression_cvterm_id, expression_id, cvterm_id, rank, cvterm_type_id) FROM stdin;
\.


--
-- Data for Name: expression_cvtermprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY expression_cvtermprop (expression_cvtermprop_id, expression_cvterm_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: expression_image; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY expression_image (expression_image_id, expression_id, eimage_id) FROM stdin;
\.


--
-- Data for Name: expression_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY expression_pub (expression_pub_id, expression_id, pub_id) FROM stdin;
\.


--
-- Data for Name: expressionprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY expressionprop (expressionprop_id, expression_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: feature; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature (feature_id, dbxref_id, organism_id, name, uniquename, residues, seqlen, md5checksum, type_id, is_analysis, is_obsolete, timeaccessioned, timelastmodified) FROM stdin;
\.


--
-- Data for Name: feature_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_cvterm (feature_cvterm_id, feature_id, cvterm_id, pub_id, is_not, rank) FROM stdin;
\.


--
-- Data for Name: feature_cvterm_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_cvterm_dbxref (feature_cvterm_dbxref_id, feature_cvterm_id, dbxref_id) FROM stdin;
\.


--
-- Data for Name: feature_cvterm_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_cvterm_pub (feature_cvterm_pub_id, feature_cvterm_id, pub_id) FROM stdin;
\.


--
-- Data for Name: feature_cvtermprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_cvtermprop (feature_cvtermprop_id, feature_cvterm_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: feature_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_dbxref (feature_dbxref_id, feature_id, dbxref_id, is_current) FROM stdin;
\.


--
-- Data for Name: feature_expression; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_expression (feature_expression_id, expression_id, feature_id, pub_id) FROM stdin;
\.


--
-- Data for Name: feature_expressionprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_expressionprop (feature_expressionprop_id, feature_expression_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: feature_genotype; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_genotype (feature_genotype_id, feature_id, genotype_id, chromosome_id, rank, cgroup, cvterm_id) FROM stdin;
\.


--
-- Data for Name: feature_phenotype; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_phenotype (feature_phenotype_id, feature_id, phenotype_id) FROM stdin;
\.


--
-- Data for Name: feature_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_pub (feature_pub_id, feature_id, pub_id) FROM stdin;
\.


--
-- Data for Name: feature_pubprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_pubprop (feature_pubprop_id, feature_pub_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: feature_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_relationship (feature_relationship_id, subject_id, object_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: feature_relationship_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_relationship_pub (feature_relationship_pub_id, feature_relationship_id, pub_id) FROM stdin;
\.


--
-- Data for Name: feature_relationshipprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_relationshipprop (feature_relationshipprop_id, feature_relationship_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: feature_relationshipprop_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_relationshipprop_pub (feature_relationshipprop_pub_id, feature_relationshipprop_id, pub_id) FROM stdin;
\.


--
-- Data for Name: feature_synonym; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY feature_synonym (feature_synonym_id, synonym_id, feature_id, pub_id, is_current, is_internal) FROM stdin;
\.


--
-- Data for Name: featureloc; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featureloc (featureloc_id, feature_id, srcfeature_id, fmin, is_fmin_partial, fmax, is_fmax_partial, strand, phase, residue_info, locgroup, rank) FROM stdin;
\.


--
-- Data for Name: featureloc_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featureloc_pub (featureloc_pub_id, featureloc_id, pub_id) FROM stdin;
\.


--
-- Data for Name: featuremap; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featuremap (featuremap_id, name, description, unittype_id) FROM stdin;
\.


--
-- Data for Name: featuremap_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featuremap_pub (featuremap_pub_id, featuremap_id, pub_id) FROM stdin;
\.


--
-- Data for Name: featurepos; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featurepos (featurepos_id, featuremap_id, feature_id, map_feature_id, mappos) FROM stdin;
\.


--
-- Data for Name: featureprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featureprop (featureprop_id, feature_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: featureprop_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featureprop_pub (featureprop_pub_id, featureprop_id, pub_id) FROM stdin;
\.


--
-- Data for Name: featurerange; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY featurerange (featurerange_id, featuremap_id, feature_id, leftstartf_id, leftendf_id, rightstartf_id, rightendf_id, rangestr) FROM stdin;
\.


--
-- Data for Name: genotype; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY genotype (genotype_id, name, uniquename, description) FROM stdin;
1	\N	TN7.4	\N
2	\N	TN7.19	\N
3	\N	TN7.18	\N
4	\N	TN7.23	\N
5	\N	TN7.22	\N
6	\N	TN7.2	\N
7	\N	TN8.4	\N
8	\N	TN8.28	\N
9	\N	TN8.24	\N
10	\N	TN8.22	\N
11	\N	TN8.23	\N
12	\N	TN9.24	\N
13	\N	TN9.4	\N
14	\N	TN9.22	\N
15	\N	TN9.3	\N
16	\N	TN9.17	\N
17	\N	TN1.18	\N
18	\N	TN1.1	\N
19	\N	TN1.3	\N
20	\N	TN1.15	\N
21	\N	TN7.13	\N
22	\N	TN7.17	\N
23	\N	TN7.20	\N
24	\N	TN7.11	\N
25	\N	TN8.15	\N
26	\N	TN8.21	\N
27	\N	TN8.25	\N
28	\N	TN8.3	\N
29	\N	TN9.5	\N
30	\N	TN9.15	\N
31	\N	TN9.12	\N
32	\N	TN9.21	\N
33	\N	TN9.20	\N
34	\N	TN1.17	\N
35	\N	TN1.13	\N
36	\N	TN1.16	\N
37	\N	TN1.21	\N
38	\N	TN1.5	\N
39	\N	TN1.11	\N
40	\N	TN8.5	\N
\.


--
-- Data for Name: library; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY library (library_id, organism_id, name, uniquename, type_id, is_obsolete, timeaccessioned, timelastmodified) FROM stdin;
\.


--
-- Data for Name: library_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY library_cvterm (library_cvterm_id, library_id, cvterm_id, pub_id) FROM stdin;
\.


--
-- Data for Name: library_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY library_dbxref (library_dbxref_id, library_id, dbxref_id, is_current) FROM stdin;
\.


--
-- Data for Name: library_feature; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY library_feature (library_feature_id, library_id, feature_id) FROM stdin;
\.


--
-- Data for Name: library_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY library_pub (library_pub_id, library_id, pub_id) FROM stdin;
\.


--
-- Data for Name: library_synonym; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY library_synonym (library_synonym_id, synonym_id, library_id, pub_id, is_current, is_internal) FROM stdin;
\.


--
-- Data for Name: libraryprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY libraryprop (libraryprop_id, library_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: libraryprop_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY libraryprop_pub (libraryprop_pub_id, libraryprop_id, pub_id) FROM stdin;
\.


--
-- Data for Name: magedocumentation; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY magedocumentation (magedocumentation_id, mageml_id, tableinfo_id, row_id, mageidentifier) FROM stdin;
\.


--
-- Data for Name: mageml; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY mageml (mageml_id, mage_package, mage_ml) FROM stdin;
\.


--
-- Data for Name: materialized_view; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY materialized_view (materialized_view_id, last_update, refresh_time, name, mv_schema, mv_table, mv_specs, indexed, query, special_index) FROM stdin;
\.


--
-- Data for Name: nd_experiment; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment (nd_experiment_id, nd_geolocation_id, type_id) FROM stdin;
1	3	2310
2	3	2310
3	3	2310
4	3	2310
5	3	2310
6	3	2310
7	3	2310
8	3	2310
9	3	2310
10	3	2310
11	3	2310
12	3	2310
13	3	2310
14	3	2310
15	3	2310
16	3	2310
17	3	2310
18	3	2310
19	3	2310
20	3	2310
21	3	2310
22	3	2310
23	3	2310
24	3	2310
25	3	2310
26	3	2310
27	3	2310
28	3	2310
29	3	2310
30	3	2310
31	3	2310
32	3	2310
33	3	2310
34	3	2310
35	3	2310
36	3	2310
37	3	2310
38	3	2310
39	3	2310
40	3	2310
41	3	2310
42	3	2310
43	3	2310
44	3	2310
45	3	2310
46	3	2310
47	3	2310
48	3	2310
49	3	2310
50	3	2310
51	3	2310
52	3	2310
53	3	2310
54	3	2310
55	3	2310
56	3	2310
57	3	2310
58	3	2310
59	3	2310
60	3	2310
61	3	2310
62	3	2310
63	3	2310
64	3	2310
65	3	2310
66	3	2310
67	3	2310
68	3	2310
69	3	2310
70	3	2310
71	3	2310
72	3	2310
73	3	2310
74	3	2310
75	3	2310
76	3	2310
77	3	2310
78	3	2310
79	3	2310
80	3	2310
81	3	2310
82	3	2310
83	3	2310
84	3	2310
85	3	2310
86	3	2310
87	3	2310
88	3	2310
89	3	2310
90	3	2310
91	3	2310
92	3	2310
93	3	2310
94	3	2310
95	3	2310
96	3	2310
97	3	2310
98	3	2310
99	3	2310
100	3	2310
101	3	2310
102	3	2310
103	3	2310
104	3	2310
105	3	2310
106	3	2310
107	3	2310
108	3	2310
109	3	2310
110	3	2310
111	3	2310
112	3	2310
113	3	2310
114	3	2310
115	3	2310
116	3	2310
117	3	2310
118	3	2310
119	3	2310
120	3	2310
121	3	2310
122	3	2310
123	3	2310
124	3	2310
125	3	2310
126	3	2310
127	3	2310
128	3	2310
129	3	2310
130	3	2310
131	3	2310
132	3	2310
133	3	2310
134	3	2310
135	3	2310
136	3	2310
137	3	2310
138	3	2310
139	3	2310
140	3	2310
141	3	2310
142	3	2310
143	3	2310
144	3	2310
145	3	2310
146	3	2310
147	3	2310
148	3	2310
149	3	2310
150	3	2310
151	3	2310
152	3	2310
153	3	2310
154	3	2310
155	3	2310
156	3	2310
157	3	2310
158	3	2310
159	3	2310
160	3	2310
161	3	2310
162	3	2310
163	3	2310
164	3	2310
165	3	2310
166	3	2310
167	3	2310
168	3	2310
169	3	2310
170	3	2310
171	3	2310
172	3	2310
173	3	2310
174	3	2310
175	3	2310
176	3	2310
177	3	2310
178	3	2310
179	3	2310
180	3	2310
181	3	2310
182	3	2310
183	3	2310
184	3	2310
185	3	2310
186	3	2310
187	3	2310
188	3	2310
189	3	2310
190	3	2310
191	3	2310
192	3	2310
193	3	2310
194	3	2310
195	3	2310
196	3	2310
197	3	2310
198	3	2310
199	3	2310
200	3	2310
201	3	2310
202	3	2310
203	3	2310
204	3	2310
205	3	2310
206	3	2310
207	3	2310
208	3	2310
209	3	2310
210	3	2310
211	3	2310
212	3	2310
213	3	2310
214	3	2310
215	3	2310
216	3	2310
217	3	2310
218	3	2310
219	3	2310
220	3	2310
221	3	2310
222	3	2310
223	3	2310
224	3	2310
225	3	2310
226	3	2310
227	3	2310
228	3	2310
229	3	2310
230	3	2310
231	3	2310
232	3	2310
233	3	2310
234	3	2310
235	3	2310
236	3	2310
237	3	2310
238	3	2310
239	3	2310
240	3	2310
\.


--
-- Data for Name: nd_experiment_contact; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_contact (nd_experiment_contact_id, nd_experiment_id, contact_id) FROM stdin;
\.


--
-- Data for Name: nd_experiment_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_dbxref (nd_experiment_dbxref_id, nd_experiment_id, dbxref_id) FROM stdin;
\.


--
-- Data for Name: nd_experiment_genotype; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_genotype (nd_experiment_genotype_id, nd_experiment_id, genotype_id) FROM stdin;
\.


--
-- Data for Name: nd_experiment_phenotype; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_phenotype (nd_experiment_phenotype_id, nd_experiment_id, phenotype_id) FROM stdin;
1	1	38
2	1	39
3	1	40
4	1	41
5	1	42
6	1	43
7	1	44
8	1	45
9	1	46
10	1	47
11	1	48
12	1	49
13	1	50
14	1	51
15	1	52
16	1	53
17	1	54
18	1	55
19	1	56
20	1	57
21	1	58
22	1	59
23	1	60
24	1	61
25	1	62
26	1	63
27	1	64
28	1	65
29	1	66
30	1	67
31	1	68
32	1	69
33	1	70
34	1	71
35	1	72
36	1	73
37	1	74
38	1	75
39	1	76
40	1	77
41	1	78
42	1	79
43	1	80
44	1	81
45	1	82
46	1	83
47	1	84
48	1	85
49	1	86
50	1	87
51	1	88
52	1	89
53	1	90
54	1	91
55	1	92
56	1	93
57	1	94
58	1	95
59	1	96
60	1	97
61	1	98
62	1	99
63	1	100
64	1	101
65	1	102
66	1	103
67	1	104
68	1	105
69	1	106
70	1	107
71	1	108
72	1	109
73	1	110
74	1	111
75	1	112
76	1	113
77	1	114
78	1	115
79	1	116
80	1	117
81	1	118
82	1	119
83	1	120
84	1	121
85	1	122
86	1	123
87	1	124
88	1	125
89	1	126
90	1	127
91	1	128
92	1	129
93	1	130
94	2	38
95	2	39
96	2	40
97	2	41
98	2	42
99	2	43
100	2	44
101	2	45
102	2	46
103	2	47
104	2	48
105	2	49
106	2	50
107	2	51
108	2	52
109	2	53
110	2	54
111	2	55
112	2	56
113	2	57
114	2	58
115	2	59
116	2	60
117	2	61
118	2	62
119	2	63
120	2	64
121	2	65
122	2	66
123	2	67
124	2	68
125	2	69
126	2	70
127	2	71
128	2	72
129	2	73
130	2	74
131	2	75
132	2	76
133	2	77
134	2	78
135	2	79
136	2	80
137	2	81
138	2	82
139	2	83
140	2	84
141	2	85
142	2	86
143	2	87
144	2	88
145	2	89
146	2	90
147	2	91
148	2	92
149	2	93
150	2	94
151	2	95
152	2	96
153	2	97
154	2	98
155	2	99
156	2	100
157	2	101
158	2	102
159	2	103
160	2	104
161	2	105
162	2	106
163	2	107
164	2	108
165	2	109
166	2	110
167	2	111
168	2	112
169	2	113
170	2	114
171	2	115
172	2	116
173	2	117
174	2	118
175	2	119
176	2	120
177	2	121
178	2	122
179	2	123
180	2	124
181	2	125
182	2	126
183	2	127
184	2	128
185	2	129
186	2	130
187	3	38
188	3	39
189	3	40
190	3	41
191	3	42
192	3	43
193	3	44
194	3	45
195	3	46
196	3	47
197	3	48
198	3	49
199	3	50
200	3	51
201	3	52
202	3	53
203	3	54
204	3	55
205	3	56
206	3	57
207	3	58
208	3	59
209	3	60
210	3	61
211	3	62
212	3	63
213	3	64
214	3	65
215	3	66
216	3	67
217	3	68
218	3	69
219	3	70
220	3	71
221	3	72
222	3	73
223	3	74
224	3	75
225	3	76
226	3	77
227	3	78
228	3	79
229	3	80
230	3	81
231	3	82
232	3	83
233	3	84
234	3	85
235	3	86
236	3	87
237	3	88
238	3	89
239	3	90
240	3	91
241	3	92
242	3	93
243	3	94
244	3	95
245	3	96
246	3	97
247	3	98
248	3	99
249	3	100
250	3	101
251	3	102
252	3	103
253	3	104
254	3	105
255	3	106
256	3	107
257	3	108
258	3	109
259	3	110
260	3	111
261	3	112
262	3	113
263	3	114
264	3	115
265	3	116
266	3	117
267	3	118
268	3	119
269	3	120
270	3	121
271	3	122
272	3	123
273	3	124
274	3	125
275	3	126
276	3	127
277	3	128
278	3	129
279	3	130
280	4	38
281	4	39
282	4	40
283	4	41
284	4	42
285	4	43
286	4	44
287	4	45
288	4	46
289	4	47
290	4	48
291	4	49
292	4	50
293	4	51
294	4	52
295	4	53
296	4	54
297	4	55
298	4	56
299	4	57
300	4	58
301	4	59
302	4	60
303	4	61
304	4	62
305	4	63
306	4	64
307	4	65
308	4	66
309	4	67
310	4	68
311	4	69
312	4	70
313	4	71
314	4	72
315	4	73
316	4	74
317	4	75
318	4	76
319	4	77
320	4	78
321	4	79
322	4	80
323	4	81
324	4	82
325	4	83
326	4	84
327	4	85
328	4	86
329	4	87
330	4	88
331	4	89
332	4	90
333	4	91
334	4	92
335	4	93
336	4	94
337	4	95
338	4	96
339	4	97
340	4	98
341	4	99
342	4	100
343	4	101
344	4	102
345	4	103
346	4	104
347	4	105
348	4	106
349	4	107
350	4	108
351	4	109
352	4	110
353	4	111
354	4	112
355	4	113
356	4	114
357	4	115
358	4	116
359	4	117
360	4	118
361	4	119
362	4	120
363	4	121
364	4	122
365	4	123
366	4	124
367	4	125
368	4	126
369	4	127
370	4	128
371	4	129
372	4	130
373	5	38
374	5	39
375	5	40
376	5	41
377	5	42
378	5	43
379	5	44
380	5	45
381	5	46
382	5	47
383	5	48
384	5	49
385	5	50
386	5	51
387	5	52
388	5	53
389	5	54
390	5	55
391	5	56
392	5	57
393	5	58
394	5	59
395	5	60
396	5	61
397	5	62
398	5	63
399	5	64
400	5	65
401	5	66
402	5	67
403	5	68
404	5	69
405	5	70
406	5	71
407	5	72
408	5	73
409	5	74
410	5	75
411	5	76
412	5	77
413	5	78
414	5	79
415	5	80
416	5	81
417	5	82
418	5	83
419	5	84
420	5	85
421	5	86
422	5	87
423	5	88
424	5	89
425	5	90
426	5	91
427	5	92
428	5	93
429	5	94
430	5	95
431	5	96
432	5	97
433	5	98
434	5	99
435	5	100
436	5	101
437	5	102
438	5	103
439	5	104
440	5	105
441	5	106
442	5	107
443	5	108
444	5	109
445	5	110
446	5	111
447	5	112
448	5	113
449	5	114
450	5	115
451	5	116
452	5	117
453	5	118
454	5	119
455	5	120
456	5	121
457	5	122
458	5	123
459	5	124
460	5	125
461	5	126
462	5	127
463	5	128
464	5	129
465	5	130
466	6	38
467	6	39
468	6	40
469	6	41
470	6	42
471	6	43
472	6	44
473	6	45
474	6	46
475	6	47
476	6	48
477	6	49
478	6	50
479	6	51
480	6	52
481	6	53
482	6	54
483	6	55
484	6	56
485	6	57
486	6	58
487	6	59
488	6	60
489	6	61
490	6	62
491	6	63
492	6	64
493	6	65
494	6	66
495	6	67
496	6	68
497	6	69
498	6	70
499	6	71
500	6	72
501	6	73
502	6	74
503	6	75
504	6	76
505	6	77
506	6	78
507	6	79
508	6	80
509	6	81
510	6	82
511	6	83
512	6	84
513	6	85
514	6	86
515	6	87
516	6	88
517	6	89
518	6	90
519	6	91
520	6	92
521	6	93
522	6	94
523	6	95
524	6	96
525	6	97
526	6	98
527	6	99
528	6	100
529	6	101
530	6	102
531	6	103
532	6	104
533	6	105
534	6	106
535	6	107
536	6	108
537	6	109
538	6	110
539	6	111
540	6	112
541	6	113
542	6	114
543	6	115
544	6	116
545	6	117
546	6	118
547	6	119
548	6	120
549	6	121
550	6	122
551	6	123
552	6	124
553	6	125
554	6	126
555	6	127
556	6	128
557	6	129
558	6	130
559	7	38
560	7	39
561	7	40
562	7	41
563	7	42
564	7	43
565	7	44
566	7	45
567	7	46
568	7	47
569	7	48
570	7	49
571	7	50
572	7	51
573	7	52
574	7	53
575	7	54
576	7	55
577	7	56
578	7	57
579	7	58
580	7	59
581	7	60
582	7	61
583	7	62
584	7	63
585	7	64
586	7	65
587	7	66
588	7	67
589	7	68
590	7	69
591	7	70
592	7	71
593	7	72
594	7	73
595	7	74
596	7	75
597	7	76
598	7	77
599	7	78
600	7	79
601	7	80
602	7	81
603	7	82
604	7	83
605	7	84
606	7	85
607	7	86
608	7	87
609	7	88
610	7	89
611	7	90
612	7	91
613	7	92
614	7	93
615	7	94
616	7	95
617	7	96
618	7	97
619	7	98
620	7	99
621	7	100
622	7	101
623	7	102
624	7	103
625	7	104
626	7	105
627	7	106
628	7	107
629	7	108
630	7	109
631	7	110
632	7	111
633	7	112
634	7	113
635	7	114
636	7	115
637	7	116
638	7	117
639	7	118
640	7	119
641	7	120
642	7	121
643	7	122
644	7	123
645	7	124
646	7	125
647	7	126
648	7	127
649	7	128
650	7	129
651	7	130
652	8	38
653	8	39
654	8	40
655	8	41
656	8	42
657	8	43
658	8	44
659	8	45
660	8	46
661	8	47
662	8	48
663	8	49
664	8	50
665	8	51
666	8	52
667	8	53
668	8	54
669	8	55
670	8	56
671	8	57
672	8	58
673	8	59
674	8	60
675	8	61
676	8	62
677	8	63
678	8	64
679	8	65
680	8	66
681	8	67
682	8	68
683	8	69
684	8	70
685	8	71
686	8	72
687	8	73
688	8	74
689	8	75
690	8	76
691	8	77
692	8	78
693	8	79
694	8	80
695	8	81
696	8	82
697	8	83
698	8	84
699	8	85
700	8	86
701	8	87
702	8	88
703	8	89
704	8	90
705	8	91
706	8	92
707	8	93
708	8	94
709	8	95
710	8	96
711	8	97
712	8	98
713	8	99
714	8	100
715	8	101
716	8	102
717	8	103
718	8	104
719	8	105
720	8	106
721	8	107
722	8	108
723	8	109
724	8	110
725	8	111
726	8	112
727	8	113
728	8	114
729	8	115
730	8	116
731	8	117
732	8	118
733	8	119
734	8	120
735	8	121
736	8	122
737	8	123
738	8	124
739	8	125
740	8	126
741	8	127
742	8	128
743	8	129
744	8	130
745	9	38
746	9	39
747	9	40
748	9	41
749	9	42
750	9	43
751	9	44
752	9	45
753	9	46
754	9	47
755	9	48
756	9	49
757	9	50
758	9	51
759	9	52
760	9	53
761	9	54
762	9	55
763	9	56
764	9	57
765	9	58
766	9	59
767	9	60
768	9	61
769	9	62
770	9	63
771	9	64
772	9	65
773	9	66
774	9	67
775	9	68
776	9	69
777	9	70
778	9	71
779	9	72
780	9	73
781	9	74
782	9	75
783	9	76
784	9	77
785	9	78
786	9	79
787	9	80
788	9	81
789	9	82
790	9	83
791	9	84
792	9	85
793	9	86
794	9	87
795	9	88
796	9	89
797	9	90
798	9	91
799	9	92
800	9	93
801	9	94
802	9	95
803	9	96
804	9	97
805	9	98
806	9	99
807	9	100
808	9	101
809	9	102
810	9	103
811	9	104
812	9	105
813	9	106
814	9	107
815	9	108
816	9	109
817	9	110
818	9	111
819	9	112
820	9	113
821	9	114
822	9	115
823	9	116
824	9	117
825	9	118
826	9	119
827	9	120
828	9	121
829	9	122
830	9	123
831	9	124
832	9	125
833	9	126
834	9	127
835	9	128
836	9	129
837	9	130
838	10	38
839	10	39
840	10	40
841	10	41
842	10	42
843	10	43
844	10	44
845	10	45
846	10	46
847	10	47
848	10	48
849	10	49
850	10	50
851	10	51
852	10	52
853	10	53
854	10	54
855	10	55
856	10	56
857	10	57
858	10	58
859	10	59
860	10	60
861	10	61
862	10	62
863	10	63
864	10	64
865	10	65
866	10	66
867	10	67
868	10	68
869	10	69
870	10	70
871	10	71
872	10	72
873	10	73
874	10	74
875	10	75
876	10	76
877	10	77
878	10	78
879	10	79
880	10	80
881	10	81
882	10	82
883	10	83
884	10	84
885	10	85
886	10	86
887	10	87
888	10	88
889	10	89
890	10	90
891	10	91
892	10	92
893	10	93
894	10	94
895	10	95
896	10	96
897	10	97
898	10	98
899	10	99
900	10	100
901	10	101
902	10	102
903	10	103
904	10	104
905	10	105
906	10	106
907	10	107
908	10	108
909	10	109
910	10	110
911	10	111
912	10	112
913	10	113
914	10	114
915	10	115
916	10	116
917	10	117
918	10	118
919	10	119
920	10	120
921	10	121
922	10	122
923	10	123
924	10	124
925	10	125
926	10	126
927	10	127
928	10	128
929	10	129
930	10	130
931	11	38
932	11	39
933	11	40
934	11	41
935	11	42
936	11	43
937	11	44
938	11	45
939	11	46
940	11	47
941	11	48
942	11	49
943	11	50
944	11	51
945	11	52
946	11	53
947	11	54
948	11	55
949	11	56
950	11	57
951	11	58
952	11	59
953	11	60
954	11	61
955	11	62
956	11	63
957	11	64
958	11	65
959	11	66
960	11	67
961	11	68
962	11	69
963	11	70
964	11	71
965	11	72
966	11	73
967	11	74
968	11	75
969	11	76
970	11	77
971	11	78
972	11	79
973	11	80
974	11	81
975	11	82
976	11	83
977	11	84
978	11	85
979	11	86
980	11	87
981	11	88
982	11	89
983	11	90
984	11	91
985	11	92
986	11	93
987	11	94
988	11	95
989	11	96
990	11	97
991	11	98
992	11	99
993	11	100
994	11	101
995	11	102
996	11	103
997	11	104
998	11	105
999	11	106
1000	11	107
1001	11	108
1002	11	109
1003	11	110
1004	11	111
1005	11	112
1006	11	113
1007	11	114
1008	11	115
1009	11	116
1010	11	117
1011	11	118
1012	11	119
1013	11	120
1014	11	121
1015	11	122
1016	11	123
1017	11	124
1018	11	125
1019	11	126
1020	11	127
1021	11	128
1022	11	129
1023	11	130
1024	12	38
1025	12	39
1026	12	40
1027	12	41
1028	12	42
1029	12	43
1030	12	44
1031	12	45
1032	12	46
1033	12	47
1034	12	48
1035	12	49
1036	12	50
1037	12	51
1038	12	52
1039	12	53
1040	12	54
1041	12	55
1042	12	56
1043	12	57
1044	12	58
1045	12	59
1046	12	60
1047	12	61
1048	12	62
1049	12	63
1050	12	64
1051	12	65
1052	12	66
1053	12	67
1054	12	68
1055	12	69
1056	12	70
1057	12	71
1058	12	72
1059	12	73
1060	12	74
1061	12	75
1062	12	76
1063	12	77
1064	12	78
1065	12	79
1066	12	80
1067	12	81
1068	12	82
1069	12	83
1070	12	84
1071	12	85
1072	12	86
1073	12	87
1074	12	88
1075	12	89
1076	12	90
1077	12	91
1078	12	92
1079	12	93
1080	12	94
1081	12	95
1082	12	96
1083	12	97
1084	12	98
1085	12	99
1086	12	100
1087	12	101
1088	12	102
1089	12	103
1090	12	104
1091	12	105
1092	12	106
1093	12	107
1094	12	108
1095	12	109
1096	12	110
1097	12	111
1098	12	112
1099	12	113
1100	12	114
1101	12	115
1102	12	116
1103	12	117
1104	12	118
1105	12	119
1106	12	120
1107	12	121
1108	12	122
1109	12	123
1110	12	124
1111	12	125
1112	12	126
1113	12	127
1114	12	128
1115	12	129
1116	12	130
1117	13	38
1118	13	39
1119	13	40
1120	13	41
1121	13	42
1122	13	43
1123	13	44
1124	13	45
1125	13	46
1126	13	47
1127	13	48
1128	13	49
1129	13	50
1130	13	51
1131	13	52
1132	13	53
1133	13	54
1134	13	55
1135	13	56
1136	13	57
1137	13	58
1138	13	59
1139	13	60
1140	13	61
1141	13	62
1142	13	63
1143	13	64
1144	13	65
1145	13	66
1146	13	67
1147	13	68
1148	13	69
1149	13	70
1150	13	71
1151	13	72
1152	13	73
1153	13	74
1154	13	75
1155	13	76
1156	13	77
1157	13	78
1158	13	79
1159	13	80
1160	13	81
1161	13	82
1162	13	83
1163	13	84
1164	13	85
1165	13	86
1166	13	87
1167	13	88
1168	13	89
1169	13	90
1170	13	91
1171	13	92
1172	13	93
1173	13	94
1174	13	95
1175	13	96
1176	13	97
1177	13	98
1178	13	99
1179	13	100
1180	13	101
1181	13	102
1182	13	103
1183	13	104
1184	13	105
1185	13	106
1186	13	107
1187	13	108
1188	13	109
1189	13	110
1190	13	111
1191	13	112
1192	13	113
1193	13	114
1194	13	115
1195	13	116
1196	13	117
1197	13	118
1198	13	119
1199	13	120
1200	13	121
1201	13	122
1202	13	123
1203	13	124
1204	13	125
1205	13	126
1206	13	127
1207	13	128
1208	13	129
1209	13	130
1210	14	38
1211	14	39
1212	14	40
1213	14	41
1214	14	42
1215	14	43
1216	14	44
1217	14	45
1218	14	46
1219	14	47
1220	14	48
1221	14	49
1222	14	50
1223	14	51
1224	14	52
1225	14	53
1226	14	54
1227	14	55
1228	14	56
1229	14	57
1230	14	58
1231	14	59
1232	14	60
1233	14	61
1234	14	62
1235	14	63
1236	14	64
1237	14	65
1238	14	66
1239	14	67
1240	14	68
1241	14	69
1242	14	70
1243	14	71
1244	14	72
1245	14	73
1246	14	74
1247	14	75
1248	14	76
1249	14	77
1250	14	78
1251	14	79
1252	14	80
1253	14	81
1254	14	82
1255	14	83
1256	14	84
1257	14	85
1258	14	86
1259	14	87
1260	14	88
1261	14	89
1262	14	90
1263	14	91
1264	14	92
1265	14	93
1266	14	94
1267	14	95
1268	14	96
1269	14	97
1270	14	98
1271	14	99
1272	14	100
1273	14	101
1274	14	102
1275	14	103
1276	14	104
1277	14	105
1278	14	106
1279	14	107
1280	14	108
1281	14	109
1282	14	110
1283	14	111
1284	14	112
1285	14	113
1286	14	114
1287	14	115
1288	14	116
1289	14	117
1290	14	118
1291	14	119
1292	14	120
1293	14	121
1294	14	122
1295	14	123
1296	14	124
1297	14	125
1298	14	126
1299	14	127
1300	14	128
1301	14	129
1302	14	130
1303	15	38
1304	15	39
1305	15	40
1306	15	41
1307	15	42
1308	15	43
1309	15	44
1310	15	45
1311	15	46
1312	15	47
1313	15	48
1314	15	49
1315	15	50
1316	15	51
1317	15	52
1318	15	53
1319	15	54
1320	15	55
1321	15	56
1322	15	57
1323	15	58
1324	15	59
1325	15	60
1326	15	61
1327	15	62
1328	15	63
1329	15	64
1330	15	65
1331	15	66
1332	15	67
1333	15	68
1334	15	69
1335	15	70
1336	15	71
1337	15	72
1338	15	73
1339	15	74
1340	15	75
1341	15	76
1342	15	77
1343	15	78
1344	15	79
1345	15	80
1346	15	81
1347	15	82
1348	15	83
1349	15	84
1350	15	85
1351	15	86
1352	15	87
1353	15	88
1354	15	89
1355	15	90
1356	15	91
1357	15	92
1358	15	93
1359	15	94
1360	15	95
1361	15	96
1362	15	97
1363	15	98
1364	15	99
1365	15	100
1366	15	101
1367	15	102
1368	15	103
1369	15	104
1370	15	105
1371	15	106
1372	15	107
1373	15	108
1374	15	109
1375	15	110
1376	15	111
1377	15	112
1378	15	113
1379	15	114
1380	15	115
1381	15	116
1382	15	117
1383	15	118
1384	15	119
1385	15	120
1386	15	121
1387	15	122
1388	15	123
1389	15	124
1390	15	125
1391	15	126
1392	15	127
1393	15	128
1394	15	129
1395	15	130
1396	16	38
1397	16	39
1398	16	40
1399	16	41
1400	16	42
1401	16	43
1402	16	44
1403	16	45
1404	16	46
1405	16	47
1406	16	48
1407	16	49
1408	16	50
1409	16	51
1410	16	52
1411	16	53
1412	16	54
1413	16	55
1414	16	56
1415	16	57
1416	16	58
1417	16	59
1418	16	60
1419	16	61
1420	16	62
1421	16	63
1422	16	64
1423	16	65
1424	16	66
1425	16	67
1426	16	68
1427	16	69
1428	16	70
1429	16	71
1430	16	72
1431	16	73
1432	16	74
1433	16	75
1434	16	76
1435	16	77
1436	16	78
1437	16	79
1438	16	80
1439	16	81
1440	16	82
1441	16	83
1442	16	84
1443	16	85
1444	16	86
1445	16	87
1446	16	88
1447	16	89
1448	16	90
1449	16	91
1450	16	92
1451	16	93
1452	16	94
1453	16	95
1454	16	96
1455	16	97
1456	16	98
1457	16	99
1458	16	100
1459	16	101
1460	16	102
1461	16	103
1462	16	104
1463	16	105
1464	16	106
1465	16	107
1466	16	108
1467	16	109
1468	16	110
1469	16	111
1470	16	112
1471	16	113
1472	16	114
1473	16	115
1474	16	116
1475	16	117
1476	16	118
1477	16	119
1478	16	120
1479	16	121
1480	16	122
1481	16	123
1482	16	124
1483	16	125
1484	16	126
1485	16	127
1486	16	128
1487	16	129
1488	16	130
1489	17	38
1490	17	39
1491	17	40
1492	17	41
1493	17	42
1494	17	43
1495	17	44
1496	17	45
1497	17	46
1498	17	47
1499	17	48
1500	17	49
1501	17	50
1502	17	51
1503	17	52
1504	17	53
1505	17	54
1506	17	55
1507	17	56
1508	17	57
1509	17	58
1510	17	59
1511	17	60
1512	17	61
1513	17	62
1514	17	63
1515	17	64
1516	17	65
1517	17	66
1518	17	67
1519	17	68
1520	17	69
1521	17	70
1522	17	71
1523	17	72
1524	17	73
1525	17	74
1526	17	75
1527	17	76
1528	17	77
1529	17	78
1530	17	79
1531	17	80
1532	17	81
1533	17	82
1534	17	83
1535	17	84
1536	17	85
1537	17	86
1538	17	87
1539	17	88
1540	17	89
1541	17	90
1542	17	91
1543	17	92
1544	17	93
1545	17	94
1546	17	95
1547	17	96
1548	17	97
1549	17	98
1550	17	99
1551	17	100
1552	17	101
1553	17	102
1554	17	103
1555	17	104
1556	17	105
1557	17	106
1558	17	107
1559	17	108
1560	17	109
1561	17	110
1562	17	111
1563	17	112
1564	17	113
1565	17	114
1566	17	115
1567	17	116
1568	17	117
1569	17	118
1570	17	119
1571	17	120
1572	17	121
1573	17	122
1574	17	123
1575	17	124
1576	17	125
1577	17	126
1578	17	127
1579	17	128
1580	17	129
1581	17	130
1582	18	38
1583	18	39
1584	18	40
1585	18	41
1586	18	42
1587	18	43
1588	18	44
1589	18	45
1590	18	46
1591	18	47
1592	18	48
1593	18	49
1594	18	50
1595	18	51
1596	18	52
1597	18	53
1598	18	54
1599	18	55
1600	18	56
1601	18	57
1602	18	58
1603	18	59
1604	18	60
1605	18	61
1606	18	62
1607	18	63
1608	18	64
1609	18	65
1610	18	66
1611	18	67
1612	18	68
1613	18	69
1614	18	70
1615	18	71
1616	18	72
1617	18	73
1618	18	74
1619	18	75
1620	18	76
1621	18	77
1622	18	78
1623	18	79
1624	18	80
1625	18	81
1626	18	82
1627	18	83
1628	18	84
1629	18	85
1630	18	86
1631	18	87
1632	18	88
1633	18	89
1634	18	90
1635	18	91
1636	18	92
1637	18	93
1638	18	94
1639	18	95
1640	18	96
1641	18	97
1642	18	98
1643	18	99
1644	18	100
1645	18	101
1646	18	102
1647	18	103
1648	18	104
1649	18	105
1650	18	106
1651	18	107
1652	18	108
1653	18	109
1654	18	110
1655	18	111
1656	18	112
1657	18	113
1658	18	114
1659	18	115
1660	18	116
1661	18	117
1662	18	118
1663	18	119
1664	18	120
1665	18	121
1666	18	122
1667	18	123
1668	18	124
1669	18	125
1670	18	126
1671	18	127
1672	18	128
1673	18	129
1674	18	130
1675	19	38
1676	19	39
1677	19	40
1678	19	41
1679	19	42
1680	19	43
1681	19	44
1682	19	45
1683	19	46
1684	19	47
1685	19	48
1686	19	49
1687	19	50
1688	19	51
1689	19	52
1690	19	53
1691	19	54
1692	19	55
1693	19	56
1694	19	57
1695	19	58
1696	19	59
1697	19	60
1698	19	61
1699	19	62
1700	19	63
1701	19	64
1702	19	65
1703	19	66
1704	19	67
1705	19	68
1706	19	69
1707	19	70
1708	19	71
1709	19	72
1710	19	73
1711	19	74
1712	19	75
1713	19	76
1714	19	77
1715	19	78
1716	19	79
1717	19	80
1718	19	81
1719	19	82
1720	19	83
1721	19	84
1722	19	85
1723	19	86
1724	19	87
1725	19	88
1726	19	89
1727	19	90
1728	19	91
1729	19	92
1730	19	93
1731	19	94
1732	19	95
1733	19	96
1734	19	97
1735	19	98
1736	19	99
1737	19	100
1738	19	101
1739	19	102
1740	19	103
1741	19	104
1742	19	105
1743	19	106
1744	19	107
1745	19	108
1746	19	109
1747	19	110
1748	19	111
1749	19	112
1750	19	113
1751	19	114
1752	19	115
1753	19	116
1754	19	117
1755	19	118
1756	19	119
1757	19	120
1758	19	121
1759	19	122
1760	19	123
1761	19	124
1762	19	125
1763	19	126
1764	19	127
1765	19	128
1766	19	129
1767	19	130
1768	20	38
1769	20	39
1770	20	40
1771	20	41
1772	20	42
1773	20	43
1774	20	44
1775	20	45
1776	20	46
1777	20	47
1778	20	48
1779	20	49
1780	20	50
1781	20	51
1782	20	52
1783	20	53
1784	20	54
1785	20	55
1786	20	56
1787	20	57
1788	20	58
1789	20	59
1790	20	60
1791	20	61
1792	20	62
1793	20	63
1794	20	64
1795	20	65
1796	20	66
1797	20	67
1798	20	68
1799	20	69
1800	20	70
1801	20	71
1802	20	72
1803	20	73
1804	20	74
1805	20	75
1806	20	76
1807	20	77
1808	20	78
1809	20	79
1810	20	80
1811	20	81
1812	20	82
1813	20	83
1814	20	84
1815	20	85
1816	20	86
1817	20	87
1818	20	88
1819	20	89
1820	20	90
1821	20	91
1822	20	92
1823	20	93
1824	20	94
1825	20	95
1826	20	96
1827	20	97
1828	20	98
1829	20	99
1830	20	100
1831	20	101
1832	20	102
1833	20	103
1834	20	104
1835	20	105
1836	20	106
1837	20	107
1838	20	108
1839	20	109
1840	20	110
1841	20	111
1842	20	112
1843	20	113
1844	20	114
1845	20	115
1846	20	116
1847	20	117
1848	20	118
1849	20	119
1850	20	120
1851	20	121
1852	20	122
1853	20	123
1854	20	124
1855	20	125
1856	20	126
1857	20	127
1858	20	128
1859	20	129
1860	20	130
1861	21	38
1862	21	39
1863	21	40
1864	21	41
1865	21	42
1866	21	43
1867	21	44
1868	21	45
1869	21	46
1870	21	47
1871	21	48
1872	21	49
1873	21	50
1874	21	51
1875	21	52
1876	21	53
1877	21	54
1878	21	55
1879	21	56
1880	21	57
1881	21	58
1882	21	59
1883	21	60
1884	21	61
1885	21	62
1886	21	63
1887	21	64
1888	21	65
1889	21	66
1890	21	67
1891	21	68
1892	21	69
1893	21	70
1894	21	71
1895	21	72
1896	21	73
1897	21	74
1898	21	75
1899	21	76
1900	21	77
1901	21	78
1902	21	79
1903	21	80
1904	21	81
1905	21	82
1906	21	83
1907	21	84
1908	21	85
1909	21	86
1910	21	87
1911	21	88
1912	21	89
1913	21	90
1914	21	91
1915	21	92
1916	21	93
1917	21	94
1918	21	95
1919	21	96
1920	21	97
1921	21	98
1922	21	99
1923	21	100
1924	21	101
1925	21	102
1926	21	103
1927	21	104
1928	21	105
1929	21	106
1930	21	107
1931	21	108
1932	21	109
1933	21	110
1934	21	111
1935	21	112
1936	21	113
1937	21	114
1938	21	115
1939	21	116
1940	21	117
1941	21	118
1942	21	119
1943	21	120
1944	21	121
1945	21	122
1946	21	123
1947	21	124
1948	21	125
1949	21	126
1950	21	127
1951	21	128
1952	21	129
1953	21	130
1954	22	38
1955	22	39
1956	22	40
1957	22	41
1958	22	42
1959	22	43
1960	22	44
1961	22	45
1962	22	46
1963	22	47
1964	22	48
1965	22	49
1966	22	50
1967	22	51
1968	22	52
1969	22	53
1970	22	54
1971	22	55
1972	22	56
1973	22	57
1974	22	58
1975	22	59
1976	22	60
1977	22	61
1978	22	62
1979	22	63
1980	22	64
1981	22	65
1982	22	66
1983	22	67
1984	22	68
1985	22	69
1986	22	70
1987	22	71
1988	22	72
1989	22	73
1990	22	74
1991	22	75
1992	22	76
1993	22	77
1994	22	78
1995	22	79
1996	22	80
1997	22	81
1998	22	82
1999	22	83
2000	22	84
2001	22	85
2002	22	86
2003	22	87
2004	22	88
2005	22	89
2006	22	90
2007	22	91
2008	22	92
2009	22	93
2010	22	94
2011	22	95
2012	22	96
2013	22	97
2014	22	98
2015	22	99
2016	22	100
2017	22	101
2018	22	102
2019	22	103
2020	22	104
2021	22	105
2022	22	106
2023	22	107
2024	22	108
2025	22	109
2026	22	110
2027	22	111
2028	22	112
2029	22	113
2030	22	114
2031	22	115
2032	22	116
2033	22	117
2034	22	118
2035	22	119
2036	22	120
2037	22	121
2038	22	122
2039	22	123
2040	22	124
2041	22	125
2042	22	126
2043	22	127
2044	22	128
2045	22	129
2046	22	130
2047	23	38
2048	23	39
2049	23	40
2050	23	41
2051	23	42
2052	23	43
2053	23	44
2054	23	45
2055	23	46
2056	23	47
2057	23	48
2058	23	49
2059	23	50
2060	23	51
2061	23	52
2062	23	53
2063	23	54
2064	23	55
2065	23	56
2066	23	57
2067	23	58
2068	23	59
2069	23	60
2070	23	61
2071	23	62
2072	23	63
2073	23	64
2074	23	65
2075	23	66
2076	23	67
2077	23	68
2078	23	69
2079	23	70
2080	23	71
2081	23	72
2082	23	73
2083	23	74
2084	23	75
2085	23	76
2086	23	77
2087	23	78
2088	23	79
2089	23	80
2090	23	81
2091	23	82
2092	23	83
2093	23	84
2094	23	85
2095	23	86
2096	23	87
2097	23	88
2098	23	89
2099	23	90
2100	23	91
2101	23	92
2102	23	93
2103	23	94
2104	23	95
2105	23	96
2106	23	97
2107	23	98
2108	23	99
2109	23	100
2110	23	101
2111	23	102
2112	23	103
2113	23	104
2114	23	105
2115	23	106
2116	23	107
2117	23	108
2118	23	109
2119	23	110
2120	23	111
2121	23	112
2122	23	113
2123	23	114
2124	23	115
2125	23	116
2126	23	117
2127	23	118
2128	23	119
2129	23	120
2130	23	121
2131	23	122
2132	23	123
2133	23	124
2134	23	125
2135	23	126
2136	23	127
2137	23	128
2138	23	129
2139	23	130
2140	24	38
2141	24	39
2142	24	40
2143	24	41
2144	24	42
2145	24	43
2146	24	44
2147	24	45
2148	24	46
2149	24	47
2150	24	48
2151	24	49
2152	24	50
2153	24	51
2154	24	52
2155	24	53
2156	24	54
2157	24	55
2158	24	56
2159	24	57
2160	24	58
2161	24	59
2162	24	60
2163	24	61
2164	24	62
2165	24	63
2166	24	64
2167	24	65
2168	24	66
2169	24	67
2170	24	68
2171	24	69
2172	24	70
2173	24	71
2174	24	72
2175	24	73
2176	24	74
2177	24	75
2178	24	76
2179	24	77
2180	24	78
2181	24	79
2182	24	80
2183	24	81
2184	24	82
2185	24	83
2186	24	84
2187	24	85
2188	24	86
2189	24	87
2190	24	88
2191	24	89
2192	24	90
2193	24	91
2194	24	92
2195	24	93
2196	24	94
2197	24	95
2198	24	96
2199	24	97
2200	24	98
2201	24	99
2202	24	100
2203	24	101
2204	24	102
2205	24	103
2206	24	104
2207	24	105
2208	24	106
2209	24	107
2210	24	108
2211	24	109
2212	24	110
2213	24	111
2214	24	112
2215	24	113
2216	24	114
2217	24	115
2218	24	116
2219	24	117
2220	24	118
2221	24	119
2222	24	120
2223	24	121
2224	24	122
2225	24	123
2226	24	124
2227	24	125
2228	24	126
2229	24	127
2230	24	128
2231	24	129
2232	24	130
2233	25	38
2234	25	39
2235	25	40
2236	25	41
2237	25	42
2238	25	43
2239	25	44
2240	25	45
2241	25	46
2242	25	47
2243	25	48
2244	25	49
2245	25	50
2246	25	51
2247	25	52
2248	25	53
2249	25	54
2250	25	55
2251	25	56
2252	25	57
2253	25	58
2254	25	59
2255	25	60
2256	25	61
2257	25	62
2258	25	63
2259	25	64
2260	25	65
2261	25	66
2262	25	67
2263	25	68
2264	25	69
2265	25	70
2266	25	71
2267	25	72
2268	25	73
2269	25	74
2270	25	75
2271	25	76
2272	25	77
2273	25	78
2274	25	79
2275	25	80
2276	25	81
2277	25	82
2278	25	83
2279	25	84
2280	25	85
2281	25	86
2282	25	87
2283	25	88
2284	25	89
2285	25	90
2286	25	91
2287	25	92
2288	25	93
2289	25	94
2290	25	95
2291	25	96
2292	25	97
2293	25	98
2294	25	99
2295	25	100
2296	25	101
2297	25	102
2298	25	103
2299	25	104
2300	25	105
2301	25	106
2302	25	107
2303	25	108
2304	25	109
2305	25	110
2306	25	111
2307	25	112
2308	25	113
2309	25	114
2310	25	115
2311	25	116
2312	25	117
2313	25	118
2314	25	119
2315	25	120
2316	25	121
2317	25	122
2318	25	123
2319	25	124
2320	25	125
2321	25	126
2322	25	127
2323	25	128
2324	25	129
2325	25	130
2326	26	38
2327	26	39
2328	26	40
2329	26	41
2330	26	42
2331	26	43
2332	26	44
2333	26	45
2334	26	46
2335	26	47
2336	26	48
2337	26	49
2338	26	50
2339	26	51
2340	26	52
2341	26	53
2342	26	54
2343	26	55
2344	26	56
2345	26	57
2346	26	58
2347	26	59
2348	26	60
2349	26	61
2350	26	62
2351	26	63
2352	26	64
2353	26	65
2354	26	66
2355	26	67
2356	26	68
2357	26	69
2358	26	70
2359	26	71
2360	26	72
2361	26	73
2362	26	74
2363	26	75
2364	26	76
2365	26	77
2366	26	78
2367	26	79
2368	26	80
2369	26	81
2370	26	82
2371	26	83
2372	26	84
2373	26	85
2374	26	86
2375	26	87
2376	26	88
2377	26	89
2378	26	90
2379	26	91
2380	26	92
2381	26	93
2382	26	94
2383	26	95
2384	26	96
2385	26	97
2386	26	98
2387	26	99
2388	26	100
2389	26	101
2390	26	102
2391	26	103
2392	26	104
2393	26	105
2394	26	106
2395	26	107
2396	26	108
2397	26	109
2398	26	110
2399	26	111
2400	26	112
2401	26	113
2402	26	114
2403	26	115
2404	26	116
2405	26	117
2406	26	118
2407	26	119
2408	26	120
2409	26	121
2410	26	122
2411	26	123
2412	26	124
2413	26	125
2414	26	126
2415	26	127
2416	26	128
2417	26	129
2418	26	130
2419	27	38
2420	27	39
2421	27	40
2422	27	41
2423	27	42
2424	27	43
2425	27	44
2426	27	45
2427	27	46
2428	27	47
2429	27	48
2430	27	49
2431	27	50
2432	27	51
2433	27	52
2434	27	53
2435	27	54
2436	27	55
2437	27	56
2438	27	57
2439	27	58
2440	27	59
2441	27	60
2442	27	61
2443	27	62
2444	27	63
2445	27	64
2446	27	65
2447	27	66
2448	27	67
2449	27	68
2450	27	69
2451	27	70
2452	27	71
2453	27	72
2454	27	73
2455	27	74
2456	27	75
2457	27	76
2458	27	77
2459	27	78
2460	27	79
2461	27	80
2462	27	81
2463	27	82
2464	27	83
2465	27	84
2466	27	85
2467	27	86
2468	27	87
2469	27	88
2470	27	89
2471	27	90
2472	27	91
2473	27	92
2474	27	93
2475	27	94
2476	27	95
2477	27	96
2478	27	97
2479	27	98
2480	27	99
2481	27	100
2482	27	101
2483	27	102
2484	27	103
2485	27	104
2486	27	105
2487	27	106
2488	27	107
2489	27	108
2490	27	109
2491	27	110
2492	27	111
2493	27	112
2494	27	113
2495	27	114
2496	27	115
2497	27	116
2498	27	117
2499	27	118
2500	27	119
2501	27	120
2502	27	121
2503	27	122
2504	27	123
2505	27	124
2506	27	125
2507	27	126
2508	27	127
2509	27	128
2510	27	129
2511	27	130
2512	28	38
2513	28	39
2514	28	40
2515	28	41
2516	28	42
2517	28	43
2518	28	44
2519	28	45
2520	28	46
2521	28	47
2522	28	48
2523	28	49
2524	28	50
2525	28	51
2526	28	52
2527	28	53
2528	28	54
2529	28	55
2530	28	56
2531	28	57
2532	28	58
2533	28	59
2534	28	60
2535	28	61
2536	28	62
2537	28	63
2538	28	64
2539	28	65
2540	28	66
2541	28	67
2542	28	68
2543	28	69
2544	28	70
2545	28	71
2546	28	72
2547	28	73
2548	28	74
2549	28	75
2550	28	76
2551	28	77
2552	28	78
2553	28	79
2554	28	80
2555	28	81
2556	28	82
2557	28	83
2558	28	84
2559	28	85
2560	28	86
2561	28	87
2562	28	88
2563	28	89
2564	28	90
2565	28	91
2566	28	92
2567	28	93
2568	28	94
2569	28	95
2570	28	96
2571	28	97
2572	28	98
2573	28	99
2574	28	100
2575	28	101
2576	28	102
2577	28	103
2578	28	104
2579	28	105
2580	28	106
2581	28	107
2582	28	108
2583	28	109
2584	28	110
2585	28	111
2586	28	112
2587	28	113
2588	28	114
2589	28	115
2590	28	116
2591	28	117
2592	28	118
2593	28	119
2594	28	120
2595	28	121
2596	28	122
2597	28	123
2598	28	124
2599	28	125
2600	28	126
2601	28	127
2602	28	128
2603	28	129
2604	28	130
2605	29	38
2606	29	39
2607	29	40
2608	29	41
2609	29	42
2610	29	43
2611	29	44
2612	29	45
2613	29	46
2614	29	47
2615	29	48
2616	29	49
2617	29	50
2618	29	51
2619	29	52
2620	29	53
2621	29	54
2622	29	55
2623	29	56
2624	29	57
2625	29	58
2626	29	59
2627	29	60
2628	29	61
2629	29	62
2630	29	63
2631	29	64
2632	29	65
2633	29	66
2634	29	67
2635	29	68
2636	29	69
2637	29	70
2638	29	71
2639	29	72
2640	29	73
2641	29	74
2642	29	75
2643	29	76
2644	29	77
2645	29	78
2646	29	79
2647	29	80
2648	29	81
2649	29	82
2650	29	83
2651	29	84
2652	29	85
2653	29	86
2654	29	87
2655	29	88
2656	29	89
2657	29	90
2658	29	91
2659	29	92
2660	29	93
2661	29	94
2662	29	95
2663	29	96
2664	29	97
2665	29	98
2666	29	99
2667	29	100
2668	29	101
2669	29	102
2670	29	103
2671	29	104
2672	29	105
2673	29	106
2674	29	107
2675	29	108
2676	29	109
2677	29	110
2678	29	111
2679	29	112
2680	29	113
2681	29	114
2682	29	115
2683	29	116
2684	29	117
2685	29	118
2686	29	119
2687	29	120
2688	29	121
2689	29	122
2690	29	123
2691	29	124
2692	29	125
2693	29	126
2694	29	127
2695	29	128
2696	29	129
2697	29	130
2698	30	38
2699	30	39
2700	30	40
2701	30	41
2702	30	42
2703	30	43
2704	30	44
2705	30	45
2706	30	46
2707	30	47
2708	30	48
2709	30	49
2710	30	50
2711	30	51
2712	30	52
2713	30	53
2714	30	54
2715	30	55
2716	30	56
2717	30	57
2718	30	58
2719	30	59
2720	30	60
2721	30	61
2722	30	62
2723	30	63
2724	30	64
2725	30	65
2726	30	66
2727	30	67
2728	30	68
2729	30	69
2730	30	70
2731	30	71
2732	30	72
2733	30	73
2734	30	74
2735	30	75
2736	30	76
2737	30	77
2738	30	78
2739	30	79
2740	30	80
2741	30	81
2742	30	82
2743	30	83
2744	30	84
2745	30	85
2746	30	86
2747	30	87
2748	30	88
2749	30	89
2750	30	90
2751	30	91
2752	30	92
2753	30	93
2754	30	94
2755	30	95
2756	30	96
2757	30	97
2758	30	98
2759	30	99
2760	30	100
2761	30	101
2762	30	102
2763	30	103
2764	30	104
2765	30	105
2766	30	106
2767	30	107
2768	30	108
2769	30	109
2770	30	110
2771	30	111
2772	30	112
2773	30	113
2774	30	114
2775	30	115
2776	30	116
2777	30	117
2778	30	118
2779	30	119
2780	30	120
2781	30	121
2782	30	122
2783	30	123
2784	30	124
2785	30	125
2786	30	126
2787	30	127
2788	30	128
2789	30	129
2790	30	130
2791	31	38
2792	31	39
2793	31	40
2794	31	41
2795	31	42
2796	31	43
2797	31	44
2798	31	45
2799	31	46
2800	31	47
2801	31	48
2802	31	49
2803	31	50
2804	31	51
2805	31	52
2806	31	53
2807	31	54
2808	31	55
2809	31	56
2810	31	57
2811	31	58
2812	31	59
2813	31	60
2814	31	61
2815	31	62
2816	31	63
2817	31	64
2818	31	65
2819	31	66
2820	31	67
2821	31	68
2822	31	69
2823	31	70
2824	31	71
2825	31	72
2826	31	73
2827	31	74
2828	31	75
2829	31	76
2830	31	77
2831	31	78
2832	31	79
2833	31	80
2834	31	81
2835	31	82
2836	31	83
2837	31	84
2838	31	85
2839	31	86
2840	31	87
2841	31	88
2842	31	89
2843	31	90
2844	31	91
2845	31	92
2846	31	93
2847	31	94
2848	31	95
2849	31	96
2850	31	97
2851	31	98
2852	31	99
2853	31	100
2854	31	101
2855	31	102
2856	31	103
2857	31	104
2858	31	105
2859	31	106
2860	31	107
2861	31	108
2862	31	109
2863	31	110
2864	31	111
2865	31	112
2866	31	113
2867	31	114
2868	31	115
2869	31	116
2870	31	117
2871	31	118
2872	31	119
2873	31	120
2874	31	121
2875	31	122
2876	31	123
2877	31	124
2878	31	125
2879	31	126
2880	31	127
2881	31	128
2882	31	129
2883	31	130
2884	32	38
2885	32	39
2886	32	40
2887	32	41
2888	32	42
2889	32	43
2890	32	44
2891	32	45
2892	32	46
2893	32	47
2894	32	48
2895	32	49
2896	32	50
2897	32	51
2898	32	52
2899	32	53
2900	32	54
2901	32	55
2902	32	56
2903	32	57
2904	32	58
2905	32	59
2906	32	60
2907	32	61
2908	32	62
2909	32	63
2910	32	64
2911	32	65
2912	32	66
2913	32	67
2914	32	68
2915	32	69
2916	32	70
2917	32	71
2918	32	72
2919	32	73
2920	32	74
2921	32	75
2922	32	76
2923	32	77
2924	32	78
2925	32	79
2926	32	80
2927	32	81
2928	32	82
2929	32	83
2930	32	84
2931	32	85
2932	32	86
2933	32	87
2934	32	88
2935	32	89
2936	32	90
2937	32	91
2938	32	92
2939	32	93
2940	32	94
2941	32	95
2942	32	96
2943	32	97
2944	32	98
2945	32	99
2946	32	100
2947	32	101
2948	32	102
2949	32	103
2950	32	104
2951	32	105
2952	32	106
2953	32	107
2954	32	108
2955	32	109
2956	32	110
2957	32	111
2958	32	112
2959	32	113
2960	32	114
2961	32	115
2962	32	116
2963	32	117
2964	32	118
2965	32	119
2966	32	120
2967	32	121
2968	32	122
2969	32	123
2970	32	124
2971	32	125
2972	32	126
2973	32	127
2974	32	128
2975	32	129
2976	32	130
2977	33	38
2978	33	39
2979	33	40
2980	33	41
2981	33	42
2982	33	43
2983	33	44
2984	33	45
2985	33	46
2986	33	47
2987	33	48
2988	33	49
2989	33	50
2990	33	51
2991	33	52
2992	33	53
2993	33	54
2994	33	55
2995	33	56
2996	33	57
2997	33	58
2998	33	59
2999	33	60
3000	33	61
3001	33	62
3002	33	63
3003	33	64
3004	33	65
3005	33	66
3006	33	67
3007	33	68
3008	33	69
3009	33	70
3010	33	71
3011	33	72
3012	33	73
3013	33	74
3014	33	75
3015	33	76
3016	33	77
3017	33	78
3018	33	79
3019	33	80
3020	33	81
3021	33	82
3022	33	83
3023	33	84
3024	33	85
3025	33	86
3026	33	87
3027	33	88
3028	33	89
3029	33	90
3030	33	91
3031	33	92
3032	33	93
3033	33	94
3034	33	95
3035	33	96
3036	33	97
3037	33	98
3038	33	99
3039	33	100
3040	33	101
3041	33	102
3042	33	103
3043	33	104
3044	33	105
3045	33	106
3046	33	107
3047	33	108
3048	33	109
3049	33	110
3050	33	111
3051	33	112
3052	33	113
3053	33	114
3054	33	115
3055	33	116
3056	33	117
3057	33	118
3058	33	119
3059	33	120
3060	33	121
3061	33	122
3062	33	123
3063	33	124
3064	33	125
3065	33	126
3066	33	127
3067	33	128
3068	33	129
3069	33	130
3070	34	38
3071	34	39
3072	34	40
3073	34	41
3074	34	42
3075	34	43
3076	34	44
3077	34	45
3078	34	46
3079	34	47
3080	34	48
3081	34	49
3082	34	50
3083	34	51
3084	34	52
3085	34	53
3086	34	54
3087	34	55
3088	34	56
3089	34	57
3090	34	58
3091	34	59
3092	34	60
3093	34	61
3094	34	62
3095	34	63
3096	34	64
3097	34	65
3098	34	66
3099	34	67
3100	34	68
3101	34	69
3102	34	70
3103	34	71
3104	34	72
3105	34	73
3106	34	74
3107	34	75
3108	34	76
3109	34	77
3110	34	78
3111	34	79
3112	34	80
3113	34	81
3114	34	82
3115	34	83
3116	34	84
3117	34	85
3118	34	86
3119	34	87
3120	34	88
3121	34	89
3122	34	90
3123	34	91
3124	34	92
3125	34	93
3126	34	94
3127	34	95
3128	34	96
3129	34	97
3130	34	98
3131	34	99
3132	34	100
3133	34	101
3134	34	102
3135	34	103
3136	34	104
3137	34	105
3138	34	106
3139	34	107
3140	34	108
3141	34	109
3142	34	110
3143	34	111
3144	34	112
3145	34	113
3146	34	114
3147	34	115
3148	34	116
3149	34	117
3150	34	118
3151	34	119
3152	34	120
3153	34	121
3154	34	122
3155	34	123
3156	34	124
3157	34	125
3158	34	126
3159	34	127
3160	34	128
3161	34	129
3162	34	130
3163	35	38
3164	35	39
3165	35	40
3166	35	41
3167	35	42
3168	35	43
3169	35	44
3170	35	45
3171	35	46
3172	35	47
3173	35	48
3174	35	49
3175	35	50
3176	35	51
3177	35	52
3178	35	53
3179	35	54
3180	35	55
3181	35	56
3182	35	57
3183	35	58
3184	35	59
3185	35	60
3186	35	61
3187	35	62
3188	35	63
3189	35	64
3190	35	65
3191	35	66
3192	35	67
3193	35	68
3194	35	69
3195	35	70
3196	35	71
3197	35	72
3198	35	73
3199	35	74
3200	35	75
3201	35	76
3202	35	77
3203	35	78
3204	35	79
3205	35	80
3206	35	81
3207	35	82
3208	35	83
3209	35	84
3210	35	85
3211	35	86
3212	35	87
3213	35	88
3214	35	89
3215	35	90
3216	35	91
3217	35	92
3218	35	93
3219	35	94
3220	35	95
3221	35	96
3222	35	97
3223	35	98
3224	35	99
3225	35	100
3226	35	101
3227	35	102
3228	35	103
3229	35	104
3230	35	105
3231	35	106
3232	35	107
3233	35	108
3234	35	109
3235	35	110
3236	35	111
3237	35	112
3238	35	113
3239	35	114
3240	35	115
3241	35	116
3242	35	117
3243	35	118
3244	35	119
3245	35	120
3246	35	121
3247	35	122
3248	35	123
3249	35	124
3250	35	125
3251	35	126
3252	35	127
3253	35	128
3254	35	129
3255	35	130
3256	36	38
3257	36	39
3258	36	40
3259	36	41
3260	36	42
3261	36	43
3262	36	44
3263	36	45
3264	36	46
3265	36	47
3266	36	48
3267	36	49
3268	36	50
3269	36	51
3270	36	52
3271	36	53
3272	36	54
3273	36	55
3274	36	56
3275	36	57
3276	36	58
3277	36	59
3278	36	60
3279	36	61
3280	36	62
3281	36	63
3282	36	64
3283	36	65
3284	36	66
3285	36	67
3286	36	68
3287	36	69
3288	36	70
3289	36	71
3290	36	72
3291	36	73
3292	36	74
3293	36	75
3294	36	76
3295	36	77
3296	36	78
3297	36	79
3298	36	80
3299	36	81
3300	36	82
3301	36	83
3302	36	84
3303	36	85
3304	36	86
3305	36	87
3306	36	88
3307	36	89
3308	36	90
3309	36	91
3310	36	92
3311	36	93
3312	36	94
3313	36	95
3314	36	96
3315	36	97
3316	36	98
3317	36	99
3318	36	100
3319	36	101
3320	36	102
3321	36	103
3322	36	104
3323	36	105
3324	36	106
3325	36	107
3326	36	108
3327	36	109
3328	36	110
3329	36	111
3330	36	112
3331	36	113
3332	36	114
3333	36	115
3334	36	116
3335	36	117
3336	36	118
3337	36	119
3338	36	120
3339	36	121
3340	36	122
3341	36	123
3342	36	124
3343	36	125
3344	36	126
3345	36	127
3346	36	128
3347	36	129
3348	36	130
3349	37	38
3350	37	39
3351	37	40
3352	37	41
3353	37	42
3354	37	43
3355	37	44
3356	37	45
3357	37	46
3358	37	47
3359	37	48
3360	37	49
3361	37	50
3362	37	51
3363	37	52
3364	37	53
3365	37	54
3366	37	55
3367	37	56
3368	37	57
3369	37	58
3370	37	59
3371	37	60
3372	37	61
3373	37	62
3374	37	63
3375	37	64
3376	37	65
3377	37	66
3378	37	67
3379	37	68
3380	37	69
3381	37	70
3382	37	71
3383	37	72
3384	37	73
3385	37	74
3386	37	75
3387	37	76
3388	37	77
3389	37	78
3390	37	79
3391	37	80
3392	37	81
3393	37	82
3394	37	83
3395	37	84
3396	37	85
3397	37	86
3398	37	87
3399	37	88
3400	37	89
3401	37	90
3402	37	91
3403	37	92
3404	37	93
3405	37	94
3406	37	95
3407	37	96
3408	37	97
3409	37	98
3410	37	99
3411	37	100
3412	37	101
3413	37	102
3414	37	103
3415	37	104
3416	37	105
3417	37	106
3418	37	107
3419	37	108
3420	37	109
3421	37	110
3422	37	111
3423	37	112
3424	37	113
3425	37	114
3426	37	115
3427	37	116
3428	37	117
3429	37	118
3430	37	119
3431	37	120
3432	37	121
3433	37	122
3434	37	123
3435	37	124
3436	37	125
3437	37	126
3438	37	127
3439	37	128
3440	37	129
3441	37	130
3442	38	38
3443	38	39
3444	38	40
3445	38	41
3446	38	42
3447	38	43
3448	38	44
3449	38	45
3450	38	46
3451	38	47
3452	38	48
3453	38	49
3454	38	50
3455	38	51
3456	38	52
3457	38	53
3458	38	54
3459	38	55
3460	38	56
3461	38	57
3462	38	58
3463	38	59
3464	38	60
3465	38	61
3466	38	62
3467	38	63
3468	38	64
3469	38	65
3470	38	66
3471	38	67
3472	38	68
3473	38	69
3474	38	70
3475	38	71
3476	38	72
3477	38	73
3478	38	74
3479	38	75
3480	38	76
3481	38	77
3482	38	78
3483	38	79
3484	38	80
3485	38	81
3486	38	82
3487	38	83
3488	38	84
3489	38	85
3490	38	86
3491	38	87
3492	38	88
3493	38	89
3494	38	90
3495	38	91
3496	38	92
3497	38	93
3498	38	94
3499	38	95
3500	38	96
3501	38	97
3502	38	98
3503	38	99
3504	38	100
3505	38	101
3506	38	102
3507	38	103
3508	38	104
3509	38	105
3510	38	106
3511	38	107
3512	38	108
3513	38	109
3514	38	110
3515	38	111
3516	38	112
3517	38	113
3518	38	114
3519	38	115
3520	38	116
3521	38	117
3522	38	118
3523	38	119
3524	38	120
3525	38	121
3526	38	122
3527	38	123
3528	38	124
3529	38	125
3530	38	126
3531	38	127
3532	38	128
3533	38	129
3534	38	130
3535	39	38
3536	39	39
3537	39	40
3538	39	41
3539	39	42
3540	39	43
3541	39	44
3542	39	45
3543	39	46
3544	39	47
3545	39	48
3546	39	49
3547	39	50
3548	39	51
3549	39	52
3550	39	53
3551	39	54
3552	39	55
3553	39	56
3554	39	57
3555	39	58
3556	39	59
3557	39	60
3558	39	61
3559	39	62
3560	39	63
3561	39	64
3562	39	65
3563	39	66
3564	39	67
3565	39	68
3566	39	69
3567	39	70
3568	39	71
3569	39	72
3570	39	73
3571	39	74
3572	39	75
3573	39	76
3574	39	77
3575	39	78
3576	39	79
3577	39	80
3578	39	81
3579	39	82
3580	39	83
3581	39	84
3582	39	85
3583	39	86
3584	39	87
3585	39	88
3586	39	89
3587	39	90
3588	39	91
3589	39	92
3590	39	93
3591	39	94
3592	39	95
3593	39	96
3594	39	97
3595	39	98
3596	39	99
3597	39	100
3598	39	101
3599	39	102
3600	39	103
3601	39	104
3602	39	105
3603	39	106
3604	39	107
3605	39	108
3606	39	109
3607	39	110
3608	39	111
3609	39	112
3610	39	113
3611	39	114
3612	39	115
3613	39	116
3614	39	117
3615	39	118
3616	39	119
3617	39	120
3618	39	121
3619	39	122
3620	39	123
3621	39	124
3622	39	125
3623	39	126
3624	39	127
3625	39	128
3626	39	129
3627	39	130
3628	40	38
3629	40	39
3630	40	40
3631	40	41
3632	40	42
3633	40	43
3634	40	44
3635	40	45
3636	40	46
3637	40	47
3638	40	48
3639	40	49
3640	40	50
3641	40	51
3642	40	52
3643	40	53
3644	40	54
3645	40	55
3646	40	56
3647	40	57
3648	40	58
3649	40	59
3650	40	60
3651	40	61
3652	40	62
3653	40	63
3654	40	64
3655	40	65
3656	40	66
3657	40	67
3658	40	68
3659	40	69
3660	40	70
3661	40	71
3662	40	72
3663	40	73
3664	40	74
3665	40	75
3666	40	76
3667	40	77
3668	40	78
3669	40	79
3670	40	80
3671	40	81
3672	40	82
3673	40	83
3674	40	84
3675	40	85
3676	40	86
3677	40	87
3678	40	88
3679	40	89
3680	40	90
3681	40	91
3682	40	92
3683	40	93
3684	40	94
3685	40	95
3686	40	96
3687	40	97
3688	40	98
3689	40	99
3690	40	100
3691	40	101
3692	40	102
3693	40	103
3694	40	104
3695	40	105
3696	40	106
3697	40	107
3698	40	108
3699	40	109
3700	40	110
3701	40	111
3702	40	112
3703	40	113
3704	40	114
3705	40	115
3706	40	116
3707	40	117
3708	40	118
3709	40	119
3710	40	120
3711	40	121
3712	40	122
3713	40	123
3714	40	124
3715	40	125
3716	40	126
3717	40	127
3718	40	128
3719	40	129
3720	40	130
3721	41	38
3722	41	39
3723	41	40
3724	41	41
3725	41	42
3726	41	43
3727	41	44
3728	41	45
3729	41	46
3730	41	47
3731	41	48
3732	41	49
3733	41	50
3734	41	51
3735	41	52
3736	41	53
3737	41	54
3738	41	55
3739	41	56
3740	41	57
3741	41	58
3742	41	59
3743	41	60
3744	41	61
3745	41	62
3746	41	63
3747	41	64
3748	41	65
3749	41	66
3750	41	67
3751	41	68
3752	41	69
3753	41	70
3754	41	71
3755	41	72
3756	41	73
3757	41	74
3758	41	75
3759	41	76
3760	41	77
3761	41	78
3762	41	79
3763	41	80
3764	41	81
3765	41	82
3766	41	83
3767	41	84
3768	41	85
3769	41	86
3770	41	87
3771	41	88
3772	41	89
3773	41	90
3774	41	91
3775	41	92
3776	41	93
3777	41	94
3778	41	95
3779	41	96
3780	41	97
3781	41	98
3782	41	99
3783	41	100
3784	41	101
3785	41	102
3786	41	103
3787	41	104
3788	41	105
3789	41	106
3790	41	107
3791	41	108
3792	41	109
3793	41	110
3794	41	111
3795	41	112
3796	41	113
3797	41	114
3798	41	115
3799	41	116
3800	41	117
3801	41	118
3802	41	119
3803	41	120
3804	41	121
3805	41	122
3806	41	123
3807	41	124
3808	41	125
3809	41	126
3810	41	127
3811	41	128
3812	41	129
3813	41	130
3814	42	38
3815	42	39
3816	42	40
3817	42	41
3818	42	42
3819	42	43
3820	42	44
3821	42	45
3822	42	46
3823	42	47
3824	42	48
3825	42	49
3826	42	50
3827	42	51
3828	42	52
3829	42	53
3830	42	54
3831	42	55
3832	42	56
3833	42	57
3834	42	58
3835	42	59
3836	42	60
3837	42	61
3838	42	62
3839	42	63
3840	42	64
3841	42	65
3842	42	66
3843	42	67
3844	42	68
3845	42	69
3846	42	70
3847	42	71
3848	42	72
3849	42	73
3850	42	74
3851	42	75
3852	42	76
3853	42	77
3854	42	78
3855	42	79
3856	42	80
3857	42	81
3858	42	82
3859	42	83
3860	42	84
3861	42	85
3862	42	86
3863	42	87
3864	42	88
3865	42	89
3866	42	90
3867	42	91
3868	42	92
3869	42	93
3870	42	94
3871	42	95
3872	42	96
3873	42	97
3874	42	98
3875	42	99
3876	42	100
3877	42	101
3878	42	102
3879	42	103
3880	42	104
3881	42	105
3882	42	106
3883	42	107
3884	42	108
3885	42	109
3886	42	110
3887	42	111
3888	42	112
3889	42	113
3890	42	114
3891	42	115
3892	42	116
3893	42	117
3894	42	118
3895	42	119
3896	42	120
3897	42	121
3898	42	122
3899	42	123
3900	42	124
3901	42	125
3902	42	126
3903	42	127
3904	42	128
3905	42	129
3906	42	130
3907	43	38
3908	43	39
3909	43	40
3910	43	41
3911	43	42
3912	43	43
3913	43	44
3914	43	45
3915	43	46
3916	43	47
3917	43	48
3918	43	49
3919	43	50
3920	43	51
3921	43	52
3922	43	53
3923	43	54
3924	43	55
3925	43	56
3926	43	57
3927	43	58
3928	43	59
3929	43	60
3930	43	61
3931	43	62
3932	43	63
3933	43	64
3934	43	65
3935	43	66
3936	43	67
3937	43	68
3938	43	69
3939	43	70
3940	43	71
3941	43	72
3942	43	73
3943	43	74
3944	43	75
3945	43	76
3946	43	77
3947	43	78
3948	43	79
3949	43	80
3950	43	81
3951	43	82
3952	43	83
3953	43	84
3954	43	85
3955	43	86
3956	43	87
3957	43	88
3958	43	89
3959	43	90
3960	43	91
3961	43	92
3962	43	93
3963	43	94
3964	43	95
3965	43	96
3966	43	97
3967	43	98
3968	43	99
3969	43	100
3970	43	101
3971	43	102
3972	43	103
3973	43	104
3974	43	105
3975	43	106
3976	43	107
3977	43	108
3978	43	109
3979	43	110
3980	43	111
3981	43	112
3982	43	113
3983	43	114
3984	43	115
3985	43	116
3986	43	117
3987	43	118
3988	43	119
3989	43	120
3990	43	121
3991	43	122
3992	43	123
3993	43	124
3994	43	125
3995	43	126
3996	43	127
3997	43	128
3998	43	129
3999	43	130
4000	44	38
4001	44	39
4002	44	40
4003	44	41
4004	44	42
4005	44	43
4006	44	44
4007	44	45
4008	44	46
4009	44	47
4010	44	48
4011	44	49
4012	44	50
4013	44	51
4014	44	52
4015	44	53
4016	44	54
4017	44	55
4018	44	56
4019	44	57
4020	44	58
4021	44	59
4022	44	60
4023	44	61
4024	44	62
4025	44	63
4026	44	64
4027	44	65
4028	44	66
4029	44	67
4030	44	68
4031	44	69
4032	44	70
4033	44	71
4034	44	72
4035	44	73
4036	44	74
4037	44	75
4038	44	76
4039	44	77
4040	44	78
4041	44	79
4042	44	80
4043	44	81
4044	44	82
4045	44	83
4046	44	84
4047	44	85
4048	44	86
4049	44	87
4050	44	88
4051	44	89
4052	44	90
4053	44	91
4054	44	92
4055	44	93
4056	44	94
4057	44	95
4058	44	96
4059	44	97
4060	44	98
4061	44	99
4062	44	100
4063	44	101
4064	44	102
4065	44	103
4066	44	104
4067	44	105
4068	44	106
4069	44	107
4070	44	108
4071	44	109
4072	44	110
4073	44	111
4074	44	112
4075	44	113
4076	44	114
4077	44	115
4078	44	116
4079	44	117
4080	44	118
4081	44	119
4082	44	120
4083	44	121
4084	44	122
4085	44	123
4086	44	124
4087	44	125
4088	44	126
4089	44	127
4090	44	128
4091	44	129
4092	44	130
4093	45	38
4094	45	39
4095	45	40
4096	45	41
4097	45	42
4098	45	43
4099	45	44
4100	45	45
4101	45	46
4102	45	47
4103	45	48
4104	45	49
4105	45	50
4106	45	51
4107	45	52
4108	45	53
4109	45	54
4110	45	55
4111	45	56
4112	45	57
4113	45	58
4114	45	59
4115	45	60
4116	45	61
4117	45	62
4118	45	63
4119	45	64
4120	45	65
4121	45	66
4122	45	67
4123	45	68
4124	45	69
4125	45	70
4126	45	71
4127	45	72
4128	45	73
4129	45	74
4130	45	75
4131	45	76
4132	45	77
4133	45	78
4134	45	79
4135	45	80
4136	45	81
4137	45	82
4138	45	83
4139	45	84
4140	45	85
4141	45	86
4142	45	87
4143	45	88
4144	45	89
4145	45	90
4146	45	91
4147	45	92
4148	45	93
4149	45	94
4150	45	95
4151	45	96
4152	45	97
4153	45	98
4154	45	99
4155	45	100
4156	45	101
4157	45	102
4158	45	103
4159	45	104
4160	45	105
4161	45	106
4162	45	107
4163	45	108
4164	45	109
4165	45	110
4166	45	111
4167	45	112
4168	45	113
4169	45	114
4170	45	115
4171	45	116
4172	45	117
4173	45	118
4174	45	119
4175	45	120
4176	45	121
4177	45	122
4178	45	123
4179	45	124
4180	45	125
4181	45	126
4182	45	127
4183	45	128
4184	45	129
4185	45	130
4186	46	38
4187	46	39
4188	46	40
4189	46	41
4190	46	42
4191	46	43
4192	46	44
4193	46	45
4194	46	46
4195	46	47
4196	46	48
4197	46	49
4198	46	50
4199	46	51
4200	46	52
4201	46	53
4202	46	54
4203	46	55
4204	46	56
4205	46	57
4206	46	58
4207	46	59
4208	46	60
4209	46	61
4210	46	62
4211	46	63
4212	46	64
4213	46	65
4214	46	66
4215	46	67
4216	46	68
4217	46	69
4218	46	70
4219	46	71
4220	46	72
4221	46	73
4222	46	74
4223	46	75
4224	46	76
4225	46	77
4226	46	78
4227	46	79
4228	46	80
4229	46	81
4230	46	82
4231	46	83
4232	46	84
4233	46	85
4234	46	86
4235	46	87
4236	46	88
4237	46	89
4238	46	90
4239	46	91
4240	46	92
4241	46	93
4242	46	94
4243	46	95
4244	46	96
4245	46	97
4246	46	98
4247	46	99
4248	46	100
4249	46	101
4250	46	102
4251	46	103
4252	46	104
4253	46	105
4254	46	106
4255	46	107
4256	46	108
4257	46	109
4258	46	110
4259	46	111
4260	46	112
4261	46	113
4262	46	114
4263	46	115
4264	46	116
4265	46	117
4266	46	118
4267	46	119
4268	46	120
4269	46	121
4270	46	122
4271	46	123
4272	46	124
4273	46	125
4274	46	126
4275	46	127
4276	46	128
4277	46	129
4278	46	130
4279	47	38
4280	47	39
4281	47	40
4282	47	41
4283	47	42
4284	47	43
4285	47	44
4286	47	45
4287	47	46
4288	47	47
4289	47	48
4290	47	49
4291	47	50
4292	47	51
4293	47	52
4294	47	53
4295	47	54
4296	47	55
4297	47	56
4298	47	57
4299	47	58
4300	47	59
4301	47	60
4302	47	61
4303	47	62
4304	47	63
4305	47	64
4306	47	65
4307	47	66
4308	47	67
4309	47	68
4310	47	69
4311	47	70
4312	47	71
4313	47	72
4314	47	73
4315	47	74
4316	47	75
4317	47	76
4318	47	77
4319	47	78
4320	47	79
4321	47	80
4322	47	81
4323	47	82
4324	47	83
4325	47	84
4326	47	85
4327	47	86
4328	47	87
4329	47	88
4330	47	89
4331	47	90
4332	47	91
4333	47	92
4334	47	93
4335	47	94
4336	47	95
4337	47	96
4338	47	97
4339	47	98
4340	47	99
4341	47	100
4342	47	101
4343	47	102
4344	47	103
4345	47	104
4346	47	105
4347	47	106
4348	47	107
4349	47	108
4350	47	109
4351	47	110
4352	47	111
4353	47	112
4354	47	113
4355	47	114
4356	47	115
4357	47	116
4358	47	117
4359	47	118
4360	47	119
4361	47	120
4362	47	121
4363	47	122
4364	47	123
4365	47	124
4366	47	125
4367	47	126
4368	47	127
4369	47	128
4370	47	129
4371	47	130
4372	48	38
4373	48	39
4374	48	40
4375	48	41
4376	48	42
4377	48	43
4378	48	44
4379	48	45
4380	48	46
4381	48	47
4382	48	48
4383	48	49
4384	48	50
4385	48	51
4386	48	52
4387	48	53
4388	48	54
4389	48	55
4390	48	56
4391	48	57
4392	48	58
4393	48	59
4394	48	60
4395	48	61
4396	48	62
4397	48	63
4398	48	64
4399	48	65
4400	48	66
4401	48	67
4402	48	68
4403	48	69
4404	48	70
4405	48	71
4406	48	72
4407	48	73
4408	48	74
4409	48	75
4410	48	76
4411	48	77
4412	48	78
4413	48	79
4414	48	80
4415	48	81
4416	48	82
4417	48	83
4418	48	84
4419	48	85
4420	48	86
4421	48	87
4422	48	88
4423	48	89
4424	48	90
4425	48	91
4426	48	92
4427	48	93
4428	48	94
4429	48	95
4430	48	96
4431	48	97
4432	48	98
4433	48	99
4434	48	100
4435	48	101
4436	48	102
4437	48	103
4438	48	104
4439	48	105
4440	48	106
4441	48	107
4442	48	108
4443	48	109
4444	48	110
4445	48	111
4446	48	112
4447	48	113
4448	48	114
4449	48	115
4450	48	116
4451	48	117
4452	48	118
4453	48	119
4454	48	120
4455	48	121
4456	48	122
4457	48	123
4458	48	124
4459	48	125
4460	48	126
4461	48	127
4462	48	128
4463	48	129
4464	48	130
4465	49	38
4466	49	39
4467	49	40
4468	49	41
4469	49	42
4470	49	43
4471	49	44
4472	49	45
4473	49	46
4474	49	47
4475	49	48
4476	49	49
4477	49	50
4478	49	51
4479	49	52
4480	49	53
4481	49	54
4482	49	55
4483	49	56
4484	49	57
4485	49	58
4486	49	59
4487	49	60
4488	49	61
4489	49	62
4490	49	63
4491	49	64
4492	49	65
4493	49	66
4494	49	67
4495	49	68
4496	49	69
4497	49	70
4498	49	71
4499	49	72
4500	49	73
4501	49	74
4502	49	75
4503	49	76
4504	49	77
4505	49	78
4506	49	79
4507	49	80
4508	49	81
4509	49	82
4510	49	83
4511	49	84
4512	49	85
4513	49	86
4514	49	87
4515	49	88
4516	49	89
4517	49	90
4518	49	91
4519	49	92
4520	49	93
4521	49	94
4522	49	95
4523	49	96
4524	49	97
4525	49	98
4526	49	99
4527	49	100
4528	49	101
4529	49	102
4530	49	103
4531	49	104
4532	49	105
4533	49	106
4534	49	107
4535	49	108
4536	49	109
4537	49	110
4538	49	111
4539	49	112
4540	49	113
4541	49	114
4542	49	115
4543	49	116
4544	49	117
4545	49	118
4546	49	119
4547	49	120
4548	49	121
4549	49	122
4550	49	123
4551	49	124
4552	49	125
4553	49	126
4554	49	127
4555	49	128
4556	49	129
4557	49	130
4558	50	38
4559	50	39
4560	50	40
4561	50	41
4562	50	42
4563	50	43
4564	50	44
4565	50	45
4566	50	46
4567	50	47
4568	50	48
4569	50	49
4570	50	50
4571	50	51
4572	50	52
4573	50	53
4574	50	54
4575	50	55
4576	50	56
4577	50	57
4578	50	58
4579	50	59
4580	50	60
4581	50	61
4582	50	62
4583	50	63
4584	50	64
4585	50	65
4586	50	66
4587	50	67
4588	50	68
4589	50	69
4590	50	70
4591	50	71
4592	50	72
4593	50	73
4594	50	74
4595	50	75
4596	50	76
4597	50	77
4598	50	78
4599	50	79
4600	50	80
4601	50	81
4602	50	82
4603	50	83
4604	50	84
4605	50	85
4606	50	86
4607	50	87
4608	50	88
4609	50	89
4610	50	90
4611	50	91
4612	50	92
4613	50	93
4614	50	94
4615	50	95
4616	50	96
4617	50	97
4618	50	98
4619	50	99
4620	50	100
4621	50	101
4622	50	102
4623	50	103
4624	50	104
4625	50	105
4626	50	106
4627	50	107
4628	50	108
4629	50	109
4630	50	110
4631	50	111
4632	50	112
4633	50	113
4634	50	114
4635	50	115
4636	50	116
4637	50	117
4638	50	118
4639	50	119
4640	50	120
4641	50	121
4642	50	122
4643	50	123
4644	50	124
4645	50	125
4646	50	126
4647	50	127
4648	50	128
4649	50	129
4650	50	130
4651	51	38
4652	51	39
4653	51	40
4654	51	41
4655	51	42
4656	51	43
4657	51	44
4658	51	45
4659	51	46
4660	51	47
4661	51	48
4662	51	49
4663	51	50
4664	51	51
4665	51	52
4666	51	53
4667	51	54
4668	51	55
4669	51	56
4670	51	57
4671	51	58
4672	51	59
4673	51	60
4674	51	61
4675	51	62
4676	51	63
4677	51	64
4678	51	65
4679	51	66
4680	51	67
4681	51	68
4682	51	69
4683	51	70
4684	51	71
4685	51	72
4686	51	73
4687	51	74
4688	51	75
4689	51	76
4690	51	77
4691	51	78
4692	51	79
4693	51	80
4694	51	81
4695	51	82
4696	51	83
4697	51	84
4698	51	85
4699	51	86
4700	51	87
4701	51	88
4702	51	89
4703	51	90
4704	51	91
4705	51	92
4706	51	93
4707	51	94
4708	51	95
4709	51	96
4710	51	97
4711	51	98
4712	51	99
4713	51	100
4714	51	101
4715	51	102
4716	51	103
4717	51	104
4718	51	105
4719	51	106
4720	51	107
4721	51	108
4722	51	109
4723	51	110
4724	51	111
4725	51	112
4726	51	113
4727	51	114
4728	51	115
4729	51	116
4730	51	117
4731	51	118
4732	51	119
4733	51	120
4734	51	121
4735	51	122
4736	51	123
4737	51	124
4738	51	125
4739	51	126
4740	51	127
4741	51	128
4742	51	129
4743	51	130
4744	52	38
4745	52	39
4746	52	40
4747	52	41
4748	52	42
4749	52	43
4750	52	44
4751	52	45
4752	52	46
4753	52	47
4754	52	48
4755	52	49
4756	52	50
4757	52	51
4758	52	52
4759	52	53
4760	52	54
4761	52	55
4762	52	56
4763	52	57
4764	52	58
4765	52	59
4766	52	60
4767	52	61
4768	52	62
4769	52	63
4770	52	64
4771	52	65
4772	52	66
4773	52	67
4774	52	68
4775	52	69
4776	52	70
4777	52	71
4778	52	72
4779	52	73
4780	52	74
4781	52	75
4782	52	76
4783	52	77
4784	52	78
4785	52	79
4786	52	80
4787	52	81
4788	52	82
4789	52	83
4790	52	84
4791	52	85
4792	52	86
4793	52	87
4794	52	88
4795	52	89
4796	52	90
4797	52	91
4798	52	92
4799	52	93
4800	52	94
4801	52	95
4802	52	96
4803	52	97
4804	52	98
4805	52	99
4806	52	100
4807	52	101
4808	52	102
4809	52	103
4810	52	104
4811	52	105
4812	52	106
4813	52	107
4814	52	108
4815	52	109
4816	52	110
4817	52	111
4818	52	112
4819	52	113
4820	52	114
4821	52	115
4822	52	116
4823	52	117
4824	52	118
4825	52	119
4826	52	120
4827	52	121
4828	52	122
4829	52	123
4830	52	124
4831	52	125
4832	52	126
4833	52	127
4834	52	128
4835	52	129
4836	52	130
4837	53	38
4838	53	39
4839	53	40
4840	53	41
4841	53	42
4842	53	43
4843	53	44
4844	53	45
4845	53	46
4846	53	47
4847	53	48
4848	53	49
4849	53	50
4850	53	51
4851	53	52
4852	53	53
4853	53	54
4854	53	55
4855	53	56
4856	53	57
4857	53	58
4858	53	59
4859	53	60
4860	53	61
4861	53	62
4862	53	63
4863	53	64
4864	53	65
4865	53	66
4866	53	67
4867	53	68
4868	53	69
4869	53	70
4870	53	71
4871	53	72
4872	53	73
4873	53	74
4874	53	75
4875	53	76
4876	53	77
4877	53	78
4878	53	79
4879	53	80
4880	53	81
4881	53	82
4882	53	83
4883	53	84
4884	53	85
4885	53	86
4886	53	87
4887	53	88
4888	53	89
4889	53	90
4890	53	91
4891	53	92
4892	53	93
4893	53	94
4894	53	95
4895	53	96
4896	53	97
4897	53	98
4898	53	99
4899	53	100
4900	53	101
4901	53	102
4902	53	103
4903	53	104
4904	53	105
4905	53	106
4906	53	107
4907	53	108
4908	53	109
4909	53	110
4910	53	111
4911	53	112
4912	53	113
4913	53	114
4914	53	115
4915	53	116
4916	53	117
4917	53	118
4918	53	119
4919	53	120
4920	53	121
4921	53	122
4922	53	123
4923	53	124
4924	53	125
4925	53	126
4926	53	127
4927	53	128
4928	53	129
4929	53	130
4930	54	38
4931	54	39
4932	54	40
4933	54	41
4934	54	42
4935	54	43
4936	54	44
4937	54	45
4938	54	46
4939	54	47
4940	54	48
4941	54	49
4942	54	50
4943	54	51
4944	54	52
4945	54	53
4946	54	54
4947	54	55
4948	54	56
4949	54	57
4950	54	58
4951	54	59
4952	54	60
4953	54	61
4954	54	62
4955	54	63
4956	54	64
4957	54	65
4958	54	66
4959	54	67
4960	54	68
4961	54	69
4962	54	70
4963	54	71
4964	54	72
4965	54	73
4966	54	74
4967	54	75
4968	54	76
4969	54	77
4970	54	78
4971	54	79
4972	54	80
4973	54	81
4974	54	82
4975	54	83
4976	54	84
4977	54	85
4978	54	86
4979	54	87
4980	54	88
4981	54	89
4982	54	90
4983	54	91
4984	54	92
4985	54	93
4986	54	94
4987	54	95
4988	54	96
4989	54	97
4990	54	98
4991	54	99
4992	54	100
4993	54	101
4994	54	102
4995	54	103
4996	54	104
4997	54	105
4998	54	106
4999	54	107
5000	54	108
5001	54	109
5002	54	110
5003	54	111
5004	54	112
5005	54	113
5006	54	114
5007	54	115
5008	54	116
5009	54	117
5010	54	118
5011	54	119
5012	54	120
5013	54	121
5014	54	122
5015	54	123
5016	54	124
5017	54	125
5018	54	126
5019	54	127
5020	54	128
5021	54	129
5022	54	130
5023	55	38
5024	55	39
5025	55	40
5026	55	41
5027	55	42
5028	55	43
5029	55	44
5030	55	45
5031	55	46
5032	55	47
5033	55	48
5034	55	49
5035	55	50
5036	55	51
5037	55	52
5038	55	53
5039	55	54
5040	55	55
5041	55	56
5042	55	57
5043	55	58
5044	55	59
5045	55	60
5046	55	61
5047	55	62
5048	55	63
5049	55	64
5050	55	65
5051	55	66
5052	55	67
5053	55	68
5054	55	69
5055	55	70
5056	55	71
5057	55	72
5058	55	73
5059	55	74
5060	55	75
5061	55	76
5062	55	77
5063	55	78
5064	55	79
5065	55	80
5066	55	81
5067	55	82
5068	55	83
5069	55	84
5070	55	85
5071	55	86
5072	55	87
5073	55	88
5074	55	89
5075	55	90
5076	55	91
5077	55	92
5078	55	93
5079	55	94
5080	55	95
5081	55	96
5082	55	97
5083	55	98
5084	55	99
5085	55	100
5086	55	101
5087	55	102
5088	55	103
5089	55	104
5090	55	105
5091	55	106
5092	55	107
5093	55	108
5094	55	109
5095	55	110
5096	55	111
5097	55	112
5098	55	113
5099	55	114
5100	55	115
5101	55	116
5102	55	117
5103	55	118
5104	55	119
5105	55	120
5106	55	121
5107	55	122
5108	55	123
5109	55	124
5110	55	125
5111	55	126
5112	55	127
5113	55	128
5114	55	129
5115	55	130
5116	56	38
5117	56	39
5118	56	40
5119	56	41
5120	56	42
5121	56	43
5122	56	44
5123	56	45
5124	56	46
5125	56	47
5126	56	48
5127	56	49
5128	56	50
5129	56	51
5130	56	52
5131	56	53
5132	56	54
5133	56	55
5134	56	56
5135	56	57
5136	56	58
5137	56	59
5138	56	60
5139	56	61
5140	56	62
5141	56	63
5142	56	64
5143	56	65
5144	56	66
5145	56	67
5146	56	68
5147	56	69
5148	56	70
5149	56	71
5150	56	72
5151	56	73
5152	56	74
5153	56	75
5154	56	76
5155	56	77
5156	56	78
5157	56	79
5158	56	80
5159	56	81
5160	56	82
5161	56	83
5162	56	84
5163	56	85
5164	56	86
5165	56	87
5166	56	88
5167	56	89
5168	56	90
5169	56	91
5170	56	92
5171	56	93
5172	56	94
5173	56	95
5174	56	96
5175	56	97
5176	56	98
5177	56	99
5178	56	100
5179	56	101
5180	56	102
5181	56	103
5182	56	104
5183	56	105
5184	56	106
5185	56	107
5186	56	108
5187	56	109
5188	56	110
5189	56	111
5190	56	112
5191	56	113
5192	56	114
5193	56	115
5194	56	116
5195	56	117
5196	56	118
5197	56	119
5198	56	120
5199	56	121
5200	56	122
5201	56	123
5202	56	124
5203	56	125
5204	56	126
5205	56	127
5206	56	128
5207	56	129
5208	56	130
5209	57	38
5210	57	39
5211	57	40
5212	57	41
5213	57	42
5214	57	43
5215	57	44
5216	57	45
5217	57	46
5218	57	47
5219	57	48
5220	57	49
5221	57	50
5222	57	51
5223	57	52
5224	57	53
5225	57	54
5226	57	55
5227	57	56
5228	57	57
5229	57	58
5230	57	59
5231	57	60
5232	57	61
5233	57	62
5234	57	63
5235	57	64
5236	57	65
5237	57	66
5238	57	67
5239	57	68
5240	57	69
5241	57	70
5242	57	71
5243	57	72
5244	57	73
5245	57	74
5246	57	75
5247	57	76
5248	57	77
5249	57	78
5250	57	79
5251	57	80
5252	57	81
5253	57	82
5254	57	83
5255	57	84
5256	57	85
5257	57	86
5258	57	87
5259	57	88
5260	57	89
5261	57	90
5262	57	91
5263	57	92
5264	57	93
5265	57	94
5266	57	95
5267	57	96
5268	57	97
5269	57	98
5270	57	99
5271	57	100
5272	57	101
5273	57	102
5274	57	103
5275	57	104
5276	57	105
5277	57	106
5278	57	107
5279	57	108
5280	57	109
5281	57	110
5282	57	111
5283	57	112
5284	57	113
5285	57	114
5286	57	115
5287	57	116
5288	57	117
5289	57	118
5290	57	119
5291	57	120
5292	57	121
5293	57	122
5294	57	123
5295	57	124
5296	57	125
5297	57	126
5298	57	127
5299	57	128
5300	57	129
5301	57	130
5302	58	38
5303	58	39
5304	58	40
5305	58	41
5306	58	42
5307	58	43
5308	58	44
5309	58	45
5310	58	46
5311	58	47
5312	58	48
5313	58	49
5314	58	50
5315	58	51
5316	58	52
5317	58	53
5318	58	54
5319	58	55
5320	58	56
5321	58	57
5322	58	58
5323	58	59
5324	58	60
5325	58	61
5326	58	62
5327	58	63
5328	58	64
5329	58	65
5330	58	66
5331	58	67
5332	58	68
5333	58	69
5334	58	70
5335	58	71
5336	58	72
5337	58	73
5338	58	74
5339	58	75
5340	58	76
5341	58	77
5342	58	78
5343	58	79
5344	58	80
5345	58	81
5346	58	82
5347	58	83
5348	58	84
5349	58	85
5350	58	86
5351	58	87
5352	58	88
5353	58	89
5354	58	90
5355	58	91
5356	58	92
5357	58	93
5358	58	94
5359	58	95
5360	58	96
5361	58	97
5362	58	98
5363	58	99
5364	58	100
5365	58	101
5366	58	102
5367	58	103
5368	58	104
5369	58	105
5370	58	106
5371	58	107
5372	58	108
5373	58	109
5374	58	110
5375	58	111
5376	58	112
5377	58	113
5378	58	114
5379	58	115
5380	58	116
5381	58	117
5382	58	118
5383	58	119
5384	58	120
5385	58	121
5386	58	122
5387	58	123
5388	58	124
5389	58	125
5390	58	126
5391	58	127
5392	58	128
5393	58	129
5394	58	130
5395	59	38
5396	59	39
5397	59	40
5398	59	41
5399	59	42
5400	59	43
5401	59	44
5402	59	45
5403	59	46
5404	59	47
5405	59	48
5406	59	49
5407	59	50
5408	59	51
5409	59	52
5410	59	53
5411	59	54
5412	59	55
5413	59	56
5414	59	57
5415	59	58
5416	59	59
5417	59	60
5418	59	61
5419	59	62
5420	59	63
5421	59	64
5422	59	65
5423	59	66
5424	59	67
5425	59	68
5426	59	69
5427	59	70
5428	59	71
5429	59	72
5430	59	73
5431	59	74
5432	59	75
5433	59	76
5434	59	77
5435	59	78
5436	59	79
5437	59	80
5438	59	81
5439	59	82
5440	59	83
5441	59	84
5442	59	85
5443	59	86
5444	59	87
5445	59	88
5446	59	89
5447	59	90
5448	59	91
5449	59	92
5450	59	93
5451	59	94
5452	59	95
5453	59	96
5454	59	97
5455	59	98
5456	59	99
5457	59	100
5458	59	101
5459	59	102
5460	59	103
5461	59	104
5462	59	105
5463	59	106
5464	59	107
5465	59	108
5466	59	109
5467	59	110
5468	59	111
5469	59	112
5470	59	113
5471	59	114
5472	59	115
5473	59	116
5474	59	117
5475	59	118
5476	59	119
5477	59	120
5478	59	121
5479	59	122
5480	59	123
5481	59	124
5482	59	125
5483	59	126
5484	59	127
5485	59	128
5486	59	129
5487	59	130
5488	60	38
5489	60	39
5490	60	40
5491	60	41
5492	60	42
5493	60	43
5494	60	44
5495	60	45
5496	60	46
5497	60	47
5498	60	48
5499	60	49
5500	60	50
5501	60	51
5502	60	52
5503	60	53
5504	60	54
5505	60	55
5506	60	56
5507	60	57
5508	60	58
5509	60	59
5510	60	60
5511	60	61
5512	60	62
5513	60	63
5514	60	64
5515	60	65
5516	60	66
5517	60	67
5518	60	68
5519	60	69
5520	60	70
5521	60	71
5522	60	72
5523	60	73
5524	60	74
5525	60	75
5526	60	76
5527	60	77
5528	60	78
5529	60	79
5530	60	80
5531	60	81
5532	60	82
5533	60	83
5534	60	84
5535	60	85
5536	60	86
5537	60	87
5538	60	88
5539	60	89
5540	60	90
5541	60	91
5542	60	92
5543	60	93
5544	60	94
5545	60	95
5546	60	96
5547	60	97
5548	60	98
5549	60	99
5550	60	100
5551	60	101
5552	60	102
5553	60	103
5554	60	104
5555	60	105
5556	60	106
5557	60	107
5558	60	108
5559	60	109
5560	60	110
5561	60	111
5562	60	112
5563	60	113
5564	60	114
5565	60	115
5566	60	116
5567	60	117
5568	60	118
5569	60	119
5570	60	120
5571	60	121
5572	60	122
5573	60	123
5574	60	124
5575	60	125
5576	60	126
5577	60	127
5578	60	128
5579	60	129
5580	60	130
5581	61	38
5582	61	39
5583	61	40
5584	61	41
5585	61	42
5586	61	43
5587	61	44
5588	61	45
5589	61	46
5590	61	47
5591	61	48
5592	61	49
5593	61	50
5594	61	51
5595	61	52
5596	61	53
5597	61	54
5598	61	55
5599	61	56
5600	61	57
5601	61	58
5602	61	59
5603	61	60
5604	61	61
5605	61	62
5606	61	63
5607	61	64
5608	61	65
5609	61	66
5610	61	67
5611	61	68
5612	61	69
5613	61	70
5614	61	71
5615	61	72
5616	61	73
5617	61	74
5618	61	75
5619	61	76
5620	61	77
5621	61	78
5622	61	79
5623	61	80
5624	61	81
5625	61	82
5626	61	83
5627	61	84
5628	61	85
5629	61	86
5630	61	87
5631	61	88
5632	61	89
5633	61	90
5634	61	91
5635	61	92
5636	61	93
5637	61	94
5638	61	95
5639	61	96
5640	61	97
5641	61	98
5642	61	99
5643	61	100
5644	61	101
5645	61	102
5646	61	103
5647	61	104
5648	61	105
5649	61	106
5650	61	107
5651	61	108
5652	61	109
5653	61	110
5654	61	111
5655	61	112
5656	61	113
5657	61	114
5658	61	115
5659	61	116
5660	61	117
5661	61	118
5662	61	119
5663	61	120
5664	61	121
5665	61	122
5666	61	123
5667	61	124
5668	61	125
5669	61	126
5670	61	127
5671	61	128
5672	61	129
5673	61	130
5674	62	38
5675	62	39
5676	62	40
5677	62	41
5678	62	42
5679	62	43
5680	62	44
5681	62	45
5682	62	46
5683	62	47
5684	62	48
5685	62	49
5686	62	50
5687	62	51
5688	62	52
5689	62	53
5690	62	54
5691	62	55
5692	62	56
5693	62	57
5694	62	58
5695	62	59
5696	62	60
5697	62	61
5698	62	62
5699	62	63
5700	62	64
5701	62	65
5702	62	66
5703	62	67
5704	62	68
5705	62	69
5706	62	70
5707	62	71
5708	62	72
5709	62	73
5710	62	74
5711	62	75
5712	62	76
5713	62	77
5714	62	78
5715	62	79
5716	62	80
5717	62	81
5718	62	82
5719	62	83
5720	62	84
5721	62	85
5722	62	86
5723	62	87
5724	62	88
5725	62	89
5726	62	90
5727	62	91
5728	62	92
5729	62	93
5730	62	94
5731	62	95
5732	62	96
5733	62	97
5734	62	98
5735	62	99
5736	62	100
5737	62	101
5738	62	102
5739	62	103
5740	62	104
5741	62	105
5742	62	106
5743	62	107
5744	62	108
5745	62	109
5746	62	110
5747	62	111
5748	62	112
5749	62	113
5750	62	114
5751	62	115
5752	62	116
5753	62	117
5754	62	118
5755	62	119
5756	62	120
5757	62	121
5758	62	122
5759	62	123
5760	62	124
5761	62	125
5762	62	126
5763	62	127
5764	62	128
5765	62	129
5766	62	130
5767	63	38
5768	63	39
5769	63	40
5770	63	41
5771	63	42
5772	63	43
5773	63	44
5774	63	45
5775	63	46
5776	63	47
5777	63	48
5778	63	49
5779	63	50
5780	63	51
5781	63	52
5782	63	53
5783	63	54
5784	63	55
5785	63	56
5786	63	57
5787	63	58
5788	63	59
5789	63	60
5790	63	61
5791	63	62
5792	63	63
5793	63	64
5794	63	65
5795	63	66
5796	63	67
5797	63	68
5798	63	69
5799	63	70
5800	63	71
5801	63	72
5802	63	73
5803	63	74
5804	63	75
5805	63	76
5806	63	77
5807	63	78
5808	63	79
5809	63	80
5810	63	81
5811	63	82
5812	63	83
5813	63	84
5814	63	85
5815	63	86
5816	63	87
5817	63	88
5818	63	89
5819	63	90
5820	63	91
5821	63	92
5822	63	93
5823	63	94
5824	63	95
5825	63	96
5826	63	97
5827	63	98
5828	63	99
5829	63	100
5830	63	101
5831	63	102
5832	63	103
5833	63	104
5834	63	105
5835	63	106
5836	63	107
5837	63	108
5838	63	109
5839	63	110
5840	63	111
5841	63	112
5842	63	113
5843	63	114
5844	63	115
5845	63	116
5846	63	117
5847	63	118
5848	63	119
5849	63	120
5850	63	121
5851	63	122
5852	63	123
5853	63	124
5854	63	125
5855	63	126
5856	63	127
5857	63	128
5858	63	129
5859	63	130
5860	64	38
5861	64	39
5862	64	40
5863	64	41
5864	64	42
5865	64	43
5866	64	44
5867	64	45
5868	64	46
5869	64	47
5870	64	48
5871	64	49
5872	64	50
5873	64	51
5874	64	52
5875	64	53
5876	64	54
5877	64	55
5878	64	56
5879	64	57
5880	64	58
5881	64	59
5882	64	60
5883	64	61
5884	64	62
5885	64	63
5886	64	64
5887	64	65
5888	64	66
5889	64	67
5890	64	68
5891	64	69
5892	64	70
5893	64	71
5894	64	72
5895	64	73
5896	64	74
5897	64	75
5898	64	76
5899	64	77
5900	64	78
5901	64	79
5902	64	80
5903	64	81
5904	64	82
5905	64	83
5906	64	84
5907	64	85
5908	64	86
5909	64	87
5910	64	88
5911	64	89
5912	64	90
5913	64	91
5914	64	92
5915	64	93
5916	64	94
5917	64	95
5918	64	96
5919	64	97
5920	64	98
5921	64	99
5922	64	100
5923	64	101
5924	64	102
5925	64	103
5926	64	104
5927	64	105
5928	64	106
5929	64	107
5930	64	108
5931	64	109
5932	64	110
5933	64	111
5934	64	112
5935	64	113
5936	64	114
5937	64	115
5938	64	116
5939	64	117
5940	64	118
5941	64	119
5942	64	120
5943	64	121
5944	64	122
5945	64	123
5946	64	124
5947	64	125
5948	64	126
5949	64	127
5950	64	128
5951	64	129
5952	64	130
5953	65	38
5954	65	39
5955	65	40
5956	65	41
5957	65	42
5958	65	43
5959	65	44
5960	65	45
5961	65	46
5962	65	47
5963	65	48
5964	65	49
5965	65	50
5966	65	51
5967	65	52
5968	65	53
5969	65	54
5970	65	55
5971	65	56
5972	65	57
5973	65	58
5974	65	59
5975	65	60
5976	65	61
5977	65	62
5978	65	63
5979	65	64
5980	65	65
5981	65	66
5982	65	67
5983	65	68
5984	65	69
5985	65	70
5986	65	71
5987	65	72
5988	65	73
5989	65	74
5990	65	75
5991	65	76
5992	65	77
5993	65	78
5994	65	79
5995	65	80
5996	65	81
5997	65	82
5998	65	83
5999	65	84
6000	65	85
6001	65	86
6002	65	87
6003	65	88
6004	65	89
6005	65	90
6006	65	91
6007	65	92
6008	65	93
6009	65	94
6010	65	95
6011	65	96
6012	65	97
6013	65	98
6014	65	99
6015	65	100
6016	65	101
6017	65	102
6018	65	103
6019	65	104
6020	65	105
6021	65	106
6022	65	107
6023	65	108
6024	65	109
6025	65	110
6026	65	111
6027	65	112
6028	65	113
6029	65	114
6030	65	115
6031	65	116
6032	65	117
6033	65	118
6034	65	119
6035	65	120
6036	65	121
6037	65	122
6038	65	123
6039	65	124
6040	65	125
6041	65	126
6042	65	127
6043	65	128
6044	65	129
6045	65	130
6046	66	38
6047	66	39
6048	66	40
6049	66	41
6050	66	42
6051	66	43
6052	66	44
6053	66	45
6054	66	46
6055	66	47
6056	66	48
6057	66	49
6058	66	50
6059	66	51
6060	66	52
6061	66	53
6062	66	54
6063	66	55
6064	66	56
6065	66	57
6066	66	58
6067	66	59
6068	66	60
6069	66	61
6070	66	62
6071	66	63
6072	66	64
6073	66	65
6074	66	66
6075	66	67
6076	66	68
6077	66	69
6078	66	70
6079	66	71
6080	66	72
6081	66	73
6082	66	74
6083	66	75
6084	66	76
6085	66	77
6086	66	78
6087	66	79
6088	66	80
6089	66	81
6090	66	82
6091	66	83
6092	66	84
6093	66	85
6094	66	86
6095	66	87
6096	66	88
6097	66	89
6098	66	90
6099	66	91
6100	66	92
6101	66	93
6102	66	94
6103	66	95
6104	66	96
6105	66	97
6106	66	98
6107	66	99
6108	66	100
6109	66	101
6110	66	102
6111	66	103
6112	66	104
6113	66	105
6114	66	106
6115	66	107
6116	66	108
6117	66	109
6118	66	110
6119	66	111
6120	66	112
6121	66	113
6122	66	114
6123	66	115
6124	66	116
6125	66	117
6126	66	118
6127	66	119
6128	66	120
6129	66	121
6130	66	122
6131	66	123
6132	66	124
6133	66	125
6134	66	126
6135	66	127
6136	66	128
6137	66	129
6138	66	130
6139	67	38
6140	67	39
6141	67	40
6142	67	41
6143	67	42
6144	67	43
6145	67	44
6146	67	45
6147	67	46
6148	67	47
6149	67	48
6150	67	49
6151	67	50
6152	67	51
6153	67	52
6154	67	53
6155	67	54
6156	67	55
6157	67	56
6158	67	57
6159	67	58
6160	67	59
6161	67	60
6162	67	61
6163	67	62
6164	67	63
6165	67	64
6166	67	65
6167	67	66
6168	67	67
6169	67	68
6170	67	69
6171	67	70
6172	67	71
6173	67	72
6174	67	73
6175	67	74
6176	67	75
6177	67	76
6178	67	77
6179	67	78
6180	67	79
6181	67	80
6182	67	81
6183	67	82
6184	67	83
6185	67	84
6186	67	85
6187	67	86
6188	67	87
6189	67	88
6190	67	89
6191	67	90
6192	67	91
6193	67	92
6194	67	93
6195	67	94
6196	67	95
6197	67	96
6198	67	97
6199	67	98
6200	67	99
6201	67	100
6202	67	101
6203	67	102
6204	67	103
6205	67	104
6206	67	105
6207	67	106
6208	67	107
6209	67	108
6210	67	109
6211	67	110
6212	67	111
6213	67	112
6214	67	113
6215	67	114
6216	67	115
6217	67	116
6218	67	117
6219	67	118
6220	67	119
6221	67	120
6222	67	121
6223	67	122
6224	67	123
6225	67	124
6226	67	125
6227	67	126
6228	67	127
6229	67	128
6230	67	129
6231	67	130
6232	68	38
6233	68	39
6234	68	40
6235	68	41
6236	68	42
6237	68	43
6238	68	44
6239	68	45
6240	68	46
6241	68	47
6242	68	48
6243	68	49
6244	68	50
6245	68	51
6246	68	52
6247	68	53
6248	68	54
6249	68	55
6250	68	56
6251	68	57
6252	68	58
6253	68	59
6254	68	60
6255	68	61
6256	68	62
6257	68	63
6258	68	64
6259	68	65
6260	68	66
6261	68	67
6262	68	68
6263	68	69
6264	68	70
6265	68	71
6266	68	72
6267	68	73
6268	68	74
6269	68	75
6270	68	76
6271	68	77
6272	68	78
6273	68	79
6274	68	80
6275	68	81
6276	68	82
6277	68	83
6278	68	84
6279	68	85
6280	68	86
6281	68	87
6282	68	88
6283	68	89
6284	68	90
6285	68	91
6286	68	92
6287	68	93
6288	68	94
6289	68	95
6290	68	96
6291	68	97
6292	68	98
6293	68	99
6294	68	100
6295	68	101
6296	68	102
6297	68	103
6298	68	104
6299	68	105
6300	68	106
6301	68	107
6302	68	108
6303	68	109
6304	68	110
6305	68	111
6306	68	112
6307	68	113
6308	68	114
6309	68	115
6310	68	116
6311	68	117
6312	68	118
6313	68	119
6314	68	120
6315	68	121
6316	68	122
6317	68	123
6318	68	124
6319	68	125
6320	68	126
6321	68	127
6322	68	128
6323	68	129
6324	68	130
6325	69	38
6326	69	39
6327	69	40
6328	69	41
6329	69	42
6330	69	43
6331	69	44
6332	69	45
6333	69	46
6334	69	47
6335	69	48
6336	69	49
6337	69	50
6338	69	51
6339	69	52
6340	69	53
6341	69	54
6342	69	55
6343	69	56
6344	69	57
6345	69	58
6346	69	59
6347	69	60
6348	69	61
6349	69	62
6350	69	63
6351	69	64
6352	69	65
6353	69	66
6354	69	67
6355	69	68
6356	69	69
6357	69	70
6358	69	71
6359	69	72
6360	69	73
6361	69	74
6362	69	75
6363	69	76
6364	69	77
6365	69	78
6366	69	79
6367	69	80
6368	69	81
6369	69	82
6370	69	83
6371	69	84
6372	69	85
6373	69	86
6374	69	87
6375	69	88
6376	69	89
6377	69	90
6378	69	91
6379	69	92
6380	69	93
6381	69	94
6382	69	95
6383	69	96
6384	69	97
6385	69	98
6386	69	99
6387	69	100
6388	69	101
6389	69	102
6390	69	103
6391	69	104
6392	69	105
6393	69	106
6394	69	107
6395	69	108
6396	69	109
6397	69	110
6398	69	111
6399	69	112
6400	69	113
6401	69	114
6402	69	115
6403	69	116
6404	69	117
6405	69	118
6406	69	119
6407	69	120
6408	69	121
6409	69	122
6410	69	123
6411	69	124
6412	69	125
6413	69	126
6414	69	127
6415	69	128
6416	69	129
6417	69	130
6418	70	38
6419	70	39
6420	70	40
6421	70	41
6422	70	42
6423	70	43
6424	70	44
6425	70	45
6426	70	46
6427	70	47
6428	70	48
6429	70	49
6430	70	50
6431	70	51
6432	70	52
6433	70	53
6434	70	54
6435	70	55
6436	70	56
6437	70	57
6438	70	58
6439	70	59
6440	70	60
6441	70	61
6442	70	62
6443	70	63
6444	70	64
6445	70	65
6446	70	66
6447	70	67
6448	70	68
6449	70	69
6450	70	70
6451	70	71
6452	70	72
6453	70	73
6454	70	74
6455	70	75
6456	70	76
6457	70	77
6458	70	78
6459	70	79
6460	70	80
6461	70	81
6462	70	82
6463	70	83
6464	70	84
6465	70	85
6466	70	86
6467	70	87
6468	70	88
6469	70	89
6470	70	90
6471	70	91
6472	70	92
6473	70	93
6474	70	94
6475	70	95
6476	70	96
6477	70	97
6478	70	98
6479	70	99
6480	70	100
6481	70	101
6482	70	102
6483	70	103
6484	70	104
6485	70	105
6486	70	106
6487	70	107
6488	70	108
6489	70	109
6490	70	110
6491	70	111
6492	70	112
6493	70	113
6494	70	114
6495	70	115
6496	70	116
6497	70	117
6498	70	118
6499	70	119
6500	70	120
6501	70	121
6502	70	122
6503	70	123
6504	70	124
6505	70	125
6506	70	126
6507	70	127
6508	70	128
6509	70	129
6510	70	130
6511	71	38
6512	71	39
6513	71	40
6514	71	41
6515	71	42
6516	71	43
6517	71	44
6518	71	45
6519	71	46
6520	71	47
6521	71	48
6522	71	49
6523	71	50
6524	71	51
6525	71	52
6526	71	53
6527	71	54
6528	71	55
6529	71	56
6530	71	57
6531	71	58
6532	71	59
6533	71	60
6534	71	61
6535	71	62
6536	71	63
6537	71	64
6538	71	65
6539	71	66
6540	71	67
6541	71	68
6542	71	69
6543	71	70
6544	71	71
6545	71	72
6546	71	73
6547	71	74
6548	71	75
6549	71	76
6550	71	77
6551	71	78
6552	71	79
6553	71	80
6554	71	81
6555	71	82
6556	71	83
6557	71	84
6558	71	85
6559	71	86
6560	71	87
6561	71	88
6562	71	89
6563	71	90
6564	71	91
6565	71	92
6566	71	93
6567	71	94
6568	71	95
6569	71	96
6570	71	97
6571	71	98
6572	71	99
6573	71	100
6574	71	101
6575	71	102
6576	71	103
6577	71	104
6578	71	105
6579	71	106
6580	71	107
6581	71	108
6582	71	109
6583	71	110
6584	71	111
6585	71	112
6586	71	113
6587	71	114
6588	71	115
6589	71	116
6590	71	117
6591	71	118
6592	71	119
6593	71	120
6594	71	121
6595	71	122
6596	71	123
6597	71	124
6598	71	125
6599	71	126
6600	71	127
6601	71	128
6602	71	129
6603	71	130
6604	72	38
6605	72	39
6606	72	40
6607	72	41
6608	72	42
6609	72	43
6610	72	44
6611	72	45
6612	72	46
6613	72	47
6614	72	48
6615	72	49
6616	72	50
6617	72	51
6618	72	52
6619	72	53
6620	72	54
6621	72	55
6622	72	56
6623	72	57
6624	72	58
6625	72	59
6626	72	60
6627	72	61
6628	72	62
6629	72	63
6630	72	64
6631	72	65
6632	72	66
6633	72	67
6634	72	68
6635	72	69
6636	72	70
6637	72	71
6638	72	72
6639	72	73
6640	72	74
6641	72	75
6642	72	76
6643	72	77
6644	72	78
6645	72	79
6646	72	80
6647	72	81
6648	72	82
6649	72	83
6650	72	84
6651	72	85
6652	72	86
6653	72	87
6654	72	88
6655	72	89
6656	72	90
6657	72	91
6658	72	92
6659	72	93
6660	72	94
6661	72	95
6662	72	96
6663	72	97
6664	72	98
6665	72	99
6666	72	100
6667	72	101
6668	72	102
6669	72	103
6670	72	104
6671	72	105
6672	72	106
6673	72	107
6674	72	108
6675	72	109
6676	72	110
6677	72	111
6678	72	112
6679	72	113
6680	72	114
6681	72	115
6682	72	116
6683	72	117
6684	72	118
6685	72	119
6686	72	120
6687	72	121
6688	72	122
6689	72	123
6690	72	124
6691	72	125
6692	72	126
6693	72	127
6694	72	128
6695	72	129
6696	72	130
6697	73	38
6698	73	39
6699	73	40
6700	73	41
6701	73	42
6702	73	43
6703	73	44
6704	73	45
6705	73	46
6706	73	47
6707	73	48
6708	73	49
6709	73	50
6710	73	51
6711	73	52
6712	73	53
6713	73	54
6714	73	55
6715	73	56
6716	73	57
6717	73	58
6718	73	59
6719	73	60
6720	73	61
6721	73	62
6722	73	63
6723	73	64
6724	73	65
6725	73	66
6726	73	67
6727	73	68
6728	73	69
6729	73	70
6730	73	71
6731	73	72
6732	73	73
6733	73	74
6734	73	75
6735	73	76
6736	73	77
6737	73	78
6738	73	79
6739	73	80
6740	73	81
6741	73	82
6742	73	83
6743	73	84
6744	73	85
6745	73	86
6746	73	87
6747	73	88
6748	73	89
6749	73	90
6750	73	91
6751	73	92
6752	73	93
6753	73	94
6754	73	95
6755	73	96
6756	73	97
6757	73	98
6758	73	99
6759	73	100
6760	73	101
6761	73	102
6762	73	103
6763	73	104
6764	73	105
6765	73	106
6766	73	107
6767	73	108
6768	73	109
6769	73	110
6770	73	111
6771	73	112
6772	73	113
6773	73	114
6774	73	115
6775	73	116
6776	73	117
6777	73	118
6778	73	119
6779	73	120
6780	73	121
6781	73	122
6782	73	123
6783	73	124
6784	73	125
6785	73	126
6786	73	127
6787	73	128
6788	73	129
6789	73	130
6790	74	38
6791	74	39
6792	74	40
6793	74	41
6794	74	42
6795	74	43
6796	74	44
6797	74	45
6798	74	46
6799	74	47
6800	74	48
6801	74	49
6802	74	50
6803	74	51
6804	74	52
6805	74	53
6806	74	54
6807	74	55
6808	74	56
6809	74	57
6810	74	58
6811	74	59
6812	74	60
6813	74	61
6814	74	62
6815	74	63
6816	74	64
6817	74	65
6818	74	66
6819	74	67
6820	74	68
6821	74	69
6822	74	70
6823	74	71
6824	74	72
6825	74	73
6826	74	74
6827	74	75
6828	74	76
6829	74	77
6830	74	78
6831	74	79
6832	74	80
6833	74	81
6834	74	82
6835	74	83
6836	74	84
6837	74	85
6838	74	86
6839	74	87
6840	74	88
6841	74	89
6842	74	90
6843	74	91
6844	74	92
6845	74	93
6846	74	94
6847	74	95
6848	74	96
6849	74	97
6850	74	98
6851	74	99
6852	74	100
6853	74	101
6854	74	102
6855	74	103
6856	74	104
6857	74	105
6858	74	106
6859	74	107
6860	74	108
6861	74	109
6862	74	110
6863	74	111
6864	74	112
6865	74	113
6866	74	114
6867	74	115
6868	74	116
6869	74	117
6870	74	118
6871	74	119
6872	74	120
6873	74	121
6874	74	122
6875	74	123
6876	74	124
6877	74	125
6878	74	126
6879	74	127
6880	74	128
6881	74	129
6882	74	130
6883	75	38
6884	75	39
6885	75	40
6886	75	41
6887	75	42
6888	75	43
6889	75	44
6890	75	45
6891	75	46
6892	75	47
6893	75	48
6894	75	49
6895	75	50
6896	75	51
6897	75	52
6898	75	53
6899	75	54
6900	75	55
6901	75	56
6902	75	57
6903	75	58
6904	75	59
6905	75	60
6906	75	61
6907	75	62
6908	75	63
6909	75	64
6910	75	65
6911	75	66
6912	75	67
6913	75	68
6914	75	69
6915	75	70
6916	75	71
6917	75	72
6918	75	73
6919	75	74
6920	75	75
6921	75	76
6922	75	77
6923	75	78
6924	75	79
6925	75	80
6926	75	81
6927	75	82
6928	75	83
6929	75	84
6930	75	85
6931	75	86
6932	75	87
6933	75	88
6934	75	89
6935	75	90
6936	75	91
6937	75	92
6938	75	93
6939	75	94
6940	75	95
6941	75	96
6942	75	97
6943	75	98
6944	75	99
6945	75	100
6946	75	101
6947	75	102
6948	75	103
6949	75	104
6950	75	105
6951	75	106
6952	75	107
6953	75	108
6954	75	109
6955	75	110
6956	75	111
6957	75	112
6958	75	113
6959	75	114
6960	75	115
6961	75	116
6962	75	117
6963	75	118
6964	75	119
6965	75	120
6966	75	121
6967	75	122
6968	75	123
6969	75	124
6970	75	125
6971	75	126
6972	75	127
6973	75	128
6974	75	129
6975	75	130
6976	76	38
6977	76	39
6978	76	40
6979	76	41
6980	76	42
6981	76	43
6982	76	44
6983	76	45
6984	76	46
6985	76	47
6986	76	48
6987	76	49
6988	76	50
6989	76	51
6990	76	52
6991	76	53
6992	76	54
6993	76	55
6994	76	56
6995	76	57
6996	76	58
6997	76	59
6998	76	60
6999	76	61
7000	76	62
7001	76	63
7002	76	64
7003	76	65
7004	76	66
7005	76	67
7006	76	68
7007	76	69
7008	76	70
7009	76	71
7010	76	72
7011	76	73
7012	76	74
7013	76	75
7014	76	76
7015	76	77
7016	76	78
7017	76	79
7018	76	80
7019	76	81
7020	76	82
7021	76	83
7022	76	84
7023	76	85
7024	76	86
7025	76	87
7026	76	88
7027	76	89
7028	76	90
7029	76	91
7030	76	92
7031	76	93
7032	76	94
7033	76	95
7034	76	96
7035	76	97
7036	76	98
7037	76	99
7038	76	100
7039	76	101
7040	76	102
7041	76	103
7042	76	104
7043	76	105
7044	76	106
7045	76	107
7046	76	108
7047	76	109
7048	76	110
7049	76	111
7050	76	112
7051	76	113
7052	76	114
7053	76	115
7054	76	116
7055	76	117
7056	76	118
7057	76	119
7058	76	120
7059	76	121
7060	76	122
7061	76	123
7062	76	124
7063	76	125
7064	76	126
7065	76	127
7066	76	128
7067	76	129
7068	76	130
7069	77	38
7070	77	39
7071	77	40
7072	77	41
7073	77	42
7074	77	43
7075	77	44
7076	77	45
7077	77	46
7078	77	47
7079	77	48
7080	77	49
7081	77	50
7082	77	51
7083	77	52
7084	77	53
7085	77	54
7086	77	55
7087	77	56
7088	77	57
7089	77	58
7090	77	59
7091	77	60
7092	77	61
7093	77	62
7094	77	63
7095	77	64
7096	77	65
7097	77	66
7098	77	67
7099	77	68
7100	77	69
7101	77	70
7102	77	71
7103	77	72
7104	77	73
7105	77	74
7106	77	75
7107	77	76
7108	77	77
7109	77	78
7110	77	79
7111	77	80
7112	77	81
7113	77	82
7114	77	83
7115	77	84
7116	77	85
7117	77	86
7118	77	87
7119	77	88
7120	77	89
7121	77	90
7122	77	91
7123	77	92
7124	77	93
7125	77	94
7126	77	95
7127	77	96
7128	77	97
7129	77	98
7130	77	99
7131	77	100
7132	77	101
7133	77	102
7134	77	103
7135	77	104
7136	77	105
7137	77	106
7138	77	107
7139	77	108
7140	77	109
7141	77	110
7142	77	111
7143	77	112
7144	77	113
7145	77	114
7146	77	115
7147	77	116
7148	77	117
7149	77	118
7150	77	119
7151	77	120
7152	77	121
7153	77	122
7154	77	123
7155	77	124
7156	77	125
7157	77	126
7158	77	127
7159	77	128
7160	77	129
7161	77	130
7162	78	38
7163	78	39
7164	78	40
7165	78	41
7166	78	42
7167	78	43
7168	78	44
7169	78	45
7170	78	46
7171	78	47
7172	78	48
7173	78	49
7174	78	50
7175	78	51
7176	78	52
7177	78	53
7178	78	54
7179	78	55
7180	78	56
7181	78	57
7182	78	58
7183	78	59
7184	78	60
7185	78	61
7186	78	62
7187	78	63
7188	78	64
7189	78	65
7190	78	66
7191	78	67
7192	78	68
7193	78	69
7194	78	70
7195	78	71
7196	78	72
7197	78	73
7198	78	74
7199	78	75
7200	78	76
7201	78	77
7202	78	78
7203	78	79
7204	78	80
7205	78	81
7206	78	82
7207	78	83
7208	78	84
7209	78	85
7210	78	86
7211	78	87
7212	78	88
7213	78	89
7214	78	90
7215	78	91
7216	78	92
7217	78	93
7218	78	94
7219	78	95
7220	78	96
7221	78	97
7222	78	98
7223	78	99
7224	78	100
7225	78	101
7226	78	102
7227	78	103
7228	78	104
7229	78	105
7230	78	106
7231	78	107
7232	78	108
7233	78	109
7234	78	110
7235	78	111
7236	78	112
7237	78	113
7238	78	114
7239	78	115
7240	78	116
7241	78	117
7242	78	118
7243	78	119
7244	78	120
7245	78	121
7246	78	122
7247	78	123
7248	78	124
7249	78	125
7250	78	126
7251	78	127
7252	78	128
7253	78	129
7254	78	130
7255	79	38
7256	79	39
7257	79	40
7258	79	41
7259	79	42
7260	79	43
7261	79	44
7262	79	45
7263	79	46
7264	79	47
7265	79	48
7266	79	49
7267	79	50
7268	79	51
7269	79	52
7270	79	53
7271	79	54
7272	79	55
7273	79	56
7274	79	57
7275	79	58
7276	79	59
7277	79	60
7278	79	61
7279	79	62
7280	79	63
7281	79	64
7282	79	65
7283	79	66
7284	79	67
7285	79	68
7286	79	69
7287	79	70
7288	79	71
7289	79	72
7290	79	73
7291	79	74
7292	79	75
7293	79	76
7294	79	77
7295	79	78
7296	79	79
7297	79	80
7298	79	81
7299	79	82
7300	79	83
7301	79	84
7302	79	85
7303	79	86
7304	79	87
7305	79	88
7306	79	89
7307	79	90
7308	79	91
7309	79	92
7310	79	93
7311	79	94
7312	79	95
7313	79	96
7314	79	97
7315	79	98
7316	79	99
7317	79	100
7318	79	101
7319	79	102
7320	79	103
7321	79	104
7322	79	105
7323	79	106
7324	79	107
7325	79	108
7326	79	109
7327	79	110
7328	79	111
7329	79	112
7330	79	113
7331	79	114
7332	79	115
7333	79	116
7334	79	117
7335	79	118
7336	79	119
7337	79	120
7338	79	121
7339	79	122
7340	79	123
7341	79	124
7342	79	125
7343	79	126
7344	79	127
7345	79	128
7346	79	129
7347	79	130
7348	80	38
7349	80	39
7350	80	40
7351	80	41
7352	80	42
7353	80	43
7354	80	44
7355	80	45
7356	80	46
7357	80	47
7358	80	48
7359	80	49
7360	80	50
7361	80	51
7362	80	52
7363	80	53
7364	80	54
7365	80	55
7366	80	56
7367	80	57
7368	80	58
7369	80	59
7370	80	60
7371	80	61
7372	80	62
7373	80	63
7374	80	64
7375	80	65
7376	80	66
7377	80	67
7378	80	68
7379	80	69
7380	80	70
7381	80	71
7382	80	72
7383	80	73
7384	80	74
7385	80	75
7386	80	76
7387	80	77
7388	80	78
7389	80	79
7390	80	80
7391	80	81
7392	80	82
7393	80	83
7394	80	84
7395	80	85
7396	80	86
7397	80	87
7398	80	88
7399	80	89
7400	80	90
7401	80	91
7402	80	92
7403	80	93
7404	80	94
7405	80	95
7406	80	96
7407	80	97
7408	80	98
7409	80	99
7410	80	100
7411	80	101
7412	80	102
7413	80	103
7414	80	104
7415	80	105
7416	80	106
7417	80	107
7418	80	108
7419	80	109
7420	80	110
7421	80	111
7422	80	112
7423	80	113
7424	80	114
7425	80	115
7426	80	116
7427	80	117
7428	80	118
7429	80	119
7430	80	120
7431	80	121
7432	80	122
7433	80	123
7434	80	124
7435	80	125
7436	80	126
7437	80	127
7438	80	128
7439	80	129
7440	80	130
7441	81	38
7442	81	39
7443	81	40
7444	81	41
7445	81	42
7446	81	43
7447	81	44
7448	81	45
7449	81	46
7450	81	47
7451	81	48
7452	81	49
7453	81	50
7454	81	51
7455	81	52
7456	81	53
7457	81	54
7458	81	55
7459	81	56
7460	81	57
7461	81	58
7462	81	59
7463	81	60
7464	81	61
7465	81	62
7466	81	63
7467	81	64
7468	81	65
7469	81	66
7470	81	67
7471	81	68
7472	81	69
7473	81	70
7474	81	71
7475	81	72
7476	81	73
7477	81	74
7478	81	75
7479	81	76
7480	81	77
7481	81	78
7482	81	79
7483	81	80
7484	81	81
7485	81	82
7486	81	83
7487	81	84
7488	81	85
7489	81	86
7490	81	87
7491	81	88
7492	81	89
7493	81	90
7494	81	91
7495	81	92
7496	81	93
7497	81	94
7498	81	95
7499	81	96
7500	81	97
7501	81	98
7502	81	99
7503	81	100
7504	81	101
7505	81	102
7506	81	103
7507	81	104
7508	81	105
7509	81	106
7510	81	107
7511	81	108
7512	81	109
7513	81	110
7514	81	111
7515	81	112
7516	81	113
7517	81	114
7518	81	115
7519	81	116
7520	81	117
7521	81	118
7522	81	119
7523	81	120
7524	81	121
7525	81	122
7526	81	123
7527	81	124
7528	81	125
7529	81	126
7530	81	127
7531	81	128
7532	81	129
7533	81	130
7534	82	38
7535	82	39
7536	82	40
7537	82	41
7538	82	42
7539	82	43
7540	82	44
7541	82	45
7542	82	46
7543	82	47
7544	82	48
7545	82	49
7546	82	50
7547	82	51
7548	82	52
7549	82	53
7550	82	54
7551	82	55
7552	82	56
7553	82	57
7554	82	58
7555	82	59
7556	82	60
7557	82	61
7558	82	62
7559	82	63
7560	82	64
7561	82	65
7562	82	66
7563	82	67
7564	82	68
7565	82	69
7566	82	70
7567	82	71
7568	82	72
7569	82	73
7570	82	74
7571	82	75
7572	82	76
7573	82	77
7574	82	78
7575	82	79
7576	82	80
7577	82	81
7578	82	82
7579	82	83
7580	82	84
7581	82	85
7582	82	86
7583	82	87
7584	82	88
7585	82	89
7586	82	90
7587	82	91
7588	82	92
7589	82	93
7590	82	94
7591	82	95
7592	82	96
7593	82	97
7594	82	98
7595	82	99
7596	82	100
7597	82	101
7598	82	102
7599	82	103
7600	82	104
7601	82	105
7602	82	106
7603	82	107
7604	82	108
7605	82	109
7606	82	110
7607	82	111
7608	82	112
7609	82	113
7610	82	114
7611	82	115
7612	82	116
7613	82	117
7614	82	118
7615	82	119
7616	82	120
7617	82	121
7618	82	122
7619	82	123
7620	82	124
7621	82	125
7622	82	126
7623	82	127
7624	82	128
7625	82	129
7626	82	130
7627	83	38
7628	83	39
7629	83	40
7630	83	41
7631	83	42
7632	83	43
7633	83	44
7634	83	45
7635	83	46
7636	83	47
7637	83	48
7638	83	49
7639	83	50
7640	83	51
7641	83	52
7642	83	53
7643	83	54
7644	83	55
7645	83	56
7646	83	57
7647	83	58
7648	83	59
7649	83	60
7650	83	61
7651	83	62
7652	83	63
7653	83	64
7654	83	65
7655	83	66
7656	83	67
7657	83	68
7658	83	69
7659	83	70
7660	83	71
7661	83	72
7662	83	73
7663	83	74
7664	83	75
7665	83	76
7666	83	77
7667	83	78
7668	83	79
7669	83	80
7670	83	81
7671	83	82
7672	83	83
7673	83	84
7674	83	85
7675	83	86
7676	83	87
7677	83	88
7678	83	89
7679	83	90
7680	83	91
7681	83	92
7682	83	93
7683	83	94
7684	83	95
7685	83	96
7686	83	97
7687	83	98
7688	83	99
7689	83	100
7690	83	101
7691	83	102
7692	83	103
7693	83	104
7694	83	105
7695	83	106
7696	83	107
7697	83	108
7698	83	109
7699	83	110
7700	83	111
7701	83	112
7702	83	113
7703	83	114
7704	83	115
7705	83	116
7706	83	117
7707	83	118
7708	83	119
7709	83	120
7710	83	121
7711	83	122
7712	83	123
7713	83	124
7714	83	125
7715	83	126
7716	83	127
7717	83	128
7718	83	129
7719	83	130
7720	84	38
7721	84	39
7722	84	40
7723	84	41
7724	84	42
7725	84	43
7726	84	44
7727	84	45
7728	84	46
7729	84	47
7730	84	48
7731	84	49
7732	84	50
7733	84	51
7734	84	52
7735	84	53
7736	84	54
7737	84	55
7738	84	56
7739	84	57
7740	84	58
7741	84	59
7742	84	60
7743	84	61
7744	84	62
7745	84	63
7746	84	64
7747	84	65
7748	84	66
7749	84	67
7750	84	68
7751	84	69
7752	84	70
7753	84	71
7754	84	72
7755	84	73
7756	84	74
7757	84	75
7758	84	76
7759	84	77
7760	84	78
7761	84	79
7762	84	80
7763	84	81
7764	84	82
7765	84	83
7766	84	84
7767	84	85
7768	84	86
7769	84	87
7770	84	88
7771	84	89
7772	84	90
7773	84	91
7774	84	92
7775	84	93
7776	84	94
7777	84	95
7778	84	96
7779	84	97
7780	84	98
7781	84	99
7782	84	100
7783	84	101
7784	84	102
7785	84	103
7786	84	104
7787	84	105
7788	84	106
7789	84	107
7790	84	108
7791	84	109
7792	84	110
7793	84	111
7794	84	112
7795	84	113
7796	84	114
7797	84	115
7798	84	116
7799	84	117
7800	84	118
7801	84	119
7802	84	120
7803	84	121
7804	84	122
7805	84	123
7806	84	124
7807	84	125
7808	84	126
7809	84	127
7810	84	128
7811	84	129
7812	84	130
7813	85	38
7814	85	39
7815	85	40
7816	85	41
7817	85	42
7818	85	43
7819	85	44
7820	85	45
7821	85	46
7822	85	47
7823	85	48
7824	85	49
7825	85	50
7826	85	51
7827	85	52
7828	85	53
7829	85	54
7830	85	55
7831	85	56
7832	85	57
7833	85	58
7834	85	59
7835	85	60
7836	85	61
7837	85	62
7838	85	63
7839	85	64
7840	85	65
7841	85	66
7842	85	67
7843	85	68
7844	85	69
7845	85	70
7846	85	71
7847	85	72
7848	85	73
7849	85	74
7850	85	75
7851	85	76
7852	85	77
7853	85	78
7854	85	79
7855	85	80
7856	85	81
7857	85	82
7858	85	83
7859	85	84
7860	85	85
7861	85	86
7862	85	87
7863	85	88
7864	85	89
7865	85	90
7866	85	91
7867	85	92
7868	85	93
7869	85	94
7870	85	95
7871	85	96
7872	85	97
7873	85	98
7874	85	99
7875	85	100
7876	85	101
7877	85	102
7878	85	103
7879	85	104
7880	85	105
7881	85	106
7882	85	107
7883	85	108
7884	85	109
7885	85	110
7886	85	111
7887	85	112
7888	85	113
7889	85	114
7890	85	115
7891	85	116
7892	85	117
7893	85	118
7894	85	119
7895	85	120
7896	85	121
7897	85	122
7898	85	123
7899	85	124
7900	85	125
7901	85	126
7902	85	127
7903	85	128
7904	85	129
7905	85	130
7906	86	38
7907	86	39
7908	86	40
7909	86	41
7910	86	42
7911	86	43
7912	86	44
7913	86	45
7914	86	46
7915	86	47
7916	86	48
7917	86	49
7918	86	50
7919	86	51
7920	86	52
7921	86	53
7922	86	54
7923	86	55
7924	86	56
7925	86	57
7926	86	58
7927	86	59
7928	86	60
7929	86	61
7930	86	62
7931	86	63
7932	86	64
7933	86	65
7934	86	66
7935	86	67
7936	86	68
7937	86	69
7938	86	70
7939	86	71
7940	86	72
7941	86	73
7942	86	74
7943	86	75
7944	86	76
7945	86	77
7946	86	78
7947	86	79
7948	86	80
7949	86	81
7950	86	82
7951	86	83
7952	86	84
7953	86	85
7954	86	86
7955	86	87
7956	86	88
7957	86	89
7958	86	90
7959	86	91
7960	86	92
7961	86	93
7962	86	94
7963	86	95
7964	86	96
7965	86	97
7966	86	98
7967	86	99
7968	86	100
7969	86	101
7970	86	102
7971	86	103
7972	86	104
7973	86	105
7974	86	106
7975	86	107
7976	86	108
7977	86	109
7978	86	110
7979	86	111
7980	86	112
7981	86	113
7982	86	114
7983	86	115
7984	86	116
7985	86	117
7986	86	118
7987	86	119
7988	86	120
7989	86	121
7990	86	122
7991	86	123
7992	86	124
7993	86	125
7994	86	126
7995	86	127
7996	86	128
7997	86	129
7998	86	130
7999	87	38
8000	87	39
8001	87	40
8002	87	41
8003	87	42
8004	87	43
8005	87	44
8006	87	45
8007	87	46
8008	87	47
8009	87	48
8010	87	49
8011	87	50
8012	87	51
8013	87	52
8014	87	53
8015	87	54
8016	87	55
8017	87	56
8018	87	57
8019	87	58
8020	87	59
8021	87	60
8022	87	61
8023	87	62
8024	87	63
8025	87	64
8026	87	65
8027	87	66
8028	87	67
8029	87	68
8030	87	69
8031	87	70
8032	87	71
8033	87	72
8034	87	73
8035	87	74
8036	87	75
8037	87	76
8038	87	77
8039	87	78
8040	87	79
8041	87	80
8042	87	81
8043	87	82
8044	87	83
8045	87	84
8046	87	85
8047	87	86
8048	87	87
8049	87	88
8050	87	89
8051	87	90
8052	87	91
8053	87	92
8054	87	93
8055	87	94
8056	87	95
8057	87	96
8058	87	97
8059	87	98
8060	87	99
8061	87	100
8062	87	101
8063	87	102
8064	87	103
8065	87	104
8066	87	105
8067	87	106
8068	87	107
8069	87	108
8070	87	109
8071	87	110
8072	87	111
8073	87	112
8074	87	113
8075	87	114
8076	87	115
8077	87	116
8078	87	117
8079	87	118
8080	87	119
8081	87	120
8082	87	121
8083	87	122
8084	87	123
8085	87	124
8086	87	125
8087	87	126
8088	87	127
8089	87	128
8090	87	129
8091	87	130
8092	88	38
8093	88	39
8094	88	40
8095	88	41
8096	88	42
8097	88	43
8098	88	44
8099	88	45
8100	88	46
8101	88	47
8102	88	48
8103	88	49
8104	88	50
8105	88	51
8106	88	52
8107	88	53
8108	88	54
8109	88	55
8110	88	56
8111	88	57
8112	88	58
8113	88	59
8114	88	60
8115	88	61
8116	88	62
8117	88	63
8118	88	64
8119	88	65
8120	88	66
8121	88	67
8122	88	68
8123	88	69
8124	88	70
8125	88	71
8126	88	72
8127	88	73
8128	88	74
8129	88	75
8130	88	76
8131	88	77
8132	88	78
8133	88	79
8134	88	80
8135	88	81
8136	88	82
8137	88	83
8138	88	84
8139	88	85
8140	88	86
8141	88	87
8142	88	88
8143	88	89
8144	88	90
8145	88	91
8146	88	92
8147	88	93
8148	88	94
8149	88	95
8150	88	96
8151	88	97
8152	88	98
8153	88	99
8154	88	100
8155	88	101
8156	88	102
8157	88	103
8158	88	104
8159	88	105
8160	88	106
8161	88	107
8162	88	108
8163	88	109
8164	88	110
8165	88	111
8166	88	112
8167	88	113
8168	88	114
8169	88	115
8170	88	116
8171	88	117
8172	88	118
8173	88	119
8174	88	120
8175	88	121
8176	88	122
8177	88	123
8178	88	124
8179	88	125
8180	88	126
8181	88	127
8182	88	128
8183	88	129
8184	88	130
8185	89	38
8186	89	39
8187	89	40
8188	89	41
8189	89	42
8190	89	43
8191	89	44
8192	89	45
8193	89	46
8194	89	47
8195	89	48
8196	89	49
8197	89	50
8198	89	51
8199	89	52
8200	89	53
8201	89	54
8202	89	55
8203	89	56
8204	89	57
8205	89	58
8206	89	59
8207	89	60
8208	89	61
8209	89	62
8210	89	63
8211	89	64
8212	89	65
8213	89	66
8214	89	67
8215	89	68
8216	89	69
8217	89	70
8218	89	71
8219	89	72
8220	89	73
8221	89	74
8222	89	75
8223	89	76
8224	89	77
8225	89	78
8226	89	79
8227	89	80
8228	89	81
8229	89	82
8230	89	83
8231	89	84
8232	89	85
8233	89	86
8234	89	87
8235	89	88
8236	89	89
8237	89	90
8238	89	91
8239	89	92
8240	89	93
8241	89	94
8242	89	95
8243	89	96
8244	89	97
8245	89	98
8246	89	99
8247	89	100
8248	89	101
8249	89	102
8250	89	103
8251	89	104
8252	89	105
8253	89	106
8254	89	107
8255	89	108
8256	89	109
8257	89	110
8258	89	111
8259	89	112
8260	89	113
8261	89	114
8262	89	115
8263	89	116
8264	89	117
8265	89	118
8266	89	119
8267	89	120
8268	89	121
8269	89	122
8270	89	123
8271	89	124
8272	89	125
8273	89	126
8274	89	127
8275	89	128
8276	89	129
8277	89	130
8278	90	38
8279	90	39
8280	90	40
8281	90	41
8282	90	42
8283	90	43
8284	90	44
8285	90	45
8286	90	46
8287	90	47
8288	90	48
8289	90	49
8290	90	50
8291	90	51
8292	90	52
8293	90	53
8294	90	54
8295	90	55
8296	90	56
8297	90	57
8298	90	58
8299	90	59
8300	90	60
8301	90	61
8302	90	62
8303	90	63
8304	90	64
8305	90	65
8306	90	66
8307	90	67
8308	90	68
8309	90	69
8310	90	70
8311	90	71
8312	90	72
8313	90	73
8314	90	74
8315	90	75
8316	90	76
8317	90	77
8318	90	78
8319	90	79
8320	90	80
8321	90	81
8322	90	82
8323	90	83
8324	90	84
8325	90	85
8326	90	86
8327	90	87
8328	90	88
8329	90	89
8330	90	90
8331	90	91
8332	90	92
8333	90	93
8334	90	94
8335	90	95
8336	90	96
8337	90	97
8338	90	98
8339	90	99
8340	90	100
8341	90	101
8342	90	102
8343	90	103
8344	90	104
8345	90	105
8346	90	106
8347	90	107
8348	90	108
8349	90	109
8350	90	110
8351	90	111
8352	90	112
8353	90	113
8354	90	114
8355	90	115
8356	90	116
8357	90	117
8358	90	118
8359	90	119
8360	90	120
8361	90	121
8362	90	122
8363	90	123
8364	90	124
8365	90	125
8366	90	126
8367	90	127
8368	90	128
8369	90	129
8370	90	130
8371	91	38
8372	91	39
8373	91	40
8374	91	41
8375	91	42
8376	91	43
8377	91	44
8378	91	45
8379	91	46
8380	91	47
8381	91	48
8382	91	49
8383	91	50
8384	91	51
8385	91	52
8386	91	53
8387	91	54
8388	91	55
8389	91	56
8390	91	57
8391	91	58
8392	91	59
8393	91	60
8394	91	61
8395	91	62
8396	91	63
8397	91	64
8398	91	65
8399	91	66
8400	91	67
8401	91	68
8402	91	69
8403	91	70
8404	91	71
8405	91	72
8406	91	73
8407	91	74
8408	91	75
8409	91	76
8410	91	77
8411	91	78
8412	91	79
8413	91	80
8414	91	81
8415	91	82
8416	91	83
8417	91	84
8418	91	85
8419	91	86
8420	91	87
8421	91	88
8422	91	89
8423	91	90
8424	91	91
8425	91	92
8426	91	93
8427	91	94
8428	91	95
8429	91	96
8430	91	97
8431	91	98
8432	91	99
8433	91	100
8434	91	101
8435	91	102
8436	91	103
8437	91	104
8438	91	105
8439	91	106
8440	91	107
8441	91	108
8442	91	109
8443	91	110
8444	91	111
8445	91	112
8446	91	113
8447	91	114
8448	91	115
8449	91	116
8450	91	117
8451	91	118
8452	91	119
8453	91	120
8454	91	121
8455	91	122
8456	91	123
8457	91	124
8458	91	125
8459	91	126
8460	91	127
8461	91	128
8462	91	129
8463	91	130
8464	92	38
8465	92	39
8466	92	40
8467	92	41
8468	92	42
8469	92	43
8470	92	44
8471	92	45
8472	92	46
8473	92	47
8474	92	48
8475	92	49
8476	92	50
8477	92	51
8478	92	52
8479	92	53
8480	92	54
8481	92	55
8482	92	56
8483	92	57
8484	92	58
8485	92	59
8486	92	60
8487	92	61
8488	92	62
8489	92	63
8490	92	64
8491	92	65
8492	92	66
8493	92	67
8494	92	68
8495	92	69
8496	92	70
8497	92	71
8498	92	72
8499	92	73
8500	92	74
8501	92	75
8502	92	76
8503	92	77
8504	92	78
8505	92	79
8506	92	80
8507	92	81
8508	92	82
8509	92	83
8510	92	84
8511	92	85
8512	92	86
8513	92	87
8514	92	88
8515	92	89
8516	92	90
8517	92	91
8518	92	92
8519	92	93
8520	92	94
8521	92	95
8522	92	96
8523	92	97
8524	92	98
8525	92	99
8526	92	100
8527	92	101
8528	92	102
8529	92	103
8530	92	104
8531	92	105
8532	92	106
8533	92	107
8534	92	108
8535	92	109
8536	92	110
8537	92	111
8538	92	112
8539	92	113
8540	92	114
8541	92	115
8542	92	116
8543	92	117
8544	92	118
8545	92	119
8546	92	120
8547	92	121
8548	92	122
8549	92	123
8550	92	124
8551	92	125
8552	92	126
8553	92	127
8554	92	128
8555	92	129
8556	92	130
8557	93	38
8558	93	39
8559	93	40
8560	93	41
8561	93	42
8562	93	43
8563	93	44
8564	93	45
8565	93	46
8566	93	47
8567	93	48
8568	93	49
8569	93	50
8570	93	51
8571	93	52
8572	93	53
8573	93	54
8574	93	55
8575	93	56
8576	93	57
8577	93	58
8578	93	59
8579	93	60
8580	93	61
8581	93	62
8582	93	63
8583	93	64
8584	93	65
8585	93	66
8586	93	67
8587	93	68
8588	93	69
8589	93	70
8590	93	71
8591	93	72
8592	93	73
8593	93	74
8594	93	75
8595	93	76
8596	93	77
8597	93	78
8598	93	79
8599	93	80
8600	93	81
8601	93	82
8602	93	83
8603	93	84
8604	93	85
8605	93	86
8606	93	87
8607	93	88
8608	93	89
8609	93	90
8610	93	91
8611	93	92
8612	93	93
8613	93	94
8614	93	95
8615	93	96
8616	93	97
8617	93	98
8618	93	99
8619	93	100
8620	93	101
8621	93	102
8622	93	103
8623	93	104
8624	93	105
8625	93	106
8626	93	107
8627	93	108
8628	93	109
8629	93	110
8630	93	111
8631	93	112
8632	93	113
8633	93	114
8634	93	115
8635	93	116
8636	93	117
8637	93	118
8638	93	119
8639	93	120
8640	93	121
8641	93	122
8642	93	123
8643	93	124
8644	93	125
8645	93	126
8646	93	127
8647	93	128
8648	93	129
8649	93	130
8650	94	38
8651	94	39
8652	94	40
8653	94	41
8654	94	42
8655	94	43
8656	94	44
8657	94	45
8658	94	46
8659	94	47
8660	94	48
8661	94	49
8662	94	50
8663	94	51
8664	94	52
8665	94	53
8666	94	54
8667	94	55
8668	94	56
8669	94	57
8670	94	58
8671	94	59
8672	94	60
8673	94	61
8674	94	62
8675	94	63
8676	94	64
8677	94	65
8678	94	66
8679	94	67
8680	94	68
8681	94	69
8682	94	70
8683	94	71
8684	94	72
8685	94	73
8686	94	74
8687	94	75
8688	94	76
8689	94	77
8690	94	78
8691	94	79
8692	94	80
8693	94	81
8694	94	82
8695	94	83
8696	94	84
8697	94	85
8698	94	86
8699	94	87
8700	94	88
8701	94	89
8702	94	90
8703	94	91
8704	94	92
8705	94	93
8706	94	94
8707	94	95
8708	94	96
8709	94	97
8710	94	98
8711	94	99
8712	94	100
8713	94	101
8714	94	102
8715	94	103
8716	94	104
8717	94	105
8718	94	106
8719	94	107
8720	94	108
8721	94	109
8722	94	110
8723	94	111
8724	94	112
8725	94	113
8726	94	114
8727	94	115
8728	94	116
8729	94	117
8730	94	118
8731	94	119
8732	94	120
8733	94	121
8734	94	122
8735	94	123
8736	94	124
8737	94	125
8738	94	126
8739	94	127
8740	94	128
8741	94	129
8742	94	130
8743	95	38
8744	95	39
8745	95	40
8746	95	41
8747	95	42
8748	95	43
8749	95	44
8750	95	45
8751	95	46
8752	95	47
8753	95	48
8754	95	49
8755	95	50
8756	95	51
8757	95	52
8758	95	53
8759	95	54
8760	95	55
8761	95	56
8762	95	57
8763	95	58
8764	95	59
8765	95	60
8766	95	61
8767	95	62
8768	95	63
8769	95	64
8770	95	65
8771	95	66
8772	95	67
8773	95	68
8774	95	69
8775	95	70
8776	95	71
8777	95	72
8778	95	73
8779	95	74
8780	95	75
8781	95	76
8782	95	77
8783	95	78
8784	95	79
8785	95	80
8786	95	81
8787	95	82
8788	95	83
8789	95	84
8790	95	85
8791	95	86
8792	95	87
8793	95	88
8794	95	89
8795	95	90
8796	95	91
8797	95	92
8798	95	93
8799	95	94
8800	95	95
8801	95	96
8802	95	97
8803	95	98
8804	95	99
8805	95	100
8806	95	101
8807	95	102
8808	95	103
8809	95	104
8810	95	105
8811	95	106
8812	95	107
8813	95	108
8814	95	109
8815	95	110
8816	95	111
8817	95	112
8818	95	113
8819	95	114
8820	95	115
8821	95	116
8822	95	117
8823	95	118
8824	95	119
8825	95	120
8826	95	121
8827	95	122
8828	95	123
8829	95	124
8830	95	125
8831	95	126
8832	95	127
8833	95	128
8834	95	129
8835	95	130
8836	96	38
8837	96	39
8838	96	40
8839	96	41
8840	96	42
8841	96	43
8842	96	44
8843	96	45
8844	96	46
8845	96	47
8846	96	48
8847	96	49
8848	96	50
8849	96	51
8850	96	52
8851	96	53
8852	96	54
8853	96	55
8854	96	56
8855	96	57
8856	96	58
8857	96	59
8858	96	60
8859	96	61
8860	96	62
8861	96	63
8862	96	64
8863	96	65
8864	96	66
8865	96	67
8866	96	68
8867	96	69
8868	96	70
8869	96	71
8870	96	72
8871	96	73
8872	96	74
8873	96	75
8874	96	76
8875	96	77
8876	96	78
8877	96	79
8878	96	80
8879	96	81
8880	96	82
8881	96	83
8882	96	84
8883	96	85
8884	96	86
8885	96	87
8886	96	88
8887	96	89
8888	96	90
8889	96	91
8890	96	92
8891	96	93
8892	96	94
8893	96	95
8894	96	96
8895	96	97
8896	96	98
8897	96	99
8898	96	100
8899	96	101
8900	96	102
8901	96	103
8902	96	104
8903	96	105
8904	96	106
8905	96	107
8906	96	108
8907	96	109
8908	96	110
8909	96	111
8910	96	112
8911	96	113
8912	96	114
8913	96	115
8914	96	116
8915	96	117
8916	96	118
8917	96	119
8918	96	120
8919	96	121
8920	96	122
8921	96	123
8922	96	124
8923	96	125
8924	96	126
8925	96	127
8926	96	128
8927	96	129
8928	96	130
8929	97	38
8930	97	39
8931	97	40
8932	97	41
8933	97	42
8934	97	43
8935	97	44
8936	97	45
8937	97	46
8938	97	47
8939	97	48
8940	97	49
8941	97	50
8942	97	51
8943	97	52
8944	97	53
8945	97	54
8946	97	55
8947	97	56
8948	97	57
8949	97	58
8950	97	59
8951	97	60
8952	97	61
8953	97	62
8954	97	63
8955	97	64
8956	97	65
8957	97	66
8958	97	67
8959	97	68
8960	97	69
8961	97	70
8962	97	71
8963	97	72
8964	97	73
8965	97	74
8966	97	75
8967	97	76
8968	97	77
8969	97	78
8970	97	79
8971	97	80
8972	97	81
8973	97	82
8974	97	83
8975	97	84
8976	97	85
8977	97	86
8978	97	87
8979	97	88
8980	97	89
8981	97	90
8982	97	91
8983	97	92
8984	97	93
8985	97	94
8986	97	95
8987	97	96
8988	97	97
8989	97	98
8990	97	99
8991	97	100
8992	97	101
8993	97	102
8994	97	103
8995	97	104
8996	97	105
8997	97	106
8998	97	107
8999	97	108
9000	97	109
9001	97	110
9002	97	111
9003	97	112
9004	97	113
9005	97	114
9006	97	115
9007	97	116
9008	97	117
9009	97	118
9010	97	119
9011	97	120
9012	97	121
9013	97	122
9014	97	123
9015	97	124
9016	97	125
9017	97	126
9018	97	127
9019	97	128
9020	97	129
9021	97	130
9022	98	38
9023	98	39
9024	98	40
9025	98	41
9026	98	42
9027	98	43
9028	98	44
9029	98	45
9030	98	46
9031	98	47
9032	98	48
9033	98	49
9034	98	50
9035	98	51
9036	98	52
9037	98	53
9038	98	54
9039	98	55
9040	98	56
9041	98	57
9042	98	58
9043	98	59
9044	98	60
9045	98	61
9046	98	62
9047	98	63
9048	98	64
9049	98	65
9050	98	66
9051	98	67
9052	98	68
9053	98	69
9054	98	70
9055	98	71
9056	98	72
9057	98	73
9058	98	74
9059	98	75
9060	98	76
9061	98	77
9062	98	78
9063	98	79
9064	98	80
9065	98	81
9066	98	82
9067	98	83
9068	98	84
9069	98	85
9070	98	86
9071	98	87
9072	98	88
9073	98	89
9074	98	90
9075	98	91
9076	98	92
9077	98	93
9078	98	94
9079	98	95
9080	98	96
9081	98	97
9082	98	98
9083	98	99
9084	98	100
9085	98	101
9086	98	102
9087	98	103
9088	98	104
9089	98	105
9090	98	106
9091	98	107
9092	98	108
9093	98	109
9094	98	110
9095	98	111
9096	98	112
9097	98	113
9098	98	114
9099	98	115
9100	98	116
9101	98	117
9102	98	118
9103	98	119
9104	98	120
9105	98	121
9106	98	122
9107	98	123
9108	98	124
9109	98	125
9110	98	126
9111	98	127
9112	98	128
9113	98	129
9114	98	130
9115	99	38
9116	99	39
9117	99	40
9118	99	41
9119	99	42
9120	99	43
9121	99	44
9122	99	45
9123	99	46
9124	99	47
9125	99	48
9126	99	49
9127	99	50
9128	99	51
9129	99	52
9130	99	53
9131	99	54
9132	99	55
9133	99	56
9134	99	57
9135	99	58
9136	99	59
9137	99	60
9138	99	61
9139	99	62
9140	99	63
9141	99	64
9142	99	65
9143	99	66
9144	99	67
9145	99	68
9146	99	69
9147	99	70
9148	99	71
9149	99	72
9150	99	73
9151	99	74
9152	99	75
9153	99	76
9154	99	77
9155	99	78
9156	99	79
9157	99	80
9158	99	81
9159	99	82
9160	99	83
9161	99	84
9162	99	85
9163	99	86
9164	99	87
9165	99	88
9166	99	89
9167	99	90
9168	99	91
9169	99	92
9170	99	93
9171	99	94
9172	99	95
9173	99	96
9174	99	97
9175	99	98
9176	99	99
9177	99	100
9178	99	101
9179	99	102
9180	99	103
9181	99	104
9182	99	105
9183	99	106
9184	99	107
9185	99	108
9186	99	109
9187	99	110
9188	99	111
9189	99	112
9190	99	113
9191	99	114
9192	99	115
9193	99	116
9194	99	117
9195	99	118
9196	99	119
9197	99	120
9198	99	121
9199	99	122
9200	99	123
9201	99	124
9202	99	125
9203	99	126
9204	99	127
9205	99	128
9206	99	129
9207	99	130
9208	100	38
9209	100	39
9210	100	40
9211	100	41
9212	100	42
9213	100	43
9214	100	44
9215	100	45
9216	100	46
9217	100	47
9218	100	48
9219	100	49
9220	100	50
9221	100	51
9222	100	52
9223	100	53
9224	100	54
9225	100	55
9226	100	56
9227	100	57
9228	100	58
9229	100	59
9230	100	60
9231	100	61
9232	100	62
9233	100	63
9234	100	64
9235	100	65
9236	100	66
9237	100	67
9238	100	68
9239	100	69
9240	100	70
9241	100	71
9242	100	72
9243	100	73
9244	100	74
9245	100	75
9246	100	76
9247	100	77
9248	100	78
9249	100	79
9250	100	80
9251	100	81
9252	100	82
9253	100	83
9254	100	84
9255	100	85
9256	100	86
9257	100	87
9258	100	88
9259	100	89
9260	100	90
9261	100	91
9262	100	92
9263	100	93
9264	100	94
9265	100	95
9266	100	96
9267	100	97
9268	100	98
9269	100	99
9270	100	100
9271	100	101
9272	100	102
9273	100	103
9274	100	104
9275	100	105
9276	100	106
9277	100	107
9278	100	108
9279	100	109
9280	100	110
9281	100	111
9282	100	112
9283	100	113
9284	100	114
9285	100	115
9286	100	116
9287	100	117
9288	100	118
9289	100	119
9290	100	120
9291	100	121
9292	100	122
9293	100	123
9294	100	124
9295	100	125
9296	100	126
9297	100	127
9298	100	128
9299	100	129
9300	100	130
9301	101	38
9302	101	39
9303	101	40
9304	101	41
9305	101	42
9306	101	43
9307	101	44
9308	101	45
9309	101	46
9310	101	47
9311	101	48
9312	101	49
9313	101	50
9314	101	51
9315	101	52
9316	101	53
9317	101	54
9318	101	55
9319	101	56
9320	101	57
9321	101	58
9322	101	59
9323	101	60
9324	101	61
9325	101	62
9326	101	63
9327	101	64
9328	101	65
9329	101	66
9330	101	67
9331	101	68
9332	101	69
9333	101	70
9334	101	71
9335	101	72
9336	101	73
9337	101	74
9338	101	75
9339	101	76
9340	101	77
9341	101	78
9342	101	79
9343	101	80
9344	101	81
9345	101	82
9346	101	83
9347	101	84
9348	101	85
9349	101	86
9350	101	87
9351	101	88
9352	101	89
9353	101	90
9354	101	91
9355	101	92
9356	101	93
9357	101	94
9358	101	95
9359	101	96
9360	101	97
9361	101	98
9362	101	99
9363	101	100
9364	101	101
9365	101	102
9366	101	103
9367	101	104
9368	101	105
9369	101	106
9370	101	107
9371	101	108
9372	101	109
9373	101	110
9374	101	111
9375	101	112
9376	101	113
9377	101	114
9378	101	115
9379	101	116
9380	101	117
9381	101	118
9382	101	119
9383	101	120
9384	101	121
9385	101	122
9386	101	123
9387	101	124
9388	101	125
9389	101	126
9390	101	127
9391	101	128
9392	101	129
9393	101	130
9394	102	38
9395	102	39
9396	102	40
9397	102	41
9398	102	42
9399	102	43
9400	102	44
9401	102	45
9402	102	46
9403	102	47
9404	102	48
9405	102	49
9406	102	50
9407	102	51
9408	102	52
9409	102	53
9410	102	54
9411	102	55
9412	102	56
9413	102	57
9414	102	58
9415	102	59
9416	102	60
9417	102	61
9418	102	62
9419	102	63
9420	102	64
9421	102	65
9422	102	66
9423	102	67
9424	102	68
9425	102	69
9426	102	70
9427	102	71
9428	102	72
9429	102	73
9430	102	74
9431	102	75
9432	102	76
9433	102	77
9434	102	78
9435	102	79
9436	102	80
9437	102	81
9438	102	82
9439	102	83
9440	102	84
9441	102	85
9442	102	86
9443	102	87
9444	102	88
9445	102	89
9446	102	90
9447	102	91
9448	102	92
9449	102	93
9450	102	94
9451	102	95
9452	102	96
9453	102	97
9454	102	98
9455	102	99
9456	102	100
9457	102	101
9458	102	102
9459	102	103
9460	102	104
9461	102	105
9462	102	106
9463	102	107
9464	102	108
9465	102	109
9466	102	110
9467	102	111
9468	102	112
9469	102	113
9470	102	114
9471	102	115
9472	102	116
9473	102	117
9474	102	118
9475	102	119
9476	102	120
9477	102	121
9478	102	122
9479	102	123
9480	102	124
9481	102	125
9482	102	126
9483	102	127
9484	102	128
9485	102	129
9486	102	130
9487	103	38
9488	103	39
9489	103	40
9490	103	41
9491	103	42
9492	103	43
9493	103	44
9494	103	45
9495	103	46
9496	103	47
9497	103	48
9498	103	49
9499	103	50
9500	103	51
9501	103	52
9502	103	53
9503	103	54
9504	103	55
9505	103	56
9506	103	57
9507	103	58
9508	103	59
9509	103	60
9510	103	61
9511	103	62
9512	103	63
9513	103	64
9514	103	65
9515	103	66
9516	103	67
9517	103	68
9518	103	69
9519	103	70
9520	103	71
9521	103	72
9522	103	73
9523	103	74
9524	103	75
9525	103	76
9526	103	77
9527	103	78
9528	103	79
9529	103	80
9530	103	81
9531	103	82
9532	103	83
9533	103	84
9534	103	85
9535	103	86
9536	103	87
9537	103	88
9538	103	89
9539	103	90
9540	103	91
9541	103	92
9542	103	93
9543	103	94
9544	103	95
9545	103	96
9546	103	97
9547	103	98
9548	103	99
9549	103	100
9550	103	101
9551	103	102
9552	103	103
9553	103	104
9554	103	105
9555	103	106
9556	103	107
9557	103	108
9558	103	109
9559	103	110
9560	103	111
9561	103	112
9562	103	113
9563	103	114
9564	103	115
9565	103	116
9566	103	117
9567	103	118
9568	103	119
9569	103	120
9570	103	121
9571	103	122
9572	103	123
9573	103	124
9574	103	125
9575	103	126
9576	103	127
9577	103	128
9578	103	129
9579	103	130
9580	104	38
9581	104	39
9582	104	40
9583	104	41
9584	104	42
9585	104	43
9586	104	44
9587	104	45
9588	104	46
9589	104	47
9590	104	48
9591	104	49
9592	104	50
9593	104	51
9594	104	52
9595	104	53
9596	104	54
9597	104	55
9598	104	56
9599	104	57
9600	104	58
9601	104	59
9602	104	60
9603	104	61
9604	104	62
9605	104	63
9606	104	64
9607	104	65
9608	104	66
9609	104	67
9610	104	68
9611	104	69
9612	104	70
9613	104	71
9614	104	72
9615	104	73
9616	104	74
9617	104	75
9618	104	76
9619	104	77
9620	104	78
9621	104	79
9622	104	80
9623	104	81
9624	104	82
9625	104	83
9626	104	84
9627	104	85
9628	104	86
9629	104	87
9630	104	88
9631	104	89
9632	104	90
9633	104	91
9634	104	92
9635	104	93
9636	104	94
9637	104	95
9638	104	96
9639	104	97
9640	104	98
9641	104	99
9642	104	100
9643	104	101
9644	104	102
9645	104	103
9646	104	104
9647	104	105
9648	104	106
9649	104	107
9650	104	108
9651	104	109
9652	104	110
9653	104	111
9654	104	112
9655	104	113
9656	104	114
9657	104	115
9658	104	116
9659	104	117
9660	104	118
9661	104	119
9662	104	120
9663	104	121
9664	104	122
9665	104	123
9666	104	124
9667	104	125
9668	104	126
9669	104	127
9670	104	128
9671	104	129
9672	104	130
9673	105	38
9674	105	39
9675	105	40
9676	105	41
9677	105	42
9678	105	43
9679	105	44
9680	105	45
9681	105	46
9682	105	47
9683	105	48
9684	105	49
9685	105	50
9686	105	51
9687	105	52
9688	105	53
9689	105	54
9690	105	55
9691	105	56
9692	105	57
9693	105	58
9694	105	59
9695	105	60
9696	105	61
9697	105	62
9698	105	63
9699	105	64
9700	105	65
9701	105	66
9702	105	67
9703	105	68
9704	105	69
9705	105	70
9706	105	71
9707	105	72
9708	105	73
9709	105	74
9710	105	75
9711	105	76
9712	105	77
9713	105	78
9714	105	79
9715	105	80
9716	105	81
9717	105	82
9718	105	83
9719	105	84
9720	105	85
9721	105	86
9722	105	87
9723	105	88
9724	105	89
9725	105	90
9726	105	91
9727	105	92
9728	105	93
9729	105	94
9730	105	95
9731	105	96
9732	105	97
9733	105	98
9734	105	99
9735	105	100
9736	105	101
9737	105	102
9738	105	103
9739	105	104
9740	105	105
9741	105	106
9742	105	107
9743	105	108
9744	105	109
9745	105	110
9746	105	111
9747	105	112
9748	105	113
9749	105	114
9750	105	115
9751	105	116
9752	105	117
9753	105	118
9754	105	119
9755	105	120
9756	105	121
9757	105	122
9758	105	123
9759	105	124
9760	105	125
9761	105	126
9762	105	127
9763	105	128
9764	105	129
9765	105	130
9766	106	38
9767	106	39
9768	106	40
9769	106	41
9770	106	42
9771	106	43
9772	106	44
9773	106	45
9774	106	46
9775	106	47
9776	106	48
9777	106	49
9778	106	50
9779	106	51
9780	106	52
9781	106	53
9782	106	54
9783	106	55
9784	106	56
9785	106	57
9786	106	58
9787	106	59
9788	106	60
9789	106	61
9790	106	62
9791	106	63
9792	106	64
9793	106	65
9794	106	66
9795	106	67
9796	106	68
9797	106	69
9798	106	70
9799	106	71
9800	106	72
9801	106	73
9802	106	74
9803	106	75
9804	106	76
9805	106	77
9806	106	78
9807	106	79
9808	106	80
9809	106	81
9810	106	82
9811	106	83
9812	106	84
9813	106	85
9814	106	86
9815	106	87
9816	106	88
9817	106	89
9818	106	90
9819	106	91
9820	106	92
9821	106	93
9822	106	94
9823	106	95
9824	106	96
9825	106	97
9826	106	98
9827	106	99
9828	106	100
9829	106	101
9830	106	102
9831	106	103
9832	106	104
9833	106	105
9834	106	106
9835	106	107
9836	106	108
9837	106	109
9838	106	110
9839	106	111
9840	106	112
9841	106	113
9842	106	114
9843	106	115
9844	106	116
9845	106	117
9846	106	118
9847	106	119
9848	106	120
9849	106	121
9850	106	122
9851	106	123
9852	106	124
9853	106	125
9854	106	126
9855	106	127
9856	106	128
9857	106	129
9858	106	130
9859	107	38
9860	107	39
9861	107	40
9862	107	41
9863	107	42
9864	107	43
9865	107	44
9866	107	45
9867	107	46
9868	107	47
9869	107	48
9870	107	49
9871	107	50
9872	107	51
9873	107	52
9874	107	53
9875	107	54
9876	107	55
9877	107	56
9878	107	57
9879	107	58
9880	107	59
9881	107	60
9882	107	61
9883	107	62
9884	107	63
9885	107	64
9886	107	65
9887	107	66
9888	107	67
9889	107	68
9890	107	69
9891	107	70
9892	107	71
9893	107	72
9894	107	73
9895	107	74
9896	107	75
9897	107	76
9898	107	77
9899	107	78
9900	107	79
9901	107	80
9902	107	81
9903	107	82
9904	107	83
9905	107	84
9906	107	85
9907	107	86
9908	107	87
9909	107	88
9910	107	89
9911	107	90
9912	107	91
9913	107	92
9914	107	93
9915	107	94
9916	107	95
9917	107	96
9918	107	97
9919	107	98
9920	107	99
9921	107	100
9922	107	101
9923	107	102
9924	107	103
9925	107	104
9926	107	105
9927	107	106
9928	107	107
9929	107	108
9930	107	109
9931	107	110
9932	107	111
9933	107	112
9934	107	113
9935	107	114
9936	107	115
9937	107	116
9938	107	117
9939	107	118
9940	107	119
9941	107	120
9942	107	121
9943	107	122
9944	107	123
9945	107	124
9946	107	125
9947	107	126
9948	107	127
9949	107	128
9950	107	129
9951	107	130
9952	108	38
9953	108	39
9954	108	40
9955	108	41
9956	108	42
9957	108	43
9958	108	44
9959	108	45
9960	108	46
9961	108	47
9962	108	48
9963	108	49
9964	108	50
9965	108	51
9966	108	52
9967	108	53
9968	108	54
9969	108	55
9970	108	56
9971	108	57
9972	108	58
9973	108	59
9974	108	60
9975	108	61
9976	108	62
9977	108	63
9978	108	64
9979	108	65
9980	108	66
9981	108	67
9982	108	68
9983	108	69
9984	108	70
9985	108	71
9986	108	72
9987	108	73
9988	108	74
9989	108	75
9990	108	76
9991	108	77
9992	108	78
9993	108	79
9994	108	80
9995	108	81
9996	108	82
9997	108	83
9998	108	84
9999	108	85
10000	108	86
10001	108	87
10002	108	88
10003	108	89
10004	108	90
10005	108	91
10006	108	92
10007	108	93
10008	108	94
10009	108	95
10010	108	96
10011	108	97
10012	108	98
10013	108	99
10014	108	100
10015	108	101
10016	108	102
10017	108	103
10018	108	104
10019	108	105
10020	108	106
10021	108	107
10022	108	108
10023	108	109
10024	108	110
10025	108	111
10026	108	112
10027	108	113
10028	108	114
10029	108	115
10030	108	116
10031	108	117
10032	108	118
10033	108	119
10034	108	120
10035	108	121
10036	108	122
10037	108	123
10038	108	124
10039	108	125
10040	108	126
10041	108	127
10042	108	128
10043	108	129
10044	108	130
10045	109	38
10046	109	39
10047	109	40
10048	109	41
10049	109	42
10050	109	43
10051	109	44
10052	109	45
10053	109	46
10054	109	47
10055	109	48
10056	109	49
10057	109	50
10058	109	51
10059	109	52
10060	109	53
10061	109	54
10062	109	55
10063	109	56
10064	109	57
10065	109	58
10066	109	59
10067	109	60
10068	109	61
10069	109	62
10070	109	63
10071	109	64
10072	109	65
10073	109	66
10074	109	67
10075	109	68
10076	109	69
10077	109	70
10078	109	71
10079	109	72
10080	109	73
10081	109	74
10082	109	75
10083	109	76
10084	109	77
10085	109	78
10086	109	79
10087	109	80
10088	109	81
10089	109	82
10090	109	83
10091	109	84
10092	109	85
10093	109	86
10094	109	87
10095	109	88
10096	109	89
10097	109	90
10098	109	91
10099	109	92
10100	109	93
10101	109	94
10102	109	95
10103	109	96
10104	109	97
10105	109	98
10106	109	99
10107	109	100
10108	109	101
10109	109	102
10110	109	103
10111	109	104
10112	109	105
10113	109	106
10114	109	107
10115	109	108
10116	109	109
10117	109	110
10118	109	111
10119	109	112
10120	109	113
10121	109	114
10122	109	115
10123	109	116
10124	109	117
10125	109	118
10126	109	119
10127	109	120
10128	109	121
10129	109	122
10130	109	123
10131	109	124
10132	109	125
10133	109	126
10134	109	127
10135	109	128
10136	109	129
10137	109	130
10138	110	38
10139	110	39
10140	110	40
10141	110	41
10142	110	42
10143	110	43
10144	110	44
10145	110	45
10146	110	46
10147	110	47
10148	110	48
10149	110	49
10150	110	50
10151	110	51
10152	110	52
10153	110	53
10154	110	54
10155	110	55
10156	110	56
10157	110	57
10158	110	58
10159	110	59
10160	110	60
10161	110	61
10162	110	62
10163	110	63
10164	110	64
10165	110	65
10166	110	66
10167	110	67
10168	110	68
10169	110	69
10170	110	70
10171	110	71
10172	110	72
10173	110	73
10174	110	74
10175	110	75
10176	110	76
10177	110	77
10178	110	78
10179	110	79
10180	110	80
10181	110	81
10182	110	82
10183	110	83
10184	110	84
10185	110	85
10186	110	86
10187	110	87
10188	110	88
10189	110	89
10190	110	90
10191	110	91
10192	110	92
10193	110	93
10194	110	94
10195	110	95
10196	110	96
10197	110	97
10198	110	98
10199	110	99
10200	110	100
10201	110	101
10202	110	102
10203	110	103
10204	110	104
10205	110	105
10206	110	106
10207	110	107
10208	110	108
10209	110	109
10210	110	110
10211	110	111
10212	110	112
10213	110	113
10214	110	114
10215	110	115
10216	110	116
10217	110	117
10218	110	118
10219	110	119
10220	110	120
10221	110	121
10222	110	122
10223	110	123
10224	110	124
10225	110	125
10226	110	126
10227	110	127
10228	110	128
10229	110	129
10230	110	130
10231	111	38
10232	111	39
10233	111	40
10234	111	41
10235	111	42
10236	111	43
10237	111	44
10238	111	45
10239	111	46
10240	111	47
10241	111	48
10242	111	49
10243	111	50
10244	111	51
10245	111	52
10246	111	53
10247	111	54
10248	111	55
10249	111	56
10250	111	57
10251	111	58
10252	111	59
10253	111	60
10254	111	61
10255	111	62
10256	111	63
10257	111	64
10258	111	65
10259	111	66
10260	111	67
10261	111	68
10262	111	69
10263	111	70
10264	111	71
10265	111	72
10266	111	73
10267	111	74
10268	111	75
10269	111	76
10270	111	77
10271	111	78
10272	111	79
10273	111	80
10274	111	81
10275	111	82
10276	111	83
10277	111	84
10278	111	85
10279	111	86
10280	111	87
10281	111	88
10282	111	89
10283	111	90
10284	111	91
10285	111	92
10286	111	93
10287	111	94
10288	111	95
10289	111	96
10290	111	97
10291	111	98
10292	111	99
10293	111	100
10294	111	101
10295	111	102
10296	111	103
10297	111	104
10298	111	105
10299	111	106
10300	111	107
10301	111	108
10302	111	109
10303	111	110
10304	111	111
10305	111	112
10306	111	113
10307	111	114
10308	111	115
10309	111	116
10310	111	117
10311	111	118
10312	111	119
10313	111	120
10314	111	121
10315	111	122
10316	111	123
10317	111	124
10318	111	125
10319	111	126
10320	111	127
10321	111	128
10322	111	129
10323	111	130
10324	112	38
10325	112	39
10326	112	40
10327	112	41
10328	112	42
10329	112	43
10330	112	44
10331	112	45
10332	112	46
10333	112	47
10334	112	48
10335	112	49
10336	112	50
10337	112	51
10338	112	52
10339	112	53
10340	112	54
10341	112	55
10342	112	56
10343	112	57
10344	112	58
10345	112	59
10346	112	60
10347	112	61
10348	112	62
10349	112	63
10350	112	64
10351	112	65
10352	112	66
10353	112	67
10354	112	68
10355	112	69
10356	112	70
10357	112	71
10358	112	72
10359	112	73
10360	112	74
10361	112	75
10362	112	76
10363	112	77
10364	112	78
10365	112	79
10366	112	80
10367	112	81
10368	112	82
10369	112	83
10370	112	84
10371	112	85
10372	112	86
10373	112	87
10374	112	88
10375	112	89
10376	112	90
10377	112	91
10378	112	92
10379	112	93
10380	112	94
10381	112	95
10382	112	96
10383	112	97
10384	112	98
10385	112	99
10386	112	100
10387	112	101
10388	112	102
10389	112	103
10390	112	104
10391	112	105
10392	112	106
10393	112	107
10394	112	108
10395	112	109
10396	112	110
10397	112	111
10398	112	112
10399	112	113
10400	112	114
10401	112	115
10402	112	116
10403	112	117
10404	112	118
10405	112	119
10406	112	120
10407	112	121
10408	112	122
10409	112	123
10410	112	124
10411	112	125
10412	112	126
10413	112	127
10414	112	128
10415	112	129
10416	112	130
10417	113	38
10418	113	39
10419	113	40
10420	113	41
10421	113	42
10422	113	43
10423	113	44
10424	113	45
10425	113	46
10426	113	47
10427	113	48
10428	113	49
10429	113	50
10430	113	51
10431	113	52
10432	113	53
10433	113	54
10434	113	55
10435	113	56
10436	113	57
10437	113	58
10438	113	59
10439	113	60
10440	113	61
10441	113	62
10442	113	63
10443	113	64
10444	113	65
10445	113	66
10446	113	67
10447	113	68
10448	113	69
10449	113	70
10450	113	71
10451	113	72
10452	113	73
10453	113	74
10454	113	75
10455	113	76
10456	113	77
10457	113	78
10458	113	79
10459	113	80
10460	113	81
10461	113	82
10462	113	83
10463	113	84
10464	113	85
10465	113	86
10466	113	87
10467	113	88
10468	113	89
10469	113	90
10470	113	91
10471	113	92
10472	113	93
10473	113	94
10474	113	95
10475	113	96
10476	113	97
10477	113	98
10478	113	99
10479	113	100
10480	113	101
10481	113	102
10482	113	103
10483	113	104
10484	113	105
10485	113	106
10486	113	107
10487	113	108
10488	113	109
10489	113	110
10490	113	111
10491	113	112
10492	113	113
10493	113	114
10494	113	115
10495	113	116
10496	113	117
10497	113	118
10498	113	119
10499	113	120
10500	113	121
10501	113	122
10502	113	123
10503	113	124
10504	113	125
10505	113	126
10506	113	127
10507	113	128
10508	113	129
10509	113	130
10510	114	38
10511	114	39
10512	114	40
10513	114	41
10514	114	42
10515	114	43
10516	114	44
10517	114	45
10518	114	46
10519	114	47
10520	114	48
10521	114	49
10522	114	50
10523	114	51
10524	114	52
10525	114	53
10526	114	54
10527	114	55
10528	114	56
10529	114	57
10530	114	58
10531	114	59
10532	114	60
10533	114	61
10534	114	62
10535	114	63
10536	114	64
10537	114	65
10538	114	66
10539	114	67
10540	114	68
10541	114	69
10542	114	70
10543	114	71
10544	114	72
10545	114	73
10546	114	74
10547	114	75
10548	114	76
10549	114	77
10550	114	78
10551	114	79
10552	114	80
10553	114	81
10554	114	82
10555	114	83
10556	114	84
10557	114	85
10558	114	86
10559	114	87
10560	114	88
10561	114	89
10562	114	90
10563	114	91
10564	114	92
10565	114	93
10566	114	94
10567	114	95
10568	114	96
10569	114	97
10570	114	98
10571	114	99
10572	114	100
10573	114	101
10574	114	102
10575	114	103
10576	114	104
10577	114	105
10578	114	106
10579	114	107
10580	114	108
10581	114	109
10582	114	110
10583	114	111
10584	114	112
10585	114	113
10586	114	114
10587	114	115
10588	114	116
10589	114	117
10590	114	118
10591	114	119
10592	114	120
10593	114	121
10594	114	122
10595	114	123
10596	114	124
10597	114	125
10598	114	126
10599	114	127
10600	114	128
10601	114	129
10602	114	130
10603	115	38
10604	115	39
10605	115	40
10606	115	41
10607	115	42
10608	115	43
10609	115	44
10610	115	45
10611	115	46
10612	115	47
10613	115	48
10614	115	49
10615	115	50
10616	115	51
10617	115	52
10618	115	53
10619	115	54
10620	115	55
10621	115	56
10622	115	57
10623	115	58
10624	115	59
10625	115	60
10626	115	61
10627	115	62
10628	115	63
10629	115	64
10630	115	65
10631	115	66
10632	115	67
10633	115	68
10634	115	69
10635	115	70
10636	115	71
10637	115	72
10638	115	73
10639	115	74
10640	115	75
10641	115	76
10642	115	77
10643	115	78
10644	115	79
10645	115	80
10646	115	81
10647	115	82
10648	115	83
10649	115	84
10650	115	85
10651	115	86
10652	115	87
10653	115	88
10654	115	89
10655	115	90
10656	115	91
10657	115	92
10658	115	93
10659	115	94
10660	115	95
10661	115	96
10662	115	97
10663	115	98
10664	115	99
10665	115	100
10666	115	101
10667	115	102
10668	115	103
10669	115	104
10670	115	105
10671	115	106
10672	115	107
10673	115	108
10674	115	109
10675	115	110
10676	115	111
10677	115	112
10678	115	113
10679	115	114
10680	115	115
10681	115	116
10682	115	117
10683	115	118
10684	115	119
10685	115	120
10686	115	121
10687	115	122
10688	115	123
10689	115	124
10690	115	125
10691	115	126
10692	115	127
10693	115	128
10694	115	129
10695	115	130
10696	116	38
10697	116	39
10698	116	40
10699	116	41
10700	116	42
10701	116	43
10702	116	44
10703	116	45
10704	116	46
10705	116	47
10706	116	48
10707	116	49
10708	116	50
10709	116	51
10710	116	52
10711	116	53
10712	116	54
10713	116	55
10714	116	56
10715	116	57
10716	116	58
10717	116	59
10718	116	60
10719	116	61
10720	116	62
10721	116	63
10722	116	64
10723	116	65
10724	116	66
10725	116	67
10726	116	68
10727	116	69
10728	116	70
10729	116	71
10730	116	72
10731	116	73
10732	116	74
10733	116	75
10734	116	76
10735	116	77
10736	116	78
10737	116	79
10738	116	80
10739	116	81
10740	116	82
10741	116	83
10742	116	84
10743	116	85
10744	116	86
10745	116	87
10746	116	88
10747	116	89
10748	116	90
10749	116	91
10750	116	92
10751	116	93
10752	116	94
10753	116	95
10754	116	96
10755	116	97
10756	116	98
10757	116	99
10758	116	100
10759	116	101
10760	116	102
10761	116	103
10762	116	104
10763	116	105
10764	116	106
10765	116	107
10766	116	108
10767	116	109
10768	116	110
10769	116	111
10770	116	112
10771	116	113
10772	116	114
10773	116	115
10774	116	116
10775	116	117
10776	116	118
10777	116	119
10778	116	120
10779	116	121
10780	116	122
10781	116	123
10782	116	124
10783	116	125
10784	116	126
10785	116	127
10786	116	128
10787	116	129
10788	116	130
10789	117	38
10790	117	39
10791	117	40
10792	117	41
10793	117	42
10794	117	43
10795	117	44
10796	117	45
10797	117	46
10798	117	47
10799	117	48
10800	117	49
10801	117	50
10802	117	51
10803	117	52
10804	117	53
10805	117	54
10806	117	55
10807	117	56
10808	117	57
10809	117	58
10810	117	59
10811	117	60
10812	117	61
10813	117	62
10814	117	63
10815	117	64
10816	117	65
10817	117	66
10818	117	67
10819	117	68
10820	117	69
10821	117	70
10822	117	71
10823	117	72
10824	117	73
10825	117	74
10826	117	75
10827	117	76
10828	117	77
10829	117	78
10830	117	79
10831	117	80
10832	117	81
10833	117	82
10834	117	83
10835	117	84
10836	117	85
10837	117	86
10838	117	87
10839	117	88
10840	117	89
10841	117	90
10842	117	91
10843	117	92
10844	117	93
10845	117	94
10846	117	95
10847	117	96
10848	117	97
10849	117	98
10850	117	99
10851	117	100
10852	117	101
10853	117	102
10854	117	103
10855	117	104
10856	117	105
10857	117	106
10858	117	107
10859	117	108
10860	117	109
10861	117	110
10862	117	111
10863	117	112
10864	117	113
10865	117	114
10866	117	115
10867	117	116
10868	117	117
10869	117	118
10870	117	119
10871	117	120
10872	117	121
10873	117	122
10874	117	123
10875	117	124
10876	117	125
10877	117	126
10878	117	127
10879	117	128
10880	117	129
10881	117	130
10882	118	38
10883	118	39
10884	118	40
10885	118	41
10886	118	42
10887	118	43
10888	118	44
10889	118	45
10890	118	46
10891	118	47
10892	118	48
10893	118	49
10894	118	50
10895	118	51
10896	118	52
10897	118	53
10898	118	54
10899	118	55
10900	118	56
10901	118	57
10902	118	58
10903	118	59
10904	118	60
10905	118	61
10906	118	62
10907	118	63
10908	118	64
10909	118	65
10910	118	66
10911	118	67
10912	118	68
10913	118	69
10914	118	70
10915	118	71
10916	118	72
10917	118	73
10918	118	74
10919	118	75
10920	118	76
10921	118	77
10922	118	78
10923	118	79
10924	118	80
10925	118	81
10926	118	82
10927	118	83
10928	118	84
10929	118	85
10930	118	86
10931	118	87
10932	118	88
10933	118	89
10934	118	90
10935	118	91
10936	118	92
10937	118	93
10938	118	94
10939	118	95
10940	118	96
10941	118	97
10942	118	98
10943	118	99
10944	118	100
10945	118	101
10946	118	102
10947	118	103
10948	118	104
10949	118	105
10950	118	106
10951	118	107
10952	118	108
10953	118	109
10954	118	110
10955	118	111
10956	118	112
10957	118	113
10958	118	114
10959	118	115
10960	118	116
10961	118	117
10962	118	118
10963	118	119
10964	118	120
10965	118	121
10966	118	122
10967	118	123
10968	118	124
10969	118	125
10970	118	126
10971	118	127
10972	118	128
10973	118	129
10974	118	130
10975	119	38
10976	119	39
10977	119	40
10978	119	41
10979	119	42
10980	119	43
10981	119	44
10982	119	45
10983	119	46
10984	119	47
10985	119	48
10986	119	49
10987	119	50
10988	119	51
10989	119	52
10990	119	53
10991	119	54
10992	119	55
10993	119	56
10994	119	57
10995	119	58
10996	119	59
10997	119	60
10998	119	61
10999	119	62
11000	119	63
11001	119	64
11002	119	65
11003	119	66
11004	119	67
11005	119	68
11006	119	69
11007	119	70
11008	119	71
11009	119	72
11010	119	73
11011	119	74
11012	119	75
11013	119	76
11014	119	77
11015	119	78
11016	119	79
11017	119	80
11018	119	81
11019	119	82
11020	119	83
11021	119	84
11022	119	85
11023	119	86
11024	119	87
11025	119	88
11026	119	89
11027	119	90
11028	119	91
11029	119	92
11030	119	93
11031	119	94
11032	119	95
11033	119	96
11034	119	97
11035	119	98
11036	119	99
11037	119	100
11038	119	101
11039	119	102
11040	119	103
11041	119	104
11042	119	105
11043	119	106
11044	119	107
11045	119	108
11046	119	109
11047	119	110
11048	119	111
11049	119	112
11050	119	113
11051	119	114
11052	119	115
11053	119	116
11054	119	117
11055	119	118
11056	119	119
11057	119	120
11058	119	121
11059	119	122
11060	119	123
11061	119	124
11062	119	125
11063	119	126
11064	119	127
11065	119	128
11066	119	129
11067	119	130
11068	120	38
11069	120	39
11070	120	40
11071	120	41
11072	120	42
11073	120	43
11074	120	44
11075	120	45
11076	120	46
11077	120	47
11078	120	48
11079	120	49
11080	120	50
11081	120	51
11082	120	52
11083	120	53
11084	120	54
11085	120	55
11086	120	56
11087	120	57
11088	120	58
11089	120	59
11090	120	60
11091	120	61
11092	120	62
11093	120	63
11094	120	64
11095	120	65
11096	120	66
11097	120	67
11098	120	68
11099	120	69
11100	120	70
11101	120	71
11102	120	72
11103	120	73
11104	120	74
11105	120	75
11106	120	76
11107	120	77
11108	120	78
11109	120	79
11110	120	80
11111	120	81
11112	120	82
11113	120	83
11114	120	84
11115	120	85
11116	120	86
11117	120	87
11118	120	88
11119	120	89
11120	120	90
11121	120	91
11122	120	92
11123	120	93
11124	120	94
11125	120	95
11126	120	96
11127	120	97
11128	120	98
11129	120	99
11130	120	100
11131	120	101
11132	120	102
11133	120	103
11134	120	104
11135	120	105
11136	120	106
11137	120	107
11138	120	108
11139	120	109
11140	120	110
11141	120	111
11142	120	112
11143	120	113
11144	120	114
11145	120	115
11146	120	116
11147	120	117
11148	120	118
11149	120	119
11150	120	120
11151	120	121
11152	120	122
11153	120	123
11154	120	124
11155	120	125
11156	120	126
11157	120	127
11158	120	128
11159	120	129
11160	120	130
11161	121	38
11162	121	39
11163	121	40
11164	121	41
11165	121	42
11166	121	43
11167	121	44
11168	121	45
11169	121	46
11170	121	47
11171	121	48
11172	121	49
11173	121	50
11174	121	51
11175	121	52
11176	121	53
11177	121	54
11178	121	55
11179	121	56
11180	121	57
11181	121	58
11182	121	59
11183	121	60
11184	121	61
11185	121	62
11186	121	63
11187	121	64
11188	121	65
11189	121	66
11190	121	67
11191	121	68
11192	121	69
11193	121	70
11194	121	71
11195	121	72
11196	121	73
11197	121	74
11198	121	75
11199	121	76
11200	121	77
11201	121	78
11202	121	79
11203	121	80
11204	121	81
11205	121	82
11206	121	83
11207	121	84
11208	121	85
11209	121	86
11210	121	87
11211	121	88
11212	121	89
11213	121	90
11214	121	91
11215	121	92
11216	121	93
11217	121	94
11218	121	95
11219	121	96
11220	121	97
11221	121	98
11222	121	99
11223	121	100
11224	121	101
11225	121	102
11226	121	103
11227	121	104
11228	121	105
11229	121	106
11230	121	107
11231	121	108
11232	121	109
11233	121	110
11234	121	111
11235	121	112
11236	121	113
11237	121	114
11238	121	115
11239	121	116
11240	121	117
11241	121	118
11242	121	119
11243	121	120
11244	121	121
11245	121	122
11246	121	123
11247	121	124
11248	121	125
11249	121	126
11250	121	127
11251	121	128
11252	121	129
11253	121	130
11254	122	38
11255	122	39
11256	122	40
11257	122	41
11258	122	42
11259	122	43
11260	122	44
11261	122	45
11262	122	46
11263	122	47
11264	122	48
11265	122	49
11266	122	50
11267	122	51
11268	122	52
11269	122	53
11270	122	54
11271	122	55
11272	122	56
11273	122	57
11274	122	58
11275	122	59
11276	122	60
11277	122	61
11278	122	62
11279	122	63
11280	122	64
11281	122	65
11282	122	66
11283	122	67
11284	122	68
11285	122	69
11286	122	70
11287	122	71
11288	122	72
11289	122	73
11290	122	74
11291	122	75
11292	122	76
11293	122	77
11294	122	78
11295	122	79
11296	122	80
11297	122	81
11298	122	82
11299	122	83
11300	122	84
11301	122	85
11302	122	86
11303	122	87
11304	122	88
11305	122	89
11306	122	90
11307	122	91
11308	122	92
11309	122	93
11310	122	94
11311	122	95
11312	122	96
11313	122	97
11314	122	98
11315	122	99
11316	122	100
11317	122	101
11318	122	102
11319	122	103
11320	122	104
11321	122	105
11322	122	106
11323	122	107
11324	122	108
11325	122	109
11326	122	110
11327	122	111
11328	122	112
11329	122	113
11330	122	114
11331	122	115
11332	122	116
11333	122	117
11334	122	118
11335	122	119
11336	122	120
11337	122	121
11338	122	122
11339	122	123
11340	122	124
11341	122	125
11342	122	126
11343	122	127
11344	122	128
11345	122	129
11346	122	130
11347	123	38
11348	123	39
11349	123	40
11350	123	41
11351	123	42
11352	123	43
11353	123	44
11354	123	45
11355	123	46
11356	123	47
11357	123	48
11358	123	49
11359	123	50
11360	123	51
11361	123	52
11362	123	53
11363	123	54
11364	123	55
11365	123	56
11366	123	57
11367	123	58
11368	123	59
11369	123	60
11370	123	61
11371	123	62
11372	123	63
11373	123	64
11374	123	65
11375	123	66
11376	123	67
11377	123	68
11378	123	69
11379	123	70
11380	123	71
11381	123	72
11382	123	73
11383	123	74
11384	123	75
11385	123	76
11386	123	77
11387	123	78
11388	123	79
11389	123	80
11390	123	81
11391	123	82
11392	123	83
11393	123	84
11394	123	85
11395	123	86
11396	123	87
11397	123	88
11398	123	89
11399	123	90
11400	123	91
11401	123	92
11402	123	93
11403	123	94
11404	123	95
11405	123	96
11406	123	97
11407	123	98
11408	123	99
11409	123	100
11410	123	101
11411	123	102
11412	123	103
11413	123	104
11414	123	105
11415	123	106
11416	123	107
11417	123	108
11418	123	109
11419	123	110
11420	123	111
11421	123	112
11422	123	113
11423	123	114
11424	123	115
11425	123	116
11426	123	117
11427	123	118
11428	123	119
11429	123	120
11430	123	121
11431	123	122
11432	123	123
11433	123	124
11434	123	125
11435	123	126
11436	123	127
11437	123	128
11438	123	129
11439	123	130
11440	124	38
11441	124	39
11442	124	40
11443	124	41
11444	124	42
11445	124	43
11446	124	44
11447	124	45
11448	124	46
11449	124	47
11450	124	48
11451	124	49
11452	124	50
11453	124	51
11454	124	52
11455	124	53
11456	124	54
11457	124	55
11458	124	56
11459	124	57
11460	124	58
11461	124	59
11462	124	60
11463	124	61
11464	124	62
11465	124	63
11466	124	64
11467	124	65
11468	124	66
11469	124	67
11470	124	68
11471	124	69
11472	124	70
11473	124	71
11474	124	72
11475	124	73
11476	124	74
11477	124	75
11478	124	76
11479	124	77
11480	124	78
11481	124	79
11482	124	80
11483	124	81
11484	124	82
11485	124	83
11486	124	84
11487	124	85
11488	124	86
11489	124	87
11490	124	88
11491	124	89
11492	124	90
11493	124	91
11494	124	92
11495	124	93
11496	124	94
11497	124	95
11498	124	96
11499	124	97
11500	124	98
11501	124	99
11502	124	100
11503	124	101
11504	124	102
11505	124	103
11506	124	104
11507	124	105
11508	124	106
11509	124	107
11510	124	108
11511	124	109
11512	124	110
11513	124	111
11514	124	112
11515	124	113
11516	124	114
11517	124	115
11518	124	116
11519	124	117
11520	124	118
11521	124	119
11522	124	120
11523	124	121
11524	124	122
11525	124	123
11526	124	124
11527	124	125
11528	124	126
11529	124	127
11530	124	128
11531	124	129
11532	124	130
11533	125	38
11534	125	39
11535	125	40
11536	125	41
11537	125	42
11538	125	43
11539	125	44
11540	125	45
11541	125	46
11542	125	47
11543	125	48
11544	125	49
11545	125	50
11546	125	51
11547	125	52
11548	125	53
11549	125	54
11550	125	55
11551	125	56
11552	125	57
11553	125	58
11554	125	59
11555	125	60
11556	125	61
11557	125	62
11558	125	63
11559	125	64
11560	125	65
11561	125	66
11562	125	67
11563	125	68
11564	125	69
11565	125	70
11566	125	71
11567	125	72
11568	125	73
11569	125	74
11570	125	75
11571	125	76
11572	125	77
11573	125	78
11574	125	79
11575	125	80
11576	125	81
11577	125	82
11578	125	83
11579	125	84
11580	125	85
11581	125	86
11582	125	87
11583	125	88
11584	125	89
11585	125	90
11586	125	91
11587	125	92
11588	125	93
11589	125	94
11590	125	95
11591	125	96
11592	125	97
11593	125	98
11594	125	99
11595	125	100
11596	125	101
11597	125	102
11598	125	103
11599	125	104
11600	125	105
11601	125	106
11602	125	107
11603	125	108
11604	125	109
11605	125	110
11606	125	111
11607	125	112
11608	125	113
11609	125	114
11610	125	115
11611	125	116
11612	125	117
11613	125	118
11614	125	119
11615	125	120
11616	125	121
11617	125	122
11618	125	123
11619	125	124
11620	125	125
11621	125	126
11622	125	127
11623	125	128
11624	125	129
11625	125	130
11626	126	38
11627	126	39
11628	126	40
11629	126	41
11630	126	42
11631	126	43
11632	126	44
11633	126	45
11634	126	46
11635	126	47
11636	126	48
11637	126	49
11638	126	50
11639	126	51
11640	126	52
11641	126	53
11642	126	54
11643	126	55
11644	126	56
11645	126	57
11646	126	58
11647	126	59
11648	126	60
11649	126	61
11650	126	62
11651	126	63
11652	126	64
11653	126	65
11654	126	66
11655	126	67
11656	126	68
11657	126	69
11658	126	70
11659	126	71
11660	126	72
11661	126	73
11662	126	74
11663	126	75
11664	126	76
11665	126	77
11666	126	78
11667	126	79
11668	126	80
11669	126	81
11670	126	82
11671	126	83
11672	126	84
11673	126	85
11674	126	86
11675	126	87
11676	126	88
11677	126	89
11678	126	90
11679	126	91
11680	126	92
11681	126	93
11682	126	94
11683	126	95
11684	126	96
11685	126	97
11686	126	98
11687	126	99
11688	126	100
11689	126	101
11690	126	102
11691	126	103
11692	126	104
11693	126	105
11694	126	106
11695	126	107
11696	126	108
11697	126	109
11698	126	110
11699	126	111
11700	126	112
11701	126	113
11702	126	114
11703	126	115
11704	126	116
11705	126	117
11706	126	118
11707	126	119
11708	126	120
11709	126	121
11710	126	122
11711	126	123
11712	126	124
11713	126	125
11714	126	126
11715	126	127
11716	126	128
11717	126	129
11718	126	130
11719	127	38
11720	127	39
11721	127	40
11722	127	41
11723	127	42
11724	127	43
11725	127	44
11726	127	45
11727	127	46
11728	127	47
11729	127	48
11730	127	49
11731	127	50
11732	127	51
11733	127	52
11734	127	53
11735	127	54
11736	127	55
11737	127	56
11738	127	57
11739	127	58
11740	127	59
11741	127	60
11742	127	61
11743	127	62
11744	127	63
11745	127	64
11746	127	65
11747	127	66
11748	127	67
11749	127	68
11750	127	69
11751	127	70
11752	127	71
11753	127	72
11754	127	73
11755	127	74
11756	127	75
11757	127	76
11758	127	77
11759	127	78
11760	127	79
11761	127	80
11762	127	81
11763	127	82
11764	127	83
11765	127	84
11766	127	85
11767	127	86
11768	127	87
11769	127	88
11770	127	89
11771	127	90
11772	127	91
11773	127	92
11774	127	93
11775	127	94
11776	127	95
11777	127	96
11778	127	97
11779	127	98
11780	127	99
11781	127	100
11782	127	101
11783	127	102
11784	127	103
11785	127	104
11786	127	105
11787	127	106
11788	127	107
11789	127	108
11790	127	109
11791	127	110
11792	127	111
11793	127	112
11794	127	113
11795	127	114
11796	127	115
11797	127	116
11798	127	117
11799	127	118
11800	127	119
11801	127	120
11802	127	121
11803	127	122
11804	127	123
11805	127	124
11806	127	125
11807	127	126
11808	127	127
11809	127	128
11810	127	129
11811	127	130
11812	128	38
11813	128	39
11814	128	40
11815	128	41
11816	128	42
11817	128	43
11818	128	44
11819	128	45
11820	128	46
11821	128	47
11822	128	48
11823	128	49
11824	128	50
11825	128	51
11826	128	52
11827	128	53
11828	128	54
11829	128	55
11830	128	56
11831	128	57
11832	128	58
11833	128	59
11834	128	60
11835	128	61
11836	128	62
11837	128	63
11838	128	64
11839	128	65
11840	128	66
11841	128	67
11842	128	68
11843	128	69
11844	128	70
11845	128	71
11846	128	72
11847	128	73
11848	128	74
11849	128	75
11850	128	76
11851	128	77
11852	128	78
11853	128	79
11854	128	80
11855	128	81
11856	128	82
11857	128	83
11858	128	84
11859	128	85
11860	128	86
11861	128	87
11862	128	88
11863	128	89
11864	128	90
11865	128	91
11866	128	92
11867	128	93
11868	128	94
11869	128	95
11870	128	96
11871	128	97
11872	128	98
11873	128	99
11874	128	100
11875	128	101
11876	128	102
11877	128	103
11878	128	104
11879	128	105
11880	128	106
11881	128	107
11882	128	108
11883	128	109
11884	128	110
11885	128	111
11886	128	112
11887	128	113
11888	128	114
11889	128	115
11890	128	116
11891	128	117
11892	128	118
11893	128	119
11894	128	120
11895	128	121
11896	128	122
11897	128	123
11898	128	124
11899	128	125
11900	128	126
11901	128	127
11902	128	128
11903	128	129
11904	128	130
11905	129	38
11906	129	39
11907	129	40
11908	129	41
11909	129	42
11910	129	43
11911	129	44
11912	129	45
11913	129	46
11914	129	47
11915	129	48
11916	129	49
11917	129	50
11918	129	51
11919	129	52
11920	129	53
11921	129	54
11922	129	55
11923	129	56
11924	129	57
11925	129	58
11926	129	59
11927	129	60
11928	129	61
11929	129	62
11930	129	63
11931	129	64
11932	129	65
11933	129	66
11934	129	67
11935	129	68
11936	129	69
11937	129	70
11938	129	71
11939	129	72
11940	129	73
11941	129	74
11942	129	75
11943	129	76
11944	129	77
11945	129	78
11946	129	79
11947	129	80
11948	129	81
11949	129	82
11950	129	83
11951	129	84
11952	129	85
11953	129	86
11954	129	87
11955	129	88
11956	129	89
11957	129	90
11958	129	91
11959	129	92
11960	129	93
11961	129	94
11962	129	95
11963	129	96
11964	129	97
11965	129	98
11966	129	99
11967	129	100
11968	129	101
11969	129	102
11970	129	103
11971	129	104
11972	129	105
11973	129	106
11974	129	107
11975	129	108
11976	129	109
11977	129	110
11978	129	111
11979	129	112
11980	129	113
11981	129	114
11982	129	115
11983	129	116
11984	129	117
11985	129	118
11986	129	119
11987	129	120
11988	129	121
11989	129	122
11990	129	123
11991	129	124
11992	129	125
11993	129	126
11994	129	127
11995	129	128
11996	129	129
11997	129	130
11998	130	38
11999	130	39
12000	130	40
12001	130	41
12002	130	42
12003	130	43
12004	130	44
12005	130	45
12006	130	46
12007	130	47
12008	130	48
12009	130	49
12010	130	50
12011	130	51
12012	130	52
12013	130	53
12014	130	54
12015	130	55
12016	130	56
12017	130	57
12018	130	58
12019	130	59
12020	130	60
12021	130	61
12022	130	62
12023	130	63
12024	130	64
12025	130	65
12026	130	66
12027	130	67
12028	130	68
12029	130	69
12030	130	70
12031	130	71
12032	130	72
12033	130	73
12034	130	74
12035	130	75
12036	130	76
12037	130	77
12038	130	78
12039	130	79
12040	130	80
12041	130	81
12042	130	82
12043	130	83
12044	130	84
12045	130	85
12046	130	86
12047	130	87
12048	130	88
12049	130	89
12050	130	90
12051	130	91
12052	130	92
12053	130	93
12054	130	94
12055	130	95
12056	130	96
12057	130	97
12058	130	98
12059	130	99
12060	130	100
12061	130	101
12062	130	102
12063	130	103
12064	130	104
12065	130	105
12066	130	106
12067	130	107
12068	130	108
12069	130	109
12070	130	110
12071	130	111
12072	130	112
12073	130	113
12074	130	114
12075	130	115
12076	130	116
12077	130	117
12078	130	118
12079	130	119
12080	130	120
12081	130	121
12082	130	122
12083	130	123
12084	130	124
12085	130	125
12086	130	126
12087	130	127
12088	130	128
12089	130	129
12090	130	130
12091	131	38
12092	131	39
12093	131	40
12094	131	41
12095	131	42
12096	131	43
12097	131	44
12098	131	45
12099	131	46
12100	131	47
12101	131	48
12102	131	49
12103	131	50
12104	131	51
12105	131	52
12106	131	53
12107	131	54
12108	131	55
12109	131	56
12110	131	57
12111	131	58
12112	131	59
12113	131	60
12114	131	61
12115	131	62
12116	131	63
12117	131	64
12118	131	65
12119	131	66
12120	131	67
12121	131	68
12122	131	69
12123	131	70
12124	131	71
12125	131	72
12126	131	73
12127	131	74
12128	131	75
12129	131	76
12130	131	77
12131	131	78
12132	131	79
12133	131	80
12134	131	81
12135	131	82
12136	131	83
12137	131	84
12138	131	85
12139	131	86
12140	131	87
12141	131	88
12142	131	89
12143	131	90
12144	131	91
12145	131	92
12146	131	93
12147	131	94
12148	131	95
12149	131	96
12150	131	97
12151	131	98
12152	131	99
12153	131	100
12154	131	101
12155	131	102
12156	131	103
12157	131	104
12158	131	105
12159	131	106
12160	131	107
12161	131	108
12162	131	109
12163	131	110
12164	131	111
12165	131	112
12166	131	113
12167	131	114
12168	131	115
12169	131	116
12170	131	117
12171	131	118
12172	131	119
12173	131	120
12174	131	121
12175	131	122
12176	131	123
12177	131	124
12178	131	125
12179	131	126
12180	131	127
12181	131	128
12182	131	129
12183	131	130
12184	132	38
12185	132	39
12186	132	40
12187	132	41
12188	132	42
12189	132	43
12190	132	44
12191	132	45
12192	132	46
12193	132	47
12194	132	48
12195	132	49
12196	132	50
12197	132	51
12198	132	52
12199	132	53
12200	132	54
12201	132	55
12202	132	56
12203	132	57
12204	132	58
12205	132	59
12206	132	60
12207	132	61
12208	132	62
12209	132	63
12210	132	64
12211	132	65
12212	132	66
12213	132	67
12214	132	68
12215	132	69
12216	132	70
12217	132	71
12218	132	72
12219	132	73
12220	132	74
12221	132	75
12222	132	76
12223	132	77
12224	132	78
12225	132	79
12226	132	80
12227	132	81
12228	132	82
12229	132	83
12230	132	84
12231	132	85
12232	132	86
12233	132	87
12234	132	88
12235	132	89
12236	132	90
12237	132	91
12238	132	92
12239	132	93
12240	132	94
12241	132	95
12242	132	96
12243	132	97
12244	132	98
12245	132	99
12246	132	100
12247	132	101
12248	132	102
12249	132	103
12250	132	104
12251	132	105
12252	132	106
12253	132	107
12254	132	108
12255	132	109
12256	132	110
12257	132	111
12258	132	112
12259	132	113
12260	132	114
12261	132	115
12262	132	116
12263	132	117
12264	132	118
12265	132	119
12266	132	120
12267	132	121
12268	132	122
12269	132	123
12270	132	124
12271	132	125
12272	132	126
12273	132	127
12274	132	128
12275	132	129
12276	132	130
12277	133	38
12278	133	39
12279	133	40
12280	133	41
12281	133	42
12282	133	43
12283	133	44
12284	133	45
12285	133	46
12286	133	47
12287	133	48
12288	133	49
12289	133	50
12290	133	51
12291	133	52
12292	133	53
12293	133	54
12294	133	55
12295	133	56
12296	133	57
12297	133	58
12298	133	59
12299	133	60
12300	133	61
12301	133	62
12302	133	63
12303	133	64
12304	133	65
12305	133	66
12306	133	67
12307	133	68
12308	133	69
12309	133	70
12310	133	71
12311	133	72
12312	133	73
12313	133	74
12314	133	75
12315	133	76
12316	133	77
12317	133	78
12318	133	79
12319	133	80
12320	133	81
12321	133	82
12322	133	83
12323	133	84
12324	133	85
12325	133	86
12326	133	87
12327	133	88
12328	133	89
12329	133	90
12330	133	91
12331	133	92
12332	133	93
12333	133	94
12334	133	95
12335	133	96
12336	133	97
12337	133	98
12338	133	99
12339	133	100
12340	133	101
12341	133	102
12342	133	103
12343	133	104
12344	133	105
12345	133	106
12346	133	107
12347	133	108
12348	133	109
12349	133	110
12350	133	111
12351	133	112
12352	133	113
12353	133	114
12354	133	115
12355	133	116
12356	133	117
12357	133	118
12358	133	119
12359	133	120
12360	133	121
12361	133	122
12362	133	123
12363	133	124
12364	133	125
12365	133	126
12366	133	127
12367	133	128
12368	133	129
12369	133	130
12370	134	38
12371	134	39
12372	134	40
12373	134	41
12374	134	42
12375	134	43
12376	134	44
12377	134	45
12378	134	46
12379	134	47
12380	134	48
12381	134	49
12382	134	50
12383	134	51
12384	134	52
12385	134	53
12386	134	54
12387	134	55
12388	134	56
12389	134	57
12390	134	58
12391	134	59
12392	134	60
12393	134	61
12394	134	62
12395	134	63
12396	134	64
12397	134	65
12398	134	66
12399	134	67
12400	134	68
12401	134	69
12402	134	70
12403	134	71
12404	134	72
12405	134	73
12406	134	74
12407	134	75
12408	134	76
12409	134	77
12410	134	78
12411	134	79
12412	134	80
12413	134	81
12414	134	82
12415	134	83
12416	134	84
12417	134	85
12418	134	86
12419	134	87
12420	134	88
12421	134	89
12422	134	90
12423	134	91
12424	134	92
12425	134	93
12426	134	94
12427	134	95
12428	134	96
12429	134	97
12430	134	98
12431	134	99
12432	134	100
12433	134	101
12434	134	102
12435	134	103
12436	134	104
12437	134	105
12438	134	106
12439	134	107
12440	134	108
12441	134	109
12442	134	110
12443	134	111
12444	134	112
12445	134	113
12446	134	114
12447	134	115
12448	134	116
12449	134	117
12450	134	118
12451	134	119
12452	134	120
12453	134	121
12454	134	122
12455	134	123
12456	134	124
12457	134	125
12458	134	126
12459	134	127
12460	134	128
12461	134	129
12462	134	130
12463	135	38
12464	135	39
12465	135	40
12466	135	41
12467	135	42
12468	135	43
12469	135	44
12470	135	45
12471	135	46
12472	135	47
12473	135	48
12474	135	49
12475	135	50
12476	135	51
12477	135	52
12478	135	53
12479	135	54
12480	135	55
12481	135	56
12482	135	57
12483	135	58
12484	135	59
12485	135	60
12486	135	61
12487	135	62
12488	135	63
12489	135	64
12490	135	65
12491	135	66
12492	135	67
12493	135	68
12494	135	69
12495	135	70
12496	135	71
12497	135	72
12498	135	73
12499	135	74
12500	135	75
12501	135	76
12502	135	77
12503	135	78
12504	135	79
12505	135	80
12506	135	81
12507	135	82
12508	135	83
12509	135	84
12510	135	85
12511	135	86
12512	135	87
12513	135	88
12514	135	89
12515	135	90
12516	135	91
12517	135	92
12518	135	93
12519	135	94
12520	135	95
12521	135	96
12522	135	97
12523	135	98
12524	135	99
12525	135	100
12526	135	101
12527	135	102
12528	135	103
12529	135	104
12530	135	105
12531	135	106
12532	135	107
12533	135	108
12534	135	109
12535	135	110
12536	135	111
12537	135	112
12538	135	113
12539	135	114
12540	135	115
12541	135	116
12542	135	117
12543	135	118
12544	135	119
12545	135	120
12546	135	121
12547	135	122
12548	135	123
12549	135	124
12550	135	125
12551	135	126
12552	135	127
12553	135	128
12554	135	129
12555	135	130
12556	136	38
12557	136	39
12558	136	40
12559	136	41
12560	136	42
12561	136	43
12562	136	44
12563	136	45
12564	136	46
12565	136	47
12566	136	48
12567	136	49
12568	136	50
12569	136	51
12570	136	52
12571	136	53
12572	136	54
12573	136	55
12574	136	56
12575	136	57
12576	136	58
12577	136	59
12578	136	60
12579	136	61
12580	136	62
12581	136	63
12582	136	64
12583	136	65
12584	136	66
12585	136	67
12586	136	68
12587	136	69
12588	136	70
12589	136	71
12590	136	72
12591	136	73
12592	136	74
12593	136	75
12594	136	76
12595	136	77
12596	136	78
12597	136	79
12598	136	80
12599	136	81
12600	136	82
12601	136	83
12602	136	84
12603	136	85
12604	136	86
12605	136	87
12606	136	88
12607	136	89
12608	136	90
12609	136	91
12610	136	92
12611	136	93
12612	136	94
12613	136	95
12614	136	96
12615	136	97
12616	136	98
12617	136	99
12618	136	100
12619	136	101
12620	136	102
12621	136	103
12622	136	104
12623	136	105
12624	136	106
12625	136	107
12626	136	108
12627	136	109
12628	136	110
12629	136	111
12630	136	112
12631	136	113
12632	136	114
12633	136	115
12634	136	116
12635	136	117
12636	136	118
12637	136	119
12638	136	120
12639	136	121
12640	136	122
12641	136	123
12642	136	124
12643	136	125
12644	136	126
12645	136	127
12646	136	128
12647	136	129
12648	136	130
12649	137	38
12650	137	39
12651	137	40
12652	137	41
12653	137	42
12654	137	43
12655	137	44
12656	137	45
12657	137	46
12658	137	47
12659	137	48
12660	137	49
12661	137	50
12662	137	51
12663	137	52
12664	137	53
12665	137	54
12666	137	55
12667	137	56
12668	137	57
12669	137	58
12670	137	59
12671	137	60
12672	137	61
12673	137	62
12674	137	63
12675	137	64
12676	137	65
12677	137	66
12678	137	67
12679	137	68
12680	137	69
12681	137	70
12682	137	71
12683	137	72
12684	137	73
12685	137	74
12686	137	75
12687	137	76
12688	137	77
12689	137	78
12690	137	79
12691	137	80
12692	137	81
12693	137	82
12694	137	83
12695	137	84
12696	137	85
12697	137	86
12698	137	87
12699	137	88
12700	137	89
12701	137	90
12702	137	91
12703	137	92
12704	137	93
12705	137	94
12706	137	95
12707	137	96
12708	137	97
12709	137	98
12710	137	99
12711	137	100
12712	137	101
12713	137	102
12714	137	103
12715	137	104
12716	137	105
12717	137	106
12718	137	107
12719	137	108
12720	137	109
12721	137	110
12722	137	111
12723	137	112
12724	137	113
12725	137	114
12726	137	115
12727	137	116
12728	137	117
12729	137	118
12730	137	119
12731	137	120
12732	137	121
12733	137	122
12734	137	123
12735	137	124
12736	137	125
12737	137	126
12738	137	127
12739	137	128
12740	137	129
12741	137	130
12742	138	38
12743	138	39
12744	138	40
12745	138	41
12746	138	42
12747	138	43
12748	138	44
12749	138	45
12750	138	46
12751	138	47
12752	138	48
12753	138	49
12754	138	50
12755	138	51
12756	138	52
12757	138	53
12758	138	54
12759	138	55
12760	138	56
12761	138	57
12762	138	58
12763	138	59
12764	138	60
12765	138	61
12766	138	62
12767	138	63
12768	138	64
12769	138	65
12770	138	66
12771	138	67
12772	138	68
12773	138	69
12774	138	70
12775	138	71
12776	138	72
12777	138	73
12778	138	74
12779	138	75
12780	138	76
12781	138	77
12782	138	78
12783	138	79
12784	138	80
12785	138	81
12786	138	82
12787	138	83
12788	138	84
12789	138	85
12790	138	86
12791	138	87
12792	138	88
12793	138	89
12794	138	90
12795	138	91
12796	138	92
12797	138	93
12798	138	94
12799	138	95
12800	138	96
12801	138	97
12802	138	98
12803	138	99
12804	138	100
12805	138	101
12806	138	102
12807	138	103
12808	138	104
12809	138	105
12810	138	106
12811	138	107
12812	138	108
12813	138	109
12814	138	110
12815	138	111
12816	138	112
12817	138	113
12818	138	114
12819	138	115
12820	138	116
12821	138	117
12822	138	118
12823	138	119
12824	138	120
12825	138	121
12826	138	122
12827	138	123
12828	138	124
12829	138	125
12830	138	126
12831	138	127
12832	138	128
12833	138	129
12834	138	130
12835	139	38
12836	139	39
12837	139	40
12838	139	41
12839	139	42
12840	139	43
12841	139	44
12842	139	45
12843	139	46
12844	139	47
12845	139	48
12846	139	49
12847	139	50
12848	139	51
12849	139	52
12850	139	53
12851	139	54
12852	139	55
12853	139	56
12854	139	57
12855	139	58
12856	139	59
12857	139	60
12858	139	61
12859	139	62
12860	139	63
12861	139	64
12862	139	65
12863	139	66
12864	139	67
12865	139	68
12866	139	69
12867	139	70
12868	139	71
12869	139	72
12870	139	73
12871	139	74
12872	139	75
12873	139	76
12874	139	77
12875	139	78
12876	139	79
12877	139	80
12878	139	81
12879	139	82
12880	139	83
12881	139	84
12882	139	85
12883	139	86
12884	139	87
12885	139	88
12886	139	89
12887	139	90
12888	139	91
12889	139	92
12890	139	93
12891	139	94
12892	139	95
12893	139	96
12894	139	97
12895	139	98
12896	139	99
12897	139	100
12898	139	101
12899	139	102
12900	139	103
12901	139	104
12902	139	105
12903	139	106
12904	139	107
12905	139	108
12906	139	109
12907	139	110
12908	139	111
12909	139	112
12910	139	113
12911	139	114
12912	139	115
12913	139	116
12914	139	117
12915	139	118
12916	139	119
12917	139	120
12918	139	121
12919	139	122
12920	139	123
12921	139	124
12922	139	125
12923	139	126
12924	139	127
12925	139	128
12926	139	129
12927	139	130
12928	140	38
12929	140	39
12930	140	40
12931	140	41
12932	140	42
12933	140	43
12934	140	44
12935	140	45
12936	140	46
12937	140	47
12938	140	48
12939	140	49
12940	140	50
12941	140	51
12942	140	52
12943	140	53
12944	140	54
12945	140	55
12946	140	56
12947	140	57
12948	140	58
12949	140	59
12950	140	60
12951	140	61
12952	140	62
12953	140	63
12954	140	64
12955	140	65
12956	140	66
12957	140	67
12958	140	68
12959	140	69
12960	140	70
12961	140	71
12962	140	72
12963	140	73
12964	140	74
12965	140	75
12966	140	76
12967	140	77
12968	140	78
12969	140	79
12970	140	80
12971	140	81
12972	140	82
12973	140	83
12974	140	84
12975	140	85
12976	140	86
12977	140	87
12978	140	88
12979	140	89
12980	140	90
12981	140	91
12982	140	92
12983	140	93
12984	140	94
12985	140	95
12986	140	96
12987	140	97
12988	140	98
12989	140	99
12990	140	100
12991	140	101
12992	140	102
12993	140	103
12994	140	104
12995	140	105
12996	140	106
12997	140	107
12998	140	108
12999	140	109
13000	140	110
13001	140	111
13002	140	112
13003	140	113
13004	140	114
13005	140	115
13006	140	116
13007	140	117
13008	140	118
13009	140	119
13010	140	120
13011	140	121
13012	140	122
13013	140	123
13014	140	124
13015	140	125
13016	140	126
13017	140	127
13018	140	128
13019	140	129
13020	140	130
13021	141	38
13022	141	39
13023	141	40
13024	141	41
13025	141	42
13026	141	43
13027	141	44
13028	141	45
13029	141	46
13030	141	47
13031	141	48
13032	141	49
13033	141	50
13034	141	51
13035	141	52
13036	141	53
13037	141	54
13038	141	55
13039	141	56
13040	141	57
13041	141	58
13042	141	59
13043	141	60
13044	141	61
13045	141	62
13046	141	63
13047	141	64
13048	141	65
13049	141	66
13050	141	67
13051	141	68
13052	141	69
13053	141	70
13054	141	71
13055	141	72
13056	141	73
13057	141	74
13058	141	75
13059	141	76
13060	141	77
13061	141	78
13062	141	79
13063	141	80
13064	141	81
13065	141	82
13066	141	83
13067	141	84
13068	141	85
13069	141	86
13070	141	87
13071	141	88
13072	141	89
13073	141	90
13074	141	91
13075	141	92
13076	141	93
13077	141	94
13078	141	95
13079	141	96
13080	141	97
13081	141	98
13082	141	99
13083	141	100
13084	141	101
13085	141	102
13086	141	103
13087	141	104
13088	141	105
13089	141	106
13090	141	107
13091	141	108
13092	141	109
13093	141	110
13094	141	111
13095	141	112
13096	141	113
13097	141	114
13098	141	115
13099	141	116
13100	141	117
13101	141	118
13102	141	119
13103	141	120
13104	141	121
13105	141	122
13106	141	123
13107	141	124
13108	141	125
13109	141	126
13110	141	127
13111	141	128
13112	141	129
13113	141	130
13114	142	38
13115	142	39
13116	142	40
13117	142	41
13118	142	42
13119	142	43
13120	142	44
13121	142	45
13122	142	46
13123	142	47
13124	142	48
13125	142	49
13126	142	50
13127	142	51
13128	142	52
13129	142	53
13130	142	54
13131	142	55
13132	142	56
13133	142	57
13134	142	58
13135	142	59
13136	142	60
13137	142	61
13138	142	62
13139	142	63
13140	142	64
13141	142	65
13142	142	66
13143	142	67
13144	142	68
13145	142	69
13146	142	70
13147	142	71
13148	142	72
13149	142	73
13150	142	74
13151	142	75
13152	142	76
13153	142	77
13154	142	78
13155	142	79
13156	142	80
13157	142	81
13158	142	82
13159	142	83
13160	142	84
13161	142	85
13162	142	86
13163	142	87
13164	142	88
13165	142	89
13166	142	90
13167	142	91
13168	142	92
13169	142	93
13170	142	94
13171	142	95
13172	142	96
13173	142	97
13174	142	98
13175	142	99
13176	142	100
13177	142	101
13178	142	102
13179	142	103
13180	142	104
13181	142	105
13182	142	106
13183	142	107
13184	142	108
13185	142	109
13186	142	110
13187	142	111
13188	142	112
13189	142	113
13190	142	114
13191	142	115
13192	142	116
13193	142	117
13194	142	118
13195	142	119
13196	142	120
13197	142	121
13198	142	122
13199	142	123
13200	142	124
13201	142	125
13202	142	126
13203	142	127
13204	142	128
13205	142	129
13206	142	130
13207	143	38
13208	143	39
13209	143	40
13210	143	41
13211	143	42
13212	143	43
13213	143	44
13214	143	45
13215	143	46
13216	143	47
13217	143	48
13218	143	49
13219	143	50
13220	143	51
13221	143	52
13222	143	53
13223	143	54
13224	143	55
13225	143	56
13226	143	57
13227	143	58
13228	143	59
13229	143	60
13230	143	61
13231	143	62
13232	143	63
13233	143	64
13234	143	65
13235	143	66
13236	143	67
13237	143	68
13238	143	69
13239	143	70
13240	143	71
13241	143	72
13242	143	73
13243	143	74
13244	143	75
13245	143	76
13246	143	77
13247	143	78
13248	143	79
13249	143	80
13250	143	81
13251	143	82
13252	143	83
13253	143	84
13254	143	85
13255	143	86
13256	143	87
13257	143	88
13258	143	89
13259	143	90
13260	143	91
13261	143	92
13262	143	93
13263	143	94
13264	143	95
13265	143	96
13266	143	97
13267	143	98
13268	143	99
13269	143	100
13270	143	101
13271	143	102
13272	143	103
13273	143	104
13274	143	105
13275	143	106
13276	143	107
13277	143	108
13278	143	109
13279	143	110
13280	143	111
13281	143	112
13282	143	113
13283	143	114
13284	143	115
13285	143	116
13286	143	117
13287	143	118
13288	143	119
13289	143	120
13290	143	121
13291	143	122
13292	143	123
13293	143	124
13294	143	125
13295	143	126
13296	143	127
13297	143	128
13298	143	129
13299	143	130
13300	144	38
13301	144	39
13302	144	40
13303	144	41
13304	144	42
13305	144	43
13306	144	44
13307	144	45
13308	144	46
13309	144	47
13310	144	48
13311	144	49
13312	144	50
13313	144	51
13314	144	52
13315	144	53
13316	144	54
13317	144	55
13318	144	56
13319	144	57
13320	144	58
13321	144	59
13322	144	60
13323	144	61
13324	144	62
13325	144	63
13326	144	64
13327	144	65
13328	144	66
13329	144	67
13330	144	68
13331	144	69
13332	144	70
13333	144	71
13334	144	72
13335	144	73
13336	144	74
13337	144	75
13338	144	76
13339	144	77
13340	144	78
13341	144	79
13342	144	80
13343	144	81
13344	144	82
13345	144	83
13346	144	84
13347	144	85
13348	144	86
13349	144	87
13350	144	88
13351	144	89
13352	144	90
13353	144	91
13354	144	92
13355	144	93
13356	144	94
13357	144	95
13358	144	96
13359	144	97
13360	144	98
13361	144	99
13362	144	100
13363	144	101
13364	144	102
13365	144	103
13366	144	104
13367	144	105
13368	144	106
13369	144	107
13370	144	108
13371	144	109
13372	144	110
13373	144	111
13374	144	112
13375	144	113
13376	144	114
13377	144	115
13378	144	116
13379	144	117
13380	144	118
13381	144	119
13382	144	120
13383	144	121
13384	144	122
13385	144	123
13386	144	124
13387	144	125
13388	144	126
13389	144	127
13390	144	128
13391	144	129
13392	144	130
13393	145	38
13394	145	39
13395	145	40
13396	145	41
13397	145	42
13398	145	43
13399	145	44
13400	145	45
13401	145	46
13402	145	47
13403	145	48
13404	145	49
13405	145	50
13406	145	51
13407	145	52
13408	145	53
13409	145	54
13410	145	55
13411	145	56
13412	145	57
13413	145	58
13414	145	59
13415	145	60
13416	145	61
13417	145	62
13418	145	63
13419	145	64
13420	145	65
13421	145	66
13422	145	67
13423	145	68
13424	145	69
13425	145	70
13426	145	71
13427	145	72
13428	145	73
13429	145	74
13430	145	75
13431	145	76
13432	145	77
13433	145	78
13434	145	79
13435	145	80
13436	145	81
13437	145	82
13438	145	83
13439	145	84
13440	145	85
13441	145	86
13442	145	87
13443	145	88
13444	145	89
13445	145	90
13446	145	91
13447	145	92
13448	145	93
13449	145	94
13450	145	95
13451	145	96
13452	145	97
13453	145	98
13454	145	99
13455	145	100
13456	145	101
13457	145	102
13458	145	103
13459	145	104
13460	145	105
13461	145	106
13462	145	107
13463	145	108
13464	145	109
13465	145	110
13466	145	111
13467	145	112
13468	145	113
13469	145	114
13470	145	115
13471	145	116
13472	145	117
13473	145	118
13474	145	119
13475	145	120
13476	145	121
13477	145	122
13478	145	123
13479	145	124
13480	145	125
13481	145	126
13482	145	127
13483	145	128
13484	145	129
13485	145	130
13486	146	38
13487	146	39
13488	146	40
13489	146	41
13490	146	42
13491	146	43
13492	146	44
13493	146	45
13494	146	46
13495	146	47
13496	146	48
13497	146	49
13498	146	50
13499	146	51
13500	146	52
13501	146	53
13502	146	54
13503	146	55
13504	146	56
13505	146	57
13506	146	58
13507	146	59
13508	146	60
13509	146	61
13510	146	62
13511	146	63
13512	146	64
13513	146	65
13514	146	66
13515	146	67
13516	146	68
13517	146	69
13518	146	70
13519	146	71
13520	146	72
13521	146	73
13522	146	74
13523	146	75
13524	146	76
13525	146	77
13526	146	78
13527	146	79
13528	146	80
13529	146	81
13530	146	82
13531	146	83
13532	146	84
13533	146	85
13534	146	86
13535	146	87
13536	146	88
13537	146	89
13538	146	90
13539	146	91
13540	146	92
13541	146	93
13542	146	94
13543	146	95
13544	146	96
13545	146	97
13546	146	98
13547	146	99
13548	146	100
13549	146	101
13550	146	102
13551	146	103
13552	146	104
13553	146	105
13554	146	106
13555	146	107
13556	146	108
13557	146	109
13558	146	110
13559	146	111
13560	146	112
13561	146	113
13562	146	114
13563	146	115
13564	146	116
13565	146	117
13566	146	118
13567	146	119
13568	146	120
13569	146	121
13570	146	122
13571	146	123
13572	146	124
13573	146	125
13574	146	126
13575	146	127
13576	146	128
13577	146	129
13578	146	130
13579	147	38
13580	147	39
13581	147	40
13582	147	41
13583	147	42
13584	147	43
13585	147	44
13586	147	45
13587	147	46
13588	147	47
13589	147	48
13590	147	49
13591	147	50
13592	147	51
13593	147	52
13594	147	53
13595	147	54
13596	147	55
13597	147	56
13598	147	57
13599	147	58
13600	147	59
13601	147	60
13602	147	61
13603	147	62
13604	147	63
13605	147	64
13606	147	65
13607	147	66
13608	147	67
13609	147	68
13610	147	69
13611	147	70
13612	147	71
13613	147	72
13614	147	73
13615	147	74
13616	147	75
13617	147	76
13618	147	77
13619	147	78
13620	147	79
13621	147	80
13622	147	81
13623	147	82
13624	147	83
13625	147	84
13626	147	85
13627	147	86
13628	147	87
13629	147	88
13630	147	89
13631	147	90
13632	147	91
13633	147	92
13634	147	93
13635	147	94
13636	147	95
13637	147	96
13638	147	97
13639	147	98
13640	147	99
13641	147	100
13642	147	101
13643	147	102
13644	147	103
13645	147	104
13646	147	105
13647	147	106
13648	147	107
13649	147	108
13650	147	109
13651	147	110
13652	147	111
13653	147	112
13654	147	113
13655	147	114
13656	147	115
13657	147	116
13658	147	117
13659	147	118
13660	147	119
13661	147	120
13662	147	121
13663	147	122
13664	147	123
13665	147	124
13666	147	125
13667	147	126
13668	147	127
13669	147	128
13670	147	129
13671	147	130
13672	148	38
13673	148	39
13674	148	40
13675	148	41
13676	148	42
13677	148	43
13678	148	44
13679	148	45
13680	148	46
13681	148	47
13682	148	48
13683	148	49
13684	148	50
13685	148	51
13686	148	52
13687	148	53
13688	148	54
13689	148	55
13690	148	56
13691	148	57
13692	148	58
13693	148	59
13694	148	60
13695	148	61
13696	148	62
13697	148	63
13698	148	64
13699	148	65
13700	148	66
13701	148	67
13702	148	68
13703	148	69
13704	148	70
13705	148	71
13706	148	72
13707	148	73
13708	148	74
13709	148	75
13710	148	76
13711	148	77
13712	148	78
13713	148	79
13714	148	80
13715	148	81
13716	148	82
13717	148	83
13718	148	84
13719	148	85
13720	148	86
13721	148	87
13722	148	88
13723	148	89
13724	148	90
13725	148	91
13726	148	92
13727	148	93
13728	148	94
13729	148	95
13730	148	96
13731	148	97
13732	148	98
13733	148	99
13734	148	100
13735	148	101
13736	148	102
13737	148	103
13738	148	104
13739	148	105
13740	148	106
13741	148	107
13742	148	108
13743	148	109
13744	148	110
13745	148	111
13746	148	112
13747	148	113
13748	148	114
13749	148	115
13750	148	116
13751	148	117
13752	148	118
13753	148	119
13754	148	120
13755	148	121
13756	148	122
13757	148	123
13758	148	124
13759	148	125
13760	148	126
13761	148	127
13762	148	128
13763	148	129
13764	148	130
13765	149	38
13766	149	39
13767	149	40
13768	149	41
13769	149	42
13770	149	43
13771	149	44
13772	149	45
13773	149	46
13774	149	47
13775	149	48
13776	149	49
13777	149	50
13778	149	51
13779	149	52
13780	149	53
13781	149	54
13782	149	55
13783	149	56
13784	149	57
13785	149	58
13786	149	59
13787	149	60
13788	149	61
13789	149	62
13790	149	63
13791	149	64
13792	149	65
13793	149	66
13794	149	67
13795	149	68
13796	149	69
13797	149	70
13798	149	71
13799	149	72
13800	149	73
13801	149	74
13802	149	75
13803	149	76
13804	149	77
13805	149	78
13806	149	79
13807	149	80
13808	149	81
13809	149	82
13810	149	83
13811	149	84
13812	149	85
13813	149	86
13814	149	87
13815	149	88
13816	149	89
13817	149	90
13818	149	91
13819	149	92
13820	149	93
13821	149	94
13822	149	95
13823	149	96
13824	149	97
13825	149	98
13826	149	99
13827	149	100
13828	149	101
13829	149	102
13830	149	103
13831	149	104
13832	149	105
13833	149	106
13834	149	107
13835	149	108
13836	149	109
13837	149	110
13838	149	111
13839	149	112
13840	149	113
13841	149	114
13842	149	115
13843	149	116
13844	149	117
13845	149	118
13846	149	119
13847	149	120
13848	149	121
13849	149	122
13850	149	123
13851	149	124
13852	149	125
13853	149	126
13854	149	127
13855	149	128
13856	149	129
13857	149	130
13858	150	38
13859	150	39
13860	150	40
13861	150	41
13862	150	42
13863	150	43
13864	150	44
13865	150	45
13866	150	46
13867	150	47
13868	150	48
13869	150	49
13870	150	50
13871	150	51
13872	150	52
13873	150	53
13874	150	54
13875	150	55
13876	150	56
13877	150	57
13878	150	58
13879	150	59
13880	150	60
13881	150	61
13882	150	62
13883	150	63
13884	150	64
13885	150	65
13886	150	66
13887	150	67
13888	150	68
13889	150	69
13890	150	70
13891	150	71
13892	150	72
13893	150	73
13894	150	74
13895	150	75
13896	150	76
13897	150	77
13898	150	78
13899	150	79
13900	150	80
13901	150	81
13902	150	82
13903	150	83
13904	150	84
13905	150	85
13906	150	86
13907	150	87
13908	150	88
13909	150	89
13910	150	90
13911	150	91
13912	150	92
13913	150	93
13914	150	94
13915	150	95
13916	150	96
13917	150	97
13918	150	98
13919	150	99
13920	150	100
13921	150	101
13922	150	102
13923	150	103
13924	150	104
13925	150	105
13926	150	106
13927	150	107
13928	150	108
13929	150	109
13930	150	110
13931	150	111
13932	150	112
13933	150	113
13934	150	114
13935	150	115
13936	150	116
13937	150	117
13938	150	118
13939	150	119
13940	150	120
13941	150	121
13942	150	122
13943	150	123
13944	150	124
13945	150	125
13946	150	126
13947	150	127
13948	150	128
13949	150	129
13950	150	130
13951	151	38
13952	151	39
13953	151	40
13954	151	41
13955	151	42
13956	151	43
13957	151	44
13958	151	45
13959	151	46
13960	151	47
13961	151	48
13962	151	49
13963	151	50
13964	151	51
13965	151	52
13966	151	53
13967	151	54
13968	151	55
13969	151	56
13970	151	57
13971	151	58
13972	151	59
13973	151	60
13974	151	61
13975	151	62
13976	151	63
13977	151	64
13978	151	65
13979	151	66
13980	151	67
13981	151	68
13982	151	69
13983	151	70
13984	151	71
13985	151	72
13986	151	73
13987	151	74
13988	151	75
13989	151	76
13990	151	77
13991	151	78
13992	151	79
13993	151	80
13994	151	81
13995	151	82
13996	151	83
13997	151	84
13998	151	85
13999	151	86
14000	151	87
14001	151	88
14002	151	89
14003	151	90
14004	151	91
14005	151	92
14006	151	93
14007	151	94
14008	151	95
14009	151	96
14010	151	97
14011	151	98
14012	151	99
14013	151	100
14014	151	101
14015	151	102
14016	151	103
14017	151	104
14018	151	105
14019	151	106
14020	151	107
14021	151	108
14022	151	109
14023	151	110
14024	151	111
14025	151	112
14026	151	113
14027	151	114
14028	151	115
14029	151	116
14030	151	117
14031	151	118
14032	151	119
14033	151	120
14034	151	121
14035	151	122
14036	151	123
14037	151	124
14038	151	125
14039	151	126
14040	151	127
14041	151	128
14042	151	129
14043	151	130
14044	152	38
14045	152	39
14046	152	40
14047	152	41
14048	152	42
14049	152	43
14050	152	44
14051	152	45
14052	152	46
14053	152	47
14054	152	48
14055	152	49
14056	152	50
14057	152	51
14058	152	52
14059	152	53
14060	152	54
14061	152	55
14062	152	56
14063	152	57
14064	152	58
14065	152	59
14066	152	60
14067	152	61
14068	152	62
14069	152	63
14070	152	64
14071	152	65
14072	152	66
14073	152	67
14074	152	68
14075	152	69
14076	152	70
14077	152	71
14078	152	72
14079	152	73
14080	152	74
14081	152	75
14082	152	76
14083	152	77
14084	152	78
14085	152	79
14086	152	80
14087	152	81
14088	152	82
14089	152	83
14090	152	84
14091	152	85
14092	152	86
14093	152	87
14094	152	88
14095	152	89
14096	152	90
14097	152	91
14098	152	92
14099	152	93
14100	152	94
14101	152	95
14102	152	96
14103	152	97
14104	152	98
14105	152	99
14106	152	100
14107	152	101
14108	152	102
14109	152	103
14110	152	104
14111	152	105
14112	152	106
14113	152	107
14114	152	108
14115	152	109
14116	152	110
14117	152	111
14118	152	112
14119	152	113
14120	152	114
14121	152	115
14122	152	116
14123	152	117
14124	152	118
14125	152	119
14126	152	120
14127	152	121
14128	152	122
14129	152	123
14130	152	124
14131	152	125
14132	152	126
14133	152	127
14134	152	128
14135	152	129
14136	152	130
14137	153	38
14138	153	39
14139	153	40
14140	153	41
14141	153	42
14142	153	43
14143	153	44
14144	153	45
14145	153	46
14146	153	47
14147	153	48
14148	153	49
14149	153	50
14150	153	51
14151	153	52
14152	153	53
14153	153	54
14154	153	55
14155	153	56
14156	153	57
14157	153	58
14158	153	59
14159	153	60
14160	153	61
14161	153	62
14162	153	63
14163	153	64
14164	153	65
14165	153	66
14166	153	67
14167	153	68
14168	153	69
14169	153	70
14170	153	71
14171	153	72
14172	153	73
14173	153	74
14174	153	75
14175	153	76
14176	153	77
14177	153	78
14178	153	79
14179	153	80
14180	153	81
14181	153	82
14182	153	83
14183	153	84
14184	153	85
14185	153	86
14186	153	87
14187	153	88
14188	153	89
14189	153	90
14190	153	91
14191	153	92
14192	153	93
14193	153	94
14194	153	95
14195	153	96
14196	153	97
14197	153	98
14198	153	99
14199	153	100
14200	153	101
14201	153	102
14202	153	103
14203	153	104
14204	153	105
14205	153	106
14206	153	107
14207	153	108
14208	153	109
14209	153	110
14210	153	111
14211	153	112
14212	153	113
14213	153	114
14214	153	115
14215	153	116
14216	153	117
14217	153	118
14218	153	119
14219	153	120
14220	153	121
14221	153	122
14222	153	123
14223	153	124
14224	153	125
14225	153	126
14226	153	127
14227	153	128
14228	153	129
14229	153	130
14230	154	38
14231	154	39
14232	154	40
14233	154	41
14234	154	42
14235	154	43
14236	154	44
14237	154	45
14238	154	46
14239	154	47
14240	154	48
14241	154	49
14242	154	50
14243	154	51
14244	154	52
14245	154	53
14246	154	54
14247	154	55
14248	154	56
14249	154	57
14250	154	58
14251	154	59
14252	154	60
14253	154	61
14254	154	62
14255	154	63
14256	154	64
14257	154	65
14258	154	66
14259	154	67
14260	154	68
14261	154	69
14262	154	70
14263	154	71
14264	154	72
14265	154	73
14266	154	74
14267	154	75
14268	154	76
14269	154	77
14270	154	78
14271	154	79
14272	154	80
14273	154	81
14274	154	82
14275	154	83
14276	154	84
14277	154	85
14278	154	86
14279	154	87
14280	154	88
14281	154	89
14282	154	90
14283	154	91
14284	154	92
14285	154	93
14286	154	94
14287	154	95
14288	154	96
14289	154	97
14290	154	98
14291	154	99
14292	154	100
14293	154	101
14294	154	102
14295	154	103
14296	154	104
14297	154	105
14298	154	106
14299	154	107
14300	154	108
14301	154	109
14302	154	110
14303	154	111
14304	154	112
14305	154	113
14306	154	114
14307	154	115
14308	154	116
14309	154	117
14310	154	118
14311	154	119
14312	154	120
14313	154	121
14314	154	122
14315	154	123
14316	154	124
14317	154	125
14318	154	126
14319	154	127
14320	154	128
14321	154	129
14322	154	130
14323	155	38
14324	155	39
14325	155	40
14326	155	41
14327	155	42
14328	155	43
14329	155	44
14330	155	45
14331	155	46
14332	155	47
14333	155	48
14334	155	49
14335	155	50
14336	155	51
14337	155	52
14338	155	53
14339	155	54
14340	155	55
14341	155	56
14342	155	57
14343	155	58
14344	155	59
14345	155	60
14346	155	61
14347	155	62
14348	155	63
14349	155	64
14350	155	65
14351	155	66
14352	155	67
14353	155	68
14354	155	69
14355	155	70
14356	155	71
14357	155	72
14358	155	73
14359	155	74
14360	155	75
14361	155	76
14362	155	77
14363	155	78
14364	155	79
14365	155	80
14366	155	81
14367	155	82
14368	155	83
14369	155	84
14370	155	85
14371	155	86
14372	155	87
14373	155	88
14374	155	89
14375	155	90
14376	155	91
14377	155	92
14378	155	93
14379	155	94
14380	155	95
14381	155	96
14382	155	97
14383	155	98
14384	155	99
14385	155	100
14386	155	101
14387	155	102
14388	155	103
14389	155	104
14390	155	105
14391	155	106
14392	155	107
14393	155	108
14394	155	109
14395	155	110
14396	155	111
14397	155	112
14398	155	113
14399	155	114
14400	155	115
14401	155	116
14402	155	117
14403	155	118
14404	155	119
14405	155	120
14406	155	121
14407	155	122
14408	155	123
14409	155	124
14410	155	125
14411	155	126
14412	155	127
14413	155	128
14414	155	129
14415	155	130
14416	156	38
14417	156	39
14418	156	40
14419	156	41
14420	156	42
14421	156	43
14422	156	44
14423	156	45
14424	156	46
14425	156	47
14426	156	48
14427	156	49
14428	156	50
14429	156	51
14430	156	52
14431	156	53
14432	156	54
14433	156	55
14434	156	56
14435	156	57
14436	156	58
14437	156	59
14438	156	60
14439	156	61
14440	156	62
14441	156	63
14442	156	64
14443	156	65
14444	156	66
14445	156	67
14446	156	68
14447	156	69
14448	156	70
14449	156	71
14450	156	72
14451	156	73
14452	156	74
14453	156	75
14454	156	76
14455	156	77
14456	156	78
14457	156	79
14458	156	80
14459	156	81
14460	156	82
14461	156	83
14462	156	84
14463	156	85
14464	156	86
14465	156	87
14466	156	88
14467	156	89
14468	156	90
14469	156	91
14470	156	92
14471	156	93
14472	156	94
14473	156	95
14474	156	96
14475	156	97
14476	156	98
14477	156	99
14478	156	100
14479	156	101
14480	156	102
14481	156	103
14482	156	104
14483	156	105
14484	156	106
14485	156	107
14486	156	108
14487	156	109
14488	156	110
14489	156	111
14490	156	112
14491	156	113
14492	156	114
14493	156	115
14494	156	116
14495	156	117
14496	156	118
14497	156	119
14498	156	120
14499	156	121
14500	156	122
14501	156	123
14502	156	124
14503	156	125
14504	156	126
14505	156	127
14506	156	128
14507	156	129
14508	156	130
14509	157	38
14510	157	39
14511	157	40
14512	157	41
14513	157	42
14514	157	43
14515	157	44
14516	157	45
14517	157	46
14518	157	47
14519	157	48
14520	157	49
14521	157	50
14522	157	51
14523	157	52
14524	157	53
14525	157	54
14526	157	55
14527	157	56
14528	157	57
14529	157	58
14530	157	59
14531	157	60
14532	157	61
14533	157	62
14534	157	63
14535	157	64
14536	157	65
14537	157	66
14538	157	67
14539	157	68
14540	157	69
14541	157	70
14542	157	71
14543	157	72
14544	157	73
14545	157	74
14546	157	75
14547	157	76
14548	157	77
14549	157	78
14550	157	79
14551	157	80
14552	157	81
14553	157	82
14554	157	83
14555	157	84
14556	157	85
14557	157	86
14558	157	87
14559	157	88
14560	157	89
14561	157	90
14562	157	91
14563	157	92
14564	157	93
14565	157	94
14566	157	95
14567	157	96
14568	157	97
14569	157	98
14570	157	99
14571	157	100
14572	157	101
14573	157	102
14574	157	103
14575	157	104
14576	157	105
14577	157	106
14578	157	107
14579	157	108
14580	157	109
14581	157	110
14582	157	111
14583	157	112
14584	157	113
14585	157	114
14586	157	115
14587	157	116
14588	157	117
14589	157	118
14590	157	119
14591	157	120
14592	157	121
14593	157	122
14594	157	123
14595	157	124
14596	157	125
14597	157	126
14598	157	127
14599	157	128
14600	157	129
14601	157	130
14602	158	38
14603	158	39
14604	158	40
14605	158	41
14606	158	42
14607	158	43
14608	158	44
14609	158	45
14610	158	46
14611	158	47
14612	158	48
14613	158	49
14614	158	50
14615	158	51
14616	158	52
14617	158	53
14618	158	54
14619	158	55
14620	158	56
14621	158	57
14622	158	58
14623	158	59
14624	158	60
14625	158	61
14626	158	62
14627	158	63
14628	158	64
14629	158	65
14630	158	66
14631	158	67
14632	158	68
14633	158	69
14634	158	70
14635	158	71
14636	158	72
14637	158	73
14638	158	74
14639	158	75
14640	158	76
14641	158	77
14642	158	78
14643	158	79
14644	158	80
14645	158	81
14646	158	82
14647	158	83
14648	158	84
14649	158	85
14650	158	86
14651	158	87
14652	158	88
14653	158	89
14654	158	90
14655	158	91
14656	158	92
14657	158	93
14658	158	94
14659	158	95
14660	158	96
14661	158	97
14662	158	98
14663	158	99
14664	158	100
14665	158	101
14666	158	102
14667	158	103
14668	158	104
14669	158	105
14670	158	106
14671	158	107
14672	158	108
14673	158	109
14674	158	110
14675	158	111
14676	158	112
14677	158	113
14678	158	114
14679	158	115
14680	158	116
14681	158	117
14682	158	118
14683	158	119
14684	158	120
14685	158	121
14686	158	122
14687	158	123
14688	158	124
14689	158	125
14690	158	126
14691	158	127
14692	158	128
14693	158	129
14694	158	130
14695	159	38
14696	159	39
14697	159	40
14698	159	41
14699	159	42
14700	159	43
14701	159	44
14702	159	45
14703	159	46
14704	159	47
14705	159	48
14706	159	49
14707	159	50
14708	159	51
14709	159	52
14710	159	53
14711	159	54
14712	159	55
14713	159	56
14714	159	57
14715	159	58
14716	159	59
14717	159	60
14718	159	61
14719	159	62
14720	159	63
14721	159	64
14722	159	65
14723	159	66
14724	159	67
14725	159	68
14726	159	69
14727	159	70
14728	159	71
14729	159	72
14730	159	73
14731	159	74
14732	159	75
14733	159	76
14734	159	77
14735	159	78
14736	159	79
14737	159	80
14738	159	81
14739	159	82
14740	159	83
14741	159	84
14742	159	85
14743	159	86
14744	159	87
14745	159	88
14746	159	89
14747	159	90
14748	159	91
14749	159	92
14750	159	93
14751	159	94
14752	159	95
14753	159	96
14754	159	97
14755	159	98
14756	159	99
14757	159	100
14758	159	101
14759	159	102
14760	159	103
14761	159	104
14762	159	105
14763	159	106
14764	159	107
14765	159	108
14766	159	109
14767	159	110
14768	159	111
14769	159	112
14770	159	113
14771	159	114
14772	159	115
14773	159	116
14774	159	117
14775	159	118
14776	159	119
14777	159	120
14778	159	121
14779	159	122
14780	159	123
14781	159	124
14782	159	125
14783	159	126
14784	159	127
14785	159	128
14786	159	129
14787	159	130
14788	160	38
14789	160	39
14790	160	40
14791	160	41
14792	160	42
14793	160	43
14794	160	44
14795	160	45
14796	160	46
14797	160	47
14798	160	48
14799	160	49
14800	160	50
14801	160	51
14802	160	52
14803	160	53
14804	160	54
14805	160	55
14806	160	56
14807	160	57
14808	160	58
14809	160	59
14810	160	60
14811	160	61
14812	160	62
14813	160	63
14814	160	64
14815	160	65
14816	160	66
14817	160	67
14818	160	68
14819	160	69
14820	160	70
14821	160	71
14822	160	72
14823	160	73
14824	160	74
14825	160	75
14826	160	76
14827	160	77
14828	160	78
14829	160	79
14830	160	80
14831	160	81
14832	160	82
14833	160	83
14834	160	84
14835	160	85
14836	160	86
14837	160	87
14838	160	88
14839	160	89
14840	160	90
14841	160	91
14842	160	92
14843	160	93
14844	160	94
14845	160	95
14846	160	96
14847	160	97
14848	160	98
14849	160	99
14850	160	100
14851	160	101
14852	160	102
14853	160	103
14854	160	104
14855	160	105
14856	160	106
14857	160	107
14858	160	108
14859	160	109
14860	160	110
14861	160	111
14862	160	112
14863	160	113
14864	160	114
14865	160	115
14866	160	116
14867	160	117
14868	160	118
14869	160	119
14870	160	120
14871	160	121
14872	160	122
14873	160	123
14874	160	124
14875	160	125
14876	160	126
14877	160	127
14878	160	128
14879	160	129
14880	160	130
14881	161	38
14882	161	39
14883	161	40
14884	161	41
14885	161	42
14886	161	43
14887	161	44
14888	161	45
14889	161	46
14890	161	47
14891	161	48
14892	161	49
14893	161	50
14894	161	51
14895	161	52
14896	161	53
14897	161	54
14898	161	55
14899	161	56
14900	161	57
14901	161	58
14902	161	59
14903	161	60
14904	161	61
14905	161	62
14906	161	63
14907	161	64
14908	161	65
14909	161	66
14910	161	67
14911	161	68
14912	161	69
14913	161	70
14914	161	71
14915	161	72
14916	161	73
14917	161	74
14918	161	75
14919	161	76
14920	161	77
14921	161	78
14922	161	79
14923	161	80
14924	161	81
14925	161	82
14926	161	83
14927	161	84
14928	161	85
14929	161	86
14930	161	87
14931	161	88
14932	161	89
14933	161	90
14934	161	91
14935	161	92
14936	161	93
14937	161	94
14938	161	95
14939	161	96
14940	161	97
14941	161	98
14942	161	99
14943	161	100
14944	161	101
14945	161	102
14946	161	103
14947	161	104
14948	161	105
14949	161	106
14950	161	107
14951	161	108
14952	161	109
14953	161	110
14954	161	111
14955	161	112
14956	161	113
14957	161	114
14958	161	115
14959	161	116
14960	161	117
14961	161	118
14962	161	119
14963	161	120
14964	161	121
14965	161	122
14966	161	123
14967	161	124
14968	161	125
14969	161	126
14970	161	127
14971	161	128
14972	161	129
14973	161	130
14974	162	38
14975	162	39
14976	162	40
14977	162	41
14978	162	42
14979	162	43
14980	162	44
14981	162	45
14982	162	46
14983	162	47
14984	162	48
14985	162	49
14986	162	50
14987	162	51
14988	162	52
14989	162	53
14990	162	54
14991	162	55
14992	162	56
14993	162	57
14994	162	58
14995	162	59
14996	162	60
14997	162	61
14998	162	62
14999	162	63
15000	162	64
15001	162	65
15002	162	66
15003	162	67
15004	162	68
15005	162	69
15006	162	70
15007	162	71
15008	162	72
15009	162	73
15010	162	74
15011	162	75
15012	162	76
15013	162	77
15014	162	78
15015	162	79
15016	162	80
15017	162	81
15018	162	82
15019	162	83
15020	162	84
15021	162	85
15022	162	86
15023	162	87
15024	162	88
15025	162	89
15026	162	90
15027	162	91
15028	162	92
15029	162	93
15030	162	94
15031	162	95
15032	162	96
15033	162	97
15034	162	98
15035	162	99
15036	162	100
15037	162	101
15038	162	102
15039	162	103
15040	162	104
15041	162	105
15042	162	106
15043	162	107
15044	162	108
15045	162	109
15046	162	110
15047	162	111
15048	162	112
15049	162	113
15050	162	114
15051	162	115
15052	162	116
15053	162	117
15054	162	118
15055	162	119
15056	162	120
15057	162	121
15058	162	122
15059	162	123
15060	162	124
15061	162	125
15062	162	126
15063	162	127
15064	162	128
15065	162	129
15066	162	130
15067	163	38
15068	163	39
15069	163	40
15070	163	41
15071	163	42
15072	163	43
15073	163	44
15074	163	45
15075	163	46
15076	163	47
15077	163	48
15078	163	49
15079	163	50
15080	163	51
15081	163	52
15082	163	53
15083	163	54
15084	163	55
15085	163	56
15086	163	57
15087	163	58
15088	163	59
15089	163	60
15090	163	61
15091	163	62
15092	163	63
15093	163	64
15094	163	65
15095	163	66
15096	163	67
15097	163	68
15098	163	69
15099	163	70
15100	163	71
15101	163	72
15102	163	73
15103	163	74
15104	163	75
15105	163	76
15106	163	77
15107	163	78
15108	163	79
15109	163	80
15110	163	81
15111	163	82
15112	163	83
15113	163	84
15114	163	85
15115	163	86
15116	163	87
15117	163	88
15118	163	89
15119	163	90
15120	163	91
15121	163	92
15122	163	93
15123	163	94
15124	163	95
15125	163	96
15126	163	97
15127	163	98
15128	163	99
15129	163	100
15130	163	101
15131	163	102
15132	163	103
15133	163	104
15134	163	105
15135	163	106
15136	163	107
15137	163	108
15138	163	109
15139	163	110
15140	163	111
15141	163	112
15142	163	113
15143	163	114
15144	163	115
15145	163	116
15146	163	117
15147	163	118
15148	163	119
15149	163	120
15150	163	121
15151	163	122
15152	163	123
15153	163	124
15154	163	125
15155	163	126
15156	163	127
15157	163	128
15158	163	129
15159	163	130
15160	164	38
15161	164	39
15162	164	40
15163	164	41
15164	164	42
15165	164	43
15166	164	44
15167	164	45
15168	164	46
15169	164	47
15170	164	48
15171	164	49
15172	164	50
15173	164	51
15174	164	52
15175	164	53
15176	164	54
15177	164	55
15178	164	56
15179	164	57
15180	164	58
15181	164	59
15182	164	60
15183	164	61
15184	164	62
15185	164	63
15186	164	64
15187	164	65
15188	164	66
15189	164	67
15190	164	68
15191	164	69
15192	164	70
15193	164	71
15194	164	72
15195	164	73
15196	164	74
15197	164	75
15198	164	76
15199	164	77
15200	164	78
15201	164	79
15202	164	80
15203	164	81
15204	164	82
15205	164	83
15206	164	84
15207	164	85
15208	164	86
15209	164	87
15210	164	88
15211	164	89
15212	164	90
15213	164	91
15214	164	92
15215	164	93
15216	164	94
15217	164	95
15218	164	96
15219	164	97
15220	164	98
15221	164	99
15222	164	100
15223	164	101
15224	164	102
15225	164	103
15226	164	104
15227	164	105
15228	164	106
15229	164	107
15230	164	108
15231	164	109
15232	164	110
15233	164	111
15234	164	112
15235	164	113
15236	164	114
15237	164	115
15238	164	116
15239	164	117
15240	164	118
15241	164	119
15242	164	120
15243	164	121
15244	164	122
15245	164	123
15246	164	124
15247	164	125
15248	164	126
15249	164	127
15250	164	128
15251	164	129
15252	164	130
15253	165	38
15254	165	39
15255	165	40
15256	165	41
15257	165	42
15258	165	43
15259	165	44
15260	165	45
15261	165	46
15262	165	47
15263	165	48
15264	165	49
15265	165	50
15266	165	51
15267	165	52
15268	165	53
15269	165	54
15270	165	55
15271	165	56
15272	165	57
15273	165	58
15274	165	59
15275	165	60
15276	165	61
15277	165	62
15278	165	63
15279	165	64
15280	165	65
15281	165	66
15282	165	67
15283	165	68
15284	165	69
15285	165	70
15286	165	71
15287	165	72
15288	165	73
15289	165	74
15290	165	75
15291	165	76
15292	165	77
15293	165	78
15294	165	79
15295	165	80
15296	165	81
15297	165	82
15298	165	83
15299	165	84
15300	165	85
15301	165	86
15302	165	87
15303	165	88
15304	165	89
15305	165	90
15306	165	91
15307	165	92
15308	165	93
15309	165	94
15310	165	95
15311	165	96
15312	165	97
15313	165	98
15314	165	99
15315	165	100
15316	165	101
15317	165	102
15318	165	103
15319	165	104
15320	165	105
15321	165	106
15322	165	107
15323	165	108
15324	165	109
15325	165	110
15326	165	111
15327	165	112
15328	165	113
15329	165	114
15330	165	115
15331	165	116
15332	165	117
15333	165	118
15334	165	119
15335	165	120
15336	165	121
15337	165	122
15338	165	123
15339	165	124
15340	165	125
15341	165	126
15342	165	127
15343	165	128
15344	165	129
15345	165	130
15346	166	38
15347	166	39
15348	166	40
15349	166	41
15350	166	42
15351	166	43
15352	166	44
15353	166	45
15354	166	46
15355	166	47
15356	166	48
15357	166	49
15358	166	50
15359	166	51
15360	166	52
15361	166	53
15362	166	54
15363	166	55
15364	166	56
15365	166	57
15366	166	58
15367	166	59
15368	166	60
15369	166	61
15370	166	62
15371	166	63
15372	166	64
15373	166	65
15374	166	66
15375	166	67
15376	166	68
15377	166	69
15378	166	70
15379	166	71
15380	166	72
15381	166	73
15382	166	74
15383	166	75
15384	166	76
15385	166	77
15386	166	78
15387	166	79
15388	166	80
15389	166	81
15390	166	82
15391	166	83
15392	166	84
15393	166	85
15394	166	86
15395	166	87
15396	166	88
15397	166	89
15398	166	90
15399	166	91
15400	166	92
15401	166	93
15402	166	94
15403	166	95
15404	166	96
15405	166	97
15406	166	98
15407	166	99
15408	166	100
15409	166	101
15410	166	102
15411	166	103
15412	166	104
15413	166	105
15414	166	106
15415	166	107
15416	166	108
15417	166	109
15418	166	110
15419	166	111
15420	166	112
15421	166	113
15422	166	114
15423	166	115
15424	166	116
15425	166	117
15426	166	118
15427	166	119
15428	166	120
15429	166	121
15430	166	122
15431	166	123
15432	166	124
15433	166	125
15434	166	126
15435	166	127
15436	166	128
15437	166	129
15438	166	130
15439	167	38
15440	167	39
15441	167	40
15442	167	41
15443	167	42
15444	167	43
15445	167	44
15446	167	45
15447	167	46
15448	167	47
15449	167	48
15450	167	49
15451	167	50
15452	167	51
15453	167	52
15454	167	53
15455	167	54
15456	167	55
15457	167	56
15458	167	57
15459	167	58
15460	167	59
15461	167	60
15462	167	61
15463	167	62
15464	167	63
15465	167	64
15466	167	65
15467	167	66
15468	167	67
15469	167	68
15470	167	69
15471	167	70
15472	167	71
15473	167	72
15474	167	73
15475	167	74
15476	167	75
15477	167	76
15478	167	77
15479	167	78
15480	167	79
15481	167	80
15482	167	81
15483	167	82
15484	167	83
15485	167	84
15486	167	85
15487	167	86
15488	167	87
15489	167	88
15490	167	89
15491	167	90
15492	167	91
15493	167	92
15494	167	93
15495	167	94
15496	167	95
15497	167	96
15498	167	97
15499	167	98
15500	167	99
15501	167	100
15502	167	101
15503	167	102
15504	167	103
15505	167	104
15506	167	105
15507	167	106
15508	167	107
15509	167	108
15510	167	109
15511	167	110
15512	167	111
15513	167	112
15514	167	113
15515	167	114
15516	167	115
15517	167	116
15518	167	117
15519	167	118
15520	167	119
15521	167	120
15522	167	121
15523	167	122
15524	167	123
15525	167	124
15526	167	125
15527	167	126
15528	167	127
15529	167	128
15530	167	129
15531	167	130
15532	168	38
15533	168	39
15534	168	40
15535	168	41
15536	168	42
15537	168	43
15538	168	44
15539	168	45
15540	168	46
15541	168	47
15542	168	48
15543	168	49
15544	168	50
15545	168	51
15546	168	52
15547	168	53
15548	168	54
15549	168	55
15550	168	56
15551	168	57
15552	168	58
15553	168	59
15554	168	60
15555	168	61
15556	168	62
15557	168	63
15558	168	64
15559	168	65
15560	168	66
15561	168	67
15562	168	68
15563	168	69
15564	168	70
15565	168	71
15566	168	72
15567	168	73
15568	168	74
15569	168	75
15570	168	76
15571	168	77
15572	168	78
15573	168	79
15574	168	80
15575	168	81
15576	168	82
15577	168	83
15578	168	84
15579	168	85
15580	168	86
15581	168	87
15582	168	88
15583	168	89
15584	168	90
15585	168	91
15586	168	92
15587	168	93
15588	168	94
15589	168	95
15590	168	96
15591	168	97
15592	168	98
15593	168	99
15594	168	100
15595	168	101
15596	168	102
15597	168	103
15598	168	104
15599	168	105
15600	168	106
15601	168	107
15602	168	108
15603	168	109
15604	168	110
15605	168	111
15606	168	112
15607	168	113
15608	168	114
15609	168	115
15610	168	116
15611	168	117
15612	168	118
15613	168	119
15614	168	120
15615	168	121
15616	168	122
15617	168	123
15618	168	124
15619	168	125
15620	168	126
15621	168	127
15622	168	128
15623	168	129
15624	168	130
15625	169	38
15626	169	39
15627	169	40
15628	169	41
15629	169	42
15630	169	43
15631	169	44
15632	169	45
15633	169	46
15634	169	47
15635	169	48
15636	169	49
15637	169	50
15638	169	51
15639	169	52
15640	169	53
15641	169	54
15642	169	55
15643	169	56
15644	169	57
15645	169	58
15646	169	59
15647	169	60
15648	169	61
15649	169	62
15650	169	63
15651	169	64
15652	169	65
15653	169	66
15654	169	67
15655	169	68
15656	169	69
15657	169	70
15658	169	71
15659	169	72
15660	169	73
15661	169	74
15662	169	75
15663	169	76
15664	169	77
15665	169	78
15666	169	79
15667	169	80
15668	169	81
15669	169	82
15670	169	83
15671	169	84
15672	169	85
15673	169	86
15674	169	87
15675	169	88
15676	169	89
15677	169	90
15678	169	91
15679	169	92
15680	169	93
15681	169	94
15682	169	95
15683	169	96
15684	169	97
15685	169	98
15686	169	99
15687	169	100
15688	169	101
15689	169	102
15690	169	103
15691	169	104
15692	169	105
15693	169	106
15694	169	107
15695	169	108
15696	169	109
15697	169	110
15698	169	111
15699	169	112
15700	169	113
15701	169	114
15702	169	115
15703	169	116
15704	169	117
15705	169	118
15706	169	119
15707	169	120
15708	169	121
15709	169	122
15710	169	123
15711	169	124
15712	169	125
15713	169	126
15714	169	127
15715	169	128
15716	169	129
15717	169	130
15718	170	38
15719	170	39
15720	170	40
15721	170	41
15722	170	42
15723	170	43
15724	170	44
15725	170	45
15726	170	46
15727	170	47
15728	170	48
15729	170	49
15730	170	50
15731	170	51
15732	170	52
15733	170	53
15734	170	54
15735	170	55
15736	170	56
15737	170	57
15738	170	58
15739	170	59
15740	170	60
15741	170	61
15742	170	62
15743	170	63
15744	170	64
15745	170	65
15746	170	66
15747	170	67
15748	170	68
15749	170	69
15750	170	70
15751	170	71
15752	170	72
15753	170	73
15754	170	74
15755	170	75
15756	170	76
15757	170	77
15758	170	78
15759	170	79
15760	170	80
15761	170	81
15762	170	82
15763	170	83
15764	170	84
15765	170	85
15766	170	86
15767	170	87
15768	170	88
15769	170	89
15770	170	90
15771	170	91
15772	170	92
15773	170	93
15774	170	94
15775	170	95
15776	170	96
15777	170	97
15778	170	98
15779	170	99
15780	170	100
15781	170	101
15782	170	102
15783	170	103
15784	170	104
15785	170	105
15786	170	106
15787	170	107
15788	170	108
15789	170	109
15790	170	110
15791	170	111
15792	170	112
15793	170	113
15794	170	114
15795	170	115
15796	170	116
15797	170	117
15798	170	118
15799	170	119
15800	170	120
15801	170	121
15802	170	122
15803	170	123
15804	170	124
15805	170	125
15806	170	126
15807	170	127
15808	170	128
15809	170	129
15810	170	130
15811	171	38
15812	171	39
15813	171	40
15814	171	41
15815	171	42
15816	171	43
15817	171	44
15818	171	45
15819	171	46
15820	171	47
15821	171	48
15822	171	49
15823	171	50
15824	171	51
15825	171	52
15826	171	53
15827	171	54
15828	171	55
15829	171	56
15830	171	57
15831	171	58
15832	171	59
15833	171	60
15834	171	61
15835	171	62
15836	171	63
15837	171	64
15838	171	65
15839	171	66
15840	171	67
15841	171	68
15842	171	69
15843	171	70
15844	171	71
15845	171	72
15846	171	73
15847	171	74
15848	171	75
15849	171	76
15850	171	77
15851	171	78
15852	171	79
15853	171	80
15854	171	81
15855	171	82
15856	171	83
15857	171	84
15858	171	85
15859	171	86
15860	171	87
15861	171	88
15862	171	89
15863	171	90
15864	171	91
15865	171	92
15866	171	93
15867	171	94
15868	171	95
15869	171	96
15870	171	97
15871	171	98
15872	171	99
15873	171	100
15874	171	101
15875	171	102
15876	171	103
15877	171	104
15878	171	105
15879	171	106
15880	171	107
15881	171	108
15882	171	109
15883	171	110
15884	171	111
15885	171	112
15886	171	113
15887	171	114
15888	171	115
15889	171	116
15890	171	117
15891	171	118
15892	171	119
15893	171	120
15894	171	121
15895	171	122
15896	171	123
15897	171	124
15898	171	125
15899	171	126
15900	171	127
15901	171	128
15902	171	129
15903	171	130
15904	172	38
15905	172	39
15906	172	40
15907	172	41
15908	172	42
15909	172	43
15910	172	44
15911	172	45
15912	172	46
15913	172	47
15914	172	48
15915	172	49
15916	172	50
15917	172	51
15918	172	52
15919	172	53
15920	172	54
15921	172	55
15922	172	56
15923	172	57
15924	172	58
15925	172	59
15926	172	60
15927	172	61
15928	172	62
15929	172	63
15930	172	64
15931	172	65
15932	172	66
15933	172	67
15934	172	68
15935	172	69
15936	172	70
15937	172	71
15938	172	72
15939	172	73
15940	172	74
15941	172	75
15942	172	76
15943	172	77
15944	172	78
15945	172	79
15946	172	80
15947	172	81
15948	172	82
15949	172	83
15950	172	84
15951	172	85
15952	172	86
15953	172	87
15954	172	88
15955	172	89
15956	172	90
15957	172	91
15958	172	92
15959	172	93
15960	172	94
15961	172	95
15962	172	96
15963	172	97
15964	172	98
15965	172	99
15966	172	100
15967	172	101
15968	172	102
15969	172	103
15970	172	104
15971	172	105
15972	172	106
15973	172	107
15974	172	108
15975	172	109
15976	172	110
15977	172	111
15978	172	112
15979	172	113
15980	172	114
15981	172	115
15982	172	116
15983	172	117
15984	172	118
15985	172	119
15986	172	120
15987	172	121
15988	172	122
15989	172	123
15990	172	124
15991	172	125
15992	172	126
15993	172	127
15994	172	128
15995	172	129
15996	172	130
15997	173	38
15998	173	39
15999	173	40
16000	173	41
16001	173	42
16002	173	43
16003	173	44
16004	173	45
16005	173	46
16006	173	47
16007	173	48
16008	173	49
16009	173	50
16010	173	51
16011	173	52
16012	173	53
16013	173	54
16014	173	55
16015	173	56
16016	173	57
16017	173	58
16018	173	59
16019	173	60
16020	173	61
16021	173	62
16022	173	63
16023	173	64
16024	173	65
16025	173	66
16026	173	67
16027	173	68
16028	173	69
16029	173	70
16030	173	71
16031	173	72
16032	173	73
16033	173	74
16034	173	75
16035	173	76
16036	173	77
16037	173	78
16038	173	79
16039	173	80
16040	173	81
16041	173	82
16042	173	83
16043	173	84
16044	173	85
16045	173	86
16046	173	87
16047	173	88
16048	173	89
16049	173	90
16050	173	91
16051	173	92
16052	173	93
16053	173	94
16054	173	95
16055	173	96
16056	173	97
16057	173	98
16058	173	99
16059	173	100
16060	173	101
16061	173	102
16062	173	103
16063	173	104
16064	173	105
16065	173	106
16066	173	107
16067	173	108
16068	173	109
16069	173	110
16070	173	111
16071	173	112
16072	173	113
16073	173	114
16074	173	115
16075	173	116
16076	173	117
16077	173	118
16078	173	119
16079	173	120
16080	173	121
16081	173	122
16082	173	123
16083	173	124
16084	173	125
16085	173	126
16086	173	127
16087	173	128
16088	173	129
16089	173	130
16090	174	38
16091	174	39
16092	174	40
16093	174	41
16094	174	42
16095	174	43
16096	174	44
16097	174	45
16098	174	46
16099	174	47
16100	174	48
16101	174	49
16102	174	50
16103	174	51
16104	174	52
16105	174	53
16106	174	54
16107	174	55
16108	174	56
16109	174	57
16110	174	58
16111	174	59
16112	174	60
16113	174	61
16114	174	62
16115	174	63
16116	174	64
16117	174	65
16118	174	66
16119	174	67
16120	174	68
16121	174	69
16122	174	70
16123	174	71
16124	174	72
16125	174	73
16126	174	74
16127	174	75
16128	174	76
16129	174	77
16130	174	78
16131	174	79
16132	174	80
16133	174	81
16134	174	82
16135	174	83
16136	174	84
16137	174	85
16138	174	86
16139	174	87
16140	174	88
16141	174	89
16142	174	90
16143	174	91
16144	174	92
16145	174	93
16146	174	94
16147	174	95
16148	174	96
16149	174	97
16150	174	98
16151	174	99
16152	174	100
16153	174	101
16154	174	102
16155	174	103
16156	174	104
16157	174	105
16158	174	106
16159	174	107
16160	174	108
16161	174	109
16162	174	110
16163	174	111
16164	174	112
16165	174	113
16166	174	114
16167	174	115
16168	174	116
16169	174	117
16170	174	118
16171	174	119
16172	174	120
16173	174	121
16174	174	122
16175	174	123
16176	174	124
16177	174	125
16178	174	126
16179	174	127
16180	174	128
16181	174	129
16182	174	130
16183	175	38
16184	175	39
16185	175	40
16186	175	41
16187	175	42
16188	175	43
16189	175	44
16190	175	45
16191	175	46
16192	175	47
16193	175	48
16194	175	49
16195	175	50
16196	175	51
16197	175	52
16198	175	53
16199	175	54
16200	175	55
16201	175	56
16202	175	57
16203	175	58
16204	175	59
16205	175	60
16206	175	61
16207	175	62
16208	175	63
16209	175	64
16210	175	65
16211	175	66
16212	175	67
16213	175	68
16214	175	69
16215	175	70
16216	175	71
16217	175	72
16218	175	73
16219	175	74
16220	175	75
16221	175	76
16222	175	77
16223	175	78
16224	175	79
16225	175	80
16226	175	81
16227	175	82
16228	175	83
16229	175	84
16230	175	85
16231	175	86
16232	175	87
16233	175	88
16234	175	89
16235	175	90
16236	175	91
16237	175	92
16238	175	93
16239	175	94
16240	175	95
16241	175	96
16242	175	97
16243	175	98
16244	175	99
16245	175	100
16246	175	101
16247	175	102
16248	175	103
16249	175	104
16250	175	105
16251	175	106
16252	175	107
16253	175	108
16254	175	109
16255	175	110
16256	175	111
16257	175	112
16258	175	113
16259	175	114
16260	175	115
16261	175	116
16262	175	117
16263	175	118
16264	175	119
16265	175	120
16266	175	121
16267	175	122
16268	175	123
16269	175	124
16270	175	125
16271	175	126
16272	175	127
16273	175	128
16274	175	129
16275	175	130
16276	176	38
16277	176	39
16278	176	40
16279	176	41
16280	176	42
16281	176	43
16282	176	44
16283	176	45
16284	176	46
16285	176	47
16286	176	48
16287	176	49
16288	176	50
16289	176	51
16290	176	52
16291	176	53
16292	176	54
16293	176	55
16294	176	56
16295	176	57
16296	176	58
16297	176	59
16298	176	60
16299	176	61
16300	176	62
16301	176	63
16302	176	64
16303	176	65
16304	176	66
16305	176	67
16306	176	68
16307	176	69
16308	176	70
16309	176	71
16310	176	72
16311	176	73
16312	176	74
16313	176	75
16314	176	76
16315	176	77
16316	176	78
16317	176	79
16318	176	80
16319	176	81
16320	176	82
16321	176	83
16322	176	84
16323	176	85
16324	176	86
16325	176	87
16326	176	88
16327	176	89
16328	176	90
16329	176	91
16330	176	92
16331	176	93
16332	176	94
16333	176	95
16334	176	96
16335	176	97
16336	176	98
16337	176	99
16338	176	100
16339	176	101
16340	176	102
16341	176	103
16342	176	104
16343	176	105
16344	176	106
16345	176	107
16346	176	108
16347	176	109
16348	176	110
16349	176	111
16350	176	112
16351	176	113
16352	176	114
16353	176	115
16354	176	116
16355	176	117
16356	176	118
16357	176	119
16358	176	120
16359	176	121
16360	176	122
16361	176	123
16362	176	124
16363	176	125
16364	176	126
16365	176	127
16366	176	128
16367	176	129
16368	176	130
16369	177	38
16370	177	39
16371	177	40
16372	177	41
16373	177	42
16374	177	43
16375	177	44
16376	177	45
16377	177	46
16378	177	47
16379	177	48
16380	177	49
16381	177	50
16382	177	51
16383	177	52
16384	177	53
16385	177	54
16386	177	55
16387	177	56
16388	177	57
16389	177	58
16390	177	59
16391	177	60
16392	177	61
16393	177	62
16394	177	63
16395	177	64
16396	177	65
16397	177	66
16398	177	67
16399	177	68
16400	177	69
16401	177	70
16402	177	71
16403	177	72
16404	177	73
16405	177	74
16406	177	75
16407	177	76
16408	177	77
16409	177	78
16410	177	79
16411	177	80
16412	177	81
16413	177	82
16414	177	83
16415	177	84
16416	177	85
16417	177	86
16418	177	87
16419	177	88
16420	177	89
16421	177	90
16422	177	91
16423	177	92
16424	177	93
16425	177	94
16426	177	95
16427	177	96
16428	177	97
16429	177	98
16430	177	99
16431	177	100
16432	177	101
16433	177	102
16434	177	103
16435	177	104
16436	177	105
16437	177	106
16438	177	107
16439	177	108
16440	177	109
16441	177	110
16442	177	111
16443	177	112
16444	177	113
16445	177	114
16446	177	115
16447	177	116
16448	177	117
16449	177	118
16450	177	119
16451	177	120
16452	177	121
16453	177	122
16454	177	123
16455	177	124
16456	177	125
16457	177	126
16458	177	127
16459	177	128
16460	177	129
16461	177	130
16462	178	38
16463	178	39
16464	178	40
16465	178	41
16466	178	42
16467	178	43
16468	178	44
16469	178	45
16470	178	46
16471	178	47
16472	178	48
16473	178	49
16474	178	50
16475	178	51
16476	178	52
16477	178	53
16478	178	54
16479	178	55
16480	178	56
16481	178	57
16482	178	58
16483	178	59
16484	178	60
16485	178	61
16486	178	62
16487	178	63
16488	178	64
16489	178	65
16490	178	66
16491	178	67
16492	178	68
16493	178	69
16494	178	70
16495	178	71
16496	178	72
16497	178	73
16498	178	74
16499	178	75
16500	178	76
16501	178	77
16502	178	78
16503	178	79
16504	178	80
16505	178	81
16506	178	82
16507	178	83
16508	178	84
16509	178	85
16510	178	86
16511	178	87
16512	178	88
16513	178	89
16514	178	90
16515	178	91
16516	178	92
16517	178	93
16518	178	94
16519	178	95
16520	178	96
16521	178	97
16522	178	98
16523	178	99
16524	178	100
16525	178	101
16526	178	102
16527	178	103
16528	178	104
16529	178	105
16530	178	106
16531	178	107
16532	178	108
16533	178	109
16534	178	110
16535	178	111
16536	178	112
16537	178	113
16538	178	114
16539	178	115
16540	178	116
16541	178	117
16542	178	118
16543	178	119
16544	178	120
16545	178	121
16546	178	122
16547	178	123
16548	178	124
16549	178	125
16550	178	126
16551	178	127
16552	178	128
16553	178	129
16554	178	130
16555	179	38
16556	179	39
16557	179	40
16558	179	41
16559	179	42
16560	179	43
16561	179	44
16562	179	45
16563	179	46
16564	179	47
16565	179	48
16566	179	49
16567	179	50
16568	179	51
16569	179	52
16570	179	53
16571	179	54
16572	179	55
16573	179	56
16574	179	57
16575	179	58
16576	179	59
16577	179	60
16578	179	61
16579	179	62
16580	179	63
16581	179	64
16582	179	65
16583	179	66
16584	179	67
16585	179	68
16586	179	69
16587	179	70
16588	179	71
16589	179	72
16590	179	73
16591	179	74
16592	179	75
16593	179	76
16594	179	77
16595	179	78
16596	179	79
16597	179	80
16598	179	81
16599	179	82
16600	179	83
16601	179	84
16602	179	85
16603	179	86
16604	179	87
16605	179	88
16606	179	89
16607	179	90
16608	179	91
16609	179	92
16610	179	93
16611	179	94
16612	179	95
16613	179	96
16614	179	97
16615	179	98
16616	179	99
16617	179	100
16618	179	101
16619	179	102
16620	179	103
16621	179	104
16622	179	105
16623	179	106
16624	179	107
16625	179	108
16626	179	109
16627	179	110
16628	179	111
16629	179	112
16630	179	113
16631	179	114
16632	179	115
16633	179	116
16634	179	117
16635	179	118
16636	179	119
16637	179	120
16638	179	121
16639	179	122
16640	179	123
16641	179	124
16642	179	125
16643	179	126
16644	179	127
16645	179	128
16646	179	129
16647	179	130
16648	180	38
16649	180	39
16650	180	40
16651	180	41
16652	180	42
16653	180	43
16654	180	44
16655	180	45
16656	180	46
16657	180	47
16658	180	48
16659	180	49
16660	180	50
16661	180	51
16662	180	52
16663	180	53
16664	180	54
16665	180	55
16666	180	56
16667	180	57
16668	180	58
16669	180	59
16670	180	60
16671	180	61
16672	180	62
16673	180	63
16674	180	64
16675	180	65
16676	180	66
16677	180	67
16678	180	68
16679	180	69
16680	180	70
16681	180	71
16682	180	72
16683	180	73
16684	180	74
16685	180	75
16686	180	76
16687	180	77
16688	180	78
16689	180	79
16690	180	80
16691	180	81
16692	180	82
16693	180	83
16694	180	84
16695	180	85
16696	180	86
16697	180	87
16698	180	88
16699	180	89
16700	180	90
16701	180	91
16702	180	92
16703	180	93
16704	180	94
16705	180	95
16706	180	96
16707	180	97
16708	180	98
16709	180	99
16710	180	100
16711	180	101
16712	180	102
16713	180	103
16714	180	104
16715	180	105
16716	180	106
16717	180	107
16718	180	108
16719	180	109
16720	180	110
16721	180	111
16722	180	112
16723	180	113
16724	180	114
16725	180	115
16726	180	116
16727	180	117
16728	180	118
16729	180	119
16730	180	120
16731	180	121
16732	180	122
16733	180	123
16734	180	124
16735	180	125
16736	180	126
16737	180	127
16738	180	128
16739	180	129
16740	180	130
16741	181	38
16742	181	39
16743	181	40
16744	181	41
16745	181	42
16746	181	43
16747	181	44
16748	181	45
16749	181	46
16750	181	47
16751	181	48
16752	181	49
16753	181	50
16754	181	51
16755	181	52
16756	181	53
16757	181	54
16758	181	55
16759	181	56
16760	181	57
16761	181	58
16762	181	59
16763	181	60
16764	181	61
16765	181	62
16766	181	63
16767	181	64
16768	181	65
16769	181	66
16770	181	67
16771	181	68
16772	181	69
16773	181	70
16774	181	71
16775	181	72
16776	181	73
16777	181	74
16778	181	75
16779	181	76
16780	181	77
16781	181	78
16782	181	79
16783	181	80
16784	181	81
16785	181	82
16786	181	83
16787	181	84
16788	181	85
16789	181	86
16790	181	87
16791	181	88
16792	181	89
16793	181	90
16794	181	91
16795	181	92
16796	181	93
16797	181	94
16798	181	95
16799	181	96
16800	181	97
16801	181	98
16802	181	99
16803	181	100
16804	181	101
16805	181	102
16806	181	103
16807	181	104
16808	181	105
16809	181	106
16810	181	107
16811	181	108
16812	181	109
16813	181	110
16814	181	111
16815	181	112
16816	181	113
16817	181	114
16818	181	115
16819	181	116
16820	181	117
16821	181	118
16822	181	119
16823	181	120
16824	181	121
16825	181	122
16826	181	123
16827	181	124
16828	181	125
16829	181	126
16830	181	127
16831	181	128
16832	181	129
16833	181	130
16834	182	38
16835	182	39
16836	182	40
16837	182	41
16838	182	42
16839	182	43
16840	182	44
16841	182	45
16842	182	46
16843	182	47
16844	182	48
16845	182	49
16846	182	50
16847	182	51
16848	182	52
16849	182	53
16850	182	54
16851	182	55
16852	182	56
16853	182	57
16854	182	58
16855	182	59
16856	182	60
16857	182	61
16858	182	62
16859	182	63
16860	182	64
16861	182	65
16862	182	66
16863	182	67
16864	182	68
16865	182	69
16866	182	70
16867	182	71
16868	182	72
16869	182	73
16870	182	74
16871	182	75
16872	182	76
16873	182	77
16874	182	78
16875	182	79
16876	182	80
16877	182	81
16878	182	82
16879	182	83
16880	182	84
16881	182	85
16882	182	86
16883	182	87
16884	182	88
16885	182	89
16886	182	90
16887	182	91
16888	182	92
16889	182	93
16890	182	94
16891	182	95
16892	182	96
16893	182	97
16894	182	98
16895	182	99
16896	182	100
16897	182	101
16898	182	102
16899	182	103
16900	182	104
16901	182	105
16902	182	106
16903	182	107
16904	182	108
16905	182	109
16906	182	110
16907	182	111
16908	182	112
16909	182	113
16910	182	114
16911	182	115
16912	182	116
16913	182	117
16914	182	118
16915	182	119
16916	182	120
16917	182	121
16918	182	122
16919	182	123
16920	182	124
16921	182	125
16922	182	126
16923	182	127
16924	182	128
16925	182	129
16926	182	130
16927	183	38
16928	183	39
16929	183	40
16930	183	41
16931	183	42
16932	183	43
16933	183	44
16934	183	45
16935	183	46
16936	183	47
16937	183	48
16938	183	49
16939	183	50
16940	183	51
16941	183	52
16942	183	53
16943	183	54
16944	183	55
16945	183	56
16946	183	57
16947	183	58
16948	183	59
16949	183	60
16950	183	61
16951	183	62
16952	183	63
16953	183	64
16954	183	65
16955	183	66
16956	183	67
16957	183	68
16958	183	69
16959	183	70
16960	183	71
16961	183	72
16962	183	73
16963	183	74
16964	183	75
16965	183	76
16966	183	77
16967	183	78
16968	183	79
16969	183	80
16970	183	81
16971	183	82
16972	183	83
16973	183	84
16974	183	85
16975	183	86
16976	183	87
16977	183	88
16978	183	89
16979	183	90
16980	183	91
16981	183	92
16982	183	93
16983	183	94
16984	183	95
16985	183	96
16986	183	97
16987	183	98
16988	183	99
16989	183	100
16990	183	101
16991	183	102
16992	183	103
16993	183	104
16994	183	105
16995	183	106
16996	183	107
16997	183	108
16998	183	109
16999	183	110
17000	183	111
17001	183	112
17002	183	113
17003	183	114
17004	183	115
17005	183	116
17006	183	117
17007	183	118
17008	183	119
17009	183	120
17010	183	121
17011	183	122
17012	183	123
17013	183	124
17014	183	125
17015	183	126
17016	183	127
17017	183	128
17018	183	129
17019	183	130
17020	184	38
17021	184	39
17022	184	40
17023	184	41
17024	184	42
17025	184	43
17026	184	44
17027	184	45
17028	184	46
17029	184	47
17030	184	48
17031	184	49
17032	184	50
17033	184	51
17034	184	52
17035	184	53
17036	184	54
17037	184	55
17038	184	56
17039	184	57
17040	184	58
17041	184	59
17042	184	60
17043	184	61
17044	184	62
17045	184	63
17046	184	64
17047	184	65
17048	184	66
17049	184	67
17050	184	68
17051	184	69
17052	184	70
17053	184	71
17054	184	72
17055	184	73
17056	184	74
17057	184	75
17058	184	76
17059	184	77
17060	184	78
17061	184	79
17062	184	80
17063	184	81
17064	184	82
17065	184	83
17066	184	84
17067	184	85
17068	184	86
17069	184	87
17070	184	88
17071	184	89
17072	184	90
17073	184	91
17074	184	92
17075	184	93
17076	184	94
17077	184	95
17078	184	96
17079	184	97
17080	184	98
17081	184	99
17082	184	100
17083	184	101
17084	184	102
17085	184	103
17086	184	104
17087	184	105
17088	184	106
17089	184	107
17090	184	108
17091	184	109
17092	184	110
17093	184	111
17094	184	112
17095	184	113
17096	184	114
17097	184	115
17098	184	116
17099	184	117
17100	184	118
17101	184	119
17102	184	120
17103	184	121
17104	184	122
17105	184	123
17106	184	124
17107	184	125
17108	184	126
17109	184	127
17110	184	128
17111	184	129
17112	184	130
17113	185	38
17114	185	39
17115	185	40
17116	185	41
17117	185	42
17118	185	43
17119	185	44
17120	185	45
17121	185	46
17122	185	47
17123	185	48
17124	185	49
17125	185	50
17126	185	51
17127	185	52
17128	185	53
17129	185	54
17130	185	55
17131	185	56
17132	185	57
17133	185	58
17134	185	59
17135	185	60
17136	185	61
17137	185	62
17138	185	63
17139	185	64
17140	185	65
17141	185	66
17142	185	67
17143	185	68
17144	185	69
17145	185	70
17146	185	71
17147	185	72
17148	185	73
17149	185	74
17150	185	75
17151	185	76
17152	185	77
17153	185	78
17154	185	79
17155	185	80
17156	185	81
17157	185	82
17158	185	83
17159	185	84
17160	185	85
17161	185	86
17162	185	87
17163	185	88
17164	185	89
17165	185	90
17166	185	91
17167	185	92
17168	185	93
17169	185	94
17170	185	95
17171	185	96
17172	185	97
17173	185	98
17174	185	99
17175	185	100
17176	185	101
17177	185	102
17178	185	103
17179	185	104
17180	185	105
17181	185	106
17182	185	107
17183	185	108
17184	185	109
17185	185	110
17186	185	111
17187	185	112
17188	185	113
17189	185	114
17190	185	115
17191	185	116
17192	185	117
17193	185	118
17194	185	119
17195	185	120
17196	185	121
17197	185	122
17198	185	123
17199	185	124
17200	185	125
17201	185	126
17202	185	127
17203	185	128
17204	185	129
17205	185	130
17206	186	38
17207	186	39
17208	186	40
17209	186	41
17210	186	42
17211	186	43
17212	186	44
17213	186	45
17214	186	46
17215	186	47
17216	186	48
17217	186	49
17218	186	50
17219	186	51
17220	186	52
17221	186	53
17222	186	54
17223	186	55
17224	186	56
17225	186	57
17226	186	58
17227	186	59
17228	186	60
17229	186	61
17230	186	62
17231	186	63
17232	186	64
17233	186	65
17234	186	66
17235	186	67
17236	186	68
17237	186	69
17238	186	70
17239	186	71
17240	186	72
17241	186	73
17242	186	74
17243	186	75
17244	186	76
17245	186	77
17246	186	78
17247	186	79
17248	186	80
17249	186	81
17250	186	82
17251	186	83
17252	186	84
17253	186	85
17254	186	86
17255	186	87
17256	186	88
17257	186	89
17258	186	90
17259	186	91
17260	186	92
17261	186	93
17262	186	94
17263	186	95
17264	186	96
17265	186	97
17266	186	98
17267	186	99
17268	186	100
17269	186	101
17270	186	102
17271	186	103
17272	186	104
17273	186	105
17274	186	106
17275	186	107
17276	186	108
17277	186	109
17278	186	110
17279	186	111
17280	186	112
17281	186	113
17282	186	114
17283	186	115
17284	186	116
17285	186	117
17286	186	118
17287	186	119
17288	186	120
17289	186	121
17290	186	122
17291	186	123
17292	186	124
17293	186	125
17294	186	126
17295	186	127
17296	186	128
17297	186	129
17298	186	130
17299	187	38
17300	187	39
17301	187	40
17302	187	41
17303	187	42
17304	187	43
17305	187	44
17306	187	45
17307	187	46
17308	187	47
17309	187	48
17310	187	49
17311	187	50
17312	187	51
17313	187	52
17314	187	53
17315	187	54
17316	187	55
17317	187	56
17318	187	57
17319	187	58
17320	187	59
17321	187	60
17322	187	61
17323	187	62
17324	187	63
17325	187	64
17326	187	65
17327	187	66
17328	187	67
17329	187	68
17330	187	69
17331	187	70
17332	187	71
17333	187	72
17334	187	73
17335	187	74
17336	187	75
17337	187	76
17338	187	77
17339	187	78
17340	187	79
17341	187	80
17342	187	81
17343	187	82
17344	187	83
17345	187	84
17346	187	85
17347	187	86
17348	187	87
17349	187	88
17350	187	89
17351	187	90
17352	187	91
17353	187	92
17354	187	93
17355	187	94
17356	187	95
17357	187	96
17358	187	97
17359	187	98
17360	187	99
17361	187	100
17362	187	101
17363	187	102
17364	187	103
17365	187	104
17366	187	105
17367	187	106
17368	187	107
17369	187	108
17370	187	109
17371	187	110
17372	187	111
17373	187	112
17374	187	113
17375	187	114
17376	187	115
17377	187	116
17378	187	117
17379	187	118
17380	187	119
17381	187	120
17382	187	121
17383	187	122
17384	187	123
17385	187	124
17386	187	125
17387	187	126
17388	187	127
17389	187	128
17390	187	129
17391	187	130
17392	188	38
17393	188	39
17394	188	40
17395	188	41
17396	188	42
17397	188	43
17398	188	44
17399	188	45
17400	188	46
17401	188	47
17402	188	48
17403	188	49
17404	188	50
17405	188	51
17406	188	52
17407	188	53
17408	188	54
17409	188	55
17410	188	56
17411	188	57
17412	188	58
17413	188	59
17414	188	60
17415	188	61
17416	188	62
17417	188	63
17418	188	64
17419	188	65
17420	188	66
17421	188	67
17422	188	68
17423	188	69
17424	188	70
17425	188	71
17426	188	72
17427	188	73
17428	188	74
17429	188	75
17430	188	76
17431	188	77
17432	188	78
17433	188	79
17434	188	80
17435	188	81
17436	188	82
17437	188	83
17438	188	84
17439	188	85
17440	188	86
17441	188	87
17442	188	88
17443	188	89
17444	188	90
17445	188	91
17446	188	92
17447	188	93
17448	188	94
17449	188	95
17450	188	96
17451	188	97
17452	188	98
17453	188	99
17454	188	100
17455	188	101
17456	188	102
17457	188	103
17458	188	104
17459	188	105
17460	188	106
17461	188	107
17462	188	108
17463	188	109
17464	188	110
17465	188	111
17466	188	112
17467	188	113
17468	188	114
17469	188	115
17470	188	116
17471	188	117
17472	188	118
17473	188	119
17474	188	120
17475	188	121
17476	188	122
17477	188	123
17478	188	124
17479	188	125
17480	188	126
17481	188	127
17482	188	128
17483	188	129
17484	188	130
17485	189	38
17486	189	39
17487	189	40
17488	189	41
17489	189	42
17490	189	43
17491	189	44
17492	189	45
17493	189	46
17494	189	47
17495	189	48
17496	189	49
17497	189	50
17498	189	51
17499	189	52
17500	189	53
17501	189	54
17502	189	55
17503	189	56
17504	189	57
17505	189	58
17506	189	59
17507	189	60
17508	189	61
17509	189	62
17510	189	63
17511	189	64
17512	189	65
17513	189	66
17514	189	67
17515	189	68
17516	189	69
17517	189	70
17518	189	71
17519	189	72
17520	189	73
17521	189	74
17522	189	75
17523	189	76
17524	189	77
17525	189	78
17526	189	79
17527	189	80
17528	189	81
17529	189	82
17530	189	83
17531	189	84
17532	189	85
17533	189	86
17534	189	87
17535	189	88
17536	189	89
17537	189	90
17538	189	91
17539	189	92
17540	189	93
17541	189	94
17542	189	95
17543	189	96
17544	189	97
17545	189	98
17546	189	99
17547	189	100
17548	189	101
17549	189	102
17550	189	103
17551	189	104
17552	189	105
17553	189	106
17554	189	107
17555	189	108
17556	189	109
17557	189	110
17558	189	111
17559	189	112
17560	189	113
17561	189	114
17562	189	115
17563	189	116
17564	189	117
17565	189	118
17566	189	119
17567	189	120
17568	189	121
17569	189	122
17570	189	123
17571	189	124
17572	189	125
17573	189	126
17574	189	127
17575	189	128
17576	189	129
17577	189	130
17578	190	38
17579	190	39
17580	190	40
17581	190	41
17582	190	42
17583	190	43
17584	190	44
17585	190	45
17586	190	46
17587	190	47
17588	190	48
17589	190	49
17590	190	50
17591	190	51
17592	190	52
17593	190	53
17594	190	54
17595	190	55
17596	190	56
17597	190	57
17598	190	58
17599	190	59
17600	190	60
17601	190	61
17602	190	62
17603	190	63
17604	190	64
17605	190	65
17606	190	66
17607	190	67
17608	190	68
17609	190	69
17610	190	70
17611	190	71
17612	190	72
17613	190	73
17614	190	74
17615	190	75
17616	190	76
17617	190	77
17618	190	78
17619	190	79
17620	190	80
17621	190	81
17622	190	82
17623	190	83
17624	190	84
17625	190	85
17626	190	86
17627	190	87
17628	190	88
17629	190	89
17630	190	90
17631	190	91
17632	190	92
17633	190	93
17634	190	94
17635	190	95
17636	190	96
17637	190	97
17638	190	98
17639	190	99
17640	190	100
17641	190	101
17642	190	102
17643	190	103
17644	190	104
17645	190	105
17646	190	106
17647	190	107
17648	190	108
17649	190	109
17650	190	110
17651	190	111
17652	190	112
17653	190	113
17654	190	114
17655	190	115
17656	190	116
17657	190	117
17658	190	118
17659	190	119
17660	190	120
17661	190	121
17662	190	122
17663	190	123
17664	190	124
17665	190	125
17666	190	126
17667	190	127
17668	190	128
17669	190	129
17670	190	130
17671	191	38
17672	191	39
17673	191	40
17674	191	41
17675	191	42
17676	191	43
17677	191	44
17678	191	45
17679	191	46
17680	191	47
17681	191	48
17682	191	49
17683	191	50
17684	191	51
17685	191	52
17686	191	53
17687	191	54
17688	191	55
17689	191	56
17690	191	57
17691	191	58
17692	191	59
17693	191	60
17694	191	61
17695	191	62
17696	191	63
17697	191	64
17698	191	65
17699	191	66
17700	191	67
17701	191	68
17702	191	69
17703	191	70
17704	191	71
17705	191	72
17706	191	73
17707	191	74
17708	191	75
17709	191	76
17710	191	77
17711	191	78
17712	191	79
17713	191	80
17714	191	81
17715	191	82
17716	191	83
17717	191	84
17718	191	85
17719	191	86
17720	191	87
17721	191	88
17722	191	89
17723	191	90
17724	191	91
17725	191	92
17726	191	93
17727	191	94
17728	191	95
17729	191	96
17730	191	97
17731	191	98
17732	191	99
17733	191	100
17734	191	101
17735	191	102
17736	191	103
17737	191	104
17738	191	105
17739	191	106
17740	191	107
17741	191	108
17742	191	109
17743	191	110
17744	191	111
17745	191	112
17746	191	113
17747	191	114
17748	191	115
17749	191	116
17750	191	117
17751	191	118
17752	191	119
17753	191	120
17754	191	121
17755	191	122
17756	191	123
17757	191	124
17758	191	125
17759	191	126
17760	191	127
17761	191	128
17762	191	129
17763	191	130
17764	192	38
17765	192	39
17766	192	40
17767	192	41
17768	192	42
17769	192	43
17770	192	44
17771	192	45
17772	192	46
17773	192	47
17774	192	48
17775	192	49
17776	192	50
17777	192	51
17778	192	52
17779	192	53
17780	192	54
17781	192	55
17782	192	56
17783	192	57
17784	192	58
17785	192	59
17786	192	60
17787	192	61
17788	192	62
17789	192	63
17790	192	64
17791	192	65
17792	192	66
17793	192	67
17794	192	68
17795	192	69
17796	192	70
17797	192	71
17798	192	72
17799	192	73
17800	192	74
17801	192	75
17802	192	76
17803	192	77
17804	192	78
17805	192	79
17806	192	80
17807	192	81
17808	192	82
17809	192	83
17810	192	84
17811	192	85
17812	192	86
17813	192	87
17814	192	88
17815	192	89
17816	192	90
17817	192	91
17818	192	92
17819	192	93
17820	192	94
17821	192	95
17822	192	96
17823	192	97
17824	192	98
17825	192	99
17826	192	100
17827	192	101
17828	192	102
17829	192	103
17830	192	104
17831	192	105
17832	192	106
17833	192	107
17834	192	108
17835	192	109
17836	192	110
17837	192	111
17838	192	112
17839	192	113
17840	192	114
17841	192	115
17842	192	116
17843	192	117
17844	192	118
17845	192	119
17846	192	120
17847	192	121
17848	192	122
17849	192	123
17850	192	124
17851	192	125
17852	192	126
17853	192	127
17854	192	128
17855	192	129
17856	192	130
17857	193	38
17858	193	39
17859	193	40
17860	193	41
17861	193	42
17862	193	43
17863	193	44
17864	193	45
17865	193	46
17866	193	47
17867	193	48
17868	193	49
17869	193	50
17870	193	51
17871	193	52
17872	193	53
17873	193	54
17874	193	55
17875	193	56
17876	193	57
17877	193	58
17878	193	59
17879	193	60
17880	193	61
17881	193	62
17882	193	63
17883	193	64
17884	193	65
17885	193	66
17886	193	67
17887	193	68
17888	193	69
17889	193	70
17890	193	71
17891	193	72
17892	193	73
17893	193	74
17894	193	75
17895	193	76
17896	193	77
17897	193	78
17898	193	79
17899	193	80
17900	193	81
17901	193	82
17902	193	83
17903	193	84
17904	193	85
17905	193	86
17906	193	87
17907	193	88
17908	193	89
17909	193	90
17910	193	91
17911	193	92
17912	193	93
17913	193	94
17914	193	95
17915	193	96
17916	193	97
17917	193	98
17918	193	99
17919	193	100
17920	193	101
17921	193	102
17922	193	103
17923	193	104
17924	193	105
17925	193	106
17926	193	107
17927	193	108
17928	193	109
17929	193	110
17930	193	111
17931	193	112
17932	193	113
17933	193	114
17934	193	115
17935	193	116
17936	193	117
17937	193	118
17938	193	119
17939	193	120
17940	193	121
17941	193	122
17942	193	123
17943	193	124
17944	193	125
17945	193	126
17946	193	127
17947	193	128
17948	193	129
17949	193	130
17950	194	38
17951	194	39
17952	194	40
17953	194	41
17954	194	42
17955	194	43
17956	194	44
17957	194	45
17958	194	46
17959	194	47
17960	194	48
17961	194	49
17962	194	50
17963	194	51
17964	194	52
17965	194	53
17966	194	54
17967	194	55
17968	194	56
17969	194	57
17970	194	58
17971	194	59
17972	194	60
17973	194	61
17974	194	62
17975	194	63
17976	194	64
17977	194	65
17978	194	66
17979	194	67
17980	194	68
17981	194	69
17982	194	70
17983	194	71
17984	194	72
17985	194	73
17986	194	74
17987	194	75
17988	194	76
17989	194	77
17990	194	78
17991	194	79
17992	194	80
17993	194	81
17994	194	82
17995	194	83
17996	194	84
17997	194	85
17998	194	86
17999	194	87
18000	194	88
18001	194	89
18002	194	90
18003	194	91
18004	194	92
18005	194	93
18006	194	94
18007	194	95
18008	194	96
18009	194	97
18010	194	98
18011	194	99
18012	194	100
18013	194	101
18014	194	102
18015	194	103
18016	194	104
18017	194	105
18018	194	106
18019	194	107
18020	194	108
18021	194	109
18022	194	110
18023	194	111
18024	194	112
18025	194	113
18026	194	114
18027	194	115
18028	194	116
18029	194	117
18030	194	118
18031	194	119
18032	194	120
18033	194	121
18034	194	122
18035	194	123
18036	194	124
18037	194	125
18038	194	126
18039	194	127
18040	194	128
18041	194	129
18042	194	130
18043	195	38
18044	195	39
18045	195	40
18046	195	41
18047	195	42
18048	195	43
18049	195	44
18050	195	45
18051	195	46
18052	195	47
18053	195	48
18054	195	49
18055	195	50
18056	195	51
18057	195	52
18058	195	53
18059	195	54
18060	195	55
18061	195	56
18062	195	57
18063	195	58
18064	195	59
18065	195	60
18066	195	61
18067	195	62
18068	195	63
18069	195	64
18070	195	65
18071	195	66
18072	195	67
18073	195	68
18074	195	69
18075	195	70
18076	195	71
18077	195	72
18078	195	73
18079	195	74
18080	195	75
18081	195	76
18082	195	77
18083	195	78
18084	195	79
18085	195	80
18086	195	81
18087	195	82
18088	195	83
18089	195	84
18090	195	85
18091	195	86
18092	195	87
18093	195	88
18094	195	89
18095	195	90
18096	195	91
18097	195	92
18098	195	93
18099	195	94
18100	195	95
18101	195	96
18102	195	97
18103	195	98
18104	195	99
18105	195	100
18106	195	101
18107	195	102
18108	195	103
18109	195	104
18110	195	105
18111	195	106
18112	195	107
18113	195	108
18114	195	109
18115	195	110
18116	195	111
18117	195	112
18118	195	113
18119	195	114
18120	195	115
18121	195	116
18122	195	117
18123	195	118
18124	195	119
18125	195	120
18126	195	121
18127	195	122
18128	195	123
18129	195	124
18130	195	125
18131	195	126
18132	195	127
18133	195	128
18134	195	129
18135	195	130
18136	196	38
18137	196	39
18138	196	40
18139	196	41
18140	196	42
18141	196	43
18142	196	44
18143	196	45
18144	196	46
18145	196	47
18146	196	48
18147	196	49
18148	196	50
18149	196	51
18150	196	52
18151	196	53
18152	196	54
18153	196	55
18154	196	56
18155	196	57
18156	196	58
18157	196	59
18158	196	60
18159	196	61
18160	196	62
18161	196	63
18162	196	64
18163	196	65
18164	196	66
18165	196	67
18166	196	68
18167	196	69
18168	196	70
18169	196	71
18170	196	72
18171	196	73
18172	196	74
18173	196	75
18174	196	76
18175	196	77
18176	196	78
18177	196	79
18178	196	80
18179	196	81
18180	196	82
18181	196	83
18182	196	84
18183	196	85
18184	196	86
18185	196	87
18186	196	88
18187	196	89
18188	196	90
18189	196	91
18190	196	92
18191	196	93
18192	196	94
18193	196	95
18194	196	96
18195	196	97
18196	196	98
18197	196	99
18198	196	100
18199	196	101
18200	196	102
18201	196	103
18202	196	104
18203	196	105
18204	196	106
18205	196	107
18206	196	108
18207	196	109
18208	196	110
18209	196	111
18210	196	112
18211	196	113
18212	196	114
18213	196	115
18214	196	116
18215	196	117
18216	196	118
18217	196	119
18218	196	120
18219	196	121
18220	196	122
18221	196	123
18222	196	124
18223	196	125
18224	196	126
18225	196	127
18226	196	128
18227	196	129
18228	196	130
18229	197	38
18230	197	39
18231	197	40
18232	197	41
18233	197	42
18234	197	43
18235	197	44
18236	197	45
18237	197	46
18238	197	47
18239	197	48
18240	197	49
18241	197	50
18242	197	51
18243	197	52
18244	197	53
18245	197	54
18246	197	55
18247	197	56
18248	197	57
18249	197	58
18250	197	59
18251	197	60
18252	197	61
18253	197	62
18254	197	63
18255	197	64
18256	197	65
18257	197	66
18258	197	67
18259	197	68
18260	197	69
18261	197	70
18262	197	71
18263	197	72
18264	197	73
18265	197	74
18266	197	75
18267	197	76
18268	197	77
18269	197	78
18270	197	79
18271	197	80
18272	197	81
18273	197	82
18274	197	83
18275	197	84
18276	197	85
18277	197	86
18278	197	87
18279	197	88
18280	197	89
18281	197	90
18282	197	91
18283	197	92
18284	197	93
18285	197	94
18286	197	95
18287	197	96
18288	197	97
18289	197	98
18290	197	99
18291	197	100
18292	197	101
18293	197	102
18294	197	103
18295	197	104
18296	197	105
18297	197	106
18298	197	107
18299	197	108
18300	197	109
18301	197	110
18302	197	111
18303	197	112
18304	197	113
18305	197	114
18306	197	115
18307	197	116
18308	197	117
18309	197	118
18310	197	119
18311	197	120
18312	197	121
18313	197	122
18314	197	123
18315	197	124
18316	197	125
18317	197	126
18318	197	127
18319	197	128
18320	197	129
18321	197	130
18322	198	38
18323	198	39
18324	198	40
18325	198	41
18326	198	42
18327	198	43
18328	198	44
18329	198	45
18330	198	46
18331	198	47
18332	198	48
18333	198	49
18334	198	50
18335	198	51
18336	198	52
18337	198	53
18338	198	54
18339	198	55
18340	198	56
18341	198	57
18342	198	58
18343	198	59
18344	198	60
18345	198	61
18346	198	62
18347	198	63
18348	198	64
18349	198	65
18350	198	66
18351	198	67
18352	198	68
18353	198	69
18354	198	70
18355	198	71
18356	198	72
18357	198	73
18358	198	74
18359	198	75
18360	198	76
18361	198	77
18362	198	78
18363	198	79
18364	198	80
18365	198	81
18366	198	82
18367	198	83
18368	198	84
18369	198	85
18370	198	86
18371	198	87
18372	198	88
18373	198	89
18374	198	90
18375	198	91
18376	198	92
18377	198	93
18378	198	94
18379	198	95
18380	198	96
18381	198	97
18382	198	98
18383	198	99
18384	198	100
18385	198	101
18386	198	102
18387	198	103
18388	198	104
18389	198	105
18390	198	106
18391	198	107
18392	198	108
18393	198	109
18394	198	110
18395	198	111
18396	198	112
18397	198	113
18398	198	114
18399	198	115
18400	198	116
18401	198	117
18402	198	118
18403	198	119
18404	198	120
18405	198	121
18406	198	122
18407	198	123
18408	198	124
18409	198	125
18410	198	126
18411	198	127
18412	198	128
18413	198	129
18414	198	130
18415	199	38
18416	199	39
18417	199	40
18418	199	41
18419	199	42
18420	199	43
18421	199	44
18422	199	45
18423	199	46
18424	199	47
18425	199	48
18426	199	49
18427	199	50
18428	199	51
18429	199	52
18430	199	53
18431	199	54
18432	199	55
18433	199	56
18434	199	57
18435	199	58
18436	199	59
18437	199	60
18438	199	61
18439	199	62
18440	199	63
18441	199	64
18442	199	65
18443	199	66
18444	199	67
18445	199	68
18446	199	69
18447	199	70
18448	199	71
18449	199	72
18450	199	73
18451	199	74
18452	199	75
18453	199	76
18454	199	77
18455	199	78
18456	199	79
18457	199	80
18458	199	81
18459	199	82
18460	199	83
18461	199	84
18462	199	85
18463	199	86
18464	199	87
18465	199	88
18466	199	89
18467	199	90
18468	199	91
18469	199	92
18470	199	93
18471	199	94
18472	199	95
18473	199	96
18474	199	97
18475	199	98
18476	199	99
18477	199	100
18478	199	101
18479	199	102
18480	199	103
18481	199	104
18482	199	105
18483	199	106
18484	199	107
18485	199	108
18486	199	109
18487	199	110
18488	199	111
18489	199	112
18490	199	113
18491	199	114
18492	199	115
18493	199	116
18494	199	117
18495	199	118
18496	199	119
18497	199	120
18498	199	121
18499	199	122
18500	199	123
18501	199	124
18502	199	125
18503	199	126
18504	199	127
18505	199	128
18506	199	129
18507	199	130
18508	200	38
18509	200	39
18510	200	40
18511	200	41
18512	200	42
18513	200	43
18514	200	44
18515	200	45
18516	200	46
18517	200	47
18518	200	48
18519	200	49
18520	200	50
18521	200	51
18522	200	52
18523	200	53
18524	200	54
18525	200	55
18526	200	56
18527	200	57
18528	200	58
18529	200	59
18530	200	60
18531	200	61
18532	200	62
18533	200	63
18534	200	64
18535	200	65
18536	200	66
18537	200	67
18538	200	68
18539	200	69
18540	200	70
18541	200	71
18542	200	72
18543	200	73
18544	200	74
18545	200	75
18546	200	76
18547	200	77
18548	200	78
18549	200	79
18550	200	80
18551	200	81
18552	200	82
18553	200	83
18554	200	84
18555	200	85
18556	200	86
18557	200	87
18558	200	88
18559	200	89
18560	200	90
18561	200	91
18562	200	92
18563	200	93
18564	200	94
18565	200	95
18566	200	96
18567	200	97
18568	200	98
18569	200	99
18570	200	100
18571	200	101
18572	200	102
18573	200	103
18574	200	104
18575	200	105
18576	200	106
18577	200	107
18578	200	108
18579	200	109
18580	200	110
18581	200	111
18582	200	112
18583	200	113
18584	200	114
18585	200	115
18586	200	116
18587	200	117
18588	200	118
18589	200	119
18590	200	120
18591	200	121
18592	200	122
18593	200	123
18594	200	124
18595	200	125
18596	200	126
18597	200	127
18598	200	128
18599	200	129
18600	200	130
18601	201	38
18602	201	39
18603	201	40
18604	201	41
18605	201	42
18606	201	43
18607	201	44
18608	201	45
18609	201	46
18610	201	47
18611	201	48
18612	201	49
18613	201	50
18614	201	51
18615	201	52
18616	201	53
18617	201	54
18618	201	55
18619	201	56
18620	201	57
18621	201	58
18622	201	59
18623	201	60
18624	201	61
18625	201	62
18626	201	63
18627	201	64
18628	201	65
18629	201	66
18630	201	67
18631	201	68
18632	201	69
18633	201	70
18634	201	71
18635	201	72
18636	201	73
18637	201	74
18638	201	75
18639	201	76
18640	201	77
18641	201	78
18642	201	79
18643	201	80
18644	201	81
18645	201	82
18646	201	83
18647	201	84
18648	201	85
18649	201	86
18650	201	87
18651	201	88
18652	201	89
18653	201	90
18654	201	91
18655	201	92
18656	201	93
18657	201	94
18658	201	95
18659	201	96
18660	201	97
18661	201	98
18662	201	99
18663	201	100
18664	201	101
18665	201	102
18666	201	103
18667	201	104
18668	201	105
18669	201	106
18670	201	107
18671	201	108
18672	201	109
18673	201	110
18674	201	111
18675	201	112
18676	201	113
18677	201	114
18678	201	115
18679	201	116
18680	201	117
18681	201	118
18682	201	119
18683	201	120
18684	201	121
18685	201	122
18686	201	123
18687	201	124
18688	201	125
18689	201	126
18690	201	127
18691	201	128
18692	201	129
18693	201	130
18694	202	38
18695	202	39
18696	202	40
18697	202	41
18698	202	42
18699	202	43
18700	202	44
18701	202	45
18702	202	46
18703	202	47
18704	202	48
18705	202	49
18706	202	50
18707	202	51
18708	202	52
18709	202	53
18710	202	54
18711	202	55
18712	202	56
18713	202	57
18714	202	58
18715	202	59
18716	202	60
18717	202	61
18718	202	62
18719	202	63
18720	202	64
18721	202	65
18722	202	66
18723	202	67
18724	202	68
18725	202	69
18726	202	70
18727	202	71
18728	202	72
18729	202	73
18730	202	74
18731	202	75
18732	202	76
18733	202	77
18734	202	78
18735	202	79
18736	202	80
18737	202	81
18738	202	82
18739	202	83
18740	202	84
18741	202	85
18742	202	86
18743	202	87
18744	202	88
18745	202	89
18746	202	90
18747	202	91
18748	202	92
18749	202	93
18750	202	94
18751	202	95
18752	202	96
18753	202	97
18754	202	98
18755	202	99
18756	202	100
18757	202	101
18758	202	102
18759	202	103
18760	202	104
18761	202	105
18762	202	106
18763	202	107
18764	202	108
18765	202	109
18766	202	110
18767	202	111
18768	202	112
18769	202	113
18770	202	114
18771	202	115
18772	202	116
18773	202	117
18774	202	118
18775	202	119
18776	202	120
18777	202	121
18778	202	122
18779	202	123
18780	202	124
18781	202	125
18782	202	126
18783	202	127
18784	202	128
18785	202	129
18786	202	130
18787	203	38
18788	203	39
18789	203	40
18790	203	41
18791	203	42
18792	203	43
18793	203	44
18794	203	45
18795	203	46
18796	203	47
18797	203	48
18798	203	49
18799	203	50
18800	203	51
18801	203	52
18802	203	53
18803	203	54
18804	203	55
18805	203	56
18806	203	57
18807	203	58
18808	203	59
18809	203	60
18810	203	61
18811	203	62
18812	203	63
18813	203	64
18814	203	65
18815	203	66
18816	203	67
18817	203	68
18818	203	69
18819	203	70
18820	203	71
18821	203	72
18822	203	73
18823	203	74
18824	203	75
18825	203	76
18826	203	77
18827	203	78
18828	203	79
18829	203	80
18830	203	81
18831	203	82
18832	203	83
18833	203	84
18834	203	85
18835	203	86
18836	203	87
18837	203	88
18838	203	89
18839	203	90
18840	203	91
18841	203	92
18842	203	93
18843	203	94
18844	203	95
18845	203	96
18846	203	97
18847	203	98
18848	203	99
18849	203	100
18850	203	101
18851	203	102
18852	203	103
18853	203	104
18854	203	105
18855	203	106
18856	203	107
18857	203	108
18858	203	109
18859	203	110
18860	203	111
18861	203	112
18862	203	113
18863	203	114
18864	203	115
18865	203	116
18866	203	117
18867	203	118
18868	203	119
18869	203	120
18870	203	121
18871	203	122
18872	203	123
18873	203	124
18874	203	125
18875	203	126
18876	203	127
18877	203	128
18878	203	129
18879	203	130
18880	204	38
18881	204	39
18882	204	40
18883	204	41
18884	204	42
18885	204	43
18886	204	44
18887	204	45
18888	204	46
18889	204	47
18890	204	48
18891	204	49
18892	204	50
18893	204	51
18894	204	52
18895	204	53
18896	204	54
18897	204	55
18898	204	56
18899	204	57
18900	204	58
18901	204	59
18902	204	60
18903	204	61
18904	204	62
18905	204	63
18906	204	64
18907	204	65
18908	204	66
18909	204	67
18910	204	68
18911	204	69
18912	204	70
18913	204	71
18914	204	72
18915	204	73
18916	204	74
18917	204	75
18918	204	76
18919	204	77
18920	204	78
18921	204	79
18922	204	80
18923	204	81
18924	204	82
18925	204	83
18926	204	84
18927	204	85
18928	204	86
18929	204	87
18930	204	88
18931	204	89
18932	204	90
18933	204	91
18934	204	92
18935	204	93
18936	204	94
18937	204	95
18938	204	96
18939	204	97
18940	204	98
18941	204	99
18942	204	100
18943	204	101
18944	204	102
18945	204	103
18946	204	104
18947	204	105
18948	204	106
18949	204	107
18950	204	108
18951	204	109
18952	204	110
18953	204	111
18954	204	112
18955	204	113
18956	204	114
18957	204	115
18958	204	116
18959	204	117
18960	204	118
18961	204	119
18962	204	120
18963	204	121
18964	204	122
18965	204	123
18966	204	124
18967	204	125
18968	204	126
18969	204	127
18970	204	128
18971	204	129
18972	204	130
18973	205	38
18974	205	39
18975	205	40
18976	205	41
18977	205	42
18978	205	43
18979	205	44
18980	205	45
18981	205	46
18982	205	47
18983	205	48
18984	205	49
18985	205	50
18986	205	51
18987	205	52
18988	205	53
18989	205	54
18990	205	55
18991	205	56
18992	205	57
18993	205	58
18994	205	59
18995	205	60
18996	205	61
18997	205	62
18998	205	63
18999	205	64
19000	205	65
19001	205	66
19002	205	67
19003	205	68
19004	205	69
19005	205	70
19006	205	71
19007	205	72
19008	205	73
19009	205	74
19010	205	75
19011	205	76
19012	205	77
19013	205	78
19014	205	79
19015	205	80
19016	205	81
19017	205	82
19018	205	83
19019	205	84
19020	205	85
19021	205	86
19022	205	87
19023	205	88
19024	205	89
19025	205	90
19026	205	91
19027	205	92
19028	205	93
19029	205	94
19030	205	95
19031	205	96
19032	205	97
19033	205	98
19034	205	99
19035	205	100
19036	205	101
19037	205	102
19038	205	103
19039	205	104
19040	205	105
19041	205	106
19042	205	107
19043	205	108
19044	205	109
19045	205	110
19046	205	111
19047	205	112
19048	205	113
19049	205	114
19050	205	115
19051	205	116
19052	205	117
19053	205	118
19054	205	119
19055	205	120
19056	205	121
19057	205	122
19058	205	123
19059	205	124
19060	205	125
19061	205	126
19062	205	127
19063	205	128
19064	205	129
19065	205	130
19066	206	38
19067	206	39
19068	206	40
19069	206	41
19070	206	42
19071	206	43
19072	206	44
19073	206	45
19074	206	46
19075	206	47
19076	206	48
19077	206	49
19078	206	50
19079	206	51
19080	206	52
19081	206	53
19082	206	54
19083	206	55
19084	206	56
19085	206	57
19086	206	58
19087	206	59
19088	206	60
19089	206	61
19090	206	62
19091	206	63
19092	206	64
19093	206	65
19094	206	66
19095	206	67
19096	206	68
19097	206	69
19098	206	70
19099	206	71
19100	206	72
19101	206	73
19102	206	74
19103	206	75
19104	206	76
19105	206	77
19106	206	78
19107	206	79
19108	206	80
19109	206	81
19110	206	82
19111	206	83
19112	206	84
19113	206	85
19114	206	86
19115	206	87
19116	206	88
19117	206	89
19118	206	90
19119	206	91
19120	206	92
19121	206	93
19122	206	94
19123	206	95
19124	206	96
19125	206	97
19126	206	98
19127	206	99
19128	206	100
19129	206	101
19130	206	102
19131	206	103
19132	206	104
19133	206	105
19134	206	106
19135	206	107
19136	206	108
19137	206	109
19138	206	110
19139	206	111
19140	206	112
19141	206	113
19142	206	114
19143	206	115
19144	206	116
19145	206	117
19146	206	118
19147	206	119
19148	206	120
19149	206	121
19150	206	122
19151	206	123
19152	206	124
19153	206	125
19154	206	126
19155	206	127
19156	206	128
19157	206	129
19158	206	130
19159	207	38
19160	207	39
19161	207	40
19162	207	41
19163	207	42
19164	207	43
19165	207	44
19166	207	45
19167	207	46
19168	207	47
19169	207	48
19170	207	49
19171	207	50
19172	207	51
19173	207	52
19174	207	53
19175	207	54
19176	207	55
19177	207	56
19178	207	57
19179	207	58
19180	207	59
19181	207	60
19182	207	61
19183	207	62
19184	207	63
19185	207	64
19186	207	65
19187	207	66
19188	207	67
19189	207	68
19190	207	69
19191	207	70
19192	207	71
19193	207	72
19194	207	73
19195	207	74
19196	207	75
19197	207	76
19198	207	77
19199	207	78
19200	207	79
19201	207	80
19202	207	81
19203	207	82
19204	207	83
19205	207	84
19206	207	85
19207	207	86
19208	207	87
19209	207	88
19210	207	89
19211	207	90
19212	207	91
19213	207	92
19214	207	93
19215	207	94
19216	207	95
19217	207	96
19218	207	97
19219	207	98
19220	207	99
19221	207	100
19222	207	101
19223	207	102
19224	207	103
19225	207	104
19226	207	105
19227	207	106
19228	207	107
19229	207	108
19230	207	109
19231	207	110
19232	207	111
19233	207	112
19234	207	113
19235	207	114
19236	207	115
19237	207	116
19238	207	117
19239	207	118
19240	207	119
19241	207	120
19242	207	121
19243	207	122
19244	207	123
19245	207	124
19246	207	125
19247	207	126
19248	207	127
19249	207	128
19250	207	129
19251	207	130
19252	208	38
19253	208	39
19254	208	40
19255	208	41
19256	208	42
19257	208	43
19258	208	44
19259	208	45
19260	208	46
19261	208	47
19262	208	48
19263	208	49
19264	208	50
19265	208	51
19266	208	52
19267	208	53
19268	208	54
19269	208	55
19270	208	56
19271	208	57
19272	208	58
19273	208	59
19274	208	60
19275	208	61
19276	208	62
19277	208	63
19278	208	64
19279	208	65
19280	208	66
19281	208	67
19282	208	68
19283	208	69
19284	208	70
19285	208	71
19286	208	72
19287	208	73
19288	208	74
19289	208	75
19290	208	76
19291	208	77
19292	208	78
19293	208	79
19294	208	80
19295	208	81
19296	208	82
19297	208	83
19298	208	84
19299	208	85
19300	208	86
19301	208	87
19302	208	88
19303	208	89
19304	208	90
19305	208	91
19306	208	92
19307	208	93
19308	208	94
19309	208	95
19310	208	96
19311	208	97
19312	208	98
19313	208	99
19314	208	100
19315	208	101
19316	208	102
19317	208	103
19318	208	104
19319	208	105
19320	208	106
19321	208	107
19322	208	108
19323	208	109
19324	208	110
19325	208	111
19326	208	112
19327	208	113
19328	208	114
19329	208	115
19330	208	116
19331	208	117
19332	208	118
19333	208	119
19334	208	120
19335	208	121
19336	208	122
19337	208	123
19338	208	124
19339	208	125
19340	208	126
19341	208	127
19342	208	128
19343	208	129
19344	208	130
19345	209	38
19346	209	39
19347	209	40
19348	209	41
19349	209	42
19350	209	43
19351	209	44
19352	209	45
19353	209	46
19354	209	47
19355	209	48
19356	209	49
19357	209	50
19358	209	51
19359	209	52
19360	209	53
19361	209	54
19362	209	55
19363	209	56
19364	209	57
19365	209	58
19366	209	59
19367	209	60
19368	209	61
19369	209	62
19370	209	63
19371	209	64
19372	209	65
19373	209	66
19374	209	67
19375	209	68
19376	209	69
19377	209	70
19378	209	71
19379	209	72
19380	209	73
19381	209	74
19382	209	75
19383	209	76
19384	209	77
19385	209	78
19386	209	79
19387	209	80
19388	209	81
19389	209	82
19390	209	83
19391	209	84
19392	209	85
19393	209	86
19394	209	87
19395	209	88
19396	209	89
19397	209	90
19398	209	91
19399	209	92
19400	209	93
19401	209	94
19402	209	95
19403	209	96
19404	209	97
19405	209	98
19406	209	99
19407	209	100
19408	209	101
19409	209	102
19410	209	103
19411	209	104
19412	209	105
19413	209	106
19414	209	107
19415	209	108
19416	209	109
19417	209	110
19418	209	111
19419	209	112
19420	209	113
19421	209	114
19422	209	115
19423	209	116
19424	209	117
19425	209	118
19426	209	119
19427	209	120
19428	209	121
19429	209	122
19430	209	123
19431	209	124
19432	209	125
19433	209	126
19434	209	127
19435	209	128
19436	209	129
19437	209	130
19438	210	38
19439	210	39
19440	210	40
19441	210	41
19442	210	42
19443	210	43
19444	210	44
19445	210	45
19446	210	46
19447	210	47
19448	210	48
19449	210	49
19450	210	50
19451	210	51
19452	210	52
19453	210	53
19454	210	54
19455	210	55
19456	210	56
19457	210	57
19458	210	58
19459	210	59
19460	210	60
19461	210	61
19462	210	62
19463	210	63
19464	210	64
19465	210	65
19466	210	66
19467	210	67
19468	210	68
19469	210	69
19470	210	70
19471	210	71
19472	210	72
19473	210	73
19474	210	74
19475	210	75
19476	210	76
19477	210	77
19478	210	78
19479	210	79
19480	210	80
19481	210	81
19482	210	82
19483	210	83
19484	210	84
19485	210	85
19486	210	86
19487	210	87
19488	210	88
19489	210	89
19490	210	90
19491	210	91
19492	210	92
19493	210	93
19494	210	94
19495	210	95
19496	210	96
19497	210	97
19498	210	98
19499	210	99
19500	210	100
19501	210	101
19502	210	102
19503	210	103
19504	210	104
19505	210	105
19506	210	106
19507	210	107
19508	210	108
19509	210	109
19510	210	110
19511	210	111
19512	210	112
19513	210	113
19514	210	114
19515	210	115
19516	210	116
19517	210	117
19518	210	118
19519	210	119
19520	210	120
19521	210	121
19522	210	122
19523	210	123
19524	210	124
19525	210	125
19526	210	126
19527	210	127
19528	210	128
19529	210	129
19530	210	130
19531	211	38
19532	211	39
19533	211	40
19534	211	41
19535	211	42
19536	211	43
19537	211	44
19538	211	45
19539	211	46
19540	211	47
19541	211	48
19542	211	49
19543	211	50
19544	211	51
19545	211	52
19546	211	53
19547	211	54
19548	211	55
19549	211	56
19550	211	57
19551	211	58
19552	211	59
19553	211	60
19554	211	61
19555	211	62
19556	211	63
19557	211	64
19558	211	65
19559	211	66
19560	211	67
19561	211	68
19562	211	69
19563	211	70
19564	211	71
19565	211	72
19566	211	73
19567	211	74
19568	211	75
19569	211	76
19570	211	77
19571	211	78
19572	211	79
19573	211	80
19574	211	81
19575	211	82
19576	211	83
19577	211	84
19578	211	85
19579	211	86
19580	211	87
19581	211	88
19582	211	89
19583	211	90
19584	211	91
19585	211	92
19586	211	93
19587	211	94
19588	211	95
19589	211	96
19590	211	97
19591	211	98
19592	211	99
19593	211	100
19594	211	101
19595	211	102
19596	211	103
19597	211	104
19598	211	105
19599	211	106
19600	211	107
19601	211	108
19602	211	109
19603	211	110
19604	211	111
19605	211	112
19606	211	113
19607	211	114
19608	211	115
19609	211	116
19610	211	117
19611	211	118
19612	211	119
19613	211	120
19614	211	121
19615	211	122
19616	211	123
19617	211	124
19618	211	125
19619	211	126
19620	211	127
19621	211	128
19622	211	129
19623	211	130
19624	212	38
19625	212	39
19626	212	40
19627	212	41
19628	212	42
19629	212	43
19630	212	44
19631	212	45
19632	212	46
19633	212	47
19634	212	48
19635	212	49
19636	212	50
19637	212	51
19638	212	52
19639	212	53
19640	212	54
19641	212	55
19642	212	56
19643	212	57
19644	212	58
19645	212	59
19646	212	60
19647	212	61
19648	212	62
19649	212	63
19650	212	64
19651	212	65
19652	212	66
19653	212	67
19654	212	68
19655	212	69
19656	212	70
19657	212	71
19658	212	72
19659	212	73
19660	212	74
19661	212	75
19662	212	76
19663	212	77
19664	212	78
19665	212	79
19666	212	80
19667	212	81
19668	212	82
19669	212	83
19670	212	84
19671	212	85
19672	212	86
19673	212	87
19674	212	88
19675	212	89
19676	212	90
19677	212	91
19678	212	92
19679	212	93
19680	212	94
19681	212	95
19682	212	96
19683	212	97
19684	212	98
19685	212	99
19686	212	100
19687	212	101
19688	212	102
19689	212	103
19690	212	104
19691	212	105
19692	212	106
19693	212	107
19694	212	108
19695	212	109
19696	212	110
19697	212	111
19698	212	112
19699	212	113
19700	212	114
19701	212	115
19702	212	116
19703	212	117
19704	212	118
19705	212	119
19706	212	120
19707	212	121
19708	212	122
19709	212	123
19710	212	124
19711	212	125
19712	212	126
19713	212	127
19714	212	128
19715	212	129
19716	212	130
19717	213	38
19718	213	39
19719	213	40
19720	213	41
19721	213	42
19722	213	43
19723	213	44
19724	213	45
19725	213	46
19726	213	47
19727	213	48
19728	213	49
19729	213	50
19730	213	51
19731	213	52
19732	213	53
19733	213	54
19734	213	55
19735	213	56
19736	213	57
19737	213	58
19738	213	59
19739	213	60
19740	213	61
19741	213	62
19742	213	63
19743	213	64
19744	213	65
19745	213	66
19746	213	67
19747	213	68
19748	213	69
19749	213	70
19750	213	71
19751	213	72
19752	213	73
19753	213	74
19754	213	75
19755	213	76
19756	213	77
19757	213	78
19758	213	79
19759	213	80
19760	213	81
19761	213	82
19762	213	83
19763	213	84
19764	213	85
19765	213	86
19766	213	87
19767	213	88
19768	213	89
19769	213	90
19770	213	91
19771	213	92
19772	213	93
19773	213	94
19774	213	95
19775	213	96
19776	213	97
19777	213	98
19778	213	99
19779	213	100
19780	213	101
19781	213	102
19782	213	103
19783	213	104
19784	213	105
19785	213	106
19786	213	107
19787	213	108
19788	213	109
19789	213	110
19790	213	111
19791	213	112
19792	213	113
19793	213	114
19794	213	115
19795	213	116
19796	213	117
19797	213	118
19798	213	119
19799	213	120
19800	213	121
19801	213	122
19802	213	123
19803	213	124
19804	213	125
19805	213	126
19806	213	127
19807	213	128
19808	213	129
19809	213	130
19810	214	38
19811	214	39
19812	214	40
19813	214	41
19814	214	42
19815	214	43
19816	214	44
19817	214	45
19818	214	46
19819	214	47
19820	214	48
19821	214	49
19822	214	50
19823	214	51
19824	214	52
19825	214	53
19826	214	54
19827	214	55
19828	214	56
19829	214	57
19830	214	58
19831	214	59
19832	214	60
19833	214	61
19834	214	62
19835	214	63
19836	214	64
19837	214	65
19838	214	66
19839	214	67
19840	214	68
19841	214	69
19842	214	70
19843	214	71
19844	214	72
19845	214	73
19846	214	74
19847	214	75
19848	214	76
19849	214	77
19850	214	78
19851	214	79
19852	214	80
19853	214	81
19854	214	82
19855	214	83
19856	214	84
19857	214	85
19858	214	86
19859	214	87
19860	214	88
19861	214	89
19862	214	90
19863	214	91
19864	214	92
19865	214	93
19866	214	94
19867	214	95
19868	214	96
19869	214	97
19870	214	98
19871	214	99
19872	214	100
19873	214	101
19874	214	102
19875	214	103
19876	214	104
19877	214	105
19878	214	106
19879	214	107
19880	214	108
19881	214	109
19882	214	110
19883	214	111
19884	214	112
19885	214	113
19886	214	114
19887	214	115
19888	214	116
19889	214	117
19890	214	118
19891	214	119
19892	214	120
19893	214	121
19894	214	122
19895	214	123
19896	214	124
19897	214	125
19898	214	126
19899	214	127
19900	214	128
19901	214	129
19902	214	130
19903	215	38
19904	215	39
19905	215	40
19906	215	41
19907	215	42
19908	215	43
19909	215	44
19910	215	45
19911	215	46
19912	215	47
19913	215	48
19914	215	49
19915	215	50
19916	215	51
19917	215	52
19918	215	53
19919	215	54
19920	215	55
19921	215	56
19922	215	57
19923	215	58
19924	215	59
19925	215	60
19926	215	61
19927	215	62
19928	215	63
19929	215	64
19930	215	65
19931	215	66
19932	215	67
19933	215	68
19934	215	69
19935	215	70
19936	215	71
19937	215	72
19938	215	73
19939	215	74
19940	215	75
19941	215	76
19942	215	77
19943	215	78
19944	215	79
19945	215	80
19946	215	81
19947	215	82
19948	215	83
19949	215	84
19950	215	85
19951	215	86
19952	215	87
19953	215	88
19954	215	89
19955	215	90
19956	215	91
19957	215	92
19958	215	93
19959	215	94
19960	215	95
19961	215	96
19962	215	97
19963	215	98
19964	215	99
19965	215	100
19966	215	101
19967	215	102
19968	215	103
19969	215	104
19970	215	105
19971	215	106
19972	215	107
19973	215	108
19974	215	109
19975	215	110
19976	215	111
19977	215	112
19978	215	113
19979	215	114
19980	215	115
19981	215	116
19982	215	117
19983	215	118
19984	215	119
19985	215	120
19986	215	121
19987	215	122
19988	215	123
19989	215	124
19990	215	125
19991	215	126
19992	215	127
19993	215	128
19994	215	129
19995	215	130
19996	216	38
19997	216	39
19998	216	40
19999	216	41
20000	216	42
20001	216	43
20002	216	44
20003	216	45
20004	216	46
20005	216	47
20006	216	48
20007	216	49
20008	216	50
20009	216	51
20010	216	52
20011	216	53
20012	216	54
20013	216	55
20014	216	56
20015	216	57
20016	216	58
20017	216	59
20018	216	60
20019	216	61
20020	216	62
20021	216	63
20022	216	64
20023	216	65
20024	216	66
20025	216	67
20026	216	68
20027	216	69
20028	216	70
20029	216	71
20030	216	72
20031	216	73
20032	216	74
20033	216	75
20034	216	76
20035	216	77
20036	216	78
20037	216	79
20038	216	80
20039	216	81
20040	216	82
20041	216	83
20042	216	84
20043	216	85
20044	216	86
20045	216	87
20046	216	88
20047	216	89
20048	216	90
20049	216	91
20050	216	92
20051	216	93
20052	216	94
20053	216	95
20054	216	96
20055	216	97
20056	216	98
20057	216	99
20058	216	100
20059	216	101
20060	216	102
20061	216	103
20062	216	104
20063	216	105
20064	216	106
20065	216	107
20066	216	108
20067	216	109
20068	216	110
20069	216	111
20070	216	112
20071	216	113
20072	216	114
20073	216	115
20074	216	116
20075	216	117
20076	216	118
20077	216	119
20078	216	120
20079	216	121
20080	216	122
20081	216	123
20082	216	124
20083	216	125
20084	216	126
20085	216	127
20086	216	128
20087	216	129
20088	216	130
20089	217	38
20090	217	39
20091	217	40
20092	217	41
20093	217	42
20094	217	43
20095	217	44
20096	217	45
20097	217	46
20098	217	47
20099	217	48
20100	217	49
20101	217	50
20102	217	51
20103	217	52
20104	217	53
20105	217	54
20106	217	55
20107	217	56
20108	217	57
20109	217	58
20110	217	59
20111	217	60
20112	217	61
20113	217	62
20114	217	63
20115	217	64
20116	217	65
20117	217	66
20118	217	67
20119	217	68
20120	217	69
20121	217	70
20122	217	71
20123	217	72
20124	217	73
20125	217	74
20126	217	75
20127	217	76
20128	217	77
20129	217	78
20130	217	79
20131	217	80
20132	217	81
20133	217	82
20134	217	83
20135	217	84
20136	217	85
20137	217	86
20138	217	87
20139	217	88
20140	217	89
20141	217	90
20142	217	91
20143	217	92
20144	217	93
20145	217	94
20146	217	95
20147	217	96
20148	217	97
20149	217	98
20150	217	99
20151	217	100
20152	217	101
20153	217	102
20154	217	103
20155	217	104
20156	217	105
20157	217	106
20158	217	107
20159	217	108
20160	217	109
20161	217	110
20162	217	111
20163	217	112
20164	217	113
20165	217	114
20166	217	115
20167	217	116
20168	217	117
20169	217	118
20170	217	119
20171	217	120
20172	217	121
20173	217	122
20174	217	123
20175	217	124
20176	217	125
20177	217	126
20178	217	127
20179	217	128
20180	217	129
20181	217	130
20182	218	38
20183	218	39
20184	218	40
20185	218	41
20186	218	42
20187	218	43
20188	218	44
20189	218	45
20190	218	46
20191	218	47
20192	218	48
20193	218	49
20194	218	50
20195	218	51
20196	218	52
20197	218	53
20198	218	54
20199	218	55
20200	218	56
20201	218	57
20202	218	58
20203	218	59
20204	218	60
20205	218	61
20206	218	62
20207	218	63
20208	218	64
20209	218	65
20210	218	66
20211	218	67
20212	218	68
20213	218	69
20214	218	70
20215	218	71
20216	218	72
20217	218	73
20218	218	74
20219	218	75
20220	218	76
20221	218	77
20222	218	78
20223	218	79
20224	218	80
20225	218	81
20226	218	82
20227	218	83
20228	218	84
20229	218	85
20230	218	86
20231	218	87
20232	218	88
20233	218	89
20234	218	90
20235	218	91
20236	218	92
20237	218	93
20238	218	94
20239	218	95
20240	218	96
20241	218	97
20242	218	98
20243	218	99
20244	218	100
20245	218	101
20246	218	102
20247	218	103
20248	218	104
20249	218	105
20250	218	106
20251	218	107
20252	218	108
20253	218	109
20254	218	110
20255	218	111
20256	218	112
20257	218	113
20258	218	114
20259	218	115
20260	218	116
20261	218	117
20262	218	118
20263	218	119
20264	218	120
20265	218	121
20266	218	122
20267	218	123
20268	218	124
20269	218	125
20270	218	126
20271	218	127
20272	218	128
20273	218	129
20274	218	130
20275	219	38
20276	219	39
20277	219	40
20278	219	41
20279	219	42
20280	219	43
20281	219	44
20282	219	45
20283	219	46
20284	219	47
20285	219	48
20286	219	49
20287	219	50
20288	219	51
20289	219	52
20290	219	53
20291	219	54
20292	219	55
20293	219	56
20294	219	57
20295	219	58
20296	219	59
20297	219	60
20298	219	61
20299	219	62
20300	219	63
20301	219	64
20302	219	65
20303	219	66
20304	219	67
20305	219	68
20306	219	69
20307	219	70
20308	219	71
20309	219	72
20310	219	73
20311	219	74
20312	219	75
20313	219	76
20314	219	77
20315	219	78
20316	219	79
20317	219	80
20318	219	81
20319	219	82
20320	219	83
20321	219	84
20322	219	85
20323	219	86
20324	219	87
20325	219	88
20326	219	89
20327	219	90
20328	219	91
20329	219	92
20330	219	93
20331	219	94
20332	219	95
20333	219	96
20334	219	97
20335	219	98
20336	219	99
20337	219	100
20338	219	101
20339	219	102
20340	219	103
20341	219	104
20342	219	105
20343	219	106
20344	219	107
20345	219	108
20346	219	109
20347	219	110
20348	219	111
20349	219	112
20350	219	113
20351	219	114
20352	219	115
20353	219	116
20354	219	117
20355	219	118
20356	219	119
20357	219	120
20358	219	121
20359	219	122
20360	219	123
20361	219	124
20362	219	125
20363	219	126
20364	219	127
20365	219	128
20366	219	129
20367	219	130
20368	220	38
20369	220	39
20370	220	40
20371	220	41
20372	220	42
20373	220	43
20374	220	44
20375	220	45
20376	220	46
20377	220	47
20378	220	48
20379	220	49
20380	220	50
20381	220	51
20382	220	52
20383	220	53
20384	220	54
20385	220	55
20386	220	56
20387	220	57
20388	220	58
20389	220	59
20390	220	60
20391	220	61
20392	220	62
20393	220	63
20394	220	64
20395	220	65
20396	220	66
20397	220	67
20398	220	68
20399	220	69
20400	220	70
20401	220	71
20402	220	72
20403	220	73
20404	220	74
20405	220	75
20406	220	76
20407	220	77
20408	220	78
20409	220	79
20410	220	80
20411	220	81
20412	220	82
20413	220	83
20414	220	84
20415	220	85
20416	220	86
20417	220	87
20418	220	88
20419	220	89
20420	220	90
20421	220	91
20422	220	92
20423	220	93
20424	220	94
20425	220	95
20426	220	96
20427	220	97
20428	220	98
20429	220	99
20430	220	100
20431	220	101
20432	220	102
20433	220	103
20434	220	104
20435	220	105
20436	220	106
20437	220	107
20438	220	108
20439	220	109
20440	220	110
20441	220	111
20442	220	112
20443	220	113
20444	220	114
20445	220	115
20446	220	116
20447	220	117
20448	220	118
20449	220	119
20450	220	120
20451	220	121
20452	220	122
20453	220	123
20454	220	124
20455	220	125
20456	220	126
20457	220	127
20458	220	128
20459	220	129
20460	220	130
20461	221	38
20462	221	39
20463	221	40
20464	221	41
20465	221	42
20466	221	43
20467	221	44
20468	221	45
20469	221	46
20470	221	47
20471	221	48
20472	221	49
20473	221	50
20474	221	51
20475	221	52
20476	221	53
20477	221	54
20478	221	55
20479	221	56
20480	221	57
20481	221	58
20482	221	59
20483	221	60
20484	221	61
20485	221	62
20486	221	63
20487	221	64
20488	221	65
20489	221	66
20490	221	67
20491	221	68
20492	221	69
20493	221	70
20494	221	71
20495	221	72
20496	221	73
20497	221	74
20498	221	75
20499	221	76
20500	221	77
20501	221	78
20502	221	79
20503	221	80
20504	221	81
20505	221	82
20506	221	83
20507	221	84
20508	221	85
20509	221	86
20510	221	87
20511	221	88
20512	221	89
20513	221	90
20514	221	91
20515	221	92
20516	221	93
20517	221	94
20518	221	95
20519	221	96
20520	221	97
20521	221	98
20522	221	99
20523	221	100
20524	221	101
20525	221	102
20526	221	103
20527	221	104
20528	221	105
20529	221	106
20530	221	107
20531	221	108
20532	221	109
20533	221	110
20534	221	111
20535	221	112
20536	221	113
20537	221	114
20538	221	115
20539	221	116
20540	221	117
20541	221	118
20542	221	119
20543	221	120
20544	221	121
20545	221	122
20546	221	123
20547	221	124
20548	221	125
20549	221	126
20550	221	127
20551	221	128
20552	221	129
20553	221	130
20554	222	38
20555	222	39
20556	222	40
20557	222	41
20558	222	42
20559	222	43
20560	222	44
20561	222	45
20562	222	46
20563	222	47
20564	222	48
20565	222	49
20566	222	50
20567	222	51
20568	222	52
20569	222	53
20570	222	54
20571	222	55
20572	222	56
20573	222	57
20574	222	58
20575	222	59
20576	222	60
20577	222	61
20578	222	62
20579	222	63
20580	222	64
20581	222	65
20582	222	66
20583	222	67
20584	222	68
20585	222	69
20586	222	70
20587	222	71
20588	222	72
20589	222	73
20590	222	74
20591	222	75
20592	222	76
20593	222	77
20594	222	78
20595	222	79
20596	222	80
20597	222	81
20598	222	82
20599	222	83
20600	222	84
20601	222	85
20602	222	86
20603	222	87
20604	222	88
20605	222	89
20606	222	90
20607	222	91
20608	222	92
20609	222	93
20610	222	94
20611	222	95
20612	222	96
20613	222	97
20614	222	98
20615	222	99
20616	222	100
20617	222	101
20618	222	102
20619	222	103
20620	222	104
20621	222	105
20622	222	106
20623	222	107
20624	222	108
20625	222	109
20626	222	110
20627	222	111
20628	222	112
20629	222	113
20630	222	114
20631	222	115
20632	222	116
20633	222	117
20634	222	118
20635	222	119
20636	222	120
20637	222	121
20638	222	122
20639	222	123
20640	222	124
20641	222	125
20642	222	126
20643	222	127
20644	222	128
20645	222	129
20646	222	130
20647	223	38
20648	223	39
20649	223	40
20650	223	41
20651	223	42
20652	223	43
20653	223	44
20654	223	45
20655	223	46
20656	223	47
20657	223	48
20658	223	49
20659	223	50
20660	223	51
20661	223	52
20662	223	53
20663	223	54
20664	223	55
20665	223	56
20666	223	57
20667	223	58
20668	223	59
20669	223	60
20670	223	61
20671	223	62
20672	223	63
20673	223	64
20674	223	65
20675	223	66
20676	223	67
20677	223	68
20678	223	69
20679	223	70
20680	223	71
20681	223	72
20682	223	73
20683	223	74
20684	223	75
20685	223	76
20686	223	77
20687	223	78
20688	223	79
20689	223	80
20690	223	81
20691	223	82
20692	223	83
20693	223	84
20694	223	85
20695	223	86
20696	223	87
20697	223	88
20698	223	89
20699	223	90
20700	223	91
20701	223	92
20702	223	93
20703	223	94
20704	223	95
20705	223	96
20706	223	97
20707	223	98
20708	223	99
20709	223	100
20710	223	101
20711	223	102
20712	223	103
20713	223	104
20714	223	105
20715	223	106
20716	223	107
20717	223	108
20718	223	109
20719	223	110
20720	223	111
20721	223	112
20722	223	113
20723	223	114
20724	223	115
20725	223	116
20726	223	117
20727	223	118
20728	223	119
20729	223	120
20730	223	121
20731	223	122
20732	223	123
20733	223	124
20734	223	125
20735	223	126
20736	223	127
20737	223	128
20738	223	129
20739	223	130
20740	224	38
20741	224	39
20742	224	40
20743	224	41
20744	224	42
20745	224	43
20746	224	44
20747	224	45
20748	224	46
20749	224	47
20750	224	48
20751	224	49
20752	224	50
20753	224	51
20754	224	52
20755	224	53
20756	224	54
20757	224	55
20758	224	56
20759	224	57
20760	224	58
20761	224	59
20762	224	60
20763	224	61
20764	224	62
20765	224	63
20766	224	64
20767	224	65
20768	224	66
20769	224	67
20770	224	68
20771	224	69
20772	224	70
20773	224	71
20774	224	72
20775	224	73
20776	224	74
20777	224	75
20778	224	76
20779	224	77
20780	224	78
20781	224	79
20782	224	80
20783	224	81
20784	224	82
20785	224	83
20786	224	84
20787	224	85
20788	224	86
20789	224	87
20790	224	88
20791	224	89
20792	224	90
20793	224	91
20794	224	92
20795	224	93
20796	224	94
20797	224	95
20798	224	96
20799	224	97
20800	224	98
20801	224	99
20802	224	100
20803	224	101
20804	224	102
20805	224	103
20806	224	104
20807	224	105
20808	224	106
20809	224	107
20810	224	108
20811	224	109
20812	224	110
20813	224	111
20814	224	112
20815	224	113
20816	224	114
20817	224	115
20818	224	116
20819	224	117
20820	224	118
20821	224	119
20822	224	120
20823	224	121
20824	224	122
20825	224	123
20826	224	124
20827	224	125
20828	224	126
20829	224	127
20830	224	128
20831	224	129
20832	224	130
20833	225	38
20834	225	39
20835	225	40
20836	225	41
20837	225	42
20838	225	43
20839	225	44
20840	225	45
20841	225	46
20842	225	47
20843	225	48
20844	225	49
20845	225	50
20846	225	51
20847	225	52
20848	225	53
20849	225	54
20850	225	55
20851	225	56
20852	225	57
20853	225	58
20854	225	59
20855	225	60
20856	225	61
20857	225	62
20858	225	63
20859	225	64
20860	225	65
20861	225	66
20862	225	67
20863	225	68
20864	225	69
20865	225	70
20866	225	71
20867	225	72
20868	225	73
20869	225	74
20870	225	75
20871	225	76
20872	225	77
20873	225	78
20874	225	79
20875	225	80
20876	225	81
20877	225	82
20878	225	83
20879	225	84
20880	225	85
20881	225	86
20882	225	87
20883	225	88
20884	225	89
20885	225	90
20886	225	91
20887	225	92
20888	225	93
20889	225	94
20890	225	95
20891	225	96
20892	225	97
20893	225	98
20894	225	99
20895	225	100
20896	225	101
20897	225	102
20898	225	103
20899	225	104
20900	225	105
20901	225	106
20902	225	107
20903	225	108
20904	225	109
20905	225	110
20906	225	111
20907	225	112
20908	225	113
20909	225	114
20910	225	115
20911	225	116
20912	225	117
20913	225	118
20914	225	119
20915	225	120
20916	225	121
20917	225	122
20918	225	123
20919	225	124
20920	225	125
20921	225	126
20922	225	127
20923	225	128
20924	225	129
20925	225	130
20926	226	38
20927	226	39
20928	226	40
20929	226	41
20930	226	42
20931	226	43
20932	226	44
20933	226	45
20934	226	46
20935	226	47
20936	226	48
20937	226	49
20938	226	50
20939	226	51
20940	226	52
20941	226	53
20942	226	54
20943	226	55
20944	226	56
20945	226	57
20946	226	58
20947	226	59
20948	226	60
20949	226	61
20950	226	62
20951	226	63
20952	226	64
20953	226	65
20954	226	66
20955	226	67
20956	226	68
20957	226	69
20958	226	70
20959	226	71
20960	226	72
20961	226	73
20962	226	74
20963	226	75
20964	226	76
20965	226	77
20966	226	78
20967	226	79
20968	226	80
20969	226	81
20970	226	82
20971	226	83
20972	226	84
20973	226	85
20974	226	86
20975	226	87
20976	226	88
20977	226	89
20978	226	90
20979	226	91
20980	226	92
20981	226	93
20982	226	94
20983	226	95
20984	226	96
20985	226	97
20986	226	98
20987	226	99
20988	226	100
20989	226	101
20990	226	102
20991	226	103
20992	226	104
20993	226	105
20994	226	106
20995	226	107
20996	226	108
20997	226	109
20998	226	110
20999	226	111
21000	226	112
21001	226	113
21002	226	114
21003	226	115
21004	226	116
21005	226	117
21006	226	118
21007	226	119
21008	226	120
21009	226	121
21010	226	122
21011	226	123
21012	226	124
21013	226	125
21014	226	126
21015	226	127
21016	226	128
21017	226	129
21018	226	130
21019	227	38
21020	227	39
21021	227	40
21022	227	41
21023	227	42
21024	227	43
21025	227	44
21026	227	45
21027	227	46
21028	227	47
21029	227	48
21030	227	49
21031	227	50
21032	227	51
21033	227	52
21034	227	53
21035	227	54
21036	227	55
21037	227	56
21038	227	57
21039	227	58
21040	227	59
21041	227	60
21042	227	61
21043	227	62
21044	227	63
21045	227	64
21046	227	65
21047	227	66
21048	227	67
21049	227	68
21050	227	69
21051	227	70
21052	227	71
21053	227	72
21054	227	73
21055	227	74
21056	227	75
21057	227	76
21058	227	77
21059	227	78
21060	227	79
21061	227	80
21062	227	81
21063	227	82
21064	227	83
21065	227	84
21066	227	85
21067	227	86
21068	227	87
21069	227	88
21070	227	89
21071	227	90
21072	227	91
21073	227	92
21074	227	93
21075	227	94
21076	227	95
21077	227	96
21078	227	97
21079	227	98
21080	227	99
21081	227	100
21082	227	101
21083	227	102
21084	227	103
21085	227	104
21086	227	105
21087	227	106
21088	227	107
21089	227	108
21090	227	109
21091	227	110
21092	227	111
21093	227	112
21094	227	113
21095	227	114
21096	227	115
21097	227	116
21098	227	117
21099	227	118
21100	227	119
21101	227	120
21102	227	121
21103	227	122
21104	227	123
21105	227	124
21106	227	125
21107	227	126
21108	227	127
21109	227	128
21110	227	129
21111	227	130
21112	228	38
21113	228	39
21114	228	40
21115	228	41
21116	228	42
21117	228	43
21118	228	44
21119	228	45
21120	228	46
21121	228	47
21122	228	48
21123	228	49
21124	228	50
21125	228	51
21126	228	52
21127	228	53
21128	228	54
21129	228	55
21130	228	56
21131	228	57
21132	228	58
21133	228	59
21134	228	60
21135	228	61
21136	228	62
21137	228	63
21138	228	64
21139	228	65
21140	228	66
21141	228	67
21142	228	68
21143	228	69
21144	228	70
21145	228	71
21146	228	72
21147	228	73
21148	228	74
21149	228	75
21150	228	76
21151	228	77
21152	228	78
21153	228	79
21154	228	80
21155	228	81
21156	228	82
21157	228	83
21158	228	84
21159	228	85
21160	228	86
21161	228	87
21162	228	88
21163	228	89
21164	228	90
21165	228	91
21166	228	92
21167	228	93
21168	228	94
21169	228	95
21170	228	96
21171	228	97
21172	228	98
21173	228	99
21174	228	100
21175	228	101
21176	228	102
21177	228	103
21178	228	104
21179	228	105
21180	228	106
21181	228	107
21182	228	108
21183	228	109
21184	228	110
21185	228	111
21186	228	112
21187	228	113
21188	228	114
21189	228	115
21190	228	116
21191	228	117
21192	228	118
21193	228	119
21194	228	120
21195	228	121
21196	228	122
21197	228	123
21198	228	124
21199	228	125
21200	228	126
21201	228	127
21202	228	128
21203	228	129
21204	228	130
21205	229	38
21206	229	39
21207	229	40
21208	229	41
21209	229	42
21210	229	43
21211	229	44
21212	229	45
21213	229	46
21214	229	47
21215	229	48
21216	229	49
21217	229	50
21218	229	51
21219	229	52
21220	229	53
21221	229	54
21222	229	55
21223	229	56
21224	229	57
21225	229	58
21226	229	59
21227	229	60
21228	229	61
21229	229	62
21230	229	63
21231	229	64
21232	229	65
21233	229	66
21234	229	67
21235	229	68
21236	229	69
21237	229	70
21238	229	71
21239	229	72
21240	229	73
21241	229	74
21242	229	75
21243	229	76
21244	229	77
21245	229	78
21246	229	79
21247	229	80
21248	229	81
21249	229	82
21250	229	83
21251	229	84
21252	229	85
21253	229	86
21254	229	87
21255	229	88
21256	229	89
21257	229	90
21258	229	91
21259	229	92
21260	229	93
21261	229	94
21262	229	95
21263	229	96
21264	229	97
21265	229	98
21266	229	99
21267	229	100
21268	229	101
21269	229	102
21270	229	103
21271	229	104
21272	229	105
21273	229	106
21274	229	107
21275	229	108
21276	229	109
21277	229	110
21278	229	111
21279	229	112
21280	229	113
21281	229	114
21282	229	115
21283	229	116
21284	229	117
21285	229	118
21286	229	119
21287	229	120
21288	229	121
21289	229	122
21290	229	123
21291	229	124
21292	229	125
21293	229	126
21294	229	127
21295	229	128
21296	229	129
21297	229	130
21298	230	38
21299	230	39
21300	230	40
21301	230	41
21302	230	42
21303	230	43
21304	230	44
21305	230	45
21306	230	46
21307	230	47
21308	230	48
21309	230	49
21310	230	50
21311	230	51
21312	230	52
21313	230	53
21314	230	54
21315	230	55
21316	230	56
21317	230	57
21318	230	58
21319	230	59
21320	230	60
21321	230	61
21322	230	62
21323	230	63
21324	230	64
21325	230	65
21326	230	66
21327	230	67
21328	230	68
21329	230	69
21330	230	70
21331	230	71
21332	230	72
21333	230	73
21334	230	74
21335	230	75
21336	230	76
21337	230	77
21338	230	78
21339	230	79
21340	230	80
21341	230	81
21342	230	82
21343	230	83
21344	230	84
21345	230	85
21346	230	86
21347	230	87
21348	230	88
21349	230	89
21350	230	90
21351	230	91
21352	230	92
21353	230	93
21354	230	94
21355	230	95
21356	230	96
21357	230	97
21358	230	98
21359	230	99
21360	230	100
21361	230	101
21362	230	102
21363	230	103
21364	230	104
21365	230	105
21366	230	106
21367	230	107
21368	230	108
21369	230	109
21370	230	110
21371	230	111
21372	230	112
21373	230	113
21374	230	114
21375	230	115
21376	230	116
21377	230	117
21378	230	118
21379	230	119
21380	230	120
21381	230	121
21382	230	122
21383	230	123
21384	230	124
21385	230	125
21386	230	126
21387	230	127
21388	230	128
21389	230	129
21390	230	130
21391	231	38
21392	231	39
21393	231	40
21394	231	41
21395	231	42
21396	231	43
21397	231	44
21398	231	45
21399	231	46
21400	231	47
21401	231	48
21402	231	49
21403	231	50
21404	231	51
21405	231	52
21406	231	53
21407	231	54
21408	231	55
21409	231	56
21410	231	57
21411	231	58
21412	231	59
21413	231	60
21414	231	61
21415	231	62
21416	231	63
21417	231	64
21418	231	65
21419	231	66
21420	231	67
21421	231	68
21422	231	69
21423	231	70
21424	231	71
21425	231	72
21426	231	73
21427	231	74
21428	231	75
21429	231	76
21430	231	77
21431	231	78
21432	231	79
21433	231	80
21434	231	81
21435	231	82
21436	231	83
21437	231	84
21438	231	85
21439	231	86
21440	231	87
21441	231	88
21442	231	89
21443	231	90
21444	231	91
21445	231	92
21446	231	93
21447	231	94
21448	231	95
21449	231	96
21450	231	97
21451	231	98
21452	231	99
21453	231	100
21454	231	101
21455	231	102
21456	231	103
21457	231	104
21458	231	105
21459	231	106
21460	231	107
21461	231	108
21462	231	109
21463	231	110
21464	231	111
21465	231	112
21466	231	113
21467	231	114
21468	231	115
21469	231	116
21470	231	117
21471	231	118
21472	231	119
21473	231	120
21474	231	121
21475	231	122
21476	231	123
21477	231	124
21478	231	125
21479	231	126
21480	231	127
21481	231	128
21482	231	129
21483	231	130
21484	232	38
21485	232	39
21486	232	40
21487	232	41
21488	232	42
21489	232	43
21490	232	44
21491	232	45
21492	232	46
21493	232	47
21494	232	48
21495	232	49
21496	232	50
21497	232	51
21498	232	52
21499	232	53
21500	232	54
21501	232	55
21502	232	56
21503	232	57
21504	232	58
21505	232	59
21506	232	60
21507	232	61
21508	232	62
21509	232	63
21510	232	64
21511	232	65
21512	232	66
21513	232	67
21514	232	68
21515	232	69
21516	232	70
21517	232	71
21518	232	72
21519	232	73
21520	232	74
21521	232	75
21522	232	76
21523	232	77
21524	232	78
21525	232	79
21526	232	80
21527	232	81
21528	232	82
21529	232	83
21530	232	84
21531	232	85
21532	232	86
21533	232	87
21534	232	88
21535	232	89
21536	232	90
21537	232	91
21538	232	92
21539	232	93
21540	232	94
21541	232	95
21542	232	96
21543	232	97
21544	232	98
21545	232	99
21546	232	100
21547	232	101
21548	232	102
21549	232	103
21550	232	104
21551	232	105
21552	232	106
21553	232	107
21554	232	108
21555	232	109
21556	232	110
21557	232	111
21558	232	112
21559	232	113
21560	232	114
21561	232	115
21562	232	116
21563	232	117
21564	232	118
21565	232	119
21566	232	120
21567	232	121
21568	232	122
21569	232	123
21570	232	124
21571	232	125
21572	232	126
21573	232	127
21574	232	128
21575	232	129
21576	232	130
21577	233	38
21578	233	39
21579	233	40
21580	233	41
21581	233	42
21582	233	43
21583	233	44
21584	233	45
21585	233	46
21586	233	47
21587	233	48
21588	233	49
21589	233	50
21590	233	51
21591	233	52
21592	233	53
21593	233	54
21594	233	55
21595	233	56
21596	233	57
21597	233	58
21598	233	59
21599	233	60
21600	233	61
21601	233	62
21602	233	63
21603	233	64
21604	233	65
21605	233	66
21606	233	67
21607	233	68
21608	233	69
21609	233	70
21610	233	71
21611	233	72
21612	233	73
21613	233	74
21614	233	75
21615	233	76
21616	233	77
21617	233	78
21618	233	79
21619	233	80
21620	233	81
21621	233	82
21622	233	83
21623	233	84
21624	233	85
21625	233	86
21626	233	87
21627	233	88
21628	233	89
21629	233	90
21630	233	91
21631	233	92
21632	233	93
21633	233	94
21634	233	95
21635	233	96
21636	233	97
21637	233	98
21638	233	99
21639	233	100
21640	233	101
21641	233	102
21642	233	103
21643	233	104
21644	233	105
21645	233	106
21646	233	107
21647	233	108
21648	233	109
21649	233	110
21650	233	111
21651	233	112
21652	233	113
21653	233	114
21654	233	115
21655	233	116
21656	233	117
21657	233	118
21658	233	119
21659	233	120
21660	233	121
21661	233	122
21662	233	123
21663	233	124
21664	233	125
21665	233	126
21666	233	127
21667	233	128
21668	233	129
21669	233	130
21670	234	38
21671	234	39
21672	234	40
21673	234	41
21674	234	42
21675	234	43
21676	234	44
21677	234	45
21678	234	46
21679	234	47
21680	234	48
21681	234	49
21682	234	50
21683	234	51
21684	234	52
21685	234	53
21686	234	54
21687	234	55
21688	234	56
21689	234	57
21690	234	58
21691	234	59
21692	234	60
21693	234	61
21694	234	62
21695	234	63
21696	234	64
21697	234	65
21698	234	66
21699	234	67
21700	234	68
21701	234	69
21702	234	70
21703	234	71
21704	234	72
21705	234	73
21706	234	74
21707	234	75
21708	234	76
21709	234	77
21710	234	78
21711	234	79
21712	234	80
21713	234	81
21714	234	82
21715	234	83
21716	234	84
21717	234	85
21718	234	86
21719	234	87
21720	234	88
21721	234	89
21722	234	90
21723	234	91
21724	234	92
21725	234	93
21726	234	94
21727	234	95
21728	234	96
21729	234	97
21730	234	98
21731	234	99
21732	234	100
21733	234	101
21734	234	102
21735	234	103
21736	234	104
21737	234	105
21738	234	106
21739	234	107
21740	234	108
21741	234	109
21742	234	110
21743	234	111
21744	234	112
21745	234	113
21746	234	114
21747	234	115
21748	234	116
21749	234	117
21750	234	118
21751	234	119
21752	234	120
21753	234	121
21754	234	122
21755	234	123
21756	234	124
21757	234	125
21758	234	126
21759	234	127
21760	234	128
21761	234	129
21762	234	130
21763	235	38
21764	235	39
21765	235	40
21766	235	41
21767	235	42
21768	235	43
21769	235	44
21770	235	45
21771	235	46
21772	235	47
21773	235	48
21774	235	49
21775	235	50
21776	235	51
21777	235	52
21778	235	53
21779	235	54
21780	235	55
21781	235	56
21782	235	57
21783	235	58
21784	235	59
21785	235	60
21786	235	61
21787	235	62
21788	235	63
21789	235	64
21790	235	65
21791	235	66
21792	235	67
21793	235	68
21794	235	69
21795	235	70
21796	235	71
21797	235	72
21798	235	73
21799	235	74
21800	235	75
21801	235	76
21802	235	77
21803	235	78
21804	235	79
21805	235	80
21806	235	81
21807	235	82
21808	235	83
21809	235	84
21810	235	85
21811	235	86
21812	235	87
21813	235	88
21814	235	89
21815	235	90
21816	235	91
21817	235	92
21818	235	93
21819	235	94
21820	235	95
21821	235	96
21822	235	97
21823	235	98
21824	235	99
21825	235	100
21826	235	101
21827	235	102
21828	235	103
21829	235	104
21830	235	105
21831	235	106
21832	235	107
21833	235	108
21834	235	109
21835	235	110
21836	235	111
21837	235	112
21838	235	113
21839	235	114
21840	235	115
21841	235	116
21842	235	117
21843	235	118
21844	235	119
21845	235	120
21846	235	121
21847	235	122
21848	235	123
21849	235	124
21850	235	125
21851	235	126
21852	235	127
21853	235	128
21854	235	129
21855	235	130
21856	236	38
21857	236	39
21858	236	40
21859	236	41
21860	236	42
21861	236	43
21862	236	44
21863	236	45
21864	236	46
21865	236	47
21866	236	48
21867	236	49
21868	236	50
21869	236	51
21870	236	52
21871	236	53
21872	236	54
21873	236	55
21874	236	56
21875	236	57
21876	236	58
21877	236	59
21878	236	60
21879	236	61
21880	236	62
21881	236	63
21882	236	64
21883	236	65
21884	236	66
21885	236	67
21886	236	68
21887	236	69
21888	236	70
21889	236	71
21890	236	72
21891	236	73
21892	236	74
21893	236	75
21894	236	76
21895	236	77
21896	236	78
21897	236	79
21898	236	80
21899	236	81
21900	236	82
21901	236	83
21902	236	84
21903	236	85
21904	236	86
21905	236	87
21906	236	88
21907	236	89
21908	236	90
21909	236	91
21910	236	92
21911	236	93
21912	236	94
21913	236	95
21914	236	96
21915	236	97
21916	236	98
21917	236	99
21918	236	100
21919	236	101
21920	236	102
21921	236	103
21922	236	104
21923	236	105
21924	236	106
21925	236	107
21926	236	108
21927	236	109
21928	236	110
21929	236	111
21930	236	112
21931	236	113
21932	236	114
21933	236	115
21934	236	116
21935	236	117
21936	236	118
21937	236	119
21938	236	120
21939	236	121
21940	236	122
21941	236	123
21942	236	124
21943	236	125
21944	236	126
21945	236	127
21946	236	128
21947	236	129
21948	236	130
21949	237	38
21950	237	39
21951	237	40
21952	237	41
21953	237	42
21954	237	43
21955	237	44
21956	237	45
21957	237	46
21958	237	47
21959	237	48
21960	237	49
21961	237	50
21962	237	51
21963	237	52
21964	237	53
21965	237	54
21966	237	55
21967	237	56
21968	237	57
21969	237	58
21970	237	59
21971	237	60
21972	237	61
21973	237	62
21974	237	63
21975	237	64
21976	237	65
21977	237	66
21978	237	67
21979	237	68
21980	237	69
21981	237	70
21982	237	71
21983	237	72
21984	237	73
21985	237	74
21986	237	75
21987	237	76
21988	237	77
21989	237	78
21990	237	79
21991	237	80
21992	237	81
21993	237	82
21994	237	83
21995	237	84
21996	237	85
21997	237	86
21998	237	87
21999	237	88
22000	237	89
22001	237	90
22002	237	91
22003	237	92
22004	237	93
22005	237	94
22006	237	95
22007	237	96
22008	237	97
22009	237	98
22010	237	99
22011	237	100
22012	237	101
22013	237	102
22014	237	103
22015	237	104
22016	237	105
22017	237	106
22018	237	107
22019	237	108
22020	237	109
22021	237	110
22022	237	111
22023	237	112
22024	237	113
22025	237	114
22026	237	115
22027	237	116
22028	237	117
22029	237	118
22030	237	119
22031	237	120
22032	237	121
22033	237	122
22034	237	123
22035	237	124
22036	237	125
22037	237	126
22038	237	127
22039	237	128
22040	237	129
22041	237	130
22042	238	38
22043	238	39
22044	238	40
22045	238	41
22046	238	42
22047	238	43
22048	238	44
22049	238	45
22050	238	46
22051	238	47
22052	238	48
22053	238	49
22054	238	50
22055	238	51
22056	238	52
22057	238	53
22058	238	54
22059	238	55
22060	238	56
22061	238	57
22062	238	58
22063	238	59
22064	238	60
22065	238	61
22066	238	62
22067	238	63
22068	238	64
22069	238	65
22070	238	66
22071	238	67
22072	238	68
22073	238	69
22074	238	70
22075	238	71
22076	238	72
22077	238	73
22078	238	74
22079	238	75
22080	238	76
22081	238	77
22082	238	78
22083	238	79
22084	238	80
22085	238	81
22086	238	82
22087	238	83
22088	238	84
22089	238	85
22090	238	86
22091	238	87
22092	238	88
22093	238	89
22094	238	90
22095	238	91
22096	238	92
22097	238	93
22098	238	94
22099	238	95
22100	238	96
22101	238	97
22102	238	98
22103	238	99
22104	238	100
22105	238	101
22106	238	102
22107	238	103
22108	238	104
22109	238	105
22110	238	106
22111	238	107
22112	238	108
22113	238	109
22114	238	110
22115	238	111
22116	238	112
22117	238	113
22118	238	114
22119	238	115
22120	238	116
22121	238	117
22122	238	118
22123	238	119
22124	238	120
22125	238	121
22126	238	122
22127	238	123
22128	238	124
22129	238	125
22130	238	126
22131	238	127
22132	238	128
22133	238	129
22134	238	130
22135	239	38
22136	239	39
22137	239	40
22138	239	41
22139	239	42
22140	239	43
22141	239	44
22142	239	45
22143	239	46
22144	239	47
22145	239	48
22146	239	49
22147	239	50
22148	239	51
22149	239	52
22150	239	53
22151	239	54
22152	239	55
22153	239	56
22154	239	57
22155	239	58
22156	239	59
22157	239	60
22158	239	61
22159	239	62
22160	239	63
22161	239	64
22162	239	65
22163	239	66
22164	239	67
22165	239	68
22166	239	69
22167	239	70
22168	239	71
22169	239	72
22170	239	73
22171	239	74
22172	239	75
22173	239	76
22174	239	77
22175	239	78
22176	239	79
22177	239	80
22178	239	81
22179	239	82
22180	239	83
22181	239	84
22182	239	85
22183	239	86
22184	239	87
22185	239	88
22186	239	89
22187	239	90
22188	239	91
22189	239	92
22190	239	93
22191	239	94
22192	239	95
22193	239	96
22194	239	97
22195	239	98
22196	239	99
22197	239	100
22198	239	101
22199	239	102
22200	239	103
22201	239	104
22202	239	105
22203	239	106
22204	239	107
22205	239	108
22206	239	109
22207	239	110
22208	239	111
22209	239	112
22210	239	113
22211	239	114
22212	239	115
22213	239	116
22214	239	117
22215	239	118
22216	239	119
22217	239	120
22218	239	121
22219	239	122
22220	239	123
22221	239	124
22222	239	125
22223	239	126
22224	239	127
22225	239	128
22226	239	129
22227	239	130
22228	240	38
22229	240	39
22230	240	40
22231	240	41
22232	240	42
22233	240	43
22234	240	44
22235	240	45
22236	240	46
22237	240	47
22238	240	48
22239	240	49
22240	240	50
22241	240	51
22242	240	52
22243	240	53
22244	240	54
22245	240	55
22246	240	56
22247	240	57
22248	240	58
22249	240	59
22250	240	60
22251	240	61
22252	240	62
22253	240	63
22254	240	64
22255	240	65
22256	240	66
22257	240	67
22258	240	68
22259	240	69
22260	240	70
22261	240	71
22262	240	72
22263	240	73
22264	240	74
22265	240	75
22266	240	76
22267	240	77
22268	240	78
22269	240	79
22270	240	80
22271	240	81
22272	240	82
22273	240	83
22274	240	84
22275	240	85
22276	240	86
22277	240	87
22278	240	88
22279	240	89
22280	240	90
22281	240	91
22282	240	92
22283	240	93
22284	240	94
22285	240	95
22286	240	96
22287	240	97
22288	240	98
22289	240	99
22290	240	100
22291	240	101
22292	240	102
22293	240	103
22294	240	104
22295	240	105
22296	240	106
22297	240	107
22298	240	108
22299	240	109
22300	240	110
22301	240	111
22302	240	112
22303	240	113
22304	240	114
22305	240	115
22306	240	116
22307	240	117
22308	240	118
22309	240	119
22310	240	120
22311	240	121
22312	240	122
22313	240	123
22314	240	124
22315	240	125
22316	240	126
22317	240	127
22318	240	128
22319	240	129
22320	240	130
\.


--
-- Data for Name: nd_experiment_phenotypeprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_phenotypeprop (nd_experiment_phenotypeprop_id, nd_experiment_phenotype_id, type_id, value, rank) FROM stdin;
1	1	2314	113	0
2	2	2314	1	0
3	3	2314	0	0
4	4	2314	1	0
5	5	2314	0.5	0
6	6	2314	0.5	0
7	7	2314		0
8	8	2314		0
9	9	2314	2	0
10	10	2314	0	0
11	11	2314	1	0
12	12	2314	0.5	0
13	13	2314	0.8	0
14	14	2314	1	0
15	15	2314	0.6	0
16	16	2314	0.8	0
17	17	2314		0
18	18	2314		0
19	19	2314	2	0
20	20	2314	13	0
21	21	2314	3	0
22	22	2314	1	0
23	23	2314	7	0
24	24	2314	3	0
25	25	2314	2	0
26	26	2314	1	0
27	27	2314	0	0
28	28	2314	0	0
29	29	2314	0	0
30	30	2314	0	0
31	31	2314	0	0
32	32	2314	0	0
33	33	2314	0	0
34	34	2314	0	0
35	35	2314	B0L5	0
36	36	2314	1.2	0
37	37	2314	1.4	0
38	38	2314	27.8	0
39	39	2314	14	0
40	40	2314	39.7	0
41	41	2314	57	0
42	42	2314	21.8	0
43	43	2314	297	0
44	44	2314	3.1	0
45	45	2314	2.7	0
46	46	2314	0	0
47	47	2314	0	0
48	48	2314	0	0
49	49	2314	109	0
50	50	2314	375	0
51	51	2314	375	0
52	52	2314	30	0
53	53	2314	50	0
54	54	2314	95	0
55	55	2314	0	0
56	56	2314	3.048333333	0
57	57	2314	39.58333333	0
58	58	2314	10.53333333	0
59	59	2314	550	0
60	60	2314	650	0
61	61	2314	2	0
62	62	2314	141	0
63	63	2314	57	0
64	64	2314	5.62	0
65	65	2314	0	0
66	66	2314	0	0
67	67	2314	80	0
68	68	2314	15	0
69	69	2314	0	0
70	70	2314	277	0
71	71	2314	137	0
72	72	2314	58	0
73	73	2314	550	0
74	74	2314	650	0
75	75	2314	2	0
76	76	2314	141	0
77	77	2314	57	0
78	78	2314	5.62	0
79	79	2314	0	0
80	80	2314	0	0
81	81	2314	80	0
82	82	2314	15	0
83	83	2314	0	0
84	84	2314	125.7	0
85	85	2314	35.3	0
86	86	2314	273.8	0
87	87	2314	74.4	0
88	88	2314	85.4	0
89	89	2314	22.4	0
90	90	2314	32.27	0
91	91	2314	5.63	0
92	92	2314	32.3053	0
93	93	2314	5.7268	0
94	94	2314	.	0
95	95	2314	.	0
96	96	2314	0	0
97	97	2314	0	0
98	98	2314		0
99	99	2314		0
100	100	2314		0
101	101	2314		0
102	102	2314	0	0
103	103	2314	0	0
104	104	2314		0
105	105	2314		0
106	106	2314		0
107	107	2314		0
108	108	2314		0
109	109	2314		0
110	110	2314		0
111	111	2314		0
112	112	2314	0	0
113	113	2314	0	0
114	114	2314	0	0
115	115	2314	.	0
116	116	2314	0	0
117	117	2314	0	0
118	118	2314	0	0
119	119	2314	0	0
120	120	2314	0	0
121	121	2314	0	0
122	122	2314	0	0
123	123	2314	0	0
124	124	2314	0	0
125	125	2314	0	0
126	126	2314	0	0
127	127	2314	0	0
128	128	2314	.	0
129	129	2314	.	0
130	130	2314	.	0
131	131	2314	.	0
132	132	2314	0	0
133	133	2314	.	0
134	134	2314	0	0
135	135	2314	.	0
136	136	2314	0	0
137	137	2314	.	0
138	138	2314	.	0
139	139	2314	0	0
140	140	2314	0	0
141	141	2314	0	0
142	142	2314		0
143	143	2314		0
144	144	2314	0	0
145	145	2314		0
146	146	2314		0
147	147	2314		0
148	148	2314		0
149	149	2314		0
150	150	2314		0
151	151	2314		0
152	152	2314		0
153	153	2314		0
154	154	2314		0
155	155	2314		0
156	156	2314		0
157	157	2314		0
158	158	2314		0
159	159	2314		0
160	160	2314		0
161	161	2314		0
162	162	2314		0
163	163	2314		0
164	164	2314		0
165	165	2314		0
166	166	2314	.	0
167	167	2314	.	0
168	168	2314	.	0
169	169	2314	.	0
170	170	2314	.	0
171	171	2314	.	0
172	172	2314	.	0
173	173	2314	.	0
174	174	2314	.	0
175	175	2314	.	0
176	176	2314	.	0
177	177	2314	.	0
178	178	2314	.	0
179	179	2314	.	0
180	180	2314	.	0
181	181	2314	.	0
182	182	2314	.	0
183	183	2314	.	0
184	184	2314	.	0
185	185	2314	.	0
186	186	2314	.	0
187	187	2314	9	0
188	188	2314	37	0
189	189	2314	1	0
190	190	2314	2	0
191	191	2314	0.4	0
192	192	2314	0.6	0
193	193	2314		0
194	194	2314		0
195	195	2314	3	0
196	196	2314	0	0
197	197	2314	1.2	0
198	198	2314	0.4	0
199	199	2314	0.5	0
200	200	2314	0.7	0
201	201	2314	0.5	0
202	202	2314	0.7	0
203	203	2314		0
204	204	2314		0
205	205	2314	2	0
206	206	2314	17	0
207	207	2314	4	0
208	208	2314	1	0
209	209	2314	7	0
210	210	2314	4	0
211	211	2314	3	0
212	212	2314	2	0
213	213	2314	1	0
214	214	2314	0	0
215	215	2314	0	0
216	216	2314	0	0
217	217	2314	0	0
218	218	2314	0	0
219	219	2314	0	0
220	220	2314	0	0
221	221	2314	B0L6	0
222	222	2314	0.9	0
223	223	2314	1	0
224	224	2314	11.5	0
225	225	2314	6	0
226	226	2314	25.9	0
227	227	2314	26	0
228	228	2314	12.2	0
229	229	2314	155	0
230	230	2314	2.8	0
231	231	2314	2.7	0
232	232	2314	5	0
233	233	2314	4	0
234	234	2314	0	0
235	235	2314	76.5	0
236	236	2314	200	0
237	237	2314	200	0
238	238	2314	10	0
239	239	2314	40	0
240	240	2314	90	0
241	241	2314	0	0
242	242	2314	1.918333333	0
243	243	2314	27.13333333	0
244	244	2314	6.4	0
245	245	2314	250	0
246	246	2314	250	0
247	247	2314	2.34	0
248	248	2314	85	0
249	249	2314	41	0
250	250	2314	2.9	0
251	251	2314	0	0
252	252	2314	0	0
253	253	2314	90	0
254	254	2314	0	0
255	255	2314	0	0
256	256	2314	189	0
257	257	2314	104	0
258	258	2314	35	0
259	259	2314	250	0
260	260	2314	250	0
261	261	2314	2.34	0
262	262	2314	85	0
263	263	2314	41	0
264	264	2314	2.9	0
265	265	2314	0	0
266	266	2314	0	0
267	267	2314	90	0
268	268	2314	0	0
269	269	2314	0	0
270	270	2314	63.9	0
271	271	2314	20.1	0
272	272	2314	114.5	0
273	273	2314	41.4	0
274	274	2314	9.5	0
275	275	2314	5.5	0
276	276	2314	10.43	0
277	277	2314	2.41	0
278	278	2314	10.4501	0
279	279	2314	2.4569	0
280	280	2314	29	0
281	281	2314	1	0
282	282	2314	0	0
283	283	2314	1	0
284	284	2314	.	0
285	285	2314	.	0
286	286	2314		0
287	287	2314		0
288	288	2314	1	0
289	289	2314	0	0
290	290	2314	1.1	0
291	291	2314	0.4	0
292	292	2314	0.6	0
293	293	2314	0.8	0
294	294	2314		0
295	295	2314		0
296	296	2314		0
297	297	2314		0
298	298	2314	2	0
299	299	2314	9	0
300	300	2314	2	0
301	301	2314	1	0
302	302	2314	6	0
303	303	2314	2	0
304	304	2314	1	0
305	305	2314	0	0
306	306	2314	0	0
307	307	2314	0	0
308	308	2314	0	0
309	309	2314	0	0
310	310	2314	0	0
311	311	2314	0	0
312	312	2314	0	0
313	313	2314	0	0
314	314	2314	B0L4	0
315	315	2314	1	0
316	316	2314	1.3	0
317	317	2314	29.6	0
318	318	2314	12	0
319	319	2314	29	0
320	320	2314	22	0
321	321	2314	2.3	0
322	322	2314	98	0
323	323	2314	2.5	0
324	324	2314	2.8	0
325	325	2314	0	0
326	326	2314	0	0
327	327	2314	0	0
328	328	2314	84	0
329	329	2314	175	0
330	330	2314	175	0
331	331	2314	10	0
332	332	2314	30	0
333	333	2314	95	0
334	334	2314	0	0
335	335	2314	3.22	0
336	336	2314	44.55	0
337	337	2314	10.43333333	0
338	338	2314	225	0
339	339	2314	275	0
340	340	2314	2.31	0
341	341	2314	97	0
342	342	2314	40.5	0
343	343	2314	2.64	0
344	344	2314	14	0
345	345	2314	0	0
346	346	2314	70	0
347	347	2314	0	0
348	348	2314	0	0
349	349	2314	147	0
350	350	2314	127	0
351	351	2314	26	0
352	352	2314	225	0
353	353	2314	275	0
354	354	2314	2.31	0
355	355	2314	97	0
356	356	2314	40.5	0
357	357	2314	2.64	0
358	358	2314	14	0
359	359	2314	0	0
360	360	2314	70	0
361	361	2314	0	0
362	362	2314	0	0
363	363	2314	63.6	0
364	364	2314	15.8	0
365	365	2314	57.5	0
366	366	2314	22.7	0
367	367	2314	7.8	0
368	368	2314	5.2	0
369	369	2314	12.43	0
370	370	2314	2.55	0
371	371	2314	12.4458	0
372	372	2314	2.5779	0
373	373	2314	63	0
374	374	2314	53	0
375	375	2314	1	0
376	376	2314	2	0
377	377	2314	0.8	0
378	378	2314	0.5	0
379	379	2314	0.7	0
380	380	2314	0.8	0
381	381	2314	3	0
382	382	2314	0	0
383	383	2314	0.7	0
384	384	2314	0.5	0
385	385	2314	0.6	0
386	386	2314	0.9	0
387	387	2314	0.6	0
388	388	2314	0.8	0
389	389	2314	0.7	0
390	390	2314	1	0
391	391	2314	2	0
392	392	2314	22	0
393	393	2314	6	0
394	394	2314	1	0
395	395	2314	8	0
396	396	2314	4	0
397	397	2314	3	0
398	398	2314	3	0
399	399	2314	2	0
400	400	2314	1	0
401	401	2314	0	0
402	402	2314	1	0
403	403	2314	0	0
404	404	2314	0	0
405	405	2314	0	0
406	406	2314	0	0
407	407	2314	B0L6	0
408	408	2314	0.9	0
409	409	2314	1.1	0
410	410	2314	16.4	0
411	411	2314	10	0
412	412	2314	39.5	0
413	413	2314	56	0
414	414	2314	25.7	0
415	415	2314	295	0
416	416	2314	2.8	0
417	417	2314	2.8	0
418	418	2314	0	0
419	419	2314	0	0
420	420	2314	0	0
421	421	2314	105	0
422	422	2314	300	0
423	423	2314	300	0
424	424	2314	5	0
425	425	2314	50	0
426	426	2314	85	0
427	427	2314	4	0
428	428	2314	1.753333333	0
429	429	2314	39.48333333	0
430	430	2314	6.033333333	0
431	431	2314	200	0
432	432	2314	225	0
433	433	2314	1.88	0
434	434	2314	94	0
435	435	2314	64	0
436	436	2314	4.71	0
437	437	2314	0	0
438	438	2314	0	0
439	439	2314	75	0
440	440	2314	0	0
441	441	2314	0	0
442	442	2314	335	0
443	443	2314	109	0
444	444	2314	25	0
445	445	2314	200	0
446	446	2314	225	0
447	447	2314	1.88	0
448	448	2314	94	0
449	449	2314	64	0
450	450	2314	4.71	0
451	451	2314	0	0
452	452	2314	0	0
453	453	2314	75	0
454	454	2314	0	0
455	455	2314	0	0
456	456	2314	62.1	0
457	457	2314	16.9	0
458	458	2314	163	0
459	459	2314	47.5	0
460	460	2314	18.8	0
461	461	2314	6.4	0
462	462	2314	18.04	0
463	463	2314	3.71	0
464	464	2314	18.0569	0
465	465	2314	3.7639	0
466	466	2314	9	0
467	467	2314	17	0
468	468	2314	1	0
469	469	2314	2	0
470	470	2314	0.5	0
471	471	2314	0.5	0
472	472	2314		0
473	473	2314		0
474	474	2314	2	0
475	475	2314	0	0
476	476	2314	0.5	0
477	477	2314	0.3	0
478	478	2314	0.6	0
479	479	2314	0.6	0
480	480	2314	0.5	0
481	481	2314	0.7	0
482	482	2314		0
483	483	2314		0
484	484	2314	0	0
485	485	2314	10	0
486	486	2314	3	0
487	487	2314	1	0
488	488	2314	6	0
489	489	2314	2	0
490	490	2314	2	0
491	491	2314	0	0
492	492	2314	0	0
493	493	2314	0	0
494	494	2314	0	0
495	495	2314	0	0
496	496	2314	0	0
497	497	2314	0	0
498	498	2314	0	0
499	499	2314	0	0
500	500	2314	B0L5	0
501	501	2314	1	0
502	502	2314	1.25	0
503	503	2314	36.6	0
504	504	2314	15	0
505	505	2314	40.9	0
506	506	2314	33	0
507	507	2314	14.1	0
508	508	2314	151	0
509	509	2314	3	0
510	510	2314	2.7	0
511	511	2314	0	0
512	512	2314	0	0
513	513	2314	0	0
514	514	2314	75	0
515	515	2314	175	0
516	516	2314	175	0
517	517	2314	5	0
518	518	2314	45	0
519	519	2314	80	0
520	520	2314	0	0
521	521	2314	1.456666667	0
522	522	2314	16.46666667	0
523	523	2314	4.966666667	0
524	524	2314	75	0
525	525	2314	150	0
526	526	2314	2.12	0
527	527	2314	88	0
528	528	2314	46	0
529	529	2314	1.32	0
530	530	2314	0	0
531	531	2314	0	0
532	532	2314	40	0
533	533	2314	0	0
534	534	2314	0	0
535	535	2314	156	0
536	536	2314	34	0
537	537	2314	6	0
538	538	2314	75	0
539	539	2314	150	0
540	540	2314	2.12	0
541	541	2314	88	0
542	542	2314	46	0
543	543	2314	1.32	0
544	544	2314	0	0
545	545	2314	0	0
546	546	2314	40	0
547	547	2314	0	0
548	548	2314	0	0
549	549	2314	80.1	0
550	550	2314	29.9	0
551	551	2314	48.9	0
552	552	2314	22.7	0
553	553	2314	18.9	0
554	554	2314	9.4	0
555	555	2314	13.12	0
556	556	2314	1.39	0
557	557	2314	13.1499	0
558	558	2314	1.4221	0
559	559	2314	14	0
560	560	2314	1	0
561	561	2314	2	0
562	562	2314	3	0
563	563	2314	0.9	0
564	564	2314	1.3	0
565	565	2314	1.5	0
566	566	2314	1.7	0
567	567	2314	5	0
568	568	2314	1	0
569	569	2314	0.9	0
570	570	2314	0.6	0
571	571	2314	0.9	0
572	572	2314	1.4	0
573	573	2314	1	0
574	574	2314	1.1	0
575	575	2314	1.1	0
576	576	2314	1.3	0
577	577	2314	1	0
578	578	2314	38	0
579	579	2314	10	0
580	580	2314	1	0
581	581	2314	9	0
582	582	2314	6	0
583	583	2314	5	0
584	584	2314	5	0
585	585	2314	3	0
586	586	2314	2	0
587	587	2314	1	0
588	588	2314	2	0
589	589	2314	1	0
590	590	2314	2	0
591	591	2314	1	0
592	592	2314	1	0
593	593	2314	B0L7	0
594	594	2314	1.3	0
595	595	2314	1.5	0
596	596	2314	18.5	0
597	597	2314	12	0
598	598	2314	41.1	0
599	599	2314	59	0
600	600	2314	24.3	0
601	601	2314	458	0
602	602	2314	2.2	0
603	603	2314	2.7	0
604	604	2314	20	0
605	605	2314	33	0
606	606	2314	4	0
607	607	2314	83	0
608	608	2314	300	0
609	609	2314	300	0
610	610	2314	5	0
611	611	2314	50	0
612	612	2314	95	0
613	613	2314	0	0
614	614	2314	1.73	0
615	615	2314	12.93333333	0
616	616	2314	5.583333333	0
617	617	2314	150	0
618	618	2314	250	0
619	619	2314	2.53	0
620	620	2314	88	0
621	621	2314	70	0
622	622	2314	3.89	0
623	623	2314	7	0
624	624	2314	0	0
625	625	2314	60	0
626	626	2314	0	0
627	627	2314	0	0
628	628	2314	343	0
629	629	2314	71	0
630	630	2314	2	0
631	631	2314	150	0
632	632	2314	250	0
633	633	2314	2.53	0
634	634	2314	88	0
635	635	2314	70	0
636	636	2314	3.89	0
637	637	2314	7	0
638	638	2314	0	0
639	639	2314	60	0
640	640	2314	0	0
641	641	2314	0	0
642	642	2314	70.1	0
643	643	2314	26.1	0
644	644	2314	146.5	0
645	645	2314	51.1	0
646	646	2314	35.9	0
647	647	2314	14.1	0
648	648	2314	14.85	0
649	649	2314	3.33	0
650	650	2314	14.8761	0
651	651	2314	3.3952	0
652	652	2314	27	0
653	653	2314	10	0
654	654	2314	1	0
655	655	2314	2	0
656	656	2314	0.5	0
657	657	2314	0.7	0
658	658	2314		0
659	659	2314		0
660	660	2314	3	0
661	661	2314	0	0
662	662	2314	0.8	0
663	663	2314	0.4	0
664	664	2314	0.6	0
665	665	2314	0.9	0
666	666	2314	0.7	0
667	667	2314	0.7	0
668	668	2314		0
669	669	2314		0
670	670	2314	0	0
671	671	2314	13	0
672	672	2314	3	0
673	673	2314	2	0
674	674	2314	7	0
675	675	2314	3	0
676	676	2314	2	0
677	677	2314	1	0
678	678	2314	0	0
679	679	2314	0	0
680	680	2314	0	0
681	681	2314	0	0
682	682	2314	0	0
683	683	2314	0	0
684	684	2314	0	0
685	685	2314	0	0
686	686	2314	B0L6	0
687	687	2314	1.3	0
688	688	2314	1.3	0
689	689	2314	42.5	0
690	690	2314	16	0
691	691	2314	55.3	0
692	692	2314	40	0
693	693	2314	16.3	0
694	694	2314	212	0
695	695	2314	3.1	0
696	696	2314	2.7	0
697	697	2314	9	0
698	698	2314	18	0
699	699	2314	0	0
700	700	2314	99	0
701	701	2314	200	0
702	702	2314	200	0
703	703	2314	5	0
704	704	2314	25	0
705	705	2314	85	0
706	706	2314	0	0
707	707	2314	1.79	0
708	708	2314	23.4	0
709	709	2314	6.1	0
710	710	2314	50	0
711	711	2314	150	0
712	712	2314	2.42	0
713	713	2314	87	0
714	714	2314	34	0
715	715	2314	2.84	0
716	716	2314	0	0
717	717	2314	0	0
718	718	2314	50	0
719	719	2314	0	0
720	720	2314	0	0
721	721	2314	141	0
722	722	2314	18	0
723	723	2314	5	0
724	724	2314	50	0
725	725	2314	150	0
726	726	2314	2.42	0
727	727	2314	87	0
728	728	2314	34	0
729	729	2314	2.84	0
730	730	2314	0	0
731	731	2314	0	0
732	732	2314	50	0
733	733	2314	0	0
734	734	2314	0	0
735	735	2314	82.2	0
736	736	2314	27.3	0
737	737	2314	89	0
738	738	2314	30.6	0
739	739	2314	6.4	0
740	740	2314	2.7	0
741	741	2314	9.23	0
742	742	2314	2.49	0
743	743	2314	9.2573	0
744	744	2314	2.5233	0
745	745	2314	11	0
746	746	2314	1	0
747	747	2314	1	0
748	748	2314	2	0
749	749	2314	0.4	0
750	750	2314	0.5	0
751	751	2314		0
752	752	2314		0
753	753	2314	3	0
754	754	2314	0	0
755	755	2314	1	0
756	756	2314	0.4	0
757	757	2314	0.4	0
758	758	2314	0.6	0
759	759	2314	0.35	0
760	760	2314	0.5	0
761	761	2314		0
762	762	2314		0
763	763	2314	2	0
764	764	2314	13	0
765	765	2314	3	0
766	766	2314	2	0
767	767	2314	7	0
768	768	2314	3	0
769	769	2314	2	0
770	770	2314	1	0
771	771	2314	0	0
772	772	2314	0	0
773	773	2314	0	0
774	774	2314	0	0
775	775	2314	0	0
776	776	2314	0	0
777	777	2314	0	0
778	778	2314	0	0
779	779	2314	B0L5	0
780	780	2314	1	0
781	781	2314	1.1	0
782	782	2314	17.4	0
783	783	2314	47	0
784	784	2314	45.3	0
785	785	2314	21	0
786	786	2314	5.1	0
787	787	2314	98	0
788	788	2314	2.4	0
789	789	2314	2	0
790	790	2314	0	0
791	791	2314	0	0
792	792	2314	0	0
793	793	2314	96	0
794	794	2314	125	0
795	795	2314	125	0
796	796	2314	5	0
797	797	2314	35	0
798	798	2314	85	0
799	799	2314	0	0
800	800	2314	1.483333333	0
801	801	2314	19.36666667	0
802	802	2314	4.25	0
803	803	2314	150	0
804	804	2314	250	0
805	805	2314	2.5	0
806	806	2314	122	0
807	807	2314	39	0
808	808	2314	2.03	0
809	809	2314	0	0
810	810	2314	0	0
811	811	2314	60	0
812	812	2314	2	0
813	813	2314	15	0
814	814	2314	100	0
815	815	2314	25	0
816	816	2314	21	0
817	817	2314	150	0
818	818	2314	250	0
819	819	2314	2.5	0
820	820	2314	122	0
821	821	2314	39	0
822	822	2314	2.03	0
823	823	2314	0	0
824	824	2314	0	0
825	825	2314	60	0
826	826	2314	2	0
827	827	2314	15	0
828	828	2314	105	0
829	829	2314	32	0
830	830	2314	42.2	0
831	831	2314	16	0
832	832	2314	5.5	0
833	833	2314	4.2	0
834	834	2314	6.55	0
835	835	2314	1.19	0
836	836	2314	6.582	0
837	837	2314	1.2102	0
838	838	2314	54	0
839	839	2314	10	0
840	840	2314	2	0
841	841	2314	3	0
842	842	2314	0.5	0
843	843	2314	0.7	0
844	844	2314	1.2	0
845	845	2314	1.2	0
846	846	2314	4	0
847	847	2314	0	0
848	848	2314	1	0
849	849	2314	0.4	0
850	850	2314	0.5	0
851	851	2314	0.7	0
852	852	2314	0.6	0
853	853	2314	0.8	0
854	854	2314	0.7	0
855	855	2314	0.8	0
856	856	2314	1	0
857	857	2314	22	0
858	858	2314	6	0
859	859	2314	1	0
860	860	2314	7	0
861	861	2314	5	0
862	862	2314	4	0
863	863	2314	3	0
864	864	2314	2	0
865	865	2314	1	0
866	866	2314	0	0
867	867	2314	0	0
868	868	2314	0	0
869	869	2314	0	0
870	870	2314	0	0
871	871	2314	0	0
872	872	2314	B0L6	0
873	873	2314	1.05	0
874	874	2314	1.3	0
875	875	2314	25.8	0
876	876	2314	12	0
877	877	2314	43.1	0
878	878	2314	32	0
879	879	2314	9.5	0
880	880	2314	178	0
881	881	2314	2.7	0
882	882	2314	2.4	0
883	883	2314	0	0
884	884	2314	0	0
885	885	2314	0	0
886	886	2314	95.5	0
887	887	2314	300	0
888	888	2314	300	0
889	889	2314	15	0
890	890	2314	70	0
891	891	2314	95	0
892	892	2314	1	0
893	893	2314	1.541666667	0
894	894	2314	17.86666667	0
895	895	2314	4.783333333	0
896	896	2314	200	0
897	897	2314	200	0
898	898	2314	2.56	0
899	899	2314	95	0
900	900	2314	50	0
901	901	2314	3.32	0
902	902	2314	6	0
903	903	2314	0	0
904	904	2314	75	0
905	905	2314	0	0
906	906	2314	0	0
907	907	2314	274	0
908	908	2314	31	0
909	909	2314	24	0
910	910	2314	200	0
911	911	2314	200	0
912	912	2314	2.56	0
913	913	2314	95	0
914	914	2314	50	0
915	915	2314	3.32	0
916	916	2314	6	0
917	917	2314	0	0
918	918	2314	75	0
919	919	2314	0	0
920	920	2314	0	0
921	921	2314	74.2	0
922	922	2314	23.1	0
923	923	2314	191.2	0
924	924	2314	58.8	0
925	925	2314	20.1	0
926	926	2314	8.4	0
927	927	2314	13.59	0
928	928	2314	2.32	0
929	929	2314	13.6131	0
930	930	2314	2.3872	0
931	931	2314	9	0
932	932	2314	1	0
933	933	2314	1	0
934	934	2314	2	0
935	935	2314	0.7	0
936	936	2314	1	0
937	937	2314	0.9	0
938	938	2314	1.1	0
939	939	2314	4	0
940	940	2314	0	0
941	941	2314	1.2	0
942	942	2314	0.5	0
943	943	2314	0.7	0
944	944	2314	1.2	0
945	945	2314	0.8	0
946	946	2314	0.9	0
947	947	2314	0.8	0
948	948	2314	1	0
949	949	2314	2	0
950	950	2314	25	0
951	951	2314	5	0
952	952	2314	1	0
953	953	2314	8	0
954	954	2314	4	0
955	955	2314	4	0
956	956	2314	4	0
957	957	2314	2	0
958	958	2314	2	0
959	959	2314	0	0
960	960	2314	1	0
961	961	2314	0	0
962	962	2314	0	0
963	963	2314	0	0
964	964	2314	0	0
965	965	2314	B0L6	0
966	966	2314	1.5	0
967	967	2314	1.15	0
968	968	2314	28.1	0
969	969	2314	16	0
970	970	2314	54.7	0
971	971	2314	57	0
972	972	2314	21.4	0
973	973	2314	273	0
974	974	2314	3.2	0
975	975	2314	2.5	0
976	976	2314	3	0
977	977	2314	6	0
978	978	2314	0	0
979	979	2314	92	0
980	980	2314	275	0
981	981	2314	275	0
982	982	2314	10	0
983	983	2314	55	0
984	984	2314	90	0
985	985	2314	0	0
986	986	2314	2.2	0
987	987	2314	27.36666667	0
988	988	2314	8.416666667	0
989	989	2314	450	0
990	990	2314	450	0
991	991	2314	1.24	0
992	992	2314	114	0
993	993	2314	48	0
994	994	2314	5.36	0
995	995	2314	6	0
996	996	2314	0	0
997	997	2314	90	0
998	998	2314	0	0
999	999	2314	0	0
1000	1000	2314	170	0
1001	1001	2314	91	0
1002	1002	2314	48	0
1003	1003	2314	450	0
1004	1004	2314	450	0
1005	1005	2314	1.24	0
1006	1006	2314	114	0
1007	1007	2314	48	0
1008	1008	2314	5.36	0
1009	1009	2314	6	0
1010	1010	2314	0	0
1011	1011	2314	90	0
1012	1012	2314	0	0
1013	1013	2314	0	0
1014	1014	2314	85.1	0
1015	1015	2314	21.6	0
1016	1016	2314	350.4	0
1017	1017	2314	100.9	0
1018	1018	2314	10.7	0
1019	1019	2314	5.1	0
1020	1020	2314	16.97	0
1021	1021	2314	5.34	0
1022	1022	2314	16.9916	0
1023	1023	2314	5.446	0
1024	1024	2314	29	0
1025	1025	2314	4	0
1026	1026	2314	1	0
1027	1027	2314	2	0
1028	1028	2314	0.8	0
1029	1029	2314	0.9	0
1030	1030	2314		0
1031	1031	2314		0
1032	1032	2314	3	0
1033	1033	2314	0	0
1034	1034	2314	0.9	0
1035	1035	2314	0.5	0
1036	1036	2314	0.8	0
1037	1037	2314	1.1	0
1038	1038	2314	0.9	0
1039	1039	2314	1.1	0
1040	1040	2314		0
1041	1041	2314		0
1042	1042	2314	2	0
1043	1043	2314	16	0
1044	1044	2314	4	0
1045	1045	2314	1	0
1046	1046	2314	7	0
1047	1047	2314	4	0
1048	1048	2314	3	0
1049	1049	2314	1	0
1050	1050	2314	1	0
1051	1051	2314	0	0
1052	1052	2314	0	0
1053	1053	2314	0	0
1054	1054	2314	0	0
1055	1055	2314	0	0
1056	1056	2314	0	0
1057	1057	2314	0	0
1058	1058	2314	B0L6	0
1059	1059	2314	1.3	0
1060	1060	2314	1.3	0
1061	1061	2314	37.5	0
1062	1062	2314	16	0
1063	1063	2314	41.5	0
1064	1064	2314	54	0
1065	1065	2314	23.8	0
1066	1066	2314	317	0
1067	1067	2314	3.2	0
1068	1068	2314	2.3	0
1069	1069	2314	0	0
1070	1070	2314	0	0
1071	1071	2314	0	0
1072	1072	2314	79	0
1073	1073	2314	125	0
1074	1074	2314	200	0
1075	1075	2314	5	0
1076	1076	2314	20	0
1077	1077	2314	60	0
1078	1078	2314	0	0
1079	1079	2314	1.385	0
1080	1080	2314	20.4	0
1081	1081	2314	6.183333333	0
1082	1082	2314	10	0
1083	1083	2314	400	0
1084	1084	2314	3.42	0
1085	1085	2314	81.5	0
1086	1086	2314	61.5	0
1087	1087	2314	3.41	0
1088	1088	2314	0	0
1089	1089	2314	0	0
1090	1090	2314	5	0
1091	1091	2314	0	0
1092	1092	2314	0	0
1093	1093	2314	12	0
1094	1094	2314	9	0
1095	1095	2314	1	0
1096	1096	2314	10	0
1097	1097	2314	400	0
1098	1098	2314	3.42	0
1099	1099	2314	81.5	0
1100	1100	2314	61.5	0
1101	1101	2314	3.41	0
1102	1102	2314	0	0
1103	1103	2314	0	0
1104	1104	2314	5	0
1105	1105	2314	0	0
1106	1106	2314	0	0
1107	1107	2314	135.3	0
1108	1108	2314	58	0
1109	1109	2314	127.3	0
1110	1110	2314	52.3	0
1111	1111	2314	9.4	0
1112	1112	2314	7.5	0
1113	1113	2314	20.75	0
1114	1114	2314	2.74	0
1115	1115	2314	20.808	0
1116	1116	2314	2.7998	0
1117	1117	2314	.	0
1118	1118	2314	.	0
1119	1119	2314	1	0
1120	1120	2314	2	0
1121	1121	2314	0.2	0
1122	1122	2314	0.2	0
1123	1123	2314		0
1124	1124	2314		0
1125	1125	2314	0	0
1126	1126	2314	0	0
1127	1127	2314		0
1128	1128	2314		0
1129	1129	2314		0
1130	1130	2314		0
1131	1131	2314		0
1132	1132	2314		0
1133	1133	2314		0
1134	1134	2314		0
1135	1135	2314	0	0
1136	1136	2314	0	0
1137	1137	2314	0	0
1138	1138	2314	.	0
1139	1139	2314	0	0
1140	1140	2314	0	0
1141	1141	2314	0	0
1142	1142	2314	0	0
1143	1143	2314	0	0
1144	1144	2314	0	0
1145	1145	2314	0	0
1146	1146	2314	0	0
1147	1147	2314	0	0
1148	1148	2314	0	0
1149	1149	2314	0	0
1150	1150	2314	0	0
1151	1151	2314	.	0
1152	1152	2314	.	0
1153	1153	2314	.	0
1154	1154	2314	.	0
1155	1155	2314	0	0
1156	1156	2314	.	0
1157	1157	2314	0	0
1158	1158	2314	.	0
1159	1159	2314	0	0
1160	1160	2314	.	0
1161	1161	2314	.	0
1162	1162	2314	0	0
1163	1163	2314	0	0
1164	1164	2314	0	0
1165	1165	2314		0
1166	1166	2314		0
1167	1167	2314	0	0
1168	1168	2314		0
1169	1169	2314		0
1170	1170	2314		0
1171	1171	2314		0
1172	1172	2314		0
1173	1173	2314		0
1174	1174	2314		0
1175	1175	2314		0
1176	1176	2314		0
1177	1177	2314		0
1178	1178	2314		0
1179	1179	2314		0
1180	1180	2314		0
1181	1181	2314		0
1182	1182	2314		0
1183	1183	2314		0
1184	1184	2314		0
1185	1185	2314		0
1186	1186	2314		0
1187	1187	2314		0
1188	1188	2314		0
1189	1189	2314	.	0
1190	1190	2314	.	0
1191	1191	2314	.	0
1192	1192	2314	.	0
1193	1193	2314	.	0
1194	1194	2314	.	0
1195	1195	2314	.	0
1196	1196	2314	.	0
1197	1197	2314	.	0
1198	1198	2314	.	0
1199	1199	2314	.	0
1200	1200	2314	.	0
1201	1201	2314	.	0
1202	1202	2314	.	0
1203	1203	2314	.	0
1204	1204	2314	.	0
1205	1205	2314	.	0
1206	1206	2314	.	0
1207	1207	2314	.	0
1208	1208	2314	.	0
1209	1209	2314	.	0
1210	1210	2314	35	0
1211	1211	2314	39	0
1212	1212	2314	2	0
1213	1213	2314	3	0
1214	1214	2314	0.5	0
1215	1215	2314	0.5	0
1216	1216	2314	1	0
1217	1217	2314	1.1	0
1218	1218	2314	3	0
1219	1219	2314	0	0
1220	1220	2314	1.1	0
1221	1221	2314	0.4	0
1222	1222	2314	0.6	0
1223	1223	2314	0.6	0
1224	1224	2314	0.7	0
1225	1225	2314	0.7	0
1226	1226	2314	0.7	0
1227	1227	2314	0.8	0
1228	1228	2314	2	0
1229	1229	2314	24	0
1230	1230	2314	4	0
1231	1231	2314	1	0
1232	1232	2314	9	0
1233	1233	2314	5	0
1234	1234	2314	5	0
1235	1235	2314	4	0
1236	1236	2314	1	0
1237	1237	2314	0	0
1238	1238	2314	0	0
1239	1239	2314	0	0
1240	1240	2314	0	0
1241	1241	2314	0	0
1242	1242	2314	0	0
1243	1243	2314	0	0
1244	1244	2314	B0L7	0
1245	1245	2314	1.1	0
1246	1246	2314	1.45	0
1247	1247	2314	29.2	0
1248	1248	2314	16	0
1249	1249	2314	36.4	0
1250	1250	2314	33	0
1251	1251	2314	21.1	0
1252	1252	2314	206	0
1253	1253	2314	2.4	0
1254	1254	2314	2.2	0
1255	1255	2314	0	0
1256	1256	2314	0	0
1257	1257	2314	0	0
1258	1258	2314	75.5	0
1259	1259	2314	350	0
1260	1260	2314	350	0
1261	1261	2314	10	0
1262	1262	2314	80	0
1263	1263	2314	95	0
1264	1264	2314	0	0
1265	1265	2314	1.05	0
1266	1266	2314	12.31666667	0
1267	1267	2314	3.5	0
1268	1268	2314	75	0
1269	1269	2314	150	0
1270	1270	2314	2.87	0
1271	1271	2314	82	0
1272	1272	2314	43	0
1273	1273	2314	3.9	0
1274	1274	2314	0	0
1275	1275	2314	0	0
1276	1276	2314	40	0
1277	1277	2314	0	0
1278	1278	2314	0	0
1279	1279	2314	216	0
1280	1280	2314	41	0
1281	1281	2314	5	0
1282	1282	2314	75	0
1283	1283	2314	150	0
1284	1284	2314	2.87	0
1285	1285	2314	82	0
1286	1286	2314	43	0
1287	1287	2314	3.9	0
1288	1288	2314	0	0
1289	1289	2314	0	0
1290	1290	2314	40	0
1291	1291	2314	0	0
1292	1292	2314	0	0
1293	1293	2314	91.4	0
1294	1294	2314	34	0
1295	1295	2314	117.8	0
1296	1296	2314	41.8	0
1297	1297	2314	19.3	0
1298	1298	2314	8.9	0
1299	1299	2314	13.37	0
1300	1300	2314	1.31	0
1301	1301	2314	13.404	0
1302	1302	2314	1.3607	0
1303	1303	2314	53	0
1304	1304	2314	18	0
1305	1305	2314	1	0
1306	1306	2314	2	0
1307	1307	2314	0.5	0
1308	1308	2314	0.7	0
1309	1309	2314		0
1310	1310	2314		0
1311	1311	2314	3	0
1312	1312	2314	0	0
1313	1313	2314	0.9	0
1314	1314	2314	0.5	0
1315	1315	2314	0.7	0
1316	1316	2314	0.9	0
1317	1317	2314	0.6	0
1318	1318	2314	0.9	0
1319	1319	2314		0
1320	1320	2314		0
1321	1321	2314	2	0
1322	1322	2314	13	0
1323	1323	2314	3	0
1324	1324	2314	1	0
1325	1325	2314	6	0
1326	1326	2314	3	0
1327	1327	2314	3	0
1328	1328	2314	1	0
1329	1329	2314	0	0
1330	1330	2314	0	0
1331	1331	2314	0	0
1332	1332	2314	0	0
1333	1333	2314	0	0
1334	1334	2314	0	0
1335	1335	2314	0	0
1336	1336	2314	0	0
1337	1337	2314	B0L5	0
1338	1338	2314	1.3	0
1339	1339	2314	1.45	0
1340	1340	2314	19.4	0
1341	1341	2314	10	0
1342	1342	2314	30.2	0
1343	1343	2314	23	0
1344	1344	2314	16.3	0
1345	1345	2314	112	0
1346	1346	2314	2.8	0
1347	1347	2314	2.6	0
1348	1348	2314	0	0
1349	1349	2314	0	0
1350	1350	2314	0	0
1351	1351	2314	54.5	0
1352	1352	2314	200	0
1353	1353	2314	200	0
1354	1354	2314	10	0
1355	1355	2314	30	0
1356	1356	2314	95	0
1357	1357	2314	0	0
1358	1358	2314	1.651666667	0
1359	1359	2314	20.03333333	0
1360	1360	2314	5.316666667	0
1361	1361	2314	300	0
1362	1362	2314	325	0
1363	1363	2314	1.94	0
1364	1364	2314	59	0
1365	1365	2314	38	0
1366	1366	2314	2.26	0
1367	1367	2314	0	0
1368	1368	2314	0	0
1369	1369	2314	90	0
1370	1370	2314	0	0
1371	1371	2314	0	0
1372	1372	2314	119	0
1373	1373	2314	75	0
1374	1374	2314	15	0
1375	1375	2314	300	0
1376	1376	2314	325	0
1377	1377	2314	1.94	0
1378	1378	2314	59	0
1379	1379	2314	38	0
1380	1380	2314	2.26	0
1381	1381	2314	0	0
1382	1382	2314	0	0
1383	1383	2314	90	0
1384	1384	2314	0	0
1385	1385	2314	0	0
1386	1386	2314	50.9	0
1387	1387	2314	13.8	0
1388	1388	2314	44.7	0
1389	1389	2314	17.3	0
1390	1390	2314	9.5	0
1391	1391	2314	5.3	0
1392	1392	2314	11.16	0
1393	1393	2314	2.67	0
1394	1394	2314	11.1738	0
1395	1395	2314	2.6926	0
1396	1396	2314	38	0
1397	1397	2314	13	0
1398	1398	2314	2	0
1399	1399	2314	3	0
1400	1400	2314	0.5	0
1401	1401	2314	0.8	0
1402	1402	2314	1.5	0
1403	1403	2314	1.3	0
1404	1404	2314	4	0
1405	1405	2314	0	0
1406	1406	2314	1.1	0
1407	1407	2314	0.5	0
1408	1408	2314	0.7	0
1409	1409	2314	0.8	0
1410	1410	2314	0.8	0
1411	1411	2314	1	0
1412	1412	2314	0.8	0
1413	1413	2314	1	0
1414	1414	2314	1	0
1415	1415	2314	29	0
1416	1416	2314	7	0
1417	1417	2314	1	0
1418	1418	2314	9	0
1419	1419	2314	5	0
1420	1420	2314	5	0
1421	1421	2314	5	0
1422	1422	2314	2	0
1423	1423	2314	1	0
1424	1424	2314	0	0
1425	1425	2314	1	0
1426	1426	2314	0	0
1427	1427	2314	1	0
1428	1428	2314	0	0
1429	1429	2314	0	0
1430	1430	2314	B0L7	0
1431	1431	2314	1.2	0
1432	1432	2314	1.2	0
1433	1433	2314	26.1	0
1434	1434	2314	17	0
1435	1435	2314	48.3	0
1436	1436	2314	58	0
1437	1437	2314	26.1	0
1438	1438	2314	364	0
1439	1439	2314	3	0
1440	1440	2314	2.5	0
1441	1441	2314	0	0
1442	1442	2314	0	0
1443	1443	2314	0	0
1444	1444	2314	110	0
1445	1445	2314	500	0
1446	1446	2314	600	0
1447	1447	2314	35	0
1448	1448	2314	40	0
1449	1449	2314	85	0
1450	1450	2314	2	0
1451	1451	2314	2.061666667	0
1452	1452	2314	24.83333333	0
1453	1453	2314	6.7	0
1454	1454	2314	200	0
1455	1455	2314	450	0
1456	1456	2314	2.11	0
1457	1457	2314	92	0
1458	1458	2314	55	0
1459	1459	2314	3.79	0
1460	1460	2314	10	0
1461	1461	2314	0	0
1462	1462	2314	40	0
1463	1463	2314	0	0
1464	1464	2314	0	0
1465	1465	2314	252	0
1466	1466	2314	193	0
1467	1467	2314	110	0
1468	1468	2314	200	0
1469	1469	2314	450	0
1470	1470	2314	2.11	0
1471	1471	2314	92	0
1472	1472	2314	55	0
1473	1473	2314	3.79	0
1474	1474	2314	10	0
1475	1475	2314	0	0
1476	1476	2314	40	0
1477	1477	2314	0	0
1478	1478	2314	0	0
1479	1479	2314	157.6	0
1480	1480	2314	51.5	0
1481	1481	2314	296.4	0
1482	1482	2314	79.3	0
1483	1483	2314	79.3	0
1484	1484	2314	23.5	0
1485	1485	2314	48.67	0
1486	1486	2314	5.98	0
1487	1487	2314	48.7215	0
1488	1488	2314	6.0828	0
1489	1489	2314	46	0
1490	1490	2314	12	0
1491	1491	2314	0	0
1492	1492	2314	1	0
1493	1493	2314	0.5	0
1494	1494	2314	0.5	0
1495	1495	2314		0
1496	1496	2314		0
1497	1497	2314	2	0
1498	1498	2314	0	0
1499	1499	2314	0.8	0
1500	1500	2314	0.4	0
1501	1501	2314	0.7	0
1502	1502	2314	0.9	0
1503	1503	2314	0.5	0
1504	1504	2314	0.6	0
1505	1505	2314		0
1506	1506	2314		0
1507	1507	2314	2	0
1508	1508	2314	16	0
1509	1509	2314	3	0
1510	1510	2314	1	0
1511	1511	2314	7	0
1512	1512	2314	3	0
1513	1513	2314	3	0
1514	1514	2314	3	0
1515	1515	2314	0	0
1516	1516	2314	0	0
1517	1517	2314	0	0
1518	1518	2314	0	0
1519	1519	2314	0	0
1520	1520	2314	0	0
1521	1521	2314	0	0
1522	1522	2314	0	0
1523	1523	2314	B0L5	0
1524	1524	2314	0.9	0
1525	1525	2314	1.1	0
1526	1526	2314	30.3	0
1527	1527	2314	10	0
1528	1528	2314	38.1	0
1529	1529	2314	19	0
1530	1530	2314	8.6	0
1531	1531	2314	74	0
1532	1532	2314	2.6	0
1533	1533	2314	2.4	0
1534	1534	2314	0	0
1535	1535	2314	0	0
1536	1536	2314	0	0
1537	1537	2314	93.7	0
1538	1538	2314	100	0
1539	1539	2314	100	0
1540	1540	2314	5	0
1541	1541	2314	20	0
1542	1542	2314	90	0
1543	1543	2314	0	0
1544	1544	2314	1.8	0
1545	1545	2314	23.8	0
1546	1546	2314	6.6	0
1547	1547	2314	225	0
1548	1548	2314	225	0
1549	1549	2314	2.11	0
1550	1550	2314	107	0
1551	1551	2314	39	0
1552	1552	2314	2.37	0
1553	1553	2314	7	0
1554	1554	2314	0	0
1555	1555	2314	90	0
1556	1556	2314	0	0
1557	1557	2314	0	0
1558	1558	2314	78	0
1559	1559	2314	36	0
1560	1560	2314	8	0
1561	1561	2314	225	0
1562	1562	2314	225	0
1563	1563	2314	2.11	0
1564	1564	2314	107	0
1565	1565	2314	39	0
1566	1566	2314	2.37	0
1567	1567	2314	7	0
1568	1568	2314	0	0
1569	1569	2314	90	0
1570	1570	2314	0	0
1571	1571	2314	0	0
1572	1572	2314	68.5	0
1573	1573	2314	21.3	0
1574	1574	2314	98.2	0
1575	1575	2314	37.2	0
1576	1576	2314	11.5	0
1577	1577	2314	7.5	0
1578	1578	2314	8.8	0
1579	1579	2314	1.5	0
1580	1580	2314	8.8213	0
1581	1581	2314	1.5447	0
1582	1582	2314	2	0
1583	1583	2314	2	0
1584	1584	2314	1	0
1585	1585	2314	2	0
1586	1586	2314	0.3	0
1587	1587	2314	0.4	0
1588	1588	2314		0
1589	1589	2314		0
1590	1590	2314	2	0
1591	1591	2314	0	0
1592	1592	2314	0.6	0
1593	1593	2314	0.3	0
1594	1594	2314	0.4	0
1595	1595	2314	0.5	0
1596	1596	2314	0.3	0
1597	1597	2314	0.4	0
1598	1598	2314		0
1599	1599	2314		0
1600	1600	2314	1	0
1601	1601	2314	13	0
1602	1602	2314	3	0
1603	1603	2314	1	0
1604	1604	2314	7	0
1605	1605	2314	3	0
1606	1606	2314	2	0
1607	1607	2314	1	0
1608	1608	2314	0	0
1609	1609	2314	0	0
1610	1610	2314	0	0
1611	1611	2314	0	0
1612	1612	2314	0	0
1613	1613	2314	0	0
1614	1614	2314	0	0
1615	1615	2314	0	0
1616	1616	2314	B0L5	0
1617	1617	2314	0.7	0
1618	1618	2314	0.85	0
1619	1619	2314	25.1	0
1620	1620	2314	6	0
1621	1621	2314	23.8	0
1622	1622	2314	1	0
1623	1623	2314	0.5	0
1624	1624	2314	54	0
1625	1625	2314	2.2	0
1626	1626	2314	2.2	0
1627	1627	2314	0	0
1628	1628	2314	0	0
1629	1629	2314	0	0
1630	1630	2314	68.5	0
1631	1631	2314	40	0
1632	1632	2314	40	0
1633	1633	2314	1	0
1634	1634	2314	10	0
1635	1635	2314	60	0
1636	1636	2314	0	0
1637	1637	2314	1.55	0
1638	1638	2314	13.65	0
1639	1639	2314	4.016666667	0
1640	1640	2314	100	0
1641	1641	2314	100	0
1642	1642	2314	1.55	0
1643	1643	2314	97	0
1644	1644	2314	32	0
1645	1645	2314	0.94	0
1646	1646	2314	0	0
1647	1647	2314	0	0
1648	1648	2314	85	0
1649	1649	2314	0	0
1650	1650	2314	0	0
1651	1651	2314	10	0
1652	1652	2314	27	0
1653	1653	2314	9	0
1654	1654	2314	100	0
1655	1655	2314	100	0
1656	1656	2314	1.55	0
1657	1657	2314	97	0
1658	1658	2314	32	0
1659	1659	2314	0.94	0
1660	1660	2314	0	0
1661	1661	2314	0	0
1662	1662	2314	85	0
1663	1663	2314	0	0
1664	1664	2314	0	0
1665	1665	2314	34.6	0
1666	1666	2314	10.1	0
1667	1667	2314	8.6	0
1668	1668	2314	6.9	0
1669	1669	2314	3.2	0
1670	1670	2314	3.1	0
1671	1671	2314	2.23	0
1672	1672	2314	0.42	0
1673	1673	2314	2.2401	0
1674	1674	2314	0.43	0
1675	1675	2314	41	0
1676	1676	2314	1	0
1677	1677	2314	1	0
1678	1678	2314	2	0
1679	1679	2314	0.4	0
1680	1680	2314	0.4	0
1681	1681	2314		0
1682	1682	2314		0
1683	1683	2314	2	0
1684	1684	2314	0	0
1685	1685	2314	1	0
1686	1686	2314	0.4	0
1687	1687	2314	0.4	0
1688	1688	2314	0.6	0
1689	1689	2314	0.4	0
1690	1690	2314	0.5	0
1691	1691	2314		0
1692	1692	2314		0
1693	1693	2314	0	0
1694	1694	2314	7	0
1695	1695	2314	0	0
1696	1696	2314	2	0
1697	1697	2314	6	0
1698	1698	2314	1	0
1699	1699	2314	0	0
1700	1700	2314	0	0
1701	1701	2314	0	0
1702	1702	2314	0	0
1703	1703	2314	0	0
1704	1704	2314	0	0
1705	1705	2314	0	0
1706	1706	2314	0	0
1707	1707	2314	0	0
1708	1708	2314	0	0
1709	1709	2314	B0L5	0
1710	1710	2314	0.6	0
1711	1711	2314	0.7	0
1712	1712	2314	29.3	0
1713	1713	2314	11	0
1714	1714	2314	22.2	0
1715	1715	2314	2	0
1716	1716	2314	0.6	0
1717	1717	2314	47	0
1718	1718	2314	2.1	0
1719	1719	2314	2.1	0
1720	1720	2314	0	0
1721	1721	2314	0	0
1722	1722	2314	0	0
1723	1723	2314	83	0
1724	1724	2314	100	0
1725	1725	2314	100	0
1726	1726	2314	5	0
1727	1727	2314	20	0
1728	1728	2314	100	0
1729	1729	2314	0	0
1730	1730	2314	1.906666667	0
1731	1731	2314	22.7	0
1732	1732	2314	6.883333333	0
1733	1733	2314	0	0
1734	1734	2314	100	0
1735	1735	2314	1.83	0
1736	1736	2314	82	0
1737	1737	2314		0
1738	1738	2314		0
1739	1739	2314	0	0
1740	1740	2314	0	0
1741	1741	2314	0	0
1742	1742	2314		0
1743	1743	2314		0
1744	1744	2314	69	0
1745	1745	2314	0	0
1746	1746	2314	0	0
1747	1747	2314	0	0
1748	1748	2314	100	0
1749	1749	2314	1.83	0
1750	1750	2314	82	0
1751	1751	2314	.	0
1752	1752	2314	.	0
1753	1753	2314	0	0
1754	1754	2314	0	0
1755	1755	2314	0	0
1756	1756	2314	.	0
1757	1757	2314	.	0
1758	1758	2314	.	0
1759	1759	2314	.	0
1760	1760	2314	.	0
1761	1761	2314	.	0
1762	1762	2314	.	0
1763	1763	2314	.	0
1764	1764	2314	3.15	0
1765	1765	2314	.	0
1766	1766	2314	3.15	0
1767	1767	2314	.	0
1768	1768	2314	6	0
1769	1769	2314	7	0
1770	1770	2314	1	0
1771	1771	2314	3	0
1772	1772	2314	0.4	0
1773	1773	2314	0.4	0
1774	1774	2314	0.6	0
1775	1775	2314	0.6	0
1776	1776	2314	4	0
1777	1777	2314	0	0
1778	1778	2314	1.1	0
1779	1779	2314	0.4	0
1780	1780	2314	.	0
1781	1781	2314	.	0
1782	1782	2314	0.6	0
1783	1783	2314	0.8	0
1784	1784	2314	0.6	0
1785	1785	2314	0.8	0
1786	1786	2314	1	0
1787	1787	2314	22	0
1788	1788	2314	5	0
1789	1789	2314	1	0
1790	1790	2314	8	0
1791	1791	2314	4	0
1792	1792	2314	4	0
1793	1793	2314	3	0
1794	1794	2314	2	0
1795	1795	2314	1	0
1796	1796	2314	0	0
1797	1797	2314	0	0
1798	1798	2314	0	0
1799	1799	2314	0	0
1800	1800	2314	0	0
1801	1801	2314	0	0
1802	1802	2314	B0L6	0
1803	1803	2314	1	0
1804	1804	2314	1.1	0
1805	1805	2314	27.7	0
1806	1806	2314	16	0
1807	1807	2314	56.5	0
1808	1808	2314	48	0
1809	1809	2314	27.7	0
1810	1810	2314	314	0
1811	1811	2314	2.7	0
1812	1812	2314	2.4	0
1813	1813	2314	14	0
1814	1814	2314	16	0
1815	1815	2314	3	0
1816	1816	2314	101	0
1817	1817	2314	150	0
1818	1818	2314	150	0
1819	1819	2314	10	0
1820	1820	2314	30	0
1821	1821	2314	80	0
1822	1822	2314	1	0
1823	1823	2314	1.073333333	0
1824	1824	2314	15.11666667	0
1825	1825	2314	4.15	0
1826	1826	2314	200	0
1827	1827	2314	300	0
1828	1828	2314	1.54	0
1829	1829	2314	82.5	0
1830	1830	2314	34.5	0
1831	1831	2314	3.52	0
1832	1832	2314	0	0
1833	1833	2314	0	0
1834	1834	2314	55	0
1835	1835	2314	5	0
1836	1836	2314	0	0
1837	1837	2314	216	0
1838	1838	2314	48	0
1839	1839	2314	8	0
1840	1840	2314	200	0
1841	1841	2314	300	0
1842	1842	2314	1.54	0
1843	1843	2314	82.5	0
1844	1844	2314	34.5	0
1845	1845	2314	3.52	0
1846	1846	2314	0	0
1847	1847	2314	0	0
1848	1848	2314	55	0
1849	1849	2314	5	0
1850	1850	2314	0	0
1851	1851	2314	65.7	0
1852	1852	2314	20.7	0
1853	1853	2314	165.6	0
1854	1854	2314	55.2	0
1855	1855	2314	17.4	0
1856	1856	2314	7.3	0
1857	1857	2314	16.61	0
1858	1858	2314	3.03	0
1859	1859	2314	16.6307	0
1860	1860	2314	3.0925	0
1861	1861	2314	3	0
1862	1862	2314		0
1863	1863	2314	0	0
1864	1864	2314	1	0
1865	1865	2314		0
1866	1866	2314		0
1867	1867	2314		0
1868	1868	2314		0
1869	1869	2314	1	0
1870	1870	2314	0	0
1871	1871	2314	0.7	0
1872	1872	2314	0.3	0
1873	1873	2314	0.5	0
1874	1874	2314	0.7	0
1875	1875	2314		0
1876	1876	2314		0
1877	1877	2314		0
1878	1878	2314		0
1879	1879	2314	2	0
1880	1880	2314	6	0
1881	1881	2314	2	0
1882	1882	2314	2	0
1883	1883	2314	5	0
1884	1884	2314	1	0
1885	1885	2314	0	0
1886	1886	2314	0	0
1887	1887	2314	0	0
1888	1888	2314	0	0
1889	1889	2314	0	0
1890	1890	2314	0	0
1891	1891	2314	0	0
1892	1892	2314	0	0
1893	1893	2314	0	0
1894	1894	2314	0	0
1895	1895	2314	B0L4	0
1896	1896	2314	0.9	0
1897	1897	2314	1.3	0
1898	1898	2314	30.1	0
1899	1899	2314	11	0
1900	1900	2314	32.7	0
1901	1901	2314	12	0
1902	1902	2314	0.8	0
1903	1903	2314	53	0
1904	1904	2314	2.8	0
1905	1905	2314	2.5	0
1906	1906	2314	0	0
1907	1907	2314	0	0
1908	1908	2314	0	0
1909	1909	2314	75.5	0
1910	1910	2314	150	0
1911	1911	2314	150	0
1912	1912	2314	15	0
1913	1913	2314	10	0
1914	1914	2314	80	0
1915	1915	2314	0	0
1916	1916	2314	2.806666667	0
1917	1917	2314	27.63333333	0
1918	1918	2314	6	0
1919	1919	2314		0
1920	1920	2314		0
1921	1921	2314	2.62	0
1922	1922	2314	110	0
1923	1923	2314	29	0
1924	1924	2314	1.94	0
1925	1925	2314	0	0
1926	1926	2314	0	0
1927	1927	2314	75	0
1928	1928	2314	0	0
1929	1929	2314	40	0
1930	1930	2314	33	0
1931	1931	2314	24	0
1932	1932	2314	7	0
1933	1933	2314	.	0
1934	1934	2314	.	0
1935	1935	2314	2.62	0
1936	1936	2314	110	0
1937	1937	2314	29	0
1938	1938	2314	1.94	0
1939	1939	2314	0	0
1940	1940	2314	0	0
1941	1941	2314	75	0
1942	1942	2314	0	0
1943	1943	2314	40	0
1944	1944	2314	64.7	0
1945	1945	2314	21.8	0
1946	1946	2314	22.8	0
1947	1947	2314	17.8	0
1948	1948	2314	5.6	0
1949	1949	2314	5.1	0
1950	1950	2314	9.54	0
1951	1951	2314	0.82	0
1952	1952	2314	9.5618	0
1953	1953	2314	0.8429	0
1954	1954	2314	.	0
1955	1955	2314	.	0
1956	1956	2314	0	0
1957	1957	2314	0	0
1958	1958	2314		0
1959	1959	2314		0
1960	1960	2314		0
1961	1961	2314		0
1962	1962	2314	0	0
1963	1963	2314	0	0
1964	1964	2314	0.6	0
1965	1965	2314	0.3	0
1966	1966	2314		0
1967	1967	2314		0
1968	1968	2314		0
1969	1969	2314		0
1970	1970	2314		0
1971	1971	2314		0
1972	1972	2314	2	0
1973	1973	2314	0	0
1974	1974	2314	0	0
1975	1975	2314	.	0
1976	1976	2314		0
1977	1977	2314	0	0
1978	1978	2314	0	0
1979	1979	2314	0	0
1980	1980	2314	0	0
1981	1981	2314	0	0
1982	1982	2314	0	0
1983	1983	2314	0	0
1984	1984	2314	0	0
1985	1985	2314	0	0
1986	1986	2314	0	0
1987	1987	2314	0	0
1988	1988	2314	.	0
1989	1989	2314	.	0
1990	1990	2314	.	0
1991	1991	2314	.	0
1992	1992	2314	0	0
1993	1993	2314	.	0
1994	1994	2314	0	0
1995	1995	2314	.	0
1996	1996	2314	0	0
1997	1997	2314	.	0
1998	1998	2314	.	0
1999	1999	2314	0	0
2000	2000	2314	0	0
2001	2001	2314	0	0
2002	2002	2314		0
2003	2003	2314		0
2004	2004	2314	0	0
2005	2005	2314		0
2006	2006	2314		0
2007	2007	2314		0
2008	2008	2314		0
2009	2009	2314		0
2010	2010	2314		0
2011	2011	2314		0
2012	2012	2314		0
2013	2013	2314		0
2014	2014	2314		0
2015	2015	2314		0
2016	2016	2314		0
2017	2017	2314		0
2018	2018	2314		0
2019	2019	2314		0
2020	2020	2314		0
2021	2021	2314		0
2022	2022	2314		0
2023	2023	2314		0
2024	2024	2314		0
2025	2025	2314		0
2026	2026	2314	.	0
2027	2027	2314	.	0
2028	2028	2314	.	0
2029	2029	2314	.	0
2030	2030	2314	.	0
2031	2031	2314	.	0
2032	2032	2314	.	0
2033	2033	2314	.	0
2034	2034	2314	.	0
2035	2035	2314	.	0
2036	2036	2314	.	0
2037	2037	2314	.	0
2038	2038	2314	.	0
2039	2039	2314	.	0
2040	2040	2314	.	0
2041	2041	2314	.	0
2042	2042	2314	.	0
2043	2043	2314	.	0
2044	2044	2314	.	0
2045	2045	2314	.	0
2046	2046	2314	.	0
2047	2047	2314	35	0
2048	2048	2314	3	0
2049	2049	2314	0	0
2050	2050	2314	1	0
2051	2051	2314	0.4	0
2052	2052	2314	0.5	0
2053	2053	2314		0
2054	2054	2314		0
2055	2055	2314	2	0
2056	2056	2314	0	0
2057	2057	2314	1.3	0
2058	2058	2314	0.6	0
2059	2059	2314	0.6	0
2060	2060	2314	0.8	0
2061	2061	2314	0.6	0
2062	2062	2314	0.7	0
2063	2063	2314		0
2064	2064	2314		0
2065	2065	2314	0	0
2066	2066	2314	12	0
2067	2067	2314	3	0
2068	2068	2314	1	0
2069	2069	2314	6	0
2070	2070	2314	3	0
2071	2071	2314	2	0
2072	2072	2314	1	0
2073	2073	2314	0	0
2074	2074	2314	0	0
2075	2075	2314	0	0
2076	2076	2314	0	0
2077	2077	2314	0	0
2078	2078	2314	0	0
2079	2079	2314	0	0
2080	2080	2314	0	0
2081	2081	2314	B0L5	0
2082	2082	2314	1	0
2083	2083	2314	1.2	0
2084	2084	2314	18.6	0
2085	2085	2314	10	0
2086	2086	2314	18.6	0
2087	2087	2314	28	0
2088	2088	2314	15.8	0
2089	2089	2314	147	0
2090	2090	2314	2.3	0
2091	2091	2314	2.2	0
2092	2092	2314	0	0
2093	2093	2314	0	0
2094	2094	2314	0	0
2095	2095	2314	91	0
2096	2096	2314	175	0
2097	2097	2314	175	0
2098	2098	2314	10	0
2099	2099	2314	20	0
2100	2100	2314	80	0
2101	2101	2314	0	0
2102	2102	2314	2.493333333	0
2103	2103	2314	30.06666667	0
2104	2104	2314	8.366666667	0
2105	2105	2314	500	0
2106	2106	2314	650	0
2107	2107	2314	2.29	0
2108	2108	2314	108	0
2109	2109	2314	48.5	0
2110	2110	2314	4.42	0
2111	2111	2314	38	0
2112	2112	2314	0	0
2113	2113	2314	75	0
2114	2114	2314	0	0
2115	2115	2314	90	0
2116	2116	2314	56	0
2117	2117	2314	71	0
2118	2118	2314	39	0
2119	2119	2314	500	0
2120	2120	2314	650	0
2121	2121	2314	2.29	0
2122	2122	2314	108	0
2123	2123	2314	48.5	0
2124	2124	2314	4.42	0
2125	2125	2314	38	0
2126	2126	2314	0	0
2127	2127	2314	75	0
2128	2128	2314	0	0
2129	2129	2314	90	0
2130	2130	2314	65.1	0
2131	2131	2314	19.3	0
2132	2132	2314	39.7	0
2133	2133	2314	18.6	0
2134	2134	2314	3.3	0
2135	2135	2314	2.3	0
2136	2136	2314	30.63	0
2137	2137	2314	3.69	0
2138	2138	2314	30.6493	0
2139	2139	2314	3.7109	0
2140	2140	2314	88	0
2141	2141	2314	37	0
2142	2142	2314	0	0
2143	2143	2314	1	0
2144	2144	2314		0
2145	2145	2314		0
2146	2146	2314		0
2147	2147	2314		0
2148	2148	2314	2	0
2149	2149	2314	0	0
2150	2150	2314	1	0
2151	2151	2314	0.4	0
2152	2152	2314	0.6	0
2153	2153	2314	1.1	0
2154	2154	2314		0
2155	2155	2314		0
2156	2156	2314		0
2157	2157	2314		0
2158	2158	2314	2	0
2159	2159	2314	8	0
2160	2160	2314	2	0
2161	2161	2314	1	0
2162	2162	2314	5	0
2163	2163	2314	2	0
2164	2164	2314	1	0
2165	2165	2314	0	0
2166	2166	2314	0	0
2167	2167	2314	0	0
2168	2168	2314	0	0
2169	2169	2314	0	0
2170	2170	2314	0	0
2171	2171	2314	0	0
2172	2172	2314	0	0
2173	2173	2314	0	0
2174	2174	2314	B0L4	0
2175	2175	2314	1.1	0
2176	2176	2314	1.4	0
2177	2177	2314	27.3	0
2178	2178	2314	14	0
2179	2179	2314	27.3	0
2180	2180	2314	44	0
2181	2181	2314	17.7	0
2182	2182	2314	256	0
2183	2183	2314	2.8	0
2184	2184	2314	2.4	0
2185	2185	2314	0	0
2186	2186	2314	0	0
2187	2187	2314	0	0
2188	2188	2314	67	0
2189	2189	2314	0	0
2190	2190	2314	350	0
2191	2191	2314	0	0
2192	2192	2314	20	0
2193	2193	2314	0	0
2194	2194	2314	0	0
2195	2195	2314		0
2196	2196	2314		0
2197	2197	2314		0
2198	2198	2314		0
2199	2199	2314		0
2200	2200	2314		0
2201	2201	2314		0
2202	2202	2314		0
2203	2203	2314		0
2204	2204	2314		0
2205	2205	2314		0
2206	2206	2314		0
2207	2207	2314		0
2208	2208	2314		0
2209	2209	2314	84	0
2210	2210	2314		0
2211	2211	2314		0
2212	2212	2314	.	0
2213	2213	2314	.	0
2214	2214	2314	.	0
2215	2215	2314	.	0
2216	2216	2314	.	0
2217	2217	2314	.	0
2218	2218	2314	.	0
2219	2219	2314	.	0
2220	2220	2314	.	0
2221	2221	2314	.	0
2222	2222	2314	.	0
2223	2223	2314	.	0
2224	2224	2314	.	0
2225	2225	2314	.	0
2226	2226	2314	.	0
2227	2227	2314	.	0
2228	2228	2314	.	0
2229	2229	2314	19.9	0
2230	2230	2314	0.49	0
2231	2231	2314	19.9	0
2232	2232	2314	0.49	0
2233	2233	2314	18	0
2234	2234	2314	1	0
2235	2235	2314	1	0
2236	2236	2314	2	0
2237	2237	2314	0.6	0
2238	2238	2314	0.8	0
2239	2239	2314	0.7	0
2240	2240	2314	0.8	0
2241	2241	2314	3	0
2242	2242	2314	0	0
2243	2243	2314	0.8	0
2244	2244	2314	0.5	0
2245	2245	2314	0.6	0
2246	2246	2314	0.7	0
2247	2247	2314	0.6	0
2248	2248	2314	0.7	0
2249	2249	2314	0.7	0
2250	2250	2314	0.7	0
2251	2251	2314	1	0
2252	2252	2314	17	0
2253	2253	2314	4	0
2254	2254	2314	1	0
2255	2255	2314	8	0
2256	2256	2314	3	0
2257	2257	2314	4	0
2258	2258	2314	1	0
2259	2259	2314	1	0
2260	2260	2314	0	0
2261	2261	2314	0	0
2262	2262	2314	0	0
2263	2263	2314	0	0
2264	2264	2314	0	0
2265	2265	2314	0	0
2266	2266	2314	0	0
2267	2267	2314	B0L6	0
2268	2268	2314	1.1	0
2269	2269	2314	1.3	0
2270	2270	2314	19.5	0
2271	2271	2314	12	0
2272	2272	2314	19.5	0
2273	2273	2314	51	0
2274	2274	2314	10.5	0
2275	2275	2314	277	0
2276	2276	2314	3	0
2277	2277	2314	2.6	0
2278	2278	2314	5	0
2279	2279	2314	13	0
2280	2280	2314	0	0
2281	2281	2314	72	0
2282	2282	2314	300	0
2283	2283	2314	300	0
2284	2284	2314	10	0
2285	2285	2314	50	0
2286	2286	2314	95	0
2287	2287	2314	0	0
2288	2288	2314	2.52	0
2289	2289	2314	28.73333333	0
2290	2290	2314	8.633333333	0
2291	2291	2314	275	0
2292	2292	2314	275	0
2293	2293	2314	1.93	0
2294	2294	2314	87	0
2295	2295	2314	40	0
2296	2296	2314	3.39	0
2297	2297	2314	0	0
2298	2298	2314	0	0
2299	2299	2314	85	0
2300	2300	2314	20	0
2301	2301	2314	0	0
2302	2302	2314	173	0
2303	2303	2314	79	0
2304	2304	2314	28	0
2305	2305	2314	275	0
2306	2306	2314	275	0
2307	2307	2314	1.93	0
2308	2308	2314	87	0
2309	2309	2314	40	0
2310	2310	2314	3.39	0
2311	2311	2314	0	0
2312	2312	2314	0	0
2313	2313	2314	85	0
2314	2314	2314	20	0
2315	2315	2314	0	0
2316	2316	2314	45.9	0
2317	2317	2314	16.3	0
2318	2318	2314	123.2	0
2319	2319	2314	45.8	0
2320	2320	2314	6.7	0
2321	2321	2314	4.6	0
2322	2322	2314	10.53	0
2323	2323	2314	3.24	0
2324	2324	2314	10.5463	0
2325	2325	2314	3.2904	0
2326	2326	2314	30	0
2327	2327	2314	1	0
2328	2328	2314	1	0
2329	2329	2314	2	0
2330	2330	2314	0.6	0
2331	2331	2314	0.9	0
2332	2332	2314	0.8	0
2333	2333	2314	0.9	0
2334	2334	2314	3	0
2335	2335	2314	0	0
2336	2336	2314	0.9	0
2337	2337	2314	0.4	0
2338	2338	2314	0.7	0
2339	2339	2314	0.9	0
2340	2340	2314	0.6	0
2341	2341	2314	0.9	0
2342	2342	2314		0
2343	2343	2314		0
2344	2344	2314	0	0
2345	2345	2314	18	0
2346	2346	2314	4	0
2347	2347	2314	1	0
2348	2348	2314	8	0
2349	2349	2314	4	0
2350	2350	2314	3	0
2351	2351	2314	2	0
2352	2352	2314	1	0
2353	2353	2314	0	0
2354	2354	2314	0	0
2355	2355	2314	0	0
2356	2356	2314	0	0
2357	2357	2314	0	0
2358	2358	2314	0	0
2359	2359	2314	0	0
2360	2360	2314	B0L6	0
2361	2361	2314	1.2	0
2362	2362	2314	1.2	0
2363	2363	2314	25.2	0
2364	2364	2314	15	0
2365	2365	2314	34.3	0
2366	2366	2314	53	0
2367	2367	2314	17	0
2368	2368	2314	142	0
2369	2369	2314	2.3	0
2370	2370	2314	1.9	0
2371	2371	2314	0	0
2372	2372	2314	0	0
2373	2373	2314	0	0
2374	2374	2314	57	0
2375	2375	2314	200	0
2376	2376	2314	200	0
2377	2377	2314	0	0
2378	2378	2314	50	0
2379	2379	2314	95	0
2380	2380	2314	0	0
2381	2381	2314	2	0
2382	2382	2314	24.23333333	0
2383	2383	2314	6.783333333	0
2384	2384	2314	175	0
2385	2385	2314	175	0
2386	2386	2314	2.38	0
2387	2387	2314	67	0
2388	2388	2314	37	0
2389	2389	2314	4.13	0
2390	2390	2314	0	0
2391	2391	2314	0	0
2392	2392	2314	90	0
2393	2393	2314	0	0
2394	2394	2314	0	0
2395	2395	2314	140	0
2396	2396	2314	56	0
2397	2397	2314	3	0
2398	2398	2314	175	0
2399	2399	2314	175	0
2400	2400	2314	2.38	0
2401	2401	2314	67	0
2402	2402	2314	37	0
2403	2403	2314	4.13	0
2404	2404	2314	0	0
2405	2405	2314	0	0
2406	2406	2314	90	0
2407	2407	2314	0	0
2408	2408	2314	0	0
2409	2409	2314	71.3	0
2410	2410	2314	37.3	0
2411	2411	2314	171.1	0
2412	2412	2314	84.4	0
2413	2413	2314	8.4	0
2414	2414	2314	7.5	0
2415	2415	2314	9.5	0
2416	2416	2314	1.49	0
2417	2417	2314	9.5373	0
2418	2418	2314	1.5819	0
2419	2419	2314	32	0
2420	2420	2314	13	0
2421	2421	2314	0	0
2422	2422	2314	1	0
2423	2423	2314	0.4	0
2424	2424	2314	0.6	0
2425	2425	2314		0
2426	2426	2314		0
2427	2427	2314	2	0
2428	2428	2314	0	0
2429	2429	2314	0.9	0
2430	2430	2314	0.5	0
2431	2431	2314	0.6	0
2432	2432	2314	0.9	0
2433	2433	2314	0.6	0
2434	2434	2314	0.9	0
2435	2435	2314		0
2436	2436	2314		0
2437	2437	2314	1	0
2438	2438	2314	12	0
2439	2439	2314	3	0
2440	2440	2314	1	0
2441	2441	2314	6	0
2442	2442	2314	3	0
2443	2443	2314	2	0
2444	2444	2314	1	0
2445	2445	2314	0	0
2446	2446	2314	0	0
2447	2447	2314	0	0
2448	2448	2314	0	0
2449	2449	2314	0	0
2450	2450	2314	0	0
2451	2451	2314	0	0
2452	2452	2314	0	0
2453	2453	2314	B0L4	0
2454	2454	2314	1	0
2455	2455	2314	1.3	0
2456	2456	2314	28.3	0
2457	2457	2314	15	0
2458	2458	2314	45.1	0
2459	2459	2314	32	0
2460	2460	2314	8.2	0
2461	2461	2314	101	0
2462	2462	2314	2.6	0
2463	2463	2314	2.4	0
2464	2464	2314	0	0
2465	2465	2314	0	0
2466	2466	2314	0	0
2467	2467	2314	115.5	0
2468	2468	2314	250	0
2469	2469	2314	250	0
2470	2470	2314	15	0
2471	2471	2314	20	0
2472	2472	2314	95	0
2473	2473	2314	0	0
2474	2474	2314	3.061666667	0
2475	2475	2314	34.55	0
2476	2476	2314	8.783333333	0
2477	2477	2314	100	0
2478	2478	2314	100	0
2479	2479	2314	2.41	0
2480	2480	2314	101	0
2481	2481	2314	38	0
2482	2482	2314	3.88	0
2483	2483	2314	0	0
2484	2484	2314	0	0
2485	2485	2314	60	0
2486	2486	2314	0	0
2487	2487	2314	0	0
2488	2488	2314	113	0
2489	2489	2314	49	0
2490	2490	2314	1	0
2491	2491	2314	100	0
2492	2492	2314	100	0
2493	2493	2314	2.41	0
2494	2494	2314	101	0
2495	2495	2314	38	0
2496	2496	2314	3.88	0
2497	2497	2314	0	0
2498	2498	2314	0	0
2499	2499	2314	60	0
2500	2500	2314	0	0
2501	2501	2314	0	0
2502	2502	2314	52.4	0
2503	2503	2314	18.7	0
2504	2504	2314	82.1	0
2505	2505	2314	37	0
2506	2506	2314	14.2	0
2507	2507	2314	11.5	0
2508	2508	2314	12.17	0
2509	2509	2314	1.26	0
2510	2510	2314	12.1887	0
2511	2511	2314	1.3085	0
2512	2512	2314	6	0
2513	2513	2314	8	0
2514	2514	2314	1	0
2515	2515	2314	2	0
2516	2516	2314	0.4	0
2517	2517	2314	0.4	0
2518	2518	2314	0.8	0
2519	2519	2314	0.7	0
2520	2520	2314	3	0
2521	2521	2314	0	0
2522	2522	2314	0.9	0
2523	2523	2314	0.3	0
2524	2524	2314	0.4	0
2525	2525	2314	0.5	0
2526	2526	2314	0.6	0
2527	2527	2314	0.6	0
2528	2528	2314	0.5	0
2529	2529	2314	0.5	0
2530	2530	2314	0	0
2531	2531	2314	16	0
2532	2532	2314	4	0
2533	2533	2314	1	0
2534	2534	2314	7	0
2535	2535	2314	3	0
2536	2536	2314	3	0
2537	2537	2314	2	0
2538	2538	2314	1	0
2539	2539	2314	0	0
2540	2540	2314	0	0
2541	2541	2314	0	0
2542	2542	2314	0	0
2543	2543	2314	0	0
2544	2544	2314	0	0
2545	2545	2314	0	0
2546	2546	2314	B0L6	0
2547	2547	2314	0.7	0
2548	2548	2314	0.9	0
2549	2549	2314	16.5	0
2550	2550	2314	4	0
2551	2551	2314	32.3	0
2552	2552	2314	28	0
2553	2553	2314	3.7	0
2554	2554	2314	91	0
2555	2555	2314	2.1	0
2556	2556	2314	2	0
2557	2557	2314	0	0
2558	2558	2314	0	0
2559	2559	2314	0	0
2560	2560	2314	75	0
2561	2561	2314	150	0
2562	2562	2314	150	0
2563	2563	2314	0	0
2564	2564	2314	30	0
2565	2565	2314	90	0
2566	2566	2314	0	0
2567	2567	2314	1.901666667	0
2568	2568	2314	19.9	0
2569	2569	2314	5.616666667	0
2570	2570	2314	50	0
2571	2571	2314	50	0
2572	2572	2314		0
2573	2573	2314	70	0
2574	2574	2314	18	0
2575	2575	2314		0
2576	2576	2314	0	0
2577	2577	2314	0	0
2578	2578	2314	0	0
2579	2579	2314	0	0
2580	2580	2314	0	0
2581	2581	2314	65	0
2582	2582	2314	0	0
2583	2583	2314	0	0
2584	2584	2314	50	0
2585	2585	2314	50	0
2586	2586	2314	.	0
2587	2587	2314	70	0
2588	2588	2314	18	0
2589	2589	2314	.	0
2590	2590	2314	0	0
2591	2591	2314	0	0
2592	2592	2314	0	0
2593	2593	2314	0	0
2594	2594	2314	0	0
2595	2595	2314	.	0
2596	2596	2314	.	0
2597	2597	2314	.	0
2598	2598	2314	.	0
2599	2599	2314	.	0
2600	2600	2314	.	0
2601	2601	2314	4.77	0
2602	2602	2314	0.34	0
2603	2603	2314	4.77	0
2604	2604	2314	0.34	0
2605	2605	2314	33	0
2606	2606	2314	7	0
2607	2607	2314	1	0
2608	2608	2314	2	0
2609	2609	2314	0.6	0
2610	2610	2314	0.8	0
2611	2611	2314	0.4	0
2612	2612	2314	0.4	0
2613	2613	2314	3	0
2614	2614	2314	0	0
2615	2615	2314	0.6	0
2616	2616	2314	0.4	0
2617	2617	2314	0.7	0
2618	2618	2314	0.8	0
2619	2619	2314	0.4	0
2620	2620	2314	0.5	0
2621	2621	2314	0.5	0
2622	2622	2314	0.6	0
2623	2623	2314	2	0
2624	2624	2314	14	0
2625	2625	2314	3	0
2626	2626	2314	2	0
2627	2627	2314	8	0
2628	2628	2314	2	0
2629	2629	2314	2	0
2630	2630	2314	2	0
2631	2631	2314	0	0
2632	2632	2314	0	0
2633	2633	2314	0	0
2634	2634	2314	0	0
2635	2635	2314	0	0
2636	2636	2314	0	0
2637	2637	2314	0	0
2638	2638	2314	0	0
2639	2639	2314	B0L6	0
2640	2640	2314	1.2	0
2641	2641	2314	1.6	0
2642	2642	2314	25.7	0
2643	2643	2314	14	0
2644	2644	2314	25.7	0
2645	2645	2314	48	0
2646	2646	2314	21	0
2647	2647	2314	215	0
2648	2648	2314	2.8	0
2649	2649	2314	2.2	0
2650	2650	2314	0	0
2651	2651	2314	0	0
2652	2652	2314	0	0
2653	2653	2314	84	0
2654	2654	2314	150	0
2655	2655	2314	250	0
2656	2656	2314	10	0
2657	2657	2314	30	0
2658	2658	2314	60	0
2659	2659	2314	2	0
2660	2660	2314	1.185	0
2661	2661	2314	16.83333333	0
2662	2662	2314	4.783333333	0
2663	2663	2314	150	0
2664	2664	2314	400	0
2665	2665	2314	2.64	0
2666	2666	2314	67.5	0
2667	2667	2314	36.5	0
2668	2668	2314	4.62	0
2669	2669	2314	0	0
2670	2670	2314	0	0
2671	2671	2314	30	0
2672	2672	2314	2	0
2673	2673	2314	97	0
2674	2674	2314	115	0
2675	2675	2314	7	0
2676	2676	2314	5	0
2677	2677	2314	150	0
2678	2678	2314	400	0
2679	2679	2314	2.64	0
2680	2680	2314	67.5	0
2681	2681	2314	36.5	0
2682	2682	2314	4.62	0
2683	2683	2314	0	0
2684	2684	2314	0	0
2685	2685	2314	30	0
2686	2686	2314	2	0
2687	2687	2314	97	0
2688	2688	2314	99.8	0
2689	2689	2314	29	0
2690	2690	2314	129.8	0
2691	2691	2314	43.6	0
2692	2692	2314	28.7	0
2693	2693	2314	10.6	0
2694	2694	2314	12.68	0
2695	2695	2314	1.39	0
2696	2696	2314	12.709	0
2697	2697	2314	1.4442	0
2698	2698	2314	17	0
2699	2699	2314	14	0
2700	2700	2314	1	0
2701	2701	2314	2	0
2702	2702	2314	0.6	0
2703	2703	2314	0.7	0
2704	2704	2314	1	0
2705	2705	2314	1	0
2706	2706	2314	4	0
2707	2707	2314	0	0
2708	2708	2314	1.3	0
2709	2709	2314	0.5	0
2710	2710	2314	0.6	0
2711	2711	2314	0.8	0
2712	2712	2314	0.9	0
2713	2713	2314	1.1	0
2714	2714	2314	0.7	0
2715	2715	2314	0.9	0
2716	2716	2314	2	0
2717	2717	2314	21	0
2718	2718	2314	5	0
2719	2719	2314	1	0
2720	2720	2314	8	0
2721	2721	2314	4	0
2722	2722	2314	4	0
2723	2723	2314	3	0
2724	2724	2314	1	0
2725	2725	2314	1	0
2726	2726	2314	0	0
2727	2727	2314	0	0
2728	2728	2314	0	0
2729	2729	2314	0	0
2730	2730	2314	0	0
2731	2731	2314	0	0
2732	2732	2314	B0L7	0
2733	2733	2314	1.4	0
2734	2734	2314	1.6	0
2735	2735	2314	28.6	0
2736	2736	2314	15	0
2737	2737	2314	33.9	0
2738	2738	2314	31	0
2739	2739	2314	17.1	0
2740	2740	2314	194	0
2741	2741	2314	1.8	0
2742	2742	2314	1.6	0
2743	2743	2314	0	0
2744	2744	2314	0	0
2745	2745	2314	0	0
2746	2746	2314	57.5	0
2747	2747	2314	200	0
2748	2748	2314	200	0
2749	2749	2314	5	0
2750	2750	2314	30	0
2751	2751	2314	95	0
2752	2752	2314	0	0
2753	2753	2314	1.15	0
2754	2754	2314	13.45	0
2755	2755	2314	3.8	0
2756	2756	2314	75	0
2757	2757	2314	100	0
2758	2758	2314	2.48	0
2759	2759	2314	79	0
2760	2760	2314	36	0
2761	2761	2314	3.54	0
2762	2762	2314	0	0
2763	2763	2314	0	0
2764	2764	2314	40	0
2765	2765	2314	0	0
2766	2766	2314	0	0
2767	2767	2314	91	0
2768	2768	2314	33	0
2769	2769	2314	4	0
2770	2770	2314	75	0
2771	2771	2314	100	0
2772	2772	2314	2.48	0
2773	2773	2314	79	0
2774	2774	2314	36	0
2775	2775	2314	3.54	0
2776	2776	2314	0	0
2777	2777	2314	0	0
2778	2778	2314	40	0
2779	2779	2314	0	0
2780	2780	2314	0	0
2781	2781	2314	75.5	0
2782	2782	2314	32.8	0
2783	2783	2314	87.5	0
2784	2784	2314	46.6	0
2785	2785	2314	6.5	0
2786	2786	2314	6.5	0
2787	2787	2314	11.13	0
2788	2788	2314	1.37	0
2789	2789	2314	11.1628	0
2790	2790	2314	1.4231	0
2791	2791	2314	8	0
2792	2792	2314	12	0
2793	2793	2314	0	0
2794	2794	2314	1	0
2795	2795	2314	0.5	0
2796	2796	2314	0.7	0
2797	2797	2314		0
2798	2798	2314		0
2799	2799	2314	2	0
2800	2800	2314	0	0
2801	2801	2314	0.5	0
2802	2802	2314	0.9	0
2803	2803	2314	0.8	0
2804	2804	2314	0.1	0
2805	2805	2314	0.5	0
2806	2806	2314	0.7	0
2807	2807	2314		0
2808	2808	2314		0
2809	2809	2314	0	0
2810	2810	2314	9	0
2811	2811	2314	2	0
2812	2812	2314	2	0
2813	2813	2314	6	0
2814	2814	2314	2	0
2815	2815	2314	1	0
2816	2816	2314	0	0
2817	2817	2314	0	0
2818	2818	2314	0	0
2819	2819	2314	0	0
2820	2820	2314	0	0
2821	2821	2314	0	0
2822	2822	2314	0	0
2823	2823	2314	0	0
2824	2824	2314	0	0
2825	2825	2314	B0L5	0
2826	2826	2314	1.3	0
2827	2827	2314	1.55	0
2828	2828	2314	29.4	0
2829	2829	2314	12	0
2830	2830	2314	31.7	0
2831	2831	2314	39	0
2832	2832	2314	16.6	0
2833	2833	2314	91	0
2834	2834	2314	3.1	0
2835	2835	2314	2.3	0
2836	2836	2314	0	0
2837	2837	2314	0	0
2838	2838	2314	0	0
2839	2839	2314	70	0
2840	2840	2314	500	0
2841	2841	2314	500	0
2842	2842	2314	50	0
2843	2843	2314	40	0
2844	2844	2314	100	0
2845	2845	2314	0	0
2846	2846	2314	4.445	0
2847	2847	2314	58.75	0
2848	2848	2314	14.83333333	0
2849	2849	2314	650	0
2850	2850	2314	650	0
2851	2851	2314	3.13	0
2852	2852	2314	110	0
2853	2853	2314	76	0
2854	2854	2314	5.68	0
2855	2855	2314	0	0
2856	2856	2314	0	0
2857	2857	2314	90	0
2858	2858	2314	0	0
2859	2859	2314	0	0
2860	2860	2314	72	0
2861	2861	2314	293	0
2862	2862	2314	120	0
2863	2863	2314	650	0
2864	2864	2314	650	0
2865	2865	2314	3.13	0
2866	2866	2314	110	0
2867	2867	2314	76	0
2868	2868	2314	5.68	0
2869	2869	2314	0	0
2870	2870	2314	0	0
2871	2871	2314	90	0
2872	2872	2314	0	0
2873	2873	2314	0	0
2874	2874	2314	92.1	0
2875	2875	2314	41.4	0
2876	2876	2314	599.5	0
2877	2877	2314	215.1	0
2878	2878	2314	11.6	0
2879	2879	2314	10	0
2880	2880	2314	76.22	0
2881	2881	2314	11.6	0
2882	2882	2314	76.2614	0
2883	2883	2314	11.8251	0
2884	2884	2314	2	0
2885	2885	2314	2	0
2886	2886	2314	1	0
2887	2887	2314	2	0
2888	2888	2314	0.9	0
2889	2889	2314	1.4	0
2890	2890	2314		0
2891	2891	2314		0
2892	2892	2314	3	0
2893	2893	2314	0	0
2894	2894	2314	1	0
2895	2895	2314	0.5	0
2896	2896	2314	1.1	0
2897	2897	2314	1.5	0
2898	2898	2314	1	0
2899	2899	2314	1	0
2900	2900	2314		0
2901	2901	2314		0
2902	2902	2314	2	0
2903	2903	2314	17	0
2904	2904	2314	4	0
2905	2905	2314	1	0
2906	2906	2314	7	0
2907	2907	2314	4	0
2908	2908	2314	3	0
2909	2909	2314	2	0
2910	2910	2314	1	0
2911	2911	2314	0	0
2912	2912	2314	0	0
2913	2913	2314	0	0
2914	2914	2314	0	0
2915	2915	2314	0	0
2916	2916	2314	0	0
2917	2917	2314	0	0
2918	2918	2314	B0L6	0
2919	2919	2314	1.3	0
2920	2920	2314	1.2	0
2921	2921	2314	39.6	0
2922	2922	2314	15	0
2923	2923	2314	41.3	0
2924	2924	2314	42	0
2925	2925	2314	17.1	0
2926	2926	2314	209	0
2927	2927	2314	2.7	0
2928	2928	2314	2	0
2929	2929	2314	0	0
2930	2930	2314	2	0
2931	2931	2314	0	0
2932	2932	2314	69	0
2933	2933	2314	300	0
2934	2934	2314	300	0
2935	2935	2314	5	0
2936	2936	2314	20	0
2937	2937	2314	95	0
2938	2938	2314	0	0
2939	2939	2314	2.308333333	0
2940	2940	2314	27.81666667	0
2941	2941	2314	7.916666667	0
2942	2942	2314	300	0
2943	2943	2314	300	0
2944	2944	2314	3.3	0
2945	2945	2314	105	0
2946	2946	2314	58	0
2947	2947	2314	4.93	0
2948	2948	2314	0	0
2949	2949	2314	0	0
2950	2950	2314	90	0
2951	2951	2314	0	0
2952	2952	2314	30	0
2953	2953	2314	15	0
2954	2954	2314	37	0
2955	2955	2314	18	0
2956	2956	2314	300	0
2957	2957	2314	300	0
2958	2958	2314	3.3	0
2959	2959	2314	105	0
2960	2960	2314	58	0
2961	2961	2314	4.93	0
2962	2962	2314	0	0
2963	2963	2314	0	0
2964	2964	2314	90	0
2965	2965	2314	0	0
2966	2966	2314	30	0
2967	2967	2314	91.7	0
2968	2968	2314	60.5	0
2969	2969	2314	218.1	0
2970	2970	2314	109.9	0
2971	2971	2314	6.2	0
2972	2972	2314	6	0
2973	2973	2314	38.18	0
2974	2974	2314	4.03	0
2975	2975	2314	38.2405	0
2976	2976	2314	4.1459	0
2977	2977	2314	43	0
2978	2978	2314	8	0
2979	2979	2314	1	0
2980	2980	2314	2	0
2981	2981	2314	0.5	0
2982	2982	2314	0.8	0
2983	2983	2314		0
2984	2984	2314		0
2985	2985	2314	3	0
2986	2986	2314	0	0
2987	2987	2314	0.9	0
2988	2988	2314	0.4	0
2989	2989	2314	0.7	0
2990	2990	2314	1	0
2991	2991	2314	0.5	0
2992	2992	2314	0.7	0
2993	2993	2314		0
2994	2994	2314		0
2995	2995	2314	2	0
2996	2996	2314	11	0
2997	2997	2314	3	0
2998	2998	2314	1	0
2999	2999	2314	6	0
3000	3000	2314	2	0
3001	3001	2314	2	0
3002	3002	2314	1	0
3003	3003	2314	0	0
3004	3004	2314	0	0
3005	3005	2314	0	0
3006	3006	2314	0	0
3007	3007	2314	0	0
3008	3008	2314	0	0
3009	3009	2314	0	0
3010	3010	2314	0	0
3011	3011	2314	B0L5	0
3012	3012	2314	1.1	0
3013	3013	2314	1.4	0
3014	3014	2314	26.4	0
3015	3015	2314	13	0
3016	3016	2314	26.4	0
3017	3017	2314	49	0
3018	3018	2314	29.2	0
3019	3019	2314	221	0
3020	3020	2314	3	0
3021	3021	2314	2.4	0
3022	3022	2314	0	0
3023	3023	2314	0	0
3024	3024	2314	0	0
3025	3025	2314	84	0
3026	3026	2314	100	0
3027	3027	2314	300	0
3028	3028	2314	10	0
3029	3029	2314	30	0
3030	3030	2314	50	0
3031	3031	2314	0	0
3032	3032	2314	1.333333333	0
3033	3033	2314	17.78333333	0
3034	3034	2314	4.55	0
3035	3035	2314	0	0
3036	3036	2314	550	0
3037	3037	2314	2.5	0
3038	3038	2314	58	0
3039	3039	2314	30	0
3040	3040	2314	3.09	0
3041	3041	2314	0	0
3042	3042	2314	0	0
3043	3043	2314	0	0
3044	3044	2314	0	0
3045	3045	2314	0	0
3046	3046	2314	132	0
3047	3047	2314	0	0
3048	3048	2314	0	0
3049	3049	2314	0	0
3050	3050	2314	550	0
3051	3051	2314	2.5	0
3052	3052	2314	58	0
3053	3053	2314	30	0
3054	3054	2314	3.09	0
3055	3055	2314	0	0
3056	3056	2314	0	0
3057	3057	2314	0	0
3058	3058	2314	0	0
3059	3059	2314	0	0
3060	3060	2314	23.5	0
3061	3061	2314	21.5	0
3062	3062	2314	26.8	0
3063	3063	2314	20.7	0
3064	3064	2314	3.3	0
3065	3065	2314	3	0
3066	3066	2314	18.74	0
3067	3067	2314	0.78	0
3068	3068	2314	18.7615	0
3069	3069	2314	0.8037	0
3070	3070	2314	13	0
3071	3071	2314		0
3072	3072	2314	1	0
3073	3073	2314	2	0
3074	3074	2314	0.4	0
3075	3075	2314	0.8	0
3076	3076	2314		0
3077	3077	2314		0
3078	3078	2314	2	0
3079	3079	2314	0	0
3080	3080	2314	0.7	0
3081	3081	2314	0.3	0
3082	3082	2314	0.5	0
3083	3083	2314	0.8	0
3084	3084	2314	0.5	0
3085	3085	2314	0.7	0
3086	3086	2314		0
3087	3087	2314		0
3088	3088	2314	1	0
3089	3089	2314	10	0
3090	3090	2314	2	0
3091	3091	2314	1	0
3092	3092	2314	6	0
3093	3093	2314	2	0
3094	3094	2314	2	0
3095	3095	2314	0	0
3096	3096	2314	0	0
3097	3097	2314	0	0
3098	3098	2314	0	0
3099	3099	2314	0	0
3100	3100	2314	0	0
3101	3101	2314	0	0
3102	3102	2314	0	0
3103	3103	2314	0	0
3104	3104	2314	B0L5	0
3105	3105	2314	0.9	0
3106	3106	2314	0.7	0
3107	3107	2314	30.4	0
3108	3108	2314	11	0
3109	3109	2314	40.6	0
3110	3110	2314	28	0
3111	3111	2314	7.2	0
3112	3112	2314	94	0
3113	3113	2314	2.9	0
3114	3114	2314	2.5	0
3115	3115	2314	0	0
3116	3116	2314	0	0
3117	3117	2314	0	0
3118	3118	2314	66	0
3119	3119	2314	175	0
3120	3120	2314	175	0
3121	3121	2314	20	0
3122	3122	2314	30	0
3123	3123	2314	95	0
3124	3124	2314	0	0
3125	3125	2314	2.4	0
3126	3126	2314	25.76666667	0
3127	3127	2314	6.65	0
3128	3128	2314	225	0
3129	3129	2314	225	0
3130	3130	2314	2.32	0
3131	3131	2314	86	0
3132	3132	2314	66	0
3133	3133	2314	2.83	0
3134	3134	2314	0	0
3135	3135	2314	0	0
3136	3136	2314	80	0
3137	3137	2314	0	0
3138	3138	2314	0	0
3139	3139	2314	64	0
3140	3140	2314	64	0
3141	3141	2314	2	0
3142	3142	2314	225	0
3143	3143	2314	225	0
3144	3144	2314	2.32	0
3145	3145	2314	86	0
3146	3146	2314	66	0
3147	3147	2314	2.83	0
3148	3148	2314	0	0
3149	3149	2314	0	0
3150	3150	2314	80	0
3151	3151	2314	0	0
3152	3152	2314	0	0
3153	3153	2314	50.7	0
3154	3154	2314	13.2	0
3155	3155	2314	58.8	0
3156	3156	2314	38.6	0
3157	3157	2314	4.7	0
3158	3158	2314	4.4	0
3159	3159	2314	10.91	0
3160	3160	2314	2.02	0
3161	3161	2314	10.9232	0
3162	3162	2314	2.063	0
3163	3163	2314	11	0
3164	3164	2314	3	0
3165	3165	2314	1	0
3166	3166	2314	2	0
3167	3167	2314	0.3	0
3168	3168	2314	0.4	0
3169	3169	2314		0
3170	3170	2314		0
3171	3171	2314	1	0
3172	3172	2314	0	0
3173	3173	2314	0.7	0
3174	3174	2314	0.4	0
3175	3175	2314	.	0
3176	3176	2314	.	0
3177	3177	2314		0
3178	3178	2314		0
3179	3179	2314		0
3180	3180	2314		0
3181	3181	2314	1	0
3182	3182	2314	0	0
3183	3183	2314	0	0
3184	3184	2314	.	0
3185	3185	2314		0
3186	3186	2314	0	0
3187	3187	2314	0	0
3188	3188	2314	0	0
3189	3189	2314	0	0
3190	3190	2314	0	0
3191	3191	2314	0	0
3192	3192	2314	0	0
3193	3193	2314	0	0
3194	3194	2314	0	0
3195	3195	2314	0	0
3196	3196	2314	0	0
3197	3197	2314	B0L4	0
3198	3198	2314	0.5	0
3199	3199	2314	0.7	0
3200	3200	2314	18.7	0
3201	3201	2314	3	0
3202	3202	2314	18.7	0
3203	3203	2314	0	0
3204	3204	2314	.	0
3205	3205	2314	22	0
3206	3206	2314	1.7	0
3207	3207	2314	1.9	0
3208	3208	2314	0	0
3209	3209	2314	0	0
3210	3210	2314	0	0
3211	3211	2314	111	0
3212	3212	2314	60	0
3213	3213	2314	100	0
3214	3214	2314	2	0
3215	3215	2314	25	0
3216	3216	2314	70	0
3217	3217	2314	0	0
3218	3218	2314	2.261666667	0
3219	3219	2314	26.33333333	0
3220	3220	2314	7.75	0
3221	3221	2314	225	0
3222	3222	2314	250	0
3223	3223	2314	2.88	0
3224	3224	2314	106	0
3225	3225	2314	40	0
3226	3226	2314	3.87	0
3227	3227	2314	0	0
3228	3228	2314	0	0
3229	3229	2314	80	0
3230	3230	2314	15	0
3231	3231	2314	0	0
3232	3232	2314	37	0
3233	3233	2314	46	0
3234	3234	2314	53	0
3235	3235	2314	225	0
3236	3236	2314	250	0
3237	3237	2314	2.88	0
3238	3238	2314	106	0
3239	3239	2314	40	0
3240	3240	2314	3.87	0
3241	3241	2314	0	0
3242	3242	2314	0	0
3243	3243	2314	80	0
3244	3244	2314	15	0
3245	3245	2314	0	0
3246	3246	2314	101.2	0
3247	3247	2314	34	0
3248	3248	2314	157.7	0
3249	3249	2314	45	0
3250	3250	2314	3.4	0
3251	3251	2314	2.9	0
3252	3252	2314	10.75	0
3253	3253	2314	1.07	0
3254	3254	2314	10.784	0
3255	3255	2314	1.1179	0
3256	3256	2314	28	0
3257	3257	2314	1	0
3258	3258	2314	1	0
3259	3259	2314	3	0
3260	3260	2314	0.5	0
3261	3261	2314	0.7	0
3262	3262	2314	1	0
3263	3263	2314	1.3	0
3264	3264	2314	3	0
3265	3265	2314	0	0
3266	3266	2314	1	0
3267	3267	2314	0.6	0
3268	3268	2314	0.4	0
3269	3269	2314	0.8	0
3270	3270	2314	0.7	0
3271	3271	2314	1	0
3272	3272	2314	0.7	0
3273	3273	2314	1	0
3274	3274	2314	2	0
3275	3275	2314	22	0
3276	3276	2314	5	0
3277	3277	2314	1	0
3278	3278	2314	8	0
3279	3279	2314	4	0
3280	3280	2314	4	0
3281	3281	2314	3	0
3282	3282	2314	2	0
3283	3283	2314	1	0
3284	3284	2314	0	0
3285	3285	2314	0	0
3286	3286	2314	0	0
3287	3287	2314	0	0
3288	3288	2314	0	0
3289	3289	2314	0	0
3290	3290	2314	B0L6	0
3291	3291	2314	1.1	0
3292	3292	2314	1.4	0
3293	3293	2314	32.2	0
3294	3294	2314	16	0
3295	3295	2314	32.2	0
3296	3296	2314	43	0
3297	3297	2314	30.2	0
3298	3298	2314	361	0
3299	3299	2314	3.2	0
3300	3300	2314	2.3	0
3301	3301	2314	25	0
3302	3302	2314	18	0
3303	3303	2314	9	0
3304	3304	2314	81	0
3305	3305	2314	150	0
3306	3306	2314	150	0
3307	3307	2314	15	0
3308	3308	2314	25	0
3309	3309	2314	80	0
3310	3310	2314	0	0
3311	3311	2314	1.932	0
3312	3312	2314	25.46	0
3313	3313	2314	7.1	0
3314	3314	2314		0
3315	3315	2314		0
3316	3316	2314	2.38	0
3317	3317	2314	99	0
3318	3318	2314	45	0
3319	3319	2314	3.98	0
3320	3320	2314	0	0
3321	3321	2314	0	0
3322	3322	2314	95	0
3323	3323	2314	5	0
3324	3324	2314	0	0
3325	3325	2314	186	0
3326	3326	2314	67	0
3327	3327	2314	34	0
3328	3328	2314		0
3329	3329	2314		0
3330	3330	2314	2.38	0
3331	3331	2314	99	0
3332	3332	2314	45	0
3333	3333	2314	3.98	0
3334	3334	2314	0	0
3335	3335	2314	0	0
3336	3336	2314	95	0
3337	3337	2314	5	0
3338	3338	2314	0	0
3339	3339	2314	79.5	0
3340	3340	2314	31.6	0
3341	3341	2314	163.2	0
3342	3342	2314	64.4	0
3343	3343	2314	5.1	0
3344	3344	2314	3.9	0
3345	3345	2314	11.98	0
3346	3346	2314	2.57	0
3347	3347	2314	12.0116	0
3348	3348	2314	2.6383	0
3349	3349	2314	14	0
3350	3350	2314	2	0
3351	3351	2314	1	0
3352	3352	2314	2	0
3353	3353	2314	0.2	0
3354	3354	2314	0.2	0
3355	3355	2314		0
3356	3356	2314		0
3357	3357	2314	2	0
3358	3358	2314	0	0
3359	3359	2314	0.5	0
3360	3360	2314	0.3	0
3361	3361	2314	0.2	0
3362	3362	2314	0.3	0
3363	3363	2314	0.3	0
3364	3364	2314	0.3	0
3365	3365	2314		0
3366	3366	2314		0
3367	3367	2314	1	0
3368	3368	2314	5	0
3369	3369	2314	0	0
3370	3370	2314	.	0
3371	3371	2314	5	0
3372	3372	2314	0	0
3373	3373	2314	0	0
3374	3374	2314	0	0
3375	3375	2314	0	0
3376	3376	2314	0	0
3377	3377	2314	0	0
3378	3378	2314	0	0
3379	3379	2314	0	0
3380	3380	2314	0	0
3381	3381	2314	0	0
3382	3382	2314	0	0
3383	3383	2314	B0L4	0
3384	3384	2314	0.6	0
3385	3385	2314	0.7	0
3386	3386	2314	22.1	0
3387	3387	2314	6	0
3388	3388	2314	23.2	0
3389	3389	2314	1	0
3390	3390	2314	0.3	0
3391	3391	2314	33	0
3392	3392	2314	1.7	0
3393	3393	2314	1.6	0
3394	3394	2314	0	0
3395	3395	2314	0	0
3396	3396	2314	0	0
3397	3397	2314	53.5	0
3398	3398	2314	100	0
3399	3399	2314	100	0
3400	3400	2314	10	0
3401	3401	2314	15	0
3402	3402	2314	95	0
3403	3403	2314	0	0
3404	3404	2314	2.571666667	0
3405	3405	2314	31	0
3406	3406	2314	6.733333333	0
3407	3407	2314	75	0
3408	3408	2314	75	0
3409	3409	2314	2.44	0
3410	3410	2314	70.5	0
3411	3411	2314	39	0
3412	3412	2314	2.02	0
3413	3413	2314	0	0
3414	3414	2314	0	0
3415	3415	2314	70	0
3416	3416	2314	20	0
3417	3417	2314	0	0
3418	3418	2314	33	0
3419	3419	2314	13	0
3420	3420	2314	9	0
3421	3421	2314	75	0
3422	3422	2314	75	0
3423	3423	2314	2.44	0
3424	3424	2314	70.5	0
3425	3425	2314	39	0
3426	3426	2314	2.02	0
3427	3427	2314	0	0
3428	3428	2314	0	0
3429	3429	2314	70	0
3430	3430	2314	20	0
3431	3431	2314	0	0
3432	3432	2314	57.8	0
3433	3433	2314	24	0
3434	3434	2314	52	0
3435	3435	2314	28	0
3436	3436	2314	7	0
3437	3437	2314	6.8	0
3438	3438	2314	4.26	0
3439	3439	2314	0.55	0
3440	3440	2314	4.284	0
3441	3441	2314	0.5848	0
3442	3442	2314	.	0
3443	3443	2314	.	0
3444	3444	2314	0	0
3445	3445	2314	0	0
3446	3446	2314		0
3447	3447	2314		0
3448	3448	2314		0
3449	3449	2314		0
3450	3450	2314	0	0
3451	3451	2314	0	0
3452	3452	2314	0.5	0
3453	3453	2314	0.3	0
3454	3454	2314		0
3455	3455	2314		0
3456	3456	2314		0
3457	3457	2314		0
3458	3458	2314		0
3459	3459	2314		0
3460	3460	2314	2	0
3461	3461	2314	1	0
3462	3462	2314	0	0
3463	3463	2314	.	0
3464	3464	2314	1	0
3465	3465	2314	0	0
3466	3466	2314	0	0
3467	3467	2314	0	0
3468	3468	2314	0	0
3469	3469	2314	0	0
3470	3470	2314	0	0
3471	3471	2314	0	0
3472	3472	2314	0	0
3473	3473	2314	0	0
3474	3474	2314	0	0
3475	3475	2314	0	0
3476	3476	2314	B0L1	0
3477	3477	2314	0.1	0
3478	3478	2314	0.1	0
3479	3479	2314	.	0
3480	3480	2314	0	0
3481	3481	2314	.	0
3482	3482	2314	0	0
3483	3483	2314	.	0
3484	3484	2314	0	0
3485	3485	2314	0	0
3486	3486	2314	0	0
3487	3487	2314	0	0
3488	3488	2314	0	0
3489	3489	2314	0	0
3490	3490	2314		0
3491	3491	2314		0
3492	3492	2314	0	0
3493	3493	2314		0
3494	3494	2314		0
3495	3495	2314		0
3496	3496	2314		0
3497	3497	2314		0
3498	3498	2314		0
3499	3499	2314		0
3500	3500	2314		0
3501	3501	2314		0
3502	3502	2314		0
3503	3503	2314		0
3504	3504	2314		0
3505	3505	2314		0
3506	3506	2314		0
3507	3507	2314		0
3508	3508	2314		0
3509	3509	2314		0
3510	3510	2314		0
3511	3511	2314		0
3512	3512	2314		0
3513	3513	2314		0
3514	3514	2314	.	0
3515	3515	2314	.	0
3516	3516	2314	.	0
3517	3517	2314	.	0
3518	3518	2314	.	0
3519	3519	2314	.	0
3520	3520	2314	.	0
3521	3521	2314	.	0
3522	3522	2314	.	0
3523	3523	2314	.	0
3524	3524	2314	.	0
3525	3525	2314	.	0
3526	3526	2314	.	0
3527	3527	2314	.	0
3528	3528	2314	.	0
3529	3529	2314	.	0
3530	3530	2314	.	0
3531	3531	2314	.	0
3532	3532	2314	.	0
3533	3533	2314	.	0
3534	3534	2314	.	0
3535	3535	2314	15	0
3536	3536	2314	8	0
3537	3537	2314	1	0
3538	3538	2314	2	0
3539	3539	2314	0.4	0
3540	3540	2314	0.6	0
3541	3541	2314		0
3542	3542	2314		0
3543	3543	2314	2	0
3544	3544	2314	0	0
3545	3545	2314	0.7	0
3546	3546	2314	0.3	0
3547	3547	2314	0.5	0
3548	3548	2314	0.7	0
3549	3549	2314	0.5	0
3550	3550	2314	0.6	0
3551	3551	2314		0
3552	3552	2314		0
3553	3553	2314	2	0
3554	3554	2314	10	0
3555	3555	2314	2	0
3556	3556	2314	1	0
3557	3557	2314	6	0
3558	3558	2314	2	0
3559	3559	2314	2	0
3560	3560	2314	0	0
3561	3561	2314	0	0
3562	3562	2314	0	0
3563	3563	2314	0	0
3564	3564	2314	0	0
3565	3565	2314	0	0
3566	3566	2314	0	0
3567	3567	2314	0	0
3568	3568	2314	0	0
3569	3569	2314	B0L5	0
3570	3570	2314	1	0
3571	3571	2314	1.1	0
3572	3572	2314	25	0
3573	3573	2314	13	0
3574	3574	2314	25	0
3575	3575	2314	50	0
3576	3576	2314	20.1	0
3577	3577	2314	261	0
3578	3578	2314	2.8	0
3579	3579	2314	2.4	0
3580	3580	2314	0	0
3581	3581	2314	0	0
3582	3582	2314	0	0
3583	3583	2314	61	0
3584	3584	2314	200	0
3585	3585	2314	200	0
3586	3586	2314	5	0
3587	3587	2314	50	0
3588	3588	2314	95	0
3589	3589	2314	0	0
3590	3590	2314	1.656666667	0
3591	3591	2314	21.21666667	0
3592	3592	2314	7.016666667	0
3593	3593	2314	275	0
3594	3594	2314	325	0
3595	3595	2314	2.12	0
3596	3596	2314	90.5	0
3597	3597	2314	40.5	0
3598	3598	2314	3.29	0
3599	3599	2314	0	0
3600	3600	2314	0	0
3601	3601	2314	80	0
3602	3602	2314	5	0
3603	3603	2314	0	0
3604	3604	2314	121	0
3605	3605	2314	71	0
3606	3606	2314	12	0
3607	3607	2314	275	0
3608	3608	2314	325	0
3609	3609	2314	2.12	0
3610	3610	2314	90.5	0
3611	3611	2314	40.5	0
3612	3612	2314	3.29	0
3613	3613	2314	0	0
3614	3614	2314	0	0
3615	3615	2314	80	0
3616	3616	2314	5	0
3617	3617	2314	0	0
3618	3618	2314	85.1	0
3619	3619	2314	31.1	0
3620	3620	2314	93.2	0
3621	3621	2314	41.2	0
3622	3622	2314	9.8	0
3623	3623	2314	6.7	0
3624	3624	2314	11.84	0
3625	3625	2314	2.83	0
3626	3626	2314	11.8711	0
3627	3627	2314	2.8779	0
3628	3628	2314	10	0
3629	3629	2314	3	0
3630	3630	2314	0	0
3631	3631	2314	1	0
3632	3632	2314	0.2	0
3633	3633	2314	0.2	0
3634	3634	2314		0
3635	3635	2314		0
3636	3636	2314	1	0
3637	3637	2314	0	0
3638	3638	2314	.	0
3639	3639	2314	.	0
3640	3640	2314	0.2	0
3641	3641	2314	0.3	0
3642	3642	2314		0
3643	3643	2314		0
3644	3644	2314		0
3645	3645	2314		0
3646	3646	2314	0	0
3647	3647	2314	4	0
3648	3648	2314	0	0
3649	3649	2314	.	0
3650	3650	2314	4	0
3651	3651	2314	0	0
3652	3652	2314	0	0
3653	3653	2314	0	0
3654	3654	2314	0	0
3655	3655	2314	0	0
3656	3656	2314	0	0
3657	3657	2314	0	0
3658	3658	2314	0	0
3659	3659	2314	0	0
3660	3660	2314	0	0
3661	3661	2314	0	0
3662	3662	2314	B0L3	0
3663	3663	2314	0.6	0
3664	3664	2314	0.7	0
3665	3665	2314	26.2	0
3666	3666	2314	4	0
3667	3667	2314	26.2	0
3668	3668	2314	0	0
3669	3669	2314	.	0
3670	3670	2314	14	0
3671	3671	2314	2	0
3672	3672	2314	1.6	0
3673	3673	2314	0	0
3674	3674	2314	0	0
3675	3675	2314	0	0
3676	3676	2314	106	0
3677	3677	2314	50	0
3678	3678	2314	50	0
3679	3679	2314	5	0
3680	3680	2314	4	0
3681	3681	2314	75	0
3682	3682	2314	0	0
3683	3683	2314	3.331666667	0
3684	3684	2314	38.36666667	0
3685	3685	2314	9.633333333	0
3686	3686	2314	200	0
3687	3687	2314	350	0
3688	3688	2314	2.52	0
3689	3689	2314	109	0
3690	3690	2314	41.5	0
3691	3691	2314	2.46	0
3692	3692	2314	0	0
3693	3693	2314	0	0
3694	3694	2314	75	0
3695	3695	2314	10	0
3696	3696	2314	80	0
3697	3697	2314	3	0
3698	3698	2314	37	0
3699	3699	2314	10	0
3700	3700	2314	200	0
3701	3701	2314	350	0
3702	3702	2314	2.52	0
3703	3703	2314	109	0
3704	3704	2314	41.5	0
3705	3705	2314	2.46	0
3706	3706	2314	0	0
3707	3707	2314	0	0
3708	3708	2314	75	0
3709	3709	2314	10	0
3710	3710	2314	80	0
3711	3711	2314	101.8	0
3712	3712	2314	26.1	0
3713	3713	2314	30.7	0
3714	3714	2314	11.9	0
3715	3715	2314	3	0
3716	3716	2314	1.6	0
3717	3717	2314	6.3	0
3718	3718	2314	0.52	0
3719	3719	2314	6.3261	0
3720	3720	2314	0.5335	0
3721	3721	2314	42	0
3722	3722	2314	29	0
3723	3723	2314	0	0
3724	3724	2314	0	0
3725	3725	2314		0
3726	3726	2314		0
3727	3727	2314		0
3728	3728	2314		0
3729	3729	2314	2	0
3730	3730	2314	0	0
3731	3731	2314	0.8	0
3732	3732	2314	0.4	0
3733	3733	2314	0.4	0
3734	3734	2314	0.6	0
3735	3735	2314		0
3736	3736	2314		0
3737	3737	2314		0
3738	3738	2314		0
3739	3739	2314	2	0
3740	3740	2314	8	0
3741	3741	2314	2	0
3742	3742	2314	1	0
3743	3743	2314	6	0
3744	3744	2314	1	0
3745	3745	2314	1	0
3746	3746	2314	0	0
3747	3747	2314	0	0
3748	3748	2314	0	0
3749	3749	2314	0	0
3750	3750	2314	0	0
3751	3751	2314	0	0
3752	3752	2314	0	0
3753	3753	2314	0	0
3754	3754	2314	0	0
3755	3755	2314	B0L4	0
3756	3756	2314	1	0
3757	3757	2314	1.25	0
3758	3758	2314	26.2	0
3759	3759	2314	12	0
3760	3760	2314	33	0
3761	3761	2314	26	0
3762	3762	2314	4.9	0
3763	3763	2314	109	0
3764	3764	2314	2.5	0
3765	3765	2314	2.5	0
3766	3766	2314	0	0
3767	3767	2314	0	0
3768	3768	2314	0	0
3769	3769	2314	94	0
3770	3770	2314	275	0
3771	3771	2314	275	0
3772	3772	2314	10	0
3773	3773	2314	20	0
3774	3774	2314	70	0
3775	3775	2314	0	0
3776	3776	2314	1.67	0
3777	3777	2314	16.06666667	0
3778	3778	2314	5.233333333	0
3779	3779	2314	175	0
3780	3780	2314	175	0
3781	3781	2314		0
3782	3782	2314	83	0
3783	3783	2314	28	0
3784	3784	2314		0
3785	3785	2314	0	0
3786	3786	2314	0	0
3787	3787	2314	0	0
3788	3788	2314	0	0
3789	3789	2314	0	0
3790	3790	2314	94	0
3791	3791	2314	0	0
3792	3792	2314	0	0
3793	3793	2314	175	0
3794	3794	2314	175	0
3795	3795	2314	.	0
3796	3796	2314	83	0
3797	3797	2314	28	0
3798	3798	2314	.	0
3799	3799	2314	0	0
3800	3800	2314	0	0
3801	3801	2314	0	0
3802	3802	2314	0	0
3803	3803	2314	0	0
3804	3804	2314	.	0
3805	3805	2314	.	0
3806	3806	2314	.	0
3807	3807	2314	.	0
3808	3808	2314	.	0
3809	3809	2314	.	0
3810	3810	2314	9.55	0
3811	3811	2314	0.37	0
3812	3812	2314	9.55	0
3813	3813	2314	0.37	0
3814	3814	2314	64	0
3815	3815	2314	13	0
3816	3816	2314	1	0
3817	3817	2314	2	0
3818	3818	2314	0.5	0
3819	3819	2314	0.7	0
3820	3820	2314		0
3821	3821	2314		0
3822	3822	2314	3	0
3823	3823	2314	0	0
3824	3824	2314	1.2	0
3825	3825	2314	0.6	0
3826	3826	2314	0.5	0
3827	3827	2314	0.7	0
3828	3828	2314	0.7	0
3829	3829	2314	1	0
3830	3830	2314		0
3831	3831	2314		0
3832	3832	2314	2	0
3833	3833	2314	16	0
3834	3834	2314	4	0
3835	3835	2314	1	0
3836	3836	2314	7	0
3837	3837	2314	3	0
3838	3838	2314	3	0
3839	3839	2314	2	0
3840	3840	2314	1	0
3841	3841	2314	0	0
3842	3842	2314	0	0
3843	3843	2314	0	0
3844	3844	2314	0	0
3845	3845	2314	0	0
3846	3846	2314	0	0
3847	3847	2314	0	0
3848	3848	2314	B0L5	0
3849	3849	2314	1.2	0
3850	3850	2314	1.4	0
3851	3851	2314	31.2	0
3852	3852	2314	14	0
3853	3853	2314	47.6	0
3854	3854	2314	35	0
3855	3855	2314	11.1	0
3856	3856	2314	217	0
3857	3857	2314	2.8	0
3858	3858	2314	2.5	0
3859	3859	2314	0	0
3860	3860	2314	0	0
3861	3861	2314	0	0
3862	3862	2314	71	0
3863	3863	2314	350	0
3864	3864	2314	350	0
3865	3865	2314	30	0
3866	3866	2314	40	0
3867	3867	2314	95	0
3868	3868	2314	0	0
3869	3869	2314	1.941666667	0
3870	3870	2314	20.33333333	0
3871	3871	2314	5.816666667	0
3872	3872	2314	75	0
3873	3873	2314	100	0
3874	3874	2314	2.54	0
3875	3875	2314	91	0
3876	3876	2314	34	0
3877	3877	2314	2.74	0
3878	3878	2314	0	0
3879	3879	2314	0	0
3880	3880	2314	85	0
3881	3881	2314	0	0
3882	3882	2314	0	0
3883	3883	2314	173	0
3884	3884	2314	72	0
3885	3885	2314	9	0
3886	3886	2314	75	0
3887	3887	2314	100	0
3888	3888	2314	2.54	0
3889	3889	2314	91	0
3890	3890	2314	34	0
3891	3891	2314	2.74	0
3892	3892	2314	0	0
3893	3893	2314	0	0
3894	3894	2314	85	0
3895	3895	2314	0	0
3896	3896	2314	0	0
3897	3897	2314	53.3	0
3898	3898	2314	30.5	0
3899	3899	2314	21.3	0
3900	3900	2314	19.8	0
3901	3901	2314	7.7	0
3902	3902	2314	6.9	0
3903	3903	2314	13.74	0
3904	3904	2314	1.67	0
3905	3905	2314	13.7705	0
3906	3906	2314	1.6967	0
3907	3907	2314	3	0
3908	3908	2314	4	0
3909	3909	2314	1	0
3910	3910	2314	3	0
3911	3911	2314	0.4	0
3912	3912	2314	0.5	0
3913	3913	2314	0.5	0
3914	3914	2314	0.6	0
3915	3915	2314	4	0
3916	3916	2314	0	0
3917	3917	2314	1	0
3918	3918	2314	0.4	0
3919	3919	2314	0.3	0
3920	3920	2314	0.5	0
3921	3921	2314	0.4	0
3922	3922	2314	0.5	0
3923	3923	2314	0.4	0
3924	3924	2314	0.5	0
3925	3925	2314	1	0
3926	3926	2314	11	0
3927	3927	2314	3	0
3928	3928	2314	1	0
3929	3929	2314	6	0
3930	3930	2314	1	0
3931	3931	2314	3	0
3932	3932	2314	1	0
3933	3933	2314	0	0
3934	3934	2314	0	0
3935	3935	2314	0	0
3936	3936	2314	0	0
3937	3937	2314	0	0
3938	3938	2314	0	0
3939	3939	2314	0	0
3940	3940	2314	0	0
3941	3941	2314	.	0
3942	3942	2314	.	0
3943	3943	2314	.	0
3944	3944	2314	24.5	0
3945	3945	2314	12	0
3946	3946	2314	29.7	0
3947	3947	2314	27	0
3948	3948	2314	13.1	0
3949	3949	2314	144	0
3950	3950	2314	2.6	0
3951	3951	2314	2.3	0
3952	3952	2314	0	0
3953	3953	2314	0	0
3954	3954	2314	0	0
3955	3955	2314	64	0
3956	3956	2314	225	0
3957	3957	2314	225	0
3958	3958	2314	25	0
3959	3959	2314	40	0
3960	3960	2314	70	0
3961	3961	2314	0	0
3962	3962	2314	1.875	0
3963	3963	2314	23.4	0
3964	3964	2314	5.516666667	0
3965	3965	2314	75	0
3966	3966	2314	150	0
3967	3967	2314	3.17	0
3968	3968	2314	89	0
3969	3969	2314	43	0
3970	3970	2314	4.92	0
3971	3971	2314	1	0
3972	3972	2314	0	0
3973	3973	2314	50	0
3974	3974	2314	0	0
3975	3975	2314	0	0
3976	3976	2314	85	0
3977	3977	2314	22	0
3978	3978	2314	14	0
3979	3979	2314	75	0
3980	3980	2314	150	0
3981	3981	2314	3.17	0
3982	3982	2314	89	0
3983	3983	2314	43	0
3984	3984	2314	4.92	0
3985	3985	2314	1	0
3986	3986	2314	0	0
3987	3987	2314	50	0
3988	3988	2314	0	0
3989	3989	2314	0	0
3990	3990	2314	29.7	0
3991	3991	2314	22.5	0
3992	3992	2314	148.5	0
3993	3993	2314	77.7	0
3994	3994	2314	5.4	0
3995	3995	2314	5.2	0
3996	3996	2314	13.47	0
3997	3997	2314	1.79	0
3998	3998	2314	13.4925	0
3999	3999	2314	1.8729	0
4000	4000	2314	2	0
4001	4001	2314	1	0
4002	4002	2314	1	0
4003	4003	2314	2	0
4004	4004	2314	0.2	0
4005	4005	2314	0.2	0
4006	4006	2314	0.6	0
4007	4007	2314	0.7	0
4008	4008	2314	3	0
4009	4009	2314	0	0
4010	4010	2314	0.9	0
4011	4011	2314	0.3	0
4012	4012	2314	0.3	0
4013	4013	2314	0.3	0
4014	4014	2314	0.5	0
4015	4015	2314	0.7	0
4016	4016	2314	0.5	0
4017	4017	2314	0.6	0
4018	4018	2314	0	0
4019	4019	2314	14	0
4020	4020	2314	3	0
4021	4021	2314	1	0
4022	4022	2314	6	0
4023	4023	2314	3	0
4024	4024	2314	3	0
4025	4025	2314	2	0
4026	4026	2314	0	0
4027	4027	2314	0	0
4028	4028	2314	0	0
4029	4029	2314	0	0
4030	4030	2314	0	0
4031	4031	2314	0	0
4032	4032	2314	0	0
4033	4033	2314	0	0
4034	4034	2314	B0L5	0
4035	4035	2314	1.2	0
4036	4036	2314	1.5	0
4037	4037	2314	30.2	0
4038	4038	2314	11	0
4039	4039	2314	43.7	0
4040	4040	2314	30	0
4041	4041	2314	10.9	0
4042	4042	2314	144	0
4043	4043	2314	3.4	0
4044	4044	2314	3	0
4045	4045	2314	0	0
4046	4046	2314	2	0
4047	4047	2314	0	0
4048	4048	2314	97	0
4049	4049	2314	180	0
4050	4050	2314	180	0
4051	4051	2314	10	0
4052	4052	2314	15	0
4053	4053	2314	70	0
4054	4054	2314	0	0
4055	4055	2314	1.211666667	0
4056	4056	2314	13.28333333	0
4057	4057	2314	3.116666667	0
4058	4058	2314	30	0
4059	4059	2314	150	0
4060	4060	2314	2.01	0
4061	4061	2314	103	0
4062	4062	2314	47	0
4063	4063	2314	3.99	0
4064	4064	2314	0	0
4065	4065	2314	0	0
4066	4066	2314	20	0
4067	4067	2314	0	0
4068	4068	2314	0	0
4069	4069	2314	97	0
4070	4070	2314	11	0
4071	4071	2314	2	0
4072	4072	2314	30	0
4073	4073	2314	150	0
4074	4074	2314	2.01	0
4075	4075	2314	103	0
4076	4076	2314	47	0
4077	4077	2314	3.99	0
4078	4078	2314	0	0
4079	4079	2314	0	0
4080	4080	2314	20	0
4081	4081	2314	0	0
4082	4082	2314	0	0
4083	4083	2314	82.1	0
4084	4084	2314	24.3	0
4085	4085	2314	201	0
4086	4086	2314	65	0
4087	4087	2314	8.2	0
4088	4088	2314	4.9	0
4089	4089	2314	7.67	0
4090	4090	2314	1.03	0
4091	4091	2314	7.6943	0
4092	4092	2314	1.0999	0
4093	4093	2314	29	0
4094	4094	2314	25	0
4095	4095	2314	0	0
4096	4096	2314	1	0
4097	4097	2314	0.5	0
4098	4098	2314	0.6	0
4099	4099	2314		0
4100	4100	2314		0
4101	4101	2314	2	0
4102	4102	2314	0	0
4103	4103	2314	1.1	0
4104	4104	2314	0.4	0
4105	4105	2314	0.6	0
4106	4106	2314	0.8	0
4107	4107	2314		0
4108	4108	2314		0
4109	4109	2314		0
4110	4110	2314		0
4111	4111	2314	2	0
4112	4112	2314	12	0
4113	4113	2314	3	0
4114	4114	2314	1	0
4115	4115	2314	7	0
4116	4116	2314	2	0
4117	4117	2314	2	0
4118	4118	2314	1	0
4119	4119	2314	0	0
4120	4120	2314	0	0
4121	4121	2314	0	0
4122	4122	2314	0	0
4123	4123	2314	0	0
4124	4124	2314	0	0
4125	4125	2314	0	0
4126	4126	2314	0	0
4127	4127	2314	B0L5	0
4128	4128	2314	1	0
4129	4129	2314	1.15	0
4130	4130	2314	31.3	0
4131	4131	2314	13	0
4132	4132	2314	34.8	0
4133	4133	2314	23	0
4134	4134	2314	5.5	0
4135	4135	2314	87	0
4136	4136	2314	2.3	0
4137	4137	2314	2.1	0
4138	4138	2314	0	0
4139	4139	2314	0	0
4140	4140	2314	0	0
4141	4141	2314	79	0
4142	4142	2314	175	0
4143	4143	2314	175	0
4144	4144	2314	15	0
4145	4145	2314	50	0
4146	4146	2314	95	0
4147	4147	2314	0	0
4148	4148	2314	2.583333333	0
4149	4149	2314	28.28333333	0
4150	4150	2314	6.75	0
4151	4151	2314	75	0
4152	4152	2314	100	0
4153	4153	2314	3.04	0
4154	4154	2314	85	0
4155	4155	2314	45	0
4156	4156	2314	3.65	0
4157	4157	2314	0	0
4158	4158	2314	0	0
4159	4159	2314	85	0
4160	4160	2314	0	0
4161	4161	2314	0	0
4162	4162	2314	115	0
4163	4163	2314	37	0
4164	4164	2314	11	0
4165	4165	2314	75	0
4166	4166	2314	100	0
4167	4167	2314	3.04	0
4168	4168	2314	85	0
4169	4169	2314	45	0
4170	4170	2314	3.65	0
4171	4171	2314	0	0
4172	4172	2314	0	0
4173	4173	2314	85	0
4174	4174	2314	0	0
4175	4175	2314	0	0
4176	4176	2314	44.1	0
4177	4177	2314	27.4	0
4178	4178	2314	67.8	0
4179	4179	2314	44	0
4180	4180	2314	11.6	0
4181	4181	2314	10.7	0
4182	4182	2314	8.68	0
4183	4183	2314	1.97	0
4184	4184	2314	8.7074	0
4185	4185	2314	2.0247	0
4186	4186	2314	15	0
4187	4187	2314	11	0
4188	4188	2314	0	0
4189	4189	2314	2	0
4190	4190	2314	0.4	0
4191	4191	2314	0.4	0
4192	4192	2314		0
4193	4193	2314		0
4194	4194	2314	2	0
4195	4195	2314	0	0
4196	4196	2314	0.7	0
4197	4197	2314	0.4	0
4198	4198	2314	0.5	0
4199	4199	2314	0.7	0
4200	4200	2314	0.4	0
4201	4201	2314	0.5	0
4202	4202	2314		0
4203	4203	2314		0
4204	4204	2314	2	0
4205	4205	2314	12	0
4206	4206	2314	3	0
4207	4207	2314	1	0
4208	4208	2314	6	0
4209	4209	2314	3	0
4210	4210	2314	2	0
4211	4211	2314	1	0
4212	4212	2314	0	0
4213	4213	2314	0	0
4214	4214	2314	0	0
4215	4215	2314	0	0
4216	4216	2314	0	0
4217	4217	2314	0	0
4218	4218	2314	0	0
4219	4219	2314	0	0
4220	4220	2314	B0L4	0
4221	4221	2314	1.05	0
4222	4222	2314	1.2	0
4223	4223	2314	22.5	0
4224	4224	2314	13	0
4225	4225	2314	37.2	0
4226	4226	2314	36	0
4227	4227	2314	7.3	0
4228	4228	2314	178	0
4229	4229	2314	2.7	0
4230	4230	2314	2.3	0
4231	4231	2314	0	0
4232	4232	2314	0	0
4233	4233	2314	0	0
4234	4234	2314	71	0
4235	4235	2314	20	0
4236	4236	2314	20	0
4237	4237	2314	4	0
4238	4238	2314	3	0
4239	4239	2314	15	0
4240	4240	2314	2	0
4241	4241	2314	0.756666667	0
4242	4242	2314	13.8	0
4243	4243	2314	3.75	0
4244	4244	2314	0	0
4245	4245	2314	200	0
4246	4246	2314	1.57	0
4247	4247	2314	69	0
4248	4248	2314	21	0
4249	4249	2314	1.41	0
4250	4250	2314	0	0
4251	4251	2314	0	0
4252	4252	2314	0	0
4253	4253	2314	0	0
4254	4254	2314	0	0
4255	4255	2314	86	0
4256	4256	2314		0
4257	4257	2314		0
4258	4258	2314	0	0
4259	4259	2314	200	0
4260	4260	2314	1.57	0
4261	4261	2314	69	0
4262	4262	2314	21	0
4263	4263	2314	1.41	0
4264	4264	2314	0	0
4265	4265	2314	0	0
4266	4266	2314	0	0
4267	4267	2314	0	0
4268	4268	2314	0	0
4269	4269	2314	27	0
4270	4270	2314	24.3	0
4271	4271	2314	27.2	0
4272	4272	2314	13	0
4273	4273	2314	12.9	0
4274	4274	2314	10.8	0
4275	4275	2314	5	0
4276	4276	2314	0.14	0
4277	4277	2314	5.0243	0
4278	4278	2314	0.1638	0
4279	4279	2314	4	0
4280	4280	2314	68	0
4281	4281	2314	1	0
4282	4282	2314	2	0
4283	4283	2314	0.5	0
4284	4284	2314	0.9	0
4285	4285	2314	0.9	0
4286	4286	2314	0.9	0
4287	4287	2314	3	0
4288	4288	2314	0	0
4289	4289	2314	0.9	0
4290	4290	2314	0.3	0
4291	4291	2314	0.7	0
4292	4292	2314	0.9	0
4293	4293	2314	0.7	0
4294	4294	2314	0.9	0
4295	4295	2314	0.7	0
4296	4296	2314	0.9	0
4297	4297	2314	1	0
4298	4298	2314	20	0
4299	4299	2314	5	0
4300	4300	2314	1	0
4301	4301	2314	8	0
4302	4302	2314	4	0
4303	4303	2314	4	0
4304	4304	2314	2	0
4305	4305	2314	1	0
4306	4306	2314	0	0
4307	4307	2314	0	0
4308	4308	2314	1	0
4309	4309	2314	0	0
4310	4310	2314	0	0
4311	4311	2314	0	0
4312	4312	2314	0	0
4313	4313	2314	B0L6	0
4314	4314	2314	1.3	0
4315	4315	2314	1.2	0
4316	4316	2314	28.2	0
4317	4317	2314	16	0
4318	4318	2314	43	0
4319	4319	2314	51	0
4320	4320	2314	21.4	0
4321	4321	2314	361	0
4322	4322	2314	2.7	0
4323	4323	2314	1.8	0
4324	4324	2314	7	0
4325	4325	2314	2	0
4326	4326	2314	4	0
4327	4327	2314	62.5	0
4328	4328	2314	125	0
4329	4329	2314	125	0
4330	4330	2314	5	0
4331	4331	2314	15	0
4332	4332	2314	95	0
4333	4333	2314	0	0
4334	4334	2314	1.511666667	0
4335	4335	2314	23.5	0
4336	4336	2314	6.083333333	0
4337	4337	2314	100	0
4338	4338	2314	300	0
4339	4339	2314	2.02	0
4340	4340	2314	76	0
4341	4341	2314	44	0
4342	4342	2314	2.97	0
4343	4343	2314	0	0
4344	4344	2314	0	0
4345	4345	2314	50	0
4346	4346	2314	0	0
4347	4347	2314	0	0
4348	4348	2314	120	0
4349	4349	2314	32	0
4350	4350	2314	0	0
4351	4351	2314	100	0
4352	4352	2314	300	0
4353	4353	2314	2.02	0
4354	4354	2314	76	0
4355	4355	2314	44	0
4356	4356	2314	2.97	0
4357	4357	2314	0	0
4358	4358	2314	0	0
4359	4359	2314	50	0
4360	4360	2314	0	0
4361	4361	2314	0	0
4362	4362	2314	67.7	0
4363	4363	2314	28	0
4364	4364	2314	170.5	0
4365	4365	2314	63.5	0
4366	4366	2314	26.2	0
4367	4367	2314	14.2	0
4368	4368	2314	6.79	0
4369	4369	2314	1.16	0
4370	4370	2314	6.818	0
4371	4371	2314	1.2377	0
4372	4372	2314	6	0
4373	4373	2314	48	0
4374	4374	2314	1	0
4375	4375	2314	2	0
4376	4376	2314	0.5	0
4377	4377	2314	0.8	0
4378	4378	2314		0
4379	4379	2314		0
4380	4380	2314	3	0
4381	4381	2314	0	0
4382	4382	2314	0.7	0
4383	4383	2314	0.3	0
4384	4384	2314	0.6	0
4385	4385	2314	0.8	0
4386	4386	2314	0.6	0
4387	4387	2314	0.9	0
4388	4388	2314		0
4389	4389	2314		0
4390	4390	2314	1	0
4391	4391	2314	18	0
4392	4392	2314	4	0
4393	4393	2314	1	0
4394	4394	2314	8	0
4395	4395	2314	3	0
4396	4396	2314	4	0
4397	4397	2314	2	0
4398	4398	2314	1	0
4399	4399	2314	0	0
4400	4400	2314	0	0
4401	4401	2314	0	0
4402	4402	2314	0	0
4403	4403	2314	0	0
4404	4404	2314	0	0
4405	4405	2314	0	0
4406	4406	2314	B0L6	0
4407	4407	2314	1	0
4408	4408	2314	1.3	0
4409	4409	2314	28.7	0
4410	4410	2314	13	0
4411	4411	2314	44.8	0
4412	4412	2314	34	0
4413	4413	2314	12.3	0
4414	4414	2314	202	0
4415	4415	2314	2.2	0
4416	4416	2314	2.2	0
4417	4417	2314	2	0
4418	4418	2314	4	0
4419	4419	2314	0	0
4420	4420	2314	85.5	0
4421	4421	2314	150	0
4422	4422	2314	150	0
4423	4423	2314	15	0
4424	4424	2314	20	0
4425	4425	2314	80	0
4426	4426	2314	2	0
4427	4427	2314	1.618333333	0
4428	4428	2314	17.91666667	0
4429	4429	2314	5.016666667	0
4430	4430	2314	250	0
4431	4431	2314	250	0
4432	4432	2314	2.77	0
4433	4433	2314	93	0
4434	4434	2314	31	0
4435	4435	2314	2.25	0
4436	4436	2314	18	0
4437	4437	2314	0	0
4438	4438	2314	80	0
4439	4439	2314	5	0
4440	4440	2314	0	0
4441	4441	2314	103	0
4442	4442	2314	109	0
4443	4443	2314	21	0
4444	4444	2314	250	0
4445	4445	2314	250	0
4446	4446	2314	2.77	0
4447	4447	2314	93	0
4448	4448	2314	31	0
4449	4449	2314	2.25	0
4450	4450	2314	18	0
4451	4451	2314	0	0
4452	4452	2314	80	0
4453	4453	2314	5	0
4454	4454	2314	0	0
4455	4455	2314	75	0
4456	4456	2314	28.5	0
4457	4457	2314	94	0
4458	4458	2314	34.3	0
4459	4459	2314	15.9	0
4460	4460	2314	10.6	0
4461	4461	2314	12.17	0
4462	4462	2314	1.87	0
4463	4463	2314	12.1985	0
4464	4464	2314	1.9149	0
4465	4465	2314	4	0
4466	4466	2314	15	0
4467	4467	2314	1	0
4468	4468	2314	2	0
4469	4469	2314	0.6	0
4470	4470	2314	0.5	0
4471	4471	2314		0
4472	4472	2314		0
4473	4473	2314	3	0
4474	4474	2314	0	0
4475	4475	2314	0.9	0
4476	4476	2314	0.4	0
4477	4477	2314	0.4	0
4478	4478	2314	0.5	0
4479	4479	2314	0.4	0
4480	4480	2314	0.6	0
4481	4481	2314		0
4482	4482	2314		0
4483	4483	2314	0	0
4484	4484	2314	10	0
4485	4485	2314	2	0
4486	4486	2314	2	0
4487	4487	2314	6	0
4488	4488	2314	2	0
4489	4489	2314	2	0
4490	4490	2314	0	0
4491	4491	2314	0	0
4492	4492	2314	0	0
4493	4493	2314	0	0
4494	4494	2314	0	0
4495	4495	2314	0	0
4496	4496	2314	0	0
4497	4497	2314	0	0
4498	4498	2314	0	0
4499	4499	2314	B0L5	0
4500	4500	2314	0.7	0
4501	4501	2314	1	0
4502	4502	2314	27.1	0
4503	4503	2314	14	0
4504	4504	2314	39.1	0
4505	4505	2314	21	0
4506	4506	2314	8	0
4507	4507	2314	117	0
4508	4508	2314	2.1	0
4509	4509	2314	2.3	0
4510	4510	2314	2	0
4511	4511	2314	2	0
4512	4512	2314	0	0
4513	4513	2314	84.5	0
4514	4514	2314	150	0
4515	4515	2314	150	0
4516	4516	2314	2	0
4517	4517	2314	20	0
4518	4518	2314	75	0
4519	4519	2314	0	0
4520	4520	2314	1.425	0
4521	4521	2314	12.86666667	0
4522	4522	2314	5.283333333	0
4523	4523	2314	50	0
4524	4524	2314	200	0
4525	4525	2314	2.23	0
4526	4526	2314	84	0
4527	4527	2314	27	0
4528	4528	2314	2.46	0
4529	4529	2314	0	0
4530	4530	2314	0	0
4531	4531	2314	30	0
4532	4532	2314	0	0
4533	4533	2314	0	0
4534	4534	2314	88	0
4535	4535	2314	7	0
4536	4536	2314	2	0
4537	4537	2314	50	0
4538	4538	2314	200	0
4539	4539	2314	2.23	0
4540	4540	2314	84	0
4541	4541	2314	27	0
4542	4542	2314	2.46	0
4543	4543	2314	0	0
4544	4544	2314	0	0
4545	4545	2314	30	0
4546	4546	2314	0	0
4547	4547	2314	0	0
4548	4548	2314	107.9	0
4549	4549	2314	37.2	0
4550	4550	2314	128.5	0
4551	4551	2314	37.3	0
4552	4552	2314	12.4	0
4553	4553	2314	5.6	0
4554	4554	2314	6.11	0
4555	4555	2314	1.11	0
4556	4556	2314	6.1472	0
4557	4557	2314	1.1529	0
4558	4558	2314	.	0
4559	4559	2314	.	0
4560	4560	2314	1	0
4561	4561	2314	1	0
4562	4562	2314	0.2	0
4563	4563	2314	0.2	0
4564	4564	2314		0
4565	4565	2314		0
4566	4566	2314	0	0
4567	4567	2314	0	0
4568	4568	2314		0
4569	4569	2314		0
4570	4570	2314		0
4571	4571	2314		0
4572	4572	2314		0
4573	4573	2314		0
4574	4574	2314		0
4575	4575	2314		0
4576	4576	2314	0	0
4577	4577	2314	0	0
4578	4578	2314	0	0
4579	4579	2314	.	0
4580	4580	2314	0	0
4581	4581	2314	0	0
4582	4582	2314	0	0
4583	4583	2314	0	0
4584	4584	2314	0	0
4585	4585	2314	0	0
4586	4586	2314	0	0
4587	4587	2314	0	0
4588	4588	2314	0	0
4589	4589	2314	0	0
4590	4590	2314	0	0
4591	4591	2314	0	0
4592	4592	2314	.	0
4593	4593	2314	.	0
4594	4594	2314	.	0
4595	4595	2314	.	0
4596	4596	2314	0	0
4597	4597	2314	.	0
4598	4598	2314	0	0
4599	4599	2314	.	0
4600	4600	2314	0	0
4601	4601	2314	.	0
4602	4602	2314	.	0
4603	4603	2314	0	0
4604	4604	2314	0	0
4605	4605	2314	0	0
4606	4606	2314		0
4607	4607	2314		0
4608	4608	2314	0	0
4609	4609	2314		0
4610	4610	2314		0
4611	4611	2314		0
4612	4612	2314		0
4613	4613	2314		0
4614	4614	2314		0
4615	4615	2314		0
4616	4616	2314		0
4617	4617	2314		0
4618	4618	2314		0
4619	4619	2314		0
4620	4620	2314		0
4621	4621	2314		0
4622	4622	2314		0
4623	4623	2314		0
4624	4624	2314		0
4625	4625	2314		0
4626	4626	2314		0
4627	4627	2314		0
4628	4628	2314		0
4629	4629	2314		0
4630	4630	2314	.	0
4631	4631	2314	.	0
4632	4632	2314	.	0
4633	4633	2314	.	0
4634	4634	2314	.	0
4635	4635	2314	.	0
4636	4636	2314	.	0
4637	4637	2314	.	0
4638	4638	2314	.	0
4639	4639	2314	.	0
4640	4640	2314	.	0
4641	4641	2314	.	0
4642	4642	2314	.	0
4643	4643	2314	.	0
4644	4644	2314	.	0
4645	4645	2314	.	0
4646	4646	2314	.	0
4647	4647	2314		0
4648	4648	2314		0
4649	4649	2314	.	0
4650	4650	2314	.	0
4651	4651	2314	50	0
4652	4652	2314	2	0
4653	4653	2314	1	0
4654	4654	2314	2	0
4655	4655	2314	0.5	0
4656	4656	2314	0.6	0
4657	4657	2314	0.8	0
4658	4658	2314	0.9	0
4659	4659	2314	3	0
4660	4660	2314	0	0
4661	4661	2314	0.8	0
4662	4662	2314	0.4	0
4663	4663	2314	0.6	0
4664	4664	2314	0.6	0
4665	4665	2314	0.6	0
4666	4666	2314	0.8	0
4667	4667	2314		0
4668	4668	2314		0
4669	4669	2314	2	0
4670	4670	2314	15	0
4671	4671	2314	3	0
4672	4672	2314	1	0
4673	4673	2314	8	0
4674	4674	2314	3	0
4675	4675	2314	3	0
4676	4676	2314	1	0
4677	4677	2314	0	0
4678	4678	2314	0	0
4679	4679	2314	0	0
4680	4680	2314	0	0
4681	4681	2314	0	0
4682	4682	2314	0	0
4683	4683	2314	0	0
4684	4684	2314	0	0
4685	4685	2314	B0L6	0
4686	4686	2314	1	0
4687	4687	2314	1.2	0
4688	4688	2314	33.3	0
4689	4689	2314	13	0
4690	4690	2314	41.1	0
4691	4691	2314	34	0
4692	4692	2314	15.8	0
4693	4693	2314	217	0
4694	4694	2314	2.5	0
4695	4695	2314	2.2	0
4696	4696	2314	0	0
4697	4697	2314	0	0
4698	4698	2314	0	0
4699	4699	2314	65	0
4700	4700	2314	125	0
4701	4701	2314	125	0
4702	4702	2314	15	0
4703	4703	2314	5	0
4704	4704	2314	80	0
4705	4705	2314	1	0
4706	4706	2314	1.416666667	0
4707	4707	2314	18.11666667	0
4708	4708	2314	4.416666667	0
4709	4709	2314	200	0
4710	4710	2314	250	0
4711	4711	2314	2.45	0
4712	4712	2314	68	0
4713	4713	2314	34	0
4714	4714	2314	2.45	0
4715	4715	2314	4	0
4716	4716	2314	0	0
4717	4717	2314	75	0
4718	4718	2314	0	0
4719	4719	2314	0	0
4720	4720	2314	70	0
4721	4721	2314	32	0
4722	4722	2314	43	0
4723	4723	2314	200	0
4724	4724	2314	250	0
4725	4725	2314	2.45	0
4726	4726	2314	68	0
4727	4727	2314	34	0
4728	4728	2314	2.45	0
4729	4729	2314	4	0
4730	4730	2314	0	0
4731	4731	2314	75	0
4732	4732	2314	0	0
4733	4733	2314	0	0
4734	4734	2314	83.4	0
4735	4735	2314	30.7	0
4736	4736	2314	121.1	0
4737	4737	2314	43	0
4738	4738	2314	16.4	0
4739	4739	2314	8	0
4740	4740	2314	8.09	0
4741	4741	2314	1.79	0
4742	4742	2314	8.1207	0
4743	4743	2314	1.841	0
4744	4744	2314	8	0
4745	4745	2314	2	0
4746	4746	2314	1	0
4747	4747	2314	2	0
4748	4748	2314	0.4	0
4749	4749	2314	0.4	0
4750	4750	2314	0.7	0
4751	4751	2314	0.8	0
4752	4752	2314	4	0
4753	4753	2314	0	0
4754	4754	2314	1.2	0
4755	4755	2314	0.6	0
4756	4756	2314	.	0
4757	4757	2314	.	0
4758	4758	2314	0.6	0
4759	4759	2314	0.7	0
4760	4760	2314	0.8	0
4761	4761	2314	1	0
4762	4762	2314	2	0
4763	4763	2314	17	0
4764	4764	2314	4	0
4765	4765	2314	1	0
4766	4766	2314	7	0
4767	4767	2314	3	0
4768	4768	2314	4	0
4769	4769	2314	2	0
4770	4770	2314	1	0
4771	4771	2314	0	0
4772	4772	2314	0	0
4773	4773	2314	0	0
4774	4774	2314	0	0
4775	4775	2314	0	0
4776	4776	2314	0	0
4777	4777	2314	0	0
4778	4778	2314	B0L5	0
4779	4779	2314	1.3	0
4780	4780	2314	1.5	0
4781	4781	2314	24.2	0
4782	4782	2314	14	0
4783	4783	2314	40.1	0
4784	4784	2314	38	0
4785	4785	2314	22.3	0
4786	4786	2314	297	0
4787	4787	2314	2.9	0
4788	4788	2314	2.6	0
4789	4789	2314	0	0
4790	4790	2314	8	0
4791	4791	2314	0	0
4792	4792	2314	87.5	0
4793	4793	2314	400	0
4794	4794	2314	400	0
4795	4795	2314	30	0
4796	4796	2314	30	0
4797	4797	2314	95	0
4798	4798	2314	0	0
4799	4799	2314	2.998333333	0
4800	4800	2314	42.16666667	0
4801	4801	2314	10.5	0
4802	4802	2314	300	0
4803	4803	2314	350	0
4804	4804	2314	4.23	0
4805	4805	2314	88	0
4806	4806	2314	51	0
4807	4807	2314	6.44	0
4808	4808	2314	0	0
4809	4809	2314	0	0
4810	4810	2314	90	0
4811	4811	2314	0	0
4812	4812	2314	0	0
4813	4813	2314	153	0
4814	4814	2314	208	0
4815	4815	2314	63	0
4816	4816	2314	300	0
4817	4817	2314	350	0
4818	4818	2314	4.23	0
4819	4819	2314	88	0
4820	4820	2314	51	0
4821	4821	2314	6.44	0
4822	4822	2314	0	0
4823	4823	2314	0	0
4824	4824	2314	90	0
4825	4825	2314	0	0
4826	4826	2314	0	0
4827	4827	2314	122	0
4828	4828	2314	66.6	0
4829	4829	2314	313.7	0
4830	4830	2314	148.5	0
4831	4831	2314	17.8	0
4832	4832	2314	16.6	0
4833	4833	2314	36.29	0
4834	4834	2314	5.39	0
4835	4835	2314	36.3566	0
4836	4836	2314	5.5551	0
4837	4837	2314	3	0
4838	4838	2314	109	0
4839	4839	2314	1	0
4840	4840	2314	2	0
4841	4841	2314	0.5	0
4842	4842	2314	1	0
4843	4843	2314	1.2	0
4844	4844	2314	1.2	0
4845	4845	2314	4	0
4846	4846	2314	0	0
4847	4847	2314	1.1	0
4848	4848	2314	0.4	0
4849	4849	2314	0.6	0
4850	4850	2314	1	0
4851	4851	2314	0.8	0
4852	4852	2314	1.1	0
4853	4853	2314	0.8	0
4854	4854	2314	1.1	0
4855	4855	2314	2	0
4856	4856	2314	20	0
4857	4857	2314	5	0
4858	4858	2314	1	0
4859	4859	2314	8	0
4860	4860	2314	4	0
4861	4861	2314	4	0
4862	4862	2314	2	0
4863	4863	2314	1	0
4864	4864	2314	0	0
4865	4865	2314	0	0
4866	4866	2314	0	0
4867	4867	2314	1	0
4868	4868	2314	0	0
4869	4869	2314	0	0
4870	4870	2314	0	0
4871	4871	2314	B0L6	0
4872	4872	2314	1.5	0
4873	4873	2314	1.6	0
4874	4874	2314	35.5	0
4875	4875	2314	15	0
4876	4876	2314	46.2	0
4877	4877	2314	75	0
4878	4878	2314	35.5	0
4879	4879	2314	580	0
4880	4880	2314	3.2	0
4881	4881	2314	2.4	0
4882	4882	2314	27	0
4883	4883	2314	32	0
4884	4884	2314	0	0
4885	4885	2314	68.5	0
4886	4886	2314	0	0
4887	4887	2314	150	0
4888	4888	2314	0	0
4889	4889	2314	2	0
4890	4890	2314	0	0
4891	4891	2314	0	0
4892	4892	2314		0
4893	4893	2314		0
4894	4894	2314		0
4895	4895	2314		0
4896	4896	2314		0
4897	4897	2314		0
4898	4898	2314		0
4899	4899	2314		0
4900	4900	2314		0
4901	4901	2314		0
4902	4902	2314		0
4903	4903	2314		0
4904	4904	2314		0
4905	4905	2314		0
4906	4906	2314	204	0
4907	4907	2314		0
4908	4908	2314		0
4909	4909	2314	.	0
4910	4910	2314	.	0
4911	4911	2314	.	0
4912	4912	2314	.	0
4913	4913	2314	.	0
4914	4914	2314	.	0
4915	4915	2314	.	0
4916	4916	2314	.	0
4917	4917	2314	.	0
4918	4918	2314	.	0
4919	4919	2314	.	0
4920	4920	2314	.	0
4921	4921	2314	.	0
4922	4922	2314	.	0
4923	4923	2314	.	0
4924	4924	2314	.	0
4925	4925	2314	.	0
4926	4926	2314	24.34	0
4927	4927	2314	2.04	0
4928	4928	2314	24.34	0
4929	4929	2314	2.04	0
4930	4930	2314	7	0
4931	4931	2314	10	0
4932	4932	2314	2	0
4933	4933	2314	2	0
4934	4934	2314	0.4	0
4935	4935	2314	0.4	0
4936	4936	2314	1.1	0
4937	4937	2314	1	0
4938	4938	2314	5	0
4939	4939	2314	0	0
4940	4940	2314	1.2	0
4941	4941	2314	0.5	0
4942	4942	2314	0.5	0
4943	4943	2314	0.5	0
4944	4944	2314	0.9	0
4945	4945	2314	1	0
4946	4946	2314	0.6	0
4947	4947	2314	0.8	0
4948	4948	2314	1	0
4949	4949	2314	19	0
4950	4950	2314	4	0
4951	4951	2314	1	0
4952	4952	2314	7	0
4953	4953	2314	5	0
4954	4954	2314	4	0
4955	4955	2314	2	0
4956	4956	2314	1	0
4957	4957	2314	0	0
4958	4958	2314	0	0
4959	4959	2314	0	0
4960	4960	2314	0	0
4961	4961	2314	0	0
4962	4962	2314	0	0
4963	4963	2314	0	0
4964	4964	2314	B0L6	0
4965	4965	2314	1.5	0
4966	4966	2314	1.5	0
4967	4967	2314	32.8	0
4968	4968	2314	16	0
4969	4969	2314	40.7	0
4970	4970	2314	46	0
4971	4971	2314	15.4	0
4972	4972	2314	288	0
4973	4973	2314	2.8	0
4974	4974	2314	2	0
4975	4975	2314	0	0
4976	4976	2314	0	0
4977	4977	2314	0	0
4978	4978	2314	74	0
4979	4979	2314	250	0
4980	4980	2314	250	0
4981	4981	2314	25	0
4982	4982	2314	35	0
4983	4983	2314	90	0
4984	4984	2314	1	0
4985	4985	2314	2.49	0
4986	4986	2314	25.38333333	0
4987	4987	2314	8.566666667	0
4988	4988	2314	200	0
4989	4989	2314	600	0
4990	4990	2314	2.2	0
4991	4991	2314	86	0
4992	4992	2314	57	0
4993	4993	2314	2.12	0
4994	4994	2314	0	0
4995	4995	2314	0	0
4996	4996	2314	50	0
4997	4997	2314	0	0
4998	4998	2314	0	0
4999	4999	2314	57	0
5000	5000	2314	91	0
5001	5001	2314	32	0
5002	5002	2314	200	0
5003	5003	2314	600	0
5004	5004	2314	2.2	0
5005	5005	2314	86	0
5006	5006	2314	57	0
5007	5007	2314	2.12	0
5008	5008	2314	0	0
5009	5009	2314	0	0
5010	5010	2314	50	0
5011	5011	2314	0	0
5012	5012	2314	0	0
5013	5013	2314	151.2	0
5014	5014	2314	66.7	0
5015	5015	2314	182.2	0
5016	5016	2314	70	0
5017	5017	2314	44.3	0
5018	5018	2314	19.6	0
5019	5019	2314	25.39	0
5020	5020	2314	3.1	0
5021	5021	2314	25.4567	0
5022	5022	2314	3.1896	0
5023	5023	2314	23	0
5024	5024	2314	36	0
5025	5025	2314	2	0
5026	5026	2314	2	0
5027	5027	2314	0.4	0
5028	5028	2314	0.6	0
5029	5029	2314	0.6	0
5030	5030	2314	0.8	0
5031	5031	2314	4	0
5032	5032	2314	0	0
5033	5033	2314	1.3	0
5034	5034	2314	0.5	0
5035	5035	2314	.	0
5036	5036	2314	.	0
5037	5037	2314	0.5	0
5038	5038	2314	0.6	0
5039	5039	2314	0.4	0
5040	5040	2314	0.8	0
5041	5041	2314	2	0
5042	5042	2314	9	0
5043	5043	2314	2	0
5044	5044	2314	2	0
5045	5045	2314	6	0
5046	5046	2314	2	0
5047	5047	2314	1	0
5048	5048	2314	0	0
5049	5049	2314	0	0
5050	5050	2314	0	0
5051	5051	2314	0	0
5052	5052	2314	0	0
5053	5053	2314	0	0
5054	5054	2314	0	0
5055	5055	2314	0	0
5056	5056	2314	0	0
5057	5057	2314	B0L5	0
5058	5058	2314	1.1	0
5059	5059	2314	1.1	0
5060	5060	2314	33.5	0
5061	5061	2314	12	0
5062	5062	2314	35.3	0
5063	5063	2314	25	0
5064	5064	2314	9.3	0
5065	5065	2314	78	0
5066	5066	2314	2.4	0
5067	5067	2314	2.2	0
5068	5068	2314	0	0
5069	5069	2314	0	0
5070	5070	2314	0	0
5071	5071	2314	82.5	0
5072	5072	2314	175	0
5073	5073	2314	175	0
5074	5074	2314	20	0
5075	5075	2314	40	0
5076	5076	2314	85	0
5077	5077	2314	0	0
5078	5078	2314	1.405	0
5079	5079	2314	17.31666667	0
5080	5080	2314	5.516666667	0
5081	5081	2314	250	0
5082	5082	2314	300	0
5083	5083	2314	2.47	0
5084	5084	2314	92	0
5085	5085	2314	50	0
5086	5086	2314	4.11	0
5087	5087	2314	0	0
5088	5088	2314	0	0
5089	5089	2314	55	0
5090	5090	2314	20	0
5091	5091	2314	3	0
5092	5092	2314	113	0
5093	5093	2314	60	0
5094	5094	2314	1	0
5095	5095	2314	250	0
5096	5096	2314	300	0
5097	5097	2314	2.47	0
5098	5098	2314	92	0
5099	5099	2314	50	0
5100	5100	2314	4.11	0
5101	5101	2314	0	0
5102	5102	2314	0	0
5103	5103	2314	55	0
5104	5104	2314	20	0
5105	5105	2314	3	0
5106	5106	2314	100.2	0
5107	5107	2314	38	0
5108	5108	2314	126.8	0
5109	5109	2314	45	0
5110	5110	2314	38	0
5111	5111	2314	17.3	0
5112	5112	2314	8.49	0
5113	5113	2314	1.74	0
5114	5114	2314	8.528	0
5115	5115	2314	1.8023	0
5116	5116	2314	21	0
5117	5117	2314	30	0
5118	5118	2314	1	0
5119	5119	2314	2	0
5120	5120	2314	0.5	0
5121	5121	2314	0.4	0
5122	5122	2314	0.8	0
5123	5123	2314	0.9	0
5124	5124	2314	4	0
5125	5125	2314	0	0
5126	5126	2314	0.8	0
5127	5127	2314	0.3	0
5128	5128	2314	0.4	0
5129	5129	2314	0.4	0
5130	5130	2314	0.5	0
5131	5131	2314	0.7	0
5132	5132	2314	0.5	0
5133	5133	2314	0.6	0
5134	5134	2314	0	0
5135	5135	2314	19	0
5136	5136	2314	4	0
5137	5137	2314	1	0
5138	5138	2314	8	0
5139	5139	2314	4	0
5140	5140	2314	4	0
5141	5141	2314	2	0
5142	5142	2314	1	0
5143	5143	2314	0	0
5144	5144	2314	0	0
5145	5145	2314	0	0
5146	5146	2314	0	0
5147	5147	2314	0	0
5148	5148	2314	0	0
5149	5149	2314	0	0
5150	5150	2314	B0L6	0
5151	5151	2314	1	0
5152	5152	2314	1.2	0
5153	5153	2314	27.3	0
5154	5154	2314	15	0
5155	5155	2314	38.5	0
5156	5156	2314	30	0
5157	5157	2314	9.8	0
5158	5158	2314	205	0
5159	5159	2314	2.6	0
5160	5160	2314	2.1	0
5161	5161	2314	16	0
5162	5162	2314	20	0
5163	5163	2314	6	0
5164	5164	2314	82	0
5165	5165	2314	250	0
5166	5166	2314	250	0
5167	5167	2314	10	0
5168	5168	2314	20	0
5169	5169	2314	95	0
5170	5170	2314	0	0
5171	5171	2314	1.401666667	0
5172	5172	2314	17.6	0
5173	5173	2314	4.45	0
5174	5174	2314	100	0
5175	5175	2314	100	0
5176	5176	2314	2.39	0
5177	5177	2314	85	0
5178	5178	2314	45	0
5179	5179	2314	3.65	0
5180	5180	2314	0	0
5181	5181	2314	0	0
5182	5182	2314	90	0
5183	5183	2314	0	0
5184	5184	2314	0	0
5185	5185	2314	121	0
5186	5186	2314	64	0
5187	5187	2314	25	0
5188	5188	2314	100	0
5189	5189	2314	100	0
5190	5190	2314	2.39	0
5191	5191	2314	85	0
5192	5192	2314	45	0
5193	5193	2314	3.65	0
5194	5194	2314	0	0
5195	5195	2314	0	0
5196	5196	2314	90	0
5197	5197	2314	0	0
5198	5198	2314	0	0
5199	5199	2314	37.1	0
5200	5200	2314	29.1	0
5201	5201	2314	131.7	0
5202	5202	2314	63.4	0
5203	5203	2314	7.5	0
5204	5204	2314	6.5	0
5205	5205	2314	13.93	0
5206	5206	2314	1.51	0
5207	5207	2314	13.9591	0
5208	5208	2314	1.5799	0
5209	5209	2314	9	0
5210	5210	2314	1	0
5211	5211	2314	1	0
5212	5212	2314	2	0
5213	5213	2314	0.3	0
5214	5214	2314	0.5	0
5215	5215	2314	0.7	0
5216	5216	2314	0.7	0
5217	5217	2314	3	0
5218	5218	2314	0	0
5219	5219	2314	1	0
5220	5220	2314	0.4	0
5221	5221	2314	0.3	0
5222	5222	2314	0.5	0
5223	5223	2314	0.4	0
5224	5224	2314	0.4	0
5225	5225	2314	0.5	0
5226	5226	2314	0.7	0
5227	5227	2314	1	0
5228	5228	2314	11	0
5229	5229	2314	3	0
5230	5230	2314	1	0
5231	5231	2314	6	0
5232	5232	2314	1	0
5233	5233	2314	3	0
5234	5234	2314	1	0
5235	5235	2314	0	0
5236	5236	2314	0	0
5237	5237	2314	0	0
5238	5238	2314	0	0
5239	5239	2314	0	0
5240	5240	2314	0	0
5241	5241	2314	0	0
5242	5242	2314	0	0
5243	5243	2314	B0L5	0
5244	5244	2314	0.8	0
5245	5245	2314	1	0
5246	5246	2314	29	0
5247	5247	2314	15	0
5248	5248	2314	32.8	0
5249	5249	2314	22	0
5250	5250	2314	4.7	0
5251	5251	2314	126	0
5252	5252	2314	2.4	0
5253	5253	2314	2.1	0
5254	5254	2314	9	0
5255	5255	2314	10	0
5256	5256	2314	7	0
5257	5257	2314	52	0
5258	5258	2314	100	0
5259	5259	2314	100	0
5260	5260	2314	10	0
5261	5261	2314	20	0
5262	5262	2314	90	0
5263	5263	2314	0	0
5264	5264	2314	1.53	0
5265	5265	2314	16.16666667	0
5266	5266	2314	6.55	0
5267	5267	2314	50	0
5268	5268	2314	125	0
5269	5269	2314	2.18	0
5270	5270	2314	58	0
5271	5271	2314	36.5	0
5272	5272	2314	2.84	0
5273	5273	2314	0	0
5274	5274	2314	0	0
5275	5275	2314	60	0
5276	5276	2314	30	0
5277	5277	2314	0	0
5278	5278	2314	90	0
5279	5279	2314	25	0
5280	5280	2314	0	0
5281	5281	2314	50	0
5282	5282	2314	125	0
5283	5283	2314	2.18	0
5284	5284	2314	58	0
5285	5285	2314	36.5	0
5286	5286	2314	2.84	0
5287	5287	2314	0	0
5288	5288	2314	0	0
5289	5289	2314	60	0
5290	5290	2314	30	0
5291	5291	2314	0	0
5292	5292	2314	96.7	0
5293	5293	2314	39.2	0
5294	5294	2314	49.5	0
5295	5295	2314	23.5	0
5296	5296	2314	6.7	0
5297	5297	2314	6.3	0
5298	5298	2314	5.34	0
5299	5299	2314	1.5	0
5300	5300	2314	5.3792	0
5301	5301	2314	1.5298	0
5302	5302	2314	7	0
5303	5303	2314	1	0
5304	5304	2314	1	0
5305	5305	2314	2	0
5306	5306	2314	0.3	0
5307	5307	2314	0.4	0
5308	5308	2314		0
5309	5309	2314		0
5310	5310	2314	2	0
5311	5311	2314	0	0
5312	5312	2314	0.7	0
5313	5313	2314	0.3	0
5314	5314	2314	0.4	0
5315	5315	2314	0.6	0
5316	5316	2314	0.3	0
5317	5317	2314	0.4	0
5318	5318	2314		0
5319	5319	2314		0
5320	5320	2314	0	0
5321	5321	2314	5	0
5322	5322	2314	0	0
5323	5323	2314	.	0
5324	5324	2314	5	0
5325	5325	2314	0	0
5326	5326	2314	0	0
5327	5327	2314	0	0
5328	5328	2314	0	0
5329	5329	2314	0	0
5330	5330	2314	0	0
5331	5331	2314	0	0
5332	5332	2314	0	0
5333	5333	2314	0	0
5334	5334	2314	0	0
5335	5335	2314	0	0
5336	5336	2314	B0L4	0
5337	5337	2314	0.7	0
5338	5338	2314	0.95	0
5339	5339	2314	29.1	0
5340	5340	2314	12	0
5341	5341	2314	26.2	0
5342	5342	2314	10	0
5343	5343	2314	2.8	0
5344	5344	2314	69	0
5345	5345	2314	2.3	0
5346	5346	2314	2.2	0
5347	5347	2314	0	0
5348	5348	2314	7	0
5349	5349	2314	0	0
5350	5350	2314	89	0
5351	5351	2314	150	0
5352	5352	2314	150	0
5353	5353	2314	10	0
5354	5354	2314	30	0
5355	5355	2314	85	0
5356	5356	2314	0	0
5357	5357	2314	1.546666667	0
5358	5358	2314	14.18333333	0
5359	5359	2314	4.1	0
5360	5360	2314	200	0
5361	5361	2314	200	0
5362	5362	2314	2.18	0
5363	5363	2314	111	0
5364	5364	2314	35	0
5365	5365	2314	2.04	0
5366	5366	2314	0	0
5367	5367	2314	0	0
5368	5368	2314	90	0
5369	5369	2314	0	0
5370	5370	2314	0	0
5371	5371	2314	75	0
5372	5372	2314	24	0
5373	5373	2314	8	0
5374	5374	2314	200	0
5375	5375	2314	200	0
5376	5376	2314	2.18	0
5377	5377	2314	111	0
5378	5378	2314	35	0
5379	5379	2314	2.04	0
5380	5380	2314	0	0
5381	5381	2314	0	0
5382	5382	2314	90	0
5383	5383	2314	0	0
5384	5384	2314	0	0
5385	5385	2314	49.1	0
5386	5386	2314	19.5	0
5387	5387	2314	65.4	0
5388	5388	2314	29.6	0
5389	5389	2314	22.1	0
5390	5390	2314	13	0
5391	5391	2314	5.99	0
5392	5392	2314	0.6	0
5393	5393	2314	6.0095	0
5394	5394	2314	0.6426	0
5395	5395	2314	6	0
5396	5396	2314	2	0
5397	5397	2314	1	0
5398	5398	2314	2	0
5399	5399	2314	0.4	0
5400	5400	2314	0.5	0
5401	5401	2314		0
5402	5402	2314		0
5403	5403	2314	2	0
5404	5404	2314	0	0
5405	5405	2314	0.6	0
5406	5406	2314	0.3	0
5407	5407	2314	0.5	0
5408	5408	2314	0.6	0
5409	5409	2314	0.4	0
5410	5410	2314	0.5	0
5411	5411	2314		0
5412	5412	2314		0
5413	5413	2314	2	0
5414	5414	2314	12	0
5415	5415	2314	3	0
5416	5416	2314	1	0
5417	5417	2314	6	0
5418	5418	2314	3	0
5419	5419	2314	2	0
5420	5420	2314	1	0
5421	5421	2314	0	0
5422	5422	2314	0	0
5423	5423	2314	0	0
5424	5424	2314	0	0
5425	5425	2314	0	0
5426	5426	2314	0	0
5427	5427	2314	0	0
5428	5428	2314	0	0
5429	5429	2314	B0L5	0
5430	5430	2314	0.8	0
5431	5431	2314	1.1	0
5432	5432	2314	15.8	0
5433	5433	2314	11	0
5434	5434	2314	28	0
5435	5435	2314	33	0
5436	5436	2314	8.5	0
5437	5437	2314	186	0
5438	5438	2314	2.2	0
5439	5439	2314	2	0
5440	5440	2314	0	0
5441	5441	2314	6	0
5442	5442	2314	0	0
5443	5443	2314	68.9	0
5444	5444	2314	150	0
5445	5445	2314	150	0
5446	5446	2314	35	0
5447	5447	2314	25	0
5448	5448	2314	85	0
5449	5449	2314	0	0
5450	5450	2314	2.618333333	0
5451	5451	2314	30.93333333	0
5452	5452	2314	7.25	0
5453	5453	2314	125	0
5454	5454	2314	125	0
5455	5455	2314	2.08	0
5456	5456	2314	108	0
5457	5457	2314	30	0
5458	5458	2314	1.39	0
5459	5459	2314	0	0
5460	5460	2314	0	0
5461	5461	2314	85	0
5462	5462	2314	0	0
5463	5463	2314	0	0
5464	5464	2314	120	0
5465	5465	2314	97	0
5466	5466	2314	12	0
5467	5467	2314	125	0
5468	5468	2314	125	0
5469	5469	2314	2.08	0
5470	5470	2314	108	0
5471	5471	2314	30	0
5472	5472	2314	1.39	0
5473	5473	2314	0	0
5474	5474	2314	0	0
5475	5475	2314	85	0
5476	5476	2314	0	0
5477	5477	2314	0	0
5478	5478	2314	49.6	0
5479	5479	2314	29	0
5480	5480	2314	14.5	0
5481	5481	2314	13.7	0
5482	5482	2314	7	0
5483	5483	2314	6.6	0
5484	5484	2314	11.08	0
5485	5485	2314	2.07	0
5486	5486	2314	11.109	0
5487	5487	2314	2.0903	0
5488	5488	2314	4	0
5489	5489	2314	17	0
5490	5490	2314	1	0
5491	5491	2314	1	0
5492	5492	2314	0.4	0
5493	5493	2314	0.5	0
5494	5494	2314		0
5495	5495	2314		0
5496	5496	2314	2	0
5497	5497	2314	0	0
5498	5498	2314	0.7	0
5499	5499	2314	0.3	0
5500	5500	2314	0.4	0
5501	5501	2314	0.6	0
5502	5502	2314	0.4	0
5503	5503	2314	0.4	0
5504	5504	2314		0
5505	5505	2314		0
5506	5506	2314	0	0
5507	5507	2314	7	0
5508	5508	2314	1	0
5509	5509	2314	2	0
5510	5510	2314	6	0
5511	5511	2314	1	0
5512	5512	2314	0	0
5513	5513	2314	0	0
5514	5514	2314	0	0
5515	5515	2314	0	0
5516	5516	2314	0	0
5517	5517	2314	0	0
5518	5518	2314	0	0
5519	5519	2314	0	0
5520	5520	2314	0	0
5521	5521	2314	0	0
5522	5522	2314	B0L4	0
5523	5523	2314	1	0
5524	5524	2314	1.1	0
5525	5525	2314	26.8	0
5526	5526	2314	11	0
5527	5527	2314	30.7	0
5528	5528	2314	13	0
5529	5529	2314	2.3	0
5530	5530	2314	75	0
5531	5531	2314	2.5	0
5532	5532	2314	2.1	0
5533	5533	2314	0	0
5534	5534	2314	4	0
5535	5535	2314	0	0
5536	5536	2314	42.5	0
5537	5537	2314	20	0
5538	5538	2314	20	0
5539	5539	2314	0	0
5540	5540	2314	0	0
5541	5541	2314	20	0
5542	5542	2314	0	0
5543	5543	2314		0
5544	5544	2314		0
5545	5545	2314		0
5546	5546	2314	0	0
5547	5547	2314	150	0
5548	5548	2314		0
5549	5549	2314	48	0
5550	5550	2314		0
5551	5551	2314		0
5552	5552	2314		0
5553	5553	2314		0
5554	5554	2314	0	0
5555	5555	2314	0	0
5556	5556	2314	0	0
5557	5557	2314	19	0
5558	5558	2314	0	0
5559	5559	2314	0	0
5560	5560	2314	0	0
5561	5561	2314	150	0
5562	5562	2314	.	0
5563	5563	2314	48	0
5564	5564	2314	.	0
5565	5565	2314	.	0
5566	5566	2314	.	0
5567	5567	2314	.	0
5568	5568	2314	0	0
5569	5569	2314	0	0
5570	5570	2314	0	0
5571	5571	2314	.	0
5572	5572	2314	.	0
5573	5573	2314	.	0
5574	5574	2314	.	0
5575	5575	2314	.	0
5576	5576	2314	.	0
5577	5577	2314	1.45	0
5578	5578	2314	.	0
5579	5579	2314	1.45	0
5580	5580	2314	.	0
5581	5581	2314	11	0
5582	5582	2314	2	0
5583	5583	2314	1	0
5584	5584	2314	3	0
5585	5585	2314	0.3	0
5586	5586	2314	0.3	0
5587	5587	2314	0.6	0
5588	5588	2314	0.7	0
5589	5589	2314	3	0
5590	5590	2314	0	0
5591	5591	2314	1	0
5592	5592	2314	0.4	0
5593	5593	2314	.	0
5594	5594	2314	.	0
5595	5595	2314	0.4	0
5596	5596	2314	0.6	0
5597	5597	2314	0.3	0
5598	5598	2314	0.5	0
5599	5599	2314	0	0
5600	5600	2314	11	0
5601	5601	2314	3	0
5602	5602	2314	2	0
5603	5603	2314	6	0
5604	5604	2314	2	0
5605	5605	2314	2	0
5606	5606	2314	1	0
5607	5607	2314	0	0
5608	5608	2314	0	0
5609	5609	2314	0	0
5610	5610	2314	0	0
5611	5611	2314	0	0
5612	5612	2314	0	0
5613	5613	2314	0	0
5614	5614	2314	0	0
5615	5615	2314	B0L5	0
5616	5616	2314	0.7	0
5617	5617	2314	1.1	0
5618	5618	2314	24.7	0
5619	5619	2314	11	0
5620	5620	2314	30.8	0
5621	5621	2314	25	0
5622	5622	2314	6.6	0
5623	5623	2314	115	0
5624	5624	2314	2.4	0
5625	5625	2314	2.3	0
5626	5626	2314	0	0
5627	5627	2314	0	0
5628	5628	2314	0	0
5629	5629	2314	75	0
5630	5630	2314	600	0
5631	5631	2314	600	0
5632	5632	2314	20	0
5633	5633	2314	60	0
5634	5634	2314	95	0
5635	5635	2314	0	0
5636	5636	2314	2.166666667	0
5637	5637	2314	26.3	0
5638	5638	2314	6.5	0
5639	5639	2314	250	0
5640	5640	2314	300	0
5641	5641	2314	2.27	0
5642	5642	2314	105	0
5643	5643	2314	60	0
5644	5644	2314	4.85	0
5645	5645	2314	0	0
5646	5646	2314	0	0
5647	5647	2314	90	0
5648	5648	2314	0	0
5649	5649	2314	0	0
5650	5650	2314	331	0
5651	5651	2314	296	0
5652	5652	2314	53	0
5653	5653	2314	250	0
5654	5654	2314	300	0
5655	5655	2314	2.27	0
5656	5656	2314	105	0
5657	5657	2314	60	0
5658	5658	2314	4.85	0
5659	5659	2314	0	0
5660	5660	2314	0	0
5661	5661	2314	90	0
5662	5662	2314	0	0
5663	5663	2314	0	0
5664	5664	2314	63.2	0
5665	5665	2314	29.2	0
5666	5666	2314	237.3	0
5667	5667	2314	85.6	0
5668	5668	2314	37.6	0
5669	5669	2314	18.8	0
5670	5670	2314	31.89	0
5671	5671	2314	5.01	0
5672	5672	2314	31.9192	0
5673	5673	2314	5.1144	0
5674	5674	2314	.	0
5675	5675	2314	.	0
5676	5676	2314	1	0
5677	5677	2314	0	0
5678	5678	2314		0
5679	5679	2314		0
5680	5680	2314		0
5681	5681	2314		0
5682	5682	2314	0	0
5683	5683	2314	0	0
5684	5684	2314	0.5	0
5685	5685	2314	0.2	0
5686	5686	2314		0
5687	5687	2314		0
5688	5688	2314		0
5689	5689	2314		0
5690	5690	2314		0
5691	5691	2314		0
5692	5692	2314	0	0
5693	5693	2314	0	0
5694	5694	2314	0	0
5695	5695	2314	.	0
5696	5696	2314	0	0
5697	5697	2314	.	0
5698	5698	2314	.	0
5699	5699	2314	0	0
5700	5700	2314	0	0
5701	5701	2314	0	0
5702	5702	2314	0	0
5703	5703	2314	0	0
5704	5704	2314	0	0
5705	5705	2314	0	0
5706	5706	2314	0	0
5707	5707	2314	0	0
5708	5708	2314	.	0
5709	5709	2314	.	0
5710	5710	2314	.	0
5711	5711	2314	.	0
5712	5712	2314	0	0
5713	5713	2314	.	0
5714	5714	2314	0	0
5715	5715	2314	.	0
5716	5716	2314	0	0
5717	5717	2314	.	0
5718	5718	2314	.	0
5719	5719	2314	0	0
5720	5720	2314	0	0
5721	5721	2314	0	0
5722	5722	2314		0
5723	5723	2314		0
5724	5724	2314	0	0
5725	5725	2314		0
5726	5726	2314		0
5727	5727	2314		0
5728	5728	2314		0
5729	5729	2314		0
5730	5730	2314		0
5731	5731	2314		0
5732	5732	2314		0
5733	5733	2314		0
5734	5734	2314		0
5735	5735	2314		0
5736	5736	2314		0
5737	5737	2314		0
5738	5738	2314		0
5739	5739	2314		0
5740	5740	2314		0
5741	5741	2314		0
5742	5742	2314		0
5743	5743	2314		0
5744	5744	2314		0
5745	5745	2314		0
5746	5746	2314	.	0
5747	5747	2314	.	0
5748	5748	2314	.	0
5749	5749	2314	.	0
5750	5750	2314	.	0
5751	5751	2314	.	0
5752	5752	2314	.	0
5753	5753	2314	.	0
5754	5754	2314	.	0
5755	5755	2314	.	0
5756	5756	2314	.	0
5757	5757	2314	.	0
5758	5758	2314	.	0
5759	5759	2314	.	0
5760	5760	2314	.	0
5761	5761	2314	.	0
5762	5762	2314	.	0
5763	5763	2314		0
5764	5764	2314		0
5765	5765	2314	.	0
5766	5766	2314	.	0
5767	5767	2314	.	0
5768	5768	2314	.	0
5769	5769	2314	2	0
5770	5770	2314	2	0
5771	5771	2314	0.4	0
5772	5772	2314	0.4	0
5773	5773	2314	0.5	0
5774	5774	2314	0.4	0
5775	5775	2314	3	0
5776	5776	2314	0	0
5777	5777	2314	.	0
5778	5778	2314	.	0
5779	5779	2314	0.3	0
5780	5780	2314	0.4	0
5781	5781	2314	0.2	0
5782	5782	2314	0.4	0
5783	5783	2314		0
5784	5784	2314		0
5785	5785	2314	1	0
5786	5786	2314	4	0
5787	5787	2314	0	0
5788	5788	2314	.	0
5789	5789	2314	4	0
5790	5790	2314	.	0
5791	5791	2314	.	0
5792	5792	2314	0	0
5793	5793	2314	0	0
5794	5794	2314	0	0
5795	5795	2314	0	0
5796	5796	2314	0	0
5797	5797	2314	0	0
5798	5798	2314	0	0
5799	5799	2314	0	0
5800	5800	2314	0	0
5801	5801	2314	B0L3	0
5802	5802	2314	0.1	0
5803	5803	2314	0.2	0
5804	5804	2314	.	0
5805	5805	2314	0	0
5806	5806	2314	.	0
5807	5807	2314	0	0
5808	5808	2314	.	0
5809	5809	2314	0	0
5810	5810	2314	.	0
5811	5811	2314	.	0
5812	5812	2314	0	0
5813	5813	2314	0	0
5814	5814	2314	0	0
5815	5815	2314		0
5816	5816	2314		0
5817	5817	2314	0	0
5818	5818	2314		0
5819	5819	2314		0
5820	5820	2314		0
5821	5821	2314		0
5822	5822	2314		0
5823	5823	2314		0
5824	5824	2314		0
5825	5825	2314		0
5826	5826	2314		0
5827	5827	2314		0
5828	5828	2314		0
5829	5829	2314		0
5830	5830	2314		0
5831	5831	2314		0
5832	5832	2314		0
5833	5833	2314		0
5834	5834	2314		0
5835	5835	2314		0
5836	5836	2314		0
5837	5837	2314		0
5838	5838	2314		0
5839	5839	2314	.	0
5840	5840	2314	.	0
5841	5841	2314	.	0
5842	5842	2314	.	0
5843	5843	2314	.	0
5844	5844	2314	.	0
5845	5845	2314	.	0
5846	5846	2314	.	0
5847	5847	2314	.	0
5848	5848	2314	.	0
5849	5849	2314	.	0
5850	5850	2314	.	0
5851	5851	2314	.	0
5852	5852	2314	.	0
5853	5853	2314	.	0
5854	5854	2314	.	0
5855	5855	2314	.	0
5856	5856	2314		0
5857	5857	2314		0
5858	5858	2314	.	0
5859	5859	2314	.	0
5860	5860	2314	9	0
5861	5861	2314	23	0
5862	5862	2314	1	0
5863	5863	2314	2	0
5864	5864	2314	0.7	0
5865	5865	2314	1	0
5866	5866	2314		0
5867	5867	2314		0
5868	5868	2314	3	0
5869	5869	2314	0	0
5870	5870	2314	1	0
5871	5871	2314	0.5	0
5872	5872	2314	0.8	0
5873	5873	2314	1.2	0
5874	5874	2314	0.6	0
5875	5875	2314	0.9	0
5876	5876	2314		0
5877	5877	2314		0
5878	5878	2314	0	0
5879	5879	2314	16	0
5880	5880	2314	4	0
5881	5881	2314	1	0
5882	5882	2314	7	0
5883	5883	2314	3	0
5884	5884	2314	3	0
5885	5885	2314	2	0
5886	5886	2314	1	0
5887	5887	2314	0	0
5888	5888	2314	0	0
5889	5889	2314	0	0
5890	5890	2314	0	0
5891	5891	2314	0	0
5892	5892	2314	0	0
5893	5893	2314	0	0
5894	5894	2314	B0L6	0
5895	5895	2314	1.1	0
5896	5896	2314	1.3	0
5897	5897	2314	16.3	0
5898	5898	2314	11	0
5899	5899	2314	36.1	0
5900	5900	2314	41	0
5901	5901	2314	20.7	0
5902	5902	2314	253	0
5903	5903	2314	2.9	0
5904	5904	2314	2.7	0
5905	5905	2314	0	0
5906	5906	2314	0	0
5907	5907	2314	0	0
5908	5908	2314	108.5	0
5909	5909	2314	700	0
5910	5910	2314	700	0
5911	5911	2314	10	0
5912	5912	2314	80	0
5913	5913	2314	90	0
5914	5914	2314	0	0
5915	5915	2314	2.448333333	0
5916	5916	2314	26	0
5917	5917	2314	7.8	0
5918	5918	2314	150	0
5919	5919	2314	200	0
5920	5920	2314	1.98	0
5921	5921	2314	120	0
5922	5922	2314	49	0
5923	5923	2314	4.37	0
5924	5924	2314	0	0
5925	5925	2314	9	0
5926	5926	2314	70	0
5927	5927	2314	0	0
5928	5928	2314	0	0
5929	5929	2314	496	0
5930	5930	2314	357	0
5931	5931	2314	90	0
5932	5932	2314	150	0
5933	5933	2314	200	0
5934	5934	2314	1.98	0
5935	5935	2314	120	0
5936	5936	2314	49	0
5937	5937	2314	4.37	0
5938	5938	2314	0	0
5939	5939	2314	9	0
5940	5940	2314	70	0
5941	5941	2314	0	0
5942	5942	2314	0	0
5943	5943	2314	37.5	0
5944	5944	2314	14.7	0
5945	5945	2314	133	0
5946	5946	2314	71.2	0
5947	5947	2314	29.8	0
5948	5948	2314	21.6	0
5949	5949	2314	36.55	0
5950	5950	2314	8.33	0
5951	5951	2314	36.5647	0
5952	5952	2314	8.4228	0
5953	5953	2314	15	0
5954	5954	2314	11	0
5955	5955	2314	2	0
5956	5956	2314	0	0
5957	5957	2314		0
5958	5958	2314		0
5959	5959	2314		0
5960	5960	2314		0
5961	5961	2314	1	0
5962	5962	2314	0	0
5963	5963	2314	0.9	0
5964	5964	2314	0.5	0
5965	5965	2314	0.5	0
5966	5966	2314	0.8	0
5967	5967	2314		0
5968	5968	2314		0
5969	5969	2314		0
5970	5970	2314		0
5971	5971	2314	2	0
5972	5972	2314	7	0
5973	5973	2314	2	0
5974	5974	2314	1	0
5975	5975	2314	5	0
5976	5976	2314	1	0
5977	5977	2314	1	0
5978	5978	2314	0	0
5979	5979	2314	0	0
5980	5980	2314	0	0
5981	5981	2314	0	0
5982	5982	2314	0	0
5983	5983	2314	0	0
5984	5984	2314	0	0
5985	5985	2314	0	0
5986	5986	2314	0	0
5987	5987	2314	B0L4	0
5988	5988	2314	0.8	0
5989	5989	2314	1	0
5990	5990	2314	23.9	0
5991	5991	2314	13	0
5992	5992	2314	29.5	0
5993	5993	2314	19	0
5994	5994	2314	7.1	0
5995	5995	2314	106	0
5996	5996	2314	2.4	0
5997	5997	2314	2.3	0
5998	5998	2314	0	0
5999	5999	2314	0	0
6000	6000	2314	0	0
6001	6001	2314	96	0
6002	6002	2314	250	0
6003	6003	2314	250	0
6004	6004	2314	5	0
6005	6005	2314	50	0
6006	6006	2314	75	0
6007	6007	2314	0	0
6008	6008	2314	2.065	0
6009	6009	2314	23.31666667	0
6010	6010	2314	7.4	0
6011	6011	2314	100	0
6012	6012	2314	110	0
6013	6013	2314	3.05	0
6014	6014	2314	100	0
6015	6015	2314	44	0
6016	6016	2314	4	0
6017	6017	2314	0	0
6018	6018	2314	0	0
6019	6019	2314	85	0
6020	6020	2314	0	0
6021	6021	2314	0	0
6022	6022	2314	224	0
6023	6023	2314	126	0
6024	6024	2314	20	0
6025	6025	2314	100	0
6026	6026	2314	110	0
6027	6027	2314	3.05	0
6028	6028	2314	100	0
6029	6029	2314	44	0
6030	6030	2314	4	0
6031	6031	2314	0	0
6032	6032	2314	0	0
6033	6033	2314	85	0
6034	6034	2314	0	0
6035	6035	2314	0	0
6036	6036	2314	68.7	0
6037	6037	2314	44	0
6038	6038	2314	126.4	0
6039	6039	2314	76.9	0
6040	6040	2314	10.1	0
6041	6041	2314	8.6	0
6042	6042	2314	17.2	0
6043	6043	2314	2.68	0
6044	6044	2314	17.244	0
6045	6045	2314	2.7655	0
6046	6046	2314	2	0
6047	6047	2314	1	0
6048	6048	2314	0	0
6049	6049	2314	3	0
6050	6050	2314	0.7	0
6051	6051	2314	0.7	0
6052	6052	2314	0.8	0
6053	6053	2314	1	0
6054	6054	2314	4	0
6055	6055	2314	0	0
6056	6056	2314	1.2	0
6057	6057	2314	0.6	0
6058	6058	2314	.	0
6059	6059	2314	.	0
6060	6060	2314	0.6	0
6061	6061	2314	0.8	0
6062	6062	2314	0.7	0
6063	6063	2314	1	0
6064	6064	2314	1	0
6065	6065	2314	23	0
6066	6066	2314	7	0
6067	6067	2314	1	0
6068	6068	2314	7	0
6069	6069	2314	4	0
6070	6070	2314	4	0
6071	6071	2314	3	0
6072	6072	2314	2	0
6073	6073	2314	1	0
6074	6074	2314	0	0
6075	6075	2314	1	0
6076	6076	2314	0	0
6077	6077	2314	1	0
6078	6078	2314	0	0
6079	6079	2314	0	0
6080	6080	2314	B0L6	0
6081	6081	2314	1.1	0
6082	6082	2314	1.2	0
6083	6083	2314	14.1	0
6084	6084	2314	11	0
6085	6085	2314	41.8	0
6086	6086	2314	48	0
6087	6087	2314	19.2	0
6088	6088	2314	251	0
6089	6089	2314	2.9	0
6090	6090	2314	2.3	0
6091	6091	2314	2	0
6092	6092	2314	2	0
6093	6093	2314	0	0
6094	6094	2314	78	0
6095	6095	2314	400	0
6096	6096	2314	400	0
6097	6097	2314	0	0
6098	6098	2314	50	0
6099	6099	2314	90	0
6100	6100	2314	0	0
6101	6101	2314	2.093333333	0
6102	6102	2314	26.7	0
6103	6103	2314	6.666666667	0
6104	6104	2314	150	0
6105	6105	2314	200	0
6106	6106	2314	3.27	0
6107	6107	2314	98	0
6108	6108	2314	52	0
6109	6109	2314	5.75	0
6110	6110	2314	0	0
6111	6111	2314	1	0
6112	6112	2314	70	0
6113	6113	2314	0	0
6114	6114	2314	0	0
6115	6115	2314	283	0
6116	6116	2314	39	0
6117	6117	2314	9	0
6118	6118	2314	150	0
6119	6119	2314	200	0
6120	6120	2314	3.27	0
6121	6121	2314	98	0
6122	6122	2314	52	0
6123	6123	2314	5.75	0
6124	6124	2314	0	0
6125	6125	2314	1	0
6126	6126	2314	70	0
6127	6127	2314	0	0
6128	6128	2314	0	0
6129	6129	2314	82.7	0
6130	6130	2314	44.1	0
6131	6131	2314	292.9	0
6132	6132	2314	113.1	0
6133	6133	2314	10.8	0
6134	6134	2314	10.2	0
6135	6135	2314	18.1	0
6136	6136	2314	3.82	0
6137	6137	2314	18.1441	0
6138	6138	2314	3.9433	0
6139	6139	2314	19	0
6140	6140	2314	85	0
6141	6141	2314	2	0
6142	6142	2314	3	0
6143	6143	2314	0.7	0
6144	6144	2314	0.9	0
6145	6145	2314	1.3	0
6146	6146	2314	1.5	0
6147	6147	2314	3	0
6148	6148	2314	0	0
6149	6149	2314	1	0
6150	6150	2314	0.6	0
6151	6151	2314	0.7	0
6152	6152	2314	1	0
6153	6153	2314	0.8	0
6154	6154	2314	1.1	0
6155	6155	2314	1	0
6156	6156	2314	1.2	0
6157	6157	2314	1	0
6158	6158	2314	24	0
6159	6159	2314	6	0
6160	6160	2314	1	0
6161	6161	2314	9	0
6162	6162	2314	4	0
6163	6163	2314	4	0
6164	6164	2314	3	0
6165	6165	2314	2	0
6166	6166	2314	1	0
6167	6167	2314	0	0
6168	6168	2314	0	0
6169	6169	2314	0	0
6170	6170	2314	1	0
6171	6171	2314	0	0
6172	6172	2314	0	0
6173	6173	2314	B0L7	0
6174	6174	2314	1.5	0
6175	6175	2314	1.7	0
6176	6176	2314	35.7	0
6177	6177	2314	15	0
6178	6178	2314	49.7	0
6179	6179	2314	65	0
6180	6180	2314	33.3	0
6181	6181	2314	426	0
6182	6182	2314	2.9	0
6183	6183	2314	2.4	0
6184	6184	2314	28	0
6185	6185	2314	29	0
6186	6186	2314	6	0
6187	6187	2314	85.5	0
6188	6188	2314	225	0
6189	6189	2314	225	0
6190	6190	2314	5	0
6191	6191	2314	70	0
6192	6192	2314	90	0
6193	6193	2314	1	0
6194	6194	2314	2.293333333	0
6195	6195	2314	24.61666667	0
6196	6196	2314	5.766666667	0
6197	6197	2314	100	0
6198	6198	2314	250	0
6199	6199	2314	2.39	0
6200	6200	2314	100	0
6201	6201	2314	51	0
6202	6202	2314	3.78	0
6203	6203	2314	14	0
6204	6204	2314	0	0
6205	6205	2314	30	0
6206	6206	2314	25	0
6207	6207	2314	0	0
6208	6208	2314	245	0
6209	6209	2314	62	0
6210	6210	2314	13	0
6211	6211	2314	100	0
6212	6212	2314	250	0
6213	6213	2314	2.39	0
6214	6214	2314	100	0
6215	6215	2314	51	0
6216	6216	2314	3.78	0
6217	6217	2314	14	0
6218	6218	2314	0	0
6219	6219	2314	30	0
6220	6220	2314	25	0
6221	6221	2314	0	0
6222	6222	2314	42.8	0
6223	6223	2314	22.6	0
6224	6224	2314	113.5	0
6225	6225	2314	52.4	0
6226	6226	2314	7.7	0
6227	6227	2314	6.7	0
6228	6228	2314	12.28	0
6229	6229	2314	2.15	0
6230	6230	2314	12.3026	0
6231	6231	2314	2.2091	0
6232	6232	2314	23	0
6233	6233	2314	6	0
6234	6234	2314	1	0
6235	6235	2314	2	0
6236	6236	2314	0.4	0
6237	6237	2314	0.4	0
6238	6238	2314	0.4	0
6239	6239	2314	0.4	0
6240	6240	2314	3	0
6241	6241	2314	0	0
6242	6242	2314	0.5	0
6243	6243	2314	0.3	0
6244	6244	2314	0.4	0
6245	6245	2314	0.6	0
6246	6246	2314	0.3	0
6247	6247	2314	0.5	0
6248	6248	2314		0
6249	6249	2314		0
6250	6250	2314	0	0
6251	6251	2314	8	0
6252	6252	2314	1	0
6253	6253	2314	2	0
6254	6254	2314	7	0
6255	6255	2314	1	0
6256	6256	2314	.	0
6257	6257	2314	0	0
6258	6258	2314	0	0
6259	6259	2314	0	0
6260	6260	2314	0	0
6261	6261	2314	0	0
6262	6262	2314	0	0
6263	6263	2314	0	0
6264	6264	2314	0	0
6265	6265	2314	0	0
6266	6266	2314	B0L5	0
6267	6267	2314	0.8	0
6268	6268	2314	1	0
6269	6269	2314	15.8	0
6270	6270	2314	10	0
6271	6271	2314	24.4	0
6272	6272	2314	13	0
6273	6273	2314	5.1	0
6274	6274	2314	78	0
6275	6275	2314	2	0
6276	6276	2314	1.8	0
6277	6277	2314	0	0
6278	6278	2314	0	0
6279	6279	2314	0	0
6280	6280	2314	77	0
6281	6281	2314	250	0
6282	6282	2314	250	0
6283	6283	2314	2	0
6284	6284	2314	40	0
6285	6285	2314	95	0
6286	6286	2314	0	0
6287	6287	2314	1.771666667	0
6288	6288	2314	21.51666667	0
6289	6289	2314	6.333333333	0
6290	6290	2314	150	0
6291	6291	2314	150	0
6292	6292	2314	2.8	0
6293	6293	2314	90	0
6294	6294	2314	45	0
6295	6295	2314	3.15	0
6296	6296	2314	23	0
6297	6297	2314	0	0
6298	6298	2314	90	0
6299	6299	2314	10	0
6300	6300	2314	0	0
6301	6301	2314	106	0
6302	6302	2314	74	0
6303	6303	2314	48	0
6304	6304	2314	150	0
6305	6305	2314	150	0
6306	6306	2314	2.8	0
6307	6307	2314	90	0
6308	6308	2314	45	0
6309	6309	2314	3.15	0
6310	6310	2314	23	0
6311	6311	2314	0	0
6312	6312	2314	90	0
6313	6313	2314	10	0
6314	6314	2314	0	0
6315	6315	2314	36.9	0
6316	6316	2314	19.6	0
6317	6317	2314	61.1	0
6318	6318	2314	37.7	0
6319	6319	2314	7.7	0
6320	6320	2314	7.2	0
6321	6321	2314	13.91	0
6322	6322	2314	3	0
6323	6323	2314	13.9296	0
6324	6324	2314	3.0449	0
6325	6325	2314	.	0
6326	6326	2314	.	0
6327	6327	2314	1	0
6328	6328	2314	1	0
6329	6329	2314	0.2	0
6330	6330	2314	0.2	0
6331	6331	2314		0
6332	6332	2314		0
6333	6333	2314	1	0
6334	6334	2314	0	0
6335	6335	2314	0.5	0
6336	6336	2314	0.2	0
6337	6337	2314	0.3	0
6338	6338	2314	0.3	0
6339	6339	2314		0
6340	6340	2314		0
6341	6341	2314		0
6342	6342	2314		0
6343	6343	2314	0	0
6344	6344	2314	0	0
6345	6345	2314	0	0
6346	6346	2314	.	0
6347	6347	2314	0	0
6348	6348	2314	.	0
6349	6349	2314	.	0
6350	6350	2314	0	0
6351	6351	2314	0	0
6352	6352	2314	0	0
6353	6353	2314	0	0
6354	6354	2314	0	0
6355	6355	2314	0	0
6356	6356	2314	0	0
6357	6357	2314	0	0
6358	6358	2314	0	0
6359	6359	2314	.	0
6360	6360	2314	.	0
6361	6361	2314	.	0
6362	6362	2314	.	0
6363	6363	2314	0	0
6364	6364	2314	.	0
6365	6365	2314	0	0
6366	6366	2314	.	0
6367	6367	2314	0	0
6368	6368	2314	.	0
6369	6369	2314	.	0
6370	6370	2314	0	0
6371	6371	2314	0	0
6372	6372	2314	0	0
6373	6373	2314		0
6374	6374	2314		0
6375	6375	2314	0	0
6376	6376	2314		0
6377	6377	2314		0
6378	6378	2314		0
6379	6379	2314		0
6380	6380	2314		0
6381	6381	2314		0
6382	6382	2314		0
6383	6383	2314		0
6384	6384	2314		0
6385	6385	2314		0
6386	6386	2314		0
6387	6387	2314		0
6388	6388	2314		0
6389	6389	2314		0
6390	6390	2314		0
6391	6391	2314		0
6392	6392	2314		0
6393	6393	2314		0
6394	6394	2314		0
6395	6395	2314		0
6396	6396	2314		0
6397	6397	2314	.	0
6398	6398	2314	.	0
6399	6399	2314	.	0
6400	6400	2314	.	0
6401	6401	2314	.	0
6402	6402	2314	.	0
6403	6403	2314	.	0
6404	6404	2314	.	0
6405	6405	2314	.	0
6406	6406	2314	.	0
6407	6407	2314	.	0
6408	6408	2314	.	0
6409	6409	2314	.	0
6410	6410	2314	.	0
6411	6411	2314	.	0
6412	6412	2314	.	0
6413	6413	2314	.	0
6414	6414	2314		0
6415	6415	2314		0
6416	6416	2314	.	0
6417	6417	2314	.	0
6418	6418	2314	10	0
6419	6419	2314	1	0
6420	6420	2314	2	0
6421	6421	2314	2	0
6422	6422	2314	0.5	0
6423	6423	2314	0.4	0
6424	6424	2314	0.6	0
6425	6425	2314	0.6	0
6426	6426	2314	3	0
6427	6427	2314	0	0
6428	6428	2314	0.6	0
6429	6429	2314	0.4	0
6430	6430	2314	0.4	0
6431	6431	2314	0.4	0
6432	6432	2314		0
6433	6433	2314		0
6434	6434	2314		0
6435	6435	2314		0
6436	6436	2314	0	0
6437	6437	2314	5	0
6438	6438	2314	0	0
6439	6439	2314	.	0
6440	6440	2314	5	0
6441	6441	2314	.	0
6442	6442	2314	.	0
6443	6443	2314	0	0
6444	6444	2314	0	0
6445	6445	2314	0	0
6446	6446	2314	0	0
6447	6447	2314	0	0
6448	6448	2314	0	0
6449	6449	2314	0	0
6450	6450	2314	0	0
6451	6451	2314	0	0
6452	6452	2314	B0L4	0
6453	6453	2314	0.5	0
6454	6454	2314	0.6	0
6455	6455	2314	24.1	0
6456	6456	2314	9	0
6457	6457	2314	18.2	0
6458	6458	2314	1	0
6459	6459	2314	0.5	0
6460	6460	2314	35	0
6461	6461	2314	2.1	0
6462	6462	2314	1.8	0
6463	6463	2314	0	0
6464	6464	2314	0	0
6465	6465	2314	0	0
6466	6466	2314	75.5	0
6467	6467	2314	100	0
6468	6468	2314	100	0
6469	6469	2314	10	0
6470	6470	2314	10	0
6471	6471	2314	90	0
6472	6472	2314	0	0
6473	6473	2314	2.256666667	0
6474	6474	2314	24.38333333	0
6475	6475	2314	6	0
6476	6476	2314	100	0
6477	6477	2314	100	0
6478	6478	2314	2.52	0
6479	6479	2314	85	0
6480	6480	2314	40	0
6481	6481	2314	2.19	0
6482	6482	2314	0	0
6483	6483	2314	0	0
6484	6484	2314	80	0
6485	6485	2314	10	0
6486	6486	2314	0	0
6487	6487	2314	45	0
6488	6488	2314	45	0
6489	6489	2314	45	0
6490	6490	2314	100	0
6491	6491	2314	100	0
6492	6492	2314	2.52	0
6493	6493	2314	85	0
6494	6494	2314	40	0
6495	6495	2314	2.19	0
6496	6496	2314	0	0
6497	6497	2314	0	0
6498	6498	2314	80	0
6499	6499	2314	10	0
6500	6500	2314	0	0
6501	6501	2314	42.1	0
6502	6502	2314	28.8	0
6503	6503	2314	19.9	0
6504	6504	2314	16.6	0
6505	6505	2314	3.4	0
6506	6506	2314	2.6	0
6507	6507	2314	7.49	0
6508	6508	2314	1.44	0
6509	6509	2314	7.5188	0
6510	6510	2314	1.4592	0
6511	6511	2314	2	0
6512	6512	2314	26	0
6513	6513	2314	1	0
6514	6514	2314	3	0
6515	6515	2314	0.7	0
6516	6516	2314	1	0
6517	6517	2314	1.3	0
6518	6518	2314	1.3	0
6519	6519	2314	3	0
6520	6520	2314	0	0
6521	6521	2314	1	0
6522	6522	2314	0.6	0
6523	6523	2314	0.7	0
6524	6524	2314	1.1	0
6525	6525	2314	0.8	0
6526	6526	2314	0.9	0
6527	6527	2314	0.9	0
6528	6528	2314	0.9	0
6529	6529	2314	2	0
6530	6530	2314	21	0
6531	6531	2314	5	0
6532	6532	2314	1	0
6533	6533	2314	8	0
6534	6534	2314	5	0
6535	6535	2314	4	0
6536	6536	2314	2	0
6537	6537	2314	1	0
6538	6538	2314	0	0
6539	6539	2314	0	0
6540	6540	2314	1	0
6541	6541	2314	0	0
6542	6542	2314	0	0
6543	6543	2314	0	0
6544	6544	2314	0	0
6545	6545	2314	B0L7	0
6546	6546	2314	1.1	0
6547	6547	2314	1.2	0
6548	6548	2314	15.5	0
6549	6549	2314	12	0
6550	6550	2314	40.7	0
6551	6551	2314	53	0
6552	6552	2314	22.9	0
6553	6553	2314	289	0
6554	6554	2314	1.5	0
6555	6555	2314	1.2	0
6556	6556	2314	7	0
6557	6557	2314	19	0
6558	6558	2314	2	0
6559	6559	2314	91	0
6560	6560	2314	225	0
6561	6561	2314	225	0
6562	6562	2314	5	0
6563	6563	2314	30	0
6564	6564	2314	80	0
6565	6565	2314	1	0
6566	6566	2314	2.101666667	0
6567	6567	2314	19.46666667	0
6568	6568	2314	6	0
6569	6569	2314	0	0
6570	6570	2314	200	0
6571	6571	2314		0
6572	6572	2314	68	0
6573	6573	2314	28	0
6574	6574	2314		0
6575	6575	2314	0	0
6576	6576	2314	0	0
6577	6577	2314	0	0
6578	6578	2314	0	0
6579	6579	2314	0	0
6580	6580	2314	100	0
6581	6581	2314	0	0
6582	6582	2314	0	0
6583	6583	2314	0	0
6584	6584	2314	200	0
6585	6585	2314	.	0
6586	6586	2314	68	0
6587	6587	2314	28	0
6588	6588	2314	.	0
6589	6589	2314	0	0
6590	6590	2314	0	0
6591	6591	2314	0	0
6592	6592	2314	0	0
6593	6593	2314	0	0
6594	6594	2314	.	0
6595	6595	2314	.	0
6596	6596	2314	.	0
6597	6597	2314	.	0
6598	6598	2314	.	0
6599	6599	2314	.	0
6600	6600	2314	6.51	0
6601	6601	2314	0.32	0
6602	6602	2314	6.51	0
6603	6603	2314	0.32	0
6604	6604	2314	8	0
6605	6605	2314	2	0
6606	6606	2314	0	0
6607	6607	2314	1	0
6608	6608	2314		0
6609	6609	2314		0
6610	6610	2314		0
6611	6611	2314		0
6612	6612	2314	2	0
6613	6613	2314	0	0
6614	6614	2314	0.8	0
6615	6615	2314	0.3	0
6616	6616	2314	0.3	0
6617	6617	2314	0.3	0
6618	6618	2314		0
6619	6619	2314		0
6620	6620	2314		0
6621	6621	2314		0
6622	6622	2314	1	0
6623	6623	2314	5	0
6624	6624	2314	0	0
6625	6625	2314	.	0
6626	6626	2314	5	0
6627	6627	2314	.	0
6628	6628	2314	.	0
6629	6629	2314	0	0
6630	6630	2314	0	0
6631	6631	2314	0	0
6632	6632	2314	0	0
6633	6633	2314	0	0
6634	6634	2314	0	0
6635	6635	2314	0	0
6636	6636	2314	0	0
6637	6637	2314	0	0
6638	6638	2314	B0L4	0
6639	6639	2314	0.6	0
6640	6640	2314	0.7	0
6641	6641	2314	19.1	0
6642	6642	2314	9	0
6643	6643	2314	20.8	0
6644	6644	2314	7	0
6645	6645	2314	1.1	0
6646	6646	2314	62	0
6647	6647	2314	1.9	0
6648	6648	2314	1.6	0
6649	6649	2314	0	0
6650	6650	2314	0	0
6651	6651	2314	0	0
6652	6652	2314	104	0
6653	6653	2314	150	0
6654	6654	2314	150	0
6655	6655	2314	10	0
6656	6656	2314	25	0
6657	6657	2314	80	0
6658	6658	2314	1	0
6659	6659	2314	2.681666667	0
6660	6660	2314	32.85	0
6661	6661	2314	7.933333333	0
6662	6662	2314	100	0
6663	6663	2314	100	0
6664	6664	2314	1.85	0
6665	6665	2314	135	0
6666	6666	2314	34	0
6667	6667	2314	4.56	0
6668	6668	2314	1	0
6669	6669	2314	0	0
6670	6670	2314	90	0
6671	6671	2314	0	0
6672	6672	2314	0	0
6673	6673	2314	79	0
6674	6674	2314	120	0
6675	6675	2314	49	0
6676	6676	2314	100	0
6677	6677	2314	100	0
6678	6678	2314	1.85	0
6679	6679	2314	135	0
6680	6680	2314	34	0
6681	6681	2314	4.56	0
6682	6682	2314	1	0
6683	6683	2314	0	0
6684	6684	2314	90	0
6685	6685	2314	0	0
6686	6686	2314	0	0
6687	6687	2314	24.6	0
6688	6688	2314	15.4	0
6689	6689	2314	273.8	0
6690	6690	2314	110.3	0
6691	6691	2314	7.6	0
6692	6692	2314	7.1	0
6693	6693	2314	12.69	0
6694	6694	2314	2.58	0
6695	6695	2314	12.7054	0
6696	6696	2314	2.6974	0
6697	6697	2314	4	0
6698	6698	2314	15	0
6699	6699	2314	2	0
6700	6700	2314	3	0
6701	6701	2314	0.6	0
6702	6702	2314	1	0
6703	6703	2314	1.4	0
6704	6704	2314	1.5	0
6705	6705	2314	4	0
6706	6706	2314	0	0
6707	6707	2314	1.2	0
6708	6708	2314	0.5	0
6709	6709	2314	0.7	0
6710	6710	2314	1	0
6711	6711	2314	1	0
6712	6712	2314	1.1	0
6713	6713	2314	1	0
6714	6714	2314	1.2	0
6715	6715	2314	2	0
6716	6716	2314	18	0
6717	6717	2314	4	0
6718	6718	2314	1	0
6719	6719	2314	8	0
6720	6720	2314	4	0
6721	6721	2314	3	0
6722	6722	2314	2	0
6723	6723	2314	1	0
6724	6724	2314	0	0
6725	6725	2314	0	0
6726	6726	2314	0	0
6727	6727	2314	0	0
6728	6728	2314	0	0
6729	6729	2314	0	0
6730	6730	2314	0	0
6731	6731	2314	B0L6	0
6732	6732	2314	1.5	0
6733	6733	2314	1.5	0
6734	6734	2314	34.6	0
6735	6735	2314	16	0
6736	6736	2314	35.7	0
6737	6737	2314	42	0
6738	6738	2314	23.4	0
6739	6739	2314	285	0
6740	6740	2314	2.9	0
6741	6741	2314	1.9	0
6742	6742	2314	6	0
6743	6743	2314	4	0
6744	6744	2314	0	0
6745	6745	2314	104.5	0
6746	6746	2314	800	0
6747	6747	2314	800	0
6748	6748	2314	20	0
6749	6749	2314	50	0
6750	6750	2314	90	0
6751	6751	2314	0	0
6752	6752	2314	1.441666667	0
6753	6753	2314	19.43333333	0
6754	6754	2314	5.866666667	0
6755	6755	2314	200	0
6756	6756	2314	500	0
6757	6757	2314	4.07	0
6758	6758	2314	109	0
6759	6759	2314	54	0
6760	6760	2314	5.77	0
6761	6761	2314	5	0
6762	6762	2314	0	0
6763	6763	2314	60	0
6764	6764	2314	0	0
6765	6765	2314	60	0
6766	6766	2314	155	0
6767	6767	2314	135	0
6768	6768	2314	73	0
6769	6769	2314	200	0
6770	6770	2314	500	0
6771	6771	2314	4.07	0
6772	6772	2314	109	0
6773	6773	2314	54	0
6774	6774	2314	5.77	0
6775	6775	2314	5	0
6776	6776	2314	0	0
6777	6777	2314	60	0
6778	6778	2314	0	0
6779	6779	2314	60	0
6780	6780	2314	142.1	0
6781	6781	2314	67.2	0
6782	6782	2314	231.6	0
6783	6783	2314	99.2	0
6784	6784	2314	8.3	0
6785	6785	2314	7.9	0
6786	6786	2314	53.14	0
6787	6787	2314	6.15	0
6788	6788	2314	53.2072	0
6789	6789	2314	6.2571	0
6790	6790	2314	21	0
6791	6791	2314	15	0
6792	6792	2314	1	0
6793	6793	2314	2	0
6794	6794	2314	0.5	0
6795	6795	2314	0.6	0
6796	6796	2314		0
6797	6797	2314		0
6798	6798	2314	3	0
6799	6799	2314	0	0
6800	6800	2314	0.9	0
6801	6801	2314	0.4	0
6802	6802	2314	0.6	0
6803	6803	2314	0.6	0
6804	6804	2314	0.5	0
6805	6805	2314	0.6	0
6806	6806	2314		0
6807	6807	2314		0
6808	6808	2314	1	0
6809	6809	2314	6	0
6810	6810	2314	0	0
6811	6811	2314	.	0
6812	6812	2314	6	0
6813	6813	2314	.	0
6814	6814	2314	.	0
6815	6815	2314	0	0
6816	6816	2314	0	0
6817	6817	2314	0	0
6818	6818	2314	0	0
6819	6819	2314	0	0
6820	6820	2314	0	0
6821	6821	2314	0	0
6822	6822	2314	0	0
6823	6823	2314	0	0
6824	6824	2314	B0L5	0
6825	6825	2314	1	0
6826	6826	2314	1.3	0
6827	6827	2314	23.2	0
6828	6828	2314	7	0
6829	6829	2314	18.1	0
6830	6830	2314	9	0
6831	6831	2314	3.1	0
6832	6832	2314	63	0
6833	6833	2314	2.5	0
6834	6834	2314	2.1	0
6835	6835	2314	0	0
6836	6836	2314	0	0
6837	6837	2314	0	0
6838	6838	2314	52	0
6839	6839	2314	250	0
6840	6840	2314	250	0
6841	6841	2314	0	0
6842	6842	2314	40	0
6843	6843	2314	95	0
6844	6844	2314	0	0
6845	6845	2314	1.956666667	0
6846	6846	2314	23.9	0
6847	6847	2314	6.45	0
6848	6848	2314	75	0
6849	6849	2314	150	0
6850	6850	2314	2.22	0
6851	6851	2314	62	0
6852	6852	2314	41	0
6853	6853	2314	3.86	0
6854	6854	2314	29	0
6855	6855	2314	0	0
6856	6856	2314	60	0
6857	6857	2314	0	0
6858	6858	2314	0	0
6859	6859	2314	114	0
6860	6860	2314	82	0
6861	6861	2314	40	0
6862	6862	2314	75	0
6863	6863	2314	150	0
6864	6864	2314	2.22	0
6865	6865	2314	62	0
6866	6866	2314	41	0
6867	6867	2314	3.86	0
6868	6868	2314	29	0
6869	6869	2314	0	0
6870	6870	2314	60	0
6871	6871	2314	0	0
6872	6872	2314	0	0
6873	6873	2314	46.9	0
6874	6874	2314	25.6	0
6875	6875	2314	62.7	0
6876	6876	2314	35.5	0
6877	6877	2314	4.1	0
6878	6878	2314	3.5	0
6879	6879	2314	11.41	0
6880	6880	2314	1.9	0
6881	6881	2314	11.4356	0
6882	6882	2314	1.939	0
6883	6883	2314	54	0
6884	6884	2314	4	0
6885	6885	2314	1	0
6886	6886	2314	2	0
6887	6887	2314	0.6	0
6888	6888	2314	0.5	0
6889	6889	2314	0.9	0
6890	6890	2314	0.9	0
6891	6891	2314	3	0
6892	6892	2314	0	0
6893	6893	2314	1.2	0
6894	6894	2314	0.6	0
6895	6895	2314	.	0
6896	6896	2314	.	0
6897	6897	2314	0.6	0
6898	6898	2314	0.7	0
6899	6899	2314	0.6	0
6900	6900	2314	0.7	0
6901	6901	2314	1	0
6902	6902	2314	14	0
6903	6903	2314	2	0
6904	6904	2314	2	0
6905	6905	2314	7	0
6906	6906	2314	4	0
6907	6907	2314	3	0
6908	6908	2314	0	0
6909	6909	2314	0	0
6910	6910	2314	0	0
6911	6911	2314	0	0
6912	6912	2314	0	0
6913	6913	2314	0	0
6914	6914	2314	0	0
6915	6915	2314	0	0
6916	6916	2314	0	0
6917	6917	2314	B0L6	0
6918	6918	2314	1.1	0
6919	6919	2314	1.3	0
6920	6920	2314	21.2	0
6921	6921	2314	13	0
6922	6922	2314	34	0
6923	6923	2314	27	0
6924	6924	2314	15.6	0
6925	6925	2314	154	0
6926	6926	2314	2.9	0
6927	6927	2314	2.4	0
6928	6928	2314	0	0
6929	6929	2314	0	0
6930	6930	2314	0	0
6931	6931	2314	70.5	0
6932	6932	2314	200	0
6933	6933	2314	200	0
6934	6934	2314	10	0
6935	6935	2314	20	0
6936	6936	2314	90	0
6937	6937	2314	0	0
6938	6938	2314	1.713333333	0
6939	6939	2314	23.78333333	0
6940	6940	2314	6.666666667	0
6941	6941	2314	250	0
6942	6942	2314	250	0
6943	6943	2314	2.98	0
6944	6944	2314	79	0
6945	6945	2314	29	0
6946	6946	2314	4.68	0
6947	6947	2314	62	0
6948	6948	2314	0	0
6949	6949	2314	80	0
6950	6950	2314	0	0
6951	6951	2314	0	0
6952	6952	2314	29	0
6953	6953	2314	54	0
6954	6954	2314	33	0
6955	6955	2314	250	0
6956	6956	2314	250	0
6957	6957	2314	2.98	0
6958	6958	2314	79	0
6959	6959	2314	29	0
6960	6960	2314	4.68	0
6961	6961	2314	62	0
6962	6962	2314	0	0
6963	6963	2314	80	0
6964	6964	2314	0	0
6965	6965	2314	0	0
6966	6966	2314	116.7	0
6967	6967	2314	52.2	0
6968	6968	2314	170.8	0
6969	6969	2314	97.3	0
6970	6970	2314	11	0
6971	6971	2314	8.6	0
6972	6972	2314	25.87	0
6973	6973	2314	3.94	0
6974	6974	2314	25.9222	0
6975	6975	2314	4.0459	0
6976	6976	2314	4	0
6977	6977	2314	17	0
6978	6978	2314	1	0
6979	6979	2314	2	0
6980	6980	2314	0.5	0
6981	6981	2314	0.6	0
6982	6982	2314	0.7	0
6983	6983	2314	0.7	0
6984	6984	2314	4	0
6985	6985	2314	0	0
6986	6986	2314	0.9	0
6987	6987	2314	0.4	0
6988	6988	2314	0.5	0
6989	6989	2314	0.7	0
6990	6990	2314	0.5	0
6991	6991	2314	0.6	0
6992	6992	2314	0.6	0
6993	6993	2314	0.7	0
6994	6994	2314	2	0
6995	6995	2314	16	0
6996	6996	2314	4	0
6997	6997	2314	1	0
6998	6998	2314	7	0
6999	6999	2314	3	0
7000	7000	2314	3	0
7001	7001	2314	2	0
7002	7002	2314	1	0
7003	7003	2314	0	0
7004	7004	2314	0	0
7005	7005	2314	0	0
7006	7006	2314	0	0
7007	7007	2314	0	0
7008	7008	2314	0	0
7009	7009	2314	0	0
7010	7010	2314	B0L6	0
7011	7011	2314	0.9	0
7012	7012	2314	1.2	0
7013	7013	2314	23.6	0
7014	7014	2314	14	0
7015	7015	2314	33.7	0
7016	7016	2314	22	0
7017	7017	2314	14.8	0
7018	7018	2314	148	0
7019	7019	2314	2.2	0
7020	7020	2314	2.1	0
7021	7021	2314	0	0
7022	7022	2314	4	0
7023	7023	2314	0	0
7024	7024	2314	67	0
7025	7025	2314	30	0
7026	7026	2314	30	0
7027	7027	2314	5	0
7028	7028	2314	4	0
7029	7029	2314	30	0
7030	7030	2314	6	0
7031	7031	2314	2.03	0
7032	7032	2314	26.86666667	0
7033	7033	2314	5.45	0
7034	7034	2314	175	0
7035	7035	2314	200	0
7036	7036	2314	2.8	0
7037	7037	2314	83	0
7038	7038	2314	38	0
7039	7039	2314	2.3	0
7040	7040	2314	1	0
7041	7041	2314	0	0
7042	7042	2314	70	0
7043	7043	2314	5	0
7044	7044	2314	0	0
7045	7045	2314	75	0
7046	7046	2314	42	0
7047	7047	2314	39	0
7048	7048	2314	175	0
7049	7049	2314	200	0
7050	7050	2314	2.8	0
7051	7051	2314	83	0
7052	7052	2314	38	0
7053	7053	2314	2.3	0
7054	7054	2314	1	0
7055	7055	2314	0	0
7056	7056	2314	70	0
7057	7057	2314	5	0
7058	7058	2314	0	0
7059	7059	2314	82.1	0
7060	7060	2314	39.4	0
7061	7061	2314	31.8	0
7062	7062	2314	22.1	0
7063	7063	2314	6.6	0
7064	7064	2314	5.9	0
7065	7065	2314	9.26	0
7066	7066	2314	0.92	0
7067	7067	2314	9.2994	0
7068	7068	2314	0.948	0
7069	7069	2314	7	0
7070	7070	2314	54	0
7071	7071	2314	1	0
7072	7072	2314	2	0
7073	7073	2314	0.3	0
7074	7074	2314	0.3	0
7075	7075	2314		0
7076	7076	2314		0
7077	7077	2314	3	0
7078	7078	2314	0	0
7079	7079	2314	0.8	0
7080	7080	2314	0.4	0
7081	7081	2314	.	0
7082	7082	2314	.	0
7083	7083	2314	0.4	0
7084	7084	2314	0.4	0
7085	7085	2314		0
7086	7086	2314		0
7087	7087	2314	1	0
7088	7088	2314	9	0
7089	7089	2314	2	0
7090	7090	2314	1	0
7091	7091	2314	6	0
7092	7092	2314	2	0
7093	7093	2314	1	0
7094	7094	2314	0	0
7095	7095	2314	0	0
7096	7096	2314	0	0
7097	7097	2314	0	0
7098	7098	2314	0	0
7099	7099	2314	0	0
7100	7100	2314	0	0
7101	7101	2314	0	0
7102	7102	2314	0	0
7103	7103	2314	B0L5	0
7104	7104	2314	0.7	0
7105	7105	2314	1.1	0
7106	7106	2314	24.3	0
7107	7107	2314	14	0
7108	7108	2314	27	0
7109	7109	2314	14	0
7110	7110	2314	4.6	0
7111	7111	2314	77	0
7112	7112	2314	2	0
7113	7113	2314	2.1	0
7114	7114	2314	3	0
7115	7115	2314	4	0
7116	7116	2314	0	0
7117	7117	2314	118.5	0
7118	7118	2314	150	0
7119	7119	2314	150	0
7120	7120	2314	5	0
7121	7121	2314	25	0
7122	7122	2314	60	0
7123	7123	2314	3	0
7124	7124	2314	1.588333333	0
7125	7125	2314	15.73333333	0
7126	7126	2314	4.816666667	0
7127	7127	2314	0	0
7128	7128	2314	250	0
7129	7129	2314		0
7130	7130	2314	68	0
7131	7131	2314	35	0
7132	7132	2314		0
7133	7133	2314	0	0
7134	7134	2314	0	0
7135	7135	2314	0	0
7136	7136	2314	0	0
7137	7137	2314	0	0
7138	7138	2314	128	0
7139	7139	2314	0	0
7140	7140	2314	0	0
7141	7141	2314	0	0
7142	7142	2314	250	0
7143	7143	2314	.	0
7144	7144	2314	68	0
7145	7145	2314	35	0
7146	7146	2314	.	0
7147	7147	2314	0	0
7148	7148	2314	0	0
7149	7149	2314	0	0
7150	7150	2314	0	0
7151	7151	2314	0	0
7152	7152	2314	.	0
7153	7153	2314	.	0
7154	7154	2314	.	0
7155	7155	2314	.	0
7156	7156	2314	.	0
7157	7157	2314	.	0
7158	7158	2314	7.19	0
7159	7159	2314	0.34	0
7160	7160	2314	7.19	0
7161	7161	2314	0.34	0
7162	7162	2314	.	0
7163	7163	2314	.	0
7164	7164	2314	0	0
7165	7165	2314	0	0
7166	7166	2314		0
7167	7167	2314		0
7168	7168	2314		0
7169	7169	2314		0
7170	7170	2314	0	0
7171	7171	2314	0	0
7172	7172	2314	0.4	0
7173	7173	2314	0.2	0
7174	7174	2314		0
7175	7175	2314		0
7176	7176	2314		0
7177	7177	2314		0
7178	7178	2314		0
7179	7179	2314		0
7180	7180	2314	0	0
7181	7181	2314	0	0
7182	7182	2314	0	0
7183	7183	2314	.	0
7184	7184	2314	0	0
7185	7185	2314	.	0
7186	7186	2314	.	0
7187	7187	2314	0	0
7188	7188	2314	0	0
7189	7189	2314	0	0
7190	7190	2314	0	0
7191	7191	2314	0	0
7192	7192	2314	0	0
7193	7193	2314	0	0
7194	7194	2314	0	0
7195	7195	2314	0	0
7196	7196	2314	.	0
7197	7197	2314	.	0
7198	7198	2314	.	0
7199	7199	2314	.	0
7200	7200	2314	0	0
7201	7201	2314	.	0
7202	7202	2314	0	0
7203	7203	2314	.	0
7204	7204	2314	0	0
7205	7205	2314	.	0
7206	7206	2314	.	0
7207	7207	2314	0	0
7208	7208	2314	0	0
7209	7209	2314	0	0
7210	7210	2314		0
7211	7211	2314		0
7212	7212	2314	0	0
7213	7213	2314		0
7214	7214	2314		0
7215	7215	2314		0
7216	7216	2314		0
7217	7217	2314		0
7218	7218	2314		0
7219	7219	2314		0
7220	7220	2314		0
7221	7221	2314		0
7222	7222	2314		0
7223	7223	2314		0
7224	7224	2314		0
7225	7225	2314		0
7226	7226	2314		0
7227	7227	2314		0
7228	7228	2314		0
7229	7229	2314		0
7230	7230	2314		0
7231	7231	2314		0
7232	7232	2314		0
7233	7233	2314		0
7234	7234	2314	.	0
7235	7235	2314	.	0
7236	7236	2314	.	0
7237	7237	2314	.	0
7238	7238	2314	.	0
7239	7239	2314	.	0
7240	7240	2314	.	0
7241	7241	2314	.	0
7242	7242	2314	.	0
7243	7243	2314	.	0
7244	7244	2314	.	0
7245	7245	2314	.	0
7246	7246	2314	.	0
7247	7247	2314	.	0
7248	7248	2314	.	0
7249	7249	2314	.	0
7250	7250	2314	.	0
7251	7251	2314		0
7252	7252	2314		0
7253	7253	2314	.	0
7254	7254	2314	.	0
7255	7255	2314	38	0
7256	7256	2314	12	0
7257	7257	2314	0	0
7258	7258	2314	2	0
7259	7259	2314	0.4	0
7260	7260	2314	0.5	0
7261	7261	2314	0.6	0
7262	7262	2314	0.6	0
7263	7263	2314	3	0
7264	7264	2314	0	0
7265	7265	2314	0.7	0
7266	7266	2314	0.3	0
7267	7267	2314	0.5	0
7268	7268	2314	0.6	0
7269	7269	2314	0.5	0
7270	7270	2314	0.6	0
7271	7271	2314	0.5	0
7272	7272	2314	0.7	0
7273	7273	2314	1	0
7274	7274	2314	16	0
7275	7275	2314	4	0
7276	7276	2314	1	0
7277	7277	2314	6	0
7278	7278	2314	4	0
7279	7279	2314	3	0
7280	7280	2314	2	0
7281	7281	2314	1	0
7282	7282	2314	0	0
7283	7283	2314	0	0
7284	7284	2314	0	0
7285	7285	2314	0	0
7286	7286	2314	0	0
7287	7287	2314	0	0
7288	7288	2314	0	0
7289	7289	2314	B0L5	0
7290	7290	2314	0.7	0
7291	7291	2314	0.8	0
7292	7292	2314	21.3	0
7293	7293	2314	7	0
7294	7294	2314	32.1	0
7295	7295	2314	23	0
7296	7296	2314	9.8	0
7297	7297	2314	114	0
7298	7298	2314	3.1	0
7299	7299	2314	2.3	0
7300	7300	2314	0	0
7301	7301	2314	0	0
7302	7302	2314	0	0
7303	7303	2314	63.5	0
7304	7304	2314	50	0
7305	7305	2314	50	0
7306	7306	2314	5	0
7307	7307	2314	5	0
7308	7308	2314	50	0
7309	7309	2314	3	0
7310	7310	2314	1.818333333	0
7311	7311	2314	24.31666667	0
7312	7312	2314	5.65	0
7313	7313	2314	50	0
7314	7314	2314	150	0
7315	7315	2314	2.55	0
7316	7316	2314	55	0
7317	7317	2314	30	0
7318	7318	2314	4.18	0
7319	7319	2314	0	0
7320	7320	2314	0	0
7321	7321	2314	20	0
7322	7322	2314	0	0
7323	7323	2314	0	0
7324	7324	2314	77	0
7325	7325	2314	29	0
7326	7326	2314	11	0
7327	7327	2314	50	0
7328	7328	2314	150	0
7329	7329	2314	2.55	0
7330	7330	2314	55	0
7331	7331	2314	30	0
7332	7332	2314	4.18	0
7333	7333	2314	0	0
7334	7334	2314	0	0
7335	7335	2314	20	0
7336	7336	2314	0	0
7337	7337	2314	0	0
7338	7338	2314	48.5	0
7339	7339	2314	29.5	0
7340	7340	2314	154.1	0
7341	7341	2314	69.9	0
7342	7342	2314	7.3	0
7343	7343	2314	6.9	0
7344	7344	2314	7.17	0
7345	7345	2314	0.95	0
7346	7346	2314	7.1995	0
7347	7347	2314	1.0268	0
7348	7348	2314	131	0
7349	7349	2314	17	0
7350	7350	2314	1	0
7351	7351	2314	2	0
7352	7352	2314	0.5	0
7353	7353	2314	0.6	0
7354	7354	2314		0
7355	7355	2314		0
7356	7356	2314	2	0
7357	7357	2314	0	0
7358	7358	2314	0.8	0
7359	7359	2314	0.4	0
7360	7360	2314	0.5	0
7361	7361	2314	0.6	0
7362	7362	2314	0.5	0
7363	7363	2314	0.6	0
7364	7364	2314		0
7365	7365	2314		0
7366	7366	2314	1	0
7367	7367	2314	11	0
7368	7368	2314	3	0
7369	7369	2314	1	0
7370	7370	2314	6	0
7371	7371	2314	2	0
7372	7372	2314	2	0
7373	7373	2314	1	0
7374	7374	2314	0	0
7375	7375	2314	0	0
7376	7376	2314	0	0
7377	7377	2314	0	0
7378	7378	2314	0	0
7379	7379	2314	0	0
7380	7380	2314	0	0
7381	7381	2314	0	0
7382	7382	2314	B0L5	0
7383	7383	2314	1	0
7384	7384	2314	0.8	0
7385	7385	2314	25.5	0
7386	7386	2314	14	0
7387	7387	2314	34.2	0
7388	7388	2314	38	0
7389	7389	2314	20.3	0
7390	7390	2314	224	0
7391	7391	2314	2.8	0
7392	7392	2314	2.1	0
7393	7393	2314	0	0
7394	7394	2314	0	0
7395	7395	2314	0	0
7396	7396	2314	93	0
7397	7397	2314	200	0
7398	7398	2314	200	0
7399	7399	2314	25	0
7400	7400	2314	25	0
7401	7401	2314	85	0
7402	7402	2314	1	0
7403	7403	2314	2.656666667	0
7404	7404	2314	32.78333333	0
7405	7405	2314	7.366666667	0
7406	7406	2314	50	0
7407	7407	2314	150	0
7408	7408	2314	2.22	0
7409	7409	2314	107	0
7410	7410	2314	41	0
7411	7411	2314	3.37	0
7412	7412	2314	7	0
7413	7413	2314	0	0
7414	7414	2314	75	0
7415	7415	2314	0	0
7416	7416	2314	0	0
7417	7417	2314	177	0
7418	7418	2314	28	0
7419	7419	2314	11	0
7420	7420	2314	50	0
7421	7421	2314	150	0
7422	7422	2314	2.22	0
7423	7423	2314	107	0
7424	7424	2314	41	0
7425	7425	2314	3.37	0
7426	7426	2314	7	0
7427	7427	2314	0	0
7428	7428	2314	75	0
7429	7429	2314	0	0
7430	7430	2314	0	0
7431	7431	2314	26.5	0
7432	7432	2314	18.3	0
7433	7433	2314	132.9	0
7434	7434	2314	57.8	0
7435	7435	2314	4.6	0
7436	7436	2314	4	0
7437	7437	2314	12.93	0
7438	7438	2314	2.48	0
7439	7439	2314	12.9483	0
7440	7440	2314	2.5418	0
7441	7441	2314		0
7442	7442	2314	14	0
7443	7443	2314	0	0
7444	7444	2314	1	0
7445	7445	2314	0.5	0
7446	7446	2314	0.7	0
7447	7447	2314		0
7448	7448	2314		0
7449	7449	2314	2	0
7450	7450	2314	0	0
7451	7451	2314	0.9	0
7452	7452	2314	0.4	0
7453	7453	2314	0.6	0
7454	7454	2314	0.8	0
7455	7455	2314		0
7456	7456	2314		0
7457	7457	2314		0
7458	7458	2314		0
7459	7459	2314	2	0
7460	7460	2314	10	0
7461	7461	2314	2	0
7462	7462	2314	1	0
7463	7463	2314	6	0
7464	7464	2314	2	0
7465	7465	2314	2	0
7466	7466	2314	.	0
7467	7467	2314	.	0
7468	7468	2314	0	0
7469	7469	2314	0	0
7470	7470	2314	0	0
7471	7471	2314	0	0
7472	7472	2314	0	0
7473	7473	2314	0	0
7474	7474	2314	0	0
7475	7475	2314	B0L4	0
7476	7476	2314	1.1	0
7477	7477	2314	1.5	0
7478	7478	2314	35.5	0
7479	7479	2314	13	0
7480	7480	2314	34.2	0
7481	7481	2314	19	0
7482	7482	2314	7.2	0
7483	7483	2314	115	0
7484	7484	2314	3	0
7485	7485	2314	2.4	0
7486	7486	2314	0	0
7487	7487	2314	0	0
7488	7488	2314	0	0
7489	7489	2314	83.2	0
7490	7490	2314	400	0
7491	7491	2314	400	0
7492	7492	2314	50	0
7493	7493	2314	50	0
7494	7494	2314	100	0
7495	7495	2314	0	0
7496	7496	2314	2.72	0
7497	7497	2314	34.7	0
7498	7498	2314	8.2	0
7499	7499	2314	500	0
7500	7500	2314	600	0
7501	7501	2314	2.49	0
7502	7502	2314	84	0
7503	7503	2314	61	0
7504	7504	2314	4.96	0
7505	7505	2314	0	0
7506	7506	2314	10	0
7507	7507	2314	80	0
7508	7508	2314	0	0
7509	7509	2314	0	0
7510	7510	2314	247	0
7511	7511	2314	45	0
7512	7512	2314	13	0
7513	7513	2314	500	0
7514	7514	2314	600	0
7515	7515	2314	2.49	0
7516	7516	2314	84	0
7517	7517	2314	61	0
7518	7518	2314	4.96	0
7519	7519	2314	0	0
7520	7520	2314	10	0
7521	7521	2314	80	0
7522	7522	2314	0	0
7523	7523	2314	0	0
7524	7524	2314	115.9	0
7525	7525	2314	37.7	0
7526	7526	2314	147.8	0
7527	7527	2314	48.9	0
7528	7528	2314	42.4	0
7529	7529	2314	17.4	0
7530	7530	2314	27.93	0
7531	7531	2314	3.53	0
7532	7532	2314	27.9677	0
7533	7533	2314	3.5963	0
7534	7534	2314	27	0
7535	7535	2314	4	0
7536	7536	2314	2	0
7537	7537	2314	3	0
7538	7538	2314	0.6	0
7539	7539	2314	1	0
7540	7540	2314	1.1	0
7541	7541	2314	1.3	0
7542	7542	2314	3	0
7543	7543	2314	0	0
7544	7544	2314	1	0
7545	7545	2314	0.3	0
7546	7546	2314	0.7	0
7547	7547	2314	0.9	0
7548	7548	2314	0.7	0
7549	7549	2314	1	0
7550	7550	2314	0.7	0
7551	7551	2314	1	0
7552	7552	2314	2	0
7553	7553	2314	23	0
7554	7554	2314	6	0
7555	7555	2314	1	0
7556	7556	2314	8	0
7557	7557	2314	4	0
7558	7558	2314	4	0
7559	7559	2314	3	0
7560	7560	2314	2	0
7561	7561	2314	0	0
7562	7562	2314	0	0
7563	7563	2314	1	0
7564	7564	2314	0	0
7565	7565	2314	1	0
7566	7566	2314	0	0
7567	7567	2314	0	0
7568	7568	2314	B0L7	0
7569	7569	2314	1.2	0
7570	7570	2314	1.4	0
7571	7571	2314	23.7	0
7572	7572	2314	13	0
7573	7573	2314	42.2	0
7574	7574	2314	43	0
7575	7575	2314	24.9	0
7576	7576	2314	253	0
7577	7577	2314	2.6	0
7578	7578	2314	2.4	0
7579	7579	2314	0	0
7580	7580	2314	0	0
7581	7581	2314	0	0
7582	7582	2314	89.5	0
7583	7583	2314	500	0
7584	7584	2314	500	0
7585	7585	2314	25	0
7586	7586	2314	25	0
7587	7587	2314	95	0
7588	7588	2314	0	0
7589	7589	2314	1.878333333	0
7590	7590	2314	24.46666667	0
7591	7591	2314	7.65	0
7592	7592	2314	400	0
7593	7593	2314	400	0
7594	7594	2314	2	0
7595	7595	2314	110	0
7596	7596	2314	24	0
7597	7597	2314	4.67	0
7598	7598	2314	0	0
7599	7599	2314	0	0
7600	7600	2314	90	0
7601	7601	2314	0	0
7602	7602	2314	20	0
7603	7603	2314	30	0
7604	7604	2314	43	0
7605	7605	2314	48	0
7606	7606	2314	400	0
7607	7607	2314	400	0
7608	7608	2314	2	0
7609	7609	2314	110	0
7610	7610	2314	24	0
7611	7611	2314	4.67	0
7612	7612	2314	0	0
7613	7613	2314	0	0
7614	7614	2314	90	0
7615	7615	2314	0	0
7616	7616	2314	20	0
7617	7617	2314	91.8	0
7618	7618	2314	24.7	0
7619	7619	2314	173.4	0
7620	7620	2314	73	0
7621	7621	2314	10.6	0
7622	7622	2314	5.6	0
7623	7623	2314	28.2	0
7624	7624	2314	3.31	0
7625	7625	2314	28.2247	0
7626	7626	2314	3.3886	0
7627	7627	2314	3	0
7628	7628	2314	29	0
7629	7629	2314	2	0
7630	7630	2314	3	0
7631	7631	2314	0.4	0
7632	7632	2314	0.9	0
7633	7633	2314	1.2	0
7634	7634	2314	1.3	0
7635	7635	2314	5	0
7636	7636	2314	0	0
7637	7637	2314	1.2	0
7638	7638	2314	0.5	0
7639	7639	2314	0.8	0
7640	7640	2314	0.5	0
7641	7641	2314	0.8	0
7642	7642	2314	0.9	0
7643	7643	2314	0.7	0
7644	7644	2314	0.9	0
7645	7645	2314	2	0
7646	7646	2314	24	0
7647	7647	2314	5	0
7648	7648	2314	1	0
7649	7649	2314	9	0
7650	7650	2314	5	0
7651	7651	2314	4	0
7652	7652	2314	3	0
7653	7653	2314	2	0
7654	7654	2314	0	0
7655	7655	2314	0	0
7656	7656	2314	1	0
7657	7657	2314	0	0
7658	7658	2314	0	0
7659	7659	2314	0	0
7660	7660	2314	0	0
7661	7661	2314	B0L7	0
7662	7662	2314	1.1	0
7663	7663	2314	1.3	0
7664	7664	2314	12.5	0
7665	7665	2314	9	0
7666	7666	2314	45.8	0
7667	7667	2314	68	0
7668	7668	2314	22.8	0
7669	7669	2314	372	0
7670	7670	2314	2.8	0
7671	7671	2314	2.7	0
7672	7672	2314	3	0
7673	7673	2314	3	0
7674	7674	2314	0	0
7675	7675	2314	75	0
7676	7676	2314	225	0
7677	7677	2314	225	0
7678	7678	2314	15	0
7679	7679	2314	40	0
7680	7680	2314	85	0
7681	7681	2314	0	0
7682	7682	2314	1.765	0
7683	7683	2314	24.6	0
7684	7684	2314	5.766666667	0
7685	7685	2314	300	0
7686	7686	2314	300	0
7687	7687	2314	2.99	0
7688	7688	2314	94	0
7689	7689	2314	38.5	0
7690	7690	2314	4.64	0
7691	7691	2314	0	0
7692	7692	2314	0	0
7693	7693	2314	90	0
7694	7694	2314	0	0
7695	7695	2314	0	0
7696	7696	2314	231	0
7697	7697	2314	97	0
7698	7698	2314	45	0
7699	7699	2314	300	0
7700	7700	2314	300	0
7701	7701	2314	2.99	0
7702	7702	2314	94	0
7703	7703	2314	38.5	0
7704	7704	2314	4.64	0
7705	7705	2314	0	0
7706	7706	2314	0	0
7707	7707	2314	90	0
7708	7708	2314	0	0
7709	7709	2314	0	0
7710	7710	2314	135.7	0
7711	7711	2314	48.3	0
7712	7712	2314	252.3	0
7713	7713	2314	89.4	0
7714	7714	2314	4.6	0
7715	7715	2314	3.3	0
7716	7716	2314	15.54	0
7717	7717	2314	2.83	0
7718	7718	2314	15.5883	0
7719	7719	2314	2.9227	0
7720	7720	2314	20	0
7721	7721	2314	24	0
7722	7722	2314	0	0
7723	7723	2314	1	0
7724	7724	2314		0
7725	7725	2314		0
7726	7726	2314		0
7727	7727	2314		0
7728	7728	2314	2	0
7729	7729	2314	0	0
7730	7730	2314	0.4	0
7731	7731	2314	0.8	0
7732	7732	2314	0.5	0
7733	7733	2314	0.8	0
7734	7734	2314		0
7735	7735	2314		0
7736	7736	2314		0
7737	7737	2314		0
7738	7738	2314	2	0
7739	7739	2314	5	0
7740	7740	2314	0	0
7741	7741	2314	.	0
7742	7742	2314	5	0
7743	7743	2314	.	0
7744	7744	2314	.	0
7745	7745	2314	.	0
7746	7746	2314	.	0
7747	7747	2314	0	0
7748	7748	2314	0	0
7749	7749	2314	0	0
7750	7750	2314	0	0
7751	7751	2314	0	0
7752	7752	2314	0	0
7753	7753	2314	0	0
7754	7754	2314	B0L4	0
7755	7755	2314	0.9	0
7756	7756	2314	1.2	0
7757	7757	2314	34.5	0
7758	7758	2314	13	0
7759	7759	2314	36.1	0
7760	7760	2314	23	0
7761	7761	2314	7.8	0
7762	7762	2314	94	0
7763	7763	2314	3	0
7764	7764	2314	2.3	0
7765	7765	2314	0	0
7766	7766	2314	0	0
7767	7767	2314	0	0
7768	7768	2314	89	0
7769	7769	2314	150	0
7770	7770	2314	150	0
7771	7771	2314	10	0
7772	7772	2314	25	0
7773	7773	2314	60	0
7774	7774	2314	1	0
7775	7775	2314	1.391666667	0
7776	7776	2314	15.93333333	0
7777	7777	2314	4.25	0
7778	7778	2314	150	0
7779	7779	2314	350	0
7780	7780	2314	2.47	0
7781	7781	2314	89.5	0
7782	7782	2314	53	0
7783	7783	2314	2.75	0
7784	7784	2314	0	0
7785	7785	2314	0	0
7786	7786	2314	50	0
7787	7787	2314	10	0
7788	7788	2314	0	0
7789	7789	2314	133	0
7790	7790	2314	26	0
7791	7791	2314	13	0
7792	7792	2314	150	0
7793	7793	2314	350	0
7794	7794	2314	2.47	0
7795	7795	2314	89.5	0
7796	7796	2314	53	0
7797	7797	2314	2.75	0
7798	7798	2314	0	0
7799	7799	2314	0	0
7800	7800	2314	50	0
7801	7801	2314	10	0
7802	7802	2314	0	0
7803	7803	2314	87.8	0
7804	7804	2314	29.7	0
7805	7805	2314	61.8	0
7806	7806	2314	23.4	0
7807	7807	2314	10	0
7808	7808	2314	7.1	0
7809	7809	2314	9	0
7810	7810	2314	1.89	0
7811	7811	2314	9.0297	0
7812	7812	2314	1.9205	0
7813	7813	2314	27	0
7814	7814	2314	2	0
7815	7815	2314	1	0
7816	7816	2314	2	0
7817	7817	2314	0.5	0
7818	7818	2314	0.7	0
7819	7819	2314		0
7820	7820	2314		0
7821	7821	2314	2	0
7822	7822	2314	0	0
7823	7823	2314	0.8	0
7824	7824	2314	0.3	0
7825	7825	2314	0.5	0
7826	7826	2314	0.8	0
7827	7827	2314	0.4	0
7828	7828	2314	0.6	0
7829	7829	2314		0
7830	7830	2314		0
7831	7831	2314	1	0
7832	7832	2314	9	0
7833	7833	2314	1	0
7834	7834	2314	1	0
7835	7835	2314	7	0
7836	7836	2314	2	0
7837	7837	2314	.	0
7838	7838	2314	.	0
7839	7839	2314	.	0
7840	7840	2314	0	0
7841	7841	2314	0	0
7842	7842	2314	0	0
7843	7843	2314	0	0
7844	7844	2314	0	0
7845	7845	2314	0	0
7846	7846	2314	0	0
7847	7847	2314	B0L6	0
7848	7848	2314	1	0
7849	7849	2314	1.2	0
7850	7850	2314	21.2	0
7851	7851	2314	8	0
7852	7852	2314	29.3	0
7853	7853	2314	18	0
7854	7854	2314	1.5	0
7855	7855	2314	68	0
7856	7856	2314	2.6	0
7857	7857	2314	2.5	0
7858	7858	2314	0	0
7859	7859	2314	0	0
7860	7860	2314	0	0
7861	7861	2314	60.5	0
7862	7862	2314	200	0
7863	7863	2314	200	0
7864	7864	2314	20	0
7865	7865	2314	20	0
7866	7866	2314	100	0
7867	7867	2314	0	0
7868	7868	2314	2.02	0
7869	7869	2314	25.83333333	0
7870	7870	2314	6.966666667	0
7871	7871	2314	350	0
7872	7872	2314	350	0
7873	7873	2314	2.34	0
7874	7874	2314	67.5	0
7875	7875	2314	40	0
7876	7876	2314	3.33	0
7877	7877	2314	0	0
7878	7878	2314	8	0
7879	7879	2314	95	0
7880	7880	2314	0	0
7881	7881	2314	10	0
7882	7882	2314	72	0
7883	7883	2314	152	0
7884	7884	2314	34	0
7885	7885	2314	350	0
7886	7886	2314	350	0
7887	7887	2314	2.34	0
7888	7888	2314	67.5	0
7889	7889	2314	40	0
7890	7890	2314	3.33	0
7891	7891	2314	0	0
7892	7892	2314	8	0
7893	7893	2314	95	0
7894	7894	2314	0	0
7895	7895	2314	10	0
7896	7896	2314	80.9	0
7897	7897	2314	31.3	0
7898	7898	2314	78.2	0
7899	7899	2314	29.6	0
7900	7900	2314	6.8	0
7901	7901	2314	2.8	0
7902	7902	2314	13.07	0
7903	7903	2314	3.25	0
7904	7904	2314	13.1013	0
7905	7905	2314	3.2824	0
7906	7906	2314	83	0
7907	7907	2314	11	0
7908	7908	2314	1	0
7909	7909	2314	2	0
7910	7910	2314	0.6	0
7911	7911	2314	0.8	0
7912	7912	2314	0.9	0
7913	7913	2314	1	0
7914	7914	2314	3	0
7915	7915	2314	0	0
7916	7916	2314	0.9	0
7917	7917	2314	0.4	0
7918	7918	2314	0.6	0
7919	7919	2314	0.8	0
7920	7920	2314	0.6	0
7921	7921	2314	0.7	0
7922	7922	2314		0
7923	7923	2314		0
7924	7924	2314	1	0
7925	7925	2314	16	0
7926	7926	2314	4	0
7927	7927	2314	1	0
7928	7928	2314	7	0
7929	7929	2314	3	0
7930	7930	2314	3	0
7931	7931	2314	2	0
7932	7932	2314	1	0
7933	7933	2314	0	0
7934	7934	2314	0	0
7935	7935	2314	0	0
7936	7936	2314	0	0
7937	7937	2314	0	0
7938	7938	2314	0	0
7939	7939	2314	0	0
7940	7940	2314	B0L6	0
7941	7941	2314	1.1	0
7942	7942	2314	1.4	0
7943	7943	2314	33.8	0
7944	7944	2314	16	0
7945	7945	2314	45.9	0
7946	7946	2314	45	0
7947	7947	2314	27.4	0
7948	7948	2314	364	0
7949	7949	2314	3.3	0
7950	7950	2314	2.8	0
7951	7951	2314	0	0
7952	7952	2314	0	0
7953	7953	2314	0	0
7954	7954	2314	84	0
7955	7955	2314	250	0
7956	7956	2314	250	0
7957	7957	2314	5	0
7958	7958	2314	40	0
7959	7959	2314	80	0
7960	7960	2314	1	0
7961	7961	2314	1.156666667	0
7962	7962	2314	13.98333333	0
7963	7963	2314	4.1	0
7964	7964	2314	150	0
7965	7965	2314	200	0
7966	7966	2314	1.8	0
7967	7967	2314	96	0
7968	7968	2314	38	0
7969	7969	2314	2.88	0
7970	7970	2314	0	0
7971	7971	2314	0	0
7972	7972	2314	60	0
7973	7973	2314	0	0
7974	7974	2314	0	0
7975	7975	2314	140	0
7976	7976	2314	57	0
7977	7977	2314	22	0
7978	7978	2314	150	0
7979	7979	2314	200	0
7980	7980	2314	1.8	0
7981	7981	2314	96	0
7982	7982	2314	38	0
7983	7983	2314	2.88	0
7984	7984	2314	0	0
7985	7985	2314	0	0
7986	7986	2314	60	0
7987	7987	2314	0	0
7988	7988	2314	0	0
7989	7989	2314	58.5	0
7990	7990	2314	23.8	0
7991	7991	2314	161.1	0
7992	7992	2314	69.4	0
7993	7993	2314	10.9	0
7994	7994	2314	7.1	0
7995	7995	2314	13.51	0
7996	7996	2314	1.54	0
7997	7997	2314	13.5338	0
7998	7998	2314	1.6165	0
7999	7999	2314	24	0
8000	8000	2314	15	0
8001	8001	2314	2	0
8002	8002	2314	3	0
8003	8003	2314	0.8	0
8004	8004	2314	1.2	0
8005	8005	2314	1.4	0
8006	8006	2314	1.5	0
8007	8007	2314	4	0
8008	8008	2314	0	0
8009	8009	2314	1	0
8010	8010	2314	0.5	0
8011	8011	2314	0.9	0
8012	8012	2314	1.3	0
8013	8013	2314	0.8	0
8014	8014	2314	1	0
8015	8015	2314	0.9	0
8016	8016	2314	1.1	0
8017	8017	2314	2	0
8018	8018	2314	24	0
8019	8019	2314	6	0
8020	8020	2314	1	0
8021	8021	2314	9	0
8022	8022	2314	5	0
8023	8023	2314	4	0
8024	8024	2314	3	0
8025	8025	2314	1	0
8026	8026	2314	0	0
8027	8027	2314	0	0
8028	8028	2314	1	0
8029	8029	2314	0	0
8030	8030	2314	1	0
8031	8031	2314	0	0
8032	8032	2314	0	0
8033	8033	2314	B0L7	0
8034	8034	2314	1.2	0
8035	8035	2314	1.3	0
8036	8036	2314	20.5	0
8037	8037	2314	14	0
8038	8038	2314	52.3	0
8039	8039	2314	61	0
8040	8040	2314	31.5	0
8041	8041	2314	364	0
8042	8042	2314	3.3	0
8043	8043	2314	2.6	0
8044	8044	2314	10	0
8045	8045	2314	14	0
8046	8046	2314	0	0
8047	8047	2314	72.5	0
8048	8048	2314	300	0
8049	8049	2314	300	0
8050	8050	2314	40	0
8051	8051	2314	40	0
8052	8052	2314	90	0
8053	8053	2314	0	0
8054	8054	2314	2.496666667	0
8055	8055	2314	29.7	0
8056	8056	2314	8.35	0
8057	8057	2314	200	0
8058	8058	2314	200	0
8059	8059	2314	2.68	0
8060	8060	2314	76	0
8061	8061	2314	68	0
8062	8062	2314	2.24	0
8063	8063	2314	0	0
8064	8064	2314	0	0
8065	8065	2314	80	0
8066	8066	2314	0	0
8067	8067	2314	0	0
8068	8068	2314	216	0
8069	8069	2314	64	0
8070	8070	2314	2	0
8071	8071	2314	200	0
8072	8072	2314	200	0
8073	8073	2314	2.68	0
8074	8074	2314	76	0
8075	8075	2314	68	0
8076	8076	2314	2.24	0
8077	8077	2314	0	0
8078	8078	2314	0	0
8079	8079	2314	80	0
8080	8080	2314	0	0
8081	8081	2314	0	0
8082	8082	2314	115.2	0
8083	8083	2314	39	0
8084	8084	2314	136.5	0
8085	8085	2314	43.7	0
8086	8086	2314	30.8	0
8087	8087	2314	11.3	0
8088	8088	2314	12.33	0
8089	8089	2314	2.24	0
8090	8090	2314	12.369	0
8091	8091	2314	2.295	0
8092	8092	2314	47	0
8093	8093	2314	90	0
8094	8094	2314	2	0
8095	8095	2314	3	0
8096	8096	2314	0.8	0
8097	8097	2314	0.8	0
8098	8098	2314	1.6	0
8099	8099	2314	1.6	0
8100	8100	2314	4	0
8101	8101	2314	0	0
8102	8102	2314	1.4	0
8103	8103	2314	0.6	0
8104	8104	2314	0.7	0
8105	8105	2314	0.8	0
8106	8106	2314	0.8	0
8107	8107	2314	1.1	0
8108	8108	2314	0.9	0
8109	8109	2314	1.1	0
8110	8110	2314	1	0
8111	8111	2314	22	0
8112	8112	2314	5	0
8113	8113	2314	1	0
8114	8114	2314	8	0
8115	8115	2314	4	0
8116	8116	2314	4	0
8117	8117	2314	3	0
8118	8118	2314	2	0
8119	8119	2314	1	0
8120	8120	2314	0	0
8121	8121	2314	0	0
8122	8122	2314	0	0
8123	8123	2314	0	0
8124	8124	2314	0	0
8125	8125	2314	0	0
8126	8126	2314	B0L7	0
8127	8127	2314	1.4	0
8128	8128	2314	1.6	0
8129	8129	2314	35.3	0
8130	8130	2314	18	0
8131	8131	2314	65.5	0
8132	8132	2314	80	0
8133	8133	2314	38.8	0
8134	8134	2314	423	0
8135	8135	2314	3.1	0
8136	8136	2314	2.5	0
8137	8137	2314	61	0
8138	8138	2314	38	0
8139	8139	2314	9	0
8140	8140	2314	98	0
8141	8141	2314	300	0
8142	8142	2314	300	0
8143	8143	2314	10	0
8144	8144	2314	40	0
8145	8145	2314	70	0
8146	8146	2314	4	0
8147	8147	2314	1.658333333	0
8148	8148	2314	20.41666667	0
8149	8149	2314	5.7	0
8150	8150	2314	300	0
8151	8151	2314	350	0
8152	8152	2314	2.21	0
8153	8153	2314	88	0
8154	8154	2314	40	0
8155	8155	2314	4.84	0
8156	8156	2314	0	0
8157	8157	2314	0	0
8158	8158	2314	85	0
8159	8159	2314	0	0
8160	8160	2314	0	0
8161	8161	2314	404	0
8162	8162	2314	63	0
8163	8163	2314	26	0
8164	8164	2314	300	0
8165	8165	2314	350	0
8166	8166	2314	2.21	0
8167	8167	2314	88	0
8168	8168	2314	40	0
8169	8169	2314	4.84	0
8170	8170	2314	0	0
8171	8171	2314	0	0
8172	8172	2314	85	0
8173	8173	2314	0	0
8174	8174	2314	0	0
8175	8175	2314	80.6	0
8176	8176	2314	28.4	0
8177	8177	2314	330.2	0
8178	8178	2314	103.3	0
8179	8179	2314	2.6	0
8180	8180	2314	2.2	0
8181	8181	2314	19.72	0
8182	8182	2314	3.53	0
8183	8183	2314	19.7484	0
8184	8184	2314	3.6355	0
8185	8185	2314	6	0
8186	8186	2314	3	0
8187	8187	2314	2	0
8188	8188	2314	3	0
8189	8189	2314	0.6	0
8190	8190	2314	1	0
8191	8191	2314	1.1	0
8192	8192	2314	1.1	0
8193	8193	2314	3	0
8194	8194	2314	0	0
8195	8195	2314	1	0
8196	8196	2314	0.3	0
8197	8197	2314	0.6	0
8198	8198	2314	1	0
8199	8199	2314	0.6	0
8200	8200	2314	1	0
8201	8201	2314	0.6	0
8202	8202	2314	0.8	0
8203	8203	2314	1	0
8204	8204	2314	15	0
8205	8205	2314	4	0
8206	8206	2314	1	0
8207	8207	2314	7	0
8208	8208	2314	1	0
8209	8209	2314	4	0
8210	8210	2314	2	0
8211	8211	2314	1	0
8212	8212	2314	0	0
8213	8213	2314	0	0
8214	8214	2314	0	0
8215	8215	2314	0	0
8216	8216	2314	0	0
8217	8217	2314	0	0
8218	8218	2314	0	0
8219	8219	2314	B0L6	0
8220	8220	2314	1.2	0
8221	8221	2314	1.3	0
8222	8222	2314	30.2	0
8223	8223	2314	16	0
8224	8224	2314	43.5	0
8225	8225	2314	43	0
8226	8226	2314	28.5	0
8227	8227	2314	235	0
8228	8228	2314	3	0
8229	8229	2314	2.4	0
8230	8230	2314	10	0
8231	8231	2314	6	0
8232	8232	2314	0	0
8233	8233	2314	96	0
8234	8234	2314	225	0
8235	8235	2314	225	0
8236	8236	2314	5	0
8237	8237	2314	50	0
8238	8238	2314	90	0
8239	8239	2314	0	0
8240	8240	2314	2.4	0
8241	8241	2314	26.425	0
8242	8242	2314	5.175	0
8243	8243	2314	125	0
8244	8244	2314	200	0
8245	8245	2314	1.92	0
8246	8246	2314	102	0
8247	8247	2314	59	0
8248	8248	2314	3.13	0
8249	8249	2314	0	0
8250	8250	2314	0	0
8251	8251	2314	70	0
8252	8252	2314	0	0
8253	8253	2314	0	0
8254	8254	2314	186	0
8255	8255	2314	54	0
8256	8256	2314	0	0
8257	8257	2314	125	0
8258	8258	2314	200	0
8259	8259	2314	1.92	0
8260	8260	2314	102	0
8261	8261	2314	59	0
8262	8262	2314	3.13	0
8263	8263	2314	0	0
8264	8264	2314	0	0
8265	8265	2314	70	0
8266	8266	2314	0	0
8267	8267	2314	0	0
8268	8268	2314	65.4	0
8269	8269	2314	21.1	0
8270	8270	2314	101.7	0
8271	8271	2314	43.1	0
8272	8272	2314	8	0
8273	8273	2314	4	0
8274	8274	2314	10.52	0
8275	8275	2314	1.43	0
8276	8276	2314	10.5411	0
8277	8277	2314	1.4771	0
8278	8278	2314	37	0
8279	8279	2314	44	0
8280	8280	2314	2	0
8281	8281	2314	3	0
8282	8282	2314	0.7	0
8283	8283	2314	0.5	0
8284	8284	2314	1.3	0
8285	8285	2314	1.2	0
8286	8286	2314	4	0
8287	8287	2314	0	0
8288	8288	2314	1.3	0
8289	8289	2314	0.6	0
8290	8290	2314	0.5	0
8291	8291	2314	0.8	0
8292	8292	2314	0.8	0
8293	8293	2314	0.8	0
8294	8294	2314	0.7	0
8295	8295	2314	0.9	0
8296	8296	2314	2	0
8297	8297	2314	24	0
8298	8298	2314	4	0
8299	8299	2314	1	0
8300	8300	2314	10	0
8301	8301	2314	4	0
8302	8302	2314	4	0
8303	8303	2314	4	0
8304	8304	2314	2	0
8305	8305	2314	0	0
8306	8306	2314	0	0
8307	8307	2314	0	0
8308	8308	2314	0	0
8309	8309	2314	0	0
8310	8310	2314	0	0
8311	8311	2314	0	0
8312	8312	2314	B0L9	0
8313	8313	2314	1.2	0
8314	8314	2314	1.4	0
8315	8315	2314	27	0
8316	8316	2314	13	0
8317	8317	2314	39.4	0
8318	8318	2314	46	0
8319	8319	2314	18.7	0
8320	8320	2314	242	0
8321	8321	2314	2.7	0
8322	8322	2314	2	0
8323	8323	2314	18	0
8324	8324	2314	16	0
8325	8325	2314	3	0
8326	8326	2314	57	0
8327	8327	2314	20	0
8328	8328	2314	100	0
8329	8329	2314	1	0
8330	8330	2314	5	0
8331	8331	2314	20	0
8332	8332	2314	7	0
8333	8333	2314	1.293333333	0
8334	8334	2314	17.03333333	0
8335	8335	2314	4.666666667	0
8336	8336	2314	0	0
8337	8337	2314	300	0
8338	8338	2314	1.63	0
8339	8339	2314	51	0
8340	8340	2314	26	0
8341	8341	2314	2.32	0
8342	8342	2314	0	0
8343	8343	2314	0	0
8344	8344	2314	0	0
8345	8345	2314	0	0
8346	8346	2314	0	0
8347	8347	2314	76	0
8348	8348	2314	0	0
8349	8349	2314	0	0
8350	8350	2314	0	0
8351	8351	2314	300	0
8352	8352	2314	1.63	0
8353	8353	2314	51	0
8354	8354	2314	26	0
8355	8355	2314	2.32	0
8356	8356	2314	0	0
8357	8357	2314	0	0
8358	8358	2314	0	0
8359	8359	2314	0	0
8360	8360	2314	0	0
8361	8361	2314	16.2	0
8362	8362	2314	15.4	0
8363	8363	2314	40.9	0
8364	8364	2314	18	0
8365	8365	2314	4.9	0
8366	8366	2314	4.3	0
8367	8367	2314	4.18	0
8368	8368	2314	0.3	0
8369	8369	2314	4.1954	0
8370	8370	2314	0.3223	0
8371	8371	2314	30	0
8372	8372	2314	11	0
8373	8373	2314	2	0
8374	8374	2314	2	0
8375	8375	2314	0.3	0
8376	8376	2314	0.4	0
8377	8377	2314		0
8378	8378	2314		0
8379	8379	2314	3	0
8380	8380	2314	0	0
8381	8381	2314	0.7	0
8382	8382	2314	0.3	0
8383	8383	2314	0.3	0
8384	8384	2314	0.6	0
8385	8385	2314	0.4	0
8386	8386	2314	0.5	0
8387	8387	2314		0
8388	8388	2314		0
8389	8389	2314	1	0
8390	8390	2314	9	0
8391	8391	2314	2	0
8392	8392	2314	2	0
8393	8393	2314	6	0
8394	8394	2314	2	0
8395	8395	2314	1	0
8396	8396	2314	.	0
8397	8397	2314	.	0
8398	8398	2314	0	0
8399	8399	2314	0	0
8400	8400	2314	0	0
8401	8401	2314	0	0
8402	8402	2314	0	0
8403	8403	2314	0	0
8404	8404	2314	0	0
8405	8405	2314	B0L5	0
8406	8406	2314	0.9	0
8407	8407	2314	1.05	0
8408	8408	2314	27	0
8409	8409	2314	12	0
8410	8410	2314	39.8	0
8411	8411	2314	39	0
8412	8412	2314	8	0
8413	8413	2314	145	0
8414	8414	2314	2.9	0
8415	8415	2314	2.4	0
8416	8416	2314	0	0
8417	8417	2314	0	0
8418	8418	2314	0	0
8419	8419	2314	105	0
8420	8420	2314	225	0
8421	8421	2314	225	0
8422	8422	2314	10	0
8423	8423	2314	60	0
8424	8424	2314	90	0
8425	8425	2314	0	0
8426	8426	2314	2.26	0
8427	8427	2314	29.06666667	0
8428	8428	2314	6.833333333	0
8429	8429	2314	250	0
8430	8430	2314	250	0
8431	8431	2314	1.95	0
8432	8432	2314	104	0
8433	8433	2314	41	0
8434	8434	2314	4.34	0
8435	8435	2314	10	0
8436	8436	2314	0	0
8437	8437	2314	85	0
8438	8438	2314	0	0
8439	8439	2314	0	0
8440	8440	2314	144	0
8441	8441	2314	73	0
8442	8442	2314	25	0
8443	8443	2314	250	0
8444	8444	2314	250	0
8445	8445	2314	1.95	0
8446	8446	2314	104	0
8447	8447	2314	41	0
8448	8448	2314	4.34	0
8449	8449	2314	10	0
8450	8450	2314	0	0
8451	8451	2314	85	0
8452	8452	2314	0	0
8453	8453	2314	0	0
8454	8454	2314	50.9	0
8455	8455	2314	18.5	0
8456	8456	2314	176.2	0
8457	8457	2314	60	0
8458	8458	2314	4.7	0
8459	8459	2314	3.2	0
8460	8460	2314	13.18	0
8461	8461	2314	2.08	0
8462	8462	2314	13.1985	0
8463	8463	2314	2.1432	0
8464	8464	2314	3	0
8465	8465	2314	24	0
8466	8466	2314	0	0
8467	8467	2314	3	0
8468	8468	2314	0.8	0
8469	8469	2314	0.9	0
8470	8470	2314		0
8471	8471	2314		0
8472	8472	2314	2	0
8473	8473	2314	0	0
8474	8474	2314	1.2	0
8475	8475	2314	0.5	0
8476	8476	2314	1	0
8477	8477	2314	1.2	0
8478	8478	2314	0.5	0
8479	8479	2314	0.7	0
8480	8480	2314		0
8481	8481	2314		0
8482	8482	2314	2	0
8483	8483	2314	12	0
8484	8484	2314	3	0
8485	8485	2314	1	0
8486	8486	2314	6	0
8487	8487	2314	3	0
8488	8488	2314	2	0
8489	8489	2314	1	0
8490	8490	2314	.	0
8491	8491	2314	0	0
8492	8492	2314	0	0
8493	8493	2314	0	0
8494	8494	2314	0	0
8495	8495	2314	0	0
8496	8496	2314	0	0
8497	8497	2314	0	0
8498	8498	2314	B0L5	0
8499	8499	2314	1.5	0
8500	8500	2314	1.5	0
8501	8501	2314	37.9	0
8502	8502	2314	16	0
8503	8503	2314	43.2	0
8504	8504	2314	45	0
8505	8505	2314	8.1	0
8506	8506	2314	316	0
8507	8507	2314	2.7	0
8508	8508	2314	2	0
8509	8509	2314	0	0
8510	8510	2314	3	0
8511	8511	2314	0	0
8512	8512	2314	71.5	0
8513	8513	2314	200	0
8514	8514	2314	200	0
8515	8515	2314	10	0
8516	8516	2314	10	0
8517	8517	2314	80	0
8518	8518	2314	0	0
8519	8519	2314	1.673333333	0
8520	8520	2314	20.3	0
8521	8521	2314	6.333333333	0
8522	8522	2314	250	0
8523	8523	2314	250	0
8524	8524	2314	3.3	0
8525	8525	2314	82	0
8526	8526	2314	56	0
8527	8527	2314	2.47	0
8528	8528	2314	0	0
8529	8529	2314	0	0
8530	8530	2314	50	0
8531	8531	2314	0	0
8532	8532	2314	75	0
8533	8533	2314	59	0
8534	8534	2314	36	0
8535	8535	2314	27	0
8536	8536	2314	250	0
8537	8537	2314	250	0
8538	8538	2314	3.3	0
8539	8539	2314	82	0
8540	8540	2314	56	0
8541	8541	2314	2.47	0
8542	8542	2314	0	0
8543	8543	2314	0	0
8544	8544	2314	50	0
8545	8545	2314	0	0
8546	8546	2314	75	0
8547	8547	2314	130.8	0
8548	8548	2314	49.5	0
8549	8549	2314	204.5	0
8550	8550	2314	72.7	0
8551	8551	2314	14.9	0
8552	8552	2314	7.1	0
8553	8553	2314	21.67	0
8554	8554	2314	3.64	0
8555	8555	2314	21.7195	0
8556	8556	2314	3.7198	0
8557	8557	2314	4	0
8558	8558	2314	17	0
8559	8559	2314	2	0
8560	8560	2314	2	0
8561	8561	2314	0.3	0
8562	8562	2314	0.3	0
8563	8563	2314	0.6	0
8564	8564	2314	0.8	0
8565	8565	2314	3	0
8566	8566	2314	0	0
8567	8567	2314	0.8	0
8568	8568	2314	0.3	0
8569	8569	2314	0.4	0
8570	8570	2314	0.4	0
8571	8571	2314	0.5	0
8572	8572	2314	0.6	0
8573	8573	2314	0.5	0
8574	8574	2314	0.6	0
8575	8575	2314	1	0
8576	8576	2314	9	0
8577	8577	2314	2	0
8578	8578	2314	2	0
8579	8579	2314	6	0
8580	8580	2314	2	0
8581	8581	2314	1	0
8582	8582	2314	.	0
8583	8583	2314	.	0
8584	8584	2314	0	0
8585	8585	2314	0	0
8586	8586	2314	0	0
8587	8587	2314	0	0
8588	8588	2314	0	0
8589	8589	2314	0	0
8590	8590	2314	0	0
8591	8591	2314	B0L5	0
8592	8592	2314	1.1	0
8593	8593	2314	1.2	0
8594	8594	2314	29.5	0
8595	8595	2314	12	0
8596	8596	2314	29.8	0
8597	8597	2314	30	0
8598	8598	2314	11.7	0
8599	8599	2314	154	0
8600	8600	2314	3	0
8601	8601	2314	2.3	0
8602	8602	2314	0	0
8603	8603	2314	0	0
8604	8604	2314	0	0
8605	8605	2314	55	0
8606	8606	2314	150	0
8607	8607	2314	150	0
8608	8608	2314	10	0
8609	8609	2314	10	0
8610	8610	2314	75	0
8611	8611	2314	0	0
8612	8612	2314	2.16	0
8613	8613	2314	25.81666667	0
8614	8614	2314	5.483333333	0
8615	8615	2314	100	0
8616	8616	2314	300	0
8617	8617	2314	2.39	0
8618	8618	2314	65.5	0
8619	8619	2314	36	0
8620	8620	2314	3.28	0
8621	8621	2314	0	0
8622	8622	2314	0	0
8623	8623	2314	75	0
8624	8624	2314	0	0
8625	8625	2314	0	0
8626	8626	2314	46	0
8627	8627	2314	48	0
8628	8628	2314	23	0
8629	8629	2314	100	0
8630	8630	2314	300	0
8631	8631	2314	2.39	0
8632	8632	2314	65.5	0
8633	8633	2314	36	0
8634	8634	2314	3.28	0
8635	8635	2314	0	0
8636	8636	2314	0	0
8637	8637	2314	75	0
8638	8638	2314	0	0
8639	8639	2314	0	0
8640	8640	2314	90.2	0
8641	8641	2314	22.8	0
8642	8642	2314	69.5	0
8643	8643	2314	26.7	0
8644	8644	2314	6.3	0
8645	8645	2314	3	0
8646	8646	2314	7.38	0
8647	8647	2314	1.14	0
8648	8648	2314	7.4028	0
8649	8649	2314	1.1697	0
8650	8650	2314	63	0
8651	8651	2314	3	0
8652	8652	2314	1	0
8653	8653	2314	2	0
8654	8654	2314	0.6	0
8655	8655	2314	0.8	0
8656	8656	2314	0.7	0
8657	8657	2314	1	0
8658	8658	2314	3	0
8659	8659	2314	0	0
8660	8660	2314	0.8	0
8661	8661	2314	0.4	0
8662	8662	2314	0.6	0
8663	8663	2314	0.8	0
8664	8664	2314	0.7	0
8665	8665	2314	0.8	0
8666	8666	2314		0
8667	8667	2314		0
8668	8668	2314	2	0
8669	8669	2314	15	0
8670	8670	2314	4	0
8671	8671	2314	1	0
8672	8672	2314	7	0
8673	8673	2314	2	0
8674	8674	2314	3	0
8675	8675	2314	2	0
8676	8676	2314	1	0
8677	8677	2314	0	0
8678	8678	2314	0	0
8679	8679	2314	0	0
8680	8680	2314	0	0
8681	8681	2314	0	0
8682	8682	2314	0	0
8683	8683	2314	0	0
8684	8684	2314	B0L6	0
8685	8685	2314	1.1	0
8686	8686	2314	1.2	0
8687	8687	2314	39.2	0
8688	8688	2314	17	0
8689	8689	2314	31.4	0
8690	8690	2314	32	0
8691	8691	2314	15.3	0
8692	8692	2314	187	0
8693	8693	2314	2.4	0
8694	8694	2314	2	0
8695	8695	2314	0	0
8696	8696	2314	0	0
8697	8697	2314	0	0
8698	8698	2314	88.5	0
8699	8699	2314	250	0
8700	8700	2314	250	0
8701	8701	2314	25	0
8702	8702	2314	25	0
8703	8703	2314	90	0
8704	8704	2314	1	0
8705	8705	2314	2.021666667	0
8706	8706	2314	21.95	0
8707	8707	2314	5.683333333	0
8708	8708	2314	100	0
8709	8709	2314	100	0
8710	8710	2314	2.66	0
8711	8711	2314	113	0
8712	8712	2314	36	0
8713	8713	2314	2.94	0
8714	8714	2314	0	0
8715	8715	2314	0	0
8716	8716	2314	50	0
8717	8717	2314	0	0
8718	8718	2314	75	0
8719	8719	2314	59	0
8720	8720	2314	16	0
8721	8721	2314	13	0
8722	8722	2314	100	0
8723	8723	2314	100	0
8724	8724	2314	2.66	0
8725	8725	2314	113	0
8726	8726	2314	36	0
8727	8727	2314	2.94	0
8728	8728	2314	0	0
8729	8729	2314	0	0
8730	8730	2314	50	0
8731	8731	2314	0	0
8732	8732	2314	75	0
8733	8733	2314	110.9	0
8734	8734	2314	37.7	0
8735	8735	2314	98.2	0
8736	8736	2314	35.6	0
8737	8737	2314	8.4	0
8738	8738	2314	4.9	0
8739	8739	2314	9.77	0
8740	8740	2314	1.09	0
8741	8741	2314	9.8077	0
8742	8742	2314	1.1305	0
8743	8743	2314	56	0
8744	8744	2314	23	0
8745	8745	2314	1	0
8746	8746	2314	3	0
8747	8747	2314	0.7	0
8748	8748	2314	1	0
8749	8749	2314	1.1	0
8750	8750	2314	1.2	0
8751	8751	2314	3	0
8752	8752	2314	0	0
8753	8753	2314	1.3	0
8754	8754	2314	0.6	0
8755	8755	2314	0.7	0
8756	8756	2314	1	0
8757	8757	2314	0.8	0
8758	8758	2314	1	0
8759	8759	2314	0.8	0
8760	8760	2314	1	0
8761	8761	2314	2	0
8762	8762	2314	20	0
8763	8763	2314	4	0
8764	8764	2314	1	0
8765	8765	2314	8	0
8766	8766	2314	4	0
8767	8767	2314	4	0
8768	8768	2314	3	0
8769	8769	2314	1	0
8770	8770	2314	0	0
8771	8771	2314	0	0
8772	8772	2314	0	0
8773	8773	2314	0	0
8774	8774	2314	0	0
8775	8775	2314	0	0
8776	8776	2314	0	0
8777	8777	2314	B0L7	0
8778	8778	2314	1.3	0
8779	8779	2314	1.4	0
8780	8780	2314	29.9	0
8781	8781	2314	15	0
8782	8782	2314	45.6	0
8783	8783	2314	55	0
8784	8784	2314	30.3	0
8785	8785	2314	294	0
8786	8786	2314	3	0
8787	8787	2314	2.7	0
8788	8788	2314	0	0
8789	8789	2314	0	0
8790	8790	2314	0	0
8791	8791	2314	79	0
8792	8792	2314	400	0
8793	8793	2314	400	0
8794	8794	2314	30	0
8795	8795	2314	30	0
8796	8796	2314	95	0
8797	8797	2314	0	0
8798	8798	2314	2.185	0
8799	8799	2314	28.03333333	0
8800	8800	2314	7.066666667	0
8801	8801	2314	350	0
8802	8802	2314	400	0
8803	8803	2314	2.8	0
8804	8804	2314	101	0
8805	8805	2314	41	0
8806	8806	2314	4.91	0
8807	8807	2314	0	0
8808	8808	2314	0	0
8809	8809	2314	85	0
8810	8810	2314	0	0
8811	8811	2314	40	0
8812	8812	2314	90	0
8813	8813	2314	82	0
8814	8814	2314	15	0
8815	8815	2314	350	0
8816	8816	2314	400	0
8817	8817	2314	2.8	0
8818	8818	2314	101	0
8819	8819	2314	41	0
8820	8820	2314	4.91	0
8821	8821	2314	0	0
8822	8822	2314	0	0
8823	8823	2314	85	0
8824	8824	2314	0	0
8825	8825	2314	40	0
8826	8826	2314	105.6	0
8827	8827	2314	26.9	0
8828	8828	2314	313	0
8829	8829	2314	112	0
8830	8830	2314	15.7	0
8831	8831	2314	8.2	0
8832	8832	2314	27.84	0
8833	8833	2314	4.03	0
8834	8834	2314	27.8669	0
8835	8835	2314	4.1502	0
8836	8836	2314	26	0
8837	8837	2314	112	0
8838	8838	2314	2	0
8839	8839	2314	3	0
8840	8840	2314	0.5	0
8841	8841	2314	1	0
8842	8842	2314	1.2	0
8843	8843	2314	1	0
8844	8844	2314	3	0
8845	8845	2314	0	0
8846	8846	2314	0.8	0
8847	8847	2314	0.3	0
8848	8848	2314	0.6	0
8849	8849	2314	0.9	0
8850	8850	2314	0.6	0
8851	8851	2314	0.8	0
8852	8852	2314		0
8853	8853	2314		0
8854	8854	2314	2	0
8855	8855	2314	16	0
8856	8856	2314	4	0
8857	8857	2314	1	0
8858	8858	2314	7	0
8859	8859	2314	3	0
8860	8860	2314	3	0
8861	8861	2314	2	0
8862	8862	2314	1	0
8863	8863	2314	0	0
8864	8864	2314	0	0
8865	8865	2314	0	0
8866	8866	2314	0	0
8867	8867	2314	0	0
8868	8868	2314	0	0
8869	8869	2314	0	0
8870	8870	2314	B0L6	0
8871	8871	2314	1.2	0
8872	8872	2314	1.4	0
8873	8873	2314	31.5	0
8874	8874	2314	15	0
8875	8875	2314	63	0
8876	8876	2314	67	0
8877	8877	2314	27.4	0
8878	8878	2314	404	0
8879	8879	2314	2.8	0
8880	8880	2314	2.3	0
8881	8881	2314	59	0
8882	8882	2314	36	0
8883	8883	2314	13	0
8884	8884	2314	105	0
8885	8885	2314	350	0
8886	8886	2314	350	0
8887	8887	2314	5	0
8888	8888	2314	60	0
8889	8889	2314	80	0
8890	8890	2314	1	0
8891	8891	2314	1.986666667	0
8892	8892	2314	25.25	0
8893	8893	2314	6.733333333	0
8894	8894	2314	350	0
8895	8895	2314	475	0
8896	8896	2314	3.23	0
8897	8897	2314	111	0
8898	8898	2314	52	0
8899	8899	2314	5.26	0
8900	8900	2314	5	0
8901	8901	2314	0	0
8902	8902	2314	75	0
8903	8903	2314	0	0
8904	8904	2314	0	0
8905	8905	2314	293	0
8906	8906	2314	114	0
8907	8907	2314	30	0
8908	8908	2314	350	0
8909	8909	2314	475	0
8910	8910	2314	3.23	0
8911	8911	2314	111	0
8912	8912	2314	52	0
8913	8913	2314	5.26	0
8914	8914	2314	5	0
8915	8915	2314	0	0
8916	8916	2314	75	0
8917	8917	2314	0	0
8918	8918	2314	0	0
8919	8919	2314	109.8	0
8920	8920	2314	38.8	0
8921	8921	2314	238.2	0
8922	8922	2314	79.8	0
8923	8923	2314	33.3	0
8924	8924	2314	13	0
8925	8925	2314	21.96	0
8926	8926	2314	3.94	0
8927	8927	2314	21.9988	0
8928	8928	2314	4.0328	0
8929	8929	2314	28	0
8930	8930	2314	3	0
8931	8931	2314	2	0
8932	8932	2314	3	0
8933	8933	2314	0.7	0
8934	8934	2314	1	0
8935	8935	2314	1.1	0
8936	8936	2314	1.4	0
8937	8937	2314	4	0
8938	8938	2314	0	0
8939	8939	2314	0.4	0
8940	8940	2314	0.9	0
8941	8941	2314	0.6	0
8942	8942	2314	1	0
8943	8943	2314	0.8	0
8944	8944	2314	0.9	0
8945	8945	2314	0.8	0
8946	8946	2314	1.1	0
8947	8947	2314	0	0
8948	8948	2314	22	0
8949	8949	2314	5	0
8950	8950	2314	1	0
8951	8951	2314	8	0
8952	8952	2314	4	0
8953	8953	2314	4	0
8954	8954	2314	3	0
8955	8955	2314	2	0
8956	8956	2314	1	0
8957	8957	2314	0	0
8958	8958	2314	0	0
8959	8959	2314	0	0
8960	8960	2314	0	0
8961	8961	2314	0	0
8962	8962	2314	0	0
8963	8963	2314	B0L6	0
8964	8964	2314	1.3	0
8965	8965	2314	1.4	0
8966	8966	2314	33.2	0
8967	8967	2314	16	0
8968	8968	2314	57.5	0
8969	8969	2314	57	0
8970	8970	2314	35	0
8971	8971	2314	475	0
8972	8972	2314	2.6	0
8973	8973	2314	2	0
8974	8974	2314	54	0
8975	8975	2314	42	0
8976	8976	2314	22	0
8977	8977	2314	114	0
8978	8978	2314	300	0
8979	8979	2314	300	0
8980	8980	2314	10	0
8981	8981	2314	40	0
8982	8982	2314	70	0
8983	8983	2314	3	0
8984	8984	2314	1.77	0
8985	8985	2314	20.15	0
8986	8986	2314	5.15	0
8987	8987	2314	300	0
8988	8988	2314	325	0
8989	8989	2314	2.4	0
8990	8990	2314	94	0
8991	8991	2314	42	0
8992	8992	2314	4.16	0
8993	8993	2314	11	0
8994	8994	2314	0	0
8995	8995	2314	80	0
8996	8996	2314	0	0
8997	8997	2314	0	0
8998	8998	2314	309	0
8999	8999	2314	84	0
9000	9000	2314	32	0
9001	9001	2314	300	0
9002	9002	2314	325	0
9003	9003	2314	2.4	0
9004	9004	2314	94	0
9005	9005	2314	42	0
9006	9006	2314	4.16	0
9007	9007	2314	11	0
9008	9008	2314	0	0
9009	9009	2314	80	0
9010	9010	2314	0	0
9011	9011	2314	0	0
9012	9012	2314	82	0
9013	9013	2314	29.1	0
9014	9014	2314	265.9	0
9015	9015	2314	95	0
9016	9016	2314	8.7	0
9017	9017	2314	4.4	0
9018	9018	2314	20.58	0
9019	9019	2314	3.32	0
9020	9020	2314	20.6091	0
9021	9021	2314	3.4194	0
9022	9022	2314	3	0
9023	9023	2314	1	0
9024	9024	2314	1	0
9025	9025	2314	2	0
9026	9026	2314	0.2	0
9027	9027	2314	0.2	0
9028	9028	2314		0
9029	9029	2314		0
9030	9030	2314	2	0
9031	9031	2314	0	0
9032	9032	2314	.	0
9033	9033	2314	.	0
9034	9034	2314	0.2	0
9035	9035	2314	0.2	0
9036	9036	2314	0.2	0
9037	9037	2314	0.2	0
9038	9038	2314		0
9039	9039	2314		0
9040	9040	2314	0	0
9041	9041	2314	5	0
9042	9042	2314	0	0
9043	9043	2314	.	0
9044	9044	2314	5	0
9045	9045	2314	.	0
9046	9046	2314	.	0
9047	9047	2314	.	0
9048	9048	2314	.	0
9049	9049	2314	0	0
9050	9050	2314	0	0
9051	9051	2314	0	0
9052	9052	2314	0	0
9053	9053	2314	0	0
9054	9054	2314	0	0
9055	9055	2314	0	0
9056	9056	2314	B0L4	0
9057	9057	2314	0.5	0
9058	9058	2314	0.6	0
9059	9059	2314	31.4	0
9060	9060	2314	4	0
9061	9061	2314	3.4	0
9062	9062	2314	0	0
9063	9063	2314	.	0
9064	9064	2314	12	0
9065	9065	2314	2.1	0
9066	9066	2314	1.8	0
9067	9067	2314	0	0
9068	9068	2314	0	0
9069	9069	2314	0	0
9070	9070	2314	66	0
9071	9071	2314	50	0
9072	9072	2314	50	0
9073	9073	2314	5	0
9074	9074	2314	10	0
9075	9075	2314	90	0
9076	9076	2314	0	0
9077	9077	2314	3.396666667	0
9078	9078	2314	41.11666667	0
9079	9079	2314	7.933333333	0
9080	9080	2314	150	0
9081	9081	2314	150	0
9082	9082	2314	2.75	0
9083	9083	2314	95	0
9084	9084	2314	36	0
9085	9085	2314	2.27	0
9086	9086	2314	0	0
9087	9087	2314	0	0
9088	9088	2314	85	0
9089	9089	2314	0	0
9090	9090	2314	0	0
9091	9091	2314	27	0
9092	9092	2314	20	0
9093	9093	2314	13	0
9094	9094	2314	150	0
9095	9095	2314	150	0
9096	9096	2314	2.75	0
9097	9097	2314	95	0
9098	9098	2314	36	0
9099	9099	2314	2.27	0
9100	9100	2314	0	0
9101	9101	2314	0	0
9102	9102	2314	85	0
9103	9103	2314	0	0
9104	9104	2314	0	0
9105	9105	2314	80.7	0
9106	9106	2314	26.2	0
9107	9107	2314	61.3	0
9108	9108	2314	18.1	0
9109	9109	2314	1.4	0
9110	9110	2314	0.7	0
9111	9111	2314	4.18	0
9112	9112	2314	1.02	0
9113	9113	2314	4.2062	0
9114	9114	2314	1.0388	0
9115	9115	2314	.	0
9116	9116	2314	.	0
9117	9117	2314	0	0
9118	9118	2314	0	0
9119	9119	2314		0
9120	9120	2314		0
9121	9121	2314		0
9122	9122	2314		0
9123	9123	2314	0	0
9124	9124	2314	0	0
9125	9125	2314		0
9126	9126	2314		0
9127	9127	2314		0
9128	9128	2314		0
9129	9129	2314		0
9130	9130	2314		0
9131	9131	2314		0
9132	9132	2314		0
9133	9133	2314	0	0
9134	9134	2314	0	0
9135	9135	2314		0
9136	9136	2314	.	0
9137	9137	2314	0	0
9138	9138	2314	.	0
9139	9139	2314	.	0
9140	9140	2314	.	0
9141	9141	2314	.	0
9142	9142	2314	0	0
9143	9143	2314	0	0
9144	9144	2314	0	0
9145	9145	2314	0	0
9146	9146	2314	0	0
9147	9147	2314	0	0
9148	9148	2314	0	0
9149	9149	2314	.	0
9150	9150	2314	.	0
9151	9151	2314	.	0
9152	9152	2314	.	0
9153	9153	2314	0	0
9154	9154	2314	.	0
9155	9155	2314	0	0
9156	9156	2314	.	0
9157	9157	2314	0	0
9158	9158	2314	.	0
9159	9159	2314	.	0
9160	9160	2314	0	0
9161	9161	2314	0	0
9162	9162	2314	0	0
9163	9163	2314		0
9164	9164	2314		0
9165	9165	2314	0	0
9166	9166	2314		0
9167	9167	2314		0
9168	9168	2314		0
9169	9169	2314		0
9170	9170	2314		0
9171	9171	2314		0
9172	9172	2314		0
9173	9173	2314		0
9174	9174	2314		0
9175	9175	2314		0
9176	9176	2314		0
9177	9177	2314		0
9178	9178	2314		0
9179	9179	2314		0
9180	9180	2314		0
9181	9181	2314		0
9182	9182	2314		0
9183	9183	2314		0
9184	9184	2314		0
9185	9185	2314		0
9186	9186	2314		0
9187	9187	2314	.	0
9188	9188	2314	.	0
9189	9189	2314	.	0
9190	9190	2314	.	0
9191	9191	2314	.	0
9192	9192	2314	.	0
9193	9193	2314	.	0
9194	9194	2314	.	0
9195	9195	2314	.	0
9196	9196	2314	.	0
9197	9197	2314	.	0
9198	9198	2314	.	0
9199	9199	2314	.	0
9200	9200	2314	.	0
9201	9201	2314	.	0
9202	9202	2314	.	0
9203	9203	2314	.	0
9204	9204	2314		0
9205	9205	2314		0
9206	9206	2314	.	0
9207	9207	2314	.	0
9208	9208	2314	67	0
9209	9209	2314	20	0
9210	9210	2314	2	0
9211	9211	2314	3	0
9212	9212	2314	0.4	0
9213	9213	2314	0.8	0
9214	9214	2314	1.2	0
9215	9215	2314	1.4	0
9216	9216	2314	4	0
9217	9217	2314	0	0
9218	9218	2314	0.9	0
9219	9219	2314	0.5	0
9220	9220	2314	0.5	0
9221	9221	2314	0.8	0
9222	9222	2314	0.9	0
9223	9223	2314	0.9	0
9224	9224	2314	0.8	0
9225	9225	2314	1	0
9226	9226	2314	1	0
9227	9227	2314	24	0
9228	9228	2314	6	0
9229	9229	2314	1	0
9230	9230	2314	8	0
9231	9231	2314	4	0
9232	9232	2314	4	0
9233	9233	2314	3	0
9234	9234	2314	2	0
9235	9235	2314	2	0
9236	9236	2314	0	0
9237	9237	2314	1	0
9238	9238	2314	0	0
9239	9239	2314	0	0
9240	9240	2314	0	0
9241	9241	2314	0	0
9242	9242	2314	B0L7	0
9243	9243	2314	1.3	0
9244	9244	2314	1.5	0
9245	9245	2314	24.5	0
9246	9246	2314	16	0
9247	9247	2314	40.6	0
9248	9248	2314	53	0
9249	9249	2314	23.2	0
9250	9250	2314	323	0
9251	9251	2314	3	0
9252	9252	2314	2.3	0
9253	9253	2314	0	0
9254	9254	2314	0	0
9255	9255	2314	0	0
9256	9256	2314	64.4	0
9257	9257	2314	350	0
9258	9258	2314	350	0
9259	9259	2314	50	0
9260	9260	2314	50	0
9261	9261	2314	95	0
9262	9262	2314	0	0
9263	9263	2314	1.783333333	0
9264	9264	2314	22.26666667	0
9265	9265	2314	5.883333333	0
9266	9266	2314	300	0
9267	9267	2314	450	0
9268	9268	2314	2.29	0
9269	9269	2314	79	0
9270	9270	2314	39	0
9271	9271	2314	3.84	0
9272	9272	2314	0	0
9273	9273	2314	0	0
9274	9274	2314	85	0
9275	9275	2314	0	0
9276	9276	2314	0	0
9277	9277	2314	119	0
9278	9278	2314	77	0
9279	9279	2314	65	0
9280	9280	2314	300	0
9281	9281	2314	450	0
9282	9282	2314	2.29	0
9283	9283	2314	79	0
9284	9284	2314	39	0
9285	9285	2314	3.84	0
9286	9286	2314	0	0
9287	9287	2314	0	0
9288	9288	2314	85	0
9289	9289	2314	0	0
9290	9290	2314	0	0
9291	9291	2314	95	0
9292	9292	2314	29.1	0
9293	9293	2314	161.9	0
9294	9294	2314	63.7	0
9295	9295	2314	11.2	0
9296	9296	2314	7.2	0
9297	9297	2314	19.42	0
9298	9298	2314	2.95	0
9299	9299	2314	19.4491	0
9300	9300	2314	3.0209	0
9301	9301	2314	115	0
9302	9302	2314	9	0
9303	9303	2314	1	0
9304	9304	2314	2	0
9305	9305	2314	0.5	0
9306	9306	2314	0.8	0
9307	9307	2314		0
9308	9308	2314		0
9309	9309	2314	2	0
9310	9310	2314	0	0
9311	9311	2314	0.7	0
9312	9312	2314	0.5	0
9313	9313	2314	0.6	0
9314	9314	2314	0.9	0
9315	9315	2314	0.5	0
9316	9316	2314	0.7	0
9317	9317	2314		0
9318	9318	2314		0
9319	9319	2314	2	0
9320	9320	2314	11	0
9321	9321	2314	3	0
9322	9322	2314	1	0
9323	9323	2314	5	0
9324	9324	2314	3	0
9325	9325	2314	2	0
9326	9326	2314	1	0
9327	9327	2314	0	0
9328	9328	2314	0	0
9329	9329	2314	0	0
9330	9330	2314	0	0
9331	9331	2314	0	0
9332	9332	2314	0	0
9333	9333	2314	0	0
9334	9334	2314	0	0
9335	9335	2314	B0L4	0
9336	9336	2314	1.2	0
9337	9337	2314	1.4	0
9338	9338	2314	25.6	0
9339	9339	2314	10	0
9340	9340	2314	38.6	0
9341	9341	2314	22	0
9342	9342	2314	11.2	0
9343	9343	2314	157	0
9344	9344	2314	2.9	0
9345	9345	2314	2.8	0
9346	9346	2314	0	0
9347	9347	2314	0	0
9348	9348	2314	0	0
9349	9349	2314	76.5	0
9350	9350	2314	350	0
9351	9351	2314	350	0
9352	9352	2314	25	0
9353	9353	2314	25	0
9354	9354	2314	85	0
9355	9355	2314	0	0
9356	9356	2314	2.746666667	0
9357	9357	2314	36.6	0
9358	9358	2314	7.783333333	0
9359	9359	2314	275	0
9360	9360	2314	300	0
9361	9361	2314	2.61	0
9362	9362	2314	100	0
9363	9363	2314	51	0
9364	9364	2314	3.55	0
9365	9365	2314	0	0
9366	9366	2314	0	0
9367	9367	2314	80	0
9368	9368	2314	0	0
9369	9369	2314	50	0
9370	9370	2314	97	0
9371	9371	2314	89	0
9372	9372	2314	25	0
9373	9373	2314	275	0
9374	9374	2314	300	0
9375	9375	2314	2.61	0
9376	9376	2314	100	0
9377	9377	2314	51	0
9378	9378	2314	3.55	0
9379	9379	2314	0	0
9380	9380	2314	0	0
9381	9381	2314	80	0
9382	9382	2314	0	0
9383	9383	2314	50	0
9384	9384	2314	84.5	0
9385	9385	2314	30.4	0
9386	9386	2314	158.5	0
9387	9387	2314	66.7	0
9388	9388	2314	6.9	0
9389	9389	2314	6.2	0
9390	9390	2314	20.36	0
9391	9391	2314	1.93	0
9392	9392	2314	20.3904	0
9393	9393	2314	2.0029	0
9394	9394	2314	18	0
9395	9395	2314	11	0
9396	9396	2314	0	0
9397	9397	2314	0	0
9398	9398	2314		0
9399	9399	2314		0
9400	9400	2314		0
9401	9401	2314		0
9402	9402	2314	1	0
9403	9403	2314	0	0
9404	9404	2314	0.4	0
9405	9405	2314	0.2	0
9406	9406	2314	.	0
9407	9407	2314	.	0
9408	9408	2314		0
9409	9409	2314		0
9410	9410	2314		0
9411	9411	2314		0
9412	9412	2314	1	0
9413	9413	2314	3	0
9414	9414	2314	0	0
9415	9415	2314	.	0
9416	9416	2314	3	0
9417	9417	2314	0	0
9418	9418	2314	0	0
9419	9419	2314	0	0
9420	9420	2314	0	0
9421	9421	2314	0	0
9422	9422	2314	0	0
9423	9423	2314	0	0
9424	9424	2314	0	0
9425	9425	2314	0	0
9426	9426	2314	0	0
9427	9427	2314	0	0
9428	9428	2314	B0L3	0
9429	9429	2314	0.4	0
9430	9430	2314	0.6	0
9431	9431	2314	19.8	0
9432	9432	2314	5	0
9433	9433	2314	1.2	0
9434	9434	2314	0	0
9435	9435	2314	.	0
9436	9436	2314	19	0
9437	9437	2314	2.1	0
9438	9438	2314	2.2	0
9439	9439	2314	0	0
9440	9440	2314	0	0
9441	9441	2314	0	0
9442	9442	2314	68	0
9443	9443	2314	225	0
9444	9444	2314	225	0
9445	9445	2314	15	0
9446	9446	2314	35	0
9447	9447	2314	85	0
9448	9448	2314	0	0
9449	9449	2314	2.77	0
9450	9450	2314	31.88333333	0
9451	9451	2314	7.4	0
9452	9452	2314	50	0
9453	9453	2314	125	0
9454	9454	2314	2.87	0
9455	9455	2314	74	0
9456	9456	2314	74	0
9457	9457	2314	3.04	0
9458	9458	2314	0	0
9459	9459	2314	0	0
9460	9460	2314	50	0
9461	9461	2314	10	0
9462	9462	2314	0	0
9463	9463	2314	111	0
9464	9464	2314	28	0
9465	9465	2314	5	0
9466	9466	2314	50	0
9467	9467	2314	125	0
9468	9468	2314	2.87	0
9469	9469	2314	74	0
9470	9470	2314	74	0
9471	9471	2314	3.04	0
9472	9472	2314	0	0
9473	9473	2314	0	0
9474	9474	2314	50	0
9475	9475	2314	10	0
9476	9476	2314	0	0
9477	9477	2314	111.7	0
9478	9478	2314	50.9	0
9479	9479	2314	153.3	0
9480	9480	2314	56.4	0
9481	9481	2314	19.9	0
9482	9482	2314	11.6	0
9483	9483	2314	9.99	0
9484	9484	2314	1.21	0
9485	9485	2314	10.0409	0
9486	9486	2314	1.278	0
9487	9487	2314	17	0
9488	9488	2314	6	0
9489	9489	2314	2	0
9490	9490	2314	2	0
9491	9491	2314	0.5	0
9492	9492	2314	0.5	0
9493	9493	2314	0.8	0
9494	9494	2314	0.8	0
9495	9495	2314	3	0
9496	9496	2314	0	0
9497	9497	2314	.	0
9498	9498	2314	.	0
9499	9499	2314	0.5	0
9500	9500	2314	0.5	0
9501	9501	2314	0.5	0
9502	9502	2314	0.7	0
9503	9503	2314	0.5	0
9504	9504	2314	0.6	0
9505	9505	2314	0	0
9506	9506	2314	10	0
9507	9507	2314	2	0
9508	9508	2314	2	0
9509	9509	2314	6	0
9510	9510	2314	2	0
9511	9511	2314	2	0
9512	9512	2314	0	0
9513	9513	2314	0	0
9514	9514	2314	0	0
9515	9515	2314	0	0
9516	9516	2314	0	0
9517	9517	2314	0	0
9518	9518	2314	0	0
9519	9519	2314	0	0
9520	9520	2314	0	0
9521	9521	2314	B0L5	0
9522	9522	2314	1.1	0
9523	9523	2314	1.35	0
9524	9524	2314	33.8	0
9525	9525	2314	14	0
9526	9526	2314	41.4	0
9527	9527	2314	23	0
9528	9528	2314	7.6	0
9529	9529	2314	102	0
9530	9530	2314	3	0
9531	9531	2314	2.7	0
9532	9532	2314	0	0
9533	9533	2314	0	0
9534	9534	2314	0	0
9535	9535	2314	63.5	0
9536	9536	2314	175	0
9537	9537	2314	175	0
9538	9538	2314	15	0
9539	9539	2314	35	0
9540	9540	2314	60	0
9541	9541	2314	0	0
9542	9542	2314	1.69	0
9543	9543	2314	20.96666667	0
9544	9544	2314	6.366666667	0
9545	9545	2314	200	0
9546	9546	2314	200	0
9547	9547	2314	2.43	0
9548	9548	2314	90	0
9549	9549	2314	51	0
9550	9550	2314	4.45	0
9551	9551	2314	5	0
9552	9552	2314	0	0
9553	9553	2314	80	0
9554	9554	2314	0	0
9555	9555	2314	40	0
9556	9556	2314	121	0
9557	9557	2314	75	0
9558	9558	2314	16	0
9559	9559	2314	200	0
9560	9560	2314	200	0
9561	9561	2314	2.43	0
9562	9562	2314	90	0
9563	9563	2314	51	0
9564	9564	2314	4.45	0
9565	9565	2314	5	0
9566	9566	2314	0	0
9567	9567	2314	80	0
9568	9568	2314	0	0
9569	9569	2314	40	0
9570	9570	2314	84.8	0
9571	9571	2314	32.1	0
9572	9572	2314	229	0
9573	9573	2314	78.7	0
9574	9574	2314	15.8	0
9575	9575	2314	8.4	0
9576	9576	2314	12.37	0
9577	9577	2314	3.38	0
9578	9578	2314	12.4021	0
9579	9579	2314	3.4671	0
9580	9580	2314	12	0
9581	9581	2314	55	0
9582	9582	2314	2	0
9583	9583	2314	3	0
9584	9584	2314	0.6	0
9585	9585	2314	0.9	0
9586	9586	2314	1.5	0
9587	9587	2314	1.5	0
9588	9588	2314	4	0
9589	9589	2314	0	0
9590	9590	2314	1.2	0
9591	9591	2314	0.5	0
9592	9592	2314	0.8	0
9593	9593	2314	1	0
9594	9594	2314	0.9	0
9595	9595	2314	1.1	0
9596	9596	2314	0.8	0
9597	9597	2314	1.2	0
9598	9598	2314	2	0
9599	9599	2314	20	0
9600	9600	2314	4	0
9601	9601	2314	1	0
9602	9602	2314	8	0
9603	9603	2314	4	0
9604	9604	2314	4	0
9605	9605	2314	3	0
9606	9606	2314	1	0
9607	9607	2314	0	0
9608	9608	2314	0	0
9609	9609	2314	0	0
9610	9610	2314	0	0
9611	9611	2314	0	0
9612	9612	2314	0	0
9613	9613	2314	0	0
9614	9614	2314	B0L7	0
9615	9615	2314	1.5	0
9616	9616	2314	1.8	0
9617	9617	2314	38.6	0
9618	9618	2314	17	0
9619	9619	2314	43.1	0
9620	9620	2314	16	0
9621	9621	2314	29.4	0
9622	9622	2314	270	0
9623	9623	2314	2.9	0
9624	9624	2314	2.7	0
9625	9625	2314	20	0
9626	9626	2314	17	0
9627	9627	2314	3	0
9628	9628	2314	73.5	0
9629	9629	2314	200	0
9630	9630	2314	200	0
9631	9631	2314	15	0
9632	9632	2314	40	0
9633	9633	2314	85	0
9634	9634	2314	0	0
9635	9635	2314	2.573333333	0
9636	9636	2314	31.75	0
9637	9637	2314	9.016666667	0
9638	9638	2314	350	0
9639	9639	2314	400	0
9640	9640	2314	2.4	0
9641	9641	2314	98	0
9642	9642	2314	46	0
9643	9643	2314	4.55	0
9644	9644	2314	2	0
9645	9645	2314	0	0
9646	9646	2314	80	0
9647	9647	2314	0	0
9648	9648	2314	0	0
9649	9649	2314	223	0
9650	9650	2314	217	0
9651	9651	2314	14	0
9652	9652	2314	350	0
9653	9653	2314	400	0
9654	9654	2314	2.4	0
9655	9655	2314	98	0
9656	9656	2314	46	0
9657	9657	2314	4.55	0
9658	9658	2314	2	0
9659	9659	2314	0	0
9660	9660	2314	80	0
9661	9661	2314	0	0
9662	9662	2314	0	0
9663	9663	2314	91.9	0
9664	9664	2314	25.2	0
9665	9665	2314	283.5	0
9666	9666	2314	84.9	0
9667	9667	2314	43.5	0
9668	9668	2314	15.3	0
9669	9669	2314	19.58	0
9670	9670	2314	3.84	0
9671	9671	2314	19.6052	0
9672	9672	2314	3.9402	0
9673	9673	2314	2	0
9674	9674	2314	3	0
9675	9675	2314	1	0
9676	9676	2314	2	0
9677	9677	2314	0.4	0
9678	9678	2314	0.6	0
9679	9679	2314	0.6	0
9680	9680	2314	0.6	0
9681	9681	2314	3	0
9682	9682	2314	0	0
9683	9683	2314	0.5	0
9684	9684	2314	1.1	0
9685	9685	2314	.	0
9686	9686	2314	.	0
9687	9687	2314	0.4	0
9688	9688	2314	0.5	0
9689	9689	2314	0.5	0
9690	9690	2314	0.6	0
9691	9691	2314	2	0
9692	9692	2314	16	0
9693	9693	2314	4	0
9694	9694	2314	1	0
9695	9695	2314	6	0
9696	9696	2314	3	0
9697	9697	2314	4	0
9698	9698	2314	2	0
9699	9699	2314	1	0
9700	9700	2314	0	0
9701	9701	2314	0	0
9702	9702	2314	0	0
9703	9703	2314	0	0
9704	9704	2314	0	0
9705	9705	2314	0	0
9706	9706	2314	0	0
9707	9707	2314	B0L5	0
9708	9708	2314	1.1	0
9709	9709	2314	1.2	0
9710	9710	2314	22.3	0
9711	9711	2314	16	0
9712	9712	2314	42.2	0
9713	9713	2314	62	0
9714	9714	2314	8.2	0
9715	9715	2314	256	0
9716	9716	2314	2.8	0
9717	9717	2314	2.4	0
9718	9718	2314	0	0
9719	9719	2314	2	0
9720	9720	2314	0	0
9721	9721	2314	57.5	0
9722	9722	2314	225	0
9723	9723	2314	225	0
9724	9724	2314	5	0
9725	9725	2314	35	0
9726	9726	2314	85	0
9727	9727	2314	0	0
9728	9728	2314	1.894	0
9729	9729	2314	20.86	0
9730	9730	2314	6.78	0
9731	9731	2314	175	0
9732	9732	2314	200	0
9733	9733	2314	2.6	0
9734	9734	2314	73	0
9735	9735	2314	51	0
9736	9736	2314	4.29	0
9737	9737	2314	0	0
9738	9738	2314	0	0
9739	9739	2314	80	0
9740	9740	2314	0	0
9741	9741	2314	0	0
9742	9742	2314	110	0
9743	9743	2314	70	0
9744	9744	2314	5	0
9745	9745	2314	175	0
9746	9746	2314	200	0
9747	9747	2314	2.6	0
9748	9748	2314	73	0
9749	9749	2314	51	0
9750	9750	2314	4.29	0
9751	9751	2314	0	0
9752	9752	2314	0	0
9753	9753	2314	80	0
9754	9754	2314	0	0
9755	9755	2314	0	0
9756	9756	2314	70.1	0
9757	9757	2314	25.1	0
9758	9758	2314	190.4	0
9759	9759	2314	75.3	0
9760	9760	2314	18.8	0
9761	9761	2314	8.6	0
9762	9762	2314	12	0
9763	9763	2314	3.08	0
9764	9764	2314	12.0251	0
9765	9765	2314	3.1639	0
9766	9766	2314	.	0
9767	9767	2314	.	0
9768	9768	2314	1	0
9769	9769	2314	1	0
9770	9770	2314	0.1	0
9771	9771	2314	0.1	0
9772	9772	2314		0
9773	9773	2314		0
9774	9774	2314	1	0
9775	9775	2314	0	0
9776	9776	2314	.	0
9777	9777	2314	.	0
9778	9778	2314	0.2	0
9779	9779	2314	0.2	0
9780	9780	2314		0
9781	9781	2314		0
9782	9782	2314		0
9783	9783	2314		0
9784	9784	2314	0	0
9785	9785	2314	0	0
9786	9786	2314	.	0
9787	9787	2314	.	0
9788	9788	2314	0	0
9789	9789	2314	.	0
9790	9790	2314	.	0
9791	9791	2314	.	0
9792	9792	2314	.	0
9793	9793	2314	0	0
9794	9794	2314	0	0
9795	9795	2314	0	0
9796	9796	2314	0	0
9797	9797	2314	0	0
9798	9798	2314	0	0
9799	9799	2314	0	0
9800	9800	2314	.	0
9801	9801	2314	.	0
9802	9802	2314	.	0
9803	9803	2314	.	0
9804	9804	2314	0	0
9805	9805	2314	.	0
9806	9806	2314	0	0
9807	9807	2314	.	0
9808	9808	2314	0	0
9809	9809	2314	.	0
9810	9810	2314	.	0
9811	9811	2314	0	0
9812	9812	2314	0	0
9813	9813	2314	0	0
9814	9814	2314		0
9815	9815	2314		0
9816	9816	2314	0	0
9817	9817	2314		0
9818	9818	2314		0
9819	9819	2314		0
9820	9820	2314		0
9821	9821	2314		0
9822	9822	2314		0
9823	9823	2314		0
9824	9824	2314		0
9825	9825	2314		0
9826	9826	2314		0
9827	9827	2314		0
9828	9828	2314		0
9829	9829	2314		0
9830	9830	2314		0
9831	9831	2314		0
9832	9832	2314		0
9833	9833	2314		0
9834	9834	2314		0
9835	9835	2314		0
9836	9836	2314		0
9837	9837	2314		0
9838	9838	2314	.	0
9839	9839	2314	.	0
9840	9840	2314	.	0
9841	9841	2314	.	0
9842	9842	2314	.	0
9843	9843	2314	.	0
9844	9844	2314	.	0
9845	9845	2314	.	0
9846	9846	2314	.	0
9847	9847	2314	.	0
9848	9848	2314	.	0
9849	9849	2314	.	0
9850	9850	2314	.	0
9851	9851	2314	.	0
9852	9852	2314	.	0
9853	9853	2314	.	0
9854	9854	2314	.	0
9855	9855	2314		0
9856	9856	2314		0
9857	9857	2314	.	0
9858	9858	2314	.	0
9859	9859	2314	16	0
9860	9860	2314	10	0
9861	9861	2314	2	0
9862	9862	2314	3	0
9863	9863	2314	0.4	0
9864	9864	2314	0.5	0
9865	9865	2314	1	0
9866	9866	2314	1.2	0
9867	9867	2314	4	0
9868	9868	2314	0	0
9869	9869	2314	0.7	0
9870	9870	2314	0.4	0
9871	9871	2314	0.4	0
9872	9872	2314	0.4	0
9873	9873	2314	0.5	0
9874	9874	2314	0.8	0
9875	9875	2314	0.6	0
9876	9876	2314	0.7	0
9877	9877	2314	1	0
9878	9878	2314	14	0
9879	9879	2314	4	0
9880	9880	2314	2	0
9881	9881	2314	6	0
9882	9882	2314	4	0
9883	9883	2314	2	0
9884	9884	2314	1	0
9885	9885	2314	1	0
9886	9886	2314	0	0
9887	9887	2314	0	0
9888	9888	2314	0	0
9889	9889	2314	0	0
9890	9890	2314	0	0
9891	9891	2314	0	0
9892	9892	2314	0	0
9893	9893	2314	B0L5	0
9894	9894	2314	1.3	0
9895	9895	2314	1.6	0
9896	9896	2314	40	0
9897	9897	2314	15	0
9898	9898	2314	50.1	0
9899	9899	2314	32	0
9900	9900	2314	24.8	0
9901	9901	2314	285	0
9902	9902	2314	2.5	0
9903	9903	2314	2.3	0
9904	9904	2314	0	0
9905	9905	2314	0	0
9906	9906	2314	0	0
9907	9907	2314	98.5	0
9908	9908	2314	175	0
9909	9909	2314	175	0
9910	9910	2314	10	0
9911	9911	2314	45	0
9912	9912	2314	75	0
9913	9913	2314	1	0
9914	9914	2314	2.456666667	0
9915	9915	2314	29.83333333	0
9916	9916	2314	7.45	0
9917	9917	2314	75	0
9918	9918	2314	275	0
9919	9919	2314	2.48	0
9920	9920	2314	114	0
9921	9921	2314	36	0
9922	9922	2314	3.78	0
9923	9923	2314	26	0
9924	9924	2314	0	0
9925	9925	2314	60	0
9926	9926	2314	10	0
9927	9927	2314	0	0
9928	9928	2314	99	0
9929	9929	2314	45	0
9930	9930	2314	2	0
9931	9931	2314	75	0
9932	9932	2314	275	0
9933	9933	2314	2.48	0
9934	9934	2314	114	0
9935	9935	2314	36	0
9936	9936	2314	3.78	0
9937	9937	2314	26	0
9938	9938	2314	0	0
9939	9939	2314	60	0
9940	9940	2314	10	0
9941	9941	2314	0	0
9942	9942	2314	66.1	0
9943	9943	2314	21.9	0
9944	9944	2314	216.1	0
9945	9945	2314	71.2	0
9946	9946	2314	15.8	0
9947	9947	2314	8.7	0
9948	9948	2314	9.93	0
9949	9949	2314	1.73	0
9950	9950	2314	9.9519	0
9951	9951	2314	1.8099	0
9952	9952	2314	24	0
9953	9953	2314	2	0
9954	9954	2314	1	0
9955	9955	2314	2	0
9956	9956	2314	0.4	0
9957	9957	2314	0.5	0
9958	9958	2314	0.6	0
9959	9959	2314	0.7	0
9960	9960	2314	3	0
9961	9961	2314	0	0
9962	9962	2314	0.7	0
9963	9963	2314	0.3	0
9964	9964	2314	0.4	0
9965	9965	2314	0.7	0
9966	9966	2314	0.4	0
9967	9967	2314	0.7	0
9968	9968	2314		0
9969	9969	2314		0
9970	9970	2314	2	0
9971	9971	2314	16	0
9972	9972	2314	4	0
9973	9973	2314	1	0
9974	9974	2314	8	0
9975	9975	2314	2	0
9976	9976	2314	3	0
9977	9977	2314	2	0
9978	9978	2314	1	0
9979	9979	2314	0	0
9980	9980	2314	0	0
9981	9981	2314	0	0
9982	9982	2314	0	0
9983	9983	2314	0	0
9984	9984	2314	0	0
9985	9985	2314	0	0
9986	9986	2314	B0L6	0
9987	9987	2314	1	0
9988	9988	2314	1.2	0
9989	9989	2314	24.7	0
9990	9990	2314	12	0
9991	9991	2314	43.3	0
9992	9992	2314	28	0
9993	9993	2314	7.3	0
9994	9994	2314	143	0
9995	9995	2314	2.3	0
9996	9996	2314	2.2	0
9997	9997	2314	0	0
9998	9998	2314	0	0
9999	9999	2314	0	0
10000	10000	2314	62	0
10001	10001	2314	350	0
10002	10002	2314	350	0
10003	10003	2314	20	0
10004	10004	2314	45	0
10005	10005	2314	85	0
10006	10006	2314	0	0
10007	10007	2314	2.235	0
10008	10008	2314	27.7	0
10009	10009	2314	5	0
10010	10010	2314	225	0
10011	10011	2314	225	0
10012	10012	2314	2.16	0
10013	10013	2314	89	0
10014	10014	2314	37	0
10015	10015	2314	3.53	0
10016	10016	2314	0	0
10017	10017	2314	0	0
10018	10018	2314	80	0
10019	10019	2314	5	0
10020	10020	2314	0	0
10021	10021	2314	95	0
10022	10022	2314	62	0
10023	10023	2314	28	0
10024	10024	2314	225	0
10025	10025	2314	225	0
10026	10026	2314	2.16	0
10027	10027	2314	89	0
10028	10028	2314	37	0
10029	10029	2314	3.53	0
10030	10030	2314	0	0
10031	10031	2314	0	0
10032	10032	2314	80	0
10033	10033	2314	5	0
10034	10034	2314	0	0
10035	10035	2314	81.1	0
10036	10036	2314	24.6	0
10037	10037	2314	187.3	0
10038	10038	2314	56.8	0
10039	10039	2314	8.3	0
10040	10040	2314	4.5	0
10041	10041	2314	13.53	0
10042	10042	2314	2.48	0
10043	10043	2314	13.5546	0
10044	10044	2314	2.5413	0
10045	10045	2314	3	0
10046	10046	2314	8	0
10047	10047	2314	1	0
10048	10048	2314	2	0
10049	10049	2314	0.4	0
10050	10050	2314	0.6	0
10051	10051	2314		0
10052	10052	2314		0
10053	10053	2314	3	0
10054	10054	2314	0	0
10055	10055	2314	0.7	0
10056	10056	2314	0.4	0
10057	10057	2314	0.4	0
10058	10058	2314	0.6	0
10059	10059	2314	0.5	0
10060	10060	2314	0.6	0
10061	10061	2314		0
10062	10062	2314		0
10063	10063	2314	2	0
10064	10064	2314	12	0
10065	10065	2314	3	0
10066	10066	2314	1	0
10067	10067	2314	6	0
10068	10068	2314	3	0
10069	10069	2314	2	0
10070	10070	2314	1	0
10071	10071	2314	0	0
10072	10072	2314	0	0
10073	10073	2314	0	0
10074	10074	2314	0	0
10075	10075	2314	0	0
10076	10076	2314	0	0
10077	10077	2314	0	0
10078	10078	2314	0	0
10079	10079	2314	B0L5	0
10080	10080	2314	0.9	0
10081	10081	2314	1	0
10082	10082	2314	24.2	0
10083	10083	2314	13	0
10084	10084	2314	37.5	0
10085	10085	2314	25	0
10086	10086	2314	26.5	0
10087	10087	2314	207	0
10088	10088	2314	2.6	0
10089	10089	2314	2.2	0
10090	10090	2314	0	0
10091	10091	2314	3	0
10092	10092	2314	0	0
10093	10093	2314	75	0
10094	10094	2314	250	0
10095	10095	2314	250	0
10096	10096	2314	20	0
10097	10097	2314	25	0
10098	10098	2314	90	0
10099	10099	2314	0	0
10100	10100	2314	2.42	0
10101	10101	2314	28.96666667	0
10102	10102	2314	5.55	0
10103	10103	2314	175	0
10104	10104	2314	175	0
10105	10105	2314	2.11	0
10106	10106	2314	95	0
10107	10107	2314	35	0
10108	10108	2314	2.79	0
10109	10109	2314	0	0
10110	10110	2314	0	0
10111	10111	2314	90	0
10112	10112	2314	0	0
10113	10113	2314	0	0
10114	10114	2314	54	0
10115	10115	2314	65	0
10116	10116	2314	34	0
10117	10117	2314	175	0
10118	10118	2314	175	0
10119	10119	2314	2.11	0
10120	10120	2314	95	0
10121	10121	2314	35	0
10122	10122	2314	2.79	0
10123	10123	2314	0	0
10124	10124	2314	0	0
10125	10125	2314	90	0
10126	10126	2314	0	0
10127	10127	2314	0	0
10128	10128	2314	37.9	0
10129	10129	2314	22.5	0
10130	10130	2314	42	0
10131	10131	2314	31.3	0
10132	10132	2314	5.1	0
10133	10133	2314	5	0
10134	10134	2314	11.39	0
10135	10135	2314	1.14	0
10136	10136	2314	11.4125	0
10137	10137	2314	1.1763	0
10138	10138	2314	40	0
10139	10139	2314	53	0
10140	10140	2314	2	0
10141	10141	2314	3	0
10142	10142	2314	0.5	0
10143	10143	2314	0.6	0
10144	10144	2314	1.2	0
10145	10145	2314	1.3	0
10146	10146	2314	4	0
10147	10147	2314	0	0
10148	10148	2314	0.9	0
10149	10149	2314	0.5	0
10150	10150	2314	0.5	0
10151	10151	2314	0.8	0
10152	10152	2314	0.7	0
10153	10153	2314	0.8	0
10154	10154	2314	0.9	0
10155	10155	2314	1.2	0
10156	10156	2314	2	0
10157	10157	2314	24	0
10158	10158	2314	6	0
10159	10159	2314	1	0
10160	10160	2314	8	0
10161	10161	2314	5	0
10162	10162	2314	4	0
10163	10163	2314	3	0
10164	10164	2314	2	0
10165	10165	2314	0	0
10166	10166	2314	0	0
10167	10167	2314	1	0
10168	10168	2314	0	0
10169	10169	2314	1	0
10170	10170	2314	0	0
10171	10171	2314	0	0
10172	10172	2314	B0L6	0
10173	10173	2314	1.5	0
10174	10174	2314	1.7	0
10175	10175	2314	42.6	0
10176	10176	2314	13	0
10177	10177	2314	63.3	0
10178	10178	2314	50	0
10179	10179	2314	40.2	0
10180	10180	2314	376	0
10181	10181	2314	3.2	0
10182	10182	2314	2.6	0
10183	10183	2314	0	0
10184	10184	2314	0	0
10185	10185	2314	0	0
10186	10186	2314	82	0
10187	10187	2314	225	0
10188	10188	2314	225	0
10189	10189	2314	15	0
10190	10190	2314	40	0
10191	10191	2314	85	0
10192	10192	2314	0	0
10193	10193	2314	1.503333333	0
10194	10194	2314	20.06666667	0
10195	10195	2314	5.85	0
10196	10196	2314	200	0
10197	10197	2314	275	0
10198	10198	2314	3.19	0
10199	10199	2314	93	0
10200	10200	2314	39	0
10201	10201	2314	3.85	0
10202	10202	2314	0	0
10203	10203	2314	0	0
10204	10204	2314	80	0
10205	10205	2314	0	0
10206	10206	2314	25	0
10207	10207	2314	139	0
10208	10208	2314	73	0
10209	10209	2314	14	0
10210	10210	2314	200	0
10211	10211	2314	275	0
10212	10212	2314	3.19	0
10213	10213	2314	93	0
10214	10214	2314	39	0
10215	10215	2314	3.85	0
10216	10216	2314	0	0
10217	10217	2314	0	0
10218	10218	2314	80	0
10219	10219	2314	0	0
10220	10220	2314	25	0
10221	10221	2314	110.6	0
10222	10222	2314	34.4	0
10223	10223	2314	265.9	0
10224	10224	2314	90.4	0
10225	10225	2314	22.8	0
10226	10226	2314	10.3	0
10227	10227	2314	19.15	0
10228	10228	2314	3.54	0
10229	10229	2314	19.1844	0
10230	10230	2314	3.6407	0
10231	10231	2314	10	0
10232	10232	2314	14	0
10233	10233	2314	1	0
10234	10234	2314	2	0
10235	10235	2314	0.4	0
10236	10236	2314	0.7	0
10237	10237	2314		0
10238	10238	2314		0
10239	10239	2314	3	0
10240	10240	2314	0	0
10241	10241	2314	0.8	0
10242	10242	2314	0.3	0
10243	10243	2314	0.5	0
10244	10244	2314	0.6	0
10245	10245	2314	0.4	0
10246	10246	2314	0.6	0
10247	10247	2314		0
10248	10248	2314		0
10249	10249	2314	2	0
10250	10250	2314	9	0
10251	10251	2314	1	0
10252	10252	2314	2	0
10253	10253	2314	7	0
10254	10254	2314	2	0
10255	10255	2314	0	0
10256	10256	2314	0	0
10257	10257	2314	0	0
10258	10258	2314	0	0
10259	10259	2314	0	0
10260	10260	2314	0	0
10261	10261	2314	0	0
10262	10262	2314	0	0
10263	10263	2314	0	0
10264	10264	2314	0	0
10265	10265	2314	B0L6	0
10266	10266	2314	1.15	0
10267	10267	2314	1.3	0
10268	10268	2314	33.1	0
10269	10269	2314	14	0
10270	10270	2314	32	0
10271	10271	2314	19	0
10272	10272	2314	6.1	0
10273	10273	2314	182	0
10274	10274	2314	3.5	0
10275	10275	2314	2.9	0
10276	10276	2314	0	0
10277	10277	2314	0	0
10278	10278	2314	0	0
10279	10279	2314	60.5	0
10280	10280	2314	300	0
10281	10281	2314	300	0
10282	10282	2314	15	0
10283	10283	2314	55	0
10284	10284	2314	90	0
10285	10285	2314	0	0
10286	10286	2314	2.755	0
10287	10287	2314	31.25	0
10288	10288	2314	9	0
10289	10289	2314	230	0
10290	10290	2314	275	0
10291	10291	2314	2.46	0
10292	10292	2314	70	0
10293	10293	2314	56	0
10294	10294	2314	4.13	0
10295	10295	2314	8	0
10296	10296	2314	0	0
10297	10297	2314	75	0
10298	10298	2314	0	0
10299	10299	2314	0	0
10300	10300	2314	180	0
10301	10301	2314	101	0
10302	10302	2314	10	0
10303	10303	2314	230	0
10304	10304	2314	275	0
10305	10305	2314	2.46	0
10306	10306	2314	70	0
10307	10307	2314	56	0
10308	10308	2314	4.13	0
10309	10309	2314	8	0
10310	10310	2314	0	0
10311	10311	2314	75	0
10312	10312	2314	0	0
10313	10313	2314	0	0
10314	10314	2314	65.1	0
10315	10315	2314	20.2	0
10316	10316	2314	96.8	0
10317	10317	2314	37.6	0
10318	10318	2314	19.3	0
10319	10319	2314	7.3	0
10320	10320	2314	15.73	0
10321	10321	2314	3.95	0
10322	10322	2314	15.7502	0
10323	10323	2314	3.9949	0
10324	10324	2314	20	0
10325	10325	2314	32	0
10326	10326	2314	1	0
10327	10327	2314	2	0
10328	10328	2314	0.6	0
10329	10329	2314	0.9	0
10330	10330	2314	1	0
10331	10331	2314	0.9	0
10332	10332	2314	3	0
10333	10333	2314	0	0
10334	10334	2314	1.1	0
10335	10335	2314	0.5	0
10336	10336	2314	0.5	0
10337	10337	2314	0.8	0
10338	10338	2314	0.7	0
10339	10339	2314	0.9	0
10340	10340	2314		0
10341	10341	2314		0
10342	10342	2314	2	0
10343	10343	2314	14	0
10344	10344	2314	3	0
10345	10345	2314	1	0
10346	10346	2314	7	0
10347	10347	2314	3	0
10348	10348	2314	3	0
10349	10349	2314	1	0
10350	10350	2314	0	0
10351	10351	2314	0	0
10352	10352	2314	0	0
10353	10353	2314	0	0
10354	10354	2314	0	0
10355	10355	2314	0	0
10356	10356	2314	0	0
10357	10357	2314	0	0
10358	10358	2314	B0L5	0
10359	10359	2314	1.3	0
10360	10360	2314	1.4	0
10361	10361	2314	28	0
10362	10362	2314	12	0
10363	10363	2314	36.8	0
10364	10364	2314	31	0
10365	10365	2314	19.8	0
10366	10366	2314	251	0
10367	10367	2314	2.8	0
10368	10368	2314	2.9	0
10369	10369	2314	25	0
10370	10370	2314	24	0
10371	10371	2314	0	0
10372	10372	2314	68.5	0
10373	10373	2314	450	0
10374	10374	2314	450	0
10375	10375	2314	20	0
10376	10376	2314	45	0
10377	10377	2314	95	0
10378	10378	2314	0	0
10379	10379	2314	3.705	0
10380	10380	2314	55.3	0
10381	10381	2314	13.38333333	0
10382	10382	2314	650	0
10383	10383	2314	650	0
10384	10384	2314	2.6	0
10385	10385	2314	105	0
10386	10386	2314	45	0
10387	10387	2314	5.47	0
10388	10388	2314	0	0
10389	10389	2314	0	0
10390	10390	2314	95	0
10391	10391	2314	0	0
10392	10392	2314	0	0
10393	10393	2314	69	0
10394	10394	2314	197	0
10395	10395	2314	121	0
10396	10396	2314	650	0
10397	10397	2314	650	0
10398	10398	2314	2.6	0
10399	10399	2314	105	0
10400	10400	2314	45	0
10401	10401	2314	5.47	0
10402	10402	2314	0	0
10403	10403	2314	0	0
10404	10404	2314	95	0
10405	10405	2314	0	0
10406	10406	2314	0	0
10407	10407	2314	88.6	0
10408	10408	2314	43.4	0
10409	10409	2314	247.1	0
10410	10410	2314	114.1	0
10411	10411	2314	8.1	0
10412	10412	2314	7.6	0
10413	10413	2314	42.86	0
10414	10414	2314	5.64	0
10415	10415	2314	42.9034	0
10416	10416	2314	5.7617	0
10417	10417	2314	20	0
10418	10418	2314	16	0
10419	10419	2314	1	0
10420	10420	2314	2	0
10421	10421	2314	0.5	0
10422	10422	2314	0.8	0
10423	10423	2314		0
10424	10424	2314		0
10425	10425	2314	3	0
10426	10426	2314	0	0
10427	10427	2314	0.9	0
10428	10428	2314	0.5	0
10429	10429	2314	0.4	0
10430	10430	2314	0.8	0
10431	10431	2314	0.4	0
10432	10432	2314	0.5	0
10433	10433	2314		0
10434	10434	2314		0
10435	10435	2314	2	0
10436	10436	2314	15	0
10437	10437	2314	4	0
10438	10438	2314	1	0
10439	10439	2314	7	0
10440	10440	2314	2	0
10441	10441	2314	3	0
10442	10442	2314	2	0
10443	10443	2314	1	0
10444	10444	2314	0	0
10445	10445	2314	0	0
10446	10446	2314	0	0
10447	10447	2314	0	0
10448	10448	2314	0	0
10449	10449	2314	0	0
10450	10450	2314	0	0
10451	10451	2314	B0L6	0
10452	10452	2314	1.4	0
10453	10453	2314	1.6	0
10454	10454	2314	27.5	0
10455	10455	2314	14	0
10456	10456	2314	41.2	0
10457	10457	2314	50	0
10458	10458	2314	32.2	0
10459	10459	2314	310	0
10460	10460	2314	3	0
10461	10461	2314	2.5	0
10462	10462	2314	0	0
10463	10463	2314	0	0
10464	10464	2314	0	0
10465	10465	2314	58	0
10466	10466	2314	225	0
10467	10467	2314	225	0
10468	10468	2314	10	0
10469	10469	2314	40	0
10470	10470	2314	85	0
10471	10471	2314	0	0
10472	10472	2314	1.36	0
10473	10473	2314	17.41666667	0
10474	10474	2314	4.733333333	0
10475	10475	2314	200	0
10476	10476	2314	200	0
10477	10477	2314	3.52	0
10478	10478	2314	82	0
10479	10479	2314	54	0
10480	10480	2314	4.75	0
10481	10481	2314	0	0
10482	10482	2314	0	0
10483	10483	2314	70	0
10484	10484	2314	0	0
10485	10485	2314	80	0
10486	10486	2314	141	0
10487	10487	2314	82	0
10488	10488	2314	0	0
10489	10489	2314	200	0
10490	10490	2314	200	0
10491	10491	2314	3.52	0
10492	10492	2314	82	0
10493	10493	2314	54	0
10494	10494	2314	4.75	0
10495	10495	2314	0	0
10496	10496	2314	0	0
10497	10497	2314	70	0
10498	10498	2314	0	0
10499	10499	2314	80	0
10500	10500	2314	150.9	0
10501	10501	2314	49.6	0
10502	10502	2314	195.6	0
10503	10503	2314	68.8	0
10504	10504	2314	23.2	0
10505	10505	2314	12.1	0
10506	10506	2314	15.1	0
10507	10507	2314	3.91	0
10508	10508	2314	15.1496	0
10509	10509	2314	3.9909	0
10510	10510	2314		0
10511	10511	2314		0
10512	10512	2314	1	0
10513	10513	2314	2	0
10514	10514	2314	0.2	0
10515	10515	2314	0.2	0
10516	10516	2314		0
10517	10517	2314		0
10518	10518	2314	2	0
10519	10519	2314	0	0
10520	10520	2314	0.8	0
10521	10521	2314	0.4	0
10522	10522	2314	0.4	0
10523	10523	2314	0.5	0
10524	10524	2314	0.2	0
10525	10525	2314	0.2	0
10526	10526	2314		0
10527	10527	2314		0
10528	10528	2314	0	0
10529	10529	2314	4	0
10530	10530	2314	0	0
10531	10531	2314	.	0
10532	10532	2314	4	0
10533	10533	2314	0	0
10534	10534	2314	0	0
10535	10535	2314	0	0
10536	10536	2314	0	0
10537	10537	2314	0	0
10538	10538	2314	0	0
10539	10539	2314	0	0
10540	10540	2314	0	0
10541	10541	2314	0	0
10542	10542	2314	0	0
10543	10543	2314	0	0
10544	10544	2314	B0L3	0
10545	10545	2314	0.4	0
10546	10546	2314	0.4	0
10547	10547	2314	2.8	0
10548	10548	2314	0	0
10549	10549	2314	.	0
10550	10550	2314	0	0
10551	10551	2314	.	0
10552	10552	2314	6	0
10553	10553	2314	0.5	0
10554	10554	2314	0.6	0
10555	10555	2314	17	0
10556	10556	2314	14	0
10557	10557	2314	0	0
10558	10558	2314		0
10559	10559	2314		0
10560	10560	2314	0	0
10561	10561	2314		0
10562	10562	2314		0
10563	10563	2314	0	0
10564	10564	2314		0
10565	10565	2314		0
10566	10566	2314		0
10567	10567	2314		0
10568	10568	2314		0
10569	10569	2314		0
10570	10570	2314		0
10571	10571	2314		0
10572	10572	2314		0
10573	10573	2314		0
10574	10574	2314		0
10575	10575	2314		0
10576	10576	2314	0	0
10577	10577	2314	0	0
10578	10578	2314	0	0
10579	10579	2314		0
10580	10580	2314		0
10581	10581	2314		0
10582	10582	2314	.	0
10583	10583	2314	.	0
10584	10584	2314	.	0
10585	10585	2314	.	0
10586	10586	2314	.	0
10587	10587	2314	.	0
10588	10588	2314	.	0
10589	10589	2314	.	0
10590	10590	2314	0	0
10591	10591	2314	0	0
10592	10592	2314	0	0
10593	10593	2314	.	0
10594	10594	2314	.	0
10595	10595	2314	.	0
10596	10596	2314	.	0
10597	10597	2314	.	0
10598	10598	2314	.	0
10599	10599	2314	0	0
10600	10600	2314	0.03	0
10601	10601	2314	.	0
10602	10602	2314	.	0
10603	10603	2314	31	0
10604	10604	2314	21	0
10605	10605	2314	1	0
10606	10606	2314	2	0
10607	10607	2314	0.4	0
10608	10608	2314	0.6	0
10609	10609	2314		0
10610	10610	2314		0
10611	10611	2314	2	0
10612	10612	2314	0	0
10613	10613	2314	0.9	0
10614	10614	2314	0.4	0
10615	10615	2314	0.5	0
10616	10616	2314	0.7	0
10617	10617	2314	0.4	0
10618	10618	2314	0.5	0
10619	10619	2314		0
10620	10620	2314		0
10621	10621	2314	2	0
10622	10622	2314	9	0
10623	10623	2314	2	0
10624	10624	2314	1	0
10625	10625	2314	6	0
10626	10626	2314	1	0
10627	10627	2314	2	0
10628	10628	2314	0	0
10629	10629	2314	0	0
10630	10630	2314	0	0
10631	10631	2314	0	0
10632	10632	2314	0	0
10633	10633	2314	0	0
10634	10634	2314	0	0
10635	10635	2314	0	0
10636	10636	2314	0	0
10637	10637	2314	B0L5	0
10638	10638	2314	1	0
10639	10639	2314	1.5	0
10640	10640	2314	24.3	0
10641	10641	2314	12	0
10642	10642	2314	34.4	0
10643	10643	2314	24	0
10644	10644	2314	18.3	0
10645	10645	2314	141	0
10646	10646	2314	3.1	0
10647	10647	2314	2.5	0
10648	10648	2314	0	0
10649	10649	2314	0	0
10650	10650	2314	0	0
10651	10651	2314	71.5	0
10652	10652	2314	225	0
10653	10653	2314	225	0
10654	10654	2314	20	0
10655	10655	2314	30	0
10656	10656	2314	85	0
10657	10657	2314	0	0
10658	10658	2314	1.406666667	0
10659	10659	2314	15.71666667	0
10660	10660	2314	4.566666667	0
10661	10661	2314	250	0
10662	10662	2314	250	0
10663	10663	2314	3.02	0
10664	10664	2314	63	0
10665	10665	2314	48	0
10666	10666	2314	2.24	0
10667	10667	2314	0	0
10668	10668	2314	0	0
10669	10669	2314	70	0
10670	10670	2314	0	0
10671	10671	2314	0	0
10672	10672	2314	144	0
10673	10673	2314	91	0
10674	10674	2314	23	0
10675	10675	2314	250	0
10676	10676	2314	250	0
10677	10677	2314	3.02	0
10678	10678	2314	63	0
10679	10679	2314	48	0
10680	10680	2314	2.24	0
10681	10681	2314	0	0
10682	10682	2314	0	0
10683	10683	2314	70	0
10684	10684	2314	0	0
10685	10685	2314	0	0
10686	10686	2314	92.2	0
10687	10687	2314	25.5	0
10688	10688	2314	69.3	0
10689	10689	2314	27	0
10690	10690	2314	7.5	0
10691	10691	2314	6.9	0
10692	10692	2314	19.67	0
10693	10693	2314	2.2	0
10694	10694	2314	19.6955	0
10695	10695	2314	2.2339	0
10696	10696	2314	.	0
10697	10697	2314	.	0
10698	10698	2314	1	0
10699	10699	2314	2	0
10700	10700	2314	0.3	0
10701	10701	2314	0.3	0
10702	10702	2314		0
10703	10703	2314		0
10704	10704	2314	1	0
10705	10705	2314	0	0
10706	10706	2314	0.3	0
10707	10707	2314	0.2	0
10708	10708	2314	0.3	0
10709	10709	2314	0.3	0
10710	10710	2314		0
10711	10711	2314		0
10712	10712	2314		0
10713	10713	2314		0
10714	10714	2314	0	0
10715	10715	2314	3	0
10716	10716	2314	0	0
10717	10717	2314	.	0
10718	10718	2314	3	0
10719	10719	2314	0	0
10720	10720	2314	0	0
10721	10721	2314	0	0
10722	10722	2314	0	0
10723	10723	2314	0	0
10724	10724	2314	0	0
10725	10725	2314	0	0
10726	10726	2314	0	0
10727	10727	2314	0	0
10728	10728	2314	0	0
10729	10729	2314	0	0
10730	10730	2314	B0L3	0
10731	10731	2314	0.35	0
10732	10732	2314	0.4	0
10733	10733	2314	8.5	0
10734	10734	2314	2	0
10735	10735	2314	3.5	0
10736	10736	2314	0	0
10737	10737	2314	.	0
10738	10738	2314	10	0
10739	10739	2314	0.9	0
10740	10740	2314	1	0
10741	10741	2314	0	0
10742	10742	2314	0	0
10743	10743	2314	0	0
10744	10744	2314		0
10745	10745	2314		0
10746	10746	2314	0	0
10747	10747	2314		0
10748	10748	2314		0
10749	10749	2314		0
10750	10750	2314		0
10751	10751	2314		0
10752	10752	2314		0
10753	10753	2314		0
10754	10754	2314		0
10755	10755	2314		0
10756	10756	2314		0
10757	10757	2314		0
10758	10758	2314		0
10759	10759	2314		0
10760	10760	2314		0
10761	10761	2314		0
10762	10762	2314		0
10763	10763	2314		0
10764	10764	2314		0
10765	10765	2314		0
10766	10766	2314		0
10767	10767	2314		0
10768	10768	2314	.	0
10769	10769	2314	.	0
10770	10770	2314	.	0
10771	10771	2314	.	0
10772	10772	2314	.	0
10773	10773	2314	.	0
10774	10774	2314	.	0
10775	10775	2314	.	0
10776	10776	2314	.	0
10777	10777	2314	.	0
10778	10778	2314	.	0
10779	10779	2314	.	0
10780	10780	2314	.	0
10781	10781	2314	.	0
10782	10782	2314	.	0
10783	10783	2314	.	0
10784	10784	2314	.	0
10785	10785	2314		0
10786	10786	2314		0
10787	10787	2314	.	0
10788	10788	2314	.	0
10789	10789	2314	23	0
10790	10790	2314	32	0
10791	10791	2314	2	0
10792	10792	2314	3	0
10793	10793	2314	0.3	0
10794	10794	2314	0.3	0
10795	10795	2314	1	0
10796	10796	2314	1	0
10797	10797	2314	4	0
10798	10798	2314	0	0
10799	10799	2314	1.1	0
10800	10800	2314	0.4	0
10801	10801	2314	0.3	0
10802	10802	2314	0.4	0
10803	10803	2314	0.5	0
10804	10804	2314	0.6	0
10805	10805	2314	0.6	0
10806	10806	2314	0.7	0
10807	10807	2314	2	0
10808	10808	2314	19	0
10809	10809	2314	4	0
10810	10810	2314	1	0
10811	10811	2314	9	0
10812	10812	2314	3	0
10813	10813	2314	4	0
10814	10814	2314	2	0
10815	10815	2314	1	0
10816	10816	2314	0	0
10817	10817	2314	0	0
10818	10818	2314	0	0
10819	10819	2314	0	0
10820	10820	2314	0	0
10821	10821	2314	0	0
10822	10822	2314	0	0
10823	10823	2314	B0L8	0
10824	10824	2314	1	0
10825	10825	2314	1.1	0
10826	10826	2314	18	0
10827	10827	2314	14	0
10828	10828	2314	31	0
10829	10829	2314	45	0
10830	10830	2314	15.7	0
10831	10831	2314	276	0
10832	10832	2314	2.5	0
10833	10833	2314	2.1	0
10834	10834	2314	0	0
10835	10835	2314	0	0
10836	10836	2314	0	0
10837	10837	2314	65.5	0
10838	10838	2314	200	0
10839	10839	2314	200	0
10840	10840	2314	10	0
10841	10841	2314	40	0
10842	10842	2314	70	0
10843	10843	2314	0	0
10844	10844	2314	1.67	0
10845	10845	2314	17.98333333	0
10846	10846	2314	5.983333333	0
10847	10847	2314	100	0
10848	10848	2314	125	0
10849	10849	2314	2.32	0
10850	10850	2314	78	0
10851	10851	2314	31.5	0
10852	10852	2314	3.46	0
10853	10853	2314	0	0
10854	10854	2314	0	0
10855	10855	2314	75	0
10856	10856	2314	0	0
10857	10857	2314	0	0
10858	10858	2314	129	0
10859	10859	2314	28	0
10860	10860	2314	25	0
10861	10861	2314	100	0
10862	10862	2314	125	0
10863	10863	2314	2.32	0
10864	10864	2314	78	0
10865	10865	2314	31.5	0
10866	10866	2314	3.46	0
10867	10867	2314	0	0
10868	10868	2314	0	0
10869	10869	2314	75	0
10870	10870	2314	0	0
10871	10871	2314	0	0
10872	10872	2314	71.5	0
10873	10873	2314	30.4	0
10874	10874	2314	101	0
10875	10875	2314	72.3	0
10876	10876	2314	7.5	0
10877	10877	2314	7.3	0
10878	10878	2314	10.57	0
10879	10879	2314	1.52	0
10880	10880	2314	10.6004	0
10881	10881	2314	1.5996	0
10882	10882	2314	30	0
10883	10883	2314	4	0
10884	10884	2314	2	0
10885	10885	2314	3	0
10886	10886	2314	0.5	0
10887	10887	2314	0.7	0
10888	10888	2314	1.1	0
10889	10889	2314	1	0
10890	10890	2314	4	0
10891	10891	2314	0	0
10892	10892	2314	1	0
10893	10893	2314	0.4	0
10894	10894	2314	0.5	0
10895	10895	2314	0.8	0
10896	10896	2314	0.5	0
10897	10897	2314	0.6	0
10898	10898	2314	0.6	0
10899	10899	2314	0.8	0
10900	10900	2314	1	0
10901	10901	2314	18	0
10902	10902	2314	5	0
10903	10903	2314	1	0
10904	10904	2314	8	0
10905	10905	2314	3	0
10906	10906	2314	3	0
10907	10907	2314	2	0
10908	10908	2314	1	0
10909	10909	2314	1	0
10910	10910	2314	0	0
10911	10911	2314	0	0
10912	10912	2314	0	0
10913	10913	2314	0	0
10914	10914	2314	0	0
10915	10915	2314	0	0
10916	10916	2314	B0L6	0
10917	10917	2314	1	0
10918	10918	2314	1.3	0
10919	10919	2314	26.5	0
10920	10920	2314	16	0
10921	10921	2314	37.6	0
10922	10922	2314	43	0
10923	10923	2314	6.5	0
10924	10924	2314	155	0
10925	10925	2314	2.6	0
10926	10926	2314	2.3	0
10927	10927	2314	29	0
10928	10928	2314	9	0
10929	10929	2314	8	0
10930	10930	2314	71.5	0
10931	10931	2314	125	0
10932	10932	2314	125	0
10933	10933	2314	15	0
10934	10934	2314	30	0
10935	10935	2314	85	0
10936	10936	2314	0	0
10937	10937	2314	2.24	0
10938	10938	2314	31.71666667	0
10939	10939	2314	7.666666667	0
10940	10940	2314	125	0
10941	10941	2314	125	0
10942	10942	2314	2.21	0
10943	10943	2314	105	0
10944	10944	2314	44	0
10945	10945	2314	3.56	0
10946	10946	2314	0	0
10947	10947	2314	0	0
10948	10948	2314	90	0
10949	10949	2314	0	0
10950	10950	2314	0	0
10951	10951	2314	89	0
10952	10952	2314	71	0
10953	10953	2314	44	0
10954	10954	2314	125	0
10955	10955	2314	125	0
10956	10956	2314	2.21	0
10957	10957	2314	105	0
10958	10958	2314	44	0
10959	10959	2314	3.56	0
10960	10960	2314	0	0
10961	10961	2314	0	0
10962	10962	2314	90	0
10963	10963	2314	0	0
10964	10964	2314	0	0
10965	10965	2314	78	0
10966	10966	2314	23	0
10967	10967	2314	157.2	0
10968	10968	2314	60.6	0
10969	10969	2314	4.7	0
10970	10970	2314	4.1	0
10971	10971	2314	10.78	0
10972	10972	2314	1.27	0
10973	10973	2314	10.803	0
10974	10974	2314	1.3347	0
10975	10975	2314	21	0
10976	10976	2314	2	0
10977	10977	2314	2	0
10978	10978	2314	3	0
10979	10979	2314	0.6	0
10980	10980	2314	1	0
10981	10981	2314	1.2	0
10982	10982	2314	1.1	0
10983	10983	2314	4	0
10984	10984	2314	0	0
10985	10985	2314	1	0
10986	10986	2314	0.4	0
10987	10987	2314	0.7	0
10988	10988	2314	1.1	0
10989	10989	2314	0.6	0
10990	10990	2314	0.8	0
10991	10991	2314	0.7	0
10992	10992	2314	1.1	0
10993	10993	2314	2	0
10994	10994	2314	22	0
10995	10995	2314	5	0
10996	10996	2314	1	0
10997	10997	2314	8	0
10998	10998	2314	4	0
10999	10999	2314	4	0
11000	11000	2314	3	0
11001	11001	2314	2	0
11002	11002	2314	0	0
11003	11003	2314	0	0
11004	11004	2314	1	0
11005	11005	2314	0	0
11006	11006	2314	0	0
11007	11007	2314	0	0
11008	11008	2314	0	0
11009	11009	2314	B0L7	0
11010	11010	2314	1	0
11011	11011	2314	1.2	0
11012	11012	2314	22.7	0
11013	11013	2314	12	0
11014	11014	2314	39.5	0
11015	11015	2314	31	0
11016	11016	2314	16.8	0
11017	11017	2314	221	0
11018	11018	2314	2.9	0
11019	11019	2314	2.1	0
11020	11020	2314	16	0
11021	11021	2314	0	0
11022	11022	2314	0	0
11023	11023	2314	66	0
11024	11024	2314	175	0
11025	11025	2314	175	0
11026	11026	2314	15	0
11027	11027	2314	15	0
11028	11028	2314	80	0
11029	11029	2314	0	0
11030	11030	2314	1.578333333	0
11031	11031	2314	18.21666667	0
11032	11032	2314	3.933333333	0
11033	11033	2314	175	0
11034	11034	2314	175	0
11035	11035	2314	2.2	0
11036	11036	2314	75	0
11037	11037	2314	44	0
11038	11038	2314	3.81	0
11039	11039	2314	0	0
11040	11040	2314	0	0
11041	11041	2314	80	0
11042	11042	2314	0	0
11043	11043	2314	0	0
11044	11044	2314	157	0
11045	11045	2314	68	0
11046	11046	2314	16	0
11047	11047	2314	175	0
11048	11048	2314	175	0
11049	11049	2314	2.2	0
11050	11050	2314	75	0
11051	11051	2314	44	0
11052	11052	2314	3.81	0
11053	11053	2314	0	0
11054	11054	2314	0	0
11055	11055	2314	80	0
11056	11056	2314	0	0
11057	11057	2314	0	0
11058	11058	2314	71.1	0
11059	11059	2314	23.4	0
11060	11060	2314	222.8	0
11061	11061	2314	90	0
11062	11062	2314	9.2	0
11063	11063	2314	8.8	0
11064	11064	2314	10.55	0
11065	11065	2314	1.71	0
11066	11066	2314	10.5734	0
11067	11067	2314	1.8088	0
11068	11068	2314	6	0
11069	11069	2314	2	0
11070	11070	2314	0	0
11071	11071	2314	1	0
11072	11072	2314	0.5	0
11073	11073	2314	0.4	0
11074	11074	2314		0
11075	11075	2314		0
11076	11076	2314	2	0
11077	11077	2314	0	0
11078	11078	2314	0.8	0
11079	11079	2314	0.3	0
11080	11080	2314	0.5	0
11081	11081	2314	0.6	0
11082	11082	2314		0
11083	11083	2314		0
11084	11084	2314		0
11085	11085	2314		0
11086	11086	2314	2	0
11087	11087	2314	9	0
11088	11088	2314	2	0
11089	11089	2314	1	0
11090	11090	2314	6	0
11091	11091	2314	2	0
11092	11092	2314	1	0
11093	11093	2314	0	0
11094	11094	2314	0	0
11095	11095	2314	0	0
11096	11096	2314	0	0
11097	11097	2314	0	0
11098	11098	2314	0	0
11099	11099	2314	0	0
11100	11100	2314	0	0
11101	11101	2314	0	0
11102	11102	2314	B0L4	0
11103	11103	2314	1.3	0
11104	11104	2314	1.3	0
11105	11105	2314	44.3	0
11106	11106	2314	14	0
11107	11107	2314	48	0
11108	11108	2314	26	0
11109	11109	2314	10.2	0
11110	11110	2314	138	0
11111	11111	2314	3.3	0
11112	11112	2314	2.5	0
11113	11113	2314	5	0
11114	11114	2314	11	0
11115	11115	2314	0	0
11116	11116	2314	79	0
11117	11117	2314	20	0
11118	11118	2314	20	0
11119	11119	2314	0	0
11120	11120	2314		0
11121	11121	2314	30	0
11122	11122	2314	0	0
11123	11123	2314	1.251666667	0
11124	11124	2314	15.53333333	0
11125	11125	2314	3.7	0
11126	11126	2314	30	0
11127	11127	2314	175	0
11128	11128	2314	2.79	0
11129	11129	2314	95	0
11130	11130	2314	33	0
11131	11131	2314	1.9	0
11132	11132	2314	0	0
11133	11133	2314	0	0
11134	11134	2314	20	0
11135	11135	2314	0	0
11136	11136	2314	0	0
11137	11137	2314	24	0
11138	11138	2314	3	0
11139	11139	2314	1	0
11140	11140	2314	30	0
11141	11141	2314	175	0
11142	11142	2314	2.79	0
11143	11143	2314	95	0
11144	11144	2314	33	0
11145	11145	2314	1.9	0
11146	11146	2314	0	0
11147	11147	2314	0	0
11148	11148	2314	20	0
11149	11149	2314	0	0
11150	11150	2314	0	0
11151	11151	2314	83.8	0
11152	11152	2314	17.8	0
11153	11153	2314	51.6	0
11154	11154	2314	16.7	0
11155	11155	2314	4.7	0
11156	11156	2314	2.7	0
11157	11157	2314	3.13	0
11158	11158	2314	0.22	0
11159	11159	2314	3.1478	0
11160	11160	2314	0.2394	0
11161	11161	2314	27	0
11162	11162	2314	27	0
11163	11163	2314	1	0
11164	11164	2314	2	0
11165	11165	2314	0.5	0
11166	11166	2314	0.8	0
11167	11167	2314	0.7	0
11168	11168	2314	0.8	0
11169	11169	2314	4	0
11170	11170	2314	0	0
11171	11171	2314	0.8	0
11172	11172	2314	0.3	0
11173	11173	2314	0.6	0
11174	11174	2314	1	0
11175	11175	2314	0.6	0
11176	11176	2314	1	0
11177	11177	2314	0.6	0
11178	11178	2314	1	0
11179	11179	2314	0	0
11180	11180	2314	18	0
11181	11181	2314	4	0
11182	11182	2314	1	0
11183	11183	2314	8	0
11184	11184	2314	4	0
11185	11185	2314	3	0
11186	11186	2314	2	0
11187	11187	2314	1	0
11188	11188	2314	0	0
11189	11189	2314	0	0
11190	11190	2314	0	0
11191	11191	2314	0	0
11192	11192	2314	0	0
11193	11193	2314	0	0
11194	11194	2314	0	0
11195	11195	2314	B0L6	0
11196	11196	2314	1.2	0
11197	11197	2314	1.45	0
11198	11198	2314	24.2	0
11199	11199	2314	13	0
11200	11200	2314	39.2	0
11201	11201	2314	32	0
11202	11202	2314	11.9	0
11203	11203	2314	158	0
11204	11204	2314	3	0
11205	11205	2314	2.5	0
11206	11206	2314	0	0
11207	11207	2314	0	0
11208	11208	2314	0	0
11209	11209	2314	68.5	0
11210	11210	2314	125	0
11211	11211	2314	125	0
11212	11212	2314	15	0
11213	11213	2314	25	0
11214	11214	2314	75	0
11215	11215	2314	0	0
11216	11216	2314	1.41	0
11217	11217	2314	19.28333333	0
11218	11218	2314	4.466666667	0
11219	11219	2314	100	0
11220	11220	2314	300	0
11221	11221	2314	2.43	0
11222	11222	2314	67.5	0
11223	11223	2314	59	0
11224	11224	2314	2.12	0
11225	11225	2314	0	0
11226	11226	2314	0	0
11227	11227	2314	30	0
11228	11228	2314	0	0
11229	11229	2314	0	0
11230	11230	2314	129	0
11231	11231	2314	12	0
11232	11232	2314	12	0
11233	11233	2314	100	0
11234	11234	2314	300	0
11235	11235	2314	2.43	0
11236	11236	2314	67.5	0
11237	11237	2314	59	0
11238	11238	2314	2.12	0
11239	11239	2314	0	0
11240	11240	2314	0	0
11241	11241	2314	30	0
11242	11242	2314	0	0
11243	11243	2314	0	0
11244	11244	2314	96.2	0
11245	11245	2314	30.8	0
11246	11246	2314	193.1	0
11247	11247	2314	67.8	0
11248	11248	2314	38.9	0
11249	11249	2314	19.4	0
11250	11250	2314	10.4	0
11251	11251	2314	1.65	0
11252	11252	2314	10.4308	0
11253	11253	2314	1.7372	0
11254	11254	2314	4	0
11255	11255	2314		0
11256	11256	2314	0	0
11257	11257	2314	2	0
11258	11258	2314	0.5	0
11259	11259	2314	0.6	0
11260	11260	2314		0
11261	11261	2314		0
11262	11262	2314	2	0
11263	11263	2314	0	0
11264	11264	2314	0.9	0
11265	11265	2314	0.4	0
11266	11266	2314	0.6	0
11267	11267	2314	0.6	0
11268	11268	2314	0.4	0
11269	11269	2314	0.6	0
11270	11270	2314		0
11271	11271	2314		0
11272	11272	2314	1	0
11273	11273	2314	12	0
11274	11274	2314	3	0
11275	11275	2314	1	0
11276	11276	2314	7	0
11277	11277	2314	1	0
11278	11278	2314	3	0
11279	11279	2314	1	0
11280	11280	2314	0	0
11281	11281	2314	0	0
11282	11282	2314	0	0
11283	11283	2314	0	0
11284	11284	2314	0	0
11285	11285	2314	0	0
11286	11286	2314	0	0
11287	11287	2314	0	0
11288	11288	2314	B0L5	0
11289	11289	2314	0.9	0
11290	11290	2314	1.1	0
11291	11291	2314	21.2	0
11292	11292	2314	11	0
11293	11293	2314	31.1	0
11294	11294	2314	17	0
11295	11295	2314	1.4	0
11296	11296	2314	79	0
11297	11297	2314	2.1	0
11298	11298	2314	2.1	0
11299	11299	2314	0	0
11300	11300	2314	0	0
11301	11301	2314	0	0
11302	11302	2314	45	0
11303	11303	2314	200	0
11304	11304	2314	200	0
11305	11305	2314	0	0
11306	11306	2314		0
11307	11307	2314	95	0
11308	11308	2314	0	0
11309	11309	2314	1.683333333	0
11310	11310	2314	24.96666667	0
11311	11311	2314	5.766666667	0
11312	11312	2314	500	0
11313	11313	2314	575	0
11314	11314	2314	2.51	0
11315	11315	2314	6	0
11316	11316	2314	35.5	0
11317	11317	2314	2.68	0
11318	11318	2314	0	0
11319	11319	2314	0	0
11320	11320	2314	95	0
11321	11321	2314	5	0
11322	11322	2314	75	0
11323	11323	2314	0	0
11324	11324	2314	0	0
11325	11325	2314	7	0
11326	11326	2314	500	0
11327	11327	2314	575	0
11328	11328	2314	2.51	0
11329	11329	2314	6	0
11330	11330	2314	35.5	0
11331	11331	2314	2.68	0
11332	11332	2314	0	0
11333	11333	2314	0	0
11334	11334	2314	95	0
11335	11335	2314	5	0
11336	11336	2314	75	0
11337	11337	2314	64.9	0
11338	11338	2314	18.5	0
11339	11339	2314	93.5	0
11340	11340	2314	38.5	0
11341	11341	2314	7.3	0
11342	11342	2314	6	0
11343	11343	2314	15.65	0
11344	11344	2314	2.65	0
11345	11345	2314	15.6685	0
11346	11346	2314	2.6945	0
11347	11347	2314	11	0
11348	11348	2314	19	0
11349	11349	2314	1	0
11350	11350	2314	2	0
11351	11351	2314	0.4	0
11352	11352	2314	0.5	0
11353	11353	2314		0
11354	11354	2314		0
11355	11355	2314	3	0
11356	11356	2314	0	0
11357	11357	2314	0.8	0
11358	11358	2314	0.4	0
11359	11359	2314	0.6	0
11360	11360	2314	0.6	0
11361	11361	2314	0.5	0
11362	11362	2314	0.7	0
11363	11363	2314		0
11364	11364	2314		0
11365	11365	2314	1	0
11366	11366	2314	11	0
11367	11367	2314	2	0
11368	11368	2314	2	0
11369	11369	2314	7	0
11370	11370	2314	3	0
11371	11371	2314	1	0
11372	11372	2314	0	0
11373	11373	2314	0	0
11374	11374	2314	0	0
11375	11375	2314	0	0
11376	11376	2314	0	0
11377	11377	2314	0	0
11378	11378	2314	0	0
11379	11379	2314	0	0
11380	11380	2314	0	0
11381	11381	2314	B0L5	0
11382	11382	2314	1	0
11383	11383	2314	1.3	0
11384	11384	2314	22.7	0
11385	11385	2314	8	0
11386	11386	2314	27.8	0
11387	11387	2314	8	0
11388	11388	2314	2.6	0
11389	11389	2314	43	0
11390	11390	2314	2.2	0
11391	11391	2314	2.2	0
11392	11392	2314	0	0
11393	11393	2314	0	0
11394	11394	2314	0	0
11395	11395	2314	73	0
11396	11396	2314	175	0
11397	11397	2314	175	0
11398	11398	2314	20	0
11399	11399	2314	20	0
11400	11400	2314	75	0
11401	11401	2314	0	0
11402	11402	2314	1.58	0
11403	11403	2314	19.48333333	0
11404	11404	2314	4.216666667	0
11405	11405	2314	5	0
11406	11406	2314	200	0
11407	11407	2314	2.09	0
11408	11408	2314	63	0
11409	11409	2314	29.5	0
11410	11410	2314	3.3	0
11411	11411	2314	0	0
11412	11412	2314	0	0
11413	11413	2314	5	0
11414	11414	2314	0	0
11415	11415	2314	0	0
11416	11416	2314	104	0
11417	11417	2314	0	0
11418	11418	2314	0	0
11419	11419	2314	5	0
11420	11420	2314	200	0
11421	11421	2314	2.09	0
11422	11422	2314	63	0
11423	11423	2314	29.5	0
11424	11424	2314	3.3	0
11425	11425	2314	0	0
11426	11426	2314	0	0
11427	11427	2314	5	0
11428	11428	2314	0	0
11429	11429	2314	0	0
11430	11430	2314	93.3	0
11431	11431	2314	35.9	0
11432	11432	2314	116.9	0
11433	11433	2314	45.4	0
11434	11434	2314	40.3	0
11435	11435	2314	16.4	0
11436	11436	2314	7.18	0
11437	11437	2314	1.94	0
11438	11438	2314	7.2159	0
11439	11439	2314	2.0018	0
11440	11440	2314	8	0
11441	11441	2314	2	0
11442	11442	2314	1	0
11443	11443	2314	2	0
11444	11444	2314	0.3	0
11445	11445	2314	0.5	0
11446	11446	2314		0
11447	11447	2314		0
11448	11448	2314	3	0
11449	11449	2314	0	0
11450	11450	2314	1	0
11451	11451	2314	0.4	0
11452	11452	2314	0.3	0
11453	11453	2314	0.5	0
11454	11454	2314	0.5	0
11455	11455	2314	0.6	0
11456	11456	2314		0
11457	11457	2314		0
11458	11458	2314	2	0
11459	11459	2314	17	0
11460	11460	2314	4	0
11461	11461	2314	1	0
11462	11462	2314	7	0
11463	11463	2314	4	0
11464	11464	2314	3	0
11465	11465	2314	2	0
11466	11466	2314	1	0
11467	11467	2314	0	0
11468	11468	2314	0	0
11469	11469	2314	0	0
11470	11470	2314	0	0
11471	11471	2314	0	0
11472	11472	2314	0	0
11473	11473	2314	0	0
11474	11474	2314	B0L5	0
11475	11475	2314	0.9	0
11476	11476	2314	1	0
11477	11477	2314	8.8	0
11478	11478	2314	6	0
11479	11479	2314	18.7	0
11480	11480	2314	23	0
11481	11481	2314	6	0
11482	11482	2314	101	0
11483	11483	2314	1.9	0
11484	11484	2314	1.8	0
11485	11485	2314	0	0
11486	11486	2314	0	0
11487	11487	2314	0	0
11488	11488	2314	38	0
11489	11489	2314	50	0
11490	11490	2314	50	0
11491	11491	2314	4	0
11492	11492	2314	5	0
11493	11493	2314	95	0
11494	11494	2314	0	0
11495	11495	2314	1.678333333	0
11496	11496	2314	23.25	0
11497	11497	2314	6.2	0
11498	11498	2314	250	0
11499	11499	2314	250	0
11500	11500	2314	2.18	0
11501	11501	2314	83	0
11502	11502	2314	15	0
11503	11503	2314	2.29	0
11504	11504	2314	0	0
11505	11505	2314	0	0
11506	11506	2314	100	0
11507	11507	2314	0	0
11508	11508	2314	0	0
11509	11509	2314	60	0
11510	11510	2314	117	0
11511	11511	2314	45	0
11512	11512	2314	250	0
11513	11513	2314	250	0
11514	11514	2314	2.18	0
11515	11515	2314	83	0
11516	11516	2314	15	0
11517	11517	2314	2.29	0
11518	11518	2314	0	0
11519	11519	2314	0	0
11520	11520	2314	100	0
11521	11521	2314	0	0
11522	11522	2314	0	0
11523	11523	2314	62.1	0
11524	11524	2314	21.1	0
11525	11525	2314	103.1	0
11526	11526	2314	40.6	0
11527	11527	2314	9.7	0
11528	11528	2314	8.5	0
11529	11529	2314	8.09	0
11530	11530	2314	2.63	0
11531	11531	2314	8.1111	0
11532	11532	2314	2.6791	0
11533	11533	2314	.	0
11534	11534	2314	.	0
11535	11535	2314	1	0
11536	11536	2314	2	0
11537	11537	2314	0.2	0
11538	11538	2314	0.3	0
11539	11539	2314		0
11540	11540	2314		0
11541	11541	2314	0	0
11542	11542	2314	0	0
11543	11543	2314		0
11544	11544	2314		0
11545	11545	2314		0
11546	11546	2314		0
11547	11547	2314		0
11548	11548	2314		0
11549	11549	2314		0
11550	11550	2314		0
11551	11551	2314	0	0
11552	11552	2314	0	0
11553	11553	2314	0	0
11554	11554	2314	.	0
11555	11555	2314	0	0
11556	11556	2314	0	0
11557	11557	2314	0	0
11558	11558	2314	0	0
11559	11559	2314	0	0
11560	11560	2314	0	0
11561	11561	2314	0	0
11562	11562	2314	0	0
11563	11563	2314	0	0
11564	11564	2314	0	0
11565	11565	2314	0	0
11566	11566	2314	0	0
11567	11567	2314		0
11568	11568	2314		0
11569	11569	2314		0
11570	11570	2314	.	0
11571	11571	2314	0	0
11572	11572	2314	.	0
11573	11573	2314	0	0
11574	11574	2314	.	0
11575	11575	2314	0	0
11576	11576	2314	.	0
11577	11577	2314	.	0
11578	11578	2314	0	0
11579	11579	2314	0	0
11580	11580	2314	0	0
11581	11581	2314		0
11582	11582	2314		0
11583	11583	2314	0	0
11584	11584	2314		0
11585	11585	2314		0
11586	11586	2314		0
11587	11587	2314		0
11588	11588	2314		0
11589	11589	2314		0
11590	11590	2314		0
11591	11591	2314		0
11592	11592	2314		0
11593	11593	2314		0
11594	11594	2314		0
11595	11595	2314		0
11596	11596	2314		0
11597	11597	2314		0
11598	11598	2314		0
11599	11599	2314		0
11600	11600	2314		0
11601	11601	2314		0
11602	11602	2314		0
11603	11603	2314		0
11604	11604	2314		0
11605	11605	2314	.	0
11606	11606	2314	.	0
11607	11607	2314	.	0
11608	11608	2314	.	0
11609	11609	2314	.	0
11610	11610	2314	.	0
11611	11611	2314	.	0
11612	11612	2314	.	0
11613	11613	2314	.	0
11614	11614	2314	.	0
11615	11615	2314	.	0
11616	11616	2314	.	0
11617	11617	2314	.	0
11618	11618	2314	.	0
11619	11619	2314	.	0
11620	11620	2314	.	0
11621	11621	2314	.	0
11622	11622	2314		0
11623	11623	2314		0
11624	11624	2314	.	0
11625	11625	2314	.	0
11626	11626	2314	16	0
11627	11627	2314	13	0
11628	11628	2314	1	0
11629	11629	2314	2	0
11630	11630	2314	0.6	0
11631	11631	2314	0.9	0
11632	11632	2314	0.8	0
11633	11633	2314	1	0
11634	11634	2314	3	0
11635	11635	2314	0	0
11636	11636	2314	0.6	0
11637	11637	2314	0.3	0
11638	11638	2314	0.7	0
11639	11639	2314	1	0
11640	11640	2314	0.8	0
11641	11641	2314	0.9	0
11642	11642	2314	0.7	0
11643	11643	2314	1	0
11644	11644	2314	1	0
11645	11645	2314	20	0
11646	11646	2314	4	0
11647	11647	2314	1	0
11648	11648	2314	8	0
11649	11649	2314	4	0
11650	11650	2314	3	0
11651	11651	2314	3	0
11652	11652	2314	2	0
11653	11653	2314	0	0
11654	11654	2314	0	0
11655	11655	2314	0	0
11656	11656	2314	0	0
11657	11657	2314	0	0
11658	11658	2314	0	0
11659	11659	2314	0	0
11660	11660	2314	B0L6	0
11661	11661	2314	1	0
11662	11662	2314	1.3	0
11663	11663	2314	17.1	0
11664	11664	2314	7	0
11665	11665	2314	32.8	0
11666	11666	2314	31	0
11667	11667	2314	10.3	0
11668	11668	2314	149	0
11669	11669	2314	2.6	0
11670	11670	2314	2.4	0
11671	11671	2314	0	0
11672	11672	2314	0	0
11673	11673	2314	0	0
11674	11674	2314	59	0
11675	11675	2314	250	0
11676	11676	2314	250	0
11677	11677	2314	10	0
11678	11678	2314	40	0
11679	11679	2314	95	0
11680	11680	2314	0	0
11681	11681	2314	2.206666667	0
11682	11682	2314	28.05	0
11683	11683	2314	5.716666667	0
11684	11684	2314	350	0
11685	11685	2314	500	0
11686	11686	2314	2.88	0
11687	11687	2314	67.5	0
11688	11688	2314	56.5	0
11689	11689	2314	4.02	0
11690	11690	2314	0	0
11691	11691	2314	0	0
11692	11692	2314	60	0
11693	11693	2314	0	0
11694	11694	2314	0	0
11695	11695	2314	131	0
11696	11696	2314	105	0
11697	11697	2314	17	0
11698	11698	2314	350	0
11699	11699	2314	500	0
11700	11700	2314	2.88	0
11701	11701	2314	67.5	0
11702	11702	2314	56.5	0
11703	11703	2314	4.02	0
11704	11704	2314	0	0
11705	11705	2314	0	0
11706	11706	2314	60	0
11707	11707	2314	0	0
11708	11708	2314	0	0
11709	11709	2314	65	0
11710	11710	2314	15.4	0
11711	11711	2314	142	0
11712	11712	2314	46.1	0
11713	11713	2314	19.4	0
11714	11714	2314	10.4	0
11715	11715	2314	15.09	0
11716	11716	2314	5.44	0
11717	11717	2314	15.1054	0
11718	11718	2314	5.4965	0
11719	11719	2314	3	0
11720	11720	2314	1	0
11721	11721	2314	1	0
11722	11722	2314	3	0
11723	11723	2314	0.6	0
11724	11724	2314	1.1	0
11725	11725	2314	1	0
11726	11726	2314	1.1	0
11727	11727	2314	5	0
11728	11728	2314	0	0
11729	11729	2314	1.2	0
11730	11730	2314	0.5	0
11731	11731	2314	0.6	0
11732	11732	2314	1.2	0
11733	11733	2314	0.6	0
11734	11734	2314	0.8	0
11735	11735	2314	0.8	0
11736	11736	2314	1	0
11737	11737	2314	2	0
11738	11738	2314	32	0
11739	11739	2314	9	0
11740	11740	2314	1	0
11741	11741	2314	9	0
11742	11742	2314	6	0
11743	11743	2314	5	0
11744	11744	2314	4	0
11745	11745	2314	3	0
11746	11746	2314	1	0
11747	11747	2314	0	0
11748	11748	2314	1	0
11749	11749	2314	1	0
11750	11750	2314	1	0
11751	11751	2314	1	0
11752	11752	2314	0	0
11753	11753	2314	B0L7	0
11754	11754	2314	1.3	0
11755	11755	2314	1.5	0
11756	11756	2314	14.3	0
11757	11757	2314	13	0
11758	11758	2314	51.2	0
11759	11759	2314	60	0
11760	11760	2314	19.9	0
11761	11761	2314	309	0
11762	11762	2314	2.8	0
11763	11763	2314	2.1	0
11764	11764	2314	10	0
11765	11765	2314	18	0
11766	11766	2314	0	0
11767	11767	2314	80	0
11768	11768	2314	225	0
11769	11769	2314	225	0
11770	11770	2314	5	0
11771	11771	2314	30	0
11772	11772	2314	95	0
11773	11773	2314	0	0
11774	11774	2314	1.758333333	0
11775	11775	2314	21.08333333	0
11776	11776	2314	5.666666667	0
11777	11777	2314	250	0
11778	11778	2314	400	0
11779	11779	2314	2.26	0
11780	11780	2314	87.5	0
11781	11781	2314	46	0
11782	11782	2314	4.31	0
11783	11783	2314	6	0
11784	11784	2314	0	0
11785	11785	2314	55	0
11786	11786	2314	10	0
11787	11787	2314	2	0
11788	11788	2314	163	0
11789	11789	2314	79	0
11790	11790	2314	36	0
11791	11791	2314	250	0
11792	11792	2314	400	0
11793	11793	2314	2.26	0
11794	11794	2314	87.5	0
11795	11795	2314	46	0
11796	11796	2314	4.31	0
11797	11797	2314	6	0
11798	11798	2314	0	0
11799	11799	2314	55	0
11800	11800	2314	10	0
11801	11801	2314	2	0
11802	11802	2314	72.3	0
11803	11803	2314	22.2	0
11804	11804	2314	108.8	0
11805	11805	2314	42	0
11806	11806	2314	20.3	0
11807	11807	2314	11	0
11808	11808	2314	14.01	0
11809	11809	2314	2.65	0
11810	11810	2314	14.0322	0
11811	11811	2314	2.703	0
11812	11812	2314	4	0
11813	11813	2314	1	0
11814	11814	2314	1	0
11815	11815	2314	2	0
11816	11816	2314	0.3	0
11817	11817	2314	0.5	0
11818	11818	2314		0
11819	11819	2314		0
11820	11820	2314	3	0
11821	11821	2314	0	0
11822	11822	2314	0.8	0
11823	11823	2314	0.4	0
11824	11824	2314	0.4	0
11825	11825	2314	0.7	0
11826	11826	2314	0.5	0
11827	11827	2314	0.7	0
11828	11828	2314		0
11829	11829	2314		0
11830	11830	2314	0	0
11831	11831	2314	13	0
11832	11832	2314	3	0
11833	11833	2314	1	0
11834	11834	2314	6	0
11835	11835	2314	3	0
11836	11836	2314	3	0
11837	11837	2314	1	0
11838	11838	2314	0	0
11839	11839	2314	0	0
11840	11840	2314	0	0
11841	11841	2314	0	0
11842	11842	2314	0	0
11843	11843	2314	0	0
11844	11844	2314	0	0
11845	11845	2314	0	0
11846	11846	2314	B0L4	0
11847	11847	2314	1.1	0
11848	11848	2314	1.35	0
11849	11849	2314	26.5	0
11850	11850	2314	14	0
11851	11851	2314	39.7	0
11852	11852	2314	20	0
11853	11853	2314	8.6	0
11854	11854	2314	120	0
11855	11855	2314	2.3	0
11856	11856	2314	2.4	0
11857	11857	2314	0	0
11858	11858	2314	4	0
11859	11859	2314	0	0
11860	11860	2314	58	0
11861	11861	2314	150	0
11862	11862	2314	150	0
11863	11863	2314	5	0
11864	11864	2314	15	0
11865	11865	2314	90	0
11866	11866	2314	0	0
11867	11867	2314	2.443333333	0
11868	11868	2314	25.85	0
11869	11869	2314	5.55	0
11870	11870	2314	225	0
11871	11871	2314	250	0
11872	11872	2314	1.31	0
11873	11873	2314	93	0
11874	11874	2314	34	0
11875	11875	2314	1.45	0
11876	11876	2314	0	0
11877	11877	2314	0	0
11878	11878	2314	85	0
11879	11879	2314	0	0
11880	11880	2314	0	0
11881	11881	2314	65	0
11882	11882	2314	27	0
11883	11883	2314	3	0
11884	11884	2314	225	0
11885	11885	2314	250	0
11886	11886	2314	1.31	0
11887	11887	2314	93	0
11888	11888	2314	34	0
11889	11889	2314	1.45	0
11890	11890	2314	0	0
11891	11891	2314	0	0
11892	11892	2314	85	0
11893	11893	2314	0	0
11894	11894	2314	0	0
11895	11895	2314	61.2	0
11896	11896	2314	17.6	0
11897	11897	2314	37.2	0
11898	11898	2314	13.1	0
11899	11899	2314	8.2	0
11900	11900	2314	3.4	0
11901	11901	2314	7.14	0
11902	11902	2314	1.88	0
11903	11903	2314	7.1576	0
11904	11904	2314	1.8965	0
11905	11905	2314	.	0
11906	11906	2314	.	0
11907	11907	2314	1	0
11908	11908	2314	2	0
11909	11909	2314	.	0
11910	11910	2314	.	0
11911	11911	2314		0
11912	11912	2314		0
11913	11913	2314	0	0
11914	11914	2314	0	0
11915	11915	2314		0
11916	11916	2314		0
11917	11917	2314		0
11918	11918	2314		0
11919	11919	2314		0
11920	11920	2314		0
11921	11921	2314		0
11922	11922	2314		0
11923	11923	2314	0	0
11924	11924	2314	0	0
11925	11925	2314	0	0
11926	11926	2314	.	0
11927	11927	2314		0
11928	11928	2314	0	0
11929	11929	2314	0	0
11930	11930	2314	0	0
11931	11931	2314	0	0
11932	11932	2314	0	0
11933	11933	2314	0	0
11934	11934	2314	0	0
11935	11935	2314	0	0
11936	11936	2314	0	0
11937	11937	2314	0	0
11938	11938	2314	0	0
11939	11939	2314		0
11940	11940	2314		0
11941	11941	2314		0
11942	11942	2314	.	0
11943	11943	2314	0	0
11944	11944	2314	.	0
11945	11945	2314	0	0
11946	11946	2314	.	0
11947	11947	2314	0	0
11948	11948	2314	.	0
11949	11949	2314	.	0
11950	11950	2314	0	0
11951	11951	2314	0	0
11952	11952	2314	0	0
11953	11953	2314		0
11954	11954	2314		0
11955	11955	2314	0	0
11956	11956	2314		0
11957	11957	2314		0
11958	11958	2314		0
11959	11959	2314		0
11960	11960	2314		0
11961	11961	2314		0
11962	11962	2314		0
11963	11963	2314		0
11964	11964	2314		0
11965	11965	2314		0
11966	11966	2314		0
11967	11967	2314		0
11968	11968	2314		0
11969	11969	2314		0
11970	11970	2314		0
11971	11971	2314		0
11972	11972	2314		0
11973	11973	2314		0
11974	11974	2314		0
11975	11975	2314		0
11976	11976	2314		0
11977	11977	2314	.	0
11978	11978	2314	.	0
11979	11979	2314	.	0
11980	11980	2314	.	0
11981	11981	2314	.	0
11982	11982	2314	.	0
11983	11983	2314	.	0
11984	11984	2314	.	0
11985	11985	2314	.	0
11986	11986	2314	.	0
11987	11987	2314	.	0
11988	11988	2314	.	0
11989	11989	2314	.	0
11990	11990	2314	.	0
11991	11991	2314	.	0
11992	11992	2314	.	0
11993	11993	2314	.	0
11994	11994	2314		0
11995	11995	2314		0
11996	11996	2314	.	0
11997	11997	2314	.	0
11998	11998	2314	22	0
11999	11999	2314	27	0
12000	12000	2314	2	0
12001	12001	2314	2	0
12002	12002	2314	0.4	0
12003	12003	2314	0.4	0
12004	12004	2314	0.5	0
12005	12005	2314	0.5	0
12006	12006	2314	3	0
12007	12007	2314	0	0
12008	12008	2314	0.6	0
12009	12009	2314	0.3	0
12010	12010	2314	.	0
12011	12011	2314	.	0
12012	12012	2314	0.4	0
12013	12013	2314	0.5	0
12014	12014	2314	0.5	0
12015	12015	2314	0.6	0
12016	12016	2314	0	0
12017	12017	2314	7	0
12018	12018	2314	2	0
12019	12019	2314	2	0
12020	12020	2314	5	0
12021	12021	2314	1	0
12022	12022	2314	1	0
12023	12023	2314	0	0
12024	12024	2314	0	0
12025	12025	2314	0	0
12026	12026	2314	0	0
12027	12027	2314	0	0
12028	12028	2314	0	0
12029	12029	2314	0	0
12030	12030	2314	0	0
12031	12031	2314	0	0
12032	12032	2314	B0L4	0
12033	12033	2314	0.9	0
12034	12034	2314	1.2	0
12035	12035	2314	25.6	0
12036	12036	2314	12	0
12037	12037	2314	26.3	0
12038	12038	2314	17	0
12039	12039	2314	10.3	0
12040	12040	2314	110	0
12041	12041	2314	2.6	0
12042	12042	2314	2	0
12043	12043	2314	0	0
12044	12044	2314	0	0
12045	12045	2314	0	0
12046	12046	2314	91	0
12047	12047	2314	275	0
12048	12048	2314	275	0
12049	12049	2314	10	0
12050	12050	2314	25	0
12051	12051	2314	90	0
12052	12052	2314	0	0
12053	12053	2314	2.541666667	0
12054	12054	2314	34.71666667	0
12055	12055	2314	8.533333333	0
12056	12056	2314	250	0
12057	12057	2314	250	0
12058	12058	2314	2.91	0
12059	12059	2314	97	0
12060	12060	2314	43	0
12061	12061	2314	3.48	0
12062	12062	2314	0	0
12063	12063	2314	0	0
12064	12064	2314	80	0
12065	12065	2314	0	0
12066	12066	2314	0	0
12067	12067	2314	56	0
12068	12068	2314	64	0
12069	12069	2314	15	0
12070	12070	2314	250	0
12071	12071	2314	250	0
12072	12072	2314	2.91	0
12073	12073	2314	97	0
12074	12074	2314	43	0
12075	12075	2314	3.48	0
12076	12076	2314	0	0
12077	12077	2314	0	0
12078	12078	2314	80	0
12079	12079	2314	0	0
12080	12080	2314	0	0
12081	12081	2314	123	0
12082	12082	2314	36.1	0
12083	12083	2314	231.5	0
12084	12084	2314	70.1	0
12085	12085	2314	27.5	0
12086	12086	2314	9.3	0
12087	12087	2314	14.88	0
12088	12088	2314	4.85	0
12089	12089	2314	14.9161	0
12090	12090	2314	4.9294	0
12091	12091	2314	5	0
12092	12092	2314	6	0
12093	12093	2314	1	0
12094	12094	2314	2	0
12095	12095	2314	0.5	0
12096	12096	2314	0.6	0
12097	12097	2314		0
12098	12098	2314		0
12099	12099	2314	3	0
12100	12100	2314	0	0
12101	12101	2314	1	0
12102	12102	2314	0.5	0
12103	12103	2314	0.5	0
12104	12104	2314	0.7	0
12105	12105	2314	0.5	0
12106	12106	2314	0.7	0
12107	12107	2314		0
12108	12108	2314		0
12109	12109	2314	2	0
12110	12110	2314	7	0
12111	12111	2314	1	0
12112	12112	2314	2	0
12113	12113	2314	6	0
12114	12114	2314	1	0
12115	12115	2314	0	0
12116	12116	2314	0	0
12117	12117	2314	0	0
12118	12118	2314	0	0
12119	12119	2314	0	0
12120	12120	2314	0	0
12121	12121	2314	0	0
12122	12122	2314	0	0
12123	12123	2314	0	0
12124	12124	2314	0	0
12125	12125	2314	B0L5	0
12126	12126	2314	1.2	0
12127	12127	2314	1.5	0
12128	12128	2314	23.4	0
12129	12129	2314	10	0
12130	12130	2314	24.3	0
12131	12131	2314	16	0
12132	12132	2314	7.3	0
12133	12133	2314	88	0
12134	12134	2314	2.8	0
12135	12135	2314	2.5	0
12136	12136	2314	0	0
12137	12137	2314	0	0
12138	12138	2314	0	0
12139	12139	2314	54	0
12140	12140	2314	225	0
12141	12141	2314	225	0
12142	12142	2314	5	0
12143	12143	2314	30	0
12144	12144	2314	90	0
12145	12145	2314	0	0
12146	12146	2314	3.495	0
12147	12147	2314	51.06666667	0
12148	12148	2314	9.566666667	0
12149	12149	2314	250	0
12150	12150	2314	325	0
12151	12151	2314	2.08	0
12152	12152	2314	65	0
12153	12153	2314	27	0
12154	12154	2314	3.8	0
12155	12155	2314	0	0
12156	12156	2314	0	0
12157	12157	2314	55	0
12158	12158	2314	20	0
12159	12159	2314	0	0
12160	12160	2314	43	0
12161	12161	2314	64	0
12162	12162	2314	15	0
12163	12163	2314	250	0
12164	12164	2314	325	0
12165	12165	2314	2.08	0
12166	12166	2314	65	0
12167	12167	2314	27	0
12168	12168	2314	3.8	0
12169	12169	2314	0	0
12170	12170	2314	0	0
12171	12171	2314	55	0
12172	12172	2314	20	0
12173	12173	2314	0	0
12174	12174	2314	84.1	0
12175	12175	2314	23.9	0
12176	12176	2314	66.8	0
12177	12177	2314	21.8	0
12178	12178	2314	12.5	0
12179	12179	2314	5.6	0
12180	12180	2314	9.66	0
12181	12181	2314	3.64	0
12182	12182	2314	9.6839	0
12183	12183	2314	3.6674	0
12184	12184	2314	17	0
12185	12185	2314	14	0
12186	12186	2314	1	0
12187	12187	2314	2	0
12188	12188	2314	0.4	0
12189	12189	2314	0.5	0
12190	12190	2314	0.8	0
12191	12191	2314	0.6	0
12192	12192	2314	3	0
12193	12193	2314	0	0
12194	12194	2314	1.1	0
12195	12195	2314	0.7	0
12196	12196	2314	0.5	0
12197	12197	2314	0.6	0
12198	12198	2314	0.5	0
12199	12199	2314	0.7	0
12200	12200	2314		0
12201	12201	2314		0
12202	12202	2314	2	0
12203	12203	2314	13	0
12204	12204	2314	3	0
12205	12205	2314	1	0
12206	12206	2314	7	0
12207	12207	2314	2	0
12208	12208	2314	3	0
12209	12209	2314	1	0
12210	12210	2314	0	0
12211	12211	2314	0	0
12212	12212	2314	0	0
12213	12213	2314	0	0
12214	12214	2314	0	0
12215	12215	2314	0	0
12216	12216	2314	0	0
12217	12217	2314	0	0
12218	12218	2314	B0L5	0
12219	12219	2314	1.3	0
12220	12220	2314	1.5	0
12221	12221	2314	25.6	0
12222	12222	2314	12	0
12223	12223	2314	33.3	0
12224	12224	2314	16	0
12225	12225	2314	8.9	0
12226	12226	2314	90	0
12227	12227	2314	2.5	0
12228	12228	2314	2.2	0
12229	12229	2314	0	0
12230	12230	2314	0	0
12231	12231	2314	0	0
12232	12232	2314	76	0
12233	12233	2314	200	0
12234	12234	2314	200	0
12235	12235	2314	30	0
12236	12236	2314	15	0
12237	12237	2314	95	0
12238	12238	2314	0	0
12239	12239	2314	3.276666667	0
12240	12240	2314	52.15	0
12241	12241	2314	11.58333333	0
12242	12242	2314	200	0
12243	12243	2314	250	0
12244	12244	2314	3.35	0
12245	12245	2314	82	0
12246	12246	2314	37	0
12247	12247	2314	2.61	0
12248	12248	2314	0	0
12249	12249	2314	0	0
12250	12250	2314	90	0
12251	12251	2314	0	0
12252	12252	2314	0	0
12253	12253	2314	39	0
12254	12254	2314	59	0
12255	12255	2314	62	0
12256	12256	2314	200	0
12257	12257	2314	250	0
12258	12258	2314	3.35	0
12259	12259	2314	82	0
12260	12260	2314	37	0
12261	12261	2314	2.61	0
12262	12262	2314	0	0
12263	12263	2314	0	0
12264	12264	2314	90	0
12265	12265	2314	0	0
12266	12266	2314	0	0
12267	12267	2314	47.8	0
12268	12268	2314	20.9	0
12269	12269	2314	171.2	0
12270	12270	2314	62.4	0
12271	12271	2314	25.9	0
12272	12272	2314	11.3	0
12273	12273	2314	18.96	0
12274	12274	2314	2.98	0
12275	12275	2314	18.9809	0
12276	12276	2314	3.0537	0
12277	12277	2314	22	0
12278	12278	2314	11	0
12279	12279	2314	2	0
12280	12280	2314	3	0
12281	12281	2314	0.3	0
12282	12282	2314	0.6	0
12283	12283	2314	0.9	0
12284	12284	2314	1	0
12285	12285	2314	4	0
12286	12286	2314	0	0
12287	12287	2314	1.3	0
12288	12288	2314	0.5	0
12289	12289	2314	0.4	0
12290	12290	2314	0.7	0
12291	12291	2314	0.7	0
12292	12292	2314	0.6	0
12293	12293	2314	0.7	0
12294	12294	2314	0.8	0
12295	12295	2314	1	0
12296	12296	2314	15	0
12297	12297	2314	3	0
12298	12298	2314	1	0
12299	12299	2314	7	0
12300	12300	2314	3	0
12301	12301	2314	3	0
12302	12302	2314	2	0
12303	12303	2314	0	0
12304	12304	2314	0	0
12305	12305	2314	0	0
12306	12306	2314	0	0
12307	12307	2314	0	0
12308	12308	2314	0	0
12309	12309	2314	0	0
12310	12310	2314	0	0
12311	12311	2314	B0L6	0
12312	12312	2314	1.35	0
12313	12313	2314	1.4	0
12314	12314	2314	22.8	0
12315	12315	2314	12	0
12316	12316	2314	34.2	0
12317	12317	2314	30	0
12318	12318	2314	21.7	0
12319	12319	2314	141	0
12320	12320	2314	2.8	0
12321	12321	2314	2.1	0
12322	12322	2314	0	0
12323	12323	2314	0	0
12324	12324	2314	0	0
12325	12325	2314	70	0
12326	12326	2314	225	0
12327	12327	2314	225	0
12328	12328	2314	10	0
12329	12329	2314	25	0
12330	12330	2314	70	0
12331	12331	2314	0	0
12332	12332	2314	2.601666667	0
12333	12333	2314	32.68333333	0
12334	12334	2314	6.383333333	0
12335	12335	2314	100	0
12336	12336	2314	175	0
12337	12337	2314	2.18	0
12338	12338	2314	101	0
12339	12339	2314	52	0
12340	12340	2314	3.62	0
12341	12341	2314	0	0
12342	12342	2314	0	0
12343	12343	2314	50	0
12344	12344	2314	0	0
12345	12345	2314	30	0
12346	12346	2314	53	0
12347	12347	2314	74	0
12348	12348	2314	16	0
12349	12349	2314	100	0
12350	12350	2314	175	0
12351	12351	2314	2.18	0
12352	12352	2314	101	0
12353	12353	2314	52	0
12354	12354	2314	3.62	0
12355	12355	2314	0	0
12356	12356	2314	0	0
12357	12357	2314	50	0
12358	12358	2314	0	0
12359	12359	2314	30	0
12360	12360	2314	109.7	0
12361	12361	2314	22.2	0
12362	12362	2314	243.2	0
12363	12363	2314	60.4	0
12364	12364	2314	21.8	0
12365	12365	2314	6.8	0
12366	12366	2314	13.48	0
12367	12367	2314	2.87	0
12368	12368	2314	13.5022	0
12369	12369	2314	2.9372	0
12370	12370	2314	28	0
12371	12371	2314	24	0
12372	12372	2314	1	0
12373	12373	2314	2	0
12374	12374	2314	0.6	0
12375	12375	2314	1	0
12376	12376	2314	0.7	0
12377	12377	2314	0.9	0
12378	12378	2314	3	0
12379	12379	2314	0	0
12380	12380	2314	0.9	0
12381	12381	2314	0.4	0
12382	12382	2314	0.7	0
12383	12383	2314	1	0
12384	12384	2314	0.7	0
12385	12385	2314	0.7	0
12386	12386	2314		0
12387	12387	2314		0
12388	12388	2314	1	0
12389	12389	2314	15	0
12390	12390	2314	4	0
12391	12391	2314	1	0
12392	12392	2314	7	0
12393	12393	2314	3	0
12394	12394	2314	3	0
12395	12395	2314	1	0
12396	12396	2314	1	0
12397	12397	2314	0	0
12398	12398	2314	0	0
12399	12399	2314	0	0
12400	12400	2314	0	0
12401	12401	2314	0	0
12402	12402	2314	0	0
12403	12403	2314	0	0
12404	12404	2314	B0L5	0
12405	12405	2314	1.4	0
12406	12406	2314	1.4	0
12407	12407	2314	28	0
12408	12408	2314	14	0
12409	12409	2314	29.6	0
12410	12410	2314	25	0
12411	12411	2314	9.9	0
12412	12412	2314	145	0
12413	12413	2314	2.6	0
12414	12414	2314	1.9	0
12415	12415	2314	0	0
12416	12416	2314	0	0
12417	12417	2314	0	0
12418	12418	2314	53	0
12419	12419	2314	225	0
12420	12420	2314	225	0
12421	12421	2314	10	0
12422	12422	2314	10	0
12423	12423	2314	85	0
12424	12424	2314	0	0
12425	12425	2314	1.5	0
12426	12426	2314	21.11666667	0
12427	12427	2314	5.416666667	0
12428	12428	2314	400	0
12429	12429	2314	550	0
12430	12430	2314	3.35	0
12431	12431	2314	65	0
12432	12432	2314	41	0
12433	12433	2314	5.14	0
12434	12434	2314	3	0
12435	12435	2314	0	0
12436	12436	2314	75	0
12437	12437	2314	10	0
12438	12438	2314	0	0
12439	12439	2314	32	0
12440	12440	2314	17	0
12441	12441	2314	39	0
12442	12442	2314	400	0
12443	12443	2314	550	0
12444	12444	2314	3.35	0
12445	12445	2314	65	0
12446	12446	2314	41	0
12447	12447	2314	5.14	0
12448	12448	2314	3	0
12449	12449	2314	0	0
12450	12450	2314	75	0
12451	12451	2314	10	0
12452	12452	2314	0	0
12453	12453	2314	175.1	0
12454	12454	2314	74.1	0
12455	12455	2314	223.1	0
12456	12456	2314	89.4	0
12457	12457	2314	28.5	0
12458	12458	2314	16.4	0
12459	12459	2314	24.84	0
12460	12460	2314	2.99	0
12461	12461	2314	24.9141	0
12462	12462	2314	3.0958	0
12463	12463	2314	8	0
12464	12464	2314	25	0
12465	12465	2314	1	0
12466	12466	2314	3	0
12467	12467	2314	0.5	0
12468	12468	2314	0.7	0
12469	12469	2314	1.1	0
12470	12470	2314	1.1	0
12471	12471	2314	5	0
12472	12472	2314	0	0
12473	12473	2314	1.2	0
12474	12474	2314	0.5	0
12475	12475	2314	0.5	0
12476	12476	2314	0.7	0
12477	12477	2314	0.7	0
12478	12478	2314	0.8	0
12479	12479	2314	0.8	0
12480	12480	2314	1	0
12481	12481	2314	1	0
12482	12482	2314	24	0
12483	12483	2314	5	0
12484	12484	2314	1	0
12485	12485	2314	9	0
12486	12486	2314	5	0
12487	12487	2314	5	0
12488	12488	2314	3	0
12489	12489	2314	1	0
12490	12490	2314	1	0
12491	12491	2314	0	0
12492	12492	2314	0	0
12493	12493	2314	0	0
12494	12494	2314	0	0
12495	12495	2314	0	0
12496	12496	2314	0	0
12497	12497	2314	B0L7	0
12498	12498	2314	1.3	0
12499	12499	2314	1.5	0
12500	12500	2314	25.5	0
12501	12501	2314	17	0
12502	12502	2314	34.7	0
12503	12503	2314	34	0
12504	12504	2314	15.1	0
12505	12505	2314	217	0
12506	12506	2314	2.2	0
12507	12507	2314	1.9	0
12508	12508	2314	0	0
12509	12509	2314	0	0
12510	12510	2314	0	0
12511	12511	2314	53.5	0
12512	12512	2314	125	0
12513	12513	2314	125	0
12514	12514	2314	10	0
12515	12515	2314	25	0
12516	12516	2314	75	0
12517	12517	2314	1	0
12518	12518	2314	1.055	0
12519	12519	2314	14.78333333	0
12520	12520	2314	3.5	0
12521	12521	2314	200	0
12522	12522	2314	500	0
12523	12523	2314	1.9	0
12524	12524	2314	76	0
12525	12525	2314	36.5	0
12526	12526	2314	3.33	0
12527	12527	2314	0	0
12528	12528	2314	0	0
12529	12529	2314	40	0
12530	12530	2314	0	0
12531	12531	2314	5	0
12532	12532	2314	92	0
12533	12533	2314	35	0
12534	12534	2314	46	0
12535	12535	2314	200	0
12536	12536	2314	500	0
12537	12537	2314	1.9	0
12538	12538	2314	76	0
12539	12539	2314	36.5	0
12540	12540	2314	3.33	0
12541	12541	2314	0	0
12542	12542	2314	0	0
12543	12543	2314	40	0
12544	12544	2314	0	0
12545	12545	2314	5	0
12546	12546	2314	68	0
12547	12547	2314	23.8	0
12548	12548	2314	127.2	0
12549	12549	2314	48.9	0
12550	12550	2314	22.7	0
12551	12551	2314	12.5	0
12552	12552	2314	10.61	0
12553	12553	2314	1.59	0
12554	12554	2314	10.6338	0
12555	12555	2314	1.6514	0
12556	12556	2314	4	0
12557	12557	2314	7	0
12558	12558	2314	1	0
12559	12559	2314	2	0
12560	12560	2314	0.4	0
12561	12561	2314	0.4	0
12562	12562	2314		0
12563	12563	2314		0
12564	12564	2314	2	0
12565	12565	2314	0	0
12566	12566	2314	0.6	0
12567	12567	2314	0.2	0
12568	12568	2314	0.3	0
12569	12569	2314	0.4	0
12570	12570	2314		0
12571	12571	2314		0
12572	12572	2314		0
12573	12573	2314		0
12574	12574	2314	0	0
12575	12575	2314	11	0
12576	12576	2314	2	0
12577	12577	2314	2	0
12578	12578	2314	6	0
12579	12579	2314	3	0
12580	12580	2314	2	0
12581	12581	2314	0	0
12582	12582	2314	0	0
12583	12583	2314	0	0
12584	12584	2314	0	0
12585	12585	2314	0	0
12586	12586	2314	0	0
12587	12587	2314	0	0
12588	12588	2314	0	0
12589	12589	2314	0	0
12590	12590	2314	B0L4	0
12591	12591	2314	0.7	0
12592	12592	2314	0.9	0
12593	12593	2314	16.3	0
12594	12594	2314	9	0
12595	12595	2314	21.4	0
12596	12596	2314	9	0
12597	12597	2314	1.7	0
12598	12598	2314	51	0
12599	12599	2314	1.8	0
12600	12600	2314	1.8	0
12601	12601	2314	0	0
12602	12602	2314	4	0
12603	12603	2314	0	0
12604	12604	2314	66	0
12605	12605	2314	175	0
12606	12606	2314	175	0
12607	12607	2314	5	0
12608	12608	2314	10	0
12609	12609	2314	90	0
12610	12610	2314	1	0
12611	12611	2314	2.765	0
12612	12612	2314	36.18333333	0
12613	12613	2314	6.8	0
12614	12614	2314	250	0
12615	12615	2314	250	0
12616	12616	2314	2.1	0
12617	12617	2314	94	0
12618	12618	2314	34	0
12619	12619	2314	2.14	0
12620	12620	2314	0	0
12621	12621	2314	0	0
12622	12622	2314	90	0
12623	12623	2314	0	0
12624	12624	2314	0	0
12625	12625	2314	33	0
12626	12626	2314	65	0
12627	12627	2314	16	0
12628	12628	2314	250	0
12629	12629	2314	250	0
12630	12630	2314	2.1	0
12631	12631	2314	94	0
12632	12632	2314	34	0
12633	12633	2314	2.14	0
12634	12634	2314	0	0
12635	12635	2314	0	0
12636	12636	2314	90	0
12637	12637	2314	0	0
12638	12638	2314	0	0
12639	12639	2314	77.8	0
12640	12640	2314	26.5	0
12641	12641	2314	105.6	0
12642	12642	2314	39.4	0
12643	12643	2314	8.4	0
12644	12644	2314	4.8	0
12645	12645	2314	9.16	0
12646	12646	2314	5.6	0
12647	12647	2314	9.1865	0
12648	12648	2314	5.6442	0
12649	12649	2314	2	0
12650	12650	2314	14	0
12651	12651	2314	2	0
12652	12652	2314	2	0
12653	12653	2314	0.4	0
12654	12654	2314	0.4	0
12655	12655	2314	0.5	0
12656	12656	2314	0.5	0
12657	12657	2314	4	0
12658	12658	2314	0	0
12659	12659	2314	0.6	0
12660	12660	2314	0.3	0
12661	12661	2314	.	0
12662	12662	2314	.	0
12663	12663	2314	0.5	0
12664	12664	2314	0.5	0
12665	12665	2314		0
12666	12666	2314		0
12667	12667	2314	0	0
12668	12668	2314	11	0
12669	12669	2314	3	0
12670	12670	2314	1	0
12671	12671	2314	6	0
12672	12672	2314	2	0
12673	12673	2314	2	0
12674	12674	2314	1	0
12675	12675	2314	0	0
12676	12676	2314	0	0
12677	12677	2314	0	0
12678	12678	2314	0	0
12679	12679	2314	0	0
12680	12680	2314	0	0
12681	12681	2314	0	0
12682	12682	2314	0	0
12683	12683	2314	B0L5	0
12684	12684	2314	0.8	0
12685	12685	2314	1.05	0
12686	12686	2314	15.3	0
12687	12687	2314	5	0
12688	12688	2314	21.2	0
12689	12689	2314	6	0
12690	12690	2314	1.7	0
12691	12691	2314	47	0
12692	12692	2314	2	0
12693	12693	2314	1.9	0
12694	12694	2314	0	0
12695	12695	2314	2	0
12696	12696	2314	0	0
12697	12697	2314	59	0
12698	12698	2314	100	0
12699	12699	2314	100	0
12700	12700	2314	5	0
12701	12701	2314	10	0
12702	12702	2314	95	0
12703	12703	2314	0	0
12704	12704	2314	2.023333333	0
12705	12705	2314	26.6	0
12706	12706	2314	5.616666667	0
12707	12707	2314	250	0
12708	12708	2314	325	0
12709	12709	2314	2.65	0
12710	12710	2314	91	0
12711	12711	2314	31.5	0
12712	12712	2314	3.73	0
12713	12713	2314	15	0
12714	12714	2314	0	0
12715	12715	2314	85	0
12716	12716	2314	40	0
12717	12717	2314	15	0
12718	12718	2314	38	0
12719	12719	2314	46	0
12720	12720	2314	39	0
12721	12721	2314	250	0
12722	12722	2314	325	0
12723	12723	2314	2.65	0
12724	12724	2314	91	0
12725	12725	2314	31.5	0
12726	12726	2314	3.73	0
12727	12727	2314	15	0
12728	12728	2314	0	0
12729	12729	2314	85	0
12730	12730	2314	40	0
12731	12731	2314	15	0
12732	12732	2314	70.5	0
12733	12733	2314	32.5	0
12734	12734	2314	71.2	0
12735	12735	2314	37.2	0
12736	12736	2314	5.8	0
12737	12737	2314	5	0
12738	12738	2314	8.95	0
12739	12739	2314	2.39	0
12740	12740	2314	8.9825	0
12741	12741	2314	2.4322	0
12742	12742	2314	21	0
12743	12743	2314	4	0
12744	12744	2314	2	0
12745	12745	2314	3	0
12746	12746	2314	0.3	0
12747	12747	2314	0.5	0
12748	12748	2314	1.2	0
12749	12749	2314	1.4	0
12750	12750	2314	5	0
12751	12751	2314	0	0
12752	12752	2314	1.1	0
12753	12753	2314	0.5	0
12754	12754	2314	0.3	0
12755	12755	2314	0.5	0
12756	12756	2314	0.8	0
12757	12757	2314	1.1	0
12758	12758	2314	0.7	0
12759	12759	2314	0.9	0
12760	12760	2314	1	0
12761	12761	2314	27	0
12762	12762	2314	9	0
12763	12763	2314	1	0
12764	12764	2314	7	0
12765	12765	2314	5	0
12766	12766	2314	5	0
12767	12767	2314	3	0
12768	12768	2314	2	0
12769	12769	2314	1	0
12770	12770	2314	0	0
12771	12771	2314	1	0
12772	12772	2314	1	0
12773	12773	2314	1	0
12774	12774	2314	1	0
12775	12775	2314	0	0
12776	12776	2314	B0L5	0
12777	12777	2314	1.1	0
12778	12778	2314	1.35	0
12779	12779	2314	22.8	0
12780	12780	2314	15	0
12781	12781	2314	44.3	0
12782	12782	2314	45	0
12783	12783	2314	16.5	0
12784	12784	2314	259	0
12785	12785	2314	2.3	0
12786	12786	2314	1.7	0
12787	12787	2314	37	0
12788	12788	2314	19	0
12789	12789	2314	11	0
12790	12790	2314	69	0
12791	12791	2314	175	0
12792	12792	2314	175	0
12793	12793	2314	10	0
12794	12794	2314	20	0
12795	12795	2314	85	0
12796	12796	2314	0	0
12797	12797	2314	2.038333333	0
12798	12798	2314	29.68333333	0
12799	12799	2314	6.366666667	0
12800	12800	2314	200	0
12801	12801	2314	300	0
12802	12802	2314	1.88	0
12803	12803	2314	69	0
12804	12804	2314	35	0
12805	12805	2314	2.81	0
12806	12806	2314	0	0
12807	12807	2314	0	0
12808	12808	2314	80	0
12809	12809	2314	0	0
12810	12810	2314	0	0
12811	12811	2314	108	0
12812	12812	2314	46	0
12813	12813	2314	27	0
12814	12814	2314	200	0
12815	12815	2314	300	0
12816	12816	2314	1.88	0
12817	12817	2314	69	0
12818	12818	2314	35	0
12819	12819	2314	2.81	0
12820	12820	2314	0	0
12821	12821	2314	0	0
12822	12822	2314	80	0
12823	12823	2314	0	0
12824	12824	2314	0	0
12825	12825	2314	88.3	0
12826	12826	2314	32.8	0
12827	12827	2314	136.9	0
12828	12828	2314	51.2	0
12829	12829	2314	29	0
12830	12830	2314	13.8	0
12831	12831	2314	9.76	0
12832	12832	2314	2.43	0
12833	12833	2314	9.7928	0
12834	12834	2314	2.495	0
12835	12835	2314	27	0
12836	12836	2314	10	0
12837	12837	2314	1	0
12838	12838	2314	2	0
12839	12839	2314	0.3	0
12840	12840	2314	0.3	0
12841	12841	2314	0.7	0
12842	12842	2314	0.7	0
12843	12843	2314	3	0
12844	12844	2314	0	0
12845	12845	2314	0.8	0
12846	12846	2314	0.4	0
12847	12847	2314	0.3	0
12848	12848	2314	0.5	0
12849	12849	2314	0.5	0
12850	12850	2314	0.7	0
12851	12851	2314		0
12852	12852	2314		0
12853	12853	2314	1	0
12854	12854	2314	14	0
12855	12855	2314	3	0
12856	12856	2314	1	0
12857	12857	2314	6	0
12858	12858	2314	4	0
12859	12859	2314	3	0
12860	12860	2314	1	0
12861	12861	2314	0	0
12862	12862	2314	0	0
12863	12863	2314	0	0
12864	12864	2314	0	0
12865	12865	2314	0	0
12866	12866	2314	0	0
12867	12867	2314	0	0
12868	12868	2314	0	0
12869	12869	2314	B0L4	0
12870	12870	2314	0.9	0
12871	12871	2314	1	0
12872	12872	2314	25	0
12873	12873	2314	10	0
12874	12874	2314	35.9	0
12875	12875	2314	23	0
12876	12876	2314	7.2	0
12877	12877	2314	138	0
12878	12878	2314	2.5	0
12879	12879	2314	2.1	0
12880	12880	2314	0	0
12881	12881	2314	0	0
12882	12882	2314	0	0
12883	12883	2314	63	0
12884	12884	2314	225	0
12885	12885	2314	225	0
12886	12886	2314	5	0
12887	12887	2314	40	0
12888	12888	2314	90	0
12889	12889	2314	0	0
12890	12890	2314	1.995	0
12891	12891	2314	27.66666667	0
12892	12892	2314	6.15	0
12893	12893	2314	175	0
12894	12894	2314	175	0
12895	12895	2314	2.51	0
12896	12896	2314	70	0
12897	12897	2314	36	0
12898	12898	2314	3.6	0
12899	12899	2314	2	0
12900	12900	2314	0	0
12901	12901	2314	90	0
12902	12902	2314	0	0
12903	12903	2314	0	0
12904	12904	2314	122	0
12905	12905	2314	50	0
12906	12906	2314	10	0
12907	12907	2314	175	0
12908	12908	2314	175	0
12909	12909	2314	2.51	0
12910	12910	2314	70	0
12911	12911	2314	36	0
12912	12912	2314	3.6	0
12913	12913	2314	2	0
12914	12914	2314	0	0
12915	12915	2314	90	0
12916	12916	2314	0	0
12917	12917	2314	0	0
12918	12918	2314	89.1	0
12919	12919	2314	29	0
12920	12920	2314	170.5	0
12921	12921	2314	59.7	0
12922	12922	2314	14.6	0
12923	12923	2314	7.2	0
12924	12924	2314	10.77	0
12925	12925	2314	2.57	0
12926	12926	2314	10.799	0
12927	12927	2314	2.6369	0
12928	12928	2314	14	0
12929	12929	2314	37	0
12930	12930	2314	1	0
12931	12931	2314	2	0
12932	12932	2314	0.5	0
12933	12933	2314	0.7	0
12934	12934	2314	0.8	0
12935	12935	2314	1.5	0
12936	12936	2314	4	0
12937	12937	2314	0	0
12938	12938	2314	0.8	0
12939	12939	2314	0.4	0
12940	12940	2314	0.6	0
12941	12941	2314	0.8	0
12942	12942	2314	0.6	0
12943	12943	2314	0.8	0
12944	12944	2314	0.5	0
12945	12945	2314	0.9	0
12946	12946	2314	1	0
12947	12947	2314	18	0
12948	12948	2314	5	0
12949	12949	2314	1	0
12950	12950	2314	8	0
12951	12951	2314	2	0
12952	12952	2314	3	0
12953	12953	2314	3	0
12954	12954	2314	1	0
12955	12955	2314	1	0
12956	12956	2314	0	0
12957	12957	2314	0	0
12958	12958	2314	0	0
12959	12959	2314	0	0
12960	12960	2314	0	0
12961	12961	2314	0	0
12962	12962	2314	B0L6	0
12963	12963	2314	1	0
12964	12964	2314	1	0
12965	12965	2314	25.3	0
12966	12966	2314	13	0
12967	12967	2314	38.4	0
12968	12968	2314	28	0
12969	12969	2314	6.6	0
12970	12970	2314	165	0
12971	12971	2314	2.3	0
12972	12972	2314	2	0
12973	12973	2314	17	0
12974	12974	2314	14	0
12975	12975	2314	4	0
12976	12976	2314	61	0
12977	12977	2314	150	0
12978	12978	2314	150	0
12979	12979	2314	5	0
12980	12980	2314	10	0
12981	12981	2314	95	0
12982	12982	2314	0	0
12983	12983	2314	2.475	0
12984	12984	2314	31.23333333	0
12985	12985	2314	6.166666667	0
12986	12986	2314	220	0
12987	12987	2314	220	0
12988	12988	2314	2.63	0
12989	12989	2314	92	0
12990	12990	2314	38	0
12991	12991	2314	3.93	0
12992	12992	2314	0	0
12993	12993	2314	0	0
12994	12994	2314	90	0
12995	12995	2314	0	0
12996	12996	2314	0	0
12997	12997	2314	86	0
12998	12998	2314	49	0
12999	12999	2314	13	0
13000	13000	2314	220	0
13001	13001	2314	220	0
13002	13002	2314	2.63	0
13003	13003	2314	92	0
13004	13004	2314	38	0
13005	13005	2314	3.93	0
13006	13006	2314	0	0
13007	13007	2314	0	0
13008	13008	2314	90	0
13009	13009	2314	0	0
13010	13010	2314	0	0
13011	13011	2314	64.2	0
13012	13012	2314	14.9	0
13013	13013	2314	192.6	0
13014	13014	2314	57.4	0
13015	13015	2314	15.6	0
13016	13016	2314	6.3	0
13017	13017	2314	9.42	0
13018	13018	2314	1.74	0
13019	13019	2314	9.4349	0
13020	13020	2314	1.8037	0
13021	13021	2314	14	0
13022	13022	2314	3	0
13023	13023	2314	1	0
13024	13024	2314	2	0
13025	13025	2314	0.3	0
13026	13026	2314	0.3	0
13027	13027	2314		0
13028	13028	2314		0
13029	13029	2314	2	0
13030	13030	2314	0	0
13031	13031	2314	0.6	0
13032	13032	2314	0.25	0
13033	13033	2314	0.3	0
13034	13034	2314	0.4	0
13035	13035	2314	0.4	0
13036	13036	2314	0.4	0
13037	13037	2314		0
13038	13038	2314		0
13039	13039	2314	1	0
13040	13040	2314	7	0
13041	13041	2314	1	0
13042	13042	2314	2	0
13043	13043	2314	6	0
13044	13044	2314	1	0
13045	13045	2314	0	0
13046	13046	2314	0	0
13047	13047	2314	0	0
13048	13048	2314	0	0
13049	13049	2314	0	0
13050	13050	2314	0	0
13051	13051	2314	0	0
13052	13052	2314	0	0
13053	13053	2314	0	0
13054	13054	2314	0	0
13055	13055	2314	B0L5	0
13056	13056	2314	0.8	0
13057	13057	2314	0.95	0
13058	13058	2314	21.5	0
13059	13059	2314	6	0
13060	13060	2314	19	0
13061	13061	2314	0	0
13062	13062	2314	.	0
13063	13063	2314	32	0
13064	13064	2314	2.3	0
13065	13065	2314	2	0
13066	13066	2314	0	0
13067	13067	2314	0	0
13068	13068	2314	0	0
13069	13069	2314	74	0
13070	13070	2314	175	0
13071	13071	2314	175	0
13072	13072	2314	5	0
13073	13073	2314	15	0
13074	13074	2314	95	0
13075	13075	2314	0	0
13076	13076	2314	4.043333333	0
13077	13077	2314	45.73333333	0
13078	13078	2314	11.56666667	0
13079	13079	2314	300	0
13080	13080	2314	300	0
13081	13081	2314	2.79	0
13082	13082	2314	85	0
13083	13083	2314	38	0
13084	13084	2314	4.49	0
13085	13085	2314	0	0
13086	13086	2314	0	0
13087	13087	2314	95	0
13088	13088	2314	0	0
13089	13089	2314	0	0
13090	13090	2314	47	0
13091	13091	2314	70	0
13092	13092	2314	14	0
13093	13093	2314	300	0
13094	13094	2314	300	0
13095	13095	2314	2.79	0
13096	13096	2314	85	0
13097	13097	2314	38	0
13098	13098	2314	4.49	0
13099	13099	2314	0	0
13100	13100	2314	0	0
13101	13101	2314	95	0
13102	13102	2314	0	0
13103	13103	2314	0	0
13104	13104	2314	114.9	0
13105	13105	2314	30.3	0
13106	13106	2314	89.6	0
13107	13107	2314	21.8	0
13108	13108	2314	12.9	0
13109	13109	2314	3.5	0
13110	13110	2314	12.08	0
13111	13111	2314	1.48	0
13112	13112	2314	12.1103	0
13113	13113	2314	1.5053	0
13114	13114	2314	51	0
13115	13115	2314	15	0
13116	13116	2314	1	0
13117	13117	2314	3	0
13118	13118	2314	0.8	0
13119	13119	2314	1	0
13120	13120	2314	0.9	0
13121	13121	2314	1.1	0
13122	13122	2314	3	0
13123	13123	2314	0	0
13124	13124	2314	1.1	0
13125	13125	2314	0.5	0
13126	13126	2314	0.9	0
13127	13127	2314	1.1	0
13128	13128	2314	0.8	0
13129	13129	2314	1.1	0
13130	13130	2314	0.8	0
13131	13131	2314	1.1	0
13132	13132	2314	1	0
13133	13133	2314	20	0
13134	13134	2314	4	0
13135	13135	2314	1	0
13136	13136	2314	8	0
13137	13137	2314	4	0
13138	13138	2314	4	0
13139	13139	2314	3	0
13140	13140	2314	1	0
13141	13141	2314	0	0
13142	13142	2314	0	0
13143	13143	2314	0	0
13144	13144	2314	0	0
13145	13145	2314	0	0
13146	13146	2314	0	0
13147	13147	2314	0	0
13148	13148	2314	B0L6	0
13149	13149	2314	1.3	0
13150	13150	2314	1.7	0
13151	13151	2314	33.9	0
13152	13152	2314	14	0
13153	13153	2314	43.5	0
13154	13154	2314	35	0
13155	13155	2314	21.4	0
13156	13156	2314	187	0
13157	13157	2314	2.7	0
13158	13158	2314	2.3	0
13159	13159	2314	0	0
13160	13160	2314	0	0
13161	13161	2314	0	0
13162	13162	2314	76	0
13163	13163	2314	250	0
13164	13164	2314	250	0
13165	13165	2314	5	0
13166	13166	2314	50	0
13167	13167	2314	100	0
13168	13168	2314	0	0
13169	13169	2314	2.023333333	0
13170	13170	2314	26.31666667	0
13171	13171	2314	5.966666667	0
13172	13172	2314	175	0
13173	13173	2314	175	0
13174	13174	2314	2.84	0
13175	13175	2314	81	0
13176	13176	2314	40	0
13177	13177	2314	3.57	0
13178	13178	2314	0	0
13179	13179	2314	0	0
13180	13180	2314	60	0
13181	13181	2314	0	0
13182	13182	2314	0	0
13183	13183	2314	123	0
13184	13184	2314		0
13185	13185	2314		0
13186	13186	2314	175	0
13187	13187	2314	175	0
13188	13188	2314	2.84	0
13189	13189	2314	81	0
13190	13190	2314	40	0
13191	13191	2314	3.57	0
13192	13192	2314	0	0
13193	13193	2314	0	0
13194	13194	2314	60	0
13195	13195	2314	0	0
13196	13196	2314	0	0
13197	13197	2314	72.4	0
13198	13198	2314	35.1	0
13199	13199	2314	78.8	0
13200	13200	2314	44.9	0
13201	13201	2314	7.2	0
13202	13202	2314	4.9	0
13203	13203	2314	15.71	0
13204	13204	2314	2.06	0
13205	13205	2314	15.7451	0
13206	13206	2314	2.1098	0
13207	13207	2314	22	0
13208	13208	2314	29	0
13209	13209	2314	1	0
13210	13210	2314	2	0
13211	13211	2314	0.7	0
13212	13212	2314	0.9	0
13213	13213	2314		0
13214	13214	2314		0
13215	13215	2314	3	0
13216	13216	2314	0	0
13217	13217	2314	1.2	0
13218	13218	2314	0.5	0
13219	13219	2314	0.6	0
13220	13220	2314	0.9	0
13221	13221	2314	0.4	0
13222	13222	2314	0.7	0
13223	13223	2314		0
13224	13224	2314		0
13225	13225	2314	0	0
13226	13226	2314	14	0
13227	13227	2314	3	0
13228	13228	2314	1	0
13229	13229	2314	6	0
13230	13230	2314	4	0
13231	13231	2314	3	0
13232	13232	2314	1	0
13233	13233	2314	0	0
13234	13234	2314	0	0
13235	13235	2314	0	0
13236	13236	2314	0	0
13237	13237	2314	0	0
13238	13238	2314	0	0
13239	13239	2314	0	0
13240	13240	2314	0	0
13241	13241	2314	B0L5	0
13242	13242	2314	0.9	0
13243	13243	2314	1	0
13244	13244	2314	26.5	0
13245	13245	2314	13	0
13246	13246	2314	39.2	0
13247	13247	2314	18	0
13248	13248	2314	12	0
13249	13249	2314	91	0
13250	13250	2314	3	0
13251	13251	2314	2.2	0
13252	13252	2314	0	0
13253	13253	2314	0	0
13254	13254	2314	0	0
13255	13255	2314	67.5	0
13256	13256	2314	150	0
13257	13257	2314	150	0
13258	13258	2314	10	0
13259	13259	2314	30	0
13260	13260	2314	85	0
13261	13261	2314	0	0
13262	13262	2314	1.975	0
13263	13263	2314	25.05	0
13264	13264	2314	5.566666667	0
13265	13265	2314	150	0
13266	13266	2314	150	0
13267	13267	2314	2.17	0
13268	13268	2314	73	0
13269	13269	2314	41	0
13270	13270	2314	3.41	0
13271	13271	2314	0	0
13272	13272	2314	0	0
13273	13273	2314	70	0
13274	13274	2314	0	0
13275	13275	2314	0	0
13276	13276	2314	104	0
13277	13277	2314	41	0
13278	13278	2314	9	0
13279	13279	2314	150	0
13280	13280	2314	150	0
13281	13281	2314	2.17	0
13282	13282	2314	73	0
13283	13283	2314	41	0
13284	13284	2314	3.41	0
13285	13285	2314	0	0
13286	13286	2314	0	0
13287	13287	2314	70	0
13288	13288	2314	0	0
13289	13289	2314	0	0
13290	13290	2314	65.2	0
13291	13291	2314	20.8	0
13292	13292	2314	110.7	0
13293	13293	2314	41.1	0
13294	13294	2314	17.7	0
13295	13295	2314	8.2	0
13296	13296	2314	10.2	0
13297	13297	2314	1.84	0
13298	13298	2314	10.2208	0
13299	13299	2314	1.8893	0
13300	13300	2314	23	0
13301	13301	2314	21	0
13302	13302	2314	1	0
13303	13303	2314	3	0
13304	13304	2314	0.8	0
13305	13305	2314	1.1	0
13306	13306	2314	0.8	0
13307	13307	2314	0.7	0
13308	13308	2314	4	0
13309	13309	2314	0	0
13310	13310	2314	0.8	0
13311	13311	2314	0.5	0
13312	13312	2314	0.7	0
13313	13313	2314	1.1	0
13314	13314	2314	0.6	0
13315	13315	2314	0.9	0
13316	13316	2314	0.8	0
13317	13317	2314	1	0
13318	13318	2314	1	0
13319	13319	2314	20	0
13320	13320	2314	4	0
13321	13321	2314	1	0
13322	13322	2314	8	0
13323	13323	2314	4	0
13324	13324	2314	4	0
13325	13325	2314	3	0
13326	13326	2314	1	0
13327	13327	2314	0	0
13328	13328	2314	0	0
13329	13329	2314	0	0
13330	13330	2314	0	0
13331	13331	2314	0	0
13332	13332	2314	0	0
13333	13333	2314	0	0
13334	13334	2314	B0L6	0
13335	13335	2314	1.3	0
13336	13336	2314	1.5	0
13337	13337	2314	29	0
13338	13338	2314	12	0
13339	13339	2314	37.7	0
13340	13340	2314	41	0
13341	13341	2314	18.4	0
13342	13342	2314	226	0
13343	13343	2314	2.6	0
13344	13344	2314	2.5	0
13345	13345	2314	0	0
13346	13346	2314	0	0
13347	13347	2314	0	0
13348	13348	2314	71	0
13349	13349	2314	150	0
13350	13350	2314	150	0
13351	13351	2314	5	0
13352	13352	2314	40	0
13353	13353	2314	90	0
13354	13354	2314	0	0
13355	13355	2314	2.143333333	0
13356	13356	2314	24.95	0
13357	13357	2314	6.25	0
13358	13358	2314	200	0
13359	13359	2314	250	0
13360	13360	2314	2.13	0
13361	13361	2314	85.5	0
13362	13362	2314	36	0
13363	13363	2314	3.22	0
13364	13364	2314	0	0
13365	13365	2314	0	0
13366	13366	2314	85	0
13367	13367	2314	0	0
13368	13368	2314	0	0
13369	13369	2314	146	0
13370	13370	2314	63	0
13371	13371	2314	6	0
13372	13372	2314	200	0
13373	13373	2314	250	0
13374	13374	2314	2.13	0
13375	13375	2314	85.5	0
13376	13376	2314	36	0
13377	13377	2314	3.22	0
13378	13378	2314	0	0
13379	13379	2314	0	0
13380	13380	2314	85	0
13381	13381	2314	0	0
13382	13382	2314	0	0
13383	13383	2314	45.1	0
13384	13384	2314	21.2	0
13385	13385	2314	115.8	0
13386	13386	2314	53.3	0
13387	13387	2314	13.7	0
13388	13388	2314	9.8	0
13389	13389	2314	12.96	0
13390	13390	2314	1.46	0
13391	13391	2314	12.9812	0
13392	13392	2314	1.5231	0
13393	13393	2314	2	0
13394	13394	2314	1	0
13395	13395	2314	1	0
13396	13396	2314	2	0
13397	13397	2314	0.5	0
13398	13398	2314	0.4	0
13399	13399	2314		0
13400	13400	2314		0
13401	13401	2314	2	0
13402	13402	2314	0	0
13403	13403	2314	0.8	0
13404	13404	2314	4	0
13405	13405	2314	0.5	0
13406	13406	2314	0.6	0
13407	13407	2314	0.5	0
13408	13408	2314	0.6	0
13409	13409	2314		0
13410	13410	2314		0
13411	13411	2314	2	0
13412	13412	2314	12	0
13413	13413	2314	3	0
13414	13414	2314	1	0
13415	13415	2314	6	0
13416	13416	2314	3	0
13417	13417	2314	2	0
13418	13418	2314	1	0
13419	13419	2314	0	0
13420	13420	2314	0	0
13421	13421	2314	0	0
13422	13422	2314	0	0
13423	13423	2314	0	0
13424	13424	2314	0	0
13425	13425	2314	0	0
13426	13426	2314	0	0
13427	13427	2314	B0L5	0
13428	13428	2314	0.6	0
13429	13429	2314	0.7	0
13430	13430	2314	17.2	0
13431	13431	2314	7	0
13432	13432	2314	34.7	0
13433	13433	2314	34	0
13434	13434	2314	6.3	0
13435	13435	2314	142	0
13436	13436	2314	2.3	0
13437	13437	2314	2.1	0
13438	13438	2314	0	0
13439	13439	2314	2	0
13440	13440	2314	0	0
13441	13441	2314	110	0
13442	13442	2314	275	0
13443	13443	2314	275	0
13444	13444	2314	10	0
13445	13445	2314	50	0
13446	13446	2314	95	0
13447	13447	2314	0	0
13448	13448	2314	2.545	0
13449	13449	2314	34.13333333	0
13450	13450	2314	8.15	0
13451	13451	2314	150	0
13452	13452	2314	250	0
13453	13453	2314	2.69	0
13454	13454	2314	78	0
13455	13455	2314	51	0
13456	13456	2314	3.54	0
13457	13457	2314	0	0
13458	13458	2314	0	0
13459	13459	2314	60	0
13460	13460	2314	0	0
13461	13461	2314	0	0
13462	13462	2314	148	0
13463	13463	2314	33	0
13464	13464	2314	24	0
13465	13465	2314	150	0
13466	13466	2314	250	0
13467	13467	2314	2.69	0
13468	13468	2314	78	0
13469	13469	2314	51	0
13470	13470	2314	3.54	0
13471	13471	2314	0	0
13472	13472	2314	0	0
13473	13473	2314	60	0
13474	13474	2314	0	0
13475	13475	2314	0	0
13476	13476	2314	115	0
13477	13477	2314	34.1	0
13478	13478	2314	241.1	0
13479	13479	2314	58.8	0
13480	13480	2314	10.1	0
13481	13481	2314	2.8	0
13482	13482	2314	11.79	0
13483	13483	2314	3.12	0
13484	13484	2314	11.8241	0
13485	13485	2314	3.1816	0
13486	13486	2314	4	0
13487	13487	2314	11	0
13488	13488	2314	1	0
13489	13489	2314	2	0
13490	13490	2314	0.5	0
13491	13491	2314	0.7	0
13492	13492	2314	1	0
13493	13493	2314	1.2	0
13494	13494	2314	3	0
13495	13495	2314	0	0
13496	13496	2314	1.2	0
13497	13497	2314	0.5	0
13498	13498	2314	0.3	0
13499	13499	2314	0.4	0
13500	13500	2314	0.7	0
13501	13501	2314	0.8	0
13502	13502	2314	0.6	0
13503	13503	2314	0.7	0
13504	13504	2314	1	0
13505	13505	2314	14	0
13506	13506	2314	3	0
13507	13507	2314	2	0
13508	13508	2314	6	0
13509	13509	2314	4	0
13510	13510	2314	3	0
13511	13511	2314	1	0
13512	13512	2314	0	0
13513	13513	2314	0	0
13514	13514	2314	0	0
13515	13515	2314	0	0
13516	13516	2314	0	0
13517	13517	2314	0	0
13518	13518	2314	0	0
13519	13519	2314	0	0
13520	13520	2314	B0L4	0
13521	13521	2314	1.3	0
13522	13522	2314	1.4	0
13523	13523	2314	27	0
13524	13524	2314	14	0
13525	13525	2314	37.4	0
13526	13526	2314	31	0
13527	13527	2314	7.5	0
13528	13528	2314	159	0
13529	13529	2314	2.9	0
13530	13530	2314	2.6	0
13531	13531	2314	19	0
13532	13532	2314	9	0
13533	13533	2314	0	0
13534	13534	2314	98.5	0
13535	13535	2314	150	0
13536	13536	2314	150	0
13537	13537	2314	5	0
13538	13538	2314	20	0
13539	13539	2314	90	0
13540	13540	2314	0	0
13541	13541	2314	2.535	0
13542	13542	2314	27.81666667	0
13543	13543	2314	7.583333333	0
13544	13544	2314	300	0
13545	13545	2314	325	0
13546	13546	2314	2.46	0
13547	13547	2314	85	0
13548	13548	2314	39	0
13549	13549	2314	4.05	0
13550	13550	2314	0	0
13551	13551	2314	0	0
13552	13552	2314	90	0
13553	13553	2314	0	0
13554	13554	2314	0	0
13555	13555	2314	129	0
13556	13556	2314	61	0
13557	13557	2314	25	0
13558	13558	2314	300	0
13559	13559	2314	325	0
13560	13560	2314	2.46	0
13561	13561	2314	85	0
13562	13562	2314	39	0
13563	13563	2314	4.05	0
13564	13564	2314	0	0
13565	13565	2314	0	0
13566	13566	2314	90	0
13567	13567	2314	0	0
13568	13568	2314	0	0
13569	13569	2314	139.2	0
13570	13570	2314	36.2	0
13571	13571	2314	131.9	0
13572	13572	2314	31.3	0
13573	13573	2314	15.9	0
13574	13574	2314	4.3	0
13575	13575	2314	11.98	0
13576	13576	2314	4.07	0
13577	13577	2314	12.0162	0
13578	13578	2314	4.1056	0
13579	13579	2314	30	0
13580	13580	2314	1	0
13581	13581	2314	1	0
13582	13582	2314	2	0
13583	13583	2314	0.5	0
13584	13584	2314	0.6	0
13585	13585	2314		0
13586	13586	2314		0
13587	13587	2314	3	0
13588	13588	2314	0	0
13589	13589	2314	0.8	0
13590	13590	2314	0.4	0
13591	13591	2314	0.5	0
13592	13592	2314	0.8	0
13593	13593	2314	0.5	0
13594	13594	2314	0.6	0
13595	13595	2314		0
13596	13596	2314		0
13597	13597	2314	0	0
13598	13598	2314	11	0
13599	13599	2314	3	0
13600	13600	2314	1	0
13601	13601	2314	6	0
13602	13602	2314	1	0
13603	13603	2314	3	0
13604	13604	2314	1	0
13605	13605	2314	0	0
13606	13606	2314	0	0
13607	13607	2314	0	0
13608	13608	2314	0	0
13609	13609	2314	0	0
13610	13610	2314	0	0
13611	13611	2314	0	0
13612	13612	2314	0	0
13613	13613	2314	B0L4	0
13614	13614	2314	1.15	0
13615	13615	2314	1.15	0
13616	13616	2314	25	0
13617	13617	2314	12	0
13618	13618	2314	30.6	0
13619	13619	2314	20	0
13620	13620	2314	16.4	0
13621	13621	2314	88	0
13622	13622	2314	2.3	0
13623	13623	2314	1.8	0
13624	13624	2314	0	0
13625	13625	2314	0	0
13626	13626	2314	0	0
13627	13627	2314	60.5	0
13628	13628	2314	50	0
13629	13629	2314	50	0
13630	13630	2314	5	0
13631	13631	2314	20	0
13632	13632	2314	75	0
13633	13633	2314	0	0
13634	13634	2314	1.253333333	0
13635	13635	2314	17.23333333	0
13636	13636	2314	4.266666667	0
13637	13637	2314	40	0
13638	13638	2314	150	0
13639	13639	2314	2.11	0
13640	13640	2314	68	0
13641	13641	2314	38	0
13642	13642	2314	2.08	0
13643	13643	2314	0	0
13644	13644	2314	0	0
13645	13645	2314	30	0
13646	13646	2314	0	0
13647	13647	2314	0	0
13648	13648	2314	66	0
13649	13649	2314	3	0
13650	13650	2314	0	0
13651	13651	2314	40	0
13652	13652	2314	150	0
13653	13653	2314	2.11	0
13654	13654	2314	68	0
13655	13655	2314	38	0
13656	13656	2314	2.08	0
13657	13657	2314	0	0
13658	13658	2314	0	0
13659	13659	2314	30	0
13660	13660	2314	0	0
13661	13661	2314	0	0
13662	13662	2314	63.2	0
13663	13663	2314	24.6	0
13664	13664	2314	24.1	0
13665	13665	2314	13.5	0
13666	13666	2314	7.9	0
13667	13667	2314	5.8	0
13668	13668	2314	3.09	0
13669	13669	2314	0.59	0
13670	13670	2314	3.1146	0
13671	13671	2314	0.6093	0
13672	13672	2314	11	0
13673	13673	2314	16	0
13674	13674	2314	0	0
13675	13675	2314	2	0
13676	13676	2314	0.3	0
13677	13677	2314	0.4	0
13678	13678	2314		0
13679	13679	2314		0
13680	13680	2314	2	0
13681	13681	2314	0	0
13682	13682	2314	.	0
13683	13683	2314	.	0
13684	13684	2314	0.4	0
13685	13685	2314	0.5	0
13686	13686	2314	0.2	0
13687	13687	2314	0.3	0
13688	13688	2314		0
13689	13689	2314		0
13690	13690	2314	0	0
13691	13691	2314	5	0
13692	13692	2314	0	0
13693	13693	2314	.	0
13694	13694	2314	5	0
13695	13695	2314	0	0
13696	13696	2314	0	0
13697	13697	2314	0	0
13698	13698	2314	0	0
13699	13699	2314	0	0
13700	13700	2314	0	0
13701	13701	2314	0	0
13702	13702	2314	0	0
13703	13703	2314	0	0
13704	13704	2314	0	0
13705	13705	2314	0	0
13706	13706	2314	B0L4	0
13707	13707	2314	0.6	0
13708	13708	2314	0.7	0
13709	13709	2314	23.5	0
13710	13710	2314	8	0
13711	13711	2314	17.3	0
13712	13712	2314	0	0
13713	13713	2314	.	0
13714	13714	2314	28	0
13715	13715	2314	2.1	0
13716	13716	2314	1.8	0
13717	13717	2314	0	0
13718	13718	2314	0	0
13719	13719	2314	0	0
13720	13720	2314	84	0
13721	13721	2314	150	0
13722	13722	2314	150	0
13723	13723	2314	5	0
13724	13724	2314	20	0
13725	13725	2314	90	0
13726	13726	2314	0	0
13727	13727	2314	2.49	0
13728	13728	2314	28.96666667	0
13729	13729	2314	8.383333333	0
13730	13730	2314	225	0
13731	13731	2314	225	0
13732	13732	2314	3	0
13733	13733	2314	99	0
13734	13734	2314	31	0
13735	13735	2314	3.88	0
13736	13736	2314	0	0
13737	13737	2314	0	0
13738	13738	2314	85	0
13739	13739	2314	0	0
13740	13740	2314	0	0
13741	13741	2314	44	0
13742	13742	2314	22	0
13743	13743	2314	41	0
13744	13744	2314	225	0
13745	13745	2314	225	0
13746	13746	2314	3	0
13747	13747	2314	99	0
13748	13748	2314	31	0
13749	13749	2314	3.88	0
13750	13750	2314	0	0
13751	13751	2314	0	0
13752	13752	2314	85	0
13753	13753	2314	0	0
13754	13754	2314	0	0
13755	13755	2314	200.9	0
13756	13756	2314	55	0
13757	13757	2314	183	0
13758	13758	2314	45.3	0
13759	13759	2314	13	0
13760	13760	2314	3.4	0
13761	13761	2314	8.67	0
13762	13762	2314	1.73	0
13763	13763	2314	8.725	0
13764	13764	2314	1.7787	0
13765	13765	2314	4	0
13766	13766	2314	8	0
13767	13767	2314	1	0
13768	13768	2314	2	0
13769	13769	2314	0.4	0
13770	13770	2314	0.6	0
13771	13771	2314		0
13772	13772	2314		0
13773	13773	2314	4	0
13774	13774	2314	0	0
13775	13775	2314	1	0
13776	13776	2314	0.4	0
13777	13777	2314	0.4	0
13778	13778	2314	0.7	0
13779	13779	2314	0.4	0
13780	13780	2314	0.7	0
13781	13781	2314	0.5	0
13782	13782	2314	0.7	0
13783	13783	2314	2	0
13784	13784	2314	17	0
13785	13785	2314	4	0
13786	13786	2314	1	0
13787	13787	2314	7	0
13788	13788	2314	2	0
13789	13789	2314	4	0
13790	13790	2314	3	0
13791	13791	2314	1	0
13792	13792	2314	0	0
13793	13793	2314	0	0
13794	13794	2314	0	0
13795	13795	2314	0	0
13796	13796	2314	0	0
13797	13797	2314	0	0
13798	13798	2314	0	0
13799	13799	2314	B0L5	0
13800	13800	2314	1	0
13801	13801	2314	1.1	0
13802	13802	2314	16.6	0
13803	13803	2314	8	0
13804	13804	2314	30.2	0
13805	13805	2314	25	0
13806	13806	2314	11.4	0
13807	13807	2314	124	0
13808	13808	2314	1.9	0
13809	13809	2314	1.6	0
13810	13810	2314	4	0
13811	13811	2314	5	0
13812	13812	2314	0	0
13813	13813	2314	37.5	0
13814	13814	2314	40	0
13815	13815	2314	40	0
13816	13816	2314	5	0
13817	13817	2314	10	0
13818	13818	2314	95	0
13819	13819	2314	0	0
13820	13820	2314	1.526666667	0
13821	13821	2314	20.45	0
13822	13822	2314	3.916666667	0
13823	13823	2314	150	0
13824	13824	2314	150	0
13825	13825	2314	1.49	0
13826	13826	2314	62	0
13827	13827	2314	42	0
13828	13828	2314	2.42	0
13829	13829	2314	0	0
13830	13830	2314	0	0
13831	13831	2314	90	0
13832	13832	2314	0	0
13833	13833	2314	0	0
13834	13834	2314	18	0
13835	13835	2314	48	0
13836	13836	2314	14	0
13837	13837	2314	150	0
13838	13838	2314	150	0
13839	13839	2314	1.49	0
13840	13840	2314	62	0
13841	13841	2314	42	0
13842	13842	2314	2.42	0
13843	13843	2314	0	0
13844	13844	2314	0	0
13845	13845	2314	90	0
13846	13846	2314	0	0
13847	13847	2314	0	0
13848	13848	2314	24.4	0
13849	13849	2314	9.9	0
13850	13850	2314	63.5	0
13851	13851	2314	28.5	0
13852	13852	2314	3.6	0
13853	13853	2314	3.5	0
13854	13854	2314	4.18	0
13855	13855	2314	0.73	0
13856	13856	2314	4.1899	0
13857	13857	2314	0.762	0
13858	13858	2314	27	0
13859	13859	2314	10	0
13860	13860	2314	2	0
13861	13861	2314	3	0
13862	13862	2314	0.6	0
13863	13863	2314	1	0
13864	13864	2314	1.3	0
13865	13865	2314	1.4	0
13866	13866	2314	5	0
13867	13867	2314	0	0
13868	13868	2314	1.2	0
13869	13869	2314	0.5	0
13870	13870	2314	0.7	0
13871	13871	2314	1.1	0
13872	13872	2314	0.7	0
13873	13873	2314	1.1	0
13874	13874	2314	0.8	0
13875	13875	2314	1.1	0
13876	13876	2314	2	0
13877	13877	2314	29	0
13878	13878	2314	8	0
13879	13879	2314	1	0
13880	13880	2314	9	0
13881	13881	2314	5	0
13882	13882	2314	5	0
13883	13883	2314	4	0
13884	13884	2314	2	0
13885	13885	2314	1	0
13886	13886	2314	0	0
13887	13887	2314	1	0
13888	13888	2314	1	0
13889	13889	2314	1	0
13890	13890	2314	0	0
13891	13891	2314	0	0
13892	13892	2314	B0L7	0
13893	13893	2314	1.2	0
13894	13894	2314	1.2	0
13895	13895	2314	32.2	0
13896	13896	2314	16	0
13897	13897	2314	55.7	0
13898	13898	2314	50	0
13899	13899	2314	28.4	0
13900	13900	2314	294	0
13901	13901	2314	2.6	0
13902	13902	2314	2.3	0
13903	13903	2314	14	0
13904	13904	2314	10	0
13905	13905	2314	3	0
13906	13906	2314	90	0
13907	13907	2314	200	0
13908	13908	2314	200	0
13909	13909	2314	15	0
13910	13910	2314	25	0
13911	13911	2314	85	0
13912	13912	2314	1	0
13913	13913	2314	1.836666667	0
13914	13914	2314	26.51666667	0
13915	13915	2314	7.2	0
13916	13916	2314	50	0
13917	13917	2314	200	0
13918	13918	2314	2.47	0
13919	13919	2314	57	0
13920	13920	2314	79	0
13921	13921	2314	4.72	0
13922	13922	2314	0	0
13923	13923	2314	0	0
13924	13924	2314	25	0
13925	13925	2314	0	0
13926	13926	2314	0	0
13927	13927	2314	268	0
13928	13928	2314	8	0
13929	13929	2314	3	0
13930	13930	2314	50	0
13931	13931	2314	200	0
13932	13932	2314	2.47	0
13933	13933	2314	57	0
13934	13934	2314	79	0
13935	13935	2314	4.72	0
13936	13936	2314	0	0
13937	13937	2314	0	0
13938	13938	2314	25	0
13939	13939	2314	0	0
13940	13940	2314	0	0
13941	13941	2314	123.1	0
13942	13942	2314	37.3	0
13943	13943	2314	290.1	0
13944	13944	2314	76.9	0
13945	13945	2314	119.1	0
13946	13946	2314	30.9	0
13947	13947	2314	12.12	0
13948	13948	2314	2.42	0
13949	13949	2314	12.1573	0
13950	13950	2314	2.5278	0
13951	13951	2314	29	0
13952	13952	2314	3	0
13953	13953	2314	1	0
13954	13954	2314	2	0
13955	13955	2314	0.5	0
13956	13956	2314	0.8	0
13957	13957	2314	1	0
13958	13958	2314	1.1	0
13959	13959	2314	5	0
13960	13960	2314	1	0
13961	13961	2314	1	0
13962	13962	2314	0.5	0
13963	13963	2314	0.6	0
13964	13964	2314	0.8	0
13965	13965	2314	0.7	0
13966	13966	2314	0.9	0
13967	13967	2314	0.7	0
13968	13968	2314	0.9	0
13969	13969	2314	2	0
13970	13970	2314	21	0
13971	13971	2314	4	0
13972	13972	2314	1	0
13973	13973	2314	8	0
13974	13974	2314	5	0
13975	13975	2314	4	0
13976	13976	2314	3	0
13977	13977	2314	1	0
13978	13978	2314	0	0
13979	13979	2314	0	0
13980	13980	2314	0	0
13981	13981	2314	0	0
13982	13982	2314	0	0
13983	13983	2314	0	0
13984	13984	2314	0	0
13985	13985	2314	B0L7	0
13986	13986	2314	1	0
13987	13987	2314	1.1	0
13988	13988	2314	21.8	0
13989	13989	2314	13	0
13990	13990	2314	41.5	0
13991	13991	2314	32	0
13992	13992	2314	7.4	0
13993	13993	2314	158	0
13994	13994	2314	3.1	0
13995	13995	2314	2.6	0
13996	13996	2314	0	0
13997	13997	2314	0	0
13998	13998	2314	0	0
13999	13999	2314	73.5	0
14000	14000	2314	125	0
14001	14001	2314	125	0
14002	14002	2314	15	0
14003	14003	2314	30	0
14004	14004	2314	100	0
14005	14005	2314	0	0
14006	14006	2314	2.41	0
14007	14007	2314	35.36666667	0
14008	14008	2314	7.65	0
14009	14009	2314	225	0
14010	14010	2314	225	0
14011	14011	2314	2.54	0
14012	14012	2314	77	0
14013	14013	2314	33	0
14014	14014	2314	4.8	0
14015	14015	2314	0	0
14016	14016	2314	0	0
14017	14017	2314	90	0
14018	14018	2314	0	0
14019	14019	2314	0	0
14020	14020	2314	81	0
14021	14021	2314	55	0
14022	14022	2314	23	0
14023	14023	2314	225	0
14024	14024	2314	225	0
14025	14025	2314	2.54	0
14026	14026	2314	77	0
14027	14027	2314	33	0
14028	14028	2314	4.8	0
14029	14029	2314	0	0
14030	14030	2314	0	0
14031	14031	2314	90	0
14032	14032	2314	0	0
14033	14033	2314	0	0
14034	14034	2314	81.4	0
14035	14035	2314	24.5	0
14036	14036	2314	284.2	0
14037	14037	2314	98.6	0
14038	14038	2314	5.2	0
14039	14039	2314	5.1	0
14040	14040	2314	12.87	0
14041	14041	2314	2.37	0
14042	14042	2314	12.8945	0
14043	14043	2314	2.4737	0
14044	14044	2314	13	0
14045	14045	2314	32	0
14046	14046	2314	1	0
14047	14047	2314	2	0
14048	14048	2314	0.5	0
14049	14049	2314	0.6	0
14050	14050	2314	0.8	0
14051	14051	2314	0.8	0
14052	14052	2314	3	0
14053	14053	2314	0	0
14054	14054	2314	0.9	0
14055	14055	2314	0.4	0
14056	14056	2314	0.5	0
14057	14057	2314	0.6	0
14058	14058	2314	0.8	0
14059	14059	2314	1.1	0
14060	14060	2314		0
14061	14061	2314		0
14062	14062	2314	0	0
14063	14063	2314	11	0
14064	14064	2314	2	0
14065	14065	2314	2	0
14066	14066	2314	7	0
14067	14067	2314	3	0
14068	14068	2314	1	0
14069	14069	2314	0	0
14070	14070	2314	0	0
14071	14071	2314	0	0
14072	14072	2314	0	0
14073	14073	2314	0	0
14074	14074	2314	0	0
14075	14075	2314	0	0
14076	14076	2314	0	0
14077	14077	2314	0	0
14078	14078	2314	B0L6	0
14079	14079	2314	1.3	0
14080	14080	2314	1.35	0
14081	14081	2314	30.6	0
14082	14082	2314	15	0
14083	14083	2314	37.2	0
14084	14084	2314	19	0
14085	14085	2314	7.8	0
14086	14086	2314	141	0
14087	14087	2314	2.6	0
14088	14088	2314	1.9	0
14089	14089	2314	0	0
14090	14090	2314	0	0
14091	14091	2314	0	0
14092	14092	2314	67	0
14093	14093	2314	150	0
14094	14094	2314	150	0
14095	14095	2314	10	0
14096	14096	2314	0	0
14097	14097	2314	95	0
14098	14098	2314	0	0
14099	14099	2314	1.816666667	0
14100	14100	2314	24.98333333	0
14101	14101	2314	6.333333333	0
14102	14102	2314	200	0
14103	14103	2314	300	0
14104	14104	2314	3.53	0
14105	14105	2314	104	0
14106	14106	2314	54	0
14107	14107	2314	4.53	0
14108	14108	2314	0	0
14109	14109	2314	0	0
14110	14110	2314	75	0
14111	14111	2314	0	0
14112	14112	2314	0	0
14113	14113	2314	34	0
14114	14114	2314	31	0
14115	14115	2314	20	0
14116	14116	2314	200	0
14117	14117	2314	300	0
14118	14118	2314	3.53	0
14119	14119	2314	104	0
14120	14120	2314	54	0
14121	14121	2314	4.53	0
14122	14122	2314	0	0
14123	14123	2314	0	0
14124	14124	2314	75	0
14125	14125	2314	0	0
14126	14126	2314	0	0
14127	14127	2314	100.3	0
14128	14128	2314	42.2	0
14129	14129	2314	374.6	0
14130	14130	2314	127.1	0
14131	14131	2314	10.4	0
14132	14132	2314	6.7	0
14133	14133	2314	20.63	0
14134	14134	2314	3.55	0
14135	14135	2314	20.6722	0
14136	14136	2314	3.6838	0
14137	14137	2314	18	0
14138	14138	2314	8	0
14139	14139	2314	2	0
14140	14140	2314	2	0
14141	14141	2314	0.4	0
14142	14142	2314	0.4	0
14143	14143	2314	0.8	0
14144	14144	2314	0.9	0
14145	14145	2314	4	0
14146	14146	2314	0	0
14147	14147	2314	1.1	0
14148	14148	2314	0.4	0
14149	14149	2314	.	0
14150	14150	2314	.	0
14151	14151	2314	0.6	0
14152	14152	2314	0.7	0
14153	14153	2314	0.2	0
14154	14154	2314	0.3	0
14155	14155	2314	0	0
14156	14156	2314	10	0
14157	14157	2314	3	0
14158	14158	2314	1	0
14159	14159	2314	6	0
14160	14160	2314	2	0
14161	14161	2314	1	0
14162	14162	2314	1	0
14163	14163	2314	0	0
14164	14164	2314	0	0
14165	14165	2314	0	0
14166	14166	2314	0	0
14167	14167	2314	0	0
14168	14168	2314	0	0
14169	14169	2314	0	0
14170	14170	2314	0	0
14171	14171	2314	B0L4	0
14172	14172	2314	1.3	0
14173	14173	2314	1.5	0
14174	14174	2314	28.2	0
14175	14175	2314	9	0
14176	14176	2314	26.3	0
14177	14177	2314	11	0
14178	14178	2314	3.4	0
14179	14179	2314	58	0
14180	14180	2314	2.7	0
14181	14181	2314	2	0
14182	14182	2314	0	0
14183	14183	2314	0	0
14184	14184	2314	0	0
14185	14185	2314	56	0
14186	14186	2314	50	0
14187	14187	2314	50	0
14188	14188	2314	1	0
14189	14189	2314	0	0
14190	14190	2314	50	0
14191	14191	2314	2	0
14192	14192	2314	1.706666667	0
14193	14193	2314	24.61666667	0
14194	14194	2314	5.6	0
14195	14195	2314	150	0
14196	14196	2314	250	0
14197	14197	2314	2.11	0
14198	14198	2314	52.5	0
14199	14199	2314	42	0
14200	14200	2314	2.45	0
14201	14201	2314	0	0
14202	14202	2314	0	0
14203	14203	2314	70	0
14204	14204	2314	0	0
14205	14205	2314	0	0
14206	14206	2314	14	0
14207	14207	2314	4	0
14208	14208	2314	16	0
14209	14209	2314	150	0
14210	14210	2314	250	0
14211	14211	2314	2.11	0
14212	14212	2314	52.5	0
14213	14213	2314	42	0
14214	14214	2314	2.45	0
14215	14215	2314	0	0
14216	14216	2314	0	0
14217	14217	2314	70	0
14218	14218	2314	0	0
14219	14219	2314	0	0
14220	14220	2314	42.4	0
14221	14221	2314	13.7	0
14222	14222	2314	25.3	0
14223	14223	2314	18.9	0
14224	14224	2314	3	0
14225	14225	2314	2.4	0
14226	14226	2314	3.84	0
14227	14227	2314	0.74	0
14228	14228	2314	3.8537	0
14229	14229	2314	0.7613	0
14230	14230	2314	18	0
14231	14231	2314	38	0
14232	14232	2314	2	0
14233	14233	2314	3	0
14234	14234	2314	0.5	0
14235	14235	2314	0.6	0
14236	14236	2314	0.9	0
14237	14237	2314	0.8	0
14238	14238	2314	4	0
14239	14239	2314	0	0
14240	14240	2314	0.9	0
14241	14241	2314	0.4	0
14242	14242	2314	0.5	0
14243	14243	2314	0.6	0
14244	14244	2314	0.6	0
14245	14245	2314	0.7	0
14246	14246	2314	0.7	0
14247	14247	2314	0.8	0
14248	14248	2314	0	0
14249	14249	2314	12	0
14250	14250	2314	2	0
14251	14251	2314	2	0
14252	14252	2314	8	0
14253	14253	2314	3	0
14254	14254	2314	1	0
14255	14255	2314	0	0
14256	14256	2314	0	0
14257	14257	2314	0	0
14258	14258	2314	0	0
14259	14259	2314	0	0
14260	14260	2314	0	0
14261	14261	2314	0	0
14262	14262	2314	0	0
14263	14263	2314	0	0
14264	14264	2314	B0L6	0
14265	14265	2314	1	0
14266	14266	2314	1.1	0
14267	14267	2314	32.1	0
14268	14268	2314	16	0
14269	14269	2314	31.4	0
14270	14270	2314	18	0
14271	14271	2314	3.5	0
14272	14272	2314	90	0
14273	14273	2314	2.3	0
14274	14274	2314	2.2	0
14275	14275	2314	0	0
14276	14276	2314	0	0
14277	14277	2314	0	0
14278	14278	2314	94	0
14279	14279	2314	225	0
14280	14280	2314	225	0
14281	14281	2314	20	0
14282	14282	2314	30	0
14283	14283	2314	80	0
14284	14284	2314	0	0
14285	14285	2314	1.67	0
14286	14286	2314	19.88333333	0
14287	14287	2314	5.3	0
14288	14288	2314	350	0
14289	14289	2314	400	0
14290	14290	2314	3.35	0
14291	14291	2314	99	0
14292	14292	2314	38	0
14293	14293	2314	4.25	0
14294	14294	2314	0	0
14295	14295	2314	0	0
14296	14296	2314	70	0
14297	14297	2314	0	0
14298	14298	2314	0	0
14299	14299	2314	114	0
14300	14300	2314	81	0
14301	14301	2314	40	0
14302	14302	2314	350	0
14303	14303	2314	400	0
14304	14304	2314	3.35	0
14305	14305	2314	99	0
14306	14306	2314	38	0
14307	14307	2314	4.25	0
14308	14308	2314	0	0
14309	14309	2314	0	0
14310	14310	2314	70	0
14311	14311	2314	0	0
14312	14312	2314	0	0
14313	14313	2314	116.8	0
14314	14314	2314	31.4	0
14315	14315	2314	246.6	0
14316	14316	2314	56.7	0
14317	14317	2314	117.7	0
14318	14318	2314	26.1	0
14319	14319	2314	16.89	0
14320	14320	2314	3.07	0
14321	14321	2314	16.9214	0
14322	14322	2314	3.1528	0
14323	14323	2314	17	0
14324	14324	2314	1	0
14325	14325	2314	1	0
14326	14326	2314	2	0
14327	14327	2314	0.5	0
14328	14328	2314	0.7	0
14329	14329	2314		0
14330	14330	2314		0
14331	14331	2314	3	0
14332	14332	2314	0	0
14333	14333	2314	1	0
14334	14334	2314	0.4	0
14335	14335	2314	0.6	0
14336	14336	2314	0.8	0
14337	14337	2314	0.7	0
14338	14338	2314	0.9	0
14339	14339	2314		0
14340	14340	2314		0
14341	14341	2314	2	0
14342	14342	2314	12	0
14343	14343	2314	3	0
14344	14344	2314	1	0
14345	14345	2314	7	0
14346	14346	2314	1	0
14347	14347	2314	3	0
14348	14348	2314	1	0
14349	14349	2314	0	0
14350	14350	2314	0	0
14351	14351	2314	0	0
14352	14352	2314	0	0
14353	14353	2314	0	0
14354	14354	2314	0	0
14355	14355	2314	0	0
14356	14356	2314	0	0
14357	14357	2314	B0L5	0
14358	14358	2314	1.4	0
14359	14359	2314	1.4	0
14360	14360	2314	25.5	0
14361	14361	2314	12	0
14362	14362	2314	38	0
14363	14363	2314	20	0
14364	14364	2314	11.5	0
14365	14365	2314	95	0
14366	14366	2314	3	0
14367	14367	2314	2.4	0
14368	14368	2314	0	0
14369	14369	2314	0	0
14370	14370	2314	0	0
14371	14371	2314	76	0
14372	14372	2314	200	0
14373	14373	2314	200	0
14374	14374	2314	5	0
14375	14375	2314	15	0
14376	14376	2314	85	0
14377	14377	2314	0	0
14378	14378	2314	2.791666667	0
14379	14379	2314	37.15	0
14380	14380	2314	8	0
14381	14381	2314	200	0
14382	14382	2314	300	0
14383	14383	2314	2.65	0
14384	14384	2314	80	0
14385	14385	2314	50	0
14386	14386	2314	3	0
14387	14387	2314	0	0
14388	14388	2314	0	0
14389	14389	2314	60	0
14390	14390	2314	0	0
14391	14391	2314	0	0
14392	14392	2314	29	0
14393	14393	2314	38	0
14394	14394	2314	35	0
14395	14395	2314	200	0
14396	14396	2314	300	0
14397	14397	2314	2.65	0
14398	14398	2314	80	0
14399	14399	2314	50	0
14400	14400	2314	3	0
14401	14401	2314	0	0
14402	14402	2314	0	0
14403	14403	2314	60	0
14404	14404	2314	0	0
14405	14405	2314	0	0
14406	14406	2314	146.2	0
14407	14407	2314	39.3	0
14408	14408	2314	143.4	0
14409	14409	2314	39.3	0
14410	14410	2314	13.9	0
14411	14411	2314	4.4	0
14412	14412	2314	22.53	0
14413	14413	2314	2.9	0
14414	14414	2314	22.5693	0
14415	14415	2314	2.9437	0
14416	14416	2314	24	0
14417	14417	2314	22	0
14418	14418	2314	1	0
14419	14419	2314	3	0
14420	14420	2314	0.6	0
14421	14421	2314	0.9	0
14422	14422	2314	1	0
14423	14423	2314	1.1	0
14424	14424	2314	4	0
14425	14425	2314	0	0
14426	14426	2314	0.8	0
14427	14427	2314	0.4	0
14428	14428	2314	0.6	0
14429	14429	2314	0.9	0
14430	14430	2314	0.7	0
14431	14431	2314	1	0
14432	14432	2314	0.8	0
14433	14433	2314	1.1	0
14434	14434	2314	1	0
14435	14435	2314	20	0
14436	14436	2314	4	0
14437	14437	2314	1	0
14438	14438	2314	7	0
14439	14439	2314	4	0
14440	14440	2314	4	0
14441	14441	2314	3	0
14442	14442	2314	2	0
14443	14443	2314	0	0
14444	14444	2314	0	0
14445	14445	2314	0	0
14446	14446	2314	0	0
14447	14447	2314	0	0
14448	14448	2314	0	0
14449	14449	2314	0	0
14450	14450	2314	B0L6	0
14451	14451	2314	1.1	0
14452	14452	2314	1.2	0
14453	14453	2314	23.1	0
14454	14454	2314	15	0
14455	14455	2314	45.3	0
14456	14456	2314	43	0
14457	14457	2314	21.8	0
14458	14458	2314	241	0
14459	14459	2314	3	0
14460	14460	2314	2.3	0
14461	14461	2314	33	0
14462	14462	2314	21	0
14463	14463	2314	18	0
14464	14464	2314	93	0
14465	14465	2314	175	0
14466	14466	2314	175	0
14467	14467	2314	10	0
14468	14468	2314	30	0
14469	14469	2314	80	0
14470	14470	2314	2	0
14471	14471	2314	1.895	0
14472	14472	2314	27.36666667	0
14473	14473	2314	6.766666667	0
14474	14474	2314	300	0
14475	14475	2314	325	0
14476	14476	2314	2.31	0
14477	14477	2314	91	0
14478	14478	2314	55	0
14479	14479	2314	3.78	0
14480	14480	2314	0	0
14481	14481	2314	0	0
14482	14482	2314	85	0
14483	14483	2314	0	0
14484	14484	2314	0	0
14485	14485	2314	129	0
14486	14486	2314	72	0
14487	14487	2314	36	0
14488	14488	2314	300	0
14489	14489	2314	325	0
14490	14490	2314	2.31	0
14491	14491	2314	91	0
14492	14492	2314	55	0
14493	14493	2314	3.78	0
14494	14494	2314	0	0
14495	14495	2314	0	0
14496	14496	2314	85	0
14497	14497	2314	0	0
14498	14498	2314	0	0
14499	14499	2314	177.6	0
14500	14500	2314	54.2	0
14501	14501	2314	120.9	0
14502	14502	2314	33.1	0
14503	14503	2314	20.1	0
14504	14504	2314	6	0
14505	14505	2314	15.85	0
14506	14506	2314	4.62	0
14507	14507	2314	15.9042	0
14508	14508	2314	4.6591	0
14509	14509	2314	31	0
14510	14510	2314	1	0
14511	14511	2314	1	0
14512	14512	2314	3	0
14513	14513	2314	0.6	0
14514	14514	2314	1	0
14515	14515	2314	1.3	0
14516	14516	2314	1.4	0
14517	14517	2314	4	0
14518	14518	2314	0	0
14519	14519	2314	1.2	0
14520	14520	2314	0.6	0
14521	14521	2314	0.8	0
14522	14522	2314	1	0
14523	14523	2314	0.9	0
14524	14524	2314	1.2	0
14525	14525	2314	0.8	0
14526	14526	2314	1.2	0
14527	14527	2314	1	0
14528	14528	2314	25	0
14529	14529	2314	7	0
14530	14530	2314	1	0
14531	14531	2314	8	0
14532	14532	2314	5	0
14533	14533	2314	4	0
14534	14534	2314	3	0
14535	14535	2314	2	0
14536	14536	2314	1	0
14537	14537	2314	0	0
14538	14538	2314	1	0
14539	14539	2314	1	0
14540	14540	2314	0	0
14541	14541	2314	0	0
14542	14542	2314	0	0
14543	14543	2314	B0L6	0
14544	14544	2314	1.2	0
14545	14545	2314	1.5	0
14546	14546	2314	25.9	0
14547	14547	2314	13	0
14548	14548	2314	44.3	0
14549	14549	2314	39	0
14550	14550	2314	16.2	0
14551	14551	2314	195	0
14552	14552	2314	2.8	0
14553	14553	2314	2.5	0
14554	14554	2314	21	0
14555	14555	2314	17	0
14556	14556	2314	9	0
14557	14557	2314	65.5	0
14558	14558	2314	100	0
14559	14559	2314	100	0
14560	14560	2314	5	0
14561	14561	2314	10	0
14562	14562	2314	90	0
14563	14563	2314	1	0
14564	14564	2314	1.831666667	0
14565	14565	2314	25.25	0
14566	14566	2314	4.2	0
14567	14567	2314	250	0
14568	14568	2314	100	0
14569	14569	2314	2.17	0
14570	14570	2314	73.5	0
14571	14571	2314	26	0
14572	14572	2314	1.99	0
14573	14573	2314	0	0
14574	14574	2314	0	0
14575	14575	2314	85	0
14576	14576	2314	0	0
14577	14577	2314	0	0
14578	14578	2314	51	0
14579	14579	2314	31	0
14580	14580	2314	7	0
14581	14581	2314	250	0
14582	14582	2314	100	0
14583	14583	2314	2.17	0
14584	14584	2314	73.5	0
14585	14585	2314	26	0
14586	14586	2314	1.99	0
14587	14587	2314	0	0
14588	14588	2314	0	0
14589	14589	2314	85	0
14590	14590	2314	0	0
14591	14591	2314	0	0
14592	14592	2314	68	0
14593	14593	2314	27.9	0
14594	14594	2314	29.1	0
14595	14595	2314	14.6	0
14596	14596	2314	5.8	0
14597	14597	2314	5.2	0
14598	14598	2314	6.42	0
14599	14599	2314	1.31	0
14600	14600	2314	6.4479	0
14601	14601	2314	1.3298	0
14602	14602	2314	61	0
14603	14603	2314	3	0
14604	14604	2314	2	0
14605	14605	2314	3	0
14606	14606	2314	0.8	0
14607	14607	2314	0.9	0
14608	14608	2314	1	0
14609	14609	2314	1	0
14610	14610	2314	5	0
14611	14611	2314	0	0
14612	14612	2314	0.9	0
14613	14613	2314	0.4	0
14614	14614	2314	0.7	0
14615	14615	2314	1	0
14616	14616	2314	0.6	0
14617	14617	2314	0.7	0
14618	14618	2314	0.9	0
14619	14619	2314	1	0
14620	14620	2314	0	0
14621	14621	2314	24	0
14622	14622	2314	6	0
14623	14623	2314	1	0
14624	14624	2314	8	0
14625	14625	2314	5	0
14626	14626	2314	4	0
14627	14627	2314	3	0
14628	14628	2314	2	0
14629	14629	2314	0	0
14630	14630	2314	0	0
14631	14631	2314	1	0
14632	14632	2314	1	0
14633	14633	2314	0	0
14634	14634	2314	0	0
14635	14635	2314	0	0
14636	14636	2314	B0L6	0
14637	14637	2314	1.2	0
14638	14638	2314	1.2	0
14639	14639	2314	15.7	0
14640	14640	2314	9	0
14641	14641	2314	36.6	0
14642	14642	2314	42	0
14643	14643	2314	21.2	0
14644	14644	2314	196	0
14645	14645	2314	2.7	0
14646	14646	2314	1.9	0
14647	14647	2314	0	0
14648	14648	2314	0	0
14649	14649	2314	0	0
14650	14650	2314	69.5	0
14651	14651	2314	100	0
14652	14652	2314	100	0
14653	14653	2314	5	0
14654	14654	2314	20	0
14655	14655	2314	90	0
14656	14656	2314	0	0
14657	14657	2314	2.521666667	0
14658	14658	2314	36.96666667	0
14659	14659	2314	6.783333333	0
14660	14660	2314	200	0
14661	14661	2314	200	0
14662	14662	2314	2.58	0
14663	14663	2314	84	0
14664	14664	2314	38	0
14665	14665	2314	3.83	0
14666	14666	2314	0	0
14667	14667	2314	0	0
14668	14668	2314	80	0
14669	14669	2314	0	0
14670	14670	2314	0	0
14671	14671	2314	99	0
14672	14672	2314	35	0
14673	14673	2314	28	0
14674	14674	2314	200	0
14675	14675	2314	200	0
14676	14676	2314	2.58	0
14677	14677	2314	84	0
14678	14678	2314	38	0
14679	14679	2314	3.83	0
14680	14680	2314	0	0
14681	14681	2314	0	0
14682	14682	2314	80	0
14683	14683	2314	0	0
14684	14684	2314	0	0
14685	14685	2314	88.9	0
14686	14686	2314	26.7	0
14687	14687	2314	171.9	0
14688	14688	2314	59	0
14689	14689	2314	7.6	0
14690	14690	2314	4.3	0
14691	14691	2314	9.92	0
14692	14692	2314	1.79	0
14693	14693	2314	9.9467	0
14694	14694	2314	1.8533	0
14695	14695	2314	7	0
14696	14696	2314	5	0
14697	14697	2314	1	0
14698	14698	2314	2	0
14699	14699	2314	0.4	0
14700	14700	2314	0.5	0
14701	14701	2314	0.7	0
14702	14702	2314	1	0
14703	14703	2314	3	0
14704	14704	2314	0	0
14705	14705	2314	0.9	0
14706	14706	2314	0.4	0
14707	14707	2314	0.5	0
14708	14708	2314	0.6	0
14709	14709	2314	0.5	0
14710	14710	2314	0.6	0
14711	14711	2314	0.5	0
14712	14712	2314	0.7	0
14713	14713	2314	0	0
14714	14714	2314	14	0
14715	14715	2314	4	0
14716	14716	2314	1	0
14717	14717	2314	6	0
14718	14718	2314	2	0
14719	14719	2314	3	0
14720	14720	2314	2	0
14721	14721	2314	1	0
14722	14722	2314	0	0
14723	14723	2314	0	0
14724	14724	2314	0	0
14725	14725	2314	0	0
14726	14726	2314	0	0
14727	14727	2314	0	0
14728	14728	2314	0	0
14729	14729	2314	B0L4	0
14730	14730	2314	1	0
14731	14731	2314	1.1	0
14732	14732	2314	25.2	0
14733	14733	2314	14	0
14734	14734	2314	34.5	0
14735	14735	2314	13	0
14736	14736	2314	4.3	0
14737	14737	2314	81	0
14738	14738	2314	2.1	0
14739	14739	2314	1.8	0
14740	14740	2314	14	0
14741	14741	2314	10	0
14742	14742	2314	3	0
14743	14743	2314	95	0
14744	14744	2314	100	0
14745	14745	2314	100	0
14746	14746	2314	10	0
14747	14747	2314	30	0
14748	14748	2314	95	0
14749	14749	2314	0	0
14750	14750	2314	2.071666667	0
14751	14751	2314	32.86666667	0
14752	14752	2314	8.7	0
14753	14753	2314	175	0
14754	14754	2314	175	0
14755	14755	2314	1.25	0
14756	14756	2314	133	0
14757	14757	2314	37	0
14758	14758	2314	2.79	0
14759	14759	2314	0	0
14760	14760	2314	0	0
14761	14761	2314	90	0
14762	14762	2314	0	0
14763	14763	2314	0	0
14764	14764	2314	55	0
14765	14765	2314	22	0
14766	14766	2314	17	0
14767	14767	2314	175	0
14768	14768	2314	175	0
14769	14769	2314	1.25	0
14770	14770	2314	133	0
14771	14771	2314	37	0
14772	14772	2314	2.79	0
14773	14773	2314	0	0
14774	14774	2314	0	0
14775	14775	2314	90	0
14776	14776	2314	0	0
14777	14777	2314	0	0
14778	14778	2314	87.8	0
14779	14779	2314	22.9	0
14780	14780	2314	58.36	0
14781	14781	2314	18.2	0
14782	14782	2314	5.9	0
14783	14783	2314	2.1	0
14784	14784	2314	5.51	0
14785	14785	2314	1.18	0
14786	14786	2314	5.5329	0
14787	14787	2314	1.2003	0
14788	14788	2314	13	0
14789	14789	2314	1	0
14790	14790	2314	1	0
14791	14791	2314	2	0
14792	14792	2314	0.6	0
14793	14793	2314	0.9	0
14794	14794	2314		0
14795	14795	2314		0
14796	14796	2314	3	0
14797	14797	2314	0	0
14798	14798	2314	0.9	0
14799	14799	2314	0.4	0
14800	14800	2314	0.8	0
14801	14801	2314	1	0
14802	14802	2314	0.7	0
14803	14803	2314	0.7	0
14804	14804	2314		0
14805	14805	2314		0
14806	14806	2314	0	0
14807	14807	2314	15	0
14808	14808	2314	3	0
14809	14809	2314	1	0
14810	14810	2314	7	0
14811	14811	2314	4	0
14812	14812	2314	3	0
14813	14813	2314	1	0
14814	14814	2314	0	0
14815	14815	2314	0	0
14816	14816	2314	0	0
14817	14817	2314	0	0
14818	14818	2314	0	0
14819	14819	2314	0	0
14820	14820	2314	0	0
14821	14821	2314	0	0
14822	14822	2314	B0L5	0
14823	14823	2314	1.3	0
14824	14824	2314	1.2	0
14825	14825	2314	26.1	0
14826	14826	2314	15	0
14827	14827	2314	45.7	0
14828	14828	2314	33	0
14829	14829	2314	8.5	0
14830	14830	2314	165	0
14831	14831	2314	2.7	0
14832	14832	2314	2.2	0
14833	14833	2314	20	0
14834	14834	2314	8	0
14835	14835	2314	8	0
14836	14836	2314	89	0
14837	14837	2314	50	0
14838	14838	2314	125	0
14839	14839	2314	5	0
14840	14840	2314	20	0
14841	14841	2314	70	0
14842	14842	2314	0	0
14843	14843	2314	1.265	0
14844	14844	2314	13.26666667	0
14845	14845	2314	4.466666667	0
14846	14846	2314	40	0
14847	14847	2314	100	0
14848	14848	2314	2.53	0
14849	14849	2314	76	0
14850	14850	2314	48	0
14851	14851	2314	3.22	0
14852	14852	2314	0	0
14853	14853	2314	0	0
14854	14854	2314	25	0
14855	14855	2314	0	0
14856	14856	2314	0	0
14857	14857	2314	77	0
14858	14858	2314	0	0
14859	14859	2314	0	0
14860	14860	2314	40	0
14861	14861	2314	100	0
14862	14862	2314	2.53	0
14863	14863	2314	76	0
14864	14864	2314	48	0
14865	14865	2314	3.22	0
14866	14866	2314	0	0
14867	14867	2314	0	0
14868	14868	2314	25	0
14869	14869	2314	0	0
14870	14870	2314	0	0
14871	14871	2314	106.9	0
14872	14872	2314	28.4	0
14873	14873	2314	84.4	0
14874	14874	2314	21.5	0
14875	14875	2314	30.5	0
14876	14876	2314	8.9	0
14877	14877	2314	5.73	0
14878	14878	2314	1.22	0
14879	14879	2314	5.7584	0
14880	14880	2314	1.2504	0
14881	14881	2314	10	0
14882	14882	2314	4	0
14883	14883	2314	1	0
14884	14884	2314	2	0
14885	14885	2314	0.6	0
14886	14886	2314	0.8	0
14887	14887	2314		0
14888	14888	2314		0
14889	14889	2314	3	0
14890	14890	2314	0	0
14891	14891	2314	0.9	0
14892	14892	2314	0.4	0
14893	14893	2314	0.7	0
14894	14894	2314	1	0
14895	14895	2314	0.7	0
14896	14896	2314	0.9	0
14897	14897	2314		0
14898	14898	2314		0
14899	14899	2314	2	0
14900	14900	2314	15	0
14901	14901	2314	3	0
14902	14902	2314	1	0
14903	14903	2314	7	0
14904	14904	2314	3	0
14905	14905	2314	3	0
14906	14906	2314	1	0
14907	14907	2314	1	0
14908	14908	2314	0	0
14909	14909	2314	0	0
14910	14910	2314	0	0
14911	14911	2314	0	0
14912	14912	2314	0	0
14913	14913	2314	0	0
14914	14914	2314	0	0
14915	14915	2314	B0L5	0
14916	14916	2314	1	0
14917	14917	2314	1.3	0
14918	14918	2314	28.6	0
14919	14919	2314	13	0
14920	14920	2314	33.9	0
14921	14921	2314	20	0
14922	14922	2314	6.3	0
14923	14923	2314	167	0
14924	14924	2314	2.6	0
14925	14925	2314	2.7	0
14926	14926	2314	0	0
14927	14927	2314	0	0
14928	14928	2314	0	0
14929	14929	2314	54	0
14930	14930	2314	250	0
14931	14931	2314	250	0
14932	14932	2314	20	0
14933	14933	2314	50	0
14934	14934	2314	100	0
14935	14935	2314	0	0
14936	14936	2314	2.228333333	0
14937	14937	2314	27.71666667	0
14938	14938	2314	7.9	0
14939	14939	2314	150	0
14940	14940	2314	175	0
14941	14941	2314	3.06	0
14942	14942	2314	83	0
14943	14943	2314	60	0
14944	14944	2314	4.81	0
14945	14945	2314	0	0
14946	14946	2314	0	0
14947	14947	2314	90	0
14948	14948	2314	0	0
14949	14949	2314	0	0
14950	14950	2314	255	0
14951	14951	2314	171	0
14952	14952	2314	48	0
14953	14953	2314	150	0
14954	14954	2314	175	0
14955	14955	2314	3.06	0
14956	14956	2314	83	0
14957	14957	2314	60	0
14958	14958	2314	4.81	0
14959	14959	2314	0	0
14960	14960	2314	0	0
14961	14961	2314	90	0
14962	14962	2314	0	0
14963	14963	2314	0	0
14964	14964	2314	106.6	0
14965	14965	2314	30.8	0
14966	14966	2314	134.9	0
14967	14967	2314	40.4	0
14968	14968	2314	26.6	0
14969	14969	2314	9.4	0
14970	14970	2314	19.84	0
14971	14971	2314	4.48	0
14972	14972	2314	19.8708	0
14973	14973	2314	4.5298	0
14974	14974	2314		0
14975	14975	2314		0
14976	14976	2314	0	0
14977	14977	2314	1	0
14978	14978	2314		0
14979	14979	2314		0
14980	14980	2314		0
14981	14981	2314		0
14982	14982	2314	1	0
14983	14983	2314	0	0
14984	14984	2314	.	0
14985	14985	2314	.	0
14986	14986	2314		0
14987	14987	2314		0
14988	14988	2314		0
14989	14989	2314		0
14990	14990	2314		0
14991	14991	2314		0
14992	14992	2314	0	0
14993	14993	2314	0	0
14994	14994	2314	0	0
14995	14995	2314	.	0
14996	14996	2314	0	0
14997	14997	2314	.	0
14998	14998	2314	.	0
14999	14999	2314	.	0
15000	15000	2314	.	0
15001	15001	2314	0	0
15002	15002	2314	0	0
15003	15003	2314	0	0
15004	15004	2314	0	0
15005	15005	2314	0	0
15006	15006	2314	0	0
15007	15007	2314	0	0
15008	15008	2314	.	0
15009	15009	2314	.	0
15010	15010	2314	.	0
15011	15011	2314	.	0
15012	15012	2314	0	0
15013	15013	2314	.	0
15014	15014	2314	0	0
15015	15015	2314	.	0
15016	15016	2314	0	0
15017	15017	2314	.	0
15018	15018	2314	.	0
15019	15019	2314	0	0
15020	15020	2314	0	0
15021	15021	2314	0	0
15022	15022	2314		0
15023	15023	2314		0
15024	15024	2314	0	0
15025	15025	2314		0
15026	15026	2314		0
15027	15027	2314		0
15028	15028	2314		0
15029	15029	2314		0
15030	15030	2314		0
15031	15031	2314		0
15032	15032	2314		0
15033	15033	2314		0
15034	15034	2314		0
15035	15035	2314		0
15036	15036	2314		0
15037	15037	2314		0
15038	15038	2314		0
15039	15039	2314		0
15040	15040	2314		0
15041	15041	2314		0
15042	15042	2314		0
15043	15043	2314		0
15044	15044	2314		0
15045	15045	2314		0
15046	15046	2314	.	0
15047	15047	2314	.	0
15048	15048	2314	.	0
15049	15049	2314	.	0
15050	15050	2314	.	0
15051	15051	2314	.	0
15052	15052	2314	.	0
15053	15053	2314	.	0
15054	15054	2314	.	0
15055	15055	2314	.	0
15056	15056	2314	.	0
15057	15057	2314	.	0
15058	15058	2314	.	0
15059	15059	2314	.	0
15060	15060	2314	.	0
15061	15061	2314	.	0
15062	15062	2314	.	0
15063	15063	2314		0
15064	15064	2314		0
15065	15065	2314	.	0
15066	15066	2314	.	0
15067	15067	2314	19	0
15068	15068	2314		0
15069	15069	2314	0	0
15070	15070	2314	0	0
15071	15071	2314		0
15072	15072	2314		0
15073	15073	2314		0
15074	15074	2314		0
15075	15075	2314	2	0
15076	15076	2314	0	0
15077	15077	2314	0.7	0
15078	15078	2314	0.4	0
15079	15079	2314	0.5	0
15080	15080	2314	0.9	0
15081	15081	2314		0
15082	15082	2314		0
15083	15083	2314		0
15084	15084	2314		0
15085	15085	2314	2	0
15086	15086	2314	10	0
15087	15087	2314	2	0
15088	15088	2314	1	0
15089	15089	2314	6	0
15090	15090	2314	2	0
15091	15091	2314	2	0
15092	15092	2314	0	0
15093	15093	2314	0	0
15094	15094	2314	0	0
15095	15095	2314	0	0
15096	15096	2314	0	0
15097	15097	2314	0	0
15098	15098	2314	0	0
15099	15099	2314	0	0
15100	15100	2314	0	0
15101	15101	2314	B0L5	0
15102	15102	2314	0.9	0
15103	15103	2314	1.2	0
15104	15104	2314	9.5	0
15105	15105	2314	6	0
15106	15106	2314	23.6	0
15107	15107	2314	11	0
15108	15108	2314	4.8	0
15109	15109	2314	107	0
15110	15110	2314	2.5	0
15111	15111	2314	2.4	0
15112	15112	2314	0	0
15113	15113	2314	0	0
15114	15114	2314	0	0
15115	15115	2314	63.5	0
15116	15116	2314	200	0
15117	15117	2314	200	0
15118	15118	2314	20	0
15119	15119	2314	15	0
15120	15120	2314	100	0
15121	15121	2314	0	0
15122	15122	2314	2.516666667	0
15123	15123	2314	32.4	0
15124	15124	2314	7.65	0
15125	15125	2314	250	0
15126	15126	2314	250	0
15127	15127	2314	3.41	0
15128	15128	2314	75	0
15129	15129	2314	35	0
15130	15130	2314	4.14	0
15131	15131	2314	0	0
15132	15132	2314	0	0
15133	15133	2314	80	0
15134	15134	2314	0	0
15135	15135	2314	0	0
15136	15136	2314	40	0
15137	15137	2314	64	0
15138	15138	2314	68	0
15139	15139	2314	250	0
15140	15140	2314	250	0
15141	15141	2314	3.41	0
15142	15142	2314	75	0
15143	15143	2314	35	0
15144	15144	2314	4.14	0
15145	15145	2314	0	0
15146	15146	2314	0	0
15147	15147	2314	80	0
15148	15148	2314	0	0
15149	15149	2314	0	0
15150	15150	2314	111.9	0
15151	15151	2314	41.9	0
15152	15152	2314	291.2	0
15153	15153	2314	105.4	0
15154	15154	2314	14.3	0
15155	15155	2314	9.9	0
15156	15156	2314	14.37	0
15157	15157	2314	4	0
15158	15158	2314	14.4119	0
15159	15159	2314	4.1153	0
15160	15160	2314	13	0
15161	15161	2314	1	0
15162	15162	2314	1	0
15163	15163	2314	2	0
15164	15164	2314	0.4	0
15165	15165	2314	0.5	0
15166	15166	2314		0
15167	15167	2314		0
15168	15168	2314	2	0
15169	15169	2314	0	0
15170	15170	2314	0.7	0
15171	15171	2314	0.3	0
15172	15172	2314	0.5	0
15173	15173	2314	0.5	0
15174	15174	2314	0.4	0
15175	15175	2314	0.5	0
15176	15176	2314		0
15177	15177	2314		0
15178	15178	2314	2	0
15179	15179	2314	6	0
15180	15180	2314	1	0
15181	15181	2314	2	0
15182	15182	2314	5	0
15183	15183	2314	1	0
15184	15184	2314	0	0
15185	15185	2314	0	0
15186	15186	2314	0	0
15187	15187	2314	0	0
15188	15188	2314	0	0
15189	15189	2314	0	0
15190	15190	2314	0	0
15191	15191	2314	0	0
15192	15192	2314	0	0
15193	15193	2314	0	0
15194	15194	2314	B0L4	0
15195	15195	2314	0.85	0
15196	15196	2314	1.1	0
15197	15197	2314	20.7	0
15198	15198	2314	11	0
15199	15199	2314	24.2	0
15200	15200	2314	11	0
15201	15201	2314	5.9	0
15202	15202	2314	87	0
15203	15203	2314	2.2	0
15204	15204	2314	2.3	0
15205	15205	2314	0	0
15206	15206	2314	0	0
15207	15207	2314	0	0
15208	15208	2314	59	0
15209	15209	2314	300	0
15210	15210	2314	300	0
15211	15211	2314	20	0
15212	15212	2314	40	0
15213	15213	2314	100	0
15214	15214	2314	0	0
15215	15215	2314	2.956666667	0
15216	15216	2314	34.21666667	0
15217	15217	2314	9	0
15218	15218	2314		0
15219	15219	2314		0
15220	15220	2314	2.57	0
15221	15221	2314	71	0
15222	15222	2314	43	0
15223	15223	2314	2.86	0
15224	15224	2314	18	0
15225	15225	2314	0	0
15226	15226	2314	90	0
15227	15227	2314	0	0
15228	15228	2314	0	0
15229	15229	2314	143	0
15230	15230	2314	96	0
15231	15231	2314	53	0
15232	15232	2314	.	0
15233	15233	2314	.	0
15234	15234	2314	2.57	0
15235	15235	2314	71	0
15236	15236	2314	43	0
15237	15237	2314	2.86	0
15238	15238	2314	18	0
15239	15239	2314	0	0
15240	15240	2314	90	0
15241	15241	2314	0	0
15242	15242	2314	0	0
15243	15243	2314	73.4	0
15244	15244	2314	23.4	0
15245	15245	2314	89.3	0
15246	15246	2314	43.5	0
15247	15247	2314	21	0
15248	15248	2314	13.7	0
15249	15249	2314	13.88	0
15250	15250	2314	2.62	0
15251	15251	2314	13.9034	0
15252	15252	2314	2.6772	0
15253	15253	2314	8	0
15254	15254	2314	2	0
15255	15255	2314	2	0
15256	15256	2314	3	0
15257	15257	2314	0.8	0
15258	15258	2314	1.2	0
15259	15259	2314	1.2	0
15260	15260	2314	1.1	0
15261	15261	2314	5	0
15262	15262	2314	0	0
15263	15263	2314	1.2	0
15264	15264	2314	0.6	0
15265	15265	2314	0.8	0
15266	15266	2314	1.2	0
15267	15267	2314	0.8	0
15268	15268	2314	1	0
15269	15269	2314	0.8	0
15270	15270	2314	1	0
15271	15271	2314	2	0
15272	15272	2314	24	0
15273	15273	2314	7	0
15274	15274	2314	1	0
15275	15275	2314	8	0
15276	15276	2314	5	0
15277	15277	2314	4	0
15278	15278	2314	3	0
15279	15279	2314	2	0
15280	15280	2314	0	0
15281	15281	2314	0	0
15282	15282	2314	1	0
15283	15283	2314	1	0
15284	15284	2314	0	0
15285	15285	2314	0	0
15286	15286	2314	0	0
15287	15287	2314	B0L6	0
15288	15288	2314	1.25	0
15289	15289	2314	1.55	0
15290	15290	2314	18.1	0
15291	15291	2314	11	0
15292	15292	2314	49	0
15293	15293	2314	51	0
15294	15294	2314	28.7	0
15295	15295	2314	281	0
15296	15296	2314	2.2	0
15297	15297	2314	1.9	0
15298	15298	2314	2	0
15299	15299	2314	6	0
15300	15300	2314	0	0
15301	15301	2314	62	0
15302	15302	2314	300	0
15303	15303	2314	300	0
15304	15304	2314	5	0
15305	15305	2314	50	0
15306	15306	2314	95	0
15307	15307	2314	0	0
15308	15308	2314	2.063333333	0
15309	15309	2314	28.93333333	0
15310	15310	2314	6.916666667	0
15311	15311	2314	150	0
15312	15312	2314	175	0
15313	15313	2314	2.58	0
15314	15314	2314	72	0
15315	15315	2314	49	0
15316	15316	2314	3.18	0
15317	15317	2314	39	0
15318	15318	2314	0	0
15319	15319	2314	60	0
15320	15320	2314	0	0
15321	15321	2314	0	0
15322	15322	2314	211	0
15323	15323	2314	80	0
15324	15324	2314	0	0
15325	15325	2314	150	0
15326	15326	2314	175	0
15327	15327	2314	2.58	0
15328	15328	2314	72	0
15329	15329	2314	49	0
15330	15330	2314	3.18	0
15331	15331	2314	39	0
15332	15332	2314	0	0
15333	15333	2314	60	0
15334	15334	2314	0	0
15335	15335	2314	0	0
15336	15336	2314	47.1	0
15337	15337	2314	16.8	0
15338	15338	2314	100.6	0
15339	15339	2314	42.8	0
15340	15340	2314	18.3	0
15341	15341	2314	8.1	0
15342	15342	2314	13.91	0
15343	15343	2314	2.77	0
15344	15344	2314	13.9268	0
15345	15345	2314	2.8209	0
15346	15346	2314	10	0
15347	15347	2314	6	0
15348	15348	2314	1	0
15349	15349	2314	2	0
15350	15350	2314	0.5	0
15351	15351	2314	0.8	0
15352	15352	2314		0
15353	15353	2314		0
15354	15354	2314	3	0
15355	15355	2314	0	0
15356	15356	2314	1	0
15357	15357	2314	0.4	0
15358	15358	2314	0.6	0
15359	15359	2314	0.8	0
15360	15360	2314	0.6	0
15361	15361	2314	0.7	0
15362	15362	2314		0
15363	15363	2314		0
15364	15364	2314	1	0
15365	15365	2314	18	0
15366	15366	2314	4	0
15367	15367	2314	1	0
15368	15368	2314	7	0
15369	15369	2314	3	0
15370	15370	2314	3	0
15371	15371	2314	2	0
15372	15372	2314	2	0
15373	15373	2314	1	0
15374	15374	2314	0	0
15375	15375	2314	0	0
15376	15376	2314	0	0
15377	15377	2314	0	0
15378	15378	2314	0	0
15379	15379	2314	0	0
15380	15380	2314	B0L4	0
15381	15381	2314	1.2	0
15382	15382	2314	1.4	0
15383	15383	2314	34.5	0
15384	15384	2314	14	0
15385	15385	2314	40.3	0
15386	15386	2314	32	0
15387	15387	2314	5.5	0
15388	15388	2314	150	0
15389	15389	2314	2.5	0
15390	15390	2314	2.4	0
15391	15391	2314	3	0
15392	15392	2314	7	0
15393	15393	2314	0	0
15394	15394	2314	73	0
15395	15395	2314	175	0
15396	15396	2314	175	0
15397	15397	2314	10	0
15398	15398	2314	30	0
15399	15399	2314	90	0
15400	15400	2314	0	0
15401	15401	2314	2.065	0
15402	15402	2314	23.11666667	0
15403	15403	2314	7.683333333	0
15404	15404	2314	100	0
15405	15405	2314	125	0
15406	15406	2314	2.84	0
15407	15407	2314	80	0
15408	15408	2314	48	0
15409	15409	2314	2.49	0
15410	15410	2314	0	0
15411	15411	2314	0	0
15412	15412	2314	80	0
15413	15413	2314	0	0
15414	15414	2314	0	0
15415	15415	2314	186	0
15416	15416	2314	28	0
15417	15417	2314	6	0
15418	15418	2314	100	0
15419	15419	2314	125	0
15420	15420	2314	2.84	0
15421	15421	2314	80	0
15422	15422	2314	48	0
15423	15423	2314	2.49	0
15424	15424	2314	0	0
15425	15425	2314	0	0
15426	15426	2314	80	0
15427	15427	2314	0	0
15428	15428	2314	0	0
15429	15429	2314	85	0
15430	15430	2314	31.4	0
15431	15431	2314	49.9	0
15432	15432	2314	17.7	0
15433	15433	2314	15.4	0
15434	15434	2314	6.5	0
15435	15435	2314	7.89	0
15436	15436	2314	2.31	0
15437	15437	2314	7.9214	0
15438	15438	2314	2.3342	0
15439	15439	2314	8	0
15440	15440	2314	23	0
15441	15441	2314	1	0
15442	15442	2314	2	0
15443	15443	2314	0.5	0
15444	15444	2314	0.8	0
15445	15445	2314	0.8	0
15446	15446	2314	0.8	0
15447	15447	2314	3	0
15448	15448	2314	0	0
15449	15449	2314	1	0
15450	15450	2314	0.35	0
15451	15451	2314	0.6	0
15452	15452	2314	1	0
15453	15453	2314	0.5	0
15454	15454	2314	0.7	0
15455	15455	2314		0
15456	15456	2314		0
15457	15457	2314	1	0
15458	15458	2314	18	0
15459	15459	2314	4	0
15460	15460	2314	1	0
15461	15461	2314	7	0
15462	15462	2314	3	0
15463	15463	2314	3	0
15464	15464	2314	2	0
15465	15465	2314	2	0
15466	15466	2314	1	0
15467	15467	2314	0	0
15468	15468	2314	0	0
15469	15469	2314	0	0
15470	15470	2314	0	0
15471	15471	2314	0	0
15472	15472	2314	0	0
15473	15473	2314	B0L6	0
15474	15474	2314	1.1	0
15475	15475	2314	1.35	0
15476	15476	2314	28.5	0
15477	15477	2314	14	0
15478	15478	2314	43.3	0
15479	15479	2314	33	0
15480	15480	2314	12.4	0
15481	15481	2314	205	0
15482	15482	2314	2.2	0
15483	15483	2314	2.2	0
15484	15484	2314	0	0
15485	15485	2314	0	0
15486	15486	2314	0	0
15487	15487	2314	99	0
15488	15488	2314	175	0
15489	15489	2314	175	0
15490	15490	2314	10	0
15491	15491	2314	25	0
15492	15492	2314	85	0
15493	15493	2314	0	0
15494	15494	2314	2.236666667	0
15495	15495	2314	27.13333333	0
15496	15496	2314	7.616666667	0
15497	15497	2314	125	0
15498	15498	2314	150	0
15499	15499	2314	2.54	0
15500	15500	2314	97	0
15501	15501	2314	50	0
15502	15502	2314	4.43	0
15503	15503	2314	0	0
15504	15504	2314	0	0
15505	15505	2314	70	0
15506	15506	2314	0	0
15507	15507	2314	0	0
15508	15508	2314	106	0
15509	15509	2314	47	0
15510	15510	2314	13	0
15511	15511	2314	125	0
15512	15512	2314	150	0
15513	15513	2314	2.54	0
15514	15514	2314	97	0
15515	15515	2314	50	0
15516	15516	2314	4.43	0
15517	15517	2314	0	0
15518	15518	2314	0	0
15519	15519	2314	70	0
15520	15520	2314	0	0
15521	15521	2314	0	0
15522	15522	2314	108.5	0
15523	15523	2314	29.6	0
15524	15524	2314	184.9	0
15525	15525	2314	55.8	0
15526	15526	2314	52.4	0
15527	15527	2314	18.6	0
15528	15528	2314	11.32	0
15529	15529	2314	2.05	0
15530	15530	2314	11.3496	0
15531	15531	2314	2.1244	0
15532	15532	2314	23	0
15533	15533	2314	4	0
15534	15534	2314	1	0
15535	15535	2314	2	0
15536	15536	2314	0.4	0
15537	15537	2314	0.5	0
15538	15538	2314	0.6	0
15539	15539	2314	0.7	0
15540	15540	2314	3	0
15541	15541	2314	0	0
15542	15542	2314	0.6	0
15543	15543	2314	0.3	0
15544	15544	2314	0.4	0
15545	15545	2314	0.5	0
15546	15546	2314	0.4	0
15547	15547	2314	0.4	0
15548	15548	2314		0
15549	15549	2314		0
15550	15550	2314	0	0
15551	15551	2314	13	0
15552	15552	2314	3	0
15553	15553	2314	1	0
15554	15554	2314	8	0
15555	15555	2314	2	0
15556	15556	2314	2	0
15557	15557	2314	1	0
15558	15558	2314	0	0
15559	15559	2314	0	0
15560	15560	2314	0	0
15561	15561	2314	0	0
15562	15562	2314	0	0
15563	15563	2314	0	0
15564	15564	2314	0	0
15565	15565	2314	0	0
15566	15566	2314	B0L6	0
15567	15567	2314	0.9	0
15568	15568	2314	1	0
15569	15569	2314	16.1	0
15570	15570	2314	6	0
15571	15571	2314	24.9	0
15572	15572	2314	19	0
15573	15573	2314	3.8	0
15574	15574	2314	88	0
15575	15575	2314	1.8	0
15576	15576	2314	1.7	0
15577	15577	2314	0	0
15578	15578	2314	0	0
15579	15579	2314	0	0
15580	15580	2314	57	0
15581	15581	2314	0	0
15582	15582	2314	90	0
15583	15583	2314	0	0
15584	15584	2314	2	0
15585	15585	2314	0	0
15586	15586	2314	0	0
15587	15587	2314		0
15588	15588	2314		0
15589	15589	2314		0
15590	15590	2314		0
15591	15591	2314		0
15592	15592	2314		0
15593	15593	2314		0
15594	15594	2314		0
15595	15595	2314		0
15596	15596	2314		0
15597	15597	2314		0
15598	15598	2314		0
15599	15599	2314		0
15600	15600	2314		0
15601	15601	2314	19	0
15602	15602	2314		0
15603	15603	2314		0
15604	15604	2314	.	0
15605	15605	2314	.	0
15606	15606	2314	.	0
15607	15607	2314	.	0
15608	15608	2314	.	0
15609	15609	2314	.	0
15610	15610	2314	.	0
15611	15611	2314	.	0
15612	15612	2314	.	0
15613	15613	2314	.	0
15614	15614	2314	.	0
15615	15615	2314	.	0
15616	15616	2314	.	0
15617	15617	2314	.	0
15618	15618	2314	.	0
15619	15619	2314	.	0
15620	15620	2314	.	0
15621	15621	2314	3.13	0
15622	15622	2314	0.36	0
15623	15623	2314	3.13	0
15624	15624	2314	0.36	0
15625	15625	2314	2	0
15626	15626	2314	7	0
15627	15627	2314	1	0
15628	15628	2314	2	0
15629	15629	2314	0.4	0
15630	15630	2314	0.5	0
15631	15631	2314		0
15632	15632	2314		0
15633	15633	2314	3	0
15634	15634	2314	0	0
15635	15635	2314	0.5	0
15636	15636	2314	0.2	0
15637	15637	2314	0.4	0
15638	15638	2314	0.6	0
15639	15639	2314	0.3	0
15640	15640	2314	0.5	0
15641	15641	2314		0
15642	15642	2314		0
15643	15643	2314	1	0
15644	15644	2314	11	0
15645	15645	2314	3	0
15646	15646	2314	1	0
15647	15647	2314	6	0
15648	15648	2314	1	0
15649	15649	2314	2	0
15650	15650	2314	1	0
15651	15651	2314	1	0
15652	15652	2314	0	0
15653	15653	2314	0	0
15654	15654	2314	0	0
15655	15655	2314	0	0
15656	15656	2314	0	0
15657	15657	2314	0	0
15658	15658	2314	0	0
15659	15659	2314	B0L4	0
15660	15660	2314	0.8	0
15661	15661	2314	1	0
15662	15662	2314	16.2	0
15663	15663	2314	8	0
15664	15664	2314	26.1	0
15665	15665	2314	14	0
15666	15666	2314	3.5	0
15667	15667	2314	71	0
15668	15668	2314	2.2	0
15669	15669	2314	1.9	0
15670	15670	2314	0	0
15671	15671	2314	0	0
15672	15672	2314	0	0
15673	15673	2314	81	0
15674	15674	2314	200	0
15675	15675	2314	200	0
15676	15676	2314	10	0
15677	15677	2314	50	0
15678	15678	2314	100	0
15679	15679	2314	0	0
15680	15680	2314	1.946666667	0
15681	15681	2314	22.65	0
15682	15682	2314	5.433333333	0
15683	15683	2314	300	0
15684	15684	2314	400	0
15685	15685	2314	2.27	0
15686	15686	2314	99	0
15687	15687	2314	32	0
15688	15688	2314	3.78	0
15689	15689	2314	0	0
15690	15690	2314	0	0
15691	15691	2314	80	0
15692	15692	2314	0	0
15693	15693	2314	0	0
15694	15694	2314	87	0
15695	15695	2314	31	0
15696	15696	2314	0	0
15697	15697	2314	300	0
15698	15698	2314	400	0
15699	15699	2314	2.27	0
15700	15700	2314	99	0
15701	15701	2314	32	0
15702	15702	2314	3.78	0
15703	15703	2314	0	0
15704	15704	2314	0	0
15705	15705	2314	80	0
15706	15706	2314	0	0
15707	15707	2314	0	0
15708	15708	2314	86.1	0
15709	15709	2314	26.1	0
15710	15710	2314	173.7	0
15711	15711	2314	63.5	0
15712	15712	2314	23.3	0
15713	15713	2314	15	0
15714	15714	2314	11.74	0
15715	15715	2314	0.37	0
15716	15716	2314	11.7661	0
15717	15717	2314	.	0
15718	15718	2314	26	0
15719	15719	2314	6	0
15720	15720	2314	2	0
15721	15721	2314	3	0
15722	15722	2314	0.5	0
15723	15723	2314	0.8	0
15724	15724	2314	1.1	0
15725	15725	2314	1.3	0
15726	15726	2314	3	0
15727	15727	2314	0	0
15728	15728	2314	1	0
15729	15729	2314	0.5	0
15730	15730	2314	0.6	0
15731	15731	2314	0.9	0
15732	15732	2314	0.8	0
15733	15733	2314	0.8	0
15734	15734	2314	0.7	0
15735	15735	2314	0.9	0
15736	15736	2314	2	0
15737	15737	2314	21	0
15738	15738	2314	5	0
15739	15739	2314	1	0
15740	15740	2314	8	0
15741	15741	2314	3	0
15742	15742	2314	4	0
15743	15743	2314	3	0
15744	15744	2314	2	0
15745	15745	2314	0	0
15746	15746	2314	0	0
15747	15747	2314	0	0
15748	15748	2314	0	0
15749	15749	2314	1	0
15750	15750	2314	0	0
15751	15751	2314	0	0
15752	15752	2314	B0L6	0
15753	15753	2314	1	0
15754	15754	2314	1.2	0
15755	15755	2314	21.5	0
15756	15756	2314	9	0
15757	15757	2314	31.2	0
15758	15758	2314	43	0
15759	15759	2314	5.8	0
15760	15760	2314	171	0
15761	15761	2314	2.5	0
15762	15762	2314	1.9	0
15763	15763	2314	15	0
15764	15764	2314	11	0
15765	15765	2314	0	0
15766	15766	2314	76	0
15767	15767	2314	80	0
15768	15768	2314	80	0
15769	15769	2314	5	0
15770	15770	2314	15	0
15771	15771	2314	75	0
15772	15772	2314	0	0
15773	15773	2314	1.555	0
15774	15774	2314	19.18333333	0
15775	15775	2314	5.583333333	0
15776	15776	2314	25	0
15777	15777	2314	50	0
15778	15778	2314	1.69	0
15779	15779	2314	76	0
15780	15780	2314	51	0
15781	15781	2314	3.91	0
15782	15782	2314	0	0
15783	15783	2314	0	0
15784	15784	2314	60	0
15785	15785	2314	0	0
15786	15786	2314	0	0
15787	15787	2314	68	0
15788	15788	2314	13	0
15789	15789	2314	4	0
15790	15790	2314	25	0
15791	15791	2314	50	0
15792	15792	2314	1.69	0
15793	15793	2314	76	0
15794	15794	2314	51	0
15795	15795	2314	3.91	0
15796	15796	2314	0	0
15797	15797	2314	0	0
15798	15798	2314	60	0
15799	15799	2314	0	0
15800	15800	2314	0	0
15801	15801	2314	59.3	0
15802	15802	2314	18.2	0
15803	15803	2314	109.3	0
15804	15804	2314	34.5	0
15805	15805	2314	29.6	0
15806	15806	2314	9.6	0
15807	15807	2314	3.67	0
15808	15808	2314	0.9	0
15809	15809	2314	3.6882	0
15810	15810	2314	0.9441	0
15811	15811	2314	25	0
15812	15812	2314	1	0
15813	15813	2314	0	0
15814	15814	2314	1	0
15815	15815	2314	0.3	0
15816	15816	2314	0.3	0
15817	15817	2314		0
15818	15818	2314		0
15819	15819	2314	2	0
15820	15820	2314	0	0
15821	15821	2314	0.8	0
15822	15822	2314	0.3	0
15823	15823	2314	0.3	0
15824	15824	2314	0.5	0
15825	15825	2314		0
15826	15826	2314		0
15827	15827	2314		0
15828	15828	2314		0
15829	15829	2314	0	0
15830	15830	2314	5	0
15831	15831	2314	0	0
15832	15832	2314	.	0
15833	15833	2314	5	0
15834	15834	2314	.	0
15835	15835	2314	.	0
15836	15836	2314	0	0
15837	15837	2314	0	0
15838	15838	2314	0	0
15839	15839	2314	0	0
15840	15840	2314	0	0
15841	15841	2314	0	0
15842	15842	2314	0	0
15843	15843	2314	0	0
15844	15844	2314	0	0
15845	15845	2314	B0L4	0
15846	15846	2314	0.65	0
15847	15847	2314	0.8	0
15848	15848	2314	15.8	0
15849	15849	2314	10	0
15850	15850	2314	18.3	0
15851	15851	2314	5	0
15852	15852	2314	0.9	0
15853	15853	2314	57	0
15854	15854	2314	1.8	0
15855	15855	2314	1.8	0
15856	15856	2314	0	0
15857	15857	2314	0	0
15858	15858	2314	0	0
15859	15859	2314	85	0
15860	15860	2314	150	0
15861	15861	2314	150	0
15862	15862	2314	15	0
15863	15863	2314	25	0
15864	15864	2314	100	0
15865	15865	2314	0	0
15866	15866	2314	1.93	0
15867	15867	2314	24.11666667	0
15868	15868	2314	5.55	0
15869	15869	2314	300	0
15870	15870	2314	400	0
15871	15871	2314	2.35	0
15872	15872	2314	94.5	0
15873	15873	2314	48	0
15874	15874	2314	3.71	0
15875	15875	2314	0	0
15876	15876	2314	0	0
15877	15877	2314	95	0
15878	15878	2314	0	0
15879	15879	2314	0	0
15880	15880	2314	41	0
15881	15881	2314	77	0
15882	15882	2314	25	0
15883	15883	2314	300	0
15884	15884	2314	400	0
15885	15885	2314	2.35	0
15886	15886	2314	94.5	0
15887	15887	2314	48	0
15888	15888	2314	3.71	0
15889	15889	2314	0	0
15890	15890	2314	0	0
15891	15891	2314	95	0
15892	15892	2314	0	0
15893	15893	2314	0	0
15894	15894	2314	87.4	0
15895	15895	2314	21.1	0
15896	15896	2314	148.1	0
15897	15897	2314	58.2	0
15898	15898	2314	4.4	0
15899	15899	2314	3	0
15900	15900	2314	7.85	0
15901	15901	2314	1.36	0
15902	15902	2314	7.8711	0
15903	15903	2314	1.4212	0
15904	15904	2314	9	0
15905	15905	2314		0
15906	15906	2314	0	0
15907	15907	2314	1	0
15908	15908	2314		0
15909	15909	2314		0
15910	15910	2314		0
15911	15911	2314		0
15912	15912	2314	2	0
15913	15913	2314	0	0
15914	15914	2314	1.1	0
15915	15915	2314	0.5	0
15916	15916	2314	.	0
15917	15917	2314	.	0
15918	15918	2314		0
15919	15919	2314		0
15920	15920	2314		0
15921	15921	2314		0
15922	15922	2314	2	0
15923	15923	2314	5	0
15924	15924	2314	0	0
15925	15925	2314	.	0
15926	15926	2314	5	0
15927	15927	2314	.	0
15928	15928	2314	.	0
15929	15929	2314	0	0
15930	15930	2314	0	0
15931	15931	2314	0	0
15932	15932	2314	0	0
15933	15933	2314	0	0
15934	15934	2314	0	0
15935	15935	2314	0	0
15936	15936	2314	0	0
15937	15937	2314	0	0
15938	15938	2314	B0L4	0
15939	15939	2314	0.4	0
15940	15940	2314	0.5	0
15941	15941	2314	25.2	0
15942	15942	2314	5	0
15943	15943	2314	8.7	0
15944	15944	2314	0	0
15945	15945	2314	.	0
15946	15946	2314	17	0
15947	15947	2314	2.5	0
15948	15948	2314	2.1	0
15949	15949	2314	0	0
15950	15950	2314	0	0
15951	15951	2314	0	0
15952	15952	2314	42	0
15953	15953	2314	50	0
15954	15954	2314	50	0
15955	15955	2314	10	0
15956	15956	2314	0	0
15957	15957	2314	100	0
15958	15958	2314	0	0
15959	15959	2314	3.491666667	0
15960	15960	2314	46.21666667	0
15961	15961	2314	9.766666667	0
15962	15962	2314	25	0
15963	15963	2314	50	0
15964	15964	2314	2.21	0
15965	15965	2314	109	0
15966	15966	2314	39	0
15967	15967	2314	2.23	0
15968	15968	2314	0	0
15969	15969	2314	0	0
15970	15970	2314	50	0
15971	15971	2314	0	0
15972	15972	2314	0	0
15973	15973	2314	20	0
15974	15974	2314	12	0
15975	15975	2314	0	0
15976	15976	2314	25	0
15977	15977	2314	50	0
15978	15978	2314	2.21	0
15979	15979	2314	109	0
15980	15980	2314	39	0
15981	15981	2314	2.23	0
15982	15982	2314	0	0
15983	15983	2314	0	0
15984	15984	2314	50	0
15985	15985	2314	0	0
15986	15986	2314	0	0
15987	15987	2314	88.2	0
15988	15988	2314	24.7	0
15989	15989	2314	72.3	0
15990	15990	2314	24.2	0
15991	15991	2314	17	0
15992	15992	2314	5.5	0
15993	15993	2314	3.42	0
15994	15994	2314	0.68	0
15995	15995	2314	3.4447	0
15996	15996	2314	0.7097	0
15997	15997	2314	4	0
15998	15998	2314	2	0
15999	15999	2314	1	0
16000	16000	2314	1	0
16001	16001	2314	0.5	0
16002	16002	2314	0.7	0
16003	16003	2314		0
16004	16004	2314		0
16005	16005	2314	3	0
16006	16006	2314	0	0
16007	16007	2314	0.7	0
16008	16008	2314	0.4	0
16009	16009	2314	0.6	0
16010	16010	2314	0.8	0
16011	16011	2314	0.4	0
16012	16012	2314	0.7	0
16013	16013	2314		0
16014	16014	2314		0
16015	16015	2314	2	0
16016	16016	2314	12	0
16017	16017	2314	3	0
16018	16018	2314	1	0
16019	16019	2314	6	0
16020	16020	2314	2	0
16021	16021	2314	2	0
16022	16022	2314	1	0
16023	16023	2314	1	0
16024	16024	2314	0	0
16025	16025	2314	0	0
16026	16026	2314	0	0
16027	16027	2314	0	0
16028	16028	2314	0	0
16029	16029	2314	0	0
16030	16030	2314	0	0
16031	16031	2314	B0L4	0
16032	16032	2314	1.5	0
16033	16033	2314	1.6	0
16034	16034	2314	27.2	0
16035	16035	2314	12	0
16036	16036	2314	30.9	0
16037	16037	2314	25	0
16038	16038	2314	3.5	0
16039	16039	2314	121	0
16040	16040	2314	3.2	0
16041	16041	2314	2.6	0
16042	16042	2314	0	0
16043	16043	2314	0	0
16044	16044	2314	0	0
16045	16045	2314	107	0
16046	16046	2314	175	0
16047	16047	2314	175	0
16048	16048	2314	10	0
16049	16049	2314	30	0
16050	16050	2314	80	0
16051	16051	2314	0	0
16052	16052	2314	2.846666667	0
16053	16053	2314	37.38333333	0
16054	16054	2314	9	0
16055	16055	2314	100	0
16056	16056	2314	200	0
16057	16057	2314	2.38	0
16058	16058	2314	107	0
16059	16059	2314	38	0
16060	16060	2314	3.91	0
16061	16061	2314	0	0
16062	16062	2314	0	0
16063	16063	2314		0
16064	16064	2314	0	0
16065	16065	2314	0	0
16066	16066	2314	83	0
16067	16067	2314	38	0
16068	16068	2314	9	0
16069	16069	2314	100	0
16070	16070	2314	200	0
16071	16071	2314	2.38	0
16072	16072	2314	107	0
16073	16073	2314	38	0
16074	16074	2314	3.91	0
16075	16075	2314	0	0
16076	16076	2314	0	0
16077	16077	2314	.	0
16078	16078	2314	0	0
16079	16079	2314	0	0
16080	16080	2314	103.5	0
16081	16081	2314	25.8	0
16082	16082	2314	61.1	0
16083	16083	2314	15.9	0
16084	16084	2314	13	0
16085	16085	2314	4.6	0
16086	16086	2314	8.67	0
16087	16087	2314	2.12	0
16088	16088	2314	8.6958	0
16089	16089	2314	2.1405	0
16090	16090	2314	20	0
16091	16091	2314	5	0
16092	16092	2314	0	0
16093	16093	2314	2	0
16094	16094	2314	0.2	0
16095	16095	2314	0.2	0
16096	16096	2314		0
16097	16097	2314		0
16098	16098	2314	3	0
16099	16099	2314	1	0
16100	16100	2314	1.4	0
16101	16101	2314	0.7	0
16102	16102	2314	.	0
16103	16103	2314	.	0
16104	16104	2314	0.7	0
16105	16105	2314	0.7	0
16106	16106	2314	0.6	0
16107	16107	2314	0.7	0
16108	16108	2314	2	0
16109	16109	2314	25	0
16110	16110	2314	7	0
16111	16111	2314	1	0
16112	16112	2314	7	0
16113	16113	2314	6	0
16114	16114	2314	6	0
16115	16115	2314	2	0
16116	16116	2314	0	0
16117	16117	2314	0	0
16118	16118	2314	0	0
16119	16119	2314	1	0
16120	16120	2314	1	0
16121	16121	2314	1	0
16122	16122	2314	1	0
16123	16123	2314	0	0
16124	16124	2314	B0L6	0
16125	16125	2314	0.9	0
16126	16126	2314	1	0
16127	16127	2314	30.5	0
16128	16128	2314	15	0
16129	16129	2314	43.5	0
16130	16130	2314	40	0
16131	16131	2314	28.5	0
16132	16132	2314	305	0
16133	16133	2314	2.8	0
16134	16134	2314	2.2	0
16135	16135	2314	0	0
16136	16136	2314	0	0
16137	16137	2314	0	0
16138	16138	2314	80	0
16139	16139	2314	250	0
16140	16140	2314	250	0
16141	16141	2314	5	0
16142	16142	2314	25	0
16143	16143	2314	75	0
16144	16144	2314	0	0
16145	16145	2314	2.186666667	0
16146	16146	2314	32.31666667	0
16147	16147	2314	8.433333333	0
16148	16148	2314	150	0
16149	16149	2314	200	0
16150	16150	2314	3.35	0
16151	16151	2314	92	0
16152	16152	2314	59	0
16153	16153	2314	4.36	0
16154	16154	2314	4	0
16155	16155	2314	0	0
16156	16156	2314	80	0
16157	16157	2314	0	0
16158	16158	2314	0	0
16159	16159	2314	51	0
16160	16160	2314	65	0
16161	16161	2314	54	0
16162	16162	2314	150	0
16163	16163	2314	200	0
16164	16164	2314	3.35	0
16165	16165	2314	92	0
16166	16166	2314	59	0
16167	16167	2314	4.36	0
16168	16168	2314	4	0
16169	16169	2314	0	0
16170	16170	2314	80	0
16171	16171	2314	0	0
16172	16172	2314	0	0
16173	16173	2314	168.4	0
16174	16174	2314	55.8	0
16175	16175	2314	295.1	0
16176	16176	2314	101.5	0
16177	16177	2314	78.2	0
16178	16178	2314	25.4	0
16179	16179	2314	24.35	0
16180	16180	2314	4.93	0
16181	16181	2314	24.4058	0
16182	16182	2314	5.0569	0
16183	16183	2314	6	0
16184	16184	2314	23	0
16185	16185	2314	1	0
16186	16186	2314	2	0
16187	16187	2314	0.5	0
16188	16188	2314	0.6	0
16189	16189	2314	0.7	0
16190	16190	2314	0.9	0
16191	16191	2314	3	0
16192	16192	2314	0	0
16193	16193	2314	1	0
16194	16194	2314	0.3	0
16195	16195	2314	0.5	0
16196	16196	2314	0.6	0
16197	16197	2314	0.6	0
16198	16198	2314	0.7	0
16199	16199	2314	0.6	0
16200	16200	2314	0.7	0
16201	16201	2314	2	0
16202	16202	2314	18	0
16203	16203	2314	4	0
16204	16204	2314	1	0
16205	16205	2314	7	0
16206	16206	2314	3	0
16207	16207	2314	3	0
16208	16208	2314	2	0
16209	16209	2314	2	0
16210	16210	2314	1	0
16211	16211	2314	0	0
16212	16212	2314	0	0
16213	16213	2314	0	0
16214	16214	2314	0	0
16215	16215	2314	0	0
16216	16216	2314	0	0
16217	16217	2314	B0L5	0
16218	16218	2314	1.1	0
16219	16219	2314	1.4	0
16220	16220	2314	18.5	0
16221	16221	2314	13	0
16222	16222	2314	39.3	0
16223	16223	2314	31	0
16224	16224	2314	11.5	0
16225	16225	2314	179	0
16226	16226	2314	2.8	0
16227	16227	2314	2.3	0
16228	16228	2314	15	0
16229	16229	2314	11	0
16230	16230	2314	3	0
16231	16231	2314	78	0
16232	16232	2314	250	0
16233	16233	2314	250	0
16234	16234	2314	5	0
16235	16235	2314	25	0
16236	16236	2314	90	0
16237	16237	2314	1	0
16238	16238	2314	3.2	0
16239	16239	2314	41.16666667	0
16240	16240	2314	10.33333333	0
16241	16241	2314	300	0
16242	16242	2314	400	0
16243	16243	2314	2.72	0
16244	16244	2314	88	0
16245	16245	2314	29	0
16246	16246	2314	3.91	0
16247	16247	2314	15	0
16248	16248	2314	0	0
16249	16249	2314	75	0
16250	16250	2314	0	0
16251	16251	2314	0	0
16252	16252	2314	119	0
16253	16253	2314	47	0
16254	16254	2314	15	0
16255	16255	2314	300	0
16256	16256	2314	400	0
16257	16257	2314	2.72	0
16258	16258	2314	88	0
16259	16259	2314	29	0
16260	16260	2314	3.91	0
16261	16261	2314	15	0
16262	16262	2314	0	0
16263	16263	2314	75	0
16264	16264	2314	0	0
16265	16265	2314	0	0
16266	16266	2314	77.6	0
16267	16267	2314	22.1	0
16268	16268	2314	72.7	0
16269	16269	2314	26.5	0
16270	16270	2314	14.5	0
16271	16271	2314	5.2	0
16272	16272	2314	11.14	0
16273	16273	2314	1.96	0
16274	16274	2314	11.1621	0
16275	16275	2314	1.9917	0
16276	16276	2314	2	0
16277	16277	2314	22	0
16278	16278	2314	0	0
16279	16279	2314	2	0
16280	16280	2314	0.5	0
16281	16281	2314	0.7	0
16282	16282	2314		0
16283	16283	2314		0
16284	16284	2314	3	0
16285	16285	2314	0	0
16286	16286	2314	0.9	0
16287	16287	2314	0.4	0
16288	16288	2314	0.9	0
16289	16289	2314	0.9	0
16290	16290	2314	0.4	0
16291	16291	2314	0.6	0
16292	16292	2314		0
16293	16293	2314		0
16294	16294	2314	1	0
16295	16295	2314	14	0
16296	16296	2314	3	0
16297	16297	2314	1	0
16298	16298	2314	8	0
16299	16299	2314	2	0
16300	16300	2314	2	0
16301	16301	2314	1	0
16302	16302	2314	1	0
16303	16303	2314	0	0
16304	16304	2314	0	0
16305	16305	2314	0	0
16306	16306	2314	0	0
16307	16307	2314	0	0
16308	16308	2314	0	0
16309	16309	2314	0	0
16310	16310	2314	B0L6	0
16311	16311	2314	1	0
16312	16312	2314	1.05	0
16313	16313	2314	19.9	0
16314	16314	2314	9	0
16315	16315	2314	26.8	0
16316	16316	2314	15	0
16317	16317	2314	3.7	0
16318	16318	2314	107	0
16319	16319	2314	2.4	0
16320	16320	2314	2.3	0
16321	16321	2314	0	0
16322	16322	2314	2	0
16323	16323	2314	0	0
16324	16324	2314	70	0
16325	16325	2314	150	0
16326	16326	2314	150	0
16327	16327	2314	5	0
16328	16328	2314	10	0
16329	16329	2314	95	0
16330	16330	2314	1	0
16331	16331	2314	2.576666667	0
16332	16332	2314	35.83333333	0
16333	16333	2314	9.733333333	0
16334	16334	2314	125	0
16335	16335	2314	125	0
16336	16336	2314	2.73	0
16337	16337	2314	68	0
16338	16338	2314	34	0
16339	16339	2314	3.99	0
16340	16340	2314	0	0
16341	16341	2314	0	0
16342	16342	2314	80	0
16343	16343	2314	0	0
16344	16344	2314	0	0
16345	16345	2314	43	0
16346	16346	2314	42	0
16347	16347	2314	7	0
16348	16348	2314	125	0
16349	16349	2314	125	0
16350	16350	2314	2.73	0
16351	16351	2314	68	0
16352	16352	2314	34	0
16353	16353	2314	3.99	0
16354	16354	2314	0	0
16355	16355	2314	0	0
16356	16356	2314	80	0
16357	16357	2314	0	0
16358	16358	2314	0	0
16359	16359	2314	71.1	0
16360	16360	2314	26.8	0
16361	16361	2314	83.3	0
16362	16362	2314	41.8	0
16363	16363	2314	6.5	0
16364	16364	2314	4.4	0
16365	16365	2314	8.86	0
16366	16366	2314	2.61	0
16367	16367	2314	8.8868	0
16368	16368	2314	2.6562	0
16369	16369	2314	60	0
16370	16370	2314	6	0
16371	16371	2314	1	0
16372	16372	2314	2	0
16373	16373	2314	0.7	0
16374	16374	2314	0.8	0
16375	16375	2314	0.7	0
16376	16376	2314	0.7	0
16377	16377	2314	3	0
16378	16378	2314	0	0
16379	16379	2314	0.9	0
16380	16380	2314	0.5	0
16381	16381	2314	0.7	0
16382	16382	2314	0.9	0
16383	16383	2314	0.6	0
16384	16384	2314	0.8	0
16385	16385	2314	0.6	0
16386	16386	2314	0.8	0
16387	16387	2314	2	0
16388	16388	2314	17	0
16389	16389	2314	4	0
16390	16390	2314	1	0
16391	16391	2314	7	0
16392	16392	2314	4	0
16393	16393	2314	3	0
16394	16394	2314	2	0
16395	16395	2314	1	0
16396	16396	2314	0	0
16397	16397	2314	0	0
16398	16398	2314	0	0
16399	16399	2314	0	0
16400	16400	2314	0	0
16401	16401	2314	0	0
16402	16402	2314	0	0
16403	16403	2314	B0L6	0
16404	16404	2314	1	0
16405	16405	2314	1.2	0
16406	16406	2314	21.4	0
16407	16407	2314	7	0
16408	16408	2314	38.9	0
16409	16409	2314	37	0
16410	16410	2314	5.6	0
16411	16411	2314	160	0
16412	16412	2314	2.9	0
16413	16413	2314	2.5	0
16414	16414	2314	0	0
16415	16415	2314	0	0
16416	16416	2314	0	0
16417	16417	2314	75	0
16418	16418	2314	100	0
16419	16419	2314	100	0
16420	16420	2314	5	0
16421	16421	2314	25	0
16422	16422	2314	70	0
16423	16423	2314	0	0
16424	16424	2314	1.73	0
16425	16425	2314	22.8	0
16426	16426	2314	5.5	0
16427	16427	2314	100	0
16428	16428	2314	150	0
16429	16429	2314	2.65	0
16430	16430	2314	97	0
16431	16431	2314	42	0
16432	16432	2314	3.4	0
16433	16433	2314	20	0
16434	16434	2314	0	0
16435	16435	2314	70	0
16436	16436	2314	0	0
16437	16437	2314	0	0
16438	16438	2314	68	0
16439	16439	2314	59	0
16440	16440	2314	5	0
16441	16441	2314	100	0
16442	16442	2314	150	0
16443	16443	2314	2.65	0
16444	16444	2314	97	0
16445	16445	2314	42	0
16446	16446	2314	3.4	0
16447	16447	2314	20	0
16448	16448	2314	0	0
16449	16449	2314	70	0
16450	16450	2314	0	0
16451	16451	2314	0	0
16452	16452	2314	64.2	0
16453	16453	2314	27	0
16454	16454	2314	104.5	0
16455	16455	2314	46.1	0
16456	16456	2314	6.9	0
16457	16457	2314	6	0
16458	16458	2314	8.1	0
16459	16459	2314	2.01	0
16460	16460	2314	8.127	0
16461	16461	2314	2.0621	0
16462	16462	2314	8	0
16463	16463	2314	33	0
16464	16464	2314	1	0
16465	16465	2314	3	0
16466	16466	2314	0.8	0
16467	16467	2314	1.1	0
16468	16468	2314	0.9	0
16469	16469	2314	1.1	0
16470	16470	2314	3	0
16471	16471	2314	0	0
16472	16472	2314	1	0
16473	16473	2314	0.5	0
16474	16474	2314	0.8	0
16475	16475	2314	1.1	0
16476	16476	2314	0.7	0
16477	16477	2314	0.7	0
16478	16478	2314	0.7	0
16479	16479	2314	0.9	0
16480	16480	2314	1	0
16481	16481	2314	19	0
16482	16482	2314	5	0
16483	16483	2314	1	0
16484	16484	2314	7	0
16485	16485	2314	4	0
16486	16486	2314	3	0
16487	16487	2314	3	0
16488	16488	2314	1	0
16489	16489	2314	0	0
16490	16490	2314	0	0
16491	16491	2314	1	0
16492	16492	2314	0	0
16493	16493	2314	0	0
16494	16494	2314	0	0
16495	16495	2314	0	0
16496	16496	2314	B0L5	0
16497	16497	2314	1.25	0
16498	16498	2314	1.3	0
16499	16499	2314	29.3	0
16500	16500	2314	14	0
16501	16501	2314	42.8	0
16502	16502	2314	59	0
16503	16503	2314	27.1	0
16504	16504	2314	285	0
16505	16505	2314	2.7	0
16506	16506	2314	2.3	0
16507	16507	2314	4	0
16508	16508	2314	4	0
16509	16509	2314	0	0
16510	16510	2314	91	0
16511	16511	2314	150	0
16512	16512	2314	150	0
16513	16513	2314	10	0
16514	16514	2314	20	0
16515	16515	2314	80	0
16516	16516	2314	2	0
16517	16517	2314	1.731666667	0
16518	16518	2314	22.58333333	0
16519	16519	2314	6.233333333	0
16520	16520	2314	100	0
16521	16521	2314	150	0
16522	16522	2314	2.66	0
16523	16523	2314	100	0
16524	16524	2314	61	0
16525	16525	2314	3.42	0
16526	16526	2314	21	0
16527	16527	2314	0	0
16528	16528	2314	75	0
16529	16529	2314	0	0
16530	16530	2314	0	0
16531	16531	2314	135	0
16532	16532	2314	37	0
16533	16533	2314	25	0
16534	16534	2314	100	0
16535	16535	2314	150	0
16536	16536	2314	2.66	0
16537	16537	2314	100	0
16538	16538	2314	61	0
16539	16539	2314	3.42	0
16540	16540	2314	21	0
16541	16541	2314	0	0
16542	16542	2314	75	0
16543	16543	2314	0	0
16544	16544	2314	0	0
16545	16545	2314	64.7	0
16546	16546	2314	24.8	0
16547	16547	2314	137	0
16548	16548	2314	43.5	0
16549	16549	2314	11.6	0
16550	16550	2314	5.2	0
16551	16551	2314	11.21	0
16552	16552	2314	2.44	0
16553	16553	2314	11.2348	0
16554	16554	2314	2.4887	0
16555	16555	2314	36	0
16556	16556	2314	2	0
16557	16557	2314	2	0
16558	16558	2314	3	0
16559	16559	2314	0.5	0
16560	16560	2314	0.6	0
16561	16561	2314	1.3	0
16562	16562	2314	1.4	0
16563	16563	2314	3	0
16564	16564	2314	0	0
16565	16565	2314	1	0
16566	16566	2314	0.5	0
16567	16567	2314	0.4	0
16568	16568	2314	0.6	0
16569	16569	2314	0.8	0
16570	16570	2314	0.9	0
16571	16571	2314	0.7	0
16572	16572	2314	0.9	0
16573	16573	2314	2	0
16574	16574	2314	25	0
16575	16575	2314	5	0
16576	16576	2314	1	0
16577	16577	2314	8	0
16578	16578	2314	4	0
16579	16579	2314	4	0
16580	16580	2314	3	0
16581	16581	2314	3	0
16582	16582	2314	2	0
16583	16583	2314	0	0
16584	16584	2314	1	0
16585	16585	2314	0	0
16586	16586	2314	0	0
16587	16587	2314	0	0
16588	16588	2314	0	0
16589	16589	2314	B0L6	0
16590	16590	2314	1.3	0
16591	16591	2314	1.6	0
16592	16592	2314	26	0
16593	16593	2314	12	0
16594	16594	2314	43.3	0
16595	16595	2314	38	0
16596	16596	2314	10.5	0
16597	16597	2314	199	0
16598	16598	2314	2.3	0
16599	16599	2314	1.9	0
16600	16600	2314	32	0
16601	16601	2314	19	0
16602	16602	2314	17	0
16603	16603	2314	76.5	0
16604	16604	2314	30	0
16605	16605	2314	30	0
16606	16606	2314	1	0
16607	16607	2314	1	0
16608	16608	2314	25	0
16609	16609	2314	5	0
16610	16610	2314	2.063333333	0
16611	16611	2314	30.35	0
16612	16612	2314	7.333333333	0
16613	16613	2314	50	0
16614	16614	2314	150	0
16615	16615	2314	2.97	0
16616	16616	2314	85	0
16617	16617	2314	39	0
16618	16618	2314	2.95	0
16619	16619	2314	0	0
16620	16620	2314	0	0
16621	16621	2314	20	0
16622	16622	2314	5	0
16623	16623	2314	0	0
16624	16624	2314	41	0
16625	16625	2314	9	0
16626	16626	2314	14	0
16627	16627	2314	50	0
16628	16628	2314	150	0
16629	16629	2314	2.97	0
16630	16630	2314	85	0
16631	16631	2314	39	0
16632	16632	2314	2.95	0
16633	16633	2314	0	0
16634	16634	2314	0	0
16635	16635	2314	20	0
16636	16636	2314	5	0
16637	16637	2314	0	0
16638	16638	2314	90.7	0
16639	16639	2314	33.4	0
16640	16640	2314	105.1	0
16641	16641	2314	43.5	0
16642	16642	2314	6.6	0
16643	16643	2314	5.9	0
16644	16644	2314	4.32	0
16645	16645	2314	0.53	0
16646	16646	2314	4.3534	0
16647	16647	2314	0.5794	0
16648	16648	2314	6	0
16649	16649	2314	1	0
16650	16650	2314	1	0
16651	16651	2314	2	0
16652	16652	2314	0.4	0
16653	16653	2314	0.8	0
16654	16654	2314	0.9	0
16655	16655	2314	1	0
16656	16656	2314	3	0
16657	16657	2314	0	0
16658	16658	2314	0.7	0
16659	16659	2314	0.35	0
16660	16660	2314	0.5	0
16661	16661	2314	0.8	0
16662	16662	2314	0.6	0
16663	16663	2314	0.7	0
16664	16664	2314		0
16665	16665	2314		0
16666	16666	2314	1	0
16667	16667	2314	16	0
16668	16668	2314	4	0
16669	16669	2314	1	0
16670	16670	2314	7	0
16671	16671	2314	3	0
16672	16672	2314	3	0
16673	16673	2314	1	0
16674	16674	2314	1	0
16675	16675	2314	1	0
16676	16676	2314	0	0
16677	16677	2314	0	0
16678	16678	2314	0	0
16679	16679	2314	0	0
16680	16680	2314	0	0
16681	16681	2314	0	0
16682	16682	2314	B0L6	0
16683	16683	2314	1.2	0
16684	16684	2314	1.1	0
16685	16685	2314	29	0
16686	16686	2314	12	0
16687	16687	2314	42.3	0
16688	16688	2314	41	0
16689	16689	2314	20.1	0
16690	16690	2314	186	0
16691	16691	2314	2.9	0
16692	16692	2314	2.4	0
16693	16693	2314	17	0
16694	16694	2314	14	0
16695	16695	2314	6	0
16696	16696	2314	73	0
16697	16697	2314	50	0
16698	16698	2314	100	0
16699	16699	2314	5	0
16700	16700	2314	10	0
16701	16701	2314	70	0
16702	16702	2314	2	0
16703	16703	2314	2.026666667	0
16704	16704	2314	30.76666667	0
16705	16705	2314	6.783333333	0
16706	16706	2314	75	0
16707	16707	2314	125	0
16708	16708	2314	2.28	0
16709	16709	2314	93	0
16710	16710	2314	69	0
16711	16711	2314	3.25	0
16712	16712	2314	0	0
16713	16713	2314	0	0
16714	16714	2314	70	0
16715	16715	2314	0	0
16716	16716	2314	0	0
16717	16717	2314	98	0
16718	16718	2314	44	0
16719	16719	2314	21	0
16720	16720	2314	75	0
16721	16721	2314	125	0
16722	16722	2314	2.28	0
16723	16723	2314	93	0
16724	16724	2314	69	0
16725	16725	2314	3.25	0
16726	16726	2314	0	0
16727	16727	2314	0	0
16728	16728	2314	70	0
16729	16729	2314	0	0
16730	16730	2314	0	0
16731	16731	2314	121.9	0
16732	16732	2314	26.7	0
16733	16733	2314	123.8	0
16734	16734	2314	35.6	0
16735	16735	2314	11.4	0
16736	16736	2314	4.6	0
16737	16737	2314	9.03	0
16738	16738	2314	1.66	0
16739	16739	2314	9.0567	0
16740	16740	2314	1.7002	0
16741	16741	2314	20	0
16742	16742	2314	17	0
16743	16743	2314	1	0
16744	16744	2314	2	0
16745	16745	2314	0.3	0
16746	16746	2314	0.4	0
16747	16747	2314	0.6	0
16748	16748	2314	0.6	0
16749	16749	2314	3	0
16750	16750	2314	0	0
16751	16751	2314	0.8	0
16752	16752	2314	0.5	0
16753	16753	2314	0.4	0
16754	16754	2314	0.5	0
16755	16755	2314	0.5	0
16756	16756	2314	0.6	0
16757	16757	2314		0
16758	16758	2314		0
16759	16759	2314	1	0
16760	16760	2314	10	0
16761	16761	2314	2	0
16762	16762	2314	2	0
16763	16763	2314	7	0
16764	16764	2314	2	0
16765	16765	2314	1	0
16766	16766	2314	0	0
16767	16767	2314	0	0
16768	16768	2314	0	0
16769	16769	2314	0	0
16770	16770	2314	0	0
16771	16771	2314	0	0
16772	16772	2314	0	0
16773	16773	2314	0	0
16774	16774	2314	0	0
16775	16775	2314	B0L6	0
16776	16776	2314	0.9	0
16777	16777	2314	1.2	0
16778	16778	2314	25.4	0
16779	16779	2314	13	0
16780	16780	2314	30.5	0
16781	16781	2314	22	0
16782	16782	2314	4.3	0
16783	16783	2314	106	0
16784	16784	2314	2.7	0
16785	16785	2314	2.4	0
16786	16786	2314	0	0
16787	16787	2314	0	0
16788	16788	2314	0	0
16789	16789	2314	43	0
16790	16790	2314	30	0
16791	16791	2314	100	0
16792	16792	2314	5	0
16793	16793	2314	20	0
16794	16794	2314	50	0
16795	16795	2314	0	0
16796	16796	2314	1.386666667	0
16797	16797	2314	22.08333333	0
16798	16798	2314	5.266666667	0
16799	16799	2314	100	0
16800	16800	2314	135	0
16801	16801	2314	2.18	0
16802	16802	2314	62	0
16803	16803	2314	38	0
16804	16804	2314	3.16	0
16805	16805	2314	30	0
16806	16806	2314	0	0
16807	16807	2314	60	0
16808	16808	2314	0	0
16809	16809	2314	80	0
16810	16810	2314	45	0
16811	16811	2314	32	0
16812	16812	2314	12	0
16813	16813	2314	100	0
16814	16814	2314	135	0
16815	16815	2314	2.18	0
16816	16816	2314	62	0
16817	16817	2314	38	0
16818	16818	2314	3.16	0
16819	16819	2314	30	0
16820	16820	2314	0	0
16821	16821	2314	60	0
16822	16822	2314	0	0
16823	16823	2314	80	0
16824	16824	2314	102.9	0
16825	16825	2314	32.2	0
16826	16826	2314	112.5	0
16827	16827	2314	44.9	0
16828	16828	2314	10	0
16829	16829	2314	8.5	0
16830	16830	2314	6.13	0
16831	16831	2314	1	0
16832	16832	2314	6.1622	0
16833	16833	2314	1.0534	0
16834	16834	2314	17	0
16835	16835	2314	4	0
16836	16836	2314	1	0
16837	16837	2314	0	0
16838	16838	2314		0
16839	16839	2314		0
16840	16840	2314		0
16841	16841	2314		0
16842	16842	2314	1	0
16843	16843	2314	0	0
16844	16844	2314	1	0
16845	16845	2314	0.6	0
16846	16846	2314	.	0
16847	16847	2314	.	0
16848	16848	2314	0.5	0
16849	16849	2314	0.5	0
16850	16850	2314		0
16851	16851	2314		0
16852	16852	2314	1	0
16853	16853	2314	7	0
16854	16854	2314	1	0
16855	16855	2314	2	0
16856	16856	2314	6	0
16857	16857	2314	1	0
16858	16858	2314	0	0
16859	16859	2314	0	0
16860	16860	2314	0	0
16861	16861	2314	0	0
16862	16862	2314	0	0
16863	16863	2314	0	0
16864	16864	2314	0	0
16865	16865	2314	0	0
16866	16866	2314	0	0
16867	16867	2314	0	0
16868	16868	2314	B0L4	0
16869	16869	2314	1	0
16870	16870	2314	1.3	0
16871	16871	2314	25.2	0
16872	16872	2314	11	0
16873	16873	2314	23.1	0
16874	16874	2314	5	0
16875	16875	2314	1.7	0
16876	16876	2314	65	0
16877	16877	2314	2.5	0
16878	16878	2314	2.2	0
16879	16879	2314	0	0
16880	16880	2314	0	0
16881	16881	2314	0	0
16882	16882	2314	64	0
16883	16883	2314	175	0
16884	16884	2314	175	0
16885	16885	2314	10	0
16886	16886	2314	40	0
16887	16887	2314	95	0
16888	16888	2314	0	0
16889	16889	2314	2.405	0
16890	16890	2314	27.96666667	0
16891	16891	2314	6.983333333	0
16892	16892	2314	300	0
16893	16893	2314	320	0
16894	16894	2314	2.71	0
16895	16895	2314	84.5	0
16896	16896	2314	37.5	0
16897	16897	2314	3.11	0
16898	16898	2314	0	0
16899	16899	2314	0	0
16900	16900	2314	95	0
16901	16901	2314	15	0
16902	16902	2314	0	0
16903	16903	2314	79	0
16904	16904	2314	63	0
16905	16905	2314	20	0
16906	16906	2314	300	0
16907	16907	2314	320	0
16908	16908	2314	2.71	0
16909	16909	2314	84.5	0
16910	16910	2314	37.5	0
16911	16911	2314	3.11	0
16912	16912	2314	0	0
16913	16913	2314	0	0
16914	16914	2314	95	0
16915	16915	2314	15	0
16916	16916	2314	0	0
16917	16917	2314	85.8	0
16918	16918	2314	43.6	0
16919	16919	2314	89.1	0
16920	16920	2314	53	0
16921	16921	2314	8.5	0
16922	16922	2314	7.4	0
16923	16923	2314	10.21	0
16924	16924	2314	2.05	0
16925	16925	2314	10.2536	0
16926	16926	2314	2.1104	0
16927	16927	2314		0
16928	16928	2314		0
16929	16929	2314	0	0
16930	16930	2314	0	0
16931	16931	2314		0
16932	16932	2314		0
16933	16933	2314		0
16934	16934	2314		0
16935	16935	2314	0	0
16936	16936	2314	0	0
16937	16937	2314		0
16938	16938	2314		0
16939	16939	2314		0
16940	16940	2314		0
16941	16941	2314		0
16942	16942	2314		0
16943	16943	2314		0
16944	16944	2314		0
16945	16945	2314	1	0
16946	16946	2314	2	0
16947	16947	2314	0	0
16948	16948	2314	.	0
16949	16949	2314	2	0
16950	16950	2314	0	0
16951	16951	2314	0	0
16952	16952	2314	0	0
16953	16953	2314	0	0
16954	16954	2314	0	0
16955	16955	2314	0	0
16956	16956	2314	0	0
16957	16957	2314	0	0
16958	16958	2314	0	0
16959	16959	2314	0	0
16960	16960	2314	0	0
16961	16961	2314	B0L2	0
16962	16962	2314	0.3	0
16963	16963	2314	0.4	0
16964	16964	2314	7.6	0
16965	16965	2314	2	0
16966	16966	2314	2.7	0
16967	16967	2314	0	0
16968	16968	2314	.	0
16969	16969	2314	11	0
16970	16970	2314	1.1	0
16971	16971	2314	1.2	0
16972	16972	2314	0	0
16973	16973	2314	0	0
16974	16974	2314	0	0
16975	16975	2314	14.5	0
16976	16976	2314	40	0
16977	16977	2314	40	0
16978	16978	2314	0	0
16979	16979	2314	0	0
16980	16980	2314	100	0
16981	16981	2314	0	0
16982	16982	2314	1.566666667	0
16983	16983	2314	21.96666667	0
16984	16984	2314	4.166666667	0
16985	16985	2314	50	0
16986	16986	2314	50	0
16987	16987	2314	1.76	0
16988	16988	2314	34	0
16989	16989	2314	45	0
16990	16990	2314	2.32	0
16991	16991	2314	36	0
16992	16992	2314	0	0
16993	16993	2314	100	0
16994	16994	2314	0	0
16995	16995	2314	10	0
16996	16996	2314	0	0
16997	16997	2314	0	0
16998	16998	2314	0	0
16999	16999	2314	50	0
17000	17000	2314	50	0
17001	17001	2314	1.76	0
17002	17002	2314	34	0
17003	17003	2314	45	0
17004	17004	2314	2.32	0
17005	17005	2314	36	0
17006	17006	2314	0	0
17007	17007	2314	100	0
17008	17008	2314	0	0
17009	17009	2314	10	0
17010	17010	2314	30.3	0
17011	17011	2314	13.5	0
17012	17012	2314	58.9	0
17013	17013	2314	30.1	0
17014	17014	2314	2.5	0
17015	17015	2314	2.4	0
17016	17016	2314	2.85	0
17017	17017	2314	0.43	0
17018	17018	2314	2.8635	0
17019	17019	2314	0.4625	0
17020	17020	2314	.	0
17021	17021	2314	15	0
17022	17022	2314	0	0
17023	17023	2314	0	0
17024	17024	2314		0
17025	17025	2314		0
17026	17026	2314		0
17027	17027	2314		0
17028	17028	2314	2	0
17029	17029	2314	0	0
17030	17030	2314	1	0
17031	17031	2314	0.4	0
17032	17032	2314	0.5	0
17033	17033	2314	0.7	0
17034	17034	2314		0
17035	17035	2314		0
17036	17036	2314		0
17037	17037	2314		0
17038	17038	2314	2	0
17039	17039	2314	4	0
17040	17040	2314	0	0
17041	17041	2314	.	0
17042	17042	2314	4	0
17043	17043	2314	0	0
17044	17044	2314	0	0
17045	17045	2314	0	0
17046	17046	2314	0	0
17047	17047	2314	0	0
17048	17048	2314	0	0
17049	17049	2314	0	0
17050	17050	2314	0	0
17051	17051	2314	0	0
17052	17052	2314	0	0
17053	17053	2314	0	0
17054	17054	2314	B0L3	0
17055	17055	2314	0.6	0
17056	17056	2314	1	0
17057	17057	2314	23.5	0
17058	17058	2314	10	0
17059	17059	2314	22.1	0
17060	17060	2314	6	0
17061	17061	2314	1.8	0
17062	17062	2314	59	0
17063	17063	2314	2.2	0
17064	17064	2314	2.4	0
17065	17065	2314	0	0
17066	17066	2314	0	0
17067	17067	2314	0	0
17068	17068	2314	63	0
17069	17069	2314	100	0
17070	17070	2314	100	0
17071	17071	2314	10	0
17072	17072	2314	10	0
17073	17073	2314	60	0
17074	17074	2314	0	0
17075	17075	2314	1.741666667	0
17076	17076	2314	21.21666667	0
17077	17077	2314	4.7	0
17078	17078	2314	60	0
17079	17079	2314	180	0
17080	17080	2314	2.55	0
17081	17081	2314	81	0
17082	17082	2314	47	0
17083	17083	2314	2.79	0
17084	17084	2314	0	0
17085	17085	2314	0	0
17086	17086	2314	70	0
17087	17087	2314	0	0
17088	17088	2314	40	0
17089	17089	2314	63	0
17090	17090	2314	31	0
17091	17091	2314	20	0
17092	17092	2314	60	0
17093	17093	2314	180	0
17094	17094	2314	2.55	0
17095	17095	2314	81	0
17096	17096	2314	47	0
17097	17097	2314	2.79	0
17098	17098	2314	0	0
17099	17099	2314	0	0
17100	17100	2314	70	0
17101	17101	2314	0	0
17102	17102	2314	40	0
17103	17103	2314	61.6	0
17104	17104	2314	24.1	0
17105	17105	2314	100.3	0
17106	17106	2314	46	0
17107	17107	2314	4.4	0
17108	17108	2314	4.2	0
17109	17109	2314	6.58	0
17110	17110	2314	1.99	0
17111	17111	2314	6.6041	0
17112	17112	2314	2.0402	0
17113	17113	2314	3	0
17114	17114	2314	17	0
17115	17115	2314	0	0
17116	17116	2314	0	0
17117	17117	2314		0
17118	17118	2314		0
17119	17119	2314		0
17120	17120	2314		0
17121	17121	2314	2	0
17122	17122	2314	0	0
17123	17123	2314	1	0
17124	17124	2314	0.5	0
17125	17125	2314	0.7	0
17126	17126	2314	1	0
17127	17127	2314		0
17128	17128	2314		0
17129	17129	2314		0
17130	17130	2314		0
17131	17131	2314	2	0
17132	17132	2314	8	0
17133	17133	2314	2	0
17134	17134	2314	1	0
17135	17135	2314	6	0
17136	17136	2314	1	0
17137	17137	2314	1	0
17138	17138	2314	0	0
17139	17139	2314	0	0
17140	17140	2314	0	0
17141	17141	2314	0	0
17142	17142	2314	0	0
17143	17143	2314	0	0
17144	17144	2314	0	0
17145	17145	2314	0	0
17146	17146	2314	0	0
17147	17147	2314	B0L4	0
17148	17148	2314	1.2	0
17149	17149	2314	1.3	0
17150	17150	2314	25.5	0
17151	17151	2314	14	0
17152	17152	2314	28.6	0
17153	17153	2314	17	0
17154	17154	2314	5.8	0
17155	17155	2314	87	0
17156	17156	2314	2.4	0
17157	17157	2314	2.2	0
17158	17158	2314	0	0
17159	17159	2314	3	0
17160	17160	2314	0	0
17161	17161	2314	56	0
17162	17162	2314	75	0
17163	17163	2314	75	0
17164	17164	2314	5	0
17165	17165	2314	5	0
17166	17166	2314	95	0
17167	17167	2314	0	0
17168	17168	2314	4.088333333	0
17169	17169	2314	54.66666667	0
17170	17170	2314	12.75	0
17171	17171	2314	200	0
17172	17172	2314	300	0
17173	17173	2314	2.24	0
17174	17174	2314	62.5	0
17175	17175	2314	35.5	0
17176	17176	2314	4.47	0
17177	17177	2314	0	0
17178	17178	2314	0	0
17179	17179	2314	85	0
17180	17180	2314	5	0
17181	17181	2314	0	0
17182	17182	2314	42	0
17183	17183	2314	48	0
17184	17184	2314	0	0
17185	17185	2314	200	0
17186	17186	2314	300	0
17187	17187	2314	2.24	0
17188	17188	2314	62.5	0
17189	17189	2314	35.5	0
17190	17190	2314	4.47	0
17191	17191	2314	0	0
17192	17192	2314	0	0
17193	17193	2314	85	0
17194	17194	2314	5	0
17195	17195	2314	0	0
17196	17196	2314	86.6	0
17197	17197	2314	21.4	0
17198	17198	2314	87.3	0
17199	17199	2314	39.6	0
17200	17200	2314	9	0
17201	17201	2314	6.3	0
17202	17202	2314	6.27	0
17203	17203	2314	1.41	0
17204	17204	2314	6.2914	0
17205	17205	2314	1.4559	0
17206	17206	2314	4	0
17207	17207	2314	13	0
17208	17208	2314	1	0
17209	17209	2314	2	0
17210	17210	2314	0.4	0
17211	17211	2314	0.6	0
17212	17212	2314		0
17213	17213	2314		0
17214	17214	2314	2	0
17215	17215	2314	0	0
17216	17216	2314	0.6	0
17217	17217	2314	0.2	0
17218	17218	2314	0.5	0
17219	17219	2314	0.7	0
17220	17220	2314	0.4	0
17221	17221	2314	0.5	0
17222	17222	2314		0
17223	17223	2314		0
17224	17224	2314	1	0
17225	17225	2314	10	0
17226	17226	2314	3	0
17227	17227	2314	1	0
17228	17228	2314	6	0
17229	17229	2314	2	0
17230	17230	2314	1	0
17231	17231	2314	1	0
17232	17232	2314	0	0
17233	17233	2314	0	0
17234	17234	2314	0	0
17235	17235	2314	0	0
17236	17236	2314	0	0
17237	17237	2314	0	0
17238	17238	2314	0	0
17239	17239	2314	0	0
17240	17240	2314	B0L4	0
17241	17241	2314	0.8	0
17242	17242	2314	1.1	0
17243	17243	2314	12.2	0
17244	17244	2314	5	0
17245	17245	2314	27.7	0
17246	17246	2314	11	0
17247	17247	2314	4.1	0
17248	17248	2314	87	0
17249	17249	2314	2.3	0
17250	17250	2314	2.2	0
17251	17251	2314	2	0
17252	17252	2314	2	0
17253	17253	2314	0	0
17254	17254	2314	61	0
17255	17255	2314	125	0
17256	17256	2314	125	0
17257	17257	2314	25	0
17258	17258	2314	25	0
17259	17259	2314	95	0
17260	17260	2314	0	0
17261	17261	2314	1.986666667	0
17262	17262	2314	24.68333333	0
17263	17263	2314	6.933333333	0
17264	17264	2314	175	0
17265	17265	2314	175	0
17266	17266	2314	1.83	0
17267	17267	2314	68	0
17268	17268	2314	41	0
17269	17269	2314	2.71	0
17270	17270	2314	0	0
17271	17271	2314	0	0
17272	17272	2314	90	0
17273	17273	2314	0	0
17274	17274	2314	10	0
17275	17275	2314	102	0
17276	17276	2314	100	0
17277	17277	2314	59	0
17278	17278	2314	175	0
17279	17279	2314	175	0
17280	17280	2314	1.83	0
17281	17281	2314	68	0
17282	17282	2314	41	0
17283	17283	2314	2.71	0
17284	17284	2314	0	0
17285	17285	2314	0	0
17286	17286	2314	90	0
17287	17287	2314	0	0
17288	17288	2314	10	0
17289	17289	2314	58.5	0
17290	17290	2314	15.8	0
17291	17291	2314	32.5	0
17292	17292	2314	25.9	0
17293	17293	2314	7.3	0
17294	17294	2314	6.8	0
17295	17295	2314	10.18	0
17296	17296	2314	2.51	0
17297	17297	2314	10.1958	0
17298	17298	2314	2.5427	0
17299	17299	2314	20	0
17300	17300	2314	15	0
17301	17301	2314	0	0
17302	17302	2314	2	0
17303	17303	2314	0.4	0
17304	17304	2314	0.6	0
17305	17305	2314		0
17306	17306	2314		0
17307	17307	2314	2	0
17308	17308	2314	0	0
17309	17309	2314	1	0
17310	17310	2314	0.4	0
17311	17311	2314	0.6	0
17312	17312	2314	0.8	0
17313	17313	2314	0.5	0
17314	17314	2314	0.6	0
17315	17315	2314		0
17316	17316	2314		0
17317	17317	2314	1	0
17318	17318	2314	12	0
17319	17319	2314	3	0
17320	17320	2314	1	0
17321	17321	2314	7	0
17322	17322	2314	2	0
17323	17323	2314	2	0
17324	17324	2314	1	0
17325	17325	2314	0	0
17326	17326	2314	0	0
17327	17327	2314	0	0
17328	17328	2314	0	0
17329	17329	2314	0	0
17330	17330	2314	0	0
17331	17331	2314	0	0
17332	17332	2314	0	0
17333	17333	2314	B0L5	0
17334	17334	2314	1.1	0
17335	17335	2314	1.2	0
17336	17336	2314	13.5	0
17337	17337	2314	8	0
17338	17338	2314	23.3	0
17339	17339	2314	21	0
17340	17340	2314	7.4	0
17341	17341	2314	131	0
17342	17342	2314	2.5	0
17343	17343	2314	2.1	0
17344	17344	2314	0	0
17345	17345	2314	0	0
17346	17346	2314	0	0
17347	17347	2314	54	0
17348	17348	2314	225	0
17349	17349	2314	225	0
17350	17350	2314	15	0
17351	17351	2314	15	0
17352	17352	2314	95	0
17353	17353	2314	0	0
17354	17354	2314	1.736666667	0
17355	17355	2314	18.91666667	0
17356	17356	2314	5.516666667	0
17357	17357	2314	150	0
17358	17358	2314	175	0
17359	17359	2314	2.47	0
17360	17360	2314	60	0
17361	17361	2314	41	0
17362	17362	2314	2.81	0
17363	17363	2314	0	0
17364	17364	2314	0	0
17365	17365	2314	70	0
17366	17366	2314	0	0
17367	17367	2314	0	0
17368	17368	2314	122	0
17369	17369	2314	46	0
17370	17370	2314	3	0
17371	17371	2314	150	0
17372	17372	2314	175	0
17373	17373	2314	2.47	0
17374	17374	2314	60	0
17375	17375	2314	41	0
17376	17376	2314	2.81	0
17377	17377	2314	0	0
17378	17378	2314	0	0
17379	17379	2314	70	0
17380	17380	2314	0	0
17381	17381	2314	0	0
17382	17382	2314	58.9	0
17383	17383	2314	27.6	0
17384	17384	2314	111.5	0
17385	17385	2314	54	0
17386	17386	2314	4.1	0
17387	17387	2314	3.9	0
17388	17388	2314	9.61	0
17389	17389	2314	1.96	0
17390	17390	2314	9.6376	0
17391	17391	2314	2.0179	0
17392	17392	2314	5	0
17393	17393	2314	1	0
17394	17394	2314	1	0
17395	17395	2314	2	0
17396	17396	2314	0.7	0
17397	17397	2314	0.7	0
17398	17398	2314	0.8	0
17399	17399	2314	0.8	0
17400	17400	2314	3	0
17401	17401	2314	0	0
17402	17402	2314	0.8	0
17403	17403	2314	0.35	0
17404	17404	2314	0.6	0
17405	17405	2314	0.9	0
17406	17406	2314	0.6	0
17407	17407	2314	0.9	0
17408	17408	2314	0.6	0
17409	17409	2314	0.8	0
17410	17410	2314	1	0
17411	17411	2314	16	0
17412	17412	2314	4	0
17413	17413	2314	1	0
17414	17414	2314	7	0
17415	17415	2314	3	0
17416	17416	2314	3	0
17417	17417	2314	2	0
17418	17418	2314	1	0
17419	17419	2314	0	0
17420	17420	2314	0	0
17421	17421	2314	0	0
17422	17422	2314	0	0
17423	17423	2314	0	0
17424	17424	2314	0	0
17425	17425	2314	0	0
17426	17426	2314	B0L5	0
17427	17427	2314	1.05	0
17428	17428	2314	1.05	0
17429	17429	2314	24.9	0
17430	17430	2314	14	0
17431	17431	2314	38.7	0
17432	17432	2314	38	0
17433	17433	2314	16.4	0
17434	17434	2314	222	0
17435	17435	2314	2.2	0
17436	17436	2314	1.6	0
17437	17437	2314	13	0
17438	17438	2314	2	0
17439	17439	2314	0	0
17440	17440	2314	53	0
17441	17441	2314	0	0
17442	17442	2314	100	0
17443	17443	2314	0	0
17444	17444	2314	1	0
17445	17445	2314	0	0
17446	17446	2314	0	0
17447	17447	2314		0
17448	17448	2314		0
17449	17449	2314		0
17450	17450	2314		0
17451	17451	2314		0
17452	17452	2314		0
17453	17453	2314		0
17454	17454	2314		0
17455	17455	2314		0
17456	17456	2314		0
17457	17457	2314		0
17458	17458	2314		0
17459	17459	2314		0
17460	17460	2314		0
17461	17461	2314	53	0
17462	17462	2314		0
17463	17463	2314		0
17464	17464	2314	.	0
17465	17465	2314	.	0
17466	17466	2314	.	0
17467	17467	2314	.	0
17468	17468	2314	.	0
17469	17469	2314	.	0
17470	17470	2314	.	0
17471	17471	2314	.	0
17472	17472	2314	.	0
17473	17473	2314	.	0
17474	17474	2314	.	0
17475	17475	2314	.	0
17476	17476	2314	.	0
17477	17477	2314	.	0
17478	17478	2314	.	0
17479	17479	2314	.	0
17480	17480	2314	.	0
17481	17481	2314	4.12	0
17482	17482	2314	0.3	0
17483	17483	2314	4.12	0
17484	17484	2314	0.3	0
17485	17485	2314	2	0
17486	17486	2314		0
17487	17487	2314	1	0
17488	17488	2314	2	0
17489	17489	2314	0.4	0
17490	17490	2314	0.8	0
17491	17491	2314		0
17492	17492	2314		0
17493	17493	2314	2	0
17494	17494	2314	0	0
17495	17495	2314	1.2	0
17496	17496	2314	0.5	0
17497	17497	2314	0.8	0
17498	17498	2314	0.9	0
17499	17499	2314	0.6	0
17500	17500	2314	0.8	0
17501	17501	2314		0
17502	17502	2314		0
17503	17503	2314	2	0
17504	17504	2314	12	0
17505	17505	2314	3	0
17506	17506	2314	1	0
17507	17507	2314	6	0
17508	17508	2314	3	0
17509	17509	2314	2	0
17510	17510	2314	1	0
17511	17511	2314	0	0
17512	17512	2314	0	0
17513	17513	2314	0	0
17514	17514	2314	0	0
17515	17515	2314	0	0
17516	17516	2314	0	0
17517	17517	2314	0	0
17518	17518	2314	0	0
17519	17519	2314	B0L5	0
17520	17520	2314	1.1	0
17521	17521	2314	1.4	0
17522	17522	2314	21.8	0
17523	17523	2314	13	0
17524	17524	2314	39.1	0
17525	17525	2314	31	0
17526	17526	2314	18.3	0
17527	17527	2314	185	0
17528	17528	2314	2.4	0
17529	17529	2314	2.3	0
17530	17530	2314	0	0
17531	17531	2314	2	0
17532	17532	2314	0	0
17533	17533	2314	69	0
17534	17534	2314	250	0
17535	17535	2314	250	0
17536	17536	2314	15	0
17537	17537	2314	40	0
17538	17538	2314	95	0
17539	17539	2314	0	0
17540	17540	2314	1.865	0
17541	17541	2314	24.4	0
17542	17542	2314	6.816666667	0
17543	17543	2314	400	0
17544	17544	2314	450	0
17545	17545	2314	2.95	0
17546	17546	2314	77.5	0
17547	17547	2314	42	0
17548	17548	2314	2.83	0
17549	17549	2314	5	0
17550	17550	2314	0	0
17551	17551	2314	95	0
17552	17552	2314	6	0
17553	17553	2314	0	0
17554	17554	2314	142	0
17555	17555	2314	78	0
17556	17556	2314	13	0
17557	17557	2314	400	0
17558	17558	2314	450	0
17559	17559	2314	2.95	0
17560	17560	2314	77.5	0
17561	17561	2314	42	0
17562	17562	2314	2.83	0
17563	17563	2314	5	0
17564	17564	2314	0	0
17565	17565	2314	95	0
17566	17566	2314	6	0
17567	17567	2314	0	0
17568	17568	2314	85.6	0
17569	17569	2314	33.5	0
17570	17570	2314	77.4	0
17571	17571	2314	35.9	0
17572	17572	2314	5	0
17573	17573	2314	3.9	0
17574	17574	2314	13.17	0
17575	17575	2314	5.17	0
17576	17576	2314	13.2035	0
17577	17577	2314	5.2098	0
17578	17578	2314		0
17579	17579	2314		0
17580	17580	2314	0	0
17581	17581	2314	1	0
17582	17582	2314		0
17583	17583	2314		0
17584	17584	2314		0
17585	17585	2314		0
17586	17586	2314	1	0
17587	17587	2314	0	0
17588	17588	2314	0.4	0
17589	17589	2314	0.3	0
17590	17590	2314		0
17591	17591	2314		0
17592	17592	2314		0
17593	17593	2314		0
17594	17594	2314		0
17595	17595	2314		0
17596	17596	2314	1	0
17597	17597	2314	3	0
17598	17598	2314	0	0
17599	17599	2314	.	0
17600	17600	2314	3	0
17601	17601	2314	0	0
17602	17602	2314	0	0
17603	17603	2314	0	0
17604	17604	2314	0	0
17605	17605	2314	0	0
17606	17606	2314	0	0
17607	17607	2314	0	0
17608	17608	2314	0	0
17609	17609	2314	0	0
17610	17610	2314	0	0
17611	17611	2314	0	0
17612	17612	2314	B0L2	0
17613	17613	2314	0.25	0
17614	17614	2314	0.35	0
17615	17615	2314	20.3	0
17616	17616	2314	2	0
17617	17617	2314	0.7	0
17618	17618	2314	0	0
17619	17619	2314	.	0
17620	17620	2314	14	0
17621	17621	2314	1.9	0
17622	17622	2314	1.6	0
17623	17623	2314	0	0
17624	17624	2314	0	0
17625	17625	2314	0	0
17626	17626	2314	39	0
17627	17627	2314	50	0
17628	17628	2314	50	0
17629	17629	2314	0	0
17630	17630	2314	0	0
17631	17631	2314	80	0
17632	17632	2314	0	0
17633	17633	2314	1.991666667	0
17634	17634	2314	20.75	0
17635	17635	2314	4.333333333	0
17636	17636	2314	125	0
17637	17637	2314	125	0
17638	17638	2314	2.67	0
17639	17639	2314	72	0
17640	17640	2314	38	0
17641	17641	2314	3.26	0
17642	17642	2314	0	0
17643	17643	2314	0	0
17644	17644	2314	90	0
17645	17645	2314	0	0
17646	17646	2314	10	0
17647	17647	2314	0	0
17648	17648	2314	5	0
17649	17649	2314	31	0
17650	17650	2314	125	0
17651	17651	2314	125	0
17652	17652	2314	2.67	0
17653	17653	2314	72	0
17654	17654	2314	38	0
17655	17655	2314	3.26	0
17656	17656	2314	0	0
17657	17657	2314	0	0
17658	17658	2314	90	0
17659	17659	2314	0	0
17660	17660	2314	10	0
17661	17661	2314	118.9	0
17662	17662	2314	49.3	0
17663	17663	2314	75.4	0
17664	17664	2314	38	0
17665	17665	2314	12.3	0
17666	17666	2314	10.8	0
17667	17667	2314	6.2	0
17668	17668	2314	1.19	0
17669	17669	2314	6.2493	0
17670	17670	2314	1.2388	0
17671	17671	2314	22	0
17672	17672	2314	2	0
17673	17673	2314	0	0
17674	17674	2314	1	0
17675	17675	2314	0.5	0
17676	17676	2314	0.8	0
17677	17677	2314		0
17678	17678	2314		0
17679	17679	2314	2	0
17680	17680	2314	0	0
17681	17681	2314	0.9	0
17682	17682	2314	0.5	0
17683	17683	2314	0.9	0
17684	17684	2314	1.1	0
17685	17685	2314	0.6	0
17686	17686	2314	0.7	0
17687	17687	2314		0
17688	17688	2314		0
17689	17689	2314	2	0
17690	17690	2314	11	0
17691	17691	2314	3	0
17692	17692	2314	1	0
17693	17693	2314	6	0
17694	17694	2314	2	0
17695	17695	2314	2	0
17696	17696	2314	1	0
17697	17697	2314	0	0
17698	17698	2314	0	0
17699	17699	2314	0	0
17700	17700	2314	0	0
17701	17701	2314	0	0
17702	17702	2314	0	0
17703	17703	2314	0	0
17704	17704	2314	0	0
17705	17705	2314	B0L5	0
17706	17706	2314	1.3	0
17707	17707	2314	1.4	0
17708	17708	2314	20.4	0
17709	17709	2314	12	0
17710	17710	2314	24.6	0
17711	17711	2314	17	0
17712	17712	2314	12.8	0
17713	17713	2314	103	0
17714	17714	2314	2.5	0
17715	17715	2314	1.8	0
17716	17716	2314	0	0
17717	17717	2314	0	0
17718	17718	2314	0	0
17719	17719	2314	65.5	0
17720	17720	2314	250	0
17721	17721	2314	250	0
17722	17722	2314	30	0
17723	17723	2314	30	0
17724	17724	2314	95	0
17725	17725	2314	0	0
17726	17726	2314	1.813333333	0
17727	17727	2314	24.1	0
17728	17728	2314	5.716666667	0
17729	17729	2314	250	0
17730	17730	2314	250	0
17731	17731	2314	2.96	0
17732	17732	2314	84	0
17733	17733	2314	35	0
17734	17734	2314	3.91	0
17735	17735	2314	0	0
17736	17736	2314	0	0
17737	17737	2314	90	0
17738	17738	2314	0	0
17739	17739	2314	0	0
17740	17740	2314	43	0
17741	17741	2314	54	0
17742	17742	2314	52	0
17743	17743	2314	250	0
17744	17744	2314	250	0
17745	17745	2314	2.96	0
17746	17746	2314	84	0
17747	17747	2314	35	0
17748	17748	2314	3.91	0
17749	17749	2314	0	0
17750	17750	2314	0	0
17751	17751	2314	90	0
17752	17752	2314	0	0
17753	17753	2314	0	0
17754	17754	2314	44.3	0
17755	17755	2314	24.2	0
17756	17756	2314	195.7	0
17757	17757	2314	81.7	0
17758	17758	2314	9.3	0
17759	17759	2314	9.1	0
17760	17760	2314	13.27	0
17761	17761	2314	2.5	0
17762	17762	2314	13.2942	0
17763	17763	2314	2.5908	0
17764	17764	2314	6	0
17765	17765	2314	2	0
17766	17766	2314	1	0
17767	17767	2314	2	0
17768	17768	2314	0.7	0
17769	17769	2314	1	0
17770	17770	2314	1.1	0
17771	17771	2314	1.1	0
17772	17772	2314	3	0
17773	17773	2314	0	0
17774	17774	2314	1	0
17775	17775	2314	0.4	0
17776	17776	2314	0.8	0
17777	17777	2314	1.1	0
17778	17778	2314	0.7	0
17779	17779	2314	0.9	0
17780	17780	2314		0
17781	17781	2314		0
17782	17782	2314	2	0
17783	17783	2314	18	0
17784	17784	2314	4	0
17785	17785	2314	1	0
17786	17786	2314	8	0
17787	17787	2314	4	0
17788	17788	2314	3	0
17789	17789	2314	2	0
17790	17790	2314	1	0
17791	17791	2314	0	0
17792	17792	2314	0	0
17793	17793	2314	0	0
17794	17794	2314	0	0
17795	17795	2314	0	0
17796	17796	2314	0	0
17797	17797	2314	0	0
17798	17798	2314	B0L6	0
17799	17799	2314	1.3	0
17800	17800	2314	1.5	0
17801	17801	2314	24.2	0
17802	17802	2314	17	0
17803	17803	2314	35.3	0
17804	17804	2314	54	0
17805	17805	2314	15.7	0
17806	17806	2314	294	0
17807	17807	2314	2.3	0
17808	17808	2314	2	0
17809	17809	2314	2	0
17810	17810	2314	4	0
17811	17811	2314	0	0
17812	17812	2314	44.5	0
17813	17813	2314	75	0
17814	17814	2314	75	0
17815	17815	2314	20	0
17816	17816	2314	20	0
17817	17817	2314	60	0
17818	17818	2314	3	0
17819	17819	2314	1.245	0
17820	17820	2314	16.48333333	0
17821	17821	2314	3.6	0
17822	17822	2314	25	0
17823	17823	2314	150	0
17824	17824	2314	2.72	0
17825	17825	2314	54	0
17826	17826	2314	33	0
17827	17827	2314	3.4	0
17828	17828	2314	0	0
17829	17829	2314	0	0
17830	17830	2314	30	0
17831	17831	2314	0	0
17832	17832	2314	0	0
17833	17833	2314	119	0
17834	17834	2314	22	0
17835	17835	2314	11	0
17836	17836	2314	25	0
17837	17837	2314	150	0
17838	17838	2314	2.72	0
17839	17839	2314	54	0
17840	17840	2314	33	0
17841	17841	2314	3.4	0
17842	17842	2314	0	0
17843	17843	2314	0	0
17844	17844	2314	30	0
17845	17845	2314	0	0
17846	17846	2314	0	0
17847	17847	2314	95.3	0
17848	17848	2314	35.6	0
17849	17849	2314	155.4	0
17850	17850	2314	70	0
17851	17851	2314	5.5	0
17852	17852	2314	5.3	0
17853	17853	2314	7.26	0
17854	17854	2314	0.93	0
17855	17855	2314	7.2956	0
17856	17856	2314	1.0053	0
17857	17857	2314	37	0
17858	17858	2314	2	0
17859	17859	2314	1	0
17860	17860	2314	2	0
17861	17861	2314	0.9	0
17862	17862	2314	1.2	0
17863	17863	2314	1	0
17864	17864	2314	1.1	0
17865	17865	2314	3	0
17866	17866	2314	0	0
17867	17867	2314	1.3	0
17868	17868	2314	0.6	0
17869	17869	2314	0.9	0
17870	17870	2314	1.2	0
17871	17871	2314	0.8	0
17872	17872	2314	1.1	0
17873	17873	2314	0.7	0
17874	17874	2314	1.1	0
17875	17875	2314	2	0
17876	17876	2314	17	0
17877	17877	2314	4	0
17878	17878	2314	1	0
17879	17879	2314	7	0
17880	17880	2314	4	0
17881	17881	2314	3	0
17882	17882	2314	2	0
17883	17883	2314	1	0
17884	17884	2314	0	0
17885	17885	2314	0	0
17886	17886	2314	0	0
17887	17887	2314	0	0
17888	17888	2314	0	0
17889	17889	2314	0	0
17890	17890	2314	0	0
17891	17891	2314	B0L6	0
17892	17892	2314	1.3	0
17893	17893	2314	1.5	0
17894	17894	2314	30.4	0
17895	17895	2314	13	0
17896	17896	2314	40.4	0
17897	17897	2314	37	0
17898	17898	2314	24	0
17899	17899	2314	194	0
17900	17900	2314	3	0
17901	17901	2314	2.4	0
17902	17902	2314	23	0
17903	17903	2314	20	0
17904	17904	2314	7	0
17905	17905	2314	64	0
17906	17906	2314	300	0
17907	17907	2314	300	0
17908	17908	2314	10	0
17909	17909	2314	25	0
17910	17910	2314	95	0
17911	17911	2314	0	0
17912	17912	2314	2.496666667	0
17913	17913	2314	39.1	0
17914	17914	2314	9.783333333	0
17915	17915	2314	200	0
17916	17916	2314	300	0
17917	17917	2314	2.32	0
17918	17918	2314	84	0
17919	17919	2314	62.5	0
17920	17920	2314	3.41	0
17921	17921	2314	20	0
17922	17922	2314	0	0
17923	17923	2314	80	0
17924	17924	2314	0	0
17925	17925	2314	0	0
17926	17926	2314	95	0
17927	17927	2314	76	0
17928	17928	2314	25	0
17929	17929	2314	200	0
17930	17930	2314	300	0
17931	17931	2314	2.32	0
17932	17932	2314	84	0
17933	17933	2314	62.5	0
17934	17934	2314	3.41	0
17935	17935	2314	20	0
17936	17936	2314	0	0
17937	17937	2314	80	0
17938	17938	2314	0	0
17939	17939	2314	0	0
17940	17940	2314	164.9	0
17941	17941	2314	55	0
17942	17942	2314	250.9	0
17943	17943	2314	87.7	0
17944	17944	2314	48.9	0
17945	17945	2314	20.2	0
17946	17946	2314	25.37	0
17947	17947	2314	3.61	0
17948	17948	2314	25.425	0
17949	17949	2314	3.7179	0
17950	17950	2314	52	0
17951	17951	2314	27	0
17952	17952	2314	1	0
17953	17953	2314	2	0
17954	17954	2314	0.7	0
17955	17955	2314	1.1	0
17956	17956	2314	0.8	0
17957	17957	2314	0.9	0
17958	17958	2314	3	0
17959	17959	2314	0	0
17960	17960	2314	0.8	0
17961	17961	2314	0.4	0
17962	17962	2314	0.9	0
17963	17963	2314	1	0
17964	17964	2314	0.9	0
17965	17965	2314	1	0
17966	17966	2314		0
17967	17967	2314		0
17968	17968	2314	2	0
17969	17969	2314	16	0
17970	17970	2314	4	0
17971	17971	2314	1	0
17972	17972	2314	7	0
17973	17973	2314	3	0
17974	17974	2314	3	0
17975	17975	2314	2	0
17976	17976	2314	1	0
17977	17977	2314	0	0
17978	17978	2314	0	0
17979	17979	2314	0	0
17980	17980	2314	0	0
17981	17981	2314	0	0
17982	17982	2314	0	0
17983	17983	2314	0	0
17984	17984	2314	B0L5	0
17985	17985	2314	1.5	0
17986	17986	2314	1.6	0
17987	17987	2314	28.6	0
17988	17988	2314	12	0
17989	17989	2314	41.4	0
17990	17990	2314	35	0
17991	17991	2314	22.3	0
17992	17992	2314	157	0
17993	17993	2314	3	0
17994	17994	2314	2	0
17995	17995	2314	0	0
17996	17996	2314	0	0
17997	17997	2314	0	0
17998	17998	2314	76	0
17999	17999	2314	100	0
18000	18000	2314	150	0
18001	18001	2314	10	0
18002	18002	2314	5	0
18003	18003	2314	60	0
18004	18004	2314	0	0
18005	18005	2314	1.708333333	0
18006	18006	2314	25.11666667	0
18007	18007	2314	5.366666667	0
18008	18008	2314	50	0
18009	18009	2314	200	0
18010	18010	2314	1.57	0
18011	18011	2314	72	0
18012	18012	2314	36	0
18013	18013	2314	2.83	0
18014	18014	2314	0	0
18015	18015	2314	0	0
18016	18016	2314	30	0
18017	18017	2314	0	0
18018	18018	2314	50	0
18019	18019	2314	93	0
18020	18020	2314	0	0
18021	18021	2314	0	0
18022	18022	2314	50	0
18023	18023	2314	200	0
18024	18024	2314	1.57	0
18025	18025	2314	72	0
18026	18026	2314	36	0
18027	18027	2314	2.83	0
18028	18028	2314	0	0
18029	18029	2314	0	0
18030	18030	2314	30	0
18031	18031	2314	0	0
18032	18032	2314	50	0
18033	18033	2314	46.2	0
18034	18034	2314	16.4	0
18035	18035	2314	89.9	0
18036	18036	2314	36.4	0
18037	18037	2314	10.5	0
18038	18038	2314	8.5	0
18039	18039	2314	5.38	0
18040	18040	2314	1.62	0
18041	18041	2314	5.3964	0
18042	18042	2314	1.6649	0
18043	18043	2314	10	0
18044	18044	2314	1	0
18045	18045	2314	1	0
18046	18046	2314	2	0
18047	18047	2314	0.6	0
18048	18048	2314	0.8	0
18049	18049	2314		0
18050	18050	2314		0
18051	18051	2314	3	0
18052	18052	2314	0	0
18053	18053	2314	1.2	0
18054	18054	2314	0.5	0
18055	18055	2314	0.7	0
18056	18056	2314	1	0
18057	18057	2314	0.6	0
18058	18058	2314	0.8	0
18059	18059	2314		0
18060	18060	2314		0
18061	18061	2314	1	0
18062	18062	2314	13	0
18063	18063	2314	3	0
18064	18064	2314	1	0
18065	18065	2314	7	0
18066	18066	2314	3	0
18067	18067	2314	2	0
18068	18068	2314	1	0
18069	18069	2314	0	0
18070	18070	2314	0	0
18071	18071	2314	0	0
18072	18072	2314	0	0
18073	18073	2314	0	0
18074	18074	2314	0	0
18075	18075	2314	0	0
18076	18076	2314	0	0
18077	18077	2314	B0L5	0
18078	18078	2314	1.3	0
18079	18079	2314	1.3	0
18080	18080	2314	35.3	0
18081	18081	2314	15	0
18082	18082	2314	35.1	0
18083	18083	2314	28	0
18084	18084	2314	10.3	0
18085	18085	2314	181	0
18086	18086	2314	2	0
18087	18087	2314	1.7	0
18088	18088	2314	0	0
18089	18089	2314	0	0
18090	18090	2314	0	0
18091	18091	2314	81	0
18092	18092	2314	400	0
18093	18093	2314	400	0
18094	18094	2314	25	0
18095	18095	2314	40	0
18096	18096	2314	80	0
18097	18097	2314	0	0
18098	18098	2314	1.878333333	0
18099	18099	2314	26.06666667	0
18100	18100	2314	7.9	0
18101	18101	2314	250	0
18102	18102	2314	300	0
18103	18103	2314	4.34	0
18104	18104	2314	87	0
18105	18105	2314	51	0
18106	18106	2314	4.87	0
18107	18107	2314	12	0
18108	18108	2314	0	0
18109	18109	2314	55	0
18110	18110	2314	0	0
18111	18111	2314	70	0
18112	18112	2314	65	0
18113	18113	2314	65	0
18114	18114	2314	16	0
18115	18115	2314	250	0
18116	18116	2314	300	0
18117	18117	2314	4.34	0
18118	18118	2314	87	0
18119	18119	2314	51	0
18120	18120	2314	4.87	0
18121	18121	2314	12	0
18122	18122	2314	0	0
18123	18123	2314	55	0
18124	18124	2314	0	0
18125	18125	2314	70	0
18126	18126	2314	156.8	0
18127	18127	2314	80.2	0
18128	18128	2314	289.8	0
18129	18129	2314	118.3	0
18130	18130	2314	26.9	0
18131	18131	2314	17.3	0
18132	18132	2314	24.18	0
18133	18133	2314	7.99	0
18134	18134	2314	24.2602	0
18135	18135	2314	8.1256	0
18136	18136	2314	4	0
18137	18137	2314	33	0
18138	18138	2314	1	0
18139	18139	2314	1	0
18140	18140	2314	0.3	0
18141	18141	2314	0.6	0
18142	18142	2314		0
18143	18143	2314		0
18144	18144	2314	.	0
18145	18145	2314	0	0
18146	18146	2314	.	0
18147	18147	2314	.	0
18148	18148	2314	.	0
18149	18149	2314		0
18150	18150	2314		0
18151	18151	2314		0
18152	18152	2314		0
18153	18153	2314		0
18154	18154	2314	2	0
18155	18155	2314	5	0
18156	18156	2314	0	0
18157	18157	2314	.	0
18158	18158	2314	5	0
18159	18159	2314	0	0
18160	18160	2314	0	0
18161	18161	2314	0	0
18162	18162	2314	0	0
18163	18163	2314	0	0
18164	18164	2314	0	0
18165	18165	2314	0	0
18166	18166	2314	0	0
18167	18167	2314	0	0
18168	18168	2314	0	0
18169	18169	2314	0	0
18170	18170	2314	B0L4	0
18171	18171	2314	0.9	0
18172	18172	2314	1	0
18173	18173	2314	25.2	0
18174	18174	2314	8	0
18175	18175	2314	25.7	0
18176	18176	2314	4	0
18177	18177	2314	3.1	0
18178	18178	2314	49	0
18179	18179	2314	2.5	0
18180	18180	2314	2.2	0
18181	18181	2314	2	0
18182	18182	2314	2	0
18183	18183	2314	0	0
18184	18184	2314	46.5	0
18185	18185	2314	25	0
18186	18186	2314	25	0
18187	18187	2314	5	0
18188	18188	2314	5	0
18189	18189	2314	60	0
18190	18190	2314	2	0
18191	18191	2314	1.213333333	0
18192	18192	2314	14.55	0
18193	18193	2314	4.166666667	0
18194	18194	2314	25	0
18195	18195	2314	75	0
18196	18196	2314	2.19	0
18197	18197	2314	58	0
18198	18198	2314	36	0
18199	18199	2314	2.09	0
18200	18200	2314	0	0
18201	18201	2314	0	0
18202	18202	2314	30	0
18203	18203	2314	0	0
18204	18204	2314	0	0
18205	18205	2314	71	0
18206	18206	2314	12	0
18207	18207	2314	2	0
18208	18208	2314	25	0
18209	18209	2314	75	0
18210	18210	2314	2.19	0
18211	18211	2314	58	0
18212	18212	2314	36	0
18213	18213	2314	2.09	0
18214	18214	2314	0	0
18215	18215	2314	0	0
18216	18216	2314	30	0
18217	18217	2314	0	0
18218	18218	2314	0	0
18219	18219	2314	53.9	0
18220	18220	2314	16.1	0
18221	18221	2314	23.8	0
18222	18222	2314	15.3	0
18223	18223	2314	5.6	0
18224	18224	2314	4.4	0
18225	18225	2314	3.41	0
18226	18226	2314	1.06	0
18227	18227	2314	3.4261	0
18228	18228	2314	1.0797	0
18229	18229	2314	12	0
18230	18230	2314	3	0
18231	18231	2314	2	0
18232	18232	2314	3	0
18233	18233	2314	0.6	0
18234	18234	2314	0.8	0
18235	18235	2314	1.2	0
18236	18236	2314	1	0
18237	18237	2314	3	0
18238	18238	2314	0	0
18239	18239	2314	.	0
18240	18240	2314	.	0
18241	18241	2314	0.6	0
18242	18242	2314	0.7	0
18243	18243	2314	0.7	0
18244	18244	2314	1	0
18245	18245	2314	0.6	0
18246	18246	2314	0.9	0
18247	18247	2314	0	0
18248	18248	2314	22	0
18249	18249	2314	6	0
18250	18250	2314	1	0
18251	18251	2314	8	0
18252	18252	2314	3	0
18253	18253	2314	4	0
18254	18254	2314	3	0
18255	18255	2314	2	0
18256	18256	2314	1	0
18257	18257	2314	0	0
18258	18258	2314	0	0
18259	18259	2314	1	0
18260	18260	2314	0	0
18261	18261	2314	0	0
18262	18262	2314	0	0
18263	18263	2314	B0L6	0
18264	18264	2314	1.2	0
18265	18265	2314	1.4	0
18266	18266	2314	26.4	0
18267	18267	2314	12	0
18268	18268	2314	42.2	0
18269	18269	2314	34	0
18270	18270	2314	20.3	0
18271	18271	2314	206	0
18272	18272	2314	2.3	0
18273	18273	2314	1.9	0
18274	18274	2314	24	0
18275	18275	2314	14	0
18276	18276	2314	6	0
18277	18277	2314	64	0
18278	18278	2314	100	0
18279	18279	2314	100	0
18280	18280	2314	20	0
18281	18281	2314	10	0
18282	18282	2314	95	0
18283	18283	2314	0	0
18284	18284	2314	2.896666667	0
18285	18285	2314	42.08333333	0
18286	18286	2314	10.45	0
18287	18287	2314	350	0
18288	18288	2314	400	0
18289	18289	2314	2.41	0
18290	18290	2314	77	0
18291	18291	2314	32.5	0
18292	18292	2314	3.78	0
18293	18293	2314	0	0
18294	18294	2314	0	0
18295	18295	2314	95	0
18296	18296	2314	10	0
18297	18297	2314	0	0
18298	18298	2314	79	0
18299	18299	2314	46	0
18300	18300	2314	16	0
18301	18301	2314	350	0
18302	18302	2314	400	0
18303	18303	2314	2.41	0
18304	18304	2314	77	0
18305	18305	2314	32.5	0
18306	18306	2314	3.78	0
18307	18307	2314	0	0
18308	18308	2314	0	0
18309	18309	2314	95	0
18310	18310	2314	10	0
18311	18311	2314	0	0
18312	18312	2314	98.6	0
18313	18313	2314	33.8	0
18314	18314	2314	143.1	0
18315	18315	2314	63.1	0
18316	18316	2314	3.3	0
18317	18317	2314	3.2	0
18318	18318	2314	8.35	0
18319	18319	2314	2.83	0
18320	18320	2314	8.3838	0
18321	18321	2314	2.8963	0
18322	18322	2314	3	0
18323	18323	2314	10	0
18324	18324	2314	1	0
18325	18325	2314	2	0
18326	18326	2314	0.4	0
18327	18327	2314	0.5	0
18328	18328	2314	0.6	0
18329	18329	2314	0.6	0
18330	18330	2314	2	0
18331	18331	2314	0	0
18332	18332	2314	0.6	0
18333	18333	2314	0.3	0
18334	18334	2314	0.4	0
18335	18335	2314	0.6	0
18336	18336	2314		0
18337	18337	2314		0
18338	18338	2314		0
18339	18339	2314		0
18340	18340	2314	2	0
18341	18341	2314	14	0
18342	18342	2314	3	0
18343	18343	2314	1	0
18344	18344	2314	7	0
18345	18345	2314	3	0
18346	18346	2314	3	0
18347	18347	2314	1	0
18348	18348	2314	0	0
18349	18349	2314	0	0
18350	18350	2314	0	0
18351	18351	2314	0	0
18352	18352	2314	0	0
18353	18353	2314	0	0
18354	18354	2314	0	0
18355	18355	2314	0	0
18356	18356	2314	B0L5	0
18357	18357	2314	0.8	0
18358	18358	2314	1	0
18359	18359	2314	21.8	0
18360	18360	2314	9	0
18361	18361	2314	33	0
18362	18362	2314	26	0
18363	18363	2314	4.4	0
18364	18364	2314	110	0
18365	18365	2314	2.1	0
18366	18366	2314	1.7	0
18367	18367	2314	9	0
18368	18368	2314	9	0
18369	18369	2314	0	0
18370	18370	2314	63	0
18371	18371	2314	50	0
18372	18372	2314	50	0
18373	18373	2314	10	0
18374	18374	2314	10	0
18375	18375	2314	70	0
18376	18376	2314	0	0
18377	18377	2314	1.581666667	0
18378	18378	2314	17.88333333	0
18379	18379	2314	4.5	0
18380	18380	2314	50	0
18381	18381	2314	50	0
18382	18382	2314	2.02	0
18383	18383	2314	88	0
18384	18384	2314	41	0
18385	18385	2314	2.47	0
18386	18386	2314	0	0
18387	18387	2314	0	0
18388	18388	2314	90	0
18389	18389	2314	0	0
18390	18390	2314	0	0
18391	18391	2314	16	0
18392	18392	2314	43	0
18393	18393	2314	28	0
18394	18394	2314	50	0
18395	18395	2314	50	0
18396	18396	2314	2.02	0
18397	18397	2314	88	0
18398	18398	2314	41	0
18399	18399	2314	2.47	0
18400	18400	2314	0	0
18401	18401	2314	0	0
18402	18402	2314	90	0
18403	18403	2314	0	0
18404	18404	2314	0	0
18405	18405	2314	42.7	0
18406	18406	2314	24.6	0
18407	18407	2314	37.4	0
18408	18408	2314	29.3	0
18409	18409	2314	5.4	0
18410	18410	2314	5	0
18411	18411	2314	4.79	0
18412	18412	2314	1.84	0
18413	18413	2314	4.8146	0
18414	18414	2314	1.8743	0
18415	18415	2314	24	0
18416	18416	2314	9	0
18417	18417	2314	0	0
18418	18418	2314	0	0
18419	18419	2314		0
18420	18420	2314		0
18421	18421	2314		0
18422	18422	2314		0
18423	18423	2314	1	0
18424	18424	2314	0	0
18425	18425	2314	0.8	0
18426	18426	2314	0.5	0
18427	18427	2314	0.5	0
18428	18428	2314	0.7	0
18429	18429	2314		0
18430	18430	2314		0
18431	18431	2314		0
18432	18432	2314		0
18433	18433	2314	2	0
18434	18434	2314	6	0
18435	18435	2314	1	0
18436	18436	2314	2	0
18437	18437	2314	5	0
18438	18438	2314	1	0
18439	18439	2314	0	0
18440	18440	2314	0	0
18441	18441	2314	0	0
18442	18442	2314	0	0
18443	18443	2314	0	0
18444	18444	2314	0	0
18445	18445	2314	0	0
18446	18446	2314	0	0
18447	18447	2314	0	0
18448	18448	2314	0	0
18449	18449	2314	B0L4	0
18450	18450	2314	0.8	0
18451	18451	2314	0.9	0
18452	18452	2314	18.1	0
18453	18453	2314	8	0
18454	18454	2314	18.3	0
18455	18455	2314	9	0
18456	18456	2314	1.7	0
18457	18457	2314	53	0
18458	18458	2314	1.8	0
18459	18459	2314	1.8	0
18460	18460	2314	0	0
18461	18461	2314	0	0
18462	18462	2314	0	0
18463	18463	2314	29	0
18464	18464	2314	0	0
18465	18465	2314	0	0
18466	18466	2314	0	0
18467	18467	2314	0	0
18468	18468	2314	5	0
18469	18469	2314	0	0
18470	18470	2314		0
18471	18471	2314		0
18472	18472	2314		0
18473	18473	2314		0
18474	18474	2314		0
18475	18475	2314		0
18476	18476	2314		0
18477	18477	2314		0
18478	18478	2314		0
18479	18479	2314		0
18480	18480	2314		0
18481	18481	2314		0
18482	18482	2314		0
18483	18483	2314		0
18484	18484	2314	23	0
18485	18485	2314		0
18486	18486	2314		0
18487	18487	2314	.	0
18488	18488	2314	.	0
18489	18489	2314	.	0
18490	18490	2314	.	0
18491	18491	2314	.	0
18492	18492	2314	.	0
18493	18493	2314	.	0
18494	18494	2314	.	0
18495	18495	2314	.	0
18496	18496	2314	.	0
18497	18497	2314	.	0
18498	18498	2314	.	0
18499	18499	2314	.	0
18500	18500	2314	.	0
18501	18501	2314	.	0
18502	18502	2314	.	0
18503	18503	2314	.	0
18504	18504	2314	1.26	0
18505	18505	2314	0.19	0
18506	18506	2314	1.26	0
18507	18507	2314	0.19	0
18508	18508	2314	11	0
18509	18509	2314	23	0
18510	18510	2314	1	0
18511	18511	2314	2	0
18512	18512	2314	0.4	0
18513	18513	2314	0.4	0
18514	18514	2314	0.9	0
18515	18515	2314	1	0
18516	18516	2314	4	0
18517	18517	2314	0	0
18518	18518	2314	0.8	0
18519	18519	2314	0.3	0
18520	18520	2314	0.5	0
18521	18521	2314	0.5	0
18522	18522	2314	0.6	0
18523	18523	2314	0.7	0
18524	18524	2314	0.6	0
18525	18525	2314	0.7	0
18526	18526	2314	1	0
18527	18527	2314	19	0
18528	18528	2314	4	0
18529	18529	2314	1	0
18530	18530	2314	8	0
18531	18531	2314	3	0
18532	18532	2314	4	0
18533	18533	2314	3	0
18534	18534	2314	1	0
18535	18535	2314	0	0
18536	18536	2314	0	0
18537	18537	2314	0	0
18538	18538	2314	0	0
18539	18539	2314	0	0
18540	18540	2314	0	0
18541	18541	2314	0	0
18542	18542	2314	B0L6	0
18543	18543	2314	1.1	0
18544	18544	2314	1.5	0
18545	18545	2314	19.2	0
18546	18546	2314	13	0
18547	18547	2314	40.2	0
18548	18548	2314	37	0
18549	18549	2314	12.7	0
18550	18550	2314	140	0
18551	18551	2314	2.2	0
18552	18552	2314	2	0
18553	18553	2314	18	0
18554	18554	2314	15	0
18555	18555	2314	11	0
18556	18556	2314	58	0
18557	18557	2314	100	0
18558	18558	2314	100	0
18559	18559	2314	10	0
18560	18560	2314	25	0
18561	18561	2314	95	0
18562	18562	2314	0	0
18563	18563	2314	2.198333333	0
18564	18564	2314	27.11666667	0
18565	18565	2314	7.766666667	0
18566	18566	2314	400	0
18567	18567	2314	420	0
18568	18568	2314	2.74	0
18569	18569	2314	67	0
18570	18570	2314	41	0
18571	18571	2314	4.3	0
18572	18572	2314	0	0
18573	18573	2314	0	0
18574	18574	2314	95	0
18575	18575	2314	15	0
18576	18576	2314	0	0
18577	18577	2314	84	0
18578	18578	2314	65	0
18579	18579	2314	10	0
18580	18580	2314	400	0
18581	18581	2314	420	0
18582	18582	2314	2.74	0
18583	18583	2314	67	0
18584	18584	2314	41	0
18585	18585	2314	4.3	0
18586	18586	2314	0	0
18587	18587	2314	0	0
18588	18588	2314	95	0
18589	18589	2314	15	0
18590	18590	2314	0	0
18591	18591	2314	103.3	0
18592	18592	2314	31	0
18593	18593	2314	130.6	0
18594	18594	2314	56.6	0
18595	18595	2314	15.1	0
18596	18596	2314	12.6	0
18597	18597	2314	8.27	0
18598	18598	2314	1.91	0
18599	18599	2314	8.301	0
18600	18600	2314	1.9792	0
18601	18601	2314	.	0
18602	18602	2314	.	0
18603	18603	2314	0	0
18604	18604	2314	1	0
18605	18605	2314	0.3	0
18606	18606	2314	0.3	0
18607	18607	2314		0
18608	18608	2314		0
18609	18609	2314	1	0
18610	18610	2314	0	0
18611	18611	2314	0.8	0
18612	18612	2314	0.4	0
18613	18613	2314	0.4	0
18614	18614	2314	0.4	0
18615	18615	2314		0
18616	18616	2314		0
18617	18617	2314		0
18618	18618	2314		0
18619	18619	2314	2	0
18620	18620	2314	3	0
18621	18621	2314	1	0
18622	18622	2314	3	0
18623	18623	2314	2	0
18624	18624	2314	1	0
18625	18625	2314	0	0
18626	18626	2314	0	0
18627	18627	2314	0	0
18628	18628	2314	0	0
18629	18629	2314	0	0
18630	18630	2314	0	0
18631	18631	2314	0	0
18632	18632	2314	0	0
18633	18633	2314	0	0
18634	18634	2314	0	0
18635	18635	2314	B0L1	0
18636	18636	2314	0.1	0
18637	18637	2314	0.1	0
18638	18638	2314	.	0
18639	18639	2314	0	0
18640	18640	2314	.	0
18641	18641	2314	0	0
18642	18642	2314	.	0
18643	18643	2314	0	0
18644	18644	2314	.	0
18645	18645	2314	.	0
18646	18646	2314	0	0
18647	18647	2314	0	0
18648	18648	2314	0	0
18649	18649	2314		0
18650	18650	2314		0
18651	18651	2314	0	0
18652	18652	2314		0
18653	18653	2314		0
18654	18654	2314		0
18655	18655	2314		0
18656	18656	2314		0
18657	18657	2314		0
18658	18658	2314		0
18659	18659	2314		0
18660	18660	2314		0
18661	18661	2314		0
18662	18662	2314		0
18663	18663	2314		0
18664	18664	2314		0
18665	18665	2314		0
18666	18666	2314		0
18667	18667	2314		0
18668	18668	2314		0
18669	18669	2314		0
18670	18670	2314		0
18671	18671	2314		0
18672	18672	2314		0
18673	18673	2314	.	0
18674	18674	2314	.	0
18675	18675	2314	.	0
18676	18676	2314	.	0
18677	18677	2314	.	0
18678	18678	2314	.	0
18679	18679	2314	.	0
18680	18680	2314	.	0
18681	18681	2314	.	0
18682	18682	2314	.	0
18683	18683	2314	.	0
18684	18684	2314	.	0
18685	18685	2314	.	0
18686	18686	2314	.	0
18687	18687	2314	.	0
18688	18688	2314	.	0
18689	18689	2314	.	0
18690	18690	2314		0
18691	18691	2314		0
18692	18692	2314	.	0
18693	18693	2314	.	0
18694	18694	2314	10	0
18695	18695	2314	2	0
18696	18696	2314	0	0
18697	18697	2314	1	0
18698	18698	2314	0.4	0
18699	18699	2314	0.4	0
18700	18700	2314		0
18701	18701	2314		0
18702	18702	2314	2	0
18703	18703	2314	0	0
18704	18704	2314	0.8	0
18705	18705	2314	0.4	0
18706	18706	2314	0.5	0
18707	18707	2314	0.8	0
18708	18708	2314		0
18709	18709	2314		0
18710	18710	2314		0
18711	18711	2314		0
18712	18712	2314	2	0
18713	18713	2314	7	0
18714	18714	2314	2	0
18715	18715	2314	1	0
18716	18716	2314	5	0
18717	18717	2314	1	0
18718	18718	2314	1	0
18719	18719	2314	0	0
18720	18720	2314	0	0
18721	18721	2314	0	0
18722	18722	2314	0	0
18723	18723	2314	0	0
18724	18724	2314	0	0
18725	18725	2314	0	0
18726	18726	2314	0	0
18727	18727	2314	0	0
18728	18728	2314	B0L4	0
18729	18729	2314	0.8	0
18730	18730	2314	1.1	0
18731	18731	2314	24.6	0
18732	18732	2314	10	0
18733	18733	2314	24.7	0
18734	18734	2314	13	0
18735	18735	2314	2.1	0
18736	18736	2314	70	0
18737	18737	2314	2.2	0
18738	18738	2314	2.2	0
18739	18739	2314	0	0
18740	18740	2314	0	0
18741	18741	2314	0	0
18742	18742	2314	74	0
18743	18743	2314	125	0
18744	18744	2314	125	0
18745	18745	2314	10	0
18746	18746	2314	10	0
18747	18747	2314	90	0
18748	18748	2314	0	0
18749	18749	2314	2.931666667	0
18750	18750	2314	33.45	0
18751	18751	2314	7.366666667	0
18752	18752	2314	300	0
18753	18753	2314	350	0
18754	18754	2314	2.03	0
18755	18755	2314	86.5	0
18756	18756	2314	42.5	0
18757	18757	2314	3.2	0
18758	18758	2314	0	0
18759	18759	2314	3	0
18760	18760	2314	85	0
18761	18761	2314	20	0
18762	18762	2314	0	0
18763	18763	2314	25	0
18764	18764	2314	137	0
18765	18765	2314	14	0
18766	18766	2314	300	0
18767	18767	2314	350	0
18768	18768	2314	2.03	0
18769	18769	2314	86.5	0
18770	18770	2314	42.5	0
18771	18771	2314	3.2	0
18772	18772	2314	0	0
18773	18773	2314	3	0
18774	18774	2314	85	0
18775	18775	2314	20	0
18776	18776	2314	0	0
18777	18777	2314	67.9	0
18778	18778	2314	19.7	0
18779	18779	2314	45.1	0
18780	18780	2314	22.6	0
18781	18781	2314	5.1	0
18782	18782	2314	4.2	0
18783	18783	2314	10.15	0
18784	18784	2314	1.9	0
18785	18785	2314	10.1697	0
18786	18786	2314	1.9268	0
18787	18787	2314	9	0
18788	18788	2314	23	0
18789	18789	2314	2	0
18790	18790	2314	3	0
18791	18791	2314	0.5	0
18792	18792	2314	0.5	0
18793	18793	2314	0.9	0
18794	18794	2314	0.9	0
18795	18795	2314	3	0
18796	18796	2314	0	0
18797	18797	2314	0.9	0
18798	18798	2314	0.5	0
18799	18799	2314	0.5	0
18800	18800	2314	0.4	0
18801	18801	2314	0.5	0
18802	18802	2314	0.6	0
18803	18803	2314		0
18804	18804	2314		0
18805	18805	2314	0	0
18806	18806	2314	13	0
18807	18807	2314	3	0
18808	18808	2314	1	0
18809	18809	2314	7	0
18810	18810	2314	2	0
18811	18811	2314	3	0
18812	18812	2314	1	0
18813	18813	2314	0	0
18814	18814	2314	0	0
18815	18815	2314	0	0
18816	18816	2314	0	0
18817	18817	2314	0	0
18818	18818	2314	0	0
18819	18819	2314	0	0
18820	18820	2314	0	0
18821	18821	2314	B0L6	0
18822	18822	2314	1.1	0
18823	18823	2314	1.3	0
18824	18824	2314	27.5	0
18825	18825	2314	14	0
18826	18826	2314	32.1	0
18827	18827	2314	28	0
18828	18828	2314	10.3	0
18829	18829	2314	145	0
18830	18830	2314	2.8	0
18831	18831	2314	2.3	0
18832	18832	2314	0	0
18833	18833	2314	0	0
18834	18834	2314	0	0
18835	18835	2314	73	0
18836	18836	2314	0	0
18837	18837	2314	225	0
18838	18838	2314	0	0
18839	18839	2314	30	0
18840	18840	2314	0	0
18841	18841	2314	0	0
18842	18842	2314		0
18843	18843	2314		0
18844	18844	2314		0
18845	18845	2314		0
18846	18846	2314		0
18847	18847	2314		0
18848	18848	2314		0
18849	18849	2314		0
18850	18850	2314		0
18851	18851	2314		0
18852	18852	2314		0
18853	18853	2314		0
18854	18854	2314		0
18855	18855	2314		0
18856	18856	2314	48	0
18857	18857	2314		0
18858	18858	2314		0
18859	18859	2314		0
18860	18860	2314		0
18861	18861	2314		0
18862	18862	2314	.	0
18863	18863	2314	.	0
18864	18864	2314	.	0
18865	18865	2314	.	0
18866	18866	2314	.	0
18867	18867	2314	.	0
18868	18868	2314	.	0
18869	18869	2314	.	0
18870	18870	2314	.	0
18871	18871	2314	.	0
18872	18872	2314	.	0
18873	18873	2314	.	0
18874	18874	2314	.	0
18875	18875	2314	.	0
18876	18876	2314	13.59	0
18877	18877	2314	1.12	0
18878	18878	2314	13.59	0
18879	18879	2314	1.12	0
18880	18880	2314	4	0
18881	18881	2314	10	0
18882	18882	2314	1	0
18883	18883	2314	2	0
18884	18884	2314	0.4	0
18885	18885	2314	0.5	0
18886	18886	2314		0
18887	18887	2314		0
18888	18888	2314	2	0
18889	18889	2314	0	0
18890	18890	2314	0.7	0
18891	18891	2314	0.4	0
18892	18892	2314	0.4	0
18893	18893	2314	0.6	0
18894	18894	2314	0.5	0
18895	18895	2314	0.6	0
18896	18896	2314		0
18897	18897	2314		0
18898	18898	2314	2	0
18899	18899	2314	8	0
18900	18900	2314	2	0
18901	18901	2314	2	0
18902	18902	2314	6	0
18903	18903	2314	1	0
18904	18904	2314	1	0
18905	18905	2314	0	0
18906	18906	2314	0	0
18907	18907	2314	0	0
18908	18908	2314	0	0
18909	18909	2314	0	0
18910	18910	2314	0	0
18911	18911	2314	0	0
18912	18912	2314	0	0
18913	18913	2314	0	0
18914	18914	2314	B0L5	0
18915	18915	2314	1	0
18916	18916	2314	1.2	0
18917	18917	2314	21.5	0
18918	18918	2314	11	0
18919	18919	2314	30.2	0
18920	18920	2314	11	0
18921	18921	2314	7.2	0
18922	18922	2314	67	0
18923	18923	2314	2.6	0
18924	18924	2314	2.4	0
18925	18925	2314	0	0
18926	18926	2314	2	0
18927	18927	2314	0	0
18928	18928	2314	59.5	0
18929	18929	2314	200	0
18930	18930	2314	200	0
18931	18931	2314	15	0
18932	18932	2314	20	0
18933	18933	2314	85	0
18934	18934	2314	0	0
18935	18935	2314	3.556666667	0
18936	18936	2314	44.55	0
18937	18937	2314	10.58333333	0
18938	18938	2314	150	0
18939	18939	2314		0
18940	18940	2314	2.36	0
18941	18941	2314	84	0
18942	18942	2314	49	0
18943	18943	2314	2.74	0
18944	18944	2314	0	0
18945	18945	2314	0	0
18946	18946	2314	90	0
18947	18947	2314	0	0
18948	18948	2314	0	0
18949	18949	2314	89	0
18950	18950	2314	123	0
18951	18951	2314	5	0
18952	18952	2314	150	0
18953	18953	2314		0
18954	18954	2314	2.36	0
18955	18955	2314	84	0
18956	18956	2314	49	0
18957	18957	2314	2.74	0
18958	18958	2314	0	0
18959	18959	2314	0	0
18960	18960	2314	90	0
18961	18961	2314	0	0
18962	18962	2314	0	0
18963	18963	2314	93.7	0
18964	18964	2314	26.8	0
18965	18965	2314	34.1	0
18966	18966	2314	20.9	0
18967	18967	2314	4.9	0
18968	18968	2314	4.8	0
18969	18969	2314	13.31	0
18970	18970	2314	3.42	0
18971	18971	2314	13.3368	0
18972	18972	2314	3.4457	0
18973	18973	2314	15	0
18974	18974	2314	1	0
18975	18975	2314	2	0
18976	18976	2314	2	0
18977	18977	2314	0.5	0
18978	18978	2314	0.6	0
18979	18979	2314	0.7	0
18980	18980	2314	0.9	0
18981	18981	2314	3	0
18982	18982	2314	0	0
18983	18983	2314	1	0
18984	18984	2314	0.4	0
18985	18985	2314	0.5	0
18986	18986	2314	0.7	0
18987	18987	2314	0.3	0
18988	18988	2314	0.5	0
18989	18989	2314	0.4	0
18990	18990	2314	0.5	0
18991	18991	2314	0	0
18992	18992	2314	10	0
18993	18993	2314	3	0
18994	18994	2314	1	0
18995	18995	2314	7	0
18996	18996	2314	2	0
18997	18997	2314	0	0
18998	18998	2314	1	0
18999	18999	2314	0	0
19000	19000	2314	0	0
19001	19001	2314	0	0
19002	19002	2314	0	0
19003	19003	2314	0	0
19004	19004	2314	0	0
19005	19005	2314	0	0
19006	19006	2314	0	0
19007	19007	2314	B0L6	0
19008	19008	2314	1	0
19009	19009	2314	1.3	0
19010	19010	2314	17	0
19011	19011	2314	7	0
19012	19012	2314	23.5	0
19013	19013	2314	3	0
19014	19014	2314	1.2	0
19015	19015	2314	44	0
19016	19016	2314	1.9	0
19017	19017	2314	2	0
19018	19018	2314	0	0
19019	19019	2314	0	0
19020	19020	2314	0	0
19021	19021	2314	48.5	0
19022	19022	2314	75	0
19023	19023	2314	75	0
19024	19024	2314	15	0
19025	19025	2314	0	0
19026	19026	2314	95	0
19027	19027	2314	0	0
19028	19028	2314	2.251666667	0
19029	19029	2314	29.16666667	0
19030	19030	2314	6.4	0
19031	19031	2314	200	0
19032	19032	2314		0
19033	19033	2314	2.58	0
19034	19034	2314	61	0
19035	19035	2314	42	0
19036	19036	2314	2.38	0
19037	19037	2314	60	0
19038	19038	2314	0	0
19039	19039	2314	90	0
19040	19040	2314	0	0
19041	19041	2314	0	0
19042	19042	2314	4	0
19043	19043	2314	117	0
19044	19044	2314	68	0
19045	19045	2314	200	0
19046	19046	2314		0
19047	19047	2314	2.58	0
19048	19048	2314	61	0
19049	19049	2314	42	0
19050	19050	2314	2.38	0
19051	19051	2314	60	0
19052	19052	2314	0	0
19053	19053	2314	90	0
19054	19054	2314	0	0
19055	19055	2314	0	0
19056	19056	2314	70	0
19057	19057	2314	22.8	0
19058	19058	2314	47.5	0
19059	19059	2314	28.3	0
19060	19060	2314	.	0
19061	19061	2314	.	0
19062	19062	2314	12.58	0
19063	19063	2314	4.14	0
19064	19064	2314	12.6028	0
19065	19065	2314	4.14	0
19066	19066	2314	34	0
19067	19067	2314	9	0
19068	19068	2314	2	0
19069	19069	2314	2	0
19070	19070	2314	0.3	0
19071	19071	2314	0.5	0
19072	19072	2314	0.5	0
19073	19073	2314	0.8	0
19074	19074	2314	3	0
19075	19075	2314	0	0
19076	19076	2314	1	0
19077	19077	2314	0.4	0
19078	19078	2314	0.5	0
19079	19079	2314	0.4	0
19080	19080	2314	0.5	0
19081	19081	2314	0.7	0
19082	19082	2314		0
19083	19083	2314		0
19084	19084	2314	0	0
19085	19085	2314	11	0
19086	19086	2314	3	0
19087	19087	2314	1	0
19088	19088	2314	5	0
19089	19089	2314	3	0
19090	19090	2314	2	0
19091	19091	2314	1	0
19092	19092	2314	0	0
19093	19093	2314	0	0
19094	19094	2314	0	0
19095	19095	2314	0	0
19096	19096	2314	0	0
19097	19097	2314	0	0
19098	19098	2314	0	0
19099	19099	2314	0	0
19100	19100	2314	B0L4	0
19101	19101	2314	0.9	0
19102	19102	2314	1.3	0
19103	19103	2314	20.2	0
19104	19104	2314	10	0
19105	19105	2314	30.2	0
19106	19106	2314	24	0
19107	19107	2314	6.3	0
19108	19108	2314	115	0
19109	19109	2314	2.6	0
19110	19110	2314	2.3	0
19111	19111	2314	0	0
19112	19112	2314	0	0
19113	19113	2314	0	0
19114	19114	2314	76	0
19115	19115	2314	175	0
19116	19116	2314	175	0
19117	19117	2314	10	0
19118	19118	2314	30	0
19119	19119	2314	85	0
19120	19120	2314	0	0
19121	19121	2314	2.833333333	0
19122	19122	2314	34.23333333	0
19123	19123	2314	7.466666667	0
19124	19124	2314	100	0
19125	19125	2314	150	0
19126	19126	2314	2.25	0
19127	19127	2314	109	0
19128	19128	2314	29	0
19129	19129	2314	2.98	0
19130	19130	2314	0	0
19131	19131	2314	0	0
19132	19132	2314	85	0
19133	19133	2314	0	0
19134	19134	2314	0	0
19135	19135	2314	99	0
19136	19136	2314	73	0
19137	19137	2314	2	0
19138	19138	2314	100	0
19139	19139	2314	150	0
19140	19140	2314	2.25	0
19141	19141	2314	109	0
19142	19142	2314	29	0
19143	19143	2314	2.98	0
19144	19144	2314	0	0
19145	19145	2314	0	0
19146	19146	2314	85	0
19147	19147	2314	0	0
19148	19148	2314	0	0
19149	19149	2314	51.5	0
19150	19150	2314	22.6	0
19151	19151	2314	43.7	0
19152	19152	2314	30.9	0
19153	19153	2314	8.7	0
19154	19154	2314	8.1	0
19155	19155	2314	11.98	0
19156	19156	2314	1.92	0
19157	19157	2314	12.0026	0
19158	19158	2314	1.959	0
19159	19159	2314	14	0
19160	19160	2314	5	0
19161	19161	2314	0	0
19162	19162	2314	1	0
19163	19163	2314	0.4	0
19164	19164	2314	0.5	0
19165	19165	2314		0
19166	19166	2314		0
19167	19167	2314	2	0
19168	19168	2314	0	0
19169	19169	2314	0.6	0
19170	19170	2314	0.3	0
19171	19171	2314	0.4	0
19172	19172	2314	0.6	0
19173	19173	2314		0
19174	19174	2314		0
19175	19175	2314		0
19176	19176	2314		0
19177	19177	2314	1	0
19178	19178	2314	7	0
19179	19179	2314	2	0
19180	19180	2314	1	0
19181	19181	2314	5	0
19182	19182	2314	1	0
19183	19183	2314	1	0
19184	19184	2314	0	0
19185	19185	2314	0	0
19186	19186	2314	0	0
19187	19187	2314	0	0
19188	19188	2314	0	0
19189	19189	2314	0	0
19190	19190	2314	0	0
19191	19191	2314	0	0
19192	19192	2314	0	0
19193	19193	2314	B0L4	0
19194	19194	2314	0.9	0
19195	19195	2314	1.1	0
19196	19196	2314	13.7	0
19197	19197	2314	7	0
19198	19198	2314	30.7	0
19199	19199	2314	25	0
19200	19200	2314	4.6	0
19201	19201	2314	126	0
19202	19202	2314	2.2	0
19203	19203	2314	2.1	0
19204	19204	2314	0	0
19205	19205	2314	0	0
19206	19206	2314	0	0
19207	19207	2314	76.5	0
19208	19208	2314	150	0
19209	19209	2314	150	0
19210	19210	2314	10	0
19211	19211	2314	40	0
19212	19212	2314	95	0
19213	19213	2314	0	0
19214	19214	2314	3.738333333	0
19215	19215	2314	48.03333333	0
19216	19216	2314	9.35	0
19217	19217	2314	150	0
19218	19218	2314		0
19219	19219	2314	2.59	0
19220	19220	2314	87	0
19221	19221	2314	60	0
19222	19222	2314	2.55	0
19223	19223	2314	0	0
19224	19224	2314	0	0
19225	19225	2314	75	0
19226	19226	2314	0	0
19227	19227	2314	0	0
19228	19228	2314	118	0
19229	19229	2314	62	0
19230	19230	2314	9	0
19231	19231	2314	150	0
19232	19232	2314		0
19233	19233	2314	2.59	0
19234	19234	2314	87	0
19235	19235	2314	60	0
19236	19236	2314	2.55	0
19237	19237	2314	0	0
19238	19238	2314	0	0
19239	19239	2314	75	0
19240	19240	2314	0	0
19241	19241	2314	0	0
19242	19242	2314	45.7	0
19243	19243	2314	17.3	0
19244	19244	2314	45.4	0
19245	19245	2314	23.3	0
19246	19246	2314	6.8	0
19247	19247	2314	5.3	0
19248	19248	2314	10.37	0
19249	19249	2314	2.69	0
19250	19250	2314	10.3873	0
19251	19251	2314	2.7186	0
19252	19252	2314	12	0
19253	19253	2314	1	0
19254	19254	2314	2	0
19255	19255	2314	2	0
19256	19256	2314	0.8	0
19257	19257	2314	0.9	0
19258	19258	2314	1.2	0
19259	19259	2314	1.2	0
19260	19260	2314	3	0
19261	19261	2314	0	0
19262	19262	2314	1.1	0
19263	19263	2314	0.5	0
19264	19264	2314	0.6	0
19265	19265	2314	1	0
19266	19266	2314	0.7	0
19267	19267	2314	0.9	0
19268	19268	2314	0.8	0
19269	19269	2314	1	0
19270	19270	2314	2	0
19271	19271	2314	20	0
19272	19272	2314	4	0
19273	19273	2314	1	0
19274	19274	2314	8	0
19275	19275	2314	4	0
19276	19276	2314	4	0
19277	19277	2314	3	0
19278	19278	2314	1	0
19279	19279	2314	0	0
19280	19280	2314	0	0
19281	19281	2314	0	0
19282	19282	2314	0	0
19283	19283	2314	0	0
19284	19284	2314	0	0
19285	19285	2314	0	0
19286	19286	2314	B0L6	0
19287	19287	2314	1.5	0
19288	19288	2314	1.6	0
19289	19289	2314	21.5	0
19290	19290	2314	14	0
19291	19291	2314	41.1	0
19292	19292	2314	40	0
19293	19293	2314	13.8	0
19294	19294	2314	245	0
19295	19295	2314	2.8	0
19296	19296	2314	2.3	0
19297	19297	2314	10	0
19298	19298	2314	16	0
19299	19299	2314	1	0
19300	19300	2314	94	0
19301	19301	2314	200	0
19302	19302	2314	200	0
19303	19303	2314	5	0
19304	19304	2314	40	0
19305	19305	2314	85	0
19306	19306	2314	0	0
19307	19307	2314	2.563333333	0
19308	19308	2314	33.45	0
19309	19309	2314	7.816666667	0
19310	19310	2314	75	0
19311	19311	2314	100	0
19312	19312	2314	2.31	0
19313	19313	2314	92	0
19314	19314	2314	49	0
19315	19315	2314	2.22	0
19316	19316	2314	0	0
19317	19317	2314	0	0
19318	19318	2314	70	0
19319	19319	2314	0	0
19320	19320	2314	0	0
19321	19321	2314	148	0
19322	19322	2314	83	0
19323	19323	2314	3	0
19324	19324	2314	75	0
19325	19325	2314	100	0
19326	19326	2314	2.31	0
19327	19327	2314	92	0
19328	19328	2314	49	0
19329	19329	2314	2.22	0
19330	19330	2314	0	0
19331	19331	2314	0	0
19332	19332	2314	70	0
19333	19333	2314	0	0
19334	19334	2314	0	0
19335	19335	2314	96.2	0
19336	19336	2314	33.6	0
19337	19337	2314	70.1	0
19338	19338	2314	34.6	0
19339	19339	2314	6.2	0
19340	19340	2314	5.8	0
19341	19341	2314	11.5	0
19342	19342	2314	1.73	0
19343	19343	2314	11.5336	0
19344	19344	2314	1.7704	0
19345	19345	2314	20	0
19346	19346	2314	6	0
19347	19347	2314	1	0
19348	19348	2314	2	0
19349	19349	2314	0.6	0
19350	19350	2314	0.9	0
19351	19351	2314		0
19352	19352	2314		0
19353	19353	2314	3	0
19354	19354	2314	0	0
19355	19355	2314	0.8	0
19356	19356	2314	0.4	0
19357	19357	2314	0.7	0
19358	19358	2314	1	0
19359	19359	2314	0.6	0
19360	19360	2314	0.8	0
19361	19361	2314		0
19362	19362	2314		0
19363	19363	2314	0	0
19364	19364	2314	13	0
19365	19365	2314	3	0
19366	19366	2314	1	0
19367	19367	2314	7	0
19368	19368	2314	2	0
19369	19369	2314	3	0
19370	19370	2314	1	0
19371	19371	2314	0	0
19372	19372	2314	0	0
19373	19373	2314	0	0
19374	19374	2314	0	0
19375	19375	2314	0	0
19376	19376	2314	0	0
19377	19377	2314	0	0
19378	19378	2314	0	0
19379	19379	2314	B0L5	0
19380	19380	2314	1.2	0
19381	19381	2314	1.3	0
19382	19382	2314	19.6	0
19383	19383	2314	14	0
19384	19384	2314	29.9	0
19385	19385	2314	28	0
19386	19386	2314	10.7	0
19387	19387	2314	174	0
19388	19388	2314	2.2	0
19389	19389	2314	1.7	0
19390	19390	2314	0	0
19391	19391	2314	0	0
19392	19392	2314	0	0
19393	19393	2314	58	0
19394	19394	2314	150	0
19395	19395	2314	150	0
19396	19396	2314	15	0
19397	19397	2314	4	0
19398	19398	2314	80	0
19399	19399	2314	0	0
19400	19400	2314	1.281666667	0
19401	19401	2314	17.6	0
19402	19402	2314	3.55	0
19403	19403	2314	150	0
19404	19404	2314	250	0
19405	19405	2314	2.31	0
19406	19406	2314	73.5	0
19407	19407	2314	32.5	0
19408	19408	2314	2.94	0
19409	19409	2314	0	0
19410	19410	2314	0	0
19411	19411	2314	75	0
19412	19412	2314	0	0
19413	19413	2314	0	0
19414	19414	2314	86	0
19415	19415	2314	37	0
19416	19416	2314	0	0
19417	19417	2314	150	0
19418	19418	2314	250	0
19419	19419	2314	2.31	0
19420	19420	2314	73.5	0
19421	19421	2314	32.5	0
19422	19422	2314	2.94	0
19423	19423	2314	0	0
19424	19424	2314	0	0
19425	19425	2314	75	0
19426	19426	2314	0	0
19427	19427	2314	0	0
19428	19428	2314	41.9	0
19429	19429	2314	20.4	0
19430	19430	2314	63.3	0
19431	19431	2314	28.6	0
19432	19432	2314	3.7	0
19433	19433	2314	3.3	0
19434	19434	2314	5.15	0
19435	19435	2314	0.87	0
19436	19436	2314	5.1704	0
19437	19437	2314	0.9019	0
19438	19438	2314	4	0
19439	19439	2314	8	0
19440	19440	2314	2	0
19441	19441	2314	3	0
19442	19442	2314	0.6	0
19443	19443	2314	1.1	0
19444	19444	2314	1.1	0
19445	19445	2314	1.4	0
19446	19446	2314	4	0
19447	19447	2314	0	0
19448	19448	2314	0.9	0
19449	19449	2314	0.5	0
19450	19450	2314	0.7	0
19451	19451	2314	1.2	0
19452	19452	2314	0.8	0
19453	19453	2314	1	0
19454	19454	2314	0.7	0
19455	19455	2314	0.8	0
19456	19456	2314	2	0
19457	19457	2314	27	0
19458	19458	2314	7	0
19459	19459	2314	1	0
19460	19460	2314	10	0
19461	19461	2314	5	0
19462	19462	2314	4	0
19463	19463	2314	3	0
19464	19464	2314	2	0
19465	19465	2314	0	0
19466	19466	2314	0	0
19467	19467	2314	1	0
19468	19468	2314	1	0
19469	19469	2314	1	0
19470	19470	2314	0	0
19471	19471	2314	0	0
19472	19472	2314	B0L8	0
19473	19473	2314	1.2	0
19474	19474	2314	1.1	0
19475	19475	2314	15.3	0
19476	19476	2314	10	0
19477	19477	2314	37.1	0
19478	19478	2314	36	0
19479	19479	2314	20	0
19480	19480	2314	230	0
19481	19481	2314	2.2	0
19482	19482	2314	1.8	0
19483	19483	2314	4	0
19484	19484	2314	7	0
19485	19485	2314	0	0
19486	19486	2314	67.5	0
19487	19487	2314	150	0
19488	19488	2314	150	0
19489	19489	2314	5	0
19490	19490	2314	25	0
19491	19491	2314	98	0
19492	19492	2314	0	0
19493	19493	2314	1.713333333	0
19494	19494	2314	20.98333333	0
19495	19495	2314	4.55	0
19496	19496	2314	80	0
19497	19497	2314		0
19498	19498	2314	1.89	0
19499	19499	2314	78	0
19500	19500	2314	38	0
19501	19501	2314	3.32	0
19502	19502	2314	0	0
19503	19503	2314	0	0
19504	19504	2314	70	0
19505	19505	2314	0	0
19506	19506	2314	0	0
19507	19507	2314	100	0
19508	19508	2314	28	0
19509	19509	2314	29	0
19510	19510	2314	80	0
19511	19511	2314		0
19512	19512	2314	1.89	0
19513	19513	2314	78	0
19514	19514	2314	38	0
19515	19515	2314	3.32	0
19516	19516	2314	0	0
19517	19517	2314	0	0
19518	19518	2314	70	0
19519	19519	2314	0	0
19520	19520	2314	0	0
19521	19521	2314	.	0
19522	19522	2314	.	0
19523	19523	2314	64.3	0
19524	19524	2314	36	0
19525	19525	2314	7.3	0
19526	19526	2314	6.3	0
19527	19527	2314	6.95	0
19528	19528	2314	1.34	0
19529	19529	2314	6.95	0
19530	19530	2314	1.3823	0
19531	19531	2314	2	0
19532	19532	2314	2	0
19533	19533	2314	2	0
19534	19534	2314	3	0
19535	19535	2314	0.5	0
19536	19536	2314	0.8	0
19537	19537	2314	1.2	0
19538	19538	2314	1.5	0
19539	19539	2314	3	0
19540	19540	2314	0	0
19541	19541	2314	1	0
19542	19542	2314	0.4	0
19543	19543	2314	0.7	0
19544	19544	2314	1	0
19545	19545	2314	0.7	0
19546	19546	2314	1	0
19547	19547	2314	0.6	0
19548	19548	2314	0.9	0
19549	19549	2314	2	0
19550	19550	2314	19	0
19551	19551	2314	4	0
19552	19552	2314	1	0
19553	19553	2314	7	0
19554	19554	2314	4	0
19555	19555	2314	4	0
19556	19556	2314	3	0
19557	19557	2314	1	0
19558	19558	2314	0	0
19559	19559	2314	0	0
19560	19560	2314	0	0
19561	19561	2314	0	0
19562	19562	2314	0	0
19563	19563	2314	0	0
19564	19564	2314	0	0
19565	19565	2314	B0L6	0
19566	19566	2314	1.2	0
19567	19567	2314	1.45	0
19568	19568	2314	25.5	0
19569	19569	2314	11	0
19570	19570	2314	44.6	0
19571	19571	2314	41	0
19572	19572	2314	19.5	0
19573	19573	2314	175	0
19574	19574	2314	2.8	0
19575	19575	2314	2.4	0
19576	19576	2314	0	0
19577	19577	2314	2	0
19578	19578	2314	0	0
19579	19579	2314	93	0
19580	19580	2314	250	0
19581	19581	2314	250	0
19582	19582	2314	15	0
19583	19583	2314	35	0
19584	19584	2314	95	0
19585	19585	2314	0	0
19586	19586	2314	2.586666667	0
19587	19587	2314	30.35	0
19588	19588	2314	6.983333333	0
19589	19589	2314	220	0
19590	19590	2314		0
19591	19591	2314	2.85	0
19592	19592	2314	93	0
19593	19593	2314	60	0
19594	19594	2314	3.39	0
19595	19595	2314	25	0
19596	19596	2314	0	0
19597	19597	2314	90	0
19598	19598	2314	0	0
19599	19599	2314	0	0
19600	19600	2314	175	0
19601	19601	2314	92	0
19602	19602	2314	12	0
19603	19603	2314	220	0
19604	19604	2314		0
19605	19605	2314	2.85	0
19606	19606	2314	93	0
19607	19607	2314	60	0
19608	19608	2314	3.39	0
19609	19609	2314	25	0
19610	19610	2314	0	0
19611	19611	2314	90	0
19612	19612	2314	0	0
19613	19613	2314	0	0
19614	19614	2314	84.5	0
19615	19615	2314	27	0
19616	19616	2314	134.7	0
19617	19617	2314	52	0
19618	19618	2314	8.4	0
19619	19619	2314	6.3	0
19620	19620	2314	11.87	0
19621	19621	2314	2.73	0
19622	19622	2314	11.897	0
19623	19623	2314	2.7883	0
19624	19624	2314	5	0
19625	19625	2314	1	0
19626	19626	2314	2	0
19627	19627	2314	2	0
19628	19628	2314	0.3	0
19629	19629	2314	0.3	0
19630	19630	2314	1.1	0
19631	19631	2314	1	0
19632	19632	2314	3	0
19633	19633	2314	0	0
19634	19634	2314	0.8	0
19635	19635	2314	0.5	0
19636	19636	2314	.	0
19637	19637	2314	.	0
19638	19638	2314	0.5	0
19639	19639	2314	0.8	0
19640	19640	2314		0
19641	19641	2314		0
19642	19642	2314	0	0
19643	19643	2314	14	0
19644	19644	2314	4	0
19645	19645	2314	1	0
19646	19646	2314	6	0
19647	19647	2314	2	0
19648	19648	2314	3	0
19649	19649	2314	2	0
19650	19650	2314	1	0
19651	19651	2314	0	0
19652	19652	2314	0	0
19653	19653	2314	0	0
19654	19654	2314	0	0
19655	19655	2314	0	0
19656	19656	2314	0	0
19657	19657	2314	0	0
19658	19658	2314	B0L5	0
19659	19659	2314	1.15	0
19660	19660	2314	1.4	0
19661	19661	2314	28.7	0
19662	19662	2314	13	0
19663	19663	2314	32.6	0
19664	19664	2314	20	0
19665	19665	2314	4.4	0
19666	19666	2314	88	0
19667	19667	2314	2.8	0
19668	19668	2314	1.9	0
19669	19669	2314	0	0
19670	19670	2314	0	0
19671	19671	2314	0	0
19672	19672	2314	58.5	0
19673	19673	2314	150	0
19674	19674	2314	150	0
19675	19675	2314	15	0
19676	19676	2314	20	0
19677	19677	2314	85	0
19678	19678	2314	0	0
19679	19679	2314	2.441666667	0
19680	19680	2314	32.71666667	0
19681	19681	2314	7.616666667	0
19682	19682	2314	220	0
19683	19683	2314		0
19684	19684	2314	2.17	0
19685	19685	2314	87	0
19686	19686	2314	36	0
19687	19687	2314	3.39	0
19688	19688	2314	0	0
19689	19689	2314	0	0
19690	19690	2314	80	0
19691	19691	2314	10	0
19692	19692	2314	0	0
19693	19693	2314	50	0
19694	19694	2314	35	0
19695	19695	2314	11	0
19696	19696	2314	220	0
19697	19697	2314		0
19698	19698	2314	2.17	0
19699	19699	2314	87	0
19700	19700	2314	36	0
19701	19701	2314	3.39	0
19702	19702	2314	0	0
19703	19703	2314	0	0
19704	19704	2314	80	0
19705	19705	2314	10	0
19706	19706	2314	0	0
19707	19707	2314	57.7	0
19708	19708	2314	22.7	0
19709	19709	2314	88.1	0
19710	19710	2314	46	0
19711	19711	2314	5.5	0
19712	19712	2314	5.4	0
19713	19713	2314	10.75	0
19714	19714	2314	1.88	0
19715	19715	2314	10.7727	0
19716	19716	2314	1.9314	0
19717	19717	2314	40	0
19718	19718	2314	3	0
19719	19719	2314	2	0
19720	19720	2314	2	0
19721	19721	2314	0.4	0
19722	19722	2314	0.6	0
19723	19723	2314		0
19724	19724	2314		0
19725	19725	2314	3	0
19726	19726	2314	0	0
19727	19727	2314	1.2	0
19728	19728	2314	0.6	0
19729	19729	2314	0.8	0
19730	19730	2314	1.3	0
19731	19731	2314	0.8	0
19732	19732	2314	1.2	0
19733	19733	2314		0
19734	19734	2314		0
19735	19735	2314	1	0
19736	19736	2314	16	0
19737	19737	2314	4	0
19738	19738	2314	1	0
19739	19739	2314	7	0
19740	19740	2314	3	0
19741	19741	2314	3	0
19742	19742	2314	2	0
19743	19743	2314	1	0
19744	19744	2314	0	0
19745	19745	2314	0	0
19746	19746	2314	0	0
19747	19747	2314	0	0
19748	19748	2314	0	0
19749	19749	2314	0	0
19750	19750	2314	0	0
19751	19751	2314	B0L6	0
19752	19752	2314	1.6	0
19753	19753	2314	1.8	0
19754	19754	2314	25.5	0
19755	19755	2314	12	0
19756	19756	2314	33.2	0
19757	19757	2314	35	0
19758	19758	2314	20.1	0
19759	19759	2314	200	0
19760	19760	2314	2.9	0
19761	19761	2314	2.4	0
19762	19762	2314	0	0
19763	19763	2314	0	0
19764	19764	2314	0	0
19765	19765	2314	84.5	0
19766	19766	2314	175	0
19767	19767	2314	175	0
19768	19768	2314	10	0
19769	19769	2314	35	0
19770	19770	2314	85	0
19771	19771	2314	0	0
19772	19772	2314	2.625	0
19773	19773	2314	34.4	0
19774	19774	2314	8.2	0
19775	19775	2314	40	0
19776	19776	2314	80	0
19777	19777	2314	2.1	0
19778	19778	2314	95	0
19779	19779	2314	48	0
19780	19780	2314	3.97	0
19781	19781	2314	0	0
19782	19782	2314	0	0
19783	19783	2314	50	0
19784	19784	2314	0	0
19785	19785	2314	0	0
19786	19786	2314	58	0
19787	19787	2314	24	0
19788	19788	2314	10	0
19789	19789	2314	40	0
19790	19790	2314	80	0
19791	19791	2314	2.1	0
19792	19792	2314	95	0
19793	19793	2314	48	0
19794	19794	2314	3.97	0
19795	19795	2314	0	0
19796	19796	2314	0	0
19797	19797	2314	50	0
19798	19798	2314	0	0
19799	19799	2314	0	0
19800	19800	2314	92.2	0
19801	19801	2314	30.9	0
19802	19802	2314	226.5	0
19803	19803	2314	78.5	0
19804	19804	2314	6.9	0
19805	19805	2314	5.7	0
19806	19806	2314	11.34	0
19807	19807	2314	1.9	0
19808	19808	2314	11.3709	0
19809	19809	2314	1.9842	0
19810	19810	2314	3	0
19811	19811	2314	14	0
19812	19812	2314	1	0
19813	19813	2314	2	0
19814	19814	2314	0.4	0
19815	19815	2314	0.6	0
19816	19816	2314		0
19817	19817	2314		0
19818	19818	2314	2	0
19819	19819	2314	0	0
19820	19820	2314	0.7	0
19821	19821	2314	0.3	0
19822	19822	2314	0.5	0
19823	19823	2314	0.7	0
19824	19824	2314	0.5	0
19825	19825	2314	0.6	0
19826	19826	2314		0
19827	19827	2314		0
19828	19828	2314	0	0
19829	19829	2314	7	0
19830	19830	2314	1	0
19831	19831	2314	2	0
19832	19832	2314	6	0
19833	19833	2314	1	0
19834	19834	2314	0	0
19835	19835	2314	0	0
19836	19836	2314	0	0
19837	19837	2314	0	0
19838	19838	2314	0	0
19839	19839	2314	0	0
19840	19840	2314	0	0
19841	19841	2314	0	0
19842	19842	2314	0	0
19843	19843	2314	0	0
19844	19844	2314	B0L5	0
19845	19845	2314	1	0
19846	19846	2314	1.2	0
19847	19847	2314	23.5	0
19848	19848	2314	12	0
19849	19849	2314	28.5	0
19850	19850	2314	13	0
19851	19851	2314	4.5	0
19852	19852	2314	89	0
19853	19853	2314	2.2	0
19854	19854	2314	1.9	0
19855	19855	2314	0	0
19856	19856	2314	3	0
19857	19857	2314	0	0
19858	19858	2314	109.5	0
19859	19859	2314	150	0
19860	19860	2314	150	0
19861	19861	2314	15	0
19862	19862	2314	10	0
19863	19863	2314	95	0
19864	19864	2314	0	0
19865	19865	2314	4.098333333	0
19866	19866	2314	57.31666667	0
19867	19867	2314	11.88333333	0
19868	19868	2314	400	0
19869	19869	2314	475	0
19870	19870	2314	3.93	0
19871	19871	2314	130	0
19872	19872	2314	53.5	0
19873	19873	2314	4.77	0
19874	19874	2314	0	0
19875	19875	2314	0	0
19876	19876	2314	95	0
19877	19877	2314	20	0
19878	19878	2314	0	0
19879	19879	2314	25	0
19880	19880	2314	45	0
19881	19881	2314	87	0
19882	19882	2314	400	0
19883	19883	2314	475	0
19884	19884	2314	3.93	0
19885	19885	2314	130	0
19886	19886	2314	53.5	0
19887	19887	2314	4.77	0
19888	19888	2314	0	0
19889	19889	2314	0	0
19890	19890	2314	95	0
19891	19891	2314	20	0
19892	19892	2314	0	0
19893	19893	2314	130.5	0
19894	19894	2314	40.6	0
19895	19895	2314	142.1	0
19896	19896	2314	52.2	0
19897	19897	2314	9.7	0
19898	19898	2314	6.3	0
19899	19899	2314	20.03	0
19900	19900	2314	4.52	0
19901	19901	2314	20.0706	0
19902	19902	2314	4.5785	0
19903	19903	2314	32	0
19904	19904	2314	11	0
19905	19905	2314	2	0
19906	19906	2314	2	0
19907	19907	2314	0.7	0
19908	19908	2314	0.8	0
19909	19909	2314	0.9	0
19910	19910	2314	1.1	0
19911	19911	2314	3	0
19912	19912	2314	0	0
19913	19913	2314	1.2	0
19914	19914	2314	0.5	0
19915	19915	2314	0.5	0
19916	19916	2314	0.7	0
19917	19917	2314	.	0
19918	19918	2314	.	0
19919	19919	2314	0.8	0
19920	19920	2314	0.9	0
19921	19921	2314	0	0
19922	19922	2314	15	0
19923	19923	2314	4	0
19924	19924	2314	1	0
19925	19925	2314	7	0
19926	19926	2314	2	0
19927	19927	2314	3	0
19928	19928	2314	2	0
19929	19929	2314	1	0
19930	19930	2314	0	0
19931	19931	2314	0	0
19932	19932	2314	0	0
19933	19933	2314	0	0
19934	19934	2314	0	0
19935	19935	2314	0	0
19936	19936	2314	0	0
19937	19937	2314	B0L6	0
19938	19938	2314	1	0
19939	19939	2314	1.25	0
19940	19940	2314	24	0
19941	19941	2314	13	0
19942	19942	2314	41.1	0
19943	19943	2314	38	0
19944	19944	2314	25.3	0
19945	19945	2314	198	0
19946	19946	2314	2.8	0
19947	19947	2314	2.1	0
19948	19948	2314	0	0
19949	19949	2314	0	0
19950	19950	2314	0	0
19951	19951	2314	85	0
19952	19952	2314	150	0
19953	19953	2314	150	0
19954	19954	2314	10	0
19955	19955	2314	25	0
19956	19956	2314	50	0
19957	19957	2314	1	0
19958	19958	2314	1.29	0
19959	19959	2314	12.86666667	0
19960	19960	2314	3.983333333	0
19961	19961	2314	20	0
19962	19962	2314	175	0
19963	19963	2314	2.27	0
19964	19964	2314	88	0
19965	19965	2314	52	0
19966	19966	2314	3.34	0
19967	19967	2314	0	0
19968	19968	2314	0	0
19969	19969	2314	5	0
19970	19970	2314	10	0
19971	19971	2314	0	0
19972	19972	2314	81	0
19973	19973	2314	6	0
19974	19974	2314	1	0
19975	19975	2314	20	0
19976	19976	2314	175	0
19977	19977	2314	2.27	0
19978	19978	2314	88	0
19979	19979	2314	52	0
19980	19980	2314	3.34	0
19981	19981	2314	0	0
19982	19982	2314	0	0
19983	19983	2314	5	0
19984	19984	2314	10	0
19985	19985	2314	0	0
19986	19986	2314	58.5	0
19987	19987	2314	23.3	0
19988	19988	2314	122	0
19989	19989	2314	51.7	0
19990	19990	2314	7.1	0
19991	19991	2314	6.3	0
19992	19992	2314	9.25	0
19993	19993	2314	1.36	0
19994	19994	2314	9.2733	0
19995	19995	2314	1.418	0
19996	19996	2314	6	0
19997	19997	2314		0
19998	19998	2314	2	0
19999	19999	2314	3	0
20000	20000	2314	0.3	0
20001	20001	2314	0.5	0
20002	20002	2314	0.8	0
20003	20003	2314	0.9	0
20004	20004	2314	3	0
20005	20005	2314	0	0
20006	20006	2314	1.2	0
20007	20007	2314	0.5	0
20008	20008	2314	0.2	0
20009	20009	2314	0.3	0
20010	20010	2314	0.3	0
20011	20011	2314	0.4	0
20012	20012	2314	0.5	0
20013	20013	2314	0.6	0
20014	20014	2314	1	0
20015	20015	2314	19	0
20016	20016	2314	5	0
20017	20017	2314	1	0
20018	20018	2314	8	0
20019	20019	2314	3	0
20020	20020	2314	2	0
20021	20021	2314	3	0
20022	20022	2314	2	0
20023	20023	2314	1	0
20024	20024	2314	0	0
20025	20025	2314	0	0
20026	20026	2314	0	0
20027	20027	2314	0	0
20028	20028	2314	0	0
20029	20029	2314	0	0
20030	20030	2314	B0L6	0
20031	20031	2314	1.2	0
20032	20032	2314	1.3	0
20033	20033	2314	25.5	0
20034	20034	2314	14	0
20035	20035	2314	29.5	0
20036	20036	2314	16	0
20037	20037	2314	3.4	0
20038	20038	2314	93	0
20039	20039	2314	2.6	0
20040	20040	2314	1.9	0
20041	20041	2314	0	0
20042	20042	2314	0	0
20043	20043	2314	0	0
20044	20044	2314	61	0
20045	20045	2314	100	0
20046	20046	2314	100	0
20047	20047	2314	15	0
20048	20048	2314	0	0
20049	20049	2314	90	0
20050	20050	2314	0	0
20051	20051	2314	1.681666667	0
20052	20052	2314	15.6	0
20053	20053	2314	4.7	0
20054	20054	2314	30	0
20055	20055	2314	150	0
20056	20056	2314		0
20057	20057	2314	65	0
20058	20058	2314		0
20059	20059	2314		0
20060	20060	2314	0	0
20061	20061	2314	2	0
20062	20062	2314	20	0
20063	20063	2314	0	0
20064	20064	2314	90	0
20065	20065	2314	0	0
20066	20066	2314	2	0
20067	20067	2314	12	0
20068	20068	2314	30	0
20069	20069	2314	150	0
20070	20070	2314	.	0
20071	20071	2314	65	0
20072	20072	2314	.	0
20073	20073	2314	.	0
20074	20074	2314	0	0
20075	20075	2314	2	0
20076	20076	2314	20	0
20077	20077	2314	0	0
20078	20078	2314	90	0
20079	20079	2314	78.5	0
20080	20080	2314	29	0
20081	20081	2314	188	0
20082	20082	2314	77.5	0
20083	20083	2314	7.7	0
20084	20084	2314	6.8	0
20085	20085	2314	11.87	0
20086	20086	2314	1.61	0
20087	20087	2314	11.899	0
20088	20088	2314	1.6943	0
20089	20089	2314	130	0
20090	20090	2314	1	0
20091	20091	2314	1	0
20092	20092	2314	2	0
20093	20093	2314	0.8	0
20094	20094	2314	0.9	0
20095	20095	2314		0
20096	20096	2314		0
20097	20097	2314	2	0
20098	20098	2314	0	0
20099	20099	2314	1.2	0
20100	20100	2314	0.5	0
20101	20101	2314	0.9	0
20102	20102	2314	1	0
20103	20103	2314	0.5	0
20104	20104	2314	0.7	0
20105	20105	2314		0
20106	20106	2314		0
20107	20107	2314	2	0
20108	20108	2314	13	0
20109	20109	2314	3	0
20110	20110	2314	1	0
20111	20111	2314	7	0
20112	20112	2314	3	0
20113	20113	2314	2	0
20114	20114	2314	1	0
20115	20115	2314	0	0
20116	20116	2314	0	0
20117	20117	2314	0	0
20118	20118	2314	0	0
20119	20119	2314	0	0
20120	20120	2314	0	0
20121	20121	2314	0	0
20122	20122	2314	0	0
20123	20123	2314	B0L6	0
20124	20124	2314	1.1	0
20125	20125	2314	1.3	0
20126	20126	2314	22.2	0
20127	20127	2314	13	0
20128	20128	2314	43.4	0
20129	20129	2314	47	0
20130	20130	2314	22.2	0
20131	20131	2314	216	0
20132	20132	2314	2.7	0
20133	20133	2314	2	0
20134	20134	2314	0	0
20135	20135	2314	0	0
20136	20136	2314	0	0
20137	20137	2314	75.5	0
20138	20138	2314	150	0
20139	20139	2314	150	0
20140	20140	2314	5	0
20141	20141	2314	30	0
20142	20142	2314	70	0
20143	20143	2314	2	0
20144	20144	2314	1.811666667	0
20145	20145	2314	19.78333333	0
20146	20146	2314	5.533333333	0
20147	20147	2314	0	0
20148	20148	2314	150	0
20149	20149	2314		0
20150	20150	2314	73	0
20151	20151	2314		0
20152	20152	2314		0
20153	20153	2314		0
20154	20154	2314		0
20155	20155	2314	0	0
20156	20156	2314	0	0
20157	20157	2314	0	0
20158	20158	2314	88	0
20159	20159	2314	0	0
20160	20160	2314	0	0
20161	20161	2314	0	0
20162	20162	2314	150	0
20163	20163	2314	.	0
20164	20164	2314	73	0
20165	20165	2314	.	0
20166	20166	2314	.	0
20167	20167	2314	.	0
20168	20168	2314	.	0
20169	20169	2314	0	0
20170	20170	2314	0	0
20171	20171	2314	0	0
20172	20172	2314	.	0
20173	20173	2314	.	0
20174	20174	2314	.	0
20175	20175	2314	.	0
20176	20176	2314	.	0
20177	20177	2314	.	0
20178	20178	2314	10.63	0
20179	20179	2314	0.25	0
20180	20180	2314	10.63	0
20181	20181	2314	0.25	0
20182	20182	2314	6	0
20183	20183	2314	13	0
20184	20184	2314	1	0
20185	20185	2314	2	0
20186	20186	2314	0.5	0
20187	20187	2314	0.7	0
20188	20188	2314		0
20189	20189	2314		0
20190	20190	2314	3	0
20191	20191	2314	0	0
20192	20192	2314	0.6	0
20193	20193	2314	0.3	0
20194	20194	2314	0.5	0
20195	20195	2314	0.7	0
20196	20196	2314	0.5	0
20197	20197	2314	0.5	0
20198	20198	2314		0
20199	20199	2314		0
20200	20200	2314	2	0
20201	20201	2314	12	0
20202	20202	2314	3	0
20203	20203	2314	1	0
20204	20204	2314	6	0
20205	20205	2314	2	0
20206	20206	2314	3	0
20207	20207	2314	1	0
20208	20208	2314	0	0
20209	20209	2314	0	0
20210	20210	2314	0	0
20211	20211	2314	0	0
20212	20212	2314	0	0
20213	20213	2314	0	0
20214	20214	2314	0	0
20215	20215	2314	0	0
20216	20216	2314	B0L5	0
20217	20217	2314	1	0
20218	20218	2314	1.2	0
20219	20219	2314	19.8	0
20220	20220	2314	13	0
20221	20221	2314	30.8	0
20222	20222	2314	22	0
20223	20223	2314	85.4	0
20224	20224	2314	98	0
20225	20225	2314	2.3	0
20226	20226	2314	2.1	0
20227	20227	2314	17	0
20228	20228	2314	2	0
20229	20229	2314	7	0
20230	20230	2314	79	0
20231	20231	2314	100	0
20232	20232	2314	100	0
20233	20233	2314	10	0
20234	20234	2314	20	0
20235	20235	2314	85	0
20236	20236	2314	0	0
20237	20237	2314	1.318333333	0
20238	20238	2314	14.41666667	0
20239	20239	2314	3.483333333	0
20240	20240	2314	100	0
20241	20241	2314	225	0
20242	20242	2314	2.75	0
20243	20243	2314	87	0
20244	20244	2314	43.5	0
20245	20245	2314	1.88	0
20246	20246	2314	0	0
20247	20247	2314	0	0
20248	20248	2314	95	0
20249	20249	2314	10	0
20250	20250	2314	0	0
20251	20251	2314	81	0
20252	20252	2314	59	0
20253	20253	2314	42	0
20254	20254	2314	100	0
20255	20255	2314	225	0
20256	20256	2314	2.75	0
20257	20257	2314	87	0
20258	20258	2314	43.5	0
20259	20259	2314	1.88	0
20260	20260	2314	0	0
20261	20261	2314	0	0
20262	20262	2314	95	0
20263	20263	2314	10	0
20264	20264	2314	0	0
20265	20265	2314	86	0
20266	20266	2314	30.6	0
20267	20267	2314	15.3	0
20268	20268	2314	12	0
20269	20269	2314	4.4	0
20270	20270	2314	2.4	0
20271	20271	2314	8.88	0
20272	20272	2314	2.29	0
20273	20273	2314	8.9106	0
20274	20274	2314	2.3044	0
20275	20275	2314	25	0
20276	20276	2314	1	0
20277	20277	2314	2	0
20278	20278	2314	2	0
20279	20279	2314	0.4	0
20280	20280	2314	0.5	0
20281	20281	2314	1	0
20282	20282	2314	1.2	0
20283	20283	2314	4	0
20284	20284	2314	0	0
20285	20285	2314	1	0
20286	20286	2314	0.4	0
20287	20287	2314	0.4	0
20288	20288	2314	0.5	0
20289	20289	2314	0.6	0
20290	20290	2314	0.7	0
20291	20291	2314	0.5	0
20292	20292	2314	0.7	0
20293	20293	2314	1	0
20294	20294	2314	16	0
20295	20295	2314	4	0
20296	20296	2314	1	0
20297	20297	2314	7	0
20298	20298	2314	4	0
20299	20299	2314	3	0
20300	20300	2314	1	0
20301	20301	2314	1	0
20302	20302	2314	0	0
20303	20303	2314	0	0
20304	20304	2314	0	0
20305	20305	2314	0	0
20306	20306	2314	0	0
20307	20307	2314	0	0
20308	20308	2314	0	0
20309	20309	2314	B0L6	0
20310	20310	2314	1	0
20311	20311	2314	1.1	0
20312	20312	2314	12.1	0
20313	20313	2314	7	0
20314	20314	2314	26.6	0
20315	20315	2314	27	0
20316	20316	2314	7.6	0
20317	20317	2314	129	0
20318	20318	2314	2	0
20319	20319	2314	1.8	0
20320	20320	2314	0	0
20321	20321	2314	0	0
20322	20322	2314	0	0
20323	20323	2314	65	0
20324	20324	2314	100	0
20325	20325	2314	100	0
20326	20326	2314	10	0
20327	20327	2314	15	0
20328	20328	2314	80	0
20329	20329	2314	0	0
20330	20330	2314	2.55	0
20331	20331	2314	33.8	0
20332	20332	2314	7.083333333	0
20333	20333	2314	100	0
20334	20334	2314	150	0
20335	20335	2314	2.09	0
20336	20336	2314	76	0
20337	20337	2314	36	0
20338	20338	2314	3.51	0
20339	20339	2314	0	0
20340	20340	2314	0	0
20341	20341	2314	70	0
20342	20342	2314	0	0
20343	20343	2314	0	0
20344	20344	2314	49	0
20345	20345	2314	34	0
20346	20346	2314	47	0
20347	20347	2314	100	0
20348	20348	2314	150	0
20349	20349	2314	2.09	0
20350	20350	2314	76	0
20351	20351	2314	36	0
20352	20352	2314	3.51	0
20353	20353	2314	0	0
20354	20354	2314	0	0
20355	20355	2314	70	0
20356	20356	2314	0	0
20357	20357	2314	0	0
20358	20358	2314	60.2	0
20359	20359	2314	17.7	0
20360	20360	2314	86.3	0
20361	20361	2314	43.2	0
20362	20362	2314	7.2	0
20363	20363	2314	6.7	0
20364	20364	2314	7.94	0
20365	20365	2314	2.2	0
20366	20366	2314	7.9577	0
20367	20367	2314	2.2499	0
20368	20368	2314	28	0
20369	20369	2314	2	0
20370	20370	2314	2	0
20371	20371	2314	3	0
20372	20372	2314	0.6	0
20373	20373	2314	0.8	0
20374	20374	2314	1	0
20375	20375	2314	1.2	0
20376	20376	2314	3	0
20377	20377	2314	0	0
20378	20378	2314	0.9	0
20379	20379	2314	0.4	0
20380	20380	2314	0.5	0
20381	20381	2314	0.8	0
20382	20382	2314	0.6	0
20383	20383	2314	0.8	0
20384	20384	2314	0.6	0
20385	20385	2314	0.9	0
20386	20386	2314	1	0
20387	20387	2314	19	0
20388	20388	2314	4	0
20389	20389	2314	1	0
20390	20390	2314	8	0
20391	20391	2314	4	0
20392	20392	2314	4	0
20393	20393	2314	2	0
20394	20394	2314	1	0
20395	20395	2314	0	0
20396	20396	2314	0	0
20397	20397	2314	0	0
20398	20398	2314	0	0
20399	20399	2314	0	0
20400	20400	2314	0	0
20401	20401	2314	0	0
20402	20402	2314	B0L6	0
20403	20403	2314	1	0
20404	20404	2314	1.2	0
20405	20405	2314	27.2	0
20406	20406	2314	12	0
20407	20407	2314	38.7	0
20408	20408	2314	25	0
20409	20409	2314	6.9	0
20410	20410	2314	117	0
20411	20411	2314	2.4	0
20412	20412	2314	1.7	0
20413	20413	2314	24	0
20414	20414	2314	9	0
20415	20415	2314	8	0
20416	20416	2314	88	0
20417	20417	2314	150	0
20418	20418	2314	150	0
20419	20419	2314	5	0
20420	20420	2314	15	0
20421	20421	2314	99	0
20422	20422	2314	0	0
20423	20423	2314	2.593333333	0
20424	20424	2314	38.28333333	0
20425	20425	2314	7.883333333	0
20426	20426	2314	200	0
20427	20427	2314		0
20428	20428	2314	2.58	0
20429	20429	2314	89	0
20430	20430	2314	41	0
20431	20431	2314	3.96	0
20432	20432	2314	0	0
20433	20433	2314	0	0
20434	20434	2314	90	0
20435	20435	2314	5	0
20436	20436	2314	0	0
20437	20437	2314	30	0
20438	20438	2314	33	0
20439	20439	2314	42	0
20440	20440	2314	200	0
20441	20441	2314		0
20442	20442	2314	2.58	0
20443	20443	2314	89	0
20444	20444	2314	41	0
20445	20445	2314	3.96	0
20446	20446	2314	0	0
20447	20447	2314	0	0
20448	20448	2314	90	0
20449	20449	2314	5	0
20450	20450	2314	0	0
20451	20451	2314	76.5	0
20452	20452	2314	27.9	0
20453	20453	2314	102.9	0
20454	20454	2314	48.1	0
20455	20455	2314	7.2	0
20456	20456	2314	7.2	0
20457	20457	2314	10.48	0
20458	20458	2314	2.13	0
20459	20459	2314	10.5079	0
20460	20460	2314	2.1853	0
20461	20461	2314	15	0
20462	20462	2314	33	0
20463	20463	2314	0	0
20464	20464	2314	1	0
20465	20465	2314	0.5	0
20466	20466	2314	0.6	0
20467	20467	2314		0
20468	20468	2314		0
20469	20469	2314	2	0
20470	20470	2314	0	0
20471	20471	2314	0.8	0
20472	20472	2314	0.4	0
20473	20473	2314	0.6	0
20474	20474	2314	0.8	0
20475	20475	2314		0
20476	20476	2314		0
20477	20477	2314		0
20478	20478	2314		0
20479	20479	2314	2	0
20480	20480	2314	8	0
20481	20481	2314	2	0
20482	20482	2314	1	0
20483	20483	2314	5	0
20484	20484	2314	2	0
20485	20485	2314	1	0
20486	20486	2314	0	0
20487	20487	2314	0	0
20488	20488	2314	0	0
20489	20489	2314	0	0
20490	20490	2314	0	0
20491	20491	2314	0	0
20492	20492	2314	0	0
20493	20493	2314	0	0
20494	20494	2314	0	0
20495	20495	2314	B0L4	0
20496	20496	2314	1	0
20497	20497	2314	1.3	0
20498	20498	2314	27.8	0
20499	20499	2314	12	0
20500	20500	2314	35	0
20501	20501	2314	12	0
20502	20502	2314	5.4	0
20503	20503	2314	84	0
20504	20504	2314	3	0
20505	20505	2314	2.4	0
20506	20506	2314	0	0
20507	20507	2314	0	0
20508	20508	2314	0	0
20509	20509	2314	62.5	0
20510	20510	2314	175	0
20511	20511	2314	175	0
20512	20512	2314	20	0
20513	20513	2314	25	0
20514	20514	2314	75	0
20515	20515	2314	0	0
20516	20516	2314	1.658333333	0
20517	20517	2314	20.86666667	0
20518	20518	2314	4.466666667	0
20519	20519	2314	100	0
20520	20520	2314	300	0
20521	20521	2314	2.35	0
20522	20522	2314	55	0
20523	20523	2314	48	0
20524	20524	2314	1.47	0
20525	20525	2314	0	0
20526	20526	2314	0	0
20527	20527	2314	30	0
20528	20528	2314	0	0
20529	20529	2314	0	0
20530	20530	2314	60	0
20531	20531	2314	30	0
20532	20532	2314	7	0
20533	20533	2314	100	0
20534	20534	2314	300	0
20535	20535	2314	2.35	0
20536	20536	2314	55	0
20537	20537	2314	48	0
20538	20538	2314	1.47	0
20539	20539	2314	0	0
20540	20540	2314	0	0
20541	20541	2314	30	0
20542	20542	2314	0	0
20543	20543	2314	0	0
20544	20544	2314	.	0
20545	20545	2314	.	0
20546	20546	2314	150.9	0
20547	20547	2314	41.6	0
20548	20548	2314	40.3	0
20549	20549	2314	10.7	0
20550	20550	2314	14.34	0
20551	20551	2314	2.32	0
20552	20552	2314	14.34	0
20553	20553	2314	2.3723	0
20554	20554	2314	2	0
20555	20555	2314	8	0
20556	20556	2314	1	0
20557	20557	2314	2	0
20558	20558	2314	0.5	0
20559	20559	2314	0.8	0
20560	20560	2314		0
20561	20561	2314		0
20562	20562	2314	2	0
20563	20563	2314	0	0
20564	20564	2314	1	0
20565	20565	2314	0.4	0
20566	20566	2314	0.6	0
20567	20567	2314	0.9	0
20568	20568	2314	0.5	0
20569	20569	2314	0.7	0
20570	20570	2314		0
20571	20571	2314		0
20572	20572	2314	2	0
20573	20573	2314	13	0
20574	20574	2314	3	0
20575	20575	2314	1	0
20576	20576	2314	6	0
20577	20577	2314	3	0
20578	20578	2314	3	0
20579	20579	2314	1	0
20580	20580	2314	0	0
20581	20581	2314	0	0
20582	20582	2314	0	0
20583	20583	2314	0	0
20584	20584	2314	0	0
20585	20585	2314	0	0
20586	20586	2314	0	0
20587	20587	2314	0	0
20588	20588	2314	B0L5	0
20589	20589	2314	1.1	0
20590	20590	2314	1.35	0
20591	20591	2314	19.5	0
20592	20592	2314	7	0
20593	20593	2314	39.9	0
20594	20594	2314	29	0
20595	20595	2314	19.7	0
20596	20596	2314	124	0
20597	20597	2314	2.3	0
20598	20598	2314	2.2	0
20599	20599	2314	0	0
20600	20600	2314	0	0
20601	20601	2314	0	0
20602	20602	2314	62	0
20603	20603	2314	250	0
20604	20604	2314	250	0
20605	20605	2314	5	0
20606	20606	2314	20	0
20607	20607	2314	90	0
20608	20608	2314	0	0
20609	20609	2314	1.681666667	0
20610	20610	2314	24.56666667	0
20611	20611	2314	5.966666667	0
20612	20612	2314	600	0
20613	20613	2314	650	0
20614	20614	2314	2.31	0
20615	20615	2314	60.5	0
20616	20616	2314		0
20617	20617	2314	4.15	0
20618	20618	2314	15	0
20619	20619	2314	0	0
20620	20620	2314	98	0
20621	20621	2314	0	0
20622	20622	2314	80	0
20623	20623	2314	16	0
20624	20624	2314	11	0
20625	20625	2314	42	0
20626	20626	2314	600	0
20627	20627	2314	650	0
20628	20628	2314	2.31	0
20629	20629	2314	60.5	0
20630	20630	2314	.	0
20631	20631	2314	4.15	0
20632	20632	2314	15	0
20633	20633	2314	0	0
20634	20634	2314	98	0
20635	20635	2314	0	0
20636	20636	2314	80	0
20637	20637	2314	106	0
20638	20638	2314	26	0
20639	20639	2314	129.9	0
20640	20640	2314	46	0
20641	20641	2314	25.9	0
20642	20642	2314	9.7	0
20643	20643	2314	20.52	0
20644	20644	2314	3.56	0
20645	20645	2314	20.546	0
20646	20646	2314	3.6157	0
20647	20647	2314	2	0
20648	20648	2314	46	0
20649	20649	2314	1	0
20650	20650	2314	3	0
20651	20651	2314	0.6	0
20652	20652	2314	0.9	0
20653	20653	2314	1.1	0
20654	20654	2314	1.1	0
20655	20655	2314	3	0
20656	20656	2314	0	0
20657	20657	2314	1.3	0
20658	20658	2314	0.5	0
20659	20659	2314	0.8	0
20660	20660	2314	1.1	0
20661	20661	2314	0.6	0
20662	20662	2314	0.9	0
20663	20663	2314	0.7	0
20664	20664	2314	1.1	0
20665	20665	2314	2	0
20666	20666	2314	24	0
20667	20667	2314	5	0
20668	20668	2314	1	0
20669	20669	2314	9	0
20670	20670	2314	5	0
20671	20671	2314	4	0
20672	20672	2314	3	0
20673	20673	2314	2	0
20674	20674	2314	1	0
20675	20675	2314	0	0
20676	20676	2314	0	0
20677	20677	2314	0	0
20678	20678	2314	0	0
20679	20679	2314	0	0
20680	20680	2314	0	0
20681	20681	2314	B0L7	0
20682	20682	2314	1.1	0
20683	20683	2314	1.2	0
20684	20684	2314	11.5	0
20685	20685	2314	11	0
20686	20686	2314	31.9	0
20687	20687	2314	32	0
20688	20688	2314	10.3	0
20689	20689	2314	192	0
20690	20690	2314	2.1	0
20691	20691	2314	2.5	0
20692	20692	2314	0	0
20693	20693	2314	2	0
20694	20694	2314	0	0
20695	20695	2314	57	0
20696	20696	2314	175	0
20697	20697	2314	175	0
20698	20698	2314	15	0
20699	20699	2314	35	0
20700	20700	2314	80	0
20701	20701	2314	2	0
20702	20702	2314	1.738333333	0
20703	20703	2314	22.45	0
20704	20704	2314	5.266666667	0
20705	20705	2314	150	0
20706	20706	2314		0
20707	20707	2314	2.54	0
20708	20708	2314	84.5	0
20709	20709	2314	34	0
20710	20710	2314	3.83	0
20711	20711	2314	0	0
20712	20712	2314	0	0
20713	20713	2314	80	0
20714	20714	2314	0	0
20715	20715	2314	0	0
20716	20716	2314	159	0
20717	20717	2314	116	0
20718	20718	2314	35	0
20719	20719	2314	150	0
20720	20720	2314		0
20721	20721	2314	2.54	0
20722	20722	2314	84.5	0
20723	20723	2314	34	0
20724	20724	2314	3.83	0
20725	20725	2314	0	0
20726	20726	2314	0	0
20727	20727	2314	80	0
20728	20728	2314	0	0
20729	20729	2314	0	0
20730	20730	2314	59.7	0
20731	20731	2314	16.7	0
20732	20732	2314	170.2	0
20733	20733	2314	51.4	0
20734	20734	2314	17	0
20735	20735	2314	6	0
20736	20736	2314	11.95	0
20737	20737	2314	2.88	0
20738	20738	2314	11.9667	0
20739	20739	2314	2.9374	0
20740	20740	2314	3	0
20741	20741	2314	17	0
20742	20742	2314	0	0
20743	20743	2314	1	0
20744	20744	2314		0
20745	20745	2314		0
20746	20746	2314		0
20747	20747	2314		0
20748	20748	2314	2	0
20749	20749	2314	0	0
20750	20750	2314	0.6	0
20751	20751	2314	0.4	0
20752	20752	2314	0.7	0
20753	20753	2314	1.2	0
20754	20754	2314		0
20755	20755	2314		0
20756	20756	2314		0
20757	20757	2314		0
20758	20758	2314	1	0
20759	20759	2314	8	0
20760	20760	2314	2	0
20761	20761	2314	1	0
20762	20762	2314	5	0
20763	20763	2314	2	0
20764	20764	2314	1	0
20765	20765	2314	0	0
20766	20766	2314	0	0
20767	20767	2314	0	0
20768	20768	2314	0	0
20769	20769	2314	0	0
20770	20770	2314	0	0
20771	20771	2314	0	0
20772	20772	2314	0	0
20773	20773	2314	0	0
20774	20774	2314	B0L4	0
20775	20775	2314	1.2	0
20776	20776	2314	1.5	0
20777	20777	2314	31.2	0
20778	20778	2314	12	0
20779	20779	2314	37.5	0
20780	20780	2314	21	0
20781	20781	2314	7.3	0
20782	20782	2314	94	0
20783	20783	2314	2.8	0
20784	20784	2314	2.5	0
20785	20785	2314	0	0
20786	20786	2314	0	0
20787	20787	2314	0	0
20788	20788	2314	59.5	0
20789	20789	2314	150	0
20790	20790	2314	150	0
20791	20791	2314	15	0
20792	20792	2314	25	0
20793	20793	2314	70	0
20794	20794	2314	0	0
20795	20795	2314	1.493333333	0
20796	20796	2314	17.61666667	0
20797	20797	2314	3.966666667	0
20798	20798	2314	75	0
20799	20799	2314	250	0
20800	20800	2314	2.63	0
20801	20801	2314	81	0
20802	20802	2314	29	0
20803	20803	2314	1.95	0
20804	20804	2314	6	0
20805	20805	2314	4	0
20806	20806	2314	70	0
20807	20807	2314	0	0
20808	20808	2314	0	0
20809	20809	2314	72	0
20810	20810	2314	22	0
20811	20811	2314	5	0
20812	20812	2314	75	0
20813	20813	2314	250	0
20814	20814	2314	2.63	0
20815	20815	2314	81	0
20816	20816	2314	29	0
20817	20817	2314	1.95	0
20818	20818	2314	6	0
20819	20819	2314	4	0
20820	20820	2314	70	0
20821	20821	2314	0	0
20822	20822	2314	0	0
20823	20823	2314	82.8	0
20824	20824	2314	25.6	0
20825	20825	2314	74.3	0
20826	20826	2314	21.7	0
20827	20827	2314	19.2	0
20828	20828	2314	7	0
20829	20829	2314	9.02	0
20830	20830	2314	1.79	0
20831	20831	2314	9.0456	0
20832	20832	2314	1.8187	0
20833	20833	2314	8	0
20834	20834	2314	79	0
20835	20835	2314	1	0
20836	20836	2314	2	0
20837	20837	2314	0.6	0
20838	20838	2314	0.8	0
20839	20839	2314		0
20840	20840	2314		0
20841	20841	2314	3	0
20842	20842	2314	0	0
20843	20843	2314	1	0
20844	20844	2314	0.5	0
20845	20845	2314	0.6	0
20846	20846	2314	1	0
20847	20847	2314	0.6	0
20848	20848	2314	0.8	0
20849	20849	2314		0
20850	20850	2314		0
20851	20851	2314	2	0
20852	20852	2314	16	0
20853	20853	2314	4	0
20854	20854	2314	1	0
20855	20855	2314	7	0
20856	20856	2314	3	0
20857	20857	2314	3	0
20858	20858	2314	2	0
20859	20859	2314	1	0
20860	20860	2314	0	0
20861	20861	2314	0	0
20862	20862	2314	0	0
20863	20863	2314	0	0
20864	20864	2314	0	0
20865	20865	2314	0	0
20866	20866	2314	0	0
20867	20867	2314	B0L5	0
20868	20868	2314	1	0
20869	20869	2314	1.1	0
20870	20870	2314	21.2	0
20871	20871	2314	10	0
20872	20872	2314	44.2	0
20873	20873	2314	41	0
20874	20874	2314	8.4	0
20875	20875	2314	189	0
20876	20876	2314	2.7	0
20877	20877	2314	2.1	0
20878	20878	2314	0	0
20879	20879	2314	8	0
20880	20880	2314	0	0
20881	20881	2314	71	0
20882	20882	2314	225	0
20883	20883	2314	225	0
20884	20884	2314	15	0
20885	20885	2314	45	0
20886	20886	2314	85	0
20887	20887	2314	0	0
20888	20888	2314	2.485	0
20889	20889	2314	34.88333333	0
20890	20890	2314	7.8	0
20891	20891	2314	200	0
20892	20892	2314	250	0
20893	20893	2314	1.9	0
20894	20894	2314	89	0
20895	20895	2314	34	0
20896	20896	2314	2.06	0
20897	20897	2314	0	0
20898	20898	2314	0	0
20899	20899	2314	80	0
20900	20900	2314	0	0
20901	20901	2314	0	0
20902	20902	2314	111	0
20903	20903	2314	74	0
20904	20904	2314	3	0
20905	20905	2314	200	0
20906	20906	2314	250	0
20907	20907	2314	1.9	0
20908	20908	2314	89	0
20909	20909	2314	34	0
20910	20910	2314	2.06	0
20911	20911	2314	0	0
20912	20912	2314	0	0
20913	20913	2314	80	0
20914	20914	2314	0	0
20915	20915	2314	0	0
20916	20916	2314	52.4	0
20917	20917	2314	16.3	0
20918	20918	2314	72.1	0
20919	20919	2314	23.6	0
20920	20920	2314	7.1	0
20921	20921	2314	3.7	0
20922	20922	2314	9.17	0
20923	20923	2314	1.99	0
20924	20924	2314	9.1863	0
20925	20925	2314	2.0173	0
20926	20926	2314	4	0
20927	20927	2314	21	0
20928	20928	2314	1	0
20929	20929	2314	2	0
20930	20930	2314	0.4	0
20931	20931	2314	0.4	0
20932	20932	2314	0.7	0
20933	20933	2314	0.8	0
20934	20934	2314	3	0
20935	20935	2314	0	0
20936	20936	2314	1	0
20937	20937	2314	0.5	0
20938	20938	2314	.	0
20939	20939	2314	.	0
20940	20940	2314	0.5	0
20941	20941	2314	0.6	0
20942	20942	2314		0
20943	20943	2314		0
20944	20944	2314	1	0
20945	20945	2314	13	0
20946	20946	2314	4	0
20947	20947	2314	1	0
20948	20948	2314	6	0
20949	20949	2314	2	0
20950	20950	2314	3	0
20951	20951	2314	1	0
20952	20952	2314	1	0
20953	20953	2314	0	0
20954	20954	2314	0	0
20955	20955	2314	0	0
20956	20956	2314	0	0
20957	20957	2314	0	0
20958	20958	2314	0	0
20959	20959	2314	0	0
20960	20960	2314	B0L5	0
20961	20961	2314	0.95	0
20962	20962	2314	1.3	0
20963	20963	2314	28	0
20964	20964	2314	13	0
20965	20965	2314	39.6	0
20966	20966	2314	28	0
20967	20967	2314	11.2	0
20968	20968	2314	157	0
20969	20969	2314	2.6	0
20970	20970	2314	2.4	0
20971	20971	2314	2	0
20972	20972	2314	2	0
20973	20973	2314	0	0
20974	20974	2314	88.5	0
20975	20975	2314	75	0
20976	20976	2314	75	0
20977	20977	2314	5	0
20978	20978	2314	25	0
20979	20979	2314	75	0
20980	20980	2314	1	0
20981	20981	2314	1.786666667	0
20982	20982	2314	22.18333333	0
20983	20983	2314	5.183333333	0
20984	20984	2314	250	0
20985	20985	2314		0
20986	20986	2314	1.79	0
20987	20987	2314	86	0
20988	20988	2314	34	0
20989	20989	2314	2.33	0
20990	20990	2314	0	0
20991	20991	2314	0	0
20992	20992	2314	80	0
20993	20993	2314	15	0
20994	20994	2314	0	0
20995	20995	2314	84	0
20996	20996	2314	30	0
20997	20997	2314	23	0
20998	20998	2314	250	0
20999	20999	2314		0
21000	21000	2314	1.79	0
21001	21001	2314	86	0
21002	21002	2314	34	0
21003	21003	2314	2.33	0
21004	21004	2314	0	0
21005	21005	2314	0	0
21006	21006	2314	80	0
21007	21007	2314	15	0
21008	21008	2314	0	0
21009	21009	2314	65.1	0
21010	21010	2314	20.9	0
21011	21011	2314	180.3	0
21012	21012	2314	55.7	0
21013	21013	2314	7.4	0
21014	21014	2314	2.3	0
21015	21015	2314	7.31	0
21016	21016	2314	1.55	0
21017	21017	2314	7.3309	0
21018	21018	2314	1.608	0
21019	21019	2314	11	0
21020	21020	2314	13	0
21021	21021	2314	1	0
21022	21022	2314	1	0
21023	21023	2314	0.1	0
21024	21024	2314	0.1	0
21025	21025	2314		0
21026	21026	2314		0
21027	21027	2314	2	0
21028	21028	2314	0	0
21029	21029	2314	.	0
21030	21030	2314	.	0
21031	21031	2314	0.2	0
21032	21032	2314	0.2	0
21033	21033	2314		0
21034	21034	2314		0
21035	21035	2314		0
21036	21036	2314		0
21037	21037	2314	0	0
21038	21038	2314	4	0
21039	21039	2314	1	0
21040	21040	2314	2	0
21041	21041	2314	4	0
21042	21042	2314	0	0
21043	21043	2314	0	0
21044	21044	2314	0	0
21045	21045	2314	0	0
21046	21046	2314	0	0
21047	21047	2314	0	0
21048	21048	2314	0	0
21049	21049	2314	0	0
21050	21050	2314	0	0
21051	21051	2314	0	0
21052	21052	2314	0	0
21053	21053	2314	B0L3	0
21054	21054	2314	0.3	0
21055	21055	2314	0.4	0
21056	21056	2314	21	0
21057	21057	2314	6	0
21058	21058	2314	2.5	0
21059	21059	2314	0	0
21060	21060	2314	.	0
21061	21061	2314	17	0
21062	21062	2314	1.7	0
21063	21063	2314	1.7	0
21064	21064	2314	0	0
21065	21065	2314	0	0
21066	21066	2314	0	0
21067	21067	2314	65	0
21068	21068	2314	75	0
21069	21069	2314	75	0
21070	21070	2314	0	0
21071	21071	2314	20	0
21072	21072	2314	60	0
21073	21073	2314	0	0
21074	21074	2314	1.733333333	0
21075	21075	2314	18.46666667	0
21076	21076	2314	4.683333333	0
21077	21077	2314	70	0
21078	21078	2314		0
21079	21079	2314	3.19	0
21080	21080	2314	83	0
21081	21081	2314	46	0
21082	21082	2314	3.01	0
21083	21083	2314	0	0
21084	21084	2314	0	0
21085	21085	2314	80	0
21086	21086	2314	0	0
21087	21087	2314	0	0
21088	21088	2314	31	0
21089	21089	2314	4	0
21090	21090	2314	15	0
21091	21091	2314	70	0
21092	21092	2314		0
21093	21093	2314	3.19	0
21094	21094	2314	83	0
21095	21095	2314	46	0
21096	21096	2314	3.01	0
21097	21097	2314	0	0
21098	21098	2314	0	0
21099	21099	2314	80	0
21100	21100	2314	0	0
21101	21101	2314	0	0
21102	21102	2314	145.6	0
21103	21103	2314	42.2	0
21104	21104	2314	167.3	0
21105	21105	2314	42.1	0
21106	21106	2314	9.7	0
21107	21107	2314	3.3	0
21108	21108	2314	4.79	0
21109	21109	2314	0.66	0
21110	21110	2314	4.8322	0
21111	21111	2314	0.7054	0
21112	21112	2314	4	0
21113	21113	2314	9	0
21114	21114	2314	1	0
21115	21115	2314	2	0
21116	21116	2314	0.6	0
21117	21117	2314	0.8	0
21118	21118	2314		0
21119	21119	2314		0
21120	21120	2314	2	0
21121	21121	2314	0	0
21122	21122	2314	1	0
21123	21123	2314	0.5	0
21124	21124	2314	0.7	0
21125	21125	2314	1	0
21126	21126	2314	0.6	0
21127	21127	2314	0.7	0
21128	21128	2314		0
21129	21129	2314		0
21130	21130	2314	2	0
21131	21131	2314	12	0
21132	21132	2314	3	0
21133	21133	2314	1	0
21134	21134	2314	6	0
21135	21135	2314	3	0
21136	21136	2314	2	0
21137	21137	2314	1	0
21138	21138	2314	0	0
21139	21139	2314	0	0
21140	21140	2314	0	0
21141	21141	2314	0	0
21142	21142	2314	0	0
21143	21143	2314	0	0
21144	21144	2314	0	0
21145	21145	2314	0	0
21146	21146	2314	B0L5	0
21147	21147	2314	1.1	0
21148	21148	2314	1.2	0
21149	21149	2314	24.2	0
21150	21150	2314	12	0
21151	21151	2314	26.5	0
21152	21152	2314	16	0
21153	21153	2314	4.5	0
21154	21154	2314	84	0
21155	21155	2314	2.1	0
21156	21156	2314	1.6	0
21157	21157	2314	0	0
21158	21158	2314	0	0
21159	21159	2314	0	0
21160	21160	2314	60.6	0
21161	21161	2314	125	0
21162	21162	2314	125	0
21163	21163	2314	10	0
21164	21164	2314	15	0
21165	21165	2314	85	0
21166	21166	2314	0	0
21167	21167	2314	1.635	0
21168	21168	2314	22.26666667	0
21169	21169	2314	4.066666667	0
21170	21170	2314	150	0
21171	21171	2314		0
21172	21172	2314	1.61	0
21173	21173	2314	87	0
21174	21174	2314	37	0
21175	21175	2314	2.2	0
21176	21176	2314	0	0
21177	21177	2314	0	0
21178	21178	2314	90	0
21179	21179	2314	0	0
21180	21180	2314	0	0
21181	21181	2314	40	0
21182	21182	2314	32	0
21183	21183	2314	15	0
21184	21184	2314	150	0
21185	21185	2314		0
21186	21186	2314	1.61	0
21187	21187	2314	87	0
21188	21188	2314	37	0
21189	21189	2314	2.2	0
21190	21190	2314	0	0
21191	21191	2314	0	0
21192	21192	2314	90	0
21193	21193	2314	0	0
21194	21194	2314	0	0
21195	21195	2314	49.9	0
21196	21196	2314	11.4	0
21197	21197	2314	108.3	0
21198	21198	2314	30	0
21199	21199	2314	6.7	0
21200	21200	2314	2.2	0
21201	21201	2314	4.73	0
21202	21202	2314	1.42	0
21203	21203	2314	4.7414	0
21204	21204	2314	1.4522	0
21205	21205	2314	26	0
21206	21206	2314	1	0
21207	21207	2314	2	0
21208	21208	2314	3	0
21209	21209	2314	0.2	0
21210	21210	2314	0.2	0
21211	21211	2314	1.6	0
21212	21212	2314	1.8	0
21213	21213	2314	4	0
21214	21214	2314	1	0
21215	21215	2314	1	0
21216	21216	2314	0.6	0
21217	21217	2314	0.3	0
21218	21218	2314	0.3	0
21219	21219	2314	0.7	0
21220	21220	2314	1	0
21221	21221	2314	0.9	0
21222	21222	2314	1	0
21223	21223	2314	2	0
21224	21224	2314	23	0
21225	21225	2314	7	0
21226	21226	2314	1	0
21227	21227	2314	6	0
21228	21228	2314	5	0
21229	21229	2314	4	0
21230	21230	2314	3	0
21231	21231	2314	2	0
21232	21232	2314	1	0
21233	21233	2314	0	0
21234	21234	2314	1	0
21235	21235	2314	0	0
21236	21236	2314	1	0
21237	21237	2314	0	0
21238	21238	2314	0	0
21239	21239	2314	B0L5	0
21240	21240	2314	1.4	0
21241	21241	2314	1.55	0
21242	21242	2314	26.4	0
21243	21243	2314	16	0
21244	21244	2314	48.3	0
21245	21245	2314	49	0
21246	21246	2314	21.1	0
21247	21247	2314	264	0
21248	21248	2314	2.6	0
21249	21249	2314	2.3	0
21250	21250	2314	15	0
21251	21251	2314	11	0
21252	21252	2314	0	0
21253	21253	2314	76	0
21254	21254	2314	250	0
21255	21255	2314	250	0
21256	21256	2314	10	0
21257	21257	2314	50	0
21258	21258	2314	90	0
21259	21259	2314	0	0
21260	21260	2314	2.118333333	0
21261	21261	2314	29.95	0
21262	21262	2314	6.933333333	0
21263	21263	2314	350	0
21264	21264	2314	375	0
21265	21265	2314	2.87	0
21266	21266	2314	96.5	0
21267	21267	2314	47	0
21268	21268	2314	4.39	0
21269	21269	2314	3	0
21270	21270	2314	0	0
21271	21271	2314	95	0
21272	21272	2314	15	0
21273	21273	2314	0	0
21274	21274	2314	182	0
21275	21275	2314	74	0
21276	21276	2314	20	0
21277	21277	2314	350	0
21278	21278	2314	375	0
21279	21279	2314	2.87	0
21280	21280	2314	96.5	0
21281	21281	2314	47	0
21282	21282	2314	4.39	0
21283	21283	2314	3	0
21284	21284	2314	0	0
21285	21285	2314	95	0
21286	21286	2314	15	0
21287	21287	2314	0	0
21288	21288	2314	112.9	0
21289	21289	2314	28.6	0
21290	21290	2314	251.8	0
21291	21291	2314	74.3	0
21292	21292	2314	14.7	0
21293	21293	2314	5.6	0
21294	21294	2314	12.95	0
21295	21295	2314	3.99	0
21296	21296	2314	12.9786	0
21297	21297	2314	4.0699	0
21298	21298	2314	.	0
21299	21299	2314	.	0
21300	21300	2314	0	0
21301	21301	2314	0	0
21302	21302	2314		0
21303	21303	2314		0
21304	21304	2314		0
21305	21305	2314		0
21306	21306	2314	0	0
21307	21307	2314	0	0
21308	21308	2314	0.6	0
21309	21309	2314	0.2	0
21310	21310	2314		0
21311	21311	2314		0
21312	21312	2314		0
21313	21313	2314		0
21314	21314	2314		0
21315	21315	2314		0
21316	21316	2314	1	0
21317	21317	2314	4	0
21318	21318	2314	0	0
21319	21319	2314	.	0
21320	21320	2314	4	0
21321	21321	2314	0	0
21322	21322	2314	0	0
21323	21323	2314	0	0
21324	21324	2314	0	0
21325	21325	2314	0	0
21326	21326	2314	0	0
21327	21327	2314	0	0
21328	21328	2314	0	0
21329	21329	2314	0	0
21330	21330	2314	0	0
21331	21331	2314	0	0
21332	21332	2314	B0L2	0
21333	21333	2314	0.2	0
21334	21334	2314	0.2	0
21335	21335	2314	17	0
21336	21336	2314	0	0
21337	21337	2314	.	0
21338	21338	2314	0	0
21339	21339	2314	.	0
21340	21340	2314	10	0
21341	21341	2314	1.7	0
21342	21342	2314	1.5	0
21343	21343	2314	0	0
21344	21344	2314	0	0
21345	21345	2314	0	0
21346	21346	2314	37.5	0
21347	21347	2314	0	0
21348	21348	2314	10	0
21349	21349	2314	0	0
21350	21350	2314	0	0
21351	21351	2314	0	0
21352	21352	2314	0	0
21353	21353	2314		0
21354	21354	2314		0
21355	21355	2314		0
21356	21356	2314		0
21357	21357	2314		0
21358	21358	2314		0
21359	21359	2314		0
21360	21360	2314		0
21361	21361	2314		0
21362	21362	2314		0
21363	21363	2314		0
21364	21364	2314		0
21365	21365	2314		0
21366	21366	2314		0
21367	21367	2314		0
21368	21368	2314		0
21369	21369	2314		0
21370	21370	2314	.	0
21371	21371	2314		0
21372	21372	2314	.	0
21373	21373	2314	.	0
21374	21374	2314	.	0
21375	21375	2314	.	0
21376	21376	2314	.	0
21377	21377	2314	.	0
21378	21378	2314	.	0
21379	21379	2314	.	0
21380	21380	2314	.	0
21381	21381	2314	.	0
21382	21382	2314	.	0
21383	21383	2314	.	0
21384	21384	2314	.	0
21385	21385	2314	.	0
21386	21386	2314	.	0
21387	21387	2314	0.22	0
21388	21388	2314	0.01	0
21389	21389	2314	0.22	0
21390	21390	2314	0.01	0
21391	21391	2314	6	0
21392	21392	2314	126	0
21393	21393	2314	0	0
21394	21394	2314	1	0
21395	21395	2314	0.6	0
21396	21396	2314	0.8	0
21397	21397	2314		0
21398	21398	2314		0
21399	21399	2314	2	0
21400	21400	2314	0	0
21401	21401	2314	1	0
21402	21402	2314	0.4	0
21403	21403	2314	0.8	0
21404	21404	2314	0.9	0
21405	21405	2314	0.5	0
21406	21406	2314	0.5	0
21407	21407	2314		0
21408	21408	2314		0
21409	21409	2314	2	0
21410	21410	2314	10	0
21411	21411	2314	2	0
21412	21412	2314	1	0
21413	21413	2314	6	0
21414	21414	2314	2	0
21415	21415	2314	2	0
21416	21416	2314	0	0
21417	21417	2314	0	0
21418	21418	2314	0	0
21419	21419	2314	0	0
21420	21420	2314	0	0
21421	21421	2314	0	0
21422	21422	2314	0	0
21423	21423	2314	0	0
21424	21424	2314	0	0
21425	21425	2314	B0L5	0
21426	21426	2314	1.2	0
21427	21427	2314	1.25	0
21428	21428	2314	26.7	0
21429	21429	2314	11	0
21430	21430	2314	40.5	0
21431	21431	2314	30	0
21432	21432	2314	20.3	0
21433	21433	2314	169	0
21434	21434	2314	2.7	0
21435	21435	2314	2.4	0
21436	21436	2314	0	0
21437	21437	2314	0	0
21438	21438	2314	0	0
21439	21439	2314	73.5	0
21440	21440	2314	0	0
21441	21441	2314	200	0
21442	21442	2314	0	0
21443	21443	2314	0	0
21444	21444	2314	0	0
21445	21445	2314	0	0
21446	21446	2314		0
21447	21447	2314		0
21448	21448	2314		0
21449	21449	2314		0
21450	21450	2314		0
21451	21451	2314		0
21452	21452	2314		0
21453	21453	2314		0
21454	21454	2314		0
21455	21455	2314		0
21456	21456	2314		0
21457	21457	2314		0
21458	21458	2314		0
21459	21459	2314		0
21460	21460	2314	133	0
21461	21461	2314		0
21462	21462	2314		0
21463	21463	2314	.	0
21464	21464	2314		0
21465	21465	2314	.	0
21466	21466	2314	.	0
21467	21467	2314	.	0
21468	21468	2314	.	0
21469	21469	2314	.	0
21470	21470	2314	.	0
21471	21471	2314	.	0
21472	21472	2314	.	0
21473	21473	2314	.	0
21474	21474	2314	.	0
21475	21475	2314	.	0
21476	21476	2314	.	0
21477	21477	2314	.	0
21478	21478	2314	.	0
21479	21479	2314	.	0
21480	21480	2314	17.88	0
21481	21481	2314	1.06	0
21482	21482	2314	17.88	0
21483	21483	2314	1.06	0
21484	21484	2314	37	0
21485	21485	2314	6	0
21486	21486	2314	0	0
21487	21487	2314	1	0
21488	21488	2314		0
21489	21489	2314		0
21490	21490	2314		0
21491	21491	2314		0
21492	21492	2314	1	0
21493	21493	2314	0	0
21494	21494	2314	1	0
21495	21495	2314	0.45	0
21496	21496	2314	0.7	0
21497	21497	2314	0.9	0
21498	21498	2314		0
21499	21499	2314		0
21500	21500	2314		0
21501	21501	2314		0
21502	21502	2314	2	0
21503	21503	2314	9	0
21504	21504	2314	2	0
21505	21505	2314	1	0
21506	21506	2314	6	0
21507	21507	2314	2	0
21508	21508	2314	1	0
21509	21509	2314	0	0
21510	21510	2314	0	0
21511	21511	2314	0	0
21512	21512	2314	0	0
21513	21513	2314	0	0
21514	21514	2314	0	0
21515	21515	2314	0	0
21516	21516	2314	0	0
21517	21517	2314	0	0
21518	21518	2314	B0L4	0
21519	21519	2314	0.8	0
21520	21520	2314	0.9	0
21521	21521	2314	27.2	0
21522	21522	2314	15	0
21523	21523	2314	34.5	0
21524	21524	2314	21	0
21525	21525	2314	7.4	0
21526	21526	2314	134	0
21527	21527	2314	1.8	0
21528	21528	2314	2	0
21529	21529	2314	0	0
21530	21530	2314	0	0
21531	21531	2314	0	0
21532	21532	2314	80	0
21533	21533	2314	150	0
21534	21534	2314	250	0
21535	21535	2314	10	0
21536	21536	2314	30	0
21537	21537	2314	60	0
21538	21538	2314	0	0
21539	21539	2314	1.326666667	0
21540	21540	2314	16.58333333	0
21541	21541	2314	3.816666667	0
21542	21542	2314	15	0
21543	21543	2314	200	0
21544	21544	2314	2.07	0
21545	21545	2314	67.5	0
21546	21546	2314	42.5	0
21547	21547	2314	3.59	0
21548	21548	2314	0	0
21549	21549	2314	0	0
21550	21550	2314	5	0
21551	21551	2314	0	0
21552	21552	2314	0	0
21553	21553	2314	64	0
21554	21554	2314	15	0
21555	21555	2314	3	0
21556	21556	2314	15	0
21557	21557	2314	200	0
21558	21558	2314	2.07	0
21559	21559	2314	67.5	0
21560	21560	2314	42.5	0
21561	21561	2314	3.59	0
21562	21562	2314	0	0
21563	21563	2314	0	0
21564	21564	2314	5	0
21565	21565	2314	0	0
21566	21566	2314	0	0
21567	21567	2314	79.8	0
21568	21568	2314	19.1	0
21569	21569	2314	107	0
21570	21570	2314	33	0
21571	21571	2314	9.3	0
21572	21572	2314	2.9	0
21573	21573	2314	6.59	0
21574	21574	2314	1.01	0
21575	21575	2314	6.6091	0
21576	21576	2314	1.0459	0
21577	21577	2314	6	0
21578	21578	2314	16	0
21579	21579	2314	2	0
21580	21580	2314	3	0
21581	21581	2314	0.8	0
21582	21582	2314	1	0
21583	21583	2314	1.2	0
21584	21584	2314	1.5	0
21585	21585	2314	4	0
21586	21586	2314	0	0
21587	21587	2314	1.3	0
21588	21588	2314	0.8	0
21589	21589	2314	0.7	0
21590	21590	2314	1	0
21591	21591	2314	0.8	0
21592	21592	2314	1	0
21593	21593	2314	0.9	0
21594	21594	2314	1.1	0
21595	21595	2314	1	0
21596	21596	2314	21	0
21597	21597	2314	4	0
21598	21598	2314	1	0
21599	21599	2314	8	0
21600	21600	2314	5	0
21601	21601	2314	4	0
21602	21602	2314	3	0
21603	21603	2314	1	0
21604	21604	2314	0	0
21605	21605	2314	0	0
21606	21606	2314	0	0
21607	21607	2314	0	0
21608	21608	2314	0	0
21609	21609	2314	0	0
21610	21610	2314	0	0
21611	21611	2314	B0L6	0
21612	21612	2314	1.1	0
21613	21613	2314	1.5	0
21614	21614	2314	27.5	0
21615	21615	2314	12	0
21616	21616	2314	46.3	0
21617	21617	2314	47	0
21618	21618	2314	29.3	0
21619	21619	2314	284	0
21620	21620	2314	2.8	0
21621	21621	2314	2.3	0
21622	21622	2314	0	0
21623	21623	2314	6	0
21624	21624	2314	0	0
21625	21625	2314	76.5	0
21626	21626	2314	150	0
21627	21627	2314	275	0
21628	21628	2314	30	0
21629	21629	2314	30	0
21630	21630	2314	70	0
21631	21631	2314	1	0
21632	21632	2314	2.055	0
21633	21633	2314	27.23333333	0
21634	21634	2314	6.033333333	0
21635	21635	2314	150	0
21636	21636	2314	275	0
21637	21637	2314	2.88	0
21638	21638	2314	54	0
21639	21639	2314	32	0
21640	21640	2314	4.32	0
21641	21641	2314	0	0
21642	21642	2314	0	0
21643	21643	2314	60	0
21644	21644	2314	0	0
21645	21645	2314	0	0
21646	21646	2314	104	0
21647	21647	2314	31	0
21648	21648	2314	15	0
21649	21649	2314	150	0
21650	21650	2314	275	0
21651	21651	2314	2.88	0
21652	21652	2314	54	0
21653	21653	2314	32	0
21654	21654	2314	4.32	0
21655	21655	2314	0	0
21656	21656	2314	0	0
21657	21657	2314	60	0
21658	21658	2314	0	0
21659	21659	2314	0	0
21660	21660	2314	108.3	0
21661	21661	2314	28.1	0
21662	21662	2314	285.3	0
21663	21663	2314	81	0
21664	21664	2314	65.4	0
21665	21665	2314	17.7	0
21666	21666	2314	11.29	0
21667	21667	2314	1.88	0
21668	21668	2314	11.3181	0
21669	21669	2314	1.9787	0
21670	21670	2314	5	0
21671	21671	2314	22	0
21672	21672	2314	2	0
21673	21673	2314	3	0
21674	21674	2314	0.4	0
21675	21675	2314	0.5	0
21676	21676	2314	0.8	0
21677	21677	2314	0.8	0
21678	21678	2314	3	0
21679	21679	2314	0	0
21680	21680	2314	0.7	0
21681	21681	2314	0.3	0
21682	21682	2314	0.3	0
21683	21683	2314	0.5	0
21684	21684	2314	0.4	0
21685	21685	2314	0.6	0
21686	21686	2314		0
21687	21687	2314		0
21688	21688	2314	0	0
21689	21689	2314	13	0
21690	21690	2314	3	0
21691	21691	2314	1	0
21692	21692	2314	6	0
21693	21693	2314	3	0
21694	21694	2314	3	0
21695	21695	2314	1	0
21696	21696	2314	0	0
21697	21697	2314	0	0
21698	21698	2314	0	0
21699	21699	2314	0	0
21700	21700	2314	0	0
21701	21701	2314	0	0
21702	21702	2314	0	0
21703	21703	2314	0	0
21704	21704	2314	B0L5	0
21705	21705	2314	1.05	0
21706	21706	2314	1.3	0
21707	21707	2314	23.1	0
21708	21708	2314	14	0
21709	21709	2314	32.2	0
21710	21710	2314	33	0
21711	21711	2314	9.8	0
21712	21712	2314	167	0
21713	21713	2314	2.7	0
21714	21714	2314	2.3	0
21715	21715	2314	12	0
21716	21716	2314	9	0
21717	21717	2314	0	0
21718	21718	2314	72	0
21719	21719	2314	100	0
21720	21720	2314	100	0
21721	21721	2314	25	0
21722	21722	2314	25	0
21723	21723	2314	90	0
21724	21724	2314	0	0
21725	21725	2314	2.285	0
21726	21726	2314	30.48333333	0
21727	21727	2314	7.35	0
21728	21728	2314	300	0
21729	21729	2314	350	0
21730	21730	2314	2.21	0
21731	21731	2314	82.5	0
21732	21732	2314	39	0
21733	21733	2314	4.19	0
21734	21734	2314	0	0
21735	21735	2314	0	0
21736	21736	2314	75	0
21737	21737	2314	20	0
21738	21738	2314	0	0
21739	21739	2314	117	0
21740	21740	2314	51	0
21741	21741	2314	0	0
21742	21742	2314	300	0
21743	21743	2314	350	0
21744	21744	2314	2.21	0
21745	21745	2314	82.5	0
21746	21746	2314	39	0
21747	21747	2314	4.19	0
21748	21748	2314	0	0
21749	21749	2314	0	0
21750	21750	2314	75	0
21751	21751	2314	20	0
21752	21752	2314	0	0
21753	21753	2314	79.1	0
21754	21754	2314	26.7	0
21755	21755	2314	110.1	0
21756	21756	2314	36.5	0
21757	21757	2314	9.6	0
21758	21758	2314	5	0
21759	21759	2314	9.07	0
21760	21760	2314	2	0
21761	21761	2314	9.0967	0
21762	21762	2314	2.0415	0
21763	21763	2314	16	0
21764	21764	2314	33	0
21765	21765	2314	1	0
21766	21766	2314	3	0
21767	21767	2314	0.5	0
21768	21768	2314	0.8	0
21769	21769	2314	0.9	0
21770	21770	2314	0.9	0
21771	21771	2314	3	0
21772	21772	2314	0	0
21773	21773	2314	0.6	0
21774	21774	2314	0.4	0
21775	21775	2314	0.5	0
21776	21776	2314	0.9	0
21777	21777	2314	0.6	0
21778	21778	2314	0.8	0
21779	21779	2314	0.6	0
21780	21780	2314	0.8	0
21781	21781	2314	0	0
21782	21782	2314	16	0
21783	21783	2314	3	0
21784	21784	2314	1	0
21785	21785	2314	7	0
21786	21786	2314	3	0
21787	21787	2314	4	0
21788	21788	2314	2	0
21789	21789	2314	0	0
21790	21790	2314	0	0
21791	21791	2314	0	0
21792	21792	2314	0	0
21793	21793	2314	0	0
21794	21794	2314	0	0
21795	21795	2314	0	0
21796	21796	2314	0	0
21797	21797	2314	B0L6	0
21798	21798	2314	1	0
21799	21799	2314	1.1	0
21800	21800	2314	20.5	0
21801	21801	2314	14	0
21802	21802	2314	40.1	0
21803	21803	2314	44	0
21804	21804	2314	13	0
21805	21805	2314	242	0
21806	21806	2314	2.3	0
21807	21807	2314	1.7	0
21808	21808	2314	18	0
21809	21809	2314	14	0
21810	21810	2314	8	0
21811	21811	2314	78	0
21812	21812	2314	125	0
21813	21813	2314	125	0
21814	21814	2314	10	0
21815	21815	2314	20	0
21816	21816	2314	80	0
21817	21817	2314	1	0
21818	21818	2314	2.386666667	0
21819	21819	2314	28.11666667	0
21820	21820	2314	7.383333333	0
21821	21821	2314	250	0
21822	21822	2314		0
21823	21823	2314	2.43	0
21824	21824	2314	65	0
21825	21825	2314	44	0
21826	21826	2314	3.21	0
21827	21827	2314	0	0
21828	21828	2314	0	0
21829	21829	2314	85	0
21830	21830	2314	20	0
21831	21831	2314	0	0
21832	21832	2314	148	0
21833	21833	2314	51	0
21834	21834	2314	18	0
21835	21835	2314	250	0
21836	21836	2314		0
21837	21837	2314	2.43	0
21838	21838	2314	65	0
21839	21839	2314	44	0
21840	21840	2314	3.21	0
21841	21841	2314	0	0
21842	21842	2314	0	0
21843	21843	2314	85	0
21844	21844	2314	20	0
21845	21845	2314	0	0
21846	21846	2314	93.2	0
21847	21847	2314	31	0
21848	21848	2314	180.3	0
21849	21849	2314	63.4	0
21850	21850	2314	5.4	0
21851	21851	2314	3.5	0
21852	21852	2314	9.76	0
21853	21853	2314	1.99	0
21854	21854	2314	9.791	0
21855	21855	2314	2.0569	0
21856	21856	2314	14	0
21857	21857	2314	8	0
21858	21858	2314	2	0
21859	21859	2314	2	0
21860	21860	2314	0.6	0
21861	21861	2314	0.9	0
21862	21862	2314	0.8	0
21863	21863	2314	0.8	0
21864	21864	2314	3	0
21865	21865	2314	0	0
21866	21866	2314	0.6	0
21867	21867	2314	0.3	0
21868	21868	2314	0.5	0
21869	21869	2314	0.9	0
21870	21870	2314	0.5	0
21871	21871	2314	0.7	0
21872	21872	2314	0.5	0
21873	21873	2314	0.6	0
21874	21874	2314	0	0
21875	21875	2314	18	0
21876	21876	2314	4	0
21877	21877	2314	1	0
21878	21878	2314	8	0
21879	21879	2314	4	0
21880	21880	2314	3	0
21881	21881	2314	2	0
21882	21882	2314	1	0
21883	21883	2314	0	0
21884	21884	2314	0	0
21885	21885	2314	0	0
21886	21886	2314	0	0
21887	21887	2314	0	0
21888	21888	2314	0	0
21889	21889	2314	0	0
21890	21890	2314	B0L5	0
21891	21891	2314	1.1	0
21892	21892	2314	1.2	0
21893	21893	2314	17.4	0
21894	21894	2314	14	0
21895	21895	2314	39.3	0
21896	21896	2314	34	0
21897	21897	2314	14.4	0
21898	21898	2314	189	0
21899	21899	2314	2.6	0
21900	21900	2314	2.2	0
21901	21901	2314	14	0
21902	21902	2314	10	0
21903	21903	2314	4	0
21904	21904	2314	75.5	0
21905	21905	2314	100	0
21906	21906	2314	100	0
21907	21907	2314	10	0
21908	21908	2314	25	0
21909	21909	2314	80	0
21910	21910	2314	1	0
21911	21911	2314	2.053333333	0
21912	21912	2314	25.15	0
21913	21913	2314	6.033333333	0
21914	21914	2314	225	0
21915	21915	2314		0
21916	21916	2314	2.42	0
21917	21917	2314	87	0
21918	21918	2314	40	0
21919	21919	2314	3.26	0
21920	21920	2314	0	0
21921	21921	2314	0	0
21922	21922	2314	90	0
21923	21923	2314	0	0
21924	21924	2314	0	0
21925	21925	2314	103	0
21926	21926	2314	70	0
21927	21927	2314	14	0
21928	21928	2314	225	0
21929	21929	2314		0
21930	21930	2314	2.42	0
21931	21931	2314	87	0
21932	21932	2314	40	0
21933	21933	2314	3.26	0
21934	21934	2314	0	0
21935	21935	2314	0	0
21936	21936	2314	90	0
21937	21937	2314	0	0
21938	21938	2314	0	0
21939	21939	2314	100.3	0
21940	21940	2314	31.2	0
21941	21941	2314	131.8	0
21942	21942	2314	41.5	0
21943	21943	2314	10.1	0
21944	21944	2314	3.8	0
21945	21945	2314	9.61	0
21946	21946	2314	1.09	0
21947	21947	2314	9.6412	0
21948	21948	2314	1.1353	0
21949	21949	2314	12	0
21950	21950	2314	12	0
21951	21951	2314	2	0
21952	21952	2314	3	0
21953	21953	2314	0.8	0
21954	21954	2314	1.2	0
21955	21955	2314	1.2	0
21956	21956	2314	1.3	0
21957	21957	2314	4	0
21958	21958	2314	0	0
21959	21959	2314	1	0
21960	21960	2314	0.5	0
21961	21961	2314	0.8	0
21962	21962	2314	1.2	0
21963	21963	2314	0.7	0
21964	21964	2314	1.1	0
21965	21965	2314	0.7	0
21966	21966	2314	0.9	0
21967	21967	2314	2	0
21968	21968	2314	19	0
21969	21969	2314	4	0
21970	21970	2314	1	0
21971	21971	2314	8	0
21972	21972	2314	4	0
21973	21973	2314	4	0
21974	21974	2314	2	0
21975	21975	2314	1	0
21976	21976	2314	0	0
21977	21977	2314	0	0
21978	21978	2314	0	0
21979	21979	2314	0	0
21980	21980	2314	0	0
21981	21981	2314	0	0
21982	21982	2314	0	0
21983	21983	2314	B0L6	0
21984	21984	2314	1.15	0
21985	21985	2314	1.3	0
21986	21986	2314	17.6	0
21987	21987	2314	11	0
21988	21988	2314	50.3	0
21989	21989	2314	38	0
21990	21990	2314	18.7	0
21991	21991	2314	207	0
21992	21992	2314	3	0
21993	21993	2314	2.3	0
21994	21994	2314	18	0
21995	21995	2314	9	0
21996	21996	2314	2	0
21997	21997	2314	79	0
21998	21998	2314	100	0
21999	21999	2314	100	0
22000	22000	2314	5	0
22001	22001	2314	30	0
22002	22002	2314	75	0
22003	22003	2314	8	0
22004	22004	2314	2.233333333	0
22005	22005	2314	33.23333333	0
22006	22006	2314	7.65	0
22007	22007	2314	0	0
22008	22008	2314	150	0
22009	22009	2314		0
22010	22010	2314	65	0
22011	22011	2314	43	0
22012	22012	2314	2.97	0
22013	22013	2314	0	0
22014	22014	2314	0	0
22015	22015	2314	0	0
22016	22016	2314	0	0
22017	22017	2314	0	0
22018	22018	2314	174	0
22019	22019	2314	0	0
22020	22020	2314	0	0
22021	22021	2314	0	0
22022	22022	2314	150	0
22023	22023	2314	.	0
22024	22024	2314	65	0
22025	22025	2314	43	0
22026	22026	2314	2.97	0
22027	22027	2314	0	0
22028	22028	2314	0	0
22029	22029	2314	0	0
22030	22030	2314	0	0
22031	22031	2314	0	0
22032	22032	2314	.	0
22033	22033	2314	.	0
22034	22034	2314	158.4	0
22035	22035	2314	47.1	0
22036	22036	2314	58.5	0
22037	22037	2314	20.1	0
22038	22038	2314	12.59	0
22039	22039	2314	1.3	0
22040	22040	2314	12.59	0
22041	22041	2314	1.3672	0
22042	22042	2314	.	0
22043	22043	2314	5	0
22044	22044	2314	1	0
22045	22045	2314	2	0
22046	22046	2314	0.6	0
22047	22047	2314	0.9	0
22048	22048	2314		0
22049	22049	2314		0
22050	22050	2314	2	0
22051	22051	2314	0	0
22052	22052	2314	1	0
22053	22053	2314	0.4	0
22054	22054	2314	0.9	0
22055	22055	2314	1.2	0
22056	22056	2314	0.5	0
22057	22057	2314	0.6	0
22058	22058	2314		0
22059	22059	2314		0
22060	22060	2314	2	0
22061	22061	2314	11	0
22062	22062	2314	3	0
22063	22063	2314	1	0
22064	22064	2314	5	0
22065	22065	2314	3	0
22066	22066	2314	2	0
22067	22067	2314	1	0
22068	22068	2314	0	0
22069	22069	2314	0	0
22070	22070	2314	0	0
22071	22071	2314	0	0
22072	22072	2314	0	0
22073	22073	2314	0	0
22074	22074	2314	0	0
22075	22075	2314	0	0
22076	22076	2314	B0L4	0
22077	22077	2314	1.1	0
22078	22078	2314	1.3	0
22079	22079	2314	19.6	0
22080	22080	2314	8	0
22081	22081	2314	30.8	0
22082	22082	2314	34	0
22083	22083	2314	16.1	0
22084	22084	2314	151	0
22085	22085	2314	2.1	0
22086	22086	2314	1.8	0
22087	22087	2314	0	0
22088	22088	2314	0	0
22089	22089	2314	0	0
22090	22090	2314	77	0
22091	22091	2314	250	0
22092	22092	2314	250	0
22093	22093	2314	15	0
22094	22094	2314	50	0
22095	22095	2314	90	0
22096	22096	2314	0	0
22097	22097	2314	2.188333333	0
22098	22098	2314	28.3	0
22099	22099	2314	7.25	0
22100	22100	2314	200	0
22101	22101	2314	225	0
22102	22102	2314	2.11	0
22103	22103	2314	80	0
22104	22104	2314	38	0
22105	22105	2314	3.23	0
22106	22106	2314	12	0
22107	22107	2314	0	0
22108	22108	2314	85	0
22109	22109	2314	30	0
22110	22110	2314	0	0
22111	22111	2314	103	0
22112	22112	2314	52	0
22113	22113	2314	29	0
22114	22114	2314	200	0
22115	22115	2314	225	0
22116	22116	2314	2.11	0
22117	22117	2314	80	0
22118	22118	2314	38	0
22119	22119	2314	3.23	0
22120	22120	2314	12	0
22121	22121	2314	0	0
22122	22122	2314	85	0
22123	22123	2314	30	0
22124	22124	2314	0	0
22125	22125	2314	62.1	0
22126	22126	2314	19	0
22127	22127	2314	141.3	0
22128	22128	2314	45.1	0
22129	22129	2314	6.7	0
22130	22130	2314	2.2	0
22131	22131	2314	9.88	0
22132	22132	2314	2.59	0
22133	22133	2314	9.899	0
22134	22134	2314	2.6373	0
22135	22135	2314	14	0
22136	22136	2314	21	0
22137	22137	2314	1	0
22138	22138	2314	2	0
22139	22139	2314	0.5	0
22140	22140	2314	0.6	0
22141	22141	2314	0.9	0
22142	22142	2314	0.9	0
22143	22143	2314	3	0
22144	22144	2314	0	0
22145	22145	2314	0.8	0
22146	22146	2314	0.4	0
22147	22147	2314	0.4	0
22148	22148	2314	0.7	0
22149	22149	2314	0.6	0
22150	22150	2314	0.9	0
22151	22151	2314		0
22152	22152	2314		0
22153	22153	2314	2	0
22154	22154	2314	16	0
22155	22155	2314	4	0
22156	22156	2314	1	0
22157	22157	2314	7	0
22158	22158	2314	3	0
22159	22159	2314	3	0
22160	22160	2314	2	0
22161	22161	2314	1	0
22162	22162	2314	0	0
22163	22163	2314	0	0
22164	22164	2314	0	0
22165	22165	2314	0	0
22166	22166	2314	0	0
22167	22167	2314	0	0
22168	22168	2314	0	0
22169	22169	2314	B0L6	0
22170	22170	2314	1	0
22171	22171	2314	1.1	0
22172	22172	2314	23.4	0
22173	22173	2314	14	0
22174	22174	2314	40.5	0
22175	22175	2314	14	0
22176	22176	2314	7.3	0
22177	22177	2314	158	0
22178	22178	2314	2.5	0
22179	22179	2314	1.9	0
22180	22180	2314	11	0
22181	22181	2314	8	0
22182	22182	2314	1	0
22183	22183	2314	66.5	0
22184	22184	2314	75	0
22185	22185	2314	75	0
22186	22186	2314	2	0
22187	22187	2314	10	0
22188	22188	2314	80	0
22189	22189	2314	2	0
22190	22190	2314	1.586666667	0
22191	22191	2314	20.73333333	0
22192	22192	2314	4.583333333	0
22193	22193	2314	200	0
22194	22194	2314	250	0
22195	22195	2314	2.1	0
22196	22196	2314	69	0
22197	22197	2314	35	0
22198	22198	2314	2.52	0
22199	22199	2314	0	0
22200	22200	2314	0	0
22201	22201	2314	85	0
22202	22202	2314	0	0
22203	22203	2314	0	0
22204	22204	2314	82	0
22205	22205	2314	42	0
22206	22206	2314	7	0
22207	22207	2314	200	0
22208	22208	2314	250	0
22209	22209	2314	2.1	0
22210	22210	2314	69	0
22211	22211	2314	35	0
22212	22212	2314	2.52	0
22213	22213	2314	0	0
22214	22214	2314	0	0
22215	22215	2314	85	0
22216	22216	2314	0	0
22217	22217	2314	0	0
22218	22218	2314	107.7	0
22219	22219	2314	40.8	0
22220	22220	2314	107.2	0
22221	22221	2314	36	0
22222	22222	2314	7.3	0
22223	22223	2314	3.9	0
22224	22224	2314	7.74	0
22225	22225	2314	1.93	0
22226	22226	2314	7.7808	0
22227	22227	2314	1.9699	0
22228	22228	2314	4	0
22229	22229	2314	5	0
22230	22230	2314	1	0
22231	22231	2314	2	0
22232	22232	2314	0.5	0
22233	22233	2314	0.4	0
22234	22234	2314		0
22235	22235	2314		0
22236	22236	2314	2	0
22237	22237	2314	0	0
22238	22238	2314	0.6	0
22239	22239	2314	0.5	0
22240	22240	2314	0.5	0
22241	22241	2314	0.5	0
22242	22242	2314	0.4	0
22243	22243	2314	0.4	0
22244	22244	2314		0
22245	22245	2314		0
22246	22246	2314	0	0
22247	22247	2314	6	0
22248	22248	2314	1	0
22249	22249	2314	2	0
22250	22250	2314	5	0
22251	22251	2314	1	0
22252	22252	2314	0	0
22253	22253	2314	0	0
22254	22254	2314	0	0
22255	22255	2314	0	0
22256	22256	2314	0	0
22257	22257	2314	0	0
22258	22258	2314	0	0
22259	22259	2314	0	0
22260	22260	2314	0	0
22261	22261	2314	0	0
22262	22262	2314	B0L4	0
22263	22263	2314	1	0
22264	22264	2314	1	0
22265	22265	2314	30.8	0
22266	22266	2314	9	0
22267	22267	2314	32.8	0
22268	22268	2314	9	0
22269	22269	2314	3.2	0
22270	22270	2314	56	0
22271	22271	2314	2.4	0
22272	22272	2314	2	0
22273	22273	2314	0	0
22274	22274	2314	4	0
22275	22275	2314	0	0
22276	22276	2314	83	0
22277	22277	2314	30	0
22278	22278	2314	30	0
22279	22279	2314	4	0
22280	22280	2314	3	0
22281	22281	2314	60	0
22282	22282	2314	1	0
22283	22283	2314	1.636666667	0
22284	22284	2314	23.05	0
22285	22285	2314	4.533333333	0
22286	22286	2314	125	0
22287	22287	2314	150	0
22288	22288	2314	1.23	0
22289	22289	2314	92	0
22290	22290	2314	40	0
22291	22291	2314	2.65	0
22292	22292	2314	0	0
22293	22293	2314	0	0
22294	22294	2314	60	0
22295	22295	2314	0	0
22296	22296	2314	0	0
22297	22297	2314	27	0
22298	22298	2314	2	0
22299	22299	2314	10	0
22300	22300	2314	125	0
22301	22301	2314	150	0
22302	22302	2314	1.23	0
22303	22303	2314	92	0
22304	22304	2314	40	0
22305	22305	2314	2.65	0
22306	22306	2314	0	0
22307	22307	2314	0	0
22308	22308	2314	60	0
22309	22309	2314	0	0
22310	22310	2314	0	0
22311	22311	2314	74.9	0
22312	22312	2314	16.2	0
22313	22313	2314	58.5	0
22314	22314	2314	17.1	0
22315	22315	2314	3.9	0
22316	22316	2314	1.8	0
22317	22317	2314	2.68	0
22318	22318	2314	0.38	0
22319	22319	2314	2.6962	0
22320	22320	2314	0.3989	0
\.


--
-- Data for Name: nd_experiment_project; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_project (nd_experiment_project_id, project_id, nd_experiment_id) FROM stdin;
1	3	1
2	3	2
3	3	3
4	3	4
5	3	5
6	3	6
7	3	7
8	3	8
9	3	9
10	3	10
11	3	11
12	3	12
13	3	13
14	3	14
15	3	15
16	3	16
17	3	17
18	3	18
19	3	19
20	3	20
21	3	21
22	3	22
23	3	23
24	3	24
25	3	25
26	3	26
27	3	27
28	3	28
29	3	29
30	3	30
31	3	31
32	3	32
33	3	33
34	3	34
35	3	35
36	3	36
37	3	37
38	3	38
39	3	39
40	3	40
41	3	41
42	3	42
43	3	43
44	3	44
45	3	45
46	3	46
47	3	47
48	3	48
49	3	49
50	3	50
51	3	51
52	3	52
53	3	53
54	3	54
55	3	55
56	3	56
57	3	57
58	3	58
59	3	59
60	3	60
61	3	61
62	3	62
63	3	63
64	3	64
65	3	65
66	3	66
67	3	67
68	3	68
69	3	69
70	3	70
71	3	71
72	3	72
73	3	73
74	3	74
75	3	75
76	3	76
77	3	77
78	3	78
79	3	79
80	3	80
81	3	81
82	3	82
83	3	83
84	3	84
85	3	85
86	3	86
87	3	87
88	3	88
89	3	89
90	3	90
91	3	91
92	3	92
93	3	93
94	3	94
95	3	95
96	3	96
97	3	97
98	3	98
99	3	99
100	3	100
101	3	101
102	3	102
103	3	103
104	3	104
105	3	105
106	3	106
107	3	107
108	3	108
109	3	109
110	3	110
111	3	111
112	3	112
113	3	113
114	3	114
115	3	115
116	3	116
117	3	117
118	3	118
119	3	119
120	3	120
121	3	121
122	3	122
123	3	123
124	3	124
125	3	125
126	3	126
127	3	127
128	3	128
129	3	129
130	3	130
131	3	131
132	3	132
133	3	133
134	3	134
135	3	135
136	3	136
137	3	137
138	3	138
139	3	139
140	3	140
141	3	141
142	3	142
143	3	143
144	3	144
145	3	145
146	3	146
147	3	147
148	3	148
149	3	149
150	3	150
151	3	151
152	3	152
153	3	153
154	3	154
155	3	155
156	3	156
157	3	157
158	3	158
159	3	159
160	3	160
161	3	161
162	3	162
163	3	163
164	3	164
165	3	165
166	3	166
167	3	167
168	3	168
169	3	169
170	3	170
171	3	171
172	3	172
173	3	173
174	3	174
175	3	175
176	3	176
177	3	177
178	3	178
179	3	179
180	3	180
181	3	181
182	3	182
183	3	183
184	3	184
185	3	185
186	3	186
187	3	187
188	3	188
189	3	189
190	3	190
191	3	191
192	3	192
193	3	193
194	3	194
195	3	195
196	3	196
197	3	197
198	3	198
199	3	199
200	3	200
201	3	201
202	3	202
203	3	203
204	3	204
205	3	205
206	3	206
207	3	207
208	3	208
209	3	209
210	3	210
211	3	211
212	3	212
213	3	213
214	3	214
215	3	215
216	3	216
217	3	217
218	3	218
219	3	219
220	3	220
221	3	221
222	3	222
223	3	223
224	3	224
225	3	225
226	3	226
227	3	227
228	3	228
229	3	229
230	3	230
231	3	231
232	3	232
233	3	233
234	3	234
235	3	235
236	3	236
237	3	237
238	3	238
239	3	239
240	3	240
\.


--
-- Data for Name: nd_experiment_protocol; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_protocol (nd_experiment_protocol_id, nd_experiment_id, nd_protocol_id) FROM stdin;
\.


--
-- Data for Name: nd_experiment_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_pub (nd_experiment_pub_id, nd_experiment_id, pub_id) FROM stdin;
\.


--
-- Data for Name: nd_experiment_stock; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_stock (nd_experiment_stock_id, nd_experiment_id, stock_id, type_id) FROM stdin;
1	1	1	2312
2	2	2	2312
3	3	3	2312
4	4	4	2312
5	5	5	2312
6	6	6	2312
7	7	7	2312
8	8	8	2312
9	9	9	2312
10	10	10	2312
11	11	11	2312
12	12	12	2312
13	13	13	2312
14	14	14	2312
15	15	15	2312
16	16	16	2312
17	17	17	2312
18	18	18	2312
19	19	19	2312
20	20	20	2312
21	21	21	2312
22	22	22	2312
23	23	23	2312
24	24	24	2312
25	25	25	2312
26	26	26	2312
27	27	27	2312
28	28	28	2312
29	29	29	2312
30	30	30	2312
31	31	31	2312
32	32	32	2312
33	33	33	2312
34	34	34	2312
35	35	35	2312
36	36	36	2312
37	37	37	2312
38	38	38	2312
39	39	39	2312
40	40	40	2312
41	41	41	2312
42	42	42	2312
43	43	43	2312
44	44	44	2312
45	45	45	2312
46	46	46	2312
47	47	47	2312
48	48	48	2312
49	49	49	2312
50	50	50	2312
51	51	51	2312
52	52	52	2312
53	53	53	2312
54	54	54	2312
55	55	55	2312
56	56	56	2312
57	57	57	2312
58	58	58	2312
59	59	59	2312
60	60	60	2312
61	61	61	2312
62	62	62	2312
63	63	63	2312
64	64	64	2312
65	65	65	2312
66	66	66	2312
67	67	67	2312
68	68	68	2312
69	69	69	2312
70	70	70	2312
71	71	71	2312
72	72	72	2312
73	73	73	2312
74	74	74	2312
75	75	75	2312
76	76	76	2312
77	77	77	2312
78	78	78	2312
79	79	79	2312
80	80	80	2312
81	81	81	2312
82	82	82	2312
83	83	83	2312
84	84	84	2312
85	85	85	2312
86	86	86	2312
87	87	87	2312
88	88	88	2312
89	89	89	2312
90	90	90	2312
91	91	91	2312
92	92	92	2312
93	93	93	2312
94	94	94	2312
95	95	95	2312
96	96	96	2312
97	97	97	2312
98	98	98	2312
99	99	99	2312
100	100	100	2312
101	101	101	2312
102	102	102	2312
103	103	103	2312
104	104	104	2312
105	105	105	2312
106	106	106	2312
107	107	107	2312
108	108	108	2312
109	109	109	2312
110	110	110	2312
111	111	111	2312
112	112	112	2312
113	113	113	2312
114	114	114	2312
115	115	115	2312
116	116	116	2312
117	117	117	2312
118	118	118	2312
119	119	119	2312
120	120	120	2312
121	121	121	2312
122	122	122	2312
123	123	123	2312
124	124	124	2312
125	125	125	2312
126	126	126	2312
127	127	127	2312
128	128	128	2312
129	129	129	2312
130	130	130	2312
131	131	131	2312
132	132	132	2312
133	133	133	2312
134	134	134	2312
135	135	135	2312
136	136	136	2312
137	137	137	2312
138	138	138	2312
139	139	139	2312
140	140	140	2312
141	141	141	2312
142	142	142	2312
143	143	143	2312
144	144	144	2312
145	145	145	2312
146	146	146	2312
147	147	147	2312
148	148	148	2312
149	149	149	2312
150	150	150	2312
151	151	151	2312
152	152	152	2312
153	153	153	2312
154	154	154	2312
155	155	155	2312
156	156	156	2312
157	157	157	2312
158	158	158	2312
159	159	159	2312
160	160	160	2312
161	161	161	2312
162	162	162	2312
163	163	163	2312
164	164	164	2312
165	165	165	2312
166	166	166	2312
167	167	167	2312
168	168	168	2312
169	169	169	2312
170	170	170	2312
171	171	171	2312
172	172	172	2312
173	173	173	2312
174	174	174	2312
175	175	175	2312
176	176	176	2312
177	177	177	2312
178	178	178	2312
179	179	179	2312
180	180	180	2312
181	181	181	2312
182	182	182	2312
183	183	183	2312
184	184	184	2312
185	185	185	2312
186	186	186	2312
187	187	187	2312
188	188	188	2312
189	189	189	2312
190	190	190	2312
191	191	191	2312
192	192	192	2312
193	193	193	2312
194	194	194	2312
195	195	195	2312
196	196	196	2312
197	197	197	2312
198	198	198	2312
199	199	199	2312
200	200	200	2312
201	201	201	2312
202	202	202	2312
203	203	203	2312
204	204	204	2312
205	205	205	2312
206	206	206	2312
207	207	207	2312
208	208	208	2312
209	209	209	2312
210	210	210	2312
211	211	211	2312
212	212	212	2312
213	213	213	2312
214	214	214	2312
215	215	215	2312
216	216	216	2312
217	217	217	2312
218	218	218	2312
219	219	219	2312
220	220	220	2312
221	221	221	2312
222	222	222	2312
223	223	223	2312
224	224	224	2312
225	225	225	2312
226	226	226	2312
227	227	227	2312
228	228	228	2312
229	229	229	2312
230	230	230	2312
231	231	231	2312
232	232	232	2312
233	233	233	2312
234	234	234	2312
235	235	235	2312
236	236	236	2312
237	237	237	2312
238	238	238	2312
239	239	239	2312
240	240	240	2312
\.


--
-- Data for Name: nd_experiment_stock_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_stock_dbxref (nd_experiment_stock_dbxref_id, nd_experiment_stock_id, dbxref_id) FROM stdin;
\.


--
-- Data for Name: nd_experiment_stockprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experiment_stockprop (nd_experiment_stockprop_id, nd_experiment_stock_id, type_id, value, rank) FROM stdin;
1	1	2203	0	0
2	2	2203	0	0
3	3	2203	0	0
4	4	2203	0	0
5	5	2203	0	0
6	6	2203	0	0
7	7	2203	0	0
8	8	2203	0	0
9	9	2203	0	0
10	10	2203	0	0
11	11	2203	0	0
12	12	2203	0	0
13	13	2203	0	0
14	14	2203	0	0
15	15	2203	0	0
16	16	2203	0	0
17	17	2203	0	0
18	18	2203	0	0
19	19	2203	0	0
20	20	2203	0	0
21	21	2203	0	0
22	22	2203	0	0
23	23	2203	0	0
24	24	2203	0	0
25	25	2203	0	0
26	26	2203	0	0
27	27	2203	0	0
28	28	2203	0	0
29	29	2203	0	0
30	30	2203	0	0
31	31	2203	0	0
32	32	2203	0	0
33	33	2203	0	0
34	34	2203	0	0
35	35	2203	0	0
36	36	2203	0	0
37	37	2203	0	0
38	38	2203	0	0
39	39	2203	0	0
40	40	2203	0	0
41	41	2203	0	0
42	42	2203	0	0
43	43	2203	0	0
44	44	2203	0	0
45	45	2203	0	0
46	46	2203	0	0
47	47	2203	0	0
48	48	2203	0	0
49	49	2203	0	0
50	50	2203	0	0
51	51	2203	0	0
52	52	2203	0	0
53	53	2203	0	0
54	54	2203	0	0
55	55	2203	0	0
56	56	2203	0	0
57	57	2203	0	0
58	58	2203	0	0
59	59	2203	0	0
60	60	2203	0	0
61	61	2203	0	0
62	62	2203	0	0
63	63	2203	0	0
64	64	2203	0	0
65	65	2203	0	0
66	66	2203	0	0
67	67	2203	0	0
68	68	2203	0	0
69	69	2203	0	0
70	70	2203	0	0
71	71	2203	0	0
72	72	2203	0	0
73	73	2203	0	0
74	74	2203	0	0
75	75	2203	0	0
76	76	2203	0	0
77	77	2203	0	0
78	78	2203	0	0
79	79	2203	0	0
80	80	2203	0	0
81	81	2203	0	0
82	82	2203	0	0
83	83	2203	0	0
84	84	2203	0	0
85	85	2203	0	0
86	86	2203	0	0
87	87	2203	0	0
88	88	2203	0	0
89	89	2203	0	0
90	90	2203	0	0
91	91	2203	0	0
92	92	2203	0	0
93	93	2203	0	0
94	94	2203	0	0
95	95	2203	0	0
96	96	2203	0	0
97	97	2203	0	0
98	98	2203	0	0
99	99	2203	0	0
100	100	2203	0	0
101	101	2203	0	0
102	102	2203	0	0
103	103	2203	0	0
104	104	2203	0	0
105	105	2203	0	0
106	106	2203	0	0
107	107	2203	0	0
108	108	2203	0	0
109	109	2203	0	0
110	110	2203	0	0
111	111	2203	0	0
112	112	2203	0	0
113	113	2203	0	0
114	114	2203	0	0
115	115	2203	0	0
116	116	2203	0	0
117	117	2203	0	0
118	118	2203	0	0
119	119	2203	0	0
120	120	2203	0	0
121	121	2203	100	0
122	122	2203	100	0
123	123	2203	100	0
124	124	2203	100	0
125	125	2203	100	0
126	126	2203	100	0
127	127	2203	100	0
128	128	2203	100	0
129	129	2203	100	0
130	130	2203	100	0
131	131	2203	100	0
132	132	2203	100	0
133	133	2203	100	0
134	134	2203	100	0
135	135	2203	100	0
136	136	2203	100	0
137	137	2203	100	0
138	138	2203	100	0
139	139	2203	100	0
140	140	2203	100	0
141	141	2203	100	0
142	142	2203	100	0
143	143	2203	100	0
144	144	2203	100	0
145	145	2203	100	0
146	146	2203	100	0
147	147	2203	100	0
148	148	2203	100	0
149	149	2203	100	0
150	150	2203	100	0
151	151	2203	100	0
152	152	2203	100	0
153	153	2203	100	0
154	154	2203	100	0
155	155	2203	100	0
156	156	2203	100	0
157	157	2203	100	0
158	158	2203	100	0
159	159	2203	100	0
160	160	2203	100	0
161	161	2203	100	0
162	162	2203	100	0
163	163	2203	100	0
164	164	2203	100	0
165	165	2203	100	0
166	166	2203	100	0
167	167	2203	100	0
168	168	2203	100	0
169	169	2203	100	0
170	170	2203	100	0
171	171	2203	100	0
172	172	2203	100	0
173	173	2203	100	0
174	174	2203	100	0
175	175	2203	100	0
176	176	2203	100	0
177	177	2203	100	0
178	178	2203	100	0
179	179	2203	100	0
180	180	2203	100	0
181	181	2203	100	0
182	182	2203	100	0
183	183	2203	100	0
184	184	2203	100	0
185	185	2203	100	0
186	186	2203	100	0
187	187	2203	100	0
188	188	2203	100	0
189	189	2203	100	0
190	190	2203	100	0
191	191	2203	100	0
192	192	2203	100	0
193	193	2203	100	0
194	194	2203	100	0
195	195	2203	100	0
196	196	2203	100	0
197	197	2203	100	0
198	198	2203	100	0
199	199	2203	100	0
200	200	2203	100	0
201	201	2203	100	0
202	202	2203	100	0
203	203	2203	100	0
204	204	2203	100	0
205	205	2203	100	0
206	206	2203	100	0
207	207	2203	100	0
208	208	2203	100	0
209	209	2203	100	0
210	210	2203	100	0
211	211	2203	100	0
212	212	2203	100	0
213	213	2203	100	0
214	214	2203	100	0
215	215	2203	100	0
216	216	2203	100	0
217	217	2203	100	0
218	218	2203	100	0
219	219	2203	100	0
220	220	2203	100	0
221	221	2203	100	0
222	222	2203	100	0
223	223	2203	100	0
224	224	2203	100	0
225	225	2203	100	0
226	226	2203	100	0
227	227	2203	100	0
228	228	2203	100	0
229	229	2203	100	0
230	230	2203	100	0
231	231	2203	100	0
232	232	2203	100	0
233	233	2203	100	0
234	234	2203	100	0
235	235	2203	100	0
236	236	2203	100	0
237	237	2203	100	0
238	238	2203	100	0
239	239	2203	100	0
240	240	2203	100	0
\.


--
-- Data for Name: nd_experimentprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_experimentprop (nd_experimentprop_id, nd_experiment_id, type_id, value, rank) FROM stdin;
1	1	2196	06/03/2009	0
2	1	2187	06/10/2009	0
3	1	2188	06/14/2009	0
4	1	2189	06/25/2009	0
5	1	2190	07/24/2009	0
6	1	2191	07/24/2009	0
7	1	2192	07/24/2009	0
8	1	2193	07/24/2009	0
9	1	2194	08/18/2009	0
10	1	2195	09/21/2009	0
11	1	2186		0
12	1	2299		0
13	1	2166	11/18/2009	0
14	1	2299		1
15	1	2299		2
16	1	2299		3
17	2	2196	06/03/2009	0
18	2	2187		0
19	2	2188		0
20	2	2189		0
21	2	2190		0
22	2	2191		0
23	2	2192		0
24	2	2193		0
25	2	2194	.	0
26	2	2195	.	0
27	2	2186	06/25/2009	0
28	2	2299		0
29	2	2166	11/18/2009	0
30	2	2299		1
31	2	2299		2
32	2	2299		3
33	3	2196	06/03/2009	0
34	3	2187	06/06/2009	0
35	3	2188	06/10/2009	0
36	3	2189	06/14/2009	0
37	3	2190	07/24/2009	0
38	3	2191	07/24/2009	0
39	3	2192	08/07/2009	0
40	3	2193	08/07/2009	0
41	3	2194	07/24/2009	0
42	3	2195	09/22/2009	0
43	3	2186		0
44	3	2299		0
45	3	2166	11/10/2009	0
46	3	2299		1
47	3	2299		2
48	3	2299		3
49	4	2196	06/06/2009	0
50	4	2187	06/10/2009	0
51	4	2188	06/25/2009	0
52	4	2189	06/25/2009	0
53	4	2190	07/24/2009	0
54	4	2191	07/11/2009	0
55	4	2192	07/24/2009	0
56	4	2193	07/24/2009	0
57	4	2194	08/07/2009	0
58	4	2195	09/03/2009	0
59	4	2186		0
60	4	2299		0
61	4	2166	11/10/2009	0
62	4	2299		1
63	4	2299		2
64	4	2299		3
65	5	2196	05/27/2009	0
66	5	2187	06/04/2009	0
67	5	2188	06/10/2009	0
68	5	2189	06/14/2009	0
69	5	2190	07/11/2009	0
70	5	2191	07/11/2009	0
71	5	2192	07/24/2009	0
72	5	2193	07/24/2009	0
73	5	2194	08/18/2009	0
74	5	2195	09/30/2009	0
75	5	2186		0
76	5	2299		0
77	5	2166	11/18/2009	0
78	5	2299		1
79	5	2299		2
80	5	2299		3
81	6	2196	05/27/2009	0
82	6	2187	06/06/2009	0
83	6	2188	06/10/2009	0
84	6	2189	06/25/2009	0
85	6	2190	06/25/2009	0
86	6	2191	07/24/2009	0
87	6	2192	07/24/2009	0
88	6	2193	07/24/2009	0
89	6	2194	08/07/2009	0
90	6	2195	09/30/2009	0
91	6	2186		0
92	6	2299		0
93	6	2166	11/18/2009	0
94	6	2299		1
95	6	2299		2
96	6	2299		3
97	7	2196	05/27/2009	0
98	7	2187	06/02/2009	0
99	7	2188	06/06/2009	0
100	7	2189	06/10/2009	0
101	7	2190	07/11/2009	0
102	7	2191	07/24/2009	0
103	7	2192	07/24/2009	0
104	7	2193	07/24/2009	0
105	7	2194	07/22/2009	0
106	7	2195	08/22/2009	0
107	7	2186		0
108	7	2299		0
109	7	2166	11/10/2009	0
110	7	2299		1
111	7	2299		2
112	7	2299		3
113	8	2196	05/27/2009	0
114	8	2187	06/04/2009	0
115	8	2188	06/10/2009	0
116	8	2189	06/14/2009	0
117	8	2190	06/25/2009	0
118	8	2191	07/11/2009	0
119	8	2192	07/11/2009	0
120	8	2193	07/11/2009	0
121	8	2194	07/24/2009	0
122	8	2195	09/12/2009	0
123	8	2186		0
124	8	2299		0
125	8	2166	11/18/2009	0
126	8	2299		1
127	8	2299		2
128	8	2299		3
129	9	2196	05/26/2009	0
130	9	2187	06/04/2009	0
131	9	2188	06/10/2009	0
132	9	2189	06/14/2009	0
133	9	2190	07/24/2009	0
134	9	2191	07/11/2009	0
135	9	2192	07/24/2009	0
136	9	2193	07/24/2009	0
137	9	2194	08/18/2009	0
138	9	2195	09/12/2009	0
139	9	2186		0
140	9	2299		0
141	9	2166	11/18/2009	0
142	9	2299	3 live stems, mostly dead	1
143	9	2299		2
144	9	2299		3
145	10	2196	05/27/2009	0
146	10	2187	06/04/2009	0
147	10	2188	06/06/2009	0
148	10	2189	06/10/2009	0
149	10	2190	07/11/2009	0
150	10	2191	06/25/2009	0
151	10	2192	07/11/2009	0
152	10	2193	07/11/2009	0
153	10	2194	08/07/2009	0
154	10	2195	09/18/2009	0
155	10	2186		0
156	10	2299		0
157	10	2166	11/10/2009	0
158	10	2299		1
159	10	2299		2
160	10	2299		3
161	11	2196	05/27/2009	0
162	11	2187	06/04/2009	0
163	11	2188	06/10/2009	0
164	11	2189	06/14/2009	0
165	11	2190	07/11/2009	0
166	11	2191	07/11/2009	0
167	11	2192	07/11/2009	0
168	11	2193	07/24/2009	0
169	11	2194	07/24/2009	0
170	11	2195	09/12/2009	0
171	11	2186		0
172	11	2299		0
173	11	2166	11/18/2009	0
174	11	2299		1
175	11	2299		2
176	11	2299		3
177	12	2196	05/27/2009	0
178	12	2187	06/04/2009	0
179	12	2188	06/10/2009	0
180	12	2189	06/14/2009	0
181	12	2190	07/11/2009	0
182	12	2191	07/11/2009	0
183	12	2192	07/24/2009	0
184	12	2193	07/24/2009	0
185	12	2194	08/07/2009	0
186	12	2195	09/30/2009	0
187	12	2186		0
188	12	2299		0
189	12	2166	11/18/2009	0
190	12	2299		1
191	12	2299		2
192	12	2299		3
193	13	2196	05/27/2009	0
194	13	2187	06/04/2009	0
195	13	2188	06/10/2009	0
196	13	2189	06/25/2009	0
197	13	2190	06/14/2009	0
198	13	2191	06/14/2009	0
199	13	2192	08/07/2009	0
200	13	2193	08/07/2009	0
201	13	2194	.	0
202	13	2195	.	0
203	13	2186	06/25/2009	0
204	13	2299		0
205	13	2166	11/10/2009	0
206	13	2299		1
207	13	2299		2
208	13	2299	.	3
209	14	2196	05/27/2009	0
210	14	2187	06/04/2009	0
211	14	2188	06/06/2009	0
212	14	2189	06/10/2009	0
213	14	2190	07/11/2009	0
214	14	2191	07/11/2009	0
215	14	2192	07/24/2009	0
216	14	2193	07/24/2009	0
217	14	2194	08/18/2009	0
218	14	2195	09/22/2009	0
219	14	2186		0
220	14	2299		0
221	14	2166	11/10/2009	0
222	14	2299		1
223	14	2299		2
224	14	2299		3
225	15	2196	05/27/2009	0
226	15	2187	06/04/2009	0
227	15	2188	06/10/2009	0
228	15	2189	06/14/2009	0
229	15	2190	07/11/2009	0
230	15	2191	07/24/2009	0
231	15	2192	07/24/2009	0
232	15	2193	08/07/2009	0
233	15	2194	08/07/2009	0
234	15	2195	09/22/2009	0
235	15	2186		0
236	15	2299		0
237	15	2166	11/10/2009	0
238	15	2299		1
239	15	2299		2
240	15	2299		3
241	16	2196	05/26/2009	0
242	16	2187	06/02/2009	0
243	16	2188	06/06/2009	0
244	16	2189	06/10/2009	0
245	16	2190	07/11/2009	0
246	16	2191	07/24/2009	0
247	16	2192	07/24/2009	0
248	16	2193	07/24/2009	0
249	16	2194	08/07/2009	0
250	16	2195	09/30/2009	0
251	16	2186		0
252	16	2299		0
253	16	2166	11/18/2009	0
254	16	2299		1
255	16	2299		2
256	16	2299		3
257	17	2196	06/03/2009	0
258	17	2187	06/10/2009	0
259	17	2188	06/14/2009	0
260	17	2189	06/25/2009	0
261	17	2190	07/24/2009	0
262	17	2191	07/24/2009	0
263	17	2192	07/24/2009	0
264	17	2193	08/07/2009	0
265	17	2194	08/07/2009	0
266	17	2195	09/16/2009	0
267	17	2186		0
268	17	2299		0
269	17	2166	11/10/2009	0
270	17	2299		1
271	17	2299		2
272	17	2299		3
273	18	2196	05/27/2009	0
274	18	2187	06/06/2009	0
275	18	2188	06/10/2009	0
276	18	2189	06/25/2009	0
277	18	2190	07/11/2009	0
278	18	2191	07/11/2009	0
279	18	2192	08/07/2009	0
280	18	2193	07/24/2009	0
281	18	2194	08/07/2009	0
282	18	2195	09/29/2009	0
283	18	2186		0
284	18	2299		0
285	18	2166	11/10/2009	0
286	18	2299		1
287	18	2299		2
288	18	2299		3
289	19	2196	06/03/2009	0
290	19	2187	06/06/2009	0
291	19	2188	06/10/2009	0
292	19	2189	06/25/2009	0
293	19	2190	06/25/2009	0
294	19	2191	07/03/2009	0
295	19	2192	07/24/2009	0
296	19	2193	08/07/2009	0
297	19	2194	08/18/2009	0
298	19	2195	09/12/2009	0
299	19	2186		0
300	19	2299		0
301	19	2166	11/10/2009	0
302	19	2299	Roots have degraded	1
303	19	2299		2
304	19	2299	dead plant no roots!	3
305	20	2196	05/27/2009	0
306	20	2187	06/04/2009	0
307	20	2188	06/10/2009	0
308	20	2189	06/10/2009	0
309	20	2190	07/11/2009	0
310	20	2191	06/14/2009	0
311	20	2192	07/24/2009	0
312	20	2193	07/24/2009	0
313	20	2194	07/22/2009	0
314	20	2195	09/12/2009	0
315	20	2186		0
316	20	2299		0
317	20	2166	11/18/2009	0
318	20	2299		1
319	20	2299		2
320	20	2299		3
321	21	2196	06/02/2009	0
322	21	2187	06/10/2009	0
323	21	2188	06/14/2009	0
324	21	2189	06/25/2009	0
325	21	2190	07/24/2009	0
326	21	2191	07/24/2009	0
327	21	2192	07/24/2009	0
328	21	2193	07/24/2009	0
329	21	2194	08/07/2009	0
330	21	2195		0
331	21	2186		0
332	21	2299		0
333	21	2166	11/10/2009	0
334	21	2299	. Leaves not estimated	1
335	21	2299		2
336	21	2299		3
337	22	2196	06/10/2009	0
338	22	2187		0
339	22	2188		0
340	22	2189		0
341	22	2190	07/24/2009	0
342	22	2191		0
343	22	2192		0
344	22	2193		0
345	22	2194	.	0
346	22	2195	.	0
347	22	2186	07/24/2009	0
348	22	2299		0
349	22	2166	11/10/2009	0
350	22	2299		1
351	22	2299		2
352	22	2299		3
353	23	2196	06/03/2009	0
354	23	2187	06/10/2009	0
355	23	2188	06/14/2009	0
356	23	2189	06/25/2009	0
357	23	2190	06/14/2009	0
358	23	2191	07/24/2009	0
359	23	2192	07/11/2009	0
360	23	2193	07/24/2009	0
361	23	2194	08/18/2009	0
362	23	2195	09/21/2009	0
363	23	2186		0
364	23	2299		0
365	23	2166	11/18/2009	0
366	23	2299		1
367	23	2299		2
368	23	2299		3
369	24	2196	.	0
370	24	2187	06/06/2009	0
371	24	2188	06/14/2009	0
372	24	2189	06/25/2009	0
373	24	2190	07/24/2009	0
374	24	2191	07/24/2009	0
375	24	2192	07/24/2009	0
376	24	2193	07/24/2009	0
377	24	2194	08/07/2009	0
378	24	2195	09/21/2009	0
379	24	2186	09/30/2009	0
380	24	2299		0
381	24	2166	11/18/2009	0
382	24	2299		1
383	24	2299		2
384	24	2299	dead plant harvested 10/4/09	3
385	25	2196	06/02/2009	0
386	25	2187	06/04/2009	0
387	25	2188	06/10/2009	0
388	25	2189	06/14/2009	0
389	25	2190	07/11/2009	0
390	25	2191	08/07/2009	0
391	25	2192	08/18/2009	0
392	25	2193	08/18/2009	0
393	25	2194	07/24/2009	0
394	25	2195	08/22/2009	0
395	25	2186		0
396	25	2299		0
397	25	2166	11/18/2009	0
398	25	2299		1
399	25	2299		2
400	25	2299		3
401	26	2196	05/28/2009	0
402	26	2187	06/06/2009	0
403	26	2188	06/10/2009	0
404	26	2189	06/14/2009	0
405	26	2190	06/25/2009	0
406	26	2191	07/24/2009	0
407	26	2192	07/24/2009	0
408	26	2193	07/24/2009	0
409	26	2194	08/07/2009	0
410	26	2195	08/22/2009	0
411	26	2186		0
412	26	2299		0
413	26	2166	11/10/2009	0
414	26	2299		1
415	26	2299		2
416	26	2299		3
417	27	2196	06/02/2009	0
418	27	2187	06/06/2009	0
419	27	2188	06/14/2009	0
420	27	2189	06/25/2009	0
421	27	2190	07/11/2009	0
422	27	2191	07/11/2009	0
423	27	2192	07/24/2009	0
424	27	2193	07/24/2009	0
425	27	2194	08/07/2009	0
426	27	2195	09/12/2009	0
427	27	2186		0
428	27	2299		0
429	27	2166	11/10/2009	0
430	27	2299		1
431	27	2299		2
432	27	2299		3
433	28	2196	05/30/2009	0
434	28	2187	06/04/2009	0
435	28	2188	06/10/2009	0
436	28	2189	06/14/2009	0
437	28	2190	06/25/2009	0
438	28	2191	06/25/2009	0
439	28	2192	07/24/2009	0
440	28	2193	07/24/2009	0
441	28	2194	08/07/2009	0
442	28	2195	09/22/2009	0
443	28	2186		0
444	28	2299		0
445	28	2166	11/10/2009	0
446	28	2299	. Diameters not measured	1
447	28	2299		2
448	28	2299		3
449	29	2196	05/28/2009	0
450	29	2187	06/04/2009	0
451	29	2188	06/10/2009	0
452	29	2189	06/14/2009	0
453	29	2190	07/11/2009	0
454	29	2191	08/07/2009	0
455	29	2192	07/24/2009	0
456	29	2193	07/24/2009	0
457	29	2194	08/07/2009	0
458	29	2195	09/12/2009	0
459	29	2186		0
460	29	2299		0
461	29	2166	11/18/2009	0
462	29	2299		1
463	29	2299		2
464	29	2299		3
465	30	2196	05/28/2009	0
466	30	2187	06/04/2009	0
467	30	2188	06/10/2009	0
468	30	2189	06/14/2009	0
469	30	2190	07/11/2009	0
470	30	2191	07/03/2009	0
471	30	2192	07/24/2009	0
472	30	2193	07/24/2009	0
473	30	2194	08/07/2009	0
474	30	2195	09/26/2009	0
475	30	2186		0
476	30	2299		0
477	30	2166	11/10/2009	0
478	30	2299	Moldy (no estimated % damage)	1
479	30	2299		2
480	30	2299		3
481	31	2196	.	0
482	31	2187	06/10/2009	0
483	31	2188	06/14/2009	0
484	31	2189	06/25/2009	0
485	31	2190	06/25/2009	0
486	31	2191	07/24/2009	0
487	31	2192	07/24/2009	0
488	31	2193	07/24/2009	0
489	31	2194	08/07/2009	0
490	31	2195	09/29/2009	0
491	31	2186		0
492	31	2299		0
493	31	2166	11/10/2009	0
494	31	2299		1
495	31	2299		2
496	31	2299		3
497	32	2196	05/28/2009	0
498	32	2187	06/06/2009	0
499	32	2188	06/10/2009	0
500	32	2189	06/14/2009	0
501	32	2190	07/11/2009	0
502	32	2191	07/24/2009	0
503	32	2192	07/24/2009	0
504	32	2193	07/24/2009	0
505	32	2194	07/22/2009	0
506	32	2195	10/09/2009	0
507	32	2186		0
508	32	2299		0
509	32	2166	11/10/2009	0
510	32	2299		1
511	32	2299		2
512	32	2299		3
513	33	2196	05/28/2009	0
514	33	2187	06/10/2009	0
515	33	2188	06/10/2009	0
516	33	2189	06/14/2009	0
517	33	2190	07/24/2009	0
518	33	2191	07/11/2009	0
519	33	2192	07/24/2009	0
520	33	2193	08/07/2009	0
521	33	2194	08/07/2009	0
522	33	2195	09/21/2009	0
523	33	2186		0
524	33	2299		0
525	33	2166	11/18/2009	0
526	33	2299		1
527	33	2299		2
528	33	2299		3
529	34	2196	05/28/2009	0
530	34	2187	06/06/2009	0
531	34	2188	06/10/2009	0
532	34	2189	06/25/2009	0
533	34	2190	07/11/2009	0
534	34	2191	07/24/2009	0
535	34	2192	07/24/2009	0
536	34	2193	07/24/2009	0
537	34	2194	08/07/2009	0
538	34	2195		0
539	34	2186		0
540	34	2299		0
541	34	2166	11/10/2009	0
542	34	2299		1
543	34	2299		2
544	34	2299		3
545	35	2196	05/28/2009	0
546	35	2187	06/04/2009	0
547	35	2188	06/10/2009	0
548	35	2189	06/25/2009	0
549	35	2190	07/11/2009	0
550	35	2191	06/14/2009	0
551	35	2192	06/25/2009	0
552	35	2193	07/24/2009	0
553	35	2194	08/07/2009	0
554	35	2195	09/30/2009	0
555	35	2186		0
556	35	2299		0
557	35	2166	11/18/2009	0
558	35	2299		1
559	35	2299		2
560	35	2299		3
561	36	2196	05/28/2009	0
562	36	2187	06/04/2009	0
563	36	2188	06/10/2009	0
564	36	2189	06/10/2009	0
565	36	2190	07/11/2009	0
566	36	2191	07/11/2009	0
567	36	2192	07/24/2009	0
568	36	2193	07/24/2009	0
569	36	2194	07/22/2009	0
570	36	2195	08/16/2009	0
571	36	2186		0
572	36	2299		0
573	36	2166	11/18/2009	0
574	36	2299		1
575	36	2299		2
576	36	2299		3
577	37	2196	06/03/2009	0
578	37	2187	06/06/2009	0
579	37	2188	06/10/2009	0
580	37	2189	06/25/2009	0
581	37	2190	07/11/2009	0
582	37	2191	07/03/2009	0
583	37	2192	08/07/2009	0
584	37	2193	07/24/2009	0
585	37	2194	08/07/2009	0
586	37	2195	09/12/2009	0
587	37	2186		0
588	37	2299		0
589	37	2166	11/10/2009	0
590	37	2299		1
591	37	2299		2
592	37	2299		3
593	38	2196	06/03/2009	0
594	38	2187	.	0
595	38	2188	06/25/2009	0
596	38	2189	07/11/2009	0
597	38	2190	07/11/2009	0
598	38	2191	.	0
599	38	2192	07/24/2009	0
600	38	2193	07/24/2009	0
601	38	2194	.	0
602	38	2195	.	0
603	38	2186	07/24/2009	0
604	38	2299	NO ROOTS TINY PLANT (6/24)	0
605	38	2166	11/18/2009	0
606	38	2299		1
607	38	2299		2
608	38	2299		3
609	39	2196	05/28/2009	0
610	39	2187	06/06/2009	0
611	39	2188	06/10/2009	0
612	39	2189	06/25/2009	0
613	39	2190	07/11/2009	0
614	39	2191	07/24/2009	0
615	39	2192	07/24/2009	0
616	39	2193	07/24/2009	0
617	39	2194	08/07/2009	0
618	39	2195	09/21/2009	0
619	39	2186		0
620	39	2299		0
621	39	2166	11/18/2009	0
622	39	2299		1
623	39	2299		2
624	39	2299		3
625	40	2196	05/28/2009	0
626	40	2187	06/10/2009	0
627	40	2188	06/25/2009	0
628	40	2189	06/25/2009	0
629	40	2190	06/10/2009	0
630	40	2191	07/24/2009	0
631	40	2192	07/24/2009	0
632	40	2193	07/24/2009	0
633	40	2194	08/18/2009	0
634	40	2195	10/26/2009	0
635	40	2186		0
636	40	2299		0
637	40	2166	11/18/2009	0
638	40	2299		1
639	40	2299		2
640	40	2299		3
641	41	2196	06/02/2009	0
642	41	2187	06/10/2009	0
643	41	2188	06/14/2009	0
644	41	2189	06/25/2009	0
645	41	2190	07/11/2009	0
646	41	2191	07/24/2009	0
647	41	2192	07/24/2009	0
648	41	2193	07/24/2009	0
649	41	2194	08/07/2009	0
650	41	2195	09/22/2009	0
651	41	2186		0
652	41	2299		0
653	41	2166	11/12/2009	0
654	41	2299	. Diameters not measured	1
655	41	2299		2
656	41	2299		3
657	42	2196	05/27/2009	0
658	42	2187	06/06/2009	0
659	42	2188	06/10/2009	0
660	42	2189	06/14/2009	0
661	42	2190	07/11/2009	0
662	42	2191	07/03/2009	0
663	42	2192	07/24/2009	0
664	42	2193	07/24/2009	0
665	42	2194	08/25/2009	0
666	42	2195	09/29/2009	0
667	42	2186		0
668	42	2299		0
669	42	2166	11/12/2009	0
670	42	2299		1
671	42	2299		2
672	42	2299		3
673	43	2196	05/27/2009	0
674	43	2187	06/02/2009	0
675	43	2188	06/10/2009	0
676	43	2189	06/10/2009	0
677	43	2190	07/11/2009	0
678	43	2191	06/25/2009	0
679	43	2192	07/24/2009	0
680	43	2193	07/24/2009	0
681	43	2194	08/07/2009	0
682	43	2195	09/26/2009	0
683	43	2186		0
684	43	2299		0
685	43	2166	11/12/2009	0
686	43	2299	Moldy (no estimated % damage)	1
687	43	2299		2
688	43	2299		3
689	44	2196	05/26/2009	0
690	44	2187	06/04/2009	0
691	44	2188	06/10/2009	0
692	44	2189	06/14/2009	0
693	44	2190	06/25/2009	0
694	44	2191	06/25/2009	0
695	44	2192	07/24/2009	0
696	44	2193	07/24/2009	0
697	44	2194	07/24/2009	0
698	44	2195	09/12/2009	0
699	44	2186		0
700	44	2299		0
701	44	2166	11/12/2009	0
702	44	2299	6 dead stems	1
703	44	2299		2
704	44	2299		3
705	45	2196	06/02/2009	0
706	45	2187	06/10/2009	0
707	45	2188	06/14/2009	0
708	45	2189	06/25/2009	0
709	45	2190	07/11/2009	0
710	45	2191	07/03/2009	0
711	45	2192	07/24/2009	0
712	45	2193	07/24/2009	0
713	45	2194	08/07/2009	0
714	45	2195	09/22/2009	0
715	45	2186		0
716	45	2299		0
717	45	2166	11/12/2009	0
718	45	2299		1
719	45	2299		2
720	45	2299		3
721	46	2196	05/26/2009	0
722	46	2187	06/10/2009	0
723	46	2188	06/10/2009	0
724	46	2189	06/25/2009	0
725	46	2190	07/24/2009	0
726	46	2191	07/24/2009	0
727	46	2192	08/07/2009	0
728	46	2193	07/24/2009	0
729	46	2194	08/07/2009	0
730	46	2195	09/16/2009	0
731	46	2186		0
732	46	2299		0
733	46	2166	11/13/2009	0
734	46	2299		1
735	46	2299		2
736	46	2299		3
737	47	2196	05/26/2009	0
738	47	2187	06/02/2009	0
739	47	2188	06/10/2009	0
740	47	2189	06/14/2009	0
741	47	2190	07/11/2009	0
742	47	2191	07/11/2009	0
743	47	2192	07/24/2009	0
744	47	2193	07/24/2009	0
745	47	2194	07/22/2009	0
746	47	2195	09/22/2009	0
747	47	2186		0
748	47	2299		0
749	47	2166	11/13/2009	0
750	47	2299		1
751	47	2299		2
752	47	2299		3
753	48	2196	05/26/2009	0
754	48	2187	06/04/2009	0
755	48	2188	06/10/2009	0
756	48	2189	06/14/2009	0
757	48	2190	07/11/2009	0
758	48	2191	07/24/2009	0
759	48	2192	07/11/2009	0
760	48	2193	07/24/2009	0
761	48	2194	07/24/2009	0
762	48	2195	09/22/2009	0
763	48	2186		0
764	48	2299		0
765	48	2166	11/13/2009	0
766	48	2299		1
767	48	2299		2
768	48	2299		3
769	49	2196	05/27/2009	0
770	49	2187	06/04/2009	0
771	49	2188	06/10/2009	0
772	49	2189	06/14/2009	0
773	49	2190	06/25/2009	0
774	49	2191	06/25/2009	0
775	49	2192	07/24/2009	0
776	49	2193	07/24/2009	0
777	49	2194	07/24/2009	0
778	49	2195	09/18/2009	0
779	49	2186		0
780	49	2299		0
781	49	2166	11/13/2009	0
782	49	2299		1
783	49	2299		2
784	49	2299		3
785	50	2196	05/26/2009	0
786	50	2187	06/06/2009	0
787	50	2188	.	0
788	50	2189	.	0
789	50	2190	06/14/2009	0
790	50	2191	06/14/2009	0
791	50	2192	.	0
792	50	2193	.	0
793	50	2194	.	0
794	50	2195	.	0
795	50	2186	06/14/2009	0
796	50	2299		0
797	50	2166	11/12/2009	0
798	50	2299		1
799	50	2299		2
800	50	2299		3
801	51	2196	05/26/2009	0
802	51	2187	06/04/2009	0
803	51	2188	06/10/2009	0
804	51	2189	06/14/2009	0
805	51	2190	07/24/2009	0
806	51	2191	07/03/2009	0
807	51	2192	07/24/2009	0
808	51	2193	07/24/2009	0
809	51	2194	08/07/2009	0
810	51	2195	09/12/2009	0
811	51	2186		0
812	51	2299		0
813	51	2166	11/13/2009	0
814	51	2299		1
815	51	2299		2
816	51	2299		3
817	52	2196	05/26/2009	0
818	52	2187	06/04/2009	0
819	52	2188	06/10/2009	0
820	52	2189	06/14/2009	0
821	52	2190	07/11/2009	0
822	52	2191	06/14/2009	0
823	52	2192	07/24/2009	0
824	52	2193	07/24/2009	0
825	52	2194	07/24/2009	0
826	52	2195	09/03/2009	0
827	52	2186		0
828	52	2299		0
829	52	2166	11/12/2009	0
830	52	2299		1
831	52	2299		2
832	52	2299		3
833	53	2196	05/26/2009	0
834	53	2187	06/04/2009	0
835	53	2188	06/10/2009	0
836	53	2189	06/14/2009	0
837	53	2190	07/11/2009	0
838	53	2191	07/03/2009	0
839	53	2192	08/18/2009	0
840	53	2193	08/18/2009	0
841	53	2194	07/22/2009	0
842	53	2195	09/08/2009	0
843	53	2186	09/30/2009	0
844	53	2299		0
845	53	2166	11/13/2009	0
846	53	2299	.	1
847	53	2299		2
848	53	2299	dead plant harvested 10/4/09	3
849	54	2196	05/27/2009	0
850	54	2187	06/04/2009	0
851	54	2188	06/06/2009	0
852	54	2189	06/14/2009	0
853	54	2190	07/24/2009	0
854	54	2191	06/25/2009	0
855	54	2192	07/11/2009	0
856	54	2193	07/11/2009	0
857	54	2194	08/07/2009	0
858	54	2195	09/22/2009	0
859	54	2186		0
860	54	2299		0
861	54	2166	11/13/2009	0
862	54	2299		1
863	54	2299		2
864	54	2299		3
865	55	2196	05/26/2009	0
866	55	2187	06/04/2009	0
867	55	2188	06/06/2009	0
868	55	2189	06/14/2009	0
869	55	2190	07/11/2009	0
870	55	2191	06/14/2009	0
871	55	2192	07/24/2009	0
872	55	2193	07/24/2009	0
873	55	2194	08/07/2009	0
874	55	2195	10/09/2009	0
875	55	2186		0
876	55	2299		0
877	55	2166	11/13/2009	0
878	55	2299		1
879	55	2299		2
880	55	2299		3
881	56	2196	05/26/2009	0
882	56	2187	06/02/2009	0
883	56	2188	06/10/2009	0
884	56	2189	06/14/2009	0
885	56	2190	06/25/2009	0
886	56	2191	07/03/2009	0
887	56	2192	07/24/2009	0
888	56	2193	07/24/2009	0
889	56	2194	07/22/2009	0
890	56	2195	09/12/2009	0
891	56	2186		0
892	56	2299		0
893	56	2166	11/12/2009	0
894	56	2299		1
895	56	2299		2
896	56	2299		3
897	57	2196	05/26/2009	0
898	57	2187	06/04/2009	0
899	57	2188	06/10/2009	0
900	57	2189	06/14/2009	0
901	57	2190	07/11/2009	0
902	57	2191	06/25/2009	0
903	57	2192	07/11/2009	0
904	57	2193	07/11/2009	0
905	57	2194	07/22/2009	0
906	57	2195	09/01/2009	0
907	57	2186		0
908	57	2299		0
909	57	2166	11/13/2009	0
910	57	2299		1
911	57	2299		2
912	57	2299		3
913	58	2196	05/26/2009	0
914	58	2187	06/06/2009	0
915	58	2188	06/10/2009	0
916	58	2189	06/25/2009	0
917	58	2190	06/25/2009	0
918	58	2191	07/24/2009	0
919	58	2192	07/24/2009	0
920	58	2193	08/07/2009	0
921	58	2194	07/24/2009	0
922	58	2195	09/12/2009	0
923	58	2186		0
924	58	2299		0
925	58	2166	11/13/2009	0
926	58	2299		1
927	58	2299		2
928	58	2299		3
929	59	2196	05/26/2009	0
930	59	2187	06/04/2009	0
931	59	2188	06/10/2009	0
932	59	2189	06/25/2009	0
933	59	2190	07/11/2009	0
934	59	2191	07/24/2009	0
935	59	2192	07/24/2009	0
936	59	2193	08/07/2009	0
937	59	2194	07/24/2009	0
938	59	2195	09/12/2009	0
939	59	2186		0
940	59	2299		0
941	59	2166	11/12/2009	0
942	59	2299		1
943	59	2299		2
944	59	2299		3
945	60	2196	05/26/2009	0
946	60	2187	06/06/2009	0
947	60	2188	06/14/2009	0
948	60	2189	06/25/2009	0
949	60	2190	07/11/2009	0
950	60	2191	07/24/2009	0
951	60	2192	07/24/2009	0
952	60	2193	07/24/2009	0
953	60	2194	07/24/2009	0
954	60	2195	09/22/2009	0
955	60	2186		0
956	60	2299		0
957	60	2166	11/12/2009	0
958	60	2299	. Diameters not measured, roots degraded	1
959	60	2299		2
960	60	2299	dead plant no roots!	3
961	61	2196	05/28/2009	0
962	61	2187	06/04/2009	0
963	61	2188	06/10/2009	0
964	61	2189	06/10/2009	0
965	61	2190	06/25/2009	0
966	61	2191	06/14/2009	0
967	61	2192	07/24/2009	0
968	61	2193	07/24/2009	0
969	61	2194	08/18/2009	0
970	61	2195	09/30/2009	0
971	61	2186		0
972	61	2299		0
973	61	2166	11/17/2009	0
974	61	2299		1
975	61	2299		2
976	61	2299		3
977	62	2196	06/03/2009	0
978	62	2187	.	0
979	62	2188	.	0
980	62	2189	.	0
981	62	2190	06/25/2009	0
982	62	2191	.	0
983	62	2192	.	0
984	62	2193	.	0
985	62	2194	.	0
986	62	2195	.	0
987	62	2186	07/24/2009	0
988	62	2299		0
989	62	2166	11/17/2009	0
990	62	2299		1
991	62	2299		2
992	62	2299		3
993	63	2196	05/26/2009	0
994	63	2187	06/04/2009	0
995	63	2188	06/06/2009	0
996	63	2189	06/14/2009	0
997	63	2190	06/14/2009	0
998	63	2191	07/03/2009	0
999	63	2192	07/24/2009	0
1000	63	2193	07/24/2009	0
1001	63	2194	.	0
1002	63	2195	.	0
1003	63	2186	07/24/2009	0
1004	63	2299		0
1005	63	2166	11/11/2009	0
1006	63	2299		1
1007	63	2299		2
1008	63	2299		3
1009	64	2196	05/30/2009	0
1010	64	2187	06/06/2009	0
1011	64	2188	06/10/2009	0
1012	64	2189	06/14/2009	0
1013	64	2190	06/25/2009	0
1014	64	2191	07/24/2009	0
1015	64	2192	07/24/2009	0
1016	64	2193	07/24/2009	0
1017	64	2194	08/07/2009	0
1018	64	2195	09/30/2009	0
1019	64	2186		0
1020	64	2299		0
1021	64	2166	11/17/2009	0
1022	64	2299	Moldy (no estimated % damage)	1
1023	64	2299		2
1024	64	2299		3
1025	65	2196	06/10/2009	0
1026	65	2187	06/14/2009	0
1027	65	2188	06/25/2009	0
1028	65	2189	06/25/2009	0
1029	65	2190	07/24/2009	0
1030	65	2191	07/24/2009	0
1031	65	2192	07/24/2009	0
1032	65	2193	07/24/2009	0
1033	65	2194	08/18/2009	0
1034	65	2195	09/25/2009	0
1035	65	2186		0
1036	65	2299		0
1037	65	2166	11/17/2009	0
1038	65	2299		1
1039	65	2299		2
1040	65	2299		3
1041	66	2196	05/26/2009	0
1042	66	2187	06/06/2009	0
1043	66	2188	06/06/2009	0
1044	66	2189	06/10/2009	0
1045	66	2190	07/11/2009	0
1046	66	2191	06/14/2009	0
1047	66	2192	07/24/2009	0
1048	66	2193	07/24/2009	0
1049	66	2194	07/24/2009	0
1050	66	2195	09/12/2009	0
1051	66	2186		0
1052	66	2299		0
1053	66	2166	11/11/2009	0
1054	66	2299	2 dead stems	1
1055	66	2299		2
1056	66	2299		3
1057	67	2196	05/26/2009	0
1058	67	2187	06/04/2009	0
1059	67	2188	06/06/2009	0
1060	67	2189	06/10/2009	0
1061	67	2190	07/11/2009	0
1062	67	2191	06/25/2009	0
1063	67	2192	07/11/2009	0
1064	67	2193	07/11/2009	0
1065	67	2194	07/22/2009	0
1066	67	2195	09/25/2009	0
1067	67	2186		0
1068	67	2299		0
1069	67	2166	11/17/2009	0
1070	67	2299		1
1071	67	2299		2
1072	67	2299		3
1073	68	2196	05/26/2009	0
1074	68	2187	06/02/2009	0
1075	68	2188	06/10/2009	0
1076	68	2189	06/14/2009	0
1077	68	2190	06/25/2009	0
1078	68	2191	07/24/2009	0
1079	68	2192	08/07/2009	0
1080	68	2193	08/07/2009	0
1081	68	2194	08/07/2009	0
1082	68	2195	09/18/2009	0
1083	68	2186		0
1084	68	2299		0
1085	68	2166	11/11/2009	0
1086	68	2299		1
1087	68	2299		2
1088	68	2299		3
1089	69	2196	05/27/2009	0
1090	69	2187	06/02/2009	0
1091	69	2188	06/25/2009	0
1092	69	2189	06/25/2009	0
1093	69	2190	06/25/2009	0
1094	69	2191	06/25/2009	0
1095	69	2192	07/11/2009	0
1096	69	2193	07/11/2009	0
1097	69	2194	.	0
1098	69	2195	.	0
1099	69	2186	07/11/2009	0
1100	69	2299		0
1101	69	2166	11/11/2009	0
1102	69	2299		1
1103	69	2299		2
1104	69	2299		3
1105	70	2196	05/27/2009	0
1106	70	2187	06/02/2009	0
1107	70	2188	06/06/2009	0
1108	70	2189	06/14/2009	0
1109	70	2190	06/25/2009	0
1110	70	2191	07/24/2009	0
1111	70	2192	08/07/2009	0
1112	70	2193	07/24/2009	0
1113	70	2194	08/18/2009	0
1114	70	2195	09/30/2009	0
1115	70	2186		0
1116	70	2299		0
1117	70	2166	11/17/2009	0
1118	70	2299		1
1119	70	2299		2
1120	70	2299		3
1121	71	2196	05/26/2009	0
1122	71	2187	06/04/2009	0
1123	71	2188	06/10/2009	0
1124	71	2189	06/10/2009	0
1125	71	2190	07/11/2009	0
1126	71	2191	07/11/2009	0
1127	71	2192	07/11/2009	0
1128	71	2193	07/24/2009	0
1129	71	2194	07/22/2009	0
1130	71	2195	09/21/2009	0
1131	71	2186		0
1132	71	2299		0
1133	71	2166	11/17/2009	0
1134	71	2299	. Diameters not measured	1
1135	71	2299		2
1136	71	2299	dead plant	3
1137	72	2196	06/06/2009	0
1138	72	2187	06/10/2009	0
1139	72	2188	06/14/2009	0
1140	72	2189	06/25/2009	0
1141	72	2190	07/24/2009	0
1142	72	2191	07/24/2009	0
1143	72	2192	08/07/2009	0
1144	72	2193	08/07/2009	0
1145	72	2194	08/07/2009	0
1146	72	2195	09/21/2009	0
1147	72	2186		0
1148	72	2299		0
1149	72	2166	11/17/2009	0
1150	72	2299		1
1151	72	2299		2
1152	72	2299		3
1153	73	2196	05/26/2009	0
1154	73	2187	06/04/2009	0
1155	73	2188	06/06/2009	0
1156	73	2189	06/10/2009	0
1157	73	2190	07/11/2009	0
1158	73	2191	06/25/2009	0
1159	73	2192	07/24/2009	0
1160	73	2193	07/24/2009	0
1161	73	2194	07/22/2009	0
1162	73	2195	09/22/2009	0
1163	73	2186		0
1164	73	2299		0
1165	73	2166	11/11/2009	0
1166	73	2299		1
1167	73	2299		2
1168	73	2299		3
1169	74	2196	05/26/2009	0
1170	74	2187	06/02/2009	0
1171	74	2188	06/10/2009	0
1172	74	2189	06/14/2009	0
1173	74	2190	07/11/2009	0
1174	74	2191	07/03/2009	0
1175	74	2192	08/07/2009	0
1176	74	2193	08/07/2009	0
1177	74	2194	08/18/2009	0
1178	74	2195	09/26/2009	0
1179	74	2186		0
1180	74	2299		0
1181	74	2166	11/11/2009	0
1182	74	2299	2 dead stems	1
1183	74	2299		2
1184	74	2299		3
1185	75	2196	05/26/2009	0
1186	75	2187	06/04/2009	0
1187	75	2188	06/10/2009	0
1188	75	2189	06/14/2009	0
1189	75	2190	07/11/2009	0
1190	75	2191	06/14/2009	0
1191	75	2192	07/24/2009	0
1192	75	2193	07/24/2009	0
1193	75	2194	08/18/2009	0
1194	75	2195	09/22/2009	0
1195	75	2186		0
1196	75	2299		0
1197	75	2166	11/11/2009	0
1198	75	2299	No root outside pot	1
1199	75	2299		2
1200	75	2299		3
1201	76	2196	05/26/2009	0
1202	76	2187	06/04/2009	0
1203	76	2188	06/10/2009	0
1204	76	2189	06/14/2009	0
1205	76	2190	07/11/2009	0
1206	76	2191	07/11/2009	0
1207	76	2192	07/24/2009	0
1208	76	2193	07/24/2009	0
1209	76	2194	07/24/2009	0
1210	76	2195	09/12/2009	0
1211	76	2186		0
1212	76	2299		0
1213	76	2166	11/11/2009	0
1214	76	2299	3 dead stems	1
1215	76	2299		2
1216	76	2299		3
1217	77	2196	05/26/2009	0
1218	77	2187	06/02/2009	0
1219	77	2188	06/10/2009	0
1220	77	2189	06/14/2009	0
1221	77	2190	07/11/2009	0
1222	77	2191	06/14/2009	0
1223	77	2192	07/24/2009	0
1224	77	2193	07/24/2009	0
1225	77	2194	07/24/2009	0
1226	77	2195	09/21/2009	0
1227	77	2186		0
1228	77	2299		0
1229	77	2166	11/17/2009	0
1230	77	2299	. Diameters not measured	1
1231	77	2299		2
1232	77	2299	dead plant	3
1233	78	2196	05/28/2009	0
1234	78	2187		0
1235	78	2188		0
1236	78	2189		0
1237	78	2190	06/25/2009	0
1238	78	2191		0
1239	78	2192		0
1240	78	2193		0
1241	78	2194	.	0
1242	78	2195	.	0
1243	78	2186	06/25/2009	0
1244	78	2299		0
1245	78	2166	11/11/2009	0
1246	78	2299		1
1247	78	2299		2
1248	78	2299		3
1249	79	2196	05/26/2009	0
1250	79	2187	06/04/2009	0
1251	79	2188	06/10/2009	0
1252	79	2189	06/14/2009	0
1253	79	2190	07/11/2009	0
1254	79	2191	07/11/2009	0
1255	79	2192	07/24/2009	0
1256	79	2193	07/24/2009	0
1257	79	2194	08/07/2009	0
1258	79	2195	09/12/2009	0
1259	79	2186		0
1260	79	2299		0
1261	79	2166	11/11/2009	0
1262	79	2299	6 dead stems	1
1263	79	2299		2
1264	79	2299		3
1265	80	2196	05/27/2009	0
1266	80	2187	06/04/2009	0
1267	80	2188	06/10/2009	0
1268	80	2189	06/14/2009	0
1269	80	2190	07/24/2009	0
1270	80	2191	07/24/2009	0
1271	80	2192	07/24/2009	0
1272	80	2193	07/24/2009	0
1273	80	2194	08/18/2009	0
1274	80	2195	09/21/2009	0
1275	80	2186		0
1276	80	2299		0
1277	80	2166	11/17/2009	0
1278	80	2299		1
1279	80	2299		2
1280	80	2299		3
1281	81	2196	06/03/2009	0
1282	81	2187	06/10/2009	0
1283	81	2188	06/14/2009	0
1284	81	2189	06/25/2009	0
1285	81	2190	07/11/2009	0
1286	81	2191	07/24/2009	0
1287	81	2192	07/24/2009	0
1288	81	2193	07/24/2009	0
1289	81	2194	09/08/2009	0
1290	81	2195	10/22/2009	0
1291	81	2186		0
1292	81	2299		0
1293	81	2166	11/11/2009	0
1294	81	2299		1
1295	81	2299		2
1296	81	2299		3
1297	82	2196	05/27/2009	0
1298	82	2187	06/02/2009	0
1299	82	2188	06/06/2009	0
1300	82	2189	06/10/2009	0
1301	82	2190	07/24/2009	0
1302	82	2191	07/24/2009	0
1303	82	2192	07/24/2009	0
1304	82	2193	07/11/2009	0
1305	82	2194	08/25/2009	0
1306	82	2195	10/09/2009	0
1307	82	2186		0
1308	82	2299		0
1309	82	2166	11/11/2009	0
1310	82	2299		1
1311	82	2299		2
1312	82	2299		3
1313	83	2196	05/26/2009	0
1314	83	2187	06/02/2009	0
1315	83	2188	06/06/2009	0
1316	83	2189	06/10/2009	0
1317	83	2190	07/11/2009	0
1318	83	2191	07/11/2009	0
1319	83	2192	07/24/2009	0
1320	83	2193	07/24/2009	0
1321	83	2194	07/24/2009	0
1322	83	2195	09/25/2009	0
1323	83	2186		0
1324	83	2299		0
1325	83	2166	11/17/2009	0
1326	83	2299		1
1327	83	2299		2
1328	83	2299		3
1329	84	2196	05/28/2009	0
1330	84	2187	06/10/2009	0
1331	84	2188	06/14/2009	0
1332	84	2189	06/25/2009	0
1333	84	2190	07/24/2009	0
1334	84	2191	07/24/2009	0
1335	84	2192	08/18/2009	0
1336	84	2193	08/18/2009	0
1337	84	2194	08/07/2009	0
1338	84	2195	09/21/2009	0
1339	84	2186		0
1340	84	2299		0
1341	84	2166	11/17/2009	0
1342	84	2299		1
1343	84	2299		2
1344	84	2299		3
1345	85	2196	06/03/2009	0
1346	85	2187	06/06/2009	0
1347	85	2188	06/10/2009	0
1348	85	2189	06/25/2009	0
1349	85	2190	07/11/2009	0
1350	85	2191	07/24/2009	0
1351	85	2192	07/24/2009	0
1352	85	2193	07/24/2009	0
1353	85	2194	09/16/2009	0
1354	85	2195	10/22/2009	0
1355	85	2186		0
1356	85	2299		0
1357	85	2166	11/11/2009	0
1358	85	2299		1
1359	85	2299		2
1360	85	2299		3
1361	86	2196	05/26/2009	0
1362	86	2187	06/04/2009	0
1363	86	2188	06/10/2009	0
1364	86	2189	06/14/2009	0
1365	86	2190	07/11/2009	0
1366	86	2191	07/11/2009	0
1367	86	2192	07/11/2009	0
1368	86	2193	07/24/2009	0
1369	86	2194	08/18/2009	0
1370	86	2195	09/25/2009	0
1371	86	2186		0
1372	86	2299		0
1373	86	2166	11/17/2009	0
1374	86	2299		1
1375	86	2299		2
1376	86	2299		3
1377	87	2196	05/27/2009	0
1378	87	2187	06/04/2009	0
1379	87	2188	06/06/2009	0
1380	87	2189	06/10/2009	0
1381	87	2190	07/11/2009	0
1382	87	2191	07/11/2009	0
1383	87	2192	07/24/2009	0
1384	87	2193	07/24/2009	0
1385	87	2194	07/24/2009	0
1386	87	2195	09/12/2009	0
1387	87	2186		0
1388	87	2299		0
1389	87	2166	11/11/2009	0
1390	87	2299		1
1391	87	2299		2
1392	87	2299		3
1393	88	2196	05/26/2009	0
1394	88	2187	06/02/2009	0
1395	88	2188	06/06/2009	0
1396	88	2189	06/10/2009	0
1397	88	2190	07/11/2009	0
1398	88	2191	07/03/2009	0
1399	88	2192	07/11/2009	0
1400	88	2193	07/24/2009	0
1401	88	2194	07/24/2009	0
1402	88	2195	09/12/2009	0
1403	88	2186		0
1404	88	2299		0
1405	88	2166	11/17/2009	0
1406	88	2299		1
1407	88	2299		2
1408	88	2299		3
1409	89	2196	05/26/2009	0
1410	89	2187	06/02/2009	0
1411	89	2188	06/06/2009	0
1412	89	2189	06/10/2009	0
1413	89	2190	07/11/2009	0
1414	89	2191	07/24/2009	0
1415	89	2192	07/24/2009	0
1416	89	2193	07/24/2009	0
1417	89	2194	07/24/2009	0
1418	89	2195	09/12/2009	0
1419	89	2186		0
1420	89	2299		0
1421	89	2166	11/17/2009	0
1422	89	2299		1
1423	89	2299		2
1424	89	2299		3
1425	90	2196	05/26/2009	0
1426	90	2187	06/02/2009	0
1427	90	2188	06/06/2009	0
1428	90	2189	06/10/2009	0
1429	90	2190	07/11/2009	0
1430	90	2191	07/11/2009	0
1431	90	2192	07/24/2009	0
1432	90	2193	07/11/2009	0
1433	90	2194	07/24/2009	0
1434	90	2195	09/08/2009	0
1435	90	2186		0
1436	90	2299		0
1437	90	2166	11/17/2009	0
1438	90	2299		1
1439	90	2299		2
1440	90	2299		3
1441	91	2196	05/26/2009	0
1442	91	2187	06/04/2009	0
1443	91	2188	06/06/2009	0
1444	91	2189	06/14/2009	0
1445	91	2190	07/11/2009	0
1446	91	2191	06/25/2009	0
1447	91	2192	07/24/2009	0
1448	91	2193	07/24/2009	0
1449	91	2194	08/07/2009	0
1450	91	2195	09/25/2009	0
1451	91	2186		0
1452	91	2299		0
1453	91	2166	11/17/2009	0
1454	91	2299		1
1455	91	2299		2
1456	91	2299		3
1457	92	2196	06/03/2009	0
1458	92	2187	06/10/2009	0
1459	92	2188	06/14/2009	0
1460	92	2189	06/25/2009	0
1461	92	2190	07/24/2009	0
1462	92	2191	07/24/2009	0
1463	92	2192	07/24/2009	0
1464	92	2193	08/07/2009	0
1465	92	2194	07/24/2009	0
1466	92	2195	09/22/2009	0
1467	92	2186		0
1468	92	2299		0
1469	92	2166	11/11/2009	0
1470	92	2299		1
1471	92	2299		2
1472	92	2299		3
1473	93	2196	05/26/2009	0
1474	93	2187	06/02/2009	0
1475	93	2188	06/06/2009	0
1476	93	2189	06/14/2009	0
1477	93	2190	07/11/2009	0
1478	93	2191	06/25/2009	0
1479	93	2192	07/24/2009	0
1480	93	2193	08/07/2009	0
1481	93	2194	08/07/2009	0
1482	93	2195	09/26/2009	0
1483	93	2186		0
1484	93	2299		0
1485	93	2166	11/11/2009	0
1486	93	2299		1
1487	93	2299		2
1488	93	2299		3
1489	94	2196	05/26/2009	0
1490	94	2187	06/04/2009	0
1491	94	2188	06/10/2009	0
1492	94	2189	06/14/2009	0
1493	94	2190	07/11/2009	0
1494	94	2191	07/03/2009	0
1495	94	2192	07/24/2009	0
1496	94	2193	07/24/2009	0
1497	94	2194	08/25/2009	0
1498	94	2195	09/26/2009	0
1499	94	2186		0
1500	94	2299		0
1501	94	2166	11/11/2009	0
1502	94	2299		1
1503	94	2299		2
1504	94	2299		3
1505	95	2196	05/26/2009	0
1506	95	2187	06/04/2009	0
1507	95	2188	06/10/2009	0
1508	95	2189	06/10/2009	0
1509	95	2190	07/11/2009	0
1510	95	2191	07/24/2009	0
1511	95	2192	07/24/2009	0
1512	95	2193	07/24/2009	0
1513	95	2194	08/07/2009	0
1514	95	2195	09/26/2009	0
1515	95	2186		0
1516	95	2299		0
1517	95	2166	11/11/2009	0
1518	95	2299		1
1519	95	2299		2
1520	95	2299		3
1521	96	2196	05/26/2009	0
1522	96	2187	06/04/2009	0
1523	96	2188	06/06/2009	0
1524	96	2189	06/10/2009	0
1525	96	2190	07/11/2009	0
1526	96	2191	07/11/2009	0
1527	96	2192	07/24/2009	0
1528	96	2193	07/24/2009	0
1529	96	2194	07/22/2009	0
1530	96	2195	09/21/2009	0
1531	96	2186		0
1532	96	2299		0
1533	96	2166	11/17/2009	0
1534	96	2299		1
1535	96	2299		2
1536	96	2299		3
1537	97	2196	05/26/2009	0
1538	97	2187	06/02/2009	0
1539	97	2188	06/06/2009	0
1540	97	2189	06/10/2009	0
1541	97	2190	06/25/2009	0
1542	97	2191	07/11/2009	0
1543	97	2192	07/11/2009	0
1544	97	2193	07/24/2009	0
1545	97	2194	07/22/2009	0
1546	97	2195	08/25/2009	0
1547	97	2186		0
1548	97	2299		0
1549	97	2166	11/17/2009	0
1550	97	2299		1
1551	97	2299		2
1552	97	2299		3
1553	98	2196	05/26/2009	0
1554	98	2187	06/04/2009	0
1555	98	2188	06/06/2009	0
1556	98	2189	06/25/2009	0
1557	98	2190	06/14/2009	0
1558	98	2191	07/03/2009	0
1559	98	2192	07/24/2009	0
1560	98	2193	07/24/2009	0
1561	98	2194	08/07/2009	0
1562	98	2195	09/01/2009	0
1563	98	2186		0
1564	98	2299		0
1565	98	2166	11/17/2009	0
1566	98	2299		1
1567	98	2299		2
1568	98	2299		3
1569	99	2196	06/10/2009	0
1570	99	2187		0
1571	99	2188		0
1572	99	2189		0
1573	99	2190	06/25/2009	0
1574	99	2191		0
1575	99	2192		0
1576	99	2193		0
1577	99	2194	.	0
1578	99	2195	.	0
1579	99	2186	06/25/2009	0
1580	99	2299		0
1581	99	2166	11/11/2009	0
1582	99	2299		1
1583	99	2299		2
1584	99	2299		3
1585	100	2196	05/26/2009	0
1586	100	2187	06/02/2009	0
1587	100	2188	06/06/2009	0
1588	100	2189	06/10/2009	0
1589	100	2190	07/11/2009	0
1590	100	2191	07/11/2009	0
1591	100	2192	07/24/2009	0
1592	100	2193	07/24/2009	0
1593	100	2194	08/07/2009	0
1594	100	2195	09/26/2009	0
1595	100	2186		0
1596	100	2299		0
1597	100	2166	11/11/2009	0
1598	100	2299		1
1599	100	2299		2
1600	100	2299		3
1601	101	2196	05/27/2009	0
1602	101	2187	06/04/2009	0
1603	101	2188	06/10/2009	0
1604	101	2189	06/25/2009	0
1605	101	2190	07/11/2009	0
1606	101	2191	07/24/2009	0
1607	101	2192	07/24/2009	0
1608	101	2193	07/24/2009	0
1609	101	2194	08/25/2009	0
1610	101	2195	09/29/2009	0
1611	101	2186		0
1612	101	2299		0
1613	101	2166	11/12/2009	0
1614	101	2299		1
1615	101	2299		2
1616	101	2299		3
1617	102	2196	06/03/2009	0
1618	102	2187	06/04/2009	0
1619	102	2188	06/14/2009	0
1620	102	2189	06/25/2009	0
1621	102	2190	07/11/2009	0
1622	102	2191	06/25/2009	0
1623	102	2192	07/24/2009	0
1624	102	2193	08/07/2009	0
1625	102	2194	09/01/2009	0
1626	102	2195	10/10/2009	0
1627	102	2186		0
1628	102	2299		0
1629	102	2166	11/12/2009	0
1630	102	2299		1
1631	102	2299		2
1632	102	2299		3
1633	103	2196	05/26/2009	0
1634	103	2187	06/04/2009	0
1635	103	2188	06/06/2009	0
1636	103	2189	06/14/2009	0
1637	103	2190	06/14/2009	0
1638	103	2191	07/03/2009	0
1639	103	2192	07/24/2009	0
1640	103	2193	07/24/2009	0
1641	103	2194	08/18/2009	0
1642	103	2195	09/22/2009	0
1643	103	2186		0
1644	103	2299		0
1645	103	2166	11/13/2009	0
1646	103	2299		1
1647	103	2299		2
1648	103	2299		3
1649	104	2196	05/27/2009	0
1650	104	2187	06/04/2009	0
1651	104	2188	06/06/2009	0
1652	104	2189	06/10/2009	0
1653	104	2190	07/24/2009	0
1654	104	2191	07/11/2009	0
1655	104	2192	07/24/2009	0
1656	104	2193	07/11/2009	0
1657	104	2194	07/22/2009	0
1658	104	2195	09/12/2009	0
1659	104	2186		0
1660	104	2299		0
1661	104	2166	11/13/2009	0
1662	104	2299		1
1663	104	2299		2
1664	104	2299		3
1665	105	2196	05/26/2009	0
1666	105	2187	06/04/2009	0
1667	105	2188	06/10/2009	0
1668	105	2189	06/14/2009	0
1669	105	2190	07/11/2009	0
1670	105	2191	06/14/2009	0
1671	105	2192	07/11/2009	0
1672	105	2193	07/24/2009	0
1673	105	2194	07/24/2009	0
1674	105	2195	09/12/2009	0
1675	105	2186		0
1676	105	2299		0
1677	105	2166	11/13/2009	0
1678	105	2299		1
1679	105	2299		2
1680	105	2299		3
1681	106	2196	05/26/2009	0
1682	106	2187	06/04/2009	0
1683	106	2188	.	0
1684	106	2189	.	0
1685	106	2190	06/14/2009	0
1686	106	2191	06/25/2009	0
1687	106	2192	.	0
1688	106	2193	.	0
1689	106	2194	.	0
1690	106	2195	.	0
1691	106	2186	06/25/2009	0
1692	106	2299		0
1693	106	2166	11/13/2009	0
1694	106	2299		1
1695	106	2299		2
1696	106	2299		3
1697	107	2196	05/26/2009	0
1698	107	2187	06/02/2009	0
1699	107	2188	06/06/2009	0
1700	107	2189	06/10/2009	0
1701	107	2190	07/11/2009	0
1702	107	2191	07/03/2009	0
1703	107	2192	07/24/2009	0
1704	107	2193	07/24/2009	0
1705	107	2194	08/07/2009	0
1706	107	2195	09/22/2009	0
1707	107	2186		0
1708	107	2299		0
1709	107	2166	11/13/2009	0
1710	107	2299		1
1711	107	2299		2
1712	107	2299		3
1713	108	2196	05/27/2009	0
1714	108	2187	06/04/2009	0
1715	108	2188	06/10/2009	0
1716	108	2189	06/14/2009	0
1717	108	2190	07/11/2009	0
1718	108	2191	07/24/2009	0
1719	108	2192	07/24/2009	0
1720	108	2193	07/24/2009	0
1721	108	2194	08/07/2009	0
1722	108	2195	10/10/2009	0
1723	108	2186		0
1724	108	2299		0
1725	108	2166	11/12/2009	0
1726	108	2299		1
1727	108	2299		2
1728	108	2299		3
1729	109	2196	05/26/2009	0
1730	109	2187	06/06/2009	0
1731	109	2188	06/10/2009	0
1732	109	2189	06/14/2009	0
1733	109	2190	07/11/2009	0
1734	109	2191	07/24/2009	0
1735	109	2192	07/24/2009	0
1736	109	2193	07/24/2009	0
1737	109	2194	07/24/2009	0
1738	109	2195	09/22/2009	0
1739	109	2186		0
1740	109	2299		0
1741	109	2166	11/12/2009	0
1742	109	2299		1
1743	109	2299		2
1744	109	2299		3
1745	110	2196	05/26/2009	0
1746	110	2187	06/02/2009	0
1747	110	2188	06/06/2009	0
1748	110	2189	06/10/2009	0
1749	110	2190	07/11/2009	0
1750	110	2191	07/03/2009	0
1751	110	2192	07/24/2009	0
1752	110	2193	07/24/2009	0
1753	110	2194	08/07/2009	0
1754	110	2195	09/22/2009	0
1755	110	2186		0
1756	110	2299		0
1757	110	2166	11/13/2009	0
1758	110	2299		1
1759	110	2299		2
1760	110	2299		3
1761	111	2196	05/26/2009	0
1762	111	2187	06/06/2009	0
1763	111	2188	06/10/2009	0
1764	111	2189	06/14/2009	0
1765	111	2190	07/11/2009	0
1766	111	2191	07/24/2009	0
1767	111	2192	07/24/2009	0
1768	111	2193	08/07/2009	0
1769	111	2194	08/07/2009	0
1770	111	2195	09/22/2009	0
1771	111	2186		0
1772	111	2299		0
1773	111	2166	11/13/2009	0
1774	111	2299		1
1775	111	2299		2
1776	111	2299		3
1777	112	2196	05/26/2009	0
1778	112	2187	06/04/2009	0
1779	112	2188	06/10/2009	0
1780	112	2189	06/14/2009	0
1781	112	2190	07/11/2009	0
1782	112	2191	07/11/2009	0
1783	112	2192	07/24/2009	0
1784	112	2193	08/07/2009	0
1785	112	2194	07/22/2009	0
1786	112	2195	09/29/2009	0
1787	112	2186		0
1788	112	2299		0
1789	112	2166	11/12/2009	0
1790	112	2299		1
1791	112	2299		2
1792	112	2299		3
1793	113	2196	05/26/2009	0
1794	113	2187	06/02/2009	0
1795	113	2188	06/10/2009	0
1796	113	2189	06/14/2009	0
1797	113	2190	07/11/2009	0
1798	113	2191	07/03/2009	0
1799	113	2192	07/11/2009	0
1800	113	2193	07/24/2009	0
1801	113	2194	08/07/2009	0
1802	113	2195	09/22/2009	0
1803	113	2186		0
1804	113	2299		0
1805	113	2166	11/13/2009	0
1806	113	2299		1
1807	113	2299		2
1808	113	2299		3
1809	114	2196	05/26/2009	0
1810	114	2187	06/04/2009	0
1811	114	2188	06/10/2009	0
1812	114	2189	06/25/2009	0
1813	114	2190	06/25/2009	0
1814	114	2191	06/25/2009	0
1815	114	2192	07/24/2009	0
1816	114	2193	07/24/2009	0
1817	114	2194		0
1818	114	2195		0
1819	114	2186	09/30/2009	0
1820	114	2299		0
1821	114	2166	11/12/2009	0
1822	114	2299	. No data collecrted, dead plant	1
1823	114	2299		2
1824	114	2299	dead plant harvested 10/4/09	3
1825	115	2196	06/02/2009	0
1826	115	2187	06/06/2009	0
1827	115	2188	06/10/2009	0
1828	115	2189	06/25/2009	0
1829	115	2190	07/11/2009	0
1830	115	2191	07/24/2009	0
1831	115	2192	07/24/2009	0
1832	115	2193	07/24/2009	0
1833	115	2194	07/24/2009	0
1834	115	2195	09/22/2009	0
1835	115	2186		0
1836	115	2299		0
1837	115	2166	11/12/2009	0
1838	115	2299	3 dead stems	1
1839	115	2299		2
1840	115	2299		3
1841	116	2196	05/27/2009	0
1842	116	2187	06/02/2009	0
1843	116	2188	06/10/2009	0
1844	116	2189	06/25/2009	0
1845	116	2190	06/25/2009	0
1846	116	2191	07/24/2009	0
1847	116	2192	07/24/2009	0
1848	116	2193	08/07/2009	0
1849	116	2194	.	0
1850	116	2195	.	0
1851	116	2186	08/07/2009	0
1852	116	2299		0
1853	116	2166	11/13/2009	0
1854	116	2299		1
1855	116	2299		2
1856	116	2299		3
1857	117	2196	05/26/2009	0
1858	117	2187	06/04/2009	0
1859	117	2188	06/06/2009	0
1860	117	2189	06/10/2009	0
1861	117	2190	07/11/2009	0
1862	117	2191	07/11/2009	0
1863	117	2192	07/24/2009	0
1864	117	2193	07/24/2009	0
1865	117	2194	08/07/2009	0
1866	117	2195	09/22/2009	0
1867	117	2186		0
1868	117	2299		0
1869	117	2166	11/12/2009	0
1870	117	2299		1
1871	117	2299		2
1872	117	2299		3
1873	118	2196	05/26/2009	0
1874	118	2187	06/02/2009	0
1875	118	2188	06/06/2009	0
1876	118	2189	06/10/2009	0
1877	118	2190	07/11/2009	0
1878	118	2191	07/24/2009	0
1879	118	2192	07/24/2009	0
1880	118	2193	07/24/2009	0
1881	118	2194	07/22/2009	0
1882	118	2195	09/03/2009	0
1883	118	2186		0
1884	118	2299		0
1885	118	2166	11/12/2009	0
1886	118	2299		1
1887	118	2299		2
1888	118	2299		3
1889	119	2196	05/27/2009	0
1890	119	2187	06/04/2009	0
1891	119	2188	06/06/2009	0
1892	119	2189	06/10/2009	0
1893	119	2190	07/11/2009	0
1894	119	2191	07/11/2009	0
1895	119	2192	07/24/2009	0
1896	119	2193	07/24/2009	0
1897	119	2194	07/22/2009	0
1898	119	2195	08/25/2009	0
1899	119	2186		0
1900	119	2299		0
1901	119	2166	11/12/2009	0
1902	119	2299		1
1903	119	2299		2
1904	119	2299		3
1905	120	2196	06/02/2009	0
1906	120	2187	06/10/2009	0
1907	120	2188	06/14/2009	0
1908	120	2189	06/25/2009	0
1909	120	2190	07/24/2009	0
1910	120	2191	07/24/2009	0
1911	120	2192	07/24/2009	0
1912	120	2193	07/24/2009	0
1913	120	2194	07/22/2009	0
1914	120	2195	09/03/2009	0
1915	120	2186		0
1916	120	2299		0
1917	120	2166	11/13/2009	0
1918	120	2299	4 dead stem	1
1919	120	2299		2
1920	120	2299		3
1921	121	2196	05/26/2009	0
1922	121	2187	06/04/2009	0
1923	121	2188	06/10/2009	0
1924	121	2189	06/14/2009	0
1925	121	2190	06/25/2009	0
1926	121	2191	06/25/2009	0
1927	121	2192	06/25/2009	0
1928	121	2193	07/24/2009	0
1929	121	2194	08/07/2009	0
1930	121	2195	09/22/2009	0
1931	121	2186		0
1932	121	2299		0
1933	121	2166	11/13/2009	0
1934	121	2299		1
1935	121	2299		2
1936	121	2299		3
1937	122	2196	05/28/2009	0
1938	122	2187	06/10/2009	0
1939	122	2188	06/10/2009	0
1940	122	2189	06/25/2009	0
1941	122	2190	07/11/2009	0
1942	122	2191	07/03/2009	0
1943	122	2192	07/24/2009	0
1944	122	2193	07/24/2009	0
1945	122	2194	08/25/2009	0
1946	122	2195		0
1947	122	2186		0
1948	122	2299		0
1949	122	2166	11/13/2009	0
1950	122	2299		1
1951	122	2299		2
1952	122	2299		3
1953	123	2196	05/26/2009	0
1954	123	2187	06/04/2009	0
1955	123	2188	06/10/2009	0
1956	123	2189	06/14/2009	0
1957	123	2190	07/11/2009	0
1958	123	2191	07/11/2009	0
1959	123	2192	07/24/2009	0
1960	123	2193	07/11/2009	0
1961	123	2194	08/07/2009	0
1962	123	2195	09/29/2009	0
1963	123	2186		0
1964	123	2299		0
1965	123	2166	11/13/2009	0
1966	123	2299		1
1967	123	2299		2
1968	123	2299		3
1969	124	2196	05/26/2009	0
1970	124	2187	06/04/2009	0
1971	124	2188	06/10/2009	0
1972	124	2189	06/14/2009	0
1973	124	2190	07/11/2009	0
1974	124	2191	07/03/2009	0
1975	124	2192	07/03/2009	0
1976	124	2193	07/24/2009	0
1977	124	2194	08/07/2009	0
1978	124	2195	10/25/2009	0
1979	124	2186		0
1980	124	2299		0
1981	124	2166	11/13/2009	0
1982	124	2299		1
1983	124	2299		2
1984	124	2299		3
1985	125	2196	05/26/2009	0
1986	125	2187	06/04/2009	0
1987	125	2188	06/10/2009	0
1988	125	2189	06/25/2009	0
1989	125	2190	06/14/2009	0
1990	125	2191	06/14/2009	0
1991	125	2192	06/14/2009	0
1992	125	2193	08/07/2009	0
1993	125	2194	.	0
1994	125	2195	.	0
1995	125	2186	06/25/2009	0
1996	125	2299		0
1997	125	2166	11/13/2009	0
1998	125	2299		1
1999	125	2299		2
2000	125	2299		3
2001	126	2196	05/26/2009	0
2002	126	2187	06/04/2009	0
2003	126	2188	06/10/2009	0
2004	126	2189	06/14/2009	0
2005	126	2190	07/11/2009	0
2006	126	2191	07/11/2009	0
2007	126	2192	07/11/2009	0
2008	126	2193	07/24/2009	0
2009	126	2194	08/18/2009	0
2010	126	2195	09/30/2009	0
2011	126	2186		0
2012	126	2299		0
2013	126	2166	11/14/2009	0
2014	126	2299		1
2015	126	2299		2
2016	126	2299		3
2017	127	2196	05/27/2009	0
2018	127	2187	06/02/2009	0
2019	127	2188	06/10/2009	0
2020	127	2189	06/10/2009	0
2021	127	2190	07/11/2009	0
2022	127	2191	07/03/2009	0
2023	127	2192	07/24/2009	0
2024	127	2193	07/24/2009	0
2025	127	2194	07/22/2009	0
2026	127	2195	08/18/2009	0
2027	127	2186		0
2028	127	2299		0
2029	127	2166	11/13/2009	0
2030	127	2299		1
2031	127	2299		2
2032	127	2299		3
2033	128	2196	05/26/2009	0
2034	128	2187	06/04/2009	0
2035	128	2188	06/10/2009	0
2036	128	2189	06/14/2009	0
2037	128	2190	06/25/2009	0
2038	128	2191	06/25/2009	0
2039	128	2192	07/03/2009	0
2040	128	2193	07/24/2009	0
2041	128	2194	07/24/2009	0
2042	128	2195	09/12/2009	0
2043	128	2186		0
2044	128	2299		0
2045	128	2166	11/14/2009	0
2046	128	2299		1
2047	128	2299		2
2048	128	2299		3
2049	129	2196	05/26/2009	0
2050	129	2187	06/04/2009	0
2051	129	2188	06/10/2009	0
2052	129	2189	06/25/2009	0
2053	129	2190	06/10/2009	0
2054	129	2191	06/10/2009	0
2055	129	2192	07/11/2009	0
2056	129	2193	07/11/2009	0
2057	129	2194	.	0
2058	129	2195	.	0
2059	129	2186	07/11/2009	0
2060	129	2299		0
2061	129	2166	11/13/2009	0
2062	129	2299		1
2063	129	2299		2
2064	129	2299		3
2065	130	2196	05/27/2009	0
2066	130	2187	06/04/2009	0
2067	130	2188	06/06/2009	0
2068	130	2189	06/14/2009	0
2069	130	2190	06/25/2009	0
2070	130	2191	06/14/2009	0
2071	130	2192	07/24/2009	0
2072	130	2193	07/24/2009	0
2073	130	2194	08/18/2009	0
2074	130	2195	10/25/2009	0
2075	130	2186		0
2076	130	2299	GR PULLED BY ME 6/25/09	0
2077	130	2166	11/14/2009	0
2078	130	2299		1
2079	130	2299		2
2080	130	2299		3
2081	131	2196	05/26/2009	0
2082	131	2187	06/04/2009	0
2083	131	2188	06/10/2009	0
2084	131	2189	06/14/2009	0
2085	131	2190	07/11/2009	0
2086	131	2191	07/24/2009	0
2087	131	2192	07/24/2009	0
2088	131	2193	07/24/2009	0
2089	131	2194	08/18/2009	0
2090	131	2195	10/25/2009	0
2091	131	2186		0
2092	131	2299		0
2093	131	2166	11/14/2009	0
2094	131	2299		1
2095	131	2299		2
2096	131	2299		3
2097	132	2196	05/26/2009	0
2098	132	2187	06/04/2009	0
2099	132	2188	06/10/2009	0
2100	132	2189	06/14/2009	0
2101	132	2190	07/11/2009	0
2102	132	2191	07/03/2009	0
2103	132	2192	07/24/2009	0
2104	132	2193	07/24/2009	0
2105	132	2194	08/07/2009	0
2106	132	2195	09/30/2009	0
2107	132	2186		0
2108	132	2299		0
2109	132	2166	11/14/2009	0
2110	132	2299		1
2111	132	2299		2
2112	132	2299		3
2113	133	2196	05/26/2009	0
2114	133	2187	06/04/2009	0
2115	133	2188	06/06/2009	0
2116	133	2189	06/10/2009	0
2117	133	2190	07/11/2009	0
2118	133	2191	07/03/2009	0
2119	133	2192	07/24/2009	0
2120	133	2193	07/24/2009	0
2121	133	2194	08/07/2009	0
2122	133	2195	09/21/2009	0
2123	133	2186		0
2124	133	2299		0
2125	133	2166	11/14/2009	0
2126	133	2299		1
2127	133	2299		2
2128	133	2299		3
2129	134	2196	05/26/2009	0
2130	134	2187	06/04/2009	0
2131	134	2188	06/10/2009	0
2132	134	2189	06/14/2009	0
2133	134	2190	07/11/2009	0
2134	134	2191	07/24/2009	0
2135	134	2192	07/24/2009	0
2136	134	2193	07/24/2009	0
2137	134	2194	08/07/2009	0
2138	134	2195	09/22/2009	0
2139	134	2186		0
2140	134	2299		0
2141	134	2166	11/13/2009	0
2142	134	2299		1
2143	134	2299		2
2144	134	2299		3
2145	135	2196	05/26/2009	0
2146	135	2187	06/04/2009	0
2147	135	2188	06/10/2009	0
2148	135	2189	06/10/2009	0
2149	135	2190	07/11/2009	0
2150	135	2191	07/24/2009	0
2151	135	2192	07/24/2009	0
2152	135	2193	07/11/2009	0
2153	135	2194	08/07/2009	0
2154	135	2195	09/22/2009	0
2155	135	2186		0
2156	135	2299		0
2157	135	2166	11/13/2009	0
2158	135	2299	Some tripe damage (no estimate)	1
2159	135	2299		2
2160	135	2299		3
2161	136	2196	05/26/2009	0
2162	136	2187	06/04/2009	0
2163	136	2188	06/10/2009	0
2164	136	2189	06/25/2009	0
2165	136	2190	06/25/2009	0
2166	136	2191	06/25/2009	0
2167	136	2192	07/24/2009	0
2168	136	2193	07/24/2009	0
2169	136	2194	07/24/2009	0
2170	136	2195	09/12/2009	0
2171	136	2186		0
2172	136	2299		0
2173	136	2166	11/14/2009	0
2174	136	2299		1
2175	136	2299		2
2176	136	2299		3
2177	137	2196	05/26/2009	0
2178	137	2187	06/04/2009	0
2179	137	2188	06/06/2009	0
2180	137	2189	06/14/2009	0
2181	137	2190	06/25/2009	0
2182	137	2191	06/14/2009	0
2183	137	2192	07/24/2009	0
2184	137	2193	07/24/2009	0
2185	137	2194	07/24/2009	0
2186	137	2195	09/22/2009	0
2187	137	2186		0
2188	137	2299		0
2189	137	2166	11/13/2009	0
2190	137	2299		1
2191	137	2299		2
2192	137	2299		3
2193	138	2196	05/26/2009	0
2194	138	2187	06/04/2009	0
2195	138	2188	06/06/2009	0
2196	138	2189	06/10/2009	0
2197	138	2190	07/11/2009	0
2198	138	2191	06/25/2009	0
2199	138	2192	07/11/2009	0
2200	138	2193	07/24/2009	0
2201	138	2194	07/22/2009	0
2202	138	2195	08/16/2009	0
2203	138	2186		0
2204	138	2299		0
2205	138	2166	11/14/2009	0
2206	138	2299		1
2207	138	2299		2
2208	138	2299		3
2209	139	2196	05/27/2009	0
2210	139	2187	06/04/2009	0
2211	139	2188	06/10/2009	0
2212	139	2189	06/14/2009	0
2213	139	2190	07/11/2009	0
2214	139	2191	07/03/2009	0
2215	139	2192	07/24/2009	0
2216	139	2193	07/24/2009	0
2217	139	2194	08/07/2009	0
2218	139	2195	09/30/2009	0
2219	139	2186		0
2220	139	2299		0
2221	139	2166	11/14/2009	0
2222	139	2299		1
2223	139	2299		2
2224	139	2299		3
2225	140	2196	05/28/2009	0
2226	140	2187	06/04/2009	0
2227	140	2188	06/10/2009	0
2228	140	2189	06/14/2009	0
2229	140	2190	07/11/2009	0
2230	140	2191	07/24/2009	0
2231	140	2192	07/24/2009	0
2232	140	2193	07/24/2009	0
2233	140	2194	07/22/2009	0
2234	140	2195	09/12/2009	0
2235	140	2186		0
2236	140	2299		0
2237	140	2166	11/14/2009	0
2238	140	2299		1
2239	140	2299		2
2240	140	2299		3
2241	141	2196	05/26/2009	0
2242	141	2187	06/06/2009	0
2243	141	2188	06/10/2009	0
2244	141	2189	06/25/2009	0
2245	141	2190	07/11/2009	0
2246	141	2191	07/03/2009	0
2247	141	2192	07/24/2009	0
2248	141	2193	07/24/2009	0
2249	141	2194	08/18/2009	0
2250	141	2195	09/25/2009	0
2251	141	2186		0
2252	141	2299		0
2253	141	2166	11/18/2009	0
2254	141	2299		1
2255	141	2299		2
2256	141	2299		3
2257	142	2196	05/28/2009	0
2258	142	2187	06/04/2009	0
2259	142	2188	06/10/2009	0
2260	142	2189	06/10/2009	0
2261	142	2190	07/11/2009	0
2262	142	2191	06/25/2009	0
2263	142	2192	07/24/2009	0
2264	142	2193	07/11/2009	0
2265	142	2194	08/18/2009	0
2266	142	2195	09/29/2009	0
2267	142	2186		0
2268	142	2299		0
2269	142	2166	11/10/2009	0
2270	142	2299		1
2271	142	2299		2
2272	142	2299		3
2273	143	2196	05/26/2009	0
2274	143	2187	06/04/2009	0
2275	143	2188	06/10/2009	0
2276	143	2189	06/14/2009	0
2277	143	2190	06/25/2009	0
2278	143	2191	07/03/2009	0
2279	143	2192	07/24/2009	0
2280	143	2193	07/24/2009	0
2281	143	2194	08/07/2009	0
2282	143	2195	09/22/2009	0
2283	143	2186		0
2284	143	2299		0
2285	143	2166	11/10/2009	0
2286	143	2299		1
2287	143	2299		2
2288	143	2299		3
2289	144	2196	05/27/2009	0
2290	144	2187	06/04/2009	0
2291	144	2188	06/10/2009	0
2292	144	2189	06/10/2009	0
2293	144	2190	07/11/2009	0
2294	144	2191	07/03/2009	0
2295	144	2192	07/24/2009	0
2296	144	2193	07/11/2009	0
2297	144	2194	08/07/2009	0
2298	144	2195	09/22/2009	0
2299	144	2186		0
2300	144	2299		0
2301	144	2166	11/10/2009	0
2302	144	2299		1
2303	144	2299		2
2304	144	2299		3
2305	145	2196	05/27/2009	0
2306	145	2187	06/04/2009	0
2307	145	2188	06/10/2009	0
2308	145	2189	06/25/2009	0
2309	145	2190	07/11/2009	0
2310	145	2191	07/11/2009	0
2311	145	2192	07/24/2009	0
2312	145	2193	07/24/2009	0
2313	145	2194	07/24/2009	0
2314	145	2195	09/25/2009	0
2315	145	2186		0
2316	145	2299		0
2317	145	2166	11/18/2009	0
2318	145	2299		1
2319	145	2299		2
2320	145	2299		3
2321	146	2196	05/26/2009	0
2322	146	2187	06/04/2009	0
2323	146	2188	06/10/2009	0
2324	146	2189	06/14/2009	0
2325	146	2190	07/11/2009	0
2326	146	2191	06/25/2009	0
2327	146	2192	07/24/2009	0
2328	146	2193	07/24/2009	0
2329	146	2194	07/22/2009	0
2330	146	2195	09/12/2009	0
2331	146	2186		0
2332	146	2299		0
2333	146	2166	11/18/2009	0
2334	146	2299		1
2335	146	2299		2
2336	146	2299		3
2337	147	2196	05/27/2009	0
2338	147	2187	06/06/2009	0
2339	147	2188	06/10/2009	0
2340	147	2189	06/14/2009	0
2341	147	2190	06/25/2009	0
2342	147	2191	06/25/2009	0
2343	147	2192	07/24/2009	0
2344	147	2193	07/24/2009	0
2345	147	2194	08/07/2009	0
2346	147	2195	09/12/2009	0
2347	147	2186		0
2348	147	2299		0
2349	147	2166	11/10/2009	0
2350	147	2299		1
2351	147	2299		2
2352	147	2299		3
2353	148	2196	06/02/2009	0
2354	148	2187	06/10/2009	0
2355	148	2188	06/10/2009	0
2356	148	2189	06/25/2009	0
2357	148	2190	06/14/2009	0
2358	148	2191	07/11/2009	0
2359	148	2192	08/07/2009	0
2360	148	2193	08/18/2009	0
2361	148	2194	08/25/2009	0
2362	148	2195	10/26/2009	0
2363	148	2186		0
2364	148	2299		0
2365	148	2166	11/18/2009	0
2366	148	2299		1
2367	148	2299		2
2368	148	2299		3
2369	149	2196	05/26/2009	0
2370	149	2187	06/04/2009	0
2371	149	2188	06/10/2009	0
2372	149	2189	06/14/2009	0
2373	149	2190	07/11/2009	0
2374	149	2191	06/25/2009	0
2375	149	2192	07/11/2009	0
2376	149	2193	07/03/2009	0
2377	149	2194	07/22/2009	0
2378	149	2195	09/12/2009	0
2379	149	2186		0
2380	149	2299		0
2381	149	2166	11/10/2009	0
2382	149	2299		1
2383	149	2299		2
2384	149	2299		3
2385	150	2196	05/26/2009	0
2386	150	2187	06/02/2009	0
2387	150	2188	06/06/2009	0
2388	150	2189	06/10/2009	0
2389	150	2190	07/11/2009	0
2390	150	2191	07/03/2009	0
2391	150	2192	07/24/2009	0
2392	150	2193	07/24/2009	0
2393	150	2194	07/24/2009	0
2394	150	2195	09/12/2009	0
2395	150	2186		0
2396	150	2299		0
2397	150	2166	11/18/2009	0
2398	150	2299	2 dead stems	1
2399	150	2299		2
2400	150	2299		3
2401	151	2196	05/26/2009	0
2402	151	2187	06/04/2009	0
2403	151	2188	06/10/2009	0
2404	151	2189	06/14/2009	0
2405	151	2190	07/11/2009	0
2406	151	2191	07/24/2009	0
2407	151	2192	07/24/2009	0
2408	151	2193	07/11/2009	0
2409	151	2194	08/07/2009	0
2410	151	2195	09/12/2009	0
2411	151	2186		0
2412	151	2299		0
2413	151	2166	11/10/2009	0
2414	151	2299		1
2415	151	2299		2
2416	151	2299		3
2417	152	2196	05/26/2009	0
2418	152	2187	06/04/2009	0
2419	152	2188	06/10/2009	0
2420	152	2189	06/14/2009	0
2421	152	2190	06/25/2009	0
2422	152	2191	06/25/2009	0
2423	152	2192	07/24/2009	0
2424	152	2193	07/24/2009	0
2425	152	2194	08/07/2009	0
2426	152	2195	09/29/2009	0
2427	152	2186		0
2428	152	2299		0
2429	152	2166	11/10/2009	0
2430	152	2299		1
2431	152	2299		2
2432	152	2299		3
2433	153	2196	05/26/2009	0
2434	153	2187	06/04/2009	0
2435	153	2188	06/06/2009	0
2436	153	2189	06/14/2009	0
2437	153	2190	06/25/2009	0
2438	153	2191	06/14/2009	0
2439	153	2192	07/24/2009	0
2440	153	2193	07/24/2009	0
2441	153	2194	08/07/2009	0
2442	153	2195	09/22/2009	0
2443	153	2186		0
2444	153	2299		0
2445	153	2166	11/10/2009	0
2446	153	2299		1
2447	153	2299		2
2448	153	2299		3
2449	154	2196	05/26/2009	0
2450	154	2187	06/04/2009	0
2451	154	2188	06/06/2009	0
2452	154	2189	06/10/2009	0
2453	154	2190	06/25/2009	0
2454	154	2191	07/03/2009	0
2455	154	2192	07/24/2009	0
2456	154	2193	07/24/2009	0
2457	154	2194	08/18/2009	0
2458	154	2195	10/26/2009	0
2459	154	2186		0
2460	154	2299		0
2461	154	2166	11/18/2009	0
2462	154	2299		1
2463	154	2299		2
2464	154	2299		3
2465	155	2196	05/28/2009	0
2466	155	2187	06/06/2009	0
2467	155	2188	06/10/2009	0
2468	155	2189	06/14/2009	0
2469	155	2190	07/11/2009	0
2470	155	2191	07/03/2009	0
2471	155	2192	07/24/2009	0
2472	155	2193	07/24/2009	0
2473	155	2194	08/07/2009	0
2474	155	2195	09/21/2009	0
2475	155	2186		0
2476	155	2299		0
2477	155	2166	11/18/2009	0
2478	155	2299		1
2479	155	2299		2
2480	155	2299		3
2481	156	2196	05/27/2009	0
2482	156	2187	06/04/2009	0
2483	156	2188	06/10/2009	0
2484	156	2189	06/10/2009	0
2485	156	2190	07/11/2009	0
2486	156	2191	07/11/2009	0
2487	156	2192	07/24/2009	0
2488	156	2193	07/24/2009	0
2489	156	2194	07/22/2009	0
2490	156	2195	09/12/2009	0
2491	156	2186		0
2492	156	2299		0
2493	156	2166	11/18/2009	0
2494	156	2299		1
2495	156	2299		2
2496	156	2299		3
2497	157	2196	05/26/2009	0
2498	157	2187	06/04/2009	0
2499	157	2188	06/10/2009	0
2500	157	2189	06/10/2009	0
2501	157	2190	07/11/2009	0
2502	157	2191	07/03/2009	0
2503	157	2192	07/11/2009	0
2504	157	2193	07/24/2009	0
2505	157	2194	07/22/2009	0
2506	157	2195	08/25/2009	0
2507	157	2186		0
2508	157	2299		0
2509	157	2166	11/10/2009	0
2510	157	2299		1
2511	157	2299		2
2512	157	2299		3
2513	158	2196	05/26/2009	0
2514	158	2187	06/04/2009	0
2515	158	2188	06/06/2009	0
2516	158	2189	06/10/2009	0
2517	158	2190	06/25/2009	0
2518	158	2191	07/11/2009	0
2519	158	2192	07/24/2009	0
2520	158	2193	07/24/2009	0
2521	158	2194	08/07/2009	0
2522	158	2195	08/21/2009	0
2523	158	2186		0
2524	158	2299		0
2525	158	2166	11/10/2009	0
2526	158	2299		1
2527	158	2299		2
2528	158	2299		3
2529	159	2196	05/26/2009	0
2530	159	2187	06/04/2009	0
2531	159	2188	06/10/2009	0
2532	159	2189	06/14/2009	0
2533	159	2190	06/25/2009	0
2534	159	2191	06/25/2009	0
2535	159	2192	07/11/2009	0
2536	159	2193	07/24/2009	0
2537	159	2194	07/22/2009	0
2538	159	2195	09/12/2009	0
2539	159	2186		0
2540	159	2299		0
2541	159	2166	11/18/2009	0
2542	159	2299		1
2543	159	2299		2
2544	159	2299		3
2545	160	2196	05/27/2009	0
2546	160	2187	06/04/2009	0
2547	160	2188	06/10/2009	0
2548	160	2189	06/14/2009	0
2549	160	2190	06/25/2009	0
2550	160	2191	07/24/2009	0
2551	160	2192	07/24/2009	0
2552	160	2193	07/24/2009	0
2553	160	2194	07/22/2009	0
2554	160	2195	09/01/2009	0
2555	160	2186		0
2556	160	2299		0
2557	160	2166	11/18/2009	0
2558	160	2299		1
2559	160	2299		2
2560	160	2299		3
2561	161	2196	05/27/2009	0
2562	161	2187	06/06/2009	0
2563	161	2188	06/10/2009	0
2564	161	2189	06/14/2009	0
2565	161	2190	07/11/2009	0
2566	161	2191	07/11/2009	0
2567	161	2192	07/24/2009	0
2568	161	2193	07/24/2009	0
2569	161	2194	08/07/2009	0
2570	161	2195	09/21/2009	0
2571	161	2186		0
2572	161	2299		0
2573	161	2166	11/19/2009	0
2574	161	2299		1
2575	161	2299		2
2576	161	2299		3
2577	162	2196	05/27/2009	0
2578	162	2187	06/10/2009	0
2579	162	2188	06/14/2009	0
2580	162	2189	.	0
2581	162	2190	06/14/2009	0
2582	162	2191	06/25/2009	0
2583	162	2192	06/25/2009	0
2584	162	2193	.	0
2585	162	2194		0
2586	162	2195		0
2587	162	2186	06/25/2009	0
2588	162	2299		0
2589	162	2166	11/11/2009	0
2590	162	2299		1
2591	162	2299		2
2592	162	2299		3
2593	163	2196	05/28/2009	0
2594	163	2187	06/14/2009	0
2595	163	2188	06/14/2009	0
2596	163	2189	06/25/2009	0
2597	163	2190	07/24/2009	0
2598	163	2191	07/24/2009	0
2599	163	2192	07/24/2009	0
2600	163	2193	07/24/2009	0
2601	163	2194	08/18/2009	0
2602	163	2195		0
2603	163	2186		0
2604	163	2299		0
2605	163	2166	11/11/2009	0
2606	163	2299		1
2607	163	2299		2
2608	163	2299		3
2609	164	2196	05/27/2009	0
2610	164	2187	06/06/2009	0
2611	164	2188	06/10/2009	0
2612	164	2189	06/25/2009	0
2613	164	2190	07/11/2009	0
2614	164	2191	07/24/2009	0
2615	164	2192	07/24/2009	0
2616	164	2193	08/07/2009	0
2617	164	2194	08/18/2009	0
2618	164	2195	09/18/2009	0
2619	164	2186		0
2620	164	2299		0
2621	164	2166	11/11/2009	0
2622	164	2299	. Leaf number not estimated	1
2623	164	2299		2
2624	164	2299		3
2625	165	2196	05/26/2009	0
2626	165	2187	06/04/2009	0
2627	165	2188	06/06/2009	0
2628	165	2189	06/10/2009	0
2629	165	2190	07/11/2009	0
2630	165	2191	07/24/2009	0
2631	165	2192	07/24/2009	0
2632	165	2193	07/24/2009	0
2633	165	2194	07/24/2009	0
2634	165	2195	09/01/2009	0
2635	165	2186		0
2636	165	2299		0
2637	165	2166	11/11/2009	0
2638	165	2299	3 dead stems	1
2639	165	2299		2
2640	165	2299		3
2641	166	2196	05/27/2009	0
2642	166	2187	06/06/2009	0
2643	166	2188	06/10/2009	0
2644	166	2189	06/14/2009	0
2645	166	2190	07/11/2009	0
2646	166	2191	07/24/2009	0
2647	166	2192	07/24/2009	0
2648	166	2193	07/11/2009	0
2649	166	2194	07/24/2009	0
2650	166	2195	09/12/2009	0
2651	166	2186		0
2652	166	2299		0
2653	166	2166	11/19/2009	0
2654	166	2299		1
2655	166	2299		2
2656	166	2299		3
2657	167	2196	05/27/2009	0
2658	167	2187	06/04/2009	0
2659	167	2188	06/10/2009	0
2660	167	2189	06/14/2009	0
2661	167	2190	07/11/2009	0
2662	167	2191	07/03/2009	0
2663	167	2192	07/24/2009	0
2664	167	2193	07/24/2009	0
2665	167	2194	07/24/2009	0
2666	167	2195	09/21/2009	0
2667	167	2186		0
2668	167	2299		0
2669	167	2166	11/19/2009	0
2670	167	2299	Several dead stems	1
2671	167	2299		2
2672	167	2299		3
2673	168	2196	05/26/2009	0
2674	168	2187	06/04/2009	0
2675	168	2188	06/10/2009	0
2676	168	2189	06/14/2009	0
2677	168	2190	06/25/2009	0
2678	168	2191	07/11/2009	0
2679	168	2192	07/11/2009	0
2680	168	2193	07/24/2009	0
2681	168	2194	08/07/2009	0
2682	168	2195	09/12/2009	0
2683	168	2186	09/29/2009	0
2684	168	2299		0
2685	168	2166	11/19/2009	0
2686	168	2299		1
2687	168	2299		2
2688	168	2299	dead plant harvested 10/4/09	3
2689	169	2196	05/27/2009	0
2690	169	2187	06/04/2009	0
2691	169	2188	06/10/2009	0
2692	169	2189	06/14/2009	0
2693	169	2190	07/11/2009	0
2694	169	2191	07/24/2009	0
2695	169	2192	07/11/2009	0
2696	169	2193	07/11/2009	0
2697	169	2194	08/07/2009	0
2698	169	2195	10/09/2009	0
2699	169	2186		0
2700	169	2299		0
2701	169	2166	11/11/2009	0
2702	169	2299		1
2703	169	2299		2
2704	169	2299	** Some roots were lost prior to weighing do not use root weight!!!!	3
2705	170	2196	05/26/2009	0
2706	170	2187	06/04/2009	0
2707	170	2188	06/06/2009	0
2708	170	2189	06/10/2009	0
2709	170	2190	07/11/2009	0
2710	170	2191	07/11/2009	0
2711	170	2192	07/11/2009	0
2712	170	2193	07/11/2009	0
2713	170	2194	07/24/2009	0
2714	170	2195	09/12/2009	0
2715	170	2186		0
2716	170	2299		0
2717	170	2166	11/19/2009	0
2718	170	2299		1
2719	170	2299		2
2720	170	2299		3
2721	171	2196	06/06/2009	0
2722	171	2187	06/10/2009	0
2723	171	2188	06/14/2009	0
2724	171	2189	06/25/2009	0
2725	171	2190	06/25/2009	0
2726	171	2191	07/03/2009	0
2727	171	2192	07/24/2009	0
2728	171	2193	08/07/2009	0
2729	171	2194	08/18/2009	0
2730	171	2195	09/29/2009	0
2731	171	2186		0
2732	171	2299		0
2733	171	2166	11/11/2009	0
2734	171	2299		1
2735	171	2299		2
2736	171	2299		3
2737	172	2196	06/03/2009	0
2738	172	2187	06/10/2009	0
2739	172	2188	06/14/2009	0
2740	172	2189	06/25/2009	0
2741	172	2190	07/11/2009	0
2742	172	2191	06/14/2009	0
2743	172	2192	07/03/2009	0
2744	172	2193	08/07/2009	0
2745	172	2194	08/25/2009	0
2746	172	2195		0
2747	172	2186		0
2748	172	2299		0
2749	172	2166	11/19/2009	0
2750	172	2299		1
2751	172	2299		2
2752	172	2299		3
2753	173	2196	05/27/2009	0
2754	173	2187	06/06/2009	0
2755	173	2188	06/14/2009	0
2756	173	2189	06/14/2009	0
2757	173	2190	07/24/2009	0
2758	173	2191	07/03/2009	0
2759	173	2192	07/24/2009	0
2760	173	2193	07/24/2009	0
2761	173	2194	08/07/2009	0
2762	173	2195	09/30/2009	0
2763	173	2186		0
2764	173	2299	B02- PULLED OFF BY MISTAKE KM 6/24/09)	0
2765	173	2166	11/19/2009	0
2766	173	2299	. No estimated plant condition	1
2767	173	2299		2
2768	173	2299		3
2769	174	2196	05/26/2009	0
2770	174	2187	06/10/2009	0
2771	174	2188	06/10/2009	0
2772	174	2189	06/14/2009	0
2773	174	2190	07/11/2009	0
2774	174	2191	06/14/2009	0
2775	174	2192	07/24/2009	0
2776	174	2193	07/24/2009	0
2777	174	2194	08/07/2009	0
2778	174	2195	09/30/2009	0
2779	174	2186		0
2780	174	2299		0
2781	174	2166	11/19/2009	0
2782	174	2299	Moldy (mold damage not estimated)	1
2783	174	2299		2
2784	174	2299		3
2785	175	2196	05/28/2009	0
2786	175	2187	06/04/2009	0
2787	175	2188	06/10/2009	0
2788	175	2189	06/14/2009	0
2789	175	2190	07/24/2009	0
2790	175	2191	07/03/2009	0
2791	175	2192	07/24/2009	0
2792	175	2193	07/24/2009	0
2793	175	2194	07/22/2009	0
2794	175	2195	09/12/2009	0
2795	175	2186		0
2796	175	2299		0
2797	175	2166	11/11/2009	0
2798	175	2299	1 dead stem	1
2799	175	2299		2
2800	175	2299		3
2801	176	2196	06/03/2009	0
2802	176	2187	06/10/2009	0
2803	176	2188	06/10/2009	0
2804	176	2189	06/14/2009	0
2805	176	2190	07/11/2009	0
2806	176	2191	07/24/2009	0
2807	176	2192	07/24/2009	0
2808	176	2193	07/24/2009	0
2809	176	2194	07/24/2009	0
2810	176	2195	09/01/2009	0
2811	176	2186		0
2812	176	2299		0
2813	176	2166	11/11/2009	0
2814	176	2299		1
2815	176	2299		2
2816	176	2299		3
2817	177	2196	05/27/2009	0
2818	177	2187	06/06/2009	0
2819	177	2188	06/10/2009	0
2820	177	2189	06/14/2009	0
2821	177	2190	07/11/2009	0
2822	177	2191	07/24/2009	0
2823	177	2192	07/24/2009	0
2824	177	2193	07/24/2009	0
2825	177	2194	08/07/2009	0
2826	177	2195	09/12/2009	0
2827	177	2186		0
2828	177	2299		0
2829	177	2166	11/11/2009	0
2830	177	2299	1 dead stem	1
2831	177	2299		2
2832	177	2299		3
2833	178	2196	05/27/2009	0
2834	178	2187	06/04/2009	0
2835	178	2188	06/10/2009	0
2836	178	2189	06/10/2009	0
2837	178	2190	07/11/2009	0
2838	178	2191	07/03/2009	0
2839	178	2192	07/24/2009	0
2840	178	2193	07/24/2009	0
2841	178	2194	07/24/2009	0
2842	178	2195	09/21/2009	0
2843	178	2186		0
2844	178	2299		0
2845	178	2166	11/19/2009	0
2846	178	2299	6 dead stems	1
2847	178	2299		2
2848	178	2299		3
2849	179	2196	05/26/2009	0
2850	179	2187	06/02/2009	0
2851	179	2188	06/06/2009	0
2852	179	2189	06/10/2009	0
2853	179	2190	07/11/2009	0
2854	179	2191	07/11/2009	0
2855	179	2192	07/11/2009	0
2856	179	2193	07/24/2009	0
2857	179	2194	07/22/2009	0
2858	179	2195	08/18/2009	0
2859	179	2186		0
2860	179	2299		0
2861	179	2166	11/11/2009	0
2862	179	2299	6 dead stems	1
2863	179	2299		2
2864	179	2299		3
2865	180	2196	05/27/2009	0
2866	180	2187	06/04/2009	0
2867	180	2188	06/10/2009	0
2868	180	2189	06/14/2009	0
2869	180	2190	07/11/2009	0
2870	180	2191	07/11/2009	0
2871	180	2192	07/24/2009	0
2872	180	2193	07/24/2009	0
2873	180	2194	07/22/2009	0
2874	180	2195	09/03/2009	0
2875	180	2186		0
2876	180	2299		0
2877	180	2166	11/19/2009	0
2878	180	2299	Moldy (mold damage not estimated)	1
2879	180	2299		2
2880	180	2299		3
2881	181	2196	05/27/2009	0
2882	181	2187	06/04/2009	0
2883	181	2188	06/10/2009	0
2884	181	2189	06/14/2009	0
2885	181	2190	07/11/2009	0
2886	181	2191	07/03/2009	0
2887	181	2192	07/24/2009	0
2888	181	2193	07/24/2009	0
2889	181	2194	08/07/2009	0
2890	181	2195	09/21/2009	0
2891	181	2186		0
2892	181	2299		0
2893	181	2166	11/14/2009	0
2894	181	2299		1
2895	181	2299		2
2896	181	2299		3
2897	182	2196	06/02/2009	0
2898	182	2187	06/06/2009	0
2899	182	2188	06/14/2009	0
2900	182	2189	06/25/2009	0
2901	182	2190	07/11/2009	0
2902	182	2191	06/14/2009	0
2903	182	2192	07/24/2009	0
2904	182	2193	08/07/2009	0
2905	182	2194	08/25/2009	0
2906	182	2195	09/21/2009	0
2907	182	2186		0
2908	182	2299		0
2909	182	2166	11/14/2009	0
2910	182	2299		1
2911	182	2299		2
2912	182	2299		3
2913	183	2196	06/06/2009	0
2914	183	2187	06/25/2009	0
2915	183	2188	06/25/2009	0
2916	183	2189	06/25/2009	0
2917	183	2190	07/11/2009	0
2918	183	2191	07/24/2009	0
2919	183	2192	07/24/2009	0
2920	183	2193	08/07/2009	0
2921	183	2194		0
2922	183	2195		0
2923	183	2186		0
2924	183	2299		0
2925	183	2166	11/13/2009	0
2926	183	2299		1
2927	183	2299		2
2928	183	2299		3
2929	184	2196	06/10/2009	0
2930	184	2187	06/14/2009	0
2931	184	2188	06/25/2009	0
2932	184	2189	06/25/2009	0
2933	184	2190	07/24/2009	0
2934	184	2191	07/24/2009	0
2935	184	2192	08/07/2009	0
2936	184	2193	08/18/2009	0
2937	184	2194	08/18/2009	0
2938	184	2195	09/29/2009	0
2939	184	2186		0
2940	184	2299		0
2941	184	2166	11/13/2009	0
2942	184	2299		1
2943	184	2299		2
2944	184	2299		3
2945	185	2196	06/02/2009	0
2946	185	2187	06/14/2009	0
2947	185	2188	06/14/2009	0
2948	185	2189	06/25/2009	0
2949	185	2190	07/24/2009	0
2950	185	2191	07/24/2009	0
2951	185	2192	07/24/2009	0
2952	185	2193	08/07/2009	0
2953	185	2194	07/24/2009	0
2954	185	2195	09/21/2009	0
2955	185	2186		0
2956	185	2299		0
2957	185	2166	11/14/2009	0
2958	185	2299		1
2959	185	2299		2
2960	185	2299		3
2961	186	2196	05/28/2009	0
2962	186	2187	06/06/2009	0
2963	186	2188	06/10/2009	0
2964	186	2189	06/25/2009	0
2965	186	2190	07/11/2009	0
2966	186	2191	07/24/2009	0
2967	186	2192	07/24/2009	0
2968	186	2193	07/24/2009	0
2969	186	2194	07/24/2009	0
2970	186	2195	09/12/2009	0
2971	186	2186		0
2972	186	2299		0
2973	186	2166	11/13/2009	0
2974	186	2299		1
2975	186	2299		2
2976	186	2299		3
2977	187	2196	05/28/2009	0
2978	187	2187	06/10/2009	0
2979	187	2188	06/10/2009	0
2980	187	2189	06/25/2009	0
2981	187	2190	07/11/2009	0
2982	187	2191	07/24/2009	0
2983	187	2192	07/24/2009	0
2984	187	2193	07/24/2009	0
2985	187	2194	08/07/2009	0
2986	187	2195	09/22/2009	0
2987	187	2186		0
2988	187	2299		0
2989	187	2166	11/13/2009	0
2990	187	2299		1
2991	187	2299		2
2992	187	2299		3
2993	188	2196	05/27/2009	0
2994	188	2187	06/04/2009	0
2995	188	2188	06/10/2009	0
2996	188	2189	06/14/2009	0
2997	188	2190	07/11/2009	0
2998	188	2191	07/24/2009	0
2999	188	2192	07/24/2009	0
3000	188	2193	07/24/2009	0
3001	188	2194	07/22/2009	0
3002	188	2195	08/22/2009	0
3003	188	2186	09/29/2009	0
3004	188	2299		0
3005	188	2166	11/14/2009	0
3006	188	2299		1
3007	188	2299		2
3008	188	2299	dead plant harvested 10/4/09	3
3009	189	2196	06/03/2009	0
3010	189	2187	06/06/2009	0
3011	189	2188	06/10/2009	0
3012	189	2189	06/25/2009	0
3013	189	2190	07/11/2009	0
3014	189	2191	07/24/2009	0
3015	189	2192	07/24/2009	0
3016	189	2193	07/24/2009	0
3017	189	2194	07/24/2009	0
3018	189	2195	09/21/2009	0
3019	189	2186		0
3020	189	2299		0
3021	189	2166	11/14/2009	0
3022	189	2299		1
3023	189	2299		2
3024	189	2299		3
3025	190	2196	06/02/2009	0
3026	190	2187	06/10/2009	0
3027	190	2188	06/25/2009	0
3028	190	2189	06/25/2009	0
3029	190	2190	07/11/2009	0
3030	190	2191	08/07/2009	0
3031	190	2192	08/07/2009	0
3032	190	2193	08/18/2009	0
3033	190	2194		0
3034	190	2195		0
3035	190	2186		0
3036	190	2299		0
3037	190	2166	11/13/2009	0
3038	190	2299		1
3039	190	2299		2
3040	190	2299		3
3041	191	2196	06/03/2009	0
3042	191	2187	06/10/2009	0
3043	191	2188	06/14/2009	0
3044	191	2189	06/25/2009	0
3045	191	2190	07/24/2009	0
3046	191	2191	07/24/2009	0
3047	191	2192	07/24/2009	0
3048	191	2193	07/24/2009	0
3049	191	2194	08/25/2009	0
3050	191	2195	09/26/2009	0
3051	191	2186		0
3052	191	2299		0
3053	191	2166	11/13/2009	0
3054	191	2299		1
3055	191	2299		2
3056	191	2299		3
3057	192	2196	05/27/2009	0
3058	192	2187	06/04/2009	0
3059	192	2188	06/10/2009	0
3060	192	2189	06/14/2009	0
3061	192	2190	07/11/2009	0
3062	192	2191	07/24/2009	0
3063	192	2192	07/24/2009	0
3064	192	2193	07/24/2009	0
3065	192	2194	07/24/2009	0
3066	192	2195	09/01/2009	0
3067	192	2186		0
3068	192	2299		0
3069	192	2166	11/13/2009	0
3070	192	2299	3 dead stems	1
3071	192	2299		2
3072	192	2299		3
3073	193	2196	05/27/2009	0
3074	193	2187	06/04/2009	0
3075	193	2188	06/10/2009	0
3076	193	2189	06/14/2009	0
3077	193	2190	07/24/2009	0
3078	193	2191	07/11/2009	0
3079	193	2192	07/24/2009	0
3080	193	2193	07/24/2009	0
3081	193	2194	07/22/2009	0
3082	193	2195	09/12/2009	0
3083	193	2186		0
3084	193	2299		0
3085	193	2166	11/14/2009	0
3086	193	2299		1
3087	193	2299		2
3088	193	2299		3
3089	194	2196	05/27/2009	0
3090	194	2187	06/06/2009	0
3091	194	2188	06/10/2009	0
3092	194	2189	06/14/2009	0
3093	194	2190	07/24/2009	0
3094	194	2191	08/07/2009	0
3095	194	2192	07/24/2009	0
3096	194	2193	08/18/2009	0
3097	194	2194	08/07/2009	0
3098	194	2195	09/21/2009	0
3099	194	2186		0
3100	194	2299		0
3101	194	2166	11/14/2009	0
3102	194	2299		1
3103	194	2299		2
3104	194	2299		3
3105	195	2196	05/27/2009	0
3106	195	2187	06/06/2009	0
3107	195	2188	06/10/2009	0
3108	195	2189	06/14/2009	0
3109	195	2190	07/11/2009	0
3110	195	2191	07/24/2009	0
3111	195	2192	07/24/2009	0
3112	195	2193	07/11/2009	0
3113	195	2194	08/18/2009	0
3114	195	2195	09/21/2009	0
3115	195	2186		0
3116	195	2299		0
3117	195	2166	11/14/2009	0
3118	195	2299		1
3119	195	2299		2
3120	195	2299		3
3121	196	2196	05/27/2009	0
3122	196	2187	06/06/2009	0
3123	196	2188	06/25/2009	0
3124	196	2189	06/25/2009	0
3125	196	2190	07/11/2009	0
3126	196	2191	07/03/2009	0
3127	196	2192	07/24/2009	0
3128	196	2193	07/24/2009	0
3129	196	2194	07/24/2009	0
3130	196	2195	09/22/2009	0
3131	196	2186		0
3132	196	2299		0
3133	196	2166	11/13/2009	0
3134	196	2299		1
3135	196	2299		2
3136	196	2299		3
3137	197	2196	05/27/2009	0
3138	197	2187	06/02/2009	0
3139	197	2188	06/06/2009	0
3140	197	2189	06/10/2009	0
3141	197	2190	06/14/2009	0
3142	197	2191	07/03/2009	0
3143	197	2192	07/24/2009	0
3144	197	2193	07/24/2009	0
3145	197	2194	07/22/2009	0
3146	197	2195	09/01/2009	0
3147	197	2186		0
3148	197	2299		0
3149	197	2166	11/14/2009	0
3150	197	2299		1
3151	197	2299		2
3152	197	2299		3
3153	198	2196	05/27/2009	0
3154	198	2187	06/04/2009	0
3155	198	2188	06/10/2009	0
3156	198	2189	06/25/2009	0
3157	198	2190	07/11/2009	0
3158	198	2191	07/24/2009	0
3159	198	2192	07/24/2009	0
3160	198	2193	08/07/2009	0
3161	198	2194	07/22/2009	0
3162	198	2195	09/18/2009	0
3163	198	2186		0
3164	198	2299		0
3165	198	2166	11/13/2009	0
3166	198	2299		1
3167	198	2299		2
3168	198	2299		3
3169	199	2196	06/10/2009	0
3170	199	2187	06/14/2009	0
3171	199	2188	06/25/2009	0
3172	199	2189	06/25/2009	0
3173	199	2190	07/24/2009	0
3174	199	2191	07/24/2009	0
3175	199	2192	07/24/2009	0
3176	199	2193	07/24/2009	0
3177	199	2194	08/07/2009	0
3178	199	2195	09/08/2009	0
3179	199	2186	09/29/2009	0
3180	199	2299		0
3181	199	2166	11/13/2009	0
3182	199	2299		1
3183	199	2299		2
3184	199	2299	dead plant harvested 10/4/09	3
3185	200	2196	05/26/2009	0
3186	200	2187	06/02/2009	0
3187	200	2188	06/10/2009	0
3188	200	2189	06/14/2009	0
3189	200	2190	07/11/2009	0
3190	200	2191	07/03/2009	0
3191	200	2192	07/24/2009	0
3192	200	2193	07/24/2009	0
3193	200	2194	07/22/2009	0
3194	200	2195	09/08/2009	0
3195	200	2186		0
3196	200	2299		0
3197	200	2166	11/14/2009	0
3198	200	2299		1
3199	200	2299		2
3200	200	2299		3
3201	201	2196	06/03/2009	0
3202	201	2187	06/10/2009	0
3203	201	2188	06/25/2009	0
3204	201	2189	.	0
3205	201	2190	07/11/2009	0
3206	201	2191	06/25/2009	0
3207	201	2192	06/25/2009	0
3208	201	2193	.	0
3209	201	2194	.	0
3210	201	2195	.	0
3211	201	2186	07/11/2009	0
3212	201	2299		0
3213	201	2166	11/10/2009	0
3214	201	2299		1
3215	201	2299		2
3216	201	2299		3
3217	202	2196	06/02/2009	0
3218	202	2187	06/10/2009	0
3219	202	2188	06/14/2009	0
3220	202	2189	06/25/2009	0
3221	202	2190	07/11/2009	0
3222	202	2191	07/03/2009	0
3223	202	2192	07/24/2009	0
3224	202	2193	07/24/2009	0
3225	202	2194	08/25/2009	0
3226	202	2195	10/25/2009	0
3227	202	2186		0
3228	202	2299		0
3229	202	2166	11/17/2009	0
3230	202	2299		1
3231	202	2299		2
3232	202	2299		3
3233	203	2196	05/26/2009	0
3234	203	2187	06/02/2009	0
3235	203	2188	06/06/2009	0
3236	203	2189	06/14/2009	0
3237	203	2190	06/25/2009	0
3238	203	2191	07/03/2009	0
3239	203	2192	07/24/2009	0
3240	203	2193	07/24/2009	0
3241	203	2194	08/07/2009	0
3242	203	2195	09/30/2009	0
3243	203	2186	09/29/2009	0
3244	203	2299		0
3245	203	2166	11/17/2009	0
3246	203	2299		1
3247	203	2299		2
3248	203	2299	dead plant harvested 10/4/09	3
3249	204	2196	05/27/2009	0
3250	204	2187	06/06/2009	0
3251	204	2188	06/10/2009	0
3252	204	2189	06/25/2009	0
3253	204	2190	07/11/2009	0
3254	204	2191	07/24/2009	0
3255	204	2192	07/24/2009	0
3256	204	2193	08/07/2009	0
3257	204	2194	07/24/2009	0
3258	204	2195	09/12/2009	0
3259	204	2186		0
3260	204	2299		0
3261	204	2166	11/10/2009	0
3262	204	2299		1
3263	204	2299		2
3264	204	2299		3
3265	205	2196	05/27/2009	0
3266	205	2187	06/02/2009	0
3267	205	2188	06/06/2009	0
3268	205	2189	06/14/2009	0
3269	205	2190	06/25/2009	0
3270	205	2191	07/11/2009	0
3271	205	2192	07/24/2009	0
3272	205	2193	07/24/2009	0
3273	205	2194	09/29/2009	0
3274	205	2195	10/25/2009	0
3275	205	2186		0
3276	205	2299		0
3277	205	2166	11/17/2009	0
3278	205	2299		1
3279	205	2299	missing bottom root sample	2
3280	205	2299		3
3281	206	2196	05/27/2009	0
3282	206	2187	06/02/2009	0
3283	206	2188	06/06/2009	0
3284	206	2189	06/14/2009	0
3285	206	2190	06/25/2009	0
3286	206	2191	07/03/2009	0
3287	206	2192	07/24/2009	0
3288	206	2193	07/24/2009	0
3289	206	2194	08/18/2009	0
3290	206	2195	09/22/2009	0
3291	206	2186		0
3292	206	2299		0
3293	206	2166	11/10/2009	0
3294	206	2299		1
3295	206	2299		2
3296	206	2299		3
3297	207	2196	05/28/2009	0
3298	207	2187	06/10/2009	0
3299	207	2188	06/14/2009	0
3300	207	2189	06/25/2009	0
3301	207	2190	07/11/2009	0
3302	207	2191	07/03/2009	0
3303	207	2192	07/24/2009	0
3304	207	2193	07/24/2009	0
3305	207	2194	08/07/2009	0
3306	207	2195	09/25/2009	0
3307	207	2186		0
3308	207	2299		0
3309	207	2166	11/17/2009	0
3310	207	2299		1
3311	207	2299		2
3312	207	2299		3
3313	208	2196	05/26/2009	0
3314	208	2187	06/04/2009	0
3315	208	2188	06/06/2009	0
3316	208	2189	06/14/2009	0
3317	208	2190	07/11/2009	0
3318	208	2191	07/11/2009	0
3319	208	2192	07/24/2009	0
3320	208	2193	07/24/2009	0
3321	208	2194	07/22/2009	0
3322	208	2195	08/28/2009	0
3323	208	2186		0
3324	208	2299		0
3325	208	2166	11/10/2009	0
3326	208	2299		1
3327	208	2299		2
3328	208	2299		3
3329	209	2196	05/27/2009	0
3330	209	2187	06/04/2009	0
3331	209	2188	06/10/2009	0
3332	209	2189	06/14/2009	0
3333	209	2190	06/25/2009	0
3334	209	2191	07/03/2009	0
3335	209	2192	07/24/2009	0
3336	209	2193	07/24/2009	0
3337	209	2194	08/07/2009	0
3338	209	2195	09/03/2009	0
3339	209	2186		0
3340	209	2299		0
3341	209	2166	11/10/2009	0
3342	209	2299		1
3343	209	2299		2
3344	209	2299		3
3345	210	2196	05/26/2009	0
3346	210	2187	06/04/2009	0
3347	210	2188	06/06/2009	0
3348	210	2189	06/10/2009	0
3349	210	2190	07/11/2009	0
3350	210	2191	07/03/2009	0
3351	210	2192	07/11/2009	0
3352	210	2193	07/11/2009	0
3353	210	2194	07/22/2009	0
3354	210	2195	09/12/2009	0
3355	210	2186		0
3356	210	2299		0
3357	210	2166	11/10/2009	0
3358	210	2299		1
3359	210	2299	missing shoot sample	2
3360	210	2299		3
3361	211	2196	05/26/2009	0
3362	211	2187	06/02/2009	0
3363	211	2188	06/06/2009	0
3364	211	2189	06/10/2009	0
3365	211	2190	07/11/2009	0
3366	211	2191	07/11/2009	0
3367	211	2192	07/24/2009	0
3368	211	2193	07/24/2009	0
3369	211	2194	07/24/2009	0
3370	211	2195	08/22/2009	0
3371	211	2186		0
3372	211	2299		0
3373	211	2166	11/10/2009	0
3374	211	2299		1
3375	211	2299		2
3376	211	2299		3
3377	212	2196	05/26/2009	0
3378	212	2187	06/02/2009	0
3379	212	2188	06/06/2009	0
3380	212	2189	06/14/2009	0
3381	212	2190	06/25/2009	0
3382	212	2191	06/14/2009	0
3383	212	2192	07/11/2009	0
3384	212	2193	07/24/2009	0
3385	212	2194	08/07/2009	0
3386	212	2195	09/12/2009	0
3387	212	2186		0
3388	212	2299		0
3389	212	2166	11/10/2009	0
3390	212	2299		1
3391	212	2299		2
3392	212	2299		3
3393	213	2196	05/26/2009	0
3394	213	2187	06/02/2009	0
3395	213	2188	06/06/2009	0
3396	213	2189	06/14/2009	0
3397	213	2190	07/11/2009	0
3398	213	2191	07/24/2009	0
3399	213	2192	07/24/2009	0
3400	213	2193	07/24/2009	0
3401	213	2194	08/07/2009	0
3402	213	2195	09/30/2009	0
3403	213	2186		0
3404	213	2299		0
3405	213	2166	11/17/2009	0
3406	213	2299	4 dead stems	1
3407	213	2299		2
3408	213	2299		3
3409	214	2196	05/28/2009	0
3410	214	2187	06/04/2009	0
3411	214	2188	06/10/2009	0
3412	214	2189	06/25/2009	0
3413	214	2190	06/25/2009	0
3414	214	2191	07/24/2009	0
3415	214	2192	08/07/2009	0
3416	214	2193	07/24/2009	0
3417	214	2194	07/24/2009	0
3418	214	2195	10/25/2009	0
3419	214	2186		0
3420	214	2299		0
3421	214	2166	11/17/2009	0
3422	214	2299		1
3423	214	2299		2
3424	214	2299		3
3425	215	2196	05/26/2009	0
3426	215	2187	06/02/2009	0
3427	215	2188	06/06/2009	0
3428	215	2189	06/10/2009	0
3429	215	2190	06/25/2009	0
3430	215	2191	07/03/2009	0
3431	215	2192	06/25/2009	0
3432	215	2193	07/11/2009	0
3433	215	2194	08/07/2009	0
3434	215	2195	09/21/2009	0
3435	215	2186		0
3436	215	2299		0
3437	215	2166	11/17/2009	0
3438	215	2299		1
3439	215	2299		2
3440	215	2299		3
3441	216	2196	05/27/2009	0
3442	216	2187	06/02/2009	0
3443	216	2188	06/06/2009	0
3444	216	2189	06/10/2009	0
3445	216	2190	07/11/2009	0
3446	216	2191	06/25/2009	0
3447	216	2192	06/25/2009	0
3448	216	2193	07/11/2009	0
3449	216	2194	08/18/2009	0
3450	216	2195		0
3451	216	2186		0
3452	216	2299		0
3453	216	2166	11/17/2009	0
3454	216	2299		1
3455	216	2299		2
3456	216	2299		3
3457	217	2196	06/03/2009	0
3458	217	2187	06/06/2009	0
3459	217	2188	06/10/2009	0
3460	217	2189	06/25/2009	0
3461	217	2190	07/24/2009	0
3462	217	2191	07/24/2009	0
3463	217	2192	07/24/2009	0
3464	217	2193	07/24/2009	0
3465	217	2194	08/18/2009	0
3466	217	2195	09/21/2009	0
3467	217	2186		0
3468	217	2299		0
3469	217	2166	11/17/2009	0
3470	217	2299		1
3471	217	2299		2
3472	217	2299		3
3473	218	2196	05/28/2009	0
3474	218	2187	06/04/2009	0
3475	218	2188	06/10/2009	0
3476	218	2189	06/14/2009	0
3477	218	2190	07/11/2009	0
3478	218	2191	07/11/2009	0
3479	218	2192	07/24/2009	0
3480	218	2193	07/24/2009	0
3481	218	2194	07/22/2009	0
3482	218	2195	09/12/2009	0
3483	218	2186		0
3484	218	2299		0
3485	218	2166	11/17/2009	0
3486	218	2299		1
3487	218	2299		2
3488	218	2299		3
3489	219	2196	05/26/2009	0
3490	219	2187	06/02/2009	0
3491	219	2188	06/06/2009	0
3492	219	2189	06/14/2009	0
3493	219	2190	07/11/2009	0
3494	219	2191	06/25/2009	0
3495	219	2192	07/24/2009	0
3496	219	2193	07/11/2009	0
3497	219	2194	08/07/2009	0
3498	219	2195	09/03/2009	0
3499	219	2186		0
3500	219	2299		0
3501	219	2166	11/10/2009	0
3502	219	2299	3 dead stems	1
3503	219	2299		2
3504	219	2299		3
3505	220	2196	05/26/2009	0
3506	220	2187	06/04/2009	0
3507	220	2188	06/06/2009	0
3508	220	2189	06/10/2009	0
3509	220	2190	07/11/2009	0
3510	220	2191	07/03/2009	0
3511	220	2192	07/11/2009	0
3512	220	2193	07/24/2009	0
3513	220	2194	07/22/2009	0
3514	220	2195	09/01/2009	0
3515	220	2186		0
3516	220	2299		0
3517	220	2166	11/10/2009	0
3518	220	2299		1
3519	220	2299		2
3520	220	2299		3
3521	221	2196	06/03/2009	0
3522	221	2187	06/10/2009	0
3523	221	2188	06/14/2009	0
3524	221	2189	06/25/2009	0
3525	221	2190	07/11/2009	0
3526	221	2191	08/07/2009	0
3527	221	2192	08/18/2009	0
3528	221	2193	08/18/2009	0
3529	221	2194	08/18/2009	0
3530	221	2195	09/22/2009	0
3531	221	2186		0
3532	221	2299		0
3533	221	2166	11/11/2009	0
3534	221	2299		1
3535	221	2299	missing shoot sample	2
3536	221	2299		3
3537	222	2196	06/03/2009	0
3538	222	2187	06/06/2009	0
3539	222	2188	06/10/2009	0
3540	222	2189	06/25/2009	0
3541	222	2190	07/11/2009	0
3542	222	2191	07/03/2009	0
3543	222	2192	07/24/2009	0
3544	222	2193	07/24/2009	0
3545	222	2194	08/25/2009	0
3546	222	2195	10/25/2009	0
3547	222	2186		0
3548	222	2299		0
3549	222	2166	11/17/2009	0
3550	222	2299		1
3551	222	2299		2
3552	222	2299		3
3553	223	2196	05/27/2009	0
3554	223	2187	06/06/2009	0
3555	223	2188	06/10/2009	0
3556	223	2189	06/10/2009	0
3557	223	2190	07/11/2009	0
3558	223	2191	07/11/2009	0
3559	223	2192	07/24/2009	0
3560	223	2193	08/07/2009	0
3561	223	2194	07/24/2009	0
3562	223	2195	09/22/2009	0
3563	223	2186		0
3564	223	2299		0
3565	223	2166	11/11/2009	0
3566	223	2299		1
3567	223	2299		2
3568	223	2299		3
3569	224	2196	06/06/2009	0
3570	224	2187	06/10/2009	0
3571	224	2188	06/14/2009	0
3572	224	2189	06/25/2009	0
3573	224	2190	07/11/2009	0
3574	224	2191	07/11/2009	0
3575	224	2192	08/07/2009	0
3576	224	2193	08/18/2009	0
3577	224	2194	08/07/2009	0
3578	224	2195	09/29/2009	0
3579	224	2186		0
3580	224	2299		0
3581	224	2166	11/11/2009	0
3582	224	2299		1
3583	224	2299		2
3584	224	2299		3
3585	225	2196	05/28/2009	0
3586	225	2187	06/06/2009	0
3587	225	2188	06/10/2009	0
3588	225	2189	06/14/2009	0
3589	225	2190	07/11/2009	0
3590	225	2191	07/11/2009	0
3591	225	2192	07/24/2009	0
3592	225	2193	07/24/2009	0
3593	225	2194	07/24/2009	0
3594	225	2195	09/22/2009	0
3595	225	2186		0
3596	225	2299		0
3597	225	2166	11/11/2009	0
3598	225	2299		1
3599	225	2299		2
3600	225	2299		3
3601	226	2196	05/27/2009	0
3602	226	2187	06/04/2009	0
3603	226	2188	06/10/2009	0
3604	226	2189	06/14/2009	0
3605	226	2190	07/11/2009	0
3606	226	2191	06/25/2009	0
3607	226	2192	07/24/2009	0
3608	226	2193	07/24/2009	0
3609	226	2194	07/24/2009	0
3610	226	2195	09/12/2009	0
3611	226	2186		0
3612	226	2299		0
3613	226	2166	11/17/2009	0
3614	226	2299		1
3615	226	2299		2
3616	226	2299		3
3617	227	2196	05/26/2009	0
3618	227	2187	06/04/2009	0
3619	227	2188	06/14/2009	0
3620	227	2189	06/25/2009	0
3621	227	2190	06/14/2009	0
3622	227	2191	07/03/2009	0
3623	227	2192	08/07/2009	0
3624	227	2193	08/18/2009	0
3625	227	2194	08/18/2009	0
3626	227	2195	09/01/2009	0
3627	227	2186		0
3628	227	2299		0
3629	227	2166	11/11/2009	0
3630	227	2299		1
3631	227	2299		2
3632	227	2299		3
3633	228	2196	06/03/2009	0
3634	228	2187	06/06/2009	0
3635	228	2188	06/10/2009	0
3636	228	2189	06/25/2009	0
3637	228	2190	07/11/2009	0
3638	228	2191	07/03/2009	0
3639	228	2192	07/11/2009	0
3640	228	2193	07/24/2009	0
3641	228	2194	08/07/2009	0
3642	228	2195	09/21/2009	0
3643	228	2186		0
3644	228	2299		0
3645	228	2166	11/17/2009	0
3646	228	2299		1
3647	228	2299		2
3648	228	2299		3
3649	229	2196	05/26/2009	0
3650	229	2187	06/04/2009	0
3651	229	2188	06/06/2009	0
3652	229	2189	06/10/2009	0
3653	229	2190	07/11/2009	0
3654	229	2191	06/25/2009	0
3655	229	2192	07/11/2009	0
3656	229	2193	07/24/2009	0
3657	229	2194	07/24/2009	0
3658	229	2195	09/03/2009	0
3659	229	2186		0
3660	229	2299		0
3661	229	2166	11/17/2009	0
3662	229	2299		1
3663	229	2299		2
3664	229	2299		3
3665	230	2196	05/27/2009	0
3666	230	2187	06/14/2009	0
3667	230	2188	06/25/2009	0
3668	230	2189	06/25/2009	0
3669	230	2190	07/24/2009	0
3670	230	2191	06/25/2009	0
3671	230	2192	08/07/2009	0
3672	230	2193	08/18/2009	0
3673	230	2194	.	0
3674	230	2195	.	0
3675	230	2186	09/29/2009	0
3676	230	2299		0
3677	230	2166	11/11/2009	0
3678	230	2299		1
3679	230	2299		2
3680	230	2299	dead plant harvested 10/4/09	3
3681	231	2196	06/03/2009	0
3682	231	2187	06/10/2009	0
3683	231	2188	06/14/2009	0
3684	231	2189	06/25/2009	0
3685	231	2190	07/11/2009	0
3686	231	2191	07/24/2009	0
3687	231	2192	07/24/2009	0
3688	231	2193	08/07/2009	0
3689	231	2194	08/07/2009	0
3690	231	2195	09/22/2009	0
3691	231	2186	09/29/2009	0
3692	231	2299		0
3693	231	2166	11/11/2009	0
3694	231	2299		1
3695	231	2299		2
3696	231	2299	dead plant harvested 10/4/09	3
3697	232	2196	06/03/2009	0
3698	232	2187	06/10/2009	0
3699	232	2188	06/25/2009	0
3700	232	2189	06/25/2009	0
3701	232	2190	07/11/2009	0
3702	232	2191	07/11/2009	0
3703	232	2192	07/24/2009	0
3704	232	2193	08/07/2009	0
3705	232	2194	08/18/2009	0
3706	232	2195	09/30/2009	0
3707	232	2186		0
3708	232	2299		0
3709	232	2166	11/17/2009	0
3710	232	2299		1
3711	232	2299		2
3712	232	2299		3
3713	233	2196	05/27/2009	0
3714	233	2187	06/06/2009	0
3715	233	2188	06/06/2009	0
3716	233	2189	06/10/2009	0
3717	233	2190	07/11/2009	0
3718	233	2191	07/11/2009	0
3719	233	2192	07/24/2009	0
3720	233	2193	07/24/2009	0
3721	233	2194	07/24/2009	0
3722	233	2195	09/21/2009	0
3723	233	2186		0
3724	233	2299		0
3725	233	2166	11/17/2009	0
3726	233	2299		1
3727	233	2299		2
3728	233	2299		3
3729	234	2196	05/26/2009	0
3730	234	2187	06/02/2009	0
3731	234	2188	06/06/2009	0
3732	234	2189	06/10/2009	0
3733	234	2190	06/25/2009	0
3734	234	2191	07/03/2009	0
3735	234	2192	07/11/2009	0
3736	234	2193	07/24/2009	0
3737	234	2194	07/22/2009	0
3738	234	2195	09/12/2009	0
3739	234	2186		0
3740	234	2299		0
3741	234	2166	11/17/2009	0
3742	234	2299		1
3743	234	2299		2
3744	234	2299		3
3745	235	2196	05/26/2009	0
3746	235	2187	06/04/2009	0
3747	235	2188	06/10/2009	0
3748	235	2189	06/10/2009	0
3749	235	2190	06/25/2009	0
3750	235	2191	07/11/2009	0
3751	235	2192	07/24/2009	0
3752	235	2193	07/24/2009	0
3753	235	2194	07/22/2009	0
3754	235	2195	09/12/2009	0
3755	235	2186		0
3756	235	2299		0
3757	235	2166	11/17/2009	0
3758	235	2299		1
3759	235	2299		2
3760	235	2299		3
3761	236	2196	05/26/2009	0
3762	236	2187	06/02/2009	0
3763	236	2188	06/06/2009	0
3764	236	2189	06/14/2009	0
3765	236	2190	06/25/2009	0
3766	236	2191	07/11/2009	0
3767	236	2192	07/24/2009	0
3768	236	2193	07/24/2009	0
3769	236	2194	07/22/2009	0
3770	236	2195	09/12/2009	0
3771	236	2186		0
3772	236	2299		0
3773	236	2166	11/11/2009	0
3774	236	2299		1
3775	236	2299		2
3776	236	2299		3
3777	237	2196	05/27/2009	0
3778	237	2187	06/04/2009	0
3779	237	2188	06/06/2009	0
3780	237	2189	06/10/2009	0
3781	237	2190	07/11/2009	0
3782	237	2191	07/24/2009	0
3783	237	2192	07/24/2009	0
3784	237	2193	07/24/2009	0
3785	237	2194	07/22/2009	0
3786	237	2195	09/12/2009	0
3787	237	2186		0
3788	237	2299		0
3789	237	2166	11/11/2009	0
3790	237	2299		1
3791	237	2299	missing shoot sample	2
3792	237	2299		3
3793	238	2196	05/27/2009	0
3794	238	2187	06/06/2009	0
3795	238	2188	06/10/2009	0
3796	238	2189	06/25/2009	0
3797	238	2190	07/11/2009	0
3798	238	2191	07/24/2009	0
3799	238	2192	07/24/2009	0
3800	238	2193	08/07/2009	0
3801	238	2194	08/18/2009	0
3802	238	2195	09/25/2009	0
3803	238	2186		0
3804	238	2299		0
3805	238	2166	11/17/2009	0
3806	238	2299		1
3807	238	2299		2
3808	238	2299		3
3809	239	2196	05/26/2009	0
3810	239	2187	06/04/2009	0
3811	239	2188	06/10/2009	0
3812	239	2189	06/14/2009	0
3813	239	2190	07/11/2009	0
3814	239	2191	07/03/2009	0
3815	239	2192	07/24/2009	0
3816	239	2193	07/24/2009	0
3817	239	2194	07/22/2009	0
3818	239	2195	09/12/2009	0
3819	239	2186		0
3820	239	2299		0
3821	239	2166	11/11/2009	0
3822	239	2299		1
3823	239	2299		2
3824	239	2299		3
3825	240	2196	05/28/2009	0
3826	240	2187	06/04/2009	0
3827	240	2188	06/10/2009	0
3828	240	2189	06/25/2009	0
3829	240	2190	06/25/2009	0
3830	240	2191	07/03/2009	0
3831	240	2192	08/07/2009	0
3832	240	2193	08/18/2009	0
3833	240	2194	07/24/2009	0
3834	240	2195	09/12/2009	0
3835	240	2186		0
3836	240	2299		0
3837	240	2166	11/17/2009	0
3838	240	2299		1
3839	240	2299		2
3840	240	2299		3
\.


--
-- Data for Name: nd_geolocation; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_geolocation (nd_geolocation_id, description, latitude, longitude, geodetic_datum, altitude) FROM stdin;
3	\N	\N	\N	\N	\N
\.


--
-- Data for Name: nd_geolocationprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_geolocationprop (nd_geolocationprop_id, nd_geolocation_id, type_id, value, rank) FROM stdin;
7	3	2266	Davis	0
8	3	2267	CA	0
9	3	2268	95616	0
\.


--
-- Data for Name: nd_protocol; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_protocol (nd_protocol_id, name) FROM stdin;
\.


--
-- Data for Name: nd_protocol_reagent; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_protocol_reagent (nd_protocol_reagent_id, nd_protocol_id, reagent_id, type_id) FROM stdin;
\.


--
-- Data for Name: nd_protocolprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_protocolprop (nd_protocolprop_id, nd_protocol_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: nd_reagent; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_reagent (nd_reagent_id, name, type_id, feature_id) FROM stdin;
\.


--
-- Data for Name: nd_reagent_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_reagent_relationship (nd_reagent_relationship_id, subject_reagent_id, object_reagent_id, type_id) FROM stdin;
\.


--
-- Data for Name: nd_reagentprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY nd_reagentprop (nd_reagentprop_id, nd_reagent_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: organism; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY organism (organism_id, abbreviation, genus, species, common_name, comment) FROM stdin;
15	M.truncatula	Medicago	truncatula	\N	\N
\.


--
-- Data for Name: organism_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY organism_dbxref (organism_dbxref_id, organism_id, dbxref_id) FROM stdin;
\.


--
-- Data for Name: organismprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY organismprop (organismprop_id, organism_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: phendesc; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phendesc (phendesc_id, genotype_id, environment_id, description, type_id, pub_id) FROM stdin;
\.


--
-- Data for Name: phenotype; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phenotype (phenotype_id, uniquename, observable_id, attr_id, value, cvalue_id, assay_id) FROM stdin;
38	reproductive unit number at flowering in_units_of count unit	\N	\N	\N	\N	\N
39	fruit number at fruiting in_units_of count unit	\N	\N	\N	\N	\N
40	alive leaf number at 6/5/2009 in_units_of count unit	\N	\N	\N	\N	\N
41	leaf number at 6/9/2009 in_units_of count unit	\N	\N	\N	\N	\N
42	first leaf length at 6/9/2009 in_units_of centimeter	\N	\N	\N	\N	\N
43	first leaf width at 6/9/2009 in_units_of centimeter	\N	\N	\N	\N	\N
44	second leaf length at 6/9/2009 in_units_of centimeter	\N	\N	\N	\N	\N
45	second leaf width at 6/9/2009 in_units_of centimeter	\N	\N	\N	\N	\N
46	alive leaf number at 6/13/2009 in_units_of count unit	\N	\N	\N	\N	\N
47	branch number at 6/13/2009 in_units_of count unit	\N	\N	\N	\N	\N
48	cotyledon length at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
49	cotyledon width at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
50	first leaf length at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
51	first leaf width at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
52	second leaf length at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
53	second leaf width at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
54	third leaf length at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
55	third leaf width at 6/13/2009 in_units_of centimeter	\N	\N	\N	\N	\N
56	cotyledon number at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
57	leaf number at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
58	branch number at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
59	primary branch position at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
60	B0 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
61	B01 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
62	B02 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
63	B03 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
64	B04 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
65	B05 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
66	B06 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
67	B011 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
68	B012 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
69	B021 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
70	B022 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
71	B031 at 6/24/2009 in_units_of count unit	\N	\N	\N	\N	\N
72	leaf category for length and width measurements	\N	\N	\N	\N	\N
73	leaf length at 6/24/2009 in_units_of centimeter	\N	\N	\N	\N	\N
74	leaf width at 6/24/2009 in_units_of centimeter	\N	\N	\N	\N	\N
75	primary branch length at 7/23/209 in_units_of centimeter	\N	\N	\N	\N	\N
76	secondary branch number at 7/23/2009 in_units_of count unit	\N	\N	\N	\N	\N
77	secondary branch length at 7/23/2009 in_units_of centimeter	\N	\N	\N	\N	\N
78	tertiary branch number at 7/23/2009 in_units_of count unit	\N	\N	\N	\N	\N
79	tertiary branch length at 7/23/2009 in_units_of centimeter	\N	\N	\N	\N	\N
80	leaf number at 7/23/2009 in_units_of count unit	\N	\N	\N	\N	\N
81	leaf length at 7/23/2009 in_units_of centimeter	\N	\N	\N	\N	\N
82	leaf width at 7/23/2009 in_units_of centimeter	\N	\N	\N	\N	\N
83	flower number at 7/23/2009 in_units_of count unit	\N	\N	\N	\N	\N
84	bud number at 7/23/2009 in_units_of count unit	\N	\N	\N	\N	\N
85	fruit number at 7/23/2009 in_units_of count unit	\N	\N	\N	\N	\N
86	branch length max at 10/2/2009 in_units_of centimeter	\N	\N	\N	\N	\N
87	alive leaf number at 10/2/2009 in_units_of count unit	\N	\N	\N	\N	\N
88	leaf number at 10/2/2009 in_units_of count unit	\N	\N	\N	\N	\N
89	flower number at 10/2/2009 in_units_of count unit	\N	\N	\N	\N	\N
90	fruit number at 10/2/2009 in_units_of count unit	\N	\N	\N	\N	\N
91	plant condition at 10/2/2009 in_units_of percent	\N	\N	\N	\N	\N
92	dead stem number at 10/2/2009 in_units_of count unit	\N	\N	\N	\N	\N
93	leaf area at 10/5/2009 (square cm)	\N	\N	\N	\N	\N
94	leaf fresh weight at 10/5/2009 in_units_of milligram	\N	\N	\N	\N	\N
95	leaf dry weight at 10/5/2009 in_units_of milligram	\N	\N	\N	\N	\N
96	alive leaf number at 11/10/2009 in_units_of count unit	\N	\N	\N	\N	\N
97	leaf number at 11/10/2009 in_units_of count unit	\N	\N	\N	\N	\N
98	stem diameter at 11/10/2009 in_units_of millimolal	\N	\N	\N	\N	\N
99	branch length max at 11/10/2009 in_units_of centimeter	\N	\N	\N	\N	\N
100	root length at 11/10/2009 in_units_of centimeter	\N	\N	\N	\N	\N
101	root diameter at 11/10/2009 in_units_of millimolal	\N	\N	\N	\N	\N
102	nodule number inside pot at 11/10/2009 in_units_of count unit	\N	\N	\N	\N	\N
103	nodule number outside pot at 11/10/2009 in_units_of count unit	\N	\N	\N	\N	\N
104	plant condition at 11/10/2009 in_units_of percent	\N	\N	\N	\N	\N
105	thrip at 11/10/2009 in_units_of percent	\N	\N	\N	\N	\N
106	powdery mildew at 11/10/2009 in_units_of percent	\N	\N	\N	\N	\N
107	mature fruit number at harvest date in_units_of count unit	\N	\N	\N	\N	\N
108	green fruit number at harvest date in_units_of count unit	\N	\N	\N	\N	\N
109	flower number at harvest date in_units_of count unit	\N	\N	\N	\N	\N
110	alive leaf number at harvest date in_units_of count unit	\N	\N	\N	\N	\N
111	leaf number at harvest date in_units_of count unit	\N	\N	\N	\N	\N
112	stem diameter at harvest date in_units_of millimolal	\N	\N	\N	\N	\N
113	branch length max at harvest date in_units_of centimeter	\N	\N	\N	\N	\N
114	root length at harvest date in_units_of centimeter	\N	\N	\N	\N	\N
115	root diameter at harvest date in_units_of millimolal	\N	\N	\N	\N	\N
116	nodule number inside pot at harvest date in_units_of count unit	\N	\N	\N	\N	\N
117	nodule number outside pot at harvest date in_units_of count unit	\N	\N	\N	\N	\N
118	plant condition at harvest date in_units_of percent	\N	\N	\N	\N	\N
119	thrip at harvest date in_units_of percent	\N	\N	\N	\N	\N
120	powdery mildew at harvest date in_units_of percent	\N	\N	\N	\N	\N
121	stem fresh weight at harvest date in_units_of milligram	\N	\N	\N	\N	\N
122	stem dry weight at harvest date in_units_of milligram	\N	\N	\N	\N	\N
123	root top wet weight at harvest date in_units_of milligram	\N	\N	\N	\N	\N
124	root top dry weight at harvest date in_units_of milligram	\N	\N	\N	\N	\N
125	root bottom wet weight at harvest date in_units_of milligram	\N	\N	\N	\N	\N
126	root bottom dry weight at harvest date in_units_of milligram	\N	\N	\N	\N	\N
127	partial shoot dry weight at harvest date in_units_of gram	\N	\N	\N	\N	\N
128	partial root dry weight at harvest date in_units_of gram	\N	\N	\N	\N	\N
129	shoot dry weight at harvest date in_units_of gram	\N	\N	\N	\N	\N
130	root dry weight at harvest date in_units_of gram	\N	\N	\N	\N	\N
\.


--
-- Data for Name: phenotype_comparison; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phenotype_comparison (phenotype_comparison_id, genotype1_id, environment1_id, genotype2_id, environment2_id, phenotype1_id, phenotype2_id, pub_id, organism_id) FROM stdin;
\.


--
-- Data for Name: phenotype_comparison_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phenotype_comparison_cvterm (phenotype_comparison_cvterm_id, phenotype_comparison_id, cvterm_id, pub_id, rank) FROM stdin;
\.


--
-- Data for Name: phenotype_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phenotype_cvterm (phenotype_cvterm_id, phenotype_id, cvterm_id, rank) FROM stdin;
\.


--
-- Data for Name: phenotypeprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phenotypeprop (phenotypeprop_id, phenotype_id, type_id, value, rank) FROM stdin;
73	38	2305	2194	0
74	38	2306	2077	0
75	39	2305	2195	0
76	39	2306	2077	0
77	40	2305	6/5/2009	0
78	40	2306	2077	0
79	41	2305	6/9/2009	0
80	41	2306	2077	0
81	42	2305	6/9/2009	0
82	42	2306	1902	0
83	43	2305	6/9/2009	0
84	43	2306	1902	0
85	44	2305	6/9/2009	0
86	44	2306	1902	0
87	45	2305	6/9/2009	0
88	45	2306	1902	0
89	46	2305	6/13/2009	0
90	46	2306	2077	0
91	47	2305	6/13/2009	0
92	47	2306	2077	0
93	48	2305	6/13/2009	0
94	48	2306	1902	0
95	49	2305	6/13/2009	0
96	49	2306	1902	0
97	50	2305	6/13/2009	0
98	50	2306	1902	0
99	51	2305	6/13/2009	0
100	51	2306	1902	0
101	52	2305	6/13/2009	0
102	52	2306	1902	0
103	53	2305	6/13/2009	0
104	53	2306	1902	0
105	54	2305	6/13/2009	0
106	54	2306	1902	0
107	55	2305	6/13/2009	0
108	55	2306	1902	0
109	56	2305	6/24/2009	0
110	56	2306	2077	0
111	57	2305	6/24/2009	0
112	57	2306	2077	0
113	58	2305	6/24/2009	0
114	58	2306	2077	0
115	59	2305	6/24/2009	0
116	59	2306	2077	0
117	60	2305	6/24/2009	0
118	60	2306	2077	0
119	61	2305	6/24/2009	0
120	61	2306	2077	0
121	62	2305	6/24/2009	0
122	62	2306	2077	0
123	63	2305	6/24/2009	0
124	63	2306	2077	0
125	64	2305	6/24/2009	0
126	64	2306	2077	0
127	65	2305	6/24/2009	0
128	65	2306	2077	0
129	66	2305	6/24/2009	0
130	66	2306	2077	0
131	67	2305	6/24/2009	0
132	67	2306	2077	0
133	68	2305	6/24/2009	0
134	68	2306	2077	0
135	69	2305	6/24/2009	0
136	69	2306	2077	0
137	70	2305	6/24/2009	0
138	70	2306	2077	0
139	71	2305	6/24/2009	0
140	71	2306	2077	0
141	73	2305	6/24/2009	0
142	73	2306	1902	0
143	74	2305	6/24/2009	0
144	74	2306	1902	0
145	75	2305	7/23/209	0
146	75	2306	1902	0
147	76	2305	7/23/2009	0
148	76	2306	2077	0
149	77	2305	7/23/2009	0
150	77	2306	1902	0
151	78	2305	7/23/2009	0
152	78	2306	2077	0
153	79	2305	7/23/2009	0
154	79	2306	1902	0
155	80	2305	7/23/2009	0
156	80	2306	2077	0
157	81	2305	7/23/2009	0
158	81	2306	1902	0
159	82	2305	7/23/2009	0
160	82	2306	1902	0
161	83	2305	7/23/2009	0
162	83	2306	2077	0
163	84	2305	7/23/2009	0
164	84	2306	2077	0
165	85	2305	7/23/2009	0
166	85	2306	2077	0
167	86	2305	10/2/2009	0
168	86	2306	1902	0
169	87	2305	10/2/2009	0
170	87	2306	2077	0
171	88	2305	10/2/2009	0
172	88	2306	2077	0
173	89	2305	10/2/2009	0
174	89	2306	2077	0
175	90	2305	10/2/2009	0
176	90	2306	2077	0
177	91	2305	10/2/2009	0
178	91	2306	2075	0
179	92	2305	10/2/2009	0
180	92	2306	2077	0
181	93	2305	10/5/2009 (square cm)	0
182	94	2305	10/5/2009	0
183	94	2306	1909	0
184	95	2305	10/5/2009	0
185	95	2306	1909	0
186	96	2305	11/10/2009	0
187	96	2306	2077	0
188	97	2305	11/10/2009	0
189	97	2306	2077	0
190	98	2305	11/10/2009	0
191	98	2306	1956	0
192	99	2305	11/10/2009	0
193	99	2306	1902	0
194	100	2305	11/10/2009	0
195	100	2306	1902	0
196	101	2305	11/10/2009	0
197	101	2306	1956	0
198	102	2305	11/10/2009	0
199	102	2306	2077	0
200	103	2305	11/10/2009	0
201	103	2306	2077	0
202	104	2305	11/10/2009	0
203	104	2306	2075	0
204	105	2305	11/10/2009	0
205	105	2306	2075	0
206	106	2305	11/10/2009	0
207	106	2306	2075	0
208	107	2305	2166	0
209	107	2306	2077	0
210	108	2305	2166	0
211	108	2306	2077	0
212	109	2305	2166	0
213	109	2306	2077	0
214	110	2305	2166	0
215	110	2306	2077	0
216	111	2305	2166	0
217	111	2306	2077	0
218	112	2305	2166	0
219	112	2306	1956	0
220	113	2305	2166	0
221	113	2306	1902	0
222	114	2305	2166	0
223	114	2306	1902	0
224	115	2305	2166	0
225	115	2306	1956	0
226	116	2305	2166	0
227	116	2306	2077	0
228	117	2305	2166	0
229	117	2306	2077	0
230	118	2305	2166	0
231	118	2306	2075	0
232	119	2305	2166	0
233	119	2306	2075	0
234	120	2305	2166	0
235	120	2306	2075	0
236	121	2305	2166	0
237	121	2306	1909	0
238	122	2305	2166	0
239	122	2306	1909	0
240	123	2305	2166	0
241	123	2306	1909	0
242	124	2305	2166	0
243	124	2306	1909	0
244	125	2305	2166	0
245	125	2306	1909	0
246	126	2305	2166	0
247	126	2306	1909	0
248	127	2305	2166	0
249	127	2306	1908	0
250	128	2305	2166	0
251	128	2306	1908	0
252	129	2305	2166	0
253	129	2306	1908	0
254	130	2305	2166	0
255	130	2306	1908	0
\.


--
-- Data for Name: phenstatement; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phenstatement (phenstatement_id, genotype_id, environment_id, phenotype_id, type_id, pub_id) FROM stdin;
\.


--
-- Data for Name: phylonode; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylonode (phylonode_id, phylotree_id, parent_phylonode_id, left_idx, right_idx, type_id, feature_id, label, distance) FROM stdin;
\.


--
-- Data for Name: phylonode_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylonode_dbxref (phylonode_dbxref_id, phylonode_id, dbxref_id) FROM stdin;
\.


--
-- Data for Name: phylonode_organism; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylonode_organism (phylonode_organism_id, phylonode_id, organism_id) FROM stdin;
\.


--
-- Data for Name: phylonode_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylonode_pub (phylonode_pub_id, phylonode_id, pub_id) FROM stdin;
\.


--
-- Data for Name: phylonode_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylonode_relationship (phylonode_relationship_id, subject_id, object_id, type_id, rank, phylotree_id) FROM stdin;
\.


--
-- Data for Name: phylonodeprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylonodeprop (phylonodeprop_id, phylonode_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: phylotree; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylotree (phylotree_id, dbxref_id, name, type_id, analysis_id, comment) FROM stdin;
\.


--
-- Data for Name: phylotree_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY phylotree_pub (phylotree_pub_id, phylotree_id, pub_id) FROM stdin;
\.


--
-- Data for Name: project; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY project (project_id, name, description) FROM stdin;
3	MTE2	First generation of 40 TN lines with NaCl
\.


--
-- Data for Name: project_contact; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY project_contact (project_contact_id, project_id, contact_id) FROM stdin;
\.


--
-- Data for Name: project_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY project_pub (project_pub_id, project_id, pub_id) FROM stdin;
\.


--
-- Data for Name: project_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY project_relationship (project_relationship_id, subject_project_id, object_project_id, type_id) FROM stdin;
\.


--
-- Data for Name: projectprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY projectprop (projectprop_id, project_id, type_id, value, rank) FROM stdin;
78	3	2213	2203	0
79	3	2215	1950	0
80	3	2316	6/24/2009	0
81	3	2316	2166	1
82	3	2316	2166	2
83	3	2316	2166	3
53	3	2217	3	0
54	3	2216	0	0
55	3	2262	05/20/2009	0
56	3	2263	11/18/2009	0
57	3	2264	Ken Moriuchi	0
58	3	2264	Eric von Wettberg	1
59	3	2265	MTE1	0
60	3	2269	The we need to do a selection analysis ASAP because the field experiment bombed experiment.  40 TN lines, two salt levels, easy as pie	0
61	3	2270	Horticultural Sand	0
62	3	2271	DEE POTS- 656ml	0
63	3	2272	full strength Farheus	0
64	3	2273	Steamed soil for one day	0
65	3	2274	?	0
66	3	2275	Top watering following early misting (3 minutes every four hours between 8AM and 5PM, top watering 2x/day	0
67	3	2276	2x day, wtih 2x/week fertilization	0
68	3	2277	100 ml	0
69	3	2278	ambient	0
70	3	2279	ambient	0
71	3	2280	ambient	0
72	3	2281	ambient- Summer, Davis, Ca	0
73	3	2282	ambient	0
74	3	2283	ambient	0
75	3	2284	ambient	0
76	3	2285	sandpaper	0
77	3	2286	None	0
\.


--
-- Data for Name: protocol; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY protocol (protocol_id, type_id, pub_id, dbxref_id, name, uri, protocoldescription, hardwaredescription, softwaredescription) FROM stdin;
\.


--
-- Data for Name: protocolparam; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY protocolparam (protocolparam_id, protocol_id, name, datatype_id, unittype_id, value, rank) FROM stdin;
\.


--
-- Data for Name: pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY pub (pub_id, title, volumetitle, volume, series_name, issue, pyear, pages, miniref, uniquename, type_id, is_obsolete, publisher, pubplace) FROM stdin;
1	\N	\N	\N	\N	\N	\N	\N	null	null	1	f	\N	\N
\.


--
-- Data for Name: pub_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY pub_dbxref (pub_dbxref_id, pub_id, dbxref_id, is_current) FROM stdin;
\.


--
-- Data for Name: pub_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY pub_relationship (pub_relationship_id, subject_id, object_id, type_id) FROM stdin;
\.


--
-- Data for Name: pubauthor; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY pubauthor (pubauthor_id, pub_id, rank, editor, surname, givennames, suffix) FROM stdin;
\.


--
-- Data for Name: pubprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY pubprop (pubprop_id, pub_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: quantification; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY quantification (quantification_id, acquisition_id, operator_id, protocol_id, analysis_id, quantificationdate, name, uri) FROM stdin;
\.


--
-- Data for Name: quantification_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY quantification_relationship (quantification_relationship_id, subject_id, type_id, object_id) FROM stdin;
\.


--
-- Data for Name: quantificationprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY quantificationprop (quantificationprop_id, quantification_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: stock; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock (stock_id, dbxref_id, organism_id, name, uniquename, description, type_id, is_obsolete) FROM stdin;
1	\N	15	\N	MTE2_TN7.4_1	\N	2308	f
2	\N	15	\N	MTE2_TN7.19_26	\N	2308	f
3	\N	15	\N	MTE2_TN7.18_31	\N	2308	f
4	\N	15	\N	MTE2_TN7.23_45	\N	2308	f
5	\N	15	\N	MTE2_TN7.22_50	\N	2308	f
6	\N	15	\N	MTE2_TN7.2_56	\N	2308	f
7	\N	15	\N	MTE2_TN8.4_67	\N	2308	f
8	\N	15	\N	MTE2_TN8.28_75	\N	2308	f
9	\N	15	\N	MTE2_TN8.24_105	\N	2308	f
10	\N	15	\N	MTE2_TN8.22_109	\N	2308	f
11	\N	15	\N	MTE2_TN8.23_117	\N	2308	f
12	\N	15	\N	MTE2_TN9.24_121	\N	2308	f
13	\N	15	\N	MTE2_TN9.4_133	\N	2308	f
14	\N	15	\N	MTE2_TN9.22_140	\N	2308	f
15	\N	15	\N	MTE2_TN9.3_145	\N	2308	f
16	\N	15	\N	MTE2_TN9.17_153	\N	2308	f
17	\N	15	\N	MTE2_TN1.18_200	\N	2308	f
18	\N	15	\N	MTE2_TN1.1_213	\N	2308	f
19	\N	15	\N	MTE2_TN1.3_224	\N	2308	f
20	\N	15	\N	MTE2_TN1.15_230	\N	2308	f
21	\N	15	\N	MTE2_TN7.13_7	\N	2308	f
22	\N	15	\N	MTE2_TN7.17_14	\N	2308	f
23	\N	15	\N	MTE2_TN7.20_21	\N	2308	f
24	\N	15	\N	MTE2_TN7.11_37	\N	2308	f
25	\N	15	\N	MTE2_TN8.15_61	\N	2308	f
26	\N	15	\N	MTE2_TN8.21_81	\N	2308	f
27	\N	15	\N	MTE2_TN8.25_86	\N	2308	f
28	\N	15	\N	MTE2_TN8.3_93	\N	2308	f
29	\N	15	\N	MTE2_TN9.5_128	\N	2308	f
30	\N	15	\N	MTE2_TN9.22_141	\N	2308	f
31	\N	15	\N	MTE2_TN9.15_157	\N	2308	f
32	\N	15	\N	MTE2_TN9.12_164	\N	2308	f
33	\N	15	\N	MTE2_TN9.21_169	\N	2308	f
34	\N	15	\N	MTE2_TN9.20_175	\N	2308	f
35	\N	15	\N	MTE2_TN1.17_181	\N	2308	f
36	\N	15	\N	MTE2_TN1.13_189	\N	2308	f
37	\N	15	\N	MTE2_TN1.16_193	\N	2308	f
38	\N	15	\N	MTE2_TN1.21_207	\N	2308	f
39	\N	15	\N	MTE2_TN1.5_217	\N	2308	f
40	\N	15	\N	MTE2_TN1.11_237	\N	2308	f
41	\N	15	\N	MTE2_TN7.13_8	\N	2308	f
42	\N	15	\N	MTE2_TN7.17_13	\N	2308	f
43	\N	15	\N	MTE2_TN7.19_27	\N	2308	f
44	\N	15	\N	MTE2_TN7.11_38	\N	2308	f
45	\N	15	\N	MTE2_TN7.23_43	\N	2308	f
46	\N	15	\N	MTE2_TN8.15_62	\N	2308	f
47	\N	15	\N	MTE2_TN8.21_79	\N	2308	f
48	\N	15	\N	MTE2_TN8.25_87	\N	2308	f
49	\N	15	\N	MTE2_TN8.22_110	\N	2308	f
50	\N	15	\N	MTE2_TN9.5_127	\N	2308	f
51	\N	15	\N	MTE2_TN9.4_134	\N	2308	f
52	\N	15	\N	MTE2_TN9.15_158	\N	2308	f
53	\N	15	\N	MTE2_TN9.12_165	\N	2308	f
54	\N	15	\N	MTE2_TN9.21_170	\N	2308	f
55	\N	15	\N	MTE2_TN9.20_176	\N	2308	f
56	\N	15	\N	MTE2_TN1.17_183	\N	2308	f
57	\N	15	\N	MTE2_TN1.21_206	\N	2308	f
58	\N	15	\N	MTE2_TN1.3_225	\N	2308	f
59	\N	15	\N	MTE2_TN1.15_231	\N	2308	f
60	\N	15	\N	MTE2_TN1.11_236	\N	2308	f
61	\N	15	\N	MTE2_TN7.4_2	\N	2308	f
62	\N	15	\N	MTE2_TN7.20_20	\N	2308	f
63	\N	15	\N	MTE2_TN7.18_32	\N	2308	f
64	\N	15	\N	MTE2_TN7.22_51	\N	2308	f
65	\N	15	\N	MTE2_TN7.2_57	\N	2308	f
66	\N	15	\N	MTE2_TN8.4_69	\N	2308	f
67	\N	15	\N	MTE2_TN8.28_73	\N	2308	f
68	\N	15	\N	MTE2_TN8.3_92	\N	2308	f
69	\N	15	\N	MTE2_TN8.5_98	\N	2308	f
70	\N	15	\N	MTE2_TN8.5_99	\N	2308	f
71	\N	15	\N	MTE2_TN8.24_104	\N	2308	f
72	\N	15	\N	MTE2_TN8.23_115	\N	2308	f
73	\N	15	\N	MTE2_TN9.24_122	\N	2308	f
74	\N	15	\N	MTE2_TN9.3_146	\N	2308	f
75	\N	15	\N	MTE2_TN9.17_152	\N	2308	f
76	\N	15	\N	MTE2_TN1.13_188	\N	2308	f
77	\N	15	\N	MTE2_TN1.16_195	\N	2308	f
78	\N	15	\N	MTE2_TN1.18_201	\N	2308	f
79	\N	15	\N	MTE2_TN1.1_211	\N	2308	f
80	\N	15	\N	MTE2_TN1.5_218	\N	2308	f
81	\N	15	\N	MTE2_TN7.4_3	\N	2308	f
82	\N	15	\N	MTE2_TN7.20_19	\N	2308	f
83	\N	15	\N	MTE2_TN7.18_33	\N	2308	f
84	\N	15	\N	MTE2_TN7.11_39	\N	2308	f
85	\N	15	\N	MTE2_TN7.22_49	\N	2308	f
86	\N	15	\N	MTE2_TN7.2_55	\N	2308	f
87	\N	15	\N	MTE2_TN8.4_68	\N	2308	f
88	\N	15	\N	MTE2_TN8.28_74	\N	2308	f
89	\N	15	\N	MTE2_TN8.5_97	\N	2308	f
90	\N	15	\N	MTE2_TN8.24_103	\N	2308	f
91	\N	15	\N	MTE2_TN8.23_116	\N	2308	f
92	\N	15	\N	MTE2_TN9.24_123	\N	2308	f
93	\N	15	\N	MTE2_TN9.5_129	\N	2308	f
94	\N	15	\N	MTE2_TN9.22_139	\N	2308	f
95	\N	15	\N	MTE2_TN9.17_151	\N	2308	f
96	\N	15	\N	MTE2_TN1.17_182	\N	2308	f
97	\N	15	\N	MTE2_TN1.13_187	\N	2308	f
98	\N	15	\N	MTE2_TN1.16_194	\N	2308	f
99	\N	15	\N	MTE2_TN1.21_205	\N	2308	f
100	\N	15	\N	MTE2_TN1.5_219	\N	2308	f
101	\N	15	\N	MTE2_TN7.13_9	\N	2308	f
102	\N	15	\N	MTE2_TN7.17_15	\N	2308	f
103	\N	15	\N	MTE2_TN7.19_25	\N	2308	f
104	\N	15	\N	MTE2_TN7.23_44	\N	2308	f
105	\N	15	\N	MTE2_TN8.15_63	\N	2308	f
106	\N	15	\N	MTE2_TN8.21_80	\N	2308	f
107	\N	15	\N	MTE2_TN8.25_85	\N	2308	f
108	\N	15	\N	MTE2_TN8.3_91	\N	2308	f
109	\N	15	\N	MTE2_TN8.22_111	\N	2308	f
110	\N	15	\N	MTE2_TN9.4_135	\N	2308	f
111	\N	15	\N	MTE2_TN9.3_147	\N	2308	f
112	\N	15	\N	MTE2_TN9.15_159	\N	2308	f
113	\N	15	\N	MTE2_TN9.12_163	\N	2308	f
114	\N	15	\N	MTE2_TN9.21_171	\N	2308	f
115	\N	15	\N	MTE2_TN9.20_177	\N	2308	f
116	\N	15	\N	MTE2_TN1.18_199	\N	2308	f
117	\N	15	\N	MTE2_TN1.1_212	\N	2308	f
118	\N	15	\N	MTE2_TN1.3_223	\N	2308	f
119	\N	15	\N	MTE2_TN1.15_229	\N	2308	f
120	\N	15	\N	MTE2_TN1.11_235	\N	2308	f
121	\N	15	\N	MTE2_TN7.13_10	\N	2308	f
122	\N	15	\N	MTE2_TN7.20_24	\N	2308	f
123	\N	15	\N	MTE2_TN7.19_28	\N	2308	f
124	\N	15	\N	MTE2_TN7.18_36	\N	2308	f
125	\N	15	\N	MTE2_TN7.23_46	\N	2308	f
126	\N	15	\N	MTE2_TN7.22_53	\N	2308	f
127	\N	15	\N	MTE2_TN8.4_70	\N	2308	f
128	\N	15	\N	MTE2_TN8.25_89	\N	2308	f
129	\N	15	\N	MTE2_TN8.3_94	\N	2308	f
130	\N	15	\N	MTE2_TN9.4_136	\N	2308	f
131	\N	15	\N	MTE2_TN9.3_148	\N	2308	f
132	\N	15	\N	MTE2_TN9.15_161	\N	2308	f
133	\N	15	\N	MTE2_TN9.12_167	\N	2308	f
134	\N	15	\N	MTE2_TN9.21_172	\N	2308	f
135	\N	15	\N	MTE2_TN9.20_178	\N	2308	f
136	\N	15	\N	MTE2_TN1.16_197	\N	2308	f
137	\N	15	\N	MTE2_TN1.18_203	\N	2308	f
138	\N	15	\N	MTE2_TN1.21_208	\N	2308	f
139	\N	15	\N	MTE2_TN1.5_220	\N	2308	f
140	\N	15	\N	MTE2_TN1.15_234	\N	2308	f
141	\N	15	\N	MTE2_TN7.4_4	\N	2308	f
142	\N	15	\N	MTE2_TN7.17_17	\N	2308	f
143	\N	15	\N	MTE2_TN7.11_40	\N	2308	f
144	\N	15	\N	MTE2_TN7.2_58	\N	2308	f
145	\N	15	\N	MTE2_TN8.15_64	\N	2308	f
146	\N	15	\N	MTE2_TN8.28_78	\N	2308	f
147	\N	15	\N	MTE2_TN8.21_83	\N	2308	f
148	\N	15	\N	MTE2_TN8.5_100	\N	2308	f
149	\N	15	\N	MTE2_TN8.24_108	\N	2308	f
150	\N	15	\N	MTE2_TN8.22_112	\N	2308	f
151	\N	15	\N	MTE2_TN8.23_120	\N	2308	f
152	\N	15	\N	MTE2_TN9.24_125	\N	2308	f
153	\N	15	\N	MTE2_TN9.5_130	\N	2308	f
154	\N	15	\N	MTE2_TN9.22_142	\N	2308	f
155	\N	15	\N	MTE2_TN9.17_156	\N	2308	f
156	\N	15	\N	MTE2_TN1.17_186	\N	2308	f
157	\N	15	\N	MTE2_TN1.13_192	\N	2308	f
158	\N	15	\N	MTE2_TN1.1_214	\N	2308	f
159	\N	15	\N	MTE2_TN1.3_227	\N	2308	f
160	\N	15	\N	MTE2_TN1.11_239	\N	2308	f
161	\N	15	\N	MTE2_TN7.4_5	\N	2308	f
162	\N	15	\N	MTE2_TN7.19_30	\N	2308	f
163	\N	15	\N	MTE2_TN7.18_34	\N	2308	f
164	\N	15	\N	MTE2_TN7.2_60	\N	2308	f
165	\N	15	\N	MTE2_TN8.15_65	\N	2308	f
166	\N	15	\N	MTE2_TN8.28_77	\N	2308	f
167	\N	15	\N	MTE2_TN8.25_90	\N	2308	f
168	\N	15	\N	MTE2_TN8.3_96	\N	2308	f
169	\N	15	\N	MTE2_TN8.5_101	\N	2308	f
170	\N	15	\N	MTE2_TN8.24_107	\N	2308	f
171	\N	15	\N	MTE2_TN8.23_119	\N	2308	f
172	\N	15	\N	MTE2_TN9.5_131	\N	2308	f
173	\N	15	\N	MTE2_TN9.3_150	\N	2308	f
174	\N	15	\N	MTE2_TN9.17_155	\N	2308	f
175	\N	15	\N	MTE2_TN1.17_184	\N	2308	f
176	\N	15	\N	MTE2_TN1.18_202	\N	2308	f
177	\N	15	\N	MTE2_TN1.1_216	\N	2308	f
178	\N	15	\N	MTE2_TN1.5_221	\N	2308	f
179	\N	15	\N	MTE2_TN1.3_228	\N	2308	f
180	\N	15	\N	MTE2_TN1.11_238	\N	2308	f
181	\N	15	\N	MTE2_TN7.13_11	\N	2308	f
182	\N	15	\N	MTE2_TN7.17_16	\N	2308	f
183	\N	15	\N	MTE2_TN7.20_23	\N	2308	f
184	\N	15	\N	MTE2_TN7.11_41	\N	2308	f
185	\N	15	\N	MTE2_TN7.23_48	\N	2308	f
186	\N	15	\N	MTE2_TN7.22_54	\N	2308	f
187	\N	15	\N	MTE2_TN8.4_72	\N	2308	f
188	\N	15	\N	MTE2_TN8.21_82	\N	2308	f
189	\N	15	\N	MTE2_TN8.22_113	\N	2308	f
190	\N	15	\N	MTE2_TN9.24_124	\N	2308	f
191	\N	15	\N	MTE2_TN9.4_137	\N	2308	f
192	\N	15	\N	MTE2_TN9.22_143	\N	2308	f
193	\N	15	\N	MTE2_TN9.15_162	\N	2308	f
194	\N	15	\N	MTE2_TN9.12_166	\N	2308	f
195	\N	15	\N	MTE2_TN9.21_173	\N	2308	f
196	\N	15	\N	MTE2_TN9.20_179	\N	2308	f
197	\N	15	\N	MTE2_TN1.13_191	\N	2308	f
198	\N	15	\N	MTE2_TN1.16_196	\N	2308	f
199	\N	15	\N	MTE2_TN1.21_209	\N	2308	f
200	\N	15	\N	MTE2_TN1.15_233	\N	2308	f
201	\N	15	\N	MTE2_TN7.4_6	\N	2308	f
202	\N	15	\N	MTE2_TN7.17_18	\N	2308	f
203	\N	15	\N	MTE2_TN7.19_29	\N	2308	f
204	\N	15	\N	MTE2_TN7.23_47	\N	2308	f
205	\N	15	\N	MTE2_TN7.22_52	\N	2308	f
206	\N	15	\N	MTE2_TN7.2_59	\N	2308	f
207	\N	15	\N	MTE2_TN8.4_71	\N	2308	f
208	\N	15	\N	MTE2_TN8.28_76	\N	2308	f
209	\N	15	\N	MTE2_TN8.21_84	\N	2308	f
210	\N	15	\N	MTE2_TN8.24_106	\N	2308	f
211	\N	15	\N	MTE2_TN8.22_114	\N	2308	f
212	\N	15	\N	MTE2_TN9.4_138	\N	2308	f
213	\N	15	\N	MTE2_TN9.3_149	\N	2308	f
214	\N	15	\N	MTE2_TN9.15_160	\N	2308	f
215	\N	15	\N	MTE2_TN9.12_168	\N	2308	f
216	\N	15	\N	MTE2_TN9.21_174	\N	2308	f
217	\N	15	\N	MTE2_TN9.20_180	\N	2308	f
218	\N	15	\N	MTE2_TN1.21_210	\N	2308	f
219	\N	15	\N	MTE2_TN1.1_215	\N	2308	f
220	\N	15	\N	MTE2_TN1.3_226	\N	2308	f
221	\N	15	\N	MTE2_TN7.13_12	\N	2308	f
222	\N	15	\N	MTE2_TN7.20_22	\N	2308	f
223	\N	15	\N	MTE2_TN7.18_35	\N	2308	f
224	\N	15	\N	MTE2_TN7.11_42	\N	2308	f
225	\N	15	\N	MTE2_TN8.15_66	\N	2308	f
226	\N	15	\N	MTE2_TN8.25_88	\N	2308	f
227	\N	15	\N	MTE2_TN8.3_95	\N	2308	f
228	\N	15	\N	MTE2_TN8.5_102	\N	2308	f
229	\N	15	\N	MTE2_TN8.23_118	\N	2308	f
230	\N	15	\N	MTE2_TN9.24_126	\N	2308	f
231	\N	15	\N	MTE2_TN9.5_132	\N	2308	f
232	\N	15	\N	MTE2_TN9.22_144	\N	2308	f
233	\N	15	\N	MTE2_TN9.17_154	\N	2308	f
234	\N	15	\N	MTE2_TN1.17_185	\N	2308	f
235	\N	15	\N	MTE2_TN1.13_190	\N	2308	f
236	\N	15	\N	MTE2_TN1.16_198	\N	2308	f
237	\N	15	\N	MTE2_TN1.18_204	\N	2308	f
238	\N	15	\N	MTE2_TN1.5_222	\N	2308	f
239	\N	15	\N	MTE2_TN1.15_232	\N	2308	f
240	\N	15	\N	MTE2_TN1.11_240	\N	2308	f
\.


--
-- Data for Name: stock_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock_cvterm (stock_cvterm_id, stock_id, cvterm_id, pub_id) FROM stdin;
\.


--
-- Data for Name: stock_dbxref; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock_dbxref (stock_dbxref_id, stock_id, dbxref_id, is_current) FROM stdin;
\.


--
-- Data for Name: stock_genotype; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock_genotype (stock_genotype_id, stock_id, genotype_id) FROM stdin;
1	1	1
2	2	2
3	3	3
4	4	4
5	5	5
6	6	6
7	7	7
8	8	8
9	9	9
10	10	10
11	11	11
12	12	12
13	13	13
14	14	14
15	15	15
16	16	16
17	17	17
18	18	18
19	19	19
20	20	20
21	21	21
22	22	22
23	23	23
24	24	24
25	25	25
26	26	26
27	27	27
28	28	28
29	29	29
30	30	14
31	31	30
32	32	31
33	33	32
34	34	33
35	35	34
36	36	35
37	37	36
38	38	37
39	39	38
40	40	39
41	41	21
42	42	22
43	43	2
44	44	24
45	45	4
46	46	25
47	47	26
48	48	27
49	49	10
50	50	29
51	51	13
52	52	30
53	53	31
54	54	32
55	55	33
56	56	34
57	57	37
58	58	19
59	59	20
60	60	39
61	61	1
62	62	23
63	63	3
64	64	5
65	65	6
66	66	7
67	67	8
68	68	28
69	69	40
70	70	40
71	71	9
72	72	11
73	73	12
74	74	15
75	75	16
76	76	35
77	77	36
78	78	17
79	79	18
80	80	38
81	81	1
82	82	23
83	83	3
84	84	24
85	85	5
86	86	6
87	87	7
88	88	8
89	89	40
90	90	9
91	91	11
92	92	12
93	93	29
94	94	14
95	95	16
96	96	34
97	97	35
98	98	36
99	99	37
100	100	38
101	101	21
102	102	22
103	103	2
104	104	4
105	105	25
106	106	26
107	107	27
108	108	28
109	109	10
110	110	13
111	111	15
112	112	30
113	113	31
114	114	32
115	115	33
116	116	17
117	117	18
118	118	19
119	119	20
120	120	39
121	121	21
122	122	23
123	123	2
124	124	3
125	125	4
126	126	5
127	127	7
128	128	27
129	129	28
130	130	13
131	131	15
132	132	30
133	133	31
134	134	32
135	135	33
136	136	36
137	137	17
138	138	37
139	139	38
140	140	20
141	141	1
142	142	22
143	143	24
144	144	6
145	145	25
146	146	8
147	147	26
148	148	40
149	149	9
150	150	10
151	151	11
152	152	12
153	153	29
154	154	14
155	155	16
156	156	34
157	157	35
158	158	18
159	159	19
160	160	39
161	161	1
162	162	2
163	163	3
164	164	6
165	165	25
166	166	8
167	167	27
168	168	28
169	169	40
170	170	9
171	171	11
172	172	29
173	173	15
174	174	16
175	175	34
176	176	17
177	177	18
178	178	38
179	179	19
180	180	39
181	181	21
182	182	22
183	183	23
184	184	24
185	185	4
186	186	5
187	187	7
188	188	26
189	189	10
190	190	12
191	191	13
192	192	14
193	193	30
194	194	31
195	195	32
196	196	33
197	197	35
198	198	36
199	199	37
200	200	20
201	201	1
202	202	22
203	203	2
204	204	4
205	205	5
206	206	6
207	207	7
208	208	8
209	209	26
210	210	9
211	211	10
212	212	13
213	213	15
214	214	30
215	215	31
216	216	32
217	217	33
218	218	37
219	219	18
220	220	19
221	221	21
222	222	23
223	223	3
224	224	24
225	225	25
226	226	27
227	227	28
228	228	40
229	229	11
230	230	12
231	231	29
232	232	14
233	233	16
234	234	34
235	235	35
236	236	36
237	237	17
238	238	38
239	239	20
240	240	39
\.


--
-- Data for Name: stock_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock_pub (stock_pub_id, stock_id, pub_id) FROM stdin;
\.


--
-- Data for Name: stock_relationship; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock_relationship (stock_relationship_id, subject_id, object_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: stock_relationship_cvterm; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock_relationship_cvterm (stock_relationship_cvterm_id, stock_relatiohship_id, cvterm_id, pub_id) FROM stdin;
\.


--
-- Data for Name: stock_relationship_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stock_relationship_pub (stock_relationship_pub_id, stock_relationship_id, pub_id) FROM stdin;
\.


--
-- Data for Name: stockcollection; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stockcollection (stockcollection_id, type_id, contact_id, name, uniquename) FROM stdin;
\.


--
-- Data for Name: stockcollection_stock; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stockcollection_stock (stockcollection_stock_id, stockcollection_id, stock_id) FROM stdin;
\.


--
-- Data for Name: stockcollectionprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stockcollectionprop (stockcollectionprop_id, stockcollection_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: stockprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stockprop (stockprop_id, stock_id, type_id, value, rank) FROM stdin;
1	1	2288	1	0
2	1	2289	1	0
3	1	2292	1	0
4	1	2294	4	0
5	1	2295		0
6	2	2288	26	0
7	2	2289	2	0
8	2	2292	1	0
9	2	2294	14	0
10	2	2295		0
11	3	2288	31	0
12	3	2289	3	0
13	3	2292	1	0
14	3	2294	5	0
15	3	2295		0
16	4	2288	45	0
17	4	2289	4	0
18	4	2292	1	0
19	4	2294	11	0
20	4	2295		0
21	5	2288	50	0
22	5	2289	5	0
23	5	2292	1	0
24	5	2294	20	0
25	5	2295		0
26	6	2288	56	0
27	6	2289	6	0
28	6	2292	1	0
29	6	2294	8	0
30	6	2295		0
31	7	2288	67	0
32	7	2289	7	0
33	7	2292	1	0
34	7	2294	17	0
35	7	2295		0
36	8	2288	75	0
37	8	2289	8	0
38	8	2292	1	0
39	8	2294	10	0
40	8	2295		0
41	9	2288	105	0
42	9	2289	9	0
43	9	2292	1	0
44	9	2294	6	0
45	9	2295		0
46	10	2288	109	0
47	10	2289	10	0
48	10	2292	1	0
49	10	2294	7	0
50	10	2295		0
51	11	2288	117	0
52	11	2289	11	0
53	11	2292	1	0
54	11	2294	16	0
55	11	2295		0
56	12	2288	121	0
57	12	2289	12	0
58	12	2292	1	0
59	12	2294	2	0
60	12	2295		0
61	13	2288	133	0
62	13	2289	13	0
63	13	2292	1	0
64	13	2294	19	0
65	13	2295		0
66	14	2288	140	0
67	14	2289	14	0
68	14	2292	1	0
69	14	2294	9	0
70	14	2295		0
71	15	2288	145	0
72	15	2289	15	0
73	15	2292	1	0
74	15	2294	1	0
75	15	2295		0
76	16	2288	153	0
77	16	2289	16	0
78	16	2292	1	0
79	16	2294	18	0
80	16	2295		0
81	17	2288	200	0
82	17	2289	17	0
83	17	2292	1	0
84	17	2294	15	0
85	17	2295		0
86	18	2288	213	0
87	18	2289	18	0
88	18	2292	1	0
89	18	2294	13	0
90	18	2295		0
91	19	2288	224	0
92	19	2289	19	0
93	19	2292	1	0
94	19	2294	3	0
95	19	2295		0
96	20	2288	230	0
97	20	2289	20	0
98	20	2292	1	0
99	20	2294	12	0
100	20	2295		0
101	21	2288	7	0
102	21	2289	21	0
103	21	2292	2	0
104	21	2294	11	0
105	21	2295		0
106	22	2288	14	0
107	22	2289	22	0
108	22	2292	2	0
109	22	2294	13	0
110	22	2295		0
111	23	2288	21	0
112	23	2289	23	0
113	23	2292	2	0
114	23	2294	6	0
115	23	2295		0
116	24	2288	37	0
117	24	2289	24	0
118	24	2292	2	0
119	24	2294	20	0
120	24	2295		0
121	25	2288	61	0
122	25	2289	25	0
123	25	2292	2	0
124	25	2294	2	0
125	25	2295		0
126	26	2288	81	0
127	26	2289	26	0
128	26	2292	2	0
129	26	2294	1	0
130	26	2295		0
131	27	2288	86	0
132	27	2289	27	0
133	27	2292	2	0
134	27	2294	9	0
135	27	2295		0
136	28	2288	93	0
137	28	2289	28	0
138	28	2292	2	0
139	28	2294	7	0
140	28	2295		0
141	29	2288	128	0
142	29	2289	29	0
143	29	2292	2	0
144	29	2294	16	0
145	29	2295		0
146	30	2288	141	0
147	30	2289	30	0
148	30	2292	2	0
149	30	2294	3	0
150	30	2295		0
151	31	2288	157	0
152	31	2289	31	0
153	31	2292	2	0
154	31	2294	17	0
155	31	2295		0
156	32	2288	164	0
157	32	2289	32	0
158	32	2292	2	0
159	32	2294	19	0
160	32	2295		0
161	33	2288	169	0
162	33	2289	33	0
163	33	2292	2	0
164	33	2294	8	0
165	33	2295		0
166	34	2288	175	0
167	34	2289	34	0
168	34	2292	2	0
169	34	2294	15	0
170	34	2295		0
171	35	2288	181	0
172	35	2289	35	0
173	35	2292	2	0
174	35	2294	12	0
175	35	2295		0
176	36	2288	189	0
177	36	2289	36	0
178	36	2292	2	0
179	36	2294	10	0
180	36	2295		0
181	37	2288	193	0
182	37	2289	37	0
183	37	2292	2	0
184	37	2294	5	0
185	37	2295		0
186	38	2288	207	0
187	38	2289	38	0
188	38	2292	2	0
189	38	2294	18	0
190	38	2295		0
191	39	2288	217	0
192	39	2289	39	0
193	39	2292	2	0
194	39	2294	4	0
195	39	2295		0
196	40	2288	237	0
197	40	2289	40	0
198	40	2292	2	0
199	40	2294	14	0
200	40	2295		0
201	41	2288	8	0
202	41	2289	41	0
203	41	2292	3	0
204	41	2294	3	0
205	41	2295		0
206	42	2288	13	0
207	42	2289	42	0
208	42	2292	3	0
209	42	2294	14	0
210	42	2295		0
211	43	2288	27	0
212	43	2289	43	0
213	43	2292	3	0
214	43	2294	1	0
215	43	2295		0
216	44	2288	38	0
217	44	2289	44	0
218	44	2292	3	0
219	44	2294	12	0
220	44	2295		0
221	45	2288	43	0
222	45	2289	45	0
223	45	2292	3	0
224	45	2294	13	0
225	45	2295		0
226	46	2288	62	0
227	46	2289	46	0
228	46	2292	3	0
229	46	2294	9	0
230	46	2295		0
231	47	2288	79	0
232	47	2289	47	0
233	47	2292	3	0
234	47	2294	8	0
235	47	2295		0
236	48	2288	87	0
237	48	2289	48	0
238	48	2292	3	0
239	48	2294	7	0
240	48	2295		0
241	49	2288	110	0
242	49	2289	49	0
243	49	2292	3	0
244	49	2294	10	0
245	49	2295		0
246	50	2288	127	0
247	50	2289	50	0
248	50	2292	3	0
249	50	2294	4	0
250	50	2295		0
251	51	2288	134	0
252	51	2289	51	0
253	51	2292	3	0
254	51	2294	6	0
255	51	2295		0
256	52	2288	158	0
257	52	2289	52	0
258	52	2292	3	0
259	52	2294	2	0
260	52	2295		0
261	53	2288	165	0
262	53	2289	53	0
263	53	2292	3	0
264	53	2294	20	0
265	53	2295		0
266	54	2288	170	0
267	54	2289	54	0
268	54	2292	3	0
269	54	2294	16	0
270	54	2295		0
271	55	2288	176	0
272	55	2289	55	0
273	55	2292	3	0
274	55	2294	18	0
275	55	2295		0
276	56	2288	183	0
277	56	2289	56	0
278	56	2292	3	0
279	56	2294	11	0
280	56	2295		0
281	57	2288	206	0
282	57	2289	57	0
283	57	2292	3	0
284	57	2294	17	0
285	57	2295		0
286	58	2288	225	0
287	58	2289	58	0
288	58	2292	3	0
289	58	2294	19	0
290	58	2295		0
291	59	2288	231	0
292	59	2289	59	0
293	59	2292	3	0
294	59	2294	5	0
295	59	2295		0
296	60	2288	236	0
297	60	2289	60	0
298	60	2292	3	0
299	60	2294	15	0
300	60	2295	08/21/2009	0
301	61	2288	2	0
302	61	2289	61	0
303	61	2292	4	0
304	61	2294	11	0
305	61	2295		0
306	62	2288	20	0
307	62	2289	62	0
308	62	2292	4	0
309	62	2294	15	0
310	62	2295		0
311	63	2288	32	0
312	63	2289	63	0
313	63	2292	4	0
314	63	2294	6	0
315	63	2295		0
316	64	2288	51	0
317	64	2289	64	0
318	64	2292	4	0
319	64	2294	13	0
320	64	2295		0
321	65	2288	57	0
322	65	2289	65	0
323	65	2292	4	0
324	65	2294	19	0
325	65	2295		0
326	66	2288	69	0
327	66	2289	66	0
328	66	2292	4	0
329	66	2294	18	0
330	66	2295		0
331	67	2288	73	0
332	67	2289	67	0
333	67	2292	4	0
334	67	2294	7	0
335	67	2295		0
336	68	2288	92	0
337	68	2289	68	0
338	68	2292	4	0
339	68	2294	20	0
340	68	2295		0
341	69	2288	98	0
342	69	2289	69	0
343	69	2292	4	0
344	69	2294	12	0
345	69	2295		0
346	70	2288	99	0
347	70	2289	70	0
348	70	2292	4	0
349	70	2294	3	0
350	70	2295		0
351	71	2288	104	0
352	71	2289	71	0
353	71	2292	4	0
354	71	2294	1	0
355	71	2295		0
356	72	2288	115	0
357	72	2289	72	0
358	72	2292	4	0
359	72	2294	17	0
360	72	2295		0
361	73	2288	122	0
362	73	2289	73	0
363	73	2292	4	0
364	73	2294	8	0
365	73	2295		0
366	74	2288	146	0
367	74	2289	74	0
368	74	2292	4	0
369	74	2294	16	0
370	74	2295		0
371	75	2288	152	0
372	75	2289	75	0
373	75	2292	4	0
374	75	2294	2	0
375	75	2295		0
376	76	2288	188	0
377	76	2289	76	0
378	76	2292	4	0
379	76	2294	10	0
380	76	2295		0
381	77	2288	195	0
382	77	2289	77	0
383	77	2292	4	0
384	77	2294	9	0
385	77	2295		0
386	78	2288	201	0
387	78	2289	78	0
388	78	2292	4	0
389	78	2294	4	0
390	78	2295		0
391	79	2288	211	0
392	79	2289	79	0
393	79	2292	4	0
394	79	2294	14	0
395	79	2295		0
396	80	2288	218	0
397	80	2289	80	0
398	80	2292	4	0
399	80	2294	5	0
400	80	2295		0
401	81	2288	3	0
402	81	2289	81	0
403	81	2292	5	0
404	81	2294	15	0
405	81	2295		0
406	82	2288	19	0
407	82	2289	82	0
408	82	2292	5	0
409	82	2294	7	0
410	82	2295		0
411	83	2288	33	0
412	83	2289	83	0
413	83	2292	5	0
414	83	2294	10	0
415	83	2295		0
416	84	2288	39	0
417	84	2289	84	0
418	84	2292	5	0
419	84	2294	8	0
420	84	2295		0
421	85	2288	49	0
422	85	2289	85	0
423	85	2292	5	0
424	85	2294	11	0
425	85	2295		0
426	86	2288	55	0
427	86	2289	86	0
428	86	2292	5	0
429	86	2294	2	0
430	86	2295		0
431	87	2288	68	0
432	87	2289	87	0
433	87	2292	5	0
434	87	2294	13	0
435	87	2295		0
436	88	2288	74	0
437	88	2289	88	0
438	88	2292	5	0
439	88	2294	20	0
440	88	2295		0
441	89	2288	97	0
442	89	2289	89	0
443	89	2292	5	0
444	89	2294	6	0
445	89	2295		0
446	90	2288	103	0
447	90	2289	90	0
448	90	2292	5	0
449	90	2294	18	0
450	90	2295		0
451	91	2288	116	0
452	91	2289	91	0
453	91	2292	5	0
454	91	2294	4	0
455	91	2295		0
456	92	2288	123	0
457	92	2289	92	0
458	92	2292	5	0
459	92	2294	1	0
460	92	2295		0
461	93	2288	129	0
462	93	2289	93	0
463	93	2292	5	0
464	93	2294	9	0
465	93	2295		0
466	94	2288	139	0
467	94	2289	94	0
468	94	2292	5	0
469	94	2294	17	0
470	94	2295		0
471	95	2288	151	0
472	95	2289	95	0
473	95	2292	5	0
474	95	2294	5	0
475	95	2295		0
476	96	2288	182	0
477	96	2289	96	0
478	96	2292	5	0
479	96	2294	16	0
480	96	2295		0
481	97	2288	187	0
482	97	2289	97	0
483	97	2292	5	0
484	97	2294	12	0
485	97	2295		0
486	98	2288	194	0
487	98	2289	98	0
488	98	2292	5	0
489	98	2294	14	0
490	98	2295		0
491	99	2288	205	0
492	99	2289	99	0
493	99	2292	5	0
494	99	2294	3	0
495	99	2295		0
496	100	2288	219	0
497	100	2289	100	0
498	100	2292	5	0
499	100	2294	19	0
500	100	2295		0
501	101	2288	9	0
502	101	2289	101	0
503	101	2292	6	0
504	101	2294	7	0
505	101	2295		0
506	102	2288	15	0
507	102	2289	102	0
508	102	2292	6	0
509	102	2294	11	0
510	102	2295		0
511	103	2288	25	0
512	103	2289	103	0
513	103	2292	6	0
514	103	2294	10	0
515	103	2295		0
516	104	2288	44	0
517	104	2289	104	0
518	104	2292	6	0
519	104	2294	14	0
520	104	2295		0
521	105	2288	63	0
522	105	2289	105	0
523	105	2292	6	0
524	105	2294	20	0
525	105	2295		0
526	106	2288	80	0
527	106	2289	106	0
528	106	2292	6	0
529	106	2294	6	0
530	106	2295		0
531	107	2288	85	0
532	107	2289	107	0
533	107	2292	6	0
534	107	2294	16	0
535	107	2295		0
536	108	2288	91	0
537	108	2289	108	0
538	108	2292	6	0
539	108	2294	9	0
540	108	2295		0
541	109	2288	111	0
542	109	2289	109	0
543	109	2292	6	0
544	109	2294	3	0
545	109	2295		0
546	110	2288	135	0
547	110	2289	110	0
548	110	2292	6	0
549	110	2294	8	0
550	110	2295		0
551	111	2288	147	0
552	111	2289	111	0
553	111	2292	6	0
554	111	2294	2	0
555	111	2295		0
556	112	2288	159	0
557	112	2289	112	0
558	112	2292	6	0
559	112	2294	5	0
560	112	2295		0
561	113	2288	163	0
562	113	2289	113	0
563	113	2292	6	0
564	113	2294	4	0
565	113	2295		0
566	114	2288	171	0
567	114	2289	114	0
568	114	2292	6	0
569	114	2294	19	0
570	114	2295	08/21/2009	0
571	115	2288	177	0
572	115	2289	115	0
573	115	2292	6	0
574	115	2294	17	0
575	115	2295		0
576	116	2288	199	0
577	116	2289	116	0
578	116	2292	6	0
579	116	2294	18	0
580	116	2295		0
581	117	2288	212	0
582	117	2289	117	0
583	117	2292	6	0
584	117	2294	1	0
585	117	2295		0
586	118	2288	223	0
587	118	2289	118	0
588	118	2292	6	0
589	118	2294	15	0
590	118	2295		0
591	119	2288	229	0
592	119	2289	119	0
593	119	2292	6	0
594	119	2294	13	0
595	119	2295		0
596	120	2288	235	0
597	120	2289	120	0
598	120	2292	6	0
599	120	2294	12	0
600	120	2295		0
601	121	2288	10	0
602	121	2289	121	0
603	121	2292	7	0
604	121	2294	18	0
605	121	2295		0
606	122	2288	24	0
607	122	2289	122	0
608	122	2292	7	0
609	122	2294	4	0
610	122	2295		0
611	123	2288	28	0
612	123	2289	123	0
613	123	2292	7	0
614	123	2294	12	0
615	123	2295		0
616	124	2288	36	0
617	124	2289	124	0
618	124	2292	7	0
619	124	2294	10	0
620	124	2295		0
621	125	2288	46	0
622	125	2289	125	0
623	125	2292	7	0
624	125	2294	14	0
625	125	2295		0
626	126	2288	53	0
627	126	2289	126	0
628	126	2292	7	0
629	126	2294	3	0
630	126	2295		0
631	127	2288	70	0
632	127	2289	127	0
633	127	2292	7	0
634	127	2294	20	0
635	127	2295		0
636	128	2288	89	0
637	128	2289	128	0
638	128	2292	7	0
639	128	2294	9	0
640	128	2295		0
641	129	2288	94	0
642	129	2289	129	0
643	129	2292	7	0
644	129	2294	16	0
645	129	2295		0
646	130	2288	136	0
647	130	2289	130	0
648	130	2292	7	0
649	130	2294	11	0
650	130	2295		0
651	131	2288	148	0
652	131	2289	131	0
653	131	2292	7	0
654	131	2294	1	0
655	131	2295		0
656	132	2288	161	0
657	132	2289	132	0
658	132	2292	7	0
659	132	2294	7	0
660	132	2295		0
661	133	2288	167	0
662	133	2289	133	0
663	133	2292	7	0
664	133	2294	2	0
665	133	2295		0
666	134	2288	172	0
667	134	2289	134	0
668	134	2292	7	0
669	134	2294	5	0
670	134	2295		0
671	135	2288	178	0
672	135	2289	135	0
673	135	2292	7	0
674	135	2294	15	0
675	135	2295		0
676	136	2288	197	0
677	136	2289	136	0
678	136	2292	7	0
679	136	2294	19	0
680	136	2295		0
681	137	2288	203	0
682	137	2289	137	0
683	137	2292	7	0
684	137	2294	17	0
685	137	2295		0
686	138	2288	208	0
687	138	2289	138	0
688	138	2292	7	0
689	138	2294	6	0
690	138	2295		0
691	139	2288	220	0
692	139	2289	139	0
693	139	2292	7	0
694	139	2294	13	0
695	139	2295		0
696	140	2288	234	0
697	140	2289	140	0
698	140	2292	7	0
699	140	2294	8	0
700	140	2295		0
701	141	2288	4	0
702	141	2289	141	0
703	141	2292	8	0
704	141	2294	10	0
705	141	2295		0
706	142	2288	17	0
707	142	2289	142	0
708	142	2292	8	0
709	142	2294	11	0
710	142	2295		0
711	143	2288	40	0
712	143	2289	143	0
713	143	2292	8	0
714	143	2294	13	0
715	143	2295		0
716	144	2288	58	0
717	144	2289	144	0
718	144	2292	8	0
719	144	2294	19	0
720	144	2295		0
721	145	2288	64	0
722	145	2289	145	0
723	145	2292	8	0
724	145	2294	4	0
725	145	2295		0
726	146	2288	78	0
727	146	2289	146	0
728	146	2292	8	0
729	146	2294	9	0
730	146	2295		0
731	147	2288	83	0
732	147	2289	147	0
733	147	2292	8	0
734	147	2294	14	0
735	147	2295		0
736	148	2288	100	0
737	148	2289	148	0
738	148	2292	8	0
739	148	2294	2	0
740	148	2295		0
741	149	2288	108	0
742	149	2289	149	0
743	149	2292	8	0
744	149	2294	17	0
745	149	2295		0
746	150	2288	112	0
747	150	2289	150	0
748	150	2292	8	0
749	150	2294	8	0
750	150	2295		0
751	151	2288	120	0
752	151	2289	151	0
753	151	2292	8	0
754	151	2294	16	0
755	151	2295		0
756	152	2288	125	0
757	152	2289	152	0
758	152	2292	8	0
759	152	2294	20	0
760	152	2295		0
761	153	2288	130	0
762	153	2289	153	0
763	153	2292	8	0
764	153	2294	12	0
765	153	2295		0
766	154	2288	142	0
767	154	2289	154	0
768	154	2292	8	0
769	154	2294	3	0
770	154	2295		0
771	155	2288	156	0
772	155	2289	155	0
773	155	2292	8	0
774	155	2294	6	0
775	155	2295		0
776	156	2288	186	0
777	156	2289	156	0
778	156	2292	8	0
779	156	2294	5	0
780	156	2295		0
781	157	2288	192	0
782	157	2289	157	0
783	157	2292	8	0
784	157	2294	15	0
785	157	2295		0
786	158	2288	214	0
787	158	2289	158	0
788	158	2292	8	0
789	158	2294	18	0
790	158	2295		0
791	159	2288	227	0
792	159	2289	159	0
793	159	2292	8	0
794	159	2294	7	0
795	159	2295		0
796	160	2288	239	0
797	160	2289	160	0
798	160	2292	8	0
799	160	2294	1	0
800	160	2295		0
801	161	2288	5	0
802	161	2289	161	0
803	161	2292	9	0
804	161	2294	13	0
805	161	2295		0
806	162	2288	30	0
807	162	2289	162	0
808	162	2292	9	0
809	162	2294	4	0
810	162	2295		0
811	163	2288	34	0
812	163	2289	163	0
813	163	2292	9	0
814	163	2294	9	0
815	163	2295		0
816	164	2288	60	0
817	164	2289	164	0
818	164	2292	9	0
819	164	2294	5	0
820	164	2295		0
821	165	2288	65	0
822	165	2289	165	0
823	165	2292	9	0
824	165	2294	3	0
825	165	2295		0
826	166	2288	77	0
827	166	2289	166	0
828	166	2292	9	0
829	166	2294	12	0
830	166	2295		0
831	167	2288	90	0
832	167	2289	167	0
833	167	2292	9	0
834	167	2294	18	0
835	167	2295		0
836	168	2288	96	0
837	168	2289	168	0
838	168	2292	9	0
839	168	2294	1	0
840	168	2295		0
841	169	2288	101	0
842	169	2289	169	0
843	169	2292	9	0
844	169	2294	10	0
845	169	2295		0
846	170	2288	107	0
847	170	2289	170	0
848	170	2292	9	0
849	170	2294	2	0
850	170	2295		0
851	171	2288	119	0
852	171	2289	171	0
853	171	2292	9	0
854	171	2294	20	0
855	171	2295		0
856	172	2288	131	0
857	172	2289	172	0
858	172	2292	9	0
859	172	2294	17	0
860	172	2295		0
861	173	2288	150	0
862	173	2289	173	0
863	173	2292	9	0
864	173	2294	11	0
865	173	2295		0
866	174	2288	155	0
867	174	2289	174	0
868	174	2292	9	0
869	174	2294	6	0
870	174	2295		0
871	175	2288	184	0
872	175	2289	175	0
873	175	2292	9	0
874	175	2294	15	0
875	175	2295		0
876	176	2288	202	0
877	176	2289	176	0
878	176	2292	9	0
879	176	2294	19	0
880	176	2295		0
881	177	2288	216	0
882	177	2289	177	0
883	177	2292	9	0
884	177	2294	8	0
885	177	2295		0
886	178	2288	221	0
887	178	2289	178	0
888	178	2292	9	0
889	178	2294	7	0
890	178	2295		0
891	179	2288	228	0
892	179	2289	179	0
893	179	2292	9	0
894	179	2294	14	0
895	179	2295		0
896	180	2288	238	0
897	180	2289	180	0
898	180	2292	9	0
899	180	2294	16	0
900	180	2295		0
901	181	2288	11	0
902	181	2289	181	0
903	181	2292	10	0
904	181	2294	11	0
905	181	2295		0
906	182	2288	16	0
907	182	2289	182	0
908	182	2292	10	0
909	182	2294	8	0
910	182	2295		0
911	183	2288	23	0
912	183	2289	183	0
913	183	2292	10	0
914	183	2294	15	0
915	183	2295	08/21/2009	0
916	184	2288	41	0
917	184	2289	184	0
918	184	2292	10	0
919	184	2294	14	0
920	184	2295		0
921	185	2288	48	0
922	185	2289	185	0
923	185	2292	10	0
924	185	2294	7	0
925	185	2295		0
926	186	2288	54	0
927	186	2289	186	0
928	186	2292	10	0
929	186	2294	13	0
930	186	2295		0
931	187	2288	72	0
932	187	2289	187	0
933	187	2292	10	0
934	187	2294	18	0
935	187	2295		0
936	188	2288	82	0
937	188	2289	188	0
938	188	2292	10	0
939	188	2294	6	0
940	188	2295		0
941	189	2288	113	0
942	189	2289	189	0
943	189	2292	10	0
944	189	2294	16	0
945	189	2295		0
946	190	2288	124	0
947	190	2289	190	0
948	190	2292	10	0
949	190	2294	10	0
950	190	2295		0
951	191	2288	137	0
952	191	2289	191	0
953	191	2292	10	0
954	191	2294	20	0
955	191	2295		0
956	192	2288	143	0
957	192	2289	192	0
958	192	2292	10	0
959	192	2294	5	0
960	192	2295		0
961	193	2288	162	0
962	193	2289	193	0
963	193	2292	10	0
964	193	2294	3	0
965	193	2295		0
966	194	2288	166	0
967	194	2289	194	0
968	194	2292	10	0
969	194	2294	2	0
970	194	2295		0
971	195	2288	173	0
972	195	2289	195	0
973	195	2292	10	0
974	195	2294	1	0
975	195	2295		0
976	196	2288	179	0
977	196	2289	196	0
978	196	2292	10	0
979	196	2294	9	0
980	196	2295		0
981	197	2288	191	0
982	197	2289	197	0
983	197	2292	10	0
984	197	2294	12	0
985	197	2295		0
986	198	2288	196	0
987	198	2289	198	0
988	198	2292	10	0
989	198	2294	4	0
990	198	2295		0
991	199	2288	209	0
992	199	2289	199	0
993	199	2292	10	0
994	199	2294	19	0
995	199	2295		0
996	200	2288	233	0
997	200	2289	200	0
998	200	2292	10	0
999	200	2294	17	0
1000	200	2295		0
1001	201	2288	6	0
1002	201	2289	201	0
1003	201	2292	11	0
1004	201	2294	9	0
1005	201	2295		0
1006	202	2288	18	0
1007	202	2289	202	0
1008	202	2292	11	0
1009	202	2294	13	0
1010	202	2295		0
1011	203	2288	29	0
1012	203	2289	203	0
1013	203	2292	11	0
1014	203	2294	6	0
1015	203	2295		0
1016	204	2288	47	0
1017	204	2289	204	0
1018	204	2292	11	0
1019	204	2294	14	0
1020	204	2295		0
1021	205	2288	52	0
1022	205	2289	205	0
1023	205	2292	11	0
1024	205	2294	7	0
1025	205	2295	08/21/2009	0
1026	206	2288	59	0
1027	206	2289	206	0
1028	206	2292	11	0
1029	206	2294	10	0
1030	206	2295		0
1031	207	2288	71	0
1032	207	2289	207	0
1033	207	2292	11	0
1034	207	2294	18	0
1035	207	2295		0
1036	208	2288	76	0
1037	208	2289	208	0
1038	208	2292	11	0
1039	208	2294	5	0
1040	208	2295		0
1041	209	2288	84	0
1042	209	2289	209	0
1043	209	2292	11	0
1044	209	2294	15	0
1045	209	2295		0
1046	210	2288	106	0
1047	210	2289	210	0
1048	210	2292	11	0
1049	210	2294	19	0
1050	210	2295		0
1051	211	2288	114	0
1052	211	2289	211	0
1053	211	2292	11	0
1054	211	2294	4	0
1055	211	2295		0
1056	212	2288	138	0
1057	212	2289	212	0
1058	212	2292	11	0
1059	212	2294	3	0
1060	212	2295		0
1061	213	2288	149	0
1062	213	2289	213	0
1063	213	2292	11	0
1064	213	2294	1	0
1065	213	2295		0
1066	214	2288	160	0
1067	214	2289	214	0
1068	214	2292	11	0
1069	214	2294	16	0
1070	214	2295		0
1071	215	2288	168	0
1072	215	2289	215	0
1073	215	2292	11	0
1074	215	2294	11	0
1075	215	2295		0
1076	216	2288	174	0
1077	216	2289	216	0
1078	216	2292	11	0
1079	216	2294	2	0
1080	216	2295		0
1081	217	2288	180	0
1082	217	2289	217	0
1083	217	2292	11	0
1084	217	2294	17	0
1085	217	2295		0
1086	218	2288	210	0
1087	218	2289	218	0
1088	218	2292	11	0
1089	218	2294	12	0
1090	218	2295		0
1091	219	2288	215	0
1092	219	2289	219	0
1093	219	2292	11	0
1094	219	2294	20	0
1095	219	2295		0
1096	220	2288	226	0
1097	220	2289	220	0
1098	220	2292	11	0
1099	220	2294	8	0
1100	220	2295		0
1101	221	2288	12	0
1102	221	2289	221	0
1103	221	2292	12	0
1104	221	2294	3	0
1105	221	2295		0
1106	222	2288	22	0
1107	222	2289	222	0
1108	222	2292	12	0
1109	222	2294	12	0
1110	222	2295		0
1111	223	2288	35	0
1112	223	2289	223	0
1113	223	2292	12	0
1114	223	2294	9	0
1115	223	2295		0
1116	224	2288	42	0
1117	224	2289	224	0
1118	224	2292	12	0
1119	224	2294	14	0
1120	224	2295		0
1121	225	2288	66	0
1122	225	2289	225	0
1123	225	2292	12	0
1124	225	2294	8	0
1125	225	2295		0
1126	226	2288	88	0
1127	226	2289	226	0
1128	226	2292	12	0
1129	226	2294	7	0
1130	226	2295		0
1131	227	2288	95	0
1132	227	2289	227	0
1133	227	2292	12	0
1134	227	2294	15	0
1135	227	2295		0
1136	228	2288	102	0
1137	228	2289	228	0
1138	228	2292	12	0
1139	228	2294	11	0
1140	228	2295		0
1141	229	2288	118	0
1142	229	2289	229	0
1143	229	2292	12	0
1144	229	2294	18	0
1145	229	2295		0
1146	230	2288	126	0
1147	230	2289	230	0
1148	230	2292	12	0
1149	230	2294	4	0
1150	230	2295		0
1151	231	2288	132	0
1152	231	2289	231	0
1153	231	2292	12	0
1154	231	2294	20	0
1155	231	2295		0
1156	232	2288	144	0
1157	232	2289	232	0
1158	232	2292	12	0
1159	232	2294	6	0
1160	232	2295		0
1161	233	2288	154	0
1162	233	2289	233	0
1163	233	2292	12	0
1164	233	2294	2	0
1165	233	2295		0
1166	234	2288	185	0
1167	234	2289	234	0
1168	234	2292	12	0
1169	234	2294	17	0
1170	234	2295		0
1171	235	2288	190	0
1172	235	2289	235	0
1173	235	2292	12	0
1174	235	2294	16	0
1175	235	2295		0
1176	236	2288	198	0
1177	236	2289	236	0
1178	236	2292	12	0
1179	236	2294	5	0
1180	236	2295		0
1181	237	2288	204	0
1182	237	2289	237	0
1183	237	2292	12	0
1184	237	2294	10	0
1185	237	2295		0
1186	238	2288	222	0
1187	238	2289	238	0
1188	238	2292	12	0
1189	238	2294	1	0
1190	238	2295		0
1191	239	2288	232	0
1192	239	2289	239	0
1193	239	2292	12	0
1194	239	2294	19	0
1195	239	2295		0
1196	240	2288	240	0
1197	240	2289	240	0
1198	240	2292	12	0
1199	240	2294	13	0
1200	240	2295		0
\.


--
-- Data for Name: stockprop_pub; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY stockprop_pub (stockprop_pub_id, stockprop_id, pub_id) FROM stdin;
\.


--
-- Data for Name: study; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY study (study_id, contact_id, pub_id, dbxref_id, name, description) FROM stdin;
\.


--
-- Data for Name: study_assay; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY study_assay (study_assay_id, study_id, assay_id) FROM stdin;
\.


--
-- Data for Name: studydesign; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY studydesign (studydesign_id, study_id, description) FROM stdin;
\.


--
-- Data for Name: studydesignprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY studydesignprop (studydesignprop_id, studydesign_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: studyfactor; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY studyfactor (studyfactor_id, studydesign_id, type_id, name, description) FROM stdin;
\.


--
-- Data for Name: studyfactorvalue; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY studyfactorvalue (studyfactorvalue_id, studyfactor_id, assay_id, factorvalue, name, rank) FROM stdin;
\.


--
-- Data for Name: studyprop; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY studyprop (studyprop_id, study_id, type_id, value, rank) FROM stdin;
\.


--
-- Data for Name: studyprop_feature; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY studyprop_feature (studyprop_feature_id, studyprop_id, feature_id, type_id) FROM stdin;
\.


--
-- Data for Name: synonym; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY synonym (synonym_id, name, type_id, synonym_sgml) FROM stdin;
\.


--
-- Data for Name: tableinfo; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY tableinfo (tableinfo_id, name, primary_key_column, is_view, view_on_table_id, superclass_table_id, is_updateable, modification_date) FROM stdin;
\.


--
-- Data for Name: treatment; Type: TABLE DATA; Schema: public; Owner: gmod
--

COPY treatment (treatment_id, rank, biomaterial_id, type_id, protocol_id, name) FROM stdin;
\.


--
-- Name: acquisition_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT acquisition_c1 UNIQUE (name);


--
-- Name: acquisition_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT acquisition_pkey PRIMARY KEY (acquisition_id);


--
-- Name: acquisition_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT acquisition_relationship_c1 UNIQUE (subject_id, object_id, type_id, rank);


--
-- Name: acquisition_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT acquisition_relationship_pkey PRIMARY KEY (acquisition_relationship_id);


--
-- Name: acquisitionprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY acquisitionprop
    ADD CONSTRAINT acquisitionprop_c1 UNIQUE (acquisition_id, type_id, rank);


--
-- Name: acquisitionprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY acquisitionprop
    ADD CONSTRAINT acquisitionprop_pkey PRIMARY KEY (acquisitionprop_id);


--
-- Name: analysis_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysis
    ADD CONSTRAINT analysis_c1 UNIQUE (program, programversion, sourcename);


--
-- Name: analysis_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysis
    ADD CONSTRAINT analysis_pkey PRIMARY KEY (analysis_id);


--
-- Name: analysisfeature_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysisfeature
    ADD CONSTRAINT analysisfeature_c1 UNIQUE (feature_id, analysis_id);


--
-- Name: analysisfeature_id_type_id_rank; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysisfeatureprop
    ADD CONSTRAINT analysisfeature_id_type_id_rank UNIQUE (analysisfeature_id, type_id, rank);


--
-- Name: analysisfeature_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysisfeature
    ADD CONSTRAINT analysisfeature_pkey PRIMARY KEY (analysisfeature_id);


--
-- Name: analysisfeatureprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysisfeatureprop
    ADD CONSTRAINT analysisfeatureprop_pkey PRIMARY KEY (analysisfeatureprop_id);


--
-- Name: analysisprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysisprop
    ADD CONSTRAINT analysisprop_c1 UNIQUE (analysis_id, type_id, rank);


--
-- Name: analysisprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY analysisprop
    ADD CONSTRAINT analysisprop_pkey PRIMARY KEY (analysisprop_id);


--
-- Name: arraydesign_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT arraydesign_c1 UNIQUE (name);


--
-- Name: arraydesign_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT arraydesign_pkey PRIMARY KEY (arraydesign_id);


--
-- Name: arraydesignprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY arraydesignprop
    ADD CONSTRAINT arraydesignprop_c1 UNIQUE (arraydesign_id, type_id, rank);


--
-- Name: arraydesignprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY arraydesignprop
    ADD CONSTRAINT arraydesignprop_pkey PRIMARY KEY (arraydesignprop_id);


--
-- Name: assay_biomaterial_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT assay_biomaterial_c1 UNIQUE (assay_id, biomaterial_id, channel_id, rank);


--
-- Name: assay_biomaterial_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT assay_biomaterial_pkey PRIMARY KEY (assay_biomaterial_id);


--
-- Name: assay_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT assay_c1 UNIQUE (name);


--
-- Name: assay_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT assay_pkey PRIMARY KEY (assay_id);


--
-- Name: assay_project_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assay_project
    ADD CONSTRAINT assay_project_c1 UNIQUE (assay_id, project_id);


--
-- Name: assay_project_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assay_project
    ADD CONSTRAINT assay_project_pkey PRIMARY KEY (assay_project_id);


--
-- Name: assayprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assayprop
    ADD CONSTRAINT assayprop_c1 UNIQUE (assay_id, type_id, rank);


--
-- Name: assayprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY assayprop
    ADD CONSTRAINT assayprop_pkey PRIMARY KEY (assayprop_id);


--
-- Name: biomaterial_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT biomaterial_c1 UNIQUE (name);


--
-- Name: biomaterial_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial_dbxref
    ADD CONSTRAINT biomaterial_dbxref_c1 UNIQUE (biomaterial_id, dbxref_id);


--
-- Name: biomaterial_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial_dbxref
    ADD CONSTRAINT biomaterial_dbxref_pkey PRIMARY KEY (biomaterial_dbxref_id);


--
-- Name: biomaterial_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT biomaterial_pkey PRIMARY KEY (biomaterial_id);


--
-- Name: biomaterial_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT biomaterial_relationship_c1 UNIQUE (subject_id, object_id, type_id);


--
-- Name: biomaterial_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT biomaterial_relationship_pkey PRIMARY KEY (biomaterial_relationship_id);


--
-- Name: biomaterial_treatment_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT biomaterial_treatment_c1 UNIQUE (biomaterial_id, treatment_id);


--
-- Name: biomaterial_treatment_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT biomaterial_treatment_pkey PRIMARY KEY (biomaterial_treatment_id);


--
-- Name: biomaterialprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterialprop
    ADD CONSTRAINT biomaterialprop_c1 UNIQUE (biomaterial_id, type_id, rank);


--
-- Name: biomaterialprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY biomaterialprop
    ADD CONSTRAINT biomaterialprop_pkey PRIMARY KEY (biomaterialprop_id);


--
-- Name: cell_line_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line
    ADD CONSTRAINT cell_line_c1 UNIQUE (uniquename, organism_id);


--
-- Name: cell_line_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT cell_line_cvterm_c1 UNIQUE (cell_line_id, cvterm_id, pub_id, rank);


--
-- Name: cell_line_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT cell_line_cvterm_pkey PRIMARY KEY (cell_line_cvterm_id);


--
-- Name: cell_line_cvtermprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_cvtermprop
    ADD CONSTRAINT cell_line_cvtermprop_c1 UNIQUE (cell_line_cvterm_id, type_id, rank);


--
-- Name: cell_line_cvtermprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_cvtermprop
    ADD CONSTRAINT cell_line_cvtermprop_pkey PRIMARY KEY (cell_line_cvtermprop_id);


--
-- Name: cell_line_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_dbxref
    ADD CONSTRAINT cell_line_dbxref_c1 UNIQUE (cell_line_id, dbxref_id);


--
-- Name: cell_line_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_dbxref
    ADD CONSTRAINT cell_line_dbxref_pkey PRIMARY KEY (cell_line_dbxref_id);


--
-- Name: cell_line_feature_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT cell_line_feature_c1 UNIQUE (cell_line_id, feature_id, pub_id);


--
-- Name: cell_line_feature_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT cell_line_feature_pkey PRIMARY KEY (cell_line_feature_id);


--
-- Name: cell_line_library_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT cell_line_library_c1 UNIQUE (cell_line_id, library_id, pub_id);


--
-- Name: cell_line_library_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT cell_line_library_pkey PRIMARY KEY (cell_line_library_id);


--
-- Name: cell_line_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line
    ADD CONSTRAINT cell_line_pkey PRIMARY KEY (cell_line_id);


--
-- Name: cell_line_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_pub
    ADD CONSTRAINT cell_line_pub_c1 UNIQUE (cell_line_id, pub_id);


--
-- Name: cell_line_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_pub
    ADD CONSTRAINT cell_line_pub_pkey PRIMARY KEY (cell_line_pub_id);


--
-- Name: cell_line_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT cell_line_relationship_c1 UNIQUE (subject_id, object_id, type_id);


--
-- Name: cell_line_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT cell_line_relationship_pkey PRIMARY KEY (cell_line_relationship_id);


--
-- Name: cell_line_synonym_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT cell_line_synonym_c1 UNIQUE (synonym_id, cell_line_id, pub_id);


--
-- Name: cell_line_synonym_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT cell_line_synonym_pkey PRIMARY KEY (cell_line_synonym_id);


--
-- Name: cell_lineprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_lineprop
    ADD CONSTRAINT cell_lineprop_c1 UNIQUE (cell_line_id, type_id, rank);


--
-- Name: cell_lineprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_lineprop
    ADD CONSTRAINT cell_lineprop_pkey PRIMARY KEY (cell_lineprop_id);


--
-- Name: cell_lineprop_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_lineprop_pub
    ADD CONSTRAINT cell_lineprop_pub_c1 UNIQUE (cell_lineprop_id, pub_id);


--
-- Name: cell_lineprop_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cell_lineprop_pub
    ADD CONSTRAINT cell_lineprop_pub_pkey PRIMARY KEY (cell_lineprop_pub_id);


--
-- Name: channel_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY channel
    ADD CONSTRAINT channel_c1 UNIQUE (name);


--
-- Name: channel_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY channel
    ADD CONSTRAINT channel_pkey PRIMARY KEY (channel_id);


--
-- Name: contact_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY contact
    ADD CONSTRAINT contact_c1 UNIQUE (name);


--
-- Name: contact_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY contact
    ADD CONSTRAINT contact_pkey PRIMARY KEY (contact_id);


--
-- Name: contact_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT contact_relationship_c1 UNIQUE (subject_id, object_id, type_id);


--
-- Name: contact_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT contact_relationship_pkey PRIMARY KEY (contact_relationship_id);


--
-- Name: contactprop_contact_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY contactprop
    ADD CONSTRAINT contactprop_contact_id_key UNIQUE (contact_id, type_id, value);


--
-- Name: contactprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY contactprop
    ADD CONSTRAINT contactprop_pkey PRIMARY KEY (contactprop_id);


--
-- Name: control_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY control
    ADD CONSTRAINT control_pkey PRIMARY KEY (control_id);


--
-- Name: cv_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cv
    ADD CONSTRAINT cv_c1 UNIQUE (name);


--
-- Name: cv_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cv
    ADD CONSTRAINT cv_pkey PRIMARY KEY (cv_id);


--
-- Name: cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvterm
    ADD CONSTRAINT cvterm_c1 UNIQUE (name, cv_id, is_obsolete);


--
-- Name: cvterm_c2; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvterm
    ADD CONSTRAINT cvterm_c2 UNIQUE (dbxref_id);


--
-- Name: cvterm_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvterm_dbxref
    ADD CONSTRAINT cvterm_dbxref_c1 UNIQUE (cvterm_id, dbxref_id);


--
-- Name: cvterm_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvterm_dbxref
    ADD CONSTRAINT cvterm_dbxref_pkey PRIMARY KEY (cvterm_dbxref_id);


--
-- Name: cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvterm
    ADD CONSTRAINT cvterm_pkey PRIMARY KEY (cvterm_id);


--
-- Name: cvterm_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT cvterm_relationship_c1 UNIQUE (subject_id, object_id, type_id);


--
-- Name: cvterm_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT cvterm_relationship_pkey PRIMARY KEY (cvterm_relationship_id);


--
-- Name: cvtermpath_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT cvtermpath_c1 UNIQUE (subject_id, object_id, type_id, pathdistance);


--
-- Name: cvtermpath_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT cvtermpath_pkey PRIMARY KEY (cvtermpath_id);


--
-- Name: cvtermprop_cvterm_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvtermprop
    ADD CONSTRAINT cvtermprop_cvterm_id_key UNIQUE (cvterm_id, type_id, value, rank);


--
-- Name: cvtermprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvtermprop
    ADD CONSTRAINT cvtermprop_pkey PRIMARY KEY (cvtermprop_id);


--
-- Name: cvtermsynonym_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvtermsynonym
    ADD CONSTRAINT cvtermsynonym_c1 UNIQUE (cvterm_id, synonym);


--
-- Name: cvtermsynonym_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY cvtermsynonym
    ADD CONSTRAINT cvtermsynonym_pkey PRIMARY KEY (cvtermsynonym_id);


--
-- Name: db_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY db
    ADD CONSTRAINT db_c1 UNIQUE (name);


--
-- Name: db_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY db
    ADD CONSTRAINT db_pkey PRIMARY KEY (db_id);


--
-- Name: dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY dbxref
    ADD CONSTRAINT dbxref_c1 UNIQUE (db_id, accession, version);


--
-- Name: dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY dbxref
    ADD CONSTRAINT dbxref_pkey PRIMARY KEY (dbxref_id);


--
-- Name: dbxrefprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY dbxrefprop
    ADD CONSTRAINT dbxrefprop_c1 UNIQUE (dbxref_id, type_id, rank);


--
-- Name: dbxrefprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY dbxrefprop
    ADD CONSTRAINT dbxrefprop_pkey PRIMARY KEY (dbxrefprop_id);


--
-- Name: eimage_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY eimage
    ADD CONSTRAINT eimage_pkey PRIMARY KEY (eimage_id);


--
-- Name: element_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY element
    ADD CONSTRAINT element_c1 UNIQUE (feature_id, arraydesign_id);


--
-- Name: element_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY element
    ADD CONSTRAINT element_pkey PRIMARY KEY (element_id);


--
-- Name: element_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT element_relationship_c1 UNIQUE (subject_id, object_id, type_id, rank);


--
-- Name: element_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT element_relationship_pkey PRIMARY KEY (element_relationship_id);


--
-- Name: elementresult_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY elementresult
    ADD CONSTRAINT elementresult_c1 UNIQUE (element_id, quantification_id);


--
-- Name: elementresult_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY elementresult
    ADD CONSTRAINT elementresult_pkey PRIMARY KEY (elementresult_id);


--
-- Name: elementresult_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT elementresult_relationship_c1 UNIQUE (subject_id, object_id, type_id, rank);


--
-- Name: elementresult_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT elementresult_relationship_pkey PRIMARY KEY (elementresult_relationship_id);


--
-- Name: environment_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY environment
    ADD CONSTRAINT environment_c1 UNIQUE (uniquename);


--
-- Name: environment_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY environment_cvterm
    ADD CONSTRAINT environment_cvterm_c1 UNIQUE (environment_id, cvterm_id);


--
-- Name: environment_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY environment_cvterm
    ADD CONSTRAINT environment_cvterm_pkey PRIMARY KEY (environment_cvterm_id);


--
-- Name: environment_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY environment
    ADD CONSTRAINT environment_pkey PRIMARY KEY (environment_id);


--
-- Name: expression_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression
    ADD CONSTRAINT expression_c1 UNIQUE (uniquename);


--
-- Name: expression_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT expression_cvterm_c1 UNIQUE (expression_id, cvterm_id, cvterm_type_id);


--
-- Name: expression_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT expression_cvterm_pkey PRIMARY KEY (expression_cvterm_id);


--
-- Name: expression_cvtermprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_cvtermprop
    ADD CONSTRAINT expression_cvtermprop_c1 UNIQUE (expression_cvterm_id, type_id, rank);


--
-- Name: expression_cvtermprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_cvtermprop
    ADD CONSTRAINT expression_cvtermprop_pkey PRIMARY KEY (expression_cvtermprop_id);


--
-- Name: expression_image_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_image
    ADD CONSTRAINT expression_image_c1 UNIQUE (expression_id, eimage_id);


--
-- Name: expression_image_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_image
    ADD CONSTRAINT expression_image_pkey PRIMARY KEY (expression_image_id);


--
-- Name: expression_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression
    ADD CONSTRAINT expression_pkey PRIMARY KEY (expression_id);


--
-- Name: expression_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_pub
    ADD CONSTRAINT expression_pub_c1 UNIQUE (expression_id, pub_id);


--
-- Name: expression_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expression_pub
    ADD CONSTRAINT expression_pub_pkey PRIMARY KEY (expression_pub_id);


--
-- Name: expressionprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expressionprop
    ADD CONSTRAINT expressionprop_c1 UNIQUE (expression_id, type_id, rank);


--
-- Name: expressionprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY expressionprop
    ADD CONSTRAINT expressionprop_pkey PRIMARY KEY (expressionprop_id);


--
-- Name: feature_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT feature_c1 UNIQUE (organism_id, uniquename, type_id);


--
-- Name: feature_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT feature_cvterm_c1 UNIQUE (feature_id, cvterm_id, pub_id, rank);


--
-- Name: feature_cvterm_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvterm_dbxref
    ADD CONSTRAINT feature_cvterm_dbxref_c1 UNIQUE (feature_cvterm_id, dbxref_id);


--
-- Name: feature_cvterm_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvterm_dbxref
    ADD CONSTRAINT feature_cvterm_dbxref_pkey PRIMARY KEY (feature_cvterm_dbxref_id);


--
-- Name: feature_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT feature_cvterm_pkey PRIMARY KEY (feature_cvterm_id);


--
-- Name: feature_cvterm_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvterm_pub
    ADD CONSTRAINT feature_cvterm_pub_c1 UNIQUE (feature_cvterm_id, pub_id);


--
-- Name: feature_cvterm_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvterm_pub
    ADD CONSTRAINT feature_cvterm_pub_pkey PRIMARY KEY (feature_cvterm_pub_id);


--
-- Name: feature_cvtermprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvtermprop
    ADD CONSTRAINT feature_cvtermprop_c1 UNIQUE (feature_cvterm_id, type_id, rank);


--
-- Name: feature_cvtermprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_cvtermprop
    ADD CONSTRAINT feature_cvtermprop_pkey PRIMARY KEY (feature_cvtermprop_id);


--
-- Name: feature_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_dbxref
    ADD CONSTRAINT feature_dbxref_c1 UNIQUE (feature_id, dbxref_id);


--
-- Name: feature_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_dbxref
    ADD CONSTRAINT feature_dbxref_pkey PRIMARY KEY (feature_dbxref_id);


--
-- Name: feature_expression_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT feature_expression_c1 UNIQUE (expression_id, feature_id, pub_id);


--
-- Name: feature_expression_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT feature_expression_pkey PRIMARY KEY (feature_expression_id);


--
-- Name: feature_expressionprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_expressionprop
    ADD CONSTRAINT feature_expressionprop_c1 UNIQUE (feature_expression_id, type_id, rank);


--
-- Name: feature_expressionprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_expressionprop
    ADD CONSTRAINT feature_expressionprop_pkey PRIMARY KEY (feature_expressionprop_id);


--
-- Name: feature_genotype_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT feature_genotype_c1 UNIQUE (feature_id, genotype_id, cvterm_id, chromosome_id, rank, cgroup);


--
-- Name: feature_genotype_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT feature_genotype_pkey PRIMARY KEY (feature_genotype_id);


--
-- Name: feature_phenotype_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_phenotype
    ADD CONSTRAINT feature_phenotype_c1 UNIQUE (feature_id, phenotype_id);


--
-- Name: feature_phenotype_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_phenotype
    ADD CONSTRAINT feature_phenotype_pkey PRIMARY KEY (feature_phenotype_id);


--
-- Name: feature_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT feature_pkey PRIMARY KEY (feature_id);


--
-- Name: feature_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_pub
    ADD CONSTRAINT feature_pub_c1 UNIQUE (feature_id, pub_id);


--
-- Name: feature_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_pub
    ADD CONSTRAINT feature_pub_pkey PRIMARY KEY (feature_pub_id);


--
-- Name: feature_pubprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_pubprop
    ADD CONSTRAINT feature_pubprop_c1 UNIQUE (feature_pub_id, type_id, rank);


--
-- Name: feature_pubprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_pubprop
    ADD CONSTRAINT feature_pubprop_pkey PRIMARY KEY (feature_pubprop_id);


--
-- Name: feature_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT feature_relationship_c1 UNIQUE (subject_id, object_id, type_id, rank);


--
-- Name: feature_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT feature_relationship_pkey PRIMARY KEY (feature_relationship_id);


--
-- Name: feature_relationship_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationship_pub
    ADD CONSTRAINT feature_relationship_pub_c1 UNIQUE (feature_relationship_id, pub_id);


--
-- Name: feature_relationship_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationship_pub
    ADD CONSTRAINT feature_relationship_pub_pkey PRIMARY KEY (feature_relationship_pub_id);


--
-- Name: feature_relationshipprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationshipprop
    ADD CONSTRAINT feature_relationshipprop_c1 UNIQUE (feature_relationship_id, type_id, rank);


--
-- Name: feature_relationshipprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationshipprop
    ADD CONSTRAINT feature_relationshipprop_pkey PRIMARY KEY (feature_relationshipprop_id);


--
-- Name: feature_relationshipprop_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationshipprop_pub
    ADD CONSTRAINT feature_relationshipprop_pub_c1 UNIQUE (feature_relationshipprop_id, pub_id);


--
-- Name: feature_relationshipprop_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_relationshipprop_pub
    ADD CONSTRAINT feature_relationshipprop_pub_pkey PRIMARY KEY (feature_relationshipprop_pub_id);


--
-- Name: feature_synonym_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT feature_synonym_c1 UNIQUE (synonym_id, feature_id, pub_id);


--
-- Name: feature_synonym_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT feature_synonym_pkey PRIMARY KEY (feature_synonym_id);


--
-- Name: featureloc_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureloc
    ADD CONSTRAINT featureloc_c1 UNIQUE (feature_id, locgroup, rank);


--
-- Name: featureloc_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureloc
    ADD CONSTRAINT featureloc_pkey PRIMARY KEY (featureloc_id);


--
-- Name: featureloc_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureloc_pub
    ADD CONSTRAINT featureloc_pub_c1 UNIQUE (featureloc_id, pub_id);


--
-- Name: featureloc_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureloc_pub
    ADD CONSTRAINT featureloc_pub_pkey PRIMARY KEY (featureloc_pub_id);


--
-- Name: featuremap_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featuremap
    ADD CONSTRAINT featuremap_c1 UNIQUE (name);


--
-- Name: featuremap_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featuremap
    ADD CONSTRAINT featuremap_pkey PRIMARY KEY (featuremap_id);


--
-- Name: featuremap_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featuremap_pub
    ADD CONSTRAINT featuremap_pub_pkey PRIMARY KEY (featuremap_pub_id);


--
-- Name: featurepos_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featurepos
    ADD CONSTRAINT featurepos_pkey PRIMARY KEY (featurepos_id);


--
-- Name: featureprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureprop
    ADD CONSTRAINT featureprop_c1 UNIQUE (feature_id, type_id, rank);


--
-- Name: featureprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureprop
    ADD CONSTRAINT featureprop_pkey PRIMARY KEY (featureprop_id);


--
-- Name: featureprop_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureprop_pub
    ADD CONSTRAINT featureprop_pub_c1 UNIQUE (featureprop_id, pub_id);


--
-- Name: featureprop_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featureprop_pub
    ADD CONSTRAINT featureprop_pub_pkey PRIMARY KEY (featureprop_pub_id);


--
-- Name: featurerange_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT featurerange_pkey PRIMARY KEY (featurerange_id);


--
-- Name: genotype_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY genotype
    ADD CONSTRAINT genotype_c1 UNIQUE (uniquename);


--
-- Name: genotype_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY genotype
    ADD CONSTRAINT genotype_pkey PRIMARY KEY (genotype_id);


--
-- Name: library_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library
    ADD CONSTRAINT library_c1 UNIQUE (organism_id, uniquename, type_id);


--
-- Name: library_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT library_cvterm_c1 UNIQUE (library_id, cvterm_id, pub_id);


--
-- Name: library_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT library_cvterm_pkey PRIMARY KEY (library_cvterm_id);


--
-- Name: library_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_dbxref
    ADD CONSTRAINT library_dbxref_c1 UNIQUE (library_id, dbxref_id);


--
-- Name: library_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_dbxref
    ADD CONSTRAINT library_dbxref_pkey PRIMARY KEY (library_dbxref_id);


--
-- Name: library_feature_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_feature
    ADD CONSTRAINT library_feature_c1 UNIQUE (library_id, feature_id);


--
-- Name: library_feature_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_feature
    ADD CONSTRAINT library_feature_pkey PRIMARY KEY (library_feature_id);


--
-- Name: library_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library
    ADD CONSTRAINT library_pkey PRIMARY KEY (library_id);


--
-- Name: library_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_pub
    ADD CONSTRAINT library_pub_c1 UNIQUE (library_id, pub_id);


--
-- Name: library_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_pub
    ADD CONSTRAINT library_pub_pkey PRIMARY KEY (library_pub_id);


--
-- Name: library_synonym_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT library_synonym_c1 UNIQUE (synonym_id, library_id, pub_id);


--
-- Name: library_synonym_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT library_synonym_pkey PRIMARY KEY (library_synonym_id);


--
-- Name: libraryprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY libraryprop
    ADD CONSTRAINT libraryprop_c1 UNIQUE (library_id, type_id, rank);


--
-- Name: libraryprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY libraryprop
    ADD CONSTRAINT libraryprop_pkey PRIMARY KEY (libraryprop_id);


--
-- Name: libraryprop_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY libraryprop_pub
    ADD CONSTRAINT libraryprop_pub_c1 UNIQUE (libraryprop_id, pub_id);


--
-- Name: libraryprop_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY libraryprop_pub
    ADD CONSTRAINT libraryprop_pub_pkey PRIMARY KEY (libraryprop_pub_id);


--
-- Name: magedocumentation_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY magedocumentation
    ADD CONSTRAINT magedocumentation_pkey PRIMARY KEY (magedocumentation_id);


--
-- Name: mageml_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY mageml
    ADD CONSTRAINT mageml_pkey PRIMARY KEY (mageml_id);


--
-- Name: materialized_view_name_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY materialized_view
    ADD CONSTRAINT materialized_view_name_key UNIQUE (name);


--
-- Name: nd_experiment_contact_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_contact
    ADD CONSTRAINT nd_experiment_contact_pkey PRIMARY KEY (nd_experiment_contact_id);


--
-- Name: nd_experiment_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_dbxref
    ADD CONSTRAINT nd_experiment_dbxref_pkey PRIMARY KEY (nd_experiment_dbxref_id);


--
-- Name: nd_experiment_genotype_nd_experiment_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_genotype
    ADD CONSTRAINT nd_experiment_genotype_nd_experiment_id_key UNIQUE (nd_experiment_id);


--
-- Name: nd_experiment_genotype_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_genotype
    ADD CONSTRAINT nd_experiment_genotype_pkey PRIMARY KEY (nd_experiment_genotype_id);


--
-- Name: nd_experiment_phenotype_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_phenotype
    ADD CONSTRAINT nd_experiment_phenotype_pkey PRIMARY KEY (nd_experiment_phenotype_id);


--
-- Name: nd_experiment_phenotypeprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_phenotypeprop
    ADD CONSTRAINT nd_experiment_phenotypeprop_c1 UNIQUE (nd_experiment_phenotype_id, type_id, rank);


--
-- Name: nd_experiment_phenotypeprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_phenotypeprop
    ADD CONSTRAINT nd_experiment_phenotypeprop_pkey PRIMARY KEY (nd_experiment_phenotypeprop_id);


--
-- Name: nd_experiment_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment
    ADD CONSTRAINT nd_experiment_pkey PRIMARY KEY (nd_experiment_id);


--
-- Name: nd_experiment_project_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_project
    ADD CONSTRAINT nd_experiment_project_pkey PRIMARY KEY (nd_experiment_project_id);


--
-- Name: nd_experiment_protocol_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_protocol
    ADD CONSTRAINT nd_experiment_protocol_pkey PRIMARY KEY (nd_experiment_protocol_id);


--
-- Name: nd_experiment_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_pub
    ADD CONSTRAINT nd_experiment_pub_c1 UNIQUE (nd_experiment_id, pub_id);


--
-- Name: nd_experiment_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_pub
    ADD CONSTRAINT nd_experiment_pub_pkey PRIMARY KEY (nd_experiment_pub_id);


--
-- Name: nd_experiment_stock_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_stock_dbxref
    ADD CONSTRAINT nd_experiment_stock_dbxref_pkey PRIMARY KEY (nd_experiment_stock_dbxref_id);


--
-- Name: nd_experiment_stock_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_stock
    ADD CONSTRAINT nd_experiment_stock_pkey PRIMARY KEY (nd_experiment_stock_id);


--
-- Name: nd_experiment_stockprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_stockprop
    ADD CONSTRAINT nd_experiment_stockprop_c1 UNIQUE (nd_experiment_stock_id, type_id, rank);


--
-- Name: nd_experiment_stockprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experiment_stockprop
    ADD CONSTRAINT nd_experiment_stockprop_pkey PRIMARY KEY (nd_experiment_stockprop_id);


--
-- Name: nd_experimentprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experimentprop
    ADD CONSTRAINT nd_experimentprop_c1 UNIQUE (nd_experiment_id, type_id, rank);


--
-- Name: nd_experimentprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_experimentprop
    ADD CONSTRAINT nd_experimentprop_pkey PRIMARY KEY (nd_experimentprop_id);


--
-- Name: nd_geolocation_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_geolocation
    ADD CONSTRAINT nd_geolocation_pkey PRIMARY KEY (nd_geolocation_id);


--
-- Name: nd_geolocationprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_geolocationprop
    ADD CONSTRAINT nd_geolocationprop_c1 UNIQUE (nd_geolocation_id, type_id, rank);


--
-- Name: nd_geolocationprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_geolocationprop
    ADD CONSTRAINT nd_geolocationprop_pkey PRIMARY KEY (nd_geolocationprop_id);


--
-- Name: nd_protocol_name_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_protocol
    ADD CONSTRAINT nd_protocol_name_key UNIQUE (name);


--
-- Name: nd_protocol_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_protocol
    ADD CONSTRAINT nd_protocol_pkey PRIMARY KEY (nd_protocol_id);


--
-- Name: nd_protocol_reagent_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_protocol_reagent
    ADD CONSTRAINT nd_protocol_reagent_pkey PRIMARY KEY (nd_protocol_reagent_id);


--
-- Name: nd_protocolprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_protocolprop
    ADD CONSTRAINT nd_protocolprop_c1 UNIQUE (nd_protocol_id, type_id, rank);


--
-- Name: nd_protocolprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_protocolprop
    ADD CONSTRAINT nd_protocolprop_pkey PRIMARY KEY (nd_protocolprop_id);


--
-- Name: nd_reagent_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_reagent
    ADD CONSTRAINT nd_reagent_pkey PRIMARY KEY (nd_reagent_id);


--
-- Name: nd_reagent_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_reagent_relationship
    ADD CONSTRAINT nd_reagent_relationship_pkey PRIMARY KEY (nd_reagent_relationship_id);


--
-- Name: nd_reagentprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_reagentprop
    ADD CONSTRAINT nd_reagentprop_c1 UNIQUE (nd_reagent_id, type_id, rank);


--
-- Name: nd_reagentprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY nd_reagentprop
    ADD CONSTRAINT nd_reagentprop_pkey PRIMARY KEY (nd_reagentprop_id);


--
-- Name: organism_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY organism
    ADD CONSTRAINT organism_c1 UNIQUE (genus, species);


--
-- Name: organism_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY organism_dbxref
    ADD CONSTRAINT organism_dbxref_c1 UNIQUE (organism_id, dbxref_id);


--
-- Name: organism_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY organism_dbxref
    ADD CONSTRAINT organism_dbxref_pkey PRIMARY KEY (organism_dbxref_id);


--
-- Name: organism_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY organism
    ADD CONSTRAINT organism_pkey PRIMARY KEY (organism_id);


--
-- Name: organismprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY organismprop
    ADD CONSTRAINT organismprop_c1 UNIQUE (organism_id, type_id, rank);


--
-- Name: organismprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY organismprop
    ADD CONSTRAINT organismprop_pkey PRIMARY KEY (organismprop_id);


--
-- Name: phendesc_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT phendesc_c1 UNIQUE (genotype_id, environment_id, type_id, pub_id);


--
-- Name: phendesc_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT phendesc_pkey PRIMARY KEY (phendesc_id);


--
-- Name: phenotype_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT phenotype_c1 UNIQUE (uniquename);


--
-- Name: phenotype_comparison_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_c1 UNIQUE (genotype1_id, environment1_id, genotype2_id, environment2_id, phenotype1_id, pub_id);


--
-- Name: phenotype_comparison_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT phenotype_comparison_cvterm_c1 UNIQUE (phenotype_comparison_id, cvterm_id);


--
-- Name: phenotype_comparison_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT phenotype_comparison_cvterm_pkey PRIMARY KEY (phenotype_comparison_cvterm_id);


--
-- Name: phenotype_comparison_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_pkey PRIMARY KEY (phenotype_comparison_id);


--
-- Name: phenotype_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype_cvterm
    ADD CONSTRAINT phenotype_cvterm_c1 UNIQUE (phenotype_id, cvterm_id, rank);


--
-- Name: phenotype_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype_cvterm
    ADD CONSTRAINT phenotype_cvterm_pkey PRIMARY KEY (phenotype_cvterm_id);


--
-- Name: phenotype_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT phenotype_pkey PRIMARY KEY (phenotype_id);


--
-- Name: phenotypeprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotypeprop
    ADD CONSTRAINT phenotypeprop_c1 UNIQUE (phenotype_id, type_id, rank);


--
-- Name: phenotypeprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenotypeprop
    ADD CONSTRAINT phenotypeprop_pkey PRIMARY KEY (phenotypeprop_id);


--
-- Name: phenstatement_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT phenstatement_c1 UNIQUE (genotype_id, phenotype_id, environment_id, type_id, pub_id);


--
-- Name: phenstatement_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT phenstatement_pkey PRIMARY KEY (phenstatement_id);


--
-- Name: phylonode_dbxref_phylonode_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_dbxref
    ADD CONSTRAINT phylonode_dbxref_phylonode_id_key UNIQUE (phylonode_id, dbxref_id);


--
-- Name: phylonode_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_dbxref
    ADD CONSTRAINT phylonode_dbxref_pkey PRIMARY KEY (phylonode_dbxref_id);


--
-- Name: phylonode_organism_phylonode_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_organism
    ADD CONSTRAINT phylonode_organism_phylonode_id_key UNIQUE (phylonode_id);


--
-- Name: phylonode_organism_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_organism
    ADD CONSTRAINT phylonode_organism_pkey PRIMARY KEY (phylonode_organism_id);


--
-- Name: phylonode_phylotree_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT phylonode_phylotree_id_key UNIQUE (phylotree_id, left_idx);


--
-- Name: phylonode_phylotree_id_key1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT phylonode_phylotree_id_key1 UNIQUE (phylotree_id, right_idx);


--
-- Name: phylonode_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT phylonode_pkey PRIMARY KEY (phylonode_id);


--
-- Name: phylonode_pub_phylonode_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_pub
    ADD CONSTRAINT phylonode_pub_phylonode_id_key UNIQUE (phylonode_id, pub_id);


--
-- Name: phylonode_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_pub
    ADD CONSTRAINT phylonode_pub_pkey PRIMARY KEY (phylonode_pub_id);


--
-- Name: phylonode_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT phylonode_relationship_pkey PRIMARY KEY (phylonode_relationship_id);


--
-- Name: phylonode_relationship_subject_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT phylonode_relationship_subject_id_key UNIQUE (subject_id, object_id, type_id);


--
-- Name: phylonodeprop_phylonode_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonodeprop
    ADD CONSTRAINT phylonodeprop_phylonode_id_key UNIQUE (phylonode_id, type_id, value, rank);


--
-- Name: phylonodeprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylonodeprop
    ADD CONSTRAINT phylonodeprop_pkey PRIMARY KEY (phylonodeprop_id);


--
-- Name: phylotree_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylotree
    ADD CONSTRAINT phylotree_pkey PRIMARY KEY (phylotree_id);


--
-- Name: phylotree_pub_phylotree_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylotree_pub
    ADD CONSTRAINT phylotree_pub_phylotree_id_key UNIQUE (phylotree_id, pub_id);


--
-- Name: phylotree_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY phylotree_pub
    ADD CONSTRAINT phylotree_pub_pkey PRIMARY KEY (phylotree_pub_id);


--
-- Name: project_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project
    ADD CONSTRAINT project_c1 UNIQUE (name);


--
-- Name: project_contact_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project_contact
    ADD CONSTRAINT project_contact_c1 UNIQUE (project_id, contact_id);


--
-- Name: project_contact_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project_contact
    ADD CONSTRAINT project_contact_pkey PRIMARY KEY (project_contact_id);


--
-- Name: project_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project
    ADD CONSTRAINT project_pkey PRIMARY KEY (project_id);


--
-- Name: project_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project_pub
    ADD CONSTRAINT project_pub_c1 UNIQUE (project_id, pub_id);


--
-- Name: project_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project_pub
    ADD CONSTRAINT project_pub_pkey PRIMARY KEY (project_pub_id);


--
-- Name: project_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT project_relationship_c1 UNIQUE (subject_project_id, object_project_id, type_id);


--
-- Name: project_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT project_relationship_pkey PRIMARY KEY (project_relationship_id);


--
-- Name: projectprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY projectprop
    ADD CONSTRAINT projectprop_c1 UNIQUE (project_id, type_id, rank);


--
-- Name: projectprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY projectprop
    ADD CONSTRAINT projectprop_pkey PRIMARY KEY (projectprop_id);


--
-- Name: protocol_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT protocol_c1 UNIQUE (name);


--
-- Name: protocol_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT protocol_pkey PRIMARY KEY (protocol_id);


--
-- Name: protocolparam_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY protocolparam
    ADD CONSTRAINT protocolparam_pkey PRIMARY KEY (protocolparam_id);


--
-- Name: pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pub
    ADD CONSTRAINT pub_c1 UNIQUE (uniquename);


--
-- Name: pub_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pub_dbxref
    ADD CONSTRAINT pub_dbxref_c1 UNIQUE (pub_id, dbxref_id);


--
-- Name: pub_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pub_dbxref
    ADD CONSTRAINT pub_dbxref_pkey PRIMARY KEY (pub_dbxref_id);


--
-- Name: pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pub
    ADD CONSTRAINT pub_pkey PRIMARY KEY (pub_id);


--
-- Name: pub_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT pub_relationship_c1 UNIQUE (subject_id, object_id, type_id);


--
-- Name: pub_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT pub_relationship_pkey PRIMARY KEY (pub_relationship_id);


--
-- Name: pubauthor_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pubauthor
    ADD CONSTRAINT pubauthor_c1 UNIQUE (pub_id, rank);


--
-- Name: pubauthor_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pubauthor
    ADD CONSTRAINT pubauthor_pkey PRIMARY KEY (pubauthor_id);


--
-- Name: pubprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pubprop
    ADD CONSTRAINT pubprop_c1 UNIQUE (pub_id, type_id, rank);


--
-- Name: pubprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY pubprop
    ADD CONSTRAINT pubprop_pkey PRIMARY KEY (pubprop_id);


--
-- Name: quantification_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT quantification_c1 UNIQUE (name, analysis_id);


--
-- Name: quantification_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT quantification_pkey PRIMARY KEY (quantification_id);


--
-- Name: quantification_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT quantification_relationship_c1 UNIQUE (subject_id, object_id, type_id);


--
-- Name: quantification_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT quantification_relationship_pkey PRIMARY KEY (quantification_relationship_id);


--
-- Name: quantificationprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY quantificationprop
    ADD CONSTRAINT quantificationprop_c1 UNIQUE (quantification_id, type_id, rank);


--
-- Name: quantificationprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY quantificationprop
    ADD CONSTRAINT quantificationprop_pkey PRIMARY KEY (quantificationprop_id);


--
-- Name: stock_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT stock_c1 UNIQUE (organism_id, uniquename, type_id);


--
-- Name: stock_cvterm_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT stock_cvterm_c1 UNIQUE (stock_id, cvterm_id, pub_id);


--
-- Name: stock_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT stock_cvterm_pkey PRIMARY KEY (stock_cvterm_id);


--
-- Name: stock_dbxref_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_dbxref
    ADD CONSTRAINT stock_dbxref_c1 UNIQUE (stock_id, dbxref_id);


--
-- Name: stock_dbxref_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_dbxref
    ADD CONSTRAINT stock_dbxref_pkey PRIMARY KEY (stock_dbxref_id);


--
-- Name: stock_genotype_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_genotype
    ADD CONSTRAINT stock_genotype_c1 UNIQUE (stock_id, genotype_id);


--
-- Name: stock_genotype_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_genotype
    ADD CONSTRAINT stock_genotype_pkey PRIMARY KEY (stock_genotype_id);


--
-- Name: stock_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT stock_pkey PRIMARY KEY (stock_id);


--
-- Name: stock_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_pub
    ADD CONSTRAINT stock_pub_c1 UNIQUE (stock_id, pub_id);


--
-- Name: stock_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_pub
    ADD CONSTRAINT stock_pub_pkey PRIMARY KEY (stock_pub_id);


--
-- Name: stock_relationship_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT stock_relationship_c1 UNIQUE (subject_id, object_id, type_id, rank);


--
-- Name: stock_relationship_cvterm_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_relationship_cvterm
    ADD CONSTRAINT stock_relationship_cvterm_pkey PRIMARY KEY (stock_relationship_cvterm_id);


--
-- Name: stock_relationship_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT stock_relationship_pkey PRIMARY KEY (stock_relationship_id);


--
-- Name: stock_relationship_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_relationship_pub
    ADD CONSTRAINT stock_relationship_pub_c1 UNIQUE (stock_relationship_id, pub_id);


--
-- Name: stock_relationship_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stock_relationship_pub
    ADD CONSTRAINT stock_relationship_pub_pkey PRIMARY KEY (stock_relationship_pub_id);


--
-- Name: stockcollection_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockcollection
    ADD CONSTRAINT stockcollection_c1 UNIQUE (uniquename, type_id);


--
-- Name: stockcollection_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockcollection
    ADD CONSTRAINT stockcollection_pkey PRIMARY KEY (stockcollection_id);


--
-- Name: stockcollection_stock_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockcollection_stock
    ADD CONSTRAINT stockcollection_stock_c1 UNIQUE (stockcollection_id, stock_id);


--
-- Name: stockcollection_stock_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockcollection_stock
    ADD CONSTRAINT stockcollection_stock_pkey PRIMARY KEY (stockcollection_stock_id);


--
-- Name: stockcollectionprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockcollectionprop
    ADD CONSTRAINT stockcollectionprop_c1 UNIQUE (stockcollection_id, type_id, rank);


--
-- Name: stockcollectionprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockcollectionprop
    ADD CONSTRAINT stockcollectionprop_pkey PRIMARY KEY (stockcollectionprop_id);


--
-- Name: stockprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockprop
    ADD CONSTRAINT stockprop_c1 UNIQUE (stock_id, type_id, rank);


--
-- Name: stockprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockprop
    ADD CONSTRAINT stockprop_pkey PRIMARY KEY (stockprop_id);


--
-- Name: stockprop_pub_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockprop_pub
    ADD CONSTRAINT stockprop_pub_c1 UNIQUE (stockprop_id, pub_id);


--
-- Name: stockprop_pub_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY stockprop_pub
    ADD CONSTRAINT stockprop_pub_pkey PRIMARY KEY (stockprop_pub_id);


--
-- Name: study_assay_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY study_assay
    ADD CONSTRAINT study_assay_c1 UNIQUE (study_id, assay_id);


--
-- Name: study_assay_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY study_assay
    ADD CONSTRAINT study_assay_pkey PRIMARY KEY (study_assay_id);


--
-- Name: study_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY study
    ADD CONSTRAINT study_c1 UNIQUE (name);


--
-- Name: study_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY study
    ADD CONSTRAINT study_pkey PRIMARY KEY (study_id);


--
-- Name: studydesign_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studydesign
    ADD CONSTRAINT studydesign_pkey PRIMARY KEY (studydesign_id);


--
-- Name: studydesignprop_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studydesignprop
    ADD CONSTRAINT studydesignprop_c1 UNIQUE (studydesign_id, type_id, rank);


--
-- Name: studydesignprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studydesignprop
    ADD CONSTRAINT studydesignprop_pkey PRIMARY KEY (studydesignprop_id);


--
-- Name: studyfactor_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studyfactor
    ADD CONSTRAINT studyfactor_pkey PRIMARY KEY (studyfactor_id);


--
-- Name: studyfactorvalue_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studyfactorvalue
    ADD CONSTRAINT studyfactorvalue_pkey PRIMARY KEY (studyfactorvalue_id);


--
-- Name: studyprop_feature_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT studyprop_feature_pkey PRIMARY KEY (studyprop_feature_id);


--
-- Name: studyprop_feature_studyprop_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT studyprop_feature_studyprop_id_key UNIQUE (studyprop_id, feature_id);


--
-- Name: studyprop_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studyprop
    ADD CONSTRAINT studyprop_pkey PRIMARY KEY (studyprop_id);


--
-- Name: studyprop_study_id_key; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY studyprop
    ADD CONSTRAINT studyprop_study_id_key UNIQUE (study_id, type_id, rank);


--
-- Name: synonym_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY synonym
    ADD CONSTRAINT synonym_c1 UNIQUE (name, type_id);


--
-- Name: synonym_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY synonym
    ADD CONSTRAINT synonym_pkey PRIMARY KEY (synonym_id);


--
-- Name: tableinfo_c1; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY tableinfo
    ADD CONSTRAINT tableinfo_c1 UNIQUE (name);


--
-- Name: tableinfo_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY tableinfo
    ADD CONSTRAINT tableinfo_pkey PRIMARY KEY (tableinfo_id);


--
-- Name: treatment_pkey; Type: CONSTRAINT; Schema: public; Owner: gmod; Tablespace: 
--

ALTER TABLE ONLY treatment
    ADD CONSTRAINT treatment_pkey PRIMARY KEY (treatment_id);


--
-- Name: acquisition_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisition_idx1 ON acquisition USING btree (assay_id);


--
-- Name: acquisition_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisition_idx2 ON acquisition USING btree (protocol_id);


--
-- Name: acquisition_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisition_idx3 ON acquisition USING btree (channel_id);


--
-- Name: acquisition_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisition_relationship_idx1 ON acquisition_relationship USING btree (subject_id);


--
-- Name: acquisition_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisition_relationship_idx2 ON acquisition_relationship USING btree (type_id);


--
-- Name: acquisition_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisition_relationship_idx3 ON acquisition_relationship USING btree (object_id);


--
-- Name: acquisitionprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisitionprop_idx1 ON acquisitionprop USING btree (acquisition_id);


--
-- Name: acquisitionprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX acquisitionprop_idx2 ON acquisitionprop USING btree (type_id);


--
-- Name: analysisfeature_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX analysisfeature_idx1 ON analysisfeature USING btree (feature_id);


--
-- Name: analysisfeature_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX analysisfeature_idx2 ON analysisfeature USING btree (analysis_id);


--
-- Name: analysisprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX analysisprop_idx1 ON analysisprop USING btree (analysis_id);


--
-- Name: analysisprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX analysisprop_idx2 ON analysisprop USING btree (type_id);


--
-- Name: arraydesign_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX arraydesign_idx1 ON arraydesign USING btree (manufacturer_id);


--
-- Name: arraydesign_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX arraydesign_idx2 ON arraydesign USING btree (platformtype_id);


--
-- Name: arraydesign_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX arraydesign_idx3 ON arraydesign USING btree (substratetype_id);


--
-- Name: arraydesign_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX arraydesign_idx4 ON arraydesign USING btree (protocol_id);


--
-- Name: arraydesign_idx5; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX arraydesign_idx5 ON arraydesign USING btree (dbxref_id);


--
-- Name: arraydesignprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX arraydesignprop_idx1 ON arraydesignprop USING btree (arraydesign_id);


--
-- Name: arraydesignprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX arraydesignprop_idx2 ON arraydesignprop USING btree (type_id);


--
-- Name: assay_biomaterial_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_biomaterial_idx1 ON assay_biomaterial USING btree (assay_id);


--
-- Name: assay_biomaterial_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_biomaterial_idx2 ON assay_biomaterial USING btree (biomaterial_id);


--
-- Name: assay_biomaterial_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_biomaterial_idx3 ON assay_biomaterial USING btree (channel_id);


--
-- Name: assay_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_idx1 ON assay USING btree (arraydesign_id);


--
-- Name: assay_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_idx2 ON assay USING btree (protocol_id);


--
-- Name: assay_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_idx3 ON assay USING btree (operator_id);


--
-- Name: assay_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_idx4 ON assay USING btree (dbxref_id);


--
-- Name: assay_project_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_project_idx1 ON assay_project USING btree (assay_id);


--
-- Name: assay_project_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assay_project_idx2 ON assay_project USING btree (project_id);


--
-- Name: assayprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assayprop_idx1 ON assayprop USING btree (assay_id);


--
-- Name: assayprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX assayprop_idx2 ON assayprop USING btree (type_id);


--
-- Name: biomaterial_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_dbxref_idx1 ON biomaterial_dbxref USING btree (biomaterial_id);


--
-- Name: biomaterial_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_dbxref_idx2 ON biomaterial_dbxref USING btree (dbxref_id);


--
-- Name: biomaterial_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_idx1 ON biomaterial USING btree (taxon_id);


--
-- Name: biomaterial_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_idx2 ON biomaterial USING btree (biosourceprovider_id);


--
-- Name: biomaterial_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_idx3 ON biomaterial USING btree (dbxref_id);


--
-- Name: biomaterial_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_relationship_idx1 ON biomaterial_relationship USING btree (subject_id);


--
-- Name: biomaterial_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_relationship_idx2 ON biomaterial_relationship USING btree (object_id);


--
-- Name: biomaterial_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_relationship_idx3 ON biomaterial_relationship USING btree (type_id);


--
-- Name: biomaterial_treatment_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_treatment_idx1 ON biomaterial_treatment USING btree (biomaterial_id);


--
-- Name: biomaterial_treatment_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_treatment_idx2 ON biomaterial_treatment USING btree (treatment_id);


--
-- Name: biomaterial_treatment_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterial_treatment_idx3 ON biomaterial_treatment USING btree (unittype_id);


--
-- Name: biomaterialprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterialprop_idx1 ON biomaterialprop USING btree (biomaterial_id);


--
-- Name: biomaterialprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX biomaterialprop_idx2 ON biomaterialprop USING btree (type_id);


--
-- Name: contact_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX contact_relationship_idx1 ON contact_relationship USING btree (type_id);


--
-- Name: contact_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX contact_relationship_idx2 ON contact_relationship USING btree (subject_id);


--
-- Name: contact_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX contact_relationship_idx3 ON contact_relationship USING btree (object_id);


--
-- Name: contactprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX contactprop_idx1 ON contactprop USING btree (contactprop_id);


--
-- Name: contactprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX contactprop_idx2 ON contactprop USING btree (type_id);


--
-- Name: control_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX control_idx1 ON control USING btree (type_id);


--
-- Name: control_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX control_idx2 ON control USING btree (assay_id);


--
-- Name: control_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX control_idx3 ON control USING btree (tableinfo_id);


--
-- Name: control_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX control_idx4 ON control USING btree (row_id);


--
-- Name: INDEX cvterm_c1; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON INDEX cvterm_c1 IS 'A name can mean different things in
different contexts; for example "chromosome" in SO and GO. A name
should be unique within an ontology or cv. A name may exist twice in a
cv, in both obsolete and non-obsolete forms - these will be for
different cvterms with different OBO identifiers; so GO documentation
for more details on obsoletion. Note that occasionally multiple
obsolete terms with the same name will exist in the same cv. If this
is a possibility for the ontology under consideration (e.g. GO) then the
ID should be appended to the name to ensure uniqueness.';


--
-- Name: INDEX cvterm_c2; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON INDEX cvterm_c2 IS 'The OBO identifier is globally unique.';


--
-- Name: cvterm_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_dbxref_idx1 ON cvterm_dbxref USING btree (cvterm_id);


--
-- Name: cvterm_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_dbxref_idx2 ON cvterm_dbxref USING btree (dbxref_id);


--
-- Name: cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_idx1 ON cvterm USING btree (cv_id);


--
-- Name: cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_idx2 ON cvterm USING btree (name);


--
-- Name: cvterm_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_idx3 ON cvterm USING btree (dbxref_id);


--
-- Name: cvterm_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_relationship_idx1 ON cvterm_relationship USING btree (type_id);


--
-- Name: cvterm_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_relationship_idx2 ON cvterm_relationship USING btree (subject_id);


--
-- Name: cvterm_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvterm_relationship_idx3 ON cvterm_relationship USING btree (object_id);


--
-- Name: cvtermpath_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvtermpath_idx1 ON cvtermpath USING btree (type_id);


--
-- Name: cvtermpath_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvtermpath_idx2 ON cvtermpath USING btree (subject_id);


--
-- Name: cvtermpath_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvtermpath_idx3 ON cvtermpath USING btree (object_id);


--
-- Name: cvtermpath_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvtermpath_idx4 ON cvtermpath USING btree (cv_id);


--
-- Name: cvtermprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvtermprop_idx1 ON cvtermprop USING btree (cvterm_id);


--
-- Name: cvtermprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvtermprop_idx2 ON cvtermprop USING btree (type_id);


--
-- Name: cvtermsynonym_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX cvtermsynonym_idx1 ON cvtermsynonym USING btree (cvterm_id);


--
-- Name: dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX dbxref_idx1 ON dbxref USING btree (db_id);


--
-- Name: dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX dbxref_idx2 ON dbxref USING btree (accession);


--
-- Name: dbxref_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX dbxref_idx3 ON dbxref USING btree (version);


--
-- Name: dbxrefprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX dbxrefprop_idx1 ON dbxrefprop USING btree (dbxref_id);


--
-- Name: dbxrefprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX dbxrefprop_idx2 ON dbxrefprop USING btree (type_id);


--
-- Name: element_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_idx1 ON element USING btree (feature_id);


--
-- Name: element_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_idx2 ON element USING btree (arraydesign_id);


--
-- Name: element_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_idx3 ON element USING btree (type_id);


--
-- Name: element_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_idx4 ON element USING btree (dbxref_id);


--
-- Name: element_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_relationship_idx1 ON element_relationship USING btree (subject_id);


--
-- Name: element_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_relationship_idx2 ON element_relationship USING btree (type_id);


--
-- Name: element_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_relationship_idx3 ON element_relationship USING btree (object_id);


--
-- Name: element_relationship_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX element_relationship_idx4 ON element_relationship USING btree (value);


--
-- Name: elementresult_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX elementresult_idx1 ON elementresult USING btree (element_id);


--
-- Name: elementresult_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX elementresult_idx2 ON elementresult USING btree (quantification_id);


--
-- Name: elementresult_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX elementresult_idx3 ON elementresult USING btree (signal);


--
-- Name: elementresult_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX elementresult_relationship_idx1 ON elementresult_relationship USING btree (subject_id);


--
-- Name: elementresult_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX elementresult_relationship_idx2 ON elementresult_relationship USING btree (type_id);


--
-- Name: elementresult_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX elementresult_relationship_idx3 ON elementresult_relationship USING btree (object_id);


--
-- Name: elementresult_relationship_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX elementresult_relationship_idx4 ON elementresult_relationship USING btree (value);


--
-- Name: environment_cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX environment_cvterm_idx1 ON environment_cvterm USING btree (environment_id);


--
-- Name: environment_cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX environment_cvterm_idx2 ON environment_cvterm USING btree (cvterm_id);


--
-- Name: environment_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX environment_idx1 ON environment USING btree (uniquename);


--
-- Name: expression_cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_cvterm_idx1 ON expression_cvterm USING btree (expression_id);


--
-- Name: expression_cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_cvterm_idx2 ON expression_cvterm USING btree (cvterm_id);


--
-- Name: expression_cvterm_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_cvterm_idx3 ON expression_cvterm USING btree (cvterm_type_id);


--
-- Name: expression_cvtermprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_cvtermprop_idx1 ON expression_cvtermprop USING btree (expression_cvterm_id);


--
-- Name: expression_cvtermprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_cvtermprop_idx2 ON expression_cvtermprop USING btree (type_id);


--
-- Name: expression_image_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_image_idx1 ON expression_image USING btree (expression_id);


--
-- Name: expression_image_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_image_idx2 ON expression_image USING btree (eimage_id);


--
-- Name: expression_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_pub_idx1 ON expression_pub USING btree (expression_id);


--
-- Name: expression_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expression_pub_idx2 ON expression_pub USING btree (pub_id);


--
-- Name: expressionprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expressionprop_idx1 ON expressionprop USING btree (expression_id);


--
-- Name: expressionprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX expressionprop_idx2 ON expressionprop USING btree (type_id);


--
-- Name: feature_cvterm_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvterm_dbxref_idx1 ON feature_cvterm_dbxref USING btree (feature_cvterm_id);


--
-- Name: feature_cvterm_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvterm_dbxref_idx2 ON feature_cvterm_dbxref USING btree (dbxref_id);


--
-- Name: feature_cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvterm_idx1 ON feature_cvterm USING btree (feature_id);


--
-- Name: feature_cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvterm_idx2 ON feature_cvterm USING btree (cvterm_id);


--
-- Name: feature_cvterm_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvterm_idx3 ON feature_cvterm USING btree (pub_id);


--
-- Name: feature_cvterm_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvterm_pub_idx1 ON feature_cvterm_pub USING btree (feature_cvterm_id);


--
-- Name: feature_cvterm_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvterm_pub_idx2 ON feature_cvterm_pub USING btree (pub_id);


--
-- Name: feature_cvtermprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvtermprop_idx1 ON feature_cvtermprop USING btree (feature_cvterm_id);


--
-- Name: feature_cvtermprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_cvtermprop_idx2 ON feature_cvtermprop USING btree (type_id);


--
-- Name: feature_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_dbxref_idx1 ON feature_dbxref USING btree (feature_id);


--
-- Name: feature_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_dbxref_idx2 ON feature_dbxref USING btree (dbxref_id);


--
-- Name: feature_expression_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_expression_idx1 ON feature_expression USING btree (expression_id);


--
-- Name: feature_expression_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_expression_idx2 ON feature_expression USING btree (feature_id);


--
-- Name: feature_expression_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_expression_idx3 ON feature_expression USING btree (pub_id);


--
-- Name: feature_expressionprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_expressionprop_idx1 ON feature_expressionprop USING btree (feature_expression_id);


--
-- Name: feature_expressionprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_expressionprop_idx2 ON feature_expressionprop USING btree (type_id);


--
-- Name: feature_genotype_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_genotype_idx1 ON feature_genotype USING btree (feature_id);


--
-- Name: feature_genotype_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_genotype_idx2 ON feature_genotype USING btree (genotype_id);


--
-- Name: feature_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_idx1 ON feature USING btree (dbxref_id);


--
-- Name: feature_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_idx2 ON feature USING btree (organism_id);


--
-- Name: feature_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_idx3 ON feature USING btree (type_id);


--
-- Name: feature_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_idx4 ON feature USING btree (uniquename);


--
-- Name: feature_idx5; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_idx5 ON feature USING btree (lower((name)::text));


--
-- Name: feature_name_ind1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_name_ind1 ON feature USING btree (name);


--
-- Name: feature_phenotype_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_phenotype_idx1 ON feature_phenotype USING btree (feature_id);


--
-- Name: feature_phenotype_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_phenotype_idx2 ON feature_phenotype USING btree (phenotype_id);


--
-- Name: feature_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_pub_idx1 ON feature_pub USING btree (feature_id);


--
-- Name: feature_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_pub_idx2 ON feature_pub USING btree (pub_id);


--
-- Name: feature_pubprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_pubprop_idx1 ON feature_pubprop USING btree (feature_pub_id);


--
-- Name: feature_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationship_idx1 ON feature_relationship USING btree (subject_id);


--
-- Name: feature_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationship_idx2 ON feature_relationship USING btree (object_id);


--
-- Name: feature_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationship_idx3 ON feature_relationship USING btree (type_id);


--
-- Name: feature_relationship_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationship_pub_idx1 ON feature_relationship_pub USING btree (feature_relationship_id);


--
-- Name: feature_relationship_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationship_pub_idx2 ON feature_relationship_pub USING btree (pub_id);


--
-- Name: feature_relationshipprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationshipprop_idx1 ON feature_relationshipprop USING btree (feature_relationship_id);


--
-- Name: feature_relationshipprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationshipprop_idx2 ON feature_relationshipprop USING btree (type_id);


--
-- Name: feature_relationshipprop_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationshipprop_pub_idx1 ON feature_relationshipprop_pub USING btree (feature_relationshipprop_id);


--
-- Name: feature_relationshipprop_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_relationshipprop_pub_idx2 ON feature_relationshipprop_pub USING btree (pub_id);


--
-- Name: feature_synonym_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_synonym_idx1 ON feature_synonym USING btree (synonym_id);


--
-- Name: feature_synonym_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_synonym_idx2 ON feature_synonym USING btree (feature_id);


--
-- Name: feature_synonym_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX feature_synonym_idx3 ON feature_synonym USING btree (pub_id);


--
-- Name: featureloc_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureloc_idx1 ON featureloc USING btree (feature_id);


--
-- Name: featureloc_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureloc_idx2 ON featureloc USING btree (srcfeature_id);


--
-- Name: featureloc_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureloc_idx3 ON featureloc USING btree (srcfeature_id, fmin, fmax);


--
-- Name: featureloc_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureloc_pub_idx1 ON featureloc_pub USING btree (featureloc_id);


--
-- Name: featureloc_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureloc_pub_idx2 ON featureloc_pub USING btree (pub_id);


--
-- Name: featuremap_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featuremap_pub_idx1 ON featuremap_pub USING btree (featuremap_id);


--
-- Name: featuremap_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featuremap_pub_idx2 ON featuremap_pub USING btree (pub_id);


--
-- Name: featurepos_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurepos_idx1 ON featurepos USING btree (featuremap_id);


--
-- Name: featurepos_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurepos_idx2 ON featurepos USING btree (feature_id);


--
-- Name: featurepos_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurepos_idx3 ON featurepos USING btree (map_feature_id);


--
-- Name: INDEX featureprop_c1; Type: COMMENT; Schema: public; Owner: gmod
--

COMMENT ON INDEX featureprop_c1 IS 'For any one feature, multivalued
property-value pairs must be differentiated by rank.';


--
-- Name: featureprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureprop_idx1 ON featureprop USING btree (feature_id);


--
-- Name: featureprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureprop_idx2 ON featureprop USING btree (type_id);


--
-- Name: featureprop_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureprop_pub_idx1 ON featureprop_pub USING btree (featureprop_id);


--
-- Name: featureprop_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featureprop_pub_idx2 ON featureprop_pub USING btree (pub_id);


--
-- Name: featurerange_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurerange_idx1 ON featurerange USING btree (featuremap_id);


--
-- Name: featurerange_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurerange_idx2 ON featurerange USING btree (feature_id);


--
-- Name: featurerange_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurerange_idx3 ON featurerange USING btree (leftstartf_id);


--
-- Name: featurerange_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurerange_idx4 ON featurerange USING btree (leftendf_id);


--
-- Name: featurerange_idx5; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurerange_idx5 ON featurerange USING btree (rightstartf_id);


--
-- Name: featurerange_idx6; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX featurerange_idx6 ON featurerange USING btree (rightendf_id);


--
-- Name: genotype_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX genotype_idx1 ON genotype USING btree (uniquename);


--
-- Name: genotype_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX genotype_idx2 ON genotype USING btree (name);


--
-- Name: library_cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_cvterm_idx1 ON library_cvterm USING btree (library_id);


--
-- Name: library_cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_cvterm_idx2 ON library_cvterm USING btree (cvterm_id);


--
-- Name: library_cvterm_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_cvterm_idx3 ON library_cvterm USING btree (pub_id);


--
-- Name: library_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_dbxref_idx1 ON library_dbxref USING btree (library_id);


--
-- Name: library_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_dbxref_idx2 ON library_dbxref USING btree (dbxref_id);


--
-- Name: library_feature_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_feature_idx1 ON library_feature USING btree (library_id);


--
-- Name: library_feature_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_feature_idx2 ON library_feature USING btree (feature_id);


--
-- Name: library_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_idx1 ON library USING btree (organism_id);


--
-- Name: library_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_idx2 ON library USING btree (type_id);


--
-- Name: library_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_idx3 ON library USING btree (uniquename);


--
-- Name: library_name_ind1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_name_ind1 ON library USING btree (name);


--
-- Name: library_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_pub_idx1 ON library_pub USING btree (library_id);


--
-- Name: library_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_pub_idx2 ON library_pub USING btree (pub_id);


--
-- Name: library_synonym_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_synonym_idx1 ON library_synonym USING btree (synonym_id);


--
-- Name: library_synonym_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_synonym_idx2 ON library_synonym USING btree (library_id);


--
-- Name: library_synonym_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX library_synonym_idx3 ON library_synonym USING btree (pub_id);


--
-- Name: libraryprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX libraryprop_idx1 ON libraryprop USING btree (library_id);


--
-- Name: libraryprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX libraryprop_idx2 ON libraryprop USING btree (type_id);


--
-- Name: libraryprop_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX libraryprop_pub_idx1 ON libraryprop_pub USING btree (libraryprop_id);


--
-- Name: libraryprop_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX libraryprop_pub_idx2 ON libraryprop_pub USING btree (pub_id);


--
-- Name: magedocumentation_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX magedocumentation_idx1 ON magedocumentation USING btree (mageml_id);


--
-- Name: magedocumentation_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX magedocumentation_idx2 ON magedocumentation USING btree (tableinfo_id);


--
-- Name: magedocumentation_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX magedocumentation_idx3 ON magedocumentation USING btree (row_id);


--
-- Name: nd_experiment_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX nd_experiment_pub_idx1 ON nd_experiment_pub USING btree (nd_experiment_id);


--
-- Name: nd_experiment_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX nd_experiment_pub_idx2 ON nd_experiment_pub USING btree (pub_id);


--
-- Name: organism_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX organism_dbxref_idx1 ON organism_dbxref USING btree (organism_id);


--
-- Name: organism_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX organism_dbxref_idx2 ON organism_dbxref USING btree (dbxref_id);


--
-- Name: organismprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX organismprop_idx1 ON organismprop USING btree (organism_id);


--
-- Name: organismprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX organismprop_idx2 ON organismprop USING btree (type_id);


--
-- Name: phendesc_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phendesc_idx1 ON phendesc USING btree (genotype_id);


--
-- Name: phendesc_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phendesc_idx2 ON phendesc USING btree (environment_id);


--
-- Name: phendesc_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phendesc_idx3 ON phendesc USING btree (pub_id);


--
-- Name: phenotype_comparison_cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_comparison_cvterm_idx1 ON phenotype_comparison_cvterm USING btree (phenotype_comparison_id);


--
-- Name: phenotype_comparison_cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_comparison_cvterm_idx2 ON phenotype_comparison_cvterm USING btree (cvterm_id);


--
-- Name: phenotype_comparison_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_comparison_idx1 ON phenotype_comparison USING btree (genotype1_id);


--
-- Name: phenotype_comparison_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_comparison_idx2 ON phenotype_comparison USING btree (genotype2_id);


--
-- Name: phenotype_comparison_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_comparison_idx4 ON phenotype_comparison USING btree (pub_id);


--
-- Name: phenotype_cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_cvterm_idx1 ON phenotype_cvterm USING btree (phenotype_id);


--
-- Name: phenotype_cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_cvterm_idx2 ON phenotype_cvterm USING btree (cvterm_id);


--
-- Name: phenotype_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_idx1 ON phenotype USING btree (cvalue_id);


--
-- Name: phenotype_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_idx2 ON phenotype USING btree (observable_id);


--
-- Name: phenotype_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenotype_idx3 ON phenotype USING btree (attr_id);


--
-- Name: phenstatement_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenstatement_idx1 ON phenstatement USING btree (genotype_id);


--
-- Name: phenstatement_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phenstatement_idx2 ON phenstatement USING btree (phenotype_id);


--
-- Name: phylonode_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_dbxref_idx1 ON phylonode_dbxref USING btree (phylonode_id);


--
-- Name: phylonode_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_dbxref_idx2 ON phylonode_dbxref USING btree (dbxref_id);


--
-- Name: phylonode_organism_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_organism_idx1 ON phylonode_organism USING btree (phylonode_id);


--
-- Name: phylonode_organism_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_organism_idx2 ON phylonode_organism USING btree (organism_id);


--
-- Name: phylonode_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_pub_idx1 ON phylonode_pub USING btree (phylonode_id);


--
-- Name: phylonode_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_pub_idx2 ON phylonode_pub USING btree (pub_id);


--
-- Name: phylonode_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_relationship_idx1 ON phylonode_relationship USING btree (subject_id);


--
-- Name: phylonode_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_relationship_idx2 ON phylonode_relationship USING btree (object_id);


--
-- Name: phylonode_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonode_relationship_idx3 ON phylonode_relationship USING btree (type_id);


--
-- Name: phylonodeprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonodeprop_idx1 ON phylonodeprop USING btree (phylonode_id);


--
-- Name: phylonodeprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylonodeprop_idx2 ON phylonodeprop USING btree (type_id);


--
-- Name: phylotree_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylotree_idx1 ON phylotree USING btree (phylotree_id);


--
-- Name: phylotree_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylotree_pub_idx1 ON phylotree_pub USING btree (phylotree_id);


--
-- Name: phylotree_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX phylotree_pub_idx2 ON phylotree_pub USING btree (pub_id);


--
-- Name: project_contact_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX project_contact_idx1 ON project_contact USING btree (project_id);


--
-- Name: project_contact_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX project_contact_idx2 ON project_contact USING btree (contact_id);


--
-- Name: project_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX project_pub_idx1 ON project_pub USING btree (project_id);


--
-- Name: project_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX project_pub_idx2 ON project_pub USING btree (pub_id);


--
-- Name: protocol_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX protocol_idx1 ON protocol USING btree (type_id);


--
-- Name: protocol_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX protocol_idx2 ON protocol USING btree (pub_id);


--
-- Name: protocol_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX protocol_idx3 ON protocol USING btree (dbxref_id);


--
-- Name: protocolparam_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX protocolparam_idx1 ON protocolparam USING btree (protocol_id);


--
-- Name: protocolparam_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX protocolparam_idx2 ON protocolparam USING btree (datatype_id);


--
-- Name: protocolparam_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX protocolparam_idx3 ON protocolparam USING btree (unittype_id);


--
-- Name: pub_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pub_dbxref_idx1 ON pub_dbxref USING btree (pub_id);


--
-- Name: pub_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pub_dbxref_idx2 ON pub_dbxref USING btree (dbxref_id);


--
-- Name: pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pub_idx1 ON pub USING btree (type_id);


--
-- Name: pub_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pub_relationship_idx1 ON pub_relationship USING btree (subject_id);


--
-- Name: pub_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pub_relationship_idx2 ON pub_relationship USING btree (object_id);


--
-- Name: pub_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pub_relationship_idx3 ON pub_relationship USING btree (type_id);


--
-- Name: pubauthor_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pubauthor_idx2 ON pubauthor USING btree (pub_id);


--
-- Name: pubprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pubprop_idx1 ON pubprop USING btree (pub_id);


--
-- Name: pubprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX pubprop_idx2 ON pubprop USING btree (type_id);


--
-- Name: quantification_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantification_idx1 ON quantification USING btree (acquisition_id);


--
-- Name: quantification_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantification_idx2 ON quantification USING btree (operator_id);


--
-- Name: quantification_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantification_idx3 ON quantification USING btree (protocol_id);


--
-- Name: quantification_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantification_idx4 ON quantification USING btree (analysis_id);


--
-- Name: quantification_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantification_relationship_idx1 ON quantification_relationship USING btree (subject_id);


--
-- Name: quantification_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantification_relationship_idx2 ON quantification_relationship USING btree (type_id);


--
-- Name: quantification_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantification_relationship_idx3 ON quantification_relationship USING btree (object_id);


--
-- Name: quantificationprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantificationprop_idx1 ON quantificationprop USING btree (quantification_id);


--
-- Name: quantificationprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX quantificationprop_idx2 ON quantificationprop USING btree (type_id);


--
-- Name: stock_cvterm_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_cvterm_idx1 ON stock_cvterm USING btree (stock_id);


--
-- Name: stock_cvterm_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_cvterm_idx2 ON stock_cvterm USING btree (cvterm_id);


--
-- Name: stock_cvterm_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_cvterm_idx3 ON stock_cvterm USING btree (pub_id);


--
-- Name: stock_dbxref_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_dbxref_idx1 ON stock_dbxref USING btree (stock_id);


--
-- Name: stock_dbxref_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_dbxref_idx2 ON stock_dbxref USING btree (dbxref_id);


--
-- Name: stock_genotype_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_genotype_idx1 ON stock_genotype USING btree (stock_id);


--
-- Name: stock_genotype_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_genotype_idx2 ON stock_genotype USING btree (genotype_id);


--
-- Name: stock_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_idx1 ON stock USING btree (dbxref_id);


--
-- Name: stock_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_idx2 ON stock USING btree (organism_id);


--
-- Name: stock_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_idx3 ON stock USING btree (type_id);


--
-- Name: stock_idx4; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_idx4 ON stock USING btree (uniquename);


--
-- Name: stock_name_ind1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_name_ind1 ON stock USING btree (name);


--
-- Name: stock_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_pub_idx1 ON stock_pub USING btree (stock_id);


--
-- Name: stock_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_pub_idx2 ON stock_pub USING btree (pub_id);


--
-- Name: stock_relationship_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_relationship_idx1 ON stock_relationship USING btree (subject_id);


--
-- Name: stock_relationship_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_relationship_idx2 ON stock_relationship USING btree (object_id);


--
-- Name: stock_relationship_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_relationship_idx3 ON stock_relationship USING btree (type_id);


--
-- Name: stock_relationship_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_relationship_pub_idx1 ON stock_relationship_pub USING btree (stock_relationship_id);


--
-- Name: stock_relationship_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stock_relationship_pub_idx2 ON stock_relationship_pub USING btree (pub_id);


--
-- Name: stockcollection_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollection_idx1 ON stockcollection USING btree (contact_id);


--
-- Name: stockcollection_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollection_idx2 ON stockcollection USING btree (type_id);


--
-- Name: stockcollection_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollection_idx3 ON stockcollection USING btree (uniquename);


--
-- Name: stockcollection_name_ind1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollection_name_ind1 ON stockcollection USING btree (name);


--
-- Name: stockcollection_stock_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollection_stock_idx1 ON stockcollection_stock USING btree (stockcollection_id);


--
-- Name: stockcollection_stock_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollection_stock_idx2 ON stockcollection_stock USING btree (stock_id);


--
-- Name: stockcollectionprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollectionprop_idx1 ON stockcollectionprop USING btree (stockcollection_id);


--
-- Name: stockcollectionprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockcollectionprop_idx2 ON stockcollectionprop USING btree (type_id);


--
-- Name: stockprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockprop_idx1 ON stockprop USING btree (stock_id);


--
-- Name: stockprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockprop_idx2 ON stockprop USING btree (type_id);


--
-- Name: stockprop_pub_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockprop_pub_idx1 ON stockprop_pub USING btree (stockprop_id);


--
-- Name: stockprop_pub_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX stockprop_pub_idx2 ON stockprop_pub USING btree (pub_id);


--
-- Name: study_assay_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX study_assay_idx1 ON study_assay USING btree (study_id);


--
-- Name: study_assay_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX study_assay_idx2 ON study_assay USING btree (assay_id);


--
-- Name: study_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX study_idx1 ON study USING btree (contact_id);


--
-- Name: study_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX study_idx2 ON study USING btree (pub_id);


--
-- Name: study_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX study_idx3 ON study USING btree (dbxref_id);


--
-- Name: studydesign_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studydesign_idx1 ON studydesign USING btree (study_id);


--
-- Name: studydesignprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studydesignprop_idx1 ON studydesignprop USING btree (studydesign_id);


--
-- Name: studydesignprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studydesignprop_idx2 ON studydesignprop USING btree (type_id);


--
-- Name: studyfactor_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyfactor_idx1 ON studyfactor USING btree (studydesign_id);


--
-- Name: studyfactor_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyfactor_idx2 ON studyfactor USING btree (type_id);


--
-- Name: studyfactorvalue_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyfactorvalue_idx1 ON studyfactorvalue USING btree (studyfactor_id);


--
-- Name: studyfactorvalue_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyfactorvalue_idx2 ON studyfactorvalue USING btree (assay_id);


--
-- Name: studyprop_feature_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyprop_feature_idx1 ON studyprop_feature USING btree (studyprop_id);


--
-- Name: studyprop_feature_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyprop_feature_idx2 ON studyprop_feature USING btree (feature_id);


--
-- Name: studyprop_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyprop_idx1 ON studyprop USING btree (study_id);


--
-- Name: studyprop_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX studyprop_idx2 ON studyprop USING btree (type_id);


--
-- Name: synonym_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX synonym_idx1 ON synonym USING btree (type_id);


--
-- Name: synonym_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX synonym_idx2 ON synonym USING btree (lower((synonym_sgml)::text));


--
-- Name: treatment_idx1; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX treatment_idx1 ON treatment USING btree (biomaterial_id);


--
-- Name: treatment_idx2; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX treatment_idx2 ON treatment USING btree (type_id);


--
-- Name: treatment_idx3; Type: INDEX; Schema: public; Owner: gmod; Tablespace: 
--

CREATE INDEX treatment_idx3 ON treatment USING btree (protocol_id);


--
-- Name: acquisition_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT acquisition_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES assay(assay_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: acquisition_channel_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT acquisition_channel_id_fkey FOREIGN KEY (channel_id) REFERENCES channel(channel_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: acquisition_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT acquisition_protocol_id_fkey FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: acquisition_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT acquisition_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES acquisition(acquisition_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: acquisition_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT acquisition_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES acquisition(acquisition_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: acquisition_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT acquisition_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: acquisitionprop_acquisition_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisitionprop
    ADD CONSTRAINT acquisitionprop_acquisition_id_fkey FOREIGN KEY (acquisition_id) REFERENCES acquisition(acquisition_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: acquisitionprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisitionprop
    ADD CONSTRAINT acquisitionprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: analysisfeature_analysis_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeature
    ADD CONSTRAINT analysisfeature_analysis_id_fkey FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: analysisfeature_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeature
    ADD CONSTRAINT analysisfeature_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: analysisfeatureprop_analysisfeature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeatureprop
    ADD CONSTRAINT analysisfeatureprop_analysisfeature_id_fkey FOREIGN KEY (analysisfeature_id) REFERENCES analysisfeature(analysisfeature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: analysisfeatureprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeatureprop
    ADD CONSTRAINT analysisfeatureprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: analysisprop_analysis_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisprop
    ADD CONSTRAINT analysisprop_analysis_id_fkey FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: analysisprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisprop
    ADD CONSTRAINT analysisprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: arraydesign_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT arraydesign_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: arraydesign_manufacturer_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT arraydesign_manufacturer_id_fkey FOREIGN KEY (manufacturer_id) REFERENCES contact(contact_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: arraydesign_platformtype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT arraydesign_platformtype_id_fkey FOREIGN KEY (platformtype_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: arraydesign_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT arraydesign_protocol_id_fkey FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: arraydesign_substratetype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT arraydesign_substratetype_id_fkey FOREIGN KEY (substratetype_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: arraydesignprop_arraydesign_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesignprop
    ADD CONSTRAINT arraydesignprop_arraydesign_id_fkey FOREIGN KEY (arraydesign_id) REFERENCES arraydesign(arraydesign_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: arraydesignprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesignprop
    ADD CONSTRAINT arraydesignprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_arraydesign_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT assay_arraydesign_id_fkey FOREIGN KEY (arraydesign_id) REFERENCES arraydesign(arraydesign_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_biomaterial_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT assay_biomaterial_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES assay(assay_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_biomaterial_biomaterial_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT assay_biomaterial_biomaterial_id_fkey FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_biomaterial_channel_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT assay_biomaterial_channel_id_fkey FOREIGN KEY (channel_id) REFERENCES channel(channel_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT assay_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_operator_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT assay_operator_id_fkey FOREIGN KEY (operator_id) REFERENCES contact(contact_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_project_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_project
    ADD CONSTRAINT assay_project_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES assay(assay_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_project_project_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_project
    ADD CONSTRAINT assay_project_project_id_fkey FOREIGN KEY (project_id) REFERENCES project(project_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assay_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT assay_protocol_id_fkey FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assayprop_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assayprop
    ADD CONSTRAINT assayprop_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES assay(assay_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: assayprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assayprop
    ADD CONSTRAINT assayprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_biosourceprovider_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT biomaterial_biosourceprovider_id_fkey FOREIGN KEY (biosourceprovider_id) REFERENCES contact(contact_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_dbxref_biomaterial_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_dbxref
    ADD CONSTRAINT biomaterial_dbxref_biomaterial_id_fkey FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_dbxref
    ADD CONSTRAINT biomaterial_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT biomaterial_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT biomaterial_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES biomaterial(biomaterial_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT biomaterial_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES biomaterial(biomaterial_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT biomaterial_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_taxon_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT biomaterial_taxon_id_fkey FOREIGN KEY (taxon_id) REFERENCES organism(organism_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_treatment_biomaterial_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT biomaterial_treatment_biomaterial_id_fkey FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_treatment_treatment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT biomaterial_treatment_treatment_id_fkey FOREIGN KEY (treatment_id) REFERENCES treatment(treatment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterial_treatment_unittype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT biomaterial_treatment_unittype_id_fkey FOREIGN KEY (unittype_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterialprop_biomaterial_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterialprop
    ADD CONSTRAINT biomaterialprop_biomaterial_id_fkey FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: biomaterialprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterialprop
    ADD CONSTRAINT biomaterialprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_cvterm_cell_line_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT cell_line_cvterm_cell_line_id_fkey FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT cell_line_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_cvterm_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT cell_line_cvterm_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_cvtermprop_cell_line_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvtermprop
    ADD CONSTRAINT cell_line_cvtermprop_cell_line_cvterm_id_fkey FOREIGN KEY (cell_line_cvterm_id) REFERENCES cell_line_cvterm(cell_line_cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_cvtermprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvtermprop
    ADD CONSTRAINT cell_line_cvtermprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_dbxref_cell_line_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_dbxref
    ADD CONSTRAINT cell_line_dbxref_cell_line_id_fkey FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_dbxref
    ADD CONSTRAINT cell_line_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_feature_cell_line_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT cell_line_feature_cell_line_id_fkey FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_feature_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT cell_line_feature_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_feature_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT cell_line_feature_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_library_cell_line_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT cell_line_library_cell_line_id_fkey FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_library_library_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT cell_line_library_library_id_fkey FOREIGN KEY (library_id) REFERENCES library(library_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_library_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT cell_line_library_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line
    ADD CONSTRAINT cell_line_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_pub_cell_line_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_pub
    ADD CONSTRAINT cell_line_pub_cell_line_id_fkey FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_pub
    ADD CONSTRAINT cell_line_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT cell_line_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT cell_line_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT cell_line_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_synonym_cell_line_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT cell_line_synonym_cell_line_id_fkey FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_synonym_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT cell_line_synonym_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_line_synonym_synonym_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT cell_line_synonym_synonym_id_fkey FOREIGN KEY (synonym_id) REFERENCES synonym(synonym_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_lineprop_cell_line_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop
    ADD CONSTRAINT cell_lineprop_cell_line_id_fkey FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_lineprop_pub_cell_lineprop_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop_pub
    ADD CONSTRAINT cell_lineprop_pub_cell_lineprop_id_fkey FOREIGN KEY (cell_lineprop_id) REFERENCES cell_lineprop(cell_lineprop_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_lineprop_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop_pub
    ADD CONSTRAINT cell_lineprop_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cell_lineprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop
    ADD CONSTRAINT cell_lineprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: contact_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT contact_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES contact(contact_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: contact_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT contact_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES contact(contact_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: contact_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT contact_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: contact_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact
    ADD CONSTRAINT contact_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: contactprop_contact_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contactprop
    ADD CONSTRAINT contactprop_contact_id_fkey FOREIGN KEY (contact_id) REFERENCES contact(contact_id) ON DELETE CASCADE;


--
-- Name: contactprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contactprop
    ADD CONSTRAINT contactprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: control_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY control
    ADD CONSTRAINT control_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES assay(assay_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: control_tableinfo_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY control
    ADD CONSTRAINT control_tableinfo_id_fkey FOREIGN KEY (tableinfo_id) REFERENCES tableinfo(tableinfo_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: control_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY control
    ADD CONSTRAINT control_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvterm_cv_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm
    ADD CONSTRAINT cvterm_cv_id_fkey FOREIGN KEY (cv_id) REFERENCES cv(cv_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvterm_dbxref_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_dbxref
    ADD CONSTRAINT cvterm_dbxref_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvterm_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_dbxref
    ADD CONSTRAINT cvterm_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvterm_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm
    ADD CONSTRAINT cvterm_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvterm_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT cvterm_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvterm_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT cvterm_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvterm_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT cvterm_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvtermpath_cv_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT cvtermpath_cv_id_fkey FOREIGN KEY (cv_id) REFERENCES cv(cv_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvtermpath_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT cvtermpath_object_id_fkey FOREIGN KEY (object_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvtermpath_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT cvtermpath_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvtermpath_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT cvtermpath_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvtermprop_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermprop
    ADD CONSTRAINT cvtermprop_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: cvtermprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermprop
    ADD CONSTRAINT cvtermprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: cvtermsynonym_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermsynonym
    ADD CONSTRAINT cvtermsynonym_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: cvtermsynonym_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermsynonym
    ADD CONSTRAINT cvtermsynonym_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: dbxref_db_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY dbxref
    ADD CONSTRAINT dbxref_db_id_fkey FOREIGN KEY (db_id) REFERENCES db(db_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: dbxrefprop_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY dbxrefprop
    ADD CONSTRAINT dbxrefprop_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: dbxrefprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY dbxrefprop
    ADD CONSTRAINT dbxrefprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: element_arraydesign_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT element_arraydesign_id_fkey FOREIGN KEY (arraydesign_id) REFERENCES arraydesign(arraydesign_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: element_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT element_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: element_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT element_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: element_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT element_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES element(element_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: element_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT element_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES element(element_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: element_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT element_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: element_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT element_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: elementresult_element_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult
    ADD CONSTRAINT elementresult_element_id_fkey FOREIGN KEY (element_id) REFERENCES element(element_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: elementresult_quantification_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult
    ADD CONSTRAINT elementresult_quantification_id_fkey FOREIGN KEY (quantification_id) REFERENCES quantification(quantification_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: elementresult_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT elementresult_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES elementresult(elementresult_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: elementresult_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT elementresult_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES elementresult(elementresult_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: elementresult_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT elementresult_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) DEFERRABLE INITIALLY DEFERRED;


--
-- Name: environment_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY environment_cvterm
    ADD CONSTRAINT environment_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: environment_cvterm_environment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY environment_cvterm
    ADD CONSTRAINT environment_cvterm_environment_id_fkey FOREIGN KEY (environment_id) REFERENCES environment(environment_id) ON DELETE CASCADE;


--
-- Name: expression_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT expression_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_cvterm_cvterm_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT expression_cvterm_cvterm_type_id_fkey FOREIGN KEY (cvterm_type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_cvterm_expression_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT expression_cvterm_expression_id_fkey FOREIGN KEY (expression_id) REFERENCES expression(expression_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_cvtermprop_expression_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvtermprop
    ADD CONSTRAINT expression_cvtermprop_expression_cvterm_id_fkey FOREIGN KEY (expression_cvterm_id) REFERENCES expression_cvterm(expression_cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_cvtermprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvtermprop
    ADD CONSTRAINT expression_cvtermprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_image_eimage_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_image
    ADD CONSTRAINT expression_image_eimage_id_fkey FOREIGN KEY (eimage_id) REFERENCES eimage(eimage_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_image_expression_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_image
    ADD CONSTRAINT expression_image_expression_id_fkey FOREIGN KEY (expression_id) REFERENCES expression(expression_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_pub_expression_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_pub
    ADD CONSTRAINT expression_pub_expression_id_fkey FOREIGN KEY (expression_id) REFERENCES expression(expression_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expression_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_pub
    ADD CONSTRAINT expression_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expressionprop_expression_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expressionprop
    ADD CONSTRAINT expressionprop_expression_id_fkey FOREIGN KEY (expression_id) REFERENCES expression(expression_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: expressionprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expressionprop
    ADD CONSTRAINT expressionprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT feature_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_cvterm_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_dbxref
    ADD CONSTRAINT feature_cvterm_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_cvterm_dbxref_feature_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_dbxref
    ADD CONSTRAINT feature_cvterm_dbxref_feature_cvterm_id_fkey FOREIGN KEY (feature_cvterm_id) REFERENCES feature_cvterm(feature_cvterm_id) ON DELETE CASCADE;


--
-- Name: feature_cvterm_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT feature_cvterm_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_cvterm_pub_feature_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_pub
    ADD CONSTRAINT feature_cvterm_pub_feature_cvterm_id_fkey FOREIGN KEY (feature_cvterm_id) REFERENCES feature_cvterm(feature_cvterm_id) ON DELETE CASCADE;


--
-- Name: feature_cvterm_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT feature_cvterm_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_cvterm_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_pub
    ADD CONSTRAINT feature_cvterm_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_cvtermprop_feature_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvtermprop
    ADD CONSTRAINT feature_cvtermprop_feature_cvterm_id_fkey FOREIGN KEY (feature_cvterm_id) REFERENCES feature_cvterm(feature_cvterm_id) ON DELETE CASCADE;


--
-- Name: feature_cvtermprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvtermprop
    ADD CONSTRAINT feature_cvtermprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_dbxref
    ADD CONSTRAINT feature_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_dbxref_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_dbxref
    ADD CONSTRAINT feature_dbxref_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT feature_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_expression_expression_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT feature_expression_expression_id_fkey FOREIGN KEY (expression_id) REFERENCES expression(expression_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_expression_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT feature_expression_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_expression_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT feature_expression_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_expressionprop_feature_expression_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expressionprop
    ADD CONSTRAINT feature_expressionprop_feature_expression_id_fkey FOREIGN KEY (feature_expression_id) REFERENCES feature_expression(feature_expression_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_expressionprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expressionprop
    ADD CONSTRAINT feature_expressionprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_genotype_chromosome_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT feature_genotype_chromosome_id_fkey FOREIGN KEY (chromosome_id) REFERENCES feature(feature_id) ON DELETE SET NULL;


--
-- Name: feature_genotype_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT feature_genotype_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: feature_genotype_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT feature_genotype_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE;


--
-- Name: feature_genotype_genotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT feature_genotype_genotype_id_fkey FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id) ON DELETE CASCADE;


--
-- Name: feature_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT feature_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_phenotype_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_phenotype
    ADD CONSTRAINT feature_phenotype_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE;


--
-- Name: feature_phenotype_phenotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_phenotype
    ADD CONSTRAINT feature_phenotype_phenotype_id_fkey FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id) ON DELETE CASCADE;


--
-- Name: feature_pub_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pub
    ADD CONSTRAINT feature_pub_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pub
    ADD CONSTRAINT feature_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_pubprop_feature_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pubprop
    ADD CONSTRAINT feature_pubprop_feature_pub_id_fkey FOREIGN KEY (feature_pub_id) REFERENCES feature_pub(feature_pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_pubprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pubprop
    ADD CONSTRAINT feature_pubprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT feature_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationship_pub_feature_relationship_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship_pub
    ADD CONSTRAINT feature_relationship_pub_feature_relationship_id_fkey FOREIGN KEY (feature_relationship_id) REFERENCES feature_relationship(feature_relationship_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationship_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship_pub
    ADD CONSTRAINT feature_relationship_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT feature_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT feature_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationshipprop_feature_relationship_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop
    ADD CONSTRAINT feature_relationshipprop_feature_relationship_id_fkey FOREIGN KEY (feature_relationship_id) REFERENCES feature_relationship(feature_relationship_id) ON DELETE CASCADE;


--
-- Name: feature_relationshipprop_pub_feature_relationshipprop_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop_pub
    ADD CONSTRAINT feature_relationshipprop_pub_feature_relationshipprop_id_fkey FOREIGN KEY (feature_relationshipprop_id) REFERENCES feature_relationshipprop(feature_relationshipprop_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationshipprop_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop_pub
    ADD CONSTRAINT feature_relationshipprop_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_relationshipprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop
    ADD CONSTRAINT feature_relationshipprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_synonym_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT feature_synonym_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_synonym_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT feature_synonym_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_synonym_synonym_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT feature_synonym_synonym_id_fkey FOREIGN KEY (synonym_id) REFERENCES synonym(synonym_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: feature_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT feature_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureloc_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc
    ADD CONSTRAINT featureloc_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureloc_pub_featureloc_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc_pub
    ADD CONSTRAINT featureloc_pub_featureloc_id_fkey FOREIGN KEY (featureloc_id) REFERENCES featureloc(featureloc_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureloc_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc_pub
    ADD CONSTRAINT featureloc_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureloc_srcfeature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc
    ADD CONSTRAINT featureloc_srcfeature_id_fkey FOREIGN KEY (srcfeature_id) REFERENCES feature(feature_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featuremap_pub_featuremap_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featuremap_pub
    ADD CONSTRAINT featuremap_pub_featuremap_id_fkey FOREIGN KEY (featuremap_id) REFERENCES featuremap(featuremap_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featuremap_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featuremap_pub
    ADD CONSTRAINT featuremap_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featuremap_unittype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featuremap
    ADD CONSTRAINT featuremap_unittype_id_fkey FOREIGN KEY (unittype_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurepos_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurepos
    ADD CONSTRAINT featurepos_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurepos_featuremap_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurepos
    ADD CONSTRAINT featurepos_featuremap_id_fkey FOREIGN KEY (featuremap_id) REFERENCES featuremap(featuremap_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurepos_map_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurepos
    ADD CONSTRAINT featurepos_map_feature_id_fkey FOREIGN KEY (map_feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureprop_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop
    ADD CONSTRAINT featureprop_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureprop_pub_featureprop_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop_pub
    ADD CONSTRAINT featureprop_pub_featureprop_id_fkey FOREIGN KEY (featureprop_id) REFERENCES featureprop(featureprop_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureprop_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop_pub
    ADD CONSTRAINT featureprop_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featureprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop
    ADD CONSTRAINT featureprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurerange_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT featurerange_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurerange_featuremap_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT featurerange_featuremap_id_fkey FOREIGN KEY (featuremap_id) REFERENCES featuremap(featuremap_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurerange_leftendf_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT featurerange_leftendf_id_fkey FOREIGN KEY (leftendf_id) REFERENCES feature(feature_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurerange_leftstartf_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT featurerange_leftstartf_id_fkey FOREIGN KEY (leftstartf_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurerange_rightendf_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT featurerange_rightendf_id_fkey FOREIGN KEY (rightendf_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: featurerange_rightstartf_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT featurerange_rightstartf_id_fkey FOREIGN KEY (rightstartf_id) REFERENCES feature(feature_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: fk10615e25772f4ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_genotype
    ADD CONSTRAINT fk10615e25772f4ea FOREIGN KEY (stock_id) REFERENCES stock(stock_id);


--
-- Name: fk10615e299afce2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_genotype
    ADD CONSTRAINT fk10615e299afce2a FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id);


--
-- Name: fk10f2db073128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_contact
    ADD CONSTRAINT fk10f2db073128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fk10f2db07d03f64aa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_contact
    ADD CONSTRAINT fk10f2db07d03f64aa FOREIGN KEY (contact_id) REFERENCES contact(contact_id);


--
-- Name: fk14f8989a482b2b4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeature
    ADD CONSTRAINT fk14f8989a482b2b4a FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id);


--
-- Name: fk14f8989a4bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeature
    ADD CONSTRAINT fk14f8989a4bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fk17ba7caaf2c8574a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studydesignprop
    ADD CONSTRAINT fk17ba7caaf2c8574a FOREIGN KEY (studydesign_id) REFERENCES studydesign(studydesign_id);


--
-- Name: fk17ba7caafd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studydesignprop
    ADD CONSTRAINT fk17ba7caafd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk182e259ea6385b0a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT fk182e259ea6385b0a FOREIGN KEY (object_project_id) REFERENCES project(project_id);


--
-- Name: fk182e259ed7bc22fd; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT fk182e259ed7bc22fd FOREIGN KEY (subject_project_id) REFERENCES project(project_id);


--
-- Name: fk182e259efd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT fk182e259efd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk18f0bc1943cae1aa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactorvalue
    ADD CONSTRAINT fk18f0bc1943cae1aa FOREIGN KEY (studyfactor_id) REFERENCES studyfactor(studyfactor_id);


--
-- Name: fk18f0bc19da77160a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactorvalue
    ADD CONSTRAINT fk18f0bc19da77160a FOREIGN KEY (assay_id) REFERENCES assay(assay_id);


--
-- Name: fk1b2fdfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub
    ADD CONSTRAINT fk1b2fdfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk1c62655b73e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock_dbxref
    ADD CONSTRAINT fk1c62655b73e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk1c62655bcca2450; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock_dbxref
    ADD CONSTRAINT fk1c62655bcca2450 FOREIGN KEY (nd_experiment_stock_id) REFERENCES nd_experiment_stock(nd_experiment_stock_id);


--
-- Name: fk1ff36d3a94ecd4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesignprop
    ADD CONSTRAINT fk1ff36d3a94ecd4a FOREIGN KEY (arraydesign_id) REFERENCES arraydesign(arraydesign_id);


--
-- Name: fk1ff36d3afd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesignprop
    ADD CONSTRAINT fk1ff36d3afd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk2026169ebb403f24; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT fk2026169ebb403f24 FOREIGN KEY (object_id) REFERENCES elementresult(elementresult_id);


--
-- Name: fk2026169ef1d8aa17; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT fk2026169ef1d8aa17 FOREIGN KEY (subject_id) REFERENCES elementresult(elementresult_id);


--
-- Name: fk2026169efd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult_relationship
    ADD CONSTRAINT fk2026169efd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk2121917c73e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY dbxrefprop
    ADD CONSTRAINT fk2121917c73e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk2121917cfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY dbxrefprop
    ADD CONSTRAINT fk2121917cfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk21d764e132a35ef4; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT fk21d764e132a35ef4 FOREIGN KEY (subject_id) REFERENCES feature(feature_id);


--
-- Name: fk21d764e1fc0af401; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT fk21d764e1fc0af401 FOREIGN KEY (object_id) REFERENCES feature(feature_id);


--
-- Name: fk21d764e1fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship
    ADD CONSTRAINT fk21d764e1fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk24a45b06c78cb5a1; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT fk24a45b06c78cb5a1 FOREIGN KEY (object_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fk24a45b06fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT fk24a45b06fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk24a45b06fe252094; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_relationship
    ADD CONSTRAINT fk24a45b06fe252094 FOREIGN KEY (subject_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fk27cb783d03f64aa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contactprop
    ADD CONSTRAINT fk27cb783d03f64aa FOREIGN KEY (contact_id) REFERENCES contact(contact_id);


--
-- Name: fk27cb783fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contactprop
    ADD CONSTRAINT fk27cb783fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk2b16e8f539ec1ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocolparam
    ADD CONSTRAINT fk2b16e8f539ec1ca FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id);


--
-- Name: fk2b16e8f541c737c5; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocolparam
    ADD CONSTRAINT fk2b16e8f541c737c5 FOREIGN KEY (datatype_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk2b16e8f5e37764ab; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocolparam
    ADD CONSTRAINT fk2b16e8f5e37764ab FOREIGN KEY (unittype_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk2b241002ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop_pub
    ADD CONSTRAINT fk2b241002ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk2b241002f22d317; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop_pub
    ADD CONSTRAINT fk2b241002f22d317 FOREIGN KEY (feature_relationshipprop_id) REFERENCES feature_relationshipprop(feature_relationshipprop_id);


--
-- Name: fk2dd1254b5772f4ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection_stock
    ADD CONSTRAINT fk2dd1254b5772f4ea FOREIGN KEY (stock_id) REFERENCES stock(stock_id);


--
-- Name: fk2dd1254bd7cdcc2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection_stock
    ADD CONSTRAINT fk2dd1254bd7cdcc2a FOREIGN KEY (stockcollection_id) REFERENCES stockcollection(stockcollection_id);


--
-- Name: fk311eadf073e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_dbxref
    ADD CONSTRAINT fk311eadf073e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk311eadf0e0e8c4d7; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_dbxref
    ADD CONSTRAINT fk311eadf0e0e8c4d7 FOREIGN KEY (feature_cvterm_id) REFERENCES feature_cvterm(feature_cvterm_id);


--
-- Name: fk33de565ea999c6ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantificationprop
    ADD CONSTRAINT fk33de565ea999c6ea FOREIGN KEY (quantification_id) REFERENCES quantification(quantification_id);


--
-- Name: fk33de565efd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantificationprop
    ADD CONSTRAINT fk33de565efd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk3433c7791389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_dbxref
    ADD CONSTRAINT fk3433c7791389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk3433c77973e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_dbxref
    ADD CONSTRAINT fk3433c77973e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk360c49c88760208a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc_pub
    ADD CONSTRAINT fk360c49c88760208a FOREIGN KEY (featureloc_id) REFERENCES featureloc(featureloc_id);


--
-- Name: fk360c49c8ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc_pub
    ADD CONSTRAINT fk360c49c8ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk378dbf0b1389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY environment_cvterm
    ADD CONSTRAINT fk378dbf0b1389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk378dbf0b6429c3ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY environment_cvterm
    ADD CONSTRAINT fk378dbf0b6429c3ca FOREIGN KEY (environment_id) REFERENCES environment(environment_id);


--
-- Name: fk38b72420fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact
    ADD CONSTRAINT fk38b72420fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk38b7655d7bc51b2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY control
    ADD CONSTRAINT fk38b7655d7bc51b2a FOREIGN KEY (tableinfo_id) REFERENCES tableinfo(tableinfo_id);


--
-- Name: fk38b7655dda77160a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY control
    ADD CONSTRAINT fk38b7655dda77160a FOREIGN KEY (assay_id) REFERENCES assay(assay_id);


--
-- Name: fk38b7655dfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY control
    ADD CONSTRAINT fk38b7655dfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk38fbdf561389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_cvterm
    ADD CONSTRAINT fk38fbdf561389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk38fbdf565678baa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_cvterm
    ADD CONSTRAINT fk38fbdf565678baa FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id);


--
-- Name: fk3a8afd32d32c9810; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotypeprop
    ADD CONSTRAINT fk3a8afd32d32c9810 FOREIGN KEY (nd_experiment_phenotype_id) REFERENCES nd_experiment_phenotype(nd_experiment_phenotype_id);


--
-- Name: fk3a8afd32fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotypeprop
    ADD CONSTRAINT fk3a8afd32fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk3afa6b3575dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organismprop
    ADD CONSTRAINT fk3afa6b3575dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fk3afa6b35fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organismprop
    ADD CONSTRAINT fk3afa6b35fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk3cbb8a0cca2450; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stockprop
    ADD CONSTRAINT fk3cbb8a0cca2450 FOREIGN KEY (nd_experiment_stock_id) REFERENCES nd_experiment_stock(nd_experiment_stock_id);


--
-- Name: fk3cbb8a0fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stockprop
    ADD CONSTRAINT fk3cbb8a0fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk3e7b7b90151272; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvtermprop
    ADD CONSTRAINT fk3e7b7b90151272 FOREIGN KEY (cell_line_cvterm_id) REFERENCES cell_line_cvterm(cell_line_cvterm_id);


--
-- Name: fk3e7b7b90fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvtermprop
    ADD CONSTRAINT fk3e7b7b90fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk3f0407b24bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT fk3f0407b24bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fk3f0407b24dd792ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT fk3f0407b24dd792ca FOREIGN KEY (synonym_id) REFERENCES synonym(synonym_id);


--
-- Name: fk3f0407b2ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_synonym
    ADD CONSTRAINT fk3f0407b2ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk3f9b5efdcc746a6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeatureprop
    ADD CONSTRAINT fk3f9b5efdcc746a6a FOREIGN KEY (analysisfeature_id) REFERENCES analysisfeature(analysisfeature_id);


--
-- Name: fk3f9b5efdfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisfeatureprop
    ADD CONSTRAINT fk3f9b5efdfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk41ca5a634bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT fk41ca5a634bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fk41ca5a6399910d2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT fk41ca5a6399910d2a FOREIGN KEY (studyprop_id) REFERENCES studyprop(studyprop_id);


--
-- Name: fk41ca5a63fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT fk41ca5a63fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk42c3f72d76e5daca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT fk42c3f72d76e5daca FOREIGN KEY (library_id) REFERENCES library(library_id);


--
-- Name: fk42c3f72db391eef; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT fk42c3f72db391eef FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fk42c3f72dec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_library
    ADD CONSTRAINT fk42c3f72dec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk43519cc199206c21; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT fk43519cc199206c21 FOREIGN KEY (object_id) REFERENCES stock(stock_id);


--
-- Name: fk43519cc1cfb8d714; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT fk43519cc1cfb8d714 FOREIGN KEY (subject_id) REFERENCES stock(stock_id);


--
-- Name: fk43519cc1fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT fk43519cc1fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk4429e2c8ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pubauthor
    ADD CONSTRAINT fk4429e2c8ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk47634d5739ec1ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT fk47634d5739ec1ca FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id);


--
-- Name: fk47634d5773e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT fk47634d5773e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk47634d5783ddf68e; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT fk47634d5783ddf68e FOREIGN KEY (substratetype_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk47634d57e0d47b59; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT fk47634d57e0d47b59 FOREIGN KEY (manufacturer_id) REFERENCES contact(contact_id);


--
-- Name: fk47634d57f34dfadc; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY arraydesign
    ADD CONSTRAINT fk47634d57f34dfadc FOREIGN KEY (platformtype_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk47e5a95cda77160a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assayprop
    ADD CONSTRAINT fk47e5a95cda77160a FOREIGN KEY (assay_id) REFERENCES assay(assay_id);


--
-- Name: fk47e5a95cfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assayprop
    ADD CONSTRAINT fk47e5a95cfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk491628074bd16eee; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT fk491628074bd16eee FOREIGN KEY (leftendf_id) REFERENCES feature(feature_id);


--
-- Name: fk491628074bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT fk491628074bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fk4916280782b56cb5; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT fk4916280782b56cb5 FOREIGN KEY (leftstartf_id) REFERENCES feature(feature_id);


--
-- Name: fk49162807851cd240; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT fk49162807851cd240 FOREIGN KEY (rightstartf_id) REFERENCES feature(feature_id);


--
-- Name: fk4916280788559b0a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT fk4916280788559b0a FOREIGN KEY (featuremap_id) REFERENCES featuremap(featuremap_id);


--
-- Name: fk49162807b5abb39; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurerange
    ADD CONSTRAINT fk49162807b5abb39 FOREIGN KEY (rightendf_id) REFERENCES feature(feature_id);


--
-- Name: fk4b9a5e545772f4ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_pub
    ADD CONSTRAINT fk4b9a5e545772f4ea FOREIGN KEY (stock_id) REFERENCES stock(stock_id);


--
-- Name: fk4b9a5e54ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_pub
    ADD CONSTRAINT fk4b9a5e54ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk4ba21f795772f4ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop
    ADD CONSTRAINT fk4ba21f795772f4ea FOREIGN KEY (stock_id) REFERENCES stock(stock_id);


--
-- Name: fk4ba21f79fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop
    ADD CONSTRAINT fk4ba21f79fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk4bd68f8d39ec1ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT fk4bd68f8d39ec1ca FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id);


--
-- Name: fk4bd68f8d482795ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT fk4bd68f8d482795ca FOREIGN KEY (channel_id) REFERENCES channel(channel_id);


--
-- Name: fk4bd68f8dda77160a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition
    ADD CONSTRAINT fk4bd68f8dda77160a FOREIGN KEY (assay_id) REFERENCES assay(assay_id);


--
-- Name: fk4bffab97d7cdcc2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollectionprop
    ADD CONSTRAINT fk4bffab97d7cdcc2a FOREIGN KEY (stockcollection_id) REFERENCES stockcollection(stockcollection_id);


--
-- Name: fk4bffab97fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollectionprop
    ADD CONSTRAINT fk4bffab97fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk50b04f8b5678baa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotypeprop
    ADD CONSTRAINT fk50b04f8b5678baa FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id);


--
-- Name: fk50b04f8bfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotypeprop
    ADD CONSTRAINT fk50b04f8bfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk51cf3318d758ec8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT fk51cf3318d758ec8a FOREIGN KEY (object_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk51cf3318df1577d; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT fk51cf3318df1577d FOREIGN KEY (subject_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk51cf3318fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm_relationship
    ADD CONSTRAINT fk51cf3318fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk51e712641989c7b7; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expressionprop
    ADD CONSTRAINT fk51e712641989c7b7 FOREIGN KEY (feature_expression_id) REFERENCES feature_expression(feature_expression_id);


--
-- Name: fk51e71264fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expressionprop
    ADD CONSTRAINT fk51e71264fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk53c5dde488559b0a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featuremap_pub
    ADD CONSTRAINT fk53c5dde488559b0a FOREIGN KEY (featuremap_id) REFERENCES featuremap(featuremap_id);


--
-- Name: fk53c5dde4ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featuremap_pub
    ADD CONSTRAINT fk53c5dde4ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk544c55dc1b30f3ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop_pub
    ADD CONSTRAINT fk544c55dc1b30f3ea FOREIGN KEY (libraryprop_id) REFERENCES libraryprop(libraryprop_id);


--
-- Name: fk544c55dcec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop_pub
    ADD CONSTRAINT fk544c55dcec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk5452ca487966db3a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT fk5452ca487966db3a FOREIGN KEY (object_id) REFERENCES biomaterial(biomaterial_id);


--
-- Name: fk5452ca48afff462d; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT fk5452ca48afff462d FOREIGN KEY (subject_id) REFERENCES biomaterial(biomaterial_id);


--
-- Name: fk5452ca48fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_relationship
    ADD CONSTRAINT fk5452ca48fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk5503aee61389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT fk5503aee61389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk5503aee69ceebdca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT fk5503aee69ceebdca FOREIGN KEY (expression_id) REFERENCES expression(expression_id);


--
-- Name: fk5503aee6f793f1cf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvterm
    ADD CONSTRAINT fk5503aee6f793f1cf FOREIGN KEY (cvterm_type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk57eea470ee88198a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisitionprop
    ADD CONSTRAINT fk57eea470ee88198a FOREIGN KEY (acquisition_id) REFERENCES acquisition(acquisition_id);


--
-- Name: fk57eea470fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisitionprop
    ADD CONSTRAINT fk57eea470fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk58cea7939ec1ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT fk58cea7939ec1ca FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id);


--
-- Name: fk58cea7973e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT fk58cea7973e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk58cea7994ecd4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT fk58cea7994ecd4a FOREIGN KEY (arraydesign_id) REFERENCES arraydesign(arraydesign_id);


--
-- Name: fk58cea79cff9e1a6; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay
    ADD CONSTRAINT fk58cea79cff9e1a6 FOREIGN KEY (operator_id) REFERENCES contact(contact_id);


--
-- Name: fk5956b5e08bdde8fd; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_geolocationprop
    ADD CONSTRAINT fk5956b5e08bdde8fd FOREIGN KEY (nd_geolocation_id) REFERENCES nd_geolocation(nd_geolocation_id);


--
-- Name: fk5956b5e0fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_geolocationprop
    ADD CONSTRAINT fk5956b5e0fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk5ce1a20cd52652fd; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagentprop
    ADD CONSTRAINT fk5ce1a20cd52652fd FOREIGN KEY (nd_reagent_id) REFERENCES nd_reagent(nd_reagent_id);


--
-- Name: fk5ce1a20cfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagentprop
    ADD CONSTRAINT fk5ce1a20cfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk5df7ff631389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT fk5df7ff631389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk5df7ff6376e5daca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT fk5df7ff6376e5daca FOREIGN KEY (library_id) REFERENCES library(library_id);


--
-- Name: fk5df7ff63ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT fk5df7ff63ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk5e95029d73e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_dbxref
    ADD CONSTRAINT fk5e95029d73e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk5e95029d76e5daca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_dbxref
    ADD CONSTRAINT fk5e95029d76e5daca FOREIGN KEY (library_id) REFERENCES library(library_id);


--
-- Name: fk61276cb175dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line
    ADD CONSTRAINT fk61276cb175dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fk632ff9e466492497; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocolprop
    ADD CONSTRAINT fk632ff9e466492497 FOREIGN KEY (nd_protocol_id) REFERENCES nd_protocol(nd_protocol_id);


--
-- Name: fk632ff9e4fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocolprop
    ADD CONSTRAINT fk632ff9e4fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk64453bbd3128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock
    ADD CONSTRAINT fk64453bbd3128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fk64453bbd5772f4ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock
    ADD CONSTRAINT fk64453bbd5772f4ea FOREIGN KEY (stock_id) REFERENCES stock(stock_id);


--
-- Name: fk64453bbdfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock
    ADD CONSTRAINT fk64453bbdfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk66509d375d63fe0a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop_pub
    ADD CONSTRAINT fk66509d375d63fe0a FOREIGN KEY (stockprop_id) REFERENCES stockprop(stockprop_id);


--
-- Name: fk66509d37ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop_pub
    ADD CONSTRAINT fk66509d37ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk676c63273e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_dbxref
    ADD CONSTRAINT fk676c63273e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk676c632c20c9eea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_dbxref
    ADD CONSTRAINT fk676c632c20c9eea FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id);


--
-- Name: fk68af71673e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT fk68af71673e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk68af71675dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT fk68af71675dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fk68af716fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT fk68af716fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk68b0dc973e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study
    ADD CONSTRAINT fk68b0dc973e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk68b0dc9d03f64aa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study
    ADD CONSTRAINT fk68b0dc9d03f64aa FOREIGN KEY (contact_id) REFERENCES contact(contact_id);


--
-- Name: fk68b0dc9ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study
    ADD CONSTRAINT fk68b0dc9ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk68be949d1389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_cvterm
    ADD CONSTRAINT fk68be949d1389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk68be949dec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_cvterm
    ADD CONSTRAINT fk68be949dec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk6a71994f3128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotype
    ADD CONSTRAINT fk6a71994f3128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fk6a71994f5678baa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotype
    ADD CONSTRAINT fk6a71994f5678baa FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id);


--
-- Name: fk711747d2e4b9d8f; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop_pub
    ADD CONSTRAINT fk711747d2e4b9d8f FOREIGN KEY (cell_lineprop_id) REFERENCES cell_lineprop(cell_lineprop_id);


--
-- Name: fk711747d2ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop_pub
    ADD CONSTRAINT fk711747d2ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk717c8e6fb391eef; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_pub
    ADD CONSTRAINT fk717c8e6fb391eef FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fk717c8e6fec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_pub
    ADD CONSTRAINT fk717c8e6fec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk71844f94b391eef; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop
    ADD CONSTRAINT fk71844f94b391eef FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fk71844f94fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_lineprop
    ADD CONSTRAINT fk71844f94fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk72f3fa9b9e402fa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT fk72f3fa9b9e402fa FOREIGN KEY (subject_id) REFERENCES element(element_id);


--
-- Name: fk72f3fa9bd34b9807; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT fk72f3fa9bd34b9807 FOREIGN KEY (object_id) REFERENCES element(element_id);


--
-- Name: fk72f3fa9bfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element_relationship
    ADD CONSTRAINT fk72f3fa9bfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk72f640c81389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT fk72f640c81389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk72f640c84bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT fk72f640c84bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fk72f640c8ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm
    ADD CONSTRAINT fk72f640c8ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk72fec6887bc51b2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY magedocumentation
    ADD CONSTRAINT fk72fec6887bc51b2a FOREIGN KEY (tableinfo_id) REFERENCES tableinfo(tableinfo_id);


--
-- Name: fk72fec688b8a8172a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY magedocumentation
    ADD CONSTRAINT fk72fec688b8a8172a FOREIGN KEY (mageml_id) REFERENCES mageml(mageml_id);


--
-- Name: fk739344024bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_dbxref
    ADD CONSTRAINT fk739344024bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fk7393440273e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_dbxref
    ADD CONSTRAINT fk7393440273e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk75341e04c20c9eea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_pub
    ADD CONSTRAINT fk75341e04c20c9eea FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id);


--
-- Name: fk75341e04ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_pub
    ADD CONSTRAINT fk75341e04ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk753bdf29c20c9eea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonodeprop
    ADD CONSTRAINT fk753bdf29c20c9eea FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id);


--
-- Name: fk753bdf29fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonodeprop
    ADD CONSTRAINT fk753bdf29fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk77abf6ad1389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT fk77abf6ad1389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk77abf6adb391eef; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT fk77abf6adb391eef FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fk77abf6adec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_cvterm
    ADD CONSTRAINT fk77abf6adec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk7848f9e773e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_dbxref
    ADD CONSTRAINT fk7848f9e773e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk7848f9e7b391eef; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_dbxref
    ADD CONSTRAINT fk7848f9e7b391eef FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fk7aab23bc1389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermsynonym
    ADD CONSTRAINT fk7aab23bc1389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk7aab23bcfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermsynonym
    ADD CONSTRAINT fk7aab23bcfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk84e2b5a04a1696a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree_pub
    ADD CONSTRAINT fk84e2b5a04a1696a FOREIGN KEY (phylotree_id) REFERENCES phylotree(phylotree_id);


--
-- Name: fk84e2b5a0ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree_pub
    ADD CONSTRAINT fk84e2b5a0ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk898c7c0461bfbbea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT fk898c7c0461bfbbea FOREIGN KEY (cv_id) REFERENCES cv(cv_id);


--
-- Name: fk898c7c04d758ec8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT fk898c7c04d758ec8a FOREIGN KEY (object_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk898c7c04df1577d; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT fk898c7c04df1577d FOREIGN KEY (subject_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk898c7c04fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermpath
    ADD CONSTRAINT fk898c7c04fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk898cbb421389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermprop
    ADD CONSTRAINT fk898cbb421389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk898cbb42fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvtermprop
    ADD CONSTRAINT fk898cbb42fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk8b81a4ac61c4940a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop
    ADD CONSTRAINT fk8b81a4ac61c4940a FOREIGN KEY (study_id) REFERENCES study(study_id);


--
-- Name: fk8b81a4acfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop
    ADD CONSTRAINT fk8b81a4acfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk8d2b5a2f73e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT fk8d2b5a2f73e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk8d2b5a2f7b6c2236; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT fk8d2b5a2f7b6c2236 FOREIGN KEY (biosourceprovider_id) REFERENCES contact(contact_id);


--
-- Name: fk8d2b5a2f95db6e92; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial
    ADD CONSTRAINT fk8d2b5a2f95db6e92 FOREIGN KEY (taxon_id) REFERENCES organism(organism_id);


--
-- Name: fk93bb208361c4940a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study_assay
    ADD CONSTRAINT fk93bb208361c4940a FOREIGN KEY (study_id) REFERENCES study(study_id);


--
-- Name: fk93bb2083da77160a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study_assay
    ADD CONSTRAINT fk93bb2083da77160a FOREIGN KEY (assay_id) REFERENCES assay(assay_id);


--
-- Name: fk93bc83112628bd1; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT fk93bc83112628bd1 FOREIGN KEY (object_id) REFERENCES phylonode(phylonode_id);


--
-- Name: fk93bc831138faf6c4; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT fk93bc831138faf6c4 FOREIGN KEY (subject_id) REFERENCES phylonode(phylonode_id);


--
-- Name: fk93bc83114a1696a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT fk93bc83114a1696a FOREIGN KEY (phylotree_id) REFERENCES phylotree(phylotree_id);


--
-- Name: fk93bc8311fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT fk93bc8311fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk97d8083f482b2b4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisprop
    ADD CONSTRAINT fk97d8083f482b2b4a FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id);


--
-- Name: fk97d8083ffd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY analysisprop
    ADD CONSTRAINT fk97d8083ffd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk98293bfbfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY synonym
    ADD CONSTRAINT fk98293bfbfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk98d55b97403caa3d; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pubprop
    ADD CONSTRAINT fk98d55b97403caa3d FOREIGN KEY (feature_pub_id) REFERENCES feature_pub(feature_pub_id);


--
-- Name: fk98d55b97fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pubprop
    ADD CONSTRAINT fk98d55b97fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk9902f5643128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_pub
    ADD CONSTRAINT fk9902f5643128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fk9902f564ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_pub
    ADD CONSTRAINT fk9902f564ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk99057f06e0e8c4d7; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_pub
    ADD CONSTRAINT fk99057f06e0e8c4d7 FOREIGN KEY (feature_cvterm_id) REFERENCES feature_cvterm(feature_cvterm_id);


--
-- Name: fk99057f06ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvterm_pub
    ADD CONSTRAINT fk99057f06ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fk990ab6893128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experimentprop
    ADD CONSTRAINT fk990ab6893128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fk990ab689fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experimentprop
    ADD CONSTRAINT fk990ab689fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk990d402be0e8c4d7; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvtermprop
    ADD CONSTRAINT fk990d402be0e8c4d7 FOREIGN KEY (feature_cvterm_id) REFERENCES feature_cvterm(feature_cvterm_id);


--
-- Name: fk990d402bfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_cvtermprop
    ADD CONSTRAINT fk990d402bfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk9bb168d23128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_genotype
    ADD CONSTRAINT fk9bb168d23128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fk9bb168d299afce2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_genotype
    ADD CONSTRAINT fk9bb168d299afce2a FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id);


--
-- Name: fk9c7b94c761c4940a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studydesign
    ADD CONSTRAINT fk9c7b94c761c4940a FOREIGN KEY (study_id) REFERENCES study(study_id);


--
-- Name: fk9ce31efc4bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT fk9ce31efc4bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fk9ce31efc73e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT fk9ce31efc73e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk9ce31efc94ecd4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT fk9ce31efc94ecd4a FOREIGN KEY (arraydesign_id) REFERENCES arraydesign(arraydesign_id);


--
-- Name: fk9ce31efcfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY element
    ADD CONSTRAINT fk9ce31efcfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk9d032d23128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_dbxref
    ADD CONSTRAINT fk9d032d23128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fk9d032d273e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_dbxref
    ADD CONSTRAINT fk9d032d273e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fk9e824bb75dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library
    ADD CONSTRAINT fk9e824bb75dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fk9e824bbfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library
    ADD CONSTRAINT fk9e824bbfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fk9fa5cc58f2c8574a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactor
    ADD CONSTRAINT fk9fa5cc58f2c8574a FOREIGN KEY (studydesign_id) REFERENCES studydesign(studydesign_id);


--
-- Name: fk9fa5cc58fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactor
    ADD CONSTRAINT fk9fa5cc58fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fka0f4ee021389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT fka0f4ee021389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fka0f4ee024bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT fka0f4ee024bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fka0f4ee0299afce2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT fka0f4ee0299afce2a FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id);


--
-- Name: fka0f4ee029e265b28; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_genotype
    ADD CONSTRAINT fka0f4ee029e265b28 FOREIGN KEY (chromosome_id) REFERENCES feature(feature_id);


--
-- Name: fka2a242b26429c3ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT fka2a242b26429c3ca FOREIGN KEY (environment_id) REFERENCES environment(environment_id);


--
-- Name: fka2a242b299afce2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT fka2a242b299afce2a FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id);


--
-- Name: fka2a242b2ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT fka2a242b2ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fka2a242b2fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT fka2a242b2fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fka47c17814bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT fka47c17814bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fka47c17819ceebdca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT fka47c17819ceebdca FOREIGN KEY (expression_id) REFERENCES expression(expression_id);


--
-- Name: fka47c1781ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_expression
    ADD CONSTRAINT fka47c1781ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkaa0aecfa756d60a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_contact
    ADD CONSTRAINT fkaa0aecfa756d60a FOREIGN KEY (project_id) REFERENCES project(project_id);


--
-- Name: fkaa0aecfad03f64aa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_contact
    ADD CONSTRAINT fkaa0aecfad03f64aa FOREIGN KEY (contact_id) REFERENCES contact(contact_id);


--
-- Name: fkab03b03a46b8efb; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT fkab03b03a46b8efb FOREIGN KEY (subject_id) REFERENCES pub(pub_id);


--
-- Name: fkab03b03acdd32408; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT fkab03b03acdd32408 FOREIGN KEY (object_id) REFERENCES pub(pub_id);


--
-- Name: fkab03b03afd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT fkab03b03afd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkaf40bb29fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent
    ADD CONSTRAINT fkaf40bb29fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkafa4df5f61bfbbea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm
    ADD CONSTRAINT fkafa4df5f61bfbbea FOREIGN KEY (cv_id) REFERENCES cv(cv_id);


--
-- Name: fkafa4df5f73e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cvterm
    ADD CONSTRAINT fkafa4df5f73e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkb041e29961c4bc0a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY dbxref
    ADD CONSTRAINT fkb041e29961c4bc0a FOREIGN KEY (db_id) REFERENCES db(db_id);


--
-- Name: fkb16d68d69ceebdca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_pub
    ADD CONSTRAINT fkb16d68d69ceebdca FOREIGN KEY (expression_id) REFERENCES expression(expression_id);


--
-- Name: fkb16d68d6ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_pub
    ADD CONSTRAINT fkb16d68d6ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkb17529fb9ceebdca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expressionprop
    ADD CONSTRAINT fkb17529fb9ceebdca FOREIGN KEY (expression_id) REFERENCES expression(expression_id);


--
-- Name: fkb17529fbfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expressionprop
    ADD CONSTRAINT fkb17529fbfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb17d33fe1389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT fkb17d33fe1389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb17d33fe56213e7b; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT fkb17d33fe56213e7b FOREIGN KEY (phenotype_comparison_id) REFERENCES phenotype_comparison(phenotype_comparison_id);


--
-- Name: fkb17d33feec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT fkb17d33feec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkb248002819ada310; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT fkb248002819ada310 FOREIGN KEY (assay_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb2480028328be67b; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT fkb2480028328be67b FOREIGN KEY (cvalue_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb248002871862c06; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT fkb248002871862c06 FOREIGN KEY (observable_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb24800287e8e098; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT fkb24800287e8e098 FOREIGN KEY (attr_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb2fe51ee5678baa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT fkb2fe51ee5678baa FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id);


--
-- Name: fkb2fe51ee6429c3ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT fkb2fe51ee6429c3ca FOREIGN KEY (environment_id) REFERENCES environment(environment_id);


--
-- Name: fkb2fe51ee99afce2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT fkb2fe51ee99afce2a FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id);


--
-- Name: fkb2fe51eeec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT fkb2fe51eeec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkb2fe51eefd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT fkb2fe51eefd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb3ba1e682a78e9aa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT fkb3ba1e682a78e9aa FOREIGN KEY (treatment_id) REFERENCES treatment(treatment_id);


--
-- Name: fkb3ba1e68da257e4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT fkb3ba1e68da257e4a FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id);


--
-- Name: fkb3ba1e68e37764ab; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_treatment
    ADD CONSTRAINT fkb3ba1e68e37764ab FOREIGN KEY (unittype_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkb43a1c774dd792ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT fkb43a1c774dd792ca FOREIGN KEY (synonym_id) REFERENCES synonym(synonym_id);


--
-- Name: fkb43a1c7776e5daca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT fkb43a1c7776e5daca FOREIGN KEY (library_id) REFERENCES library(library_id);


--
-- Name: fkb43a1c77ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT fkb43a1c77ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkb504cf47e94ed0a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_image
    ADD CONSTRAINT fkb504cf47e94ed0a FOREIGN KEY (eimage_id) REFERENCES eimage(eimage_id);


--
-- Name: fkb504cf49ceebdca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_image
    ADD CONSTRAINT fkb504cf49ceebdca FOREIGN KEY (expression_id) REFERENCES expression(expression_id);


--
-- Name: fkb94d461b73e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_dbxref
    ADD CONSTRAINT fkb94d461b73e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkb94d461bec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_dbxref
    ADD CONSTRAINT fkb94d461bec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkb97996268bdde8fd; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment
    ADD CONSTRAINT fkb97996268bdde8fd FOREIGN KEY (nd_geolocation_id) REFERENCES nd_geolocation(nd_geolocation_id);


--
-- Name: fkb9799626fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment
    ADD CONSTRAINT fkb9799626fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkba904f8eb5781dd4; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent_relationship
    ADD CONSTRAINT fkba904f8eb5781dd4 FOREIGN KEY (object_reagent_id) REFERENCES nd_reagent(nd_reagent_id);


--
-- Name: fkba904f8ee6fbe5c7; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent_relationship
    ADD CONSTRAINT fkba904f8ee6fbe5c7 FOREIGN KEY (subject_reagent_id) REFERENCES nd_reagent(nd_reagent_id);


--
-- Name: fkba904f8efd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent_relationship
    ADD CONSTRAINT fkba904f8efd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkbc5a6f49c4d843b; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvtermprop
    ADD CONSTRAINT fkbc5a6f49c4d843b FOREIGN KEY (expression_cvterm_id) REFERENCES expression_cvterm(expression_cvterm_id);


--
-- Name: fkbc5a6f49fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY expression_cvtermprop
    ADD CONSTRAINT fkbc5a6f49fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkbfa22ea673e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organism_dbxref
    ADD CONSTRAINT fkbfa22ea673e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkbfa22ea675dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organism_dbxref
    ADD CONSTRAINT fkbfa22ea675dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fkc50a8e9873e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT fkc50a8e9873e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkc50a8e98ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT fkc50a8e98ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkc50a8e98fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT fkc50a8e98fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkc5a27af673e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT fkc5a27af673e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkc5a27af675dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT fkc5a27af675dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fkc5a27af6fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature
    ADD CONSTRAINT fkc5a27af6fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkc5cc02003128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_project
    ADD CONSTRAINT fkc5cc02003128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fkc5cc0200756d60a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_project
    ADD CONSTRAINT fkc5cc0200756d60a FOREIGN KEY (project_id) REFERENCES project(project_id);


--
-- Name: fkc7c06ff76f1f9d2b; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT fkc7c06ff76f1f9d2b FOREIGN KEY (object_id) REFERENCES contact(contact_id);


--
-- Name: fkc7c06ff7a5b8081e; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT fkc7c06ff7a5b8081e FOREIGN KEY (subject_id) REFERENCES contact(contact_id);


--
-- Name: fkc7c06ff7fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY contact_relationship
    ADD CONSTRAINT fkc7c06ff7fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkc7fff8d7756d60a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_pub
    ADD CONSTRAINT fkc7fff8d7756d60a FOREIGN KEY (project_id) REFERENCES project(project_id);


--
-- Name: fkc7fff8d7ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_pub
    ADD CONSTRAINT fkc7fff8d7ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkc807b9fc756d60a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY projectprop
    ADD CONSTRAINT fkc807b9fc756d60a FOREIGN KEY (project_id) REFERENCES project(project_id);


--
-- Name: fkc807b9fcfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY projectprop
    ADD CONSTRAINT fkc807b9fcfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkd0d80ec64a1696a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT fkd0d80ec64a1696a FOREIGN KEY (phylotree_id) REFERENCES phylotree(phylotree_id);


--
-- Name: fkd0d80ec64bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT fkd0d80ec64bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fkd0d80ec69975e45f; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT fkd0d80ec69975e45f FOREIGN KEY (parent_phylonode_id) REFERENCES phylonode(phylonode_id);


--
-- Name: fkd0d80ec6fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT fkd0d80ec6fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkd0dad462482b2b4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree
    ADD CONSTRAINT fkd0dad462482b2b4a FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id);


--
-- Name: fkd0dad46273e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree
    ADD CONSTRAINT fkd0dad46273e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkd0dad462fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree
    ADD CONSTRAINT fkd0dad462fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkd1050e6d4dd792ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT fkd1050e6d4dd792ca FOREIGN KEY (synonym_id) REFERENCES synonym(synonym_id);


--
-- Name: fkd1050e6db391eef; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT fkd1050e6db391eef FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fkd1050e6dec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_synonym
    ADD CONSTRAINT fkd1050e6dec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkd303188075dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd303188075dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fkd30318808d2f4f29; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd30318808d2f4f29 FOREIGN KEY (phenotype1_id) REFERENCES phenotype(phenotype_id);


--
-- Name: fkd30318808d2fc388; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd30318808d2fc388 FOREIGN KEY (phenotype2_id) REFERENCES phenotype(phenotype_id);


--
-- Name: fkd3031880a401c97f; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd3031880a401c97f FOREIGN KEY (environment1_id) REFERENCES environment(environment_id);


--
-- Name: fkd3031880a4023dde; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd3031880a4023dde FOREIGN KEY (environment2_id) REFERENCES environment(environment_id);


--
-- Name: fkd3031880b310202b; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd3031880b310202b FOREIGN KEY (genotype1_id) REFERENCES genotype(genotype_id);


--
-- Name: fkd3031880b310948a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd3031880b310948a FOREIGN KEY (genotype2_id) REFERENCES genotype(genotype_id);


--
-- Name: fkd3031880ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT fkd3031880ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkd3f0367f7786dcb7; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_pub
    ADD CONSTRAINT fkd3f0367f7786dcb7 FOREIGN KEY (stock_relationship_id) REFERENCES stock_relationship(stock_relationship_id);


--
-- Name: fkd3f0367fec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_pub
    ADD CONSTRAINT fkd3f0367fec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkd57b5353756d60a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_project
    ADD CONSTRAINT fkd57b5353756d60a FOREIGN KEY (project_id) REFERENCES project(project_id);


--
-- Name: fkd57b5353da77160a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_project
    ADD CONSTRAINT fkd57b5353da77160a FOREIGN KEY (assay_id) REFERENCES assay(assay_id);


--
-- Name: fkd9eba1f4bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_phenotype
    ADD CONSTRAINT fkd9eba1f4bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fkd9eba1f5678baa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_phenotype
    ADD CONSTRAINT fkd9eba1f5678baa FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id);


--
-- Name: fkdbae317976e5daca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_pub
    ADD CONSTRAINT fkdbae317976e5daca FOREIGN KEY (library_id) REFERENCES library(library_id);


--
-- Name: fkdbae3179ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_pub
    ADD CONSTRAINT fkdbae3179ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkdbb5f29e76e5daca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop
    ADD CONSTRAINT fkdbb5f29e76e5daca FOREIGN KEY (library_id) REFERENCES library(library_id);


--
-- Name: fkdbb5f29efd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop
    ADD CONSTRAINT fkdbb5f29efd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkddb5bdaa38121098; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT fkddb5bdaa38121098 FOREIGN KEY (object_id) REFERENCES acquisition(acquisition_id);


--
-- Name: fkddb5bdaa6eaa7b8b; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT fkddb5bdaa6eaa7b8b FOREIGN KEY (subject_id) REFERENCES acquisition(acquisition_id);


--
-- Name: fkddb5bdaafd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY acquisition_relationship
    ADD CONSTRAINT fkddb5bdaafd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkdf1641173e0df60a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop_pub
    ADD CONSTRAINT fkdf1641173e0df60a FOREIGN KEY (featureprop_id) REFERENCES featureprop(featureprop_id);


--
-- Name: fkdf164117ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop_pub
    ADD CONSTRAINT fkdf164117ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkdf58687b39ec1ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT fkdf58687b39ec1ca FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id);


--
-- Name: fkdf58687b482b2b4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT fkdf58687b482b2b4a FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id);


--
-- Name: fkdf58687bcff9e1a6; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT fkdf58687bcff9e1a6 FOREIGN KEY (operator_id) REFERENCES contact(contact_id);


--
-- Name: fkdf58687bee88198a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT fkdf58687bee88198a FOREIGN KEY (acquisition_id) REFERENCES acquisition(acquisition_id);


--
-- Name: fke1b35b724bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_feature
    ADD CONSTRAINT fke1b35b724bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fke1b35b7276e5daca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_feature
    ADD CONSTRAINT fke1b35b7276e5daca FOREIGN KEY (library_id) REFERENCES library(library_id);


--
-- Name: fke224577c5fa865a6; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT fke224577c5fa865a6 FOREIGN KEY (object_id) REFERENCES quantification(quantification_id);


--
-- Name: fke224577c9640d099; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT fke224577c9640d099 FOREIGN KEY (subject_id) REFERENCES quantification(quantification_id);


--
-- Name: fke224577cfd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT fke224577cfd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fke98cf2344bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pub
    ADD CONSTRAINT fke98cf2344bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fke98cf234ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_pub
    ADD CONSTRAINT fke98cf234ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fke994b3594bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop
    ADD CONSTRAINT fke994b3594bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fke994b359fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureprop
    ADD CONSTRAINT fke994b359fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkea4482a973e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_dbxref
    ADD CONSTRAINT fkea4482a973e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkea4482a9da257e4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterial_dbxref
    ADD CONSTRAINT fkea4482a9da257e4a FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id);


--
-- Name: fkeaefe0a81389bc6a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT fkeaefe0a81389bc6a FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkeaefe0a85772f4ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT fkeaefe0a85772f4ea FOREIGN KEY (stock_id) REFERENCES stock(stock_id);


--
-- Name: fkeaefe0a8ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT fkeaefe0a8ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkeb00e22b75dbdf8a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_organism
    ADD CONSTRAINT fkeb00e22b75dbdf8a FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: fkeb00e22bc20c9eea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_organism
    ADD CONSTRAINT fkeb00e22bc20c9eea FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id);


--
-- Name: fkeb8ce3e25772f4ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_dbxref
    ADD CONSTRAINT fkeb8ce3e25772f4ea FOREIGN KEY (stock_id) REFERENCES stock(stock_id);


--
-- Name: fkeb8ce3e273e13a2a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_dbxref
    ADD CONSTRAINT fkeb8ce3e273e13a2a FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id);


--
-- Name: fkec14ee9fc8d85877; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship_pub
    ADD CONSTRAINT fkec14ee9fc8d85877 FOREIGN KEY (feature_relationship_id) REFERENCES feature_relationship(feature_relationship_id);


--
-- Name: fkec14ee9fec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationship_pub
    ADD CONSTRAINT fkec14ee9fec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkec1cafc4c8d85877; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop
    ADD CONSTRAINT fkec1cafc4c8d85877 FOREIGN KEY (feature_relationship_id) REFERENCES feature_relationship(feature_relationship_id);


--
-- Name: fkec1cafc4fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY feature_relationshipprop
    ADD CONSTRAINT fkec1cafc4fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkee5f4234d03f64aa; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection
    ADD CONSTRAINT fkee5f4234d03f64aa FOREIGN KEY (contact_id) REFERENCES contact(contact_id);


--
-- Name: fkee5f4234fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection
    ADD CONSTRAINT fkee5f4234fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkf1fa8fe0ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pubprop
    ADD CONSTRAINT fkf1fa8fe0ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: fkf1fa8fe0fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pubprop
    ADD CONSTRAINT fkf1fa8fe0fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkf445b4913128eb37; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_protocol
    ADD CONSTRAINT fkf445b4913128eb37 FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id);


--
-- Name: fkf445b49166492497; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_protocol
    ADD CONSTRAINT fkf445b49166492497 FOREIGN KEY (nd_protocol_id) REFERENCES nd_protocol(nd_protocol_id);


--
-- Name: fkf6b94e12da257e4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterialprop
    ADD CONSTRAINT fkf6b94e12da257e4a FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id);


--
-- Name: fkf6b94e12fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY biomaterialprop
    ADD CONSTRAINT fkf6b94e12fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkf704bc8a4bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc
    ADD CONSTRAINT fkf704bc8a4bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fkf704bc8a72419dce; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featureloc
    ADD CONSTRAINT fkf704bc8a72419dce FOREIGN KEY (srcfeature_id) REFERENCES feature(feature_id);


--
-- Name: fkf704bea6e37764ab; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featuremap
    ADD CONSTRAINT fkf704bea6e37764ab FOREIGN KEY (unittype_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkf704cb9e4bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurepos
    ADD CONSTRAINT fkf704cb9e4bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fkf704cb9e7583ebad; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurepos
    ADD CONSTRAINT fkf704cb9e7583ebad FOREIGN KEY (map_feature_id) REFERENCES feature(feature_id);


--
-- Name: fkf704cb9e88559b0a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY featurepos
    ADD CONSTRAINT fkf704cb9e88559b0a FOREIGN KEY (featuremap_id) REFERENCES featuremap(featuremap_id);


--
-- Name: fkfb83b569482795ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT fkfb83b569482795ca FOREIGN KEY (channel_id) REFERENCES channel(channel_id);


--
-- Name: fkfb83b569da257e4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT fkfb83b569da257e4a FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id);


--
-- Name: fkfb83b569da77160a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY assay_biomaterial
    ADD CONSTRAINT fkfb83b569da77160a FOREIGN KEY (assay_id) REFERENCES assay(assay_id);


--
-- Name: fkfc39787839ec1ca; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY treatment
    ADD CONSTRAINT fkfc39787839ec1ca FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id);


--
-- Name: fkfc397878da257e4a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY treatment
    ADD CONSTRAINT fkfc397878da257e4a FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id);


--
-- Name: fkfc397878fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY treatment
    ADD CONSTRAINT fkfc397878fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkfc84ba194e69272a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult
    ADD CONSTRAINT fkfc84ba194e69272a FOREIGN KEY (element_id) REFERENCES element(element_id);


--
-- Name: fkfc84ba19a999c6ea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY elementresult
    ADD CONSTRAINT fkfc84ba19a999c6ea FOREIGN KEY (quantification_id) REFERENCES quantification(quantification_id);


--
-- Name: fkfd70f174169698d4; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocol_reagent
    ADD CONSTRAINT fkfd70f174169698d4 FOREIGN KEY (reagent_id) REFERENCES nd_reagent(nd_reagent_id);


--
-- Name: fkfd70f17466492497; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocol_reagent
    ADD CONSTRAINT fkfd70f17466492497 FOREIGN KEY (nd_protocol_id) REFERENCES nd_protocol(nd_protocol_id);


--
-- Name: fkfd70f174fd493aaf; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocol_reagent
    ADD CONSTRAINT fkfd70f174fd493aaf FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: fkfe7e4d684bd8ecea; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT fkfe7e4d684bd8ecea FOREIGN KEY (feature_id) REFERENCES feature(feature_id);


--
-- Name: fkfe7e4d68b391eef; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT fkfe7e4d68b391eef FOREIGN KEY (cell_line_id) REFERENCES cell_line(cell_line_id);


--
-- Name: fkfe7e4d68ec5a138a; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY cell_line_feature
    ADD CONSTRAINT fkfe7e4d68ec5a138a FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: library_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT library_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id);


--
-- Name: library_cvterm_library_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT library_cvterm_library_id_fkey FOREIGN KEY (library_id) REFERENCES library(library_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_cvterm_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_cvterm
    ADD CONSTRAINT library_cvterm_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id);


--
-- Name: library_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_dbxref
    ADD CONSTRAINT library_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_dbxref_library_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_dbxref
    ADD CONSTRAINT library_dbxref_library_id_fkey FOREIGN KEY (library_id) REFERENCES library(library_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_feature_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_feature
    ADD CONSTRAINT library_feature_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_feature_library_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_feature
    ADD CONSTRAINT library_feature_library_id_fkey FOREIGN KEY (library_id) REFERENCES library(library_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library
    ADD CONSTRAINT library_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id);


--
-- Name: library_pub_library_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_pub
    ADD CONSTRAINT library_pub_library_id_fkey FOREIGN KEY (library_id) REFERENCES library(library_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_pub
    ADD CONSTRAINT library_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_synonym_library_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT library_synonym_library_id_fkey FOREIGN KEY (library_id) REFERENCES library(library_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_synonym_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT library_synonym_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_synonym_synonym_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library_synonym
    ADD CONSTRAINT library_synonym_synonym_id_fkey FOREIGN KEY (synonym_id) REFERENCES synonym(synonym_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: library_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY library
    ADD CONSTRAINT library_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: libraryprop_library_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop
    ADD CONSTRAINT libraryprop_library_id_fkey FOREIGN KEY (library_id) REFERENCES library(library_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: libraryprop_pub_libraryprop_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop_pub
    ADD CONSTRAINT libraryprop_pub_libraryprop_id_fkey FOREIGN KEY (libraryprop_id) REFERENCES libraryprop(libraryprop_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: libraryprop_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop_pub
    ADD CONSTRAINT libraryprop_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: libraryprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY libraryprop
    ADD CONSTRAINT libraryprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: magedocumentation_mageml_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY magedocumentation
    ADD CONSTRAINT magedocumentation_mageml_id_fkey FOREIGN KEY (mageml_id) REFERENCES mageml(mageml_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: magedocumentation_tableinfo_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY magedocumentation
    ADD CONSTRAINT magedocumentation_tableinfo_id_fkey FOREIGN KEY (tableinfo_id) REFERENCES tableinfo(tableinfo_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_contact_contact_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_contact
    ADD CONSTRAINT nd_experiment_contact_contact_id_fkey FOREIGN KEY (contact_id) REFERENCES contact(contact_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_contact_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_contact
    ADD CONSTRAINT nd_experiment_contact_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_dbxref
    ADD CONSTRAINT nd_experiment_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_dbxref_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_dbxref
    ADD CONSTRAINT nd_experiment_dbxref_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_genotype_genotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_genotype
    ADD CONSTRAINT nd_experiment_genotype_genotype_id_fkey FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_genotype_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_genotype
    ADD CONSTRAINT nd_experiment_genotype_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_nd_geolocation_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment
    ADD CONSTRAINT nd_experiment_nd_geolocation_id_fkey FOREIGN KEY (nd_geolocation_id) REFERENCES nd_geolocation(nd_geolocation_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_phenotype_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotype
    ADD CONSTRAINT nd_experiment_phenotype_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_phenotype_phenotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotype
    ADD CONSTRAINT nd_experiment_phenotype_phenotype_id_fkey FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_phenotypeprop_nd_experiment_phenotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotypeprop
    ADD CONSTRAINT nd_experiment_phenotypeprop_nd_experiment_phenotype_id_fkey FOREIGN KEY (nd_experiment_phenotype_id) REFERENCES nd_experiment_phenotype(nd_experiment_phenotype_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_phenotypeprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_phenotypeprop
    ADD CONSTRAINT nd_experiment_phenotypeprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_project_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_project
    ADD CONSTRAINT nd_experiment_project_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_project_project_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_project
    ADD CONSTRAINT nd_experiment_project_project_id_fkey FOREIGN KEY (project_id) REFERENCES project(project_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_protocol_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_protocol
    ADD CONSTRAINT nd_experiment_protocol_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_protocol_nd_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_protocol
    ADD CONSTRAINT nd_experiment_protocol_nd_protocol_id_fkey FOREIGN KEY (nd_protocol_id) REFERENCES nd_protocol(nd_protocol_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_pub_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_pub
    ADD CONSTRAINT nd_experiment_pub_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_pub
    ADD CONSTRAINT nd_experiment_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_stock_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock_dbxref
    ADD CONSTRAINT nd_experiment_stock_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_stock_dbxref_nd_experiment_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock_dbxref
    ADD CONSTRAINT nd_experiment_stock_dbxref_nd_experiment_stock_id_fkey FOREIGN KEY (nd_experiment_stock_id) REFERENCES nd_experiment_stock(nd_experiment_stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_stock_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock
    ADD CONSTRAINT nd_experiment_stock_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_stock_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock
    ADD CONSTRAINT nd_experiment_stock_stock_id_fkey FOREIGN KEY (stock_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_stock_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stock
    ADD CONSTRAINT nd_experiment_stock_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_stockprop_nd_experiment_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stockprop
    ADD CONSTRAINT nd_experiment_stockprop_nd_experiment_stock_id_fkey FOREIGN KEY (nd_experiment_stock_id) REFERENCES nd_experiment_stock(nd_experiment_stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_stockprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment_stockprop
    ADD CONSTRAINT nd_experiment_stockprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experiment_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experiment
    ADD CONSTRAINT nd_experiment_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experimentprop_nd_experiment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experimentprop
    ADD CONSTRAINT nd_experimentprop_nd_experiment_id_fkey FOREIGN KEY (nd_experiment_id) REFERENCES nd_experiment(nd_experiment_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_experimentprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_experimentprop
    ADD CONSTRAINT nd_experimentprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_geolocationprop_nd_geolocation_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_geolocationprop
    ADD CONSTRAINT nd_geolocationprop_nd_geolocation_id_fkey FOREIGN KEY (nd_geolocation_id) REFERENCES nd_geolocation(nd_geolocation_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_geolocationprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_geolocationprop
    ADD CONSTRAINT nd_geolocationprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_protocol_reagent_nd_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocol_reagent
    ADD CONSTRAINT nd_protocol_reagent_nd_protocol_id_fkey FOREIGN KEY (nd_protocol_id) REFERENCES nd_protocol(nd_protocol_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_protocol_reagent_reagent_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocol_reagent
    ADD CONSTRAINT nd_protocol_reagent_reagent_id_fkey FOREIGN KEY (reagent_id) REFERENCES nd_reagent(nd_reagent_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_protocol_reagent_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocol_reagent
    ADD CONSTRAINT nd_protocol_reagent_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_protocolprop_nd_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocolprop
    ADD CONSTRAINT nd_protocolprop_nd_protocol_id_fkey FOREIGN KEY (nd_protocol_id) REFERENCES nd_protocol(nd_protocol_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_protocolprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_protocolprop
    ADD CONSTRAINT nd_protocolprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_reagent_relationship_object_reagent_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent_relationship
    ADD CONSTRAINT nd_reagent_relationship_object_reagent_id_fkey FOREIGN KEY (object_reagent_id) REFERENCES nd_reagent(nd_reagent_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_reagent_relationship_subject_reagent_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent_relationship
    ADD CONSTRAINT nd_reagent_relationship_subject_reagent_id_fkey FOREIGN KEY (subject_reagent_id) REFERENCES nd_reagent(nd_reagent_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_reagent_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent_relationship
    ADD CONSTRAINT nd_reagent_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_reagent_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagent
    ADD CONSTRAINT nd_reagent_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_reagentprop_nd_reagent_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagentprop
    ADD CONSTRAINT nd_reagentprop_nd_reagent_id_fkey FOREIGN KEY (nd_reagent_id) REFERENCES nd_reagent(nd_reagent_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: nd_reagentprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY nd_reagentprop
    ADD CONSTRAINT nd_reagentprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: organism_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organism_dbxref
    ADD CONSTRAINT organism_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: organism_dbxref_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organism_dbxref
    ADD CONSTRAINT organism_dbxref_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: organismprop_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organismprop
    ADD CONSTRAINT organismprop_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: organismprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY organismprop
    ADD CONSTRAINT organismprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: phendesc_environment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT phendesc_environment_id_fkey FOREIGN KEY (environment_id) REFERENCES environment(environment_id) ON DELETE CASCADE;


--
-- Name: phendesc_genotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT phendesc_genotype_id_fkey FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id) ON DELETE CASCADE;


--
-- Name: phendesc_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT phendesc_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE;


--
-- Name: phendesc_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phendesc
    ADD CONSTRAINT phendesc_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phenotype_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT phenotype_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL;


--
-- Name: phenotype_attr_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT phenotype_attr_id_fkey FOREIGN KEY (attr_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL;


--
-- Name: phenotype_comparison_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT phenotype_comparison_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_cvterm_phenotype_comparison_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT phenotype_comparison_cvterm_phenotype_comparison_id_fkey FOREIGN KEY (phenotype_comparison_id) REFERENCES phenotype_comparison(phenotype_comparison_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_cvterm_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison_cvterm
    ADD CONSTRAINT phenotype_comparison_cvterm_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_environment1_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_environment1_id_fkey FOREIGN KEY (environment1_id) REFERENCES environment(environment_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_environment2_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_environment2_id_fkey FOREIGN KEY (environment2_id) REFERENCES environment(environment_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_genotype1_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_genotype1_id_fkey FOREIGN KEY (genotype1_id) REFERENCES genotype(genotype_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_genotype2_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_genotype2_id_fkey FOREIGN KEY (genotype2_id) REFERENCES genotype(genotype_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_phenotype1_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_phenotype1_id_fkey FOREIGN KEY (phenotype1_id) REFERENCES phenotype(phenotype_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_phenotype2_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_phenotype2_id_fkey FOREIGN KEY (phenotype2_id) REFERENCES phenotype(phenotype_id) ON DELETE CASCADE;


--
-- Name: phenotype_comparison_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_comparison
    ADD CONSTRAINT phenotype_comparison_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE;


--
-- Name: phenotype_cvalue_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT phenotype_cvalue_id_fkey FOREIGN KEY (cvalue_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL;


--
-- Name: phenotype_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_cvterm
    ADD CONSTRAINT phenotype_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phenotype_cvterm_phenotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype_cvterm
    ADD CONSTRAINT phenotype_cvterm_phenotype_id_fkey FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id) ON DELETE CASCADE;


--
-- Name: phenotype_observable_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotype
    ADD CONSTRAINT phenotype_observable_id_fkey FOREIGN KEY (observable_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phenotypeprop_phenotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotypeprop
    ADD CONSTRAINT phenotypeprop_phenotype_id_fkey FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id) ON DELETE CASCADE;


--
-- Name: phenotypeprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenotypeprop
    ADD CONSTRAINT phenotypeprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phenstatement_environment_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT phenstatement_environment_id_fkey FOREIGN KEY (environment_id) REFERENCES environment(environment_id) ON DELETE CASCADE;


--
-- Name: phenstatement_genotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT phenstatement_genotype_id_fkey FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id) ON DELETE CASCADE;


--
-- Name: phenstatement_phenotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT phenstatement_phenotype_id_fkey FOREIGN KEY (phenotype_id) REFERENCES phenotype(phenotype_id) ON DELETE CASCADE;


--
-- Name: phenstatement_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT phenstatement_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE;


--
-- Name: phenstatement_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phenstatement
    ADD CONSTRAINT phenstatement_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phylonode_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_dbxref
    ADD CONSTRAINT phylonode_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE;


--
-- Name: phylonode_dbxref_phylonode_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_dbxref
    ADD CONSTRAINT phylonode_dbxref_phylonode_id_fkey FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id) ON DELETE CASCADE;


--
-- Name: phylonode_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT phylonode_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE;


--
-- Name: phylonode_organism_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_organism
    ADD CONSTRAINT phylonode_organism_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id) ON DELETE CASCADE;


--
-- Name: phylonode_organism_phylonode_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_organism
    ADD CONSTRAINT phylonode_organism_phylonode_id_fkey FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id) ON DELETE CASCADE;


--
-- Name: phylonode_parent_phylonode_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT phylonode_parent_phylonode_id_fkey FOREIGN KEY (parent_phylonode_id) REFERENCES phylonode(phylonode_id) ON DELETE CASCADE;


--
-- Name: phylonode_phylotree_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT phylonode_phylotree_id_fkey FOREIGN KEY (phylotree_id) REFERENCES phylotree(phylotree_id) ON DELETE CASCADE;


--
-- Name: phylonode_pub_phylonode_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_pub
    ADD CONSTRAINT phylonode_pub_phylonode_id_fkey FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id) ON DELETE CASCADE;


--
-- Name: phylonode_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_pub
    ADD CONSTRAINT phylonode_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE;


--
-- Name: phylonode_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT phylonode_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES phylonode(phylonode_id) ON DELETE CASCADE;


--
-- Name: phylonode_relationship_phylotree_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT phylonode_relationship_phylotree_id_fkey FOREIGN KEY (phylotree_id) REFERENCES phylotree(phylotree_id) ON DELETE CASCADE;


--
-- Name: phylonode_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT phylonode_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES phylonode(phylonode_id) ON DELETE CASCADE;


--
-- Name: phylonode_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode_relationship
    ADD CONSTRAINT phylonode_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phylonode_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonode
    ADD CONSTRAINT phylonode_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phylonodeprop_phylonode_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonodeprop
    ADD CONSTRAINT phylonodeprop_phylonode_id_fkey FOREIGN KEY (phylonode_id) REFERENCES phylonode(phylonode_id) ON DELETE CASCADE;


--
-- Name: phylonodeprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylonodeprop
    ADD CONSTRAINT phylonodeprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: phylotree_analysis_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree
    ADD CONSTRAINT phylotree_analysis_id_fkey FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id) ON DELETE CASCADE;


--
-- Name: phylotree_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree
    ADD CONSTRAINT phylotree_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE;


--
-- Name: phylotree_pub_phylotree_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree_pub
    ADD CONSTRAINT phylotree_pub_phylotree_id_fkey FOREIGN KEY (phylotree_id) REFERENCES phylotree(phylotree_id) ON DELETE CASCADE;


--
-- Name: phylotree_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree_pub
    ADD CONSTRAINT phylotree_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE;


--
-- Name: phylotree_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY phylotree
    ADD CONSTRAINT phylotree_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: project_contact_contact_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_contact
    ADD CONSTRAINT project_contact_contact_id_fkey FOREIGN KEY (contact_id) REFERENCES contact(contact_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: project_contact_project_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_contact
    ADD CONSTRAINT project_contact_project_id_fkey FOREIGN KEY (project_id) REFERENCES project(project_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: project_pub_project_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_pub
    ADD CONSTRAINT project_pub_project_id_fkey FOREIGN KEY (project_id) REFERENCES project(project_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: project_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_pub
    ADD CONSTRAINT project_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: project_relationship_object_project_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT project_relationship_object_project_id_fkey FOREIGN KEY (object_project_id) REFERENCES project(project_id) ON DELETE CASCADE;


--
-- Name: project_relationship_subject_project_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT project_relationship_subject_project_id_fkey FOREIGN KEY (subject_project_id) REFERENCES project(project_id) ON DELETE CASCADE;


--
-- Name: project_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY project_relationship
    ADD CONSTRAINT project_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE RESTRICT;


--
-- Name: projectprop_project_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY projectprop
    ADD CONSTRAINT projectprop_project_id_fkey FOREIGN KEY (project_id) REFERENCES project(project_id) ON DELETE CASCADE;


--
-- Name: projectprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY projectprop
    ADD CONSTRAINT projectprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: protocol_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT protocol_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: protocol_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT protocol_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: protocol_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocol
    ADD CONSTRAINT protocol_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: protocolparam_datatype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocolparam
    ADD CONSTRAINT protocolparam_datatype_id_fkey FOREIGN KEY (datatype_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: protocolparam_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocolparam
    ADD CONSTRAINT protocolparam_protocol_id_fkey FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: protocolparam_unittype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY protocolparam
    ADD CONSTRAINT protocolparam_unittype_id_fkey FOREIGN KEY (unittype_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pub_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_dbxref
    ADD CONSTRAINT pub_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pub_dbxref_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_dbxref
    ADD CONSTRAINT pub_dbxref_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pub_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT pub_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pub_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT pub_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pub_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub_relationship
    ADD CONSTRAINT pub_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pub_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pub
    ADD CONSTRAINT pub_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pubauthor_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pubauthor
    ADD CONSTRAINT pubauthor_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pubprop_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pubprop
    ADD CONSTRAINT pubprop_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: pubprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY pubprop
    ADD CONSTRAINT pubprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantification_acquisition_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT quantification_acquisition_id_fkey FOREIGN KEY (acquisition_id) REFERENCES acquisition(acquisition_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantification_analysis_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT quantification_analysis_id_fkey FOREIGN KEY (analysis_id) REFERENCES analysis(analysis_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantification_operator_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT quantification_operator_id_fkey FOREIGN KEY (operator_id) REFERENCES contact(contact_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantification_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification
    ADD CONSTRAINT quantification_protocol_id_fkey FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantification_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT quantification_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES quantification(quantification_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantification_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT quantification_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES quantification(quantification_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantification_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantification_relationship
    ADD CONSTRAINT quantification_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantificationprop_quantification_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantificationprop
    ADD CONSTRAINT quantificationprop_quantification_id_fkey FOREIGN KEY (quantification_id) REFERENCES quantification(quantification_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: quantificationprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY quantificationprop
    ADD CONSTRAINT quantificationprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT stock_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_cvterm_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT stock_cvterm_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_cvterm_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_cvterm
    ADD CONSTRAINT stock_cvterm_stock_id_fkey FOREIGN KEY (stock_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_dbxref_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_dbxref
    ADD CONSTRAINT stock_dbxref_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT stock_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_dbxref_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_dbxref
    ADD CONSTRAINT stock_dbxref_stock_id_fkey FOREIGN KEY (stock_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_genotype_genotype_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_genotype
    ADD CONSTRAINT stock_genotype_genotype_id_fkey FOREIGN KEY (genotype_id) REFERENCES genotype(genotype_id) ON DELETE CASCADE;


--
-- Name: stock_genotype_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_genotype
    ADD CONSTRAINT stock_genotype_stock_id_fkey FOREIGN KEY (stock_id) REFERENCES stock(stock_id) ON DELETE CASCADE;


--
-- Name: stock_organism_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT stock_organism_id_fkey FOREIGN KEY (organism_id) REFERENCES organism(organism_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_pub
    ADD CONSTRAINT stock_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_pub_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_pub
    ADD CONSTRAINT stock_pub_stock_id_fkey FOREIGN KEY (stock_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_relationship_cvterm_cvterm_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_cvterm
    ADD CONSTRAINT stock_relationship_cvterm_cvterm_id_fkey FOREIGN KEY (cvterm_id) REFERENCES cvterm(cvterm_id) ON DELETE RESTRICT;


--
-- Name: stock_relationship_cvterm_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_cvterm
    ADD CONSTRAINT stock_relationship_cvterm_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE RESTRICT;


--
-- Name: stock_relationship_object_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT stock_relationship_object_id_fkey FOREIGN KEY (object_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_relationship_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_pub
    ADD CONSTRAINT stock_relationship_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_relationship_pub_stock_relationship_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship_pub
    ADD CONSTRAINT stock_relationship_pub_stock_relationship_id_fkey FOREIGN KEY (stock_relationship_id) REFERENCES stock_relationship(stock_relationship_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_relationship_subject_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT stock_relationship_subject_id_fkey FOREIGN KEY (subject_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_relationship_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock_relationship
    ADD CONSTRAINT stock_relationship_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stock_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stock
    ADD CONSTRAINT stock_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockcollection_contact_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection
    ADD CONSTRAINT stockcollection_contact_id_fkey FOREIGN KEY (contact_id) REFERENCES contact(contact_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockcollection_stock_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection_stock
    ADD CONSTRAINT stockcollection_stock_stock_id_fkey FOREIGN KEY (stock_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockcollection_stock_stockcollection_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection_stock
    ADD CONSTRAINT stockcollection_stock_stockcollection_id_fkey FOREIGN KEY (stockcollection_id) REFERENCES stockcollection(stockcollection_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockcollection_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollection
    ADD CONSTRAINT stockcollection_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: stockcollectionprop_stockcollection_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollectionprop
    ADD CONSTRAINT stockcollectionprop_stockcollection_id_fkey FOREIGN KEY (stockcollection_id) REFERENCES stockcollection(stockcollection_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockcollectionprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockcollectionprop
    ADD CONSTRAINT stockcollectionprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id);


--
-- Name: stockprop_pub_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop_pub
    ADD CONSTRAINT stockprop_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockprop_pub_stockprop_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop_pub
    ADD CONSTRAINT stockprop_pub_stockprop_id_fkey FOREIGN KEY (stockprop_id) REFERENCES stockprop(stockprop_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockprop_stock_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop
    ADD CONSTRAINT stockprop_stock_id_fkey FOREIGN KEY (stock_id) REFERENCES stock(stock_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: stockprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY stockprop
    ADD CONSTRAINT stockprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: study_assay_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study_assay
    ADD CONSTRAINT study_assay_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES assay(assay_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: study_assay_study_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study_assay
    ADD CONSTRAINT study_assay_study_id_fkey FOREIGN KEY (study_id) REFERENCES study(study_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: study_contact_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study
    ADD CONSTRAINT study_contact_id_fkey FOREIGN KEY (contact_id) REFERENCES contact(contact_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: study_dbxref_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study
    ADD CONSTRAINT study_dbxref_id_fkey FOREIGN KEY (dbxref_id) REFERENCES dbxref(dbxref_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: study_pub_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY study
    ADD CONSTRAINT study_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studydesign_study_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studydesign
    ADD CONSTRAINT studydesign_study_id_fkey FOREIGN KEY (study_id) REFERENCES study(study_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studydesignprop_studydesign_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studydesignprop
    ADD CONSTRAINT studydesignprop_studydesign_id_fkey FOREIGN KEY (studydesign_id) REFERENCES studydesign(studydesign_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studydesignprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studydesignprop
    ADD CONSTRAINT studydesignprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studyfactor_studydesign_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactor
    ADD CONSTRAINT studyfactor_studydesign_id_fkey FOREIGN KEY (studydesign_id) REFERENCES studydesign(studydesign_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studyfactor_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactor
    ADD CONSTRAINT studyfactor_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studyfactorvalue_assay_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactorvalue
    ADD CONSTRAINT studyfactorvalue_assay_id_fkey FOREIGN KEY (assay_id) REFERENCES assay(assay_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studyfactorvalue_studyfactor_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyfactorvalue
    ADD CONSTRAINT studyfactorvalue_studyfactor_id_fkey FOREIGN KEY (studyfactor_id) REFERENCES studyfactor(studyfactor_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: studyprop_feature_feature_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT studyprop_feature_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE;


--
-- Name: studyprop_feature_studyprop_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT studyprop_feature_studyprop_id_fkey FOREIGN KEY (studyprop_id) REFERENCES studyprop(studyprop_id) ON DELETE CASCADE;


--
-- Name: studyprop_feature_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop_feature
    ADD CONSTRAINT studyprop_feature_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: studyprop_study_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop
    ADD CONSTRAINT studyprop_study_id_fkey FOREIGN KEY (study_id) REFERENCES study(study_id) ON DELETE CASCADE;


--
-- Name: studyprop_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY studyprop
    ADD CONSTRAINT studyprop_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE;


--
-- Name: synonym_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY synonym
    ADD CONSTRAINT synonym_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: treatment_biomaterial_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY treatment
    ADD CONSTRAINT treatment_biomaterial_id_fkey FOREIGN KEY (biomaterial_id) REFERENCES biomaterial(biomaterial_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: treatment_protocol_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY treatment
    ADD CONSTRAINT treatment_protocol_id_fkey FOREIGN KEY (protocol_id) REFERENCES protocol(protocol_id) ON DELETE SET NULL DEFERRABLE INITIALLY DEFERRED;


--
-- Name: treatment_type_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: gmod
--

ALTER TABLE ONLY treatment
    ADD CONSTRAINT treatment_type_id_fkey FOREIGN KEY (type_id) REFERENCES cvterm(cvterm_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;


--
-- Name: public; Type: ACL; Schema: -; Owner: postgres
--

REVOKE ALL ON SCHEMA public FROM PUBLIC;
REVOKE ALL ON SCHEMA public FROM postgres;
GRANT ALL ON SCHEMA public TO postgres;
GRANT ALL ON SCHEMA public TO PUBLIC;


--
-- Name: cell_line; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line FROM PUBLIC;
REVOKE ALL ON TABLE cell_line FROM gmod;
GRANT ALL ON TABLE cell_line TO gmod;
GRANT ALL ON TABLE cell_line TO PUBLIC;


--
-- Name: cell_line_cvterm; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_cvterm FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_cvterm FROM gmod;
GRANT ALL ON TABLE cell_line_cvterm TO gmod;
GRANT ALL ON TABLE cell_line_cvterm TO PUBLIC;


--
-- Name: cell_line_cvtermprop; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_cvtermprop FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_cvtermprop FROM gmod;
GRANT ALL ON TABLE cell_line_cvtermprop TO gmod;
GRANT ALL ON TABLE cell_line_cvtermprop TO PUBLIC;


--
-- Name: cell_line_dbxref; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_dbxref FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_dbxref FROM gmod;
GRANT ALL ON TABLE cell_line_dbxref TO gmod;
GRANT ALL ON TABLE cell_line_dbxref TO PUBLIC;


--
-- Name: cell_line_feature; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_feature FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_feature FROM gmod;
GRANT ALL ON TABLE cell_line_feature TO gmod;
GRANT ALL ON TABLE cell_line_feature TO PUBLIC;


--
-- Name: cell_line_library; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_library FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_library FROM gmod;
GRANT ALL ON TABLE cell_line_library TO gmod;
GRANT ALL ON TABLE cell_line_library TO PUBLIC;


--
-- Name: cell_line_pub; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_pub FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_pub FROM gmod;
GRANT ALL ON TABLE cell_line_pub TO gmod;
GRANT ALL ON TABLE cell_line_pub TO PUBLIC;


--
-- Name: cell_line_relationship; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_relationship FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_relationship FROM gmod;
GRANT ALL ON TABLE cell_line_relationship TO gmod;
GRANT ALL ON TABLE cell_line_relationship TO PUBLIC;


--
-- Name: cell_line_synonym; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_line_synonym FROM PUBLIC;
REVOKE ALL ON TABLE cell_line_synonym FROM gmod;
GRANT ALL ON TABLE cell_line_synonym TO gmod;
GRANT ALL ON TABLE cell_line_synonym TO PUBLIC;


--
-- Name: cell_lineprop; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_lineprop FROM PUBLIC;
REVOKE ALL ON TABLE cell_lineprop FROM gmod;
GRANT ALL ON TABLE cell_lineprop TO gmod;
GRANT ALL ON TABLE cell_lineprop TO PUBLIC;


--
-- Name: cell_lineprop_pub; Type: ACL; Schema: public; Owner: gmod
--

REVOKE ALL ON TABLE cell_lineprop_pub FROM PUBLIC;
REVOKE ALL ON TABLE cell_lineprop_pub FROM gmod;
GRANT ALL ON TABLE cell_lineprop_pub TO gmod;
GRANT ALL ON TABLE cell_lineprop_pub TO PUBLIC;


--
-- PostgreSQL database dump complete
--

